
<html lang="en"     class="pb-page"  data-request-id="9ab796e2-8b91-4a6f-a97a-fc28207290f3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01255;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-23"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase" /></meta><meta name="dc.Creator" content="Dongxing  Chen" /></meta><meta name="dc.Creator" content="Linjie  Li" /></meta><meta name="dc.Creator" content="Krystal  Diaz" /></meta><meta name="dc.Creator" content="Iredia D.  Iyamu" /></meta><meta name="dc.Creator" content="Ravi  Yadav" /></meta><meta name="dc.Creator" content="Nicholas  Noinaj" /></meta><meta name="dc.Creator" content="Rong  Huang" /></meta><meta name="dc.Description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the methyl transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-containing compounds. NNMT is an important regulator ..." /></meta><meta name="Description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the methyl transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-containing compounds. NNMT is an important regulator ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 14, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01255" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01255" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01255" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01255" /></link>
        
    
    

<title>Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01255" /></meta><meta property="og:title" content="Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0013.jpeg" /></meta><meta property="og:description" content="Nicotinamide N-methyltransferase (NNMT) catalyzes the methyl transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-containing compounds. NNMT is an important regulator for nicotinamide metabolism and methylation potential. Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target. Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases. Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker. LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date. The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT. Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of methyl transfer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01255"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01255">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01255&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01255&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01255&amp;href=/doi/10.1021/acs.jmedchem.9b01255" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10783-10797</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01227" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01290" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide <i>N</i>-Methyltransferase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dongxing Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongxing Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongxing++Chen">Dongxing Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Linjie Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linjie Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linjie++Li">Linjie Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Krystal Diaz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krystal Diaz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krystal++Diaz">Krystal Diaz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iredia D. Iyamu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iredia D. Iyamu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iredia+D.++Iyamu">Iredia D. Iyamu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ravi Yadav</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ravi Yadav</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Markey Center for Structural Biology, Department of Biological Sciences and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ravi++Yadav">Ravi Yadav</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Noinaj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Noinaj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Markey Center for Structural Biology, Department of Biological Sciences and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Noinaj">Nicholas Noinaj</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rong Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#92fae7f3fcf5bfe0d2e2e7e0f6e7f7bcf7f6e7"><span class="__cf_email__" data-cfemail="f39b86929d94de81b3838681978696dd969786">[email protected]</span></a>. Phone: (765) 494 3426.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Huang">Rong Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01255&amp;href=/doi/10.1021%2Facs.jmedchem.9b01255" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10783–10797</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 14, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 July 2019</li><li><span class="item_label"><b>Published</b> online</span>14 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 December 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01255</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10783%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDongxing%2BChen%252C%2BLinjie%2BLi%252C%2BKrystal%2BDiaz%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D23%26contentID%3Dacs.jmedchem.9b01255%26title%3DNovel%2BPropargyl-Linked%2BBisubstrate%2BAnalogues%2Bas%2BTight-Binding%2BInhibitors%2Bfor%2BNicotinamide%2BN-Methyltransferase%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10797%26publicationDate%3DDecember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01255"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1135</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01255" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dongxing&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Linjie&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Krystal&quot;,&quot;last_name&quot;:&quot;Diaz&quot;},{&quot;first_name&quot;:&quot;Iredia&quot;,&quot;last_name&quot;:&quot;D. Iyamu&quot;},{&quot;first_name&quot;:&quot;Ravi&quot;,&quot;last_name&quot;:&quot;Yadav&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Noinaj&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10783-10797&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01255&quot;},&quot;abstract&quot;:&quot;Nicotinamide N-methyltransferase (NNMT) catalyzes the methyl transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-containing compounds. NNMT is an important regulator for nicotinamide metabolism and methylation potential. Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target. Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases. Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker. LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date. The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT. Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which wi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01255&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01255" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01255&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01255" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01255&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01255" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01255&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01255&amp;href=/doi/10.1021/acs.jmedchem.9b01255" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01255" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01255" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26pmid%3D31724854%26genre%3Darticle%26aulast%3DChen%26date%3D2019%26atitle%3DNovel%2BPropargyl-Linked%2BBisubstrate%2BAnalogues%2Bas%2BTight-Binding%2BInhibitors%2Bfor%2BNicotinamide%2BN-Methyltransferase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D23%26spage%3D10783%26epage%3D10797%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291071" title="Nicotinamide">Nicotinamide</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/jmcmar.2019.62.issue-23/20191212/jmcmar.2019.62.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Nicotinamide <i>N</i>-methyltransferase (NNMT) catalyzes the methyl transfer from the cofactor <i>S</i>-adenosylmethionine to nicotinamide and other pyridine-containing compounds. NNMT is an important regulator for nicotinamide metabolism and methylation potential. Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target. Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases. Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker. LL320 demonstrates a <i>K</i><sub>i</sub> value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date. The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT. Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of methyl transfer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03273" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03273" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Nicotinamide <i>N</i>-methyltransferase (NNMT), a cytosolic enzyme that is predominantly expressed in the liver and adipose tissue, catalyzes the transfer of a methyl group from the cofactor <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to nicotinamide (vitamin B3) to produce <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH) and N1-methylnicotinamide (MNA).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> NNMT can also methylate pyridines, quinoline, and other structurally related heterocyclic compounds.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Its substrate, nicotinamide, is a precursor for NAD<sup>+</sup>, an important cofactor involved in redox metabolism and energy homeostasis.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> NNMT is also implicated in methyl donor balance because its expression level is inversely proportional to the SAM/SAH ratio both in vivo and in vitro.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Because of its dual involvement in the metabolism of both the NAD<sup>+</sup> precursor and the methyl donor SAM, NNMT is an important regulator for nicotinamide metabolism and methylation potential. Therefore, NNMT has been linked to various diseases including cancers, neurodegenerative diseases, obesity, and diabetes.<a onclick="showRef(event, 'ref3 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref3 ref5 ref6 ref7 ref8">(3,5−8)</a> Increased expression of NNMT enhances cell proliferation and disease progression in several cancers including colorectal, gastric, breast, renal, lung, and ovarian cancers.<a onclick="showRef(event, 'ref5 ref7 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref9 ref10">(5,7,9,10)</a> Knockdown of NNMT by siRNA or shRNA inhibits the cancer cell growth and induces apoptosis.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In addition, NNMT is upregulated in Parkinson’s disease, atherosclerosis, obesity, and Type 2 diabetes.<a onclick="showRef(event, 'ref3 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref3 ref12 ref13">(3,12,13)</a> Hence, NNMT has drawn increasing attention as a potential target for the aforementioned diseases.</div><div class="NLM_p">Potent and selective NNMT inhibitors would be valuable tools to investigate the role of its catalytic activity in its physiological and pharmacological roles. However, currently reported NNMT inhibitors only have modest activity at μM levels (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). 5-Amino-1-MQ is a representative example that mimics the product and binds to the substrate binding site.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> 5-Amino-1-MQ can rejuvenate muscle stem cells to improve the regenerative capacity of the aged skeletal muscle. Compound JBSNF000088 acts as a competitive but a slow-turnover substrate analogue.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> JBSNF000088 has been demonstrated to reduce MNA levels and drives insulin sensitization in preclinical animal models of obesity.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> SS3 is a suicide inhibitor and promotes the modification of Cys159 upon methylation.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The inhibitor HS312, containing an α-chloroacetamide group, inhibits NNMT by covalently inhibiting Cys165 that is located at the SAM/SAH binding site, but loses its engagement to NNMT in cells because of its reactivity with other cysteines in situ.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reported NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Bisubstrate analogues have the potential to offer high potency and selectivity, especially for small molecule methyltransferases such as NNMT. To achieve such potential, an optimal linker is critical to orient both substrate and SAM cofactor analogues to retain their respective interactions to the target. Three previously reported bisubstrate analogues VH45, MS2756, and MS2734 demonstrated moderate activities at the μM range.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> During the development of the potent bisubstrate inhibitor for protein N-terminal methyltransferase 1 (NTMT1), a 3-C atom linker exhibited its advantage to link the substrate and the SAM analogue to offer potent and selective inhibition at the nM level.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Because the distance between the sulfur atom and nucleophilic N atom is about 4 Å in both NNMT and NTMT1,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> we hypothesize that a 3-C atom linker may serve as an optimal linker to produce NNMT bisubstrate analogues with high potency. Here, we designed and synthesized a series of bisubstrate analogues with variable 3-C atom linkers and a 4-C atom linker to investigate the optimal linker length. Top inhibitor LL320 with a propargyl linker exhibits a <i>K</i><sub>i</sub> value at 1.6 ± 0.3 nM, which is the most potent inhibitor for NNMT to date. In addition, we obtained cocrystal structures of both LL319 (<b>1a</b>) and LL320 (<b>2a</b>) in complex with human NNMT (hNNMT), which clearly delineate the occupancies of bisubstrate analogues at both nicotinamide and SAM binding sites. Here, we reported the first case that the propargyl linker is able to construct potent and tight-binding bisubstrate inhibitors for NNMT. The propargyl linker expands the linker pool to devise bisubstrate analogues for methyltransferases and provides a valuable approach to develop potent and selective inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design</h3><div class="NLM_p">There are two adjacent binding sites that are occupied by nicotinamide and SAH from the ternary complex crystal structure of the hNNMT–nicotinamide–SAH (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The kinetic mechanism of NNMT indicates a rapid equilibrium ordered mechanism, where NNMT binds SAM first and is followed by binding nicotinamide to form a ternary complex.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The distance between the nicotinamide nitrogen atom and the sulfur atom of SAH ranges from 3.5 to 4.2 Å,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> which is about the linear distance of 2–3 methylene groups. Using a saturated 3-C-atom linker has been shown to be a feasible strategy to link both substrate and SAM analogues to generate a potent and selective inhibitor for NTMT1.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Thus, we hypothesized that bisubstrate analogues that covalently link an SAM and a nicotinamide analogue through a 3-C-atom linker would imitate the ternary complex to offer higher potency and selectivity for NNMT. In addition, there is a very narrow tunnel guiding to the S<sub>N</sub>2-type of methylation process of nicotinamide by SAM. We proposed that a more rigid unsaturated 3-C-atom such as a propargyl linker may constrain the conformation to decrease the entropy loss and to yield higher potency than a saturated 3-C-atom linker (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of NNMT bisubstrate inhibitors <b>1a–3a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To support our hypothesis, we docked proposed compound <b>1a</b> into the SAH binding site of the crystal structure of the hNNMT–nicotinamide–SAH ternary complex (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Comparison of our docking model with the ternary structure depicted that the NAH moiety of <b>1a</b> overlays well with SAH and the benzamide moiety inserts deeper into the substrate-binding pocket to attain two extra H-bonds than nicotinamide. The oxygen atom of the benzamide forms two H-bonds with Ser201 and Ser213. Also, the nitrogen atom of the benzamide interacts with Ser213 and Asp197 residues through two additional H-bonds.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking analysis of compound <b>1a</b> in the binding sites of hNNMT. (A) Docking of compound <b>1a</b> (blue) in the hNNMT (gray) structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). H-bond interactions are shown in yellow dotted lines. (B) Overlay of the docking model with the hNNMT (gray)–nicotinamide (yellow)–SAH (pink) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p">Syntheses of all bisubstrate analogues were accomplished in a convergent route through reductive amination of NAH and aldehydes (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Protected NAH was synthesized as previously described.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Compound <b>S3</b> was synthesized according to <a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Scheme S1</a>. To synthesize compound <b>1a</b>, allyl alcohol was coupled with 3-iodobenzonitrile <b>4a</b> under the Heck coupling condition to yield aldehyde <b>5a</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> which was subjected to reductive amination with protected NAH to produce <b>6a</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Oxidation by H<sub>2</sub>O<sub>2</sub> followed by TFA deprotection provided <b>1a</b>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Other analogues <b>1e–1j</b> were prepared using the similar procedure from commercially available aldehydes <b>5e–5j</b> and protected NAH or <b>S3</b> (<a class="ref internalNav" href="#sch2" aria-label="Schemes 2">Schemes 2</a> and <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). Compound <b>7a</b> was synthesized from commercially available 3-iodobenzonitrile through the Sonogashira cross-coupling reaction (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Then, <b>7a</b> was subjected to Dess–Martin oxidation to yield aldehyde <b>8a</b>, which was then subjected to reductive amination with protected NAH to produce <b>9a</b>. Subsequent oxidation by H<sub>2</sub>O<sub>2</sub> and TFA deprotection produced <b>2a</b>. Likewise, Sonogashira reaction with 3-butynl-1-ol offered <b>10</b>, which underwent hydrogenation and subsequent oxidation to provide aldehyde <b>11</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). Reductive amination of <b>11</b> with protected NAH through reductive amination afforded <b>12a</b>, which was then subjected to H<sub>2</sub>O<sub>2</sub> oxidation and acid deprotection to yield <b>3a</b>. Ester prodrugs <b>1a*</b> and <b>2a*</b> were synthesized for the cell-based study (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>). MS2756 was resynthesized as reported before and served as a positive control for the inhibition study.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Bisubstrate Inhibitors <b>1a–1d</b> and <b>1a*</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>, Bu<sub>4</sub>NCl, NaHCO<sub>3</sub>, DMF, 3 h, 45–84%; (b) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56–68%; (c) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49–61%; (d) SOCl<sub>2</sub>, EtOH, 0–50 °C, 59%.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Bisubstrate Inhibitors <b>1e–1i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56−65%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 50−58%.</p></p></figure><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Bisubstrate Inhibitor <b>1j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 68%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 52%.</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Bisubstrate Inhibitor <b>2a–2d</b> and <b>2a*</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, CH<sub>3</sub>CN, 12 h, 87−95%; (b) DMP, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt 1 h, 55−61%; (c) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56−68%; (d) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49−61%; (e) SOCl<sub>2</sub>, EtOH, 0 °C to 50 °C, 51%.</p></p></figure><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Bisubstrate Inhibitors <b>3a–3b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, CH<sub>3</sub>CN, 12 h, 90%; (b) Pd/C, H<sub>2</sub>, THF, 12 h; then DMP, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt 1 h, 60%; (c) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 50−56%; (d) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49−51%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Biochemical Characterization</h3><div class="NLM_p">The SAH hydrolase (SAHH)-coupled fluorescence assay was employed to evaluate the inhibitory activities of all synthesized compounds continuously through monitoring the production of SAH.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> Under our SAHH-coupled assay condition, the <i>K</i><sub>m</sub> values for nicotinamide and SAM were determined to be around 10 μM, which are comparable to the reported values (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Inhibition studies were performed with both SAM and nicotinamide at their respective <i>K</i><sub>m</sub> values. All compounds were incubated with NNMT for 10 min before initiating the reaction through addition of nicotinamide. Inhibitory activity of each compound was first analyzed by obtaining initial velocities at increasing concentrations and followed by nonlinear regression fit. Compounds <b>1a</b> and <b>2a</b> displayed IC<sub>50</sub> values close to the enzyme concentration, suggesting that they are tight binding inhibitors.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Therefore, both tight binding inhibitors were reanalyzed using Morrison’s quadratic equation to determine their apparent <i>K</i><sub>i</sub> values.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Compared to the control MS2756 (<i>K</i><sub>i</sub> = 10 ± 0.35 μM), <b>1a</b> (<i>K</i><sub>i</sub> = 43 ± 5.0 nM), <b>2a</b> (<i>K</i><sub>i</sub> = 6.8 ± 2.3 nM), and <b>3a</b> (<i>K</i><sub>i</sub> = 840 ± 50 nM) exhibit over 200-fold, 1000-fold, and 10-fold increased inhibition, respectively. Therefore, the 3-C-atom linker used to construct <b>1a</b> and <b>2a</b> is superior to the 4-C-atom linker used to build <b>3a</b> for development of NNMT bisubstrate inhibitors. Among them, the propargyl linker is the best linker because compound <b>2a</b> showed the highest potency (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical characterization of compound LL320 (<b>2a</b>). (A) SAHH-coupled fluorescence assay of compound LL320 ranging in concentrations from 0 to 10 μM; (B) concentration–response plots fitted to Morrison’s quadratic equation for compound LL320.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SAR Studies</h3><div class="NLM_p">After the 3-C-atom linker was established as the optimal length, structure–activity relationship (SAR) studies were carried out to understand the contribution of each portion to the potency of NNMT bisubstrate inhibitor (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Analogues of compounds <b>1a</b> were initially synthesized for the SAR studies because compound <b>1a</b> was designed as a standard approach to build a potent bisubstrate analogue for NNMT.</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SARs of NNMT inhibitors. *The values of <i>K</i><sub>i</sub> for the compounds were determined with duplicate experiments (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Nicotinamide Analogues</h3><div class="NLM_p">We began SAR analysis with the investigation on the nicotinamide portion. Our synthetic scheme can accommodate the introduction of nicotinamide analogues through a late-stage reductive amination reaction. Quinoline and 4-methylnicotinamide have shown <i>K</i><sub>m</sub> values comparable to nicotinamide,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> with isoquinoline showing about a 3-fold lower <i>K</i><sub>m</sub> value than nicotinamide.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Therefore, substituted benzamide and naphthalene moieties were introduced as a nicotinamide analogue to produce <b>1a–1j</b>. Compared with <b>1a</b>, introducing a methyl group to the ortho-position of the benzamide marginally affected the inhibition as shown in compound <b>1c</b>, while the activity decreased more than 3-fold when introducing a methyl group at the para-position of the benzamide as shown in compound <b>1b</b>. When the benzamide moiety was replaced by a naphthalene ring to produce compounds <b>1g</b> and <b>1h</b>, the inhibitory activities dropped about 100-fold to 300-fold compared with <b>1a</b>. Replacing the amide group with an ester, compound <b>1j</b> lost about 700-fold activity compared with <b>1a</b>, which further supported the importance of the amide group for binding to NNMT.</div><div class="NLM_p last">Introduction of a methoxy group at the para-position of the benzamide of <b>2a</b> produced <b>2b</b>, resulting in a 7-fold loss of activity. When a methyl group was introduced at the ortho-position of the benzamide of <b>2a</b> to generate <b>2c</b>, comparable activity was observed.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Aspartic Acid Portion</h3><div class="NLM_p last">MS2734 showed over 10-fold higher inhibitory activity than MS 2756 by replacing the aspartic amino acid with glutamic acid in the SAM portion.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> However, similar alteration of <b>1a</b> to <b>1d</b> failed to display such trends, as <b>1d</b> exhibited a 70-fold decreased activity. We also noticed that this alteration produced different effects on parent compounds as <b>1e</b>, <b>1f,</b> and <b>3b</b> showed either comparable or 3-fold decreased activities when compared to their respective parent compounds <b>1g</b>, <b>1h,</b> and <b>3a</b>.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Deconstruction Strategy</h3><div class="NLM_p last">In order to understand the contributions of each moiety of LL320, we applied a deconstruction strategy. When the aspartate portion is removed to yield <b>2d</b>, there is about a 500-fold loss of activity compared to <b>2a</b>. Furthermore, the <i>K</i><sub>i</sub> value of benzamide is more than 100 μM, resulting in over 10 000-fold loss of activity compared to <b>2a</b>. This suggests that each portion of the bisubstrate analogue is important for the potent inhibition.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Slow-Binding Inhibition</h3><div class="NLM_p">Tight-binding inhibitors exhibit slow binding characteristics. Therefore, we investigated the optimal preincubation time that will ensure the equilibrium condition reached for Morrison’s analysis. Previously, all synthesized compounds were only preincubated with NNMT for 10 min. Therefore, different preincubation times (30, 60, 90, and 120 min) were examined to ensure the steady state condition can be achieved (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). From the results (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), the system reaches equilibrium after a 60 min preincubation, as there is a negligible difference in <i>K</i><sub>i</sub> values at a longer preincubation time. In addition, rapid dilution study suggested the recovery of NNMT activity at 3 nM of LL320 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S2</a>), suggesting that compound LL320 is a reversible and tight binding inhibitor for NNMT.</div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Time-dependent inhibition study of compound LL320 (<i>n</i> = 2). (A) Concentration–response plots for different preincubation times fitted to Morrison’s quadratic equation for compound LL320; (B) nonlinear regression of Morrison <i>K</i><sub>i</sub> values with preincubation time.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Selectivity Studies</h3><div class="NLM_p">Bisubstrate analogues are known to display high selectivity to their target.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To evaluate the selectivity of <b>2a</b>, we investigated its inhibitory activity over a panel of methyltransferases including two closely-related methyltransferases to NNMT such as phenylethanolamine <i>N</i>-methyltransferase (PNMT) and indolethylamine <i>N</i>-methyltransferase (INMT), two representative members from protein lysine methyltransferase PKMT (G9a and SETD7) and protein arginine methyltransferase PRMT (PRMT1 and <i>Tb</i>PRMT7), respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S3</a>). We also included NTMT1, sharing an SAM cofactor binding pocket with NNMT. In addition, SAHH is included in the selectivity study because it has an SAH binding site and is used in the coupled fluorescence assay. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S3</a>, compound <b>2a</b> is more than 2000-fold selective over SAM-dependent protein methyltransferases including PNMT, INMT, G9a, SETD7, and PRMT1 because it barely displayed any inhibition at 100 μM. In addition, <b>2a</b> displayed about 1000-fold selectivity for <i>Tb</i>PRMT7 and NTMT1, and 100-fold selectivity for SAHH.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selectivity Evaluation of Compound LL320 (<b>2a</b>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">[LL320] (μM)</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme activity (%)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">3.7</th><th class="colsep0 rowsep0" align="center" char=".">11</th><th class="colsep0 rowsep0" align="center" char=".">33</th><th class="colsep0 rowsep0" align="center" char=".">100</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PNMT</td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="char" char=".">93</td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">INMT</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">84</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G9a</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SETD7</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT1</td><td class="colsep0 rowsep0" align="char" char=".">93</td><td class="colsep0 rowsep0" align="char" char=".">95</td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TbPRMT7</td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="left">>33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NTMT1</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="left">>33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SAHH</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="left">>3.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The values of enzyme activity are mean values of duplicate experiments (<i>n</i> = 2).</p></div></div></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Cocrystal Structure of LL319 (<b>1a</b>) and LL320 (<b>2a</b>) in Complex with NNMT</h3><div class="NLM_p">To elucidate the molecular interactions of bisubstrate analogues with NNMT, we determined the X-ray cocrystal structure of NNMT in complex with LL319 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a>) and LL320 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Both LL319 and LL320 were found to occupy both the cofactor and nicotinamide binding sites of NNMT. Specifically, the adenine part of <b>1a</b> and <b>2a</b> in the binary complex binds nearly identically with that of SAH. The inhibitor-superimposition of our NNMT–LL319 and NNMT–LL320 with the published NNMT–nicotinamide–SAH ternary complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>) gave a root-mean-square deviation value of 0.421 and 0.442 Å (across all residues of chain A), respectively. The interactions of inhibitors–NNMT are retained in a similar manner as previously observed with NNMT–SAH–nicotinamide in the ternary complex of nicotinamide/SAH and NNMT–MS2756 binary complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> For example, the carboxyl group of the NAH portion forms four H-bonds with Tyr 20, Tyr25, Tyr69, and Thr163. The amino group of NAH forms two H-bonds with Gly63 and Thr163 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D,E). Meanwhile, the adenine moiety of both <b>1a</b> and <b>2a</b> forms two H-bonds with Asp142 and Val143, and two H-bonds with two water molecules. Hydroxyl groups of the ribose also form H-bonds with both Asp85 and Asn90. Meanwhile, the benzamide portion of <b>1a</b> and <b>2a</b> also binds very similarly to the nicotinamide through H-bond interactions with Ser201, Ser213, and one water molecule. Extensive examination of H-bonding interactions of <b>2a</b> with NNMT indicates that additional H-bonds were formed between the amino group of NAH with Ser64 and Thr67, and between the benzamide portion with two surrounding Tyr residues 203 and 204. Superimposed structures also suggested that both <b>1a</b> and <b>2a</b> extended a bit further to interact with NNMT (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figure S4A</a>).</div><figure id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of NNMT in complex with LL319 and LL320. (A) Structure of NNMT (gray cartoon) bound with LL319 (green stick) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a>). An 2Fo–Fc omit map contoured to 1.0 σ is shown for LL319 as a transparent green isosurface. (B) Structure of NNMT (gray cartoon) bound with LL320 (yellow stick) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>). An Fo–Fc omit map contoured to 3.0 σ is shown for LL320 as a transparent yellow isosurface. (C) Structural alignment of LL319 (green stick) and LL320 (yellow stick) within the binding pocket of NNMT. A ligand interaction diagram is shown for LL319 (D) and LL320 (E) with NNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Permeability Evaluation</h3><div class="NLM_p last">We tested the cell permeability of compounds <b>1a–3a</b> through MS detection on their cellular levels.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> However, the compounds were barely detected at 100 μM inside cells. In order to increase the cell permeability, corresponding ethyl ester prodrugs <b>1a*</b> and <b>2a*</b> of compounds <b>1a</b> and <b>2a</b> were designed and synthesized, respectively (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>). Unfortunately, prodrug <b>1a*</b> barely exhibited enhanced cell permeability and <b>2a*</b> only displayed limited permeability at 100 μM in comparison with TAT peptide through a cellular MS assay, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Figures S5–S7</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we designed and synthesized a series of potent bisubstrate inhibitors of NNMT. This potency supports the optimal length of the methyl transfer tunnel. To our knowledge, <b>2a</b> is the most potent NNMT inhibitors reported so far with a <i>K</i><sub>i</sub> value of 1.6 ± 0.3 nM in the SAHH-coupled fluorescence assay. <b>2a</b> exhibited around 1000-fold selectivity for NNMT over other methyltransferases and almost 100-fold selectivity for SAHH. Furthermore, the cocrystal structure of NNMT in the complex with <b>1a</b> or <b>2a</b> clearly showed that the bisubstrate inhibitors occupied both cofactor SAM and substrate binding sites, which is consistent with our design strategy and the docking study. Despite their high potency and selectivity, limited cell permeability restricts them from cell-based studies. However, these tight and slow binding bisubstrate inhibitors are valuable probes for development of future cell-potent inhibitors for NNMT. More importantly, <b>2a</b> is the first example to adopt a propargyl linker to conjugate SAM and a substrate analogue to yield a methyltransferase bisubstrate inhibitor. In addition, the propargyl linker of <b>2a</b> provides insights into the orientation of the methyl transfer of NNMT. Furthermore, the strategy described here not only applies to target NNMT but also may have widespread impact on the development of potent inhibitors for other methyltransferases.</div><div class="NLM_p last">Note added: During the revision of this manuscript, Shair and co-workers developed another tight binding bisubstrate inhibitor <b>NS1</b> with alkynyl linker for NNMT.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The <i>K</i><sub>i</sub> value of the compound was around 500 pM and its analogues showed a modest effect on the cell-based study.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry General Procedures</h3><div class="NLM_p last">The reagents and solvents were purchased from commercial sources (Fisher and Sigma-Aldrich) and used directly. Analytical thin-layer chromatography (TLC) was performed on ready-to-use plates with silica gel 60 (Merck, F254). Flash column chromatography was performed over silica gel (grade 60, 230–400 mesh) on a Teledyne Isco CombiFlash purification system. Final compounds were purified by preparative reversed phase high-pressure liquid chromatography (RP-HPLC) that was performed on an Agilent 1260 Series system. The Agilent 1260 Infinity II Variable Wavelength Detector (G7114A, UV = 254 nM) and a Waters BEH C18 (130 Å, 5 μm, 10 mm × 250 mm) at a flow rate of 4 mL/min using a solvent system of 100% water with 0.1% TFA to 50% methanol over 30 min were used to purify the final compounds. NMR spectra were acquired on a Bruker AV500 instrument (500 MHz for <sup>1</sup>H NMR, 126 MHz for <sup>13</sup>C NMR). TLC-MS was carried out using an Advion CMS-L MS. Matrix-assisted laser desorption ionization mass spectrometry (MALDI-MS) data were acquired in the positive-ion mode using a Sciex 4800 MALDI TOF/TOF MS. The Agilent 1260 Infinity II Variable Wavelength Detector (G7114A, UV = 254 nM) and an Agilent ZORBAX RR SB-C18 (80 Å, 3.5 μm, 4.6 × 150 mm) at a flow rate of 1 mL/min using a solvent system of 100% water with 0.1% TFA to 40 or 60% methanol over 20 min were used to assess purity of final compounds. All the purity of target compounds showed >95% in RP-HPLC.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure A (Heck Coupling)</h3><div class="NLM_p last">Tetrabutyl ammonium chloride (4.9 mmol) and sodium bicarbonate (12.2 mmol) were taken in dry DMF (4 mL) and cooled to 0 °C and 3-iodobenzonitrile (<b>4a–c</b>) (4.9 mmol) was added. Allyl alcohol (7.3 mmol) was added to the reaction mixture, followed by a catalytic amount of Pd(OAc)<sub>2</sub> (0.15 mmol), and the mixture was stirred at 0 °C for 30 min. The reaction mixture was slowly warmed up to room temperature and further stirred for 2 h. The reaction mixture was diluted with water, and the organic product was extracted with ether. The combined extracts were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure to get the crude product. The residue was purified with the silica gel column to provide the desired product.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure B (Sonogashira Cross Coupling)</h3><div class="NLM_p last">Under nitrogen condition, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (0.1 mmol), CuI (0.2 mmol), and iodine substrates (<b>4a–c</b>) (5 mmol) were successively added to a 25 mL vial equipped with a stir bar. MeCN (5 mL) was added using a syringe, and then, propargyl alcohol (6 mmol) was added to the mixture. Et<sub>3</sub>N (15 mmol) was added at last. The reaction was stirred at room temperature for 8 h. Solvent was removed in vacuo to leave the crude mixture, which is purified by silica gel column chromatography to afford the pure product.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure C (Oxidation of Alcohols)</h3><div class="NLM_p last">To a solution of the alcohol (<b>7a–c</b>) (2.5 mmol) in DCM (5 mL) was added DMP (2.5 mmol) at 0 °C. The resulting solution was stirred for 1 h at room temperature. The reaction was quenched with saturated sodium hyposulfite solution (5 mL), extracted with DCM (3 × 25 mL), washed with Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> three times, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After the removal of solvent, the residue was purified with a silica gel column to provide the aldehydes.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedure D (Reductive Amination)</h3><div class="NLM_p last">To a solution of protected NAH, <b>S1</b> or <b>S3</b> (0.06 mmol) in 0.5 mL MeOH was added the aldehyde (<b>5a–j</b>, <b>8a–c</b> and <b>11</b>) (0.08 mmol) followed by 2 drops of AcOH. The resulting mixture was stirred for 30 min before NaBH<sub>3</sub>CN (0.1 mmol) was added. After being stirred for 2 h at rt, the reaction was quenched with saturated NaHCO<sub>3</sub> and extracted with DCM (5×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified with a silica gel column to provide the desired product.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure E (Oxidation and Deprotection of Bisubstrate Inhibitors)</h3><div class="NLM_p last">To a solution of the cyano compound (<b>6a–d</b>, <b>9a–d</b>, and <b>12a–b</b>) (0.05 mmol) in DMSO (1 mL) was added K<sub>2</sub>CO<sub>3</sub> (0.2 mmol). The mixture was cooled to 0 °C and treated with H<sub>2</sub>O<sub>2</sub> (0.2 mL). The reaction mixture was warmed to rt and stirred for 3 h at rt. The reaction was diluted with water and extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford the crude compound. The crude protected inhibitor was dissolved in DCM (0.5 mL). The resulting solution was cooled to 0 °C and treated with TFA (0.2 mL). The reaction mixture was warmed to rt and stirred for 6 h. The solution was concentrated under reduced pressure, dissolved in ddH<sub>2</sub>O, and purified by reverse HPLC to get the corresponding bisubstrate compound.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedure F (Deprotection of Bisubstrate Inhibitors)</h3><div class="NLM_p last">The protected inhibitor (<b>6e–j</b>) was dissolved in DCM (0.5 mL). The resulting solution was cooled to 0 °C and treated with TFA (0.2 mL). The reaction mixture was warmed to rt and stirred for 6 h. The solution was concentrated under reduced pressure, dissolved in ddH<sub>2</sub>O, and purified by reverse HPLC to get the corresponding bisubstrate compound.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Procedure G (Esterification of Bisubstrate Analogues)</h3><div class="NLM_p last">To a solution of bisubstrate analogue <b>1a</b> or <b>2a</b> (0.02 mmol) in anhydrous EtOH (3 mL), SOCl<sub>2</sub> (0.1 mmol) was added at 0 °C. The mixture was stirred at 50 °C for 2 h. The volatiles were removed under reduced pressure. The residue was dissolved in ddH<sub>2</sub>O and purified by RP-HPLC to provide the ester prodrugs of the respective bisubstrate inhibitors.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoylphenyl)propyl)amino)butanoic Acid (<b>1a</b>)</h3><div class="NLM_p last">Compound <b>1a</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.40 (s, 1H), 8.32 (s, 1H), 7.71–7.61 (m, 2H), 7.39–7.21 (m, 2H), 6.11–5.95 (m, 1H), 4.71–4.61 (m, 1H), 4.47–4.32 (m, 2H), 4.04–3.92 (m, 1H), 3.84–3.70 (m, 1H), 3.67–3.52 (m, 2H), 3.50–3.39 (m, 1H), 3.29–3.18 (m, 2H), 2.78–2.65 (m, 2H), 2.44–2.27 (m, 1H), 2.24–2.12 (m, 1H), 2.12–1.97 (m, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.93, 170.21, 151.82, 148.27, 145.87, 142.66, 140.50, 133.91, 131.52, 128.42, 127.20, 125.26, 119.67, 90.70, 78.73, 73.19, 72.14, 54.97, 51.19, 46.92, 46.75, 31.65, 24.72, 24.55. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>33</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 529.2523; found, 529.4907.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Ethyl (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoylphenyl)propyl)amino)butanoate (<b>1a*</b>)</h3><div class="NLM_p last">Compound <b>1a*</b> was prepared according to the general procedure G. <sup>1</sup>H NMR (500 MHz, D<sub>2</sub>O): δ 8.22 (s, 1H), 8.16 (s, 1H), 7.43 (d, <i>J</i> = 7.7 Hz, 1H), 7.34 (s, 1H), 7.21 (t, <i>J</i> = 7.5 Hz, 1H), 7.14 (d, <i>J</i> = 7.8 Hz, 1H), 5.92 (d, <i>J</i> = 3.6 Hz, 1H), 4.58 (t, <i>J</i> = 4.2 Hz, 1H), 4.39–4.29 (m, 2H), 4.20 (q, <i>J</i> = 7.1 Hz, 2H), 4.15–4.10 (m, 1H), 3.76–3.68 (m, 1H), 3.57 (d, <i>J</i> = 14.3 Hz, 1H), 3.52–3.43 (m, 1H), 3.43–3.34 (m, 1H), 3.24 (t, <i>J</i> = 8.3 Hz, 2H), 2.63–2.54 (m, 1H), 2.54–2.46 (m, 1H), 2.41–2.30 (m, 1H), 2.27–2.18 (m, 1H), 2.00–1.80 (m, 2H), 1.17 (t, <i>J</i> = 7.1 Hz, 3H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>36</sub>N<sub>8</sub>O<sub>6</sub>Na [M + Na]<sup>+</sup>, 579.2656; found, 579.1624.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(5-carbamoyl-2-methylphenyl)propyl)amino)butanoic Acid (<b>1b</b>)</h3><div class="NLM_p last">Compound <b>1b</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.33 (d, <i>J</i> = 4.3 Hz, 1H), 8.26 (s, 1H), 7.62 (d, <i>J</i> = 2.0 Hz, 1H), 7.57 (dd, <i>J</i> = 7.8, 2.0 Hz, 1H), 7.16 (d, <i>J</i> = 7.8 Hz, 1H), 6.04 (t, <i>J</i> = 4.2 Hz, 1H), 4.70 (t, <i>J</i> = 4.2 Hz, 1H), 4.48–4.38 (m, 2H), 3.91 (dd, <i>J</i> = 9.6, 3.6 Hz, 1H), 3.80–3.70 (m, 1H), 3.62 (d, <i>J</i> = 13.9 Hz, 1H), 3.54 (td, <i>J</i> = 8.5, 8.0, 4.3 Hz, 1H), 3.50–3.42 (m, 1H), 3.35 (s, 3H), 2.75–2.60 (m, 2H), 2.39–2.29 (m, 1H), 2.22 (s, 2H), 2.12 (d, <i>J</i> = 6.5 Hz, 2H), 2.03 (p, <i>J</i> = 7.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 172.35, 155.10, 150.24, 149.89, 143.20, 141.70, 139.89, 132.71, 131.47, 128.90, 126.71, 121.09, 109.28, 92.11, 80.48, 74.33, 73.65, 56.09, 53.77, 49.85, 49.51, 30.41, 26.38, 24.85, 19.30.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoyl-4-methylphenyl)propyl)amino)butanoic Acid (<b>1c</b>)</h3><div class="NLM_p last">Compound <b>1c</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.43 (s, 1H), 8.28 (s, 1H), 7.22 (s, 1H), 7.12 (d, <i>J</i> = 7.8 Hz, 1H), 7.06 (d, <i>J</i> = 7.9 Hz, 1H), 6.08 (d, <i>J</i> = 4.1 Hz, 1H), 4.73–4.68 (m, 1H), 4.49–4.35 (m, 2H), 4.08–3.93 (m, 1H), 3.80–3.71 (m, 1H), 3.63 (d, <i>J</i> = 13.8 Hz, 1H), 3.58–3.50 (m, 1H), 3.49–3.41 (m, 1H), 3.29–3.18 (m, 2H), 2.71–2.56 (m, 2H), 2.38 (s, 3H), 2.36–2.29 (m, 1H), 2.23–2.12 (m, 1H), 2.08–1.96 (m, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 174.24, 170.38, 151.72, 148.30, 145.69, 142.85, 137.44, 136.31, 133.35, 130.75, 129.54, 126.54, 120.04, 90.63, 78.77, 73.18, 72.14, 54.91, 51.08, 46.87, 46.29, 31.17, 24.67, 24.49, 18.03. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>35</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 543.2680; found, 543.3776.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>S</i>)-2-Amino-5-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoylphenyl)propyl)amino)pentanoic Acid (<b>1d</b>)</h3><div class="NLM_p last">Compound <b>1d</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.34 (d, <i>J</i> = 32.5 Hz, 2H), 7.79–7.58 (m, 2H), 7.41–7.20 (m, 2H), 6.07 (d, <i>J</i> = 3.8 Hz, 1H), 4.75–4.62 (m, 2H), 4.50–4.32 (m, 2H), 3.97 (d, <i>J</i> = 7.0 Hz, 1H), 3.78 (dd, <i>J</i> = 13.9, 9.8 Hz, 1H), 3.62 (d, <i>J</i> = 13.3 Hz, 1H), 3.36–3.32 (m, 1H), 3.27 (d, <i>J</i> = 8.2 Hz, 3H), 2.82–2.60 (m, 2H), 2.04 (dt, <i>J</i> = 12.0, 6.0 Hz, 2H), 1.97 (d, <i>J</i> = 19.4 Hz, 2H), 1.93–1.81 (m, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.92, 156.31, 148.48, 140.46, 134.01, 133.98, 133.94, 131.48, 128.45, 127.24, 126.24, 125.24, 90.80, 78.73, 73.06, 72.20, 52.59, 50.00, 46.94, 46.77, 31.67, 27.11, 24.43, 19.63.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>S</i>)-2-Amino-5-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(naphthalen-1-yl)propyl)amino)pentanoic Acid (<b>1e</b>)</h3><div class="NLM_p last">Compound <b>1e</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.29 (s, 1H), 8.19 (s, 1H), 8.01–7.87 (m, 1H), 7.87–7.75 (m, 1H), 7.70 (d, <i>J</i> = 8.2 Hz, 1H), 7.51–7.38 (m, 2H), 7.32 (t, <i>J</i> = 7.6 Hz, 1H), 7.19 (d, <i>J</i> = 6.8 Hz, 1H), 5.99 (d, <i>J</i> = 3.4 Hz, 1H), 4.66–4.55 (m, 1H), 4.49–4.30 (m, 2H), 4.00–3.88 (m, 1H), 3.84–3.73 (m, 1H), 3.59 (d, <i>J</i> = 13.8 Hz, 1H), 3.49–3.33 (m, 3H), 3.29–3.25 (m, 1H), 3.20–3.08 (m, 1H), 3.08–2.96 (m, 1H), 2.23–2.04 (m, 2H), 2.04–1.78 (m, 4H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.13, 152.86, 148.22, 145.75, 142.05, 135.86, 133.93, 131.38, 128.48, 126.89, 125.76, 125.64, 125.30, 125.08, 122.80, 119.70, 90.85, 78.26, 73.17, 72.12, 53.30, 52.83, 51.95, 48.45, 48.23, 28.87, 27.13, 19.76. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>36</sub>N<sub>7</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 550.2778; found, 550.3173.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (<i>S</i>)-2-Amino-5-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(naphthalen-2-yl)propyl)amino)pentanoic Acid (<b>1f</b>)</h3><div class="NLM_p last">Compound <b>1f</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.31 (s, 1H), 8.27 (s, 1H), 7.80–7.75 (m, 1H), 7.71 (d, <i>J</i> = 8.1 Hz, 2H), 7.54 (s, 1H), 7.46–7.38 (m, 2H), 7.19 (d, <i>J</i> = 8.2 Hz, 1H), 6.00 (d, <i>J</i> = 3.7 Hz, 1H), 4.70–4.56 (m, 1H), 4.48–4.33 (m, 2H), 3.93 (t, <i>J</i> = 6.1 Hz, 1H), 3.85–3.70 (m, 1H), 3.70–3.55 (m, 1H), 3.39–3.32 (m, 2H), 3.30–3.24 (m, 2H), 2.87–2.67 (m, 2H), 2.19–2.03 (m, 2H), 2.03–1.74 (m, 4H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.27, 153.68, 148.45, 141.67, 141.54, 137.33, 133.59, 132.25, 127.90, 127.22, 127.05, 126.25, 126.14, 125.79, 125.16, 119.80, 90.77, 78.44, 73.06, 72.17, 54.71, 53.04, 52.75, 48.37, 32.06, 27.16, 24.53, 19.72. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>36</sub>N<sub>7</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 550.2778; found, 550.3164.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(naphthalen-1-yl)propyl)amino)butanoic Acid (<b>1g</b>)</h3><div class="NLM_p last">Compound <b>1g</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.26 (s, 1H), 8.12 (s, 1H), 7.87 (d, <i>J</i> = 5.8 Hz, 1H), 7.73 (d, <i>J</i> = 6.9 Hz, 1H), 7.61 (d, <i>J</i> = 8.2 Hz, 1H), 7.35 (dd, <i>J</i> = 6.2, 3.2 Hz, 2H), 7.25 (t, <i>J</i> = 7.6 Hz, 1H), 7.14 (d, <i>J</i> = 5.4 Hz, 1H), 5.93 (s, 1H), 5.42 (s, 1H), 4.55 (s, 1H), 4.44–4.22 (m, 2H), 3.94–3.80 (m, 1H), 3.68 (s, 1H), 3.57–3.43 (m, 2H), 3.37 (s, 2H), 3.06 (d, <i>J</i> = 7.2 Hz, 2H), 3.01–2.91 (m, 1H), 2.28 (s, 1H), 2.08 (p, <i>J</i> = 8.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD_SPE): δ 172.62, 154.64, 151.16, 149.70, 143.46, 137.36, 135.35, 132.83, 129.89, 128.26, 127.19, 127.06, 126.70, 126.48, 124.29, 92.10, 80.38, 74.57, 73.58, 56.00, 54.80, 53.55, 30.31, 28.04, 26.37, 25.70.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(naphthalen-2-yl)propyl)amino)butanoic Acid (<b>1h</b>)</h3><div class="NLM_p last">Compound <b>1h</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.38–8.22 (m, 2H), 7.76 (d, <i>J</i> = 7.6 Hz, 1H), 7.74–7.68 (m, 2H), 7.55 (s, 1H), 7.45–7.36 (m, 2H), 7.25–7.18 (m, 1H), 6.04–5.96 (m, 1H), 4.68–4.59 (m, 1H), 4.48–4.35 (m, 2H), 4.03–3.90 (m, 1H), 3.83–3.69 (m, 1H), 3.69–3.59 (m, 1H), 3.59–3.50 (m, 1H), 3.50–3.42 (m, 1H), 3.39–3.32 (m, 1H), 3.28–3.22 (m, 1H), 2.86–2.73 (m, 2H), 2.42–2.28 (m, 1H), 2.22–2.05 (m, 3H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>34</sub>N<sub>7</sub>O<sub>5</sub> [M + H]<sup>+</sup>, 536.2621; found, 536.3082.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(2-(naphthalen-2-yl)ethyl)amino)butanoic Acid (<b>1i</b>)</h3><div class="NLM_p last">Compound <b>1i</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.38 (s, 1H), 8.21 (s, 1H), 7.80–7.73 (m, 1H), 7.67 (dd, <i>J</i> = 18.8, 6.8 Hz, 2H), 7.55 (s, 1H), 7.47–7.40 (m, 2H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 6.10 (d, <i>J</i> = 4.7 Hz, 1H), 4.75 (t, <i>J</i> = 4.9 Hz, 1H), 4.57 (d, <i>J</i> = 7.5 Hz, 1H), 4.44 (t, <i>J</i> = 5.1 Hz, 1H), 3.99 (dd, <i>J</i> = 9.2, 3.7 Hz, 1H), 3.85–3.72 (m, 2H), 3.72–3.63 (m, 1H), 3.63–3.50 (m, 3H), 3.35 (s, 1H), 3.25–3.12 (m, 2H), 2.51–2.38 (m, 1H), 2.21 (s, 1H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 172.43, 154.81, 149.89, 143.31, 134.84, 134.68, 133.85, 129.55, 128.64, 128.43, 128.35, 127.66, 127.45, 127.05, 121.12, 91.93, 80.63, 74.30, 73.72, 55.99, 53.25, 40.42 (2C), 30.84, 26.16.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-(methoxycarbonyl)phenyl)propyl)amino)butanoic Acid (<b>1j</b>)</h3><div class="NLM_p last">Compound <b>1j</b> was prepared according to the general procedure F. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.41 (s, 1H), 8.31 (s, 1H), 7.81 (d, <i>J</i> = 5.1 Hz, 2H), 7.35 (d, <i>J</i> = 6.8 Hz, 2H), 6.05 (d, <i>J</i> = 3.8 Hz, 1H), 4.73–4.64 (m, 1H), 4.48–4.38 (m, 2H), 4.06–3.94 (m, 1H), 3.89 (s, 3H), 3.84–3.73 (m, 1H), 3.68–3.40 (m, 4H), 3.35 (s, 1H), 2.37 (s, 1H), 2.24–2.13 (m, 1H), 2.11–1.93 (m, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 168.42, 164.26, 154.00, 149.83, 145.65, 143.60, 142.09, 134.21, 133.49, 131.58, 130.28, 129.87, 128.57, 91.89, 80.36, 74.54, 73.57, 56.29, 53.82, 52.69, 40.44 (2C), 33.08, 26.33, 26.15.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoylphenyl)prop-2-yn-1-yl)amino)butanoic Acid (<b>2a</b>)</h3><div class="NLM_p last">Compound <b>2a</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.46 (d, <i>J</i> = 11.5 Hz, 1H), 8.34 (s, 1H), 7.95 (s, 1H), 7.88 (d, <i>J</i> = 7.7 Hz, 1H), 7.58 (d, <i>J</i> = 7.6 Hz, 1H), 7.46 (t, <i>J</i> = 7.8 Hz, 1H), 6.20–6.00 (m, 1H), 4.72 (t, <i>J</i> = 4.6 Hz, 1H), 4.53–4.32 (m, 2H), 4.28–4.13 (m, 2H), 4.12–4.01 (m, 1H), 3.66–3.47 (m, 2H), 3.47–3.33 (m, 2H), 2.48–2.28 (m, 1H), 2.27–2.04 (m, 1H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.39, 169.76, 151.36, 148.37, 145.17, 142.78, 134.44, 134.16, 130.83, 128.56, 127.86, 121.85, 119.58, 90.36, 87.60, 80.18, 79.57, 73.57, 72.17, 55.80, 51.13, 48.30, 42.77, 25.48. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>29</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 525.2210; found, 525.3198.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Ethyl (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoylphenyl)prop-2-yn-1-yl)amino)butanoate (<b>2a*</b>)</h3><div class="NLM_p last">Compound <b>2a*</b> was prepared according to the general procedure G. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.46 (s, 1H), 8.37 (s, 1H), 7.95 (t, <i>J</i> = 1.8 Hz, 1H), 7.89–7.84 (m, 1H), 7.59–7.54 (m, 1H), 7.46 (t, <i>J</i> = 7.8 Hz, 1H), 6.11 (d, <i>J</i> = 3.9 Hz, 1H), 4.71 (t, <i>J</i> = 4.2 Hz, 1H), 4.44–4.38 (m, 2H), 4.31–4.23 (m, 2H), 4.19 (t, <i>J</i> = 6.3 Hz, 1H), 4.13–4.03 (m, 2H), 3.47–3.35 (m, 2H), 3.24 (t, <i>J</i> = 7.0 Hz, 2H), 2.35–2.25 (m, 1H), 2.23–2.12 (m, 1H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 171.13, 169.85, 152.78, 149.77, 146.63, 144.08, 135.73, 135.56, 132.20, 129.89, 128.95, 123.66, 121.01, 91.77, 88.05, 82.52, 82.26, 75.04, 73.57, 64.01, 57.51, 52.99, 51.52, 44.29, 27.23, 14.32. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>33</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 553.2523; found, 553.2736.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(5-carbamoyl-2-methoxyphenyl)prop-2-yn-1-yl)amino)butanoic Acid (<b>2b</b>)</h3><div class="NLM_p last">Compound <b>2b</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.45 (s, 1H), 8.30 (s, 1H), 7.97–7.87 (m, 2H), 7.09 (d, <i>J</i> = 8.7 Hz, 1H), 6.10 (d, <i>J</i> = 4.4 Hz, 1H), 4.76 (t, <i>J</i> = 4.8 Hz, 1H), 4.52–4.44 (m, 1H), 4.39 (t, <i>J</i> = 5.1 Hz, 1H), 4.16 (s, 2H), 4.04 (d, <i>J</i> = 6.5 Hz, 1H), 3.91 (s, 3H), 3.56 (t, <i>J</i> = 11.8 Hz, 1H), 3.50–3.42 (m, 1H), 3.38 (d, <i>J</i> = 6.6 Hz, 1H), 3.34 (s, 2H), 2.40–2.25 (m, 1H), 2.17–2.04 (m, 1H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 170.43, 164.43, 154.12, 148.44, 142.32, 139.19, 134.23, 131.50, 127.10, 124.76, 120.52, 112.17, 111.72, 91.39, 85.10, 81.97, 74.89, 73.66, 61.00, 57.12, 56.76, 52.92, 49.85, 44.08, 27.01.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(3-(3-carbamoyl-4-methylphenyl)prop-2-yn-1-yl)amino)butanoic Acid (<b>2c</b>)</h3><div class="NLM_p last">Compound <b>2c</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.28 (s, 1H), 8.24 (d, <i>J</i> = 4.1 Hz, 2H), 7.47 (d, <i>J</i> = 1.8 Hz, 1H), 7.34 (dd, <i>J</i> = 7.7, 2.0 Hz, 2H), 7.24 (d, <i>J</i> = 8.2 Hz, 2H), 6.02 (d, <i>J</i> = 4.5 Hz, 2H), 4.77 (d, <i>J</i> = 4.4 Hz, 1H), 4.37 (d, <i>J</i> = 6.4 Hz, 6H), 3.96 (d, <i>J</i> = 9.3 Hz, 9H), 3.24–2.94 (m, 3H), 2.44 (s, 5H), 2.24 (s, 2H), 2.01 (s, 3H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>31</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 539.2367; found, 539.1651.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 3-(3-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)amino)prop-1-yn-1-yl)benzamide (<b>2d</b>)</h3><div class="NLM_p last">Compound <b>2d</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.47–8.32 (m, 2H), 8.00–7.94 (m, 1H), 7.94–7.86 (m, 1H), 7.66–7.58 (m, 1H), 7.53–7.44 (m, 1H), 6.18–6.05 (m, 1H), 4.82–4.78 (m, 1H), 4.51–4.46 (m, 1H), 4.45–4.39 (m, 1H), 4.26 (d, <i>J</i> = 4.5 Hz, 2H), 3.78–3.59 (m, 2H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 169.65, 152.17, 148.41, 146.27, 142.82, 134.49, 134.30, 130.86, 128.65, 128.13, 121.50, 119.97, 90.72, 87.25, 80.11, 79.10, 73.53, 71.89, 48.38, 37.05. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>21</sub>N<sub>7</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>, 446.1553; found, 446.1817.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> (<i>S</i>)-2-Amino-4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(4-(3-carbamoylphenyl)butyl)amino)butanoic Acid (<b>3a</b>)</h3><div class="NLM_p last">Compound <b>3a</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.43 (d, <i>J</i> = 6.5 Hz, 1H), 8.33 (s, 1H), 7.81–7.57 (m, 2H), 7.45–7.23 (m, 2H), 6.10 (d, <i>J</i> = 3.8 Hz, 1H), 4.72 (t, <i>J</i> = 4.5 Hz, 1H), 4.51–4.39 (m, 2H), 4.03–3.91 (m, 1H), 3.79–3.67 (m, 1H), 3.62 (d, <i>J</i> = 13.8 Hz, 1H), 3.57–3.49 (m, 1H), 3.49–3.38 (m, 1H), 3.30–3.18 (m, 2H), 2.64 (t, <i>J</i> = 7.2 Hz, 2H), 2.46–2.28 (m, 1H), 2.25–2.07 (m, 1H), 1.83–1.51 (m, 4H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>): δ 171.16, 170.53, 152.16, 148.42, 146.46, 142.49, 141.83, 133.64, 131.66, 128.33, 127.32, 124.93, 119.55, 90.44, 78.76, 73.19, 72.12, 54.96, 53.20, 51.40, 48.25, 34.37, 27.57, 24.74, 22.60. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>35</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 543.2680; found, 543.3305.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> (<i>S</i>)-2-Amino-5-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)(4-(3-carbamoylphenyl)butyl)amino)pentanoic Acid (<b>3b</b>)</h3><div class="NLM_p last">Compound <b>3b</b> was prepared according to the general procedure E. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 8.38 (br s, 2H), 7.82–7.51 (m, 2H), 7.48–7.16 (m, 2H), 6.11–6.04 (m, 1H), 4.73 (t, <i>J</i> = 4.4 Hz, 1H), 4.46–4.40 (m, 2H), 3.95 (br s, 1H), 3.79–3.71 (m, 1H), 3.63–3.56 (m, 1H), 3.28–3.19 (m, 4H), 2.69–2.56 (m, 2H), 2.02–1.81 (m, 4H), 1.77–1.59 (m, 4H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>37</sub>N<sub>8</sub>O<sub>6</sub> [M + H]<sup>+</sup>, 557.2836; found, 557.4778.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 3-(3-Oxopropyl)benzonitrile (<b>5a</b>)</h3><div class="NLM_p last">Compound <b>5a</b> was synthesized according to the general procedure A. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.81 (s, 1H), 7.37–7.50 (m, 4H), 2.98 (t, <i>J</i> = 5 Hz, 2H), 2.82 (t, <i>J</i> = 5 Hz, 2H); <sup>13</sup>C NMR (150 MHz, chloroform-<i>d</i>): δ 200.4, 141.9, 133.1, 131.9, 130.1, 129.4, 118.8, 112.6, 44.7, 27.5.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 4-Methyl-3-(3-oxopropyl)benzonitrile (<b>5b</b>)</h3><div class="NLM_p last">Compound <b>5b</b> was synthesized according to the general procedure A. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.82 (d, <i>J</i> = 3.7 Hz, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 2H), 7.22 (d, <i>J</i> = 7.5 Hz, 1H), 2.93 (t, 2H), 2.77 (t, <i>J</i> = 7.6 Hz, 2H), 2.36 (s, 3H). <sup>13</sup>C NMR (126 MHz, chloroform-<i>d</i>): δ 200.45, 141.99, 140.03, 131.86, 131.09, 130.03, 119.03, 109.87, 43.28, 24.77, 19.70.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 2-Methyl-5-(3-oxopropyl)benzonitrile (<b>5c</b>)</h3><div class="NLM_p last">Compound <b>5c</b> was synthesized according to the general procedure A. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.88–9.60 (m, 1H), 7.41 (s, 1H), 7.30 (d, <i>J</i> = 8.0 Hz, 1H), 7.22 (d, <i>J</i> = 8.0 Hz, 1H), 2.92 (d, <i>J</i> = 7.3 Hz, 2H), 2.79 (d, <i>J</i> = 7.2 Hz, 2H), 2.48 (s, 3H).</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(3-cyanophenyl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6a</b>)</h3><div class="NLM_p last">Compound <b>6a</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.29 (d, <i>J</i> = 3.0 Hz, 1H), 7.91 (s, 1H), 7.52–7.27 (m, 4H), 6.11–6.04 (m, 1H), 6.04–5.86 (m, 2H), 5.60–5.35 (m, 2H), 5.12–4.94 (m, 1H), 4.51–4.26 (m, 1H), 4.23–4.04 (m, 1H), 2.96–2.26 (m, 8H), 2.07–1.89 (m, 1H), 1.86–1.65 (m, 3H), 1.61 (s, 3H), 1.53–1.26 (m, 21H).</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(5-cyano-2-methylphenyl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6b</b>)</h3><div class="NLM_p last">Compound <b>6b</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.27 (s, 1H), 7.90 (s, 1H), 7.41–7.31 (m, 2H), 7.20–7.14 (m, 1H), 6.09–6.02 (m, 1H), 5.94 (s, 2H), 5.60 (d, <i>J</i> = 8.2 Hz, 1H), 5.54–5.43 (m, 1H), 5.04–4.95 (m, 1H), 4.34–4.26 (m, 1H), 4.21–4.14 (m, 1H), 2.80–2.72 (m, 1H), 2.64–2.40 (m, 7H), 2.29 (s, 3H), 1.97–1.87 (m, 1H), 1.74–1.61 (m, 3H), 1.59 (s, 3H), 1.51–1.30 (m, 21H).</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(3-cyano-4-methylphenyl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6c</b>)</h3><div class="NLM_p last">Compound <b>6c</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.28 (s, 1H), 7.90 (s, 1H), 7.36 (s, 1H), 7.23 (d, <i>J</i> = 8.1 Hz, 1H), 7.17 (d, <i>J</i> = 7.8 Hz, 1H), 6.04 (s, 1H), 6.01 (s, 2H), 5.67 (d, <i>J</i> = 8.1 Hz, 1H), 5.49 (d, <i>J</i> = 6.5 Hz, 1H), 5.03–4.93 (m, 1H), 4.32–4.23 (m, 1H), 4.21–4.10 (m, 1H), 2.77–2.71 (m, 1H), 2.60–2.52 (m, 3H), 2.51–2.46 (m, 4H), 2.45–2.41 (m, 2H), 2.40–2.35 (m, 1H), 1.94–1.86 (m, 1H), 1.70–1.62 (m, 3H), 1.60 (s, 3H), 1.48–1.36 (m, 21H).</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(3-cyanophenyl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)pentanoate (<b>6d</b>)</h3><div class="NLM_p last">Compound <b>6d</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.30 (s, 1H), 7.92 (s, 1H), 7.49–7.44 (m, 1H), 7.42 (s, 1H), 7.35 (d, <i>J</i> = 4.8 Hz, 2H), 6.05 (s, 1H), 5.72 (s, 2H), 5.51 (d, <i>J</i> = 6.3 Hz, 1H), 5.18 (d, <i>J</i> = 8.4 Hz, 1H), 5.00–4.92 (m, 1H), 4.30 (s, 1H), 4.18–4.09 (m, 1H), 2.70–2.35 (m, 8H), 1.76–1.53 (m, 9H), 1.46–1.39 (m, 21H).</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(naphthalen-1-yl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)pentanoate (<b>6e</b>)</h3><div class="NLM_p last">Compound <b>6e</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.30 (s, 1H), 8.00 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.92 (s, 1H), 7.84 (dd, <i>J</i> = 7.6, 1.7 Hz, 1H), 7.70 (d, <i>J</i> = 8.2 Hz, 1H), 7.52–7.43 (m, 2H), 7.40–7.35 (m, 1H), 7.27 (d, <i>J</i> = 6.9 Hz, 1H), 6.07–6.00 (m, 1H), 5.70–5.61 (m, 2H), 5.50 (dd, <i>J</i> = 6.4, 2.1 Hz, 1H), 5.20 (d, <i>J</i> = 8.4 Hz, 1H), 4.97 (dd, <i>J</i> = 6.8, 3.2 Hz, 1H), 4.34 (s, 1H), 4.15 (s, 1H), 3.10–2.94 (m, 2H), 2.76–2.67 (m, 1H), 2.63–2.42 (m, 5H), 1.86–1.70 (m, 6H), 1.60 (s, 3H), 1.42 (s, 18H), 1.38 (s, 3H).</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(naphthalen-2-yl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)pentanoate (<b>6f</b>)</h3><div class="NLM_p last">Compound <b>6f</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.31 (s, 1H), 7.92 (s, 1H), 7.81–7.72 (m, 3H), 7.57 (s, 1H), 7.47–7.38 (m, 2H), 7.28 (d, <i>J</i> = 8.0 Hz, 1H), 6.03 (s, 1H), 5.78 (s, 2H), 5.49 (d, <i>J</i> = 6.4 Hz, 1H), 5.20 (d, <i>J</i> = 8.4 Hz, 1H), 4.99–4.90 (m, 1H), 4.37–4.27 (m, 1H), 4.20–4.11 (m, 1H), 2.78–2.63 (m, 3H), 2.63–2.40 (m, 5H), 1.84–1.66 (m, 3H), 1.62–1.55 (m, 4H), 1.45–1.40 (m, 20H), 1.38 (s, 3H).</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(naphthalen-1-yl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6g</b>)</h3><div class="NLM_p last">Compound <b>6g</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.30 (s, 1H), 7.99 (d, <i>J</i> = 8.2 Hz, 1H), 7.91 (s, 1H), 7.87–7.82 (m, 1H), 7.71 (d, <i>J</i> = 8.2 Hz, 1H), 7.53–7.44 (m, 2H), 7.38 (t, <i>J</i> = 7.6 Hz, 1H), 7.28–7.18 (m, 1H), 6.06 (d, <i>J</i> = 2.0 Hz, 1H), 5.92 (s, 2H), 5.64 (br s, 1H), 5.46 (s, 1H), 5.04 (br s, 1H), 4.41 (br s, 1H), 4.17 (s, 1H), 3.12–2.96 (m, 2H), 2.96–2.39 (m, 6H), 2.05–1.75 (m, 4H), 1.62 (s, 3H), 1.49–1.35 (m, 21H).</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(naphthalen-2-yl)propyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6h</b>)</h3><div class="NLM_p last">Compound <b>6h</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.30 (s, 1H), 7.86 (s, 1H), 7.81–7.69 (m, 3H), 7.54 (s, 1H), 7.47–7.37 (m, 2H), 7.24 (br s, 1H), 6.08–5.98 (m, 1H), 5.92–5.75 (m, 2H), 5.43 (br s, 1H), 5.01 (s, 1H), 4.38 (br s, 1H), 4.14 (s, 1H), 2.85–2.41 (m, 8H), 1.99 (br s, 1H), 1.82 (br s, 3H), 1.59 (s, 3H), 1.46–1.36 (m, 21H).</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(2-(naphthalen-2-yl)ethyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>6i</b>)</h3><div class="NLM_p last">Compound <b>6i</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.37 (s, 1H), 7.91 (s, 1H), 7.81–7.70 (m, 3H), 7.53 (s, 1H), 7.46–7.37 (m, 2H), 7.22 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H), 6.04 (d, <i>J</i> = 2.1 Hz, 1H), 5.71 (s, 2H), 5.61–5.44 (m, 2H), 4.95 (dd, <i>J</i> = 6.4, 3.4 Hz, 1H), 4.35 (s, 1H), 4.23–4.12 (m, 1H), 2.93–2.53 (m, 8H), 2.02–1.95 (m, 1H), 1.89–1.83 (m, 1H), 1.59 (s, 3H), 1.44 (s, 9H), 1.43 (s, 9H), 1.33 (s, 3H).</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Methyl 3-(3-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)((<i>S</i>)-4-(<i>tert</i>-butoxy)-3-((<i>tert</i>-butoxycarbonyl)amino)-4-oxobutyl)amino)propyl)benzoate (<b>6j</b>)</h3><div class="NLM_p last">Compound <b>6j</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.29 (s, 1H), 7.90–7.80 (m, 3H), 7.35–7.27 (m, 2H), 6.03 (s, 1H), 5.74 (s, 2H), 5.72–5.67 (m, 1H), 5.63–5.59 (m, 1H), 5.10–4.95 (m, 1H), 4.30–4.26 (m, 1H), 4.21–4.10 (m, 1H), 3.89 (s, 3H), 2.78–2.33 (m, 8H), 2.00–1.65 (m, 4H), 1.60 (s, 3H), 1.50–1.30 (m, 21H).</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3-(3-Hydroxyprop-1-yn-1-yl)benzonitrile (<b>7a</b>)</h3><div class="NLM_p last">Compound <b>7a</b> was synthesized according to the general procedure B. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 7.75–7.66 (m, 1H), 7.69–7.54 (m, 2H), 7.43 (t, <i>J</i> = 7.9 Hz, 1H), 4.50 (s, 2H), 1.95 (br s, 1H).</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 3-(3-Hydroxyprop-1-yn-1-yl)-4-methoxybenzonitrile (<b>7b</b>)</h3><div class="NLM_p last">Compound <b>7b</b> was synthesized according to the general procedure B. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 7.66–7.47 (m, 2H), 6.90 (d, <i>J</i> = 8.7 Hz, 1H), 4.49 (s, 2H), 3.88 (s, 3H). <sup>13</sup>C NMR (126 MHz, chloroform-<i>d</i>): δ 163.02, 137.17, 133.93, 118.40, 113.66, 111.34, 103.97, 94.02, 79.12, 56.25, 51.32.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 5-(3-Hydroxyprop-1-yn-1-yl)-2-methylbenzonitrile (<b>7c</b>)</h3><div class="NLM_p last">Compound <b>7c</b> was synthesized according to the general procedure B. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 7.60 (d, <i>J</i> = 1.4 Hz, 1H), 7.49 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.24 (d, <i>J</i> = 8.0 Hz, 1H), 4.47 (s, 2H), 2.51 (s, 3H), 2.39 (s, 1H). <sup>13</sup>C NMR (126 MHz, chloroform-<i>d</i>): δ 142.11, 135.66, 135.36, 130.51, 121.33, 117.34, 113.16, 89.09, 83.26, 51.40, 20.52.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 3-(3-Oxoprop-1-yn-1-yl)benzonitrile (<b>8a</b>)</h3><div class="NLM_p last">Compound <b>8a</b> was synthesized according to the general procedure C. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.43 (s, 1H), 7.88 (s, 1H), 7.84–7.74 (m, 2H), 7.68–7.49 (m, 1H).</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 4-Methoxy-3-(3-oxoprop-1-yn-1-yl)benzonitrile (<b>8b</b>)</h3><div class="NLM_p last">Compound <b>8b</b> was synthesized according to the general procedure C. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.44 (s, 1H), 7.91–7.60 (m, 2H), 7.01 (d, <i>J</i> = 8.8 Hz, 1H), 3.98 (s, 3H). <sup>13</sup>C NMR (126 MHz, chloroform-<i>d</i>): δ 176.48, 164.37, 138.72, 136.70, 117.73, 111.93, 110.58, 104.78, 93.01, 88.52, 56.60.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 2-Methyl-5-(3-oxoprop-1-yn-1-yl)benzonitrile (<b>8c</b>)</h3><div class="NLM_p last">Compound <b>8c</b> was synthesized according to the general procedure C. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.40 (s, 1H), 7.81 (d, <i>J</i> = 1.5 Hz, 1H), 7.68 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 2.59 (s, 3H).</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(3-cyanophenyl)prop-2-yn-1-yl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>9a</b>)</h3><div class="NLM_p last">Compound <b>9a</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.32 (s, 1H), 7.95 (s, 1H), 7.62 (s, 1H), 7.58–7.51 (m, 2H), 7.42–7.35 (m, 1H), 6.07 (s, 1H), 5.75 (s, 2H), 5.50 (d, <i>J</i> = 7.3 Hz, 1H), 5.04 (s, 1H), 4.39 (s, 1H), 4.31–4.18 (m, 1H), 3.70–3.60 (m, 2H), 2.90–2.83 (m, 1H), 2.81–2.74 (m, 1H), 2.66–2.60 (m, 2H), 2.01–1.96 (m, 1H), 1.85–1.77 (m, 1H), 1.61 (s, 3H), 1.42 (s, 18H), 1.40 (s, 3H).</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(5-cyano-2-methoxyphenyl)prop-2-yn-1-yl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>9b</b>)</h3><div class="NLM_p last">Compound <b>9b</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.32 (s, 1H), 7.96 (s, 1H), 7.61–7.50 (m, 2H), 6.89 (d, <i>J</i> = 8.7 Hz, 1H), 6.07 (s, 1H), 5.94–5.77 (m, 2H), 5.57–5.44 (m, 2H), 5.06–4.97 (m, 1H), 4.47–4.33 (m, 1H), 4.25–4.21 (m, 1H), 3.88 (s, 3H), 3.74–3.63 (m, 2H), 2.91–2.74 (m, 2H), 2.64 (t, <i>J</i> = 7.0 Hz, 2H), 2.01–1.93 (m, 1H), 1.86–1.79 (m, 1H), 1.61 (s, 3H), 1.46–1.36 (m, 21H).</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(3-(3-cyano-4-methylphenyl)prop-2-yn-1-yl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>9c</b>)</h3><div class="NLM_p last">Compound <b>9c</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.37–8.28 (m, 1H), 7.94 (d, <i>J</i> = 12.2 Hz, 1H), 7.56 (d, <i>J</i> = 11.9 Hz, 1H), 7.46–7.36 (m, 1H), 7.24–7.17 (m, 1H), 6.06 (d, <i>J</i> = 12.6 Hz, 1H), 5.84–5.70 (m, 2H), 5.55–5.44 (m, 1H), 5.34–5.25 (m, 1H), 5.08–4.96 (m, 1H), 4.45–4.31 (m, 1H), 4.31–4.17 (m, 1H), 3.73–3.55 (m, 2H), 2.90–2.72 (m, 2H), 2.66–2.56 (m, 2H), 2.54–2.47 (m, 3H), 1.99–1.92 (m, 1H), 1.85–1.77 (m, 1H), 1.63–1.58 (m, 3H), 1.46–1.34 (m, 21H).</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 3-(3-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)prop-1-yn-1-yl)benzonitrile (<b>9d</b>)</h3><div class="NLM_p last">Compound <b>9d</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.30 (s, 1H), 7.90 (s, 1H), 7.66–7.63 (m, 1H), 7.58–7.51 (m, 2H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 6.01 (d, <i>J</i> = 3.2 Hz, 3H), 5.46 (dd, <i>J</i> = 6.4, 3.2 Hz, 1H), 5.08 (dd, <i>J</i> = 6.4, 3.2 Hz, 1H), 4.42 (dt, <i>J</i> = 6.0, 3.7 Hz, 1H), 3.71–3.60 (m, 2H), 3.12–2.97 (m, 2H), 2.33 (br s, 1H), 1.61 (s, 3H), 1.38 (s, 3H).</div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 3-(4-Hydroxybut-1-yn-1-yl)benzonitrile (<b>10</b>)</h3><div class="NLM_p last">Compound <b>10</b> was synthesized according to the general procedure B. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 7.63 (s, 1H), 7.58 (d, <i>J</i> = 7.9 Hz, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 1H), 7.36 (t, <i>J</i> = 7.8 Hz, 1H), 3.80 (t, <i>J</i> = 6.3 Hz, 2H), 2.66 (t, <i>J</i> = 6.4 Hz, 2H). <sup>13</sup>C NMR (126 MHz, chloroform-<i>d</i>): δ 135.87, 135.02, 131.11, 129.21, 125.12, 118.16, 112.58, 89.76, 79.98, 60.87, 23.70.</div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 3-(4-Oxobutyl)benzonitrile (<b>11</b>)</h3><div class="NLM_p last">To a solution of <b>10</b> (50 mg, 0.29 mmol) in THF was added 10% Pd/C (10 mg). The mixture stirred for overnight under H<sub>2</sub> atmosphere. After filtration, the solvent was removed in vacuum. The residue was dissolved in DCM (5 mL). Then DMP (175 mg, 0.4 mmol) was added to the solution at 0 °C. The resulting solution was stirred for 1 h at room temperature. The reaction was quenched with saturated sodium hyposulfite solution (5 mL), extracted with DCM (3 × 5 mL), washed with Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> three times and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After removal of solvent, the residue was purified with silica gel column to get 28 mg colorless oil (60%). <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 9.77 (d, <i>J</i> = 1.3 Hz, 1H), 7.47 (d, <i>J</i> = 10.7 Hz, 2H), 7.43–7.35 (m, 2H), 2.76–2.59 (m, 2H), 2.48 (td, <i>J</i> = 7.2, 1.3 Hz, 2H), 2.06–1.81 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 201.72, 142.82, 133.12, 132.01, 130.05, 129.38, 118.98, 112.57, 42.96, 34.60, 23.31.</div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> <i>tert</i>-Butyl (<i>S</i>)-4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(4-(3-cyanophenyl)butyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)butanoate (<b>12a</b>)</h3><div class="NLM_p last">Compound <b>12a</b> was synthesized according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.33 (s, 1H), 7.90 (s, 1H), 7.49–7.41 (m, 2H), 7.40–7.31 (m, 2H), 6.04 (d, <i>J</i> = 1.6 Hz, 1H), 5.69 (s, 2H), 5.58–5.46 (m, 2H), 4.98 (s, 1H), 4.32 (s, 1H), 4.20–4.10 (m, 1H), 2.78–2.69 (m, 1H), 2.56 (t, <i>J</i> = 7.7 Hz, 4H), 2.49–2.33 (m, 3H), 1.91 (s, 1H), 1.82–1.69 (m, 3H), 1.60 (s, 3H), 1.43 (s, 9H), 1.42 (s, 9H), 1.39 (s, 3H).</div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> <i>tert</i>-Butyl (<i>S</i>)-5-((((3a<i>R</i>,4<i>R</i>,6<i>S</i>,6a<i>R</i>)-6-(6-Amino-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)(4-(3-cyanophenyl)butyl)amino)-2-((<i>tert</i>-butoxycarbonyl)amino)pentanoate (<b>12b</b>)</h3><div class="NLM_p last">Compound <b>12b</b> was prepared according to the general procedure D. <sup>1</sup>H NMR (500 MHz, chloroform-<i>d</i>): δ 8.33 (s, 1H), 7.92 (s, 1H), 7.48–7.41 (m, 2H), 7.39–7.32 (m, 2H), 6.05 (s, 1H), 5.80 (s, 2H), 5.56–5.47 (m, 1H), 5.20 (d, <i>J</i> = 8.3 Hz, 1H), 4.98–4.88 (m, 1H), 4.31 (s, 1H), 4.17–4.12 (m, 1H), 2.67–2.53 (m, 4H), 2.46–2.35 (m, 4H), 1.72–1.68 (m, 1H), 1.59 (s, 3H), 1.58–1.47 (m, 5H), 1.44–1.38 (m, 23H).</div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Protein Expression and Purification</h3><div class="NLM_p last">Expression and purification of full-length hNNMT wild type and triple mutant K100A E101A E103A (hNNMTtm) were performed, as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Briefly, full-length hNNMT (amino acids 1–270) cloned in pET28a(+) with an N-terminal TEV cleavage site was synthesized by Genscript for biochemical assays. The full-length hNNMTtm cloned in pET28a-LIC obtained from Addgene (#40734) was used for crystallization. Protein was expressed in <i>Escherichia coli</i> BL21-CodonPlus(DE3)-RIPL competent cells and induced by 0.3 mM isopropyl-D-1-thiogalactopyranoside at 16 °C for 20 h. Harvested cells were resuspended in ten volumes of 50 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> (pH = 7.4) with 500 mM NaCl, 25 mM imidazole, 5 mM β-mercaptoethanol, and 100 μM PMSF, then lysed through sonication (Qsonica Q55 cell disruptor) on ice at 80% power using 5–7 cycles of 30 s pulse, 30 s rest. The cell lysate was centrifuged at 25 000<i>g</i> for 30 min at 4 °C, and the supernatant was then applied to two 1 mL HiTrap FF Ni-NTA columns connected in series on a GE AKTA Prime purification system using 50 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> (pH = 7.4) with 500 mM NaCl and 0.5 mM TCEP, washed and eluted using a step gradient of imidazole (0.025, 0.05, 0.1, 0.25, and 0.5 M). The peak fractions were verified by SDS-PAGE analysis, and the purest fractions were combined. For enzymatic assays, combined His-NNMT were dialyzed in the dialysis buffer (25 mM Tris, pH = 7.5, 150 mM NaCl, 50 mM KCl) and concentrated to 1.5 mg/mL for biochemical assays. For crystallography study, the sample was then applied to an S200 Sephacryl HR size exclusion column (26/100, GE) using 50 mM Tris-HCl, pH = 8.0, 100 mM NaCl, 0.5 mM TCEP, 5% glycerol. Fractions containing purest NNMT were combined, concentrated to 0.8 mg/mL, and supplemented with additional TCEP to a final concentration of 1 mM. G9a, SETD7, PRMT1, <i>Tb</i>PRMT7, NTMT1, PNMT, and INMT were expressed and purified as described before.<a onclick="showRef(event, 'ref28 ref34 ref35 ref36 ref37 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref28 ref34 ref35 ref36 ref37 ref41 ref42">(28,34−37,41,42)</a></div></div><div id="sec4_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Docking Studies with Glide</h3><div class="NLM_p last">For the crystal structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>, the incomplete side-chains were reconstructed by Prime. Then, the solvent was removed, and the structure was minimized. The SAH binding site was defined as docking site with 10 Å. The 3D structure for compounds was prepared in Maestro from the 2D structure and parameterized using the OPLS3 force-field. Rigid-receptor/flexible-ligand docking was performed using the extra-precision mode in Glide. The pose with the highest docking score was exported and was analyzed by PyMOL.</div></div><div id="sec4_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> NNMT Biochemical Assays and Enzyme Kinetics Study</h3><div class="NLM_p last">The fluorescence-based SAHH-coupled assay was applied to study the IC<sub>50</sub> values of compounds to monitor the production of SAH. The assay was performed under the following conditions in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM NNMT, 10 μM AdoMet, and 10 μM ThioGlo4. After 10 min incubation with inhibitors at 37 °C, reactions were initiated by the addition of 10 μM nicotinamide (<i>K</i><sub>m</sub> value). The fluorescence signal was monitored on a BMG CLARIOstar microplate reader with excitation 400 nm and emission 465 nm. Data were processed by using GraphPad Prism software 7.0.</div></div><div id="sec4_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Time-Dependent Inhibition Study</h3><div class="NLM_p last">The fluorescence-based SAHH-coupled assay was applied to this study. The assay was performed under the following conditions in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM NNMT, and 10 μM AdoMet. The inhibitors were added at concentrations ranging from 0.15 nM to 10 μM. After incubation for 30, 60, 90, and 120 min, reactions were initiated by the addition of 10 μM nicotinamide and 10 μM Thioglo4. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation 400 nm and emission 465 nm. Data were processed by using GraphPad Prism software 7.0 with Morrison’s quadratic equation.</div></div><div id="sec4_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Rapid Dilution Study</h3><div class="NLM_p last">10 μM NNMT and compound <b>LL320</b> were preincubated for 30 min. Then, 1 μL preincubation mixture was added into 99 μL reaction mixture: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 10 μM ThioGlo4, 10 μM AdoMet, and 10 μM nicotinamide. The fluorescence signal was monitored on a BMG CLARIOstar microplate reader with excitation 400 nm and emission 465 nm. Data were processed by using GraphPad Prism software 7.0.</div></div><div id="sec4_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Selectivity Assays</h3><div class="NLM_p">A fluorescence-based SAHH-coupled assay was applied to study the effect of the compound on methyltransferase activity of PNMT, INMT, G9a, SETD7, PRMT1, <i>Tb</i>PRMT7, and NTMT1. For PNMT, the assay was performed in a final well volume of 40 μL: 25 mM potassium phosphate buffer (pH = 7.6), 1 mM EDTA, 2 mM MgCl<sub>2</sub>, 0.01% Triton X-100, 5 μM SAHH, 0.2 μM PNMT, 3 μM AdoMet, and 10 μM ThioGlo4. The inhibitor was added at four compound concentrations: 3.7, 11, 33, and 100 μM. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 40 μM norepinephrine. For INMT, the assay was performed in a final well volume of 40 μL: 25 mM HEPES (pH = 7.5), 5 μM SAHH, 0.2 μM INMT, 30 μM AdoMet, and 10 μM ThioGlo4. The inhibitor was added at four compound concentrations: 3.7, 11, 33, and 100 μM. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 2.5 mM tryptamine. For G9a, the assay was performed in a final well volume of 40 μL: 25 mM potassium phosphate buffer (pH = 7.6), 1 mM EDTA, 2 mM MgCl<sub>2</sub>, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM His-G9a, 10 μM AdoMet, and 10 μM ThioGlo4. The inhibitor was added at four compound concentrations: 3.7, 11, 33, and 100 μM. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 4 μM H3-21 peptide. For NTMT1, the assay was performed in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.1 μM NTMT1, 3 μM AdoMet, and 10 μM ThioGlo4. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 0.5 μM GPKRIA peptide. All experiments were determined in duplicate. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation 400 nm and emission 465 nm.</div><div class="NLM_p">For SETD7, the assay was performed in a final well volume of 40 μL: 25 mM potassium phosphate buffer (pH = 7.6), 0.01% Triton X-100, 5 μM SAHH, 1 μM His-SETD7, 2 μM AdoMet, and 15 μM ThioGlo1. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 90 μM H3-21 peptide. For PRMT1, the assay was also performed in a final well volume of 40 μL: 2.5 mM HEPES (pH = 7.0), 25 mM NaCl, 25 μM EDTA, 50 μM TCEP, 0.01% Triton X-100, 5 μM SAHH, 0.2 μM PRMT1, 10 μM AdoMet, and 15 μM ThioGlo1. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 5 μM H4-21 peptide. For <i>Tb</i>PRMT7, the assay was performed in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 5 μM SAHH, 0.2 μM <i>Tb</i>PRMT7, 3 μM AdoMet, and 15 μM ThioGlo1. After 10 min incubation with the inhibitor, reactions were initiated by the addition of 60 μM H4-21 peptide. All experiments were determined in duplicate. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation 380 nm and emission 505 nm.</div><div class="NLM_p last">The compound was evaluated for its effect on SAHH activity, the coupled enzyme. The assay was performed in a final well volume of 40 μL: 25 mM Tris (pH = 7.5), 50 mM KCl, 0.01% Triton X-100, 0.1 μM SAHH, and 15 μM ThioGlo1. After 10 min incubation with the compound, 0.5 μM SAH was added to initiate the reactions. The experiment was determined in duplicate. Fluorescence was monitored on a BMG CLARIOstar microplate reader with excitation 380 nm and emission 505 nm.</div></div><div id="sec4_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Cocrystallization and Structure Determination</h3><div class="NLM_p last">For cocrystallization, 20 mg/mL hNNMT triple mutant was mixed with either LL319 or LL320 at a 1:4 molar ratio in 50 mM Tris-HCl, pH = 8.0, with 0.5 mM TCEP and 5% glycerol, incubated for 1 h at 4 °C. Broad matrix crystallization screening was performed using a Mosquito-LCP high throughput crystallization robot (TTP LabTech) using a hanging-drop vapor diffusion method, incubated at 20 °C, and crystals harvested directly from the 96-well crystallization plates. Crystals containing LL319 or LL320 were grown in 0.1 M bicine, pH 9.0, and 1.6 M ammonium sulfate (space group P1); all crystals were harvested directly from the drop and flash cooled into liquid nitrogen. Data were collected on single crystals at 12.0 keV at the GM/CA ID-D beamline at the Advanced Photon Source, Argonne National Laboratory. The data were processed using the on-site automated pipeline with AutoProc and/or manually with HKL2000 and molecular replacement performed with Phaser-MR (PHENIX).<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> All model building was performed using COOT, and subsequent refinement was done using phenix.refine (PHENIX).<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Structure factors and model coordinates have been deposited in the Protein Data Bank with ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a> (LL319) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a> (LL320). Data collection parameters and refinement statistics are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">Table S1</a>. Structure-related figures were made with PyMOL (Schrödinger) and annotated and finalized with Adobe Photoshop and Illustrator.</div></div><div id="sec4_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Cell Permeability Evaluation by MALDI</h3><div class="NLM_p last">The colon cancer cell line HCT116 was cultured in Mccoy’s media supplemented with 10% fetal bovine serum and 1% antibiotic–antimycotic (Gibco). The cells were cultured in tissue culture dish (Falcon 353003). Cells were maintained in cell culture flasks until seeding into a 12 well tissue culture plate (Falcon 353047). Media was removed, and the cells were washed with DPBS (1 mL) twice followed by treating with TrypLE Express (1 mL) into a 100 × 20 mm culture flask. The reaction was quenched by the addition of 4 mL of media, and the cells were counted. Cells were seeded into a 12 well tissue culture plate (Falcon 353047) at a density of 0.1 × 10<sup>6</sup> cells/mL and incubated overnight at 37 °C, 5% CO<sub>2</sub>, and 95% humidity, with the lid on. They were then treated with the inhibitors at different concentrations, and incubation was continued for the specified time. TAT peptide (GRKKRRQRRR-NH<sub>2</sub>) was also incubated as a control experiment. The media was removed, and the cells were washed with 1× PBS three times to remove any residual compound or peptide attached to the cell surface. Then, 100 μL of 1X PBS was added, and the cells were snap frozen in liquid nitrogen twice. The cell lysate was then analyzed with MALDI using DHB matrix to identify the presence of the compound inside the cell.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01255" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01255?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01255</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">General Procedures for the Synthesis of compound <b>S3</b>; NMR spectra of compounds <b>S3</b>, <b>6a–i</b>, <b>9a–d</b>, and <b>12a–b</b>; NMR, MALDI-MS, and HPLC spectra of compounds <b>1a–j</b>, <b>1a*</b>, <b>2a–d</b>, <b>2a*</b> and <b>3a–b</b>; kinetic parameter determination for NNMT; rapid dilution assay for compound LL320 (<b>2a</b>); selectivity study of LL320 (<b>2a</b>); superimposed cocrystal structures; cell permeability evaluation of TAT peptide as the positive control; cell permeability evaluation of <b>1a*</b>; cell permeability evaluation of <b>2a*</b>; and crystallography data and refinement statistics (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf">jm9b01255_si_001.pdf (5.79 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_002.csv">jm9b01255_si_002.csv (2.09 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structures of human NNMT in the complex of <b>LL319</b> and <b>LL320</b> have been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01255" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center
for Cancer Research, Institute for Drug Discovery and , Purdue
University, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8fe7faeee1e8a2fdcffffafdebfaeaa1eaebfa"><span class="__cf_email__" data-cfemail="f49c81959a93d986b4848186908191da919081">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongxing Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center
for Cancer Research, Institute for Drug Discovery and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linjie Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center
for Cancer Research, Institute for Drug Discovery and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krystal Diaz</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center
for Cancer Research, Institute for Drug Discovery and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iredia D. Iyamu</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center
for Cancer Research, Institute for Drug Discovery and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ravi Yadav</span> - <span class="hlFld-Affiliation affiliation">Markey Center for Structural Biology,
Department of Biological Sciences and the Purdue Institute of Inflammation,
Immunology and Infectious Disease and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Noinaj</span> - <span class="hlFld-Affiliation affiliation">Markey Center for Structural Biology,
Department of Biological Sciences and the Purdue Institute of Inflammation,
Immunology and Infectious Disease and , Purdue
University, West Lafayette, Indiana 47907, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.C. and L.L. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>NIH grants R01GM117275 (R.H.), U01CA214649 (R.H.), K22 AI113078-02 (N.N.), 1R01GM127896-01 (N.N.), 1R01AI127793 (N.N.), and P30 CA023168 (Purdue University Center for Cancer Research).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04684" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04684" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We appreciate Dr. Ayad A. Al-Hamashi for his initial assistance with docking studies and Guangping Dong for initial cellular concentration evaluation. We thank Dr. Darrel L. Peterson for purification of SAHH for biochemical assays. Furthermore, we thank Dr. Lawrence M. Szewczuk for valuable discussion on tight-binding analysis. The authors acknowledge the support from NIH grants R01GM117275 (R.H.), U01CA214649 (R.H.), K22 AI113078-02 (N.N.), 1R01GM127896-01 (N.N.), 1R01AI127793 (N.N.), and P30 CA023168 (Purdue University Center for Cancer Research). We also thank supports from the Department of Medicinal Chemistry and Molecular Pharmacology (R.H.) and Department of Biological Sciences (N.N.) at Purdue University.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NNMT</td><td class="NLM_def"><p class="first last">nicotinamide <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteines</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last">SAH hydrolase</p></td></tr><tr><td class="NLM_term">PNMT</td><td class="NLM_def"><p class="first last">phenylethanolamine <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">INMT</td><td class="NLM_def"><p class="first last">indolethylamine <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">NTMT1</td><td class="NLM_def"><p class="first last">protein <i>N</i>-terminal methyltransferase 1</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aksoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szumlanski, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Human Liver Nicotinamide N-Methyltransferase. cDNA Cloning, Expression, and Biochemical Characterization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">14835</span>– <span class="NLM_lpage">14840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=8182091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=14835-14840&author=S.+Aksoyauthor=C.+L.+Szumlanskiauthor=R.+M.+Weinshilboum&title=Human+Liver+Nicotinamide+N-Methyltransferase.+cDNA+Cloning%2C+Expression%2C+and+Biochemical+Characterization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span></div><div class="casAuthors">Aksoy, Saime; Szumlanski, Carol L.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">14835-40</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide and other pyridines.  Human liver NNMT activity has a bimodal frequency distribution, an observation which raises the possibility that this enzyme might be regulated by a genetic polymorphism, a polymorphism that could have functional implications for individual differences in drug and xenobiotic toxicity.  As a first step toward testing this hypothesis, the authors set out to clone and express a cDNA for human liver NNMT.  Human liver NNMT was partially purified, photoaffinity-labeled, subjected to limited proteolysis, and partial amino acid sequence information was obtained.  The polymerase chain reaction was then used to amplify a 550-nucleotide sequence with human liver cDNA as template and primers designed on the basis of the NNMT amino acid sequence.  The 5'- and 3'-ends of a human liver NNMT cDNA were obtained by use of the rapid amplification of cDNA ends.  The combined use of these approaches resulted in the isolation of a human liver NNMT cDNA that was 969 nucleotides in length, with a 792-nucleotide open reading frame that encoded a 264-amino acid protein with a calcd. mol. mass of 29.6 kDa.  The human liver NNMT cDNA was transcribed in vitro and translated with a reticulocyte lysate system to yield a protein with a mol. mass of approx. 29 kDa that comigrated during SDS-polyacrylamide gel electrophoresis with photoaffinity-labeled human liver NNMT.  The NNMT cDNA was also subcloned into the eukaryotic expression vector p91023(B).  COS-1 cells transfected with this construct expressed a high level of NNMT enzymic activity, and the biochem. properties of this activity were similar to those of human liver NNMT.  Human liver NNMT and transfected COS-1 cell NNMT had apparent Km values for the two cosubstrates for the reaction, nicotinamide and S-adenosyl-L-methionine, of 0.43 and 0.38 mM and of 1.8 and 2.2 μM, resp.  IC50 values for the inhibition of NNMT by N1-methylnicotinamide were 60 and 30 μM for human liver and COS-1 cell-expressed NNMT, resp.  Cloning of a cDNA for human liver NNMT should make it possible to test the hypothesis that inheritance may play a role in the regulation of individual differences in human liver NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZRn3_QD5atbVg90H21EOLACvtfcHk0liFzwojFeCWbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D&md5=d5d9edf09cb329ecc01b49fd819fad47</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAksoy%26aufirst%3DS.%26aulast%3DSzumlanski%26aufirst%3DC.%2BL.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DHuman%2520Liver%2520Nicotinamide%2520N-Methyltransferase.%2520cDNA%2520Cloning%252C%2520Expression%252C%2520and%2520Biochemical%2520Characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D14835%26epage%3D14840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&author=P.+Pissios&title=Nicotinamide+N-Methyltransferase%3A+More+Than+a+Vitamin+B3+Clearance+Enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0lgMbCgkBCBECg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-Methyltransferase%253A%2520More%2520Than%2520a%2520Vitamin%2520B3%2520Clearance%2520Enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirinen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulinilkunnil, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhonen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Knockdown Protects against Diet-Induced Obesity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Gongauthor=Y.-c.+Wangauthor=Y.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-Methyltransferase+Knockdown+Protects+against+Diet-Induced+Obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0lho19aP6ZMofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.-c.%26aulast%3DCen%26aufirst%3DY.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Knockdown%2520Protects%2520against%2520Diet-Induced%2520Obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Interacts with Enzymes of the Methionine Cycle and Regulates Methyl Donor Metabolism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5775-5779&author=S.+Hongauthor=B.+Zhaiauthor=P.+Pissios&title=Nicotinamide+N-Methyltransferase+Interacts+with+Enzymes+of+the+Methionine+Cycle+and+Regulates+Methyl+Donor+Metabolism&doi=10.1021%2Facs.biochem.8b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span></div><div class="casAuthors">Hong, Shangyu; Zhai, Bo; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5775-5779</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Me donor balance is crit. for epigenetic regulation in cells and is maintained by the so-called methionine cycle proteins that regenerate S-adenosylmethionine (SAM), the universal Me donor, from homocysteine formed by the activity of methyltransferases.  Nnmt is a liver enzyme that methylates nicotinamide, but its role in regulating Me donor balance in the liver is unclear.  In this study, we assessed the effect of altered Nnmt expression on various aspects of Me donor metab. in the liver.  We found that Nnmt overexpression decreased SAM levels and the SAM/ S-adenosylhomocysteine (SAH) ratio both in vivo and in vitro.  Nnmt knockdown did not change Me donor balance in mouse primary hepatocytes but increased SAM levels and the SAM/SAH ratio when Gnmt, the dominantly expressed methyltransferase in liver, was simultaneously knocked down.  Paradoxically, expression of enzymically deficient Nnmt increased the SAM/SAH ratio, suggesting that Nnmt can regulate Me donor balance independent of its methyltransferase activity.  Proteomics anal. of Nnmt-interacting proteins in the liver identified Bhmt, Mat1a, and Ahcy, all components of the methionine cycle, and functional expts. showed that mutant Nnmt increased the level of remethylation of homocysteine to SAM.  In summary, we show that the function of Nnmt in hepatic Me donor balance is multifactorial.  On one hand, Nnmt decreases Me donor balance, consistent with its activity as a methyltransferase consuming Me donors.  On the other hand, by co-opting the enzymes of the methionine cycle, Nnmt aids the recycling of homocysteine to SAM for another round of methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD5fiz3fTxsrVg90H21EOLACvtfcHk0licqd36yYcf8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O&md5=2689739ce6ec8ef59b09af7a20cb06ce</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00561%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DB.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Interacts%2520with%2520Enzymes%2520of%2520the%2520Methionine%2520Cycle%2520and%2520Regulates%2520Methyl%2520Donor%2520Metabolism%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5775%26epage%3D5779%26doi%3D10.1021%2Facs.biochem.8b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Enhances the Capacity of Tumorigenesis Associated with the Promotion of Cell Cycle Progression in Human Colorectal Cancer Cells</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>564</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2014.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.abb.2014.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25201588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=564&publication_year=2014&pages=52-66&author=X.+Xieauthor=H.+Yuauthor=Y.+Wangauthor=Y.+Zhouauthor=G.+Liauthor=Z.+Ruanauthor=F.+Liauthor=X.+Wangauthor=H.+Liuauthor=J.+Zhang&title=Nicotinamide+N-Methyltransferase+Enhances+the+Capacity+of+Tumorigenesis+Associated+with+the+Promotion+of+Cell+Cycle+Progression+in+Human+Colorectal+Cancer+Cells&doi=10.1016%2Fj.abb.2014.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase enhances the capacity of tumorigenesis associated with the promotion of cell cycle progression in human colorectal cancer cells</span></div><div class="casAuthors">Xie, Xinyou; Yu, Haitao; Wang, Yanzhong; Zhou, Yanwen; Li, Guiling; Ruan, Zhi; Li, Fengying; Wang, Xiuhong; Liu, Huixing; Zhang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">564</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-66</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT), an enzyme involved in the biotransformation and detoxification of many drugs and xenobiotic compds., has been found to be overexpressed in several malignancies, including colorectal cancer.  However, the biol. function of NNMT and the related mechanisms in colorectal cancer have not been elucidated.  In the present study, we investigated the effects of NNMT on tumorigenesis by overexpressing NNMT in the human colorectal cancer cells line SW480 which lacks constitutive NNMT expression, and downregulating NNMT expression in HT-29 cells, which exhibit high endogenous expression of NNMT.  We found that NNMT significantly accelerates cell proliferation, enhances colony formation in vitro and tumorigenicity in mice; it also inhibits apoptosis, promotes cell cycle progression, increases ATP and 1-methylnicotinamide level and decreases ROS level.  We also showed that 1-methylnicotinamide accelerates cell growth, inhibits apoptosis, promotes cell cycle progression, attenuates ROS prodn. and increases ATP level.  Our results indicate that NNMT enhances the capacity of tumorigenesis assocd. with the inhibition of cell apoptosis and the promotion of cell cycle progression in human colorectal cancer cells and the 1-methylnicotinamide increased by NNMT mediates the cellular effects of NNMT in cells.  NNMT may play a vital role in energy balance and ROS induction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7twTbySzqd7Vg90H21EOLACvtfcHk0ljsMcUPaBxhkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltbnO&md5=5782d0bee67209263ee9e592608eccdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2014.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2014.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DRuan%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Enhances%2520the%2520Capacity%2520of%2520Tumorigenesis%2520Associated%2520with%2520the%2520Promotion%2520of%2520Cell%2520Cycle%2520Progression%2520in%2520Human%2520Colorectal%2520Cancer%2520Cells%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2014%26volume%3D564%26spage%3D52%26epage%3D66%26doi%3D10.1016%2Fj.abb.2014.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palanichamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumoorthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litzenberg, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span> <span> </span><span class="NLM_article-title">NNMT Silencing Activates Tumor Suppressor PP2A, Inactivates Oncogenic STKs, and Inhibits Tumor Forming Ability</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-1323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1158%2F1078-0432.ccr-16-1323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=27810903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2325-2334&author=K.+Palanichamyauthor=S.+Kanjiauthor=N.+Gordonauthor=K.+Thirumoorthyauthor=J.+R.+Jacobauthor=K.+T.+Litzenbergauthor=D.+Patelauthor=A.+Chakravarti&title=NNMT+Silencing+Activates+Tumor+Suppressor+PP2A%2C+Inactivates+Oncogenic+STKs%2C+and+Inhibits+Tumor+Forming+Ability&doi=10.1158%2F1078-0432.ccr-16-1323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT Silencing Activates Tumor Suppressor PP2A, Inactivates Oncogenic STKs, and Inhibits Tumor Forming Ability</span></div><div class="casAuthors">Palanichamy, Kamalakannan; Kanji, Suman; Gordon, Nicolaus; Thirumoorthy, Krishnan; Jacob, John R.; Litzenberg, Kevin T.; Patel, Disha; Chakravarti, Arnab</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-2334</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: To identify potential mol. hubs that regulate oncogenic kinases and target them to improve treatment outcomes for glioblastoma patients.  Exptl. Design: Data mining of The Cancer Genome Atlas datasets identified nicotinamide-N-Me transferase (NNMT) as a prognostic marker for glioblastoma, an enzyme linked to the reorganization of the methylome.  We tested our hypothesis that NNMT plays a crucial role by modulating protein methylation, leading to inactivation of tumor suppressors and activation of oncogenes.  Further expts. were performed to understand the underlying biochem. mechanisms using glioblastoma patient samples, established, primary, and isogenic cells.  Results: We demonstrate that NNMT outcompetes leucine carboxyl Me transferase 1 (LCMT1) for Me transfer from principal Me donor SAM in biol. systems.  Inhibiting NNMT increased the availability of Me groups for LCMT1 to methylate PP2A, resulting in the inhibition of oncogenic serine/threonine kinases (STK).  Further, NNMT inhibition retained the radiosensitizer nicotinamide and enhanced radiation sensitivity.  We have provided the biochem. rationale of how NNMT plays a vital role in inhibiting tumor suppressor PP2A while concomitantly activating STKs.  Conclusions: We report the intricate novel mechanism in which NNMT inhibits tumor suppressor PP2A by reorganizing the methylome both at epigenome and proteome levels and concomitantly activating prosurvival STKs.  In glioblastoma tumors with NNMT expression, activation of PP2A can be accomplished by FDA approved perphenazine (PPZ), which is currently used to treat mood disorders such as schizophrenia, bipolar disorder, etc.  This study forms a foundation for further glioblastoma clin. trials using PPZ with std. of care treatment.  Clin Cancer Res; 23(9); 2325-34. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqIwv0424CHbVg90H21EOLACvtfcHk0ljD-nki7VuI4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFehuro%253D&md5=dafd3575d75a59c87f691b37e8dbcd8f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-1323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-1323%26sid%3Dliteratum%253Aachs%26aulast%3DPalanichamy%26aufirst%3DK.%26aulast%3DKanji%26aufirst%3DS.%26aulast%3DGordon%26aufirst%3DN.%26aulast%3DThirumoorthy%26aufirst%3DK.%26aulast%3DJacob%26aufirst%3DJ.%2BR.%26aulast%3DLitzenberg%26aufirst%3DK.%2BT.%26aulast%3DPatel%26aufirst%3DD.%26aulast%3DChakravarti%26aufirst%3DA.%26atitle%3DNNMT%2520Silencing%2520Activates%2520Tumor%2520Suppressor%2520PP2A%252C%2520Inactivates%2520Oncogenic%2520STKs%252C%2520and%2520Inhibits%2520Tumor%2520Forming%2520Ability%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2325%26epage%3D2334%26doi%3D10.1158%2F1078-0432.ccr-16-1323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coscia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chryplewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tienda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahotko, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaženović, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastra, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perets, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span> <span> </span><span class="NLM_article-title">Proteomics Reveals NNMT as a Master Metabolic Regulator of Cancer-Associated Fibroblasts</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1173-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fs41586-019-1173-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=31043742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=723-728&author=M.+A.+Eckertauthor=F.+Cosciaauthor=A.+Chryplewiczauthor=J.+W.+Changauthor=K.+M.+Hernandezauthor=S.+Panauthor=S.+M.+Tiendaauthor=D.+A.+Nahotkoauthor=G.+Liauthor=I.+Bla%C5%BEenovi%C4%87author=R.+R.+Lastraauthor=M.+Curtisauthor=S.+D.+Yamadaauthor=R.+Peretsauthor=S.+M.+McGregorauthor=J.+Andradeauthor=O.+Fiehnauthor=R.+E.+Moelleringauthor=M.+Mannauthor=E.+Lengyel&title=Proteomics+Reveals+NNMT+as+a+Master+Metabolic+Regulator+of+Cancer-Associated+Fibroblasts&doi=10.1038%2Fs41586-019-1173-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span></div><div class="casAuthors">Eckert, Mark A.; Coscia, Fabian; Chryplewicz, Agnieszka; Chang, Jae Won; Hernandez, Kyle M.; Pan, Shawn; Tienda, Samantha M.; Nahotko, Dominik A.; Li, Gang; Blazenovic, Ivana; Lastra, Ricardo R.; Curtis, Marion; Yamada, S. Diane; Perets, Ruth; McGregor, Stephanie M.; Andrade, Jorge; Fiehn, Oliver; Moellering, Raymond E.; Mann, Matthias; Lengyel, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7758</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">High-grade serous carcinoma has a poor prognosis, owing primarily to its early dissemination throughout the abdominal cavity.  Genomic and proteomic approaches have provided snapshots of the proteogenomics of ovarian cancer1,2, but a systematic examn. of both the tumor and stromal compartments is crit. in understanding ovarian cancer metastasis.  Here we develop a label-free proteomic workflow to analyze as few as 5000 formalin-fixed, paraffin-embedded cells microdissected from each compartment.  The tumor proteome was stable during progression from in situ lesions to metastatic disease; however, the metastasis-assocd. stroma was characterized by a highly conserved proteomic signature, prominently including the methyltransferase nicotinamide N-methyltransferase (NNMT) and several of the proteins that it regulates.  Stromal NNMT expression was necessary and sufficient for functional aspects of the cancer-assocd. fibroblast (CAF) phenotype, including the expression of CAF markers and the secretion of cytokines and oncogenic extracellular matrix.  Stromal NNMT expression supported ovarian cancer migration, proliferation and in vivo growth and metastasis.  Expression of NNMT in CAFs led to depletion of S-adenosyl methionine and redn. in histone methylation assocd. with widespread gene expression changes in the tumor stroma.  This work supports the use of ultra-low-input proteomics to identify candidate drivers of disease phenotypes.  NNMT is a central, metabolic regulator of CAF differentiation and cancer progression in the stroma that may be therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlL_GCBb1LSbVg90H21EOLACvtfcHk0ljTrJVOHq0aRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D&md5=43dbbcee11b5d21c5fb77f5b8ff02701</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1173-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1173-8%26sid%3Dliteratum%253Aachs%26aulast%3DEckert%26aufirst%3DM.%2BA.%26aulast%3DCoscia%26aufirst%3DF.%26aulast%3DChryplewicz%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHernandez%26aufirst%3DK.%2BM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DTienda%26aufirst%3DS.%2BM.%26aulast%3DNahotko%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DBla%25C5%25BEenovi%25C4%2587%26aufirst%3DI.%26aulast%3DLastra%26aufirst%3DR.%2BR.%26aulast%3DCurtis%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DS.%2BD.%26aulast%3DPerets%26aufirst%3DR.%26aulast%3DMcGregor%26aufirst%3DS.%2BM.%26aulast%3DAndrade%26aufirst%3DJ.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DLengyel%26aufirst%3DE.%26atitle%3DProteomics%2520Reveals%2520NNMT%2520as%2520a%2520Master%2520Metabolic%2520Regulator%2520of%2520Cancer-Associated%2520Fibroblasts%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D723%26epage%3D728%26doi%3D10.1038%2Fs41586-019-1173-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhl, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">NNMT Promotes Epigenetic Remodeling in Cancer by Creating a Metabolic Methylation Sink</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+Promotes+Epigenetic+Remodeling+in+Cancer+by+Creating+a+Metabolic+Methylation+Sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0lj0jALQ15BS6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520Promotes%2520Epigenetic%2520Remodeling%2520in%2520Cancer%2520by%2520Creating%2520a%2520Metabolic%2520Methylation%2520Sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guidi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zizzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliante, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase in Non-Small Cell Lung Cancer: Promising Results for Targeted Anti-Cancer Therapy</span>. <i>Cell Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">865</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1007/s12013-013-9574-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1007%2Fs12013-013-9574-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=23532607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKjtbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2013&pages=865-873&author=D.+Sartiniauthor=S.+Morgantiauthor=E.+Guidiauthor=C.+Rubiniauthor=A.+Zizziauthor=R.+Giulianteauthor=V.+Pozziauthor=M.+Emanuelli&title=Nicotinamide+N-Methyltransferase+in+Non-Small+Cell+Lung+Cancer%3A+Promising+Results+for+Targeted+Anti-Cancer+Therapy&doi=10.1007%2Fs12013-013-9574-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase in Non-small Cell Lung Cancer: Promising Results for Targeted Anti-cancer Therapy</span></div><div class="casAuthors">Sartini, Davide; Morganti, Stefano; Guidi, Elena; Rubini, Corrado; Zizzi, Antonio; Giuliante, Rachela; Pozzi, Valentina; Emanuelli, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">865-873</span>CODEN:
                <span class="NLM_cas:coden">CBBIFV</span>;
        ISSN:<span class="NLM_cas:issn">1085-9195</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Lung cancer, predominantly non-small cell lung cancer (NSCLC), is currently the most common cause of malignancy-related death in the world.  Despite advances in both detection and treatment, its incidence rate is still increasing.  Therefore, effective strategies for early detection as well as mol. therapeutic targets are urgently needed.  We focused on the enzyme nicotinamide N-methyltransferase (NNMT).  NNMT expression levels were investigated in tumor, tumor-adjacent, and surrounding tissue samples of 25 patients with NSCLC by Real-Time PCR, Western blot anal., and catalytic activity assay.  NNMT enzyme activity in NSCLC was then correlated with clinicopathol. characteristics.  Results obtained showed NNMT upregulation (mRNA and protein) in tumor compared with both tumor-adjacent and surrounding tissue.  Moreover, NSCLC displayed significantly higher activity levels than those detd. in both tumor-adjacent and surrounding tissue.  Interestingly, both tumor-adjacent and surrounding tissue samples of unfavorable cases (N+) seem to display higher activity levels than those of favorable NSCLCs (N0).  The present work shows a marked increase of NNMT enzyme activity in NSCLC and suggests that normal-looking tissue of unfavorable cases seems to change toward cancer.  Further studies may establish whether NNMT could represent a target for an effective anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi37Ur7tl8Y7Vg90H21EOLACvtfcHk0lgLE9K1_7Ecmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKjtbnJ&md5=7fabae2de5172ba0611f4a995e85132f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs12013-013-9574-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12013-013-9574-z%26sid%3Dliteratum%253Aachs%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DMorganti%26aufirst%3DS.%26aulast%3DGuidi%26aufirst%3DE.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DZizzi%26aufirst%3DA.%26aulast%3DGiuliante%26aufirst%3DR.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%253A%2520Promising%2520Results%2520for%2520Targeted%2520Anti-Cancer%2520Therapy%26jtitle%3DCell%2520Biochem.%2520Biophys.%26date%3D2013%26volume%3D67%26spage%3D865%26epage%3D873%26doi%3D10.1007%2Fs12013-013-9574-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtake, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurosumi, M.</span></span> <span> </span><span class="NLM_article-title">Stat3 Up-Regulates Expression of Nicotinamide N-Methyltransferase in Human Cancer Cells</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1007/s00432-007-0318-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1007%2Fs00432-007-0318-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=17922140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsF2ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2008&pages=551-559&author=M.+Tomidaauthor=H.+Ohtakeauthor=T.+Yokotaauthor=Y.+Kobayashiauthor=M.+Kurosumi&title=Stat3+Up-Regulates+Expression+of+Nicotinamide+N-Methyltransferase+in+Human+Cancer+Cells&doi=10.1007%2Fs00432-007-0318-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 up-regulates expression of nicotinamide N-methyltransferase in human cancer cells</span></div><div class="casAuthors">Tomida, Mikio; Ohtake, Hideki; Yokota, Takashi; Kobayashi, Yasuhito; Kurosumi, Masafumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">551-559</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: To discover new mol. targets for cancer therapy and diagnosis, we surveyed signal transducers and activators of transcription 3 (Stat3)-regulated genes, because constitutive activation of Stat3 is assocd. with a wide variety of human malignancies.  Methods: We investigated the Stat3-regulated genes in 293 cells with cDNA microarray anal. and found that Nicotinamide N-methyltransferase (NNMT) was induced on stimulation of the cells with leukemia inhibitory factor.  We examd. the expression of NNMT in several types of cancer cells by real-time quant. RT-PCR.  To examine the role of Stat3, Hep-G2 hepatocellular carcinoma cells were transfected with NNMT promoter-luciferase reporter construct together with conditionally active Stat3 (Stat3ER) or dominant-neg. Stat3 expression vector and NNMT promoter activity was detd.  The expression of NNMT and activated Stat3 in 88 colon cancer tissues and 17 normal colon tissues was examd. with immunohistochem. anal.  Results: In Hep-G2 cells and SW480 colon cancer cells, NNMT expression increased on stimulation of the cells with interleukin 6.  NNMT promoter activity in Hep-G2 cells was dependent on the activation of Stat3.  MDA-MB-468 breast cancer cells and HT29 colon cancer cells expressed constitutively a high level of NNMT.  Treatment of these cells with Stat3 siRNA or curcumin, which inhibited Stat3 phosphorylation, resulted in redn. of the NNMT level.  We found a correlation between the expression of NNMT and activated Stat3 (P < 0.001) in the colon cancer tissues.  Conclusion: NNMT is a novel Stat3-regulated gene.  Its expression is enhanced with the activation of Stat3 in colon cancer tissues.  NNMT may be a potential candidate for a tumor marker of various kinds of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdYhc8dVgoOLVg90H21EOLACvtfcHk0ljllBGOEJpaHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsF2ju7Y%253D&md5=db842717175e713e708cfb5a086cd424</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00432-007-0318-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-007-0318-6%26sid%3Dliteratum%253Aachs%26aulast%3DTomida%26aufirst%3DM.%26aulast%3DOhtake%26aufirst%3DH.%26aulast%3DYokota%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DY.%26aulast%3DKurosumi%26aufirst%3DM.%26atitle%3DStat3%2520Up-Regulates%2520Expression%2520of%2520Nicotinamide%2520N-Methyltransferase%2520in%2520Human%2520Cancer%2520Cells%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2008%26volume%3D134%26spage%3D551%26epage%3D559%26doi%3D10.1007%2Fs00432-007-0318-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span> <span> </span><span class="NLM_article-title">Down-Regulation of Nicotinamide N-Methyltransferase Induces Apoptosis in Human Breast Cancer Cells via the Mitochondria-Mediated Pathway</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e89202</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0089202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1371%2Fjournal.pone.0089202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=24558488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjs73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=J.+Zhangauthor=Y.+Wangauthor=G.+Liauthor=H.+Yuauthor=X.+Xie&title=Down-Regulation+of+Nicotinamide+N-Methyltransferase+Induces+Apoptosis+in+Human+Breast+Cancer+Cells+via+the+Mitochondria-Mediated+Pathway&doi=10.1371%2Fjournal.pone.0089202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulation of nicotinamide N-methyltransferase induces apoptosis in human breast cancer cells via the mitochondria-mediated pathway</span></div><div class="casAuthors">Zhang, Jun; Wang, Yanzhong; Li, Guiling; Yu, Haitao; Xie, Xinyou</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e89202/1-e89202/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been found involved in cell proliferation of several malignancies.  However, the functional role of NNMT in breast cancer has not been elucidated.  In the present study, we showed that NNMT was selectively expressed in some breast cancer cell lines, down-regulation of NNMT expression in Bcap-37 and MDA-MB-231 cell lines by NNMT shRNA significantly inhibited cell growth in vitro, decreased tumorigenicity in mice and induced apoptosis.  The silencing reciprocal effect of NNMT was confirmed by over-expressing NNMT in the MCF-7 and SK-BR-3 breast cancer cell lines which lack constitutive expression of NNMT.  In addn., down-regulation of NNMT expression resulted in reducing expression of Bcl-2 and Bcl-xL, up-regulation of Bax, Puma, cleaved caspase-9, cleaved caspase-3 and cleaved PARP, increasing reactive oxygen species prodn. and release of cytochrome c from mitochondria, and decreasing the phosphorylation of Akt and ERK1/2.  These data suggest that down-regulation of NNMT induces apoptosis via the mitochondria-mediated pathway in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw6YRvrhclrVg90H21EOLACvtfcHk0lguEVYNv5Cn8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjs73O&md5=ba37d01dcc59df8ec842da9895bd3e57</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0089202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0089202%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26atitle%3DDown-Regulation%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Induces%2520Apoptosis%2520in%2520Human%2520Breast%2520Cancer%2520Cells%2520via%2520the%2520Mitochondria-Mediated%2520Pathway%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0089202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateuszuk, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khomich, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Słomińska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wójcik, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Łomnicka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwóźdź, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chłopicki, S.</span></span> <span> </span><span class="NLM_article-title">Activation of Nicotinamide N-Methyltrasferase and Increased Formation of 1-Methylnicotinamide (MNA) in Atherosclerosis</span>. <i>Pharmacol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/s1734-1140(09)70009-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fs1734-1140%2809%2970009-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=19307695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFCku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=76-85&author=%C5%81.+Mateuszukauthor=T.+I.+Khomichauthor=E.+S%C5%82omi%C5%84skaauthor=M.+Gajdaauthor=L.+W%C3%B3jcikauthor=M.+%C5%81omnickaauthor=P.+Gw%C3%B3%C5%BAd%C5%BAauthor=S.+Ch%C5%82opicki&title=Activation+of+Nicotinamide+N-Methyltrasferase+and+Increased+Formation+of+1-Methylnicotinamide+%28MNA%29+in+Atherosclerosis&doi=10.1016%2Fs1734-1140%2809%2970009-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of nicotinamide N-methyltrasferase and increased formation of 1-methylnicotinamide (MNA) in atherosclerosis</span></div><div class="casAuthors">Mateuszuk, Lukasz; Khomich, Tamara I.; Slominska, Ewa; Gajda, Mariusz; Wojcik, Luiza; Lomnicka, Magdalena; Gwozdz, Pawel; Chlopicki, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">PRHEDU</span>;
        ISSN:<span class="NLM_cas:issn">1734-1140</span>.
    
            (<span class="NLM_cas:orgname">Polish Academy of Sciences, Institute of Pharmacology</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NMMT) catalyzes the conversion of nicotinamide (NA) to 1-methylnicotinamide (MNA).  Recent studies have reported that exogenous MNA exerts anti-thrombotic and anti-inflammatory activity, suggesting that endogenous NMMT-derived MNA may play a biol. role in the cardiovascular system.  In the present study, we assayed changes in hepatic NNMT activity and MNA plasma levels along the progression of atherosclerosis in apoE/LDLR-/- mice, as compared to age-matched wild-type mice.  Atherosclerosis progression in apoE/LDLR-/- mice was quantified in aortic root, while hepatic NNMT activity and MNA plasma concns. were concomitantly measured in 2-, 3-, 4-, and 6-mo-old mice.  In apoE/LDLR-/- mice, atherosclerotic plaques developed in the aortic roots beginning at the age of 3 mo and gradually increased in size, macrophage content, and inflammation intensity over time, as detected by Oil-Red O staining, CD68 immunostaining, and in situ zymog. (MMP2/MMP9 activity).  Hepatic NNMT activity was upregulated approx. two-fold in apoE/LDLR-/- mice by the age of 2 mo, as compared to wild-type mice (1.03 ± 0.14 vs. 0.64 ± 0.23 pmol/min/mg, resp.).  MNA plasma concns. were also elevated approx. two-fold (0.30 ± 0.13 vs. 0.17 ± 0.04 μmol/l, resp.).  As atherosclerosis progressed, hepatic NMMT activity and MNA plasma concns. increased five-fold in 6-mo-old apoE/LDLR-/- mice at the stage of advanced atherosclerotic plaques (NMMT activity: 2.29 ± 0.34 pmol/min/mg, MNA concn.: 1.083 ± 0.33 μmol/l).  In summary, the present study demonstrated that the progression of vascular inflammation and atherosclerosis was assocd. with the upregulation of hepatic NNMT activity and subsequent increase in endogenous MNA plasma levels.  Given the anti-thrombotic and anti-inflammatory properties of exogenous MNA, robust activation of an endogenous NA-MNA pathway in atherosclerosis may play an important compensatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofiHh0pa0Le7Vg90H21EOLACvtfcHk0liMgN9TDm9CYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFCku77O&md5=5a8337af5d9fedd50445989fced30a88</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fs1734-1140%2809%2970009-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1734-1140%252809%252970009-x%26sid%3Dliteratum%253Aachs%26aulast%3DMateuszuk%26aufirst%3D%25C5%2581.%26aulast%3DKhomich%26aufirst%3DT.%2BI.%26aulast%3DS%25C5%2582omi%25C5%2584ska%26aufirst%3DE.%26aulast%3DGajda%26aufirst%3DM.%26aulast%3DW%25C3%25B3jcik%26aufirst%3DL.%26aulast%3D%25C5%2581omnicka%26aufirst%3DM.%26aulast%3DGw%25C3%25B3%25C5%25BAd%25C5%25BA%26aufirst%3DP.%26aulast%3DCh%25C5%2582opicki%26aufirst%3DS.%26atitle%3DActivation%2520of%2520Nicotinamide%2520N-Methyltrasferase%2520and%2520Increased%2520Formation%2520of%25201-Methylnicotinamide%2520%2528MNA%2529%2520in%2520Atherosclerosis%26jtitle%3DPharmacol.%2520Rep.%26date%3D2009%26volume%3D61%26spage%3D76%26epage%3D85%26doi%3D10.1016%2Fs1734-1140%2809%2970009-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasouli, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsden, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Increases Complex i Activity in SH-SY5Y Cells via Sirtuin 3</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2015.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.bbrc.2015.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26456643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=491-496&author=K.+Y.+Liuauthor=R.+J.+Mistryauthor=C.+A.+Aguirreauthor=E.+S.+Fasouliauthor=M.+G.+Thomasauthor=F.+Klamtauthor=D.+B.+Ramsdenauthor=R.+B.+Parsons&title=Nicotinamide+N-Methyltransferase+Increases+Complex+i+Activity+in+SH-SY5Y+Cells+via+Sirtuin+3&doi=10.1016%2Fj.bbrc.2015.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase increases complex I activity in SH-SY5Y cells via sirtuin 3</span></div><div class="casAuthors">Liu, Karolina Y.; Mistry, Rakhee J.; Aguirre, Carlos A.; Fasouli, Eirini S.; Thomas, Martin G.; Klamt, Fabio; Ramsden, David B.; Parsons, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT, E.C. 2.1.1.1) N-methylates nicotinamide to 1-methylnicotinamide.  We have previously shown that NNMT is significantly overexpressed in the brains of patients who have died of Parkinson's disease, and others have shown that NNMT is significantly overexpressed in a variety of diseases ranging from cancer to hepatic cirrhosis.  In vitro overexpression has revealed many cytoprotective effects of NNMT, in particular increased complex I activity and ATP synthesis.  Although this appears to be mediated by an increase in 1-methylnicotinamide prodn., the mol. mechanisms involved remain unclear.  In the present study, we have investigated the role that sirtuins 1, 2 and 3, class III DNA deacetylase enzymes known to regulate mitochondrial energy prodn. and cell cycle, have in mediating the effects of NNMT upon complex I activity.  Expression of NNMT in SH-SY5Y human neuroblastoma cells, which have no endogenous expression of NNMT, significantly increased the expression of all three sirtuins.  SiRNA-mediated silencing of sirtuin 3 expression decreased complex I activity in NNMT-expressing SH-SY5Y cells to that obsd. in wild-type SH-SY5Y, and significantly reduced cellular ATP content also.  These results demonstrate that sirtuin 3 is a key mediator of NNMT-induced complex I activity and ATP synthesis.  These results further reinforce a central role for NNMT in the regulation of energy homeostasis and provide further mechanistic insight into the consequences of enhanced NNMT expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq27Ko18UYX_bVg90H21EOLACvtfcHk0ljujl50TmDeeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqu7rJ&md5=919f548abd38d02a68681a3b72b30791</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2015.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2015.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%2BY.%26aulast%3DMistry%26aufirst%3DR.%2BJ.%26aulast%3DAguirre%26aufirst%3DC.%2BA.%26aulast%3DFasouli%26aufirst%3DE.%2BS.%26aulast%3DThomas%26aufirst%3DM.%2BG.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Increases%2520Complex%2520i%2520Activity%2520in%2520SH-SY5Y%2520Cells%2520via%2520Sirtuin%25203%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2015%26volume%3D467%26spage%3D491%26epage%3D496%26doi%3D10.1016%2Fj.bbrc.2015.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5015</span>– <span class="NLM_lpage">5028</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&author=H.+Neelakantanauthor=H.-Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-Activity+Relationship+for+Small+Molecule+Inhibitors+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0lig9IXBxwFhZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.-Y.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520Small%2520Molecule%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zope, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parveen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Inhibitor of Nicotinamide N-Methyltransferase for the Treatment of Metabolic Disorders</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3660</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22081-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fs41598-018-22081-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=29483571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=3660&author=A.+Kanntauthor=S.+Rajagopalauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=S.+Swaminathanauthor=M.+S.+Hallurauthor=R.+Kristamauthor=R.+Elvertauthor=J.+Czechauthor=A.+Pfenningerauthor=C.+Rudolphauthor=H.+Schreuderauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+Birudukotaauthor=S.+Shaikauthor=B.+R.+Zopeauthor=R.+R.+Burriauthor=N.+N.+Anandauthor=M.+K.+Thakurauthor=M.+Singhauthor=R.+Parveenauthor=S.+Kandanauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=S.+Rufauthor=S.+Dhakshinamoorthy&title=A+Small+Molecule+Inhibitor+of+Nicotinamide+N-Methyltransferase+for+the+Treatment+of+Metabolic+Disorders&doi=10.1038%2Fs41598-018-22081-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule inhibitor of Nicotinamide N-methyltransferase for the treatment of metabolic disorders</span></div><div class="casAuthors">Kannt Aimo; Elvert Ralf; Czech Jorg; Pfenninger Anja; Rudolph Christine; Schreuder Herman; Ruf Sven; Kannt Aimo; Rajagopal Sridharan; Kadnur Sanjay Venkatachalapathi; Suresh Juluri; Bhamidipati Ravi Kanth; Swaminathan Srinivasan; Hallur Mahanandeesha Siddappa; Kristam Rajendra; Chandrasekar Devaraj Venkatapura; Mane Vishal Subhash; Birudukota Swarnakumari; Shaik Shama; Zope Bharat Ravindra; Burri Raghunadha Reddy; Anand Niranjan Naranapura; Thakur Manish Kumar; Singh Manvi; Parveen Reejuana; Kandan Saravanan; Mullangi Ramesh; Yura Takeshi; Gosu Ramachandraiah; Dhakshinamoorthy Saravanakumar</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3660</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a cytosolic enzyme that catalyzes the transfer of a methyl group from the co-factor S-adenosyl-L-methionine (SAM) onto the substrate, nicotinamide (NA) to form 1-methyl-nicotinamide (MNA).  Higher NNMT expression and MNA concentrations have been associated with obesity and type-2 diabetes.  Here we report a small molecule analog of NA, JBSNF-000088, that inhibits NNMT activity, reduces MNA levels and drives insulin sensitization, glucose modulation and body weight reduction in animal models of metabolic disease.  In mice with high fat diet (HFD)-induced obesity, JBSNF-000088 treatment caused a reduction in body weight, improved insulin sensitivity and normalized glucose tolerance to the level of lean control mice.  These effects were not seen in NNMT knockout mice on HFD, confirming specificity of JBSNF-000088.  The compound also improved glucose handling in ob/ob and db/db mice albeit to a lesser extent and in the absence of weight loss.  Co-crystal structure analysis revealed the presence of the N-methylated product of JBSNF-000088 bound to the NNMT protein.  The N-methylated product was also detected in the plasma of mice treated with JBSNF-000088.  Hence, JBSNF-000088 may act as a slow-turnover substrate analog, driving the observed metabolic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdHZ3bviJkhgaK-_ZudimnfW6udTcc2eY6eiqCnqR7E7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D&md5=3f2306da18e0ab2a41a57f08ea9384fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22081-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22081-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DZope%26aufirst%3DB.%2BR.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26atitle%3DA%2520Small%2520Molecule%2520Inhibitor%2520of%2520Nicotinamide%2520N-Methyltransferase%2520for%2520the%2520Treatment%2520of%2520Metabolic%2520Disorders%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D3660%26doi%3D10.1038%2Fs41598-018-22081-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fast, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Development of a Suicide Inhibition-Based Protein Labeling Strategy for Nicotinamide N-Methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00211</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1Olt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=613-618&author=S.+Senauthor=S.+Mondalauthor=L.+Zhengauthor=A.+J.+Salingerauthor=W.+Fastauthor=E.+Weerapanaauthor=P.+R.+Thompson&title=Development+of+a+Suicide+Inhibition-Based+Protein+Labeling+Strategy+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facschembio.9b00211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Suicide Inhibition-Based Protein Labeling Strategy for Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Sen, Sudeshna; Mondal, Santanu; Zheng, Li; Salinger, Ari J.; Fast, Walter; Weerapana, Eranthie; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">613-618</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the S-adenosyl-L-methionine-dependent methylation of nicotinamide to form N-methylnicotinamide.  This enzyme detoxifies xenobiotics and regulates NAD+ biosynthesis.  Addnl., NNMT is overexpressed in various cancers.  Herein, we describe the first NNMT-targeted suicide substrates.  These compds., which include 4-chloropyridine and 4-chloronicotinamide, exploit the broad substrate scope of NNMT; methylation of the pyridine nitrogen enhances the electrophilicity of the C4 position, thereby promoting an arom. nucleophilic substitution by C159, a noncatalytic cysteine.  On the basis of this activity, we developed a suicide inhibition-based protein labeling strategy using an alkyne-substituted 4-chloropyridine that selectively labels NNMT in vitro and in cells.  In total, this study describes the first NNMT-directed activity-based probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBszUWf8y2FrVg90H21EOLACvtfcHk0lgviuVUpSVZYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1Olt7Y%253D&md5=330147d34dbe6ebac795255b594ae4ee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00211%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DS.%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DSalinger%26aufirst%3DA.%2BJ.%26aulast%3DFast%26aufirst%3DW.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DDevelopment%2520of%2520a%2520Suicide%2520Inhibition-Based%2520Protein%2520Labeling%2520Strategy%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D613%26epage%3D618%26doi%3D10.1021%2Facschembio.9b00211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibitors of Nicotinamide N-Methyltransferase (NNMT) Provide Evidence for Target Engagement Challenges in Situ</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.bmcl.2018.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=29731364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2682-2687&author=H.-Y.+Leeauthor=R.+M.+Suciuauthor=B.+D.+Horningauthor=E.+V.+Vinogradovaauthor=O.+A.+Ulanovskayaauthor=B.+F.+Cravatt&title=Covalent+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29+Provide+Evidence+for+Target+Engagement+Challenges+in+Situ&doi=10.1016%2Fj.bmcl.2018.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span></div><div class="casAuthors">Lee, Hsin-Yu; Suciu, Radu M.; Horning, Benjamin D.; Vinogradova, Ekaterina V.; Ulanovskaya, Olesya A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2682-2687</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide using S-adenosyl-L-methionine (SAM) as a Me donor and, through doing so, can modulate cellular methylation potential to impact diverse epigenetic processes.  NNMT has been implicated in a range of diseases, including cancer and metabolic disorders.  Potent, selective, and cell-active inhibitors would constitute valuable probes to study the biol. functions and therapeutic potential of NNMT.  The authors previously reported the discovery of electrophilic small mols. that inhibit NNMT by reacting with an active-site cysteine residue in the SAM-binding pocket.  Here, the authors have used activity-based protein profiling (ABPP)-guided medicinal chem. to optimize the potency and selectivity of NNMT inhibitors, culminating in the discovery of multiple alpha-chloroacetamide (αCA) compds. with sub-μM IC50 values in vitro and excellent proteomic selectivity in cell lysates.  However, these compds. showed much weaker inhibition of NNMT in cells, a feature that was not shared by off-targets of the αCAs.  The authors' results show the potential for developing potent and selective covalent inhibitors of NNMT, but also highlight challenges that may be faced in targeting this enzyme in cellular systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFjmALOSTRFbVg90H21EOLACvtfcHk0lht3MiZCeQmTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D&md5=9194d6ac5d1fd5a70683fb4af9cae74f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.-Y.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCovalent%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%2520Provide%2520Evidence%2520for%2520Target%2520Engagement%2520Challenges%2520in%2520Situ%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2682%26epage%3D2687%26doi%3D10.1016%2Fj.bmcl.2018.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Nicotinamide N-Methyltransferase Designed to Mimic the Methylation Reaction Transition State</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/c7ob01357d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1039%2Fc7ob01357d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+Nicotinamide+N-Methyltransferase+Designed+to+Mimic+the+Methylation+Reaction+Transition+State&doi=10.1039%2Fc7ob01357d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0li5OxsBoQDW0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fc7ob01357d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7ob01357d%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Designed%2520to%2520Mimic%2520the%2520Methylation%2520Reaction%2520Transition%2520State%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2Fc7ob01357d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0liSOx22PRX_Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0li7wLVFd3RYBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0ljL4g5_ycOUnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Substrate Recognition in Human Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7800</span>– <span class="NLM_lpage">7808</span>, <span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+Basis+of+Substrate+Recognition+in+Human+Nicotinamide+N-Methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0ljL4g5_ycOUnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520Basis%2520of%2520Substrate%2520Recognition%2520in%2520Human%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loring, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5524</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5524-5532&author=H.+S.+Loringauthor=P.+R.+Thompson&title=Kinetic+Mechanism+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.biochem.8b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Loring, Heather S.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">5524-5532</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the transfer of a Me group from S-adenosyl methionine to nicotinamide, pyridine and other structural analogs.  Aberrantly increased NNMT activity results in the depletion of S-adenosyl methionine (SAM), nicotinamide (NAM), and NAD+; NAM is required for NAD+ biosynthesis.  SAM depletion impairs the methylation potential of the cell, resulting in hypomethylated histones and an altered epigenetic profile.  In addn., decreased NAD+ levels neg. affect energy metab. by disrupting oxidative phosphorylation.  Due to its impact on epigenetic states and NAD+ levels, NNMT is implicated in cancer, neurodegenerative and metabolic diseases, making it an appealing target for therapeutic intervention.  To gain insights that would guide the design of inhibitors and activity-based probes, we performed detailed kinetic studies on human NNMT.  Herein, we report the kinetic mechanism of NNMT.  Our initial velocity, product inhibition, and dead-end analog inhibition studies collectively indicate that NNMT uses a rapid equil. ordered mechanism, where NNMT first binds SAM, which is followed by NAM.  Me transfer occurs and methylated NAM (MeNAM) and S-adenosyl homocysteine (SAH) are released consecutively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBKv-k7WIajbVg90H21EOLACvtfcHk0lhq4skXkA1nGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM&md5=8c67d54bd73a7fe4ad3dc51411d12d1e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00775%26sid%3Dliteratum%253Aachs%26aulast%3DLoring%26aufirst%3DH.%2BS.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5524%26epage%3D5532%26doi%3D10.1021%2Facs.biochem.8b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borch, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durst, H. D.</span></span> <span> </span><span class="NLM_article-title">Cyanohydridoborate Anion as a Selective Reducing Agent</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2897</span>– <span class="NLM_lpage">2904</span>, <span class="refDoi"> DOI: 10.1021/ja00741a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00741a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaE3MXksV2msLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1971&pages=2897-2904&author=R.+F.+Borchauthor=M.+D.+Bernsteinauthor=H.+D.+Durst&title=Cyanohydridoborate+Anion+as+a+Selective+Reducing+Agent&doi=10.1021%2Fja00741a013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydridoborate anion as a selective reducing agent</span></div><div class="casAuthors">Borch, Richard F.; Bernstein, Mark D.; Durst, H. Dupont</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2897-904</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Sodium cyanohydridoborate (NaBH3CN) reduces a wide variety of org. functional groups with remarkable selectivity.  The redn. of aldehydes and ketones is pH dependent, the reaction proceeding readily at pH 3-4.  Oximes are smoothly reduced to alkylhydroxylamines and enamines are reduced to amines under acid catalysis.  Reaction of an aldehyde or ketone with NH3, primary amine, or secondary amine at pH ∼7 in the presence of BH3CN- leads to primary, secondary, or tertiary amines, resp., via reductive amination of the carbonyl group.  Beaction of substituted pyruvic acids with NH3 and BH3CN- affords an excellent method for the synthesis of amino acids; 15N labeling can be accomplished by using 15NH3.  The hydrogens of BH3CN- can be readily exchanged for either D or T, thus permitting the synthesis of D- or T-labeled alcs., amines, and amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuf4qG_KVwrVg90H21EOLACvtfcHk0lh8tMaUYOb11w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXksV2msLs%253D&md5=0d4426a681222cd423971c97b20cf140</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja00741a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00741a013%26sid%3Dliteratum%253Aachs%26aulast%3DBorch%26aufirst%3DR.%2BF.%26aulast%3DBernstein%26aufirst%3DM.%2BD.%26aulast%3DDurst%26aufirst%3DH.%2BD.%26atitle%3DCyanohydridoborate%2520Anion%2520as%2520a%2520Selective%2520Reducing%2520Agent%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2897%26epage%3D2904%26doi%3D10.1021%2Fja00741a013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/c5ob00120j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1039%2Fc5ob00120j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2Fc5ob00120j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0ljOxt-hFcc__A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2Fc5ob00120j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5ob00120j%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2Fc5ob00120j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heck, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolley, J. P.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Vinylic Hydrogen Substitution Reactions with Aryl, Benzyl, and Styryl Halides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2320</span>– <span class="NLM_lpage">2322</span>, <span class="refDoi"> DOI: 10.1021/jo00979a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00979a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaE38XkslCqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1972&pages=2320-2322&author=R.+F.+Heckauthor=J.+P.+Nolley&title=Palladium-Catalyzed+Vinylic+Hydrogen+Substitution+Reactions+with+Aryl%2C+Benzyl%2C+and+Styryl+Halides&doi=10.1021%2Fjo00979a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed vinylic hydrogen substitution reactions with aryl, benzyl, and styryl halides</span></div><div class="casAuthors">Heck, R. F.; Nolley, J. P., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2320-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Aryl, benzyl, and styryl halides react with olefins in the presence of a hindered amine and a catalytic amt. of Pd metal to form vinylic derivs. in which the aryl, benzyl, or styryl group has replaced a vinylic H of the original olefin.  The reactions occur readily at 100°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG4X3ukMoCE7Vg90H21EOLACvtfcHk0lgYuGh3GD9BVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38XkslCqsrk%253D&md5=2e3ba21021999631fb0ea341346f11f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo00979a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00979a024%26sid%3Dliteratum%253Aachs%26aulast%3DHeck%26aufirst%3DR.%2BF.%26aulast%3DNolley%26aufirst%3DJ.%2BP.%26atitle%3DPalladium-Catalyzed%2520Vinylic%2520Hydrogen%2520Substitution%2520Reactions%2520with%2520Aryl%252C%2520Benzyl%252C%2520and%2520Styryl%2520Halides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1972%26volume%3D37%26spage%3D2320%26epage%3D2322%26doi%3D10.1021%2Fjo00979a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonogashira, K.</span></span> <span> </span><span class="NLM_article-title">Development of Pd-Cu Catalyzed Cross-Coupling of Terminal Acetylenes with Sp2-Carbon Halides</span>. <i>J. Organomet. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>653</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/s0022-328x(02)01158-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fs0022-328x%2802%2901158-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD38Xktlyhurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=653&publication_year=2002&pages=46-49&author=K.+Sonogashira&title=Development+of+Pd-Cu+Catalyzed+Cross-Coupling+of+Terminal+Acetylenes+with+Sp2-Carbon+Halides&doi=10.1016%2Fs0022-328x%2802%2901158-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Pd-Cu catalyzed cross-coupling of terminal acetylenes with sp2-carbon halides</span></div><div class="casAuthors">Sonogashira, Kenkichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organometallic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">653</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">46-49</span>CODEN:
                <span class="NLM_cas:coden">JORCAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-328X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The Pd-catalyzed cross coupling reactions between sp2-C halides and terminal acetylenes were independently reported by Heck, Cassar and the authors in 1975.  The former two methods were developed as an extension of the Heck reaction to an acetylenic CH-bond.  The authors' method was discovered based on the combination of Pd-catalyzed cross-coupling of sp2-C halides with terminal acetylenes and Cu-catalyzed alkynylation of metal complexes developed by the authors in systematic studies on transition metal acetylide chem.  The coupling reactions were used extensively as a reliable method for the synthesis of enyne-based acetylenic materials.  Some recent advances of the coupling are also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozYFGk4Kf6srVg90H21EOLACvtfcHk0lgYuGh3GD9BVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xktlyhurk%253D&md5=fd23c853c93b16bb3557bc8955243a93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs0022-328x%2802%2901158-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0022-328x%252802%252901158-0%26sid%3Dliteratum%253Aachs%26aulast%3DSonogashira%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520Pd-Cu%2520Catalyzed%2520Cross-Coupling%2520of%2520Terminal%2520Acetylenes%2520with%2520Sp2-Carbon%2520Halides%26jtitle%3DJ.%2520Organomet.%2520Chem.%26date%3D2002%26volume%3D653%26spage%3D46%26epage%3D49%26doi%3D10.1016%2Fs0022-328x%2802%2901158-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i>-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0liZNlAhocl2uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1074%2Fjbc.m114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.m114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0ljNbXz5bXOU6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.m114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibitors</span>.  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0lgQ3UoVI4f9vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">A Rapid and Efficient Assay for the Characterization of Substrates and Inhibitors of Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5307</span>– <span class="NLM_lpage">5315</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+Van%0AHarenauthor=J.+Sastre+Tora%C3%B1oauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+Rapid+and+Efficient+Assay+for+the+Characterization+of+Substrates+and+Inhibitors+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0lh1PgFxD-f4WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520Rapid%2520and%2520Efficient%2520Assay%2520for%2520the%2520Characterization%2520of%2520Substrates%2520and%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct Measurement of Intracellular Compound Concentration by RapidFire Mass Spectrometry Offers Insights into Cell Permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26336900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+Measurement+of+Intracellular+Compound+Concentration+by+RapidFire+Mass+Spectrometry+Offers+Insights+into+Cell+Permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span></div><div class="casAuthors">Gordon, Laurie J.; Allen, Morven; Artursson, Per; Hann, Michael M.; Leavens, Bill J.; Mateus, Andre; Readshaw, Simon; Valko, Klara; Wayne, Gareth J.; West, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">156-164</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">One of the key challenges facing early stage drug discovery is understanding the commonly obsd. difference between the activity of compds. in biochem. assays and cellular assays.  Traditionally, indirect or estd. cell permeability measurements such as estns. from logP or artificial membrane permeability are used to explain the differences.  The missing link is a direct measurement of intracellular compd. concn. in whole cells.  This can, in some circumstances, be estd. from the cellular activity, but this may also be problematic if cellular activity is weak or absent.  Advances in sensitivity and throughput of anal. techniques have enabled us to develop a high-throughput assay for the measurement of intracellular compd. concn. for routine use to support lead optimization.  The assay uses a RapidFire-MS based readout of compd. concn. in HeLa cells following incubation of cells with test compd.  The initial assay validation was performed by ultra-high performance liq. chromatog. tandem mass spectrometry, and the assay was subsequently transferred to RapidFire tandem mass spectrometry.  Further miniaturization and optimization were performed to streamline the process, increase sample throughput, and reduce cycle time.  This optimization has delivered a semi-automated platform with the potential of prodn. scale compd. profiling up to 100 compds. per day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLd62-6firibVg90H21EOLACvtfcHk0lgIq_OLTIjtuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D&md5=34a5d2a18c1a36973425d00b87112f20</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520Measurement%2520of%2520Intracellular%2520Compound%2520Concentration%2520by%2520RapidFire%2520Mass%2520Spectrometry%2520Offers%2520Insights%2520into%2520Cell%2520Permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Human H3K9 Methyltransferases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e8570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=H.+Wuauthor=J.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+Biology+of+Human+H3K9+Methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eYjzGK3LdOpg7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520Biology%2520of%2520Human%2520H3K9%2520Methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">A Transient Kinetic Analysis of PRMT1 Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7033</span>– <span class="NLM_lpage">7044</span>, <span class="refDoi"> DOI: 10.1021/bi200456u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200456u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7033-7044&author=Y.+Fengauthor=N.+Xieauthor=M.+Jinauthor=M.+R.+Stahleyauthor=J.+T.+Stiversauthor=Y.+G.+Zheng&title=A+Transient+Kinetic+Analysis+of+PRMT1+Catalysis&doi=10.1021%2Fbi200456u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A transient kinetic analysis of PRMT1 catalysis</span></div><div class="casAuthors">Feng, You; Xie, Nan; Jin, Miyeong; Stahley, Mary R.; Stivers, JamesT.; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">7033-7044</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications (PTMs) are important strategies used by eukaryotic organisms to modulate their phenotypes.  One of the well-studied PTMs, Arg residue methylation, is catalyzed by protein arginine methyltransferases (PRMTs) with S-adenosyl-L-methionine (SAM) as the Me donor.  The functions of PRMTs have been broadly studied in different biol. processes and diseased states, but the mol. basis for Arg methylation is not well-defined.  Here, the authors report the transient-state kinetic anal. of rat PRMT1 catalysis.  The fast assocn. and dissocn. rates suggested that PRMT1 catalysis of histone H4 methylation followed a rapid equil. sequential kinetic mechanism.  The data provided direct evidence that the chem. of Me transfer is the major rate-limiting step and that binding of cofactors SAM or S-adenosyl-L-homocysteine (SAH) affects the assocn. and dissocn. of H4 with PRMT1.  Importantly, from stopped-flow fluorescence measurements, the authors identified a crit. kinetic step suggesting a pre-catalytic conformational transition induced by substrate binding.  These results provide new insights into the mechanism of Arg methylation and the rational design of PRMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpImJRI93ID4rVg90H21EOLACvtfcHk0lhe2xoHOj8jbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D&md5=f2562d7ece9b10754b551c2b6dec7cb1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi200456u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200456u%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DStahley%26aufirst%3DM.%2BR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DA%2520Transient%2520Kinetic%2520Analysis%2520of%2520PRMT1%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7033%26epage%3D7044%26doi%3D10.1021%2Fbi200456u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oudhoff, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couzens, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benn, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braam, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, F. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">(R)-PFI-2 Is a Potent and Selective Inhibitor of SETD7 Methyltransferase Activity in Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12853</span>– <span class="NLM_lpage">12858</span>, <span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+Is+a+Potent+and+Selective+Inhibitor+of+SETD7+Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0lgUBnyLEGbvWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520Is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debler, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmack, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavropoulos, P.</span></span> <span> </span><span class="NLM_article-title">A Glutamate/Aspartate Switch Controls Product Specificity in a Protein Arginine Methyltransferase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2073</span>, <span class="refDoi"> DOI: 10.1073/pnas.1525783113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1073%2Fpnas.1525783113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26858449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2068-2073&author=E.+W.+Deblerauthor=K.+Jainauthor=R.+A.+Warmackauthor=Y.+Fengauthor=S.+G.+Clarkeauthor=G.+Blobelauthor=P.+Stavropoulos&title=A+Glutamate%2FAspartate+Switch+Controls+Product+Specificity+in+a+Protein+Arginine+Methyltransferase&doi=10.1073%2Fpnas.1525783113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamate/aspartate switch controls product specificity in a protein arginine methyltransferase</span></div><div class="casAuthors">Debler, Erik W.; Jain, Kanishk; Warmack, Rebeccah A.; Feng, You; Clarke, Steven G.; Blobel, Gunter; Stavropoulos, Pete</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2073</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosoma brucei PRMT7 (TbPRMT7) is a protein arginine methyltransferase (PRMT) that strictly monomethylates various substrates, thus classifying it as a type III PRMT.  However, the mol. basis of its unique product specificity has remained elusive.  Here, we present the structure of TbPRMT7 in complex with its cofactor product S-adenosyl-L-homocysteine (AdoHcy) at 2.8 Å resoln. and identify a glutamate residue crit. for its monomethylation behavior.  TbPRMT7 comprises the conserved methyltransferase and β-barrel domains, an N-terminal extension, and a dimerization arm.  The active site at the interface of the N-terminal extension, methyltransferase, and β-barrel domains is stabilized by the dimerization arm of the neighboring protomer, providing a structural basis for dimerization as a prerequisite for catalytic activity.  Mutagenesis of active-site residues highlights the importance of Glu181, the second of the two invariant glutamate residues of the double E loop that coordinate the target arginine in substrate peptides/proteins and that increase its nucleophilicity.  Strikingly, mutation of Glu181 to aspartate converts TbPRMT7 into a type I PRMT, producing asym. dimethylarginine (ADMA).  Isothermal titrn. calorimetry (ITC) using a histone H4 peptide showed that the Glu181Asp mutant has markedly increased affinity for monomethylated peptide with respect to the WT, suggesting that the enlarged active site can favorably accommodate monomethylated peptide and provide sufficient space for ADMA formation.  In conclusion, these findings yield valuable insights into the product specificity and the catalytic mechanism of protein arginine methyltransferases and have important implications for the rational (re)design of PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj2myjluq1B7Vg90H21EOLACvtfcHk0liT4SYmKh8FiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D&md5=93f9d17569159c432b5801e2439c6df0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1525783113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1525783113%26sid%3Dliteratum%253Aachs%26aulast%3DDebler%26aufirst%3DE.%2BW.%26aulast%3DJain%26aufirst%3DK.%26aulast%3DWarmack%26aufirst%3DR.%2BA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DStavropoulos%26aufirst%3DP.%26atitle%3DA%2520Glutamate%252FAspartate%2520Switch%2520Controls%2520Product%2520Specificity%2520in%2520a%2520Protein%2520Arginine%2520Methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D2068%26epage%3D2073%26doi%3D10.1073%2Fpnas.1525783113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span> <span> </span><span class="NLM_article-title">Fast_dp Methods</span>.  <i>Zenodo</i>, <span class="NLM_year">2014</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DFast_dp%2520Methods%26jtitle%3DZenodo%26date%3D2014%26volume%3D55%26spage%3D81%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liDC-Kwe6SsxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgc5yF45UC_Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Policarpo, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decultot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmič, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittakavi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panarese, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shair, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9837</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01238</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01238" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9837&author=R.+L.+Policarpoauthor=L.+Decultotauthor=E.+Mayauthor=P.+Kuzmi%C4%8Dauthor=S.+Carlsonauthor=D.+Huangauthor=V.+Chuauthor=B.+A.+Wrightauthor=S.+Dhakshinamoorthyauthor=A.+Kanntauthor=S.+Raniauthor=S.+Dittakaviauthor=J.+D.+Panareseauthor=R.+Gaudetauthor=M.+D.+Shair&title=High-Affinity+Alkynyl+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.9b01238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Policarpo, Rocco L.; Decultot, Ludovic; May, Elizabeth; Kuzmic, Petr; Carlson, Samuel; Huang, Danny; Chu, Vincent; Wright, Brandon A.; Dhakshinamoorthy, Saravanakumar; Kannt, Aimo; Rani, Shilpa; Dittakavi, Sreekanth; Panarese, Joseph D.; Gaudet, Rachelle; Shair, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9837-9873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM).  NNMT overexpression has been linked to diabetes, obesity, and various cancers.  In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT.  The reported nicotinamide-SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM Me transfer reaction.  NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor.  An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the Me transfer tunnel of NNMT with ideal shape complementarity.  The compds. reported in this work represent the most potent and selective NNMT inhibitors reported to date.  The rational design principle described herein could potentially be extended to other methyltransferase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzDcwfc0ibVg90H21EOLACvtfcHk0lg7bhUDKyiFQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM&md5=f62a8055840292563d07cd25b0fcb4c2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26aulast%3DPolicarpo%26aufirst%3DR.%2BL.%26aulast%3DDecultot%26aufirst%3DL.%26aulast%3DMay%26aufirst%3DE.%26aulast%3DKuzmi%25C4%258D%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DV.%26aulast%3DWright%26aufirst%3DB.%2BA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DDittakavi%26aufirst%3DS.%26aulast%3DPanarese%26aufirst%3DJ.%2BD.%26aulast%3DGaudet%26aufirst%3DR.%26aulast%3DShair%26aufirst%3DM.%2BD.%26atitle%3DHigh-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9837%26doi%3D10.1021%2Facs.jmedchem.9b01238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=342-346&author=C.+F.+Strattonauthor=M.+B.+Poulinauthor=Q.+Duauthor=V.+L.+Schramm&title=Kinetic+Isotope+Effects+and+Transition+State+Structure+for+Human+Phenylethanolamine+N-Methyltransferase&doi=10.1021%2Facschembio.6b00922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span></div><div class="casAuthors">Stratton, Christopher F.; Poulin, Myles B.; Du, Quan; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-346</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenylethanolamine N-methyltransferase (PNMT) catalyzes the S-adenosyl-L-methionine (SAM)-dependent conversion of norepinephrine to epinephrine.  Epinephrine has been assocd. with crit. processes in humans including the control of respiration and blood pressure.  Addnl., PNMT activity has been suggested to play a role in hypertension and Alzheimer's disease.  Here, labeled SAM substrates were used to measure primary methyl-14C and 36S, and secondary methyl-3H, 5'-3H, 5'-14C intrinsic kinetic isotope effects for human PNMT.  The transition state of human PNMT was modeled by matching kinetic isotope effects predicted via quantum chem. calcns. to intrinsic values.  The model showed that human PNMT catalyzes the formation of epinephrine through an early SN2 transition state in which Me transfer is rate-limiting.  This anal. provides information on the geometry and electrostatics of the human PNMT transition state structure, details that enable the design of transition state analogs as potential PNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBDuQMfV317Vg90H21EOLACvtfcHk0liqnGhBkl_ZpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL&md5=cadf5bbe757ceba13081bd8e63a2f1d6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00922%26sid%3Dliteratum%253Aachs%26aulast%3DStratton%26aufirst%3DC.%2BF.%26aulast%3DPoulin%26aufirst%3DM.%2BB.%26aulast%3DDu%26aufirst%3DQ.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DKinetic%2520Isotope%2520Effects%2520and%2520Transition%2520State%2520Structure%2520for%2520Human%2520Phenylethanolamine%2520N-Methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D342%26epage%3D346%26doi%3D10.1021%2Facschembio.6b00922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Ying Meng, Dan Yu, Nicholas Noinaj, Xiaodong Cheng, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoproteomic Study Uncovers HemK2/KMT9 As a New Target for NTMT1 Bisubstrate Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1234-1242. <a href="https://doi.org/10.1021/acschembio.1c00279" title="DOI URL">https://doi.org/10.1021/acschembio.1c00279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00279%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemoproteomic%252BStudy%252BUncovers%252BHemK2%25252FKMT9%252BAs%252Ba%252BNew%252BTarget%252Bfor%252BNTMT1%252BBisubstrate%252BInhibitors%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13042021%26date%3D18062021%26date%3D01072021%26volume%3D16%26issue%3D7%26spage%3D1234%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Charles A. Lewis, Jr., <span class="NLM_string-name hlFld-ContribAuthor">Richard Wolfenden</span>. </span><span class="cited-content_cbyCitation_article-title">The Burden Borne by Protein Methyltransferases: Rates and Equilibria of Non-enzymatic Methylation of Amino Acid Side Chains by SAM in Water. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (11)
                                     , 854-858. <a href="https://doi.org/10.1021/acs.biochem.1c00028" title="DOI URL">https://doi.org/10.1021/acs.biochem.1c00028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.1c00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.1c00028%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DThe%252BBurden%252BBorne%252Bby%252BProtein%252BMethyltransferases%25253A%252BRates%252Band%252BEquilibria%252Bof%252BNon-enzymatic%252BMethylation%252Bof%252BAmino%252BAcid%252BSide%252BChains%252Bby%252BSAM%252Bin%252BWater%26aulast%3DLewis%26aufirst%3DCharles%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08012021%26date%3D27022021%26date%3D05032021%26volume%3D60%26issue%3D11%26spage%3D854%26epage%3D858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jian Lu, Aaron G. Bart, Qian Wu, Kevin R. Criscione, Michael J. McLeish, Emily E. Scott, <span class="NLM_string-name hlFld-ContribAuthor">Gary L. Grunewald</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Drug Design of Bisubstrate Inhibitors of Phenylethanolamine N-Methyltransferase Possessing Low Nanomolar Affinity at Both Substrate Binding Domains1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13878-13898. <a href="https://doi.org/10.1021/acs.jmedchem.0c01475" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDrug%252BDesign%252Bof%252BBisubstrate%252BInhibitors%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%252BPossessing%252BLow%252BNanomolar%252BAffinity%252Bat%252BBoth%252BSubstrate%252BBinding%252BDomains1%26aulast%3DLu%26aufirst%3DJian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24082020%26date%3D04112020%26volume%3D63%26issue%3D22%26spage%3D13878%26epage%3D13898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brianna D. Mackie, Dongxing Chen, Guangping Dong, Cheng Dong, Haley Parker, Christine E. Schaner Tooley, Nicholas Noinaj, Jinrong Min, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9512-9522. <a href="https://doi.org/10.1021/acs.jmedchem.0c00689" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00689%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BPeptidomimetic%252BInhibitors%252Bof%252BNTMT1%25252F2%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BCharacterization%25252C%252Band%252BCrystallographic%252BStudies%26aulast%3DMackie%26aufirst%3DBrianna%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24042020%26date%3D05082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9512%26epage%3D9522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Niusha Mahmoodi, Rajesh K. Harijan, <span class="NLM_string-name hlFld-ContribAuthor">Vern L. Schramm</span>. </span><span class="cited-content_cbyCitation_article-title">Transition-State Analogues of Phenylethanolamine N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (33)
                                     , 14222-14233. <a href="https://doi.org/10.1021/jacs.0c05446" title="DOI URL">https://doi.org/10.1021/jacs.0c05446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c05446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c05446%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DTransition-State%252BAnalogues%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%26aulast%3DMahmoodi%26aufirst%3DNiusha%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18052020%26date%3D07082020%26date%3D24072020%26volume%3D142%26issue%3D33%26spage%3D14222%26epage%3D14233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Cheng Dong, Guangping Dong, Karthik Srinivasan, Jinrong Min, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 8419-8431. <a href="https://doi.org/10.1021/acs.jmedchem.0c00770" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProbing%252Bthe%252BPlasticity%252Bin%252Bthe%252BActive%252BSite%252Bof%252BProtein%252BN-terminal%252BMethyltransferase%252B1%252BUsing%252BBisubstrate%252BAnalogues%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D16072020%26date%3D01072020%26volume%3D63%26issue%3D15%26spage%3D8419%26epage%3D8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan C.  García-Cañaveras</span>, <span class="hlFld-ContribAuthor ">Agustín  Lahoz</span>. </span><span class="cited-content_cbyCitation_article-title">Tumor Microenvironment-Derived Metabolites: A Guide to Find New Metabolic Therapeutic Targets and Biomarkers. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (13)
                                     , 3230. <a href="https://doi.org/10.3390/cancers13133230" title="DOI URL">https://doi.org/10.3390/cancers13133230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13133230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13133230%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTumor%252BMicroenvironment-Derived%252BMetabolites%25253A%252BA%252BGuide%252Bto%252BFind%252BNew%252BMetabolic%252BTherapeutic%252BTargets%252Band%252BBiomarkers%26aulast%3DGarc%25C3%25ADa-Ca%25C3%25B1averas%26aufirst%3DJuan%2BC.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D13%26spage%3D3230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongzhi  Gao</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>, <span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>. </span><span class="cited-content_cbyCitation_article-title">Nicotinamide N-methyl transferase (NNMT): An emerging therapeutic target. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DNicotinamide%252BN-methyl%252Btransferase%252B%252528NNMT%252529%25253A%252BAn%252Bemerging%252Btherapeutic%252Btarget%26aulast%3DGao%26aufirst%3DYongzhi%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Sridharan  Rajagopal</span>, <span class="hlFld-ContribAuthor ">Mahanandeesha S.  Hallur</span>, <span class="hlFld-ContribAuthor ">Indu  Swamy</span>, <span class="hlFld-ContribAuthor ">Rajendra  Kristam</span>, <span class="hlFld-ContribAuthor ">Saravanakumar  Dhakshinamoorthy</span>, <span class="hlFld-ContribAuthor ">Joerg  Czech</span>, <span class="hlFld-ContribAuthor ">Gernot  Zech</span>, <span class="hlFld-ContribAuthor ">Herman  Schreuder</span>, <span class="hlFld-ContribAuthor ">Sven  Ruf</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Inhibitors of Nicotinamide-N-Methyltransferase for the Treatment of Metabolic Disorders. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 991. <a href="https://doi.org/10.3390/molecules26040991" title="DOI URL">https://doi.org/10.3390/molecules26040991</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040991%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNovel%252BInhibitors%252Bof%252BNicotinamide-N-Methyltransferase%252Bfor%252Bthe%252BTreatment%252Bof%252BMetabolic%252BDisorders%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Palmiro  Poltronieri</span>, <span class="hlFld-ContribAuthor ">Angela  Celetti</span>, <span class="hlFld-ContribAuthor ">Luca  Palazzo</span>. </span><span class="cited-content_cbyCitation_article-title">Mono(ADP-ribosyl)ation Enzymes and NAD+ Metabolism: A Focus on Diseases and Therapeutic Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2021,</strong> <em>10 </em>
                                    (1)
                                     , 128. <a href="https://doi.org/10.3390/cells10010128" title="DOI URL">https://doi.org/10.3390/cells10010128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells10010128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells10010128%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DMono%252528ADP-ribosyl%252529ation%252BEnzymes%252Band%252BNAD%25252B%252BMetabolism%25253A%252BA%252BFocus%252Bon%252BDiseases%252Band%252BTherapeutic%252BPerspectives%26aulast%3DPoltronieri%26aufirst%3DPalmiro%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D1%26spage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and inhibitors of nicotinamide
              N
              -methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>28 </em><a href="https://doi.org/10.1039/D1MD00016K" title="DOI URL">https://doi.org/10.1039/D1MD00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DMechanisms%252Band%252Binhibitors%252Bof%252Bnicotinamide%252BN%252B-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Dongxing  Chen</span>, <span class="hlFld-ContribAuthor ">Youchao  Deng</span>, <span class="hlFld-ContribAuthor ">Guangping  Dong</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent and dual-selective bisubstrate inhibitor for protein arginine methyltransferase 4/5. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>57 </em><a href="https://doi.org/10.1016/j.apsb.2020.10.013" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.10.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.10.013%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Band%252Bdual-selective%252Bbisubstrate%252Binhibitor%252Bfor%252Bprotein%252Barginine%252Bmethyltransferase%252B4%25252F5%26aulast%3DAl-Hamashi%26aufirst%3DAyad%2BA.%26date%3D2020%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of fluorescence polarization-based competition assay for nicotinamide N-methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2020,</strong> <em>604 </em>, 113833. <a href="https://doi.org/10.1016/j.ab.2020.113833" title="DOI URL">https://doi.org/10.1016/j.ab.2020.113833</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2020.113833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2020.113833%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DDevelopment%252Bof%252Bfluorescence%252Bpolarization-based%252Bcompetition%252Bassay%252Bfor%252Bnicotinamide%252BN-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2020%26volume%3D604%26spage%3D113833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of reported NNMT inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of NNMT bisubstrate inhibitors <b>1a–3a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking analysis of compound <b>1a</b> in the binding sites of hNNMT. (A) Docking of compound <b>1a</b> (blue) in the hNNMT (gray) structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). H-bond interactions are shown in yellow dotted lines. (B) Overlay of the docking model with the hNNMT (gray)–nicotinamide (yellow)–SAH (pink) complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Bisubstrate Inhibitors <b>1a–1d</b> and <b>1a*</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(OAc)<sub>2</sub>, Bu<sub>4</sub>NCl, NaHCO<sub>3</sub>, DMF, 3 h, 45–84%; (b) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56–68%; (c) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49–61%; (d) SOCl<sub>2</sub>, EtOH, 0–50 °C, 59%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Bisubstrate Inhibitors <b>1e–1i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56−65%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 50−58%.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Bisubstrate Inhibitor <b>1j</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 68%; (b) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 52%.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Bisubstrate Inhibitor <b>2a–2d</b> and <b>2a*</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, CH<sub>3</sub>CN, 12 h, 87−95%; (b) DMP, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt 1 h, 55−61%; (c) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 56−68%; (d) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49−61%; (e) SOCl<sub>2</sub>, EtOH, 0 °C to 50 °C, 51%.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Bisubstrate Inhibitors <b>3a–3b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, CuI, Et<sub>3</sub>N, CH<sub>3</sub>CN, 12 h, 90%; (b) Pd/C, H<sub>2</sub>, THF, 12 h; then DMP, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt 1 h, 60%; (c) protected NAH or <b>S3</b>, NaBH<sub>3</sub>CN, AcOH, MeOH, 2 h, 50−56%; (d) H<sub>2</sub>O<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, DMSO; then TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 6 h, 49−51%.</p></p></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical characterization of compound LL320 (<b>2a</b>). (A) SAHH-coupled fluorescence assay of compound LL320 ranging in concentrations from 0 to 10 μM; (B) concentration–response plots fitted to Morrison’s quadratic equation for compound LL320.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SARs of NNMT inhibitors. *The values of <i>K</i><sub>i</sub> for the compounds were determined with duplicate experiments (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Time-dependent inhibition study of compound LL320 (<i>n</i> = 2). (A) Concentration–response plots for different preincubation times fitted to Morrison’s quadratic equation for compound LL320; (B) nonlinear regression of Morrison <i>K</i><sub>i</sub> values with preincubation time.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/medium/jm9b01255_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structures of NNMT in complex with LL319 and LL320. (A) Structure of NNMT (gray cartoon) bound with LL319 (green stick) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a>). An 2Fo–Fc omit map contoured to 1.0 σ is shown for LL319 as a transparent green isosurface. (B) Structure of NNMT (gray cartoon) bound with LL320 (yellow stick) (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>). An Fo–Fc omit map contoured to 3.0 σ is shown for LL320 as a transparent yellow isosurface. (C) Structural alignment of LL319 (green stick) and LL320 (yellow stick) within the binding pocket of NNMT. A ligand interaction diagram is shown for LL319 (D) and LL320 (E) with NNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-23/acs.jmedchem.9b01255/20191205/images/large/jm9b01255_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01255&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45348" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45348" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aksoy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szumlanski, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Human Liver Nicotinamide N-Methyltransferase. cDNA Cloning, Expression, and Biochemical Characterization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">14835</span>– <span class="NLM_lpage">14840</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=8182091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=14835-14840&author=S.+Aksoyauthor=C.+L.+Szumlanskiauthor=R.+M.+Weinshilboum&title=Human+Liver+Nicotinamide+N-Methyltransferase.+cDNA+Cloning%2C+Expression%2C+and+Biochemical+Characterization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human liver nicotinamide N-methyltransferase. cDNA cloning, expression, and biochemical characterization</span></div><div class="casAuthors">Aksoy, Saime; Szumlanski, Carol L.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">14835-40</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide and other pyridines.  Human liver NNMT activity has a bimodal frequency distribution, an observation which raises the possibility that this enzyme might be regulated by a genetic polymorphism, a polymorphism that could have functional implications for individual differences in drug and xenobiotic toxicity.  As a first step toward testing this hypothesis, the authors set out to clone and express a cDNA for human liver NNMT.  Human liver NNMT was partially purified, photoaffinity-labeled, subjected to limited proteolysis, and partial amino acid sequence information was obtained.  The polymerase chain reaction was then used to amplify a 550-nucleotide sequence with human liver cDNA as template and primers designed on the basis of the NNMT amino acid sequence.  The 5'- and 3'-ends of a human liver NNMT cDNA were obtained by use of the rapid amplification of cDNA ends.  The combined use of these approaches resulted in the isolation of a human liver NNMT cDNA that was 969 nucleotides in length, with a 792-nucleotide open reading frame that encoded a 264-amino acid protein with a calcd. mol. mass of 29.6 kDa.  The human liver NNMT cDNA was transcribed in vitro and translated with a reticulocyte lysate system to yield a protein with a mol. mass of approx. 29 kDa that comigrated during SDS-polyacrylamide gel electrophoresis with photoaffinity-labeled human liver NNMT.  The NNMT cDNA was also subcloned into the eukaryotic expression vector p91023(B).  COS-1 cells transfected with this construct expressed a high level of NNMT enzymic activity, and the biochem. properties of this activity were similar to those of human liver NNMT.  Human liver NNMT and transfected COS-1 cell NNMT had apparent Km values for the two cosubstrates for the reaction, nicotinamide and S-adenosyl-L-methionine, of 0.43 and 0.38 mM and of 1.8 and 2.2 μM, resp.  IC50 values for the inhibition of NNMT by N1-methylnicotinamide were 60 and 30 μM for human liver and COS-1 cell-expressed NNMT, resp.  Cloning of a cDNA for human liver NNMT should make it possible to test the hypothesis that inheritance may play a role in the regulation of individual differences in human liver NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZRn3_QD5atbVg90H21EOLACvtfcHk0lhMIEfDGEOUbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslCqtro%253D&md5=d5d9edf09cb329ecc01b49fd819fad47</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAksoy%26aufirst%3DS.%26aulast%3DSzumlanski%26aufirst%3DC.%2BL.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DHuman%2520Liver%2520Nicotinamide%2520N-Methyltransferase.%2520cDNA%2520Cloning%252C%2520Expression%252C%2520and%2520Biochemical%2520Characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D14835%26epage%3D14840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&author=P.+Pissios&title=Nicotinamide+N-Methyltransferase%3A+More+Than+a+Vitamin+B3+Clearance+Enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0lg1TeQmx6xdGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-Methyltransferase%253A%2520More%2520Than%2520a%2520Vitamin%2520B3%2520Clearance%2520Enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirinen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulinilkunnil, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhonen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Knockdown Protects against Diet-Induced Obesity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Gongauthor=Y.-c.+Wangauthor=Y.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-Methyltransferase+Knockdown+Protects+against+Diet-Induced+Obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0ljMx00wW8D84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.-c.%26aulast%3DCen%26aufirst%3DY.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Knockdown%2520Protects%2520against%2520Diet-Induced%2520Obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Interacts with Enzymes of the Methionine Cycle and Regulates Methyl Donor Metabolism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5775-5779&author=S.+Hongauthor=B.+Zhaiauthor=P.+Pissios&title=Nicotinamide+N-Methyltransferase+Interacts+with+Enzymes+of+the+Methionine+Cycle+and+Regulates+Methyl+Donor+Metabolism&doi=10.1021%2Facs.biochem.8b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span></div><div class="casAuthors">Hong, Shangyu; Zhai, Bo; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5775-5779</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Me donor balance is crit. for epigenetic regulation in cells and is maintained by the so-called methionine cycle proteins that regenerate S-adenosylmethionine (SAM), the universal Me donor, from homocysteine formed by the activity of methyltransferases.  Nnmt is a liver enzyme that methylates nicotinamide, but its role in regulating Me donor balance in the liver is unclear.  In this study, we assessed the effect of altered Nnmt expression on various aspects of Me donor metab. in the liver.  We found that Nnmt overexpression decreased SAM levels and the SAM/ S-adenosylhomocysteine (SAH) ratio both in vivo and in vitro.  Nnmt knockdown did not change Me donor balance in mouse primary hepatocytes but increased SAM levels and the SAM/SAH ratio when Gnmt, the dominantly expressed methyltransferase in liver, was simultaneously knocked down.  Paradoxically, expression of enzymically deficient Nnmt increased the SAM/SAH ratio, suggesting that Nnmt can regulate Me donor balance independent of its methyltransferase activity.  Proteomics anal. of Nnmt-interacting proteins in the liver identified Bhmt, Mat1a, and Ahcy, all components of the methionine cycle, and functional expts. showed that mutant Nnmt increased the level of remethylation of homocysteine to SAM.  In summary, we show that the function of Nnmt in hepatic Me donor balance is multifactorial.  On one hand, Nnmt decreases Me donor balance, consistent with its activity as a methyltransferase consuming Me donors.  On the other hand, by co-opting the enzymes of the methionine cycle, Nnmt aids the recycling of homocysteine to SAM for another round of methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD5fiz3fTxsrVg90H21EOLACvtfcHk0lhmcktLj5ckEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O&md5=2689739ce6ec8ef59b09af7a20cb06ce</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00561%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DB.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Interacts%2520with%2520Enzymes%2520of%2520the%2520Methionine%2520Cycle%2520and%2520Regulates%2520Methyl%2520Donor%2520Metabolism%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5775%26epage%3D5779%26doi%3D10.1021%2Facs.biochem.8b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Enhances the Capacity of Tumorigenesis Associated with the Promotion of Cell Cycle Progression in Human Colorectal Cancer Cells</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>564</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2014.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.abb.2014.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25201588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFWltbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=564&publication_year=2014&pages=52-66&author=X.+Xieauthor=H.+Yuauthor=Y.+Wangauthor=Y.+Zhouauthor=G.+Liauthor=Z.+Ruanauthor=F.+Liauthor=X.+Wangauthor=H.+Liuauthor=J.+Zhang&title=Nicotinamide+N-Methyltransferase+Enhances+the+Capacity+of+Tumorigenesis+Associated+with+the+Promotion+of+Cell+Cycle+Progression+in+Human+Colorectal+Cancer+Cells&doi=10.1016%2Fj.abb.2014.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase enhances the capacity of tumorigenesis associated with the promotion of cell cycle progression in human colorectal cancer cells</span></div><div class="casAuthors">Xie, Xinyou; Yu, Haitao; Wang, Yanzhong; Zhou, Yanwen; Li, Guiling; Ruan, Zhi; Li, Fengying; Wang, Xiuhong; Liu, Huixing; Zhang, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">564</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-66</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT), an enzyme involved in the biotransformation and detoxification of many drugs and xenobiotic compds., has been found to be overexpressed in several malignancies, including colorectal cancer.  However, the biol. function of NNMT and the related mechanisms in colorectal cancer have not been elucidated.  In the present study, we investigated the effects of NNMT on tumorigenesis by overexpressing NNMT in the human colorectal cancer cells line SW480 which lacks constitutive NNMT expression, and downregulating NNMT expression in HT-29 cells, which exhibit high endogenous expression of NNMT.  We found that NNMT significantly accelerates cell proliferation, enhances colony formation in vitro and tumorigenicity in mice; it also inhibits apoptosis, promotes cell cycle progression, increases ATP and 1-methylnicotinamide level and decreases ROS level.  We also showed that 1-methylnicotinamide accelerates cell growth, inhibits apoptosis, promotes cell cycle progression, attenuates ROS prodn. and increases ATP level.  Our results indicate that NNMT enhances the capacity of tumorigenesis assocd. with the inhibition of cell apoptosis and the promotion of cell cycle progression in human colorectal cancer cells and the 1-methylnicotinamide increased by NNMT mediates the cellular effects of NNMT in cells.  NNMT may play a vital role in energy balance and ROS induction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7twTbySzqd7Vg90H21EOLACvtfcHk0lgrudmyPWMVjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFWltbnO&md5=5782d0bee67209263ee9e592608eccdf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2014.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2014.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DRuan%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Enhances%2520the%2520Capacity%2520of%2520Tumorigenesis%2520Associated%2520with%2520the%2520Promotion%2520of%2520Cell%2520Cycle%2520Progression%2520in%2520Human%2520Colorectal%2520Cancer%2520Cells%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2014%26volume%3D564%26spage%3D52%26epage%3D66%26doi%3D10.1016%2Fj.abb.2014.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palanichamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumoorthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litzenberg, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarti, A.</span></span> <span> </span><span class="NLM_article-title">NNMT Silencing Activates Tumor Suppressor PP2A, Inactivates Oncogenic STKs, and Inhibits Tumor Forming Ability</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2325</span>– <span class="NLM_lpage">2334</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-1323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1158%2F1078-0432.ccr-16-1323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=27810903" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2325-2334&author=K.+Palanichamyauthor=S.+Kanjiauthor=N.+Gordonauthor=K.+Thirumoorthyauthor=J.+R.+Jacobauthor=K.+T.+Litzenbergauthor=D.+Patelauthor=A.+Chakravarti&title=NNMT+Silencing+Activates+Tumor+Suppressor+PP2A%2C+Inactivates+Oncogenic+STKs%2C+and+Inhibits+Tumor+Forming+Ability&doi=10.1158%2F1078-0432.ccr-16-1323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT Silencing Activates Tumor Suppressor PP2A, Inactivates Oncogenic STKs, and Inhibits Tumor Forming Ability</span></div><div class="casAuthors">Palanichamy, Kamalakannan; Kanji, Suman; Gordon, Nicolaus; Thirumoorthy, Krishnan; Jacob, John R.; Litzenberg, Kevin T.; Patel, Disha; Chakravarti, Arnab</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2325-2334</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: To identify potential mol. hubs that regulate oncogenic kinases and target them to improve treatment outcomes for glioblastoma patients.  Exptl. Design: Data mining of The Cancer Genome Atlas datasets identified nicotinamide-N-Me transferase (NNMT) as a prognostic marker for glioblastoma, an enzyme linked to the reorganization of the methylome.  We tested our hypothesis that NNMT plays a crucial role by modulating protein methylation, leading to inactivation of tumor suppressors and activation of oncogenes.  Further expts. were performed to understand the underlying biochem. mechanisms using glioblastoma patient samples, established, primary, and isogenic cells.  Results: We demonstrate that NNMT outcompetes leucine carboxyl Me transferase 1 (LCMT1) for Me transfer from principal Me donor SAM in biol. systems.  Inhibiting NNMT increased the availability of Me groups for LCMT1 to methylate PP2A, resulting in the inhibition of oncogenic serine/threonine kinases (STK).  Further, NNMT inhibition retained the radiosensitizer nicotinamide and enhanced radiation sensitivity.  We have provided the biochem. rationale of how NNMT plays a vital role in inhibiting tumor suppressor PP2A while concomitantly activating STKs.  Conclusions: We report the intricate novel mechanism in which NNMT inhibits tumor suppressor PP2A by reorganizing the methylome both at epigenome and proteome levels and concomitantly activating prosurvival STKs.  In glioblastoma tumors with NNMT expression, activation of PP2A can be accomplished by FDA approved perphenazine (PPZ), which is currently used to treat mood disorders such as schizophrenia, bipolar disorder, etc.  This study forms a foundation for further glioblastoma clin. trials using PPZ with std. of care treatment.  Clin Cancer Res; 23(9); 2325-34. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqIwv0424CHbVg90H21EOLACvtfcHk0li78IhnaPirbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFehuro%253D&md5=dafd3575d75a59c87f691b37e8dbcd8f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-1323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-1323%26sid%3Dliteratum%253Aachs%26aulast%3DPalanichamy%26aufirst%3DK.%26aulast%3DKanji%26aufirst%3DS.%26aulast%3DGordon%26aufirst%3DN.%26aulast%3DThirumoorthy%26aufirst%3DK.%26aulast%3DJacob%26aufirst%3DJ.%2BR.%26aulast%3DLitzenberg%26aufirst%3DK.%2BT.%26aulast%3DPatel%26aufirst%3DD.%26aulast%3DChakravarti%26aufirst%3DA.%26atitle%3DNNMT%2520Silencing%2520Activates%2520Tumor%2520Suppressor%2520PP2A%252C%2520Inactivates%2520Oncogenic%2520STKs%252C%2520and%2520Inhibits%2520Tumor%2520Forming%2520Ability%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2325%26epage%3D2334%26doi%3D10.1158%2F1078-0432.ccr-16-1323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coscia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chryplewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tienda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahotko, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaženović, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastra, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perets, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span> <span> </span><span class="NLM_article-title">Proteomics Reveals NNMT as a Master Metabolic Regulator of Cancer-Associated Fibroblasts</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1173-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fs41586-019-1173-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=31043742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=723-728&author=M.+A.+Eckertauthor=F.+Cosciaauthor=A.+Chryplewiczauthor=J.+W.+Changauthor=K.+M.+Hernandezauthor=S.+Panauthor=S.+M.+Tiendaauthor=D.+A.+Nahotkoauthor=G.+Liauthor=I.+Bla%C5%BEenovi%C4%87author=R.+R.+Lastraauthor=M.+Curtisauthor=S.+D.+Yamadaauthor=R.+Peretsauthor=S.+M.+McGregorauthor=J.+Andradeauthor=O.+Fiehnauthor=R.+E.+Moelleringauthor=M.+Mannauthor=E.+Lengyel&title=Proteomics+Reveals+NNMT+as+a+Master+Metabolic+Regulator+of+Cancer-Associated+Fibroblasts&doi=10.1038%2Fs41586-019-1173-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span></div><div class="casAuthors">Eckert, Mark A.; Coscia, Fabian; Chryplewicz, Agnieszka; Chang, Jae Won; Hernandez, Kyle M.; Pan, Shawn; Tienda, Samantha M.; Nahotko, Dominik A.; Li, Gang; Blazenovic, Ivana; Lastra, Ricardo R.; Curtis, Marion; Yamada, S. Diane; Perets, Ruth; McGregor, Stephanie M.; Andrade, Jorge; Fiehn, Oliver; Moellering, Raymond E.; Mann, Matthias; Lengyel, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7758</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">High-grade serous carcinoma has a poor prognosis, owing primarily to its early dissemination throughout the abdominal cavity.  Genomic and proteomic approaches have provided snapshots of the proteogenomics of ovarian cancer1,2, but a systematic examn. of both the tumor and stromal compartments is crit. in understanding ovarian cancer metastasis.  Here we develop a label-free proteomic workflow to analyze as few as 5000 formalin-fixed, paraffin-embedded cells microdissected from each compartment.  The tumor proteome was stable during progression from in situ lesions to metastatic disease; however, the metastasis-assocd. stroma was characterized by a highly conserved proteomic signature, prominently including the methyltransferase nicotinamide N-methyltransferase (NNMT) and several of the proteins that it regulates.  Stromal NNMT expression was necessary and sufficient for functional aspects of the cancer-assocd. fibroblast (CAF) phenotype, including the expression of CAF markers and the secretion of cytokines and oncogenic extracellular matrix.  Stromal NNMT expression supported ovarian cancer migration, proliferation and in vivo growth and metastasis.  Expression of NNMT in CAFs led to depletion of S-adenosyl methionine and redn. in histone methylation assocd. with widespread gene expression changes in the tumor stroma.  This work supports the use of ultra-low-input proteomics to identify candidate drivers of disease phenotypes.  NNMT is a central, metabolic regulator of CAF differentiation and cancer progression in the stroma that may be therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlL_GCBb1LSbVg90H21EOLACvtfcHk0lgISoM3xaGNmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D&md5=43dbbcee11b5d21c5fb77f5b8ff02701</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1173-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1173-8%26sid%3Dliteratum%253Aachs%26aulast%3DEckert%26aufirst%3DM.%2BA.%26aulast%3DCoscia%26aufirst%3DF.%26aulast%3DChryplewicz%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHernandez%26aufirst%3DK.%2BM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DTienda%26aufirst%3DS.%2BM.%26aulast%3DNahotko%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DBla%25C5%25BEenovi%25C4%2587%26aufirst%3DI.%26aulast%3DLastra%26aufirst%3DR.%2BR.%26aulast%3DCurtis%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DS.%2BD.%26aulast%3DPerets%26aufirst%3DR.%26aulast%3DMcGregor%26aufirst%3DS.%2BM.%26aulast%3DAndrade%26aufirst%3DJ.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DLengyel%26aufirst%3DE.%26atitle%3DProteomics%2520Reveals%2520NNMT%2520as%2520a%2520Master%2520Metabolic%2520Regulator%2520of%2520Cancer-Associated%2520Fibroblasts%26jtitle%3DNature%26date%3D2019%26volume%3D569%26spage%3D723%26epage%3D728%26doi%3D10.1038%2Fs41586-019-1173-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhl, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">NNMT Promotes Epigenetic Remodeling in Cancer by Creating a Metabolic Methylation Sink</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+Promotes+Epigenetic+Remodeling+in+Cancer+by+Creating+a+Metabolic+Methylation+Sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0lh5af8VH4Iqrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520Promotes%2520Epigenetic%2520Remodeling%2520in%2520Cancer%2520by%2520Creating%2520a%2520Metabolic%2520Methylation%2520Sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guidi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zizzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliante, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase in Non-Small Cell Lung Cancer: Promising Results for Targeted Anti-Cancer Therapy</span>. <i>Cell Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">865</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1007/s12013-013-9574-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1007%2Fs12013-013-9574-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=23532607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKjtbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2013&pages=865-873&author=D.+Sartiniauthor=S.+Morgantiauthor=E.+Guidiauthor=C.+Rubiniauthor=A.+Zizziauthor=R.+Giulianteauthor=V.+Pozziauthor=M.+Emanuelli&title=Nicotinamide+N-Methyltransferase+in+Non-Small+Cell+Lung+Cancer%3A+Promising+Results+for+Targeted+Anti-Cancer+Therapy&doi=10.1007%2Fs12013-013-9574-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase in Non-small Cell Lung Cancer: Promising Results for Targeted Anti-cancer Therapy</span></div><div class="casAuthors">Sartini, Davide; Morganti, Stefano; Guidi, Elena; Rubini, Corrado; Zizzi, Antonio; Giuliante, Rachela; Pozzi, Valentina; Emanuelli, Monica</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">865-873</span>CODEN:
                <span class="NLM_cas:coden">CBBIFV</span>;
        ISSN:<span class="NLM_cas:issn">1085-9195</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Lung cancer, predominantly non-small cell lung cancer (NSCLC), is currently the most common cause of malignancy-related death in the world.  Despite advances in both detection and treatment, its incidence rate is still increasing.  Therefore, effective strategies for early detection as well as mol. therapeutic targets are urgently needed.  We focused on the enzyme nicotinamide N-methyltransferase (NNMT).  NNMT expression levels were investigated in tumor, tumor-adjacent, and surrounding tissue samples of 25 patients with NSCLC by Real-Time PCR, Western blot anal., and catalytic activity assay.  NNMT enzyme activity in NSCLC was then correlated with clinicopathol. characteristics.  Results obtained showed NNMT upregulation (mRNA and protein) in tumor compared with both tumor-adjacent and surrounding tissue.  Moreover, NSCLC displayed significantly higher activity levels than those detd. in both tumor-adjacent and surrounding tissue.  Interestingly, both tumor-adjacent and surrounding tissue samples of unfavorable cases (N+) seem to display higher activity levels than those of favorable NSCLCs (N0).  The present work shows a marked increase of NNMT enzyme activity in NSCLC and suggests that normal-looking tissue of unfavorable cases seems to change toward cancer.  Further studies may establish whether NNMT could represent a target for an effective anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi37Ur7tl8Y7Vg90H21EOLACvtfcHk0lh5af8VH4Iqrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKjtbnJ&md5=7fabae2de5172ba0611f4a995e85132f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs12013-013-9574-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12013-013-9574-z%26sid%3Dliteratum%253Aachs%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DMorganti%26aufirst%3DS.%26aulast%3DGuidi%26aufirst%3DE.%26aulast%3DRubini%26aufirst%3DC.%26aulast%3DZizzi%26aufirst%3DA.%26aulast%3DGiuliante%26aufirst%3DR.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DEmanuelli%26aufirst%3DM.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%253A%2520Promising%2520Results%2520for%2520Targeted%2520Anti-Cancer%2520Therapy%26jtitle%3DCell%2520Biochem.%2520Biophys.%26date%3D2013%26volume%3D67%26spage%3D865%26epage%3D873%26doi%3D10.1007%2Fs12013-013-9574-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohtake, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurosumi, M.</span></span> <span> </span><span class="NLM_article-title">Stat3 Up-Regulates Expression of Nicotinamide N-Methyltransferase in Human Cancer Cells</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1007/s00432-007-0318-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1007%2Fs00432-007-0318-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=17922140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsF2ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2008&pages=551-559&author=M.+Tomidaauthor=H.+Ohtakeauthor=T.+Yokotaauthor=Y.+Kobayashiauthor=M.+Kurosumi&title=Stat3+Up-Regulates+Expression+of+Nicotinamide+N-Methyltransferase+in+Human+Cancer+Cells&doi=10.1007%2Fs00432-007-0318-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Stat3 up-regulates expression of nicotinamide N-methyltransferase in human cancer cells</span></div><div class="casAuthors">Tomida, Mikio; Ohtake, Hideki; Yokota, Takashi; Kobayashi, Yasuhito; Kurosumi, Masafumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">551-559</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: To discover new mol. targets for cancer therapy and diagnosis, we surveyed signal transducers and activators of transcription 3 (Stat3)-regulated genes, because constitutive activation of Stat3 is assocd. with a wide variety of human malignancies.  Methods: We investigated the Stat3-regulated genes in 293 cells with cDNA microarray anal. and found that Nicotinamide N-methyltransferase (NNMT) was induced on stimulation of the cells with leukemia inhibitory factor.  We examd. the expression of NNMT in several types of cancer cells by real-time quant. RT-PCR.  To examine the role of Stat3, Hep-G2 hepatocellular carcinoma cells were transfected with NNMT promoter-luciferase reporter construct together with conditionally active Stat3 (Stat3ER) or dominant-neg. Stat3 expression vector and NNMT promoter activity was detd.  The expression of NNMT and activated Stat3 in 88 colon cancer tissues and 17 normal colon tissues was examd. with immunohistochem. anal.  Results: In Hep-G2 cells and SW480 colon cancer cells, NNMT expression increased on stimulation of the cells with interleukin 6.  NNMT promoter activity in Hep-G2 cells was dependent on the activation of Stat3.  MDA-MB-468 breast cancer cells and HT29 colon cancer cells expressed constitutively a high level of NNMT.  Treatment of these cells with Stat3 siRNA or curcumin, which inhibited Stat3 phosphorylation, resulted in redn. of the NNMT level.  We found a correlation between the expression of NNMT and activated Stat3 (P < 0.001) in the colon cancer tissues.  Conclusion: NNMT is a novel Stat3-regulated gene.  Its expression is enhanced with the activation of Stat3 in colon cancer tissues.  NNMT may be a potential candidate for a tumor marker of various kinds of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdYhc8dVgoOLVg90H21EOLACvtfcHk0lh_txQpa75NGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsF2ju7Y%253D&md5=db842717175e713e708cfb5a086cd424</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00432-007-0318-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-007-0318-6%26sid%3Dliteratum%253Aachs%26aulast%3DTomida%26aufirst%3DM.%26aulast%3DOhtake%26aufirst%3DH.%26aulast%3DYokota%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DY.%26aulast%3DKurosumi%26aufirst%3DM.%26atitle%3DStat3%2520Up-Regulates%2520Expression%2520of%2520Nicotinamide%2520N-Methyltransferase%2520in%2520Human%2520Cancer%2520Cells%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2008%26volume%3D134%26spage%3D551%26epage%3D559%26doi%3D10.1007%2Fs00432-007-0318-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span> <span> </span><span class="NLM_article-title">Down-Regulation of Nicotinamide N-Methyltransferase Induces Apoptosis in Human Breast Cancer Cells via the Mitochondria-Mediated Pathway</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e89202</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0089202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1371%2Fjournal.pone.0089202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=24558488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjs73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=J.+Zhangauthor=Y.+Wangauthor=G.+Liauthor=H.+Yuauthor=X.+Xie&title=Down-Regulation+of+Nicotinamide+N-Methyltransferase+Induces+Apoptosis+in+Human+Breast+Cancer+Cells+via+the+Mitochondria-Mediated+Pathway&doi=10.1371%2Fjournal.pone.0089202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulation of nicotinamide N-methyltransferase induces apoptosis in human breast cancer cells via the mitochondria-mediated pathway</span></div><div class="casAuthors">Zhang, Jun; Wang, Yanzhong; Li, Guiling; Yu, Haitao; Xie, Xinyou</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e89202/1-e89202/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been found involved in cell proliferation of several malignancies.  However, the functional role of NNMT in breast cancer has not been elucidated.  In the present study, we showed that NNMT was selectively expressed in some breast cancer cell lines, down-regulation of NNMT expression in Bcap-37 and MDA-MB-231 cell lines by NNMT shRNA significantly inhibited cell growth in vitro, decreased tumorigenicity in mice and induced apoptosis.  The silencing reciprocal effect of NNMT was confirmed by over-expressing NNMT in the MCF-7 and SK-BR-3 breast cancer cell lines which lack constitutive expression of NNMT.  In addn., down-regulation of NNMT expression resulted in reducing expression of Bcl-2 and Bcl-xL, up-regulation of Bax, Puma, cleaved caspase-9, cleaved caspase-3 and cleaved PARP, increasing reactive oxygen species prodn. and release of cytochrome c from mitochondria, and decreasing the phosphorylation of Akt and ERK1/2.  These data suggest that down-regulation of NNMT induces apoptosis via the mitochondria-mediated pathway in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw6YRvrhclrVg90H21EOLACvtfcHk0lgr5beoOGDSRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjs73O&md5=ba37d01dcc59df8ec842da9895bd3e57</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0089202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0089202%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DX.%26atitle%3DDown-Regulation%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Induces%2520Apoptosis%2520in%2520Human%2520Breast%2520Cancer%2520Cells%2520via%2520the%2520Mitochondria-Mediated%2520Pathway%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0089202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mateuszuk, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khomich, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Słomińska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wójcik, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Łomnicka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwóźdź, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chłopicki, S.</span></span> <span> </span><span class="NLM_article-title">Activation of Nicotinamide N-Methyltrasferase and Increased Formation of 1-Methylnicotinamide (MNA) in Atherosclerosis</span>. <i>Pharmacol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">76</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/s1734-1140(09)70009-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fs1734-1140%2809%2970009-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=19307695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFCku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=76-85&author=%C5%81.+Mateuszukauthor=T.+I.+Khomichauthor=E.+S%C5%82omi%C5%84skaauthor=M.+Gajdaauthor=L.+W%C3%B3jcikauthor=M.+%C5%81omnickaauthor=P.+Gw%C3%B3%C5%BAd%C5%BAauthor=S.+Ch%C5%82opicki&title=Activation+of+Nicotinamide+N-Methyltrasferase+and+Increased+Formation+of+1-Methylnicotinamide+%28MNA%29+in+Atherosclerosis&doi=10.1016%2Fs1734-1140%2809%2970009-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of nicotinamide N-methyltrasferase and increased formation of 1-methylnicotinamide (MNA) in atherosclerosis</span></div><div class="casAuthors">Mateuszuk, Lukasz; Khomich, Tamara I.; Slominska, Ewa; Gajda, Mariusz; Wojcik, Luiza; Lomnicka, Magdalena; Gwozdz, Pawel; Chlopicki, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">76-85</span>CODEN:
                <span class="NLM_cas:coden">PRHEDU</span>;
        ISSN:<span class="NLM_cas:issn">1734-1140</span>.
    
            (<span class="NLM_cas:orgname">Polish Academy of Sciences, Institute of Pharmacology</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NMMT) catalyzes the conversion of nicotinamide (NA) to 1-methylnicotinamide (MNA).  Recent studies have reported that exogenous MNA exerts anti-thrombotic and anti-inflammatory activity, suggesting that endogenous NMMT-derived MNA may play a biol. role in the cardiovascular system.  In the present study, we assayed changes in hepatic NNMT activity and MNA plasma levels along the progression of atherosclerosis in apoE/LDLR-/- mice, as compared to age-matched wild-type mice.  Atherosclerosis progression in apoE/LDLR-/- mice was quantified in aortic root, while hepatic NNMT activity and MNA plasma concns. were concomitantly measured in 2-, 3-, 4-, and 6-mo-old mice.  In apoE/LDLR-/- mice, atherosclerotic plaques developed in the aortic roots beginning at the age of 3 mo and gradually increased in size, macrophage content, and inflammation intensity over time, as detected by Oil-Red O staining, CD68 immunostaining, and in situ zymog. (MMP2/MMP9 activity).  Hepatic NNMT activity was upregulated approx. two-fold in apoE/LDLR-/- mice by the age of 2 mo, as compared to wild-type mice (1.03 ± 0.14 vs. 0.64 ± 0.23 pmol/min/mg, resp.).  MNA plasma concns. were also elevated approx. two-fold (0.30 ± 0.13 vs. 0.17 ± 0.04 μmol/l, resp.).  As atherosclerosis progressed, hepatic NMMT activity and MNA plasma concns. increased five-fold in 6-mo-old apoE/LDLR-/- mice at the stage of advanced atherosclerotic plaques (NMMT activity: 2.29 ± 0.34 pmol/min/mg, MNA concn.: 1.083 ± 0.33 μmol/l).  In summary, the present study demonstrated that the progression of vascular inflammation and atherosclerosis was assocd. with the upregulation of hepatic NNMT activity and subsequent increase in endogenous MNA plasma levels.  Given the anti-thrombotic and anti-inflammatory properties of exogenous MNA, robust activation of an endogenous NA-MNA pathway in atherosclerosis may play an important compensatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofiHh0pa0Le7Vg90H21EOLACvtfcHk0lgzd-kuYsdgPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFCku77O&md5=5a8337af5d9fedd50445989fced30a88</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fs1734-1140%2809%2970009-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1734-1140%252809%252970009-x%26sid%3Dliteratum%253Aachs%26aulast%3DMateuszuk%26aufirst%3D%25C5%2581.%26aulast%3DKhomich%26aufirst%3DT.%2BI.%26aulast%3DS%25C5%2582omi%25C5%2584ska%26aufirst%3DE.%26aulast%3DGajda%26aufirst%3DM.%26aulast%3DW%25C3%25B3jcik%26aufirst%3DL.%26aulast%3D%25C5%2581omnicka%26aufirst%3DM.%26aulast%3DGw%25C3%25B3%25C5%25BAd%25C5%25BA%26aufirst%3DP.%26aulast%3DCh%25C5%2582opicki%26aufirst%3DS.%26atitle%3DActivation%2520of%2520Nicotinamide%2520N-Methyltrasferase%2520and%2520Increased%2520Formation%2520of%25201-Methylnicotinamide%2520%2528MNA%2529%2520in%2520Atherosclerosis%26jtitle%3DPharmacol.%2520Rep.%26date%3D2009%26volume%3D61%26spage%3D76%26epage%3D85%26doi%3D10.1016%2Fs1734-1140%2809%2970009-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasouli, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsden, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-Methyltransferase Increases Complex i Activity in SH-SY5Y Cells via Sirtuin 3</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2015.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.bbrc.2015.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26456643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=491-496&author=K.+Y.+Liuauthor=R.+J.+Mistryauthor=C.+A.+Aguirreauthor=E.+S.+Fasouliauthor=M.+G.+Thomasauthor=F.+Klamtauthor=D.+B.+Ramsdenauthor=R.+B.+Parsons&title=Nicotinamide+N-Methyltransferase+Increases+Complex+i+Activity+in+SH-SY5Y+Cells+via+Sirtuin+3&doi=10.1016%2Fj.bbrc.2015.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase increases complex I activity in SH-SY5Y cells via sirtuin 3</span></div><div class="casAuthors">Liu, Karolina Y.; Mistry, Rakhee J.; Aguirre, Carlos A.; Fasouli, Eirini S.; Thomas, Martin G.; Klamt, Fabio; Ramsden, David B.; Parsons, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT, E.C. 2.1.1.1) N-methylates nicotinamide to 1-methylnicotinamide.  We have previously shown that NNMT is significantly overexpressed in the brains of patients who have died of Parkinson's disease, and others have shown that NNMT is significantly overexpressed in a variety of diseases ranging from cancer to hepatic cirrhosis.  In vitro overexpression has revealed many cytoprotective effects of NNMT, in particular increased complex I activity and ATP synthesis.  Although this appears to be mediated by an increase in 1-methylnicotinamide prodn., the mol. mechanisms involved remain unclear.  In the present study, we have investigated the role that sirtuins 1, 2 and 3, class III DNA deacetylase enzymes known to regulate mitochondrial energy prodn. and cell cycle, have in mediating the effects of NNMT upon complex I activity.  Expression of NNMT in SH-SY5Y human neuroblastoma cells, which have no endogenous expression of NNMT, significantly increased the expression of all three sirtuins.  SiRNA-mediated silencing of sirtuin 3 expression decreased complex I activity in NNMT-expressing SH-SY5Y cells to that obsd. in wild-type SH-SY5Y, and significantly reduced cellular ATP content also.  These results demonstrate that sirtuin 3 is a key mediator of NNMT-induced complex I activity and ATP synthesis.  These results further reinforce a central role for NNMT in the regulation of energy homeostasis and provide further mechanistic insight into the consequences of enhanced NNMT expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq27Ko18UYX_bVg90H21EOLACvtfcHk0lis3nVoU4lQQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cqu7rJ&md5=919f548abd38d02a68681a3b72b30791</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2015.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2015.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%2BY.%26aulast%3DMistry%26aufirst%3DR.%2BJ.%26aulast%3DAguirre%26aufirst%3DC.%2BA.%26aulast%3DFasouli%26aufirst%3DE.%2BS.%26aulast%3DThomas%26aufirst%3DM.%2BG.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DRamsden%26aufirst%3DD.%2BB.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26atitle%3DNicotinamide%2520N-Methyltransferase%2520Increases%2520Complex%2520i%2520Activity%2520in%2520SH-SY5Y%2520Cells%2520via%2520Sirtuin%25203%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2015%26volume%3D467%26spage%3D491%26epage%3D496%26doi%3D10.1016%2Fj.bbrc.2015.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5015</span>– <span class="NLM_lpage">5028</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&author=H.+Neelakantanauthor=H.-Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-Activity+Relationship+for+Small+Molecule+Inhibitors+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0ljivxRCHRpA4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.-Y.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-Activity%2520Relationship%2520for%2520Small%2520Molecule%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zope, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parveen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Inhibitor of Nicotinamide N-Methyltransferase for the Treatment of Metabolic Disorders</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3660</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22081-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1038%2Fs41598-018-22081-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=29483571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=3660&author=A.+Kanntauthor=S.+Rajagopalauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=S.+Swaminathanauthor=M.+S.+Hallurauthor=R.+Kristamauthor=R.+Elvertauthor=J.+Czechauthor=A.+Pfenningerauthor=C.+Rudolphauthor=H.+Schreuderauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+Birudukotaauthor=S.+Shaikauthor=B.+R.+Zopeauthor=R.+R.+Burriauthor=N.+N.+Anandauthor=M.+K.+Thakurauthor=M.+Singhauthor=R.+Parveenauthor=S.+Kandanauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=S.+Rufauthor=S.+Dhakshinamoorthy&title=A+Small+Molecule+Inhibitor+of+Nicotinamide+N-Methyltransferase+for+the+Treatment+of+Metabolic+Disorders&doi=10.1038%2Fs41598-018-22081-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule inhibitor of Nicotinamide N-methyltransferase for the treatment of metabolic disorders</span></div><div class="casAuthors">Kannt Aimo; Elvert Ralf; Czech Jorg; Pfenninger Anja; Rudolph Christine; Schreuder Herman; Ruf Sven; Kannt Aimo; Rajagopal Sridharan; Kadnur Sanjay Venkatachalapathi; Suresh Juluri; Bhamidipati Ravi Kanth; Swaminathan Srinivasan; Hallur Mahanandeesha Siddappa; Kristam Rajendra; Chandrasekar Devaraj Venkatapura; Mane Vishal Subhash; Birudukota Swarnakumari; Shaik Shama; Zope Bharat Ravindra; Burri Raghunadha Reddy; Anand Niranjan Naranapura; Thakur Manish Kumar; Singh Manvi; Parveen Reejuana; Kandan Saravanan; Mullangi Ramesh; Yura Takeshi; Gosu Ramachandraiah; Dhakshinamoorthy Saravanakumar</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3660</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a cytosolic enzyme that catalyzes the transfer of a methyl group from the co-factor S-adenosyl-L-methionine (SAM) onto the substrate, nicotinamide (NA) to form 1-methyl-nicotinamide (MNA).  Higher NNMT expression and MNA concentrations have been associated with obesity and type-2 diabetes.  Here we report a small molecule analog of NA, JBSNF-000088, that inhibits NNMT activity, reduces MNA levels and drives insulin sensitization, glucose modulation and body weight reduction in animal models of metabolic disease.  In mice with high fat diet (HFD)-induced obesity, JBSNF-000088 treatment caused a reduction in body weight, improved insulin sensitivity and normalized glucose tolerance to the level of lean control mice.  These effects were not seen in NNMT knockout mice on HFD, confirming specificity of JBSNF-000088.  The compound also improved glucose handling in ob/ob and db/db mice albeit to a lesser extent and in the absence of weight loss.  Co-crystal structure analysis revealed the presence of the N-methylated product of JBSNF-000088 bound to the NNMT protein.  The N-methylated product was also detected in the plasma of mice treated with JBSNF-000088.  Hence, JBSNF-000088 may act as a slow-turnover substrate analog, driving the observed metabolic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdHZ3bviJkhgaK-_ZudimnfW6udTcc2ebrGUe1W3FvpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D&md5=3f2306da18e0ab2a41a57f08ea9384fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22081-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22081-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DZope%26aufirst%3DB.%2BR.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26atitle%3DA%2520Small%2520Molecule%2520Inhibitor%2520of%2520Nicotinamide%2520N-Methyltransferase%2520for%2520the%2520Treatment%2520of%2520Metabolic%2520Disorders%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D3660%26doi%3D10.1038%2Fs41598-018-22081-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fast, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Development of a Suicide Inhibition-Based Protein Labeling Strategy for Nicotinamide N-Methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">613</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00211</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmt1Olt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=613-618&author=S.+Senauthor=S.+Mondalauthor=L.+Zhengauthor=A.+J.+Salingerauthor=W.+Fastauthor=E.+Weerapanaauthor=P.+R.+Thompson&title=Development+of+a+Suicide+Inhibition-Based+Protein+Labeling+Strategy+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facschembio.9b00211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Suicide Inhibition-Based Protein Labeling Strategy for Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Sen, Sudeshna; Mondal, Santanu; Zheng, Li; Salinger, Ari J.; Fast, Walter; Weerapana, Eranthie; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">613-618</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the S-adenosyl-L-methionine-dependent methylation of nicotinamide to form N-methylnicotinamide.  This enzyme detoxifies xenobiotics and regulates NAD+ biosynthesis.  Addnl., NNMT is overexpressed in various cancers.  Herein, we describe the first NNMT-targeted suicide substrates.  These compds., which include 4-chloropyridine and 4-chloronicotinamide, exploit the broad substrate scope of NNMT; methylation of the pyridine nitrogen enhances the electrophilicity of the C4 position, thereby promoting an arom. nucleophilic substitution by C159, a noncatalytic cysteine.  On the basis of this activity, we developed a suicide inhibition-based protein labeling strategy using an alkyne-substituted 4-chloropyridine that selectively labels NNMT in vitro and in cells.  In total, this study describes the first NNMT-directed activity-based probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBszUWf8y2FrVg90H21EOLACvtfcHk0lgoEUEQU-_6Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmt1Olt7Y%253D&md5=330147d34dbe6ebac795255b594ae4ee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00211%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DS.%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DSalinger%26aufirst%3DA.%2BJ.%26aulast%3DFast%26aufirst%3DW.%26aulast%3DWeerapana%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DDevelopment%2520of%2520a%2520Suicide%2520Inhibition-Based%2520Protein%2520Labeling%2520Strategy%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2019%26volume%3D14%26spage%3D613%26epage%3D618%26doi%3D10.1021%2Facschembio.9b00211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Covalent Inhibitors of Nicotinamide N-Methyltransferase (NNMT) Provide Evidence for Target Engagement Challenges in Situ</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fj.bmcl.2018.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=29731364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2682-2687&author=H.-Y.+Leeauthor=R.+M.+Suciuauthor=B.+D.+Horningauthor=E.+V.+Vinogradovaauthor=O.+A.+Ulanovskayaauthor=B.+F.+Cravatt&title=Covalent+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29+Provide+Evidence+for+Target+Engagement+Challenges+in+Situ&doi=10.1016%2Fj.bmcl.2018.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span></div><div class="casAuthors">Lee, Hsin-Yu; Suciu, Radu M.; Horning, Benjamin D.; Vinogradova, Ekaterina V.; Ulanovskaya, Olesya A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2682-2687</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide using S-adenosyl-L-methionine (SAM) as a Me donor and, through doing so, can modulate cellular methylation potential to impact diverse epigenetic processes.  NNMT has been implicated in a range of diseases, including cancer and metabolic disorders.  Potent, selective, and cell-active inhibitors would constitute valuable probes to study the biol. functions and therapeutic potential of NNMT.  The authors previously reported the discovery of electrophilic small mols. that inhibit NNMT by reacting with an active-site cysteine residue in the SAM-binding pocket.  Here, the authors have used activity-based protein profiling (ABPP)-guided medicinal chem. to optimize the potency and selectivity of NNMT inhibitors, culminating in the discovery of multiple alpha-chloroacetamide (αCA) compds. with sub-μM IC50 values in vitro and excellent proteomic selectivity in cell lysates.  However, these compds. showed much weaker inhibition of NNMT in cells, a feature that was not shared by off-targets of the αCAs.  The authors' results show the potential for developing potent and selective covalent inhibitors of NNMT, but also highlight challenges that may be faced in targeting this enzyme in cellular systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFjmALOSTRFbVg90H21EOLACvtfcHk0lj8Qm3nCs5nMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D&md5=9194d6ac5d1fd5a70683fb4af9cae74f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.-Y.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCovalent%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%2520Provide%2520Evidence%2520for%2520Target%2520Engagement%2520Challenges%2520in%2520Situ%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2682%26epage%3D2687%26doi%3D10.1016%2Fj.bmcl.2018.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Nicotinamide N-Methyltransferase Designed to Mimic the Methylation Reaction Transition State</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/c7ob01357d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1039%2Fc7ob01357d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+Nicotinamide+N-Methyltransferase+Designed+to+Mimic+the+Methylation+Reaction+Transition+State&doi=10.1039%2Fc7ob01357d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0lg85U7oKEYrWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fc7ob01357d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7ob01357d%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Designed%2520to%2520Mimic%2520the%2520Methylation%2520Reaction%2520Transition%2520State%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2Fc7ob01357d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0lgX0ExcwXGX0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lj_FenCx4wKbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0liyOi9O_2S2qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Substrate Recognition in Human Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7800</span>– <span class="NLM_lpage">7808</span>, <span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+Basis+of+Substrate+Recognition+in+Human+Nicotinamide+N-Methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0lgb_z_RXH9eQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520Basis%2520of%2520Substrate%2520Recognition%2520in%2520Human%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loring, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5524</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5524-5532&author=H.+S.+Loringauthor=P.+R.+Thompson&title=Kinetic+Mechanism+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.biochem.8b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Loring, Heather S.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">5524-5532</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the transfer of a Me group from S-adenosyl methionine to nicotinamide, pyridine and other structural analogs.  Aberrantly increased NNMT activity results in the depletion of S-adenosyl methionine (SAM), nicotinamide (NAM), and NAD+; NAM is required for NAD+ biosynthesis.  SAM depletion impairs the methylation potential of the cell, resulting in hypomethylated histones and an altered epigenetic profile.  In addn., decreased NAD+ levels neg. affect energy metab. by disrupting oxidative phosphorylation.  Due to its impact on epigenetic states and NAD+ levels, NNMT is implicated in cancer, neurodegenerative and metabolic diseases, making it an appealing target for therapeutic intervention.  To gain insights that would guide the design of inhibitors and activity-based probes, we performed detailed kinetic studies on human NNMT.  Herein, we report the kinetic mechanism of NNMT.  Our initial velocity, product inhibition, and dead-end analog inhibition studies collectively indicate that NNMT uses a rapid equil. ordered mechanism, where NNMT first binds SAM, which is followed by NAM.  Me transfer occurs and methylated NAM (MeNAM) and S-adenosyl homocysteine (SAH) are released consecutively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBKv-k7WIajbVg90H21EOLACvtfcHk0lidphFD30fZ4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM&md5=8c67d54bd73a7fe4ad3dc51411d12d1e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00775%26sid%3Dliteratum%253Aachs%26aulast%3DLoring%26aufirst%3DH.%2BS.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D5524%26epage%3D5532%26doi%3D10.1021%2Facs.biochem.8b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borch, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durst, H. D.</span></span> <span> </span><span class="NLM_article-title">Cyanohydridoborate Anion as a Selective Reducing Agent</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1971</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">2897</span>– <span class="NLM_lpage">2904</span>, <span class="refDoi"> DOI: 10.1021/ja00741a013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00741a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaE3MXksV2msLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1971&pages=2897-2904&author=R.+F.+Borchauthor=M.+D.+Bernsteinauthor=H.+D.+Durst&title=Cyanohydridoborate+Anion+as+a+Selective+Reducing+Agent&doi=10.1021%2Fja00741a013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydridoborate anion as a selective reducing agent</span></div><div class="casAuthors">Borch, Richard F.; Bernstein, Mark D.; Durst, H. Dupont</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1971</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2897-904</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Sodium cyanohydridoborate (NaBH3CN) reduces a wide variety of org. functional groups with remarkable selectivity.  The redn. of aldehydes and ketones is pH dependent, the reaction proceeding readily at pH 3-4.  Oximes are smoothly reduced to alkylhydroxylamines and enamines are reduced to amines under acid catalysis.  Reaction of an aldehyde or ketone with NH3, primary amine, or secondary amine at pH ∼7 in the presence of BH3CN- leads to primary, secondary, or tertiary amines, resp., via reductive amination of the carbonyl group.  Beaction of substituted pyruvic acids with NH3 and BH3CN- affords an excellent method for the synthesis of amino acids; 15N labeling can be accomplished by using 15NH3.  The hydrogens of BH3CN- can be readily exchanged for either D or T, thus permitting the synthesis of D- or T-labeled alcs., amines, and amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuf4qG_KVwrVg90H21EOLACvtfcHk0liWxWEc7NPmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3MXksV2msLs%253D&md5=0d4426a681222cd423971c97b20cf140</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja00741a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00741a013%26sid%3Dliteratum%253Aachs%26aulast%3DBorch%26aufirst%3DR.%2BF.%26aulast%3DBernstein%26aufirst%3DM.%2BD.%26aulast%3DDurst%26aufirst%3DH.%2BD.%26atitle%3DCyanohydridoborate%2520Anion%2520as%2520a%2520Selective%2520Reducing%2520Agent%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1971%26volume%3D93%26spage%3D2897%26epage%3D2904%26doi%3D10.1021%2Fja00741a013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/c5ob00120j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1039%2Fc5ob00120j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2Fc5ob00120j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0liWxWEc7NPmGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1039%2Fc5ob00120j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc5ob00120j%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2Fc5ob00120j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heck, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolley, J. P.</span></span> <span> </span><span class="NLM_article-title">Palladium-Catalyzed Vinylic Hydrogen Substitution Reactions with Aryl, Benzyl, and Styryl Halides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1972</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2320</span>– <span class="NLM_lpage">2322</span>, <span class="refDoi"> DOI: 10.1021/jo00979a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00979a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaE38XkslCqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1972&pages=2320-2322&author=R.+F.+Heckauthor=J.+P.+Nolley&title=Palladium-Catalyzed+Vinylic+Hydrogen+Substitution+Reactions+with+Aryl%2C+Benzyl%2C+and+Styryl+Halides&doi=10.1021%2Fjo00979a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium-catalyzed vinylic hydrogen substitution reactions with aryl, benzyl, and styryl halides</span></div><div class="casAuthors">Heck, R. F.; Nolley, J. P., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2320-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Aryl, benzyl, and styryl halides react with olefins in the presence of a hindered amine and a catalytic amt. of Pd metal to form vinylic derivs. in which the aryl, benzyl, or styryl group has replaced a vinylic H of the original olefin.  The reactions occur readily at 100°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG4X3ukMoCE7Vg90H21EOLACvtfcHk0liAbVYhzWjkPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE38XkslCqsrk%253D&md5=2e3ba21021999631fb0ea341346f11f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo00979a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00979a024%26sid%3Dliteratum%253Aachs%26aulast%3DHeck%26aufirst%3DR.%2BF.%26aulast%3DNolley%26aufirst%3DJ.%2BP.%26atitle%3DPalladium-Catalyzed%2520Vinylic%2520Hydrogen%2520Substitution%2520Reactions%2520with%2520Aryl%252C%2520Benzyl%252C%2520and%2520Styryl%2520Halides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1972%26volume%3D37%26spage%3D2320%26epage%3D2322%26doi%3D10.1021%2Fjo00979a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonogashira, K.</span></span> <span> </span><span class="NLM_article-title">Development of Pd-Cu Catalyzed Cross-Coupling of Terminal Acetylenes with Sp2-Carbon Halides</span>. <i>J. Organomet. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>653</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/s0022-328x(02)01158-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2Fs0022-328x%2802%2901158-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BD38Xktlyhurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=653&publication_year=2002&pages=46-49&author=K.+Sonogashira&title=Development+of+Pd-Cu+Catalyzed+Cross-Coupling+of+Terminal+Acetylenes+with+Sp2-Carbon+Halides&doi=10.1016%2Fs0022-328x%2802%2901158-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Pd-Cu catalyzed cross-coupling of terminal acetylenes with sp2-carbon halides</span></div><div class="casAuthors">Sonogashira, Kenkichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organometallic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">653</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">46-49</span>CODEN:
                <span class="NLM_cas:coden">JORCAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-328X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  The Pd-catalyzed cross coupling reactions between sp2-C halides and terminal acetylenes were independently reported by Heck, Cassar and the authors in 1975.  The former two methods were developed as an extension of the Heck reaction to an acetylenic CH-bond.  The authors' method was discovered based on the combination of Pd-catalyzed cross-coupling of sp2-C halides with terminal acetylenes and Cu-catalyzed alkynylation of metal complexes developed by the authors in systematic studies on transition metal acetylide chem.  The coupling reactions were used extensively as a reliable method for the synthesis of enyne-based acetylenic materials.  Some recent advances of the coupling are also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozYFGk4Kf6srVg90H21EOLACvtfcHk0lgsWi5oBA3m5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xktlyhurk%253D&md5=fd23c853c93b16bb3557bc8955243a93</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs0022-328x%2802%2901158-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0022-328x%252802%252901158-0%26sid%3Dliteratum%253Aachs%26aulast%3DSonogashira%26aufirst%3DK.%26atitle%3DDevelopment%2520of%2520Pd-Cu%2520Catalyzed%2520Cross-Coupling%2520of%2520Terminal%2520Acetylenes%2520with%2520Sp2-Carbon%2520Halides%26jtitle%3DJ.%2520Organomet.%2520Chem.%26date%3D2002%26volume%3D653%26spage%3D46%26epage%3D49%26doi%3D10.1016%2Fs0022-328x%2802%2901158-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i>-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0li0xt61lrH8IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1074%2Fjbc.m114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.m114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0ljhqz2MXevshQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.m114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibitors</span>.  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibitors%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0lgjdvuBcaumag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">A Rapid and Efficient Assay for the Characterization of Substrates and Inhibitors of Nicotinamide N-Methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5307</span>– <span class="NLM_lpage">5315</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+Van%0AHarenauthor=J.+Sastre+Tora%C3%B1oauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+Rapid+and+Efficient+Assay+for+the+Characterization+of+Substrates+and+Inhibitors+of+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0lhJzWVaiCCjwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520Rapid%2520and%2520Efficient%2520Assay%2520for%2520the%2520Characterization%2520of%2520Substrates%2520and%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavens, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mateus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wayne, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, A.</span></span> <span> </span><span class="NLM_article-title">Direct Measurement of Intracellular Compound Concentration by RapidFire Mass Spectrometry Offers Insights into Cell Permeability</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1177/1087057115604141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1177%2F1087057115604141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26336900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=156-164&author=L.+J.+Gordonauthor=M.+Allenauthor=P.+Arturssonauthor=M.+M.+Hannauthor=B.+J.+Leavensauthor=A.+Mateusauthor=S.+Readshawauthor=K.+Valkoauthor=G.+J.+Wayneauthor=A.+West&title=Direct+Measurement+of+Intracellular+Compound+Concentration+by+RapidFire+Mass+Spectrometry+Offers+Insights+into+Cell+Permeability&doi=10.1177%2F1087057115604141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of intracellular compound concentration by RapidFire mass spectrometry offers insights into cell permeability</span></div><div class="casAuthors">Gordon, Laurie J.; Allen, Morven; Artursson, Per; Hann, Michael M.; Leavens, Bill J.; Mateus, Andre; Readshaw, Simon; Valko, Klara; Wayne, Gareth J.; West, Andy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">156-164</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">One of the key challenges facing early stage drug discovery is understanding the commonly obsd. difference between the activity of compds. in biochem. assays and cellular assays.  Traditionally, indirect or estd. cell permeability measurements such as estns. from logP or artificial membrane permeability are used to explain the differences.  The missing link is a direct measurement of intracellular compd. concn. in whole cells.  This can, in some circumstances, be estd. from the cellular activity, but this may also be problematic if cellular activity is weak or absent.  Advances in sensitivity and throughput of anal. techniques have enabled us to develop a high-throughput assay for the measurement of intracellular compd. concn. for routine use to support lead optimization.  The assay uses a RapidFire-MS based readout of compd. concn. in HeLa cells following incubation of cells with test compd.  The initial assay validation was performed by ultra-high performance liq. chromatog. tandem mass spectrometry, and the assay was subsequently transferred to RapidFire tandem mass spectrometry.  Further miniaturization and optimization were performed to streamline the process, increase sample throughput, and reduce cycle time.  This optimization has delivered a semi-automated platform with the potential of prodn. scale compd. profiling up to 100 compds. per day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLd62-6firibVg90H21EOLACvtfcHk0li-M79MN92N8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVSrtLc%253D&md5=34a5d2a18c1a36973425d00b87112f20</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F1087057115604141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115604141%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DAllen%26aufirst%3DM.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeavens%26aufirst%3DB.%2BJ.%26aulast%3DMateus%26aufirst%3DA.%26aulast%3DReadshaw%26aufirst%3DS.%26aulast%3DValko%26aufirst%3DK.%26aulast%3DWayne%26aufirst%3DG.%2BJ.%26aulast%3DWest%26aufirst%3DA.%26atitle%3DDirect%2520Measurement%2520of%2520Intracellular%2520Compound%2520Concentration%2520by%2520RapidFire%2520Mass%2520Spectrometry%2520Offers%2520Insights%2520into%2520Cell%2520Permeability%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D156%26epage%3D164%26doi%3D10.1177%2F1087057115604141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antoshenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombrovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagna-Slater, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of Human H3K9 Methyltransferases</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e8570</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0008570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1371%2Fjournal.pone.0008570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20084102" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=H.+Wuauthor=J.+Minauthor=V.+V.+Luninauthor=T.+Antoshenkoauthor=L.+Dombrovskiauthor=H.+Zengauthor=A.+Allali-Hassaniauthor=V.+Campagna-Slaterauthor=M.+Vedadiauthor=C.+H.+Arrowsmithauthor=A.+N.+Plotnikovauthor=M.+Schapira&title=Structural+Biology+of+Human+H3K9+Methyltransferases&doi=10.1371%2Fjournal.pone.0008570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of human H3K9 methyltransferases</span></div><div class="casAuthors">Wu Hong; Min Jinrong; Lunin Vladimir V; Antoshenko Tatiana; Dombrovski Ludmila; Zeng Hong; Allali-Hassani Abdellah; Campagna-Slater Valerie; Vedadi Masoud; Arrowsmith Cheryl H; Plotnikov Alexander N; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e8570</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SET domain methyltransferases deposit methyl marks on specific histone tail lysine residues and play a major role in epigenetic regulation of gene transcription.  We solved the structures of the catalytic domains of GLP, G9a, Suv39H2 and PRDM2, four of the eight known human H3K9 methyltransferases in their apo conformation or in complex with the methyl donating cofactor, and peptide substrates.  We analyzed the structural determinants for methylation state specificity, and designed a G9a mutant able to tri-methylate H3K9.  We show that the I-SET domain acts as a rigid docking platform, while induced-fit of the Post-SET domain is necessary to achieve a catalytically competent conformation.  We also propose a model where long-range electrostatics bring enzyme and histone substrate together, while the presence of an arginine upstream of the target lysine is critical for binding and specificity.  ENHANCED VERSION: This article can also be viewed as an enhanced version in which the text of the article is integrated with interactive 3D representations and animated transitions.  Please note that a web plugin is required to access this enhanced functionality.  Instructions for the installation and use of the web plugin are available in Text S1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAEo3Y9d9dncRAV4D_Ins_fW6udTcc2eYD54o2TTl1Fbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FjtlGgsA%253D%253D&md5=2649472b1401c69035f99e0ca20822a3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008570%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLunin%26aufirst%3DV.%2BV.%26aulast%3DAntoshenko%26aufirst%3DT.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520Biology%2520of%2520Human%2520H3K9%2520Methyltransferases%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0008570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stivers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. G.</span></span> <span> </span><span class="NLM_article-title">A Transient Kinetic Analysis of PRMT1 Catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">7033</span>– <span class="NLM_lpage">7044</span>, <span class="refDoi"> DOI: 10.1021/bi200456u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200456u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7033-7044&author=Y.+Fengauthor=N.+Xieauthor=M.+Jinauthor=M.+R.+Stahleyauthor=J.+T.+Stiversauthor=Y.+G.+Zheng&title=A+Transient+Kinetic+Analysis+of+PRMT1+Catalysis&doi=10.1021%2Fbi200456u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A transient kinetic analysis of PRMT1 catalysis</span></div><div class="casAuthors">Feng, You; Xie, Nan; Jin, Miyeong; Stahley, Mary R.; Stivers, JamesT.; Zheng, Yujun George</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">7033-7044</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications (PTMs) are important strategies used by eukaryotic organisms to modulate their phenotypes.  One of the well-studied PTMs, Arg residue methylation, is catalyzed by protein arginine methyltransferases (PRMTs) with S-adenosyl-L-methionine (SAM) as the Me donor.  The functions of PRMTs have been broadly studied in different biol. processes and diseased states, but the mol. basis for Arg methylation is not well-defined.  Here, the authors report the transient-state kinetic anal. of rat PRMT1 catalysis.  The fast assocn. and dissocn. rates suggested that PRMT1 catalysis of histone H4 methylation followed a rapid equil. sequential kinetic mechanism.  The data provided direct evidence that the chem. of Me transfer is the major rate-limiting step and that binding of cofactors SAM or S-adenosyl-L-homocysteine (SAH) affects the assocn. and dissocn. of H4 with PRMT1.  Importantly, from stopped-flow fluorescence measurements, the authors identified a crit. kinetic step suggesting a pre-catalytic conformational transition induced by substrate binding.  These results provide new insights into the mechanism of Arg methylation and the rational design of PRMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpImJRI93ID4rVg90H21EOLACvtfcHk0liIkofMjhBmFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptFKltLc%253D&md5=f2562d7ece9b10754b551c2b6dec7cb1</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi200456u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200456u%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DStahley%26aufirst%3DM.%2BR.%26aulast%3DStivers%26aufirst%3DJ.%2BT.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26atitle%3DA%2520Transient%2520Kinetic%2520Analysis%2520of%2520PRMT1%2520Catalysis%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26spage%3D7033%26epage%3D7044%26doi%3D10.1021%2Fbi200456u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oudhoff, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tatlock, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couzens, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Awar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstenberger, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benn, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braam, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, F. M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">(R)-PFI-2 Is a Potent and Selective Inhibitor of SETD7 Methyltransferase Activity in Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12853</span>– <span class="NLM_lpage">12858</span>, <span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.-P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.-C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+S.+Braamauthor=F.+M.+V.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+Is+a+Potent+and+Selective+Inhibitor+of+SETD7+Methyltransferase+Activity+in+Cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0lgdbIxfpizEJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520Is%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520SETD7%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debler, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmack, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stavropoulos, P.</span></span> <span> </span><span class="NLM_article-title">A Glutamate/Aspartate Switch Controls Product Specificity in a Protein Arginine Methyltransferase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2073</span>, <span class="refDoi"> DOI: 10.1073/pnas.1525783113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1073%2Fpnas.1525783113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=26858449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2068-2073&author=E.+W.+Deblerauthor=K.+Jainauthor=R.+A.+Warmackauthor=Y.+Fengauthor=S.+G.+Clarkeauthor=G.+Blobelauthor=P.+Stavropoulos&title=A+Glutamate%2FAspartate+Switch+Controls+Product+Specificity+in+a+Protein+Arginine+Methyltransferase&doi=10.1073%2Fpnas.1525783113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A glutamate/aspartate switch controls product specificity in a protein arginine methyltransferase</span></div><div class="casAuthors">Debler, Erik W.; Jain, Kanishk; Warmack, Rebeccah A.; Feng, You; Clarke, Steven G.; Blobel, Gunter; Stavropoulos, Pete</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2073</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosoma brucei PRMT7 (TbPRMT7) is a protein arginine methyltransferase (PRMT) that strictly monomethylates various substrates, thus classifying it as a type III PRMT.  However, the mol. basis of its unique product specificity has remained elusive.  Here, we present the structure of TbPRMT7 in complex with its cofactor product S-adenosyl-L-homocysteine (AdoHcy) at 2.8 Å resoln. and identify a glutamate residue crit. for its monomethylation behavior.  TbPRMT7 comprises the conserved methyltransferase and β-barrel domains, an N-terminal extension, and a dimerization arm.  The active site at the interface of the N-terminal extension, methyltransferase, and β-barrel domains is stabilized by the dimerization arm of the neighboring protomer, providing a structural basis for dimerization as a prerequisite for catalytic activity.  Mutagenesis of active-site residues highlights the importance of Glu181, the second of the two invariant glutamate residues of the double E loop that coordinate the target arginine in substrate peptides/proteins and that increase its nucleophilicity.  Strikingly, mutation of Glu181 to aspartate converts TbPRMT7 into a type I PRMT, producing asym. dimethylarginine (ADMA).  Isothermal titrn. calorimetry (ITC) using a histone H4 peptide showed that the Glu181Asp mutant has markedly increased affinity for monomethylated peptide with respect to the WT, suggesting that the enlarged active site can favorably accommodate monomethylated peptide and provide sufficient space for ADMA formation.  In conclusion, these findings yield valuable insights into the product specificity and the catalytic mechanism of protein arginine methyltransferases and have important implications for the rational (re)design of PRMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj2myjluq1B7Vg90H21EOLACvtfcHk0lj2CPknJbpj1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7k%253D&md5=93f9d17569159c432b5801e2439c6df0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1525783113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1525783113%26sid%3Dliteratum%253Aachs%26aulast%3DDebler%26aufirst%3DE.%2BW.%26aulast%3DJain%26aufirst%3DK.%26aulast%3DWarmack%26aufirst%3DR.%2BA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DStavropoulos%26aufirst%3DP.%26atitle%3DA%2520Glutamate%252FAspartate%2520Switch%2520Controls%2520Product%2520Specificity%2520in%2520a%2520Protein%2520Arginine%2520Methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D2068%26epage%3D2073%26doi%3D10.1073%2Fpnas.1525783113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span> <span> </span><span class="NLM_article-title">Fast_dp Methods</span>.  <i>Zenodo</i>, <span class="NLM_year">2014</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DFast_dp%2520Methods%26jtitle%3DZenodo%26date%3D2014%26volume%3D55%26spage%3D81%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0li0eyPDPWqGwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgBXRHUPsIgbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Policarpo, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decultot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmič, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittakavi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panarese, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shair, M. D.</span></span> <span> </span><span class="NLM_article-title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9837</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01238</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01238" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9837&author=R.+L.+Policarpoauthor=L.+Decultotauthor=E.+Mayauthor=P.+Kuzmi%C4%8Dauthor=S.+Carlsonauthor=D.+Huangauthor=V.+Chuauthor=B.+A.+Wrightauthor=S.+Dhakshinamoorthyauthor=A.+Kanntauthor=S.+Raniauthor=S.+Dittakaviauthor=J.+D.+Panareseauthor=R.+Gaudetauthor=M.+D.+Shair&title=High-Affinity+Alkynyl+Bisubstrate+Inhibitors+of+Nicotinamide+N-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.9b01238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Policarpo, Rocco L.; Decultot, Ludovic; May, Elizabeth; Kuzmic, Petr; Carlson, Samuel; Huang, Danny; Chu, Vincent; Wright, Brandon A.; Dhakshinamoorthy, Saravanakumar; Kannt, Aimo; Rani, Shilpa; Dittakavi, Sreekanth; Panarese, Joseph D.; Gaudet, Rachelle; Shair, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9837-9873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM).  NNMT overexpression has been linked to diabetes, obesity, and various cancers.  In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT.  The reported nicotinamide-SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM Me transfer reaction.  NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor.  An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the Me transfer tunnel of NNMT with ideal shape complementarity.  The compds. reported in this work represent the most potent and selective NNMT inhibitors reported to date.  The rational design principle described herein could potentially be extended to other methyltransferase enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLrzDcwfc0ibVg90H21EOLACvtfcHk0ljdZJFFGswbDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWltrzM&md5=f62a8055840292563d07cd25b0fcb4c2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26aulast%3DPolicarpo%26aufirst%3DR.%2BL.%26aulast%3DDecultot%26aufirst%3DL.%26aulast%3DMay%26aufirst%3DE.%26aulast%3DKuzmi%25C4%258D%26aufirst%3DP.%26aulast%3DCarlson%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DChu%26aufirst%3DV.%26aulast%3DWright%26aufirst%3DB.%2BA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DDittakavi%26aufirst%3DS.%26aulast%3DPanarese%26aufirst%3DJ.%2BD.%26aulast%3DGaudet%26aufirst%3DR.%26aulast%3DShair%26aufirst%3DM.%2BD.%26atitle%3DHigh-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9837%26doi%3D10.1021%2Facs.jmedchem.9b01238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=342-346&author=C.+F.+Strattonauthor=M.+B.+Poulinauthor=Q.+Duauthor=V.+L.+Schramm&title=Kinetic+Isotope+Effects+and+Transition+State+Structure+for+Human+Phenylethanolamine+N-Methyltransferase&doi=10.1021%2Facschembio.6b00922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span></div><div class="casAuthors">Stratton, Christopher F.; Poulin, Myles B.; Du, Quan; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-346</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenylethanolamine N-methyltransferase (PNMT) catalyzes the S-adenosyl-L-methionine (SAM)-dependent conversion of norepinephrine to epinephrine.  Epinephrine has been assocd. with crit. processes in humans including the control of respiration and blood pressure.  Addnl., PNMT activity has been suggested to play a role in hypertension and Alzheimer's disease.  Here, labeled SAM substrates were used to measure primary methyl-14C and 36S, and secondary methyl-3H, 5'-3H, 5'-14C intrinsic kinetic isotope effects for human PNMT.  The transition state of human PNMT was modeled by matching kinetic isotope effects predicted via quantum chem. calcns. to intrinsic values.  The model showed that human PNMT catalyzes the formation of epinephrine through an early SN2 transition state in which Me transfer is rate-limiting.  This anal. provides information on the geometry and electrostatics of the human PNMT transition state structure, details that enable the design of transition state analogs as potential PNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBDuQMfV317Vg90H21EOLACvtfcHk0lhZWTlZQPN4tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL&md5=cadf5bbe757ceba13081bd8e63a2f1d6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00922%26sid%3Dliteratum%253Aachs%26aulast%3DStratton%26aufirst%3DC.%2BF.%26aulast%3DPoulin%26aufirst%3DM.%2BB.%26aulast%3DDu%26aufirst%3DQ.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DKinetic%2520Isotope%2520Effects%2520and%2520Transition%2520State%2520Structure%2520for%2520Human%2520Phenylethanolamine%2520N-Methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D342%26epage%3D346%26doi%3D10.1021%2Facschembio.6b00922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD','PDB','3ROD'); return false;">PDB: 3ROD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE','PDB','6PVE'); return false;">PDB: 6PVE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS','PDB','6PVS'); return false;">PDB: 6PVS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i84"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01255">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04447"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01255?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01255</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">General Procedures for the Synthesis of compound <b>S3</b>; NMR spectra of compounds <b>S3</b>, <b>6a–i</b>, <b>9a–d</b>, and <b>12a–b</b>; NMR, MALDI-MS, and HPLC spectra of compounds <b>1a–j</b>, <b>1a*</b>, <b>2a–d</b>, <b>2a*</b> and <b>3a–b</b>; kinetic parameter determination for NNMT; rapid dilution assay for compound LL320 (<b>2a</b>); selectivity study of LL320 (<b>2a</b>); superimposed cocrystal structures; cell permeability evaluation of TAT peptide as the positive control; cell permeability evaluation of <b>1a*</b>; cell permeability evaluation of <b>2a*</b>; and crystallography data and refinement statistics (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_001.pdf">jm9b01255_si_001.pdf (5.79 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01255/suppl_file/jm9b01255_si_002.csv">jm9b01255_si_002.csv (2.09 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structures of human NNMT in the complex of <b>LL319</b> and <b>LL320</b> have been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVE">6PVE</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6PVS">6PVS</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01255%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-23" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01255" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ab697f90a3c88","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
