
<html lang="en"     class="pb-page"  data-request-id="f341b3bc-a3f3-4cb4-8dc1-406afed1537b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02195;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-6;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors" /></meta><meta name="dc.Creator" content="Michael  Bauder" /></meta><meta name="dc.Creator" content="Christian  Meyners" /></meta><meta name="dc.Creator" content="Patrick L.  Purder" /></meta><meta name="dc.Creator" content="Stephanie  Merz" /></meta><meta name="dc.Creator" content="Wisely Oki  Sugiarto" /></meta><meta name="dc.Creator" content="Andreas M.  Voll" /></meta><meta name="dc.Creator" content="Tim  Heymann" /></meta><meta name="dc.Creator" content="Felix  Hausch" /></meta><meta name="dc.Description" content="The FK506-binding protein 51 (FKBP51) emerged as a key player in several diseases like stress-related disorders, chronic pain, and obesity. Linear analogues of FK506 called SAFit were shown to be h..." /></meta><meta name="Description" content="The FK506-binding protein 51 (FKBP51) emerged as a key player in several diseases like stress-related disorders, chronic pain, and obesity. Linear analogues of FK506 called SAFit were shown to be h..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 5, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02195" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02195" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02195" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02195" /></link>
        
    
    

<title>Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02195" /></meta><meta property="og:title" content="Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0009.jpeg" /></meta><meta property="og:description" content="The FK506-binding protein 51 (FKBP51) emerged as a key player in several diseases like stress-related disorders, chronic pain, and obesity. Linear analogues of FK506 called SAFit were shown to be highly selective for FKBP51 over its closest homologue FKBP52, allowing the proof-of-concept studies in animal models. Here, we designed and synthesized the first macrocyclic FKBP51-selective ligands to stabilize the active conformation. All macrocycles retained full FKBP51 affinity and selectivity over FKBP52 and the incorporation of polar functionalities further enhanced affinity. Six high-resolution crystal structures of macrocyclic inhibitors in complex with FKBP51 confirmed the desired selectivity-enabling binding mode. Our results show that macrocyclization is a viable strategy to target the shallow FKBP51 binding site selectively." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02195"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02195">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02195&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02195&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02195&amp;href=/doi/10.1021/acs.jmedchem.0c02195" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3320-3349</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02191" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02199" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Michael Bauder</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Bauder</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Bauder">Michael Bauder</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Meyners</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Meyners</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Meyners">Christian Meyners</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patrick L. Purder</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrick L. Purder</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrick+L.++Purder">Patrick L. Purder</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephanie Merz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephanie Merz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephanie++Merz">Stephanie Merz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wisely Oki Sugiarto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wisely Oki Sugiarto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wisely+Oki++Sugiarto">Wisely Oki Sugiarto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas M. Voll</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas M. Voll</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas+M.++Voll">Andreas M. Voll</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tim Heymann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tim Heymann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tim++Heymann">Tim Heymann</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Felix Hausch</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Felix Hausch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department Chemistry and Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +49-6151-16-21245. Email: <a href="/cdn-cgi/l/email-protection#c9a1a8bcbaaaa189bdbce4ada8bba4babda8adbde7adac"><span class="__cf_email__" data-cfemail="e981889c9a8a81a99d9cc48d889b849a9d888d9dc78d8c">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Felix++Hausch">Felix Hausch</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3710-8838" title="Orcid link">http://orcid.org/0000-0002-3710-8838</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02195&amp;href=/doi/10.1021%2Facs.jmedchem.0c02195" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3320–3349</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 5, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 December 2020</li><li><span class="item_label"><b>Published</b> online</span>5 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02195" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02195</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3320%26pageCount%3D30%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMichael%2BBauder%252C%2BChristian%2BMeyners%252C%2BPatrick%2BL.%2BPurder%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D6%26contentID%3Dacs.jmedchem.0c02195%26title%3DStructure-Based%2BDesign%2Bof%2BHigh-Affinity%2BMacrocyclic%2BFKBP51%2BInhibitors%26numPages%3D30%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3349%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02195"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1110</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02195" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of High-Affinity Macrocyclic FKBP51 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Bauder&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Meyners&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;L. Purder&quot;},{&quot;first_name&quot;:&quot;Stephanie&quot;,&quot;last_name&quot;:&quot;Merz&quot;},{&quot;first_name&quot;:&quot;Wisely&quot;,&quot;last_name&quot;:&quot;Oki Sugiarto&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;M. Voll&quot;},{&quot;first_name&quot;:&quot;Tim&quot;,&quot;last_name&quot;:&quot;Heymann&quot;},{&quot;first_name&quot;:&quot;Felix&quot;,&quot;last_name&quot;:&quot;Hausch&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3320-3349&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02195&quot;},&quot;abstract&quot;:&quot;The FK506-binding protein 51 (FKBP51) emerged as a key player in several diseases like stress-related disorders, chronic pain, and obesity. Linear analogues of FK506 called SAFit were shown to be highly selective for FKBP51 over its closest homologue FKBP52, allowing the proof-of-concept studies in animal models. Here, we designed and synthesized the first macrocyclic FKBP51-selective ligands to stabilize the active conformation. All macrocycles retained full FKBP51 affinity and selectivity over FKBP52 and the incorporation of polar functionalities further enhanced affinity. Six high-resolution crystal structures of macrocyclic inhibitors in complex with FKBP51 confirmed the desired selectivity-enabling binding mode. Our results show that macrocyclization is a viable strategy to target the shallow FKBP51 binding site selectively.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02195&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02195" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02195&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02195" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02195&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02195" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02195&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02195&amp;href=/doi/10.1021/acs.jmedchem.0c02195" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02195" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02195" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02195%26sid%3Dliteratum%253Aachs%26pmid%3D33666419%26genre%3Darticle%26aulast%3DBauder%26date%3D2021%26atitle%3DStructure-Based%2BDesign%2Bof%2BHigh-Affinity%2BMacrocyclic%2BFKBP51%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D6%26spage%3D3320%26epage%3D3349%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292017" title="Solvents">Solvents</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/jmcmar.2021.64.issue-6/20210325/jmcmar.2021.64.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The FK506-binding protein 51 (FKBP51) emerged as a key player in several diseases like stress-related disorders, chronic pain, and obesity. Linear analogues of FK506 called SAFit were shown to be highly selective for FKBP51 over its closest homologue FKBP52, allowing the proof-of-concept studies in animal models. Here, we designed and synthesized the first macrocyclic FKBP51-selective ligands to stabilize the active conformation. All macrocycles retained full FKBP51 affinity and selectivity over FKBP52 and the incorporation of polar functionalities further enhanced affinity. Six high-resolution crystal structures of macrocyclic inhibitors in complex with FKBP51 confirmed the desired selectivity-enabling binding mode. Our results show that macrocyclization is a viable strategy to target the shallow FKBP51 binding site selectively.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42111" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42111" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The FK506-binding protein 51 (FKBP51, encoded by the FKBP5 gene) is a peptidyl-prolyl isomerase (PPIase) that belongs to the immunophilin family and is best known for the modulation of the glucocorticoid receptor activity<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> and its participation as a cochaperone in the Hsp90 protein folding machinery.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> In numerous studies, FKBP51 was associated with stress-related mental disorders,<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10">(5−10)</a> obesity-induced diabetes,<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15">(11−15)</a> and several forms of chronic pain,<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> collectively suggesting FKBP51 as a new drug target.<a onclick="showRef(event, 'ref3 ref19'); return false;" href="javascript:void(0);" class="ref ref3 ref19">(3,19)</a></div><div class="NLM_p">The recent discovery of the first FKBP51-selective ligands (SAFit1 and SAFit2) provided powerful tools to study the role of FKBP51 in cellular and animal models.<a onclick="showRef(event, 'ref13 ref16 ref17 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref13 ref16 ref17 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27">(13,16,17,20−27)</a> A newly discovered transient binding pocket is the key to selectivity for FKBP51 over the highly homologous FKBP52.<a onclick="showRef(event, 'ref20 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref20 ref28 ref29 ref30 ref31">(20,28−31)</a> Nevertheless, SAFit1 and SAFit2 are far from becoming drug candidates due to their high molecular weight and the associated poor physicochemical properties.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Additionally, the binding site of FKBP51 is shallow and the creation of low molecular mass inhibitors with high ligand efficiency and sufficient potency is extremely challenging.<a onclick="showRef(event, 'ref28 ref33 ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref28 ref33 ref34 ref35 ref36">(28,33−36)</a></div><div class="NLM_p last">Macrocycles are thought to possess improved drug-relevant properties, which are especially important for difficult drug targets requiring ligands beyond the rule-of-five space.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42">(35−42)</a> Preorganizing the bioactive conformation in a ring structure is thought to improve affinity by reducing the entropic loss upon binding.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Moreover, macrocyclizations can improve cell permeability and provide enhanced proteolytic stability, thereby increasing bioavailability.<a onclick="showRef(event, 'ref39 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref41 ref42">(39,41,42)</a> However, the design of appropriate macrocycles is still far from trivial. Quite recently, macrocyclization strategies were crucial in the discovery of potent and orally bioavailable cyclophilin inhibitors.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> We here set out to investigate the suitability of linear FKBP51 ligands to macrocyclization and report on the discovery of two series of highly potent and selective macrocyclic FKBP51 inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Crystallographic Analysis and Design of the Macrocyclization Strategy</h3><div class="NLM_p">We used existing 3D information on high-affinity inhibitors in a complex with the FK1 domain of FKBP51 to identify promising attachment points for the design of macrocyclic SAFit analogues. The crystal structures of known, selective FKBP51 inhibitors served as templates for our structure-based rigidification strategy.<a onclick="showRef(event, 'ref20 ref28 ref31 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref20 ref28 ref31 ref33 ref34">(20,28,31,33,34)</a> SAFit analogues contain two aryl rings (A and B, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), which are consistently in close proximity when bound to FKBP51, as illustrated for the FKBP51-SAFit2 complex (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a>,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Structural analysis suggested the <i>meta</i>- or <i>para</i>-position of ring A and the <i>meta</i>-position of ring B as possible attachment points for linkers and modeling led us to postulate that macrocyclization within this structural framework would lead to macrocyclic ligands that could maintain all the binding interactions observed for the existing acyclic analogues.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structure of the selective FKBP51 ligands SAFit1 and SAFit2 and the macrocyclization strategy. (B) SAFit2 (red sticks) in complex with FKBP51 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a>); key interactions are highlighted (black dotted lines; distances in Å), and the macrocyclization strategy is outlined. Lys121 has been omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of <i>para</i>- and <i>meta</i>-Aryl-Substituted Cyclization Precursors</h3><div class="NLM_p">Since the precise requirements for the cyclizing linkers were unknown, we implemented a macrocyclization strategy that allows various linkers to be incorporated. The synthesis toward the cyclization precursors for <i>para</i>-aryl-linked macrocycles starts with commercially available phenol <b>1</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). After TBPDPS protection of the para hydroxyl group, the carboxylic acid <b>2</b> was preactivated to introduce the chiral auxiliary. The key step in this sequence is the introduction of the cyclohexene ring via α-alkylation of the enolate derived through deprotonation of <b>4</b>. Hydrogenation of the alkene followed by hydrolysis affords the chiral TBDPS-protected intermediate <b>6</b> in 33% yield over six steps. Amide coupling with amine <b>15</b> and TBAF-mediated desilylation provides the phenol <b>8</b>, which is further derivatized to the cyclization precursors (<b>9</b> - <b>12</b>) through alkylation with the corresponding allyl-containing building blocks.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>para</i>-Aryl-Substituted Cyclization Precursors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a) TBDPSCl, imidazole, DCM/DMF, rt; then K<sub>2</sub>CO<sub>3</sub>, THF/H<sub>2</sub>O, rt, 93%; b) C<sub>6</sub>F<sub>5</sub>OH, EDC, 4-DMAP, DCM, rt, 92%; c) (<i>S</i>)-4-phenyloxazolidin-2-one, <i>n</i>-BuLi, THF, 0 °C - rt, 81%; d) 3-bromocyclohexene, LiHMDS, THF, −78 °C - rt, 50%; e) Pd/C, H<sub>2</sub>, CH/EA, rt, 99%; f) LiOH, H<sub>2</sub>O<sub>2</sub>, THF/H<sub>2</sub>O, rt, 95%; g) <b>15</b>, HATU, DIPEA, DCM/DMF, rt, 88%; h) TBAF, THF, 0 °C - rt, 95%; (i) Bromide <i>or</i> tosylate, K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 92–98%; j) Allyl bromide, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 96%; k) (<i>S</i>)-Fmoc-Pip–OH, DCC, 4-DMAP, DCM, 0 °C - rt; then 4-methylpiperidine, DCM, rt, 67% (2 steps).</p></p></figure><div class="NLM_p">The synthesis of the cyclization precursors for <i>meta</i>-aryl-linked macrocycles turned out to be far more challenging (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) since five steps were necessary to prepare the hydroxy group in the <i>meta</i>-position of the aryl ring accessible for later derivatizations. <b>18</b> was synthesized over five steps in 51% yield from commercially available 3,4,5-trimethoxyphenylacetic acid following known procedures (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Scheme S1</a>).<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Methylation of the remaining hydroxy groups in <b>18</b> afforded <b>19</b> in a good yield of 93%. After substantial exploration of several alkylation conditions, we referred to a direct racemic alkylation protocol, where the carboxylic ester <b>19</b> is converted to the corresponding enolate with LiHMDS, alkylated with 3-bromocyclohexene, and the methyl ester <b>20</b> is afterward hydrolyzed in a mixture of aqueous LiOH and THF to generate carboxylic acid <b>21</b>. Conversion to the corresponding acyl chloride with SOCl<sub>2</sub> and subsequent Schotten–Baumann reaction with L-pipecolic acid and aqueous KOH in 1,4-dioxane provided the corresponding amide as a mixture of four diastereomers (not shown). Fortunately, the acyl chloride-activation/amidation sequence provides a good yield of 71% with an additional 28% of recovered <b>21</b> after hydrolysis of the Schotten–Baumann reaction. Pd/C-catalyzed hydrogenation affected alkene reduction and simultaneous deprotection of the benzyl ether to afford <b>22b</b> and desired <b>22a</b> as a 1:1 mixture of diastereomers that could be separated by silica gel chromatography (the assignment of diastereomers <b>22a</b>/<b>22b</b> to the desired (<i>S</i>) configuration is described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>). Alkylation to install the alkene for later RCM followed by ester cleavage afforded the carboxylic acids <b>23</b>–<b>26</b> in very good yields (81–96%) over two steps. After substantial optimization to minimize epimerization,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> esterification with alcohol <b>14</b> was achieved with EDC and an excess of 4-pyrrolidinopyridine, which afforded 59–64% of the desired esters <b>27</b>–<b>30</b> with low amounts of epimerization (<5%). Two key findings substantially contributed to the optimized conditions: (a) the excess use of 4-pyrrolidinopyridine, a more reactive analogue to standard DMAP, and (b) the use of toluene as a nonpolar solvent, probably due to decreased stabilization of the polar intermediates that lead to epimerization.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>meta</i>-Aryl-Substituted Cyclization Precursors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 93%; (b) 3-bromocyclohexene, LiHMDS, THF, −78 °C–rt, 94%; (c) LiOH, THF/H<sub>2</sub>O, rt, 91%; (d) SOCl<sub>2</sub>, DCM, reflux; then L-Pipecolic acid, KOH aq, 1,4-dioxane, 0 °C–rt, 71% + 28% of recovered substrate <b>21</b>; (e) Pd/C, H<sub>2</sub>, THF/MeOH, rt, 44% for <b>22b</b>, 41% for <b>22a</b>, separation of isomers by flash column chromatography; (f) bromide or tosylate, K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 87–99%; (g) LiOH, THF/MeOH/H<sub>2</sub>O, rt, 90–97%; (h) <b>14</b>, EDC, 4-(pyrrolidin-1-yl)pyridine, toluene, 0 °C–rt, 59–64%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Synthesis of Macrocyclic FKBP51 Ligands</h3><div class="NLM_p">The precursors for the <i>para</i>- (<b>9</b>–<b>12</b>) and the <i>meta</i>-series (<b>27</b>–<b>30</b>) were cyclized via ring-closing metathesis to give the macrocycles (<b>31</b>–<b>38</b>, <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The ring-closing metatheses always afforded mixtures of <i>E</i>/<i>Z</i>-alkene isomers, except for <b>35</b> and the longest linker of each series (<b>34</b> and <b>38</b>), where only the <i>E</i>-alkenes were observed. Fortunately, we were able to separate all <i>E</i>/<i>Z</i>-alkene mixtures and hence were able to test the influence of the alkene geometry on FKBP51 binding affinity. Hydrogenation of the alkenes afforded the saturated macrocycles (<b>39</b>–<b>46</b>).</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Pathways Towards Macrocyclic FKBP51 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Grubbs second generation catalyst (C<sub>46</sub>H<sub>65</sub>Cl<sub>2</sub>N<sub>2</sub>PRu), DCM, reflux, 33–78%; (b) Pd/C, H<sub>2</sub>, MeOH, rt, 37–45% (for <b>39</b> and <b>40</b>); (c) RhCl(PPh<sub>3</sub>)<sub>3</sub>, H<sub>2</sub>, toluene, or MeOH/THF, rt, 66–87%, (for <b>41</b>–<b>46</b>); (d) RhCl(PPh<sub>3</sub>)<sub>3</sub>, DABCO, EtOH/H<sub>2</sub>O, reflux, 16–47%; (e) I<sub>2</sub>, PPh<sub>3</sub>, imidazole, toluene or DCM, rt, 38–85%; (f) K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 26–81%; (g) Pd(OAc)<sub>2</sub>, <i>p</i>-BQ, HBF<sub>4</sub>, MeCN/H<sub>2</sub>O, rt, 33–67%; (h) OsO<sub>4</sub>, NMO, H<sub>2</sub>O/acetone, 0 °C–rt, 26–40%.</p></p></figure><div class="NLM_p">The shorter variants of the alkene macrocycles (<b>31-(</b><i>E</i><b>)</b> and <b>35-(</b><i>E</i><b>)</b>) were subjected to an OsO<sub>4</sub> dihydroxylation<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> to introduce additional functionalities into the macrocyclic frame. Fortunately, we were able to separate the resulting pair of isomers for the <i>meta</i>-series (<b>64a</b>/<b>64b</b>), but not for the <i>para</i>-series (<b>63</b>). Furthermore, the alkenes <b>31-(</b><i>E</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, <b>36-(</b><i>E</i><b>)</b>, and <b>37-(</b><i>E</i><b>)</b> were oxidized under Wacker conditions with palladium acetate and 1,4-benzoquinone<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> to obtain the corresponding ketones (<b>59</b>, <b>60</b>, <b>61a</b>/<b>61b</b>, and <b>62</b>). Interestingly, the oxidation of the short-chain alkenes (<b>31-(</b><i>E</i><b>)</b> and <b>35-(</b><i>E</i><b>)</b>) resulted in ketones <b>59</b> and <b>60</b> as the sole products of two possible regioisomers. Oxidation of the medium-length linker (<b>36-(</b><i>E</i><b>)</b>) provided major and minor product regioisomers (<b>61a</b> and <b>61b</b>), whereas oxidation of the longest linker (<b>37-(</b><i>E</i><b>)</b>) resulted in a single isolable product isomer (<b>62</b>).</div><div class="NLM_p last">The bis-O-allyl intermediates bearing an alkyl O-allyl moiety (<b>11</b>, <b>12</b>, <b>29</b>, and <b>30</b>) offered the potential for an additional approach toward cyclization. Global deallylation of these intermediates with Wilkinson catalyst yielded the diols (<b>47</b>–<b>50</b>), which were transformed into the corresponding monoiodides (<b>51</b>–<b>54</b>). Subsequent ring-closure was achieved through an intramolecular S<sub>N</sub>2 reaction<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> under high dilution to yield the smallest macrocycles of the <i>para</i>- (<b>55</b> and <b>56</b>) and <i>meta</i>-series (<b>57</b> and <b>58</b>). Taken together, we were able to synthesize 33 macrocycles of various ring sizes and linker compositions to test their binding to FKBPs.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structure-Affinity Relationship (SAR) of Macrocyclic FKBP51 Ligands</h3><div class="NLM_p">All macrocyclic compounds were tested in a competitive fluorescence polarization assay<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> for binding toward FKBP51 as well as the homologues FKBP12, FKBP12.6, and FKBP52 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). To compare the binding affinities of our constrained, macrocyclic ligands with a flexible analogue, we use the acyclic SAFit analogue <b>65</b> (structure shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which has the same number and type of nonhydrogen atoms, hydrogen bond donors, and acceptors and the same functional groups as macrocycles <b>55</b> and <b>57</b> and thereby served as a reference compound also for the larger macrocycles. Gratifyingly, almost all compounds bound to FKBP51 with a <i>K</i><sub>d</sub> < 150 nM, similar to the unfunctionalized, linear reference compound <b>65</b>. Interestingly, there appears to be a lower limit for the unfunctionalized macrocycles regarding FKBP51 binding affinity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinities of Macrocyclic FKBP51 Ligands (<i>para</i>-Linked Series) Determined by a Competitive Fluorescence Polarization Assay<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0007.gif" alt="" id="gr5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values are given as mean ± weighted standard deviation from at least two independent measurements.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">No binding can be observed within the range of solubility.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Tested as a diastereomeric mixture (dr = 34:66).</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinities of Macrocyclic FKBP51 Ligands (<i>meta</i>-Linked Series) Determined by a Competitive Fluorescence Polarization Assay<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0008.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values are given as mean ± weighted standard deviation from at least two independent measurements.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">No binding can be observed within the range of solubility.</p></div></div><div></div></div><div class="NLM_p">However, the introduction of polar groups increased affinity in several cases (e.g., <b>59</b>, <b>60</b>, <b>63</b>, or <b>64a</b>/<b>64b</b>). Gratifyingly, all compounds displayed a striking selectivity for FKBP51 over FKBP52, suggesting binding to a Phe67<sup>out</sup> conformation that was shown to be highly unfavorable for FKBP52.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the linker functionalization seemed to restore weak binding to FKBP52 for several macrocycles (e.g., <b>61a</b>, <b>63</b>, and <b>64a</b>). Moreover, a slight preference for FKBP51 over FKBP12 and FKBP12.6 was observed, similar to previous findings for acyclic SAFit analogues.<a onclick="showRef(event, 'ref20 ref28 ref33 ref34 ref52'); return false;" href="javascript:void(0);" class="ref ref20 ref28 ref33 ref34 ref52">(20,28,33,34,52)</a> This preference seems to be somewhat stronger for the <i>para</i>-linked series than for the <i>meta</i>-linked series.</div><div class="NLM_p last">The finding that the SAFit analogues are extremely tolerant to a variety of macrocyclic constraints was unexpected. Even the shortest linkers such as in <b>55</b> or <b>57</b>, which were designed to challenge the binding mode of the acyclic SAFit compounds, neither enhanced nor compromised binding.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cocrystal Structures and Binding Thermodynamics</h3><div class="NLM_p">To understand the molecular binding mode of our macrocyclic FKBP51 inhibitors and to evaluate the role of the linker, we solved cocrystal structures of six macrocycles in complex with the FK506-binding domain of FKBP51, covering a wide range of linker lengths and functionalization (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Figure S7</a>).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Both the 18-membered macrocycle <b>55</b>, which has the shortest linker of the <i>para</i>-series, and macrocyclic diol <b>64a</b> show binding modes consistent with those of linear FKBP51-selective ligands (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW6">4TW6</a>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW7">4TW7</a>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIT">5DIT</a>,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIU">5DIU</a>,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIV">5DIV</a>,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAF">6SAF</a>,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>), including the conformational rearrangement of Phe67, which underlies the strong selectivity vs the highly homologous FKBP52 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The crystal structure of <b>64a</b> also reveals that both linker hydroxy groups are solvent exposed and do not contact the protein surface. Gratifyingly, the conformations of the linkers in all cocrystal structures were well-defined (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Figures S1–S6</a>), although neither of them directly contacts FKBP51.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structures of the FK506-binding domain of FKBP51 (gray surface) in complex with the macrocyclic ligands <b>55</b> (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX">7AWX</a>, green sticks) and <b>64a</b> (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a>, blue sticks). Lys121 has been omitted for clarity. Hydrogen bonds to Ile87 and Tyr113 (yellow) are indicated as black dotted lines (distances in Å). Phe67 (yellow) has been displaced by the cyclohexyl moiety upon binding of the ligands. Novel linker moieties are highlighted in bright green and dark blue, respectively. Linker oxygens are highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A superposition of the cocrystallized macrocycles <b>35-(</b><i>E</i><b>)</b> and <b>64a</b> with cocrystallized SAFit2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and other SAFit analogues (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Figure S9</a>) revealed almost identical binding modes of the pipecolate core and the cyclohexyl ring. Superposed <b>35-(</b><i>E</i><b>)</b> shows a significant rotation of ring B (21.1°) due to the rigid <i>E</i>-configuration in the linker, which in turn affects the position of the ring C-bearing arm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The highly potent macrocycle <b>64a</b> shows an almost perfect overlay with SAFit2, only the A ring shows a minimal outward shift of 0.3 Å.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of the FK506-binding domain of FKBP51 (gray surface) in complex with the macrocyclic ligands <b>35-(</b><i>E</i><b>)</b> (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z">7B9Z</a>, yellow sticks) and <b>64a</b> (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a>, blue sticks) superposed on the cocrystal structure of SAFit2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a>, red sticks). Lys121 has been omitted for clarity. Novel linker moieties and rings A, B and C are highlighted in orange and dark blue, respectively. Linker oxygens are highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Macrocyclization restricts conformational freedom and may thus reduce the entropic penalty upon binding in a single active conformation. To explore the thermodynamic consequences of macrocyclization in more detail, we analyzed dihydroxylated macrocycle <b>64a</b> by isothermal titration calorimetry (ITC) and compared it to the acyclic ligand SAFit1.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The ITC data closely matched the affinity constants obtained by fluorescence polarization and revealed that binding of both compounds is enthalpically driven (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Figure S10</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Thermodynamic Parameters for Binding of Ligands SAFit1 and <b>64a</b> to FKBP51FK1, Obtained by ITC at 25 °C<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>G</i> [kcal/mol]</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>H</i> [kcal/mol]</th><th class="colsep0 rowsep0" align="center" char="±">–<i>T</i>Δ<i>S</i>[kcal/mol]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SAFit1</td><td class="colsep0 rowsep0" align="char" char="±">9.8 ± 3.6</td><td class="colsep0 rowsep0" align="char" char="±">–10.9 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">–11.2 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">–0.24 ± 0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64a</b></td><td class="colsep0 rowsep0" align="char" char="±">6.9 ± 5.6</td><td class="colsep0 rowsep0" align="char" char="±">–11.1 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">–11.3 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">–0.10 ± 0.10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Mean ± weighted standard deviation from three independent measurements is reported.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Macrocyclization is thought to work in part by reducing the conformational flexibility and in turn the entropic penalty of small molecules for adopting the active conformation. Since we did not observe an affinity enhancement for any of the unfunctionalized linkers, irrespective of linker size or linker rigidity, we conclude that the conformation of SAFit analogues previously observed in the crystal structures is already energetically favorable, e.g., by a hydrophobic collapse of the two A and B rings.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> In this circumstance, no energy gain can be expected by additional preorganization, as also confirmed by the results of the ITC experiments.</div><div class="NLM_p last">Remarkably, the introduction of polar modifications in the linker enhanced affinity in all cases. The macrocyclic linker, while not contacting the protein directly, thus seems to provide an efficient framework for the positioning of hydrophilic groups, as has been suggested as a recurring principle for macrocyclic natural products.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Information</h3><div class="NLM_p last">Air and water-sensitive reactions were performed under argon atmosphere with commercially available dry solvents. All commercially available chemicals and solvents were used as received. Solutions of <i>n</i>-butyllithium were titrated prior to use in toluene using 2-propanol with 1,10-phenanthroline as an indicator (red end point of titration; average of two determinations used). When indicated, solvents or reagents were degassed by sparging with argon. Chromatographic separations were performed by manual flash chromatography on silica (SiO<sub>2</sub>) from Macherey–Nagel (particle size 0.04–0.063 mm). Reverse-phase purifications were performed either with a Beckman System Gold 126 NMP programmable solvent module fitted with a Phenomenex Jupiter 4 μ Proteo 90 Å column (250 mm × 10 mm) and a Beckman System Gold 166 programmable detector module or a Interchim puriFlash 5.250 system fitted with a Uptisphere Strategy C18-HQ 5 μm column (250 mm × 21.2 mm). Eluents were 0.1% TFA in water (eluent A) and 0.1% TFA in acetonitrile (eluent B). Thin-layer chromatography (TLC) was performed on precoated aluminum plates with a fluorescence indicator from Merck (silica gel 60 F<sub>254</sub>). <sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, 2D HSQC, HMBC, COSY, and NOESY were obtained from the NMR Department of the Technical University of Darmstadt, on a Bruker DRX 500 spectrometer at room temperature, unless stated otherwise. Proton chemical shifts are expressed as parts per million (ppm, δ scale) and are referenced to residual solvent (<sup>1</sup>H: CDCl<sub>3</sub>, δ = 7.26; C<sub>6</sub>D<sub>6</sub>, δ = 7.16; DMSO-<i>d</i><sub>6</sub>, δ = 2.50; THF-<i>d</i><sub>8</sub>, δ = 3.58; MeOD, δ = 3.31. <sup>13</sup>C: CDCl<sub>3</sub>, δ = 77.16; C<sub>6</sub>D<sub>6</sub>, δ = 128.06; DMSO-<i>d</i><sub>6</sub>, δ = 39.52; THF-<i>d</i><sub>8</sub>, δ = 67.57; MeOD, δ = 49.00). Coupling constants (<i>J</i>) were given in hertz (Hz). LC–MS (liquid chromatography–mass spectrometry) measurements were performed either on an LC–MS system with a Beckmann Coulter System Gold 126 solvent module, Beckmann Coulter System Gold 508 autosampler, Beckman Coulter System 166 detector connected to a Thermo Finnigan LCQ Deca XP Plus or on an Agilent 1260 Infinity II System consisting of a 1260 Infinity II flexible pump, a vialsampler, a multicolumn thermostat, and a diode array detector connected to a 6125B MSD single quadrupole detector. Eluents were 0.1% formic acid in water (eluent A) and 0.1% formic acid in acetonitrile (eluent B). High-resolution mass spectra (HRMS) were obtained by the Mass Spectrometry Department of the Technical University of Darmstadt using a Bruker Daltonics Impact II mass spectrometer (quadrupole time-of-flight). Analytical HPLC analyses were performed on a Dionex P580 pump, Dionex ASI-100 automated sample injector fitted with a Phenomenex Kinetex 5 μm C18 100 Å column (250 mm × 4.6 mm), and a Dionex UCD 340U photodiode array detector (eluent A, 0.1% TFA in water; eluent B, 0.1% TFA in acetonitrile) or an Agilent 1260 Infinity II System consisting of a 1260 Infinity II flexible pump, a vialsampler, and a multicolumn thermostat fitted with a Poroshell 120 3 mm × 150 mm, 2.7 μm EC-C18 column or a Poroshell 120 50 mm × 2.1 mm, 1.9 μm EC-C18, a diode array detector, and a 6125B MSD single quadrupole detector (eluent A, 0.1% formic acid in water; eluent B, 0.1% formic acid in acetonitrile). Detailed measurement parameters (e.g., used gradient) were specified in the analytical data of the compounds. The pipecolic acid structural motive led to the presence of rotamers in the <sup>1</sup>H and <sup>13</sup>C NMR spectra of all linear and most of the macrocyclic compounds of this work. Due to this, the proton and carbon NMR spectra became fairly complex. When only two rotamers were present, we tried to distinguish between the major (indicated with <b>B</b>) and the minor rotamer (indicated with <b>A</b>) where possible, even though some signals overlapped. This applied to compounds <b>7</b>–<b>12</b>, <b>15</b>, <b>22a</b>, <b>22b</b>, <b>23</b>–<b>26</b>, <b>33-(</b><i>E</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, <b>36-(</b><i>Z</i><b>)</b>, <b>36-(</b><i>E</i><b>)</b>, <b>37-(</b><i>E</i><b>)</b>, <b>38-(</b><i>E</i><b>)</b>, <b>41</b>, <b>43</b>–<b>48</b>, <b>51</b>, <b>52</b>, <b>61a</b>, <b>61b</b>, and <b>62</b>. For compounds <b>42</b>, <b>58</b>, and <b>64a</b> the portion of the minor rotamer was too small (≤1:10, minor/major), and we have therefore only reported the major rotamer. If multiple rotamers were present, a detailed assignment was not possible, and we indicated signals originating from the major rotamer (alone or together with overlapping rotamer signals) with an <b>A</b>. This applied to compounds <b>27</b>–<b>30</b>, <b>49</b>, <b>50</b>, <b>53</b>, and <b>54</b>. Compounds <b>20</b> and <b>21</b> were mixtures of four diastereomers (stereocenter configurations: <i>RR</i>, <i>RS</i>, <i>SR</i>, <i>SS</i>), consisting of two pairs of enantiomers. Here the NMR signals were normalized to the major diastereomer, and all signals were reported. The geometry of the alkene bonds was determined by <sup>1</sup>H–<sup>1</sup>H-homonuclear decoupling of the two sets of vicinal methylene protons, providing informative coupling constants between the double bond protons for at least one of the two isomers. Compound <b>13</b> was prepared as previously described.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a><b>18</b> was synthesized over five steps from commercially available 3,4,5-trimethoxyphenylacetic acid.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> A dedicated reaction scheme as well as analytical data can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>. Diffraction data have been collected on BL14.1 at the BESSY II electron storage ring operated by the Helmholtz-Zentrum in Berlin (Germany),<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> the beamline PX at the Swiss Light Source (SLS) in Villigen (Switzerland),<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and the beamline P13 operated by EMBL Hamburg at the PETRA III storage ring (DESY) in Hamburg (Germany).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> A detailed description of the crystallization procedure, the structure solution, the refinement, as well as the refinement data are shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>. The purity of all tested compounds was ≥95% based on reverse phase HPLC (λ = 220 nm).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Experimental Procedures</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> General Procedure A (Alkylation with Bromides or Tosylates)</h4><div class="NLM_p last">To a stirred solution of the substrate and K<sub>2</sub>CO<sub>3</sub> in the solvent was added the corresponding bromide or tosylate, and the reaction mixture was heated at reflux until completion of the reaction. The resulting suspension was filtered through Celite and washed with EA (200 mL), and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure B (Ring-Closing Metathesis (RCM))</h4><div class="NLM_p last">A stirred solution of the substrate in DCM was degassed by sparging with argon for 15 min. Then Grubbs second generation catalyst and 1,4-benzoquinone (if indicated) were added and the mixture was heated at reflux. After completion of the reaction the mixture was either (a) filtered through silica and the solvent removed under reduced pressure or (b) quenched by the addition of tris(hydroxymethyl)phosphine solution (1 M in iPrOH) and washed with brine. The organic phase was dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography or semipreparative HPLC.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedure C (Nucleophilic Substitution with Iodine)</h4><div class="NLM_p last">To a stirred solution of the substrate, PPh<sub>3</sub>, and base in anhydrous solvent, iodine was added. The mixture was stirred at room temperature until completion of the reaction. The reaction mixture was diluted with DCM or Et<sub>2</sub>O (25 mL) and washed with aqueous Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (10 mL). The organic phase was washed with 1 M HCl solution and brine (10 mL each), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The crude product was purified by flash column chromatography or semipreparative HPLC.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure D (Intramolecular Nucleophilic Substitution)</h4><div class="NLM_p last">To a stirred solution of the substrate in MeCN was added K<sub>2</sub>CO<sub>3</sub>, and the mixture was degassed by sparging with argon for 15 min. The resulting mixture was then heated at reflux until completion of the reaction. After cooling to room temperature, the suspension was filtered through Celite and washed with EA (100 mL), and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography or semipreparative HPLC.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure E (Double Bond Hydrogenation)</h4><div class="NLM_p last">A stirred solution of the substrate in the solvent was degassed by sparging with Argon for 10 min. After the addition of Wilkinsońs catalyst [RhCl(PPh<sub>3</sub>)<sub>3</sub>] the solution was sparged with hydrogen for 15 min and was then stirred under a hydrogen atmosphere at room temperature. When the reaction was deemed complete by LC-MS analysis, the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography or semipreparative HPLC.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General Procedure F (Ester Hydrolysis)</h4><div class="NLM_p last">To a stirred solution of the substrate in H<sub>2</sub>O/MeOH/THF (v/v/v = 1:1:2) was added lithium hydroxide. The reaction mixture was stirred at room temperature until completion of the reaction and subsequently extracted with Et<sub>2</sub>O (2 × 50 mL). The organic layer was removed, and the aqueous layer was acidified with 1 M HCl solution (pH = 1–2). After extraction with Et<sub>2</sub>O (3 × 50 mL), the organic phase was washed with brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The crude product was purified by flash column chromatography.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General Procedure G (Esterification with Alcohol <b>14</b>)</h4><div class="NLM_p last">To a stirred solution of the substrate, <b>14</b> and 4-(1-pyrrolidinyl)pyridine in dry toluene at 0 °C was added EDC. After stirring for 1 h at 0 °C, the reaction mixture was allowed to reach ambient temperature and left stirring for the indicated time. During this time, the reaction was controlled via LC-MS and HPLC to monitor product formation and epimerization. Then the reaction mixture was acidified with 1 M HCl solution and extracted with DCM (3 × 30 mL). The organic phase was washed with brine, dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure H (Wacker Oxidation of Internal Alkenes)</h4><div class="NLM_p last">Palladium acetate (Pd(OAc)<sub>2</sub>) and 1,4-benzoquinone were charged in a flask under air. A mixture of MeCN and water (v/v = 9:1) was added, followed by the addition of aqueous HBF<sub>4</sub>. After the addition of the corresponding substrate, the mixture was stirred at room temperature until completion of the reaction. The reaction mixture was diluted with brine (20 mL) and extracted with DCM (3 × 30 mL). The combined organic phases were dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography or semipreparative HPLC.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-(4-((<i>tert</i>-Butyldiphenylsilyl)oxy)-3,5-dimethoxyphenyl)acetic Acid (<b>2</b>)</h3><div class="NLM_p last">3,5-Dimethoxy-4-hydroxyphenylacetic acid (10.00 g, 47.1 mmol, 1.00 equiv) and imidazole (12.83 g, 188.5 mmol, 4.00 equiv) were dissolved in a mixture of DCM/DMF (v/v = 2:1, 150 mL). TBDPSCl (38.87 g, 141.4 mmol, 3.00 equiv) is added dropwise and the reaction mixture was stirred at room temperature for 20 h. The reaction was quenched by the addition of water (50 mL) and stirred for 5 min. The layers were separated, and the organic phase was washed with water for another three times (100 mL). The organic layer was then washed with 1 M HCl and brine (50 mL each), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to give 51.2 g crude product of the bis-protected compound as a brown oil. The crude was dissolved in THF/H<sub>2</sub>O (v/v = 2:1, 150 mL), K<sub>2</sub>CO<sub>3</sub> (13.03 g, 94.2 mmol, 2.00 equiv) was added, and the reaction mixture was stirred at room temperature for 16 h. 0.1 M HCl was added to adjust a neutral pH, and the reaction mixture was extracted with Et<sub>2</sub>O (3 × 100 mL). The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 5:1) to yield <b>2</b> (19.81 g, 93%) as a white solid over two steps. TLC (CH/EA = 3:1 + 1% HCOOH, v/v): <i>R</i><sub><i>f</i></sub> = 0.31. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.78–7.64 (m, 4H), 7.40–7.29 (m, 6H), 6.35 (s, 2H), 3.51 (s, 2H), 3.43 (s, 6H), 1.11 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 178.0, 151.0, 135.3, 134.8, 133.8, 129.2, 127.1, 125.6, 106.4, 55.4, 41.3, 26.9, 20.3. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>26</sub>H<sub>31</sub>O<sub>5</sub>Si [M + H]<sup>+</sup>: 451.19, found 451.13. HPLC (5–95% sSolvent B, 1.0 mL/min, 19 min): <i>t</i><sub>R</sub> = 13.65 min, purity (254 nm) = 99%.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Perfluorophenyl 2-(4-((<i>tert</i>-butyldiphenylsilyl)oxy)-3,5-dimethoxyphenyl)acetate (<b>3</b>)</h3><div class="NLM_p last">To a stirred solution of <b>2</b> (19.81 g, 44.0 mmol, 1.00 equiv) and 2,3,4,5,6-pentafluorophenol (8.90 g, 48.4 mmol, 1.10 equiv) in DCM (220 mL, 0.2 M) were added EDC hydrochloride (9.27 g, 48.4 mmol, 1.10 equiv) and DMAP (1.61 g, 13.2 mmol, 0.30 equiv), and the reaction mixture was stirred at room temperature overnight. The reaction mixture was washed with 1 M HCl and brine (50 mL each), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 20:1) to yield <b>3</b> (24.90 g, 92%) as a colorless oil. TLC (CH/EA = 9:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.51. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.86–7.60 (m, 4H), 7.49–7.29 (m, 6H), 6.43 (s, 2H), 3.83 (s, 2H), 3.47 (s, 6H), 1.13 (s, 9H). Note: The carbon NMR signals of the pentafluorophenyl ring are broad and diffuse with low intensity. <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 167.7, 151.3, 135.3, 134.6, 134.1, 129.2, 127.1, 124.4, 106.1, 55.4, 40.5, 26.9, 20.3. HPLC (50–95% Solvent B, 1.0 mL/min, 19 min): <i>t</i><sub>R</sub> = 13.73 min, purity (254 nm) = 99%.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (<i>S</i>)-3-(2-(4-((<i>tert</i>-Butyldiphenylsilyl)oxy)-3,5-dimethoxyphenyl)acetyl)-4-phenyloxazo-lidin-2-one (<b>4</b>)</h3><div class="NLM_p last">(<i>S</i>)-4-Phenyloxazolidin-2-one (6.72 g, 40.8 mmol, 1.01 equiv) was dissolved in dry THF (350 mL, <i>c</i> = 0.1 M) and cooled in an ice bath to 0 °C. <i>n</i>-Butyllithium (2.3 M in hexanes, 17.72 mL, 40.8 mmol, 1.01 equiv) was added dropwise, and the reaction mixture was stirred for 1 h at 0 °C. Afterward, <b>3</b> (24.90 g, 40.4 mmol, 1.00 equiv) dissolved in dry THF (50 mL) was added to the above solution and stirred for 3 h at 0 °C and then overnight at room temperature. The reaction mixture was quenched by the addition of saturated NH<sub>4</sub>Cl solution, and the aqueous phase was extracted by Et<sub>2</sub>O (3 × 50 mL). The combined organic phases were dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 8:1) to yield <b>4</b> (19.58 g, 81%) as a white solid. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.49. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.77–7.61 (m, 4H), 7.41–7.25 (m, 9H), 7.15–7.08 (m, 2H), 6.31 (s, 2H), 5.40 (dd, <i>J</i> = 4.2, 8.8 Hz, 1H), 4.65 (t, <i>J</i> = 8.8 Hz, 1H), 4.30–4.12 (m, 2H), 4.05 (d, <i>J</i> = 14.4 Hz, 1H), 3.36 (s, 6H), 1.09 (s, 9H). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 170.8, 153.6, 150.8, 138.8, 135.3, 134.8, 133.5, 129.2, 129.2, 128.7, 127.1, 126.0, 125.5, 106.6, 69.9, 57.9, 55.3, 41.7, 26.9, 20.3. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>35</sub>H<sub>38</sub>NO<sub>6</sub>Si [M + H]<sup>+</sup>: 596.25, found 596.11. HPLC (50–95% solvent B, 1.0 mL/min, 19 min): <i>t</i><sub>R</sub> = 12.52 min, purity (254 nm) = 98%.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (<i>S</i>)-3-((<i>S</i>)-2-(4-((<i>tert</i>-Butyldiphenylsilyl)oxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetyl)-4-phenyloxazolidin-2-one (<b>5</b>)</h3><div class="NLM_p last">To a stirred solution of <b>4</b> (19.58 g, 32.9 mmol, 1.0 equiv) in dry THF (500 mL) was added LiHMDS (1 M in THF, 39.4 mL, 1.20 equiv) at 0 °C, and the reaction mixture was stirred at this temperature for 1 h. The mixture was then cooled to −78 °C and 3-bromocyclohex-1-ene (11.14 g, 65.7 mmol, 2.00 equiv) was added dropwise. The reaction mixture was stirred at −78 °C for 5 h and was then allowed to warm to room temperature overnight. Then the reaction mixture was quenched by the addition of saturated NH<sub>4</sub>Cl solution (100 mL), and the aqueous phase was extracted by Et<sub>2</sub>O (3 × 40 mL). The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 10:1) to yield <b>S1</b> (11.12 g, 50%) as white solid and a mixture of diastereomers (analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>). Subsequently, a stirred solution of <b>S1</b> (11.09 g, 16.41 mmol, 1.00 equiv) in a mixture of CH/EA/MeOH (v/v/v = 2:1:1, 400 mL) was degassed by sparging with argon for 15 min. After addition of Pd/C (10 wt %, 1.74 g, 1.64 mmol, 0.10 equiv), the solution was sparged with hydrogen for 15 min and was then stirred under a hydrogen atmosphere at room temperature overnight. The dark suspension was filtered through Celite, and the solvent was removed under reduced pressure to yield <b>5</b> (11.10 g, 99%) as white solid without further purification. TLC (CH/EA = 9:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.76–7.60 (m, 4H), 7.43–7.26 (m, 11H), 6.45 (s, 2H), 5.33 (dd, <i>J</i> = 3.4, 8.7 Hz, 1H), 4.69 (d, <i>J</i> = 10.6 Hz, 1H), 4.54 (t, <i>J</i> = 8.8 Hz, 1H), 4.19 (dd, <i>J</i> = 3.5, 8.8 Hz, 1H), 3.42 (s, 6H), 1.95–1.80 (m, 1H), 1.66–1.44 (m, 4H), 1.39–1.28 (m, 1H), 1.20–1.04 (m, 12H), 0.96–0.83 (m, 1H), 0.79–0.59 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 174.1, 153.6, 150.9, 139.6, 135.4, 134.6, 133.7, 130.0, 129.2, 129.1, 128.8, 127.0, 126.0, 106.1, 69.5, 58.3, 55.4, 54.5, 42.0, 31.7, 30.2, 26.9, 26.5, 26.0, 25.9, 20.2. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>48</sub>NO<sub>6</sub>Si [M + H]<sup>+</sup>: 678.33, found 678.29. HPLC (80–95% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 13.33 min, purity (220 nm) = 99%.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (<i>S</i>)-2-(4-((<i>tert</i>-butyldiphenylsilyl)oxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetic acid (<b>6</b>)</h3><div class="NLM_p last">To a stirred solution of <b>5</b> (11.21 g, 16.54 mmol, 1.00 equiv) in THF/H<sub>2</sub>O (v/v = 8:5, 130 mL) at 0 °C under air is added lithium hydroxide (792 mg, 33.07 mmol, 2.00 equiv) and H<sub>2</sub>O<sub>2</sub> (30% w/w in water, 8.45 mL, 82.68 mmol, 5.00 equiv). After stirring overnight at room temperature, the reaction mixture was quenched by the addition of 1.5 M Na<sub>2</sub>SO<sub>3</sub> solution (50 mL) and extracted with Et<sub>2</sub>O (3 × 20 mL). The aqueous phase was acidified with 1 M HCl (pH < 2) and extracted with Et<sub>2</sub>O (3 × 30 mL). The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 8:1 + 1% HCOOH) to afford <b>6</b> (8.36 g, 95%) as white solid. TLC (CH/EA = 5:1 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.40. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.71–7.65 (m, 4H), 7.36–7.30 (m, 2H), 7.30–7.25 (m, 4H), 6.36 (s, 2H), 3.41 (s, 6H), 3.02 (d, <i>J</i> = 10.5 Hz, 1H), 1.89–1.80 (m, 2H), 1.76–1.70 (m, 1H), 1.68–1.58 (m, 2H), 1.33–1.24 (m, 2H), 1.17–1.08 (m, 11H), 1.06–0.96 (m, 1H), 0.73–0.58 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 179.6, 151.0, 135.4, 134.6, 133.9, 129.8, 129.1, 127.1, 105.6, 58.8, 55.4, 41.1, 32.1, 30.3, 26.9, 26.5, 26.1, 20.2. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>32</sub>H<sub>41</sub>O<sub>5</sub>Si [M + H]<sup>+</sup>: 533.2718, found 533.2718. HPLC (70–100% solvent B, 0.8 mL/min, 2 min): <i>t</i><sub>R</sub> = 1.81 min, purity (220 nm) = 98%.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl-1-((<i>S</i>)-2-(4-((<i>tert</i>-butyldiphenyl-silyl)oxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>7</b>)</h3><div class="NLM_p last">To a stirred solution of <b>6</b> (1.82 g, 3.42 mmol, 1.00 equiv) in a mixture of DCM/DMF (v/v = 1:1, 100 mL) at room temperature were added HATU (1.37 g, 3.59 mmol, 1.05 equiv) and DIPEA (1.33 g, 10.25 mmol, 3.00 equiv) and stirred for 30 min, before <b>15</b> (1.65 g, 3.76 mmol, 1.10 equiv) was added. After stirring overnight, additional portions of HATU (0.20 equiv), DIPEA (1.00 equiv) and <b>15</b> (0.10 equiv) were added and the reaction was stirred for another 4 h. Then the reaction mixture was diluted with DCM (50 mL), washed with 1 M HCl and H<sub>2</sub>O (50 mL each). The organic phase was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 10:1) to yield <b>7</b> (2.86 g, 88%) as colorless resin. TLC (CH/EA = 5:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.37. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.76:1, <b>A</b>/<b>B</b>): δ 7.75–7.68 (m, 2.92H, <b>A</b>+<b>B</b>), 7.68–7.65 (m, 1.15H, <b>A</b>+<b>B</b>), 7.40–7.21 (m, 6.92H, <b>A</b>+<b>B</b>), 7.16 (t, <i>J</i> = 7.9 Hz, 0.57H, <b>A</b>), 6.98–6.87 (m, 1.33H, <b>A</b>+<b>B</b>), 6.84–6.75 (m, 2.21H, <b>A</b>+<b>B</b>), 6.74–6.70 (m, 0.44H, <b>A</b>), 6.69–6.67 (m, 0.44H, <b>A</b>), 6.67–6.63 (m, 1.16H, <b>B</b>), 6.57–6.53 (m, 0.56H, <b>A</b>), 6.29 (d, <i>J</i> = 1.0 Hz, 1.15H, <b>B</b>), 6.24 (d, <i>J</i> = 1.1 Hz, 0.87H, <b>A</b>), 6.14–5.98 (m, 0.94H, <b>A</b>+<b>B</b>), 5.82 (t, <i>J</i> = 6.9 Hz, 0.42H, <b>A</b>), 5.63 (dd, <i>J</i> = 5.5, 8.0 Hz, 0.56H, <b>B</b>), 5.49–5.43 (m, 1.10H, <b>B</b>), 5.43–5.38 (m, 0.47H, <b>A</b>), 5.35–5.25 (m, 0.99H, <b>A</b>+<b>B</b>), 4.73–4.67 (m, 0.42H, <b>A</b>), 4.60–4.47 (m, 2.51H, <b>A</b>+<b>B</b>), 3.88 (d, <i>J</i> = 1.5 Hz, 3.00H, <b>A</b>+<b>B</b>), 3.86 (d, <i>J</i> = 2.2 Hz, 3.88H, <b>A</b>+<b>B</b>), 3.44 (s, 2.71H, <b>A</b>+<b>B</b>), 3.29 (s, 3.34H, <b>A</b>+<b>B</b>), 3.27 (d, <i>J</i> = 10.0 Hz, 0.83H, <b>A</b>+<b>B</b>), 2.93 (d, <i>J</i> = 9.4 Hz, 0.43H, <b>A</b>), 2.79 (m, 0.54H, <b>B</b>), 2.67–2.41 (m, 2.48H, <b>A</b>+<b>B</b>), 2.34–2.23 (m, 1.04H, <b>A</b>+<b>B</b>), 2.18–2.09 (m, 0.43H, <b>A</b>), 2.06–1.95 (m, 2.11H, <b>A</b>+<b>B</b>), 1.94–1.84 (m, 1.71H, <b>A</b>+<b>B</b>), 1.75–1.50 (m, 6.79H, <b>A</b>+<b>B</b>), 1.47–1.22 (m, 6.04H, <b>A</b>+<b>B</b>), 1.18–1.13 (m, 5.74H, <b>A</b>+<b>B</b>), 1.13–1.09 (m, 6.60H, <b>A</b>+<b>B</b>), 1.04–0.77 (m, 6.03H, <b>A</b>+<b>B</b>), 0.73–0.62 (m, 1.13H, <b>A</b>+<b>B</b>), 0.61–0.51 (m, 0.46H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 4:5, <b>A</b>:<b>B</b>): δ 172.62 (<b>B</b>), 172.18 (<b>A</b>), 170.76 (<b>A</b>), 170.57 (<b>B</b>), 158.95 (<b>A</b>), 158.62 (<b>B</b>), 151.32 (<b>A</b>), 150.94 (<b>B</b>), 149.12 (<b>A</b>), 148.96 (<b>B</b>), 147.63 (<b>A</b>), 147.39 (<b>B</b>), 141.94 (<b>B</b>), 141.61 (<b>A</b>), 135.46 (<b>A</b>), 135.44 (<b>A</b>), 135.34 (<b>B</b>), 134.56 (<b>B)</b>, 134.52 (<b>B</b>), 134.26 (<b>A</b>), 134.22 (<b>A</b>), 133.78 (<b>A</b>), 133.46 (<b>A</b>), 133.44 (<b>B</b>), 133.37 (<b>A</b>), 133.32(<b>B</b>), 133.17 (<b>B</b>), 131.40 (<b>A</b>), 130.42 (<b>B</b>), 129.79 (<b>A</b>), 129.67 (<b>B</b>), 129.14 (<b>B</b>), 129.03 (<b>A</b>), 129.00 (<b>A</b>), 126.97 (<b>A</b>+<b>B</b>), 120.41 (<b>B</b>), 120.24 (<b>A</b>), 119.24 (<b>A</b>), 118.84 (<b>B</b>), 117.85 (<b>A</b>), 117.64 (<b>B</b>), 114.40 (<b>A</b>), 114.13 (<b>B</b>), 113.31 (<b>A</b>), 113.08 (<b>B</b>), 111.98 (<b>B</b>), 111.88 (<b>A</b>), 111.57 (<b>A</b>), 111.51 (<b>B</b>), 105.66 (<b>B</b>), 105.02 (<b>A</b>), 76.78 (<b>A)</b>, 75.74 (<b>B</b>), 68.91 (<b>A</b>), 68.86 (<b>B</b>), 56.08 (<b>A</b>), 56.05 (<b>B</b>), 55.99 (<b>A</b>), 55.96 (<b>B</b>), 55.86 (<b>A</b>), 55.76 (<b>B</b>), 55.54 (<b>A</b>+<b>B</b>), 55.27 (<b>B</b>), 55.21 (<b>A</b>), 52.20 (<b>B</b>), 43.65 (<b>B</b>), 41.36 (<b>B</b>), 40.91 (<b>A</b>), 39.50 (<b>A</b>), 38.14 (<b>B</b>), 37.98 (<b>A</b>), 33.31 (<b>A</b>), 33.03 (<b>A</b>), 31.57 (<b>B</b>), 31.16 (<b>A</b>), 30.63 (<b>B</b>), 30.49 (<b>B</b>), 26.96 (<b>A</b>), 26.88 (<b>A</b>+<b>B</b>), 26.77 (<b>A</b>+<b>B</b>), 26.64 (<b>B</b>), 26.63 (<b>A</b>), 26.42 (<b>A</b>), 26.39 (<b>B</b>), 26.35 (<b>B</b>), 26.26 (<b>A</b>), 25.62 (<b>B</b>), 24.51 (<b>A</b>), 21.10 (<b>B</b>), 20.90 (<b>A</b>), 20.16 (<b>A</b>+<b>B</b>). LC-MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>58</sub>H<sub>72</sub>NO<sub>9</sub>Si [M + H]<sup>+</sup>: 954.50, found 954.28. HPLC (90–95% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 10.91 min, purity (220 nm) ≥ 99%.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(4-hydroxy-3,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>8</b>)</h3><div class="NLM_p last">To a stirred solution of <b>7</b> (2.86 g, 3.00 mmol, 1.00 equiv) in THF (50 mL, <i>c</i> = 0.06 M) at 0 °C was added TBAF (1 M in THF, 3.00 mL, 3.00 mmol, 1.00 equiv), and the reaction was allowed to warm to room temperature and stirred overnight. The reaction was quenched by the addition of H<sub>2</sub>O (20 mL) and extracted with Et<sub>2</sub>O (3 × 10 mL). The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 4:1) to afford <b>8</b> (2.04 g, 95%) as light pink solid. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.58. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.37:1, <b>A</b>/<b>B</b>): δ 7.28 (t, <i>J</i> = 7.9 Hz, 0.28H, <b>A</b>), 7.10 (t, <i>J</i> = 7.9 Hz, 0.73H, <b>B</b>), 6.97–6.91 (m, 0.59H, <b>A</b>+<b>B</b>), 6.89–6.85 (m, 0.3H, <b>A</b>), 6.82–6.74 (m, 1.81H, <b>A</b>+<b>B</b>), 6.73–6.67 (m, 1.06H, <b>A</b>+<b>B</b>), 6.66–6.59 (m, 1.81H, <b>A</b>+<b>B</b>), 6.50–6.41 (m, 2.83H, <b>A</b>+<b>B</b>), 6.10–5.96 (m, 0.95H, <b>A</b>+<b>B</b>), 5.81 (dd, <i>J</i> = 6.3, 7.7 Hz, 0.27H, <b>A</b>), 5.58 (dd, <i>J</i> = 5.6, 8.0 Hz, 0.74H, <b>B</b>), 5.48–5.34 (m, 2.70H, <b>A</b>+<b>B</b>), 5.31–5.23 (m, 1.01H, <b>A</b>+<b>B</b>), 4.71 (d, <i>J</i> = 5.6 Hz, 0.27H, <b>A</b>), 4.61–4.45 (m, 2.40H, <b>A</b>+<b>B</b>), 3.96–3.89 (m, 0.77H, <b>B</b>), 3.88–3.81 (m, 8.16H, <b>A</b>+<b>B</b>), 3.71 (s, 4.52H, <b>A</b>+<b>B</b>), 3.34 (d, <i>J</i> = 9.8 Hz, 0.72H, <b>B</b>), 2.95 (d, <i>J</i> = 9.6 Hz, 0.27H, <b>A</b>), 2.79 (td, <i>J</i> = 3.0, 13.3 Hz, 0.72H, <b>B</b>), 2.65–2.51 (m, 0.87H, <b>A</b>+<b>B</b>), 2.50–2.34 (m, 1.50H, <b>A</b>+<b>B</b>), 2.32–2.23 (m, 1.03H, <b>A</b>+<b>B</b>), 2.15–2.01 (m, 2.04H, <b>A</b>+<b>B</b>), 2.01–1.92 (m, 0.76H, <b>A</b>+<b>B</b>), 1.92–1.79 (m, 1.86H, <b>A</b>+<b>B</b>), 1.73–1.49 (m, 6.52H, <b>A</b>+<b>B</b>), 1.48–1.04 (m, 8.43H, <b>A</b>+<b>B</b>), 1.04–0.68 (m, 3.22H, <b>A</b>+<b>B</b>), 0.67–0.52 (m, 0.58H, <b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:5, <b>A</b>:<b>B</b>): δ 172.67 (<b>B</b>), 172.51 (<b>A</b>), 170.77 (<b>A</b>), 170.70 (<b>B</b>), 158.98 (<b>A</b>), 158.62 (<b>B</b>), 149.12 (<b>A</b>), 149.00 (<b>B</b>), 147.64 (<b>A</b>), 147.45 (<b>B</b>), 147.25 (<b>A</b>), 147.00 (<b>B</b>), 141.89 (<b>B</b>), 141.62 (<b>A</b>), 133.76 (<b>B</b>), 133.74 (<b>B</b>), 133.45 (<b>A</b>), 133.38 (<b>B</b>), 133.15 (<b>A</b>), 129.89 (<b>A</b>), 129.58 (<b>B</b>), 129.04 (<b>A</b>+<b>B</b>), 120.40 (<b>B</b>), 120.25 (<b>A</b>), 119.32 (<b>A</b>), 118.69 (<b>B</b>), 117.91 (<b>A</b>), 117.70 (<b>B</b>), 114.34 (<b>A</b>), 114.05 (<b>B</b>), 113.45 (<b>A</b>), 113.21 (<b>B</b>), 112.05 (<b>B</b>), 111.89 (<b>A</b>), 111.58 (<b>A</b>), 111.49 (<b>B</b>), 105.70 (<b>B</b>), 105.14 (<b>A</b>), 76.90 (<b>A</b>), 75.81 (<b>B</b>), 68.94 (<b>A</b>), 68.86 (<b>B</b>), 56.66 (<b>A</b>), 56.37 (<b>B</b>), 56.07 (<b>B</b>), 56.01 (<b>B</b>), 55.90 (<b>A</b>), 55.71 (<b>A</b>), 55.05 (<b>A</b>+<b>B</b>), 52.17 (<b>B</b>), 43.69 (<b>B</b>), 41.45 (<b>B</b>), 41.27 (<b>A</b>), 39.54 (<b>A</b>), 38.15 (<b>B</b>), 37.93 (<b>A</b>), 33.14 (<b>A</b>), 32.97 (<b>B</b>), 31.59 (<b>A</b>), 31.14 (<b>B</b>), 30.78 (<b>B</b>), 30.60 (<b>A</b>), 26.94 (<b>B</b>), 26.93 (<b>A</b>), 26.73 (<b>B</b>), 26.59 (<b>A</b>), 26.36 (<b>B</b>), 26.35 (<b>A</b>), 26.32 (<b>B</b>), 26.23 (<b>A</b>), 25.64 (<b>B</b>), 24.49 (<b>A</b>), 21.07 (<b>B</b>), 20.82 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 716.37931, found 716.37915. HPLC (50–95% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 13.49 min, purity (220 nm) = 99%.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(4-(Allyloxy)-3,5-dimethoxy-phenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>9</b>)</h3><div class="NLM_p last">The substrate <b>8</b> (400 mg, 0.56 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (116 mg, 0.84 mmol, 1.50 equiv) and allyl bromide (76 mg, 0.62 mmol, 1.10 equiv) in MeCN (20 mL). Additional portions of K<sub>2</sub>CO<sub>3</sub> (1.00 equiv) and allyl bromide (2 × 2.00 equiv) were added until completion of the reaction. <b>9</b> (411 mg, 97%) was obtained as a light brown solid after purification by flash column chromatography (CH/EA = 4:1). TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.26. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.38:1, <b>A</b>/<b>B</b>): δ 7.26–7.23 (m, 0.27H, <b>A</b>), 7.06 (t, <i>J</i> = 7.9 Hz, 0.71H, <b>B</b>), 6.93–6.88 (m, 0.58H, <b>A</b>+<b>B</b>), 6.85–6.82 (m, 0.28H, <b>A</b>), 6.77–6.70 (m, 1.78H, <b>A</b>+<b>B</b>), 6.68–6.64 (m, 1.03H, <b>B</b>), 6.64–6.58 (m, 1.80H, <b>A</b>+<b>B</b>), 6.47 (s, 1.47H, <b>A</b>+<b>B</b>), 6.42–6.38 (m, 1.32H, <b>A</b>+<b>B</b>), 6.08–5.94 (m, 1.93H, <b>A</b>+<b>B</b>), 5.78 (dd, <i>J</i> = 6.2, 7.7 Hz, 0.28H, <b>A</b>), 5.54 (dd, <i>J</i> = 5.6, 8.1 Hz, 0.72H, <b>B</b>), 5.46–5.42 (m, 0.72H, <b>B</b>), 5.40–5.32 (m, 1.02H, <b>A</b>+<b>B</b>), 5.27–5.19 (m, 2.28H, <b>A</b>+<b>B</b>), 5.14–5.05 (m, 1.01H, <b>A</b>+<b>B</b>), 4.70–4.66 (m, 0.27H, <b>A</b>), 4.56–4.44 (m, 3.02H, <b>A</b>+<b>B</b>), 4.42–4.38 (m, 1.47H, <b>A</b>+<b>B</b>), 3.96–3.89 (m, 0.77H, <b>B</b>), 3.83–3.76 (m, 8.23H, <b>A</b>+<b>B</b>), 3.66 (s, 4.49H, <b>A</b>+<b>B</b>), 3.35 (d, <i>J</i> = 9.8 Hz, 0.72H, <b>B</b>), 2.95 (d, <i>J</i> = 9.6 Hz, 0.27H, <b>A</b>), 2.77 (td, <i>J</i> = 2.9, 13.4 Hz, 0.71H, <b>B</b>), 2.61–2.49 (m, 0.86H, <b>A</b>+<b>B</b>), 2.47–2.31 (m, 1.49H, <b>A</b>+<b>B</b>), 2.29–2.21 (m, 1.05H, <b>A</b>+<b>B</b>), 2.12–1.99 (m, 1.76H, <b>A</b>+<b>B</b>), 1.96–1.75 (m, 2.66H, <b>A</b>+<b>B</b>), 1.70–1.46 (m, 6.38H, <b>A</b>+<b>B</b>), 1.44–1.02 (m, 8.83H, <b>A</b>+<b>B</b>), 1.01–0.67 (m, 3.33H, <b>A</b>+<b>B</b>), 0.65–0.52 (m, 0.59H, <b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:5, <b>A</b>/<b>B</b>): δ 172.30 (<b>B</b>), 172.15 (<b>A</b>), 170.51 (<b>A</b>), 170.44 (<b>B</b>), 158.81 (<b>A</b>), 158.43 (<b>B</b>), 153.49 (<b>A</b>), 153.19 (<b>B</b>), 148.96 (<b>A</b>), 148.83 (<b>B</b>), 147.48 (<b>A</b>), 147.26 (<b>B</b>), 141.73 (<b>B</b>), 141.46 (<b>A</b>), 135.73 (<b>B</b>), 135.59 (<b>A</b>), 134.65 (<b>B</b>), 134.52 (<b>A</b>), 134.22 (<b>A</b>), 133.53 (<b>B</b>), 133.47 (<b>B</b>), 133.27 (<b>A</b>), 133.22 (<b>B</b>), 133.00 (<b>A</b>), 129.65 (<b>A</b>), 129.43 (<b>B</b>), 120.25 (<b>B</b>), 120.11 (<b>A</b>), 119.11 (<b>A</b>), 118.53 (<b>B</b>), 117.66 (<b>A</b>), 117.44 (<b>B</b>), 117.36 (<b>A</b>), 117.12 (<b>B</b>), 114.18 (<b>A</b>), 113.85 (<b>B</b>), 113.23 (<b>A</b>), 113.12 (<b>B</b>), 111.86 (<b>B</b>), 111.76 (<b>A</b>), 111.46 (<b>A</b>), 111.36 (<b>B</b>), 105.88 (<b>B</b>), 105.38 (<b>A</b>), 76.66 (<b>A</b>), 75.53 (<b>B</b>), 73.98 (<b>A</b>+<b>B</b>), 68.74 (<b>A</b>), 68.66 (<b>B</b>), 56.30 (<b>B</b>), 56.03 (<b>B</b>), 55.90 (<b>A</b>), 55.87 (<b>B</b>), 55.82 (<b>A</b>), 55.79 (<b>B</b>), 55.75 (<b>A</b>), 54.93 (<b>B</b>), 51.99 (<b>A</b>), 43.59 (<b>B</b>), 41.36 (<b>B</b>), 41.00 (<b>A</b>), 39.39 (<b>A</b>), 37.88 (<b>B</b>), 37.75 (<b>A</b>), 32.96 (<b>A</b>), 32.72 (<b>B</b>), 31.41 (<b>A</b>), 30.95 (<b>B</b>), 30.58 (<b>B</b>), 30.41 (<b>A</b>), 26.69 (<b>B</b>), 26.64 (<b>A</b>), 26.54 (<b>B</b>), 26.41 (<b>A</b>), 26.18 (<b>A</b>+<b>B</b>), 26.13 (<b>B</b>), 26.06 (<b>A</b>), 25.50 (<b>B</b>), 24.30 (<b>A</b>), 20.90 (<b>B</b>), 20.64 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 756.41061, found 756.40996. HPLC (60–90% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 15.34 min, purity (220 nm) = 99%.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(4-(But-3-en-1-yloxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>10</b>)</h3><div class="NLM_p last">The substrate <b>8</b> (300 mg, 0.42 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (174 mg, 1.26 mmol, 3.00 equiv) and 4-bromo-1-butene (146 mg, 1.05 mmol, 2.50 equiv) in MeCN (10 mL). Additional portions of K<sub>2</sub>CO<sub>3</sub> (2.00 equiv) and 4-bromo-1-butene (3 × 3.00 equiv) were added until completion of the reaction. <b>10</b> (307 mg, 95%) was obtained as yellow resin after purification by flash column chromatography (CH/EA = 5:1). TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.41. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.38:1, <b>A</b>/<b>B</b>): δ 7.29–7.24 (m, 0.35H, <b>A</b>), 7.07 (t, <i>J</i> = 7.9 Hz, 0.72H, <b>B</b>), 6.94–6.89 (m, 0.59H, <b>A</b>+<b>B</b>), 6.87–6.83 (m, 0.3H, <b>A</b>), 6.79–6.71 (m, 1.81H, <b>A</b>+<b>B</b>), 6.69–6.58 (m, 2.86H, <b>A</b>+<b>B</b>), 6.47 (s, 1.49H, <b>A</b>+<b>B</b>), 6.45–6.38 (m, 1.34H, <b>A</b>+<b>B</b>), 6.06–5.96 (m, 0.96H, <b>A</b>+<b>B</b>), 5.93–5.76 (m, 1.26H, <b>A</b>+<b>B</b>), 5.54 (dd, <i>J</i> = 5.6, 8.1 Hz, 0.73H, <b>B</b>), 5.47–5.43 (m, 0.72H, <b>B</b>), 5.41–5.33 (m, 1.00H, <b>A</b>+<b>B</b>), 5.28–5.20 (m, 1.07H, <b>A</b>+<b>B</b>), 5.13–5.07 (m, 0.29H, <b>A</b>), 5.07–4.95 (m, 1.72H, <b>A</b>+<b>B</b>), 4.69 (d, <i>J</i> = 5.7 Hz, 0.27H, <b>A</b>), 4.57–4.44 (m, 2.40H, <b>A</b>+<b>B</b>), 4.01–3.89 (m, 2.96H, <b>A</b>+<b>B</b>), 3.86–3.76 (m, 8.39H, <b>A</b>+<b>B</b>), 3.67 (s, 4.55H, <b>A</b>+<b>B</b>), 3.36 (d, <i>J</i> = 9.7 Hz, 0.72H, <b>B</b>), 2.95 (d, <i>J</i> = 9.6 Hz, 0.28H, <b>A</b>), 2.78 (td, <i>J</i> = 2.9, 13.3 Hz, 0.71H, <b>B</b>), 2.61–2.32 (m, 4.61H, <b>A</b>+<b>B</b>), 2.30–2.22 (m, 1.08H, <b>A</b>+<b>B</b>), 2.12–2.00 (m, 2.17H, <b>A</b>+<b>B</b>), 1.98–1.75 (m, 2.67H, <b>A</b>+<b>B</b>), 1.70–1.48 (m, 6.32H, <b>A</b>+<b>B</b>), 1.45–1.03 (m, 7.64H, <b>A</b>+<b>B</b>), 1.01–0.80 (m, 1.32H, <b>A</b>+<b>B</b>), 0.78–0.68 (m, 0.76H, <b>B</b>), 0.66–0.52 (m, 0.57H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:5, <b>A</b>/<b>B</b>): δ 172.34 (<b>B</b>), 172.20 (<b>A</b>), 170.57 (<b>A</b>), 170.48 (<b>B</b>), 158.85 (<b>A</b>), 158.47 (<b>B</b>), 153.53 (<b>A</b>), 153.27 (<b>B</b>), 149.00 (<b>A</b>), 148.87 (<b>B</b>), 147.52 (<b>A</b>), 147.31 (<b>B</b>), 141.79 (<b>B</b>), 141.50 (<b>A</b>), 136.08 (<b>A</b>), 136.04 (<b>B</b>), 135.02 (<b>A</b>+<b>B</b>), 134.18 (<b>A</b>), 133.51 (<b>B</b>), 133.47 (<b>B</b>), 133.31 (<b>A</b>), 133.27 (<b>B</b>), 133.04 (<b>A</b>), 129.69 (<b>A</b>), 129.47 (<b>B</b>), 120.30 (<b>B</b>), 120.15 (<b>A</b>), 119.16 (<b>A</b>), 118.56 (<b>B</b>), 117.72 (<b>A</b>), 117.50 (<b>B</b>), 116.28 (<b>A</b>), 116.17 (<b>B</b>), 114.23 (<b>A</b>), 113.88 (<b>B</b>), 113.28 (<b>A</b>), 113.17 (<b>B</b>), 111.89 (<b>B</b>), 111.79 (<b>A</b>), 111.49 (<b>A</b>), 111.39 (<b>B</b>), 105.98 (<b>B</b>), 105.49 (<b>A</b>), 76.70 (<b>A</b>), 75.57 (<b>B</b>), 72.37 (<b>A</b>), 72.30 (<b>B</b>), 68.79 (<b>A</b>), 68.71 (<b>B</b>), 56.35 (<b>B</b>), 56.08 (<b>A</b>+<b>B</b>), 55.95 (<b>A</b>), 55.91 (<b>B</b>), 55.87 (<b>A</b>), 55.84 (<b>B</b>), 55.78 (<b>A</b>), 54.98 (<b>B</b>), 52.02 (<b>A</b>+<b>B</b>), 43.63 (<b>B</b>), 41.40 (<b>B</b>), 41.05 (<b>A</b>), 39.43 (<b>A</b>), 37.93 (<b>B</b>), 37.80 (<b>A</b>), 34.55 (<b>A</b>), 34.52 (<b>B</b>), 33.00 (<b>A</b>), 32.77 (<b>B</b>), 31.45 (<b>A</b>), 30.99 (<b>B</b>), 30.63 (<b>B</b>), 30.46 (<b>A</b>), 26.74 (<b>B</b>), 26.71 (<b>A</b>), 26.59 (<b>B</b>), 26.45 (<b>A</b>), 26.22 (<b>A</b>+<b>B</b>), 26.18 (<b>B</b>), 26.10 (<b>A</b>), 25.55 (<b>B</b>), 24.35 (<b>A</b>), 20.94 (<b>B</b>), 20.69 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>60</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 770.42626, found 770.42659. HPLC (70–95% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 11.52 min, purity (220 nm) = 99%.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(4-(2-(Allyloxy)ethoxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>11</b>)</h3><div class="NLM_p last">The substrate <b>8</b> (500 mg, 0.7 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (290 mg, 2.1 mmol, 3.00 equiv) and <b>16</b> (718 mg, 2.8 mmol, 4.00 equiv) in MeCN (25 mL). <b>11</b> (550 mg, 98%) was obtained after purification by flash column chromatography (CH/EA = 4:1) as light yellow resin. TLC (CH/EA = 4:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.44. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.44:1, <b>A</b>/<b>B</b>): δ 7.29 (t, <i>J</i> = 7.9 Hz, 0.33H, <b>A</b>), 7.10 (t, <i>J</i> = 7.9 Hz, 0.70H, <b>B</b>), 6.97–6.91 (m, 0.64H, <b>A</b>+<b>B</b>), 6.90–6.85 (m, 0.32H, <b>A</b>), 6.81–6.74 (m, 1.79H, <b>A</b>+<b>B</b>), 6.71–6.67 (m, 1.03H, <b>B</b>), 6.66–6.61 (m, 1.80H, <b>A</b>+<b>B</b>), 6.48 (s, 1.41H, <b>A</b>+<b>B</b>), 6.46–6.43 (m, 0.75H, <b>A</b>+<b>B</b>), 6.41 (s, 0.63H, <b>A</b>), 6.09–5.99 (m, 0.97H, <b>A</b>+<b>B</b>), 5.97–5.86 (m, 0.93H, <b>A</b>+<b>B</b>), 5.81 (dd, <i>J</i> = 6.2, 7.7 Hz, 0.31H, <b>A</b>), 5.56 (dd, <i>J</i> = 5.5, 8.2 Hz, 0.71H, <b>B</b>), 5.49–5.44 (m, 0.69H, <b>B</b>), 5.44–5.36 (m, 1.05H, <b>A</b>+<b>B</b>), 5.31–5.22 (m, 2.27H, <b>A</b>+<b>B</b>), 5.18–5.12 (m, 0.98H, <b>A</b>+<b>B</b>), 4.70 (d, <i>J</i> = 5.7 Hz, 0.29H, <b>A</b>), 4.59–4.47 (m, 2.42H, <b>A</b>+<b>B</b>), 4.16–4.12 (m, 0.64H, <b>A</b>), 4.10–4.02 (m, 3.47H, <b>A</b>+<b>B</b>), 3.97–3.90 (m, 0.71H, <b>B</b>), 3.87–3.79 (m, 8.42H, <b>A</b>+<b>B</b>), 3.77–3.73 (m, 0.68H, <b>A</b>), 3.72–3.67 (m, 5.73H, <b>A</b>+<b>B</b>), 3.37 (d, <i>J</i> = 9.8 Hz, 0.70H, <b>B</b>), 2.97 (d, <i>J</i> = 9.6 Hz, 0.30H, <b>A</b>), 2.80 (td, <i>J</i> = 2.9, 13.3 Hz, 0.70H, <b>B</b>), 2.65–2.51 (m, 0.94H, <b>A</b>+<b>B</b>), 2.50–2.35 (m, 1.54H, <b>A</b>+<b>B</b>), 2.32–2.24 (m, 1.07H, <b>A</b>+<b>B</b>), 2.15–2.00 (m, 1.77H, <b>A</b>+<b>B</b>), 1.99–1.78 (m, 2.58H, <b>A</b>+<b>B</b>), 1.74–1.50 (m, 6.80H, <b>A</b>+<b>B</b>), 1.47–1.05 (m, 6.92H, <b>A</b>+<b>B</b>), 1.04–0.85 (m, 1.18H, <b>A</b>+<b>B</b>), 0.79–0.69 (m, 0.75H, <b>A</b>+<b>B</b>), 0.67–0.53 (m, 0.65H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:2, <b>A</b>:<b>B</b>): δ 172.50 (<b>B</b>), 172.33 (<b>A</b>), 170.74 (<b>A</b>), 170.62 (<b>B</b>), 158.99 (<b>A</b>), 158.62 (<b>B</b>), 153.57 (<b>A</b>), 153.31 (<b>B</b>), 149.14 (<b>A</b>), 149.01 (<b>B</b>), 147.66 (<b>A</b>), 147.45 (<b>B</b>), 141.95 (<b>B</b>), 141.62 (<b>A</b>), 136.19 (<b>A</b>), 135.19 (<b>B</b>), 135.13 (<b>A</b>), 134.39 (<b>A</b>), 133.70 (<b>B</b>), 133.67 (<b>B</b>), 133.40 (<b>B</b>), 133.17 (<b>A</b>), 129.84 (<b>A</b>), 129.63 (<b>B</b>), 120.43 (<b>B</b>), 120.27 (<b>A</b>), 119.31 (<b>A</b>), 118.69 (<b>B</b>), 117.93 (<b>A</b>), 117.71 (<b>B</b>), 116.85 (<b>A</b>), 116.79 (<b>B</b>), 114.38 (<b>A</b>), 114.03 (<b>B</b>), 113.46 (<b>A</b>), 113.31 (<b>B</b>), 112.01 (<b>B</b>), 111.90 (<b>A</b>), 111.59 (<b>A</b>), 111.51 (<b>B</b>), 106.11 (<b>B</b>), 105.59 (<b>A</b>), 76.91 (<b>A</b>), 75.74 (<b>B</b>), 72.33 (<b>A</b>), 72.27 (<b>B</b>), 69.61 (<b>A</b>), 69.55 (<b>B</b>), 68.96 (<b>A</b>), 68.89 (<b>B</b>), 56.52 (<b>B</b>), 56.26 (<b>A</b>+<b>B</b>), 56.11 (<b>A</b>), 56.08 (<b>B</b>), 56.02 (<b>A</b>), 56.00 (<b>B</b>), 55.94 (<b>A</b>), 55.15 (<b>B</b>), 52.18 (<b>A</b>+<b>B</b>), 43.77 (<b>B</b>), 41.55 (<b>B</b>), 41.21 (<b>A</b>), 39.58 (<b>A</b>), 38.12 (<b>B</b>), 37.96 (<b>A</b>), 33.16 (<b>A</b>), 32.94 (<b>B</b>), 31.61 (<b>A</b>), 31.15 (<b>B</b>), 30.81 (<b>B</b>), 30.63 (<b>A</b>), 26.92 (<b>B</b>), 26.88 (<b>A</b>), 26.74 (<b>B</b>), 26.60 (<b>A</b>), 26.37 (<b>A</b>+<b>B</b>), 26.32 (<b>B</b>), 25.70 (<b>B</b>), 24.50 (<b>A</b>), 21.11 (<b>B</b>), 20.84 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>47</sub>H<sub>62</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 800.43682, found 800.43719. HPLC (70–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 10.08 min, purity (220 nm) = 95%.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(4-(3-(Allyloxy)propoxy)-3,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>12</b>)</h3><div class="NLM_p last">The substrate <b>8</b> (550 mg, 0.77 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (213 mg, 1.54 mmol, 2.50 equiv) and <b>17</b> (625 mg, 2.31 mmol, 3.00 equiv) in MeCN (20 mL). Additional portions of K<sub>2</sub>CO<sub>3</sub> (1.00 equiv) and <b>17</b> (1.00 equiv) were added until completion of the reaction. <b>12</b> (576 mg, 92%) was obtained after purification by flash column chromatography (CH/EA = 4:1) as colorless resin. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.26. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.42:1, <b>A</b>/<b>B</b>): δ 7.30–7.26 (m, 0.25H, <b>A</b>), 7.11–7.06 (m, 0.69H, <b>B</b>), 6.95–6.90 (m, 0.62H, <b>B</b>), 6.88–6.85 (m, 0.31H, <b>A</b>), 6.80–6.73 (m, 1.77H, <b>A</b>+<b>B</b>), 6.70–6.66 (m, 1.04H, <b>A</b>+<b>B</b>), 6.66–6.60 (m, 1.80H, <b>A</b>+<b>B</b>), 6.47 (s, 1.43H, <b>B</b>), 6.46–6.42 (m, 0.75H, <b>A</b>+<b>B</b>), 6.41–6.39 (m, 0.63H, <b>A</b>), 6.08–5.98 (m, 0.96H, <b>A</b>+<b>B</b>), 5.95–5.83 (m, 0.95H, <b>A</b>+<b>B</b>), 5.80 (t, <i>J</i> = 6.9 Hz, 0.31H, <b>A</b>), 5.58–5.53 (m, 0.71H, <b>B</b>), 5.45 (d, <i>J</i> = 5.5 Hz, 0.70H, <b>B</b>), 5.43–5.35 (m, 1.04H, <b>A</b>+<b>B</b>), 5.30–5.20 (m, 2.05H, <b>A</b>+<b>B</b>), 5.17–5.10 (m, 1.01H, <b>A</b>+<b>B</b>), 4.72–4.68 (m, 0.30H, <b>A</b>), 4.58–4.46 (m, 2.42H, <b>A</b>+<b>B</b>), 4.07–4.02 (m, 0.70H, <b>B</b>), 4.01–3.90 (m, 4.42H, <b>A</b>+<b>B</b>), 3.86–3.77 (m, 8.55H, <b>A</b>+<b>B</b>), 3.70–3.63 (m, 5.14H, <b>A</b>+<b>B</b>), 3.63–3.58 (m, 1.51H), <b>A</b>+<b>B</b>, 3.36 (d, <i>J</i> = 9.7 Hz, 0.70H, <b>B</b>), 2.96 (d, <i>J</i> = 9.5 Hz, 0.3H, <b>A</b>), 2.83–2.75 (m, 0.69H, <b>B</b>), 2.64–2.50 (m, 0.96H, <b>A</b>+<b>B</b>), 2.49–2.34 (m, 1.46H, <b>A</b>+<b>B</b>), 2.31–2.23 (m, 1.06H, <b>A</b>+<b>B</b>), 2.13–2.04 (m, 1.58H, <b>A</b>+<b>B</b>), 2.01–1.90 (m, 2.71H, <b>A</b>+<b>B</b>), 1.90–1.78 (m, 2.00H, <b>A</b>+<b>B</b>), 1.72–1.49 (m, 6.50H, <b>A</b>+<b>B</b>), 1.46–1.25 (m, 3.15H, <b>A</b>+<b>B</b>), 1.20–1.06 (m, 2.98H, <b>A</b>+<b>B</b>), 1.03–0.82 (m, 1.21H, <b>A</b>+<b>B</b>), 0.79–0.68 (m, 0.69H, <b>A</b>+<b>B</b>), 0.66–0.52 (m, 0.57H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:5, <b>A</b>/<b>B</b>): δ 172.47 (<b>B</b>), 172.31 (<b>A</b>), 170.71 (<b>A</b>), 170.59 (<b>B</b>), 158.95 (<b>A</b>), 158.58 (<b>B</b>), 153.62 (<b>A</b>), 153.36 (<b>B</b>), 149.10 (<b>A</b>), 148.97 (<b>B</b>), 147.62 (<b>A</b>), 147.41 (<b>B</b>), 141.89 (<b>B</b>), 141.59 (<b>A</b>), 136.24 (<b>A</b>), 135.23 (<b>B</b>), 135.19 (<b>A</b>), 134.23 (<b>A</b>), 133.63 (<b>B</b>), 133.50 (<b>B</b>), 133.42 (<b>A</b>), 133.37 (<b>B</b>), 133.13 (<b>A</b>), 129.80 (<b>A</b>), 129.58 (<b>B</b>), 120.39 (<b>B</b>), 120.23 (<b>A</b>), 119.27 (<b>A</b>), 118.67 (<b>B</b>), 117.87 (<b>A</b>), 117.64 (<b>B</b>), 116.68 (<b>A</b>), 116.60 (<b>B</b>), 114.33 (<b>A</b>), 114.00 (<b>B</b>), 113.42 (<b>A</b>), 113.25 (<b>B</b>), 111.98 (<b>B</b>), 111.87 (<b>A</b>), 111.56 (<b>A</b>), 111.47 (<b>B</b>), 106.06 (<b>B</b>), 105.55 (<b>A</b>), 76.83 (<b>A</b>), 75.69 (<b>B</b>), 72.01 (<b>A</b>), 71.96 (<b>B</b>), 70.56 (<b>A</b>), 70.48 (<b>B</b>), 68.91 (<b>A</b>), 68.84 (<b>B</b>), 67.55 (<b>B</b>), 67.49 (<b>A</b>), 56.43 (<b>B</b>), 56.16 (<b>B</b>), 56.06 (<b>A</b>), 56.03 (<b>B</b>), 55.98 (<b>A</b>), 55.95 (<b>B</b>), 55.90 (<b>A</b>), 55.12 (<b>B</b>), 52.14 (<b>B</b>), 43.72 (<b>B</b>), 41.50 (<b>B</b>), 41.18 (<b>A</b>), 39.54 (<b>A</b>), 38.04 (<b>B</b>), 37.91 (<b>A</b>), 33.12 (<b>A</b>), 32.90 (<b>B</b>), 31.56 (<b>A</b>), 31.10 (<b>B</b>), 30.76 (<b>B</b>), 30.69 (<b>A</b>+<b>B</b>), 30.58 (<b>A</b>), 26.87 (<b>B</b>), 26.83 (<b>A</b>), 26.70 (<b>B</b>), 26.56 (<b>A</b>), 26.33 (<b>A</b>+<b>B</b>), 26.29 (<b>B</b>), 26.21 (<b>A</b>), 25.65 (<b>B</b>), 24.44 (<b>A</b>), 21.06 (<b>B</b>), 20.80 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>48</sub>H<sub>64</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 814.45247, found 814.45247. HPLC (70–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 11.28 min, purity (220 nm) = 99%.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propan-1-ol (<b>14</b>)</h3><div class="NLM_p last">To a stirred solution of <b>13</b> (2.00 g, 6.94 mmol, 1.00 equiv, synthesized as previously described<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>) and K<sub>2</sub>CO<sub>3</sub> (1.44 g, 10.40 mmol, 1.50 equiv) in acetone was added allyl bromide (1.68 g, 13.88 mmol, 2.00 equiv), and the reaction mixture was heated at reflux for 2 days. After the first day, another portion of allyl bromide (0.50 equiv) was added to the reaction mixture. The suspension was filtered through Celite, and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 3:1) to afford <b>14</b> (2.18 g, 96%) as cloudy, yellow oil. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.40. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.25 (t, <i>J</i> = 7.8 Hz, 1H), 6.96–6.89 (m, 2H), 6.86–6.81 (m, 1H), 6.78 (d, <i>J</i> = 8.0 Hz, 1H), 6.75–6.68 (m, 2H), 6.12–5.98 (m, 1H), 5.46–5.37 (m, 1H), 5.33–5.23 (m, 1H), 4.65 (dd, <i>J</i> = 5.2, 7.9 Hz, 1H), 4.53 (dt, <i>J</i> = 1.5, 5.3 Hz, 2H), 3.85 (s, 3H), 3.85 (s, 3H), 2.80–2.48 (m, 2H), 2.18–1.90 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 158.9, 148.9, 147.3, 146.5, 134.5, 133.3, 129.6, 120.3, 118.5, 117.7, 113.8, 112.4, 111.9, 111.4, 73.8, 68.9, 56.0, 55.9, 40.7, 31.7. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>20</sub>H<sub>24</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>: 351.15668, found 351.15644. HPLC (5–95% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 14.25 min, purity (220 nm) = 99%.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl Piperidine-2-carboxylate (<b>15</b>)</h3><div class="NLM_p last">To a stirred solution of <b>14</b> (2.24 g, 6.83 mmol, 1.00 equiv) and (<i>S</i>)-1-(((9H-fluoren-9-yl)methoxy)-carbonyl)piperidine-2-carboxylic acid (2.64 g, 7.52 mmol, 1.10 equiv) in DCM (40 mL) at 0 °C were added DMAP (167 mg, 1.37 mmol, 0.20 equiv) and DCC (1.55 g, 7.52 mmol, 1.10 equiv). The reaction mixture was stirred at 0 °C for 30 min and then overnight at room temperature. The resulting precipitate was filtered off and washed with DCM (3 × 20 mL), and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 5:1) to afford the Fmoc-protected amine (4.31 g, 6.52 mmol, 95%) as a light yellow resin. Subsequently the Fmoc-protected amine (4.31 g, 6.52 mmol, 1.00 equiv) was dissolved in a mixture of 4-methylpiperidine/DCM (v/v = 1:20, 63 mL), and the mixture was stirred overnight at room temperature. The solvent was removed under reduced pressure and the crude product directly subjected to flash column chromatography (CH/EA = 2:1 + 1% NEt<sub>3</sub>) to afford <b>15</b> (2.11 g, 70%) as a light yellow oil. TLC (DCM/MeOH = 20:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.24. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 353 K, mixture of rotamers 0.3:1, <b>A</b>/<b>B</b>): δ 7.43–7.18 (m, 1H, <b>A</b>+<b>B</b>), 7.00–6.87 (m, 4.03H, <b>A</b>+<b>B</b>), 6.82 (d, J = 2.0 Hz, 0.72H, <b>B</b>), 6.78 (d, J = 2.0 Hz, 0.26H, <b>A</b>), 6.76–6.68 (m, 0.99H, <b>A</b>+<b>B</b>), 6.19–5.99 (m, 0.86H, <b>A</b>+<b>B</b>), 5.84–5.73 (m, 0.98H, <b>A</b>+<b>B</b>), 5.54–5.38 (m, 0.94H, <b>A</b>+<b>B</b>), 5.35–5.18 (m, 0.94H, <b>A</b>+<b>B</b>), 4.71–4.55 (m, 2.02H, <b>A</b>+<b>B</b>), 3.81 (s, 2.04H, <b>B</b>), 3.80–3.73 (m, 3.70H, <b>A</b>+<b>B</b>), 3.48 (t, J = 4.9 Hz, 0.25H, <b>A</b>), 3.39 (dd, J = 3.3, 9.3 Hz, 0.69H, <b>B</b>), 3.03–2.88 (m, 1.88H, <b>A</b>+<b>B</b>), 2.74–2.49 (m, 3.05H, <b>A</b>+<b>B</b>), 2.48–2.38 (m, 0.25H, <b>A</b>), 2.31–2.18 (m, 1.03H, <b>A</b>+<b>B</b>), 2.17–2.05 (m, 1.05H, <b>A</b>+<b>B</b>), 2.00–1.82 (m, 1.12H, <b>A</b>+<b>B</b>), 1.83–1.63 (m, 1.11H, <b>A</b>+<b>B</b>), 1.62–1.35 (m, 3.99H, <b>A</b>+<b>B</b>), 1.29–1.20 (m, 0.21H, <b>A</b>+<b>B</b>), 1.01–0.80 (m, 0.28H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 353 K, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 172.22 (<b>B</b>), 172.14 (<b>A</b>), 158.42 (<b>B</b>), 158.39 (<b>A</b>), 149.20 (<b>A</b>+<b>B</b>), 147.58 (<b>A</b>+<b>B</b>), 142.20 (<b>B</b>), 142.08 (<b>A</b>), 133.83 (<b>B</b>), 133.74 (<b>A</b>+<b>B</b>), 133.69 (<b>A</b>), 129.36 (<b>B</b>), 129.29 (<b>A</b>), 120.25 (<b>B</b>), 120.20 (<b>A</b>), 118.67 (<b>A</b>), 118.51 (<b>B</b>), 117.09 (<b>B</b>), 116.99 (<b>A</b>), 114.01 (<b>A</b>+<b>B</b>), 113.18 (<b>B</b>), 113.11 (<b>A</b>), 113.07 (<b>A</b>), 112.96 (<b>A</b>+<b>B</b>), 112.87 (<b>B</b>), 74.64 (<b>B</b>), 74.54 (<b>A</b>), 68.41 (<b>B</b>), 68.39 (<b>A</b>), 60.10 (<b>A</b>), 58.21 (<b>B</b>), 55.96 (<b>A</b>+<b>B</b>), 55.82 (<b>B</b>), 55.79 (<b>A</b>), 47.20 (<b>A</b>), 45.02 (<b>B</b>), 37.41 (<b>B</b>), 37.35 (<b>A</b>), 30.68 (<b>B</b>), 30.66 (<b>A</b>), 28.89 (<b>B</b>), 28.22 (<b>A</b>), 25.62 (<b>B</b>), 24.87 (<b>A</b>), 23.59 (<b>B</b>), 21.53 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>26</sub>H<sub>34</sub>NO<sub>5</sub> [M + H]<sup>+</sup>: 440.24315, found 440.24365. HPLC (40–70% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 6.47 min, purity (220 nm) = 96%.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 2-(Allyloxy)ethyl 4-Methylbenzenesulfonate (<b>16</b>)</h3><div class="NLM_p last">To a stirred solution of 2-(allyloxy)ethanol (3.00 g, 29.37 mmol, 1.00 equiv) and pyridine (3.49 g, 44.06 mmol, 1.50 equiv) in dry DCM (50 mL) at 0 °C was added <i>p</i>-toluenesulfonyl chloride (6.72 g, 35.25 mmol, 1.20 equiv). The reaction mixture was allowed to reach ambient temperature and left stirring overnight. Then additional portions of pyridine (0.50 equiv) and <i>p</i>-toluenesulfonyl chloride (0.50 equiv) were added, and the reaction was stirred for another night. The reaction mixture was diluted with DCM (50 mL), washed with 1 M HCl solution (100 mL), 4% aqueous NaHCO<sub>3</sub>, water, and brine (50 mL each). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 8:1) to yield <b>16</b> (4.77 g, 63%) as red liquid. TLC (CH/EA = 5:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.45. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.77–7.73 (m, 2H), 7.33–7.28 (m, 2H), 5.77 (ddt, <i>J</i> = 5.5, 10.4, 17.3 Hz, 1H), 5.20–5.15 (m, 1H), 5.12–5.08 (m, 1H), 4.15–4.11 (m, 2H), 3.91–3.87 (m, 2H), 3.60–3.55 (m, 2H), 2.39 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 144.7, 134.1, 132.9, 129.7, 127.8, 117.0, 71.9, 69.3, 67.3, 21.4. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>12</sub>H<sub>17</sub>O<sub>4</sub>S [M + H]<sup>+</sup>: 257.08, found 257.05.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 3-(Allyloxy)propyl 4-Methylbenzenesulfonate (<b>17</b>)</h3><div class="NLM_p last">To a stirred solution of 1,3-propanediol (4.00 g, 52.56 mmol, 1.00 equiv) in dry THF (50 mL) at 0 °C was added sodium hydride (2.32 g, 57.82 g, 1.10 equiv) in portions. After stirring for 30 min at 0 °C, allyl bromide (9.54 g, 78.84 mmol, 1.50 equiv) was added to the gray suspension and the resulting yellow suspension was allowed to reach ambient temperature and was left stirring overnight. The reaction mixture was diluted with Et<sub>2</sub>O (50 mL) and washed with 10% aqueous K<sub>2</sub>CO<sub>3</sub> and brine (25 mL each). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure (75 mbar, 40 °C; product is volatile). The crude product was purified by flash column chromatography (CH/EA = 2:1) to yield <b>S2</b> (2.94 g, 48%, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) as a light yellow liquid. To a stirred solution of <b>S2</b> (2.94 g, 25.34 mmol, 1.00 equiv) and pyridine (4.01 g, 50.67 mmol, 2.00 equiv) in dry DCM (50 mL) at 0 °C was added <i>p</i>-toluenesulfonyl chloride (9.66 g, 50.67 mmol, 2.00 equiv). The reaction mixture was allowed to reach ambient temperature and left stirring overnight. The reaction mixture was diluted with DCM (50 mL), washed with 1 M HCl solution (100 mL), 4% aqueous NaHCO<sub>3</sub>, water, and brine (50 mL each). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 8:1 → 6:1) to yield <b>17</b> (5.06 g, 74%) as a light yellow liquid. TLC (CH/EA = 4:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.46. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.80–7.75 (m, 2H), 7.35–7.30 (m, 2H), 5.80 (ddt, <i>J</i> = 5.6, 10.4, 17.2 Hz, 1H), 5.18 (dq, <i>J</i> = 1.7, 17.3 Hz, 1H), 5.12 (dq, <i>J</i> = 1.4, 10.4 Hz, 1H), 4.13 (t, <i>J</i> = 6.2 Hz, 2H), 3.85 (dt, <i>J</i> = 1.4, 5.5 Hz, 2H), 3.43 (t, <i>J</i> = 6.0 Hz, 2H), 2.43 (s, 3H), 1.90 (p, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 144.8, 134.7, 133.2, 129.9, 128.0, 117.0, 72.0, 67.8, 65.6, 29.5, 21.7. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>13</sub>H<sub>19</sub>O<sub>4</sub>S [M + H]<sup>+</sup>: 271.10, found 271.08.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Methyl 2-(3-(benzyloxy)-4,5-dihydroxyphenyl)acetate (<b>18</b>)</h3><div class="NLM_p last">A solution of 2-(3,4,5-trimethoxyphenyl)acetic acid (60.00 g, 265.2 mmol, 1.00 equiv) and acetic acid (160 mL) in concentrated hydrobromic acid (48% w/w aq, 200 mL) was heated at reflux until the starting material was no longer detected by TLC. A saturated solution of sodium sulfate (100 mL) was added to the reaction mixture and was then extracted with EA (8 × 100 mL). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product <b>S3</b> (analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) was filtered through a plug of silica and concentrated under reduced pressure. Subsequently, crude <b>S3</b> was dissolved in MeOH (300 mL), and a catalytic amount of H<sub>2</sub>SO<sub>4</sub> (0.26 g, 2.7 mmol, 0.01 equiv) was added and the reaction mixture was stirred at room temperature for 14 h. Then ice water (300 mL) was added to the reaction mixture, and the layers were separated. The aqueous phase was extracted with Et<sub>2</sub>O (3 × 150 mL) and the combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to yield <b>S4</b> (38.22 g, 73%, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) as dark-red solid without further purification. A solution of <b>S4</b> (12.40 g, 62.6 mmol, 1.00 equiv), <i>p</i>-toluenesulfonic acid monohydrate (1.19 g, 6.3 mmol, 0.10 equiv), and 2,2-dimethoxypropane (32.59 g, 312.9 mmol, 5.00 equiv) in toluene (1250 mL) was heated at reflux for 2 h. Then additional portions of <i>p</i>-toluenesulfonic acid monohydrate (0.05 equiv) and 2,2-dimethoxypropane (1.00 equiv) were added, and the reaction was heated at reflux for another hour. Then the reaction mixture was diluted with Et<sub>2</sub>O (500 mL), washed with saturated aq NaHCO<sub>3</sub> solution and brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 8:1 → 2:1) to yield <b>S5</b> (14.21 g, 95%) as brown oil. Subsequently, to a stirred solution of <b>S5</b> (31.20 g, 131.0 mmol, 1.00 equiv) in acetone (525 mL, 0.25 M) were added K<sub>2</sub>CO<sub>3</sub> (21.72 g, 157.2 mmol, 1.20 equiv) and benzyl bromide (24.64 g, 144.1 mmol, 1.10 equiv), and the reaction mixture was heated at reflux overnight. The resulting suspension was filtered through Celite and washed with EA (500 mL), and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 20:1) to yield <b>S6</b> (42.16 g, 98%, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) as a light yellow solid. To a stirred solution of methyl 2-(7-(benzyloxy)-2,2-dimethylbenzo[d][1,3]dioxol-5-yl)acetate <b>S6</b> (7.23 g, 22.0 mmol, 1.00 equiv) in MeCN/H<sub>2</sub>O (70 mL, v/v = 1:1) was added trifluoroacetic acid (170 mL), and the reaction mixture was stirred at room temperature for 4 h. The reaction mixture was quenched by the addition of saturated aq NaHCO<sub>3</sub> solution. The aqueous phase was extracted with EA (3 × 150 mL) and the combined organic phase subsequently washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 4:1 → 3:1) to yield <b>18</b> (5.13 g, 81%) as a yellow oil. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.22. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.43–7.32 (m, 5H), 6.54 (d, <i>J</i> = 1.9 Hz, 1H), 6.50 (d, <i>J</i> = 1.9 Hz, 1H), 5.49–5.36 (m, 2H), 5.07 (s, 2H), 3.68 (s, 3H), 3.49 (s, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.4, 146.3, 144.2, 136.4, 132.0, 128.8, 128.6, 128.1, 125.8, 110.2, 105.8, 71.5, 52.2, 41.1. LC–MS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>16</sub>H<sub>17</sub>O<sub>5</sub> [M + H]<sup>+</sup>: 289.11, found 289.16.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Methyl 2-(3-(benzyloxy)-4,5-dimethoxyphenyl)acetate (<b>19</b>)</h3><div class="NLM_p last">To a stirred solution of <b>18</b> (5.13 g, 17.8 mmol, 1.00 equiv) in degassed DMF (60 mL, 0.3 M) were added K<sub>2</sub>CO<sub>3</sub> (7.37 g, 53.3 mmol, 3.00 equiv) and methyl iodide (10.10 g, 71.1 mmol, 4.00 equiv), and the reaction mixture was stirred at room temperature overnight. The resulting red solution was diluted with water (100 mL) and then extracted with Et<sub>2</sub>O (4 × 50 mL). The combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 6:1 → 3:1) to yield <b>19</b> (5.25 g, 93%) as a light yellow oil. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.42. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.46–7.43 (m, 2H), 7.39–7.36 (m, 2H), 7.33–7.29 (m, 1H), 6.55 (d, <i>J</i> = 1.9 Hz, 1H), 6.51 (d, <i>J</i> = 1.9 Hz, 1H), 5.12 (s, 2H), 3.86 (s, 6H), 3.68 (s, 3H), 3.52 (s, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.0, 153.6, 152.6, 138.1, 137.3, 129.5, 128.6, 128.0, 127.5, 108.8, 107.0, 71.3, 61.0, 56.3, 52.2, 41.5. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>18</sub>H<sub>21</sub>O<sub>5</sub> [M + H]<sup>+</sup>: 317.13835, found 317.13795.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Methyl 2-(3-(benzyloxy)-4,5-dimethoxyphenyl)-2-(cyclohex-2-en-1-yl)acetate (<b>20</b>)</h3><div class="NLM_p last">To a stirred solution of <b>19</b> (4.85 g, 15.3 mmol, 1.0 equiv) in dry THF (50 mL) was added LiHMDS (1 M in THF, 16.9 mL, 1.10 equiv) dropwise at −78 °C, and the reaction mixture was stirred at this temperature for 20 min. Then 3-bromocyclohex-1-ene (5.20 g, 30.7 mmol, 2.00 equiv) was added dropwise. The reaction mixture was allowed to warm to −40 °C over a period of 3 h (TLC indicated full conversion) before the reaction mixture was quenched by the addition of saturated NH<sub>4</sub>Cl solution (50 mL). The aqueous phase was extracted with Et<sub>2</sub>O (3 × 50 mL), and the combined organic phases were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 10:1) to yield <b>20</b> (5.72 g, 94%) as a light yellow oil and a mixture of diastereomers. TLC (CH/EA = 7:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.38. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of diastereomers): δ 7.47–7.41 (m, 3.07H), 7.38–7.33 (m, 3.00H), 7.32–7.27 (m, 1.40H), 6.62 (d, <i>J</i> = 1.9 Hz, 0.53H), 6.58–6.57 (m, 1.44H), 6.54–6.52 (m, 1.00H), 5.80–5.74 (m, 1.14H), 5.67–5.61 (m, 0.48H), 5.60–5.53 (m, 1.10H), 5.18–5.10 (m, 2.19H), 3.91–3.81 (m, 9.92H), 3.67–3.62 (m, 4.53H), 3.21–3.14 (m, 1.59H), 2.82–2.66 (m, 1.67H), 2.01–1.93 (m, 3.12H), 1.90–1.83 (m, 0.54H), 1.79–1.53 (m, 3.17H), 1.47–1.37 (m, 1.46H), 1.36–1.27 (m, 1.82H), 1.04–0.94 (m, 1.17H), 0.91–0.82 (m, 0.41H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of diastereomers): δ 174.04, 173.96, 153.45, 153.42, 152.26, 152.19, 138.19, 137.20, 133.08, 132.91, 129.35, 128.95, 128.91, 128.57, 127.95, 127.90, 127.58, 127.53, 108.22, 107.94, 106.25, 105.89, 71.20, 71.16, 60.96, 57.62, 57.54, 56.27, 51.98, 38.82, 38.67, 28.04, 26.34, 25.38, 25.28, 21.25, 20.80. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>24</sub>H<sub>29</sub>O<sub>5</sub> [M + H]<sup>+</sup>: 397.20095, found 397.20114. HPLC (55–65% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R,1</sub> = 13.39 min, <i>t</i><sub>R,2</sub> = 13.62 min (no baseline separation of peaks achieved), purity (220 nm) = 99%.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-(3-(Benzyloxy)-4,5-dimethoxyphenyl)-2-(cyclohex-2-en-1-yl)acetic Acid (<b>21</b>)</h3><div class="NLM_p last">To a stirred solution of <b>20</b> (1147 mg, 2.90 mmol, 1.00 equiv) in THF/H<sub>2</sub>O (v/v = 1:1, 50 mL) under air at room temperature was added lithium hydroxide (348 mg, 14.50 mmol, 5.00 equiv), and the mixture was heated at reflux overnight. The reaction mixture was extracted with Et<sub>2</sub>O (3 × 30 mL), the organic layer removed, and the aqueous layer subsequently acidified with 1 M HCl solution (pH = 1–2). After extraction with Et<sub>2</sub>O (3 × 30 mL), the organic phase was washed with brine, dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 6:1 + 0.2% HCOOH) to afford <b>21</b> (1.01 g, 91%) as a colorless resin and mixture of diastereomers. TLC (DCM/MeOH/HCOOH = 80:1:1, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of diastereomers): 7.45–7.40 (m, 2H), 7.38–7.31 (m, 2H), 7.30–7.25 (m, 1H), 6.67–6.65 (m, 0.5H), 6.62–6.58 (m, 1H), 6.57–6.54 (m, 0.5H), 5.83–5.77 (m, 0.5H), 5.70–5.63 (m, 1H), 5.18–5.10 (m, 2.5H), 3.89–3.83 (m, 6H), 3.19 (t, <i>J</i> = 11.3 Hz, 1H), 2.81–2.67 (m, 1H), 2.02–1.93 (m, 2.5H), 1.81–1.73 (m, 0.5H), 1.63–1.55 (m, 1H), 1.45–1.32 (m, 1.5H), 1.03–0.95 (m, 0.5H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of diastereomers): δ 179.52, 179.40, 153.51, 153.47, 152.33, 152.26, 138.42, 138.39, 137.07, 132.46, 132.30, 129.63, 129.23, 128.61, 128.57, 127.96, 127.64, 127.59, 108.40, 108.12, 106.44, 106.10, 71.27, 71.23, 60.99, 57.67, 57.62, 56.29, 38.45, 38.29, 28.01, 26.22, 25.35, 25.25, 21.19, 20.76. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>23</sub>H<sub>27</sub>O<sub>5</sub> [M + H]<sup>+</sup>: 383.18530, found 383.18513. HPLC (50–60% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.39 min, purity (220 nm) = 99% (no peak separation achieved).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (<i>S</i>)-1-((<i>S</i>)-2-Cyclohexyl-2-(3-hydroxy-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylic Acid (<b>22a</b>) and (<i>S</i>)-1-((<i>R</i>)-2-Cyclohexyl-2-(3-hydroxy-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylic Acid (<b>22b</b>)</h3><div class="NLM_p last">To a stirred solution of <b>21</b> (3.11 g, 8.14 mmol mmol, 1.00 equiv) in DCM (125 mL) at 0 °C was added DMF (6 μL, 0,08 mmol, 0.01 equiv) and thionyl chloride (0.89 mL, 12.21 mmol, 1.50 equiv), and the reaction mixture was stirred at 0 °C overnight. Then additional portions of DMF (0.05 equiv) and thionyl chloride (0.20 equiv) were added, and the reaction was stirred for one more hour. The volatiles were removed under reduced pressure to yield the crude acid chloride as a yellow oil, which was used in the following step without further purification. (<i>S</i>)-piperidine-2-carboxylic acid (1168 mg, 8.95 mmol, 1.10 equiv) was dissolved in 1 M KOH aqueous solution (18 mL, 17.9 mmol, 2.20 equiv) and cooled to 0 °C. In a separate flask, the crude acid chloride was dissolved in 1,4-dioxane (25 mL) and was added to the above aqueous solution at 0 °C. After stirring for 1 h at 0 °C, the reaction mixture was allowed to reach ambient temperature and was left stirring overnight. Then the reaction mixture was quenched and acidified with 1 M HCl solution (pH = 1–2), extracted with EA (3 × 30 mL), dried over MgSO<sub>4,</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 5:1 + 1% HCOOH) to afford product <b>S7</b> (2858 mg, 71%, white solid, mixture of diastereomers, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) and recovered starting material <b>21</b> (885 mg, 28%). Subsequently, a stirred solution of <b>S7</b> (6.15 g, 12.46 mmol, 1.00 equiv) in THF/MeOH (v/v = 1:1, 300 mL) was degassed by sparging with argon for 15 min. After addition of Pd/C (10 wt %, 1330 mg, 1.25 mmol, 0.10 equiv), the solution was sparged with hydrogen for 15 min and was then stirred under a hydrogen atmosphere at room temperature overnight. The dark suspension was filtered through Celite, and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (DCM/MeOH/HCOOH = 90:1:1) to afford the desired product <b>22a</b> (2089 mg, 41%) and the corresponding diastereomer <b>22b</b> (2210 mg, 44%) as white solids. A detailed description of the assignment of diastereomers <b>22a</b> and <b>22b</b> to the desired (<i>S</i>) configuration can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>. Data for <b>22a</b>: TLC (DCM/MeOH/HCOOH = 80:1:1, v/v/v; two runs): <i>R</i><sub><i>f</i></sub> = 0.36. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.32:1, <b>A</b>/<b>B</b>): δ 6.47–6.34 (m, 1.96H, <b>A</b>+<b>B</b>), 4.78 (d, <i>J</i> = 5.5 Hz, 0.17H, <b>A</b>), 4.58 (d, <i>J</i> = 13.7 Hz, 0.13H, <b>A</b>), 3.95–3.66 (m, 6.90H, <b>A</b>+<b>B</b>), 3.30 (d, <i>J</i> = 9.6 Hz, 0.76H, <b>B</b>), 3.10 (d, <i>J</i> = 9.7 Hz, 0.24H, <b>A</b>), 2.97–2.85 (m, 0.70H, <b>B</b>), 2.64 (td, <i>J</i> = 2.8, 13.4 Hz, 0.22H, <b>A</b>), 2.29–2.17 (m, 0.75H, <b>B</b>), 2.12–1.97 (m, 1.30H, <b>A</b>+<b>B</b>), 1.92–1.76 (m, 1.22H, <b>A</b>+<b>B</b>), 1.75–1.50 (m, 6.15H, <b>A</b>+<b>B</b>), 1.50–1.00 (m, 7.50H, <b>A</b>+<b>B</b>), 0.98–0.63 (m, 2.32H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 3:1, <b>A</b>:<b>B</b>): δ 175.65 (<b>A</b>), 175.52 (<b>B</b>), 173.75 (<b>B</b>), 172.53 (<b>A</b>), 152.84 (<b>A</b>), 152.60 (<b>B</b>), 149.30 (<b>A</b>), 148.99 (<b>B</b>), 134.73 (<b>A</b>), 134.64 (<b>B</b>), 134.60 (<b>A</b>), 133.53 (<b>B</b>), 109.29 (<b>B</b>), 108.72 (<b>A</b>), 104.27 (<b>B</b>), 103.57 (<b>A</b>), 61.04 (<b>A</b>), 60.96 (<b>B</b>), 56.13 (<b>A</b>), 56.02 (<b>B</b>), 55.79 (<b>A</b>), 55.44 (<b>A</b>), 55.22 (<b>B</b>), 53.53 (<b>A</b>), 52.52 (<b>B</b>), 43.79 (<b>B</b>), 41.12 (<b>B</b>), 39.72 (<b>A</b>), 32.89 (<b>B</b>), 32.85 (<b>A</b>), 30.77 (<b>B</b>), 30.65 (<b>A</b>), 26.92 (<b>A</b>), 26.64 (<b>B</b>), 26.30 (<b>B</b>), 26.22 (<b>A</b>), 25.34 (<b>B</b>), 24.49 (<b>A</b>), 20.96 (<b>A</b>), 20.86 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>22</sub>H<sub>32</sub>NO<sub>6</sub> [M + H]<sup>+</sup>: 406.22241, found 406.22231. HPLC (30–50% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 13.42 min, purity (220 nm) = 99%. Data for <b>22b</b>: TLC (DCM/MeOH/HCOOH = 80:1:1, v/v/v; two runs): <i>R</i><sub><i>f</i></sub> = 0.38. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.12:1, <b>A</b>/<b>B</b>): δ 6.51–6.47 (m, 0.84H, <b>A</b>+<b>B</b>), 6.47–6.43 (m, 0.86H, <b>A</b>+<b>B</b>), 6.42–6.39 (m, 0.34H, <b>A</b>), 5.48–5.43 (m, 0.82H, <b>B</b>), 4.94–4.90 (m, 0.10H, <b>A</b>), 4.53 (d, <i>J</i> = 13.4 Hz, 0.08H, <b>A</b>), 4.01–3.94 (m, 0.86H, <b>B</b>), 3.87–3.84 (m, 2.56H, <b>A</b>+<b>B</b>), 3.84–3.77 (m, 3.43H, <b>A</b>+<b>B</b>), 3.36 (d, <i>J</i> = 10.0 Hz, 0.84H, <b>B</b>), 3.23 (td, <i>J</i> = 3.1, 13.1 Hz, 0.99H, <b>A</b>+<b>B</b>), 2.80–2.72 (m, 0.10H, <b>A</b>), 2.30–2.14 (m, 1.03H, <b>A</b>+<b>B</b>), 2.14–2.01 (m, 1.04H, <b>A</b>+<b>B</b>), 1.89 (d, <i>J</i> = 12.7 Hz, 1.00H, <b>A</b>+<b>B</b>), 1.74–1.42 (m, 6.60H, <b>A</b>+<b>B</b>), 1.42–1.20 (m, 4.08H, <b>A</b>+<b>B</b>), 1.19–0.93 (m, 3.33H, <b>A</b>+<b>B</b>), 0.93–0.79 (m, 1.25H, <b>A</b>+<b>B</b>), 0.79–0.65 (m, 1.05H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:7, <b>A</b>:<b>B</b>): δ 176.28 (<b>B</b>), 176.20 (<b>A</b>), 173.87 (<b>B</b>), 172.98 (<b>A</b>), 152.68 (<b>B</b>), 149.17 (<b>B</b>), 134.87 (<b>B</b>), 134.61 (<b>B</b>), 133.67 (<b>A</b>), 109.65 (<b>A</b>), 108.95 (<b>B</b>), 104.22 (<b>A</b>), 103.90 (<b>B</b>), 61.02 (<b>B</b>), 60.85 (<b>A</b>), 56.10 (<b>B</b>), 56.03 (<b>A</b>), 55.34 (<b>A</b>), 54.78 (<b>B</b>), 53.53 (<b>A</b>), 52.32 (<b>B</b>), 43.76 (<b>B</b>), 41.35 (<b>B</b>), 41.08 (<b>A</b>), 39.97 (<b>A</b>), 32.73 (<b>A</b>), 32.61 (<b>B</b>), 30.80 (<b>B</b>), 29.78 (<b>A</b>), 28.10 (<b>A</b>), 26.66 (<b>B</b>), 26.27 (<b>B</b>), 26.21 (<b>B</b>), 25.04 (<b>B</b>), 24.97 (<b>A</b>), 21.00 (<b>A</b>), 20.86 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>22</sub>H<sub>32</sub>NO<sub>6</sub> [M + H]<sup>+</sup>: 406.22241, found 406.22269. HPLC (30–50% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 11.78 min, purity (220 nm) = 99%.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> (<i>S</i>)-1-((<i>S</i>)-2-(3-(Allyloxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylic Acid (<b>23</b>)</h3><div class="NLM_p last">The substrate <b>22a</b> (350 mg, 0.86 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (358 mg, 2.59 mmol, 3.00 equiv) and allyl bromide (313 mg, 2.59 mmol, 3.00 equiv) in MeCN (20 mL). An additional portion of allyl bromide (0.50 equiv) was added until completion of the reaction. The product <b>S8</b> (390 mg, 93%, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) was obtained after purification by flash column chromatography (CH/EA = 6:1) as a light yellow oil. Subsequently, <b>S8</b> (390 mg, 0.80 mmol, 1.00 equiv) was applied to general procedure F with LiOH (78 mg, 3.20 mmol, 4.00 equiv) in H<sub>2</sub>O/MeOH/THF (12 mL). <b>23</b> (322 mg, 90%) was obtained after purification by flash column chromatography (CH/EA = 3:1 + 0.2% HCOOH) as a white solid. TLC (CH/EA = 2:1 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.34. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.36:1, <b>A</b>/<b>B</b>): δ 8.31 (br s, 0.79H, <b>A</b>+<b>B</b>), 6.47–6.40 (m, 2.12H, <b>A</b>+<b>B</b>), 6.07–5.95 (m, 0.99H, <b>A</b>+<b>B</b>), 5.39–5.33 (m, 1.03H, <b>A</b>+<b>B</b>), 5.29 (dd, <i>J</i> = 2.4, 6.1 Hz, 0.74H, <b>B</b>), 5.25–5.19 (m, 1.04H, <b>A</b>+<b>B</b>), 4.75 (d, <i>J</i> = 5.6 Hz, 0.26H, <b>A</b>), 4.63–4.49 (m, 2.41H, <b>A</b>+<b>B</b>), 3.90–3.73 (m, 7.19H, <b>A</b>+<b>B</b>), 3.32 (d, <i>J</i> = 9.6 Hz, 0.74H, <b>B</b>), 3.09 (d, <i>J</i> = 9.7 Hz, 0.26H, <b>A</b>), 2.94–2.85 (m, 0.73H, <b>B</b>), 2.67–2.59 (m, 0.26H, <b>A</b>), 2.26–2.19 (m, 0.73H, <b>B</b>), 2.11–1.99 (m, 1.48H, <b>A</b>+<b>B</b>), 1.93–1.79 (m, 1.06H, <b>A</b>+<b>B</b>), 1.72–1.52 (m, 6.16H, <b>A</b>+<b>B</b>), 1.51–1.24 (m, 3.90H, <b>A</b>+<b>B</b>), 1.21–0.97 (m, 3.11H, <b>A</b>+<b>B</b>), 0.96–0.85 (m, 0.82H, <b>A</b>+<b>B</b>), 0.83–0.64 (m, 1.38H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 3:1, <b>A</b>/<b>B</b>): δ 175.84 (<b>A</b>), 175.71 (<b>B</b>), 173.68 (<b>B</b>), 172.26 (<b>A</b>), 153.62 (<b>A</b>), 153.28 (<b>B</b>), 152.13 (<b>A</b>), 151.94 (<b>B</b>), 137.59 (<b>A</b>), 137.55 (<b>B</b>), 134.09 (<b>A</b>), 133.57 (<b>B</b>), 133.39 (<b>A</b>), 132.98 (<b>B</b>), 117.68 (<b>A</b>), 117.53 (<b>B</b>), 108.10 (<b>B</b>), 107.58 (<b>A</b>), 106.01 (<b>B</b>), 105.33 (<b>A</b>), 70.04 (<b>B</b>), 69.96 (<b>A</b>), 60.96 (<b>A</b>), 60.88 (<b>B</b>), 56.41 (<b>A</b>), 56.26 (<b>B</b>), 55.73 (<b>A</b>), 55.39 (<b>B</b>), 52.44 (<b>B</b>), 43.77 (<b>B</b>), 41.29 (<b>B</b>), 41.20 (<b>A</b>), 39.65 (<b>A</b>), 32.93 (<b>B</b>), 30.80 (<b>B</b>), 30.68 (<b>A</b>), 26.88 (<b>A</b>), 26.66 (<b>B</b>), 26.55 (<b>B</b>), 26.33 (<b>A</b>+<b>B</b>), 26.22 (<b>A</b>), 25.37 (<b>B</b>), 24.50 (<b>A</b>), 20.98 (<b>A</b>), 20.89 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>25</sub>H<sub>36</sub>NO<sub>6</sub> [M + H]<sup>+</sup>: 446.25371, found 446.25383. HPLC (40–60% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 13.45 min, purity (220 nm) = 97%.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> (<i>S</i>)-1-((<i>S</i>)-2-(3-(But-3-en-1-yloxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylic Acid (<b>24</b>)</h3><div class="NLM_p last">The substrate <b>22a</b> (400 mg, 0.99 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (409 mg, 2.96 mmol, 3.00 equiv) and 4-bromo-1-butene (532 mg, 3.94 mmol, 4.00 equiv) in MeCN (20 mL). Additional portions of K<sub>2</sub>CO<sub>3</sub> (2 × 2.00 equiv) and 4-bromo-1-butene (2.00 + 3.00 equiv) were added until completion of the reaction. <b>S9</b> (443 mg, 87%) was obtained as a light yellow resin after purification by flash column chromatography (CH/EA = 7:1, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>). Subsequently, the substrate <b>S9</b> (360 mg, 0.70 mmol, 1.00 equiv) was applied to general procedure F with LiOH (67 mg, 2.80 mmol, 4.00 equiv) in H<sub>2</sub>O/MeOH/THF (16 mL). <b>24</b> (298 mg, 93%) was obtained as a white solid after purification by flash column chromatography (CH/EA = 4:1 + 0.5% HCOOH). TLC (CH/EA = 2:1 + 1% HCOOH, v/v): <i>R</i><sub><i>f</i></sub> = 0.37. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.36:1, <b>A</b>/<b>B</b>): δ 8.52 (br s, 0.81H, <b>A</b>+<b>B</b>), 6.51–6.37 (m, 2.16H, <b>A</b>+<b>B</b>), 5.97–5.83 (m, 0.99H, <b>A</b>+<b>B</b>), 5.33–5.27 (m, 0.73H, <b>B</b>), 5.18–5.03 (m, 2.08H, <b>A</b>+<b>B</b>), 4.76 (d, <i>J</i> = 5.7 Hz, 0.26H, <b>A</b>), 4.63–4.55 (m, 0.26H, <b>A</b>), 4.01 (q, <i>J</i> = 6.6 Hz, 2.21H, <b>A</b>+<b>B</b>), 3.92–3.84 (m, 0.77H, <b>B</b>), 3.83–3.72 (m, 6.23H, <b>A</b>+<b>B</b>), 3.32 (d, <i>J</i> = 9.6 Hz, 0.76H, <b>B</b>), 3.10 (d, <i>J</i> = 9.7 Hz, 0.26H, <b>A</b>), 2.95–2.86 (m, 0.71H, <b>B</b>), 2.68–2.60 (m, 0.28H, <b>A</b>), 2.57–2.47 (m, 2.15H, <b>A</b>+<b>B</b>), 2.26–2.19 (m, 0.76H, <b>B</b>), 2.12–1.99 (m, 1.46H, <b>A</b>+<b>B</b>), 1.93–1.79 (m, 1.09H, <b>A</b>+<b>B</b>), 1.73–1.52 (m, 6.21H, <b>A</b>+<b>B</b>), 1.50–0.98 (m, 7.17H, <b>A</b>+<b>B</b>), 0.96–0.85 (m, 0.87H, <b>B</b>), 0.83–0.65 (m, 1.38H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:5, <b>A</b>/<b>B</b>): δ 175.74 (<b>B</b>), 173.60 (<b>B</b>), 172.31 (<b>A</b>), 153.51 (<b>A</b>), 153.20 (<b>B</b>), 152.58 (<b>A</b>), 152.26 (<b>B</b>), 137.54 (<b>A</b>), 137.45 (<b>B</b>), 134.72 (<b>B</b>), 134.55 (<b>A</b>), 134.14 (<b>A</b>), 133.01 (<b>B</b>), 117.22 (<b>A</b>), 117.08 (<b>B</b>), 107.67 (<b>B</b>), 107.07 (<b>A</b>), 105.95 (<b>B</b>), 105.40 (<b>A</b>), 68.57 (<b>A</b>), 68.46 (<b>B</b>), 60.96 (<b>A</b>), 60.88 (<b>B</b>), 56.43 (<b>A</b>), 56.27 (<b>B</b>), 55.77 (<b>A</b>), 55.67 (<b>A</b>), 55.38 (<b>B</b>), 52.41 (<b>B</b>), 43.75 (<b>B</b>), 41.29 (<b>B</b>), 41.20 (<b>A</b>), 39.65 (<b>A</b>), 33.92 (<b>B</b>), 32.93 (<b>B</b>), 30.80 (<b>B</b>), 30.68 (<b>A</b>), 26.86 (<b>A</b>), 26.64 (<b>B</b>), 26.59 (<b>B</b>), 26.37 (<b>A</b>), 26.32 (<b>B</b>), 26.21 (<b>A</b>), 25.38 (<b>B</b>), 24.51 (<b>A</b>), 20.97 (<b>A</b>), 20.91 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>26</sub>H<sub>38</sub>NO<sub>6</sub> [M + H]<sup>+</sup>: 460.26936, found 460.26956. HPLC (40–60% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 16.20 min, purity (220 nm) = 95%.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> (<i>S</i>)-1-((<i>S</i>)-2-(3-(2-(Allyloxy)ethoxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylic Acid (<b>25</b>)</h3><div class="NLM_p last">The substrate <b>22a</b> (502 mg, 1.24 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (684 mg, 4.95 mmol, 4.00 equiv) and <b>16</b> (1587 mg, 6.19 mmol, 5.00 equiv) in MeCN (20 mL). <b>S10</b> (635 mg, 94%) was obtained as a light yellow oil after purification by flash column chromatography (CH/EA = 5:1, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>). Subsequently, the substrate <b>S10</b> (635 mg, 1.11 mmol, 1.00 equiv) was applied to general procedure F with LiOH (106 mg, 4.43 mmol, 4.00 equiv) in H<sub>2</sub>O/MeOH/THF (20 mL). <b>25</b> (511 mg, 94%) was obtained as a white solid after purification by flash column chromatography (CH/EA = 5:1 + 0.5% HCOOH). TLC (CH/EA = 2:1 + 1% HCOOH, v/v): <i>R</i><sub><i>f</i></sub> = 0.34. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.34:1, <b>A</b>/<b>B</b>): δ 9.33 (br s, 0.90H, <b>A</b>+<b>B</b>), 6.48–6.43 (m, 2.11H, <b>A</b>+<b>B</b>), 5.96–5.85 (m, 1.06H, <b>A</b>+<b>B</b>), 5.33–5.24 (m, 1.89H, <b>A</b>+<b>B</b>), 5.20–5.14 (m, 1.07H, <b>A</b>+<b>B</b>), 4.75 (d, <i>J</i> = 5.6 Hz, 0.26H, <b>A</b>), 4.62–4.56 (m, 0.26H, <b>A</b>), 4.17–4.05 (m, 4.11H, <b>A</b>+<b>B</b>), 3.88–3.74 (m, 10.02H, <b>A</b>+<b>B</b>), 3.32 (d, <i>J</i> = 9.5 Hz, 0.75H, <b>B</b>), 3.10 (d, <i>J</i> = 9.6 Hz, 0.24H, <b>A</b>), 2.91–2.82 (m, 0.75H, <b>B</b>), 2.67–2.60 (m, 0.25H, <b>A</b>), 2.27–2.19 (m, 0.77H, <b>B</b>), 2.11–2.00 (m, 1.33H, <b>A</b>+<b>B</b>), 1.90–1.81 (m, 0.96H, <b>A</b>+<b>B</b>), 1.70–1.56 (m, 5.93H, <b>A</b>+<b>B</b>), 1.47–1.02 (m, 7.05H, <b>A</b>+<b>B</b>), 0.95–0.67 (m, 2.01H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 175.58 (<b>A</b>), 175.28 (<b>B</b>), 173.53 (<b>B</b>), 172.29 (<b>A</b>), 153.54 (<b>A</b>), 153.24 (<b>B</b>), 152.57 (<b>A</b>), 152.20 (<b>B</b>), 137.70 (<b>A</b>), 137.68 (<b>B</b>), 134.67 (<b>A</b>), 134.63 (<b>B</b>), 134.18 (<b>A</b>), 132.98 (<b>B</b>), 117.35 (<b>B</b>), 117.29 (<b>A</b>), 108.37 (<b>B</b>), 107.57 (<b>A</b>), 106.19 (<b>B</b>), 105.56 (<b>A</b>), 72.38 (<b>B</b>), 69.04 (<b>A</b>), 68.94 (<b>B</b>), 68.74 (<b>B</b>), 68.63 (<b>A</b>), 60.95 (<b>A</b>), 60.88 (<b>B</b>), 56.42 (<b>A</b>), 56.26 (<b>B</b>), 55.75 (<b>A</b>), 55.60 (<b>A</b>), 55.40 (<b>B</b>), 52.39 (<b>B</b>), 43.74 (<b>B</b>), 41.22 (<b>B</b>), 39.64 (<b>A</b>), 32.96 (<b>B</b>), 32.91 (<b>A</b>), 30.80 (<b>B</b>), 30.67 (<b>A</b>), 26.88 (<b>A</b>), 26.65 (<b>B</b>), 26.54 (<b>B</b>), 26.33 (<b>B</b>), 26.22 (<b>A</b>), 25.36 (<b>B</b>), 24.51 (<b>A</b>), 20.97 (<b>A</b>), 20.88 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>27</sub>H<sub>40</sub>NO<sub>7</sub> [M + H]<sup>+</sup>: 490.27993, found 490.28008. HPLC (50–60% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 8.17 min, purity (220 nm) = 94%.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> (<i>S</i>)-1-((<i>S</i>)-2-(3-(3-(Allyloxy)propoxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylic Acid (<b>26</b>)</h3><div class="NLM_p last">The substrate <b>22a</b> (474 mg, 1.17 mmol, 1.00 equiv) was applied to general procedure A with K<sub>2</sub>CO<sub>3</sub> (647 mg, 4.68 mmol, 4.00 equiv) and <b>17</b> (1582 mg, 5.85 mmol, 5.00 equiv) in MeCN (20 mL). <b>S11</b> (694 mg, 99%, analytics shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) was obtained as a colorless resin after purification by flash column chromatography (CH/EA = 5:1). Subsequently, the substrate <b>S11</b> (694 mg, 1.15 mmol, 1.00 equiv) was applied to general procedure F with LiOH (110 mg, 4.60 mmol, 4.00 equiv) in H<sub>2</sub>O/MeOH/THF (20 mL). <b>26</b> (563 mg, 97%) was obtained as white solid after purification by flash column chromatography (CH/EA = 4:1 + 0.5% HCOOH). TLC (CH/EA = 2:1 + 1% HCOOH, v/v): <i>R</i><sub><i>f</i></sub> = 0.35. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.33:1, <b>A</b>/<b>B</b>): δ 8.73 (br s, 0.81H, <b>A</b>+<b>B</b>), 6.49–6.40 (m, 2.01H, <b>A</b>+<b>B</b>), 5.92–5.83 (m, 1.04H, <b>A</b>+<b>B</b>), 5.30 (dd, <i>J</i> = 2.3, 6.0 Hz, 0.74H, <b>B</b>), 5.27–5.21 (m, 1.09H, <b>A</b>+<b>B</b>), 5.16–5.11 (m, 1.04H, <b>A</b>+<b>B</b>), 4.77–4.74 (m, 0.23H, <b>A</b>), 4.61–4.55 (m, 0.23H, <b>A</b>), 4.09–4.02 (m, 2.00H, <b>A</b>+<b>B</b>), 4.00–3.94 (m, 2.45H, <b>A</b>+<b>B</b>), 3.87–3.73 (m, 7.33H, <b>A</b>+<b>B</b>), 3.64–3.58 (m, 2.14H, <b>A</b>+<b>B</b>), 3.32 (d, <i>J</i> = 9.4 Hz, 0.72H, <b>B</b>), 3.11 (d, <i>J</i> = 9.7 Hz, 0.24H, <b>A</b>), 2.88–2.81 (m, 0.70H, <b>B</b>), 2.63 (td, <i>J</i> = 2.6, 13.3 Hz, 0.23H, <b>A</b>), 2.24–2.19 (m, 0.71H, <b>B</b>), 2.08–1.99 (m, 3.12H, <b>A</b>+<b>B</b>), 1.91–1.80 (m, 1.27H, <b>A</b>+<b>B</b>), 1.68–1.54 (m, 6.02H, <b>A</b>+<b>B</b>), 1.46–1.25 (m, 3.49H, <b>A</b>+<b>B</b>), 1.22–1.01 (m, 2.50H, <b>A</b>+<b>B</b>), 0.94–0.86 (m, 0.80H, <b>B</b>), 0.81–0.67 (m, 1.19H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 175.12 (<b>A</b>), 174.90 (<b>B</b>), 173.31 (<b>B</b>), 172.30 (<b>A</b>), 153.48 (<b>A</b>), 153.12 (<b>B</b>), 152.65 (<b>A</b>), 152.25 (<b>B</b>), 137.43 (<b>B</b>), 134.85 (<b>A</b>), 134.67 (<b>B</b>), 134.22 (<b>A</b>), 133.02 (<b>B</b>), 117.28 (<b>B</b>), 116.95 (<b>A</b>), 107.84 (<b>B</b>), 107.04 (<b>A</b>), 106.02 (<b>B</b>), 105.21 (<b>A</b>), 71.97 (<b>A</b>), 71.93 (<b>B</b>), 66.92 (<b>B</b>), 66.27 (<b>A</b>), 66.21 (<b>B</b>), 60.88 (<b>A</b>), 60.83 (<b>B</b>), 56.37 (<b>A</b>), 56.22 (<b>B</b>), 55.74 (<b>A</b>), 55.60 (<b>A</b>), 55.40 (<b>B</b>), 52.30 (<b>B</b>), 43.71 (<b>B</b>), 41.14 (<b>B</b>), 39.62 (<b>A</b>), 32.97 (<b>B</b>), 32.89 (<b>A</b>), 30.79 (<b>B</b>), 30.66 (<b>A</b>), 29.83 (<b>A</b>), 29.68 (<b>B</b>), 26.84 (<b>A</b>), 26.64 (<b>B</b>), 26.54 (<b>A</b>), 26.33 (<b>B</b>), 26.20 (<b>A</b>), 25.38 (<b>B</b>), 24.50 (<b>A</b>), 20.94 (<b>A</b>), 20.88 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>28</sub>H<sub>42</sub>NO<sub>7</sub> [M + H]<sup>+</sup>: 504.29558, found 504.29563. HPLC (50–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.95 min, purity (220 nm) = 91%.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(3-(allyloxy)-4,5-dimethoxy-phenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>27</b>)</h3><div class="NLM_p last">The substrate <b>23</b> (246 mg, 0.55 mmol, 1.00 equiv) was applied to general procedure G with <b>14</b> (181 mg, 0.55 mmol, 1.00 equiv), 4-(1-pyrrolidinyl)pyridine (334 mg, 2.21 mmol, 4.00 equiv), and EDC (119 mg, 0.61 mmol, 1.10 equiv) in dry toluene (40 mL). After stirring at room temperature for 3 h, <b>27</b> (267 mg, 64%) was obtained after purification by flash column chromatography (CH/EA = 4:1 + 1% MeOH) as light yellow resin. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.35. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.29 (t, <i>J</i> = 7.9 Hz, 0.31H), 7.21–7.17 (m, 0.12H), 7.11 (t, <i>J</i> = 7.9 Hz, 0.58H, <b>A</b>), 6.97–6.90 (m, 0.66H, <b>A</b>), 6.90–6.85 (m, 0.33H), 6.81–6.74 (m, 2.03H, <b>A</b>), 6.71–6.67 (m, 0.93H, <b>A</b>), 6.67–6.58 (m, 1.80H, <b>A</b>), 6.53–6.48 (m, 1.42H, <b>A</b>), 6.46–6.40 (m, 1.21H, <b>A</b>), 6.09–5.93 (m, 1.95H, <b>A</b>), 5.82–5.78 (m, 0.29H), 5.58 (dd, <i>J</i> = 5.6, 8.1 Hz, 0.70H, <b>A</b>), 5.48–5.44 (m, 0.59H, <b>A</b>), 5.44–5.34 (m, 1.87H, <b>A</b>), 5.33–5.17 (m, 2.38H, <b>A</b>), 4.73–4.67 (m, 0.34H), 4.60–4.45 (m, 4.48H, <b>A</b>), 3.97–3.90 (m, 0.78H, <b>A</b>), 3.88–3.75 (m, 10.90H, <b>A</b>), 3.69 (s, 1.79H, <b>A</b>), 3.38–3.32 (m, 0.70H, <b>A</b>), 3.09 (d, <i>J</i> = 9.5 Hz, 0.04H), 3.01–2.91 (m, 0.37H), 2.80 (td, <i>J</i> = 2.9, 13.3 Hz, 0.58H, <b>A</b>), 2.65–2.50 (m, 1.01H, <b>A</b>), 2.50–2.34 (m, 1.42H, <b>A</b>), 2.32–2.23 (m, 1.05H, <b>A</b>), 2.15–1.79 (m, 4.32H, <b>A</b>), 1.72–1.51 (m, 6.67H, <b>A</b>), 1.47–1.05 (m, 6.68H, <b>A</b>), 1.04–0.84 (m, 1.19H, <b>A</b>), 0.80–0.68 (m, 0.81H, <b>A</b>), 0.65–0.50 (m, 0.55H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 172.81, 172.41 (<b>A</b>), 172.31, 170.76, 170.66 (<b>A</b>), 170.52, 158.98, 158.60 (<b>A</b>), 153.61, 153.37 (<b>A</b>), 153.27, 152.12, 152.03 (<b>A</b>), 151.98, 149.12, 148.99 (<b>A</b>), 147.63, 147.43 (<b>A</b>), 142.31, 141.87 (<b>A</b>), 141.62, 137.68, 137.62 (<b>A</b>), 134.19, 133.64 (<b>A</b>), 133.59 (<b>A</b>), 133.56 (<b>A</b>), 133.49, 133.44, 133.37 (<b>A</b>), 133.16, 129.86, 129.66 (<b>A</b>), 120.41 (<b>A</b>), 120.34, 120.26, 119.30, 118.69 (<b>A</b>), 118.61, 117.96, 117.73 (<b>A</b>), 117.67, 117.49 (<b>A</b>), 114.35, 114.02 (<b>A</b>), 113.49, 113.36 (<b>A</b>), 112.89, 111.97 (<b>A</b>), 111.91, 111.85, 111.54, 111.47 (<b>A</b>), 108.21 (<b>A</b>), 107.75, 106.10, 106.03 (<b>A</b>), 105.47, 75.74 (<b>A</b>), 70.05 (<b>A</b>), 68.96, 68.88 (<b>A</b>), 60.98, 60.87 (<b>A</b>), 56.47, 56.25, 56.19 (<b>A</b>), 56.10, 56.06 (<b>A</b>), 56.01, 55.97 (<b>A</b>), 55.91, 55.86, 55.15, 55.08 (<b>A</b>), 52.32, 52.18 (<b>A</b>), 43.77 (<b>A</b>), 41.58, 41.51 (<b>A</b>), 41.24, 39.57, 38.26, 38.08 (<b>A</b>), 38.01, 37.96, 33.13, 32.91 (<b>A</b>), 31.64, 31.60, 31.34, 31.14 (<b>A</b>), 30.84, 30.79 (<b>A</b>), 30.61, 26.94 (<b>A</b>), 26.73 (<b>A</b>), 26.59, 26.37 (<b>A</b>), 26.31 (<b>A</b>), 26.23, 25.71 (<b>A</b>), 24.49, 21.13 (<b>A</b>), 21.07, 20.85. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 756.41061, found 756.41056. HPLC (75–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.25 min, purity (220 nm) = 98%.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(3-(But-3-en-1-yloxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>28</b>)</h3><div class="NLM_p last">The substrate <b>24</b> (289 mg, 0.63 mmol, 1.00 equiv) was applied to general procedure G with <b>14</b> (207 mg, 0.63 mmol, 1.00 equiv), 4-(1-pyrrolidinyl)pyridine (380 mg, 2.52 mmol, 4.00 equiv), and EDC (135 mg, 0.69 mmol, 1.10 equiv) in dry toluene (50 mL). After stirring at room temperature for 4 h, <b>28</b> (287 mg, 59%) was obtained as a light yellow resin after purification by flash column chromatography (CH/EA = 4:1 + 1% MeOH). TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.39. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.28 (t, <i>J</i> = 7.9 Hz, 0.29H), 7.19 (t, <i>J</i> = 8.1 Hz, 0.10H), 7.10 (t, <i>J</i> = 7.9 Hz, 0.60H, <b>A</b>), 6.96–6.90 (m, 0.61H, <b>A</b>), 6.89–6.85 (m, 0.32H), 6.80–6.73 (m, 2.04H, <b>A</b>), 6.71–6.58 (m, 2.51H, <b>A</b>), 6.52–6.48 (m, 1.41H, <b>A</b>), 6.43–6.39 (m, 1.10H, <b>A</b>), 6.08–5.98 (m, 0.91H, <b>A</b>), 5.94–5.78 (m, 1.16H, <b>A</b>), 5.57 (dd, <i>J</i> = 5.6, 8.0 Hz, 0.72H, <b>A</b>), 5.48–5.45 (m, 0.60H, <b>A</b>), 5.43–5.36 (m, 1.01H, <b>A</b>), 5.31–5.23 (m, 1.01H, <b>A</b>), 5.19–5.03 (m, 2.03H, <b>A</b>), 4.73–4.68 (m, 0.28H), 4.58–4.46 (m, 2.39H, <b>A</b>), 4.06–4.01 (m, 0.62H), 4.01–3.91 (m, 2.26H, <b>A</b>), 3.87–3.80 (m, 8.47H, <b>A</b>), 3.79–3.76 (m, 2.56H, <b>A</b>), 3.69 (s, 1.87H, <b>A</b>), 3.39–3.33 (m, 0.73H, <b>A</b>), 2.98–2.93 (m, 0.30H), 2.79 (td, <i>J</i> = 2.9, 13.3 Hz, 0.59H, <b>A</b>), 2.63–2.34 (m, 4.30H, <b>A</b>), 2.31–2.24 (m, 0.96H, <b>A</b>), 2.14–2.03 (m, 1.63H, <b>A</b>), 1.99–1.79 (m, 2.18H, <b>A</b>), 1.71–1.51 (m, 5.98H, <b>A</b>), 1.47–1.06 (m, 6.21H, <b>A</b>), 1.02–0.85 (m, 0.99H, <b>A</b>), 0.79–0.70 (m, 0.71H, <b>A</b>), 0.64–0.53 (m, 0.44H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 172.81, 172.39 (<b>A</b>), 172.33, 170.73, 170.62 (<b>A</b>), 170.48, 158.95, 158.71, 158.55 (<b>A</b>), 153.48, 153.26 (<b>A</b>), 153.15, 152.55, 152.39 (<b>A</b>), 152.31, 149.08, 148.96 (<b>A</b>), 147.60, 147.39 (<b>A</b>), 142.31, 141.83 (<b>A</b>), 141.58, 137.62, 137.51 (<b>A</b>), 137.45, 134.72 (<b>A</b>), 134.58, 134.19, 133.61 (<b>A</b>), 133.56 (<b>A</b>), 133.41, 133.34 (<b>A</b>), 133.13, 129.82, 129.63 (<b>A</b>), 120.37 (<b>A</b>), 120.31, 120.23, 119.27, 118.64 (<b>A</b>), 118.55, 117.91, 117.68 (<b>A</b>), 117.22, 117.04 (<b>A</b>), 114.35, 114.00 (<b>A</b>), 113.45, 113.31 (<b>A</b>), 112.82, 111.95 (<b>A</b>), 111.90, 111.82, 111.51, 111.44 (<b>A</b>), 107.68 (<b>A</b>), 107.21, 106.07, 105.97 (<b>A</b>), 105.55, 75.72, 75.65 (<b>A</b>), 68.92, 68.84 (<b>A</b>), 68.62, 68.46 (<b>A</b>), 68.41, 60.95, 60.84 (<b>A</b>), 56.48, 56.25, 56.18 (<b>A</b>), 56.06, 56.02 (<b>A</b>), 55.98, 55.93 (<b>A</b>), 55.90, 55.80, 55.14, 55.05 (<b>A</b>), 52.27, 52.14 (<b>A</b>), 43.74 (<b>A</b>), 41.56, 41.51 (<b>A</b>), 41.24, 39.54, 38.23, 38.04 (<b>A</b>), 37.91, 33.93 (<b>A</b>), 33.10, 32.89 (<b>A</b>), 31.61, 31.57, 31.29, 31.08 (<b>A</b>), 30.77 (<b>A</b>), 30.58, 26.90 (<b>A</b>), 26.70 (<b>A</b>), 26.55, 26.33 (<b>A</b>), 26.28 (<b>A</b>), 26.28, 26.21, 25.68 (<b>A</b>), 25.63, 24.46, 21.10 (<b>A</b>), 21.03, 20.81. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>60</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 770.42626, found 770.42648. HPLC (75–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 10.99 min, purity (220 nm) = 97%.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(3-(2-(Allyloxy)ethoxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>29</b>)</h3><div class="NLM_p last">The substrate <b>25</b> (508 mg, 1.04 mmol, 1.00 equiv) was applied to general procedure G with <b>14</b> (341 mg, 1.04 mmol, 1.00 equiv), 4-(1-pyrrolidinyl)pyridine (628 mg, 4.15 mmol, 4.00 equiv), and EDC (223 mg, 1.14 mmol, 1.10 equiv) in dry toluene (75 mL). After stirring at room temperature for 4 h and performing the standard workup procedure, the crude product was reacted with acetic anhydride (234 mg), pyridine (75 μL), and DMAP (11 mg) in DCM (15 mL) to convert unreacted <b>14</b> into the corresponding acetate and thereby facilitating the purification of <b>29</b>. <b>29</b> (524 mg, 63%) was obtained as a colorless resin after purification by flash column chromatography (CH/EA = 5:1). TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.14. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.29 (t, <i>J</i> = 7.8 Hz, 0.30H), 7.21–7.17 (m, 0.10H), 7.11 (t, <i>J</i> = 7.9 Hz, 0.58H, <b>A</b>), 6.97–6.90 (m, 0.67H, <b>A</b>), 6.89–6.85 (m, 0.34H, <b>A</b>), 6.81–6.73 (m, 2.13H, <b>A</b>), 6.72–6.57 (m, 2.88H, <b>A</b>), 6.54–6.49 (m, 1.51H, <b>A</b>), 6.47–6.39 (m, 1.23H, <b>A</b>), 6.09–5.98 (m, 1.01H, <b>A</b>), 5.96–5.85 (m, 0.98H, <b>A</b>), 5.82–5.77 (m, 0.30H), 5.60–5.54 (m, 0.72H, <b>A</b>), 5.48–5.44 (m, 0.62H, <b>A</b>), 5.44–5.35 (m, 1.20H, <b>A</b>), 5.33–5.24 (m, 2.12H, <b>A</b>), 5.21–5.14 (m, 1.02H, <b>A</b>), 4.74–4.64 (m, 0.35H), 4.58–4.45 (m, 2.47H, <b>A</b>), 4.18–4.02 (m, 4.36H, <b>A</b>), 3.98–3.62 (m, 16.72H, <b>A</b>), 3.39–3.32 (m, 0.75H, <b>A</b>), 3.09 (d, <i>J</i> = 9.6 Hz, 0.04H), 3.01–2.91 (m, 0.36H), 2.78 (td, <i>J</i> = 2.9, 13.3 Hz, 0.59H, <b>A</b>), 2.65–2.52 (m, 1.03H, <b>A</b>), 2.50–2.33 (m, 1.46H, <b>A</b>), 2.33–2.21 (m, 1.12H, <b>A</b>), 2.15–1.77 (m, 4.52H, <b>A</b>), 1.76–1.48 (m, 7.00H, <b>A</b>), 1.48–1.04 (m, 6.93H, <b>A</b>), 1.03–0.84 (m, 1.22H, <b>A</b>), 0.80–0.68 (m, 0.85H, <b>A</b>), 0.65–0.49 (m, 0.54H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 172.40 (<b>A</b>), 172.35, 170.74, 170.63 (<b>A</b>), 170.48, 158.95, 158.72, 158.56 (<b>A</b>), 153.53, 153.34 (<b>A</b>), 153.24, 152.56, 152.31 (<b>A</b>), 152.23, 149.09, 148.97 (<b>A</b>), 147.60, 147.40 (<b>A</b>), 142.31, 141.85 (<b>A</b>), 141.60, 137.80, 137.69 (<b>A</b>), 134.81 (<b>A</b>), 134.73, 134.22, 133.64 (<b>A</b>), 133.56 (<b>A</b>), 133.47, 133.42, 133.36 (<b>A</b>), 133.15, 129.84, 129.67 (<b>A</b>), 120.39 (<b>A</b>), 120.32, 120.24, 119.28, 118.64 (<b>A</b>), 118.60, 117.92, 117.69 (<b>A</b>), 117.27, 117.09 (<b>A</b>), 114.36, 114.03 (<b>A</b>), 113.46, 113.29 (<b>A</b>), 112.84, 111.95 (<b>A</b>), 111.90, 111.83, 111.52, 111.43 (<b>A</b>), 108.28 (<b>A</b>), 107.74, 106.17, 106.06 (<b>A</b>), 105.68, 75.74, 75.70 (<b>A</b>), 72.41, 72.34 (<b>A</b>), 69.13, 68.96 (<b>A</b>), 68.93, 68.85 (<b>A</b>), 68.71 (<b>A</b>), 60.96, 60.85 (<b>A</b>), 56.47, 56.23, 56.16 (<b>A</b>), 56.07, 56.03 (<b>A</b>), 55.99, 55.95 (<b>A</b>), 55.90, 55.76, 55.11, 55.06 (<b>A</b>), 52.29, 52.16 (<b>A</b>), 43.75 (<b>A</b>), 41.56, 41.49 (<b>A</b>), 41.27, 39.55, 38.24, 38.07 (<b>A</b>), 37.92, 33.09, 32.90 (<b>A</b>), 31.59, 31.30, 31.10 (<b>A</b>), 30.78 (<b>A</b>), 30.59, 26.96, 26.91 (<b>A</b>), 26.84, 26.71 (<b>A</b>), 26.55, 26.34 (<b>A</b>), 26.29 (<b>A</b>), 26.21, 25.67 (<b>A</b>), 24.46, 21.10 (<b>A</b>), 21.04, 20.80. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>47</sub>H<sub>62</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 800.43682, found 800.43680. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 12.18 min, purity (220 nm) = 95%.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (<i>S</i>)-(<i>R</i>)-1-(3-(Allyloxy)phenyl)-3-(3,4-dimethoxyphenyl)propyl 1-((<i>S</i>)-2-(3-(3-(Allyloxy)propoxy)-4,5-dimethoxyphenyl)-2-cyclohexylacetyl)piperidine-2-carboxylate (<b>30</b>)</h3><div class="NLM_p last">The substrate <b>26</b> (545 mg, 1.08 mmol, 1.00 equiv) was applied to general procedure G with <b>14</b> (355 mg, 1.08 mmol, 1.00 equiv), 4-(1-pyrrolidinyl)pyridine (640 mg, 4.32 mmol, 4.00 equiv), and EDC (228 mg, 1.19 mmol, 1.10 equiv) in dry toluene (75 mL). After stirring at room temperature for 4 h and performing the standard workup procedure, the crude product was reacted with acetic anhydride (166 mg), pyridine (53 μL), and DMAP (8 mg) in DCM (10 mL) to convert unreacted <b>14</b> into the corresponding acetate and thereby facilitating the purification of <b>30</b>. <b>30</b> (522 mg, 59%) was obtained as a colorless resin after purification by flash column chromatography (CH/EA = 4:1). TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.40. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.30–7.26 (m, 0.19H), 7.20–7.16 (m, 0.10H), 7.09 (t, <i>J</i> = 7.9 Hz, 0.56H, <b>A</b>), 6.96–6.89 (m, 0.55H, <b>A</b>), 6.88–6.84 (m, 0.29H), 6.79–6.73 (m, 1.88H, <b>A</b>), 6.70–6.56 (m, 2.55H, <b>A</b>), 6.52–6.48 (m, 1.38H, <b>A</b>), 6.42 (s, 0.55H, <b>A</b>), 6.40–6.37 (m, 0.54H), 6.07–5.97 (m, 0.92H, <b>A</b>), 5.93–5.84 (m, 0.93H, <b>A</b>), 5.79 (dd, <i>J</i> = 6.2, 7.6 Hz, 0.27H), 5.55 (dd, <i>J</i> = 5.6, 8.1 Hz, 0.68H, <b>A</b>), 5.46 (d, <i>J</i> = 5.4 Hz, 0.55H, <b>A</b>), 5.42–5.35 (m, 1.09H, <b>A</b>), 5.29–5.22 (m, 2.01H, <b>A</b>), 5.16–5.11 (m, 0.99H, <b>A</b>), 4.70 (d, <i>J</i> = 5.6 Hz, 0.27H), 4.57–4.46 (m, 2.25H, <b>A</b>), 4.10–3.91 (m, 5.04H, <b>A</b>), 3.86–3.79 (m, 8.36H, <b>A</b>), 3.77–3.74 (m, 2.42H, <b>A</b>), 3.67–3.53 (m, 3.89H, <b>A</b>), 3.37–3.33 (m, 0.71H, <b>A</b>), 2.95 (d, <i>J</i> = 9.7 Hz, 0.30H), 2.78 (td, <i>J</i> = 2.8, 13.4 Hz, 0.54H, <b>A</b>), 2.62–2.50 (m, 0.80H, <b>A</b>), 2.48–2.33 (m, 1.22, <b>A</b>), 2.30–2.23 (m, 0.94H, <b>A</b>), 2.12–1.79 (m, 6.01H, <b>A</b>), 1.70–1.52 (m, 5.93H, <b>A</b>), 1.44–1.06 (m, 6.25H, <b>A</b>), 1.03–0.82 (m, 1.01H, <b>A</b>), 0.79–0.68 (m, 0.70H, <b>A</b>), 0.63–0.52 (m, 0.37H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 172.74, 172.34 (<b>A</b>), 172.30, 170.68, 170.59 (<b>A</b>), 170.44, 158.89, 158.66, 158.49 (<b>A</b>), 153.41, 153.26 (<b>A</b>), 153.15, 152.64, 152.40 (<b>A</b>), 152.33, 149.02, 148.89 (<b>A</b>), 147.53, 147.33 (<b>A</b>), 142.28, 141.81 (<b>A</b>), 141.53, 137.42, 137.35 (<b>A</b>), 134.95 (<b>A</b>), 134.90, 134.22, 133.57 (<b>A</b>), 133.55 (<b>A</b>), 133.47, 133.35, 133.30 (<b>A</b>), 133.09, 129.78, 129.72, 129.58 (<b>A</b>), 120.33 (<b>A</b>), 120.26, 120.18, 119.21, 118.57 (<b>A</b>), 118.49, 117.85, 117.63 (<b>A</b>), 116.88, 116.82 (<b>A</b>), 114.32, 113.95 (<b>A</b>), 113.39, 113.24 (<b>A</b>), 112.74, 111.89 (<b>A</b>), 111.83, 111.75, 111.44, 111.36 (<b>A</b>), 107.66 (<b>A</b>), 107.02, 105.71, 105.62 (<b>A</b>), 105.29, 76.82, 75.64, 75.60 (<b>A</b>), 71.95 (<b>A</b>), 68.86, 68.78 (<b>A</b>), 67.01 (<b>A</b>), 66.91, 66.29, 66.19 (<b>A</b>), 66.14, 60.87, 60.76 (<b>A</b>), 56.40, 56.16, 56.08 (<b>A</b>), 56.01, 55.97 (<b>A</b>), 55.92, 55.88 (<b>A</b>), 55.74, 55.08, 55.02 (<b>A</b>), 52.23, 52.08 (<b>A</b>), 43.70 (<b>A</b>), 41.51, 41.44 (<b>A</b>), 41.18, 39.50, 38.20, 38.00 (<b>A</b>), 37.87, 33.24, 33.05, 32.84 (<b>A</b>), 31.53, 31.24, 31.02 (<b>A</b>), 30.72 (<b>A</b>), 30.54, 29.88 (<b>A</b>), 26.86 (<b>A</b>), 26.66 (<b>A</b>), 26.51, 26.28 (<b>A</b>), 26.24, 26.23 (<b>A</b>), 26.17, 25.64 (<b>A</b>), 24.56, 24.42, 21.06 (<b>A</b>), 21.02, 20.77. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>48</sub>H<sub>63</sub>NO<sub>10</sub>Na [M + Na]<sup>+</sup>: 836.43442, found 836.43489. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 6.84 min, purity (220 nm) = 95%.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Macrocycles <b>31-(</b><i>Z</i><b>)</b> and <b>31-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>9</b> (357 mg, 0.47 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (40 mg, 0.05 mmol, 0.10 equiv) in DCM (1000 mL, 0.5 mM). After workup procedure a, <b>31-(</b><i>Z</i><b>)</b> (16 mg, 5%, white solid) and <b>31-(</b><i>E</i><b>)</b> (132 mg, 38%, white solid) were obtained as pure isomers after purification by flash column chromatography (CH/EA = 5:1) and semipreparative HPLC (70–100% solvent B). Another product fraction was obtained as a mixture of <i>E</i>/<i>Z</i> isomers (77 mg, 22%) and not further purified. Data for macrocycle <b>31-(</b><i>Z</i><b>)</b>: TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.15 (t, <i>J</i> = 7.7 Hz, 1H), 6.84–6.81 (m, 1H), 6.80–6.75 (m, 3H), 6.74 (d, <i>J</i> = 1.9 Hz, 1H), 6.72–6.68 (m, 1H), 6.32–6.29 (m, 2H), 6.06–5.99 (m, 1H), 5.89–5.79 (m, 2H), 5.50 (d, <i>J</i> = 5.4 Hz, 1H), 4.65–4.54 (m, 2H), 4.09–4.03 (m, 1H), 3.88 (s, 3H), 3.85 (s, 3H), 3.84–3.78 (m, 4H), 3.62 (s, 3H), 3.56–3.51 (m, 1H), 3.27 (d, <i>J</i> = 9.3 Hz, 1H), 2.69–2.62 (m, 1H), 2.58–2.49 (m, 2H), 2.33–2.27 (m, 1H), 2.19–2.10 (m, 1H), 2.09–1.92 (m, 2H), 1.89–1.82 (m, 1H), 1.72–1.60 (m, 5H), 1.57–1.48 (m, 1H), 1.48–1.28 (m, 3H), 1.24–1.18 (m, 1H), 1.18–1.07 (m, 2H), 0.98–0.88 (m, 1H), 0.81–0.71 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.8, 171.0, 158.8, 154.7, 154.0, 149.0, 147.5, 141.9, 135.1, 133.8, 133.7, 130.9, 129.5, 127.5, 120.4, 120.4, 117.3, 112.1, 111.5, 108.2, 107.4, 103.6, 76.0, 66.3, 62.5, 56.4, 56.3, 56.2, 56.1, 56.0, 52.7, 43.8, 42.5, 39.3, 33.3, 31.4, 30.9, 26.8, 26.7, 26.5, 25.7, 21.1. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 728.37931, found 728.37972. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.19 min, purity (220 nm) = 99%. Data for macrocycle <b>31-(</b><i>E</i><b>)</b>: TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.21. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.17 (t, <i>J</i> = 7.9 Hz, 1H), 6.84–6.81 (m, 1H), 6.80–6.76 (m, 2H), 6.71–6.65 (m, 3H), 6.42–6.39 (m, 1H), 6.27 (d, <i>J</i> = 1.8 Hz, 1H), 5.86 (ddd, <i>J</i> = 5.9, 7.4, 15.5 Hz, 1H), 5.70–5.60 (m, 2H), 5.49–5.44 (m, 1H), 4.63 (ddd, <i>J</i> = 1.1, 6.7, 25.9 Hz, 1H), 4.40–4.37 (m, 2H), 3.97–3.90 (m, 1H), 3.86 (s, 3H), 3.84 (s, 3H), 3.82 (s, 3H), 3.53 (s, 3H), 3.30 (d, <i>J</i> = 9.9 Hz, 1H), 2.88–2.79 (m, 1H), 2.64–2.46 (m, 2H), 2.29–2.22 (m, 1H), 2.13–2.02 (m, 2H), 1.94–1.82 (m, 2H), 1.72–1.54 (m, 6H), 1.48–1.38 (m, 1H), 1.37–1.23 (m, 3H), 1.17–1.06 (m, 2H), 0.94–0.84 (m, 1H), 0.78–0.67 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.2, 171.1, 158.7, 154.2, 153.0, 148.9, 147.3, 141.7, 134.0, 133.5, 133.4, 129.7, 129.5, 129.0, 120.6, 120.3, 116.9, 113.6, 111.9, 111.3, 107.4, 104.3, 75.8, 71.4, 69.2, 56.3, 56.2, 56.0, 55.8, 55.4, 52.2, 43.6, 41.7, 38.0, 32.9, 31.3, 30.6, 26.8, 26.6, 26.2, 25.6, 20.8. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 728.37931, found 728.37955. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 8.37 min, purity (220 nm) = 99%.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Macrocycles <b>32-(</b><i>Z</i><b>)</b> and <b>32-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>10</b> (304 mg, 0.40 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (34 mg, 0.04 mmol, 0.10 equiv) in DCM (800 mL, 0.5 mM). After workup procedure a, <b>32-(</b><i>Z</i><b>)</b> (50 mg, 17%, colorless resin) and <b>32-(</b><i>E</i><b>)</b> (62 mg, 21%, colorless resin) were obtained as pure isomers after purification by flash column chromatography (CH/EA = 5:1). Another product fraction was obtained as a mixture of <i>E</i>/<i>Z</i> isomers (118 mg, 40%) and not further purified. Note: The NMR spectra of compounds <b>32-(</b><i>E</i><b>)</b> and <b>32-(</b><i>Z</i><b>)</b> show a peak broadening for the ortho and meta protons and carbon signals of the 3,5-dimethoxyaryl ring. This might result out of a slow interconversion of the C2-symmetrical nuclei of the aryl ring and the attached methoxy groups, because the connected macrocycle linker probably hinders the rotation of the aryl ring. The six equivalent methoxy protons of <b>32-(</b><i>Z</i><b>)</b>therefore show a broad singlet at 3.54 ppm with an integral of only three protons. Data for macrocycle <b>32-(</b><i>Z</i><b>)</b>: TLC (CH/EA = 5:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.37. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.15 (t, <i>J</i> = 7.9 Hz, 1H), 6.86–6.82 (m, 1H), 6.80–6.75 (m, 2H), 6.71–6.63 (m, 3H), 6.39–6.36 (m, 1H), 6.32 (br s, 1H), 5.73–5.67 (m, 2H), 5.64 (dd, <i>J</i> = 6.4, 8.1 Hz, 1H), 5.46–5.43 (m, 1H), 4.45 (dd, <i>J</i> = 5.7, 12.5 Hz, 1H), 4.34 (dd, <i>J</i> = 3.9, 12.6 Hz, 1H), 4.20–4.13 (m, 1H), 4.07–4.00 (m, 1H), 3.94 (d, <i>J</i> = 13.8 Hz, 1H), 3.88–3.80 (m, 6H), 3.54 (br s, 3H), 3.32 (d, <i>J</i> = 9.9 Hz, 1H), 2.72 (td, <i>J</i> = 2.8, 13.4 Hz, 1H), 2.60–2.50 (m, 1H), 2.49–2.40 (m, 3H), 2.31–2.25 (m, 1H), 2.16–2.05 (m, 2H), 1.95–1.84 (m, 2H), 1.71–1.52 (m, 6H), 1.46–1.07 (m, 6H), 0.93–0.83 (m, 1H), 0.79–0.68 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.0, 171.0, 158.3, 154.1, 152.7, 148.9, 147.3, 140.8, 134.5, 133.8, 133.4, 130.4, 129.3, 127.1, 120.9, 120.3, 116.1, 112.1, 111.9, 111.3, 107.6, 104.3, 75.9, 71.4, 63.2, 56.3, 55.9, 55.8, 55.2, 52.1, 43.6, 41.5, 36.6, 32.8, 31.3, 30.6, 28.1, 26.9, 26.6, 26.2, 26.2, 25.6, 21.0. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 742.39496, found 742.39492. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 10.13 min, purity (220 nm) = 99%. Data for macrocycle <b>32-(</b><i>E</i><b>)</b>: TLC (CH/EA = 5:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.32. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.15 (t, <i>J</i> = 7.9 Hz, 1H), 6.83–6.75 (m, 3H), 6.66–6.61 (m, 2H), 6.57–6.23 (m, 3H), 5.70 (dt, <i>J</i> = 6.8, 15.6 Hz, 1H), 5.56–5.44 (m, 3H), 4.50–4.44 (m, 1H), 4.35–4.28 (m, 2H), 4.09–4.03 (m, 1H), 4.02–3.96 (m, 1H), 3.85–3.84 (m, 6H), 3.79–3.53 (m, 6H), 3.35 (d, <i>J</i> = 9.9 Hz, 1H), 2.99 (td, <i>J</i> = 2.7, 13.3 Hz, 1H), 2.56–2.48 (m, 1H), 2.47–2.37 (m, 2H), 2.34–2.22 (m, 2H), 2.12–2.03 (m, 2H), 1.92–1.82 (m, 2H), 1.72–1.56 (m, 6H), 1.49–1.38 (m, 1H), 1.37–1.21 (m, 3H), 1.17–1.06 (m, 2H), 0.94–0.83 (m, 1H), 0.81–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.4, 170.9, 158.5, 149.0, 147.5, 141.4, 136.1, 133.5, 133.3, 132.9, 129.5, 126.1, 120.4, 119.7, 117.3, 112.5, 112.0, 111.4, 76.3, 72.6, 68.7, 56.2, 56.1, 56.0, 55.0, 52.1, 43.6, 41.4, 37.3, 33.8, 32.9, 31.4, 30.7, 27.2, 26.7, 26.3, 26.3, 25.8, 21.1. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 742.39496, found 742.39481. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.40 min, purity (220 nm) = 99%.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Macrocycles <b>33-(</b><i>Z</i><b>)</b> and <b>33-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>11</b> (170 mg, 0.21 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (18 mg, 0.02 mmol, 0.10 equiv) in DCM (450 mL, 0.5 mM). After workup procedure a, <b>33-(</b><i>Z</i><b>)</b> (11 mg, 7%, colorless resins) and <b>33-(</b><i>E</i><b>)</b> (91 mg, 56%, colorless resin) were obtained after purification by flash column chromatography (CH/EA = 3:1). Data for macrocycle <b>33-(</b><i>Z</i><b>)</b>: TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.17 (t, <i>J</i> = 7.9 Hz, 1H), 6.87–6.84 (m, 1H), 6.83–6.79 (m, 1H), 6.79–6.75 (m, 1H), 6.66–6.63 (m, 2H), 6.53–6.50 (m, 1H), 6.37 (s, 2H), 5.91 (ddd, <i>J</i> = 5.3, 6.8, 11.9 Hz, 1H), 5.87–5.81 (m, 1H), 5.49 (dd, <i>J</i> = 5.6, 8.7 Hz, 1H), 5.45 (d, <i>J</i> = 5.1 Hz, 1H), 4.63 (dd, <i>J</i> = 6.9, 11.9 Hz, 1H), 4.54 (ddd, <i>J</i> = 6.0, 12.2, 19.4 Hz, 2H), 4.23 (dd, <i>J</i> = 5.6, 12.7 Hz, 1H), 4.16–4.05 (m, 2H), 3.93 (d, <i>J</i> = 13.6 Hz, 1H), 3.87–3.84 (m, 6H), 3.80–3.68 (m, 2H), 3.58 (s, 6H), 3.34 (d, <i>J</i> = 10.1 Hz, 1H), 2.75 (td, <i>J</i> = 3.0, 13.3 Hz, 1H), 2.58–2.51 (m, 1H), 2.45–2.38 (m, 1H), 2.35–2.29 (m, 1H), 2.14–2.06 (m, 1H), 2.04–1.95 (m, 1H), 1.92–1.85 (m, 2H), 1.71–1.58 (m, 6H), 1.46–1.08 (m, 6H), 0.91–0.82 (m, 1H), 0.79–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.5, 170.8, 158.7, 153.2, 149.0, 147.5, 141.6, 135.4, 133.6, 130.8, 129.7, 127.8, 120.5, 120.1, 116.0, 112.3, 112.0, 111.5, 105.8, 76.0, 73.4, 69.1, 66.8, 64.3, 56.1, 56.1, 56.0, 55.1, 52.0, 43.6, 40.9, 36.3, 32.9, 31.5, 30.7, 27.1, 26.8, 26.4, 26.3, 25.7, 21.2. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 772.4055, found 772.4050. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.64 min, purity (220 nm) = 99%. Data for macrocycle <b>33-(</b><i>E</i><b>)</b>: TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.20. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.58:1, <b>A</b>/<b>B</b>): δ 7.30–7.27 (m, 0.30H, <b>A</b>), 7.16–7.11 (m, 0.99H, <b>A</b>+<b>B</b>), 6.90–6.87 (m, 0.37H, <b>A</b>), 6.81–6.67 (m, 3.52H, <b>A</b>+<b>B</b>), 6.65–6.61 (m, 1.26H, <b>A</b>+<b>B</b>), 6.52–6.45 (m, 2.58H, <b>A</b>+<b>B</b>), 5.83 (dt, <i>J</i> = 5.6, 17.0 Hz, 1.00H, <b>A</b>+<b>B</b>), 5.75 (dt, <i>J</i> = 5.1, 16.0 Hz, 0.59H, <b>B</b>), 5.66–5.63 (m, 0.65H, <b>B</b>), 5.51–5.46 (m, 0.63H, <b>B</b>), 5.41 (dd, <i>J</i> = 5.5, 8.5 Hz, 0.62H, <b>B</b>), 4.80–4.76 (m, 0.41H, <b>A</b>), 4.64–4.59 (m, 0.62H, <b>B</b>), 4.55–4.50 (m, 0.61H, <b>B</b>), 4.45–4.38 (m, 0.74H, <b>A</b>), 4.28–4.24 (m, 0.72H, <b>A</b>), 4.16 (dd, <i>J</i> = 4.8, 10.1 Hz, 0.41H, <b>A</b>), 4.12–4.00 (m, 2.61H, <b>A</b>+<b>B</b>), 3.98–3.83 (m, 10.46H, <b>A</b>+<b>B</b>), 3.67–3.52 (m, 5.57H, <b>A</b>+<b>B</b>), 3.39 (d, <i>J</i> = 9.9 Hz, 0.61H, <b>B</b>), 3.01–2.93 (m, 0.99H, <b>A</b>+<b>B</b>), 2.71 (ddd, <i>J</i> = 5.4, 9.1, 14.3 Hz, 0.36H, <b>A</b>), 2.61 (ddd, <i>J</i> = 7.1, 8.8, 14.1 Hz, 0.37H, <b>A</b>), 2.51–2.38 (m, 1.30H, <b>B</b>), 2.35 (d, <i>J</i> = 10.1 Hz, 0.37H, <b>A</b>), 2.29–2.23 (m, 1.26H, <b>A</b>+<b>B</b>), 2.12–2.05 (m, 0.93H, <b>A</b>+<b>B</b>), 1.96–1.85 (m, 1.65H, <b>A</b>+<b>B</b>), 1.79–1.62 (m, 6.10H, <b>A</b>+<b>B</b>), 1.49–0.77 (m, 8.30H, <b>A</b>+<b>B</b>), 0.66–0.57 (m, 0.34H, <b>A</b>), −0.14 (qd, <i>J</i> = 3.7, 12.7 Hz, 0.34H, <b>A</b>), −0.22 – −0.31 (m, 0.34H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 2:3, <b>A</b>/<b>B</b>): δ 174.21 (<b>A</b>), 172.57 (<b>B</b>), 171.69 (<b>A</b>), 170.80 (<b>B</b>), 159.55 (<b>A</b>), 158.08 (<b>B</b>), 153.20 (<b>A</b>), 153.01 (<b>B</b>), 149.13 (<b>A</b>), 149.01 (<b>B</b>), 147.65 (<b>A</b>), 147.49 (<b>B</b>), 143.64 (<b>A</b>), 142.06 (<b>B</b>), 136.64 (<b>A</b>), 135.80 (<b>B</b>), 133.90 (<b>A</b>), 133.49 (<b>B</b>), 133.36 (<b>A</b>), 133.20 (<b>B</b>), 132.16 (<b>A</b>), 131.80 (<b>B</b>), 129.91 (<b>A</b>), 129.60 (<b>B</b>), 126.85 (<b>B</b>), 125.04 (<b>A</b>), 120.43 (<b>B</b>), 120.39 (<b>A</b>), 119.95 (<b>A</b>), 119.34 (<b>B</b>), 115.96 (<b>B</b>), 113.19 (<b>B</b>), 112.02 (<b>B</b>), 111.89 (<b>A</b>), 111.54 (<b>A</b>), 111.46 (<b>B</b>), 111.01 (<b>A</b>), 105.75 (<b>B</b>), 104.73 (<b>A</b>), 76.08 (<b>B</b>), 75.73 (<b>A</b>), 72.82 (<b>A</b>), 72.40 (<b>B</b>), 71.83 (<b>A</b>), 71.13 (<b>B</b>), 71.09 (<b>A</b>), 70.03 (<b>B</b>), 68.23 (<b>A</b>), 67.87 (<b>B</b>), 56.32 (<b>A</b>), 56.08 (<b>B</b>), 56.03 (<b>A</b>), 55.99 (<b>B</b>), 55.95 (<b>B</b>), 54.91 (<b>B</b>), 51.93 (<b>A</b>), 43.65 (<b>B</b>), 42.31 (<b>A</b>), 41.44 (<b>B</b>), 39.47 (<b>A</b>), 38.69 (<b>A</b>), 37.54 (<b>B</b>), 32.83 (<b>B</b>), 32.12 (<b>A</b>), 31.71 (<b>A</b>), 31.25 (<b>B</b>), 30.75 (<b>B</b>), 29.57 (<b>A</b>), 27.14 (<b>B</b>), 26.83 (<b>A</b>), 26.72 (<b>B</b>), 26.51 (<b>A</b>), 26.30 (<b>B</b>), 26.26 (<b>B</b>), 26.14 (<b>A</b>), 26.09 (<b>A</b>), 25.79 (<b>B</b>), 23.94 (<b>A</b>), 21.08 (<b>B</b>), 20.47 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 772.40552, found 772.40455. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 8.24 min, purity (220 nm) = 99%.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Macrocycle <b>34-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>12</b> (125 mg, 0.15 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (13 mg, 0.015 mmol, 0.10 equiv) in DCM (310 mL, 0.5 mM). After workup procedure a, <b>34-(</b><i>E</i><b>)</b> (53 mg, 44%) was obtained as a white solid after purification by flash column chromatography (CH/EA = 4:1). The corresponding <i>Z</i>-alkene was only observed in trace amounts during the reaction control via LC–MS and could not be isolated. TLC (CH/EA = 4:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.26. Note: The<sup>1</sup>H signals of the olefin protons of compound<b>34-(</b><i>E</i><b>)</b>overlap each other in different NMR solvents (CDCl<sub>3</sub>, THF-<i>d</i><sub>8</sub>, DMSO-<i>d</i><sub>6</sub>, C<sub>6</sub>D<sub>6</sub>). Finally, the (<i>E</i>)-configuration of the double bond was determined via the coupling constant of the<sup>13</sup>C satellite signals of one olefin proton (see spectra in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>, 512 scans for<sup>1</sup>H spectra in C<sub>6</sub>D<sub>6</sub>). <sup>1</sup>H NMR (500 MHz, C<sub>6</sub>D<sub>6</sub>): δ 7.08 (t, <i>J</i> = 7.9 Hz, 1H), 6.74–6.70 (m, 5H), 6.64–6.59 (m, 2H), 6.52–6.47 (m, 1H), 5.91–5.87 (m, 1H), 5.84 (dd, <i>J</i> = 5.3, 8.1 Hz, 1H), 5.56–5.48 (m, 2H), 4.33–4.24 (m, 2H), 4.13–4.07 (m, 1H), 3.99–3.94 (m, 1H), 3.91–3.86 (m, 1H), 3.70–3.64 (m, 2H), 3.55 (s, 3H), 3.46–3.43 (m, 9H), 3.35–3.28 (m, 3H), 2.97–2.90 (m, 1H), 2.60–2.51 (m, 3H), 2.33–2.28 (m, 1H), 2.25–2.21 (m, 1H), 2.08–1.99 (m, 3H), 1.91–1.85 (m, 1H), 1.77–1.73 (m, 1H), 1.67–1.62 (m, 2H), 1.56–1.52 (m, 1H), 1.44–1.28 (m, 4H), 1.20–1.14 (m, 4H), 0.97–0.91 (m, 1H), 0.87–0.78 (m, 1H). <sup>13</sup>C NMR (126 MHz, C<sub>6</sub>D<sub>6</sub>): δ 172.2, 170.7, 158.2, 154.3, 150.3, 148.7, 143.1, 138.0, 134.1, 133.7, 131.6, 129.8, 127.6, 120.9, 118.5, 115.1, 114.7, 113.3, 112.8, 107.0, 75.8, 70.8, 70.1, 67.5, 66.2, 56.1, 55.8, 55.7, 55.3, 52.2, 43.9, 41.9, 39.3, 33.1, 31.6, 31.3, 30.9, 27.1, 26.8, 26.7, 26.7, 26.0, 21.2. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>60</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 786.42117, found 786.42139. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 8.98 min, purity (220 nm) = 99%.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Macrocycle <b>35-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>27</b> (261 mg, 0.35 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second Generation catalyst (30 mg, 0.04 mmol, 0.10 equiv) in DCM (700 mL, 0.5 mM). After workup procedure b), <b>35-(</b><i>E</i><b>)</b> (57 mg, 23%, white solid) was obtained as pure product isomer after purification by flash column chromatography (CH/EA = 5:1). In addition to compound <b>35-(</b><i>E</i><b>)</b>, another product (28 mg, 11%, white solid) could be isolated from the RCM which, according to mass and NMR analysis, must be an isomer, most likely an epimer of <b>35-(</b><i>E</i><b>)</b>, due to its remarkably reduced binding to FKBP51, 12, and 12.6. Unfortunately, a proper structure elucidation was not possible. TLC (DCM/MeOH = 20:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.32. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.16:1, <b>A</b>/<b>B</b>): δ 7.24 (t, <i>J</i> = 7.8 Hz, 0.15H, <b>A</b>), 7.09 (t, <i>J</i> = 7.8 Hz, 0.85H, <b>B</b>), 7.04–7.01 (m, 0.14H, <b>A</b>), 6.92–6.87 (m, 0.30H, <b>A</b>), 6.81–6.74 (m, 2.59H, <b>A</b>+<b>B</b>), 6.70–6.51 (m, 4.36H, <b>A</b>+<b>B</b>), 6.47–6.43 (m, 0.84H, <b>B</b>), 6.16–6.13 (m, 0.14H, <b>A</b>), 6.08–5.80 (m, 2.39H, <b>A</b>+<b>B</b>), 5.56–5.52 (m, 0.85H, <b>B</b>), 5.33 (dd, <i>J</i> = 4.2, 9.8 Hz, 0.88H, <b>B</b>), 4.88–4.80 (m, 0.85H, <b>B</b>), 4.78–4.65 (m, 2.35H, <b>A</b>+<b>B</b>), 4.58–4.53 (m, 0.16H, <b>A</b>), 4.49–4.35 (m, 1.05H, <b>A</b>+<b>B</b>), 4.27 (d, <i>J</i> = 13.2 Hz, 0.85H, <b>B</b>), 3.87–3.82 (m, 7.03H, <b>A</b>+<b>B</b>), 3.79 (s, 0.61H, <b>A</b>), 3.74 (s, 5.06H, <b>A</b>+<b>B</b>), 3.50 (d, <i>J</i> = 10.2 Hz, 0.79H, <b>B</b>), 3.20 (td, <i>J</i> = 2.6, 13.1 Hz, 0.85H, <b>B</b>), 2.93 (td, <i>J</i> = 3.5, 12.8 Hz, 0.14H, <b>A</b>), 2.63–2.50 (m, 0.35H, <b>A</b>), 2.42–2.17 (m, 3.57H, <b>A</b>+<b>B</b>), 2.15–2.09 (m, 0.30H, <b>A</b>), 1.88 (d, <i>J</i> = 12.4 Hz, 0.97H, <b>A</b>+<b>B</b>), 1.76–1.46 (m, 8.06H, <b>A</b>+<b>B</b>), 1.43–1.31 (m, 3.08H, <b>A</b>+<b>B</b>), 1.28–1.11 (m, 3.00H, <b>A</b>+<b>B</b>), 0.98–0.83 (m, 1.20H, <b>A</b>+<b>B</b>), 0.80–0.68 (m, 0.84H, <b>B</b>), 0.41–0.31 (m, 0.15H, <b>A</b>), 0.10 – −0.01 (m, 0.14H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:7, <b>A</b>/<b>B</b>): δ 172.92 (<b>A</b>), 172.70 (<b>B</b>), 170.94 (<b>A</b>), 170.52 (<b>B</b>), 159.55 (<b>A</b>), 159.48 (<b>B</b>), 154.20 (<b>A</b>), 153.78 (<b>B</b>), 152.24 (<b>A</b>), 150.79 (<b>B</b>), 149.07 (<b>A</b>), 148.94 (<b>B</b>), 147.60 (<b>A</b>), 147.36 (<b>B</b>), 142.95 (<b>B</b>), 141.39 (<b>A</b>), 137.59 (<b>A</b>), 137.27 (<b>B</b>), 134.28 (<b>A</b>), 133.57 (<b>B</b>), 133.34 (<b>A</b>), 133.24 (<b>B</b>), 129.86 (<b>A</b>), 129.62 (<b>B</b>), 129.42 (<b>B</b>), 129.32 (<b>A</b>), 127.50 (<b>A</b>), 127.05 (<b>B</b>), 121.05 (<b>A</b>), 120.65 (<b>B</b>), 120.27 (<b>A</b>), 119.51 (<b>B</b>), 118.17 (<b>A</b>), 117.00 (<b>B</b>), 111.99 (<b>B</b>), 111.88 (<b>A</b>), 111.51 (<b>A</b>), 111.45 (<b>B</b>), 110.16 (<b>B</b>), 110.10 (<b>A</b>), 109.26 (<b>A</b>), 108.37 (<b>B</b>), 104.79 (<b>B</b>), 103.77 (<b>A</b>), 75.84 (<b>A</b>), 74.91 (<b>B</b>), 70.67 (<b>A</b>), 68.86 (<b>B</b>), 68.14 (<b>B</b>), 68.08 (<b>A</b>), 61.12 (<b>A</b>), 60.75 (<b>B</b>), 56.63 (<b>A</b>), 56.21 (<b>A</b>), 56.05 (<b>B</b>), 55.98 (<b>B</b>), 55.88 (<b>B</b>), 55.57 (<b>A</b>), 54.25 (<b>B</b>), 52.04 (<b>B</b>), 44.29 (<b>B</b>), 41.44 (<b>B</b>), 39.29 (<b>A</b>), 38.24 (<b>B</b>), 38.03 (<b>A</b>), 32.36 (<b>B</b>), 31.45 (<b>A</b>), 30.73 (<b>B</b>), 29.80 (<b>A</b>), 26.96 (<b>A</b>), 26.65 (<b>B</b>), 26.52 (<b>B</b>), 26.25 (<b>A</b>), 26.18 (<b>B</b>), 26.11 (<b>B</b>), 25.97 (<b>B</b>), 24.11 (<b>A</b>), 21.14 (<b>B</b>), 20.59 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 728.37931, found 728.37963. HPLC (70–80% sSolvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 7.15 min, purity (220 nm) = 99%.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Macrocycles <b>36-(</b><i>Z</i><b>)</b> and <b>36-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>28</b> (224 mg, 0.29 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (25 mg, 0.03 mmol, 0.10 equiv) and 1,4-benzoquinone (4 mg, 0.03 mmol, 0.10 equiv) in DCM (1 mM, 300 mL). After workup procedure b, <b>36-(</b><i>Z</i><b>)</b> (28 mg, 13%, white solid) and <b>36-(</b><i>E</i><b>)</b> (115 mg, 53%, white solid) were obtained as product isomers after purification by flash column chromatography (CH/EA = 4:1) and semipreparative HPLC (80–100% solvent B). Data for macrocycle <b>36-(</b><i>Z</i><b>)</b>: TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.39. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.16:1, <b>A</b>/<b>B</b>): δ 7.29 (t, <i>J</i> = 7.9 Hz, 0.15H, <b>A</b>), 7.15 (t, <i>J</i> = 8.3 Hz, 0.85H, <b>B</b>), 7.08–7.06 (m, 0.14H, <b>A</b>), 6.94–6.88 (m, 0.32H, <b>A</b>), 6.81–6.74 (m, 2.73H, <b>A</b>+<b>B</b>), 6.69–6.61 (m, 2.75H, <b>A</b>+<b>B</b>), 6.56–6.51 (m, 0.22H, <b>A</b>), 6.43–6.40 (m, 0.80H, <b>B</b>), 6.31–6.28 (m, 0.81H, <b>B</b>), 6.21–6.19 (m, 0.13H, <b>A</b>), 6.02–5.95 (m, 0.15H, <b>A</b>), 5.93–5.78 (m, 1.87H, <b>A</b>+<b>B</b>), 5.55–5.46 (m, 1.77H, <b>B</b>), 4.62–4.42 (m, 0.65H, <b>A</b>), 4.34–4.23 (m, 2.52H, <b>A</b>+<b>B</b>), 4.12–4.00 (m, 1.79H, <b>B</b>), 3.88–3.82 (m, 6.69H, <b>A</b>+<b>B</b>), 3.80–3.76 (m, 3.00H, <b>A</b>+<b>B</b>), 3.64 (s, 2.42H, <b>B</b>), 3.37 (d, <i>J</i> = 10.0 Hz, 0.84H, <b>B</b>), 2.81–2.71 (m, 1.73H, <b>A</b>+<b>B</b>), 2.66–2.54 (m, 0.38H, <b>A</b>), 2.52–2.34 (m, 3.01H, <b>A</b>+<b>B</b>), 2.31–2.24 (m, 1.02H, <b>A</b>+<b>B</b>), 2.20–2.06 (m, 1.21H, <b>A</b>+<b>B</b>), 1.92–1.84 (m, 1.64H, <b>A</b>+<b>B</b>), 1.80–1.53 (m, 7.03H, <b>A</b>+<b>B</b>), 1.50–1.24 (m, 3.83H, <b>A</b>+<b>B</b>), 1.19–1.07 (m, 2.59H, <b>A</b>+<b>B</b>), 0.92–0.82 (m, 1.05H, <b>A</b>+<b>B</b>), 0.78–0.69 (m, 0.82H, <b>B</b>), 0.30–0.21 (m, 0.13H, <b>A</b>), 0.14–0.05 (m, 0.18H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:4, <b>A</b>/<b>B</b>): δ 173.61 (<b>A</b>), 172.24 (<b>B</b>), 171.00 (<b>A</b>), 170.59 (<b>B</b>), 159.41 (<b>A</b>), 158.87 (<b>B</b>), 154.38 (<b>A</b>), 153.91 (<b>B</b>), 151.13 (<b>B</b>), 150.74 (<b>A</b>), 149.10 (<b>A</b>), 148.97 (<b>B</b>), 147.60 (<b>A</b>), 147.43 (<b>B</b>), 142.76 (<b>A</b>), 141.89 (<b>B</b>), 137.23 (<b>B</b>), 133.59 (<b>B</b>), 133.40 (<b>B</b>), 131.91 (<b>B</b>), 129.86 (<b>B</b>), 129.79 (<b>A</b>), 129.49 (<b>B</b>), 128.10 (<b>B</b>), 127.12 (<b>A</b>), 120.99 (<b>A</b>), 120.51 (<b>B</b>), 120.48 (<b>B</b>), 120.33 (<b>A</b>), 116.22 (<b>B</b>), 113.71 (<b>A</b>), 111.97 (<b>B</b>), 111.82 (<b>A</b>), 111.51 (<b>A</b>), 111.42 (<b>B</b>), 110.32 (<b>B</b>), 109.29 (<b>A</b>), 107.24 (<b>B</b>), 104.84 (<b>B</b>), 103.51 (<b>A</b>), 76.23 (<b>A</b>), 75.88 (<b>B</b>), 67.70 (<b>A</b>), 67.06 (<b>B</b>), 63.94 (<b>B</b>), 63.08 (<b>A</b>), 61.01 (<b>A</b>), 60.83 (<b>B</b>), 56.29 (<b>B</b>), 56.19 (<b>A</b>), 56.07 (<b>B</b>), 56.02 (<b>A</b>), 55.93 (<b>B</b>), 55.26 (<b>B</b>), 52.14 (<b>B</b>), 43.97 (<b>B</b>), 41.53 (<b>B</b>), 39.62 (<b>A</b>), 38.21 (<b>A</b>), 37.62 (<b>B</b>), 32.94 (<b>B</b>), 32.55 (<b>A</b>), 31.61 (<b>A</b>), 31.27 (<b>B</b>), 30.76 (<b>B</b>), 30.30 (<b>A</b>), 28.41 (<b>A</b>), 27.94 (<b>B</b>), 27.12 (<b>A</b>), 26.71 (<b>B</b>), 26.40 (<b>A</b>), 26.31 (<b>B</b>), 26.21 (<b>B</b>), 26.07 (<b>A</b>), 25.85 (<b>B</b>), 24.23 (<b>A</b>), 21.05 (<b>B</b>), 20.88 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 742.39496, found 742.39491. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.58 min, purity (220 nm) = 95%. Data for macrocycle <b>36-(</b><i>E</i><b>)</b>: TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.36. <sup>1</sup>H NMR (500 MHz, THF-<i>d</i><sub>8</sub>, mixture of rotamers 0.18:1, <b>A</b>/<b>B</b>): δ 7.20 (t, <i>J</i> = 7.8 Hz, 0.90H, <b>B</b>), 7.11–7.08 (m, 0.14H, <b>A</b>), 6.99–6.92 (m, 0.81H, <b>B</b>), 6.92–6.83 (m, 0.31H, <b>A</b>), 6.81–6.75 (m, 1.85H, <b>B</b>), 6.73–6.70 (m, 0.99H, <b>A</b>+<b>B</b>), 6.69 (d, 0.82H, <b>B</b>), 6.67–6.60 (m, 1.17H, <b>A</b>+<b>B</b>), 6.46 (d, <i>J</i> = 1.8 Hz, 0.80H, <b>B</b>), 6.37 (d, <i>J</i> = 1.9 Hz, 0.14H, <b>A</b>), 6.33 (d, <i>J</i> = 1.8 Hz, 0.79H, <b>B</b>), 5.97–5.89 (m, 0.28H, <b>A</b>), 5.79 (dt, <i>J</i> = 4.9, 16.2 Hz, 0.15H, <b>A</b>), 5.72–5.61 (m, 1.69H, <b>A</b>+<b>B</b>), 5.56 (dd, <i>J</i> = 4.8, 9.2 Hz, 0.82H, <b>B</b>), 5.29–5.25 (m, 0.80H, <b>B</b>), 4.74 (d, <i>J</i> = 5.8 Hz, 0.14H, <b>A</b>), 4.69–4.53 (m, 2.01H, <b>A</b>+<b>B</b>), 4.24–4.17 (m, 0.14H, <b>A</b>), 3.98–3.92 (m, 0.16H, <b>A</b>), 3.89–3.81 (m, 1.75H, <b>A</b>+<b>B</b>), 3.76–3.67 (m, 13.11H, <b>A</b>+<b>B</b>), 3.43 (d, <i>J</i> = 8.7 Hz, 0.87H, <b>B</b>), 2.84 (d, <i>J</i> = 9.9 Hz, 0.14H, <b>A</b>), 2.74 (td, <i>J</i> = 3.1, 13.3 Hz, 0.97H, <b>B</b>), 2.61–2.54 (m, 0.99H, <b>A</b>+<b>B</b>), 2.50–2.34 (m, 3.56H, <b>A</b>+<b>B</b>), 2.11–2.03 (m, 1.01H, <b>A</b>+<b>B</b>), 1.99–1.89 (m, 1.76H, <b>A</b>+<b>B</b>), 1.85–1.79 (m, 1.00H, <b>A</b>+<b>B</b>), 1.70–1.55 (m, 5.22H, <b>A</b>+<b>B</b>), 1.46–1.37 (m, 0.94H, <b>A</b>+<b>B</b>), 1.33–0.76 (m, 8.59H, <b>A</b>+<b>B</b>), 0.55–0.45 (m, 0.25H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, THF-<i>d</i><sub>8</sub>, mixture of rotamers 1:5, <b>A</b>/<b>B</b>): δ 172.93 (<b>B</b>), 172.50 (<b>A</b>), 171.01 (<b>A</b>), 170.82 (<b>B</b>), 160.49 (<b>A</b>), 158.96 (<b>B</b>), 155.71 (<b>A</b>), 154.13 (<b>B</b>), 153.78 (<b>B</b>), 152.91 (<b>A</b>), 150.90 (<b>B</b>), 149.38 (<b>A</b>), 149.31 (<b>B</b>), 143.39 (<b>B</b>), 143.08 (<b>A</b>), 140.29 (<b>A</b>), 138.59 (<b>B</b>), 135.26 (<b>A</b>), 134.78 (<b>B</b>), 134.63 (<b>A</b>), 134.05 (<b>B</b>), 130.50 (<b>B</b>), 130.46 (<b>B</b>), 130.20 (<b>A</b>), 128.57 (<b>B</b>), 128.09 (<b>A</b>), 121.32 (<b>A</b>), 121.15 (<b>B</b>), 118.90 (<b>B</b>), 118.09 (<b>B</b>), 117.02 (<b>A</b>), 114.42 (<b>B</b>), 113.76 (<b>B</b>), 113.50 (<b>A</b>), 113.35 (<b>B</b>), 112.60 (<b>A</b>), 108.21 (<b>B</b>), 108.16 (<b>B</b>), 104.99 (<b>A</b>), 76.99 (<b>A</b>), 76.77 (<b>B</b>), 70.74 (<b>A</b>), 68.99 (<b>B</b>), 68.89 (<b>A</b>), 68.73 (<b>B</b>), 68.10 (<b>A</b>), 60.66 (<b>A</b>), 60.54 (<b>B</b>), 57.18 (<b>A</b>), 56.52 (<b>B</b>), 56.38 (<b>B</b>), 56.28 (<b>B</b>), 56.09 (<b>B</b>), 53.14 (<b>B</b>), 44.15 (<b>B</b>), 42.35 (<b>A</b>), 42.27 (<b>B</b>), 40.01 (<b>A</b>), 39.64 (<b>B</b>), 39.29 (<b>A</b>), 34.01 (<b>B</b>), 33.80 (<b>B</b>), 33.48 (<b>A</b>), 32.60 (<b>B</b>), 32.43 (<b>A</b>), 31.71 (<b>B</b>), 31.15 (<b>A</b>), 30.80 (<b>A</b>), 28.19 (<b>A</b>), 28.03 (<b>B</b>), 27.79 (<b>B</b>), 27.61 (<b>B</b>), 27.41 (<b>A</b>), 27.21 (<b>A</b>), 26.49 (<b>B</b>), 25.98 (<b>A</b>), 22.52 (<b>A</b>), 22.35 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 742.39496, found 742.39517. HPLC (70–100% solvent B, 0.8 mL/min, 10.5 min): <i>t</i><sub>R</sub> = 6.23 min, purity (220 nm) = 99%.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Macrocycles <b>37-(</b><i>E</i><b>)</b> and <b>37-(</b><i>Z</i><b>)</b></h3><div class="NLM_p last">The substrate <b>29</b> (201 mg, 0.25 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second generation catalyst (21 mg, 0.03 mmol, 0.10 equiv) and 1,4-benzoquinone (3 mg, 0.03 mmol, 0.10 equiv) in DCM (1 mM, 250 mL). After workup procedure b, <b>37-(</b><i>E</i><b>)</b> (58 mg, 30%, white solid) and <b>37-(</b><i>Z</i><b>)</b> (7 mg, 4%, white solid) were obtained as pure product isomers after purification by flash column chromatography (CH/EA = 4:1) and semipreparative HPLC (80–100% solvent B). Another product fraction was obtained as a mixture of <i>E</i>/<i>Z</i> isomers (75 mg, 39%) and not further purified. Data for macrocycle <b>37-(</b><i>E</i><b>)</b>: TLC (CH/EA = 1:1 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.52. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.34:1, <b>A</b>/<b>B</b>): δ 7.28–7.23 (m, 0.40H, <b>A</b>), 7.19–7.15 (m, 0.78H, <b>B</b>), 7.05–7.02 (m, 0.23H, <b>A</b>), 6.91–6.87 (m, 0.28H, <b>A</b>), 6.86–6.72 (m, 2.99H, <b>A</b>+<b>B</b>), 6.71–6.61 (m, 2.06H, <b>A</b>+<b>B</b>), 6.57–6.55 (m, 0.71H, <b>B</b>), 6.43–6.37 (m, 1.34H, <b>A</b>+<b>B</b>), 6.33–6.30 (m, 0.23H, <b>A</b>), 6.00 (dt, <i>J</i> = 4.9, 17.3 Hz, 0.32H, <b>A</b>), 5.93–5.90 (m, 0.17H, <b>A</b>), 5.87 (dt, <i>J</i> = 5.2, 16.5 Hz, 1.15H, <b>B</b>), 5.83–5.80 (m, 0.40H, <b>A</b>), 5.58 (dd, <i>J</i> = 5.3, 8.5 Hz, 0.74H, <b>B</b>), 5.49 (d, <i>J</i> = 5.5 Hz, 0.78H, <b>B</b>), 4.65–4.54 (m, 1.05H, <b>A</b>+<b>B</b>), 4.53–4.42 (m, 1.57H, <b>A</b>+<b>B</b>), 4.31–4.18 (m, 0.79H, <b>A</b>), 4.15–4.03 (m, 2.82H, <b>A</b>+<b>B</b>), 4.00–3.88 (m, 2.20H, <b>A</b>+<b>B</b>), 3.86–3.79 (m, 11.69H, <b>A</b>+<b>B</b>), 3.74–3.61 (m, 4.92H, <b>A</b>+<b>B</b>), 3.33 (d, <i>J</i> = 9.5 Hz, 0.72H, <b>B</b>), 2.85–2.76 (m, 0.98H, <b>A</b>+<b>B</b>), 2.69 (d, <i>J</i> = 9.7 Hz, 0.24H, <b>A</b>), 2.65–2.48 (m, 1.31H, <b>A</b>+<b>B</b>), 2.45–2.38 (m, 0.77H, <b>B</b>), 2.32–2.18 (m, 1.51H, <b>A</b>+<b>B</b>), 2.10–1.96 (m, 2.00H, <b>A</b>+<b>B</b>), 1.90–1.83 (m, 1.51H, <b>A</b>+<b>B</b>), 1.70–1.56 (m, 5.38H, <b>A</b>+<b>B</b>), 1.51–1.23 (m, 5.20H, <b>A</b>+<b>B</b>), 1.17–0.99 (m, 2.30H, <b>A</b>+<b>B</b>), 0.94–0.85 (m, 0.96H, <b>A</b>+<b>B</b>), 0.77–0.69 (m, 0.82H, <b>B</b>), 0.48–0.38 (m, 0.23H, <b>A</b>), 0.26–0.17 (m, 0.25H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:4, <b>A</b>/<b>B</b>): δ 173.10 (<b>A</b>), 172.31 (<b>B</b>), 170.64 (<b>B</b>), 159.12 (<b>A</b>), 158.30 (<b>B</b>), 154.25 (<b>A</b>), 153.31 (<b>B</b>), 152.28 (<b>B</b>), 151.25 (<b>A</b>), 149.04 (<b>A</b>), 148.94 (<b>B</b>), 147.54 (<b>A</b>), 147.38 (<b>B</b>), 142.15 (<b>B</b>), 142.08 (<b>A</b>), 139.25 (<b>A</b>), 137.68 (<b>B</b>), 134.20 (<b>A</b>), 133.57 (<b>B</b>), 133.45 (<b>A</b>), 133.24 (<b>B</b>), 131.06 (<b>B</b>), 130.32 (<b>A</b>), 129.61 (<b>A</b>), 129.50 (<b>B</b>), 127.75 (<b>B</b>), 127.09 (<b>A</b>), 120.68 (<b>A</b>), 120.37 (<b>B</b>), 120.26 (<b>A</b>), 119.01 (<b>B</b>), 114.37 (<b>B</b>), 113.99 (<b>A</b>), 113.77 (<b>A</b>), 112.95 (<b>B</b>), 112.47 (<b>A</b>), 111.88 (<b>B</b>), 111.78 (<b>A</b>), 111.47 (<b>A</b>), 111.40 (<b>B</b>), 108.31 (<b>B</b>), 105.97 (<b>B</b>), 104.88 (<b>A</b>), 76.22 (<b>A</b>), 75.56 (<b>B</b>), 71.17 (<b>A</b>), 70.94 (<b>A</b>), 70.83 (<b>B</b>), 69.09 (<b>A</b>), 69.05 (<b>B</b>), 68.98 (<b>B</b>), 67.50 (<b>B</b>), 67.28 (<b>A</b>), 60.93 (<b>B</b>), 60.86 (<b>A</b>), 56.36 (<b>B</b>), 56.15 (<b>A</b>), 56.11 (<b>A</b>), 56.02 (<b>B</b>), 55.96 (<b>A</b>), 55.93 (<b>B</b>), 55.23 (<b>B</b>), 52.06 (<b>B</b>), 43.61 (<b>B</b>), 41.46 (<b>B</b>), 39.35 (<b>A</b>), 38.51 (<b>B</b>), 37.92 (<b>A</b>), 33.03 (<b>B</b>), 32.39 (<b>A</b>), 31.63 (<b>A</b>), 31.28 (<b>B</b>), 30.81 (<b>B</b>), 30.28 (<b>A</b>), 27.40 (<b>A</b>), 27.02 (<b>B</b>), 26.68 (<b>B</b>), 26.41 (<b>A</b>), 26.35 (<b>B</b>), 26.29 (<b>A</b>), 26.03 (<b>A</b>), 25.66 (<b>B</b>), 24.41 (<b>A</b>), 21.12 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 772.40552, found 772.40598. HPLC (70–100% solvent B, 0.80 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.42 min, purity (220 nm) = 96%. Data for macrocycle <b>37-(</b><i>Z</i><b>)</b>: TLC (CH/EA = 1:1 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.52. Note: At room temperature, the NMR spectra of compound <b>37-(</b><i>Z</i><b>)</b> show the presence of multiple rotamers (two major and two minor rotamers). Due to this, the proton and carbon NMR spectra become very complex and signals cannot be assigned to specific rotamers. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers): δ 7.28–7.24 (m, 0.98H), 7.16 (dt, <i>J</i> = 7.9, 10.4 Hz, 0.72H), 7.01 (t, <i>J</i> = 2.0 Hz, 0.15H), 6.97 (t, <i>J</i> = 2.0 Hz, 0.16H), 6.92 (dd, <i>J</i> = 2.1, 8.0 Hz, 0.29H), 6.89–6.73 (m, 3.01H), 6.72–6.66 (m, 1.03H), 6.65–6.55 (m, 2.97H), 6.50 (d, <i>J</i> = 1.9 Hz, 0.36H), 6.35 (d, <i>J</i> = 1.9 Hz, 0.30H), 6.23 (d, <i>J</i> = 2.0 Hz, 0.14H), 6.01–5.74 (m, 2.45H), 5.52 (dd, <i>J</i> = 5.6, 8.7 Hz, 0.36H), 5.45–5.36 (m, 1.00H), 4.79–4.57 (m, 2.38H), 4.52 (d, <i>J</i> = 13.7 Hz, 0.17H), 4.30–4.20 (m, 1.49H), 4.19–3.97 (m, 2.86H), 3.89–3.75 (m, 14.09H), 3.72–3.69 (m, 0.35H), 3.65–3.60 (m, 0.30H), 3.57–3.51 (m, 0.29H), 3.39 (dd, <i>J</i> = 5.7, 9.4 Hz, 0.60H), 3.25 (d, <i>J</i> = 9.6 Hz, 0.17H), 2.93–2.83 (m, 0.55H), 2.77 (td, <i>J</i> = 3.1, 13.3 Hz, 0.32H), 2.68–2.51 (m, 1.32H), 2.48–2.40 (m, 0.65H), 2.38–2.19 (m, 1.39H), 2.16–2.02 (m, 1.70H), 1.98–1.75 (m, 2.19H), 1.72–1.55 (m, 6.00H), 1.47–1.27 (m, 4.00H), 1.19–1.00 (m, 2.99H), 0.92–0.73 (m, 2.06H), 0.40–0.30 (m, 0.17H), 0.27–0.18 (m, 0.15H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers): δ 172.82, 172.54, 171.64, 170.86, 170.66, 170.33, 170.28, 159.47, 158.88, 158.85, 158.25, 154.47, 154.27, 153.46, 152.87, 152.39, 152.13, 151.94, 151.28, 149.12, 149.07, 148.99, 147.64, 147.58, 147.49, 147.46, 142.76, 141.95, 141.60, 138.28, 134.33, 134.12, 133.73, 133.66, 133.54, 133.47, 133.24, 130.30, 129.80, 129.70, 129.66, 129.54, 129.50, 129.10, 128.83, 128.76, 126.04, 121.11, 120.53, 120.42, 120.32, 119.77, 117.33, 116.71, 116.43, 116.37, 115.37, 113.69, 112.45, 112.03, 111.97, 111.91, 111.86, 111.80, 111.56, 111.49, 111.40, 109.81, 109.53, 108.82, 107.44, 106.54, 104.66, 104.46, 76.71, 76.41, 76.30, 75.63, 71.34, 70.69, 70.66, 70.26, 70.14, 69.83, 69.34, 68.60, 68.52, 68.16, 67.70, 67.39, 67.24, 67.11, 64.65, 64.43, 61.09, 60.95, 60.92, 56.41, 56.34, 56.26, 56.09, 56.07, 55.96, 55.69, 55.23, 54.81, 52.39, 52.16, 43.85, 41.26, 41.23, 40.05, 39.34, 38.37, 38.26, 38.09, 36.87, 33.55, 32.98, 32.78, 32.53, 31.81, 31.70, 31.58, 31.40, 31.21, 30.94, 30.81, 30.74, 30.33, 29.84, 27.46, 27.00, 26.79, 26.72, 26.59, 26.55, 26.46, 26.42, 26.35, 26.31, 26.06, 25.75, 25.49, 24.81, 24.34, 21.22, 20.99, 20.87, 20.79. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 772.40552, found 772.40547. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.52 min, purity (220 nm) = 99%.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Macrocycle <b>38-(</b><i>E</i><b>)</b></h3><div class="NLM_p last">The substrate <b>30</b> (134 mg, 0.17 mmol, 1.00 equiv) was applied to general procedure B with Grubbs second gGeneration catalyst (15 mg, 0.017 mmol, 0.10 equiv) and 1,4-benzoquinone (2 mg, 0.017 mmol, 0.10 equiv) in DCM (1 mM, 175 mL). After workup procedure b, <b>38-(</b><i>E</i><b>)</b> (43 mg, 33%, white solid) was obtained as pure product isomer after purification by semipreparative HPLC (85–100% solvent B). Another product fraction was obtained as a mixture of <b>38-(</b><i>E</i><b>)</b> and an unidentified isomer (probably the <i>Z</i>-alkene due to same <i>m</i>/<i>z</i> on HPLC–MS; 75 mg, 58%, ratio 1:4, unidentified isomer <i>E</i>-alkene). Unfortunately, we were not able to obtain the unidentified isomer as a pure product by column chromatography or semipreparative HPLC. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.32. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.36:1, <b>A</b>/<b>B</b>): δ 7.29–7.26 (m, 0.22H, <b>A</b>), 7.14 (t, <i>J</i> = 7.9 Hz, 0.86H, <b>B</b>), 7.03–7.01 (m, 0.23H, <b>A</b>), 6.89–6.87 (m, 0.24H, <b>A</b>), 6.87–6.84 (m, 0.24H, <b>A</b>), 6.80–6.72 (m, 2.87H, <b>A</b>+<b>B</b>), 6.69–6.64 (m, 2.16H, <b>A</b>+<b>B</b>), 6.63–6.59 (m, 1.23H, <b>A</b>+<b>B</b>), 6.47–6.43 (m, 1.48H, <b>A</b>+<b>B</b>), 6.28 (d, <i>J</i> = 1.9 Hz, 0.21H, <b>A</b>), 5.95–5.89 (m, 0.39H, <b>A</b>), 5.87–5.80 (m, 1.12H, <b>A</b>+<b>B</b>), 5.77 (dt, <i>J</i> = 5.1, 16.0 Hz, 0.92H, <b>B</b>), 5.58 (dd, <i>J</i> = 5.6, 8.3 Hz, 0.75H, <b>B</b>), 5.51 (d, <i>J</i> = 5.6 Hz, 0.73H, <b>B</b>), 4.64–4.56 (m, 0.95H, <b>A</b>+<b>B</b>), 4.40–4.36 (m, 1.43H, <b>A</b>+<b>B</b>), 4.14–4.04 (m, 2.40H, <b>A</b>+<b>B</b>), 4.02–3.97 (m, 1.07H, <b>A</b>+<b>B</b>), 3.96–3.90 (m, 1.70H, <b>A</b>+<b>B</b>), 3.87–3.83 (m, 6.67H, <b>A</b>+<b>B</b>), 3.82 (s, 0.81H, <b>A</b>), 3.81–3.79 (m, 0.93H, <b>A</b>), 3.78 (s, 2.32H, <b>B</b>), 3.66 (s, 2.67H, <b>A</b>+<b>B</b>), 3.58–3.53 (m, 0.90H, <b>A</b>+<b>B</b>), 3.35 (d, <i>J</i> = 9.8 Hz, 0.74H, <b>B</b>), 2.92 (td, <i>J</i> = 2.8, 13.5 Hz, 0.77H, <b>B</b>), 2.83–2.74 (m, 0.50H, <b>A</b>), 2.65–2.59 (m, 0.38H, <b>A</b>), 2.55–2.37 (m, 2.20H, <b>A</b>+<b>B</b>), 2.32–2.23 (m, 1.00H, <b>A</b>+<b>B</b>), 2.21–2.14 (m, 0.30H, <b>A</b>), 2.11–1.97 (m, 3.64H, <b>A</b>+<b>B</b>), 1.93–1.83 (m, 1.37H, <b>A</b>+<b>B</b>), 1.72–1.58 (m, 5.21H, <b>A</b>+<b>B</b>), 1.54–1.40 (m, 1.19H, <b>A</b>+<b>B</b>), 1.36–1.25 (m, 3.38H, <b>A</b>+<b>B</b>), 1.17–1.07 (m, 2.20H, <b>A</b>+<b>B</b>), 0.95–0.83 (m, 1.22H, <b>A</b>+<b>B</b>), 0.78–0.70 (m, 0.81H, <b>B</b>), 0.61–0.51 (m, 0.22H, <b>A</b>), 0.39–0.30 (m, 0.23H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 172.89 (<b>A</b>), 172.27 (<b>B</b>), 170.70 (<b>B</b>), 170.52 (<b>A</b>), 159.01 (<b>A</b>), 158.58 (<b>B</b>), 154.06 (<b>A</b>), 153.39 (<b>B</b>), 152.60 (<b>B</b>), 151.91 (<b>A</b>), 149.10 (<b>A</b>), 148.99 (<b>B</b>), 147.60 (<b>A</b>), 147.42 (<b>B</b>), 142.18 (<b>B</b>), 137.78 (<b>A</b>), 137.45 (<b>B</b>), 134.19 (<b>A</b>), 133.59 (<b>B</b>), 133.47 (<b>A</b>), 130.94 (<b>A</b>), 130.06 (<b>B</b>), 129.61 (<b>A</b>), 129.44 (<b>B</b>), 127.71 (<b>B</b>), 126.74 (<b>A</b>), 120.44 (<b>B</b>), 120.33 (<b>A</b>), 120.29 (<b>A</b>), 119.32 (<b>B</b>), 114.48 (<b>B</b>), 113.17 (<b>A</b>), 113.08 (<b>A</b>), 112.23 (<b>B</b>), 112.01 (<b>B</b>), 111.84 (<b>A</b>), 111.54 (<b>A</b>), 111.48 (<b>B</b>), 109.75 (<b>A</b>), 107.41 (<b>B</b>), 105.84 (<b>B</b>), 103.89 (<b>A</b>), 76.54 (<b>A</b>), 75.76 (<b>B</b>), 70.13 (<b>A</b>), 70.03 (<b>B</b>), 67.64 (<b>B</b>), 67.55 (<b>A</b>), 67.45 (<b>A</b>), 67.07 (<b>A</b>), 66.76 (<b>A</b>), 65.71 (<b>B</b>), 65.20 (<b>B</b>), 60.90 (<b>A</b>), 60.85 (<b>B</b>), 56.61 (<b>B</b>), 56.23 (<b>A</b>), 56.20 (<b>A</b>), 56.06 (<b>B</b>), 56.00 (<b>B</b>), 55.97 (<b>B</b>), 55.47 (<b>A</b>), 55.23 (<b>B</b>), 52.22 (<b>B</b>), 43.79 (<b>B</b>), 41.74 (<b>B</b>), 41.35 (<b>A</b>), 39.46 (<b>A</b>), 38.42 (<b>B</b>), 38.30 (<b>A</b>), 33.00 (<b>B</b>), 32.60 (<b>A</b>), 31.56 (<b>A</b>), 31.23 (<b>B</b>), 30.82 (<b>B</b>), 30.37 (<b>A</b>), 30.15 (<b>A</b>), 29.99 (<b>B</b>), 29.81 (<b>A</b>), 27.37 (<b>A</b>), 27.10 (<b>B</b>), 26.71 (<b>B</b>), 26.55 (<b>A</b>), 26.35 (<b>B</b>), 26.31 (<b>B</b>), 26.11 (<b>A</b>), 25.84 (<b>B</b>), 24.53 (<b>A</b>), 21.26 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>60</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 786.42117, found 786.42154. HPLC (70–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 5.96 min, purity (220 nm) = 97%.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Macrocycle <b>39</b></h3><div class="NLM_p last">A stirred solution of <b>31</b> (<i>E</i>/<i>Z</i>-alkene mixture, 44 mg, 0.06 mmol, 1.00 equiv) in MeOH (3 mL) was degassed by sparging with argon for 15 min. After addition of Pd/C (10 wt %, 6.4 mg, 0.006 mmol, 0.10 equiv), the solution was sparged with hydrogen for 15 min and was then stirred under a hydrogen atmosphere at room temperature for 45 min. The dark suspension was filtered through Celite and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 5:1) to afford <b>39</b> (20 mg, 45%) as white solid. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.33. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.15 (t, <i>J</i> = 7.9 Hz, 1H), 6.79–6.71 (m, 5H), 6.70–6.66 (m, 1H), 6.33 (d, <i>J</i> = 1.8 Hz, 1H), 6.18–6.15 (m, 1H), 5.74 (dd, <i>J</i> = 5.8, 8.1 Hz, 1H), 5.48 (d, <i>J</i> = 5.7 Hz, 1H), 4.38–4.32 (m, 1H), 4.20–4.12 (m, 1H), 4.07–4.01 (m, 1H), 3.95–3.90 (m, 1H), 3.88 (s, 3H), 3.86 (s, 3H), 3.85 (s, 3H), 3.77 (td, <i>J</i> = 3.0, 8.9 Hz, 1H), 3.56 (s, 3H), 3.28 (d, <i>J</i> = 9.9 Hz, 1H), 2.66–2.55 (m, 2H), 2.50–2.42 (m, 1H), 2.33–2.27 (m, 1H), 2.24–2.16 (m, 1H), 2.16–2.06 (m, 2H), 2.00–1.83 (m, 3H), 1.76–1.52 (m, 8H), 1.46–1.25 (m, 4H), 1.18–1.09 (m, 2H), 0.95–0.84 (m, 1H), 0.80–0.69 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.1, 171.2, 159.6, 153.7, 152.4, 149.0, 147.4, 141.5, 137.0, 134.0, 132.9, 129.3, 120.4, 120.3, 114.6, 113.6, 112.1, 111.4, 108.3, 104.4, 76.0, 72.9, 69.6, 56.6, 56.5, 56.1, 56.0, 55.8, 52.5, 43.8, 41.7, 38.9, 33.2, 31.4, 30.7, 28.4, 26.9, 26.8, 26.4, 26.4, 25.7, 24.0, 21.1. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 730.39496, found 730.39464. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 10.74 min, purity (220 nm) = 99%.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Macrocycle <b>40</b></h3><div class="NLM_p last">A stirred solution of <b>32</b> (<i>E</i>/<i>Z</i>-alkene mixture, 118 mg, 0.16 mmol, 1.00 equiv) in MeOH (3 mL) was degassed by sparging with argon for 15 min. After addition of Pd/C (10 wt %, 20 mg, 0.02 mmol, 0.10 equiv) the solution was sparged with hydrogen for 15 min and was then stirred under a hydrogen atmosphere at room temperature for 15 min. The dark suspension was filtered through Celite and the solvent was removed under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 6:1) to afford <b>40</b> (44 mg, 37%) as a white solid. TLC (CH/EA = 3:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.33. Note: The NMR spectra of compound <b>40</b> shows a peak broadening for the <i>ortho</i> and <i>meta</i> proton and carbon signals of the 3,5-dimethoxyaryl ring and the connected methoxy groups. This might result out of a slow interconversion of the C2-symmetrical nuclei of the aryl ring and the attached methoxy groups, because the connected macrocycle linker probably hinders the rotation of the aryl ring. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.16 (t, <i>J</i> = 7.9 Hz, 1H), 6.84–6.80 (m, 1H), 6.78–6.73 (m, 2H), 6.69–6.59 (m, 3H), 6.44–6.40 (m, 1H), 6.30 (br s, 1H), 5.59 (dd, <i>J</i> = 6.0, 8.3 Hz, 1H), 5.47–5.42 (m, 1H), 4.22–4.15 (m, 1H), 4.10–4.04 (m, 1H), 3.99–3.89 (m, 3H), 3.88–3.80 (m, 9H), 3.51 (br s, 3H), 3.33 (d, <i>J</i> = 9.8 Hz, 1H), 2.82 (td, <i>J</i> = 2.8, 13.4 Hz, 1H), 2.58 (ddd, <i>J</i> = 5.5, 9.6, 14.6 Hz, 1H), 2.46 (ddd, <i>J</i> = 6.6, 9.4, 13.9 Hz, 1H), 2.32–2.22 (m, 1H), 2.22–2.02 (m, 2H), 1.97–1.90 (m, 1H), 1.86 (d, <i>J</i> = 12.5 Hz, 1H), 1.76–1.50 (m, 12H), 1.45–1.38 (m, 1H), 1.35–1.23 (m, 3H), 1.19–1.06 (m, 2H), 0.95–0.83 (m, 1H), 0.79–0.68 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.4, 171.0, 158.7, 153.9, 152.7, 149.0, 147.4, 141.4, 135.2, 133.7, 133.3, 129.4, 120.4, 120.0, 114.6, 114.2, 112.0, 111.4, 107.7, 104.6, 76.2, 72.6, 68.1, 56.4, 56.2, 56.1, 55.9, 55.1, 52.1, 43.7, 41.5, 37.2, 32.9, 31.5, 30.7, 28.9, 27.6, 27.0, 26.7, 26.3, 26.3, 25.7, 22.8, 21.1. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>58</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 744.41061, found 744.41050. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 11.00 min, purity (220 nm) = 99%.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Macrocycle <b>41</b></h3><div class="NLM_p last">The substrate <b>33-(</b><i>E</i><b>)</b> (40 mg, 0.05 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (10 mg, 0.01 mmol, 0.20 equiv) in toluene (2 mL). Additional portions of RhCl(PPh<sub>3</sub>)<sub>3</sub> (2 × 0.1 equiv) and sparging with hydrogen were applied until completion of the reaction. <b>41</b> (30 mg, 75%) was obtained as light yellow resin after purification by flash column chromatography (CH/EA = 2:1). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.23. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.63:1, <b>A</b>/<b>B</b>): δ 7.31–7.26 (m, 0.34H, <b>A</b>), 7.15 (t, <i>J</i> = 7.9 Hz, 0.62H, <b>B</b>), 7.13–7.11 (m, 0.36H, <b>A</b>), 6.92–6.88 (m, 0.38H, <b>A</b>), 6.83–6.74 (m, 2.80H, <b>A</b>+<b>B</b>), 6.72–6.63 (m, 2.10H, <b>A</b>+<b>B</b>), 6.51–6.44 (m, 1.32H, <b>A</b>+<b>B</b>), 6.36 (s, 1.23H, <b>A</b>+<b>B</b>), 5.81 (dd, <i>J</i> = 5.3, 8.7 Hz, 0.39H, <b>A</b>), 5.46 (q, <i>J</i> = 5.2 Hz, 1.25H, <b>B</b>), 4.77 (dd, <i>J</i> = 2.4, 7.0 Hz, 0.37H, <b>A</b>), 4.38 (d, <i>J</i> = 13.4 Hz, 0.38H, <b>A</b>), 4.32–4.26 (m, 0.39H, <b>A</b>), 4.22–4.17 (m, 0.42H, <b>A</b>), 4.16–4.10 (m, 0.67H, <b>B</b>), 4.09–3.79 (m, 12.23H, <b>A</b>+<b>B</b>), 3.75–3.60 (m, 2.77H, <b>A</b>+<b>B</b>), 3.57 (s, 4.20H, <b>A</b>+<b>B</b>), 3.47–3.41 (m, 0.43H, <b>A</b>), 3.34 (d, <i>J</i> = 9.9 Hz, 0.62H, <b>B</b>), 3.19–3.12 (m, 0.43H, <b>A</b>), 3.06–2.96 (m, 0.39H, <b>A</b>), 2.81 (td, <i>J</i> = 3.0, 13.3 Hz, 0.60H, <b>B</b>), 2.73–2.65 (m, 0.39H, <b>A</b>), 2.63–2.51 (m, 0.99H, <b>A</b>+<b>B</b>), 2.47–2.39 (m, 0.63H, <b>B</b>), 2.34–2.26 (m, 1.29H, <b>A</b>+<b>B</b>), 2.20 (d, <i>J</i> = 13.8 Hz, 0.39H, <b>A</b>+<b>B</b>), 2.12–1.99 (m, 1.60H, <b>A</b>+<b>B</b>), 1.94–1.55 (m, 10.94H, <b>A</b>+<b>B</b>), 1.51–1.00 (m, 7.52H, <b>A</b>+<b>B</b>), 0.96–0.68 (m, 2.00H, <b>A</b>+<b>B</b>), 0.64–0.54 (m, 0.36H, <b>A</b>), −0.11 – −0.22 (m, 0.38H, <b>A</b>), −0.25 – −0.37 (m, 0.38H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:2, <b>A</b>/<b>B</b>): δ 174.10 (<b>A</b>), 172.54 (<b>B</b>), 171.85 (<b>A</b>), 170.85 (<b>B</b>), 159.82 (<b>A</b>), 159.04 (<b>B</b>), 153.26 (<b>A</b>), 153.06 (<b>B</b>), 149.12 (<b>A</b>), 148.98 (<b>B</b>), 147.64 (<b>A</b>), 147.46 (<b>B</b>), 143.27 (<b>A</b>), 141.68 (<b>B</b>), 136.68 (<b>A</b>), 135.72 (<b>B</b>), 133.81 (<b>A</b>), 133.59 (<b>B</b>), 133.35 (<b>A</b>), 133.19 (<b>B</b>), 129.90 (<b>A</b>), 129.61 (<b>B</b>), 120.42 (<b>B</b>), 120.38 (<b>A</b>), 120.18 (<b>A</b>), 119.33 (<b>B</b>), 115.26 (<b>B</b>), 113.08 (<b>A</b>), 112.50 (<b>B</b>), 111.97 (<b>B</b>), 111.86 (<b>A</b>), 111.50 (<b>A</b>), 111.40 (<b>B</b>), 111.00 (<b>A</b>), 105.83 (<b>B</b>), 76.27 (<b>B</b>), 75.73 (<b>A</b>), 73.29 (<b>A</b>), 72.77 (<b>B</b>), 71.89 (<b>A</b>), 71.47 (<b>A</b>), 70.52 (<b>B</b>), 70.02 (<b>B</b>), 67.50 (<b>B</b>), 67.46 (<b>A</b>), 56.75 (<b>A</b>), 56.07 (<b>A</b>+<b>B</b>), 55.96 (<b>B</b>), 55.91 (<b>B</b>), 55.68 (<b>A</b>), 55.04 (<b>B</b>), 51.95 (<b>B</b>), 43.55 (<b>B</b>), 41.97 (<b>A</b>), 41.05 (<b>B</b>), 39.45 (<b>A</b>), 38.74 (<b>A</b>), 36.85 (<b>B</b>), 32.87 (<b>B</b>), 32.16 (<b>A</b>), 31.73 (<b>A</b>), 31.50 (<b>B</b>), 30.74 (<b>B</b>), 29.99 (<b>A</b>), 27.32 (<b>A</b>), 27.08 (<b>B</b>), 26.88 (<b>A</b>), 26.74 (<b>B</b>), 26.45 (<b>A</b>), 26.32 (<b>B</b>), 26.27 (<b>B</b>), 26.09 (<b>A</b>), 26.03 (<b>A</b>), 25.77 (<b>A</b>), 25.70 (<b>B</b>), 23.90 (<b>A</b>), 21.09 (<b>B</b>), 20.23 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>60</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 774.42117, found 774.42144. HPLC (60–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 7.86 min, purity (220 nm) = 99%.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Macrocycle <b>42</b></h3><div class="NLM_p last">The substrate <b>34-(</b><i>E</i><b>)</b> (31 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (15 mg, 0.016 mmol, 0.40 equiv) in toluene (2 mL). <b>42</b> (27 mg, 87%) was obtained as a light brown resin after purification by flash column chromatography (CH/EA = 3:1). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.44. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.08:1. Only major rotamer is reported here): δ 7.13 (t, <i>J</i> = 7.9 Hz, 1H), 6.77–6.62 (m, 5H), 6.52–6.48 (m, 1H), 6.41 (s, 2H), 5.50–5.44 (m, 2H), 3.97–3.84 (m, 11H), 3.69–3.65 (m, 1H), 3.60–3.48 (m, 9H), 3.36 (d, <i>J</i> = 9.9 Hz, 1H), 2.85 (td, <i>J</i> = 2.9, 13.3 Hz, 1H), 2.54–2.41 (m, 2H), 2.30–2.26 (m, 1H), 2.11–2.05 (m, 1H), 1.96–1.82 (m, 6H), 1.74–1.60 (m, 8H), 1.44–1.11 (m, 7H), 0.92–0.85 (m, 1H), 0.77–0.71 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:9, <b>A</b>/<b>B</b>): δ 173.48 (<b>A</b>), 172.50 (<b>B</b>), 172.07 (<b>A</b>), 170.83 (<b>B</b>), 158.79 (<b>B</b>), 153.34 (<b>B</b>), 153.24 (<b>A</b>), 149.00 (<b>B</b>), 147.47 (<b>B</b>), 142.73 (<b>A</b>), 141.99 (<b>B</b>), 136.28 (<b>B</b>), 133.56 (<b>B</b>), 133.26 (<b>B</b>), 129.83 (<b>A</b>), 129.64 (<b>B</b>), 120.45 (<b>B</b>), 120.34 (<b>A</b>), 118.97 (<b>B</b>), 114.98 (<b>B</b>), 112.95 (<b>B</b>), 112.51 (<b>A</b>), 112.43 (<b>A</b>), 111.99 (<b>B</b>), 111.88 (<b>A</b>), 111.53 (<b>A</b>), 111.45 (<b>B</b>), 106.14 (<b>B</b>), 105.26 (<b>A</b>), 76.18 (<b>B</b>), 76.02 (<b>A</b>), 70.84 (<b>B</b>), 70.57 (<b>A</b>), 70.05 (<b>B</b>), 69.97 (<b>A</b>), 67.88 (<b>A</b>), 67.70 (<b>B</b>), 67.17 (<b>B</b>), 56.22 (<b>B</b>), 56.07 (<b>B</b>), 55.97 (<b>B</b>), 55.74 (<b>A</b>), 55.03 (<b>B</b>), 51.91 (<b>B</b>), 43.60 (<b>B</b>), 41.56 (<b>A</b>), 41.22 (<b>B</b>), 39.47 (<b>A</b>), 38.42 (<b>A</b>), 37.50 (<b>B</b>), 34.85 (<b>A</b>), 32.92 (<b>B</b>), 31.59 (<b>A</b>), 31.39 (<b>B</b>), 30.87 (<b>B</b>), 30.72 (<b>B</b>), 30.57 (<b>A</b>), 27.03 (<b>B</b>), 26.74 (<b>B</b>), 26.53 (<b>A</b>), 26.44 (<b>B</b>), 26.34 (<b>B</b>), 26.28 (<b>B</b>), 25.75 (<b>B</b>), 25.71 (<b>B</b>), 21.05 (<b>B</b>), 20.42 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>62</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 788.43682, found 788.43728. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 11.19 min, purity (220 nm) = 99%.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Macrocycle <b>43</b></h3><div class="NLM_p last">The substrate <b>35</b> (<i>E</i>/<i>Z</i>-alkene mixture, 30 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (20 mg, 0.021 mmol, 0.50 equiv) in THF/MeOH (v/v = 1:1, 10 mL). <b>43</b> (20 mg, 66%) was obtained after purification by flash column chromatography (CH/EA = 5:1) as a white solid. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.60. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.22:1, <b>A</b>/<b>B</b>): δ 7.29 (t, <i>J</i> = 7.9 Hz, 0.22H, <b>A</b>), 7.15–7.10 (m, 0.97H, <b>A</b>+<b>B</b>), 6.93–6.90 (m, 0.18H, <b>A</b>), 6.84–6.79 (m, 0.29H, <b>A</b>), 6.79–6.75 (m, 0.95H, <b>A</b>+<b>B</b>), 6.74–6.68 (m, 2.59H, <b>A</b>+<b>B</b>), 6.68–6.66 (m, 0.19H, <b>A</b>), 6.65–6.61 (m, 1.58H, <b>A</b>+<b>B</b>), 6.48 (d, <i>J</i> = 1.7 Hz, 0.18H, <b>A</b>), 6.38 (d, <i>J</i> = 1.9 Hz, 0.75H, <b>B</b>), 6.29 (d, <i>J</i> = 1.8 Hz, 0.17H, <b>A</b>), 5.92–5.87 (m, 0.75H, <b>B</b>), 5.78 (dd, <i>J</i> = 5.2, 8.6 Hz, 0.18H, <b>A</b>), 5.61 (dd, <i>J</i> = 6.6, 7.7 Hz, 0.82H, <b>B</b>), 5.53 (d, <i>J</i> = 5.5 Hz, 0.80H, <b>B</b>), 4.78 (td, <i>J</i> = 4.2, 11.3 Hz, 0.17H, <b>A</b>), 4.54–4.48 (m, 0.16H, <b>A</b>), 4.43–4.40 (m, 0.17H, <b>A</b>), 4.25–4.09 (m, 2.98H, <b>A</b>+<b>B</b>), 4.00 (d, <i>J</i> = 13.7 Hz, 0.82H, <b>B</b>), 3.96–3.90 (m, 0.81H, <b>B</b>), 3.87–3.78 (m, 10.08H, <b>A</b>+<b>B</b>), 3.59 (s, 2.43H, <b>B</b>), 3.33 (d, <i>J</i> = 10.2 Hz, 0.80H, <b>B</b>), 2.87–2.79 (m, 0.16H, <b>A</b>), 2.71–2.55 (m, 0.38H, <b>A</b>), 2.54–2.11 (m, 6.59H, <b>A</b>+<b>B</b>), 1.97–1.78 (m, 3.90H, <b>A</b>+<b>B</b>), 1.77–1.49 (m, 7.01H, <b>A</b>+<b>B</b>), 1.46–1.07 (m, 6.33H, <b>A</b>+<b>B</b>), 0.99–0.92 (m, 0.37H, <b>A</b>), 0.88–0.80 (m, 1.17H, <b>A</b>+<b>B</b>), 0.76–0.68 (m, 0.73H, <b>B</b>), 0.22–0.13 (m, 0.16H, <b>A</b>), 0.09–0.00 (m, 0.16H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:4, <b>A</b>/<b>B</b>): δ 173.61 (<b>A</b>), 171.87 (<b>B</b>), 171.31 (<b>A</b>), 170.75 (<b>B</b>), 159.45 (<b>A</b>), 159.17 (<b>B</b>), 154.23 (<b>A</b>), 153.84 (<b>B</b>), 152.77 (<b>B</b>), 150.70 (<b>A</b>), 149.12 (<b>A</b>), 148.95 (<b>B</b>), 147.62 (<b>A</b>), 147.42 (<b>B</b>), 143.49 (<b>A</b>), 141.44 (<b>B</b>), 137.44 (<b>A</b>), 137.24 (<b>B</b>), 133.93 (<b>A</b>), 133.85 (<b>B</b>), 133.56 (<b>B</b>), 133.46 (<b>A</b>), 129.85 (<b>A</b>), 129.19 (<b>B</b>), 120.48 (<b>A</b>), 120.39 (<b>B</b>), 120.35 (<b>B</b>), 114.38 (<b>B</b>), 112.70 (<b>A</b>), 111.96 (<b>B</b>), 111.86 (<b>A</b>), 111.60 (<b>A</b>), 111.56 (<b>A</b>), 111.47 (<b>B</b>), 111.40 (<b>B</b>), 108.53 (<b>B</b>), 108.45 (<b>A</b>), 103.76 (<b>B</b>), 102.97 (<b>A</b>), 76.13 (<b>A</b>), 75.42 (<b>B</b>), 69.08 (<b>A</b>), 68.09 (<b>B</b>), 67.97 (<b>A</b>), 67.84 (<b>B</b>), 60.93 (<b>A</b>), 60.75 (<b>B</b>), 56.21 (<b>B</b>), 56.07 (<b>B</b>), 56.00 (<b>B</b>), 55.95 (<b>B</b>), 54.90 (<b>B</b>), 51.74 (<b>B</b>), 43.83 (<b>B</b>), 41.58 (<b>B</b>), 41.38 (<b>A</b>), 39.48 (<b>A</b>), 38.45 (<b>A</b>), 37.06 (<b>B</b>), 32.81 (<b>B</b>), 32.65 (<b>A</b>), 31.69 (<b>A</b>), 31.24 (<b>B</b>), 30.67 (<b>B</b>), 30.41 (<b>A</b>), 29.83 (<b>A</b>), 27.84 (<b>B</b>), 26.98 (<b>A</b>), 26.77 (<b>B</b>), 26.71 (<b>A</b>), 26.52 (<b>B</b>), 26.42 (<b>A</b>), 26.37 (<b>A</b>), 26.28 (<b>B</b>), 26.20 (<b>B</b>), 26.02 (<b>A</b>), 25.88 (<b>B</b>), 24.38 (<b>B</b>), 24.29 (<b>A</b>), 24.16 (<b>A</b>), 21.06 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>56</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 730.39496, found 730.39472. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.34 min, purity (220 nm) = 99%.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Macrocycle <b>44</b></h3><div class="NLM_p last">The substrate <b>36-(</b><i>E</i><b>)</b> (25 mg, 0.034 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (16 mg, 0.017 mmol, 0.50 equiv) in THF/MeOH (v/v = 1:1, 10 mL). <b>44</b> (18 mg, 72%) was obtained as a white solid after purification by semipreparative HPLC (85–100% solvent B). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.50. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.32:1, <b>A</b>/<b>B</b>): δ 7.28 (d, <i>J</i> = 8.0 Hz, 0.21H, <b>A</b>), 7.14 (t, <i>J</i> = 7.9 Hz, 0.75H, <b>B</b>), 7.09–7.07 (m, 0.24H, <b>A</b>), 6.90–6.85 (m, 0.52H, <b>A</b>), 6.81–6.72 (m, 2.64H, <b>A</b>+<b>B</b>), 6.71–6.69 (m, 0.30H, <b>A</b>), 6.67–6.61 (m, 2.88H, <b>A</b>+<b>B</b>), 6.43 (d, <i>J</i> = 1.8 Hz, 0.75H, <b>B</b>), 6.23–6.20 (m, 1.01H, <b>A</b>+<b>B</b>), 5.87 (dd, <i>J</i> = 5.4, 8.5 Hz, 0.25H, <b>A</b>), 5.54–5.49 (m, 1.54H, <b>A</b>+<b>B</b>), 4.56–4.47 (m, 0.57H, <b>A</b>), 4.24–4.17 (m, 1.33H, <b>A</b>+<b>B</b>), 4.12–3.91 (m, 3.11H, <b>A</b>+<b>B</b>), 3.87–3.82 (m, 7.90H, <b>A</b>+<b>B</b>), 3.80 (s, 0.81H, <b>A</b>), 3.75 (s, 2.29H, <b>B</b>), 3.65 (s, 2.30H, <b>B</b>), 3.39 (d, <i>J</i> = 10.1 Hz, 0.76H, <b>B</b>), 2.94–2.86 (m, 0.24H, <b>A</b>), 2.75 (td, <i>J</i> = 2.6, 13.3 Hz, 0.75H, <b>B</b>), 2.68–2.53 (m, 0.49H, <b>A</b>), 2.51–2.33 (m, 1.64H, <b>B</b>), 2.32–2.19 (m, 1.24H, <b>A</b>+<b>B</b>), 2.14–2.07 (m, 1.00H), 2.01–1.93 (m, 1.29H, <b>A</b>+<b>B</b>), 1.90–1.55 (m, 12.08H, <b>A</b>+<b>B</b>), 1.51–1.25 (m, 4.00H, <b>A</b>+<b>B</b>), 1.20–1.07 (m, 2.19H, <b>A</b>+<b>B</b>), 1.03–0.82 (m, 1.59H, <b>A</b>+<b>B</b>), 0.78–0.69 (m, 0.77H, <b>B</b>), 0.28–0.18 (m, 0.51H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 174.15 (<b>A</b>), 172.67 (<b>B</b>), 170.75 (<b>A</b>), 170.37 (<b>B</b>), 159.64 (<b>A</b>), 159.33 (<b>B</b>), 154.40 (<b>A</b>), 153.74 (<b>B</b>), 152.11 (<b>B</b>), 151.96 (<b>A</b>), 149.11 (<b>A</b>), 148.97 (<b>B</b>), 147.61 (<b>A</b>), 147.44 (<b>B</b>), 142.39 (<b>A</b>), 141.78 (<b>B</b>), 139.23 (<b>A</b>), 137.19 (<b>B</b>), 133.62 (<b>A</b>), 133.49 (<b>A</b>), 133.39 (<b>B</b>), 133.38 (<b>B</b>), 129.81 (<b>A</b>), 129.41 (<b>B</b>), 120.48 (<b>B</b>), 120.35 (<b>A</b>), 120.29 (<b>B</b>), 113.98 (<b>B</b>), 113.25 (<b>A</b>), 113.22 (<b>A</b>), 112.70 (<b>B</b>), 112.51 (<b>A</b>), 112.02 (<b>B</b>), 111.88 (<b>A</b>), 111.59 (<b>A</b>), 111.48 (<b>B</b>), 107.43 (<b>B</b>), 104.43 (<b>B</b>), 104.38 (<b>A</b>), 76.18 (<b>A</b>), 75.90 (<b>B</b>), 71.70 (<b>A</b>), 68.20 (<b>B</b>), 67.36 (<b>B</b>), 67.06 (<b>A</b>), 60.88 (<b>B</b>), 56.22 (<b>A</b>), 56.15 (<b>B</b>), 56.08 (<b>B</b>), 56.03 (<b>A</b>), 55.93 (<b>B</b>), 55.12 (<b>B</b>), 52.24 (<b>B</b>), 44.06 (<b>B</b>), 41.60 (<b>A</b>), 41.35 (<b>B</b>), 39.85 (<b>A</b>), 38.08 (<b>A</b>), 37.20 (<b>B</b>), 32.86 (<b>B</b>), 32.43 (<b>A</b>), 31.66 (<b>A</b>), 31.28 (<b>B</b>), 30.72 (<b>B</b>), 30.14 (<b>A</b>), 28.52 (<b>A</b>), 28.44 (<b>B</b>), 28.20 (<b>A</b>), 27.63 (<b>B</b>), 27.10 (<b>A</b>), 26.70 (<b>B</b>), 26.68 (<b>B</b>), 26.37 (<b>A</b>), 26.25 (<b>B</b>), 26.13 (<b>B</b>), 25.98 (<b>A</b>), 25.82 (<b>B</b>), 24.15 (<b>A</b>), 22.19 (<b>B</b>), 21.98 (<b>A</b>), 21.02 (<b>B</b>), 20.83 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>58</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 744.41061, found 744.40994. HPLC (70–100% solvent B, 0.8 mL/min, 10.5 min): <i>t</i><sub>R</sub> = 6.32 min, purity (220 nm) = 98%.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Macrocycle <b>45</b></h3><div class="NLM_p last">The substrate <b>37-(</b><i>E</i><b>)</b> (30 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (18 mg, 0.02 mmol, 0.50 equiv) in THF/MeOH (v/v = 1:1, 10 mL). <b>45</b> (25 mg, 83%) was obtained as a white solid after purification by semipreparative HPLC (80–100% solvent B). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.48:1, <b>A</b>/<b>B</b>): δ 7.26–7.23 (m, 0.30H, <b>A</b>), 7.18 (t, <i>J</i> = 7.9 Hz, 0.59H, <b>B</b>), 7.01–6.99 (m, 0.29H, <b>A</b>), 6.90–6.83 (m, 1.25H, <b>A</b>+<b>B</b>), 6.81–6.73 (m, 1.93H, <b>A</b>+<b>B</b>), 6.71–6.61 (m, 2.15H, <b>A</b>+<b>B</b>), 6.60–6.56 (m, 0.78H, <b>A</b>+<b>B</b>), 6.52–6.49 (m, 0.59H, <b>B</b>), 6.38–6.34 (m, 0.69H, <b>B</b>), 6.28 (d, <i>J</i> = 2.0 Hz, 0.29H, <b>A</b>), 5.84 (dd, <i>J</i> = 5.5, 8.3 Hz, 0.29H, <b>A</b>), 5.52 (dd, <i>J</i> = 5.4, 8.7 Hz, 0.60H, <b>B</b>), 5.45–5.42 (m, 0.63H, <b>B</b>), 4.64–4.60 (m, 0.28H, <b>A</b>), 4.53 (d, <i>J</i> = 13.6 Hz, 0.26H, <b>A</b>), 4.27–4.21 (m, 0.34H, <b>A</b>), 4.16–3.91 (m, 4.67H, <b>A</b>+<b>B</b>), 3.87–3.79 (m, 9.44H, <b>A</b>+<b>B</b>), 3.76–3.73 (m, 3.09H, <b>A</b>+<b>B</b>), 3.70–3.64 (m, 2.06H, <b>A</b>+<b>B</b>), 3.36 (d, <i>J</i> = 9.7 Hz, 0.64H, <b>A</b>+<b>B</b>), 2.88 (td, <i>J</i> = 3.0, 13.1 Hz, 0.27H, <b>A</b>), 2.82 (td, <i>J</i> = 3.0, 13.4 Hz, 0.61H, <b>B</b>), 2.66–2.60 (m, 0.65H, <b>B</b>), 2.58–2.46 (m, 0.98H, <b>A</b>+<b>B</b>), 2.43–2.37 (m, 0.63H, <b>B</b>), 2.31–2.25 (m, 1.13H, <b>A</b>+<b>B</b>), 2.19–2.05 (m, 1.96H, <b>A</b>+<b>B</b>), 1.96–1.77 (m, 6.13H, <b>A</b>+<b>B</b>), 1.69–1.57 (m, 4.99H, <b>A</b>+<b>B</b>), 1.52–1.25 (m, 4.86H, <b>A</b>+<b>B</b>), 1.20–1.01 (m, 3.13H, <b>A</b>+<b>B</b>), 0.95–0.85 (m, 1.06H, <b>A</b>+<b>B</b>), 0.77–0.69 (m, 0.70H, <b>B</b>), 0.45–0.37 (m, 0.26H, <b>A</b>), 0.28–0.19 (m, 0.26H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:2, <b>A</b>/<b>B</b>): δ 172.99 (<b>A</b>), 172.49 (<b>B</b>), 170.75 (<b>B</b>), 159.85 (<b>A</b>), 159.18 (<b>B</b>), 154.33 (<b>A</b>), 153.32 (<b>B</b>), 152.26 (<b>B</b>), 151.31 (<b>A</b>), 149.10 (<b>A</b>), 148.98 (<b>B</b>), 147.59 (<b>A</b>), 147.43 (<b>B</b>), 141.96 (<b>A</b>), 141.88 (<b>B</b>), 139.04 (<b>A</b>), 137.72 (<b>B</b>), 134.29 (<b>A</b>), 133.55 (<b>B</b>), 133.53 (<b>A</b>), 133.47 (<b>B</b>), 129.57 (<b>B</b>), 120.53 (<b>A</b>), 120.48 (<b>B</b>), 120.30 (<b>A</b>), 119.48 (<b>B</b>), 114.92 (<b>B</b>), 114.81 (<b>A</b>), 113.01 (<b>A</b>), 111.97 (<b>B</b>), 111.84 (<b>A</b>), 111.67 (<b>B</b>), 111.53 (<b>A</b>), 111.46 (<b>B</b>), 110.88 (<b>A</b>), 108.44 (<b>B</b>), 106.18 (<b>B</b>), 104.71 (<b>A</b>), 76.32 (<b>A</b>), 75.75 (<b>B</b>), 71.17 (<b>A</b>), 71.00 (<b>B</b>), 70.62 (<b>A</b>), 69.26 (<b>B</b>), 69.16 (<b>B</b>), 67.77 (<b>A</b>), 67.55 (<b>B</b>), 60.93 (<b>B</b>), 56.39 (<b>B</b>), 56.23 (<b>A</b>), 56.08 (<b>A</b>), 56.06 (<b>B</b>), 56.02 (<b>A</b>), 55.94 (<b>B</b>), 55.88 (<b>A</b>), 55.76 (<b>A</b>), 55.15 (<b>B</b>), 52.16 (<b>B</b>), 43.74 (<b>B</b>), 41.28 (<b>B</b>), 39.36 (<b>A</b>), 38.19 (<b>A</b>), 37.46 (<b>B</b>), 32.91 (<b>B</b>), 32.49 (<b>A</b>), 31.68 (<b>A</b>), 31.32 (<b>B</b>), 30.82 (<b>B</b>), 30.37 (<b>A</b>), 27.45 (<b>A</b>), 26.97 (<b>B</b>), 26.72 (<b>B</b>), 26.55 (<b>A</b>), 26.43 (<b>B</b>), 26.37 (<b>B</b>), 26.17 (<b>A</b>), 26.04 (<b>A</b>), 25.79 (<b>B</b>), 25.67 (<b>B</b>), 24.38 (<b>A</b>), 21.08 (<b>B</b>), 21.00 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>60</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 774.42117, found 774.42152. HPLC (70–100% solvent B, 0.8 mL/min, 20 min): <i>t</i><sub>R</sub> = 6.12 min, purity (220 nm) = 98%.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Macrocycle <b>46</b></h3><div class="NLM_p last">The substrate <b>38-(</b><i>E</i><b>)</b> (75 mg, 0.095 mmol, 1.00 equiv) was applied to general procedure E with RhCl(PPh<sub>3</sub>)<sub>3</sub> (44 mg, 0.048 mmol, 0.50 equiv) in THF/MeOH (v/v = 1:1, 10 mL). <b>46</b> (33 mg, 44%) was obtained as a white solid after purification by semipreparative HPLC (85–100% solvent B). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.29. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.55:1, <b>A</b>/<b>B</b>): δ 7.29–7.24 (m, 0.64H, <b>A</b>), 7.17–7.11 (m, 0.66H, <b>B</b>), 7.01–6.98 (m, 0.36H, <b>A</b>), 6.90–6.87 (m, 0.38H, <b>A</b>), 6.86–6.83 (m, 0.39H, <b>A</b>), 6.81–6.72 (m, 2.27H, <b>A</b>+<b>B</b>), 6.71–6.68 (m, 0.34H, <b>A</b>), 6.67–6.62 (m, 2.22H, <b>A</b>+<b>B</b>), 6.59 (d, <i>J</i> = 1.8 Hz, 0.35H, <b>A</b>), 6.50–6.47 (m, 0.65H, <b>B</b>), 6.43 (d, <i>J</i> = 1.8 Hz, 0.64H, <b>B</b>), 6.29 (d, <i>J</i> = 1.8 Hz, 0.37H, <b>A</b>), 5.77 (dd, <i>J</i> = 5.6, 8.2 Hz, 0.37H, <b>A</b>), 5.52–5.48 (m, 1.35H, <b>B</b>), 4.65–4.61 (m, 0.35H, <b>A</b>), 4.52 (br d, <i>J</i> = 13.8 Hz, 0.36H, <b>A</b>), 4.37 (br s, 0.66H, <b>B</b>), 4.26–4.20 (m, 0.39H, <b>A</b>), 4.15–4.02 (m, 2.40H, <b>A</b>+<b>B</b>), 3.97–3.88 (m, 1.27H, <b>A</b>+<b>B</b>), 3.87–3.78 (m, 10.32H, <b>A</b>+<b>B</b>), 3.72–3.47 (m, 6.29H, <b>A</b>+<b>B</b>), 3.41 (d, <i>J</i> = 10.0 Hz, 0.63H, <b>B</b>), 3.00 (td, <i>J</i> = 2.7, 13.3 Hz, 0.64H, <b>B</b>), 2.87 (td, <i>J</i> = 3.1, 13.3 Hz, 0.35H, <b>A</b>), 2.68–2.53 (m, 1.11H, <b>A</b>+<b>B</b>), 2.49–2.33 (m, 1.32H, <b>A</b>+<b>B</b>), 2.32–2.25 (m, 1.00H, <b>A</b>+<b>B</b>), 2.19–1.57 (m, 15.12H, <b>A</b>+<b>B</b>), 1.51–1.02 (m, 7.50H, <b>A</b>+<b>B</b>), 0.95–0.85 (m, 1.00H, <b>A</b>+<b>B</b>), 0.79–0.70 (m, 0.63H, <b>B</b>), 0.48–0.39 (m, 0.34H, <b>A</b>), 0.18–0.09 (m, 0.36H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:2, <b>A</b>/<b>B</b>): δ 173.49 (<b>A</b>), 172.73 (<b>B</b>), 170.69 (<b>A</b>), 170.57 (<b>B</b>), 159.77 (<b>A</b>), 159.30 (<b>B</b>), 154.13 (<b>A</b>), 153.47 (<b>B</b>), 152.45 (<b>B</b>), 152.05 (<b>A</b>), 149.11 (<b>A</b>), 148.99 (<b>B</b>), 147.62 (<b>A</b>), 147.43 (<b>B</b>), 142.21 (<b>A</b>), 142.14 (<b>B</b>), 137.72 (<b>A</b>), 137.32 (<b>B</b>), 133.97 (<b>A</b>), 133.48 (<b>B</b>), 129.62 (<b>A</b>), 129.47 (<b>B</b>), 120.51 (<b>B</b>), 120.48 (<b>A</b>), 120.33 (<b>A</b>), 119.35 (<b>B</b>), 114.82 (<b>B</b>), 113.21 (<b>A</b>), 112.14 (<b>A</b>), 112.01 (<b>B</b>), 111.87 (<b>A</b>), 111.56 (<b>B</b>), 111.50 (<b>B</b>), 111.12 (<b>B</b>), 109.41 (<b>A</b>), 107.31 (<b>B</b>), 105.43 (<b>B</b>), 103.69 (<b>A</b>), 76.65 (<b>A</b>), 75.87 (<b>B</b>), 70.41 (<b>A</b>), 69.91 (<b>B</b>), 67.90 (<b>A</b>), 67.35 (<b>A</b>), 67.08 (<b>B</b>), 66.86 (<b>A</b>), 66.47 (<b>B</b>), 65.49 (<b>B</b>), 60.93 (<b>A</b>), 60.87 (<b>B</b>), 56.39 (<b>B</b>), 56.21 (<b>A</b>), 56.09 (<b>A</b>), 56.05 (<b>B</b>), 56.03 (<b>B</b>), 55.94 (<b>B</b>), 55.01 (<b>B</b>), 52.33 (<b>B</b>), 43.99 (<b>B</b>), 41.60 (<b>A</b>), 41.18 (<b>A</b>), 39.58 (<b>A</b>), 38.31 (<b>A</b>), 37.87 (<b>B</b>), 32.84 (<b>B</b>), 32.44 (<b>A</b>), 31.65 (<b>A</b>), 31.14 (<b>B</b>), 30.83 (<b>B</b>), 30.15 (<b>B</b>), 29.84 (<b>B</b>), 27.34 (<b>A</b>), 27.07 (<b>B</b>), 26.67 (<b>B</b>), 26.39 (<b>A</b>), 26.29 (<b>B</b>), 26.18 (<b>B</b>), 25.98 (<b>A</b>), 25.84 (<b>B</b>), 25.50 (<b>B</b>), 24.31 (<b>A</b>), 21.17 (<b>B</b>), 20.87 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>46</sub>H<sub>62</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 788.43682, found 788.43647. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 6.35 min, purity (220 nm) = 99%.</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(4-(2-hydroxy-ethoxy)-3,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>47</b>)</h3><div class="NLM_p last">A solution of <b>11</b> (200 mg, 0.25 mmol, 1.00 equiv), diazabicyclo[2.2.2]octane (9 mg, 0.08 mmol, 0.30 equiv) and RhCl(PPh<sub>3</sub>)<sub>3</sub> (23 mg, 0.03 mmol, 0.1 equiv) in EtOH/H<sub>2</sub>O (v/v = 9:1, 15 mL) was heated at reflux. The reaction was assayed by TLC and LC–MS analysis and was quenched by the addition of 1 M HCl solution (5 mL) at room temperature when the reaction progress had stopped. After stirring at room temperature for 2 h, the reaction mixture was extracted with EA (3 × 20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 2:1 + 1% MeOH) to yield <b>47</b> (85 mg, 47%) as a white solid. TLC (CH/EA = 1:2, v/v): <i>R</i><sub><i>f</i></sub> = 0.40. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.22:1, <b>A</b>/<b>B</b>): δ 7.78 (s, 0.13H, <b>A</b>), 7.40 (s, 0.71H, <b>B</b>), 7.18 (t, <i>J</i> = 7.8 Hz, 0.18H, <b>A</b>), 7.03 (t, <i>J</i> = 7.9 Hz, 0.81H, <b>B</b>), 6.90–6.81 (m, 0.60H, <b>B</b>), 6.79–6.70 (m, 1.90H, <b>A</b>+<b>B</b>), 6.69–6.58 (m, 2.16H, <b>A</b>+<b>B</b>), 6.54–6.41 (m, 3.83H, <b>A</b>+<b>B</b>), 5.80 (t, <i>J</i> = 6.6, 14.1 Hz, 0.17H, <b>A</b>), 5.57 (dd, <i>J</i> = 5.7, 8.0 Hz, 0.82H, <b>B</b>), 5.47–5.42 (m, 0.82H, <b>B</b>), 4.74–4.68 (m, 0.17H, <b>A</b>), 4.57–4.49 (m, 0.17H, <b>A</b>), 4.14–3.99 (m, 2.17H, <b>A</b>+<b>B</b>), 3.95–3.88 (m, 0.92H, <b>A</b>+<b>B</b>), 3.88–3.76 (m, 7.78H, <b>A</b>+<b>B</b>), 3.76–3.57 (m, 8.43H, <b>A</b>+<b>B</b>), 3.38 (d, <i>J</i> = 9.9 Hz, 0.82H, <b>B</b>), 3.04 (d, <i>J</i> = 9.7 Hz, 0.18H, <b>A</b>), 2.77–2.68 (m, 0.82H, <b>B</b>), 2.63–2.34 (m, 2.78H, <b>A</b>+<b>B</b>), 2.32–2.21 (m, 1.06H, <b>A</b>+<b>B</b>), 2.15–2.03 (m, 1.48H, <b>A</b>+<b>B</b>), 1.98–1.78 (m, 2.80H, <b>A</b>+<b>B</b>), 1.70–1.48 (m, 6.29H, <b>A</b>+<b>B</b>), 1.46–1.04 (m, 6.77H, <b>A</b>+<b>B</b>), 1.04–0.82 (m, 1.14H, <b>A</b>+<b>B</b>), 0.80–0.55 (m, 1.25H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:5, <b>A</b>/<b>B</b>): δ 172.79 (<b>A</b>), 172.55 (<b>B</b>), 170.56 (<b>B</b>), 170.36 (<b>A</b>), 157.02 (<b>A</b>), 156.67 (<b>B</b>), 153.37 (<b>A</b>), 153.01 (<b>B</b>), 149.01 (<b>A</b>), 148.89 (<b>B</b>), 147.52 (<b>A</b>), 147.35 (<b>B</b>), 141.53 (<b>A</b>), 141.48 (<b>B</b>), 135.18 (<b>A</b>+<b>B</b>), 134.74 (<b>A</b>), 134.07 (<b>B</b>), 133.58 (<b>B</b>), 133.42 (<b>A</b>), 129.83 (<b>A</b>), 129.60 (<b>B</b>), 120.34 (<b>B</b>), 120.20 (<b>A</b>), 118.55 (<b>B</b>), 118.38 (<b>A</b>), 115.56 (<b>A</b>), 115.33 (<b>B</b>), 113.53 (<b>A</b>), 112.88 (<b>B</b>), 111.93 (<b>B</b>), 111.81 (<b>A</b>), 111.52 (<b>A</b>), 111.41 (<b>B</b>), 106.00 (<b>B</b>), 105.19 (<b>A</b>), 77.00 (<b>A</b>), 75.76 (<b>B</b>), 75.27 (<b>A</b>+<b>B</b>), 61.45 (<b>B</b>), 56.38 (<b>A</b>), 56.22 (<b>B</b>), 56.02 (<b>A</b>), 55.99 (<b>B</b>), 55.93 (<b>A</b>), 55.90 (<b>B</b>), 55.77 (<b>A</b>), 55.19 (<b>B</b>), 52.25 (<b>B</b>), 43.77 (<b>B</b>), 41.21 (<b>B</b>), 39.80 (<b>A</b>), 37.93 (<b>A</b>), 37.62 (<b>B</b>), 33.09 (<b>A</b>), 32.85 (<b>B</b>), 31.56 (<b>A</b>), 31.19 (<b>B</b>), 30.68 (<b>B</b>), 30.52 (<b>A</b>), 26.79 (<b>B</b>), 26.72 (<b>A</b>), 26.58 (<b>B</b>), 26.44 (<b>A</b>), 26.20 (<b>A</b>+<b>B</b>), 26.14 (<b>B</b>), 25.50 (<b>B</b>), 24.38 (<b>A</b>), 20.83 (<b>B</b>), 20.62 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>54</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 720.37422, found 720.37412. HPLC (50–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.73 min, purity (220 nm) = 99%.</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(4-(3-hydroxy propoxy)-3,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>48</b>)</h3><div class="NLM_p last">A solution of <b>12</b> (100 mg, 0.12 mmol, 1.00 equiv), diazabicyclo[2.2.2]octane (6 mg, 0.055 mmol, 0.45 equiv) and RhCl(PPh<sub>3</sub>)<sub>3</sub> (17 mg, 0.02 mmol, 0.15 equiv) in EtOH/H<sub>2</sub>O (v/v = 9:1, 10 mL) was heated at reflux. The reaction was assayed by TLC and LC–MS analysis and was quenched by the addition of 1 M HCl solution (3 mL) at room temperature when the reaction progress had stopped. After stirring at room temperature for 20 min, the reaction mixture was extracted with EA (3 × 20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 2:1) to yield <b>48</b> (20 mg, 22%) as a white solid. TLC (CH/EA = 1:2, v/v): <i>R</i><sub><i>f</i></sub> = 0.32. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.21:1, <b>A</b>/<b>B</b>): δ 7.21 (t, <i>J</i> = 8.1 Hz, 0.15H, <b>A</b>), 7.06 (t, <i>J</i> = 7.8 Hz, 0.81H, <b>B</b>), 6.88–6.81 (m, 0.50H, <b>A</b>), 6.80–6.71 (m, 1.85H, <b>A</b>+<b>B</b>), 6.71–6.60 (m, 2.00H, <b>A</b>+<b>B</b>), 6.57–6.52 (m, 0.82H, <b>B</b>), 6.49 (s, 1.60H, <b>B</b>), 6.43 (s, 0.29H, <b>A</b>), 6.33 (t, <i>J</i> = 2.1 Hz, 0.79H, <b>B</b>), 5.81 (dd, <i>J</i> = 6.2, 7.6 Hz, 0.13H, <b>A</b>), 5.59 (dd, <i>J</i> = 5.7, 8.1 Hz, 0.79H, <b>B</b>), 5.48–5.43 (m, 0.80H, <b>B</b>), 4.72 (d, <i>J</i> = 5.5 Hz, 0.13H, <b>A</b>), 4.54 (d, <i>J</i> = 13.6 Hz, 0.11H, <b>A</b>), 4.15–4.04 (m, 1.98H, <b>A</b>+<b>B</b>), 3.95–3.78 (m, 9.91H, <b>A</b>+<b>B</b>), 3.67 (s, 4.90H, <b>A</b>+<b>B</b>), 3.38 (d, <i>J</i> = 9.9 Hz, 0.82H, <b>B</b>), 3.06 (d, <i>J</i> = 9.6 Hz, 0.18H, <b>A</b>), 2.69 (td, <i>J</i> = 3.0, 13.3 Hz, 0.93H, <b>B</b>), 2.63–2.53 (m, 0.44H, <b>A</b>), 2.53–2.34 (m, 1.98H, <b>A</b>+<b>B</b>), 2.32–2.22 (m, 1.00H, <b>A</b>+<b>B</b>), 2.18–2.02 (m, 1.42H, <b>A</b>+<b>B</b>), 2.00–1.86 (m, 3.86H, <b>A</b>+<b>B</b>), 1.86–1.75 (m, 0.90H, <b>A</b>+<b>B</b>), 1.74–1.49 (m, 6.10H, <b>A</b>+<b>B</b>), 1.47–0.95 (m, 7.18H, <b>A</b>+<b>B</b>), 0.95–0.58 (m, 2.51H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:6, <b>A</b>/<b>B</b>): δ 172.77 (<b>A</b>), 172.48 (<b>B</b>), 170.76 (<b>B</b>), 170.47 (<b>A</b>), 156.88 (<b>A</b>), 156.65 (<b>B</b>), 153.36 (<b>A</b>), 152.99 (<b>B</b>), 149.14 (<b>A</b>), 149.02 (<b>B</b>), 147.65 (<b>A</b>), 147.48 (<b>B</b>), 141.68 (<b>A</b>), 141.51 (<b>B</b>), 135.98 (<b>B</b>), 134.56 (<b>A</b>), 134.00 (<b>B</b>), 133.66 (<b>B</b>), 133.51 (<b>A</b>), 132.28 (<b>A</b>), 129.94 (<b>A</b>), 129.69 (<b>B</b>), 120.40 (<b>B</b>), 120.28 (<b>A</b>), 119.03 (<b>B</b>), 118.69 (<b>A</b>), 115.65 (<b>A</b>), 115.53 (<b>B</b>), 113.46 (<b>A</b>), 112.77 (<b>B</b>), 112.06 (<b>B</b>), 111.93 (<b>A</b>), 111.62 (<b>A</b>), 111.52 (<b>B</b>), 106.44 (<b>B</b>), 105.37 (<b>A</b>), 75.74 (<b>B</b>), 72.30 (<b>A</b>), 72.20 (<b>B</b>), 61.47 (<b>A</b>), 61.34 (<b>B</b>), 56.41 (<b>A</b>+<b>B</b>), 56.10 (<b>A</b>+<b>B</b>), 56.01 (<b>A</b>+<b>B</b>), 55.91 (<b>A</b>), 55.20 (<b>B</b>), 52.25 (<b>A</b>), 43.84 (<b>B</b>), 41.19 (<b>B</b>), 39.82 (<b>A</b>), 38.04 (<b>A</b>), 37.51 (<b>B</b>), 33.25 (<b>A</b>), 32.91 (<b>B</b>), 32.27 (<b>A</b>+<b>B</b>), 31.64 (<b>A</b>), 31.28 (<b>B</b>), 30.76 (<b>B</b>), 30.66 (<b>A</b>), 26.82 (<b>B</b>), 26.70 (<b>B</b>), 26.58 (<b>A</b>), 26.37 (<b>A</b>), 26.31 (<b>B</b>), 26.25 (<b>B</b>), 25.62 (<b>B</b>), 24.50 (<b>A</b>), 20.90 (<b>B</b>), 20.72 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>56</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 734.38987, found 734.38989. HPLC (50–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 8.71 min, purity (220 nm) = 96%.</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(3-(2-hydroxyethoxy)-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>49</b>)</h3><div class="NLM_p last">A solution of <b>29</b> (233 mg, 0.29 mmol, 1.00 equiv), diazabicyclo[2.2.2]octane (10 mg, 0.09 mmol, 0.30 equiv), and RhCl(PPh<sub>3</sub>)<sub>3</sub> (27 mg, 0.03 mmol, 0.10 equiv) in EtOH/H<sub>2</sub>O (v/v = 9:1, 30 mL) was heated at reflux. The reaction was assayed by TLC and LC–MS analysis and was quenched by the addition of 1 M HCl solution (5 mL) at room temperature when the reaction progress had stopped. After stirring at room temperature for 6 h, the reaction mixture was extracted with EA (3 × 20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 1:2) to yield <b>49</b> (79 mg, 38%) as a white solid. TLC (CH/EA = 1:2, v/v): <i>R</i><sub><i>f</i></sub> = 0.38. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.24–7.19 (m, 0.19H), 7.12–7.06 (m, 0.88H, <b>A</b>), 7.00 (br s, 0.51H, <b>A</b>), 6.88–6.85 (m, 0.24H), 6.84–6.72 (m, 2.21H, <b>A</b>), 6.70–6.54 (m, 4.27H, <b>A</b>), 6.52–6.43 (m, 1.25H, <b>A</b>), 6.23–6.20 (m, 0.74H, <b>A</b>), 5.84–5.77 (m, 0.10H, <b>A</b>), 5.62 (dd, <i>J</i> = 5.9, 8.1 Hz, 0.75H, <b>A</b>), 5.54 (t, <i>J</i> = 6.8 Hz, 0.13H, <b>A</b>), 5.48–5.41 (m, 0.93H, <b>A</b>), 4.75–4.62 (m, 0.14H), 4.53 (d, <i>J</i> = 13.7 Hz, 0.09H), 4.14–3.97 (m, 2.42H, <b>A</b>), 3.96–3.73 (m, 13.96H, <b>A</b>), 3.66 (s, 2.33H, <b>A</b>), 3.43 (d, <i>J</i> = 9.9 Hz, 0.17H), 3.36 (d, <i>J</i> = 10.1 Hz, 0.78H, <b>A</b>), 3.23–3.01 (m, 0.48H, <b>A</b>), 3.00–2.89 (m, 0.37H), 2.57 (td, <i>J</i> = 3.0, 13.3 Hz, 1.05H, <b>A</b>), 2.49–2.25 (m, 3.05H, <b>A</b>), 2.19–1.84 (m, 4.21H, <b>A</b>), 1.84–1.74 (m, 1.00H, <b>A</b>), 1.73–1.52 (m, 6.02H, <b>A</b>), 1.42–1.08 (m, 8.03H, <b>A</b>), 0.94–0.71 (m, 3.01H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 172.80, 172.14 (<b>A</b>), 170.79 (<b>A</b>), 170.40, 156.77, 156.59, 156.40 (<b>A</b>), 153.78, 153.60 (<b>A</b>), 153.18, 152.32, 152.27, 151.80 (<b>A</b>), 149.10, 149.04, 148.98 (<b>A</b>), 147.62, 147.50, 147.44 (<b>A</b>), 141.75, 141.03 (<b>A</b>), 137.90, 137.34 (<b>A</b>), 134.58, 134.49 (<b>A</b>), 133.99, 133.63 (<b>A</b>), 133.59, 133.49, 129.98, 129.79 (<b>A</b>), 120.40 (<b>A</b>), 120.33, 120.28, 119.87 (<b>A</b>), 119.03, 118.51, 115.91, 115.91 (<b>A</b>), 115.75, 113.12, 112.98, 112.69 (<b>A</b>), 112.05 (<b>A</b>), 111.89, 111.57, 111.53, 111.43 (<b>A</b>), 109.16 (<b>A</b>), 109.05, 108.60, 107.15, 106.44 (<b>A</b>), 105.90, 75.92, 75.67 (<b>A</b>), 71.86, 71.76 (<b>A</b>), 71.57, 61.50, 61.46 (<b>A</b>), 61.24 (<b>A</b>), 61.07, 56.44, 56.32 (<b>A</b>), 56.09 (<b>A</b>), 56.03, 55.97 (<b>A</b>), 55.81, 55.04 (<b>A</b>), 54.95, 52.51, 52.17 (<b>A</b>), 44.15, 43.86 (<b>A</b>), 41.39, 41.22, 41.02 (<b>A</b>), 39.77, 38.07, 37.90, 36.90 (<b>A</b>), 33.14, 32.93, 32.82 (<b>A</b>), 31.66, 31.58, 31.42, 31.33 (<b>A</b>), 30.79, 30.73 (<b>A</b>), 30.55, 29.84, 26.80 (<b>A</b>), 26.69 (<b>A</b>), 26.54, 26.30 (<b>A</b>), 26.21 (<b>A</b>), 25.76, 25.66 (<b>A</b>), 24.45, 21.12, 20.95 (<b>A</b>), 20.71. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>54</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 720.3742, found 720.3742. HPLC (50–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.91 min, purity (220 nm) = 98%.</div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(3-(3-hydroxypropoxy)-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>50</b>)</h3><div class="NLM_p last">A solution of <b>30</b> (257 mg, 0.32 mmol, 1.00 equiv), diazabicyclo[2.2.2]octane (11 mg, 0.10 mmol, 0.30 equiv), and RhCl(PPh<sub>3</sub>)<sub>3</sub> (29 mg, 0.03 mmol, 0.10 equiv) in EtOH/H<sub>2</sub>O (v/v = 9:1, 30 mL) was heated at reflux. The reaction was assayed by TLC and LC–MS analysis and was quenched by the addition of 1 M HCl solution (5 mL) at room temperature when the reaction progress had stopped. After stirring at room temperature for 2 h, the reaction mixture was extracted with EA (3 × 20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by semipreparative HPLC (50–100% solvent B) and flash column chromatography (CH/EA = 1:1) to yield <b>50</b> (36 mg, 16%) as a white solid. Note: The poor yield can partly be explained by the esterification of one of the hydroxy groups with TFA after the purification by semipreparative HPLC (the solvents H<sub>2</sub>O and MeCN contain 0.1% TFA each). To us, it is unclear why this happened, since the product fraction was freeze-dried immediately after the purification process (no standing in solution overnight and no solvent removal on a rotary evaporator at elevated temperatures in a water bath!). TLC (CH/EA = 1:2, v/v): <i>R</i><sub><i>f</i></sub> = 0.36. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.30 (br s, 0.52H, <b>A</b>), 7.19 (t, <i>J</i> = 7.8 Hz, 0.15H), 7.06 (t, <i>J</i> = 7.9 Hz, 0.84H, <b>A</b>), 6.87–6.81 (m, 0.47H), 6.79–6.70 (m, 2.06H, <b>A</b>), 6.69–6.61 (m, 2.27H, <b>A</b>), 6.60–6.47 (m, 2.99H, <b>A</b>), 6.45–6.41 (m, 0.33H), 6.32–6.30 (m, 0.73H, <b>A</b>), 5.82 (dd, <i>J</i> = 6.0, 7.7 Hz, 0.13H), 5.60 (dd, <i>J</i> = 5.6, 8.1 Hz, 0.74H, <b>A</b>), 5.52 (dd, <i>J</i> = 4.8, 8.6 Hz, 0.13H), 5.47–5.43 (m, 0.90H, <b>A</b>), 4.74–4.70 (m, 0.12H), 4.52 (d, <i>J</i> = 13.5 Hz, 0.12H), 4.17–4.13 (m, 0.38H), 4.11–4.00 (m, 1.69H, <b>A</b>), 3.95–3.88 (m, 1.07H, <b>A</b>), 3.87–3.74 (m, 12.73H, <b>A</b>), 3.73–3.64 (m, 2.83H, <b>A</b>), 3.44 (d, <i>J</i> = 10.0 Hz, 0.15H), 3.37 (d, <i>J</i> = 10.0 Hz, 0.74H, <b>A</b>), 3.10–2.92 (m, 1.01H, <b>A</b>), 2.68 (td, <i>J</i> = 2.9, 13.3 Hz, 0.77H, <b>A</b>), 2.60–2.53 (m, 0.42H), 2.50–2.34 (m, 1.63H, <b>A</b>), 2.31–2.22 (m, 1.47H, <b>A</b>), 2.13–2.03 (m, 1.47H, <b>A</b>), 1.97–1.86 (m, 3.40H, <b>A</b>), 1.85–1.78 (m, 0.77H, <b>A</b>), 1.72–1.53 (m, 6.00H, <b>A</b>), 1.45–1.36 (m, 0.76H, <b>A</b>), 1.34–1.07 (m, 4.95H, <b>A</b>), 0.93–0.84 (m, 0.93H, <b>A</b>), 0.79–0.71 (m, 0.90H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 173.03, 172.97, 172.54 (<b>A</b>), 170.71 (<b>A</b>), 170.44, 170.28, 157.06, 156.88, 156.62 (<b>A</b>), 153.63, 153.28 (<b>A</b>), 153.11, 152.65, 152.45, 152.20 (<b>A</b>), 149.03, 148.98, 148.93 (<b>A</b>), 147.55, 147.44, 147.38 (<b>A</b>), 142.07, 141.63 (<b>A</b>), 141.36, 137.50, 137.44, 137.29 (<b>A</b>), 134.29, 133.90 (<b>A</b>), 133.63 (<b>A</b>), 133.47, 129.90, 129.69 (<b>A</b>), 120.37 (<b>A</b>), 120.24, 118.90 (<b>A</b>), 118.78, 117.84, 115.74, 115.53 (<b>A</b>), 115.17, 112.98, 112.74 (<b>A</b>), 112.32, 111.96 (<b>A</b>), 111.83, 111.50, 111.42 (<b>A</b>), 107.97 (<b>A</b>), 107.31, 106.98, 106.64, 106.52 (<b>A</b>), 105.37, 75.71 (<b>A</b>), 67.85, 67.53 (<b>A</b>), 61.00, 60.96 (<b>A</b>), 60.80, 60.54, 60.42 (<b>A</b>), 56.42, 56.37 (<b>A</b>), 56.32, 56.05 (<b>A</b>), 55.98 (<b>A</b>), 55.94 (<b>A</b>), 55.84, 55.06 (<b>A</b>), 54.99, 52.64, 52.21 (<b>A</b>), 44.28, 43.83 (<b>A</b>), 41.49, 41.17 (<b>A</b>), 39.81, 38.26, 38.08, 37.47 (<b>A</b>), 33.15, 32.84 (<b>A</b>), 32.72, 32.10, 32.03 (<b>A</b>), 31.62, 31.29 (<b>A</b>), 30.71 (<b>A</b>), 30.53, 26.80 (<b>A</b>), 26.65 (<b>A</b>), 26.52, 26.26 (<b>A</b>), 26.21 (<b>A</b>), 25.71, 25.57 (<b>A</b>), 24.43, 21.16, 20.89 (<b>A</b>), 20.68. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>56</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 734.38987, found 734.39029. HPLC (50–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.09 min, purity (220 nm) = 96%.</div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(4-(2-iodoethoxy)-3,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>51</b>)</h3><div class="NLM_p last">The substrate <b>47</b> (76 mg, 0.11 mmol, 1.00 equiv) was applied to general procedure C with PPh<sub>3</sub> (36 mg, 0.14 mmol, 1.30 equiv), imidazole (15 mg, 0.21 mmol, 2.00 equiv), and iodine (38 mg, 0.15 mmol, 1.40 equiv) in DCM (3 mL). Additional portions of PPh<sub>3</sub> (1.30 + 0.50 equiv), imidazole (2.00 equiv) and iodine (1.50 + 0.50 equiv) were added until completion of the reaction. <b>51</b> (41 mg, 47%) was obtained after purification by flash column chromatography (CH/EA = 3:1) as light yellow solid. TLC (CH/EA = 2:1, v/v): R<sub>f</sub> = 0.36. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.2:1, <b>A</b>:<b>B</b>): δ 7.23 (t, <i>J</i> = 7.8 Hz, 0.19H, <b>A</b>), 7.07 (t, <i>J</i> = 7.9 Hz, 0.77H, <b>B</b>), 6.91–6.87 (m, 0.18H, <b>A</b>), 6.86–6.81 (m, 0.36H, <b>A</b>), 6.81–6.73 (m, 1.86H, <b>A</b>+<b>B</b>), 6.70–6.68 (m, 0.20H, <b>A</b>), 6.67–6.61 (m, 1.85H, <b>A</b>+<b>B</b>), 6.59–6.53 (m, 0.9H, <b>A</b>+<b>B</b>), 6.52 (s, 1.59H, <b>B</b>), 6.43 (s, 0.29H, <b>A</b>), 6.29–6.27 (m, 0.07H, <b>A</b>), 6.23 (t, <i>J</i> = 2.0 Hz, 0.79H, <b>B</b>), 5.81 (dd, <i>J</i> = 6.2, 7.7 Hz, 0.17H, <b>A</b>), 5.61 (dd, <i>J</i> = 6.0, 7.9 Hz, 0.83H, <b>B</b>), 5.47 (q, <i>J</i> = 2.0 Hz, 0.84H, <b>B</b>), 4.72 (d, <i>J</i> = 5.6 Hz, 0.14H, <b>A</b>), 4.54 (d, <i>J</i> = 13.7 Hz, 0.16H, <b>A</b>), 4.23–4.11 (m, 2.01H, <b>A</b>+<b>B</b>), 3.97–3.89 (m, 0.98H, <b>A</b>+<b>B</b>), 3.88–3.79 (m, 6.93H, <b>A</b>+<b>B</b>), 3.68 (s, 4.79H, <b>A</b>+<b>B</b>), 3.50 (s, 0.11H, <b>A</b>), 3.42–3.27 (m, 2.81H, <b>A</b>+<b>B</b>), 3.06 (d, <i>J</i> = 9.6 Hz, 0.14H, <b>A</b>), 2.66 (td, <i>J</i> = 2.9, 13.3 Hz, 0.86H, <b>B</b>), 2.63–2.52 (m, 0.39H, <b>A</b>), 2.52–2.33 (m, 1.91H, <b>A</b>+<b>B</b>), 2.32–2.23 (m, 1.05H, <b>A</b>+<b>B</b>), 2.19–2.03 (m, 1.58H, <b>A</b>+<b>B</b>), 1.96–1.83 (m, 2.10H, <b>A</b>+<b>B</b>), 1.83–1.49 (m, 7.28H, <b>A</b>+<b>B</b>), 1.47–1.07 (m, 7.28H, <b>A</b>+<b>B</b>), 1.05–0.82 (m, 1.38H, <b>A</b>+<b>B</b>), 0.82–0.57 (m, 1.21H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:8, <b>A</b>/<b>B</b>): δ 172.73 (<b>A</b>), 172.32 (<b>B</b>), 170.74 (<b>B</b>), 170.48 (<b>A</b>), 156.52 (<b>A</b>), 156.43 (<b>B</b>), 153.46 (<b>A</b>), 153.08 (<b>B</b>), 149.15 (<b>A</b>), 149.02 (<b>B</b>), 147.48 (<b>B</b>), 141.79 (<b>A</b>), 141.45 (<b>B</b>), 135.38 (<b>B</b>), 135.30 (<b>A</b>), 134.92 (<b>A</b>), 134.58 (<b>B</b>), 133.64 (<b>B</b>), 133.48 (<b>A</b>), 130.01 (<b>A</b>), 129.72 (<b>B</b>), 120.44 (<b>B</b>), 120.28 (<b>A</b>), 119.38 (<b>B</b>), 118.94 (<b>A</b>), 115.64 (<b>A</b>+<b>B</b>), 113.47 (<b>A</b>), 112.58 (<b>B</b>), 112.06 (<b>B</b>), 111.93 (<b>A</b>), 111.63 (<b>A</b>), 111.50 (<b>B</b>), 106.60 (<b>B</b>), 105.39 (<b>A</b>), 75.74 (<b>B</b>), 73.84 (<b>B</b>), 73.75 (<b>A</b>), 56.57 (<b>B</b>), 56.51 (<b>A</b>), 56.12 (<b>B</b>), 56.04 (<b>B</b>), 55.15 (<b>B</b>), 52.23 (<b>B</b>), 43.89 (<b>B</b>), 41.21 (<b>B</b>), 39.84 (<b>A</b>), 38.06 (<b>A</b>), 37.55 (<b>B</b>), 33.28 (<b>A</b>), 32.88 (<b>B</b>), 31.65 (<b>A</b>), 31.28 (<b>B</b>), 30.77 (<b>B</b>), 30.69 (<b>A</b>), 26.78 (<b>B</b>), 26.71 (<b>A</b>+<b>B</b>), 26.60 (<b>A</b>), 26.37 (<b>A</b>), 26.31 (<b>B</b>), 26.25 (<b>B</b>), 25.66 (<b>B</b>), 24.51 (<b>A</b>), 20.91 (<b>B</b>), 20.72 (<b>A</b>), 2.51 (<b>A</b>), 2.36 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>53</sub>INO<sub>9</sub> [M + H]<sup>+</sup>: 830.27598, found 830.27529. HPLC (50–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 16.75 min, purity (220 nm) = 98%.</div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(4-(3-iodoprop-oxy)-3,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>52</b>)</h3><div class="NLM_p last">The substrate <b>48</b> (41 mg, 0.06 mmol, 1.00 equiv) was applied to general procedure C with PPh<sub>3</sub> (28 mg, 0.11 mmol, 1.90 equiv), pyridine (13 μL, 0.17 mmol, 3.00 equiv), and iodine (23 mg, 0.09 mmol, 1.60 equiv) in toluene (3 mL). Additional portions of PPh<sub>3</sub> (1.5 + 0.7 equiv) and iodine (1.00 + 0.50 equiv) were added until completion of the reaction. <b>52</b> (18 mg, 38%) was obtained as a light yellow solid after purification by flash column chromatography (CH/EA = 3:1). TLC (DCM/MeOH = 50:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.21. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.16:1, <b>A</b>/<b>B</b>): δ 7.22 (t, <i>J</i> = 7.8 Hz, 0.16H, <b>A</b>), 7.07 (t, <i>J</i> = 7.8 Hz, 0.83H, <b>B</b>), 6.88 (d, <i>J</i> = 7.7 Hz, 0.15H, <b>A</b>), 6.86–6.81 (m, 0.32H, <b>A</b>), 6.80–6.72 (m, 1.87H, <b>A</b>+<b>B</b>), 6.71–6.67 (m, 0.19H, <b>A</b>), 6.67–6.62 (m, 1.87H, <b>A</b>+<b>B</b>), 6.57–6.51 (m, 2.51H, <b>A</b>+<b>B</b>), 6.42 (s, 0.31H, <b>A</b>), 6.31 (s, 0.76H, <b>B</b>), 6.17 (t, <i>J</i> = 1.9 Hz, 0.81H, <b>B</b>), 5.81 (dd, <i>J</i> = 6.1, 7.7 Hz, 0.14H, <b>A</b>), 5.61 (dd, <i>J</i> = 6.0, 8.0 Hz, 0.85H, <b>B</b>), 5.49–5.44 (m, 0.83H, <b>B</b>), 4.73 (d, <i>J</i> = 5.5 Hz, 0.14H, <b>A</b>), 4.55 (d, <i>J</i> = 13.6 Hz, 0.14H, <b>A</b>), 4.03–3.96 (m, 2.03H, <b>A</b>+<b>B</b>), 3.96–3.89 (m, 0.93H, <b>A</b>+<b>B</b>), 3.89–3.80 (m, 7.03H, <b>A</b>+<b>B</b>), 3.69 (s, 5.09H, <b>A</b>+<b>B</b>), 3.48 (t, <i>J</i> = 6.8 Hz, 0.32H, <b>A</b>), 3.43 (t, <i>J</i> = 6.8 Hz, 1.71H, <b>B</b>), 3.37 (d, <i>J</i> = 10.1 Hz, 0.86H, <b>B</b>), 3.07 (d, <i>J</i> = 9.6 Hz, 0.14H, <b>A</b>), 2.65–2.55 (m, 1.10H, <b>A</b>+<b>B</b>), 2.51–2.42 (m, 0.99H, <b>A</b>+<b>B</b>), 2.41–2.32 (m, 0.89H, <b>A</b>+<b>B</b>), 2.32–2.25 (m, 1.05H, <b>A</b>+<b>B</b>), 2.24–2.02 (m, 3.46H, <b>A</b>+<b>B</b>), 1.96–1.48 (m, 9.94H, <b>A</b>+<b>B</b>), 1.45–0.97 (m, 6.81H, <b>A</b>+<b>B</b>), 0.95–0.83 (m, 0.95H, <b>A</b>+<b>B</b>), 0.82–0.70 (m, 1.02H, <b>A</b>+<b>B</b>), 0.69–0.58 (m, 0.18H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:4, <b>A</b>/<b>B</b>): δ 172.76 (<b>A</b>), 172.23 (<b>B</b>), 170.78 (<b>B</b>), 170.50 (<b>A</b>), 156.51 (<b>A</b>+<b>B</b>), 153.63 (<b>A</b>), 153.30 (<b>B</b>), 149.15 (<b>A</b>), 149.00 (<b>B</b>), 147.47 (<b>B</b>), 141.78 (<b>A</b>), 141.32 (<b>B</b>), 135.90 (<b>B</b>), 134.35 (<b>B</b>), 133.65 (<b>B</b>), 133.49 (<b>A</b>), 130.00 (<b>A</b>), 129.70 (<b>B</b>), 120.44 (<b>B</b>), 120.28 (<b>A</b>), 119.63 (<b>B</b>), 118.88 (<b>A</b>), 115.72 (<b>B</b>), 115.60 (<b>A</b>), 113.47 (<b>A</b>), 112.60 (<b>B</b>), 112.08 (<b>B</b>), 111.92 (<b>A</b>), 111.63 (<b>A</b>), 111.49 (<b>B</b>), 106.69 (<b>B</b>), 105.45 (<b>A</b>), 75.73 (<b>B</b>), 72.92 (<b>B</b>), 72.74 (<b>A</b>), 56.61 (<b>B</b>), 56.51 (<b>A</b>), 56.13 (<b>B</b>), 56.03 (<b>B</b>), 55.94 (<b>A</b>), 55.15 (<b>B</b>), 52.19 (<b>B</b>), 43.89 (<b>B</b>), 41.15 (<b>B</b>), 39.82 (<b>A</b>), 38.08, 37.36 (<b>B</b>), 34.49 (<b>A</b>), 34.42 (<b>B</b>), 33.32 (<b>A</b>), 32.87 (<b>B</b>), 31.64 (<b>A</b>), 31.29 (<b>B</b>), 30.75 (<b>B</b>), 30.70 (<b>A</b>), 26.77 (<b>B</b>), 26.72 (<b>B</b>), 26.66 (<b>A</b>), 26.62 (<b>A</b>), 26.38 (<b>A</b>), 26.32 (<b>B</b>), 26.24 (<b>B</b>), 25.67 (<b>B</b>), 24.52 (<b>A</b>), 20.93 (<b>B</b>), 20.73 (<b>A</b>), 3.59 (<b>A</b>), 3.36 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>55</sub>NIO<sub>9</sub> [M + H]<sup>+</sup>: 844.2916, found 844.2906. HPLC (60–100% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 7.46 min, purity (220 nm) = 98%.</div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(3-(2-iodoethoxy)-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>53</b>)</h3><div class="NLM_p last">The substrate <b>49</b> (88 mg, 0.12 mmol, 1.00 equiv) was applied to general procedure C with PPh<sub>3</sub> (42 mg, 0.16 mmol, 1.30 equiv), imidazole (17 mg, 0.24 mmol, 2.00 equiv) and iodine (44 mg, 0.17 mmol, 1.40 equiv) in DCM (5 mL). Additional portions of PPh<sub>3</sub> (0.50 equiv), imidazole (0.50 equiv), and iodine (0.50 equiv) were added until completion of the reaction. <b>53</b> (74 mg, 73%) was obtained as white solid after purification by semipreparative HPLC (70–100% solvent B). TLC (DCM/MeOH = 20:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.45. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 7.24–7.20 (m, 0.21H), 7.13–7.05 (m, 0.90H, <b>A</b>), 6.89–6.83 (m, 0.51H), 6.80–6.73 (m, 1.83H, <b>A</b>), 6.71–6.60 (m, 1.92H, <b>A</b>), 6.57–6.47 (m, 2.78H, <b>A</b>), 6.43–6.41 (m, 0.13H, <b>A</b>), 6.30–6.25 (m, 0.66H, <b>A</b>), 5.82 (t, <i>J</i> = 7.0 Hz, 0.13H), 5.63–5.56 (m, 0.86H, <b>A</b>), 5.49–5.44 (m, 0.85H, <b>A</b>), 4.78–4.73 (m, 0.15H), 4.52 (d, <i>J</i> = 13.6 Hz, 0.13H), 4.30–4.23 (m, 0.35H), 4.19–4.05 (m, 1.73H, <b>A</b>), 3.99–3.90 (m, 0.73H, <b>A</b>), 3.89–3.75 (m, 10.85H, <b>A</b>), 3.70–3.65 (m, 2.20H, <b>A</b>), 3.46–3.36 (m, 1.26H, <b>A</b>), 3.34–3.25 (m, 1.75H, <b>A</b>), 3.11 (d, <i>J</i> = 9.6 Hz, 0.04H), 3.06 (d, <i>J</i> = 9.9 Hz, 0.13H), 2.98 (td, <i>J</i> = 2.6, 13.4 Hz, 0.14H), 2.70–2.50 (m, 1.19H, <b>A</b>), 2.48–2.23 (m, 2.75H, <b>A</b>), 2.14–2.02 (m, 1.10H, <b>A</b>), 1.93–1.84 (m, 1.38H, <b>A</b>), 1.82–1.52 (m, 7.08H, <b>A</b>), 1.47–1.09 (m, 5.95H, <b>A</b>), 0.95–0.85 (m, 0.85H, <b>A</b>), 0.81–0.59 (m, 1.14H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 173.47, 173.41, 172.96 (<b>A</b>), 170.32 (<b>A</b>), 170.14, 170.02, 158.82, 158.50, 156.52, 156.45 (<b>A</b>), 156.45, 153.86, 153.50 (<b>A</b>), 151.70, 151.38 (<b>A</b>), 149.03, 148.91 (<b>A</b>), 147.55, 147.37 (<b>A</b>), 141.99, 141.51, 141.23 (<b>A</b>), 137.81 (<b>A</b>), 133.86, 133.62 (<b>A</b>), 133.52 (<b>A</b>), 133.39, 133.26, 130.00, 129.77 (<b>A</b>), 120.37 (<b>A</b>), 120.32, 120.25, 119.28 (<b>A</b>), 115.73, 115.63 (<b>A</b>), 115.22, 113.36, 112.60 (<b>A</b>), 112.46, 111.92 (<b>A</b>), 111.82, 111.48, 111.42 (<b>A</b>), 108.82 (<b>A</b>), 108.26, 108.03, 107.25 (<b>A</b>), 106.82, 106.02, 75.87 (<b>A</b>), 70.26 (<b>A</b>), 70.19, 69.99, 61.33 (<b>A</b>), 61.18, 56.49 (<b>A</b>), 56.34, 56.22, 56.05 (<b>A</b>), 56.00, 55.96, 55.93 (<b>A</b>), 55.70, 54.99 (<b>A</b>), 52.79, 52.47 (<b>A</b>), 44.32, 44.11 (<b>A</b>), 41.26, 41.12, 40.91 (<b>A</b>), 38.17, 38.01, 37.33 (<b>A</b>), 33.13, 32.77 (<b>A</b>), 31.63, 31.31, 31.22 (<b>A</b>), 30.69 (<b>A</b>), 30.56, 26.70 (<b>A</b>), 26.65, 26.57 (<b>A</b>), 26.44, 26.24, 26.18 (<b>A</b>), 26.05 (<b>A</b>), 25.64, 25.52 (<b>A</b>), 20.99, 20.76 (<b>A</b>), 20.51, 1.70, 1.62, 1.55 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>53</sub>INO<sub>9</sub> [M + H]<sup>+</sup>: 830.27596, found 830.27584. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.45 min, purity (220 nm) = 98%.</div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> (<i>S</i>)-(<i>R</i>)-3-(3,4-Dimethoxyphenyl)-1-(3-hydroxyphenyl)propyl 1-((<i>S</i>)-2-Cyclohexyl-2-(3-(3-iodopropoxy)-4,5-dimethoxyphenyl)acetyl)piperidine-2-carboxylate (<b>54</b>)</h3><div class="NLM_p last">The substrate <b>50</b> (34 mg, 0.05 mmol, 1.00 equiv) was applied to general procedure C with PPh<sub>3</sub> (19 mg, 0.07 mmol, 1.50 equiv), imidazole (8 mg, 0.12 mmol, 2.50 equiv), and iodine (22 mg, 0.09 mmol, 1.80 equiv) in DCM (5 mL). <b>54</b> (33 mg, 85%) was obtained as a light yellow solid after purification by flash column chromatography (CH/EA = 2:1). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.33. <sup>1</sup>H NMR (500 MHz, THF-<i>d</i><sub>8</sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 8.39 (s, 0.18H), 8.33 (s, 0.03H), 8.23 (s, 0.10H), 8.16 (s, 0.54H, <b>A</b>), 7.15 (t, <i>J</i> = 7.8 Hz, 0.23H), 7.05 (t, <i>J</i> = 7.8 Hz, 0.11H), 6.99 (t, <i>J</i> = 7.8 Hz, 0.63H, <b>A</b>), 6.85–6.75 (m, 1.54H, <b>A</b>), 6.73–6.65 (m, 1.54H, <b>A</b>), 6.64–6.55 (m, 3.65H, <b>A</b>), 6.53–6.51 (m, 0.65H, <b>A</b>), 6.32–6.26 (m, 0.64H, <b>A</b>), 5.79 (dd, <i>J</i> = 5.9, 7.7 Hz, 0.24H, <b>A</b>), 5.54 (dd, <i>J</i> = 5.2, 8.3 Hz, 0.76H, <b>A</b>), 5.43–5.40 (m, 0.63H, <b>A</b>), 5.35 (d, <i>J</i> = 4.8 Hz, 0.12H), 4.89–4.85 (m, 0.21H), 4.53 (d, <i>J</i> = 13.5 Hz, 0.20H), 4.08 (d, <i>J</i> = 13.5 Hz, 0.71H, <b>A</b>), 4.04–3.98 (m, 0.50H), 3.96–3.83 (m, 1.52H, <b>A</b>), 3.78–3.72 (m, 7.03H, <b>A</b>), 3.70 (s, 1.11H, <b>A</b>), 3.66–3.61 (m, 3.98H, <b>A</b>), 3.51 (d, <i>J</i> = 9.9 Hz, 0.80H, <b>A</b>), 3.41 (td, <i>J</i> = 1.2, 6.8 Hz, 0.49H), 3.36–3.25 (m, 1.55H, <b>A</b>), 3.14 (d, <i>J</i> = 9.8 Hz, 0.20H), 2.90 (dd, <i>J</i> = 11.7, 14.4 Hz, 0.11H), 2.77 (td, <i>J</i> = 2.8, 13.3 Hz, 0.63H, <b>A</b>), 2.59–2.32 (m, 2.00H, <b>A</b>), 2.29–2.20 (m, 1.60H, <b>A</b>), 2.17–2.02 (m, 3.04H, <b>A</b>), 1.95–1.85 (m, 1.98H, <b>A</b>), 1.84–1.76 (m, 0.55H), 1.70–1.54 (m, 5.16H, <b>A</b>), 1.51–1.06 (m, 7.20H, <b>A</b>), 0.97–0.77 (m, 1.65H, <b>A</b>), 0.74–0.64 (m, 0.35H). <sup>13</sup>C NMR (126 MHz, THF-<i>d</i><sub><i>8</i></sub>, mixture of rotamers; major rotamer indicated with an <b>A</b>): δ 173.23, 172.85 (<b>A</b>), 172.61, 171.26, 171.17 (<b>A</b>), 171.05, 159.05, 158.60 (<b>A</b>), 158.49, 154.91, 154.68 (<b>A</b>), 154.59, 153.63, 153.46 (<b>A</b>), 153.40, 150.92, 150.85 (<b>A</b>), 149.38, 149.18 (<b>A</b>), 143.69, 143.21 (<b>A</b>), 143.00, 139.18, 139.11 (<b>A</b>), 139.07, 135.68, 134.94 (<b>A</b>), 134.86, 134.74 (<b>A</b>), 134.54, 130.37, 130.30 (<b>A</b>), 130.15, 121.22 (<b>A</b>), 121.11, 118.58, 117.97 (<b>A</b>), 117.76, 116.02, 115.42 (<b>A</b>), 115.33, 114.69, 114.56 (<b>A</b>), 114.47, 114.09, 113.93 (<b>A</b>), 113.85, 113.77, 113.39, 113.33 (<b>A</b>), 109.14 (<b>A</b>), 108.61, 107.78, 107.72 (<b>A</b>), 107.00, 77.55, 76.36 (<b>A</b>), 76.11, 69.81, 69.71 (<b>A</b>), 60.77, 60.71 (<b>A</b>), 60.66, 56.84, 56.56, 56.41 (<b>A</b>), 56.38 (<b>A</b>), 56.34, 56.27 (<b>A</b>), 55.92, 55.71, 55.50 (<b>A</b>), 53.15, 52.95 (<b>A</b>), 44.44 (<b>A</b>), 42.67, 42.57 (<b>A</b>), 42.50, 40.16, 39.42, 39.34 (<b>A</b>), 39.09, 34.92, 34.86 (<b>A</b>), 34.72, 33.84, 33.64 (<b>A</b>), 32.50, 32.22, 32.10 (<b>A</b>), 31.58 (<b>A</b>), 31.43, 27.79 (<b>A</b>), 27.72 (<b>A</b>), 27.60, 27.50, 27.43 (<b>A</b>), 27.37 (<b>A</b>), 27.32, 26.74 (<b>A</b>), 26.64, 25.98, 22.15 (<b>A</b>), 22.04, 21.99, 3.03, 2.98 (<b>A</b>), 2.93. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>55</sub>INO<sub>9</sub> [M + H]<sup>+</sup>: 844.29161, found 844.29172. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.94 min, purity (220 nm) = 96%.</div></div><div id="sec4_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Macrocycle <b>55</b></h3><div class="NLM_p last">The substrate <b>51</b> (41 mg, 0.05 mmol, 1.00 equiv) was applied to general procedure D with K<sub>2</sub>CO<sub>3</sub> (14 mg, 0.10 mmol, 2.00 equiv) in MeCN (<i>c</i> = 1 mM, 50 mL). <b>55</b> (19 mg, 54%) was obtained as a white solid after purification by flash column chromatography (CH/EA = 5:1). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.42. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.17 (t, <i>J</i> = 7.9 Hz, 1H), 6.78–6.69 (m, 5H), 6.66 (dd, <i>J</i> = 2.0, 8.2 Hz, 1H), 6.33 (d, <i>J</i> = 1.8 Hz, 1H), 6.04–5.98 (m, 1H), 5.89 (t, <i>J</i> = 6.1 Hz, 1H), 5.58 (d, <i>J</i> = 5.0 Hz, 1H), 4.47–4.35 (m, 3H), 4.22–4.15 (m, 1H), 4.04–3.97 (m, 1H), 3.88 (s, 3H), 3.85 (s, 3H), 3.84 (s, 3H), 3.56 (s, 3H), 3.29 (d, <i>J</i> = 10.0 Hz, 1H), 2.69–2.61 (m, 1H), 2.53–2.41 (m, 2H), 2.38–2.32 (m, 1H), 2.22–2.01 (m, 2H), 2.01–1.89 (m, 2H), 1.76–1.52 (m, 6H), 1.48–1.28 (m, 4H), 1.15 (td, <i>J</i> = 9.3, 12.3 Hz, 2H), 0.94–0.71 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.6, 171.0, 159.0, 154.1, 152.7, 149.0, 147.4, 141.6, 136.6, 134.0, 133.7, 129.2, 120.3, 119.8, 115.6, 113.8, 112.0, 111.4, 107.8, 103.4, 75.4, 71.7, 69.4, 56.5, 56.1, 56.1, 56.0, 52.7, 44.1, 41.5, 39.6, 33.3, 30.8, 30.7, 26.8, 26.7, 26.4, 26.4, 25.8, 21.2. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>52</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 702.36366, found 702.36395. HPLC (70–10% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.81 min, purity (220 nm) = 98%.</div></div><div id="sec4_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Macrocycle <b>56</b></h3><div class="NLM_p last">The substrate <b>52</b> (16 mg, 0.02 mmol, 1.00 equiv) was applied to general procedure D with K<sub>2</sub>CO<sub>3</sub> (5 mg, 0.04 mmol, 2.00 equiv) in MeCN (<i>c</i> = 0.5 mM, 40 mL). <b>56</b> (11 mg, 81%) was obtained after purification by flash column chromatography (CH/EA = 4:1) as a white solid. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.43. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.14 (t, <i>J</i> = 7.8 Hz, 1H), 6.78–6.66 (m, 6H), 6.38–6.34 (m, 1H), 6.25–6.21 (m, 1H), 5.76 (t, <i>J</i> = 6.9 Hz, 1H), 5.51 (d, <i>J</i> = 5.4 Hz, 1H), 4.44–4.30 (m, 2H), 3.94–3.83 (m, 9H), 3.77 (s, 3H), 3.68 (s, 3H), 3.25 (d, <i>J</i> = 9.8 Hz, 1H), 2.59–2.43 (m, 3H), 2.31 (d, <i>J</i> = 13.5 Hz, 1H), 2.24–2.17 (m, 1H), 2.13–2.07 (m, 3H), 2.01–1.94 (m, 1H), 1.88 (d, <i>J</i> = 12.4 Hz, 1H), 1.68–1.56 (m, 6H), 1.42–1.28 (m, 4H), 1.18–1.09 (m, 2H), 0.91–0.84 (m, 1H), 0.77–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.0, 171.2, 159.3, 153.4, 152.3, 149.0, 147.4, 140.9, 136.8, 133.9, 133.5, 129.2, 120.8, 120.3, 115.8, 112.6, 112.0, 111.4, 107.7, 104.5, 76.3, 69.7, 65.3, 56.7, 56.3, 56.1, 56.0, 55.5, 52.5, 43.9, 41.6, 37.6, 33.2, 31.3, 30.8, 30.4, 27.0, 26.7, 26.4, 26.4, 25.8, 21.1. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 716.37931, found 716.38054. HPLC (70–80% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 9.56 min, purity (220 nm) = 99%.</div></div><div id="sec4_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> Macrocycle <b>57</b></h3><div class="NLM_p last">The substrate <b>53</b> (49 mg, 0.06 mmol, 1.00 equiv) was applied to general procedure D with K<sub>2</sub>CO<sub>3</sub> (16 mg, 0.12 mmol, 2.00 equiv) in MeCN (<i>c</i> = 1 mM, 60 mL). <b>57</b> (17 mg, 42%) was obtained as a white solid after purification by semipreparative HPLC (85–100% Solvent B). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.38. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.14 (t, <i>J</i> = 7.9 Hz, 1H), 6.78–6.74 (m, 3H), 6.72–6.69 (m, 1H), 6.65–6.61 (m, 3H), 6.28–6.24 (m, 1H), 5.64 (dd, <i>J</i> = 5.5, 7.8 Hz, 1H), 5.58 (d, <i>J</i> = 5.5 Hz, 1H), 4.51–4.47 (m, 2H), 4.45–4.40 (m, 1H), 4.27 (dt, <i>J</i> = 4.3, 11.6 Hz, 1H), 4.11 (d, <i>J</i> = 13.7 Hz, 1H), 3.86 (s, 3H), 3.85 (s, 3H), 3.68 (s, 3H), 3.64 (s, 3H), 3.44 (d, <i>J</i> = 10.3 Hz, 1H), 2.89–2.82 (m, 1H), 2.44–2.38 (m, 2H), 2.26–2.13 (m, 2H), 1.90–1.78 (m, 3H), 1.72–1.58 (m, 6H), 1.47–1.30 (m, 4H), 1.21–1.10 (m, 2H), 0.92–0.85 (m, 1H), 0.79–0.71 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.4, 170.8, 159.1, 153.9, 151.3, 149.0, 147.4, 142.0, 138.4, 133.6, 129.4, 120.4, 119.9, 115.8, 113.9, 111.9, 111.4, 111.1, 105.5, 75.1, 68.5, 67.4, 60.7, 56.1, 56.0, 54.4, 51.8, 44.1, 41.6, 37.9, 32.7, 31.0, 30.8, 26.7, 26.4, 26.2, 26.2, 26.0, 20.8. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>41</sub>H<sub>52</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 702.36366, found 702.36404. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.54 min, purity (220 nm) = 99%.</div></div><div id="sec4_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Macrocycle <b>58</b></h3><div class="NLM_p last">The substrate <b>54</b> (32 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure D with K<sub>2</sub>CO<sub>3</sub> (11 mg, 0.08 mmol, 2.00 equiv) in MeCN (<i>c</i> = 0.3 mM, 125 mL). <b>58</b> (7 mg, 26%) was obtained as a white solid after purification by semipreparative HPLC (85–90% solvent B). TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.35. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.1:1. Only major rotamer is reported here: δ 7.16–7.12 (m, 1H), 6.82–6.79 (m, 1H), 6.78–6.75 (m, 2H), 6.71 (d, <i>J</i> = 1.9 Hz, 1H), 6.63–6.61 (m, 2H), 6.51 (d, <i>J</i> = 1.8 Hz, 1H), 6.30–6.27 (m, 1H), 5.58–5.52 (m, 2H), 4.47–4.43 (m, 1H), 4.36–4.31 (m, 1H), 4.17–4.12 (m, 1H), 4.10–4.03 (m, 2H), 3.86–3.84 (m, 6H), 3.74 (s, 3H), 3.66 (s, 3H), 3.41 (d, <i>J</i> = 10.2 Hz, 1H), 2.76 (td, <i>J</i> = 2.6, 13.3 Hz, 1H), 2.43–2.26 (m, 6H), 2.20–2.15 (m, 2H), 1.88 (br d, <i>J</i> = 12.6 Hz, 1H), 1.80–1.75 (m, 1H), 1.71–1.58 (m, 6H), 1.41–1.30 (m, 3H), 1.22–1.11 (m, 3H), 0.90–0.84 (m, 1H), 0.78–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:10. Only major rotamer is reported here: δ 172.4, 170.7, 159.7, 154.0, 151.8, 149.0, 147.4, 141.8, 137.5, 133.8, 133.5, 129.5, 121.2, 120.5, 117.1, 112.5, 112.0, 111.4, 108.3, 104.6, 75.4, 66.8, 65.0, 60.9, 56.1, 56.1, 56.0, 54.8, 52.0, 44.1, 41.3, 37.0, 32.7, 31.0, 30.7, 29.8, 26.7, 26.6, 26.3, 26.2, 25.9, 20.9. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>42</sub>H<sub>54</sub>NO<sub>9</sub> [M + H]<sup>+</sup>: 716.37931, found 716.38019. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.17 min, purity (220 nm) = 96%.</div></div><div id="sec4_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Macrocycle <b>59</b></h3><div class="NLM_p last">The substrate <b>31-(</b><i>E</i><b>)</b> (27 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure H with Pd(OAc)<sub>2</sub> (0.5 mg, 0.002 mmol, 0.05 equiv), 1,4-benzoquinone (4 mg, 0.04 mmol, 1.00 equiv), and aqueous HBF<sub>4</sub> (48 wt %, 10 μL, 0.05 mmol, 1.40 equiv) in MeCN/H<sub>2</sub>O (v/v = 7:1, 800 μL). Additional portions of Pd(OAc)<sub>2</sub> (0.05 equiv) and 1,4-benzoquinone (0.30 equiv) were added until completion of the reaction. <b>59</b> (14 mg, 52%, white solid) was obtained after purification by flash column chromatography (CH/EA = 4:1) as a single product isomer. TLC (CH/EA = 2:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.31. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.21 (t, <i>J</i> = 7.9 Hz, 1H), 6.88–6.85 (m, 1H), 6.78–6.74 (m, 2H), 6.72–6.70 (m, 1H), 6.69–6.65 (m, 2H), 6.44–6.42 (m, 1H), 6.29 (d, <i>J</i> = 1.8 Hz, 1H), 5.64 (dd, <i>J</i> = 5.5, 8.1 Hz, 1H), 5.51–5.47 (m, 1H), 4.50 (dt, <i>J</i> = 6.7, 11.4 Hz, 1H), 4.44–4.36 (m, 3H), 3.94 (d, <i>J</i> = 13.9 Hz, 1H), 3.87 (s, 3H), 3.84 (s, 3H), 3.80 (s, 3H), 3.52 (s, 3H), 3.35 (d, <i>J</i> = 9.9 Hz, 1H), 2.86–2.79 (m, 3H), 2.60–2.46 (m, 2H), 2.33–2.27 (m, 1H), 2.18–2.07 (m, 2H), 1.99–1.92 (m, 1H), 1.88 (d, <i>J</i> = 12.4 Hz, 1H), 1.71–1.61 (m, 6H), 1.49–1.41 (m, 1H), 1.38–1.29 (m, 3H), 1.17–1.09 (m, 2H), 0.94–0.86 (m, 1H), 0.78–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 207.8, 172.5, 171.0, 158.1, 154.1, 153.0, 149.1, 147.5, 142.3, 134.4, 133.9, 133.7, 129.9, 120.6, 120.4, 113.9, 113.7, 112.0, 111.5, 107.7, 104.5, 76.0, 73.2, 67.2, 56.5, 56.2, 56.1, 56.0, 55.3, 52.3, 43.8, 41.5, 39.4, 38.2, 32.9, 31.3, 30.8, 26.9, 26.7, 26.4, 25.7, 21.0. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>54</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 744.37422, found 744.37440. HPLC (60–70% solvent B, 1.5 mL/min, 20 min): <i>t</i><sub>R</sub> = 13.33 min, purity (220 nm) = 99%.</div></div><div id="sec4_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Macrocycle <b>60</b></h3><div class="NLM_p last">The substrate <b>35-(</b><i>E</i><b>)</b> (32 mg, 0.04 mmol, 1.00 equiv) was applied to general procedure H with Pd(OAc)<sub>2</sub> (1 mg, 0.004 mmol, 0.1 equiv), 1,4-benzoquinone (6 mg, 0.05 mmol, 1.20 equiv), and aqueous HBF<sub>4</sub> (48 wt %, 8 μL, 0.06 mmol, 1.40 equiv) in MeCN/H<sub>2</sub>O (v/v = 7:1, 3 mL). Additional portions of Pd(OAc)<sub>2</sub> (3 × 0.1 equiv), 1,4-benzoquinone (2 × 1.20 equiv), and aqueous HBF<sub>4</sub> (4 × 3.00 equiv) were added until completion of the reaction. <b>60</b> (15 mg, 45%, white solid) was obtained after purification by semipreparative HPLC (70–100% solvent B) as single product isomer. TLC (CH/EA = 1:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.63. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.19 (t, <i>J</i> = 7.9 Hz, 1H), 6.83–6.80 (m, 1H), 6.78–6.75 (m, 1H), 6.74–6.70 (m, 1H), 6.68 (d, <i>J</i> = 1.8 Hz, 1H), 6.64–6.61 (m, 2H), 6.45 (d, <i>J</i> = 1.8 Hz, 1H), 6.13–6.09 (m, 1H), 5.57 (dd, <i>J</i> = 5.9, 8.2 Hz, 1H), 5.52 (d, <i>J</i> = 5.5 Hz, 1H), 4.57–4.42 (m, 4H), 4.08 (d, <i>J</i> = 13.8 Hz, 1H), 3.86–3.84 (m, 6H), 3.67 (s, 3H), 3.63 (s, 3H), 3.43 (d, <i>J</i> = 10.2 Hz, 1H), 3.41–3.33 (m, 1H), 2.83–2.67 (m, 2H), 2.52–2.44 (m, 1H), 2.41–2.33 (m, 1H), 2.30 (d, <i>J</i> = 14.0 Hz, 1H), 2.15–2.03 (m, 1H), 1.96–1.74 (m, 3H), 1.73–1.59 (m, 6H), 1.49–1.40 (m, 1H), 1.37–1.22 (m, 3H), 1.20–1.08 (m, 2H), 0.94–0.83 (m, 1H), 0.80–0.70 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 207.6, 173.1, 170.1, 157.9, 154.0, 151.3, 149.0, 147.4, 142.3, 137.7, 133.3, 133.3, 129.8, 121.0, 120.5, 113.4, 113.0, 111.9, 111.5, 107.9, 105.4, 75.6, 73.5, 65.0, 60.8, 56.2, 56.1, 55.9, 55.0, 52.4, 44.3, 41.2, 38.6, 37.5, 32.8, 31.3, 30.7, 26.6, 26.5, 26.2, 26.0, 25.8, 20.8. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>53</sub>NO<sub>10</sub>Na [M + Na]<sup>+</sup>: 766.35617, found 766.35619. HPLC (70–100% solvent B, 0.8 mL/min, 10.5 min): <i>t</i><sub>R</sub> = 3.96 min, purity (220 nm) = 97%.</div></div><div id="sec4_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Macrocycles <b>61a</b> and <b>61b</b></h3><div class="NLM_p last">The substrate <b>36-(</b><i>E</i><b>)</b> (35 mg, 0.047 mmol, 1.00 equiv) was applied to general procedure H with Pd(OAc)<sub>2</sub> (1.0 mg, 0.005 mmol, 0.10 equiv), 1,4-benzoquinone (6.1 mg, 0.056 mmol, 1.20 equiv), and aqueous HBF<sub>4</sub> (48 wt %, 12 μL, 0.066 mmol, 1.40 equiv) in MeCN/H<sub>2</sub>O (v/v = 7:1, 4 mL). The two product isomers <b>61a</b> (19 mg, 53%, white solid) and <b>61b</b> (5 mg, 14%, white solid) were obtained after purification by semipreparative HPLC (70–100% solvent B). Data for macrocycle <b>61a</b>: TLC (CH/EA = 1:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.36. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.18:1, <b>A</b>/<b>B</b>): δ 7.30–7.26 (m, 0.12H, <b>A</b>), 7.16–7.11 (m, 0.88H, <b>B</b>), 7.05–7.01 (m, 0.15H, <b>A</b>), 6.90–6.87 (m, 0.16H, <b>A</b>), 6.86–6.83 (m, 0.16H, <b>A</b>), 6.81–6.75 (m, 1.04H, <b>A</b>+<b>B</b>), 6.74–6.66 (m, 2.91H, <b>A</b>+<b>B</b>), 6.64–6.60 (m, 1.81H, <b>A</b>+<b>B</b>), 6.41 (d, <i>J</i> = 1.8 Hz, 0.86H, <b>B</b>), 6.23 (d, <i>J</i> = 2.0 Hz, 0.15H, <b>A</b>), 5.88 (t, <i>J</i> = 2.0 Hz, 0.85H, <b>B</b>), 5.85 (dd, <i>J</i> = 4.9, 8.6 Hz, 0.17H, <b>A</b>), 5.55–5.49 (m, 1.78H, <b>A</b>+<b>B</b>), 4.60–4.54 (m, 0.31H, <b>A</b>), 4.51–4.41 (m, 0.98H, <b>A</b>+<b>B</b>), 4.40–4.34 (m, 0.17H, <b>A</b>), 4.34–4.28 (m, 1.19H, <b>A</b>+<b>B</b>), 4.27–4.22 (m, 0.87H, <b>B</b>), 4.17–4.12 (m, 0.94H, <b>A</b>+<b>B</b>), 4.11–3.98 (m, 3.04H, <b>A</b>+<b>B</b>), 3.87–3.83 (m, 6.21H, <b>A</b>+<b>B</b>), 3.81 (s, 0.49H, <b>A</b>), 3.68 (s, 2.60H, <b>B</b>), 3.54 (s, 2.59H, <b>B</b>), 3.40 (d, <i>J</i> = 10.3 Hz, 0.86H, <b>B</b>), 3.15–3.08 (m, 0.85H, <b>B</b>), 3.08–3.01 (m, 0.88H, <b>B</b>), 2.94–2.87 (m, 1.60H, <b>A</b>+<b>B</b>), 2.85–2.79 (m, 0.87H, <b>B</b>), 2.70–2.62 (m, 0.96H, <b>A</b>+<b>B</b>), 2.46–2.38 (m, 1.01H, <b>A</b>+<b>B</b>), 2.34–2.26 (m, 1.75H, <b>A</b>+<b>B</b>), 2.23–2.10 (m, 1.05H, <b>A</b>+<b>B</b>), 1.93–1.88 (m, 0.84H, <b>B</b>), 1.82–1.74 (m, 0.86H, <b>A</b>+<b>B</b>), 1.72–1.53 (m, 6.86H, <b>A</b>+<b>B</b>), 1.50–1.26 (m, 3.92H, <b>A</b>+<b>B</b>), 1.22–1.10 (m, 2.79H, <b>A</b>+<b>B</b>), 1.05–0.97 (m, 0.32H, <b>A</b>), 0.91–0.82 (m, 1.04H, <b>A</b>+<b>B</b>), 0.78–0.70 (m, 0.89H, <b>B</b>), 0.32–0.15 (m, 0.31H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:6, <b>A</b>/<b>B</b>): δ 208.68 (<b>B</b>), 206.66 (<b>A</b>), 173.92 (<b>A</b>), 172.38 (<b>B</b>), 170.76 (<b>A</b>), 170.40 (<b>B</b>), 159.26 (<b>A</b>), 158.94 (<b>B</b>), 154.57 (<b>A</b>), 154.03 (<b>B</b>), 151.51 (<b>B</b>), 151.30 (<b>A</b>), 149.11 (<b>A</b>), 148.95 (<b>B</b>), 147.60 (<b>A</b>), 147.42 (<b>B</b>), 141.94 (<b>B</b>), 139.30 (<b>A</b>), 137.24 (<b>B</b>), 133.84 (<b>A</b>), 133.51 (<b>B</b>), 133.35 (<b>B</b>), 129.83 (<b>A</b>), 129.32 (<b>B</b>), 120.64 (<b>B</b>), 120.49 (<b>B</b>), 120.34 (<b>A</b>), 113.68 (<b>A</b>), 113.63 (<b>B</b>), 112.89 (<b>A</b>), 112.43 (<b>A</b>), 112.03 (<b>B</b>), 111.84 (<b>A</b>), 111.69 (<b>B</b>), 111.58 (<b>A</b>), 111.45 (<b>B</b>), 107.67 (<b>B</b>), 107.13 (<b>A</b>), 104.83 (<b>A</b>), 104.55 (<b>B</b>), 75.80 (<b>A</b>), 75.72 (<b>B</b>), 66.35 (<b>A</b>), 64.52 (<b>B</b>), 63.93 (<b>A</b>), 63.80 (<b>B</b>), 60.89 (<b>A</b>), 60.76 (<b>B</b>), 56.22 (<b>A</b>), 56.07 (<b>B</b>), 55.92 (<b>B</b>), 55.07 (<b>B</b>), 52.19 (<b>B</b>), 44.60 (<b>B</b>), 44.18 (<b>B</b>), 43.65 (<b>A</b>), 42.31 (<b>A</b>), 41.66 (<b>B</b>), 41.41 (<b>A</b>), 41.22 (<b>B</b>), 39.79 (<b>A</b>), 38.52 (<b>A</b>), 37.76 (<b>B</b>), 32.78 (<b>B</b>), 32.48 (<b>A</b>), 31.59 (<b>A</b>), 31.25 (<b>B</b>), 30.66 (<b>B</b>), 30.14 (<b>A</b>), 27.12 (<b>A</b>), 26.69 (<b>B</b>), 26.42 (<b>B</b>), 26.32 (<b>A</b>), 26.23 (<b>B</b>), 26.08 (<b>B</b>), 25.97 (<b>A</b>), 25.76 (<b>B</b>), 24.14 (<b>A</b>), 20.92 (<b>B</b>), 20.82 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 758.38987, found 758.39061. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 3.64 min, purity (220 nm) = 99%. Data for macrocycle <b>61b</b>: TLC (CH/EA = 1:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.51. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.21:1, <b>A</b>/<b>B</b>): δ 7.32 (t, <i>J</i> = 7.9 Hz, 0.20H, <b>A</b>), 7.19 (t, <i>J</i> = 7.9 Hz, 0.80H, <b>B</b>), 7.03–7.00 (m, 0.20H, <b>A</b>), 6.98–6.95 (m, 0.21H, <b>A</b>), 6.91–6.87 (m, 0.22H, <b>A</b>), 6.85–6.82 (m, 0.80H, <b>B</b>), 6.81–6.75 (m, 1.78H, <b>A</b>+<b>B</b>), 6.71–6.59 (m, 3.20H, <b>A</b>+<b>B</b>), 6.38 (d, <i>J</i> = 1.8 Hz, 0.75H, <b>B</b>), 6.25–6.22 (m, 0.94H, <b>A</b>+<b>B</b>), 5.88 (dd, <i>J</i> = 5.5, 8.3 Hz, 0.20H, <b>A</b>), 5.53–5.46 (m, 1.56H, <b>B</b>), 4.61–4.49 (m, 1.63H, <b>A</b>+<b>B</b>), 4.40 (d, <i>J</i> = 15.1 Hz, 0.72H, <b>B</b>), 4.18–4.11 (m, 0.19H, <b>A</b>), 4.05–3.98 (m, 2.53H, <b>A</b>+<b>B</b>), 3.88–3.83 (m, 6.76H, <b>A</b>+<b>B</b>), 3.80 (s, 0.68H, <b>A</b>), 3.78 (s, 2.34H, <b>B</b>), 3.67 (s, 2.38H, <b>B</b>), 3.37 (d, <i>J</i> = 10.0 Hz, 0.77H, <b>B</b>), 3.18 (dt, <i>J</i> = 7.0, 16.5 Hz, 0.20H, <b>A</b>), 3.05–2.98 (m, 0.78H, <b>B</b>), 2.87–2.80 (m, 0.19H, <b>A</b>), 2.70–2.34 (m, 7.46H, <b>A</b>+<b>B</b>), 2.31–2.01 (m, 4.54H, <b>A</b>+<b>B</b>), 1.91–1.81 (m, 1.54H, <b>A</b>+<b>B</b>), 1.76–1.52 (m, 6.83H, <b>A</b>+<b>B</b>), 1.45–1.24 (m, 4.25H, <b>A</b>+<b>B</b>), 1.19–1.10 (m, 2.57H, <b>A</b>+<b>B</b>), 0.92–0.83 (m, 1.39H, <b>A</b>+<b>B</b>), 0.78–0.69 (m, 0.85H, <b>B</b>), 0.36–0.21 (m, 0.40H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:4, <b>A</b>/<b>B</b>): δ 208.85 (<b>B</b>), 208.08 (<b>A</b>), 173.22 (<b>A</b>), 172.33 (<b>B</b>), 170.71 (<b>A</b>), 170.54 (<b>B</b>), 158.47 (<b>A</b>), 158.20 (<b>B</b>), 154.57 (<b>A</b>), 153.73 (<b>B</b>), 151.99 (<b>B</b>), 151.07 (<b>A</b>), 149.15 (<b>A</b>), 149.00 (<b>B</b>), 147.69 (<b>A</b>), 147.49 (<b>B</b>), 142.85 (<b>A</b>), 142.21 (<b>B</b>), 137.31 (<b>B</b>), 134.08 (<b>A</b>), 133.69 (<b>B</b>), 133.32 (<b>B</b>), 130.11 (<b>A</b>), 129.80 (<b>B</b>), 121.62 (<b>B</b>), 121.50 (<b>A</b>), 120.50 (<b>B</b>), 120.33 (<b>A</b>), 114.22 (<b>B</b>), 113.27 (<b>B</b>), 112.02 (<b>B</b>), 111.88 (<b>A</b>), 111.59 (<b>A</b>), 111.54 (<b>A</b>), 111.48 (<b>B</b>), 107.61 (<b>B</b>), 105.15 (<b>B</b>), 104.39 (<b>A</b>), 76.01 (<b>A</b>), 75.61 (<b>B</b>), 74.04 (<b>B</b>), 73.00 (<b>A</b>), 69.84 (<b>A</b>), 67.78 (<b>B</b>), 60.92 (<b>B</b>), 56.24 (<b>B</b>), 56.08 (<b>B</b>), 55.96 (<b>B</b>), 54.97 (<b>B</b>), 52.07 (<b>B</b>), 43.97 (<b>B</b>), 41.33 (<b>A</b>), 41.26 (<b>B</b>), 39.63 (<b>A</b>), 38.37 (<b>A</b>), 37.15 (<b>B</b>), 35.49 (<b>B</b>), 34.65 (<b>A</b>), 32.81 (<b>B</b>), 32.60 (<b>A</b>), 31.64 (<b>A</b>), 31.27 (<b>B</b>), 30.73 (<b>B</b>), 30.19 (<b>A</b>), 27.15 (<b>A</b>), 26.70 (<b>B</b>), 26.48 (<b>A</b>), 26.30 (<b>B</b>), 26.21 (<b>B</b>), 26.00 (<b>A</b>), 25.76 (<b>B</b>), 24.89 (<b>B</b>), 24.28 (<b>A</b>), 23.35 (<b>A</b>), 21.07 (<b>B</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>44</sub>H<sub>56</sub>NO<sub>10</sub> [M + H]<sup>+</sup>: 758.38987, found 758.39032. HPLC (70–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.32 min, purity (220 nm) = 97%.</div></div><div id="sec4_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> Macrocycle <b>62</b></h3><div class="NLM_p last">The substrate <b>37-(</b><i>E</i><b>)</b> (38 mg, 0.05 mmol, 1.00 equiv) was applied to general procedure H with Pd(OAc)<sub>2</sub> (1.1 mg, 0.005 mmol, 0.1 equiv), 1,4-benzoquinone (6.4 mg, 0.06 mmol, 1.20 equiv), and aqueous HBF<sub>4</sub> (48 wt %, 13 μL, 0.07 mmol, 1.40 equiv) in MeCN/H<sub>2</sub>O (v/v = 7:1, 5 mL). Additional portions of Pd(OAc)<sub>2</sub> (0.10 equiv), 1,4-benzoquinone (1.20 equiv), and aqueous HBF<sub>4</sub> (2.00 equiv) were added until completion of the reaction. <b>62</b> (13 mg, 33%, white solid) was obtained after purification by semipreparative HPLC (80–90% solvent B) as a single product isomer. TLC (CH/EA = 1:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.39. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.24:1, <b>A</b>/<b>B</b>): δ 7.29–7.26 (m, 0.16H, <b>A</b>), 7.17 (t, <i>J</i> = 7.9 Hz, 0.83H, <b>B</b>), 7.00–6.97 (m, 0.19H, <b>A</b>), 6.92–6.89 (m, 0.22H, <b>A</b>), 6.87–6.84 (m, 0.25H, <b>A</b>), 6.83–6.73 (m, 2.79H, <b>A</b>+<b>B</b>), 6.71–6.63 (m, 2.75H, <b>A</b>+<b>B</b>), 6.59–6.52 (m, 0.29H, <b>A</b>), 6.39–6.34 (m, 1.00H, <b>A</b>+<b>B</b>), 6.27–6.23 (m, 0.77H, <b>B</b>), 5.80 (dd, <i>J</i> = 5.3, 8.4 Hz, 0.18H, <b>A</b>), 5.56 (dd, <i>J</i> = 5.9, 8.3 Hz, 0.80H, <b>B</b>), 5.46 (d, <i>J</i> = 5.5 Hz, 0.79H, <b>B</b>), 5.43–5.40 (m, 0.26H, <b>A</b>), 4.65–4.59 (m, 0.24H, <b>A</b>), 4.52 (d, <i>J</i> = 13.7 Hz, 0.21H, <b>A</b>), 4.41–4.10 (m, 6.04H, <b>A</b>+<b>B</b>), 4.03 (dt, <i>J</i> = 3.4, 10.9 Hz, 0.88H, <b>A</b>+<b>B</b>), 3.98–3.90 (m, 2.18H, <b>A</b>+<b>B</b>), 3.85 (d, <i>J</i> = 4.0 Hz, 6.61H, <b>A</b>+<b>B</b>), 3.80 (d, <i>J</i> = 10.6 Hz, 3.16H, <b>A</b>+<b>B</b>), 3.64 (s, 2.40H, <b>B</b>), 3.35 (d, <i>J</i> = 9.8 Hz, 0.80H, <b>B</b>), 3.14–3.03 (m, 1.00H, <b>A</b>+<b>B</b>), 2.90–2.81 (m, 1.11H, <b>A</b>+<b>B</b>), 2.72–2.56 (m, 3.02H, <b>A</b>+<b>B</b>), 2.52–2.36 (m, 1.96H, <b>A</b>+<b>B</b>), 2.32–2.25 (m, 1.09H, <b>A</b>+<b>B</b>), 2.13–2.04 (m, 1.09H, <b>A</b>+<b>B</b>), 1.97–1.76 (m, 3.05H, <b>A</b>+<b>B</b>), 1.72–1.57 (m, 5.97H, <b>A</b>+<b>B</b>), 1.48–1.25 (m, 4.99H, <b>A</b>+<b>B</b>), 1.21–1.09 (m, 2.19H, <b>A</b>+<b>B</b>), 0.95–0.83 (m, 1.09H, <b>A</b>+<b>B</b>), 0.79–0.70 (m, 1.00H, <b>A</b>+<b>B</b>), 0.39–0.30 (m, 0.17H, <b>A</b>), 0.15–0.06 (m, 0.15H, <b>A</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:3, <b>A</b>/<b>B</b>): δ 207.98 (<b>B</b>), 207.43 (<b>A</b>), 173.36 (<b>A</b>), 172.31 (<b>B</b>), 158.75 (<b>B</b>), 158.45 (<b>A</b>), 154.51 (<b>A</b>), 153.75 (<b>B</b>), 151.81 (<b>B</b>), 151.24 (<b>A</b>), 149.13 (<b>A</b>), 149.00 (<b>B</b>), 147.63 (<b>A</b>), 147.46 (<b>B</b>), 142.19 (<b>A</b>), 141.90 (<b>B</b>), 139.39 (<b>A</b>), 137.75 (<b>B</b>), 134.26 (<b>A</b>), 133.76 (<b>A</b>), 133.64 (<b>B</b>), 133.50 (<b>B</b>), 133.46 (<b>B</b>), 129.74 (<b>A</b>), 129.54 (<b>B</b>), 120.93 (<b>A</b>), 120.49 (<b>B</b>), 120.36 (<b>B</b>), 119.09 (<b>A</b>), 113.82 (<b>A</b>), 113.51 (<b>A</b>), 113.41 (<b>B</b>), 112.82 (<b>B</b>), 112.23 (<b>A</b>), 112.01 (<b>B</b>), 111.85 (<b>A</b>), 111.56 (<b>A</b>), 111.47 (<b>B</b>), 108.84 (<b>B</b>), 108.65 (<b>A</b>), 105.84 (<b>B</b>), 105.10 (<b>A</b>), 76.25 (<b>A</b>), 75.73 (<b>B</b>), 70.97 (<b>A</b>), 70.85 (<b>B</b>), 70.33, 69.60 (<b>B</b>), 63.80 (<b>A</b>), 62.72 (<b>B</b>), 60.99 (<b>B</b>), 60.94 (<b>A</b>), 56.40 (<b>B</b>), 56.33 (<b>A</b>), 56.24 (<b>A</b>), 56.08 (<b>B</b>), 56.04 (<b>B</b>), 55.96 (<b>B</b>), 55.70 (<b>A</b>), 55.29 (<b>B</b>), 54.84 (<b>A</b>), 52.56 (<b>A</b>), 52.26 (<b>B</b>), 44.27 (<b>A</b>), 43.93 (<b>B</b>), 41.38 (<b>B</b>), 41.33 (<b>A</b>), 39.53 (<b>A</b>), 39.06 (<b>A</b>), 38.57 (<b>A</b>), 38.39 (<b>A</b>), 38.27 (<b>B</b>), 37.69 (<b>B</b>), 32.99 (<b>B</b>), 32.84 (<b>A</b>), 32.40 (<b>A</b>), 31.64 (<b>A</b>), 31.41 (<b>A</b>), 31.34 (<b>B</b>), 30.85 (<b>A</b>), 30.78 (<b>B</b>), 30.22 (<b>A</b>), 27.41 (<b>A</b>), 26.79 (<b>B</b>), 26.71 (<b>B</b>), 26.46 (<b>A</b>), 26.37 (<b>B</b>), 26.28 (<b>B</b>), 26.00 (<b>A</b>), 25.64 (<b>B</b>), 24.33 (<b>A</b>), 21.05 (<b>B</b>), 20.94 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>45</sub>H<sub>58</sub>NO<sub>11</sub>H [M + H]<sup>+</sup>: 788.40044, found 788.40094. HPLC (70–100% solvent B, 0.8 mL/min, 10.5 min): <i>t</i><sub>R</sub> = 4.16 min, purity (220 nm) = 98%.</div></div><div id="sec4_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Macrocycle <b>63</b></h3><div class="NLM_p last">To a stirred solution of <b>31-(</b><i>E</i><b>)</b> (51 mg, 0.07 mmol, 1.00 equiv) in acetone (1 mL) at 0 °C was added <i>N</i>-methylmorpholine <i>N</i>-oxide (13 mg, 0.11 mmol, 1.50 equiv). The mixture was treated with OsO<sub>4</sub> (2.5 wt % in tBuOH, 144 μL, 0.014 mmol, 0.20 equiv) and H<sub>2</sub>O (1 mL). The resulting mixture was stirred at room temperature overnight. Then the reaction mixture was diluted with EA (10 mL), layers separated, and the organic phase extracted with H<sub>2</sub>O (2 × 5 mL). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 2:3 and DCM/MeOH = 50:1) to afford the desired product <b>63</b> (14 mg, 26%, white solid) as a mixture of diastereomers (dr = 34:66, determined via HPLC). Note: Attempts to separate the diastereomers via semipreparative HPLC failed. TLC (DCM/MeOH = 20:1, v/v): <i>R</i><sub><i>f</i></sub> = 0.26. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of diastereomers 0.5:1, <b>A</b>/<b>B</b>): δ 7.23–7.13 (m, 1.54H, <b>A</b>+<b>B</b>), 6.89–6.83 (m, 1.15H, <b>A</b>+<b>B</b>), 6.84–6.80 (m, 1.63H, <b>A</b>+<b>B</b>), 6.80–6.75 (m, 3.05H, <b>A</b>+<b>B</b>), 6.75–6.68 (m, 3.63H, <b>A</b>+<b>B</b>), 6.41–6.36 (m, 0.95H, <b>B</b>), 6.33–6.30 (m, 0.91H, <b>A</b>), 6.26–6.22 (m, 0.96H, <b>B</b>), 5.80–5.71 (m, 1.60H, <b>A</b>+<b>B</b>), 5.51–5.44 (m, 1.54H, <b>A</b>+<b>B</b>), 4.37 (dd, <i>J</i> = 3.0, 11.9 Hz, 0.50H, <b>A</b>), 4.23 (dd, <i>J</i> = 4.7, 10.3 Hz, 1.00H, <b>B</b>), 4.19–4.09 (m, 2.36H, <b>A</b>+<b>B</b>), 4.06–3.99 (m, 2.45H, <b>A</b>+<b>B</b>), 3.92–3.83 (m, 16.74H, <b>A</b>+<b>B</b>), 3.79 (dd, <i>J</i> = 3.6, 10.4 Hz, 1.20H, <b>B</b>), 3.72–3.67 (m, 1.61H, <b>A</b>+<b>B</b>), 3.60 (s, 1.30H, <b>A</b>), 3.52 (s, 2.84H, <b>A</b>+<b>B</b>), 3.34–3.28 (m, 1.49H, <b>A</b>+<b>B</b>), 3.00 (br s, 0.95H, <b>A</b>+<b>B</b>), 2.74–2.61 (m, 2.44H, <b>A</b>+<b>B</b>), 2.55–2.46 (m, 1.49H, <b>A</b>+<b>B</b>), 2.36–2.26 (m, 1.61H, <b>A</b>+<b>B</b>), 2.17–2.05 (m, 2.85H, <b>A</b>+<b>B</b>), 1.96–1.85 (m, 2.76H, <b>A</b>+<b>B</b>), 1.76–1.54 (m, 10.94H, <b>A</b>+<b>B</b>), 1.46–1.27 (m, 6.55H, <b>A</b>+<b>B</b>), 1.21–1.09 (m, 2.93H, <b>A</b>+<b>B</b>), 0.98–0.86 (m, 1.45H, <b>A</b>+<b>B</b>), 0.85–0.71 (m, 1.44H, <b>A</b>+<b>B</b>). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of diastereomers 1:2, <b>A</b>/<b>B</b>): δ 172.05 (<b>A</b>), 171.75 (<b>B</b>), 171.26 (<b>B</b>), 171.21 (<b>A</b>), 159.21 (<b>A</b>), 159.10 (<b>B</b>), 154.23 (<b>B</b>), 153.67 (<b>A</b>), 153.28 (<b>B</b>), 152.13 (<b>A</b>), 149.02 (<b>A</b>+<b>B</b>), 147.43 (<b>A</b>+<b>B</b>), 142.19 (<b>B</b>), 141.74 (<b>A</b>), 135.30 (<b>A</b>), 134.97 (<b>B</b>), 134.55 (<b>B</b>), 134.38 (<b>A</b>), 133.85 (<b>A</b>+<b>B</b>), 129.54 (<b>B</b>), 129.48 (<b>A</b>), 121.20 (<b>B</b>), 120.76 (<b>A</b>), 120.47 (<b>B</b>), 120.42 (<b>A</b>), 115.36 (<b>B</b>), 115.03 (<b>A</b>), 114.40 (<b>B</b>), 113.19 (<b>A</b>), 112.16 (<b>B</b>), 112.08 (<b>A</b>), 111.47 (<b>A</b>+<b>B</b>), 108.12 (<b>A</b>), 107.71 (<b>B</b>), 106.26 (<b>B</b>), 104.28 (<b>A</b>), 76.03 (<b>B</b>), 75.87 (<b>A</b>), 74.58 (<b>B</b>), 73.67 (<b>A</b>), 71.21 (<b>A</b>), 69.72 (<b>A</b>), 69.59 (<b>B</b>), 69.52 (<b>B</b>), 69.09 (<b>A</b>), 68.65 (<b>B</b>), 57.20 (<b>B</b>), 56.83 (<b>B</b>), 56.79 (<b>A</b>), 56.32 (<b>A</b>), 56.28 (<b>B</b>), 56.10 (<b>A</b>+<b>B</b>), 56.00 (<b>A</b>+<b>B</b>), 55.74 (<b>A</b>), 52.71 (<b>B</b>), 52.40 (<b>A</b>), 44.01 (<b>B</b>), 43.74 (<b>A</b>), 41.71 (<b>A</b>), 41.49 (<b>B</b>), 39.55 (<b>B</b>), 38.54 (<b>A</b>), 33.35 (<b>B</b>), 33.10 (<b>A</b>), 31.52 (<b>B</b>), 31.45 (<b>A</b>), 30.79 (<b>B</b>), 30.76 (<b>A</b>), 27.05 (<b>A</b>), 26.94 (<b>B</b>), 26.71 (<b>A</b>+<b>B</b>), 26.50 (<b>B</b>), 26.43 (<b>A</b>+<b>B</b>), 26.37 (<b>A</b>), 25.68 (<b>A</b>), 25.57 (<b>B</b>), 21.07 (<b>B</b>), 21.03 (<b>A</b>). HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>56</sub>NO<sub>11</sub> [M + H]<sup>+</sup>: 762.38479, found 762.38530. HPLC (50–60% solvent B, 1.5 mL/min, 20 min, 220 nm): <i>t</i><sub>R,1</sub> = 15.37 min (34%), <i>t</i><sub>R,2</sub> = 15.96 min (66%).</div></div><div id="sec4_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Macrocycles <b>64a</b> and <b>64b</b></h3><div class="NLM_p last">To a stirred solution of <b>35-(</b><i>E</i><b>)</b> (35 mg, 0.05 mmol, 1.00 equiv) in acetone (1 mL) at 0 °C was added <i>N</i>-methylmorpholine <i>N</i>-oxide (8 mg, 0.07 mmol, 1.50 equiv). The mixture was treated with OsO<sub>4</sub> (2.5 wt % in tBuOH, 100 μL, 0.01 mmol, 0.20 equiv) and H<sub>2</sub>O (1 mL). The resulting mixture was stirred at room temperature overnight. Then the reaction mixture was diluted with EA (10 mL), layers separated, and the organic phase extracted with H<sub>2</sub>O (2 × 5 mL). The organic phase was dried over MgSO<sub>4</sub> and concentrated under reduced pressure. The crude product was purified by flash column chromatography (CH/EA = 2:3) and semipreparative HPLC (70–100% solvent B) to afford the desired product diastereomers <b>64a</b> (8 mg, 22%) and <b>64b</b> (7 mg, 18%) as white solids. Data for macrocycle <b>64a</b>: TLC (CH/EA = 1:2 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.24. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>, mixture of rotamers 0.1:1. Only the major rotamer is reported here: δ 7.16 (t, <i>J</i> = 7.9 Hz, 1H), 6.79–6.72 (m, 4H), 6.65–6.60 (m, 2H), 6.41 (d, <i>J</i> = 1.8 Hz, 1H), 5.90–5.87 (m, 1H), 5.61 (dd, <i>J</i> = 6.4, 7.8 Hz, 1H), 5.53 (d, <i>J</i> = 5.5 Hz, 1H), 4.40–4.31 (m, 2H), 4.25 (dd, <i>J</i> = 3.0, 10.2 Hz, 1H), 4.20 (dd, <i>J</i> = 3.5, 9.6 Hz, 1H), 4.00 (br d, <i>J</i> = 13.9 Hz, 1H), 3.86 (s, 3H), 3.85 (s, 3H), 3.74 (s, 3H), 3.59 (s, 3H), 3.36 (d, <i>J</i> = 10.3 Hz, 1H), 3.11 (br s, 1H), 2.55 (t, <i>J</i> = 13.2 Hz, 1H), 2.48–2.41 (m, 1H), 2.37–2.24 (m, 2H), 2.19–2.11 (m, 1H), 1.91–1.84 (m, 2H), 1.75–1.52 (m, 9H), 1.40–1.11 (m, 7H), 0.90–0.81 (m, 2H), 0.76–0.68 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, mixture of rotamers 1:10. Only the major rotamer is reported here: δ 172.0, 170.7, 158.3, 153.9, 151.9, 149.0, 147.5, 141.7, 137.1, 134.3, 133.5, 129.5, 121.0, 120.3, 113.9, 111.9, 111.6, 111.4, 108.0, 104.7, 75.3, 74.1, 69.9, 69.7, 67.9, 60.9, 56.1, 56.0, 56.0, 54.7, 51.8, 43.9, 41.6, 37.2, 32.7, 31.2, 30.7, 26.7, 26.6, 26.2, 26.2, 25.8, 21.0. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>56</sub>NO<sub>11</sub> [M + H]<sup>+</sup>: 762.38479, found 762.38489. HPLC (50–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 4.86 min, purity (220 nm) = 95%. Data for macrocycle <b>64b</b>: TLC (CH/EA = 1:2 + 1% HCOOH, v/v/v): <i>R</i><sub><i>f</i></sub> = 0.42. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.15 (t, <i>J</i> = 7.9 Hz, 1H), 6.79–6.72 (m, 4H), 6.65–6.61 (m, 2H), 6.37 (d, <i>J</i> = 1.8 Hz, 1H), 5.72–5.68 (m, 1H), 5.61 (t, <i>J</i> = 7.2 Hz, 1H), 5.50 (d, <i>J</i> = 5.4 Hz, 1H), 4.41 (ddd, <i>J</i> = 4.7, 8.6, 20.4 Hz, 2H), 4.27–4.19 (m, 2H), 4.13 (dd, <i>J</i> = 4.5, 9.8 Hz, 1H), 4.08–4.04 (m, 1H), 3.99–3.93 (m, 1H), 3.87 (s, 3H), 3.85 (s, 3H), 3.80 (s, 3H), 3.54 (s, 3H), 3.33 (d, <i>J</i> = 10.2 Hz, 1H), 2.58 (br s, 2H), 2.47–2.40 (m, 2H), 2.36–2.25 (m, 2H), 2.19–2.10 (m, 1H), 1.93–1.85 (m, 2H), 1.71–1.53 (m, 6H), 1.41–1.31 (m, 2H), 1.25–1.11 (m, 4H), 0.88–0.82 (m, 1H), 0.76–0.68 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.8, 170.6, 158.8, 153.7, 151.6, 148.9, 147.4, 141.4, 137.5, 134.1, 133.5, 129.4, 121.0, 120.4, 114.2, 112.0, 111.5, 111.4, 109.0, 104.9, 75.7, 72.8, 72.1, 70.0, 66.6, 61.0, 56.1, 55.9, 55.9, 54.9, 51.9, 43.9, 41.5, 37.2, 32.8, 31.4, 30.7, 26.7, 26.5, 26.3, 26.2, 25.7, 21.0. HRMS (<i>m</i>/<i>z</i>): (ESI<sup>+</sup>) calculated for C<sub>43</sub>H<sub>56</sub>NO<sub>11</sub> [M + H]<sup>+</sup>: 762.38479, found 762.38465. HPLC (50–100% solvent B, 0.8 mL/min, 10.50 min): <i>t</i><sub>R</sub> = 5.49 min, purity (220 nm) = 98%.</div></div><div id="sec4_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Fluorescence Polarization Assay for the Binding of Labeled Ligands to FKBP Target Proteins</h3><div class="NLM_p last">The fluorescence polarization assay was performed according to Kozany et al.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> For fluorescence polarization assays, the fluorescent ligand <b>FT</b> (chemical structure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) was diluted from a DMSO stock in a HEPES buffer (150 mM NaCl, 20 mM HEPES, pH 8.0, 0.002% Triton X-100) at double the concentration required for the final sample. The target protein was also diluted in this assay buffer at double the highest concentration required for the final sample. This protein stock was used for a 1:1 serial dilution in a black 384-well assay plate (no. 3575, nonbinding surface, Corning Life Sciences). Finally, each protein dilution (25 μL) was mixed with the dilution of the fluorescent ligand (25 μL). After incubation at room temperature for 30 min, the fluorescence polarization was measured with a plate reader Tecan Genios Pro at excitation/emission values of 535/590 nm. For FKBP12, 12.6, −51FK1, and −52FK1, the binding assays were performed in triplicates in the plate format. The binding curves were analyzed by using Prism 6.0 (GraphPad Software). For the analysis of <i>K</i><sub>d</sub> values, data were fitted to the equation provided in Kozany et al. (Supporting Information, Appendix 2).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Dissociation constants of the fluorescent ligand <b>FT</b> for FKBP12, 12.6, −51FK1 and −52FK1 are summarized in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Competitive Fluorescence Polarization Assay for Binding to FKBP Target Proteins</h3><div class="NLM_p last">The competitive fluorescence polarization assay (FPA) was performed according to Kozany et al.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Briefly, the fluorescent tracer <b>FT</b> (chemical structure and tracer dissociation constants shown in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">Supporting Information</a>) was diluted in the assay buffer (150 mM NaCl, 20 mM HEPES, pH 8.0, 0.002% Triton X-100) to a concentration 11.9× the final concentration (11.9 nM solution used to perform FPA with FKBP12.6, −51FK1, and −52FK1; 5.95 nM solution used to perform FPA with <b>FKBP12</b>). The competitive ligand was dissolved in DMSO to a concentration 62.5× the final concentration. This was used for a 1:1 serial dilution in DMSO. Every sample of this serial dilution was diluted by a factor of 6.25 in the assay buffer supplemented with the fluorescent tracer (11.9 nM or 5.95 nM, respectively) to achieve a 10× concentrated mixture of the fluorescent tracer and competing ligand (add 2 μL of competitive ligand to 10.5 μL of fluorescent tracer). Protein aliquots (45 μL) at 1.11× the final concentration in the assay buffer were transferred into black 384-well assay plates (no. 3575, nonbinding surface, Corning Life Sciences). A mixture (5 μL) of the fluorescent tracer and competitive ligand were added to the protein. After incubation at room temperature for 30 min, the fluorescence polarization was measured with a plate reader Tecan Genios Pro at excitation/emission values of 535/590 nm. The competition curves were analyzed using Prism 6.0 (GraphPad Software). For the analysis of <i>K</i><sub>d</sub> values, data were fitted to the equation provided in Kozany et al. (Supporting Information, Appendix 3).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> At least two independent measurements were performed for each compound. <i>K</i><sub>d</sub> values represent mean values ± the weighted standard deviation which, in addition to the standard deviation (SD), also take the errors from the <i>K</i><sub>d</sub> fitting curves into account.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_m001.gif" alt="" id="_i91" /></img></span></div></div><div id="sec4_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Isothermal Titration Calorimetry Experiments</h3><div class="NLM_p last">ITC experiments were performed with a MicroCal PEAQ-ITC isothermal titration calorimeter (Malvern). All experiments were conducted at 25 °C. For the ITC experiment of SAFit1, a 10 μM solution of FKBP51FK1 formulated in 20 mM HEPES pH 8.0, 20 mM NaCl, and 0.5% DMSO was placed in the sample cell of the PEAQ-ITC instrument. The syringe was filled with a 100 μM solution of SAFit1 formulated in the same buffer. After equilibration, the compound was titrated into the sample cell by 18 injections of 2 μL each. Due to the limiting solubility, compound <b>64a</b> was measured in a reverse setup injecting the protein into a solution of the ligand. Therefore, a 10 μM solution of <b>64a</b> formulated in 20 mM HEPES pH 8.0, 20 mM NaCl, and 5% DMSO was placed in the sample cell of a PEAQ-ITC instrument. The syringe was filled with a 100 μM solution of FKBP51FK1 formulated in the same buffer. After equilibration, the compound was titrated into the sample cell by 18 injections of 2 μL each. The obtained data were analyzed using the provided software package and fitted to a one-site binding model. All titrations were performed as triplicates, and errors were calculated as weighted standard deviations which, in addition to the standard deviation, also take the errors from the fitting curves into account (see formula above at the FP assay procedure).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02195" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02195?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02195</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Further experimental details: crystallographic data collection and refinement statistics; X-ray structures of <b>56</b>, <b>33-(</b><i>Z</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, and <b>63</b> in complex with FKBP51FK1; superposition of X-ray structures with SAFit2 and linear SAFit-like ligands; ITC traces of SAFit1 and <b>64a</b>; additional synthetic representations and explanations; analytical data of intermediate compounds; additional information and binding curves from FP assay; and NMR spectra and HPLC traces of final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf">jm0c02195_si_001.pdf (15.87 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_002.csv">jm0c02195_si_002.csv (4.13 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the X-ray structures of compounds <b>33-(</b><i>Z</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, <b>55</b>, <b>56</b>, <b>63</b>, and <b>64a</b> in complex with FKBP51 (14-140)-A19T (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6W">7A6W</a> (<b>33-(</b><i>Z</i><b>)</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z">7B9Z</a> (<b>35-(</b><i>E</i><b>)</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX">7AWX</a> (<b>55</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6X">7A6X</a> (<b>56</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BA0">7BA0</a> (<b>63</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a> (<b>64a</b>)) are available from the RCSB Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02195" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21050" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21050" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felix Hausch</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3710-8838" title="Orcid link">http://orcid.org/0000-0002-3710-8838</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#452d243036262d053130682124372836312421316b2120"><span class="__cf_email__" data-cfemail="b4dcd5c1c7d7dcf4c0c199d0d5c6d9c7c0d5d0c09ad0d1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Bauder</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Meyners</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick L. Purder</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephanie Merz</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wisely Oki Sugiarto</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas M. Voll</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tim Heymann</span> - <span class="hlFld-Affiliation affiliation">Department Chemistry and
Biochemistry, Clemens-Schöpf-Institute, Technical University Darmstadt, Alarich-Weiss-Straße 4, 64287 Darmstadt, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24464" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24464" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the DFG Grant HA5565-5/1, the LOEWE cluster TRABITA, and the Pioneer Fund (ENTEGA/TU Darmstadt). Diffraction data have been collected on BL14.1 at the BESSY II electron storage ring operated by the Helmholtz-Zentrum Berlin, Germany. We would particularly like to acknowledge the help and support of Jan Wollenhaupt during the experiment. We acknowledge the Paul Scherrer Institut, Villigen, Switzerland, for provision of synchrotron radiation beamtime at beamline PX of the Swiss Light Source (SLS) and would like to thank the beamline scientists for assistance. The synchrotron MX data was collected at beamline P13 operated by EMBL Hamburg at the PETRA III storage ring (DESY, Hamburg, Germany). We would like to thank David von Stetten for the assistance in using the beamline.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CH</td><td class="NLM_def"><p class="first last">cyclohexane</p></td></tr><tr><td class="NLM_term">DCC</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-dicyclohexylcarbodiimide</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-(dimethylamino)pyridine</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-(dimethylamino)propyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">EA</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">(ESI+)</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">FKBP</td><td class="NLM_def"><p class="first last">FK506-binding protein</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)-methylene]-1H-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">LiHMDS</td><td class="NLM_def"><p class="first last">lithium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term"><i>n</i>-BuLi</td><td class="NLM_def"><p class="first last"><i>n</i>-butyllithium</p></td></tr><tr><td class="NLM_term">NMO</td><td class="NLM_def"><p class="first last">4-methylmorpholine <i>N</i>-oxide</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">TBAF</td><td class="NLM_def"><p class="first last">tetrabutylammonium fluoride</p></td></tr><tr><td class="NLM_term">TBDPSCl</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl(chloro)diphenylsilane</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, E. R.</span></span> <span> </span><span class="NLM_article-title">Chaperoning steroidal physiology: Lessons from mouse genetic models of Hsp90 and its cochaperones</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1823</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">729</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2011.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bbamcr.2011.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22155719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFSgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1823&publication_year=2012&pages=722-729&author=E.+R.+Sanchez&title=Chaperoning+steroidal+physiology%3A+Lessons+from+mouse+genetic+models+of+Hsp90+and+its+cochaperones&doi=10.1016%2Fj.bbamcr.2011.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chaperoning steroidal physiology: Lessons from mouse genetic models of Hsp90 and its cochaperones</span></div><div class="casAuthors">Sanchez, Edwin R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1823</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-729</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The mol. chaperone Hsp90 is abundant, ubiquitous, and catholic to biol. processes in eukaryotes, controlling phosphorylation cascades, protein stability and turnover, client localization and trafficking, and ligand-receptor interactions.  Not surprisingly, Hsp90 does not accomplish these activities alone.  Instead, an ever-growing no. of cochaperones have been identified, leading to an explosion of reports on their mol. and cellular effects on Hsp90 chaperoning of client substrates.  Notable among these clients are many members of the steroid receptor family, such as glucocorticoid, androgen, estrogen and progesterone receptors.  Cochaperones typically assocd. with the mature, hormone-competent states of these receptors include p23, the FK506-binding protein 52 (FKBP52), FKBP51, protein phosphatase 5 (PP5) and cyclophilin 40 (Cyp40).  The ultimate relevance of these cochaperones to steroid receptor action depends on their physiol. effects.  In recent years, the first mouse genetic models of these cochaperones have been developed.  This work will review the complex and intriguing phenotypes so far obtained in genetically-altered mice and compare them to the known mol. and cellular impacts of cochaperones on steroid receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJdJEf85X2nrVg90H21EOLACvtfcHk0lgycxFa0v0vVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFSgt7s%253D&md5=03195ad9b56e3768e96ad1d5c772f49a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2011.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2011.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DE.%2BR.%26atitle%3DChaperoning%2520steroidal%2520physiology%253A%2520Lessons%2520from%2520mouse%2520genetic%2520models%2520of%2520Hsp90%2520and%2520its%2520cochaperones%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2012%26volume%3D1823%26spage%3D722%26epage%3D729%26doi%3D10.1016%2Fj.bbamcr.2011.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Storer, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galigniana, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span> <span> </span><span class="NLM_article-title">FKBP51 and FKBP52 in signaling and disease</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.tem.2011.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21889356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=481-490&author=C.+L.+Storerauthor=C.+A.+Dickeyauthor=M.+D.+Galignianaauthor=T.+Reinauthor=M.+B.+Cox&title=FKBP51+and+FKBP52+in+signaling+and+disease&doi=10.1016%2Fj.tem.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP51 and FKBP52 in signaling and disease</span></div><div class="casAuthors">Storer, Cheryl L.; Dickey, Chad A.; Galigniana, Mario D.; Rein, Theo; Cox, Marc B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">481-490</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  FKBP51 and FKBP52 are diverse regulators of steroid hormone receptor signaling, including receptor maturation, hormone binding and nuclear translocation.  Although structurally similar, they are functionally divergent, which is largely attributed to differences in the FK1 domain and the proline-rich loop.  FKBP51 and FKBP52 have emerged as likely contributors to a variety of hormone-dependent diseases, including stress-related diseases, immune function, reproductive functions and a variety of cancers.  In addn., recent studies have implicated FKBP51 and FKBP52 in Alzheimer's disease and other protein aggregation disorders.  This review summarizes our current understanding of FKBP51 and FKBP52 interactions within the receptor-chaperone complex, their contributions to health and disease, and their potential as therapeutic targets for the treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaeNToplbXEbVg90H21EOLACvtfcHk0lgycxFa0v0vVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKltbnI&md5=235f92790216db4db59d9ea8e9fd166e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DStorer%26aufirst%3DC.%2BL.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26aulast%3DGaligniana%26aufirst%3DM.%2BD.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DCox%26aufirst%3DM.%2BB.%26atitle%3DFKBP51%2520and%2520FKBP52%2520in%2520signaling%2520and%2520disease%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2011%26volume%3D22%26spage%3D481%26epage%3D490%26doi%3D10.1016%2Fj.tem.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">The prospect of FKBP51 as a drug target</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1351</span>– <span class="NLM_lpage">1359</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fcmdc.201200137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVKlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1351-1359&author=M.+V.+Schmidtauthor=M.+Paez-Peredaauthor=F.+Holsboerauthor=F.+Hausch&title=The+prospect+of+FKBP51+as+a+drug+target&doi=10.1002%2Fcmdc.201200137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Prospect of FKBP51 as a Drug Target</span></div><div class="casAuthors">Schmidt, Mathias V.; Paez-Pereda, Marcelo; Holsboer, Florian; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1351-1359</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The FK506 binding protein 51 (FKBP51) is best known as an Hsp90-assocd. co-chaperone that regulates the responsiveness of steroid hormone receptors.  In human genetic assocn. studies, FKBP51 has repeatedly been assocd. with emotion processing and numerous stress-related affective disorderst.  It has also been implicated in contributing to the glucocorticoid hyposensitivity obsd. in New World primates.  More recently, several research groups have consistently shown a protective effect of FKBP51 knockout or knockdown on stress endocrinol. and stress-coping behavior in animal models of depression and anxiety.  The principal druggability of FKBP51 is exemplified by the prototypic FKBP ligands FK506 and rapamycin.  Moreover, FKBP51 is highly suited for X-ray co-crystallog., which should facilitate the rational drug design of improved FKBP51 ligands.  In summary, FKBP51 has emerged as a promising new drug target for stress-related disorders that should be amenable to drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVM2m01bsqHbVg90H21EOLACvtfcHk0ljwv70RV7mdhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVKlsbo%253D&md5=12a5779bf7a0571429716e571f43f93f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200137%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DThe%2520prospect%2520of%2520FKBP51%2520as%2520a%2520drug%2520target%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D1351%26epage%3D1359%26doi%3D10.1002%2Fcmdc.201200137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyners, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">The Many Faces of FKBP51</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">35</span>, <span class="refDoi"> DOI: 10.3390/biom9010035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.3390%2Fbiom9010035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=35&author=A.+Hahleauthor=S.+Merzauthor=C.+Meynersauthor=F.+Hausch&title=The+Many+Faces+of+FKBP51&doi=10.3390%2Fbiom9010035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fbiom9010035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom9010035%26sid%3Dliteratum%253Aachs%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DMerz%26aufirst%3DS.%26aulast%3DMeyners%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DThe%2520Many%2520Faces%2520of%2520FKBP51%26jtitle%3DBiomolecules%26date%3D2019%26volume%3D9%26spage%3D35%26doi%3D10.3390%2Fbiom9010035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanowski, C. P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letizia Curzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubcakova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flachskamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, M.</span></span> <span> </span><span class="NLM_article-title">Deficiency of FK506-binding protein (FKBP) 51 alters sleep architecture and recovery sleep responses to stress in mice</span>. <i>J. Sleep Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1111/jsr.12112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Fjsr.12112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=24354785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC2c3nslCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=176-185&author=S.+Albuauthor=C.+P.+N.+Romanowskiauthor=M.+Letizia+Curziauthor=V.+Jakubcakovaauthor=C.+Flachskammauthor=N.+C.+Gassenauthor=J.+Hartmannauthor=M.+V.+Schmidtauthor=U.+Schmidtauthor=T.+Reinauthor=F.+Holsboerauthor=F.+Hauschauthor=M.+Paez-Peredaauthor=M.+Kimura&title=Deficiency+of+FK506-binding+protein+%28FKBP%29+51+alters+sleep+architecture+and+recovery+sleep+responses+to+stress+in+mice&doi=10.1111%2Fjsr.12112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of FK506-binding protein (FKBP) 51 alters sleep architecture and recovery sleep responses to stress in mice</span></div><div class="casAuthors">Albu Stefana; Romanowski Christoph P N; Letizia Curzi M; Jakubcakova Vladimira; Flachskamm Cornelia; Gassen Nils C; Hartmann Jakob; Schmidt Mathias V; Schmidt Ulrike; Rein Theo; Holsboer Florian; Hausch Felix; Paez-Pereda Marcelo; Kimura Mayumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of sleep research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">176-85</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FK506-binding protein 51 (FKBP51) is a co-chaperone of the glucocorticoid receptor, functionally linked to its activity via an ultra-short negative feedback loop.  Thus, FKBP51 plays an important regulatory role in the hypothalamic-pituitary-adrenocortical (HPA) axis necessary for stress adaptation and recovery.  Previous investigations illustrated that HPA functionality is influenced by polymorphisms in the gene encoding FKBP51, which are associated with both increased protein levels and depressive episodes.  Because FKBP51 is a key molecule in stress responses, we hypothesized that its deletion impacts sleep.  To study FKBP51-involved changes in sleep, polysomnograms of FKBP51 knockout (KO) mice and wild-type (WT) littermates were compared at baseline and in the recovery phase after 6-h sleep deprivation (SD) and 1-h restraint stress (RS).  Using another set of animals, the 24-h profiles of hippocampal free corticosterone levels were also determined.  The most dominant effect of FKBP51 deletion appeared as increased nocturnal wake, where the bout length was significantly extended while non-rapid eye movement sleep (NREMS) and rapid eye movement sleep were rather suppressed.  After both SD and RS, FKBP51KO mice exhibited less recovery or rebound sleep than WTs, although slow-wave activity during NREMS was higher in KOs, particularly after SD.  Sleep compositions of KOs were nearly opposite to sleep profiles observed in human depression.  This might result from lower levels of free corticosterone in FKBP51KO mice, confirming reduced HPA reactivity.  The results indicate that an FKBP51 deletion yields a pro-resilience sleep phenotype.  FKBP51 could therefore be a therapeutic target for stress-induced mood and sleep disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTraiwR6zEpya91pGlMMoT5fW6udTcc2eYuYcN-xxhJA7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3nslCgtQ%253D%253D&md5=4add85c5be5fc854830e49c6ce943357</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fjsr.12112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjsr.12112%26sid%3Dliteratum%253Aachs%26aulast%3DAlbu%26aufirst%3DS.%26aulast%3DRomanowski%26aufirst%3DC.%2BP.%2BN.%26aulast%3DLetizia%2BCurzi%26aufirst%3DM.%26aulast%3DJakubcakova%26aufirst%3DV.%26aulast%3DFlachskamm%26aufirst%3DC.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DM.%26atitle%3DDeficiency%2520of%2520FK506-binding%2520protein%2520%2528FKBP%2529%252051%2520alters%2520sleep%2520architecture%2520and%2520recovery%2520sleep%2520responses%2520to%2520stress%2520in%2520mice%26jtitle%3DJ.%2520Sleep%2520Res.%26date%3D2014%26volume%3D23%26spage%3D176%26epage%3D185%26doi%3D10.1111%2Fjsr.12112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharf, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">The involvement of FK506-binding protein 51 (FKBP5) in the behavioral and neuroendocrine effects of chronic social defeat stress</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2011.07.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.neuropharm.2011.07.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21839098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaktb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2012&pages=332-339&author=J.+Hartmannauthor=K.+V.+Wagnerauthor=C.+Lieblauthor=S.+H.+Scharfauthor=X.+D.+Wangauthor=M.+Wolfauthor=F.+Hauschauthor=T.+Reinauthor=U.+Schmidtauthor=C.+Toumaauthor=J.+Cheung-Flynnauthor=M.+B.+Coxauthor=D.+F.+Smithauthor=F.+Holsboerauthor=M.+B.+Mullerauthor=M.+V.+Schmidt&title=The+involvement+of+FK506-binding+protein+51+%28FKBP5%29+in+the+behavioral+and+neuroendocrine+effects+of+chronic+social+defeat+stress&doi=10.1016%2Fj.neuropharm.2011.07.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of FK506-binding protein 51 (FKBP5) in the behavioral and neuroendocrine effects of chronic social defeat stress</span></div><div class="casAuthors">Hartmann, Jakob; Wagner, Klaus V.; Liebl, Claudia; Scharf, Sebastian H.; Wang, Xiao-Dong; Wolf, Miriam; Hausch, Felix; Rein, Theo; Schmidt, Ulrike; Touma, Chadi; Cheung-Flynn, Joyce; Cox, Marc B.; Smith, David F.; Holsboer, Florian; Mueller, Marianne B.; Schmidt, Mathias V.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">332-339</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chronic stress is increasingly considered to be a main risk factor for the development of a variety of psychiatric diseases such as depression.  This is further supported by an impaired neg. feedback of the hypothalamic-pituitary-adrenal (HPA) axis, which has been obsd. in the majority of depressed patients.  The effects of glucocorticoids, the main hormonal endpoint of the HPA axis, are mediated via the glucocorticoid receptor (GR) and the mineralocorticoid receptor.  The FK506-binding protein 51 (FKBP5), a co-chaperone of the Hsp90 and component of the chaperone-receptor heterocomplex, has been shown to reduce ligand sensitivity of the GR.  This study aimed to investigate the function of FKBP5 as a possible mediator of the stress response system and its potential role in the development of stress-related diseases.  Therefore, we assessed whether mice lacking the gene encoding FKBP5 (51KO mice) were less vulnerable to the adverse effects of three weeks of chronic social defeat stress.  Mice were subsequently analyzed with regards to physiol., neuroendocrine, behavioral and mRNA expression alterations.  Our results show a less vulnerable phenotype of 51KO mice with respect to physiol. and neuroendocrine parameters compared to wild-type animals.  51KO mice demonstrated lower adrenal wts. and basal corticosterone levels, a diminished response to a novel acute stimulus and an enhanced recovery, as well as more active stress-coping behavior.  These results suggest an enhanced neg. glucocorticoid feedback within the HPA axis of 51KO mice, possibly modulated by an increased sensitivity of the GR.  This article is part of a Special Issue entitled Anxiety and Depression'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwLJ6JU_-_WbVg90H21EOLACvtfcHk0liDkeNpenUyZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaktb%252FN&md5=77d3cae9170127c4784377ecc2c332c0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2011.07.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2011.07.041%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DK.%2BV.%26aulast%3DLiebl%26aufirst%3DC.%26aulast%3DScharf%26aufirst%3DS.%2BH.%26aulast%3DWang%26aufirst%3DX.%2BD.%26aulast%3DWolf%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DSmith%26aufirst%3DD.%2BF.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DM.%2BB.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DThe%2520involvement%2520of%2520FK506-binding%2520protein%252051%2520%2528FKBP5%2529%2520in%2520the%2520behavioral%2520and%2520neuroendocrine%2520effects%2520of%2520chronic%2520social%2520defeat%2520stress%26jtitle%3DNeuropharmacology%26date%3D2012%26volume%3D62%26spage%3D332%26epage%3D339%26doi%3D10.1016%2Fj.neuropharm.2011.07.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bull, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ionescu, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebertz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depping, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ising, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span> <span> </span><span class="NLM_article-title">FK506 binding protein 5 shapes stress responsiveness: Modulation of neuroendocrine reactivity and coping behavior</span>. <i>Biol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">928</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1016/j.biopsych.2011.07.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.biopsych.2011.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21907973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2011&pages=928-936&author=C.+Toumaauthor=N.+C.+Gassenauthor=L.+Herrmannauthor=J.+Cheung-Flynnauthor=D.+R.+Bullauthor=I.+A.+Ionescuauthor=J.+M.+Heinzmannauthor=A.+Knapmanauthor=A.+Siebertzauthor=A.+M.+Deppingauthor=J.+Hartmannauthor=F.+Hauschauthor=M.+V.+Schmidtauthor=F.+Holsboerauthor=M.+Isingauthor=M.+B.+Coxauthor=U.+Schmidtauthor=T.+Rein&title=FK506+binding+protein+5+shapes+stress+responsiveness%3A+Modulation+of+neuroendocrine+reactivity+and+coping+behavior&doi=10.1016%2Fj.biopsych.2011.07.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">FK506 Binding Protein 5 Shapes Stress Responsiveness: Modulation of Neuroendocrine Reactivity and Coping Behavior</span></div><div class="casAuthors">Touma, Chadi; Gassen, Nils Christian; Herrmann, Leonie; Cheung-Flynn, Joyce; Buell, Dominik R.; Ionescu, Irina A.; Heinzmann, Jan-Michael; Knapman, Alana; Siebertz, Anna; Depping, Anna-Mareike; Hartmann, Jakob; Hausch, Felix; Schmidt, Mathias V.; Holsboer, Florian; Ising, Marcus; Cox, Marc B.; Schmidt, Ulrike; Rein, Theo</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">928-936</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Background: The Hsp90 cochaperone FK506 binding protein 5 (FKBP5) is an established regulator of the glucocorticoid receptor (GR), and numerous genetic studies have linked it to stress-related diseases such as major depression or posttraumatic stress disorder.  However, translational studies including genetic animal models are lacking.  Methods: Mice deficient of FKBP5 were generated and analyzed in comparison with wildtype littermates.  They were subjected to several test paradigms characterizing their emotionality, stress reactivity, and coping behavior as well as hypothalamus-pituitary-adrenal axis function and regulation.  Moreover, protein expression of GR and FKBP5 was detd. in different brain structures 8 days after stress exposure.  The combined dexamethasone/corticotropin-releasing hormone test was performed both in mice and healthy human subjects of different FKBP5 genotypes.  The GR function was evaluated by reporter gene assays.  Results: Under basal conditions, deletion of FKBP5 did not change exploratory drive, locomotor activity, anxiety-related behavior, stress-coping, or depression-like behavior.  After exposure to different acute stressors of sufficient intensity, however, it led to a more active coping behavior.  Moreover, loss of FKBP5 decreased hypothalamus-pituitary-adrenal axis reactivity and GR expression changes in response to stressors.  In mice and humans, the FKBP5 genotype also detd. the outcome of the dexamethasone/corticotropin-releasing hormone test.  Conclusions: This study in mice and humans presents FKBP5 as a decisive factor for the physiol. stress response, shaping neuroendocrine reactivity as well as coping behavior.  This lends strong support to the concept emerging from human studies of FKBP5 as important factor governing gene-environment interactions relevant for the etiol. of affective disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreFnJ8h2ZzGLVg90H21EOLACvtfcHk0lheMFCdDCbQ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOlsr7I&md5=22d66ce41dc19d44a7f795eb0f2780a6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2011.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2011.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DHerrmann%26aufirst%3DL.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DBull%26aufirst%3DD.%2BR.%26aulast%3DIonescu%26aufirst%3DI.%2BA.%26aulast%3DHeinzmann%26aufirst%3DJ.%2BM.%26aulast%3DKnapman%26aufirst%3DA.%26aulast%3DSiebertz%26aufirst%3DA.%26aulast%3DDepping%26aufirst%3DA.%2BM.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DIsing%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DRein%26aufirst%3DT.%26atitle%3DFK506%2520binding%2520protein%25205%2520shapes%2520stress%2520responsiveness%253A%2520Modulation%2520of%2520neuroendocrine%2520reactivity%2520and%2520coping%2520behavior%26jtitle%3DBiol.%2520Psychiatry%26date%3D2011%26volume%3D70%26spage%3D928%26epage%3D936%26doi%3D10.1016%2Fj.biopsych.2011.07.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, J. C.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Erausquin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeber, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinwal, U. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span> <span> </span><span class="NLM_article-title">A new anti-depressive strategy for the elderly: ablation of FKBP5/FKBP51</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e24840</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0024840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1371%2Fjournal.pone.0024840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21935478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1yrurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=J.+C.+O%E2%80%99Learyauthor=S.+Dhariaauthor=L.+J.+Blairauthor=S.+Bradyauthor=A.+G.+Johnsonauthor=M.+Petersauthor=J.+Cheung-Flynnauthor=M.+B.+Coxauthor=G.+de+Erausquinauthor=E.+J.+Weeberauthor=U.+K.+Jinwalauthor=C.+A.+Dickey&title=A+new+anti-depressive+strategy+for+the+elderly%3A+ablation+of+FKBP5%2FFKBP51&doi=10.1371%2Fjournal.pone.0024840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A new anti-depressive strategy for the elderly: ablation of FKBP5/FKBP51</span></div><div class="casAuthors">O'Leary, John C., III; Dharia, Sheetal; Blair, Laura J.; Brady, Sarah; Johnson, Amelia G.; Peters, Melinda; Cheung-Flynn, Joyce; Cox, Marc B.; de Erausquin, Gabriel; Weeber, Edwin J.; Jinwal, Umesh K.; Dickey, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24840</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The gene FKBP5 codes for FKBP51, a co-chaperone protein of the Hsp90 complex that increases with age.  Through its assocn. with Hsp90, FKBP51 regulates the glucocorticoid receptor (GR).  Single nucleotide polymorphisms (SNPs) in the FKBP5 gene assoc. with increased recurrence of depressive episodes, increased susceptibility to post-traumatic stress disorder, bipolar disorder, attempt of suicide, and major depressive disorder in HIV patients.  Variation in one of these SNPs correlates with increased levels of FKBP51.  FKBP51 is also increased in HIV patients.  Moreover, increases in FKBP51 in the amygdala produce an anxiety phenotype in mice.  Therefore, we tested the behavioral consequences of FKBP5 deletion in aged mice.  Similar to that of naive animals treated with classical antidepressants FKBP5-/- mice showed antidepressant behavior without affecting cognition and other basic motor functions.  Reduced corticosterone levels following stress accompanied these obsd. effects on depression.  Age-dependent anxiety was also modulated by FKBP5 deletion.  Therefore, drug discovery efforts focused on depleting FKBP51 levels may yield novel antidepressant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpeKgbMac6-7Vg90H21EOLACvtfcHk0lheMFCdDCbQ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1yrurrF&md5=4301f9433a35bebc37dd6cdcea407793</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024840%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DJ.%2BC.%26aulast%3DDharia%26aufirst%3DS.%26aulast%3DBlair%26aufirst%3DL.%2BJ.%26aulast%3DBrady%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DA.%2BG.%26aulast%3DPeters%26aufirst%3DM.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3Dde%2BErausquin%26aufirst%3DG.%26aulast%3DWeeber%26aufirst%3DE.%2BJ.%26aulast%3DJinwal%26aufirst%3DU.%2BK.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520anti-depressive%2520strategy%2520for%2520the%2520elderly%253A%2520ablation%2520of%2520FKBP5%252FFKBP51%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0024840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matosin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halldorsdottir, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Molecular Mechanisms Underpinning Gene by Environment Interactions in Psychiatric Disorders: The FKBP5Model</span>. <i>Biol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.biopsych.2018.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.biopsych.2018.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29573791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlslKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=821-830&author=N.+Matosinauthor=T.+Halldorsdottirauthor=E.+B.+Binder&title=Understanding+the+Molecular+Mechanisms+Underpinning+Gene+by+Environment+Interactions+in+Psychiatric+Disorders%3A+The+FKBP5Model&doi=10.1016%2Fj.biopsych.2018.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Molecular Mechanisms Underpinning Gene by Environment Interactions in Psychiatric Disorders: The FKBP5 Model</span></div><div class="casAuthors">Matosin, Natalie; Halldorsdottir, Thorhildur; Binder, Elisabeth B.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">821-830</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Epidemiol. and genetic studies suggest common environmental and genetic risk factors for a no. of psychiatric disorders, including depression, bipolar disorder, and schizophrenia.  Genetic and environmental factors, esp. adverse life events, not only have main effects on disease development but also may interact to shape risk and resilience.  Such gene by adversity interactions have been described for FKBP5, an endogenous regulator of the stress-neuroendocrine system, conferring risk for a no. of psychiatric disorders.  In this review, we present a mol. and cellular model of the consequences of FKBP5 by early adversity interactions.  We illustrate how altered genetic and epigenetic regulation of FKBP5 may contribute to disease risk by covering evidence from clin. and preclin. studies of FKBP5 dysregulation, known cell-type and tissue-type expression patterns of FKBP5 in humans and animals, and the role of FKBP5 as a stress-responsive mol. hub modulating many cellular pathways.  FKBP5 presents the possibility to better understand the mol. and cellular factors contributing to a disease-relevant gene by environment interaction, with implications for the development of biomarkers and interventions for psychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS27sgZbIsLLVg90H21EOLACvtfcHk0liIiBQ_QoW_IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlslKisLo%253D&md5=5618792a3d10befce96db810065d44a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2018.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2018.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DMatosin%26aufirst%3DN.%26aulast%3DHalldorsdottir%26aufirst%3DT.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26atitle%3DUnderstanding%2520the%2520Molecular%2520Mechanisms%2520Underpinning%2520Gene%2520by%2520Environment%2520Interactions%2520in%2520Psychiatric%2520Disorders%253A%2520The%2520FKBP5Model%26jtitle%3DBiol.%2520Psychiatry%26date%3D2018%26volume%3D83%26spage%3D821%26epage%3D830%26doi%3D10.1016%2Fj.biopsych.2018.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lesiak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumaier, J. F.</span></span> <span> </span><span class="NLM_article-title">Sequencing the serotonergic neuron translatome reveals a new role for Fkbp5 in stress</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1038/s41380-020-0750-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41380-020-0750-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32366949" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+J.+Lesiakauthor=K.+Coffeyauthor=J.+H.+Cohenauthor=K.+J.+Liangauthor=C.+Chavkinauthor=J.+F.+Neumaier&title=Sequencing+the+serotonergic+neuron+translatome+reveals+a+new+role+for+Fkbp5+in+stress&doi=10.1038%2Fs41380-020-0750-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41380-020-0750-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41380-020-0750-4%26sid%3Dliteratum%253Aachs%26aulast%3DLesiak%26aufirst%3DA.%2BJ.%26aulast%3DCoffey%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DJ.%2BH.%26aulast%3DLiang%26aufirst%3DK.%2BJ.%26aulast%3DChavkin%26aufirst%3DC.%26aulast%3DNeumaier%26aufirst%3DJ.%2BF.%26atitle%3DSequencing%2520the%2520serotonergic%2520neuron%2520translatome%2520reveals%2520a%2520new%2520role%2520for%2520Fkbp5%2520in%2520stress%26jtitle%3DMol.%2520Psychiatry%26date%3D2020%26doi%3D10.1038%2Fs41380-020-0750-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palming, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenback, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidibeh, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">FKBP5 expression in human adipose tissue increases following dexamethasone exposure and is associated with insulin resistance</span>. <i>Metab., Clin. Exp.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1198</span>– <span class="NLM_lpage">1208</span>, <span class="refDoi"> DOI: 10.1016/j.metabol.2014.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.metabol.2014.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=24997500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=1198-1208&author=M.+J.+Pereiraauthor=J.+Palmingauthor=M.+K.+Svenssonauthor=M.+Rizellauthor=J.+Dalenbackauthor=M.+Hammarauthor=T.+Fallauthor=C.+O.+Sidibehauthor=P.+A.+Svenssonauthor=J.+W.+Eriksson&title=FKBP5+expression+in+human+adipose+tissue+increases+following+dexamethasone+exposure+and+is+associated+with+insulin+resistance&doi=10.1016%2Fj.metabol.2014.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP5 expression in human adipose tissue increases following dexamethasone exposure and is associated with insulin resistance</span></div><div class="casAuthors">Pereira, Maria J.; Palming, Jenny; Svensson, Maria K.; Rizell, Magnus; Dalenbaeck, Jan; Hammar, Maarten; Fall, Tove; Sidibeh, Cherno O.; Svensson, Per-Arne; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Metabolism, Clinical and Experimental</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1198-1208</span>CODEN:
                <span class="NLM_cas:coden">METAAJ</span>;
        ISSN:<span class="NLM_cas:issn">0026-0495</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">To study effects of dexamethasone on gene expression in human adipose tissue aiming to identify potential novel mechanisms for glucocorticoid-induced insulin resistance.S.c. and omental adipose tissue, obtained from non-diabetic donors (10 M/15 F; age: 28-60 years; BMI: 20.7-30.6 kg/m2), was incubated with or without dexamethasone (0.003-3 μmol/L) for 24 h.  Gene expression was assessed by microarray and real time-PCR and protein expression by immunoblotting.FKBP5 (FK506-binding protein 5) and CNR1 (cannabinoid receptor 1) were the most responsive genes to dexamethasone in both s.c. and omental adipose tissue (∼ 7-fold).  Dexamethasone increased FKBP5 gene and protein expression in a dose-dependent manner in both depots.  The gene product, FKBP51 protein, was 10-fold higher in the omental than in the s.c. depot, whereas the mRNA levels were similar.  Higher FKBP5 gene expression in omental adipose tissue was assocd. with reduced insulin effects on glucose uptake in both depots.  Furthermore, FKBP5 gene expression in s.c. adipose tissue was pos. correlated with serum insulin, HOMA-IR and s.c. adipocyte diam. and neg. with plasma HDL-cholesterol.  FKBP5 SNPs were found to be assocd. with type 2 diabetes and diabetes-related phenotypes in large population-based samples.  Dexamethasone exposure promotes expression of FKBP5 in adipose tissue, a gene that may be implicated in glucocorticoid-induced insulin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwQbo-ymUm3rVg90H21EOLACvtfcHk0lgZr06pTnUnAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsrvL&md5=d31678400f5b1e2b972bd684ea6dfed5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.metabol.2014.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.metabol.2014.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DPalming%26aufirst%3DJ.%26aulast%3DSvensson%26aufirst%3DM.%2BK.%26aulast%3DRizell%26aufirst%3DM.%26aulast%3DDalenback%26aufirst%3DJ.%26aulast%3DHammar%26aufirst%3DM.%26aulast%3DFall%26aufirst%3DT.%26aulast%3DSidibeh%26aufirst%3DC.%2BO.%26aulast%3DSvensson%26aufirst%3DP.%2BA.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DFKBP5%2520expression%2520in%2520human%2520adipose%2520tissue%2520increases%2520following%2520dexamethasone%2520exposure%2520and%2520is%2520associated%2520with%2520insulin%2520resistance%26jtitle%3DMetab.%252C%2520Clin.%2520Exp.%26date%3D2014%26volume%3D63%26spage%3D1198%26epage%3D1208%26doi%3D10.1016%2Fj.metabol.2014.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidibeh, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abalo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">G, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundkvist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsogiannos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillejo-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">FKBP5 expression in human adipose tissue: potential role in glucose and lipid metabolism, adipogenesis and type 2 diabetes</span>. <i>Endocrine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1007/s12020-018-1674-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1007%2Fs12020-018-1674-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30032404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKhsbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=116-128&author=C.+O.+Sidibehauthor=M.+J.+Pereiraauthor=X.+M.+Abaloauthor=J.+B.+Gauthor=S.+Skrticauthor=P.+Lundkvistauthor=P.+Katsogiannosauthor=F.+Hauschauthor=C.+Castillejo-Lopezauthor=J.+W.+Eriksson&title=FKBP5+expression+in+human+adipose+tissue%3A+potential+role+in+glucose+and+lipid+metabolism%2C+adipogenesis+and+type+2+diabetes&doi=10.1007%2Fs12020-018-1674-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP5 expression in human adipose tissue: potential role in glucose and lipid metabolism, adipogenesis and type 2 diabetes</span></div><div class="casAuthors">Sidibeh, Cherno O.; Pereira, Maria J.; Abalo, Xesus M.; J. Boersma, Gretha; Skrtic, Stanko; Lundkvist, Per; Katsogiannos, Petros; Hausch, Felix; Castillejo-Lopez, Casimiro; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-128</span>CODEN:
                <span class="NLM_cas:coden">EOCRE5</span>;
        ISSN:<span class="NLM_cas:issn">1355-008X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Here, we explore the involvement of FKBP51 in glucocorticoid-induced insulin resistance (IR) in human s.c. adipose tissue (SAT), including its potential role in type 2 diabetes (T2D).  Moreover, we assess the metabolic effects of reducing the activity of FKBP51 using the specific inhibitor SAFit1.  Methods: Human SAT was obtained by needle biopsies of the lower abdominal region.  FKBP5 gene expression was assessed in fresh SAT explants from a cohort of 20 T2D subjects group-wise matched by gender, age and BMI to 20 non-diabetic subjects.  In addn., human SAT was obtained from non-diabetic volunteers (20F/9M).  SAT was incubated for 24 h with or without the synthetic glucocorticoid dexamethasone and SAFit1.  Incubated SAT was used to measure the glucose uptake rate in isolated adipocytes.  Results: FKBP5 gene expression levels in SAT pos. correlated with several indexes of IR as well as glucose area under the curve during oral glucose tolerance test (r = 0.33, p < 0.05).  FKBP5 gene expression levels tended to be higher in T2D subjects compared to non-diabetic subjects (p = 0.088).  Moreover, FKBP5 gene expression levels were found to inversely correlate with lipolytic, lipogenic and adipogenic genes.  SAFit1 partly prevented the inhibitory effects of dexamethasone on glucose uptake.  Conclusions: FKBP5 gene expression in human SAT tends to be increased in T2D subjects and is related to elevated glucose levels.  Moreover, FKBP5 gene expression is inversely assocd. with the expression of lipolytic, lipogenic and adipogenic genes.  SAFit1 can partly prevent glucose uptake impairment by glucocorticoids, suggesting that FKBP51 might be a key factor in glucocorticoid-induced IR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZDH980jvIObVg90H21EOLACvtfcHk0lgZr06pTnUnAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKhsbrI&md5=4d3cebb9d489064e35e6b2c97fda2c45</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs12020-018-1674-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12020-018-1674-5%26sid%3Dliteratum%253Aachs%26aulast%3DSidibeh%26aufirst%3DC.%2BO.%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DAbalo%26aufirst%3DX.%2BM.%26aulast%3DG%26aufirst%3DJ.%2BB.%26aulast%3DSkrtic%26aufirst%3DS.%26aulast%3DLundkvist%26aufirst%3DP.%26aulast%3DKatsogiannos%26aufirst%3DP.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DCastillejo-Lopez%26aufirst%3DC.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DFKBP5%2520expression%2520in%2520human%2520adipose%2520tissue%253A%2520potential%2520role%2520in%2520glucose%2520and%2520lipid%2520metabolism%252C%2520adipogenesis%2520and%2520type%25202%2520diabetes%26jtitle%3DEndocrine%26date%3D2018%26volume%3D62%26spage%3D116%26epage%3D128%26doi%3D10.1007%2Fs12020-018-1674-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamihalev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dournes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uribe-Marino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labermaier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodoropoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschop, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Stress-responsive FKBP51 regulates AKT2-AS160 signaling and metabolic function</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1725</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01783-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41467-017-01783-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29170369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ksVKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1725&author=G.+Balsevichauthor=A.+S.+Hauslauthor=C.+W.+Meyerauthor=S.+Karamihalevauthor=X.+Fengauthor=M.+L.+Pohlmannauthor=C.+Dournesauthor=A.+Uribe-Marinoauthor=S.+Santarelliauthor=C.+Labermaierauthor=K.+Hafnerauthor=T.+Maoauthor=M.+Breitsamerauthor=M.+Theodoropoulouauthor=C.+Namendorfauthor=M.+Uhrauthor=M.+Paez-Peredaauthor=G.+Winterauthor=F.+Hauschauthor=A.+Chenauthor=M.+H.+Tschopauthor=T.+Reinauthor=N.+C.+Gassenauthor=M.+V.+Schmidt&title=Stress-responsive+FKBP51+regulates+AKT2-AS160+signaling+and+metabolic+function&doi=10.1038%2Fs41467-017-01783-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Stress-responsive FKBP51 regulates AKT2-AS160 signaling and metabolic function</span></div><div class="casAuthors">Balsevich Georgia; Hausl Alexander S; Karamihalev Stoyo; Pohlmann Max L; Dournes Carine; Uribe-Marino Andres; Santarelli Sara; Labermaier Christiana; Chen Alon; Schmidt Mathias V; Meyer Carola W; Tschop Matthias H; Feng Xixi; Hafner Kathrin; Mao Tianqi; Theodoropoulou Marily; Namendorf Christian; Uhr Manfred; Paez-Pereda Marcelo; Rein Theo; Gassen Nils C; Breitsamer Michaela; Winter Gerhard; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1725</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The co-chaperone FKBP5 is a stress-responsive protein-regulating stress reactivity, and its genetic variants are associated with T2D related traits and other stress-related disorders.  Here we show that FKBP51 plays a role in energy and glucose homeostasis.  Fkbp5 knockout (51KO) mice are protected from high-fat diet-induced weight gain, show improved glucose tolerance and increased insulin signaling in skeletal muscle.  Chronic treatment with a novel FKBP51 antagonist, SAFit2, recapitulates the effects of FKBP51 deletion on both body weight regulation and glucose tolerance.  Using shorter SAFit2 treatment, we show that glucose tolerance improvement precedes the reduction in body weight.  Mechanistically, we identify a novel association between FKBP51 and AS160, a substrate of AKT2 that is involved in glucose uptake.  FKBP51 antagonism increases the phosphorylation of AS160, increases glucose transporter 4 expression at the plasma membrane, and ultimately enhances glucose uptake in skeletal myotubes.  We propose FKBP51 as a mediator between stress and T2D development, and potential target for therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgr0bCoZaOtk2yPDyEFiigfW6udTcc2ebuWDLBC0EqTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ksVKktw%253D%253D&md5=f58c64e386d4662cd24bc4dcb1739f2d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01783-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01783-y%26sid%3Dliteratum%253Aachs%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BW.%26aulast%3DKaramihalev%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DPohlmann%26aufirst%3DM.%2BL.%26aulast%3DDournes%26aufirst%3DC.%26aulast%3DUribe-Marino%26aufirst%3DA.%26aulast%3DSantarelli%26aufirst%3DS.%26aulast%3DLabermaier%26aufirst%3DC.%26aulast%3DHafner%26aufirst%3DK.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DTheodoropoulou%26aufirst%3DM.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DTschop%26aufirst%3DM.%2BH.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DStress-responsive%2520FKBP51%2520regulates%2520AKT2-AS160%2520signaling%2520and%2520metabolic%2520function%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1725%26doi%3D10.1038%2Fs41467-017-01783-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stechschulte, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinds, T. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuder, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najjar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecka-Czernik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, E. R.</span></span> <span> </span><span class="NLM_article-title">FKBP51 null mice are resistant to diet-induced obesity and the PPARgamma agonist rosiglitazone</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">3888</span>– <span class="NLM_lpage">3900</span>, <span class="refDoi"> DOI: 10.1210/en.2015-1996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1210%2Fen.2015-1996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=27442117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVOlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=3888-3900&author=L.+A.+Stechschulteauthor=B.+Qiuauthor=M.+Warrierauthor=T.+D.+Hindsauthor=M.+Zhangauthor=H.+Guauthor=Y.+Xuauthor=S.+S.+Khuderauthor=L.+Russoauthor=S.+M.+Najjarauthor=B.+Lecka-Czernikauthor=W.+Yongauthor=E.+R.+Sanchez&title=FKBP51+null+mice+are+resistant+to+diet-induced+obesity+and+the+PPARgamma+agonist+rosiglitazone&doi=10.1210%2Fen.2015-1996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP51 null mice are resistant to diet-induced obesity and the PPARγ agonist rosiglitazone</span></div><div class="casAuthors">Stechschulte, Lance A.; Qiu, Bin; Warrier, Manya; Hinds, Terry D., Jr.; Zhang, Man; Gu, Hao; Xu, Yuxue; Khuder, Saja S.; Russo, Lucia; Najjar, Sonia M.; Lecka-Czernik, Beata; Yong, Weidong; Sanchez, Edwin R.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3888-3900</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">1945-7170</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">FK506-binding protein-51 (FKBP51) is a mol. cochaperone recently shown to be a pos. regulator of peroxisome proliferator-activated receptor (PPAR)γ, the master regulator of adipocyte differentiation and function.  In cellular models of adipogenesis, loss of FKBP51 not only reduced PPARγ activity but also reduced lipid accumulation, suggesting that FKBP51 knock-out (KO) mice might have insufficient development of adipose tissue and lipid storage ability.  This model was tested by examg. wild-type (WT) and FKBP51-KO mice under regular and high-fat diet conditions.  Under both diets, FKBP51-KO mice were resistant to wt. gain, hepatic steatosis, and had greatly reduced white adipose tissue (WAT) but higher amts. of brown adipose tissue.  Under high-fat diet, KOmice were highly resistant to adiposity and exhibited reduced plasma lipids and elevated glucose and insulin tolerance.  Profiling of perigonadal and scWATrevealed elevated expression of brown adipose tissue lineage genes in KO mice that correlated increased energy expenditure and a shift of substrate oxidn. to carbohydrates, as measured by indirect calorimetry.  To directly test PPARγ involvement, WT and KO mice were fed rosiglitazone agonist.  In WT mice, rosiglitazone induced whole-body wt. gain, increased WAT mass, a shift of substrate oxidn. to lipids, and elevated expression of PPARγ-regulated lipogenic genes in WAT.  In contrast, KO mice had reduced rosiglitazone responses for these parameters.  Our results identify FKBP51 as an important regulator of PPARγ in WAT and as a potential new target in the treatment of obesity and diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsfyxK2ww2rVg90H21EOLACvtfcHk0lii5Wso2KQDng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVOlsrk%253D&md5=67f7c6da620d8ee9a4df0207ddcaaa3d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1210%2Fen.2015-1996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2015-1996%26sid%3Dliteratum%253Aachs%26aulast%3DStechschulte%26aufirst%3DL.%2BA.%26aulast%3DQiu%26aufirst%3DB.%26aulast%3DWarrier%26aufirst%3DM.%26aulast%3DHinds%26aufirst%3DT.%2BD.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKhuder%26aufirst%3DS.%2BS.%26aulast%3DRusso%26aufirst%3DL.%26aulast%3DNajjar%26aufirst%3DS.%2BM.%26aulast%3DLecka-Czernik%26aufirst%3DB.%26aulast%3DYong%26aufirst%3DW.%26aulast%3DSanchez%26aufirst%3DE.%2BR.%26atitle%3DFKBP51%2520null%2520mice%2520are%2520resistant%2520to%2520diet-induced%2520obesity%2520and%2520the%2520PPARgamma%2520agonist%2520rosiglitazone%26jtitle%3DEndocrinology%26date%3D2016%26volume%3D157%26spage%3D3888%26epage%3D3900%26doi%3D10.1210%2Fen.2015-1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamble, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hetty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vranic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almby, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillejo-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abalo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">Proof-of-concept for CRISPR/Cas9 gene editing in human preadipocytes: Deletion of FKBP5 and PPARG and effects on adipocyte differentiation and metabolism</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10565</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-67293-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41598-020-67293-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32601291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlWgu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=10565&author=P.+G.+Kambleauthor=S.+Hettyauthor=M.+Vranicauthor=K.+Almbyauthor=C.+Castillejo-Lopezauthor=X.+M.+Abaloauthor=M.+J.+Pereiraauthor=J.+W.+Eriksson&title=Proof-of-concept+for+CRISPR%2FCas9+gene+editing+in+human+preadipocytes%3A+Deletion+of+FKBP5+and+PPARG+and+effects+on+adipocyte+differentiation+and+metabolism&doi=10.1038%2Fs41598-020-67293-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proof-of-concept for CRISPR/Cas9 gene editing in human preadipocytes: Deletion of FKBP5 and PPARG and effects on adipocyte differentiation and metabolism</span></div><div class="casAuthors">Kamble, Prasad G.; Hetty, Susanne; Vranic, Milica; Almby, Kristina; Castillejo-Lopez, Casimiro; Abalo, Xesus M.; Pereira, Maria J.; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10565</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: CRISPR/Cas9 has revolutionized the genome-editing field.  So far, successful application in human adipose tissue has not been convincingly shown.  We present a method for gene knockout using electroporation in preadipocytes from human adipose tissue that achieved at least 90% efficiency without any need for selection of edited cells or clonal isolation.  We knocked out the FKBP5 and PPARG genes in preadipocytes and studied the resulting phenotypes.  PPARG knockout prevented differentiation into adipocytes.  Conversely, deletion of FKBP51, the protein coded by the FKBP5 gene, did not affect adipogenesis.  Instead, it markedly modulated glucocorticoid effects on adipocyte glucose metab. and, furthermore, we show some evidence of altered transcriptional activity of glucocorticoid receptors.  This has potential implications for the development of insulin resistance and type 2 diabetes.  The reported method is simple, easy to adapt, and enables the use of human primary preadipocytes instead of animal adipose cell models to assess the role of key genes and their products in adipose tissue development, metab. and pathobiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGpZ2rNrPUbbVg90H21EOLACvtfcHk0lhmahfaST1mow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlWgu7rO&md5=830e261629f4837312f665035563a241</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-67293-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-67293-y%26sid%3Dliteratum%253Aachs%26aulast%3DKamble%26aufirst%3DP.%2BG.%26aulast%3DHetty%26aufirst%3DS.%26aulast%3DVranic%26aufirst%3DM.%26aulast%3DAlmby%26aufirst%3DK.%26aulast%3DCastillejo-Lopez%26aufirst%3DC.%26aulast%3DAbalo%26aufirst%3DX.%2BM.%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DProof-of-concept%2520for%2520CRISPR%252FCas9%2520gene%2520editing%2520in%2520human%2520preadipocytes%253A%2520Deletion%2520of%2520FKBP5%2520and%2520PPARG%2520and%2520effects%2520on%2520adipocyte%2520differentiation%2520and%2520metabolism%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D10565%26doi%3D10.1038%2Fs41598-020-67293-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiaru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tochiki, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geranton, S. M.</span></span> <span> </span><span class="NLM_article-title">The stress regulator FKBP51 drives chronic pain by modulating spinal glucocorticoid signaling</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">325ra319</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aab3376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1126%2Fscitranslmed.aab3376" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=325ra319&author=M.+Maiaruauthor=K.+K.+Tochikiauthor=M.+B.+Coxauthor=L.+V.+Annanauthor=C.+G.+Bellauthor=X.+Fengauthor=F.+Hauschauthor=S.+M.+Geranton&title=The+stress+regulator+FKBP51+drives+chronic+pain+by+modulating+spinal+glucocorticoid+signaling&doi=10.1126%2Fscitranslmed.aab3376"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab3376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab3376%26sid%3Dliteratum%253Aachs%26aulast%3DMaiaru%26aufirst%3DM.%26aulast%3DTochiki%26aufirst%3DK.%2BK.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DAnnan%26aufirst%3DL.%2BV.%26aulast%3DBell%26aufirst%3DC.%2BG.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DGeranton%26aufirst%3DS.%2BM.%26atitle%3DThe%2520stress%2520regulator%2520FKBP51%2520drives%2520chronic%2520pain%2520by%2520modulating%2520spinal%2520glucocorticoid%2520signaling%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26spage%3D325ra319%26doi%3D10.1126%2Fscitranslmed.aab3376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiaru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geranton, S. M.</span></span> <span> </span><span class="NLM_article-title">The stress regulator FKBP51: a novel and promising druggable target for the treatment of persistent pain states across sexes</span>. <i>Pain</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">1224</span>– <span class="NLM_lpage">1234</span>, <span class="refDoi"> DOI: 10.1097/j.pain.0000000000001204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1097%2Fj.pain.0000000000001204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29533388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGgs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2018&pages=1224-1234&author=M.+Maiaruauthor=O.+B.+Morganauthor=T.+Maoauthor=M.+Breitsamerauthor=H.+Bamberauthor=M.+Pohlmannauthor=M.+V.+Schmidtauthor=G.+Winterauthor=F.+Hauschauthor=S.+M.+Geranton&title=The+stress+regulator+FKBP51%3A+a+novel+and+promising+druggable+target+for+the+treatment+of+persistent+pain+states+across+sexes&doi=10.1097%2Fj.pain.0000000000001204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The stress regulator FKBP51: a novel and promising druggable target for the treatment of persistent pain states across sexes</span></div><div class="casAuthors">Maiaru, Maria; Morgan, Oakley B.; Mao, Tianqi; Breitsamer, Michaela; Bamber, Harry; Poehlmann, Max; Schmidt, Mathias V.; Winter, Gerhard; Hausch, Felix; Geranton, Sandrine M.</div><div class="citationInfo"><span class="NLM_cas:title">Pain</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1224-1234</span>CODEN:
                <span class="NLM_cas:coden">PAINDB</span>;
        ISSN:<span class="NLM_cas:issn">0304-3959</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">It is well established that FKBP51 regulates the stress system by modulating the sensitivity of the glucocorticoid receptor to stress hormones.  Recently, we have demonstrated that FKBP51 also drives long-term inflammatory pain states in male mice by modulating glucocorticoid signalling at spinal cord level.  Here, we explored the potential of FKBP51 as a new pharmacol. target for the treatment of persistent pain across the sexes.  First, we demonstrated that FKBP51 regulates long-term pain states of different etiologies independently of sex.  Deletion of FKBP51 reduced the mech. hypersensitivity seen in joint inflammatory and neuropathic pain states in female and male mice.  Furthermore, FKBP51 deletion also reduced the hypersensitivity seen in a translational model of chemotherapy-induced pain.  Interestingly, these 3 pain states were assocd. with changes in glucocorticoid signalling, as indicated by the increased expression, at spinal cord level, of the glucocorticoid receptor isoform assocd. with glucocorticoid resistance, GRβ, and increased levels of plasma corticosterone.  These pain states were also accompanied by an upregulation of interleukin-6 in the spinal cord.  Crucially, we were able to pharmacol. reduce the severity of the mech. hypersensitivity seen in these 3 models of persistent pain with the unique FKBP51 ligand SAFit2.  When SAFit2 was combined with a state-of-the-art vesicular phospholipid gel formulation for slow release, a single injection of SAFit2 offered pain relief for at least 7 days.  We therefore propose the pharmacol. blockade of FKBP51 as a new approach for the treatment of persistent pain across sexes, likely in humans as well as rodents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXPZQxtP6trVg90H21EOLACvtfcHk0lhfF0dLWWHq4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGgs7vE&md5=cce2fddf3d07d4089e42caac61d8abfd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1097%2Fj.pain.0000000000001204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fj.pain.0000000000001204%26sid%3Dliteratum%253Aachs%26aulast%3DMaiaru%26aufirst%3DM.%26aulast%3DMorgan%26aufirst%3DO.%2BB.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DBamber%26aufirst%3DH.%26aulast%3DPohlmann%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DGeranton%26aufirst%3DS.%2BM.%26atitle%3DThe%2520stress%2520regulator%2520FKBP51%253A%2520a%2520novel%2520and%2520promising%2520druggable%2520target%2520for%2520the%2520treatment%2520of%2520persistent%2520pain%2520states%2520across%2520sexes%26jtitle%3DPain%26date%3D2018%26volume%3D159%26spage%3D1224%26epage%3D1234%26doi%3D10.1097%2Fj.pain.0000000000001204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linnstaedt, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riker, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckeis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchko, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendry, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domeier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laederach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, S. A.</span></span> <span> </span><span class="NLM_article-title">A Functional riboSNitch in the 3′ Untranslated Region of FKBP5 Alters MicroRNA-320a Binding Efficiency and Mediates Vulnerability to Chronic Post-Traumatic Pain</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8407</span>– <span class="NLM_lpage">8420</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.3458-17.2018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1523%2FJNEUROSCI.3458-17.2018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30150364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1ymur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=8407-8420&author=S.+D.+Linnstaedtauthor=K.+D.+Rikerauthor=C.+A.+Rueckeisauthor=K.+M.+Kutchkoauthor=L.+Lackeyauthor=K.+R.+McCarthyauthor=Y.+H.+Tsaiauthor=J.+S.+Parkerauthor=M.+C.+Kurzauthor=P.+L.+Hendryauthor=C.+Lewandowskiauthor=E.+Datnerauthor=C.+Pearsonauthor=B.+O%E2%80%99Neilauthor=R.+Domeierauthor=S.+Kaushikauthor=A.+Laederachauthor=S.+A.+McLean&title=A+Functional+riboSNitch+in+the+3%E2%80%B2+Untranslated+Region+of+FKBP5+Alters+MicroRNA-320a+Binding+Efficiency+and+Mediates+Vulnerability+to+Chronic+Post-Traumatic+Pain&doi=10.1523%2FJNEUROSCI.3458-17.2018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A functional riboSNitch in the 3' untranslated region of FKBP5 alters MicroRNA-320a binding efficiency and mediates vulnerability to chronic post-traumatic pain</span></div><div class="casAuthors">Linnstaedt, Sarah D.; Riker, Kyle D.; Rueckeis, Xcathleen A.; Kutchko, Katrina M.; Lackey, Lela; Mccarthy, Kathleen R.; Tsai, Yi-Hsuan; Parker, Joel S.; Kurz, Michael C.; Hendry, Phyllis L.; Lewandowski, Christopher; Datner, Elizabeth; Pearson, Claire; O'neil, Brian; Domeier, Robert; Kaushik, Sangeeta; Laederach, Alain; MClean, Samuel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">8407-8420</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">1529-2401</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Previous studies have shown that common variants of the gene coding for FK506-binding protein 51 (FKBP5), a crit. regulator of glucocorticoid sensitivity, affect vulnerability to stress-related disorders.  In a previous report, FKBP5 rs1360780 was identified as a functional variant because of its effect on gene methylation.  Here we report evidence for a novel functional FKBP5 allele, rs3800373.  This study assessed the assocn. between rs3800373 and post-traumatic chronic pain in 1607 women and men from two ethnically diverse human cohorts.  The mol. mechanism through which rs3800373 affects adverse outcomes was established via in silico, in vivo, and in vitro analyses.  The rs3800373 minor allele predicted worse adverse outcomes after trauma exposure, such that individuals with the minor (risk) allele developed more severe post-traumatic chronic musculoskeletal pain.  Among these individuals, peritraumatic circulating FKBP5 expression levels increased as cortisol and glucocorticoid receptor (NR3C1) mRNA levels increased, consistent with increased glucocorticoid resistance.  Bioinformatic, in vitro, and mutational analyses indicate that the rs3800373 minor allele reduces the binding of a stress- and pain-assocd. microRNA, miR-320a, to FKBP5 via altering the FKBP5 mRNA 3'UTR secondary structure (i.e., is a riboSNitch).  This results in relatively greater FKBP5 translation, unchecked by miR-320a.  Overall, these results identify an important gene-miRNA interaction influencing chronic pain risk in vulnerable individuals and suggest that exogenous methods to achieve targeted redn. in poststress FKBP5 mRNA expression may constitute useful therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8ey_putJR7Vg90H21EOLACvtfcHk0liGzDGz9_dt1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1ymur7E&md5=e45aaa0214e4ddaa5a06c409fae2590a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3458-17.2018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3458-17.2018%26sid%3Dliteratum%253Aachs%26aulast%3DLinnstaedt%26aufirst%3DS.%2BD.%26aulast%3DRiker%26aufirst%3DK.%2BD.%26aulast%3DRueckeis%26aufirst%3DC.%2BA.%26aulast%3DKutchko%26aufirst%3DK.%2BM.%26aulast%3DLackey%26aufirst%3DL.%26aulast%3DMcCarthy%26aufirst%3DK.%2BR.%26aulast%3DTsai%26aufirst%3DY.%2BH.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DKurz%26aufirst%3DM.%2BC.%26aulast%3DHendry%26aufirst%3DP.%2BL.%26aulast%3DLewandowski%26aufirst%3DC.%26aulast%3DDatner%26aufirst%3DE.%26aulast%3DPearson%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DDomeier%26aufirst%3DR.%26aulast%3DKaushik%26aufirst%3DS.%26aulast%3DLaederach%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DS.%2BA.%26atitle%3DA%2520Functional%2520riboSNitch%2520in%2520the%25203%25E2%2580%25B2%2520Untranslated%2520Region%2520of%2520FKBP5%2520Alters%2520MicroRNA-320a%2520Binding%2520Efficiency%2520and%2520Mediates%2520Vulnerability%2520to%2520Chronic%2520Post-Traumatic%2520Pain%26jtitle%3DJ.%2520Neurosci.%26date%3D2018%26volume%3D38%26spage%3D8407%26epage%3D8420%26doi%3D10.1523%2FJNEUROSCI.3458-17.2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">FKBPs and their role in neuronal signaling</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1850</i></span>,  <span class="NLM_fpage">2035</span>– <span class="NLM_lpage">2040</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2015.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bbagen.2015.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25615537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=2035-2040&author=F.+Hausch&title=FKBPs+and+their+role+in+neuronal+signaling&doi=10.1016%2Fj.bbagen.2015.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">FKBPs and their role in neuronal signaling</span></div><div class="casAuthors">Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2035-2040</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Ligands for FK506-binding proteins, also referred to as neuroimmunophilin ligands, have repeatedly been described as neuritotrophic, neuroprotective or neuroregenerative agents.  However, the precise mol. mechanism of action underlying the obsd. effects has remained elusive, which eventually led to a reduced interest in FKBP ligand development.  A survey is presented on the pharmacol. of neuroimmunophilin ligands, of the current understanding of individual FKBP homologs in neuronal processes and an assessment of their potential as drug targets for CNS disorders.  FKBP51 is the major target accounting for the neuritotrophic effect of neuroimmunophilin ligands.  Selectivity against the homolog FKBP52 is essential for optimal neuritotrophic efficacy.  Selectivity within the FKBP family, in particular selective inhibition of FKBP12 or FKBP51, is possible.  FKBP51 is a pharmacol. tractable target for stress-related disorders.  The role of FKBPs in neurodegeneration remains to be clarified.  This article is part of a Special Issue entitled Proline-directed Foldases: Cell Signaling Catalysts and Drug Targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5ThYgkiLY7Vg90H21EOLACvtfcHk0liGzDGz9_dt1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtrw%253D&md5=462283764c62af331bfc90b3f2221835</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DFKBPs%2520and%2520their%2520role%2520in%2520neuronal%2520signaling%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D2035%26epage%3D2040%26doi%3D10.1016%2Fj.bbagen.2015.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuboni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vizarra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zannas, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draenert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, O. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of the FK506-binding protein 51 by induced fit</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnchembio.1699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25436518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCksrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=33-37&author=S.+Gaaliauthor=A.+Kirschnerauthor=S.+Cuboniauthor=J.+Hartmannauthor=C.+Kozanyauthor=G.+Balsevichauthor=C.+Namendorfauthor=P.+Fernandez-Vizarraauthor=C.+Sippelauthor=A.+S.+Zannasauthor=R.+Draenertauthor=E.+B.+Binderauthor=O.+F.+Almeidaauthor=G.+Ruhterauthor=M.+Uhrauthor=M.+V.+Schmidtauthor=C.+Toumaauthor=A.+Bracherauthor=F.+Hausch&title=Selective+inhibitors+of+the+FK506-binding+protein+51+by+induced+fit&doi=10.1038%2Fnchembio.1699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of the FK506-binding protein 51 by induced fit</span></div><div class="casAuthors">Gaali, Steffen; Kirschner, Alexander; Cuboni, Serena; Hartmann, Jakob; Kozany, Christian; Balsevich, Georgia; Namendorf, Christian; Fernandez-Vizarra, Paula; Sippel, Claudia; Zannas, Anthony S.; Draenert, Rika; Binder, Elisabeth B.; Almeida, Osborne F. X.; Ruehter, Gerd; Uhr, Manfred; Schmidt, Mathias V.; Touma, Chadi; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-37</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51, encoded by the FKBP5 gene) is an established risk factor for stress-related psychiatric disorders such as major depression.  Drug discovery for FKBP51 has been hampered by the inability to pharmacol. differentiate against the structurally similar but functional opposing homolog FKBP52, and all known FKBP ligands are unselective.  Here, we report the discovery of the potent and highly selective inhibitors of FKBP51, SAFit1 and SAFit2.  This new class of ligands achieves selectivity for FKBP51 by an induced-fit mechanism that is much less favorable for FKBP52.  By using these ligands, we demonstrate that selective inhibition of FKBP51 enhances neurite elongation in neuronal cultures and improves neuroendocrine feedback and stress-coping behavior in mice.  Our findings provide the structural and functional basis for the development of mechanistically new antidepressants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6eYhLVOE_brVg90H21EOLACvtfcHk0ljYGb3WMav2Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCksrnN&md5=3f8d9300671ae7df4763b6d4507644dc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1699%26sid%3Dliteratum%253Aachs%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DKirschner%26aufirst%3DA.%26aulast%3DCuboni%26aufirst%3DS.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DFernandez-Vizarra%26aufirst%3DP.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DZannas%26aufirst%3DA.%2BS.%26aulast%3DDraenert%26aufirst%3DR.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26aulast%3DAlmeida%26aufirst%3DO.%2BF.%26aulast%3DRuhter%26aufirst%3DG.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520FK506-binding%2520protein%252051%2520by%2520induced%2520fit%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D33%26epage%3D37%26doi%3D10.1038%2Fnchembio.1699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerholz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deussing, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of the psychiatric risk factor FKBP51 has anxiolytic properties</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">9007</span>– <span class="NLM_lpage">9016</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.4024-14.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1523%2FJNEUROSCI.4024-14.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=26085626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=9007-9016&author=J.+Hartmannauthor=K.+V.+Wagnerauthor=S.+Gaaliauthor=A.+Kirschnerauthor=C.+Kozanyauthor=G.+Ruhterauthor=N.+Dedicauthor=A.+S.+Hauslauthor=L.+Hoeijmakersauthor=S.+Westerholzauthor=C.+Namendorfauthor=T.+Gerlachauthor=M.+Uhrauthor=A.+Chenauthor=J.+M.+Deussingauthor=F.+Holsboerauthor=F.+Hauschauthor=M.+V.+Schmidt&title=Pharmacological+inhibition+of+the+psychiatric+risk+factor+FKBP51+has+anxiolytic+properties&doi=10.1523%2FJNEUROSCI.4024-14.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of the psychiatric risk factor FKBP51 has anxiolytic properties</span></div><div class="casAuthors">Hartmann, Jakob; Wagner, Klaus V.; Gaali, Steffen; Kirschner, Alexander; Kozany, Christian; Ruehter, Gerd; Dedic, Nina; Haeusl, Alexander S.; Hoeijmakers, Lianne; Westerholz, Soeren; Namendorf, Christian; Gerlach, Tamara; Uhr, Manfred; Chen, Alon; Deussing, Jan M.; Holsboer, Florian; Hausch, Felix; Schmidt, Mathias V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9007-9016</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Anxiety-related psychiatric disorders represent one of the largest health burdens worldwide.  Single nucleotide polymorphisms of the FK506 binding protein 51 (FKBP51) gene have been repeatedly assocd. with anxiety-related disorders and stress sensitivity.  Given the intimate relationship of stress and anxiety, we hypothesized that amygdala FKBP51 may mediate anxiety-related behaviors.  Mimicking the stress effect by specifically overexpressing FKBP51 in the basolateral amygdala (BLA) or central amygdala resulted in increased anxiety-related behavior, resp.  In contrast, application of a highly selective FKBP51 point mutant antagonist, following FKBP51mut BLA-overexpression, reduced the anxiogenic phenotype.  We subsequently tested a novel FKBP51 antagonist, SAFit2, in wild-type mice via BLA microinjections, which reduced anxiety-related behavior.  Remarkably, the same effect was obsd. following peripheral administration of SAFit2.  To our knowledge, this is the first in vivo study using a specific FKBP51 antagonist, thereby unraveling the role of FKBP51 and its potential as a novel drug target for the improved treatment of anxiety-related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLNKQ4xD4NzLVg90H21EOLACvtfcHk0lhtcOysfo3B1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamu73O&md5=cf5e2e0fe69e3d2ea923567b5693909f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.4024-14.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.4024-14.2015%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DK.%2BV.%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DKirschner%26aufirst%3DA.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DRuhter%26aufirst%3DG.%26aulast%3DDedic%26aufirst%3DN.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DHoeijmakers%26aufirst%3DL.%26aulast%3DWesterholz%26aufirst%3DS.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DGerlach%26aufirst%3DT.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeussing%26aufirst%3DJ.%2BM.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DPharmacological%2520inhibition%2520of%2520the%2520psychiatric%2520risk%2520factor%2520FKBP51%2520has%2520anxiolytic%2520properties%26jtitle%3DJ.%2520Neurosci.%26date%3D2015%26volume%3D35%26spage%3D9007%26epage%3D9016%26doi%3D10.1523%2FJNEUROSCI.4024-14.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Pharmacological modulation of the psychiatric risk factor FKBP51 alters efficiency of common antidepressant drugs</span>. <i>Front. Behav. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">262</span>, <span class="refDoi"> DOI: 10.3389/fnbeh.2018.00262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.3389%2Ffnbeh.2018.00262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30483074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BB3crlsVGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=262&author=M.+L.+Pohlmannauthor=A.+S.+Hauslauthor=D.+Harbichauthor=G.+Balsevichauthor=C.+Engelhardtauthor=X.+X.+Fengauthor=M.+Breitsamerauthor=F.+Hauschauthor=G.+Winterauthor=M.+V.+Schmidt&title=Pharmacological+modulation+of+the+psychiatric+risk+factor+FKBP51+alters+efficiency+of+common+antidepressant+drugs&doi=10.3389%2Ffnbeh.2018.00262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Modulation of the Psychiatric Risk Factor FKBP51 Alters Efficiency of Common Antidepressant Drugs</span></div><div class="casAuthors">Pohlmann Max L; Hausl Alexander S; Harbich Daniela; Balsevich Georgia; Engelhardt Clara; Feng Xixi; Hausch Felix; Schmidt Mathias V; Breitsamer Michaela; Winter Gerhard; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in behavioral neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">262</span>
        ISSN:<span class="NLM_cas:issn">1662-5153</span>.
    </div><div class="casAbstract">Despite a growing body of research over the last few decades, mental disorders, including anxiety disorders or depression, are still one of the most prevalent and hardest to treat health burdens worldwide.  Since pharmacological treatment with a single drug is often rather ineffective, approaches such as co-medication with functionally diverse antidepressants (ADs) have been discussed and tried more recently.  Besides classical ADs, there is a growing number of candidate targets identified as potential starting points for new treatment methods.  One of these candidates, the FK506 binding protein 51 (FKBP51) is linked to a number of psychiatric disorders in humans.  In this study, we used SAFit2-a newly developed modulator of FKBP51, which has shown promising results in rodent models for stress-related disorders delivered in a depot formulation.  We combined SAFit2 with the commonly prescribed selective serotonin reuptake inhibitor (SSRI) escitalopram and performed basic behavioral characterization in a mouse model.  Remarkably, co-application of SAFit2 lowered the efficacy of escitalopram in anxiety-related tests but improved stress coping behavior.  Given the fact that mental diseases such as anxiety disorders or depression can be divided into different sub-categories, some of which more or less prone to stress, SAFit2 could indeed be a highly beneficial co-medication in very specific cases.  This study could be a first, promising step towards the use of FKBP51 modulators as potent and specific enhancers of AD efficiency for subclasses of patients in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmSzSTA7pEe0Q0lWbKiCflfW6udTcc2eZ6yQc2WxBBOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crlsVGnuw%253D%253D&md5=cdedd03523d588d0464852c5d30534b7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3389%2Ffnbeh.2018.00262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnbeh.2018.00262%26sid%3Dliteratum%253Aachs%26aulast%3DPohlmann%26aufirst%3DM.%2BL.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DHarbich%26aufirst%3DD.%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DEngelhardt%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DPharmacological%2520modulation%2520of%2520the%2520psychiatric%2520risk%2520factor%2520FKBP51%2520alters%2520efficiency%2520of%2520common%2520antidepressant%2520drugs%26jtitle%3DFront.%2520Behav.%2520Neurosci.%26date%3D2018%26volume%3D12%26spage%3D262%26doi%3D10.3389%2Ffnbeh.2018.00262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Arrigo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tufano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guadagno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Basso De Caro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staibano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, S.</span></span> <span> </span><span class="NLM_article-title">A regulatory role for the co-chaperone FKBP51s in PD-L1 expression in glioma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">68291</span>– <span class="NLM_lpage">68304</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.18632%2Foncotarget.19309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=28978117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FltlWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=68291-68304&author=P.+D%E2%80%99Arrigoauthor=M.+Russoauthor=A.+Reaauthor=M.+Tufanoauthor=E.+Guadagnoauthor=M.+L.+Del+Basso+De+Caroauthor=R.+Pacelliauthor=F.+Hauschauthor=S.+Staibanoauthor=G.+Ilardiauthor=S.+Parisiauthor=M.+F.+Romanoauthor=S.+Romano&title=A+regulatory+role+for+the+co-chaperone+FKBP51s+in+PD-L1+expression+in+glioma&doi=10.18632%2Foncotarget.19309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A regulatory role for the co-chaperone FKBP51s in PD-L1 expression in glioma</span></div><div class="casAuthors">D'Arrigo Paolo; Russo Michele; Rea Anna; Tufano Martina; Parisi Silvia; Romano Maria Fiammetta; Romano Simona; Guadagno Elia; Del Basso De Caro Maria Laura; Pacelli Roberto; Staibano Stefania; Ilardi Gennaro; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">68291-68304</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  FKBP51 is a co-chaperone with isomerase activity, abundantly expressed in glioma.  We previously identified a spliced isoform (FKBP51s) and highlighted a role for this protein in the upregulation of Programmed Death Ligand 1 (PD-L1) expression in melanoma.  Because gliomas can express PD-L1 causing a defective host anti-tumoral immunity, we investigated whether FKBP51s was expressed in glioma and played a role in PD-L1 regulation in this tumour.  METHODS:  We used D54 and U251 glioblastoma cell lines that constitutively expressed PD-L1.  FKBP51s was measured by immunoblot, flow cytometry and microscopy.  In patient tumours, IHC and qPCR were used to measure protein and mRNA levels respectively.  FKBP51s depletion was achieved by siRNAs, and its enzymatic function was inhibited using selective inhibitors (SAFit).  We investigated protein maturation using N-glycosidase and cell fractionation approaches.  RESULTS:  FKBP51s was expressed at high levels in glioma cells.  Glycosylated-PD-L1 was increased and reduced by FKBP51s overexpression or silencing, respectively.  Naive PD-L1 was found in the endoplasmic reticulum (ER) of glioma cells complexed with FKBP51s, whereas the glycosylated form was measured in the Golgi apparatus.  SAFit reduced PD-L1 levels (constitutively expressed and ionizing radiation-induced).  SAFit reduced cell death of PBMC co-cultured with glioma.  CONCLUSIONS:  Here we addressed the mechanism of post-translational regulation of PD-L1 protein in glioma.  FKBP51s upregulated PD-L1 expression on the plasma membrane by catalysing the protein folding required for subsequent glycosylation.  Inhibition of FKBP51s isomerase activity by SAFit decreased PD-L1 levels.  These findings suggest that FKBP51s is a potential target of immunomodulatory strategies for glioblastoma treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLwCRKT7Pe3Uh5eUSa2oe_fW6udTcc2eYIbMj4eo4A67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FltlWmtQ%253D%253D&md5=727369325047c082b28529f43cfca839</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19309%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Arrigo%26aufirst%3DP.%26aulast%3DRusso%26aufirst%3DM.%26aulast%3DRea%26aufirst%3DA.%26aulast%3DTufano%26aufirst%3DM.%26aulast%3DGuadagno%26aufirst%3DE.%26aulast%3DDel%2BBasso%2BDe%2BCaro%26aufirst%3DM.%2BL.%26aulast%3DPacelli%26aufirst%3DR.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DStaibano%26aufirst%3DS.%26aulast%3DIlardi%26aufirst%3DG.%26aulast%3DParisi%26aufirst%3DS.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26aulast%3DRomano%26aufirst%3DS.%26atitle%3DA%2520regulatory%2520role%2520for%2520the%2520co-chaperone%2520FKBP51s%2520in%2520PD-L1%2520expression%2520in%2520glioma%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D68291%26epage%3D68304%26doi%3D10.18632%2Foncotarget.19309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Arrigo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digregorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tufano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedobbeleer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigorito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogister, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span> <span> </span><span class="NLM_article-title">The splicing FK506-binding protein-51 isoform plays a role in glioblastoma resistance through programmed cell death ligand-1 expression regulation</span>. <i>Cell Death Discov.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">137</span>, <span class="refDoi"> DOI: 10.1038/s41420-019-0216-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41420-019-0216-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31583120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BB3MnkvFCntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=137&author=P.+D%E2%80%99Arrigoauthor=M.+Digregorioauthor=S.+Romanoauthor=M.+Tufanoauthor=A.+Reaauthor=F.+Hauschauthor=M.+Dedobbeleerauthor=V.+Vigoritoauthor=S.+Russoauthor=M.+Bauderauthor=B.+Rogisterauthor=M.+F.+Romano&title=The+splicing+FK506-binding+protein-51+isoform+plays+a+role+in+glioblastoma+resistance+through+programmed+cell+death+ligand-1+expression+regulation&doi=10.1038%2Fs41420-019-0216-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The splicing FK506-binding protein-51 isoform plays a role in glioblastoma resistance through programmed cell death ligand-1 expression regulation</span></div><div class="casAuthors">D'Arrigo Paolo; Romano Simona; Tufano Martina; Rea Anna; Vigorito Vincenza; Russo Salvatore; Romano Maria Fiammetta; Digregorio Marina; Dedobbeleer Matthias; Rogister Bernard; Hausch Felix; Bauder Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell death discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137</span>
        ISSN:<span class="NLM_cas:issn">2058-7716</span>.
    </div><div class="casAbstract">Gliomas aberrantly express programmed cell death ligand-1 (PD-L1), which has a pivotal role in immunoevasion.  The splicing isoform of FKBP5, termed FKBP51s, is a PD-L1 foldase, assisting the immune checkpoint molecule in maturation and expression on the plasma membrane.  The concept that PD-L1 supports tumor-intrinsic properties is increasingly emerging.  The aim of the present work was to confirm the pro-tumoral effect of PD-L1 on human glioma cell survival, stemness capacity and resistance, and to address the issue of whether, by targeting its foldase either chemically or by silencing, the aggressive tumor features could be attenuated.  PD-L1-depleted glioma cells have a reduced threshold for apoptosis, while PD-L1 forced expression increases resistance.  Similar results were obtained with FKBP51s modulation.  The ability of PD-L1 to counteract cell death was hampered by FKBP51s silencing.  PD-L1 expression was particularly high in glioma cells with a cancer-stem-cell profile.  Moreover, PD-L1 sustained the spheroid formation capability of glioma cells.  Targeting of FKBP51s by small-interfering RNA (siRNA) or the specific inhibitor SAFit2, reduced the number of formed spheroids, along with PD-L1 expression.  Finally, in an orthotopic mouse model of glioblastoma, daily treatment with SAFit2 significantly reduced tumor PD-L1 expression, and tumor growth.  In treated mice, caspase-3 activation and reduced vimentin expression were observed in excised tumors.  In conclusion, targeting of FKBP51s hampers PD-L1 and its pro-tumoral properties, thereby affecting the self-renewal and growth capacities of glioblastoma cells in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTVgJP0BVo3R4xhnOQkIHOxfW6udTcc2eYIbMj4eo4A67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnkvFCntQ%253D%253D&md5=63af445bdf1c6c3005a40f32fc517b1b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41420-019-0216-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41420-019-0216-0%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Arrigo%26aufirst%3DP.%26aulast%3DDigregorio%26aufirst%3DM.%26aulast%3DRomano%26aufirst%3DS.%26aulast%3DTufano%26aufirst%3DM.%26aulast%3DRea%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DDedobbeleer%26aufirst%3DM.%26aulast%3DVigorito%26aufirst%3DV.%26aulast%3DRusso%26aufirst%3DS.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DRogister%26aufirst%3DB.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26atitle%3DThe%2520splicing%2520FK506-binding%2520protein-51%2520isoform%2520plays%2520a%2520role%2520in%2520glioblastoma%2520resistance%2520through%2520programmed%2520cell%2520death%2520ligand-1%2520expression%2520regulation%26jtitle%3DCell%2520Death%2520Discov.%26date%3D2019%26volume%3D5%26spage%3D137%26doi%3D10.1038%2Fs41420-019-0216-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konig, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalinichenko, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voll, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornhuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. P.</span></span> <span> </span><span class="NLM_article-title">The selective FKBP51 inhibitor SAFit2 reduces alcohol consumption and reinstatement of conditioned alcohol effects in mice</span>. <i>Addict. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">e12758</span> <span class="refDoi"> DOI: 10.1111/adb.12758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Fadb.12758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31173432" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&author=L.+Konigauthor=L.+S.+Kalinichenkoauthor=S.+E.+Huberauthor=A.+M.+Vollauthor=M.+Bauderauthor=J.+Kornhuberauthor=F.+Hauschauthor=C.+P.+Muller&title=The+selective+FKBP51+inhibitor+SAFit2+reduces+alcohol+consumption+and+reinstatement+of+conditioned+alcohol+effects+in+mice&doi=10.1111%2Fadb.12758"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fadb.12758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fadb.12758%26sid%3Dliteratum%253Aachs%26aulast%3DKonig%26aufirst%3DL.%26aulast%3DKalinichenko%26aufirst%3DL.%2BS.%26aulast%3DHuber%26aufirst%3DS.%2BE.%26aulast%3DVoll%26aufirst%3DA.%2BM.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DKornhuber%26aufirst%3DJ.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DC.%2BP.%26atitle%3DThe%2520selective%2520FKBP51%2520inhibitor%2520SAFit2%2520reduces%2520alcohol%2520consumption%2520and%2520reinstatement%2520of%2520conditioned%2520alcohol%2520effects%2520in%2520mice%26jtitle%3DAddict.%2520Biol.%26date%3D2020%26volume%3D25%26doi%3D10.1111%2Fadb.12758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozburn, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metten, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlumbohm, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeMoine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabbe, J. C.</span></span> <span> </span><span class="NLM_article-title">Targeting the glucocorticoid receptor reduces binge-like drinking in high drinking in the Dark (HDID-1) mice</span>. <i>Alcohol.: Clin. Exp. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1111/acer.14318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Facer.14318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32154593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2020&pages=1025-1036&author=A.+M.+Savareseauthor=A.+R.+Ozburnauthor=P.+Mettenauthor=J.+P.+Schlumbohmauthor=W.+R.+Hackauthor=K.+LeMoineauthor=H.+Huntauthor=F.+Hauschauthor=M.+Bauderauthor=J.+C.+Crabbe&title=Targeting+the+glucocorticoid+receptor+reduces+binge-like+drinking+in+high+drinking+in+the+Dark+%28HDID-1%29+mice&doi=10.1111%2Facer.14318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Glucocorticoid Receptor Reduces Binge-Like Drinking in High Drinking in the Dark (HDID-1) Mice</span></div><div class="casAuthors">Savarese, Antonia M.; Ozburn, Angela R.; Metten, Pamela; Schlumbohm, Jason P.; Hack, Wyatt R.; LeMoine, Kathryn; Hunt, Hazel; Hausch, Felix; Bauder, Michael; Crabbe, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Alcoholism: Clinical & Experimental Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1025-1036</span>CODEN:
                <span class="NLM_cas:coden">ACRSDM</span>;
        ISSN:<span class="NLM_cas:issn">0145-6008</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chronic alc. exposure can alter glucocorticoid receptor (GR) function in some brain areas that promotes escalated and compulsive-like alc. intake.  GR antagonism can prevent dependence-induced escalation in drinking, but very little is known about the role of GR in regulating high-risk nondependent alc. intake.  Here, we investigate the role of GR in regulating binge-like drinking and aversive responses to alc. in the High Drinking in the Dark (HDID-1) mice, which have been selectively bred for high blood ethanol (EtOH) concns. (BECs) in the Drinking in the Dark (DID) test, and in their founder line, the HS/NPT.  In sep. expts., male and female HDID-1 mice were administered one of several compds. that inhibited GR or its neg. regulator, FKBP51 (mifepristone [12.5, 25, 50, 100 mg/kg], CORT113176 [20, 40, 80 mg/kg], and SAFit2 [10, 20, 40 mg/kg]) during a 2-day DID task.  EtOH consumption and BECs were measured.  EtOH conditioned taste and place aversion (CTA and CPA, resp.) were measured in sep. HDID-1 mice after mifepristone administration to assess GR's role in regulating the conditioned aversive effects of EtOH.  Lastly, HS/NPT mice were administered CORT113176 during DID to assess whether dissimilar effects from those of HDID-1 would be obsd., which could suggest that selective breeding had altered sensitivity to the effects of GR antagonism on binge-like drinking.  GR antagonism (with both mifepristone and CORT113176) selectively reduced binge-like EtOH intake and BECs in the HDID-1 mice, while inhibition of FKBP51 did not alter intake or BECs.  In contrast, GR antagonism had no effect on EtOH intake or BECs in the HS/NPT mice.  Although HDID-1 mice exhibit attenuated EtOH CTA, mifepristone administration did not enhance the aversive effects of EtOH in either a CTA or CPA task.  These data suggest that the selection process increased sensitivity to GR antagonism on EtOH intake in the HDID-1 mice, and support a role for the GR as a genetic risk factor for high-risk alc. intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJwn8j-V1QLVg90H21EOLACvtfcHk0ljQZiX7dXVlYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVGktrw%253D&md5=b082e4bc8ce4336a997e6f0ad0b1f791</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Facer.14318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Facer.14318%26sid%3Dliteratum%253Aachs%26aulast%3DSavarese%26aufirst%3DA.%2BM.%26aulast%3DOzburn%26aufirst%3DA.%2BR.%26aulast%3DMetten%26aufirst%3DP.%26aulast%3DSchlumbohm%26aufirst%3DJ.%2BP.%26aulast%3DHack%26aufirst%3DW.%2BR.%26aulast%3DLeMoine%26aufirst%3DK.%26aulast%3DHunt%26aufirst%3DH.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DCrabbe%26aufirst%3DJ.%2BC.%26atitle%3DTargeting%2520the%2520glucocorticoid%2520receptor%2520reduces%2520binge-like%2520drinking%2520in%2520high%2520drinking%2520in%2520the%2520Dark%2520%2528HDID-1%2529%2520mice%26jtitle%3DAlcohol.%253A%2520Clin.%2520Exp.%2520Res.%26date%3D2020%26volume%3D44%26spage%3D1025%26epage%3D1036%26doi%3D10.1111%2Facer.14318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zannas, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pape, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arloth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emeny, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iurato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrillo-Roa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raikkonen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gieger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressler, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Myhsok, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladwig, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span> <span> </span><span class="NLM_article-title">Epigenetic upregulation of FKBP5 by aging and stress contributes to NF-kappa B-driven inflammation and cardiovascular risk</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">11370</span>– <span class="NLM_lpage">11379</span>, <span class="refDoi"> DOI: 10.1073/pnas.1816847116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1073%2Fpnas.1816847116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31113877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2itLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=11370-11379&author=A.+S.+Zannasauthor=M.+W.+Jiaauthor=K.+Hafnerauthor=J.+Baumertauthor=T.+Wiechmannauthor=J.+C.+Papeauthor=J.+Arlothauthor=M.+Kodelauthor=S.+Martinelliauthor=M.+Roitmanauthor=S.+Rohauthor=A.+Haehleauthor=R.+T.+Emenyauthor=S.+Iuratoauthor=T.+Carrillo-Roaauthor=J.+Lahtiauthor=K.+Raikkonenauthor=J.+G.+Erikssonauthor=A.+J.+Drakeauthor=M.+Waldenbergerauthor=S.+Wahlauthor=S.+Kunzeauthor=S.+Lucaeauthor=B.+Bradleyauthor=C.+Giegerauthor=F.+Hauschauthor=A.+K.+Smithauthor=K.+J.+Resslerauthor=B.+Muller-Myhsokauthor=K.+H.+Ladwigauthor=T.+Reinauthor=N.+C.+Gassenauthor=E.+B.+Binder&title=Epigenetic+upregulation+of+FKBP5+by+aging+and+stress+contributes+to+NF-kappa+B-driven+inflammation+and+cardiovascular+risk&doi=10.1073%2Fpnas.1816847116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic upregulation of FKBP5 by aging and stress contributes to NF-κB-driven inflammation and cardiovascular risk</span></div><div class="casAuthors">Zannas, Anthony S.; Jia, Meiwen; Hafner, Kathrin; Baumert, Jens; Wiechmann, Tobias; Pape, Julius C.; Arloth, Janine; Koedel, Maik; Martinelli, Silvia; Roitman, Maria; Roeh, Simone; Haehle, Andreas; Emeny, Rebecca T.; Iurato, Stella; Carrillo-Roa, Tania; Lahti, Jari; Raeikkoenen, Katri; Eriksson, Johan G.; Drake, Amanda J.; Waldenberger, Melanie; Wahl, Simone; Kunze, Sonja; Lucae, Susanne; Bradley, Bekh; Gieger, Christian; Hausch, Felix; Smith, Alicia K.; Ressler, Kerry J.; Mueller-Myhsok, Bertram; Ladwig, Karl-Heinz; Rein, Theo; Gassen, Nils C.; Binder, Elisabeth B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11370-11379</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aging and psychosocial stress are assocd. with increased inflammation and disease risk, but the underlying mol. mechanisms are unclear.  Because both aging and stress are also assocd. with lasting epigenetic changes, a plausible hypothesis is that stress along the lifespan could confer disease risk through epigenetic effects on mols. involved in inflammatory processes.  Here, by combining large-scale analyses in human cohorts with expts. in cells, we report that FKBP5, a protein implicated in stress physiol., contributes to these relations.  Across independent human cohorts (total n > 3000), aging synergized with stress-related phenotypes, measured with childhood trauma and major depression questionnaires, to epigenetically up-regulate FKBP5 expression.  These age/stress-related epigenetic effects were recapitulated in a cellular model of replicative senescence, whereby we exposed replicating human fibroblasts to stress (glucocorticoid) hormones.  Unbiased genome-wide analyses in human blood linked higher FKBP5 mRNA with a proinflammatory profile and altered NF-κB-related gene networks.  Accordingly, expts. in immune cells showed that higher FKBP5 promotes inflammation by strengthening the interactions of NF-κB regulatory kinases, whereas opposing FKBP5 either by genetic deletion (CRISPR/Cas9-mediated) or selective pharmacol. inhibition prevented the effects on NF-κB.  Further, the age/stress-related epigenetic signature enhanced FKBP5 response to NF-κB through a pos. feedback loop and was present in individuals with a history of acute myocardial infarction, a disease state linked to peripheral inflammation.  These findings suggest that aging/stress-driven FKBP5-NF-κB signaling mediates inflammation, potentially contributing to cardiovascular risk, and may thus point to novel biomarker and treatment possibilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS78Gd39hLKrVg90H21EOLACvtfcHk0liUPGodU5h_pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2itLvO&md5=4007fdc7feef32fde4cf9c1a2ab13d76</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1816847116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1816847116%26sid%3Dliteratum%253Aachs%26aulast%3DZannas%26aufirst%3DA.%2BS.%26aulast%3DJia%26aufirst%3DM.%2BW.%26aulast%3DHafner%26aufirst%3DK.%26aulast%3DBaumert%26aufirst%3DJ.%26aulast%3DWiechmann%26aufirst%3DT.%26aulast%3DPape%26aufirst%3DJ.%2BC.%26aulast%3DArloth%26aufirst%3DJ.%26aulast%3DKodel%26aufirst%3DM.%26aulast%3DMartinelli%26aufirst%3DS.%26aulast%3DRoitman%26aufirst%3DM.%26aulast%3DRoh%26aufirst%3DS.%26aulast%3DHaehle%26aufirst%3DA.%26aulast%3DEmeny%26aufirst%3DR.%2BT.%26aulast%3DIurato%26aufirst%3DS.%26aulast%3DCarrillo-Roa%26aufirst%3DT.%26aulast%3DLahti%26aufirst%3DJ.%26aulast%3DRaikkonen%26aufirst%3DK.%26aulast%3DEriksson%26aufirst%3DJ.%2BG.%26aulast%3DDrake%26aufirst%3DA.%2BJ.%26aulast%3DWaldenberger%26aufirst%3DM.%26aulast%3DWahl%26aufirst%3DS.%26aulast%3DKunze%26aufirst%3DS.%26aulast%3DLucae%26aufirst%3DS.%26aulast%3DBradley%26aufirst%3DB.%26aulast%3DGieger%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSmith%26aufirst%3DA.%2BK.%26aulast%3DRessler%26aufirst%3DK.%2BJ.%26aulast%3DMuller-Myhsok%26aufirst%3DB.%26aulast%3DLadwig%26aufirst%3DK.%2BH.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26atitle%3DEpigenetic%2520upregulation%2520of%2520FKBP5%2520by%2520aging%2520and%2520stress%2520contributes%2520to%2520NF-kappa%2520B-driven%2520inflammation%2520and%2520cardiovascular%2520risk%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D11370%26epage%3D11379%26doi%3D10.1073%2Fpnas.1816847116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Structure-affinity relationship analysis of selective FKBP51 ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7796</span>– <span class="NLM_lpage">7806</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00785</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00785" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2gsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7796-7806&author=X.+Fengauthor=C.+Sippelauthor=A.+Bracherauthor=F.+Hausch&title=Structure-affinity+relationship+analysis+of+selective+FKBP51+ligands&doi=10.1021%2Facs.jmedchem.5b00785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Affinity Relationship Analysis of Selective FKBP51 Ligands</span></div><div class="casAuthors">Feng, Xixi; Sippel, Claudia; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7796-7806</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51) is a promising drug target for the treatment of stress-related psychiatric or metabolic disorders.  Just recently, the first selective ligands for FKBP51 were reported based on an induced fit mechanism, but they are too large for a further drug development process.  The authors therefore designed and synthesized a novel series of selective ligands to explore the requirements necessary for binding to the induced-fit conformation.  All ligands of this series show no binding toward the structurally very similar antitarget FKBP52.  With the cocrystal structure of the best ligand in this novel series the authors confirmed the induced fit mechanism.  Furthermore, the structure-affinity relationship provides information about beneficial structural features, which is valuable for the development of improved FKBP51-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIuleKzzlpMbVg90H21EOLACvtfcHk0liSof8UvNlDPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2gsbfL&md5=90c480d0a6819dea97e4742e6db24583</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00785%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DStructure-affinity%2520relationship%2520analysis%2520of%2520selective%2520FKBP51%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7796%26epage%3D7806%26doi%3D10.1021%2Facs.jmedchem.5b00785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomplun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Recent progress in FKBP ligand development</span>. <i>Curr. Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.2174/1874467208666150519113313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.2174%2F1874467208666150519113313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25986570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC2MfltVKiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=27-36&author=X.+X.+Fengauthor=S.+Pomplunauthor=F.+Hausch&title=Recent+progress+in+FKBP+ligand+development&doi=10.2174%2F1874467208666150519113313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in FKBP Ligand Development</span></div><div class="casAuthors">Feng Xixi; Pomplun Sebastian; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Current molecular pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-36</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FK506-binding proteins have been implicated in numerous human diseases suggesting novel therapeutic opportunities.  In particular, the large FKBP51 has emerged as an important regulator of the stress-coping system and as an established risk factor for stress-related disorders.  The principal druggabilty of FKBPs is evidenced by the prototypical high affinity ligands FK506 and rapamycin but the development of more refined and selective chemical probes for FKBPs has been challenging.  In this review we summarize recent advances in the development of FKBP ligands, which cumulated in the first highly selective ligands for FKBP51.  The best ligand SAFit2 allowed the proof-of-concept in mice for FKBP51 inhibitors as potentially novel antidepressants.  Finally, we discuss pending issues that need to be addressed for the further development of FKBP51-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzpKzaquF5abZpSb_XWfMrfW6udTcc2eZ0vBKUrvNBabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfltVKiug%253D%253D&md5=8e40ae83c490c941c986f35a17d8987c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F1874467208666150519113313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874467208666150519113313%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DPomplun%26aufirst%3DS.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DRecent%2520progress%2520in%2520FKBP%2520ligand%2520development%26jtitle%3DCurr.%2520Mol.%2520Pharmacol.%26date%3D2015%26volume%3D9%26spage%3D27%26epage%3D36%26doi%3D10.2174%2F1874467208666150519113313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, V. R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattler, M.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of FKBP51 employ conformational selection of dynamic invisible states</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">13619</span>– <span class="NLM_lpage">13619</span>, <span class="refDoi"> DOI: 10.1002/anie.201908888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fanie.201908888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsF2ltr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=13619-13619&author=P.+K.+A.+Jagtapauthor=S.+Asamiauthor=C.+Sippelauthor=V.+R.+I.+Kailaauthor=F.+Hauschauthor=M.+Sattler&title=Selective+inhibitors+of+FKBP51+employ+conformational+selection+of+dynamic+invisible+states&doi=10.1002%2Fanie.201908888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Corrigendum: Selective Inhibitors of FKBP51 Employ Conformational Selection of Dynamic Invisible States</span></div><div class="casAuthors">Jagtap, Pravin Kumar Ankush; Asami, Sam; Sippel, Claudia; Kaila, Ville R. I.; Hausch, Felix; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">13619</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqLb9uy9ZXLVg90H21EOLACvtfcHk0liSof8UvNlDPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsF2ltr7P&md5=ed2f1bdb31da080dcfd6e5069b7cc16c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201908888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201908888%26sid%3Dliteratum%253Aachs%26aulast%3DJagtap%26aufirst%3DP.%2BK.%2BA.%26aulast%3DAsami%26aufirst%3DS.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DKaila%26aufirst%3DV.%2BR.%2BI.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DSelective%2520inhibitors%2520of%2520FKBP51%2520employ%2520conformational%2520selection%2520of%2520dynamic%2520invisible%2520states%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D13619%26epage%3D13619%26doi%3D10.1002%2Fanie.201908888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draxler, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickmeier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangel Rojas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span> <span> </span><span class="NLM_article-title">Hybrid screening approach for very small fragments: X-ray and computational screening on FKBP51</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5856</span>– <span class="NLM_lpage">5864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsFelsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5856-5864&author=S.+W.+Draxlerauthor=M.+Bauerauthor=C.+Eickmeierauthor=S.+Nadalauthor=H.+Narauthor=D.+Rangel+Rojasauthor=D.+Seeligerauthor=M.+Zeebauthor=D.+Fiegen&title=Hybrid+screening+approach+for+very+small+fragments%3A+X-ray+and+computational+screening+on+FKBP51&doi=10.1021%2Facs.jmedchem.0c00120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid Screening Approach for Very Small Fragments: X-ray and Computational Screening on FKBP51</span></div><div class="casAuthors">Draxler, Sebastian W.; Bauer, Margit; Eickmeier, Christian; Nadal, Simon; Nar, Herbert; Rangel Rojas, Daniel; Seeliger, Daniel; Zeeb, Markus; Fiegen, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5856-5864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) permits efficient sampling of the vast chem. space for hit identification.  Libraries are screened biophys. and fragment:protein co-structures are detd. by X-ray crystallog.  In parallel, computational methods can derive pharmacophore models or screen virtual libraries.  We screened 15 very small fragments (VSFs) (HA ≤ 11) computationally, using site identification by ligand competitive satn. (SILCS), and exptl., by X-ray crystallog., to map potential interaction sites on the FKBP51 FK1 domain.  We identified three hot spots and obtained 6 X-ray co-structures, giving a hit rate of 40%.  SILCS FragMaps overlapped with X-ray structures.  The compds. had millimolar affinities as detd. by 15N HSQC NMR.  VSFs identified the same interactions as known FK1 binder and provide new chem. starting points.  We propose a hybrid screening strategy starting with SILCS, followed by a pharmacophore-derived X-ray screen and 15N HSQC NMR based KD detn. to rapidly identify hits and their binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOqWmaKxGJkrVg90H21EOLACvtfcHk0ljrlLorE2dPLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsFelsLo%253D&md5=35fe6bd3207a0b8d2b08f61c1723371a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00120%26sid%3Dliteratum%253Aachs%26aulast%3DDraxler%26aufirst%3DS.%2BW.%26aulast%3DBauer%26aufirst%3DM.%26aulast%3DEickmeier%26aufirst%3DC.%26aulast%3DNadal%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRangel%2BRojas%26aufirst%3DD.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DFiegen%26aufirst%3DD.%26atitle%3DHybrid%2520screening%2520approach%2520for%2520very%2520small%2520fragments%253A%2520X-ray%2520and%2520computational%2520screening%2520on%2520FKBP51%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5856%26epage%3D5864%26doi%3D10.1021%2Facs.jmedchem.0c00120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabani, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulochana, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddesh, A. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saini, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span> <span> </span><span class="NLM_article-title">Validated LC-MS/MS method for simultaneous quantitation of SAFit-1 and SAFit-2 in mice plasma: application to a pharmacokinetic study</span>. <i>Drug Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1055/a-1164-6123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1055%2Fa-1164-6123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2020&pages=325-332&author=B.+B.+Gabaniauthor=S.+P.+Sulochanaauthor=A.+H.+A.+Siddeshauthor=V.+Kiranauthor=N.+K.+Sainiauthor=S.+K.+Samantaauthor=M.+S.+Hallurauthor=S.+Rajagopalauthor=R.+Mullangi&title=Validated+LC-MS%2FMS+method+for+simultaneous+quantitation+of+SAFit-1+and+SAFit-2+in+mice+plasma%3A+application+to+a+pharmacokinetic+study&doi=10.1055%2Fa-1164-6123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Validated LC-MS/MS Method for Simultaneous Quantitation of SAFit-1 and SAFit-2 in Mice Plasma: Application to a Pharmacokinetic Study</span></div><div class="casAuthors">Gabani, Bhavesh Babulal; Sulochana, Suresh Ponnayyan; Siddesh, Anup H. A.; Kiran, Vinay; Saini, Neeraj Kumar; Samanta, Swapan Kumar; Hallur, Mahanandeesha S.; Rajagopal, Sridharan; Mullangi, Ramesh</div><div class="citationInfo"><span class="NLM_cas:title">Drug Research (Stuttgart, Germany)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">325-332</span>CODEN:
                <span class="NLM_cas:coden">DRSGAO</span>;
        ISSN:<span class="NLM_cas:issn">2194-9379</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">SAFit-1 and SAFit-2 are selective FKBP51 (FK506-binding protein 51) ligands.  In this paper, we present the development and validation data of an LC-MS/MS method for the simultaneous quantitation of SAFit-1 and SAFit-2 in mice plasma as per FDA regulatory guideline.  SAFit-1 and SAFit-2 along with internal std. were extd. from mice plasma using liq.-liq. extn. method.  Chromatog. resoln. of SAFit-1, SAFit-2 and the internal std. (warfarin) was achieved on an X-Terra Ph column using 0.2% formic acid:acetonitrile (20:80, vol./vol.) as an eluent, which was delivered at a flow-rate of 0.9 mL/min.  The MS/MS ion transitions monitored were m/z 748.4420.4, 803.7384.3 and 309.2 163.2 for SAFit-1, SAFit-2 and the internal std., resp.  The linearity range was 2.45-2446 ng/mL for both SAFit-1 and SAFit-2.  The intra- and inter-day accuracy and intra- and inter-day precision were in the range of 0.90-1.07 and 2.38-10.8%, resp. for SAFit-1; 0.97-1.15 and 0.23-12.5%, resp. for SAFit-2.  Both SAFit-1 and SAFit-2 were found to be stable in stability studies (up to three freeze-thaw cycles and for long-term at -80°C for 30 days) and processed (bench-top for 3 h and in in-injector for 16 h) samples.  The application of the validated method was shown in a pharmacokinetic study in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojtHALoOYkXLVg90H21EOLACvtfcHk0ljrlLorE2dPLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlKisL0%253D&md5=4062795128725b3bf8152c897fc1c900</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1055%2Fa-1164-6123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fa-1164-6123%26sid%3Dliteratum%253Aachs%26aulast%3DGabani%26aufirst%3DB.%2BB.%26aulast%3DSulochana%26aufirst%3DS.%2BP.%26aulast%3DSiddesh%26aufirst%3DA.%2BH.%2BA.%26aulast%3DKiran%26aufirst%3DV.%26aulast%3DSaini%26aufirst%3DN.%2BK.%26aulast%3DSamanta%26aufirst%3DS.%2BK.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26atitle%3DValidated%2520LC-MS%252FMS%2520method%2520for%2520simultaneous%2520quantitation%2520of%2520SAFit-1%2520and%2520SAFit-2%2520in%2520mice%2520plasma%253A%2520application%2520to%2520a%2520pharmacokinetic%2520study%26jtitle%3DDrug%2520Res.%26date%3D2020%26volume%3D70%26spage%3D325%26epage%3D332%26doi%3D10.1055%2Fa-1164-6123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knaup, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staibano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">A novel decalin-based bicyclic scaffold for FKBP51-selective ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWktrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=231-240&author=X.+Fengauthor=C.+Sippelauthor=F.+H.+Knaupauthor=A.+Bracherauthor=S.+Staibanoauthor=M.+F.+Romanoauthor=F.+Hausch&title=A+novel+decalin-based+bicyclic+scaffold+for+FKBP51-selective+ligands&doi=10.1021%2Facs.jmedchem.9b01157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A novel decalin-based bicyclic scaffold for FKBP51-selective ligands</span></div><div class="casAuthors">Feng, Xixi; Sippel, Claudia; Knaup, Fabian H.; Bracher, Andreas; Staibano, Stefania; Romano, Maria F.; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibition of FKBP51 has emerged as possible novel treatment for diseases like major depressive disorder, obesity, chronic pain and certain cancers.  The current FKBP51 inhibitors are rather large, flexible and have to be further optimized.  Using a structure-based rigidification strategy, a novel promising bicyclic scaffold for FKBP51 ligands has been designed and synthesized.  The structure-activity anal. revealed the decalin scaffold as the best moiety for the selectivity-enabling subpocket of FBKP51.  The resulting compds. retain high potency for FKBP51 and excellent selectivity over the close homolog FKBP52.  With the cocrystal structure of an advanced ligand in this novel series, it was shown how the decalin locks the key selectivity-inducing cyclohexyl moiety of the ligand in a conformation typical for FKBP51-selective binding.  The best compd. I produces cell death in a HeLa-derived KB cell line, a cellular model of cervical adenocarcinoma, where FKBP51 is highly overexpressed.  This results show how FKBP51 inhibitors can be rigidified and extended while preserving FKBP51 selectivity.  Such inhibitors might be novel tools in the treatment of human cancers with deregulated FKBP51.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQvCfTxSgpFrVg90H21EOLACvtfcHk0lhmDEb6KcV3dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWktrnI&md5=8b480e29b1cdeed653b7e908f28f881a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01157%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DKnaup%26aufirst%3DF.%2BH.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DStaibano%26aufirst%3DS.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DA%2520novel%2520decalin-based%2520bicyclic%2520scaffold%2520for%2520FKBP51-selective%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D231%26epage%3D240%26doi%3D10.1021%2Facs.jmedchem.9b01157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Rapid, structure-based exploration of pipecolic acid amides as novel selective antagonists of the FK506-binding protein 51</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2410</span>– <span class="NLM_lpage">2422</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslSqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2410-2422&author=S.+Gaaliauthor=X.+Fengauthor=A.+Hahleauthor=C.+Sippelauthor=A.+Bracherauthor=F.+Hausch&title=Rapid%2C+structure-based+exploration+of+pipecolic+acid+amides+as+novel+selective+antagonists+of+the+FK506-binding+protein+51&doi=10.1021%2Facs.jmedchem.5b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, Structure-Based Exploration of Pipecolic Acid Amides as Novel Selective Antagonists of the FK506-Binding Protein 51</span></div><div class="casAuthors">Gaali, Steffen; Feng, Xixi; Haehle, Andreas; Sippel, Claudia; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2410-2422</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51) is a key regulator of stress hormone receptors and an established risk factor for stress-related disorders.  Drug development for FKBP51 has been impaired by the structurally similar but functionally opposing homolog FKBP52.  High selectivity between FKBP51 and FKBP52 can be achieved by ligands that stabilize a recently discovered FKBP51-favoring conformation.  However, drug-like parameters for these ligands remained unfavorable.  In the present study, we replaced the potentially labile pipecolic ester group of previous FKBP51 ligands by various low mol. wt. amides.  This resulted in the first series of pipecolic acid amides, which had much lower mol. wts. without affecting FKBP51 selectivity.  We discovered a geminally substituted cyclopentyl amide as a preferred FKBP51-binding motif and elucidated its binding mode to provide a new lead structure for future drug optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpotLzPLV83BrVg90H21EOLACvtfcHk0lhWa2siGlQ2Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslSqs7c%253D&md5=7a46f4b0da6344be79bd2aa2e941ba81</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01355%26sid%3Dliteratum%253Aachs%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DRapid%252C%2520structure-based%2520exploration%2520of%2520pipecolic%2520acid%2520amides%2520as%2520novel%2520selective%2520antagonists%2520of%2520the%2520FK506-binding%2520protein%252051%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2410%26epage%3D2422%26doi%3D10.1021%2Facs.jmedchem.5b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunyak, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Peptidyl-proline isomerases (PPlases): targets for natural products and natural product-inspired compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9622</span>– <span class="NLM_lpage">9644</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9622-9644&author=B.+M.+Dunyakauthor=J.+E.+Gestwicki&title=Peptidyl-proline+isomerases+%28PPlases%29%3A+targets+for+natural+products+and+natural+product-inspired+compounds&doi=10.1021%2Facs.jmedchem.6b00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl-Proline Isomerases (PPIases): Targets for Natural Products and Natural Product-Inspired Compounds</span></div><div class="casAuthors">Dunyak, Bryan M.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9622-9644</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptidyl-proline isomerases (PPIases) are a chaperone superfamily comprising the FK506-binding proteins (FKBPs), cyclophilins, and parvulins.  PPIases catalyze the cis/trans isomerization of proline, acting as a regulatory switch during folding, activation, and/or degrdn. of many proteins.  These "clients" include proteins with key roles in cancer, neurodegeneration, and psychiatric disorders, suggesting that PPIase inhibitors could be important therapeutics.  However, the active site of PPIases is shallow, solvent-exposed, and well conserved between family members, making selective inhibitor design challenging.  Despite these hurdles, macrocyclic natural products, including FK506, rapamycin, and cyclosporin, bind PPIases with nanomolar or better affinity.  De novo attempts to derive new classes of inhibitors have been somewhat less successful, often showcasing the "undruggable" features of PPIases.  Interestingly, the most potent of these next-generation mols. tend to integrate features of the natural products, including macrocyclization or proline mimicry strategies.  Here, we review recent developments and ongoing challenges in the inhibition of PPIases, with a focus on how natural products might inform the creation of potent and selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz44xLeDGfhLVg90H21EOLACvtfcHk0lhWa2siGlQ2Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms73J&md5=b4c6336096f9558926d965d73fb6b7da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00411%26sid%3Dliteratum%253Aachs%26aulast%3DDunyak%26aufirst%3DB.%2BM.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DPeptidyl-proline%2520isomerases%2520%2528PPlases%2529%253A%2520targets%2520for%2520natural%2520products%2520and%2520natural%2520product-inspired%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9622%26epage%3D9644%26doi%3D10.1021%2Facs.jmedchem.6b00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0ljp-GMobSY2hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based macrocycle design in small-molecule drug discovery and simple metrics to identify opportunities for macrocyclization of small-molecule ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-based+macrocycle+design+in+small-molecule+drug+discovery+and+simple+metrics+to+identify+opportunities+for+macrocyclization+of+small-molecule+ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0ljp-GMobSY2hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-based%2520macrocycle%2520design%2520in%2520small-molecule%2520drug%2520discovery%2520and%2520simple%2520metrics%2520to%2520identify%2520opportunities%2520for%2520macrocyclization%2520of%2520small-molecule%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lgbWBgSLVJ0UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery - an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery+-+an+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lheXbD78vkQJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery%2520-%2520an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0liAy49er4QVVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgRE__C5uuaVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lgRE__C5uuaVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent cyclophilin inhibitors based on the structural simplification of sanglifehrin A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01329</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01329" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptlWrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1000-1017&author=V.+A.+Steadmanauthor=S.+B.+Pettitauthor=K.+G.+Poullennecauthor=L.+Lazaridesauthor=A.+J.+Keatsauthor=D.+K.+Deanauthor=S.+J.+Stanwayauthor=C.+A.+Austinauthor=J.+A.+Sanvoisinauthor=G.+M.+Wattauthor=H.+G.+Fliriauthor=A.+C.+Liclicanauthor=D.+Jinauthor=M.+H.+Wongauthor=S.+A.+Leavittauthor=Y.+J.+Leeauthor=Y.+Tianauthor=C.+R.+Freyauthor=T.+C.+Applebyauthor=U.+Schmitzauthor=P.+Jansaauthor=R.+L.+Mackmanauthor=B.+E.+Schultz&title=Discovery+of+potent+cyclophilin+inhibitors+based+on+the+structural+simplification+of+sanglifehrin+A&doi=10.1021%2Facs.jmedchem.6b01329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Cyclophilin Inhibitors Based on the Structural Simplification of Sanglifehrin A</span></div><div class="casAuthors">Steadman, Victoria A.; Pettit, Simon B.; Poullennec, Karine G.; Lazarides, Linos; Keats, Andrew J.; Dean, David K.; Stanway, Steven J.; Austin, Carol A.; Sanvoisin, Jonathan A.; Watt, Gregory M.; Fliri, Hans G.; Liclican, Albert C.; Jin, Debi; Wong, Melanie H.; Leavitt, Stephanie A.; Lee, Yu-Jen; Tian, Yang; Frey, Christian R.; Appleby, Todd C.; Schmitz, Uli; Jansa, Petr; Mackman, Richard L.; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1000-1017</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilin inhibition has been a target for the treatment of hepatitis C and other diseases, but the generation of potent, drug-like mols. through chem. synthesis has been challenging.  In this study, a set of macrocyclic cyclophilin inhibitors was synthesized based on the core structure of the natural product sanglifehrin A.  Initial compd. optimization identified the valine-m-tyrosine-piperazic acid tripeptide (Val-m-Tyr-Pip) in the sanglifehrin core, stereocenters at C14 and C15, and the hydroxyl group of the m-tyrosine (m-Tyr) residue as key contributors to compd. potency.  Replacing the C18-C21 diene unit of sanglifehrin with a styryl group led to potent compds. that displayed a novel binding mode in which the styrene moiety engaged in a π-stacking interaction with Arg55 of cyclophilin A (Cyp A), and the m-Tyr residue was displaced into solvent.  This observation allowed further simplifications of the scaffold to generate new lead compds. in the search for orally bioavailable cyclophilin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorJXRBXLoC-rVg90H21EOLACvtfcHk0lh1tZTY_-x3mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXptlWrsg%253D%253D&md5=7b3c1ccee76ff6da98064783c0710fd6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01329%26sid%3Dliteratum%253Aachs%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DStanway%26aufirst%3DS.%2BJ.%26aulast%3DAustin%26aufirst%3DC.%2BA.%26aulast%3DSanvoisin%26aufirst%3DJ.%2BA.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DFliri%26aufirst%3DH.%2BG.%26aulast%3DLiclican%26aufirst%3DA.%2BC.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DFrey%26aufirst%3DC.%2BR.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520potent%2520cyclophilin%2520inhibitors%2520based%2520on%2520the%2520structural%2520simplification%2520of%2520sanglifehrin%2520A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1000%26epage%3D1017%26doi%3D10.1021%2Facs.jmedchem.6b01329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiva, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9473</span>– <span class="NLM_lpage">9499</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00802</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00802" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9473-9499&author=R.+L.+Mackmanauthor=V.+A.+Steadmanauthor=D.+K.+Deanauthor=P.+Jansaauthor=K.+G.+Poullennecauthor=T.+Applebyauthor=C.+Austinauthor=C.+A.+Blakemoreauthor=R.+Caiauthor=C.+Cannizzaroauthor=G.+Chinauthor=J.+C.+Chivaauthor=N.+A.+Dunbarauthor=H.+Fliriauthor=A.+J.+Hightonauthor=H.+Huiauthor=M.+Jiauthor=H.+Jinauthor=K.+Karkiauthor=A.+J.+Keatsauthor=L.+Lazaridesauthor=Y.+J.+Leeauthor=A.+Liclicanauthor=M.+Mishauthor=B.+Murrayauthor=S.+B.+Pettitauthor=P.+Pyunauthor=M.+Sangiauthor=R.+Santosauthor=J.+Sanvoisinauthor=U.+Schmitzauthor=A.+Schrierauthor=D.+Siegelauthor=D.+Sperandioauthor=G.+Stepanauthor=Y.+Tianauthor=G.+M.+Wattauthor=H.+Yangauthor=B.+E.+Schultz&title=Discovery+of+a+potent+and+orally+bioavailable+cyclophilin+inhibitor+derived+from+the+sanglifehrin+macrocycle&doi=10.1021%2Facs.jmedchem.8b00802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span></div><div class="casAuthors">Mackman, Richard L.; Steadman, Victoria A.; Dean, David K.; Jansa, Petr; Poullennec, Karine G.; Appleby, Todd; Austin, Carol; Blakemore, Caroline A.; Cai, Ruby; Cannizzaro, Carina; Chin, Gregory; Chiva, Jean-Yves C.; Dunbar, Neil A.; Fliri, Hans; Highton, Adrian J.; Hui, Hon; Ji, Mingzhe; Jin, Haolun; Karki, Kapil; Keats, Andrew J.; Lazarides, Linos; Lee, Yu-Jen; Liclican, Albert; Mish, Michael; Murray, Bernard; Pettit, Simon B.; Pyun, Peter; Sangi, Michael; Santos, Rex; Sanvoisin, Jonathan; Schmitz, Uli; Schrier, Adam; Siegel, Dustin; Sperandio, David; Stepan, George; Tian, Yang; Watt, Gregory M.; Yang, Hai; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9473-9499</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilins are a family of peptidyl-prolyl isomerases that are implicated in a wide range of diseases including hepatitis C.  Our aim was to discover through total synthesis an orally bioavailable, non-immunosuppressive cyclophilin (Cyp) inhibitor with potent anti-hepatitis C virus (HCV) activity that could serve as part of an all oral antiviral combination therapy.  An initial lead (I) derived from the sanglifehrin A macrocycle was optimized using structure based design to produce a potent and orally bioavailable inhibitor (II).  The macrocycle ring size was reduced by one atom, and an internal hydrogen bond drove improved permeability and drug-like properties. II demonstrates potent Cyp inhibition (Kd = 5 nM), potent anti-HCV 2a activity (EC50 = 98 nM), and high oral bioavailability in rat (100%) and dog (55%).  The synthetic accessibility and properties of II support its potential as an anti-HCV agent and for interrogating the role of Cyp inhibition in a variety of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEpEZcwXK8z7Vg90H21EOLACvtfcHk0ljq6EVkGu9QZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI&md5=616cf72e0d2766e159769166c98a9e86</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00802%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DAppleby%26aufirst%3DT.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBlakemore%26aufirst%3DC.%2BA.%26aulast%3DCai%26aufirst%3DR.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DChin%26aufirst%3DG.%26aulast%3DChiva%26aufirst%3DJ.%2BC.%26aulast%3DDunbar%26aufirst%3DN.%2BA.%26aulast%3DFliri%26aufirst%3DH.%26aulast%3DHighton%26aufirst%3DA.%2BJ.%26aulast%3DHui%26aufirst%3DH.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DMish%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPyun%26aufirst%3DP.%26aulast%3DSangi%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSanvoisin%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DSchrier%26aufirst%3DA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520orally%2520bioavailable%2520cyclophilin%2520inhibitor%2520derived%2520from%2520the%2520sanglifehrin%2520macrocycle%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9473%26epage%3D9499%26doi%3D10.1021%2Facs.jmedchem.8b00802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girao da Cruz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhazes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, M. P. M.</span></span> <span> </span><span class="NLM_article-title">Anticancer activity of phenolic acids of natural or synthetic origin: a structure-activity study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5395</span>– <span class="NLM_lpage">5401</span>, <span class="refDoi"> DOI: 10.1021/jm030956v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030956v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVClt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5395-5401&author=C.+A.+Gomesauthor=T.+Girao+da+Cruzauthor=J.+L.+Andradeauthor=N.+Milhazesauthor=F.+Borgesauthor=M.+P.+M.+Marques&title=Anticancer+activity+of+phenolic+acids+of+natural+or+synthetic+origin%3A+a+structure-activity+study&doi=10.1021%2Fjm030956v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer Activity of Phenolic Acids of Natural or Synthetic Origin: A Structure-Activity Study</span></div><div class="casAuthors">Gomes, Catarina A.; Girao da Cruz, Teresa; Andrade, Jose L.; Milhazes, Nuno; Borges, Fernanda; Marques, M. Paula M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5395-5401</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several phenolic acids-caffeic and gallic acid derivs.-were synthesized and screened for their potential antiproliferative and cytotoxic properties, in different human cancer cell lines: mammary gland and cervix adenocarcinomas and lymphoblastic leukemia.  The selected phenols were structurally related, which allowed us to gather important information regarding the structure-activity relationships underlying the biol. activity of such compds.  This is proposed to be due to a balance between the antioxidant and pro-oxidant properties of this kind of agent.  Distinct effects were found for different cell lines, which points to a significant specificity of action of the drugs tested.  It was verified, for the types of cancer investigated, that the trihydroxylated derivs. yielded better results than the dihydroxylated ones.  Tests in noncancerous cells, human lung fibroblasts, were also undertaken, in view of detg. the toxic side effects of the compds. studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEnWoIoYj87Vg90H21EOLACvtfcHk0lg96QspKPkwDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVClt7g%253D&md5=c079f974dca4ef1c55bfd29bfef10dc9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm030956v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030956v%26sid%3Dliteratum%253Aachs%26aulast%3DGomes%26aufirst%3DC.%2BA.%26aulast%3DGirao%2Bda%2BCruz%26aufirst%3DT.%26aulast%3DAndrade%26aufirst%3DJ.%2BL.%26aulast%3DMilhazes%26aufirst%3DN.%26aulast%3DBorges%26aufirst%3DF.%26aulast%3DMarques%26aufirst%3DM.%2BP.%2BM.%26atitle%3DAnticancer%2520activity%2520of%2520phenolic%2520acids%2520of%2520natural%2520or%2520synthetic%2520origin%253A%2520a%2520structure-activity%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5395%26epage%3D5401%26doi%3D10.1021%2Fjm030956v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span>; <span class="NLM_string-name">Zhang, X.</span></span> <i>Phenylacetic acid derivative and antitumor use thereof</i>. World Patent <span class="NLM_patent">2015/018182 A1</span>, February 2, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Li&author=X.+Zhang&title=Phenylacetic+acid+derivative+and+antitumor+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26jtitle%3DPhenylacetic%2520acid%2520derivative%2520and%2520antitumor%2520use%2520thereof%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bols, M.</span></span> <span> </span><span class="NLM_article-title">An inexpensive and scalable synthesis of Shld</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">6050</span>– <span class="NLM_lpage">6055</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1MfitFajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=6050-6055&author=F.+P.+Jorgensenauthor=M.+Bols&title=An+inexpensive+and+scalable+synthesis+of+Shld&doi=10.1021%2Facs.joc.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">An Inexpensive and Scalable Synthesis of Shld</span></div><div class="casAuthors">Jorgensen Frederik Praestholm; Bols Mikael</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of organic chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6050-6055</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A synthesis of the important FKBP ligand Shld is reported.  The synthesis avoids stoichiometric use of expensive and chiral reagents, maintains enantioselectivity and provides a high overall yield (39%).  The main features in the method are enantioselective alkylation for preparation of the phenyl acetic acid moiety (building block A), catalytic enantioselective reduction for obtaining the chiral diaryl-1-propanol (building block C), and direct acylation of S-pipecolic tartrate rather than use of expensive Fmoc-pipecolic acid.  The assembly of the building blocks A-C is reversed in comparison to previous synthesis, which also eliminates the need for protective groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVlUzHyK4uGrrIbt-tGZmxfW6udTcc2eYyuMB9O6I4cLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfitFajsA%253D%253D&md5=e934f7eb001e353f68552fa0bdb22568</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DF.%2BP.%26aulast%3DBols%26aufirst%3DM.%26atitle%3DAn%2520inexpensive%2520and%2520scalable%2520synthesis%2520of%2520Shld%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2018%26volume%3D83%26spage%3D6050%26epage%3D6055%26doi%3D10.1021%2Facs.joc.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, E. J.</span></span> <span> </span><span class="NLM_article-title">Enantioselective total synthesis of aspidophytine</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">6771</span>– <span class="NLM_lpage">6772</span>, <span class="refDoi"> DOI: 10.1021/ja9915201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9915201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADyaK1MXktVahsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=6771-6772&author=F.+Heauthor=Y.+Boauthor=J.+D.+Altomauthor=E.+J.+Corey&title=Enantioselective+total+synthesis+of+aspidophytine&doi=10.1021%2Fja9915201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective Total Synthesis of Aspidophytine</span></div><div class="casAuthors">He, Feng; Bo, Yunxin; Altom, Jason D.; Corey, E. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6771-6772</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The alkaloid aspidophytine was prepd. via the coupling of I and II that yielded the pentacyclic synthon III which followed by base catalyzed hydrolysis and oxidative lactonization provided the title compd. in 86% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5yFwvxG0WhrVg90H21EOLACvtfcHk0lhOgfWKCpZ93w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVahsro%253D&md5=1407fccbeb944f8babc7d7b6497aaea1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja9915201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9915201%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DF.%26aulast%3DBo%26aufirst%3DY.%26aulast%3DAltom%26aufirst%3DJ.%2BD.%26aulast%3DCorey%26aufirst%3DE.%2BJ.%26atitle%3DEnantioselective%2520total%2520synthesis%2520of%2520aspidophytine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D6771%26epage%3D6772%26doi%3D10.1021%2Fja9915201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morandi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickens, Z. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Regioselective Wacker oxidation of internal alkenes: rapid access to functionalized ketones facilitated by cross-metathesis</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">9751</span>– <span class="NLM_lpage">9754</span>, <span class="refDoi"> DOI: 10.1002/anie.201303587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fanie.201303587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhs7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=9751-9754&author=B.+Morandiauthor=Z.+K.+Wickensauthor=R.+H.+Grubbs&title=Regioselective+Wacker+oxidation+of+internal+alkenes%3A+rapid+access+to+functionalized+ketones+facilitated+by+cross-metathesis&doi=10.1002%2Fanie.201303587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective Wacker oxidation of internal alkenes: rapid access to functionalized ketones facilitated by cross-metathesis</span></div><div class="casAuthors">Morandi, Bill; Wickens, Zachary K.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9751-9754</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A new synthetic strategy to regioselectively access complex ketone products from simple starting materials is described.  A wide range of functionalized alkenes were established as efficient substrates for a regioseνlective Wacker oxidn., thus producing valuable synthetic intermediates (hydroxyketones, aminoketones, ketoesters).  It was demonstrated using both E and Z alkene isomers prepd. by cross-metathesis could be use as the starting materials for this regioselective Wacker oxidn.  An intriguing long-range inductive dependence was shown by competition expts.  The high functional-group tolerance of both the cross-metathesis step and the Wacker oxidn., combined with the predictable regioselectivity of the oxidn. step, holds great promise in the wide adoption of this strategy in org. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroWQSR7iPhhrVg90H21EOLACvtfcHk0lhOgfWKCpZ93w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhs7zN&md5=4d9a66e0d674a7935038f9f45a07cefb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303587%26sid%3Dliteratum%253Aachs%26aulast%3DMorandi%26aufirst%3DB.%26aulast%3DWickens%26aufirst%3DZ.%2BK.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DRegioselective%2520Wacker%2520oxidation%2520of%2520internal%2520alkenes%253A%2520rapid%2520access%2520to%2520functionalized%2520ketones%2520facilitated%2520by%2520cross-metathesis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D9751%26epage%3D9754%26doi%3D10.1002%2Fanie.201303587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span> <span> </span><span class="NLM_article-title">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors as potential antidiabetic agents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5468</span>– <span class="NLM_lpage">5479</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bmc.2011.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21868239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKhsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=5468-5479&author=M.+J.+Kimauthor=S.+H.+Leeauthor=S.+O.+Parkauthor=H.+Kangauthor=J.+S.+Leeauthor=K.+N.+Leeauthor=M.+E.+Jungauthor=J.+Kimauthor=J.+Lee&title=Novel+macrocyclic+C-aryl+glucoside+SGLT2+inhibitors+as+potential+antidiabetic+agents&doi=10.1016%2Fj.bmc.2011.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors as potential antidiabetic agents</span></div><div class="casAuthors">Kim, Min Ju; Lee, Suk Ho; Park, So Ok; Kang, Hyunku; Lee, Jun Sung; Lee, Ki Nam; Jung, Myung Eun; Kim, Jeongmin; Lee, Jinhwa</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5468-5479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors were designed and synthesized.  Two different synthetic routes of macrocyclization were adopted to prep. novel ansa SGLT2 inhibitors.  Among the compds. tested, [1,7]dioxacyclopentadecine macrocycles possessing methylthiophenyl at the distal ring 40 or ethoxyphenyl at the distal ring 23 showed the best in vitro inhibitory activity in this series to date (40, IC50 = 0.778 nM and 23, IC50 = 0.899 nM) against hSGLT2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2MFX8lSt4bVg90H21EOLACvtfcHk0lj4lBFZT7Mk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKhsr7I&md5=28491049fd76513d3fac62d8215448fc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BO.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DK.%2BN.%26aulast%3DJung%26aufirst%3DM.%2BE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DNovel%2520macrocyclic%2520C-aryl%2520glucoside%2520SGLT2%2520inhibitors%2520as%2520potential%2520antidiabetic%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D5468%26epage%3D5479%26doi%3D10.1016%2Fj.bmc.2011.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Fluorescent probes to characterise FK506-binding proteins</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1402</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1002/cbic.200800806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fcbic.200800806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=19418507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtlOlsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=1402-1410&author=C.+Kozanyauthor=A.+Marzauthor=C.+Kressauthor=F.+Hausch&title=Fluorescent+probes+to+characterise+FK506-binding+proteins&doi=10.1002%2Fcbic.200800806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent Probes to Characterise FK506-Binding Proteins</span></div><div class="casAuthors">Kozany, Christian; Maerz, Andreas; Kress, Christoph; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1402-1410</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Talented all-rounders: Fluorescence polarisation assays were developed for members of the FK506-binding protein family by using fluorescent rapamycin analogs (demonstrated in the figure).  These tracers retain medium to high affinity to all tested proteins (FKBP12, -12.6, -13, -25, -51, -52).  They can be used for active-site titrns., competition assays with unlabeled ligands and enable a robust, miniaturized assay adequate for high-throughput screening.  FK506-binding proteins (FKBPs) convey the immunosuppressive action of FK506 and rapamycin and mediate the neuroprotective properties of these compds., and participate in the regulation of calcium channels.  In addn., the larger homologues FKBP51 and FKBP52 act as cochaperones for Hsp90 and regulate the transactivational activity of steroid hormone receptors.  To further characterize these FKBPs, we have synthesized fluorescein-coupled rapamycin analogs.  In fluorescence polarization assays one of these compds. retained high affinity to all tested proteins (Kd: 0.1-20 nM) and could be used for active-site titrns.  To adapt the fluorescence polarization assay for high-throughput purposes, a simplified rapamycin deriv. was synthesized and labeled with fluorescein.  This probe showed moderate affinity for the FK1 domains of FKBP51 (177 nM) and FKBP52 (469 nM) and allowed a highly robust, optimized, miniaturized assay (Z'>0.7) sufficient for high-throughput screening of large compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQIvLuUjAKDbVg90H21EOLACvtfcHk0liltm327NyUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtlOlsro%253D&md5=57f00a9d8116c9bc8007f34c421c7dee</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200800806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200800806%26sid%3Dliteratum%253Aachs%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DMarz%26aufirst%3DA.%26aulast%3DKress%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DFluorescent%2520probes%2520to%2520characterise%2520FK506-binding%2520proteins%26jtitle%3DChemBioChem%26date%3D2009%26volume%3D10%26spage%3D1402%26epage%3D1410%26doi%3D10.1002%2Fcbic.200800806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pomplun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klages, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unal, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preiser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Chemogenomic profiling of human and microbial FK506-binding proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3660</span>– <span class="NLM_lpage">3673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00137</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00137" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3660-3673&author=S.+Pomplunauthor=C.+Sippelauthor=A.+Hahleauthor=D.+Tayauthor=K.+Shimaauthor=A.+Klagesauthor=C.+M.+Unalauthor=B.+Riessauthor=H.+T.+Tohauthor=G.+Hansenauthor=H.+S.+Yoonauthor=A.+Bracherauthor=P.+Preiserauthor=J.+Ruppauthor=M.+Steinertauthor=F.+Hausch&title=Chemogenomic+profiling+of+human+and+microbial+FK506-binding+proteins&doi=10.1021%2Facs.jmedchem.8b00137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic Profiling of Human and Microbial FK506-Binding Proteins</span></div><div class="casAuthors">Pomplun, Sebastian; Sippel, Claudia; Haehle, Andreas; Tay, Donald; Shima, Kensuke; Klages, Alina; Uenal, Can Murat; Riess, Benedikt; Toh, Hui Ting; Hansen, Guido; Yoon, Ho Sup; Bracher, Andreas; Preiser, Peter; Rupp, Jan; Steinert, Michael; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3660-3673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FK506-binding proteins (FKBPs) are evolutionarily conserved proteins that display peptidyl-prolyl isomerase activities and act as coreceptors for immunosuppressants.  Microbial macrophage-infectivity-potentiator (Mip)-type FKBPs can enhance infectivity.  However, developing druglike ligands for FKBPs or Mips has proven difficult, and many FKBPs and Mips still lack biol. useful ligands.  To explore the scope and potential of C5-substituted [4.3.1]-aza-bicyclic sulfonamides as a broadly applicable class of FKBP inhibitors, we developed a new synthesis method for the bicyclic core scaffold and used it to prep. an FKBP- and Mip-focused library.  This allowed us to perform a systematic structure-activity-relationship anal. across key human FKBPs and microbial Mips, yielding highly improved inhibitors for all the FKBPs studied.  A cocrystal structure confirmed the mol.-binding mode of the core structure and explained the affinity gained as a result of the preferred substituents.  The best FKBP and Mip ligands showed promising antimalarial, antileginonellal, and antichlamydial properties in cellular models of infectivity, suggesting that substituted [4.3.1]-aza-bicyclic sulfonamides could be a novel class of anti-infectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_KYmjlu3wrrVg90H21EOLACvtfcHk0liltm327NyUvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVWru7o%253D&md5=1961bd1a6f7aa868b5dd58319600b840</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00137%26sid%3Dliteratum%253Aachs%26aulast%3DPomplun%26aufirst%3DS.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DTay%26aufirst%3DD.%26aulast%3DShima%26aufirst%3DK.%26aulast%3DKlages%26aufirst%3DA.%26aulast%3DUnal%26aufirst%3DC.%2BM.%26aulast%3DRiess%26aufirst%3DB.%26aulast%3DToh%26aufirst%3DH.%2BT.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DPreiser%26aufirst%3DP.%26aulast%3DRupp%26aufirst%3DJ.%26aulast%3DSteinert%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DChemogenomic%2520profiling%2520of%2520human%2520and%2520microbial%2520FK506-binding%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3660%26epage%3D3673%26doi%3D10.1021%2Facs.jmedchem.8b00137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thost, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Structural characterization of the PPIase domain of FKBP51, a cochaperone of human Hsp90</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1107/S0907444911013862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS0907444911013862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21636895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFyrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=549-559&author=A.+Bracherauthor=C.+Kozanyauthor=A.+K.+Thostauthor=F.+Hausch&title=Structural+characterization+of+the+PPIase+domain+of+FKBP51%2C+a+cochaperone+of+human+Hsp90&doi=10.1107%2FS0907444911013862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of the PPIase domain of FKBP51, a cochaperone of human Hsp90</span></div><div class="casAuthors">Bracher, Andreas; Kozany, Christian; Thost, Ann Katrin; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">549-559</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Steroid hormone receptors are key components of mammalian stress and sex hormone systems.  Many of them rely on the Hsp90 chaperone system for full function and are further fine-tuned by Hsp90-assocd. peptidyl-prolyl isomerases such as FK506-binding proteins 51 and 52.  FK506-binding protein 51 (FKBP51) has been shown to reduce glucocorticoid receptor signaling and has been genetically assocd. with human stress resilience and with numerous psychiatric disorders.  The peptidyl-prolyl isomerase domain of FKBP51 contains a high-affinity binding site for the natural products FK506 and rapamycin and has further been shown to convey most of the inhibitory activity on the glucocorticoid receptor.  FKBP51 has therefore become a prime new target for the treatment of stress-related affective disorders that could be amenable to structure-based drug design.  Here, a series of high-resoln. structures of the peptidyl-prolyl isomerase domain of FKBP51 as well as a cocrystal structure with the prototypic ligand FK506 are described.  These structures provide a detailed picture of the drug-binding domain of FKBP51 and the mol. binding mode of its ligand as a starting point for the rational design of improved inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2WGAZjThWGLVg90H21EOLACvtfcHk0lgB9_n_EDh2sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFyrsLg%253D&md5=3ec5288a9394ab264678ace99e0fd5f7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444911013862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911013862%26sid%3Dliteratum%253Aachs%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DThost%26aufirst%3DA.%2BK.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DStructural%2520characterization%2520of%2520the%2520PPIase%2520domain%2520of%2520FKBP51%252C%2520a%2520cochaperone%2520of%2520human%2520Hsp90%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D549%26epage%3D559%26doi%3D10.1107%2FS0907444911013862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babine, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S. L.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of protein-ligand complexes: applications to drug design</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1021/cr960370z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr960370z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADyaK2sXksFSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=1359-1472&author=R.+E.+Babineauthor=S.+L.+Bender&title=Molecular+recognition+of+protein-ligand+complexes%3A+applications+to+drug+design&doi=10.1021%2Fcr960370z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein-Ligand Complexes: Applications to Drug Design</span></div><div class="casAuthors">Babine, Robert E.; Bender, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1359-1472</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 538 refs.  This article provides a comprehensive discussion of the interactions of small mol. ligands with protein receptors.  It covers the literature of the structures of complexes between small mol. ligands and several enzymes which are relevant as drug targets.  Protein targets which are not enzymes are beyond the scope of this review.  It covers about 20 yr of the literature ending in Sept. 1996; however, most of the relevant work has been published within the last five years.  Whenever possible the relevant Brookhaven Protein Data Bank (PDB) entry has been given.  A major goal of this review is to enhance the qual. understanding of protein-ligand complexes.  A major portion of this review provides an in-depth discussion of protease inhibitors.  Four different mechanistic classes of enzymes that catalyze the same reaction, selective amide hydrolysis within a protein or polypeptide framework, are discussed.  Successful strategies for inhibiting these enzymes vary from class to class and reflect the mechanistic and structural differences of the enzymes.  These enzymes have evolved to recognize proteins and polypeptides.  While inhibition strategies based upon the preferred cleavage sites are now straightforward, they result in inhibitors which are peptide-like in structure.  Peptides and peptidic compds. are usually unsuitable as drug candidates.  This has resulted in the need for compds. which act as peptide mimetics.  Many of the structures described will illustrate the structural basis for peptide mimicry.  There have been major research efforts to develop drug candidates as inhibitors of two different aspartic protease enzymes, renin and HIV-1 protease.  In the search for renin inhibitors the structure of the target protein was not known during most of this research.  As a result, traditional medicinal chem. approaches were used in an attempt to prep. drug candidates.  To date there have been no renin inhibitors which are successful drug candidates.  In contrast, during the search for inhibitors of HIV-1 protease, the structure of the protein was known and many groups have used an iterative structure-based design process.  This has resulted in many structurally distinct classes of inhibitors.  Several successful drug candidates have emerged from these studies and the use of HIV-1 protease structure in drug design represents one of the more impressive success stories in this emerging field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM7ZsftkTf4rVg90H21EOLACvtfcHk0lgB9_n_EDh2sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFSisLk%253D&md5=b533a1fbdec87724d3ec714f434afd07</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcr960370z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr960370z%26sid%3Dliteratum%253Aachs%26aulast%3DBabine%26aufirst%3DR.%2BE.%26aulast%3DBender%26aufirst%3DS.%2BL.%26atitle%3DMolecular%2520recognition%2520of%2520protein-ligand%2520complexes%253A%2520applications%2520to%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D1359%26epage%3D1472%26doi%3D10.1021%2Fcr960370z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malecki, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhringer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rower, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The macromolecular crystallography beamlines at BESSY II of the Helmholtz-Zentrum Berlin: Current status and perspectives</span>. <i>Eur. Phys. J. Plus</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">141</span>, <span class="refDoi"> DOI: 10.1140/epjp/i2015-15141-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1140%2Fepjp%2Fi2015-15141-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2015&pages=141&author=U.+Muellerauthor=R.+Forsterauthor=M.+Hellmigauthor=F.+U.+Huschmannauthor=A.+Kastnerauthor=P.+Maleckiauthor=S.+Puhringerauthor=M.+Rowerauthor=K.+Spartaauthor=M.+Steffienauthor=M.+Uhleinauthor=P.+Wilkauthor=M.+S.+Weiss&title=The+macromolecular+crystallography+beamlines+at+BESSY+II+of+the+Helmholtz-Zentrum+Berlin%3A+Current+status+and+perspectives&doi=10.1140%2Fepjp%2Fi2015-15141-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1140%2Fepjp%2Fi2015-15141-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1140%252Fepjp%252Fi2015-15141-2%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DHellmig%26aufirst%3DM.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DKastner%26aufirst%3DA.%26aulast%3DMalecki%26aufirst%3DP.%26aulast%3DPuhringer%26aufirst%3DS.%26aulast%3DRower%26aufirst%3DM.%26aulast%3DSparta%26aufirst%3DK.%26aulast%3DSteffien%26aufirst%3DM.%26aulast%3DUhlein%26aufirst%3DM.%26aulast%3DWilk%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520macromolecular%2520crystallography%2520beamlines%2520at%2520BESSY%2520II%2520of%2520the%2520Helmholtz-Zentrum%2520Berlin%253A%2520Current%2520status%2520and%2520perspectives%26jtitle%3DEur.%2520Phys.%2520J.%2520Plus%26date%3D2015%26volume%3D130%26spage%3D141%26doi%3D10.1140%2Fepjp%2Fi2015-15141-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Briese, C.</span></span> <span> </span><span class="NLM_article-title">Optimal fine phi-slicing for single-photon-counting pixel detectors</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1107/S0907444911049833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS0907444911049833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22194332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=42-56&author=M.+Muellerauthor=M.+Wangauthor=C.+Schulze-Briese&title=Optimal+fine+phi-slicing+for+single-photon-counting+pixel+detectors&doi=10.1107%2FS0907444911049833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal fine .vphi.-slicing for single-photon-counting pixel detectors</span></div><div class="casAuthors">Mueller, Marcus; Wang, Meitian; Schulze-Briese, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-56</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Fine .vphi.-slicing substantially improves scaling statistics and anomalous signal for diffraction data collection with hybrid pixel detectors.  The data-collection parameters used in a macromol. diffraction expt. have a strong impact on data quality.  A careful choice of parameters leads to better data and can make the difference between success and failure in phasing attempts, and will also result in a more accurate at. model.  The selection of parameters has to account for the application of the data in various phasing methods or high-resoln. refinement.  Furthermore, exptl. factors such as crystal characteristics, available expt. time and the properties of the X-ray source and detector have to be considered.  For many years, CCD detectors have been the prevalent type of detectors used in macromol. crystallog.  Recently, hybrid pixel X-ray detectors that operate in single-photon-counting mode have become available.  These detectors have fundamentally different characteristics compared with CCD detectors and different data-collection strategies should be applied.  Fine .vphi.-slicing is a strategy that is particularly well suited to hybrid pixel detectors because of the fast readout time and the absence of readout noise.  A large no. of data sets were systematically collected from crystals of four different proteins in order to investigate the benefit of fine .vphi.-slicing on data quality with a noise-free detector.  The results show that fine .vphi.-slicing can substantially improve scaling statistics and anomalous signal provided that the rotation angle is comparable to half the crystal mosaicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGjNTwtRdWJrVg90H21EOLACvtfcHk0lgYETXsAXDnzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E&md5=03c8d02288ee79a7f41e187df10d872d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444911049833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911049833%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSchulze-Briese%26aufirst%3DC.%26atitle%3DOptimal%2520fine%2520phi-slicing%2520for%2520single-photon-counting%2520pixel%2520detectors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D42%26epage%3D56%26doi%3D10.1107%2FS0907444911049833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompidor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpics, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cipriani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiedler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span> <span> </span><span class="NLM_article-title">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high-and low-energy phasing with variable beam focusing</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1107/S1600577516016465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS1600577516016465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=28009574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=323-332&author=M.+Cianciauthor=G.+Bourenkovauthor=G.+Pompidorauthor=I.+Karpicsauthor=J.+Kallioauthor=I.+Bentoauthor=M.+Roessleauthor=F.+Ciprianiauthor=S.+Fiedlerauthor=T.+R.+Schneider&title=P13%2C+the+EMBL+macromolecular+crystallography+beamline+at+the+low-emittance+PETRA+III+ring+for+high-and+low-energy+phasing+with+variable+beam+focusing&doi=10.1107%2FS1600577516016465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span></div><div class="casAuthors">Cianci, Michele; Bourenkov, Gleb; Pompidor, Guillaume; Karpics, Ivars; Kallio, Johanna; Bento, Isabel; Roessle, Manfred; Cipriani, Florent; Fiedler, Stefan; Schneider, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. P13 beamline is part of the European Mol. Biol. Lab. Integrated Facility for Structural Biol. at PETRA III (DESY, Hamburg, Germany) and has been in user operation since mid-2013.  P13 is tunable across the energy range from 4 to 17.5 keV to support crystallog. data acquisition exploiting a wide range of elemental absorption edges for exptl. phase detn.  An adaptive Kirkpatrick-Baez focusing system provides an X-ray beam with a high photon flux and tunable focus size to adapt to diverse exptl. situations.  Data collections at energies as low as 4 keV (λ = 3.1 Å) are possible due to a beamline design minimizing background and maximizing photon flux particularly at low energy (up to 1011 photons s-1 at 4 keV), a custom calibration of the PILATUS 6M-F detector for use at low energies, and the availability of a helium path.  At high energies, the high photon flux (5.4 × 1011 photons s-1 at 17.5 keV) combined with a large area detector mounted on a 2θ arm allows data collection to sub-at. resoln. (0.55 Å).  A peak flux of about 8.0 × 1012 photons s-1 is reached at 11 keV.  Automated sample mounting is available by means of the robotic sample changer 'MARVIN' with a dewar capacity of 160 samples.  In close proximity to the beamline, labs. have been set up for sample prepn. and characterization; a lab. specifically equipped for on-site heavy atom derivatization with a library of more than 150 compds. is available to beamline users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYDoilsi_YbVg90H21EOLACvtfcHk0lh2q0x4_REx3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D&md5=807e7e497aa2b5f524a4366caa645ee6</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS1600577516016465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577516016465%26sid%3Dliteratum%253Aachs%26aulast%3DCianci%26aufirst%3DM.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DPompidor%26aufirst%3DG.%26aulast%3DKarpics%26aufirst%3DI.%26aulast%3DKallio%26aufirst%3DJ.%26aulast%3DBento%26aufirst%3DI.%26aulast%3DRoessle%26aufirst%3DM.%26aulast%3DCipriani%26aufirst%3DF.%26aulast%3DFiedler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26atitle%3DP13%252C%2520the%2520EMBL%2520macromolecular%2520crystallography%2520beamline%2520at%2520the%2520low-emittance%2520PETRA%2520III%2520ring%2520for%2520high-and%2520low-energy%2520phasing%2520with%2520variable%2520beam%2520focusing%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2017%26volume%3D24%26spage%3D323%26epage%3D332%26doi%3D10.1107%2FS1600577516016465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas M.  Voll</span>, <span class="hlFld-ContribAuthor ">Christian  Meyners</span>, <span class="hlFld-ContribAuthor ">Martha C.  Taubert</span>, <span class="hlFld-ContribAuthor ">Thomas  Bajaj</span>, <span class="hlFld-ContribAuthor ">Tim  Heymann</span>, <span class="hlFld-ContribAuthor ">Stephanie  Merz</span>, <span class="hlFld-ContribAuthor ">Anna  Charalampidou</span>, <span class="hlFld-ContribAuthor ">Jürgen  Kolos</span>, <span class="hlFld-ContribAuthor ">Patrick L.  Purder</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Geiger</span>, <span class="hlFld-ContribAuthor ">Pablo  Wessig</span>, <span class="hlFld-ContribAuthor ">Nils C.  Gassen</span>, <span class="hlFld-ContribAuthor ">Andreas  Bracher</span>, <span class="hlFld-ContribAuthor ">Felix  Hausch</span>. </span><span class="cited-content_cbyCitation_article-title">Makrozyklische FKBP51‐Liganden enthüllen einen transienten Bindungsmodus mit erhöhter Selektivität. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (24)
                                     , 13366-13372. <a href="https://doi.org/10.1002/ange.202017352" title="DOI URL">https://doi.org/10.1002/ange.202017352</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202017352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202017352%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DMakrozyklische%252BFKBP51%2525E2%252580%252590Liganden%252Benth%2525C3%2525BCllen%252Beinen%252Btransienten%252BBindungsmodus%252Bmit%252Berh%2525C3%2525B6hter%252BSelektivit%2525C3%2525A4t%26aulast%3DVoll%26aufirst%3DAndreas%2BM.%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D24%26spage%3D13366%26epage%3D13372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas M.  Voll</span>, <span class="hlFld-ContribAuthor ">Christian  Meyners</span>, <span class="hlFld-ContribAuthor ">Martha C.  Taubert</span>, <span class="hlFld-ContribAuthor ">Thomas  Bajaj</span>, <span class="hlFld-ContribAuthor ">Tim  Heymann</span>, <span class="hlFld-ContribAuthor ">Stephanie  Merz</span>, <span class="hlFld-ContribAuthor ">Anna  Charalampidou</span>, <span class="hlFld-ContribAuthor ">Jürgen  Kolos</span>, <span class="hlFld-ContribAuthor ">Patrick L.  Purder</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Geiger</span>, <span class="hlFld-ContribAuthor ">Pablo  Wessig</span>, <span class="hlFld-ContribAuthor ">Nils C.  Gassen</span>, <span class="hlFld-ContribAuthor ">Andreas  Bracher</span>, <span class="hlFld-ContribAuthor ">Felix  Hausch</span>. </span><span class="cited-content_cbyCitation_article-title">Macrocyclic FKBP51 Ligands Define a Transient Binding Mode with Enhanced Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (24)
                                     , 13257-13263. <a href="https://doi.org/10.1002/anie.202017352" title="DOI URL">https://doi.org/10.1002/anie.202017352</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202017352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202017352%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DMacrocyclic%252BFKBP51%252BLigands%252BDefine%252Ba%252BTransient%252BBinding%252BMode%252Bwith%252BEnhanced%252BSelectivity%26aulast%3DVoll%26aufirst%3DAndreas%2BM.%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D24%26spage%3D13257%26epage%3D13263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Chemical structure of the selective FKBP51 ligands SAFit1 and SAFit2 and the macrocyclization strategy. (B) SAFit2 (red sticks) in complex with FKBP51 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a>); key interactions are highlighted (black dotted lines; distances in Å), and the macrocyclization strategy is outlined. Lys121 has been omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <i>para</i>-Aryl-Substituted Cyclization Precursors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: a) TBDPSCl, imidazole, DCM/DMF, rt; then K<sub>2</sub>CO<sub>3</sub>, THF/H<sub>2</sub>O, rt, 93%; b) C<sub>6</sub>F<sub>5</sub>OH, EDC, 4-DMAP, DCM, rt, 92%; c) (<i>S</i>)-4-phenyloxazolidin-2-one, <i>n</i>-BuLi, THF, 0 °C - rt, 81%; d) 3-bromocyclohexene, LiHMDS, THF, −78 °C - rt, 50%; e) Pd/C, H<sub>2</sub>, CH/EA, rt, 99%; f) LiOH, H<sub>2</sub>O<sub>2</sub>, THF/H<sub>2</sub>O, rt, 95%; g) <b>15</b>, HATU, DIPEA, DCM/DMF, rt, 88%; h) TBAF, THF, 0 °C - rt, 95%; (i) Bromide <i>or</i> tosylate, K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 92–98%; j) Allyl bromide, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 96%; k) (<i>S</i>)-Fmoc-Pip–OH, DCC, 4-DMAP, DCM, 0 °C - rt; then 4-methylpiperidine, DCM, rt, 67% (2 steps).</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>meta</i>-Aryl-Substituted Cyclization Precursors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 93%; (b) 3-bromocyclohexene, LiHMDS, THF, −78 °C–rt, 94%; (c) LiOH, THF/H<sub>2</sub>O, rt, 91%; (d) SOCl<sub>2</sub>, DCM, reflux; then L-Pipecolic acid, KOH aq, 1,4-dioxane, 0 °C–rt, 71% + 28% of recovered substrate <b>21</b>; (e) Pd/C, H<sub>2</sub>, THF/MeOH, rt, 44% for <b>22b</b>, 41% for <b>22a</b>, separation of isomers by flash column chromatography; (f) bromide or tosylate, K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 87–99%; (g) LiOH, THF/MeOH/H<sub>2</sub>O, rt, 90–97%; (h) <b>14</b>, EDC, 4-(pyrrolidin-1-yl)pyridine, toluene, 0 °C–rt, 59–64%.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Pathways Towards Macrocyclic FKBP51 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Grubbs second generation catalyst (C<sub>46</sub>H<sub>65</sub>Cl<sub>2</sub>N<sub>2</sub>PRu), DCM, reflux, 33–78%; (b) Pd/C, H<sub>2</sub>, MeOH, rt, 37–45% (for <b>39</b> and <b>40</b>); (c) RhCl(PPh<sub>3</sub>)<sub>3</sub>, H<sub>2</sub>, toluene, or MeOH/THF, rt, 66–87%, (for <b>41</b>–<b>46</b>); (d) RhCl(PPh<sub>3</sub>)<sub>3</sub>, DABCO, EtOH/H<sub>2</sub>O, reflux, 16–47%; (e) I<sub>2</sub>, PPh<sub>3</sub>, imidazole, toluene or DCM, rt, 38–85%; (f) K<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 26–81%; (g) Pd(OAc)<sub>2</sub>, <i>p</i>-BQ, HBF<sub>4</sub>, MeCN/H<sub>2</sub>O, rt, 33–67%; (h) OsO<sub>4</sub>, NMO, H<sub>2</sub>O/acetone, 0 °C–rt, 26–40%.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structures of the FK506-binding domain of FKBP51 (gray surface) in complex with the macrocyclic ligands <b>55</b> (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX">7AWX</a>, green sticks) and <b>64a</b> (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a>, blue sticks). Lys121 has been omitted for clarity. Hydrogen bonds to Ile87 and Tyr113 (yellow) are indicated as black dotted lines (distances in Å). Phe67 (yellow) has been displaced by the cyclohexyl moiety upon binding of the ligands. Novel linker moieties are highlighted in bright green and dark blue, respectively. Linker oxygens are highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/medium/jm0c02195_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of the FK506-binding domain of FKBP51 (gray surface) in complex with the macrocyclic ligands <b>35-(</b><i>E</i><b>)</b> (A, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z">7B9Z</a>, yellow sticks) and <b>64a</b> (B, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a>, blue sticks) superposed on the cocrystal structure of SAFit2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX">6TXX</a>, red sticks). Lys121 has been omitted for clarity. Novel linker moieties and rings A, B and C are highlighted in orange and dark blue, respectively. Linker oxygens are highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02195/20210319/images/large/jm0c02195_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02195&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04638" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04638" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, E. R.</span></span> <span> </span><span class="NLM_article-title">Chaperoning steroidal physiology: Lessons from mouse genetic models of Hsp90 and its cochaperones</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>1823</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">729</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2011.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bbamcr.2011.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22155719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFSgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1823&publication_year=2012&pages=722-729&author=E.+R.+Sanchez&title=Chaperoning+steroidal+physiology%3A+Lessons+from+mouse+genetic+models+of+Hsp90+and+its+cochaperones&doi=10.1016%2Fj.bbamcr.2011.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chaperoning steroidal physiology: Lessons from mouse genetic models of Hsp90 and its cochaperones</span></div><div class="casAuthors">Sanchez, Edwin R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">1823</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-729</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The mol. chaperone Hsp90 is abundant, ubiquitous, and catholic to biol. processes in eukaryotes, controlling phosphorylation cascades, protein stability and turnover, client localization and trafficking, and ligand-receptor interactions.  Not surprisingly, Hsp90 does not accomplish these activities alone.  Instead, an ever-growing no. of cochaperones have been identified, leading to an explosion of reports on their mol. and cellular effects on Hsp90 chaperoning of client substrates.  Notable among these clients are many members of the steroid receptor family, such as glucocorticoid, androgen, estrogen and progesterone receptors.  Cochaperones typically assocd. with the mature, hormone-competent states of these receptors include p23, the FK506-binding protein 52 (FKBP52), FKBP51, protein phosphatase 5 (PP5) and cyclophilin 40 (Cyp40).  The ultimate relevance of these cochaperones to steroid receptor action depends on their physiol. effects.  In recent years, the first mouse genetic models of these cochaperones have been developed.  This work will review the complex and intriguing phenotypes so far obtained in genetically-altered mice and compare them to the known mol. and cellular impacts of cochaperones on steroid receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJdJEf85X2nrVg90H21EOLACvtfcHk0liIZNllOLwbFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFSgt7s%253D&md5=03195ad9b56e3768e96ad1d5c772f49a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2011.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2011.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez%26aufirst%3DE.%2BR.%26atitle%3DChaperoning%2520steroidal%2520physiology%253A%2520Lessons%2520from%2520mouse%2520genetic%2520models%2520of%2520Hsp90%2520and%2520its%2520cochaperones%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2012%26volume%3D1823%26spage%3D722%26epage%3D729%26doi%3D10.1016%2Fj.bbamcr.2011.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Storer, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galigniana, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span> <span> </span><span class="NLM_article-title">FKBP51 and FKBP52 in signaling and disease</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2011.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.tem.2011.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21889356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=481-490&author=C.+L.+Storerauthor=C.+A.+Dickeyauthor=M.+D.+Galignianaauthor=T.+Reinauthor=M.+B.+Cox&title=FKBP51+and+FKBP52+in+signaling+and+disease&doi=10.1016%2Fj.tem.2011.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP51 and FKBP52 in signaling and disease</span></div><div class="casAuthors">Storer, Cheryl L.; Dickey, Chad A.; Galigniana, Mario D.; Rein, Theo; Cox, Marc B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">481-490</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  FKBP51 and FKBP52 are diverse regulators of steroid hormone receptor signaling, including receptor maturation, hormone binding and nuclear translocation.  Although structurally similar, they are functionally divergent, which is largely attributed to differences in the FK1 domain and the proline-rich loop.  FKBP51 and FKBP52 have emerged as likely contributors to a variety of hormone-dependent diseases, including stress-related diseases, immune function, reproductive functions and a variety of cancers.  In addn., recent studies have implicated FKBP51 and FKBP52 in Alzheimer's disease and other protein aggregation disorders.  This review summarizes our current understanding of FKBP51 and FKBP52 interactions within the receptor-chaperone complex, their contributions to health and disease, and their potential as therapeutic targets for the treatment of these diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaeNToplbXEbVg90H21EOLACvtfcHk0ljObNqa_LeggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKltbnI&md5=235f92790216db4db59d9ea8e9fd166e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2011.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2011.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DStorer%26aufirst%3DC.%2BL.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26aulast%3DGaligniana%26aufirst%3DM.%2BD.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DCox%26aufirst%3DM.%2BB.%26atitle%3DFKBP51%2520and%2520FKBP52%2520in%2520signaling%2520and%2520disease%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2011%26volume%3D22%26spage%3D481%26epage%3D490%26doi%3D10.1016%2Fj.tem.2011.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">The prospect of FKBP51 as a drug target</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1351</span>– <span class="NLM_lpage">1359</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201200137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fcmdc.201200137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22581765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVKlsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1351-1359&author=M.+V.+Schmidtauthor=M.+Paez-Peredaauthor=F.+Holsboerauthor=F.+Hausch&title=The+prospect+of+FKBP51+as+a+drug+target&doi=10.1002%2Fcmdc.201200137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The Prospect of FKBP51 as a Drug Target</span></div><div class="casAuthors">Schmidt, Mathias V.; Paez-Pereda, Marcelo; Holsboer, Florian; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1351-1359</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The FK506 binding protein 51 (FKBP51) is best known as an Hsp90-assocd. co-chaperone that regulates the responsiveness of steroid hormone receptors.  In human genetic assocn. studies, FKBP51 has repeatedly been assocd. with emotion processing and numerous stress-related affective disorderst.  It has also been implicated in contributing to the glucocorticoid hyposensitivity obsd. in New World primates.  More recently, several research groups have consistently shown a protective effect of FKBP51 knockout or knockdown on stress endocrinol. and stress-coping behavior in animal models of depression and anxiety.  The principal druggability of FKBP51 is exemplified by the prototypic FKBP ligands FK506 and rapamycin.  Moreover, FKBP51 is highly suited for X-ray co-crystallog., which should facilitate the rational drug design of improved FKBP51 ligands.  In summary, FKBP51 has emerged as a promising new drug target for stress-related disorders that should be amenable to drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVM2m01bsqHbVg90H21EOLACvtfcHk0ljObNqa_LeggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVKlsbo%253D&md5=12a5779bf7a0571429716e571f43f93f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200137%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DThe%2520prospect%2520of%2520FKBP51%2520as%2520a%2520drug%2520target%26jtitle%3DChemMedChem%26date%3D2012%26volume%3D7%26spage%3D1351%26epage%3D1359%26doi%3D10.1002%2Fcmdc.201200137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyners, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">The Many Faces of FKBP51</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">35</span>, <span class="refDoi"> DOI: 10.3390/biom9010035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.3390%2Fbiom9010035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=35&author=A.+Hahleauthor=S.+Merzauthor=C.+Meynersauthor=F.+Hausch&title=The+Many+Faces+of+FKBP51&doi=10.3390%2Fbiom9010035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fbiom9010035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom9010035%26sid%3Dliteratum%253Aachs%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DMerz%26aufirst%3DS.%26aulast%3DMeyners%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DThe%2520Many%2520Faces%2520of%2520FKBP51%26jtitle%3DBiomolecules%26date%3D2019%26volume%3D9%26spage%3D35%26doi%3D10.3390%2Fbiom9010035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanowski, C. P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letizia Curzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubcakova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flachskamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, M.</span></span> <span> </span><span class="NLM_article-title">Deficiency of FK506-binding protein (FKBP) 51 alters sleep architecture and recovery sleep responses to stress in mice</span>. <i>J. Sleep Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">176</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1111/jsr.12112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Fjsr.12112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=24354785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC2c3nslCgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=176-185&author=S.+Albuauthor=C.+P.+N.+Romanowskiauthor=M.+Letizia+Curziauthor=V.+Jakubcakovaauthor=C.+Flachskammauthor=N.+C.+Gassenauthor=J.+Hartmannauthor=M.+V.+Schmidtauthor=U.+Schmidtauthor=T.+Reinauthor=F.+Holsboerauthor=F.+Hauschauthor=M.+Paez-Peredaauthor=M.+Kimura&title=Deficiency+of+FK506-binding+protein+%28FKBP%29+51+alters+sleep+architecture+and+recovery+sleep+responses+to+stress+in+mice&doi=10.1111%2Fjsr.12112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Deficiency of FK506-binding protein (FKBP) 51 alters sleep architecture and recovery sleep responses to stress in mice</span></div><div class="casAuthors">Albu Stefana; Romanowski Christoph P N; Letizia Curzi M; Jakubcakova Vladimira; Flachskamm Cornelia; Gassen Nils C; Hartmann Jakob; Schmidt Mathias V; Schmidt Ulrike; Rein Theo; Holsboer Florian; Hausch Felix; Paez-Pereda Marcelo; Kimura Mayumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of sleep research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">176-85</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FK506-binding protein 51 (FKBP51) is a co-chaperone of the glucocorticoid receptor, functionally linked to its activity via an ultra-short negative feedback loop.  Thus, FKBP51 plays an important regulatory role in the hypothalamic-pituitary-adrenocortical (HPA) axis necessary for stress adaptation and recovery.  Previous investigations illustrated that HPA functionality is influenced by polymorphisms in the gene encoding FKBP51, which are associated with both increased protein levels and depressive episodes.  Because FKBP51 is a key molecule in stress responses, we hypothesized that its deletion impacts sleep.  To study FKBP51-involved changes in sleep, polysomnograms of FKBP51 knockout (KO) mice and wild-type (WT) littermates were compared at baseline and in the recovery phase after 6-h sleep deprivation (SD) and 1-h restraint stress (RS).  Using another set of animals, the 24-h profiles of hippocampal free corticosterone levels were also determined.  The most dominant effect of FKBP51 deletion appeared as increased nocturnal wake, where the bout length was significantly extended while non-rapid eye movement sleep (NREMS) and rapid eye movement sleep were rather suppressed.  After both SD and RS, FKBP51KO mice exhibited less recovery or rebound sleep than WTs, although slow-wave activity during NREMS was higher in KOs, particularly after SD.  Sleep compositions of KOs were nearly opposite to sleep profiles observed in human depression.  This might result from lower levels of free corticosterone in FKBP51KO mice, confirming reduced HPA reactivity.  The results indicate that an FKBP51 deletion yields a pro-resilience sleep phenotype.  FKBP51 could therefore be a therapeutic target for stress-induced mood and sleep disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTraiwR6zEpya91pGlMMoT5fW6udTcc2eYqLfG1c4w9x7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3nslCgtQ%253D%253D&md5=4add85c5be5fc854830e49c6ce943357</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fjsr.12112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjsr.12112%26sid%3Dliteratum%253Aachs%26aulast%3DAlbu%26aufirst%3DS.%26aulast%3DRomanowski%26aufirst%3DC.%2BP.%2BN.%26aulast%3DLetizia%2BCurzi%26aufirst%3DM.%26aulast%3DJakubcakova%26aufirst%3DV.%26aulast%3DFlachskamm%26aufirst%3DC.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DM.%26atitle%3DDeficiency%2520of%2520FK506-binding%2520protein%2520%2528FKBP%2529%252051%2520alters%2520sleep%2520architecture%2520and%2520recovery%2520sleep%2520responses%2520to%2520stress%2520in%2520mice%26jtitle%3DJ.%2520Sleep%2520Res.%26date%3D2014%26volume%3D23%26spage%3D176%26epage%3D185%26doi%3D10.1111%2Fjsr.12112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebl, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharf, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">The involvement of FK506-binding protein 51 (FKBP5) in the behavioral and neuroendocrine effects of chronic social defeat stress</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2011.07.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.neuropharm.2011.07.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21839098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaktb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2012&pages=332-339&author=J.+Hartmannauthor=K.+V.+Wagnerauthor=C.+Lieblauthor=S.+H.+Scharfauthor=X.+D.+Wangauthor=M.+Wolfauthor=F.+Hauschauthor=T.+Reinauthor=U.+Schmidtauthor=C.+Toumaauthor=J.+Cheung-Flynnauthor=M.+B.+Coxauthor=D.+F.+Smithauthor=F.+Holsboerauthor=M.+B.+Mullerauthor=M.+V.+Schmidt&title=The+involvement+of+FK506-binding+protein+51+%28FKBP5%29+in+the+behavioral+and+neuroendocrine+effects+of+chronic+social+defeat+stress&doi=10.1016%2Fj.neuropharm.2011.07.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The involvement of FK506-binding protein 51 (FKBP5) in the behavioral and neuroendocrine effects of chronic social defeat stress</span></div><div class="casAuthors">Hartmann, Jakob; Wagner, Klaus V.; Liebl, Claudia; Scharf, Sebastian H.; Wang, Xiao-Dong; Wolf, Miriam; Hausch, Felix; Rein, Theo; Schmidt, Ulrike; Touma, Chadi; Cheung-Flynn, Joyce; Cox, Marc B.; Smith, David F.; Holsboer, Florian; Mueller, Marianne B.; Schmidt, Mathias V.</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">332-339</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Chronic stress is increasingly considered to be a main risk factor for the development of a variety of psychiatric diseases such as depression.  This is further supported by an impaired neg. feedback of the hypothalamic-pituitary-adrenal (HPA) axis, which has been obsd. in the majority of depressed patients.  The effects of glucocorticoids, the main hormonal endpoint of the HPA axis, are mediated via the glucocorticoid receptor (GR) and the mineralocorticoid receptor.  The FK506-binding protein 51 (FKBP5), a co-chaperone of the Hsp90 and component of the chaperone-receptor heterocomplex, has been shown to reduce ligand sensitivity of the GR.  This study aimed to investigate the function of FKBP5 as a possible mediator of the stress response system and its potential role in the development of stress-related diseases.  Therefore, we assessed whether mice lacking the gene encoding FKBP5 (51KO mice) were less vulnerable to the adverse effects of three weeks of chronic social defeat stress.  Mice were subsequently analyzed with regards to physiol., neuroendocrine, behavioral and mRNA expression alterations.  Our results show a less vulnerable phenotype of 51KO mice with respect to physiol. and neuroendocrine parameters compared to wild-type animals.  51KO mice demonstrated lower adrenal wts. and basal corticosterone levels, a diminished response to a novel acute stimulus and an enhanced recovery, as well as more active stress-coping behavior.  These results suggest an enhanced neg. glucocorticoid feedback within the HPA axis of 51KO mice, possibly modulated by an increased sensitivity of the GR.  This article is part of a Special Issue entitled Anxiety and Depression'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwLJ6JU_-_WbVg90H21EOLACvtfcHk0lgNCutqn0oqbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaktb%252FN&md5=77d3cae9170127c4784377ecc2c332c0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2011.07.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2011.07.041%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DK.%2BV.%26aulast%3DLiebl%26aufirst%3DC.%26aulast%3DScharf%26aufirst%3DS.%2BH.%26aulast%3DWang%26aufirst%3DX.%2BD.%26aulast%3DWolf%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DSmith%26aufirst%3DD.%2BF.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DM.%2BB.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DThe%2520involvement%2520of%2520FK506-binding%2520protein%252051%2520%2528FKBP5%2529%2520in%2520the%2520behavioral%2520and%2520neuroendocrine%2520effects%2520of%2520chronic%2520social%2520defeat%2520stress%26jtitle%3DNeuropharmacology%26date%3D2012%26volume%3D62%26spage%3D332%26epage%3D339%26doi%3D10.1016%2Fj.neuropharm.2011.07.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bull, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ionescu, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzmann, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebertz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depping, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ising, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span> <span> </span><span class="NLM_article-title">FK506 binding protein 5 shapes stress responsiveness: Modulation of neuroendocrine reactivity and coping behavior</span>. <i>Biol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">928</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1016/j.biopsych.2011.07.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.biopsych.2011.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21907973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlOlsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2011&pages=928-936&author=C.+Toumaauthor=N.+C.+Gassenauthor=L.+Herrmannauthor=J.+Cheung-Flynnauthor=D.+R.+Bullauthor=I.+A.+Ionescuauthor=J.+M.+Heinzmannauthor=A.+Knapmanauthor=A.+Siebertzauthor=A.+M.+Deppingauthor=J.+Hartmannauthor=F.+Hauschauthor=M.+V.+Schmidtauthor=F.+Holsboerauthor=M.+Isingauthor=M.+B.+Coxauthor=U.+Schmidtauthor=T.+Rein&title=FK506+binding+protein+5+shapes+stress+responsiveness%3A+Modulation+of+neuroendocrine+reactivity+and+coping+behavior&doi=10.1016%2Fj.biopsych.2011.07.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">FK506 Binding Protein 5 Shapes Stress Responsiveness: Modulation of Neuroendocrine Reactivity and Coping Behavior</span></div><div class="casAuthors">Touma, Chadi; Gassen, Nils Christian; Herrmann, Leonie; Cheung-Flynn, Joyce; Buell, Dominik R.; Ionescu, Irina A.; Heinzmann, Jan-Michael; Knapman, Alana; Siebertz, Anna; Depping, Anna-Mareike; Hartmann, Jakob; Hausch, Felix; Schmidt, Mathias V.; Holsboer, Florian; Ising, Marcus; Cox, Marc B.; Schmidt, Ulrike; Rein, Theo</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">928-936</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Background: The Hsp90 cochaperone FK506 binding protein 5 (FKBP5) is an established regulator of the glucocorticoid receptor (GR), and numerous genetic studies have linked it to stress-related diseases such as major depression or posttraumatic stress disorder.  However, translational studies including genetic animal models are lacking.  Methods: Mice deficient of FKBP5 were generated and analyzed in comparison with wildtype littermates.  They were subjected to several test paradigms characterizing their emotionality, stress reactivity, and coping behavior as well as hypothalamus-pituitary-adrenal axis function and regulation.  Moreover, protein expression of GR and FKBP5 was detd. in different brain structures 8 days after stress exposure.  The combined dexamethasone/corticotropin-releasing hormone test was performed both in mice and healthy human subjects of different FKBP5 genotypes.  The GR function was evaluated by reporter gene assays.  Results: Under basal conditions, deletion of FKBP5 did not change exploratory drive, locomotor activity, anxiety-related behavior, stress-coping, or depression-like behavior.  After exposure to different acute stressors of sufficient intensity, however, it led to a more active coping behavior.  Moreover, loss of FKBP5 decreased hypothalamus-pituitary-adrenal axis reactivity and GR expression changes in response to stressors.  In mice and humans, the FKBP5 genotype also detd. the outcome of the dexamethasone/corticotropin-releasing hormone test.  Conclusions: This study in mice and humans presents FKBP5 as a decisive factor for the physiol. stress response, shaping neuroendocrine reactivity as well as coping behavior.  This lends strong support to the concept emerging from human studies of FKBP5 as important factor governing gene-environment interactions relevant for the etiol. of affective disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreFnJ8h2ZzGLVg90H21EOLACvtfcHk0lhrgCoqwru-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlOlsr7I&md5=22d66ce41dc19d44a7f795eb0f2780a6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2011.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2011.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DHerrmann%26aufirst%3DL.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DBull%26aufirst%3DD.%2BR.%26aulast%3DIonescu%26aufirst%3DI.%2BA.%26aulast%3DHeinzmann%26aufirst%3DJ.%2BM.%26aulast%3DKnapman%26aufirst%3DA.%26aulast%3DSiebertz%26aufirst%3DA.%26aulast%3DDepping%26aufirst%3DA.%2BM.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DIsing%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DSchmidt%26aufirst%3DU.%26aulast%3DRein%26aufirst%3DT.%26atitle%3DFK506%2520binding%2520protein%25205%2520shapes%2520stress%2520responsiveness%253A%2520Modulation%2520of%2520neuroendocrine%2520reactivity%2520and%2520coping%2520behavior%26jtitle%3DBiol.%2520Psychiatry%26date%3D2011%26volume%3D70%26spage%3D928%26epage%3D936%26doi%3D10.1016%2Fj.biopsych.2011.07.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, J. C.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung-Flynn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Erausquin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeber, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinwal, U. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span> <span> </span><span class="NLM_article-title">A new anti-depressive strategy for the elderly: ablation of FKBP5/FKBP51</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e24840</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0024840</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1371%2Fjournal.pone.0024840" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21935478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1yrurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=J.+C.+O%E2%80%99Learyauthor=S.+Dhariaauthor=L.+J.+Blairauthor=S.+Bradyauthor=A.+G.+Johnsonauthor=M.+Petersauthor=J.+Cheung-Flynnauthor=M.+B.+Coxauthor=G.+de+Erausquinauthor=E.+J.+Weeberauthor=U.+K.+Jinwalauthor=C.+A.+Dickey&title=A+new+anti-depressive+strategy+for+the+elderly%3A+ablation+of+FKBP5%2FFKBP51&doi=10.1371%2Fjournal.pone.0024840"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A new anti-depressive strategy for the elderly: ablation of FKBP5/FKBP51</span></div><div class="casAuthors">O'Leary, John C., III; Dharia, Sheetal; Blair, Laura J.; Brady, Sarah; Johnson, Amelia G.; Peters, Melinda; Cheung-Flynn, Joyce; Cox, Marc B.; de Erausquin, Gabriel; Weeber, Edwin J.; Jinwal, Umesh K.; Dickey, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24840</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The gene FKBP5 codes for FKBP51, a co-chaperone protein of the Hsp90 complex that increases with age.  Through its assocn. with Hsp90, FKBP51 regulates the glucocorticoid receptor (GR).  Single nucleotide polymorphisms (SNPs) in the FKBP5 gene assoc. with increased recurrence of depressive episodes, increased susceptibility to post-traumatic stress disorder, bipolar disorder, attempt of suicide, and major depressive disorder in HIV patients.  Variation in one of these SNPs correlates with increased levels of FKBP51.  FKBP51 is also increased in HIV patients.  Moreover, increases in FKBP51 in the amygdala produce an anxiety phenotype in mice.  Therefore, we tested the behavioral consequences of FKBP5 deletion in aged mice.  Similar to that of naive animals treated with classical antidepressants FKBP5-/- mice showed antidepressant behavior without affecting cognition and other basic motor functions.  Reduced corticosterone levels following stress accompanied these obsd. effects on depression.  Age-dependent anxiety was also modulated by FKBP5 deletion.  Therefore, drug discovery efforts focused on depleting FKBP51 levels may yield novel antidepressant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpeKgbMac6-7Vg90H21EOLACvtfcHk0lhrgCoqwru-Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1yrurrF&md5=4301f9433a35bebc37dd6cdcea407793</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024840&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024840%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DJ.%2BC.%26aulast%3DDharia%26aufirst%3DS.%26aulast%3DBlair%26aufirst%3DL.%2BJ.%26aulast%3DBrady%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DA.%2BG.%26aulast%3DPeters%26aufirst%3DM.%26aulast%3DCheung-Flynn%26aufirst%3DJ.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3Dde%2BErausquin%26aufirst%3DG.%26aulast%3DWeeber%26aufirst%3DE.%2BJ.%26aulast%3DJinwal%26aufirst%3DU.%2BK.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26atitle%3DA%2520new%2520anti-depressive%2520strategy%2520for%2520the%2520elderly%253A%2520ablation%2520of%2520FKBP5%252FFKBP51%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0024840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matosin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halldorsdottir, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span> <span> </span><span class="NLM_article-title">Understanding the Molecular Mechanisms Underpinning Gene by Environment Interactions in Psychiatric Disorders: The FKBP5Model</span>. <i>Biol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.biopsych.2018.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.biopsych.2018.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29573791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlslKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=821-830&author=N.+Matosinauthor=T.+Halldorsdottirauthor=E.+B.+Binder&title=Understanding+the+Molecular+Mechanisms+Underpinning+Gene+by+Environment+Interactions+in+Psychiatric+Disorders%3A+The+FKBP5Model&doi=10.1016%2Fj.biopsych.2018.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding the Molecular Mechanisms Underpinning Gene by Environment Interactions in Psychiatric Disorders: The FKBP5 Model</span></div><div class="casAuthors">Matosin, Natalie; Halldorsdottir, Thorhildur; Binder, Elisabeth B.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Psychiatry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">821-830</span>CODEN:
                <span class="NLM_cas:coden">BIPCBF</span>;
        ISSN:<span class="NLM_cas:issn">0006-3223</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Epidemiol. and genetic studies suggest common environmental and genetic risk factors for a no. of psychiatric disorders, including depression, bipolar disorder, and schizophrenia.  Genetic and environmental factors, esp. adverse life events, not only have main effects on disease development but also may interact to shape risk and resilience.  Such gene by adversity interactions have been described for FKBP5, an endogenous regulator of the stress-neuroendocrine system, conferring risk for a no. of psychiatric disorders.  In this review, we present a mol. and cellular model of the consequences of FKBP5 by early adversity interactions.  We illustrate how altered genetic and epigenetic regulation of FKBP5 may contribute to disease risk by covering evidence from clin. and preclin. studies of FKBP5 dysregulation, known cell-type and tissue-type expression patterns of FKBP5 in humans and animals, and the role of FKBP5 as a stress-responsive mol. hub modulating many cellular pathways.  FKBP5 presents the possibility to better understand the mol. and cellular factors contributing to a disease-relevant gene by environment interaction, with implications for the development of biomarkers and interventions for psychiatric disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS27sgZbIsLLVg90H21EOLACvtfcHk0lhgL8XuDaM79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlslKisLo%253D&md5=5618792a3d10befce96db810065d44a5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.biopsych.2018.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopsych.2018.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DMatosin%26aufirst%3DN.%26aulast%3DHalldorsdottir%26aufirst%3DT.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26atitle%3DUnderstanding%2520the%2520Molecular%2520Mechanisms%2520Underpinning%2520Gene%2520by%2520Environment%2520Interactions%2520in%2520Psychiatric%2520Disorders%253A%2520The%2520FKBP5Model%26jtitle%3DBiol.%2520Psychiatry%26date%3D2018%26volume%3D83%26spage%3D821%26epage%3D830%26doi%3D10.1016%2Fj.biopsych.2018.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lesiak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavkin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumaier, J. F.</span></span> <span> </span><span class="NLM_article-title">Sequencing the serotonergic neuron translatome reveals a new role for Fkbp5 in stress</span>. <i>Mol. Psychiatry</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1038/s41380-020-0750-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41380-020-0750-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32366949" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+J.+Lesiakauthor=K.+Coffeyauthor=J.+H.+Cohenauthor=K.+J.+Liangauthor=C.+Chavkinauthor=J.+F.+Neumaier&title=Sequencing+the+serotonergic+neuron+translatome+reveals+a+new+role+for+Fkbp5+in+stress&doi=10.1038%2Fs41380-020-0750-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41380-020-0750-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41380-020-0750-4%26sid%3Dliteratum%253Aachs%26aulast%3DLesiak%26aufirst%3DA.%2BJ.%26aulast%3DCoffey%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DJ.%2BH.%26aulast%3DLiang%26aufirst%3DK.%2BJ.%26aulast%3DChavkin%26aufirst%3DC.%26aulast%3DNeumaier%26aufirst%3DJ.%2BF.%26atitle%3DSequencing%2520the%2520serotonergic%2520neuron%2520translatome%2520reveals%2520a%2520new%2520role%2520for%2520Fkbp5%2520in%2520stress%26jtitle%3DMol.%2520Psychiatry%26date%3D2020%26doi%3D10.1038%2Fs41380-020-0750-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palming, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenback, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fall, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidibeh, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">FKBP5 expression in human adipose tissue increases following dexamethasone exposure and is associated with insulin resistance</span>. <i>Metab., Clin. Exp.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1198</span>– <span class="NLM_lpage">1208</span>, <span class="refDoi"> DOI: 10.1016/j.metabol.2014.05.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.metabol.2014.05.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=24997500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2014&pages=1198-1208&author=M.+J.+Pereiraauthor=J.+Palmingauthor=M.+K.+Svenssonauthor=M.+Rizellauthor=J.+Dalenbackauthor=M.+Hammarauthor=T.+Fallauthor=C.+O.+Sidibehauthor=P.+A.+Svenssonauthor=J.+W.+Eriksson&title=FKBP5+expression+in+human+adipose+tissue+increases+following+dexamethasone+exposure+and+is+associated+with+insulin+resistance&doi=10.1016%2Fj.metabol.2014.05.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP5 expression in human adipose tissue increases following dexamethasone exposure and is associated with insulin resistance</span></div><div class="casAuthors">Pereira, Maria J.; Palming, Jenny; Svensson, Maria K.; Rizell, Magnus; Dalenbaeck, Jan; Hammar, Maarten; Fall, Tove; Sidibeh, Cherno O.; Svensson, Per-Arne; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Metabolism, Clinical and Experimental</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1198-1208</span>CODEN:
                <span class="NLM_cas:coden">METAAJ</span>;
        ISSN:<span class="NLM_cas:issn">0026-0495</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">To study effects of dexamethasone on gene expression in human adipose tissue aiming to identify potential novel mechanisms for glucocorticoid-induced insulin resistance.S.c. and omental adipose tissue, obtained from non-diabetic donors (10 M/15 F; age: 28-60 years; BMI: 20.7-30.6 kg/m2), was incubated with or without dexamethasone (0.003-3 μmol/L) for 24 h.  Gene expression was assessed by microarray and real time-PCR and protein expression by immunoblotting.FKBP5 (FK506-binding protein 5) and CNR1 (cannabinoid receptor 1) were the most responsive genes to dexamethasone in both s.c. and omental adipose tissue (∼ 7-fold).  Dexamethasone increased FKBP5 gene and protein expression in a dose-dependent manner in both depots.  The gene product, FKBP51 protein, was 10-fold higher in the omental than in the s.c. depot, whereas the mRNA levels were similar.  Higher FKBP5 gene expression in omental adipose tissue was assocd. with reduced insulin effects on glucose uptake in both depots.  Furthermore, FKBP5 gene expression in s.c. adipose tissue was pos. correlated with serum insulin, HOMA-IR and s.c. adipocyte diam. and neg. with plasma HDL-cholesterol.  FKBP5 SNPs were found to be assocd. with type 2 diabetes and diabetes-related phenotypes in large population-based samples.  Dexamethasone exposure promotes expression of FKBP5 in adipose tissue, a gene that may be implicated in glucocorticoid-induced insulin resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwQbo-ymUm3rVg90H21EOLACvtfcHk0lhgL8XuDaM79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFSlsrvL&md5=d31678400f5b1e2b972bd684ea6dfed5</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.metabol.2014.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.metabol.2014.05.015%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DPalming%26aufirst%3DJ.%26aulast%3DSvensson%26aufirst%3DM.%2BK.%26aulast%3DRizell%26aufirst%3DM.%26aulast%3DDalenback%26aufirst%3DJ.%26aulast%3DHammar%26aufirst%3DM.%26aulast%3DFall%26aufirst%3DT.%26aulast%3DSidibeh%26aufirst%3DC.%2BO.%26aulast%3DSvensson%26aufirst%3DP.%2BA.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DFKBP5%2520expression%2520in%2520human%2520adipose%2520tissue%2520increases%2520following%2520dexamethasone%2520exposure%2520and%2520is%2520associated%2520with%2520insulin%2520resistance%26jtitle%3DMetab.%252C%2520Clin.%2520Exp.%26date%3D2014%26volume%3D63%26spage%3D1198%26epage%3D1208%26doi%3D10.1016%2Fj.metabol.2014.05.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidibeh, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abalo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">G, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundkvist, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsogiannos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillejo-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">FKBP5 expression in human adipose tissue: potential role in glucose and lipid metabolism, adipogenesis and type 2 diabetes</span>. <i>Endocrine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1007/s12020-018-1674-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1007%2Fs12020-018-1674-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30032404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKhsbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=116-128&author=C.+O.+Sidibehauthor=M.+J.+Pereiraauthor=X.+M.+Abaloauthor=J.+B.+Gauthor=S.+Skrticauthor=P.+Lundkvistauthor=P.+Katsogiannosauthor=F.+Hauschauthor=C.+Castillejo-Lopezauthor=J.+W.+Eriksson&title=FKBP5+expression+in+human+adipose+tissue%3A+potential+role+in+glucose+and+lipid+metabolism%2C+adipogenesis+and+type+2+diabetes&doi=10.1007%2Fs12020-018-1674-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP5 expression in human adipose tissue: potential role in glucose and lipid metabolism, adipogenesis and type 2 diabetes</span></div><div class="casAuthors">Sidibeh, Cherno O.; Pereira, Maria J.; Abalo, Xesus M.; J. Boersma, Gretha; Skrtic, Stanko; Lundkvist, Per; Katsogiannos, Petros; Hausch, Felix; Castillejo-Lopez, Casimiro; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">116-128</span>CODEN:
                <span class="NLM_cas:coden">EOCRE5</span>;
        ISSN:<span class="NLM_cas:issn">1355-008X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Here, we explore the involvement of FKBP51 in glucocorticoid-induced insulin resistance (IR) in human s.c. adipose tissue (SAT), including its potential role in type 2 diabetes (T2D).  Moreover, we assess the metabolic effects of reducing the activity of FKBP51 using the specific inhibitor SAFit1.  Methods: Human SAT was obtained by needle biopsies of the lower abdominal region.  FKBP5 gene expression was assessed in fresh SAT explants from a cohort of 20 T2D subjects group-wise matched by gender, age and BMI to 20 non-diabetic subjects.  In addn., human SAT was obtained from non-diabetic volunteers (20F/9M).  SAT was incubated for 24 h with or without the synthetic glucocorticoid dexamethasone and SAFit1.  Incubated SAT was used to measure the glucose uptake rate in isolated adipocytes.  Results: FKBP5 gene expression levels in SAT pos. correlated with several indexes of IR as well as glucose area under the curve during oral glucose tolerance test (r = 0.33, p < 0.05).  FKBP5 gene expression levels tended to be higher in T2D subjects compared to non-diabetic subjects (p = 0.088).  Moreover, FKBP5 gene expression levels were found to inversely correlate with lipolytic, lipogenic and adipogenic genes.  SAFit1 partly prevented the inhibitory effects of dexamethasone on glucose uptake.  Conclusions: FKBP5 gene expression in human SAT tends to be increased in T2D subjects and is related to elevated glucose levels.  Moreover, FKBP5 gene expression is inversely assocd. with the expression of lipolytic, lipogenic and adipogenic genes.  SAFit1 can partly prevent glucose uptake impairment by glucocorticoids, suggesting that FKBP51 might be a key factor in glucocorticoid-induced IR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZDH980jvIObVg90H21EOLACvtfcHk0lhTkl8Ww26GRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKhsbrI&md5=4d3cebb9d489064e35e6b2c97fda2c45</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs12020-018-1674-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12020-018-1674-5%26sid%3Dliteratum%253Aachs%26aulast%3DSidibeh%26aufirst%3DC.%2BO.%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DAbalo%26aufirst%3DX.%2BM.%26aulast%3DG%26aufirst%3DJ.%2BB.%26aulast%3DSkrtic%26aufirst%3DS.%26aulast%3DLundkvist%26aufirst%3DP.%26aulast%3DKatsogiannos%26aufirst%3DP.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DCastillejo-Lopez%26aufirst%3DC.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DFKBP5%2520expression%2520in%2520human%2520adipose%2520tissue%253A%2520potential%2520role%2520in%2520glucose%2520and%2520lipid%2520metabolism%252C%2520adipogenesis%2520and%2520type%25202%2520diabetes%26jtitle%3DEndocrine%26date%3D2018%26volume%3D62%26spage%3D116%26epage%3D128%26doi%3D10.1007%2Fs12020-018-1674-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamihalev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dournes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uribe-Marino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labermaier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theodoropoulou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Pereda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschop, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Stress-responsive FKBP51 regulates AKT2-AS160 signaling and metabolic function</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1725</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01783-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41467-017-01783-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29170369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ksVKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1725&author=G.+Balsevichauthor=A.+S.+Hauslauthor=C.+W.+Meyerauthor=S.+Karamihalevauthor=X.+Fengauthor=M.+L.+Pohlmannauthor=C.+Dournesauthor=A.+Uribe-Marinoauthor=S.+Santarelliauthor=C.+Labermaierauthor=K.+Hafnerauthor=T.+Maoauthor=M.+Breitsamerauthor=M.+Theodoropoulouauthor=C.+Namendorfauthor=M.+Uhrauthor=M.+Paez-Peredaauthor=G.+Winterauthor=F.+Hauschauthor=A.+Chenauthor=M.+H.+Tschopauthor=T.+Reinauthor=N.+C.+Gassenauthor=M.+V.+Schmidt&title=Stress-responsive+FKBP51+regulates+AKT2-AS160+signaling+and+metabolic+function&doi=10.1038%2Fs41467-017-01783-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Stress-responsive FKBP51 regulates AKT2-AS160 signaling and metabolic function</span></div><div class="casAuthors">Balsevich Georgia; Hausl Alexander S; Karamihalev Stoyo; Pohlmann Max L; Dournes Carine; Uribe-Marino Andres; Santarelli Sara; Labermaier Christiana; Chen Alon; Schmidt Mathias V; Meyer Carola W; Tschop Matthias H; Feng Xixi; Hafner Kathrin; Mao Tianqi; Theodoropoulou Marily; Namendorf Christian; Uhr Manfred; Paez-Pereda Marcelo; Rein Theo; Gassen Nils C; Breitsamer Michaela; Winter Gerhard; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1725</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The co-chaperone FKBP5 is a stress-responsive protein-regulating stress reactivity, and its genetic variants are associated with T2D related traits and other stress-related disorders.  Here we show that FKBP51 plays a role in energy and glucose homeostasis.  Fkbp5 knockout (51KO) mice are protected from high-fat diet-induced weight gain, show improved glucose tolerance and increased insulin signaling in skeletal muscle.  Chronic treatment with a novel FKBP51 antagonist, SAFit2, recapitulates the effects of FKBP51 deletion on both body weight regulation and glucose tolerance.  Using shorter SAFit2 treatment, we show that glucose tolerance improvement precedes the reduction in body weight.  Mechanistically, we identify a novel association between FKBP51 and AS160, a substrate of AKT2 that is involved in glucose uptake.  FKBP51 antagonism increases the phosphorylation of AS160, increases glucose transporter 4 expression at the plasma membrane, and ultimately enhances glucose uptake in skeletal myotubes.  We propose FKBP51 as a mediator between stress and T2D development, and potential target for therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgr0bCoZaOtk2yPDyEFiigfW6udTcc2eYLo8Nv44zHTrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ksVKktw%253D%253D&md5=f58c64e386d4662cd24bc4dcb1739f2d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01783-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01783-y%26sid%3Dliteratum%253Aachs%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BW.%26aulast%3DKaramihalev%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DPohlmann%26aufirst%3DM.%2BL.%26aulast%3DDournes%26aufirst%3DC.%26aulast%3DUribe-Marino%26aufirst%3DA.%26aulast%3DSantarelli%26aufirst%3DS.%26aulast%3DLabermaier%26aufirst%3DC.%26aulast%3DHafner%26aufirst%3DK.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DTheodoropoulou%26aufirst%3DM.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DPaez-Pereda%26aufirst%3DM.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DTschop%26aufirst%3DM.%2BH.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DStress-responsive%2520FKBP51%2520regulates%2520AKT2-AS160%2520signaling%2520and%2520metabolic%2520function%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1725%26doi%3D10.1038%2Fs41467-017-01783-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stechschulte, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinds, T. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuder, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najjar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecka-Czernik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, E. R.</span></span> <span> </span><span class="NLM_article-title">FKBP51 null mice are resistant to diet-induced obesity and the PPARgamma agonist rosiglitazone</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">3888</span>– <span class="NLM_lpage">3900</span>, <span class="refDoi"> DOI: 10.1210/en.2015-1996</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1210%2Fen.2015-1996" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=27442117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVOlsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=3888-3900&author=L.+A.+Stechschulteauthor=B.+Qiuauthor=M.+Warrierauthor=T.+D.+Hindsauthor=M.+Zhangauthor=H.+Guauthor=Y.+Xuauthor=S.+S.+Khuderauthor=L.+Russoauthor=S.+M.+Najjarauthor=B.+Lecka-Czernikauthor=W.+Yongauthor=E.+R.+Sanchez&title=FKBP51+null+mice+are+resistant+to+diet-induced+obesity+and+the+PPARgamma+agonist+rosiglitazone&doi=10.1210%2Fen.2015-1996"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">FKBP51 null mice are resistant to diet-induced obesity and the PPARγ agonist rosiglitazone</span></div><div class="casAuthors">Stechschulte, Lance A.; Qiu, Bin; Warrier, Manya; Hinds, Terry D., Jr.; Zhang, Man; Gu, Hao; Xu, Yuxue; Khuder, Saja S.; Russo, Lucia; Najjar, Sonia M.; Lecka-Czernik, Beata; Yong, Weidong; Sanchez, Edwin R.</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3888-3900</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">1945-7170</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">FK506-binding protein-51 (FKBP51) is a mol. cochaperone recently shown to be a pos. regulator of peroxisome proliferator-activated receptor (PPAR)γ, the master regulator of adipocyte differentiation and function.  In cellular models of adipogenesis, loss of FKBP51 not only reduced PPARγ activity but also reduced lipid accumulation, suggesting that FKBP51 knock-out (KO) mice might have insufficient development of adipose tissue and lipid storage ability.  This model was tested by examg. wild-type (WT) and FKBP51-KO mice under regular and high-fat diet conditions.  Under both diets, FKBP51-KO mice were resistant to wt. gain, hepatic steatosis, and had greatly reduced white adipose tissue (WAT) but higher amts. of brown adipose tissue.  Under high-fat diet, KOmice were highly resistant to adiposity and exhibited reduced plasma lipids and elevated glucose and insulin tolerance.  Profiling of perigonadal and scWATrevealed elevated expression of brown adipose tissue lineage genes in KO mice that correlated increased energy expenditure and a shift of substrate oxidn. to carbohydrates, as measured by indirect calorimetry.  To directly test PPARγ involvement, WT and KO mice were fed rosiglitazone agonist.  In WT mice, rosiglitazone induced whole-body wt. gain, increased WAT mass, a shift of substrate oxidn. to lipids, and elevated expression of PPARγ-regulated lipogenic genes in WAT.  In contrast, KO mice had reduced rosiglitazone responses for these parameters.  Our results identify FKBP51 as an important regulator of PPARγ in WAT and as a potential new target in the treatment of obesity and diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsfyxK2ww2rVg90H21EOLACvtfcHk0lg54oBQT4k6pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVOlsrk%253D&md5=67f7c6da620d8ee9a4df0207ddcaaa3d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1210%2Fen.2015-1996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2015-1996%26sid%3Dliteratum%253Aachs%26aulast%3DStechschulte%26aufirst%3DL.%2BA.%26aulast%3DQiu%26aufirst%3DB.%26aulast%3DWarrier%26aufirst%3DM.%26aulast%3DHinds%26aufirst%3DT.%2BD.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKhuder%26aufirst%3DS.%2BS.%26aulast%3DRusso%26aufirst%3DL.%26aulast%3DNajjar%26aufirst%3DS.%2BM.%26aulast%3DLecka-Czernik%26aufirst%3DB.%26aulast%3DYong%26aufirst%3DW.%26aulast%3DSanchez%26aufirst%3DE.%2BR.%26atitle%3DFKBP51%2520null%2520mice%2520are%2520resistant%2520to%2520diet-induced%2520obesity%2520and%2520the%2520PPARgamma%2520agonist%2520rosiglitazone%26jtitle%3DEndocrinology%26date%3D2016%26volume%3D157%26spage%3D3888%26epage%3D3900%26doi%3D10.1210%2Fen.2015-1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamble, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hetty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vranic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almby, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castillejo-Lopez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abalo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. W.</span></span> <span> </span><span class="NLM_article-title">Proof-of-concept for CRISPR/Cas9 gene editing in human preadipocytes: Deletion of FKBP5 and PPARG and effects on adipocyte differentiation and metabolism</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10565</span>, <span class="refDoi"> DOI: 10.1038/s41598-020-67293-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41598-020-67293-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32601291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlWgu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=10565&author=P.+G.+Kambleauthor=S.+Hettyauthor=M.+Vranicauthor=K.+Almbyauthor=C.+Castillejo-Lopezauthor=X.+M.+Abaloauthor=M.+J.+Pereiraauthor=J.+W.+Eriksson&title=Proof-of-concept+for+CRISPR%2FCas9+gene+editing+in+human+preadipocytes%3A+Deletion+of+FKBP5+and+PPARG+and+effects+on+adipocyte+differentiation+and+metabolism&doi=10.1038%2Fs41598-020-67293-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Proof-of-concept for CRISPR/Cas9 gene editing in human preadipocytes: Deletion of FKBP5 and PPARG and effects on adipocyte differentiation and metabolism</span></div><div class="casAuthors">Kamble, Prasad G.; Hetty, Susanne; Vranic, Milica; Almby, Kristina; Castillejo-Lopez, Casimiro; Abalo, Xesus M.; Pereira, Maria J.; Eriksson, Jan W.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10565</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: CRISPR/Cas9 has revolutionized the genome-editing field.  So far, successful application in human adipose tissue has not been convincingly shown.  We present a method for gene knockout using electroporation in preadipocytes from human adipose tissue that achieved at least 90% efficiency without any need for selection of edited cells or clonal isolation.  We knocked out the FKBP5 and PPARG genes in preadipocytes and studied the resulting phenotypes.  PPARG knockout prevented differentiation into adipocytes.  Conversely, deletion of FKBP51, the protein coded by the FKBP5 gene, did not affect adipogenesis.  Instead, it markedly modulated glucocorticoid effects on adipocyte glucose metab. and, furthermore, we show some evidence of altered transcriptional activity of glucocorticoid receptors.  This has potential implications for the development of insulin resistance and type 2 diabetes.  The reported method is simple, easy to adapt, and enables the use of human primary preadipocytes instead of animal adipose cell models to assess the role of key genes and their products in adipose tissue development, metab. and pathobiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGpZ2rNrPUbbVg90H21EOLACvtfcHk0lg54oBQT4k6pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlWgu7rO&md5=830e261629f4837312f665035563a241</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-020-67293-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-020-67293-y%26sid%3Dliteratum%253Aachs%26aulast%3DKamble%26aufirst%3DP.%2BG.%26aulast%3DHetty%26aufirst%3DS.%26aulast%3DVranic%26aufirst%3DM.%26aulast%3DAlmby%26aufirst%3DK.%26aulast%3DCastillejo-Lopez%26aufirst%3DC.%26aulast%3DAbalo%26aufirst%3DX.%2BM.%26aulast%3DPereira%26aufirst%3DM.%2BJ.%26aulast%3DEriksson%26aufirst%3DJ.%2BW.%26atitle%3DProof-of-concept%2520for%2520CRISPR%252FCas9%2520gene%2520editing%2520in%2520human%2520preadipocytes%253A%2520Deletion%2520of%2520FKBP5%2520and%2520PPARG%2520and%2520effects%2520on%2520adipocyte%2520differentiation%2520and%2520metabolism%26jtitle%3DSci.%2520Rep.%26date%3D2020%26volume%3D10%26spage%3D10565%26doi%3D10.1038%2Fs41598-020-67293-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiaru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tochiki, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annan, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geranton, S. M.</span></span> <span> </span><span class="NLM_article-title">The stress regulator FKBP51 drives chronic pain by modulating spinal glucocorticoid signaling</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">325ra319</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aab3376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1126%2Fscitranslmed.aab3376" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=325ra319&author=M.+Maiaruauthor=K.+K.+Tochikiauthor=M.+B.+Coxauthor=L.+V.+Annanauthor=C.+G.+Bellauthor=X.+Fengauthor=F.+Hauschauthor=S.+M.+Geranton&title=The+stress+regulator+FKBP51+drives+chronic+pain+by+modulating+spinal+glucocorticoid+signaling&doi=10.1126%2Fscitranslmed.aab3376"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aab3376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aab3376%26sid%3Dliteratum%253Aachs%26aulast%3DMaiaru%26aufirst%3DM.%26aulast%3DTochiki%26aufirst%3DK.%2BK.%26aulast%3DCox%26aufirst%3DM.%2BB.%26aulast%3DAnnan%26aufirst%3DL.%2BV.%26aulast%3DBell%26aufirst%3DC.%2BG.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DGeranton%26aufirst%3DS.%2BM.%26atitle%3DThe%2520stress%2520regulator%2520FKBP51%2520drives%2520chronic%2520pain%2520by%2520modulating%2520spinal%2520glucocorticoid%2520signaling%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26spage%3D325ra319%26doi%3D10.1126%2Fscitranslmed.aab3376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiaru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geranton, S. M.</span></span> <span> </span><span class="NLM_article-title">The stress regulator FKBP51: a novel and promising druggable target for the treatment of persistent pain states across sexes</span>. <i>Pain</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">1224</span>– <span class="NLM_lpage">1234</span>, <span class="refDoi"> DOI: 10.1097/j.pain.0000000000001204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1097%2Fj.pain.0000000000001204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=29533388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFGgs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2018&pages=1224-1234&author=M.+Maiaruauthor=O.+B.+Morganauthor=T.+Maoauthor=M.+Breitsamerauthor=H.+Bamberauthor=M.+Pohlmannauthor=M.+V.+Schmidtauthor=G.+Winterauthor=F.+Hauschauthor=S.+M.+Geranton&title=The+stress+regulator+FKBP51%3A+a+novel+and+promising+druggable+target+for+the+treatment+of+persistent+pain+states+across+sexes&doi=10.1097%2Fj.pain.0000000000001204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The stress regulator FKBP51: a novel and promising druggable target for the treatment of persistent pain states across sexes</span></div><div class="casAuthors">Maiaru, Maria; Morgan, Oakley B.; Mao, Tianqi; Breitsamer, Michaela; Bamber, Harry; Poehlmann, Max; Schmidt, Mathias V.; Winter, Gerhard; Hausch, Felix; Geranton, Sandrine M.</div><div class="citationInfo"><span class="NLM_cas:title">Pain</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1224-1234</span>CODEN:
                <span class="NLM_cas:coden">PAINDB</span>;
        ISSN:<span class="NLM_cas:issn">0304-3959</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">It is well established that FKBP51 regulates the stress system by modulating the sensitivity of the glucocorticoid receptor to stress hormones.  Recently, we have demonstrated that FKBP51 also drives long-term inflammatory pain states in male mice by modulating glucocorticoid signalling at spinal cord level.  Here, we explored the potential of FKBP51 as a new pharmacol. target for the treatment of persistent pain across the sexes.  First, we demonstrated that FKBP51 regulates long-term pain states of different etiologies independently of sex.  Deletion of FKBP51 reduced the mech. hypersensitivity seen in joint inflammatory and neuropathic pain states in female and male mice.  Furthermore, FKBP51 deletion also reduced the hypersensitivity seen in a translational model of chemotherapy-induced pain.  Interestingly, these 3 pain states were assocd. with changes in glucocorticoid signalling, as indicated by the increased expression, at spinal cord level, of the glucocorticoid receptor isoform assocd. with glucocorticoid resistance, GRβ, and increased levels of plasma corticosterone.  These pain states were also accompanied by an upregulation of interleukin-6 in the spinal cord.  Crucially, we were able to pharmacol. reduce the severity of the mech. hypersensitivity seen in these 3 models of persistent pain with the unique FKBP51 ligand SAFit2.  When SAFit2 was combined with a state-of-the-art vesicular phospholipid gel formulation for slow release, a single injection of SAFit2 offered pain relief for at least 7 days.  We therefore propose the pharmacol. blockade of FKBP51 as a new approach for the treatment of persistent pain across sexes, likely in humans as well as rodents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHXPZQxtP6trVg90H21EOLACvtfcHk0lgFTatpXQMXCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGgs7vE&md5=cce2fddf3d07d4089e42caac61d8abfd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1097%2Fj.pain.0000000000001204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fj.pain.0000000000001204%26sid%3Dliteratum%253Aachs%26aulast%3DMaiaru%26aufirst%3DM.%26aulast%3DMorgan%26aufirst%3DO.%2BB.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DBamber%26aufirst%3DH.%26aulast%3DPohlmann%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DGeranton%26aufirst%3DS.%2BM.%26atitle%3DThe%2520stress%2520regulator%2520FKBP51%253A%2520a%2520novel%2520and%2520promising%2520druggable%2520target%2520for%2520the%2520treatment%2520of%2520persistent%2520pain%2520states%2520across%2520sexes%26jtitle%3DPain%26date%3D2018%26volume%3D159%26spage%3D1224%26epage%3D1234%26doi%3D10.1097%2Fj.pain.0000000000001204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linnstaedt, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riker, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckeis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchko, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendry, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Domeier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laederach, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLean, S. A.</span></span> <span> </span><span class="NLM_article-title">A Functional riboSNitch in the 3′ Untranslated Region of FKBP5 Alters MicroRNA-320a Binding Efficiency and Mediates Vulnerability to Chronic Post-Traumatic Pain</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8407</span>– <span class="NLM_lpage">8420</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.3458-17.2018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1523%2FJNEUROSCI.3458-17.2018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30150364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1ymur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=8407-8420&author=S.+D.+Linnstaedtauthor=K.+D.+Rikerauthor=C.+A.+Rueckeisauthor=K.+M.+Kutchkoauthor=L.+Lackeyauthor=K.+R.+McCarthyauthor=Y.+H.+Tsaiauthor=J.+S.+Parkerauthor=M.+C.+Kurzauthor=P.+L.+Hendryauthor=C.+Lewandowskiauthor=E.+Datnerauthor=C.+Pearsonauthor=B.+O%E2%80%99Neilauthor=R.+Domeierauthor=S.+Kaushikauthor=A.+Laederachauthor=S.+A.+McLean&title=A+Functional+riboSNitch+in+the+3%E2%80%B2+Untranslated+Region+of+FKBP5+Alters+MicroRNA-320a+Binding+Efficiency+and+Mediates+Vulnerability+to+Chronic+Post-Traumatic+Pain&doi=10.1523%2FJNEUROSCI.3458-17.2018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A functional riboSNitch in the 3' untranslated region of FKBP5 alters MicroRNA-320a binding efficiency and mediates vulnerability to chronic post-traumatic pain</span></div><div class="casAuthors">Linnstaedt, Sarah D.; Riker, Kyle D.; Rueckeis, Xcathleen A.; Kutchko, Katrina M.; Lackey, Lela; Mccarthy, Kathleen R.; Tsai, Yi-Hsuan; Parker, Joel S.; Kurz, Michael C.; Hendry, Phyllis L.; Lewandowski, Christopher; Datner, Elizabeth; Pearson, Claire; O'neil, Brian; Domeier, Robert; Kaushik, Sangeeta; Laederach, Alain; MClean, Samuel A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">8407-8420</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">1529-2401</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Previous studies have shown that common variants of the gene coding for FK506-binding protein 51 (FKBP5), a crit. regulator of glucocorticoid sensitivity, affect vulnerability to stress-related disorders.  In a previous report, FKBP5 rs1360780 was identified as a functional variant because of its effect on gene methylation.  Here we report evidence for a novel functional FKBP5 allele, rs3800373.  This study assessed the assocn. between rs3800373 and post-traumatic chronic pain in 1607 women and men from two ethnically diverse human cohorts.  The mol. mechanism through which rs3800373 affects adverse outcomes was established via in silico, in vivo, and in vitro analyses.  The rs3800373 minor allele predicted worse adverse outcomes after trauma exposure, such that individuals with the minor (risk) allele developed more severe post-traumatic chronic musculoskeletal pain.  Among these individuals, peritraumatic circulating FKBP5 expression levels increased as cortisol and glucocorticoid receptor (NR3C1) mRNA levels increased, consistent with increased glucocorticoid resistance.  Bioinformatic, in vitro, and mutational analyses indicate that the rs3800373 minor allele reduces the binding of a stress- and pain-assocd. microRNA, miR-320a, to FKBP5 via altering the FKBP5 mRNA 3'UTR secondary structure (i.e., is a riboSNitch).  This results in relatively greater FKBP5 translation, unchecked by miR-320a.  Overall, these results identify an important gene-miRNA interaction influencing chronic pain risk in vulnerable individuals and suggest that exogenous methods to achieve targeted redn. in poststress FKBP5 mRNA expression may constitute useful therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8ey_putJR7Vg90H21EOLACvtfcHk0lidcGxe2VUh8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1ymur7E&md5=e45aaa0214e4ddaa5a06c409fae2590a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3458-17.2018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3458-17.2018%26sid%3Dliteratum%253Aachs%26aulast%3DLinnstaedt%26aufirst%3DS.%2BD.%26aulast%3DRiker%26aufirst%3DK.%2BD.%26aulast%3DRueckeis%26aufirst%3DC.%2BA.%26aulast%3DKutchko%26aufirst%3DK.%2BM.%26aulast%3DLackey%26aufirst%3DL.%26aulast%3DMcCarthy%26aufirst%3DK.%2BR.%26aulast%3DTsai%26aufirst%3DY.%2BH.%26aulast%3DParker%26aufirst%3DJ.%2BS.%26aulast%3DKurz%26aufirst%3DM.%2BC.%26aulast%3DHendry%26aufirst%3DP.%2BL.%26aulast%3DLewandowski%26aufirst%3DC.%26aulast%3DDatner%26aufirst%3DE.%26aulast%3DPearson%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%26aulast%3DDomeier%26aufirst%3DR.%26aulast%3DKaushik%26aufirst%3DS.%26aulast%3DLaederach%26aufirst%3DA.%26aulast%3DMcLean%26aufirst%3DS.%2BA.%26atitle%3DA%2520Functional%2520riboSNitch%2520in%2520the%25203%25E2%2580%25B2%2520Untranslated%2520Region%2520of%2520FKBP5%2520Alters%2520MicroRNA-320a%2520Binding%2520Efficiency%2520and%2520Mediates%2520Vulnerability%2520to%2520Chronic%2520Post-Traumatic%2520Pain%26jtitle%3DJ.%2520Neurosci.%26date%3D2018%26volume%3D38%26spage%3D8407%26epage%3D8420%26doi%3D10.1523%2FJNEUROSCI.3458-17.2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">FKBPs and their role in neuronal signaling</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1850</i></span>,  <span class="NLM_fpage">2035</span>– <span class="NLM_lpage">2040</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2015.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bbagen.2015.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25615537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1850&publication_year=2015&pages=2035-2040&author=F.+Hausch&title=FKBPs+and+their+role+in+neuronal+signaling&doi=10.1016%2Fj.bbagen.2015.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">FKBPs and their role in neuronal signaling</span></div><div class="casAuthors">Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1850</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2035-2040</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Ligands for FK506-binding proteins, also referred to as neuroimmunophilin ligands, have repeatedly been described as neuritotrophic, neuroprotective or neuroregenerative agents.  However, the precise mol. mechanism of action underlying the obsd. effects has remained elusive, which eventually led to a reduced interest in FKBP ligand development.  A survey is presented on the pharmacol. of neuroimmunophilin ligands, of the current understanding of individual FKBP homologs in neuronal processes and an assessment of their potential as drug targets for CNS disorders.  FKBP51 is the major target accounting for the neuritotrophic effect of neuroimmunophilin ligands.  Selectivity against the homolog FKBP52 is essential for optimal neuritotrophic efficacy.  Selectivity within the FKBP family, in particular selective inhibition of FKBP12 or FKBP51, is possible.  FKBP51 is a pharmacol. tractable target for stress-related disorders.  The role of FKBPs in neurodegeneration remains to be clarified.  This article is part of a Special Issue entitled Proline-directed Foldases: Cell Signaling Catalysts and Drug Targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5ThYgkiLY7Vg90H21EOLACvtfcHk0lidcGxe2VUh8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFSjtrw%253D&md5=462283764c62af331bfc90b3f2221835</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2015.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2015.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DFKBPs%2520and%2520their%2520role%2520in%2520neuronal%2520signaling%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2015%26volume%3D1850%26spage%3D2035%26epage%3D2040%26doi%3D10.1016%2Fj.bbagen.2015.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuboni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Vizarra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zannas, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draenert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida, O. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of the FK506-binding protein 51 by induced fit</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1699</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnchembio.1699" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25436518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCksrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=33-37&author=S.+Gaaliauthor=A.+Kirschnerauthor=S.+Cuboniauthor=J.+Hartmannauthor=C.+Kozanyauthor=G.+Balsevichauthor=C.+Namendorfauthor=P.+Fernandez-Vizarraauthor=C.+Sippelauthor=A.+S.+Zannasauthor=R.+Draenertauthor=E.+B.+Binderauthor=O.+F.+Almeidaauthor=G.+Ruhterauthor=M.+Uhrauthor=M.+V.+Schmidtauthor=C.+Toumaauthor=A.+Bracherauthor=F.+Hausch&title=Selective+inhibitors+of+the+FK506-binding+protein+51+by+induced+fit&doi=10.1038%2Fnchembio.1699"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of the FK506-binding protein 51 by induced fit</span></div><div class="casAuthors">Gaali, Steffen; Kirschner, Alexander; Cuboni, Serena; Hartmann, Jakob; Kozany, Christian; Balsevich, Georgia; Namendorf, Christian; Fernandez-Vizarra, Paula; Sippel, Claudia; Zannas, Anthony S.; Draenert, Rika; Binder, Elisabeth B.; Almeida, Osborne F. X.; Ruehter, Gerd; Uhr, Manfred; Schmidt, Mathias V.; Touma, Chadi; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-37</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51, encoded by the FKBP5 gene) is an established risk factor for stress-related psychiatric disorders such as major depression.  Drug discovery for FKBP51 has been hampered by the inability to pharmacol. differentiate against the structurally similar but functional opposing homolog FKBP52, and all known FKBP ligands are unselective.  Here, we report the discovery of the potent and highly selective inhibitors of FKBP51, SAFit1 and SAFit2.  This new class of ligands achieves selectivity for FKBP51 by an induced-fit mechanism that is much less favorable for FKBP52.  By using these ligands, we demonstrate that selective inhibition of FKBP51 enhances neurite elongation in neuronal cultures and improves neuroendocrine feedback and stress-coping behavior in mice.  Our findings provide the structural and functional basis for the development of mechanistically new antidepressants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6eYhLVOE_brVg90H21EOLACvtfcHk0liaHjzbrwkO2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCksrnN&md5=3f8d9300671ae7df4763b6d4507644dc</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1699%26sid%3Dliteratum%253Aachs%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DKirschner%26aufirst%3DA.%26aulast%3DCuboni%26aufirst%3DS.%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DFernandez-Vizarra%26aufirst%3DP.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DZannas%26aufirst%3DA.%2BS.%26aulast%3DDraenert%26aufirst%3DR.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26aulast%3DAlmeida%26aufirst%3DO.%2BF.%26aulast%3DRuhter%26aufirst%3DG.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26aulast%3DTouma%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520FK506-binding%2520protein%252051%2520by%2520induced%2520fit%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D33%26epage%3D37%26doi%3D10.1038%2Fnchembio.1699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruhter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeijmakers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerholz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deussing, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holsboer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of the psychiatric risk factor FKBP51 has anxiolytic properties</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">9007</span>– <span class="NLM_lpage">9016</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.4024-14.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1523%2FJNEUROSCI.4024-14.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=26085626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=9007-9016&author=J.+Hartmannauthor=K.+V.+Wagnerauthor=S.+Gaaliauthor=A.+Kirschnerauthor=C.+Kozanyauthor=G.+Ruhterauthor=N.+Dedicauthor=A.+S.+Hauslauthor=L.+Hoeijmakersauthor=S.+Westerholzauthor=C.+Namendorfauthor=T.+Gerlachauthor=M.+Uhrauthor=A.+Chenauthor=J.+M.+Deussingauthor=F.+Holsboerauthor=F.+Hauschauthor=M.+V.+Schmidt&title=Pharmacological+inhibition+of+the+psychiatric+risk+factor+FKBP51+has+anxiolytic+properties&doi=10.1523%2FJNEUROSCI.4024-14.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of the psychiatric risk factor FKBP51 has anxiolytic properties</span></div><div class="casAuthors">Hartmann, Jakob; Wagner, Klaus V.; Gaali, Steffen; Kirschner, Alexander; Kozany, Christian; Ruehter, Gerd; Dedic, Nina; Haeusl, Alexander S.; Hoeijmakers, Lianne; Westerholz, Soeren; Namendorf, Christian; Gerlach, Tamara; Uhr, Manfred; Chen, Alon; Deussing, Jan M.; Holsboer, Florian; Hausch, Felix; Schmidt, Mathias V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9007-9016</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Anxiety-related psychiatric disorders represent one of the largest health burdens worldwide.  Single nucleotide polymorphisms of the FK506 binding protein 51 (FKBP51) gene have been repeatedly assocd. with anxiety-related disorders and stress sensitivity.  Given the intimate relationship of stress and anxiety, we hypothesized that amygdala FKBP51 may mediate anxiety-related behaviors.  Mimicking the stress effect by specifically overexpressing FKBP51 in the basolateral amygdala (BLA) or central amygdala resulted in increased anxiety-related behavior, resp.  In contrast, application of a highly selective FKBP51 point mutant antagonist, following FKBP51mut BLA-overexpression, reduced the anxiogenic phenotype.  We subsequently tested a novel FKBP51 antagonist, SAFit2, in wild-type mice via BLA microinjections, which reduced anxiety-related behavior.  Remarkably, the same effect was obsd. following peripheral administration of SAFit2.  To our knowledge, this is the first in vivo study using a specific FKBP51 antagonist, thereby unraveling the role of FKBP51 and its potential as a novel drug target for the improved treatment of anxiety-related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLNKQ4xD4NzLVg90H21EOLACvtfcHk0ljTemF51JLHKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamu73O&md5=cf5e2e0fe69e3d2ea923567b5693909f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.4024-14.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.4024-14.2015%26sid%3Dliteratum%253Aachs%26aulast%3DHartmann%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DK.%2BV.%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DKirschner%26aufirst%3DA.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DRuhter%26aufirst%3DG.%26aulast%3DDedic%26aufirst%3DN.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DHoeijmakers%26aufirst%3DL.%26aulast%3DWesterholz%26aufirst%3DS.%26aulast%3DNamendorf%26aufirst%3DC.%26aulast%3DGerlach%26aufirst%3DT.%26aulast%3DUhr%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeussing%26aufirst%3DJ.%2BM.%26aulast%3DHolsboer%26aufirst%3DF.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DPharmacological%2520inhibition%2520of%2520the%2520psychiatric%2520risk%2520factor%2520FKBP51%2520has%2520anxiolytic%2520properties%26jtitle%3DJ.%2520Neurosci.%26date%3D2015%26volume%3D35%26spage%3D9007%26epage%3D9016%26doi%3D10.1523%2FJNEUROSCI.4024-14.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausl, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsevich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitsamer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. V.</span></span> <span> </span><span class="NLM_article-title">Pharmacological modulation of the psychiatric risk factor FKBP51 alters efficiency of common antidepressant drugs</span>. <i>Front. Behav. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">262</span>, <span class="refDoi"> DOI: 10.3389/fnbeh.2018.00262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.3389%2Ffnbeh.2018.00262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=30483074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BB3crlsVGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=262&author=M.+L.+Pohlmannauthor=A.+S.+Hauslauthor=D.+Harbichauthor=G.+Balsevichauthor=C.+Engelhardtauthor=X.+X.+Fengauthor=M.+Breitsamerauthor=F.+Hauschauthor=G.+Winterauthor=M.+V.+Schmidt&title=Pharmacological+modulation+of+the+psychiatric+risk+factor+FKBP51+alters+efficiency+of+common+antidepressant+drugs&doi=10.3389%2Ffnbeh.2018.00262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological Modulation of the Psychiatric Risk Factor FKBP51 Alters Efficiency of Common Antidepressant Drugs</span></div><div class="casAuthors">Pohlmann Max L; Hausl Alexander S; Harbich Daniela; Balsevich Georgia; Engelhardt Clara; Feng Xixi; Hausch Felix; Schmidt Mathias V; Breitsamer Michaela; Winter Gerhard; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in behavioral neuroscience</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">262</span>
        ISSN:<span class="NLM_cas:issn">1662-5153</span>.
    </div><div class="casAbstract">Despite a growing body of research over the last few decades, mental disorders, including anxiety disorders or depression, are still one of the most prevalent and hardest to treat health burdens worldwide.  Since pharmacological treatment with a single drug is often rather ineffective, approaches such as co-medication with functionally diverse antidepressants (ADs) have been discussed and tried more recently.  Besides classical ADs, there is a growing number of candidate targets identified as potential starting points for new treatment methods.  One of these candidates, the FK506 binding protein 51 (FKBP51) is linked to a number of psychiatric disorders in humans.  In this study, we used SAFit2-a newly developed modulator of FKBP51, which has shown promising results in rodent models for stress-related disorders delivered in a depot formulation.  We combined SAFit2 with the commonly prescribed selective serotonin reuptake inhibitor (SSRI) escitalopram and performed basic behavioral characterization in a mouse model.  Remarkably, co-application of SAFit2 lowered the efficacy of escitalopram in anxiety-related tests but improved stress coping behavior.  Given the fact that mental diseases such as anxiety disorders or depression can be divided into different sub-categories, some of which more or less prone to stress, SAFit2 could indeed be a highly beneficial co-medication in very specific cases.  This study could be a first, promising step towards the use of FKBP51 modulators as potent and specific enhancers of AD efficiency for subclasses of patients in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRmSzSTA7pEe0Q0lWbKiCflfW6udTcc2eao3snkDMXdhLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crlsVGnuw%253D%253D&md5=cdedd03523d588d0464852c5d30534b7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3389%2Ffnbeh.2018.00262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffnbeh.2018.00262%26sid%3Dliteratum%253Aachs%26aulast%3DPohlmann%26aufirst%3DM.%2BL.%26aulast%3DHausl%26aufirst%3DA.%2BS.%26aulast%3DHarbich%26aufirst%3DD.%26aulast%3DBalsevich%26aufirst%3DG.%26aulast%3DEngelhardt%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DBreitsamer%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DSchmidt%26aufirst%3DM.%2BV.%26atitle%3DPharmacological%2520modulation%2520of%2520the%2520psychiatric%2520risk%2520factor%2520FKBP51%2520alters%2520efficiency%2520of%2520common%2520antidepressant%2520drugs%26jtitle%3DFront.%2520Behav.%2520Neurosci.%26date%3D2018%26volume%3D12%26spage%3D262%26doi%3D10.3389%2Ffnbeh.2018.00262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Arrigo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tufano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guadagno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Basso De Caro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staibano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parisi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, S.</span></span> <span> </span><span class="NLM_article-title">A regulatory role for the co-chaperone FKBP51s in PD-L1 expression in glioma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">68291</span>– <span class="NLM_lpage">68304</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.18632%2Foncotarget.19309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=28978117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FltlWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=68291-68304&author=P.+D%E2%80%99Arrigoauthor=M.+Russoauthor=A.+Reaauthor=M.+Tufanoauthor=E.+Guadagnoauthor=M.+L.+Del+Basso+De+Caroauthor=R.+Pacelliauthor=F.+Hauschauthor=S.+Staibanoauthor=G.+Ilardiauthor=S.+Parisiauthor=M.+F.+Romanoauthor=S.+Romano&title=A+regulatory+role+for+the+co-chaperone+FKBP51s+in+PD-L1+expression+in+glioma&doi=10.18632%2Foncotarget.19309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A regulatory role for the co-chaperone FKBP51s in PD-L1 expression in glioma</span></div><div class="casAuthors">D'Arrigo Paolo; Russo Michele; Rea Anna; Tufano Martina; Parisi Silvia; Romano Maria Fiammetta; Romano Simona; Guadagno Elia; Del Basso De Caro Maria Laura; Pacelli Roberto; Staibano Stefania; Ilardi Gennaro; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">68291-68304</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  FKBP51 is a co-chaperone with isomerase activity, abundantly expressed in glioma.  We previously identified a spliced isoform (FKBP51s) and highlighted a role for this protein in the upregulation of Programmed Death Ligand 1 (PD-L1) expression in melanoma.  Because gliomas can express PD-L1 causing a defective host anti-tumoral immunity, we investigated whether FKBP51s was expressed in glioma and played a role in PD-L1 regulation in this tumour.  METHODS:  We used D54 and U251 glioblastoma cell lines that constitutively expressed PD-L1.  FKBP51s was measured by immunoblot, flow cytometry and microscopy.  In patient tumours, IHC and qPCR were used to measure protein and mRNA levels respectively.  FKBP51s depletion was achieved by siRNAs, and its enzymatic function was inhibited using selective inhibitors (SAFit).  We investigated protein maturation using N-glycosidase and cell fractionation approaches.  RESULTS:  FKBP51s was expressed at high levels in glioma cells.  Glycosylated-PD-L1 was increased and reduced by FKBP51s overexpression or silencing, respectively.  Naive PD-L1 was found in the endoplasmic reticulum (ER) of glioma cells complexed with FKBP51s, whereas the glycosylated form was measured in the Golgi apparatus.  SAFit reduced PD-L1 levels (constitutively expressed and ionizing radiation-induced).  SAFit reduced cell death of PBMC co-cultured with glioma.  CONCLUSIONS:  Here we addressed the mechanism of post-translational regulation of PD-L1 protein in glioma.  FKBP51s upregulated PD-L1 expression on the plasma membrane by catalysing the protein folding required for subsequent glycosylation.  Inhibition of FKBP51s isomerase activity by SAFit decreased PD-L1 levels.  These findings suggest that FKBP51s is a potential target of immunomodulatory strategies for glioblastoma treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLwCRKT7Pe3Uh5eUSa2oe_fW6udTcc2eb1ZqxJCk9q37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FltlWmtQ%253D%253D&md5=727369325047c082b28529f43cfca839</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19309%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Arrigo%26aufirst%3DP.%26aulast%3DRusso%26aufirst%3DM.%26aulast%3DRea%26aufirst%3DA.%26aulast%3DTufano%26aufirst%3DM.%26aulast%3DGuadagno%26aufirst%3DE.%26aulast%3DDel%2BBasso%2BDe%2BCaro%26aufirst%3DM.%2BL.%26aulast%3DPacelli%26aufirst%3DR.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DStaibano%26aufirst%3DS.%26aulast%3DIlardi%26aufirst%3DG.%26aulast%3DParisi%26aufirst%3DS.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26aulast%3DRomano%26aufirst%3DS.%26atitle%3DA%2520regulatory%2520role%2520for%2520the%2520co-chaperone%2520FKBP51s%2520in%2520PD-L1%2520expression%2520in%2520glioma%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D68291%26epage%3D68304%26doi%3D10.18632%2Foncotarget.19309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">D’Arrigo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digregorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tufano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedobbeleer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigorito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogister, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span> <span> </span><span class="NLM_article-title">The splicing FK506-binding protein-51 isoform plays a role in glioblastoma resistance through programmed cell death ligand-1 expression regulation</span>. <i>Cell Death Discov.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">137</span>, <span class="refDoi"> DOI: 10.1038/s41420-019-0216-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fs41420-019-0216-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31583120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BB3MnkvFCntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=137&author=P.+D%E2%80%99Arrigoauthor=M.+Digregorioauthor=S.+Romanoauthor=M.+Tufanoauthor=A.+Reaauthor=F.+Hauschauthor=M.+Dedobbeleerauthor=V.+Vigoritoauthor=S.+Russoauthor=M.+Bauderauthor=B.+Rogisterauthor=M.+F.+Romano&title=The+splicing+FK506-binding+protein-51+isoform+plays+a+role+in+glioblastoma+resistance+through+programmed+cell+death+ligand-1+expression+regulation&doi=10.1038%2Fs41420-019-0216-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The splicing FK506-binding protein-51 isoform plays a role in glioblastoma resistance through programmed cell death ligand-1 expression regulation</span></div><div class="casAuthors">D'Arrigo Paolo; Romano Simona; Tufano Martina; Rea Anna; Vigorito Vincenza; Russo Salvatore; Romano Maria Fiammetta; Digregorio Marina; Dedobbeleer Matthias; Rogister Bernard; Hausch Felix; Bauder Michael</div><div class="citationInfo"><span class="NLM_cas:title">Cell death discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137</span>
        ISSN:<span class="NLM_cas:issn">2058-7716</span>.
    </div><div class="casAbstract">Gliomas aberrantly express programmed cell death ligand-1 (PD-L1), which has a pivotal role in immunoevasion.  The splicing isoform of FKBP5, termed FKBP51s, is a PD-L1 foldase, assisting the immune checkpoint molecule in maturation and expression on the plasma membrane.  The concept that PD-L1 supports tumor-intrinsic properties is increasingly emerging.  The aim of the present work was to confirm the pro-tumoral effect of PD-L1 on human glioma cell survival, stemness capacity and resistance, and to address the issue of whether, by targeting its foldase either chemically or by silencing, the aggressive tumor features could be attenuated.  PD-L1-depleted glioma cells have a reduced threshold for apoptosis, while PD-L1 forced expression increases resistance.  Similar results were obtained with FKBP51s modulation.  The ability of PD-L1 to counteract cell death was hampered by FKBP51s silencing.  PD-L1 expression was particularly high in glioma cells with a cancer-stem-cell profile.  Moreover, PD-L1 sustained the spheroid formation capability of glioma cells.  Targeting of FKBP51s by small-interfering RNA (siRNA) or the specific inhibitor SAFit2, reduced the number of formed spheroids, along with PD-L1 expression.  Finally, in an orthotopic mouse model of glioblastoma, daily treatment with SAFit2 significantly reduced tumor PD-L1 expression, and tumor growth.  In treated mice, caspase-3 activation and reduced vimentin expression were observed in excised tumors.  In conclusion, targeting of FKBP51s hampers PD-L1 and its pro-tumoral properties, thereby affecting the self-renewal and growth capacities of glioblastoma cells in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTVgJP0BVo3R4xhnOQkIHOxfW6udTcc2eb1ZqxJCk9q37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnkvFCntQ%253D%253D&md5=63af445bdf1c6c3005a40f32fc517b1b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41420-019-0216-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41420-019-0216-0%26sid%3Dliteratum%253Aachs%26aulast%3DD%25E2%2580%2599Arrigo%26aufirst%3DP.%26aulast%3DDigregorio%26aufirst%3DM.%26aulast%3DRomano%26aufirst%3DS.%26aulast%3DTufano%26aufirst%3DM.%26aulast%3DRea%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DDedobbeleer%26aufirst%3DM.%26aulast%3DVigorito%26aufirst%3DV.%26aulast%3DRusso%26aufirst%3DS.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DRogister%26aufirst%3DB.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26atitle%3DThe%2520splicing%2520FK506-binding%2520protein-51%2520isoform%2520plays%2520a%2520role%2520in%2520glioblastoma%2520resistance%2520through%2520programmed%2520cell%2520death%2520ligand-1%2520expression%2520regulation%26jtitle%3DCell%2520Death%2520Discov.%26date%3D2019%26volume%3D5%26spage%3D137%26doi%3D10.1038%2Fs41420-019-0216-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konig, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalinichenko, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voll, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornhuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, C. P.</span></span> <span> </span><span class="NLM_article-title">The selective FKBP51 inhibitor SAFit2 reduces alcohol consumption and reinstatement of conditioned alcohol effects in mice</span>. <i>Addict. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">e12758</span> <span class="refDoi"> DOI: 10.1111/adb.12758</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Fadb.12758" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31173432" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&author=L.+Konigauthor=L.+S.+Kalinichenkoauthor=S.+E.+Huberauthor=A.+M.+Vollauthor=M.+Bauderauthor=J.+Kornhuberauthor=F.+Hauschauthor=C.+P.+Muller&title=The+selective+FKBP51+inhibitor+SAFit2+reduces+alcohol+consumption+and+reinstatement+of+conditioned+alcohol+effects+in+mice&doi=10.1111%2Fadb.12758"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1111%2Fadb.12758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fadb.12758%26sid%3Dliteratum%253Aachs%26aulast%3DKonig%26aufirst%3DL.%26aulast%3DKalinichenko%26aufirst%3DL.%2BS.%26aulast%3DHuber%26aufirst%3DS.%2BE.%26aulast%3DVoll%26aufirst%3DA.%2BM.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DKornhuber%26aufirst%3DJ.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DC.%2BP.%26atitle%3DThe%2520selective%2520FKBP51%2520inhibitor%2520SAFit2%2520reduces%2520alcohol%2520consumption%2520and%2520reinstatement%2520of%2520conditioned%2520alcohol%2520effects%2520in%2520mice%26jtitle%3DAddict.%2520Biol.%26date%3D2020%26volume%3D25%26doi%3D10.1111%2Fadb.12758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozburn, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metten, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlumbohm, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hack, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeMoine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabbe, J. C.</span></span> <span> </span><span class="NLM_article-title">Targeting the glucocorticoid receptor reduces binge-like drinking in high drinking in the Dark (HDID-1) mice</span>. <i>Alcohol.: Clin. Exp. Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1111/acer.14318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1111%2Facer.14318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=32154593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptVGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2020&pages=1025-1036&author=A.+M.+Savareseauthor=A.+R.+Ozburnauthor=P.+Mettenauthor=J.+P.+Schlumbohmauthor=W.+R.+Hackauthor=K.+LeMoineauthor=H.+Huntauthor=F.+Hauschauthor=M.+Bauderauthor=J.+C.+Crabbe&title=Targeting+the+glucocorticoid+receptor+reduces+binge-like+drinking+in+high+drinking+in+the+Dark+%28HDID-1%29+mice&doi=10.1111%2Facer.14318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the Glucocorticoid Receptor Reduces Binge-Like Drinking in High Drinking in the Dark (HDID-1) Mice</span></div><div class="casAuthors">Savarese, Antonia M.; Ozburn, Angela R.; Metten, Pamela; Schlumbohm, Jason P.; Hack, Wyatt R.; LeMoine, Kathryn; Hunt, Hazel; Hausch, Felix; Bauder, Michael; Crabbe, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Alcoholism: Clinical & Experimental Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1025-1036</span>CODEN:
                <span class="NLM_cas:coden">ACRSDM</span>;
        ISSN:<span class="NLM_cas:issn">0145-6008</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chronic alc. exposure can alter glucocorticoid receptor (GR) function in some brain areas that promotes escalated and compulsive-like alc. intake.  GR antagonism can prevent dependence-induced escalation in drinking, but very little is known about the role of GR in regulating high-risk nondependent alc. intake.  Here, we investigate the role of GR in regulating binge-like drinking and aversive responses to alc. in the High Drinking in the Dark (HDID-1) mice, which have been selectively bred for high blood ethanol (EtOH) concns. (BECs) in the Drinking in the Dark (DID) test, and in their founder line, the HS/NPT.  In sep. expts., male and female HDID-1 mice were administered one of several compds. that inhibited GR or its neg. regulator, FKBP51 (mifepristone [12.5, 25, 50, 100 mg/kg], CORT113176 [20, 40, 80 mg/kg], and SAFit2 [10, 20, 40 mg/kg]) during a 2-day DID task.  EtOH consumption and BECs were measured.  EtOH conditioned taste and place aversion (CTA and CPA, resp.) were measured in sep. HDID-1 mice after mifepristone administration to assess GR's role in regulating the conditioned aversive effects of EtOH.  Lastly, HS/NPT mice were administered CORT113176 during DID to assess whether dissimilar effects from those of HDID-1 would be obsd., which could suggest that selective breeding had altered sensitivity to the effects of GR antagonism on binge-like drinking.  GR antagonism (with both mifepristone and CORT113176) selectively reduced binge-like EtOH intake and BECs in the HDID-1 mice, while inhibition of FKBP51 did not alter intake or BECs.  In contrast, GR antagonism had no effect on EtOH intake or BECs in the HS/NPT mice.  Although HDID-1 mice exhibit attenuated EtOH CTA, mifepristone administration did not enhance the aversive effects of EtOH in either a CTA or CPA task.  These data suggest that the selection process increased sensitivity to GR antagonism on EtOH intake in the HDID-1 mice, and support a role for the GR as a genetic risk factor for high-risk alc. intake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJwn8j-V1QLVg90H21EOLACvtfcHk0lhQ7cPUPBlDZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptVGktrw%253D&md5=b082e4bc8ce4336a997e6f0ad0b1f791</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1111%2Facer.14318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Facer.14318%26sid%3Dliteratum%253Aachs%26aulast%3DSavarese%26aufirst%3DA.%2BM.%26aulast%3DOzburn%26aufirst%3DA.%2BR.%26aulast%3DMetten%26aufirst%3DP.%26aulast%3DSchlumbohm%26aufirst%3DJ.%2BP.%26aulast%3DHack%26aufirst%3DW.%2BR.%26aulast%3DLeMoine%26aufirst%3DK.%26aulast%3DHunt%26aufirst%3DH.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DBauder%26aufirst%3DM.%26aulast%3DCrabbe%26aufirst%3DJ.%2BC.%26atitle%3DTargeting%2520the%2520glucocorticoid%2520receptor%2520reduces%2520binge-like%2520drinking%2520in%2520high%2520drinking%2520in%2520the%2520Dark%2520%2528HDID-1%2529%2520mice%26jtitle%3DAlcohol.%253A%2520Clin.%2520Exp.%2520Res.%26date%3D2020%26volume%3D44%26spage%3D1025%26epage%3D1036%26doi%3D10.1111%2Facer.14318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zannas, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pape, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arloth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roitman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emeny, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iurato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrillo-Roa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raikkonen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gieger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressler, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Myhsok, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladwig, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, E. B.</span></span> <span> </span><span class="NLM_article-title">Epigenetic upregulation of FKBP5 by aging and stress contributes to NF-kappa B-driven inflammation and cardiovascular risk</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">11370</span>– <span class="NLM_lpage">11379</span>, <span class="refDoi"> DOI: 10.1073/pnas.1816847116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1073%2Fpnas.1816847116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=31113877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2itLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=11370-11379&author=A.+S.+Zannasauthor=M.+W.+Jiaauthor=K.+Hafnerauthor=J.+Baumertauthor=T.+Wiechmannauthor=J.+C.+Papeauthor=J.+Arlothauthor=M.+Kodelauthor=S.+Martinelliauthor=M.+Roitmanauthor=S.+Rohauthor=A.+Haehleauthor=R.+T.+Emenyauthor=S.+Iuratoauthor=T.+Carrillo-Roaauthor=J.+Lahtiauthor=K.+Raikkonenauthor=J.+G.+Erikssonauthor=A.+J.+Drakeauthor=M.+Waldenbergerauthor=S.+Wahlauthor=S.+Kunzeauthor=S.+Lucaeauthor=B.+Bradleyauthor=C.+Giegerauthor=F.+Hauschauthor=A.+K.+Smithauthor=K.+J.+Resslerauthor=B.+Muller-Myhsokauthor=K.+H.+Ladwigauthor=T.+Reinauthor=N.+C.+Gassenauthor=E.+B.+Binder&title=Epigenetic+upregulation+of+FKBP5+by+aging+and+stress+contributes+to+NF-kappa+B-driven+inflammation+and+cardiovascular+risk&doi=10.1073%2Fpnas.1816847116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic upregulation of FKBP5 by aging and stress contributes to NF-κB-driven inflammation and cardiovascular risk</span></div><div class="casAuthors">Zannas, Anthony S.; Jia, Meiwen; Hafner, Kathrin; Baumert, Jens; Wiechmann, Tobias; Pape, Julius C.; Arloth, Janine; Koedel, Maik; Martinelli, Silvia; Roitman, Maria; Roeh, Simone; Haehle, Andreas; Emeny, Rebecca T.; Iurato, Stella; Carrillo-Roa, Tania; Lahti, Jari; Raeikkoenen, Katri; Eriksson, Johan G.; Drake, Amanda J.; Waldenberger, Melanie; Wahl, Simone; Kunze, Sonja; Lucae, Susanne; Bradley, Bekh; Gieger, Christian; Hausch, Felix; Smith, Alicia K.; Ressler, Kerry J.; Mueller-Myhsok, Bertram; Ladwig, Karl-Heinz; Rein, Theo; Gassen, Nils C.; Binder, Elisabeth B.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">11370-11379</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aging and psychosocial stress are assocd. with increased inflammation and disease risk, but the underlying mol. mechanisms are unclear.  Because both aging and stress are also assocd. with lasting epigenetic changes, a plausible hypothesis is that stress along the lifespan could confer disease risk through epigenetic effects on mols. involved in inflammatory processes.  Here, by combining large-scale analyses in human cohorts with expts. in cells, we report that FKBP5, a protein implicated in stress physiol., contributes to these relations.  Across independent human cohorts (total n > 3000), aging synergized with stress-related phenotypes, measured with childhood trauma and major depression questionnaires, to epigenetically up-regulate FKBP5 expression.  These age/stress-related epigenetic effects were recapitulated in a cellular model of replicative senescence, whereby we exposed replicating human fibroblasts to stress (glucocorticoid) hormones.  Unbiased genome-wide analyses in human blood linked higher FKBP5 mRNA with a proinflammatory profile and altered NF-κB-related gene networks.  Accordingly, expts. in immune cells showed that higher FKBP5 promotes inflammation by strengthening the interactions of NF-κB regulatory kinases, whereas opposing FKBP5 either by genetic deletion (CRISPR/Cas9-mediated) or selective pharmacol. inhibition prevented the effects on NF-κB.  Further, the age/stress-related epigenetic signature enhanced FKBP5 response to NF-κB through a pos. feedback loop and was present in individuals with a history of acute myocardial infarction, a disease state linked to peripheral inflammation.  These findings suggest that aging/stress-driven FKBP5-NF-κB signaling mediates inflammation, potentially contributing to cardiovascular risk, and may thus point to novel biomarker and treatment possibilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS78Gd39hLKrVg90H21EOLACvtfcHk0li0PPT31JY81w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2itLvO&md5=4007fdc7feef32fde4cf9c1a2ab13d76</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1816847116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1816847116%26sid%3Dliteratum%253Aachs%26aulast%3DZannas%26aufirst%3DA.%2BS.%26aulast%3DJia%26aufirst%3DM.%2BW.%26aulast%3DHafner%26aufirst%3DK.%26aulast%3DBaumert%26aufirst%3DJ.%26aulast%3DWiechmann%26aufirst%3DT.%26aulast%3DPape%26aufirst%3DJ.%2BC.%26aulast%3DArloth%26aufirst%3DJ.%26aulast%3DKodel%26aufirst%3DM.%26aulast%3DMartinelli%26aufirst%3DS.%26aulast%3DRoitman%26aufirst%3DM.%26aulast%3DRoh%26aufirst%3DS.%26aulast%3DHaehle%26aufirst%3DA.%26aulast%3DEmeny%26aufirst%3DR.%2BT.%26aulast%3DIurato%26aufirst%3DS.%26aulast%3DCarrillo-Roa%26aufirst%3DT.%26aulast%3DLahti%26aufirst%3DJ.%26aulast%3DRaikkonen%26aufirst%3DK.%26aulast%3DEriksson%26aufirst%3DJ.%2BG.%26aulast%3DDrake%26aufirst%3DA.%2BJ.%26aulast%3DWaldenberger%26aufirst%3DM.%26aulast%3DWahl%26aufirst%3DS.%26aulast%3DKunze%26aufirst%3DS.%26aulast%3DLucae%26aufirst%3DS.%26aulast%3DBradley%26aufirst%3DB.%26aulast%3DGieger%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSmith%26aufirst%3DA.%2BK.%26aulast%3DRessler%26aufirst%3DK.%2BJ.%26aulast%3DMuller-Myhsok%26aufirst%3DB.%26aulast%3DLadwig%26aufirst%3DK.%2BH.%26aulast%3DRein%26aufirst%3DT.%26aulast%3DGassen%26aufirst%3DN.%2BC.%26aulast%3DBinder%26aufirst%3DE.%2BB.%26atitle%3DEpigenetic%2520upregulation%2520of%2520FKBP5%2520by%2520aging%2520and%2520stress%2520contributes%2520to%2520NF-kappa%2520B-driven%2520inflammation%2520and%2520cardiovascular%2520risk%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2019%26volume%3D116%26spage%3D11370%26epage%3D11379%26doi%3D10.1073%2Fpnas.1816847116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Structure-affinity relationship analysis of selective FKBP51 ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7796</span>– <span class="NLM_lpage">7806</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00785</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00785" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2gsbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7796-7806&author=X.+Fengauthor=C.+Sippelauthor=A.+Bracherauthor=F.+Hausch&title=Structure-affinity+relationship+analysis+of+selective+FKBP51+ligands&doi=10.1021%2Facs.jmedchem.5b00785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Affinity Relationship Analysis of Selective FKBP51 Ligands</span></div><div class="casAuthors">Feng, Xixi; Sippel, Claudia; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7796-7806</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51) is a promising drug target for the treatment of stress-related psychiatric or metabolic disorders.  Just recently, the first selective ligands for FKBP51 were reported based on an induced fit mechanism, but they are too large for a further drug development process.  The authors therefore designed and synthesized a novel series of selective ligands to explore the requirements necessary for binding to the induced-fit conformation.  All ligands of this series show no binding toward the structurally very similar antitarget FKBP52.  With the cocrystal structure of the best ligand in this novel series the authors confirmed the induced fit mechanism.  Furthermore, the structure-affinity relationship provides information about beneficial structural features, which is valuable for the development of improved FKBP51-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIuleKzzlpMbVg90H21EOLACvtfcHk0li0PPT31JY81w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2gsbfL&md5=90c480d0a6819dea97e4742e6db24583</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00785%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DStructure-affinity%2520relationship%2520analysis%2520of%2520selective%2520FKBP51%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7796%26epage%3D7806%26doi%3D10.1021%2Facs.jmedchem.5b00785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomplun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Recent progress in FKBP ligand development</span>. <i>Curr. Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.2174/1874467208666150519113313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.2174%2F1874467208666150519113313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25986570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC2MfltVKiug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=27-36&author=X.+X.+Fengauthor=S.+Pomplunauthor=F.+Hausch&title=Recent+progress+in+FKBP+ligand+development&doi=10.2174%2F1874467208666150519113313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in FKBP Ligand Development</span></div><div class="casAuthors">Feng Xixi; Pomplun Sebastian; Hausch Felix</div><div class="citationInfo"><span class="NLM_cas:title">Current molecular pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-36</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">FK506-binding proteins have been implicated in numerous human diseases suggesting novel therapeutic opportunities.  In particular, the large FKBP51 has emerged as an important regulator of the stress-coping system and as an established risk factor for stress-related disorders.  The principal druggabilty of FKBPs is evidenced by the prototypical high affinity ligands FK506 and rapamycin but the development of more refined and selective chemical probes for FKBPs has been challenging.  In this review we summarize recent advances in the development of FKBP ligands, which cumulated in the first highly selective ligands for FKBP51.  The best ligand SAFit2 allowed the proof-of-concept in mice for FKBP51 inhibitors as potentially novel antidepressants.  Finally, we discuss pending issues that need to be addressed for the further development of FKBP51-directed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQzpKzaquF5abZpSb_XWfMrfW6udTcc2eaPmODh2XYRe7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MfltVKiug%253D%253D&md5=8e40ae83c490c941c986f35a17d8987c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F1874467208666150519113313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874467208666150519113313%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%2BX.%26aulast%3DPomplun%26aufirst%3DS.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DRecent%2520progress%2520in%2520FKBP%2520ligand%2520development%26jtitle%3DCurr.%2520Mol.%2520Pharmacol.%26date%3D2015%26volume%3D9%26spage%3D27%26epage%3D36%26doi%3D10.2174%2F1874467208666150519113313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, P. K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaila, V. R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattler, M.</span></span> <span> </span><span class="NLM_article-title">Selective inhibitors of FKBP51 employ conformational selection of dynamic invisible states</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">13619</span>– <span class="NLM_lpage">13619</span>, <span class="refDoi"> DOI: 10.1002/anie.201908888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fanie.201908888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsF2ltr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=13619-13619&author=P.+K.+A.+Jagtapauthor=S.+Asamiauthor=C.+Sippelauthor=V.+R.+I.+Kailaauthor=F.+Hauschauthor=M.+Sattler&title=Selective+inhibitors+of+FKBP51+employ+conformational+selection+of+dynamic+invisible+states&doi=10.1002%2Fanie.201908888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Corrigendum: Selective Inhibitors of FKBP51 Employ Conformational Selection of Dynamic Invisible States</span></div><div class="casAuthors">Jagtap, Pravin Kumar Ankush; Asami, Sam; Sippel, Claudia; Kaila, Ville R. I.; Hausch, Felix; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">13619</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqLb9uy9ZXLVg90H21EOLACvtfcHk0liaGbzoRPx9dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsF2ltr7P&md5=ed2f1bdb31da080dcfd6e5069b7cc16c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201908888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201908888%26sid%3Dliteratum%253Aachs%26aulast%3DJagtap%26aufirst%3DP.%2BK.%2BA.%26aulast%3DAsami%26aufirst%3DS.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DKaila%26aufirst%3DV.%2BR.%2BI.%26aulast%3DHausch%26aufirst%3DF.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DSelective%2520inhibitors%2520of%2520FKBP51%2520employ%2520conformational%2520selection%2520of%2520dynamic%2520invisible%2520states%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D13619%26epage%3D13619%26doi%3D10.1002%2Fanie.201908888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Draxler, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickmeier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nadal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rangel Rojas, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeb, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiegen, D.</span></span> <span> </span><span class="NLM_article-title">Hybrid screening approach for very small fragments: X-ray and computational screening on FKBP51</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5856</span>– <span class="NLM_lpage">5864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsFelsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5856-5864&author=S.+W.+Draxlerauthor=M.+Bauerauthor=C.+Eickmeierauthor=S.+Nadalauthor=H.+Narauthor=D.+Rangel+Rojasauthor=D.+Seeligerauthor=M.+Zeebauthor=D.+Fiegen&title=Hybrid+screening+approach+for+very+small+fragments%3A+X-ray+and+computational+screening+on+FKBP51&doi=10.1021%2Facs.jmedchem.0c00120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Hybrid Screening Approach for Very Small Fragments: X-ray and Computational Screening on FKBP51</span></div><div class="casAuthors">Draxler, Sebastian W.; Bauer, Margit; Eickmeier, Christian; Nadal, Simon; Nar, Herbert; Rangel Rojas, Daniel; Seeliger, Daniel; Zeeb, Markus; Fiegen, Dennis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5856-5864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) permits efficient sampling of the vast chem. space for hit identification.  Libraries are screened biophys. and fragment:protein co-structures are detd. by X-ray crystallog.  In parallel, computational methods can derive pharmacophore models or screen virtual libraries.  We screened 15 very small fragments (VSFs) (HA ≤ 11) computationally, using site identification by ligand competitive satn. (SILCS), and exptl., by X-ray crystallog., to map potential interaction sites on the FKBP51 FK1 domain.  We identified three hot spots and obtained 6 X-ray co-structures, giving a hit rate of 40%.  SILCS FragMaps overlapped with X-ray structures.  The compds. had millimolar affinities as detd. by 15N HSQC NMR.  VSFs identified the same interactions as known FK1 binder and provide new chem. starting points.  We propose a hybrid screening strategy starting with SILCS, followed by a pharmacophore-derived X-ray screen and 15N HSQC NMR based KD detn. to rapidly identify hits and their binding poses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOqWmaKxGJkrVg90H21EOLACvtfcHk0lg5TGIMEHj5-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsFelsLo%253D&md5=35fe6bd3207a0b8d2b08f61c1723371a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00120%26sid%3Dliteratum%253Aachs%26aulast%3DDraxler%26aufirst%3DS.%2BW.%26aulast%3DBauer%26aufirst%3DM.%26aulast%3DEickmeier%26aufirst%3DC.%26aulast%3DNadal%26aufirst%3DS.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DRangel%2BRojas%26aufirst%3DD.%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DZeeb%26aufirst%3DM.%26aulast%3DFiegen%26aufirst%3DD.%26atitle%3DHybrid%2520screening%2520approach%2520for%2520very%2520small%2520fragments%253A%2520X-ray%2520and%2520computational%2520screening%2520on%2520FKBP51%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5856%26epage%3D5864%26doi%3D10.1021%2Facs.jmedchem.0c00120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabani, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulochana, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddesh, A. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saini, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span> <span> </span><span class="NLM_article-title">Validated LC-MS/MS method for simultaneous quantitation of SAFit-1 and SAFit-2 in mice plasma: application to a pharmacokinetic study</span>. <i>Drug Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1055/a-1164-6123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1055%2Fa-1164-6123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlKisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2020&pages=325-332&author=B.+B.+Gabaniauthor=S.+P.+Sulochanaauthor=A.+H.+A.+Siddeshauthor=V.+Kiranauthor=N.+K.+Sainiauthor=S.+K.+Samantaauthor=M.+S.+Hallurauthor=S.+Rajagopalauthor=R.+Mullangi&title=Validated+LC-MS%2FMS+method+for+simultaneous+quantitation+of+SAFit-1+and+SAFit-2+in+mice+plasma%3A+application+to+a+pharmacokinetic+study&doi=10.1055%2Fa-1164-6123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Validated LC-MS/MS Method for Simultaneous Quantitation of SAFit-1 and SAFit-2 in Mice Plasma: Application to a Pharmacokinetic Study</span></div><div class="casAuthors">Gabani, Bhavesh Babulal; Sulochana, Suresh Ponnayyan; Siddesh, Anup H. A.; Kiran, Vinay; Saini, Neeraj Kumar; Samanta, Swapan Kumar; Hallur, Mahanandeesha S.; Rajagopal, Sridharan; Mullangi, Ramesh</div><div class="citationInfo"><span class="NLM_cas:title">Drug Research (Stuttgart, Germany)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">325-332</span>CODEN:
                <span class="NLM_cas:coden">DRSGAO</span>;
        ISSN:<span class="NLM_cas:issn">2194-9379</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">SAFit-1 and SAFit-2 are selective FKBP51 (FK506-binding protein 51) ligands.  In this paper, we present the development and validation data of an LC-MS/MS method for the simultaneous quantitation of SAFit-1 and SAFit-2 in mice plasma as per FDA regulatory guideline.  SAFit-1 and SAFit-2 along with internal std. were extd. from mice plasma using liq.-liq. extn. method.  Chromatog. resoln. of SAFit-1, SAFit-2 and the internal std. (warfarin) was achieved on an X-Terra Ph column using 0.2% formic acid:acetonitrile (20:80, vol./vol.) as an eluent, which was delivered at a flow-rate of 0.9 mL/min.  The MS/MS ion transitions monitored were m/z 748.4420.4, 803.7384.3 and 309.2 163.2 for SAFit-1, SAFit-2 and the internal std., resp.  The linearity range was 2.45-2446 ng/mL for both SAFit-1 and SAFit-2.  The intra- and inter-day accuracy and intra- and inter-day precision were in the range of 0.90-1.07 and 2.38-10.8%, resp. for SAFit-1; 0.97-1.15 and 0.23-12.5%, resp. for SAFit-2.  Both SAFit-1 and SAFit-2 were found to be stable in stability studies (up to three freeze-thaw cycles and for long-term at -80°C for 30 days) and processed (bench-top for 3 h and in in-injector for 16 h) samples.  The application of the validated method was shown in a pharmacokinetic study in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojtHALoOYkXLVg90H21EOLACvtfcHk0lg5TGIMEHj5-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlKisL0%253D&md5=4062795128725b3bf8152c897fc1c900</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1055%2Fa-1164-6123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fa-1164-6123%26sid%3Dliteratum%253Aachs%26aulast%3DGabani%26aufirst%3DB.%2BB.%26aulast%3DSulochana%26aufirst%3DS.%2BP.%26aulast%3DSiddesh%26aufirst%3DA.%2BH.%2BA.%26aulast%3DKiran%26aufirst%3DV.%26aulast%3DSaini%26aufirst%3DN.%2BK.%26aulast%3DSamanta%26aufirst%3DS.%2BK.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26atitle%3DValidated%2520LC-MS%252FMS%2520method%2520for%2520simultaneous%2520quantitation%2520of%2520SAFit-1%2520and%2520SAFit-2%2520in%2520mice%2520plasma%253A%2520application%2520to%2520a%2520pharmacokinetic%2520study%26jtitle%3DDrug%2520Res.%26date%3D2020%26volume%3D70%26spage%3D325%26epage%3D332%26doi%3D10.1055%2Fa-1164-6123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knaup, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staibano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">A novel decalin-based bicyclic scaffold for FKBP51-selective ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWktrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=231-240&author=X.+Fengauthor=C.+Sippelauthor=F.+H.+Knaupauthor=A.+Bracherauthor=S.+Staibanoauthor=M.+F.+Romanoauthor=F.+Hausch&title=A+novel+decalin-based+bicyclic+scaffold+for+FKBP51-selective+ligands&doi=10.1021%2Facs.jmedchem.9b01157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A novel decalin-based bicyclic scaffold for FKBP51-selective ligands</span></div><div class="casAuthors">Feng, Xixi; Sippel, Claudia; Knaup, Fabian H.; Bracher, Andreas; Staibano, Stefania; Romano, Maria F.; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibition of FKBP51 has emerged as possible novel treatment for diseases like major depressive disorder, obesity, chronic pain and certain cancers.  The current FKBP51 inhibitors are rather large, flexible and have to be further optimized.  Using a structure-based rigidification strategy, a novel promising bicyclic scaffold for FKBP51 ligands has been designed and synthesized.  The structure-activity anal. revealed the decalin scaffold as the best moiety for the selectivity-enabling subpocket of FBKP51.  The resulting compds. retain high potency for FKBP51 and excellent selectivity over the close homolog FKBP52.  With the cocrystal structure of an advanced ligand in this novel series, it was shown how the decalin locks the key selectivity-inducing cyclohexyl moiety of the ligand in a conformation typical for FKBP51-selective binding.  The best compd. I produces cell death in a HeLa-derived KB cell line, a cellular model of cervical adenocarcinoma, where FKBP51 is highly overexpressed.  This results show how FKBP51 inhibitors can be rigidified and extended while preserving FKBP51 selectivity.  Such inhibitors might be novel tools in the treatment of human cancers with deregulated FKBP51.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQvCfTxSgpFrVg90H21EOLACvtfcHk0lg5TGIMEHj5-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWktrnI&md5=8b480e29b1cdeed653b7e908f28f881a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01157%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DKnaup%26aufirst%3DF.%2BH.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DStaibano%26aufirst%3DS.%26aulast%3DRomano%26aufirst%3DM.%2BF.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DA%2520novel%2520decalin-based%2520bicyclic%2520scaffold%2520for%2520FKBP51-selective%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D231%26epage%3D240%26doi%3D10.1021%2Facs.jmedchem.9b01157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Rapid, structure-based exploration of pipecolic acid amides as novel selective antagonists of the FK506-binding protein 51</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2410</span>– <span class="NLM_lpage">2422</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslSqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2410-2422&author=S.+Gaaliauthor=X.+Fengauthor=A.+Hahleauthor=C.+Sippelauthor=A.+Bracherauthor=F.+Hausch&title=Rapid%2C+structure-based+exploration+of+pipecolic+acid+amides+as+novel+selective+antagonists+of+the+FK506-binding+protein+51&doi=10.1021%2Facs.jmedchem.5b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, Structure-Based Exploration of Pipecolic Acid Amides as Novel Selective Antagonists of the FK506-Binding Protein 51</span></div><div class="casAuthors">Gaali, Steffen; Feng, Xixi; Haehle, Andreas; Sippel, Claudia; Bracher, Andreas; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2410-2422</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The FK506-binding protein 51 (FKBP51) is a key regulator of stress hormone receptors and an established risk factor for stress-related disorders.  Drug development for FKBP51 has been impaired by the structurally similar but functionally opposing homolog FKBP52.  High selectivity between FKBP51 and FKBP52 can be achieved by ligands that stabilize a recently discovered FKBP51-favoring conformation.  However, drug-like parameters for these ligands remained unfavorable.  In the present study, we replaced the potentially labile pipecolic ester group of previous FKBP51 ligands by various low mol. wt. amides.  This resulted in the first series of pipecolic acid amides, which had much lower mol. wts. without affecting FKBP51 selectivity.  We discovered a geminally substituted cyclopentyl amide as a preferred FKBP51-binding motif and elucidated its binding mode to provide a new lead structure for future drug optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpotLzPLV83BrVg90H21EOLACvtfcHk0lhtH3otq9rz7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslSqs7c%253D&md5=7a46f4b0da6344be79bd2aa2e941ba81</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01355%26sid%3Dliteratum%253Aachs%26aulast%3DGaali%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DRapid%252C%2520structure-based%2520exploration%2520of%2520pipecolic%2520acid%2520amides%2520as%2520novel%2520selective%2520antagonists%2520of%2520the%2520FK506-binding%2520protein%252051%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2410%26epage%3D2422%26doi%3D10.1021%2Facs.jmedchem.5b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dunyak, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Peptidyl-proline isomerases (PPlases): targets for natural products and natural product-inspired compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9622</span>– <span class="NLM_lpage">9644</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00411</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00411" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFKms73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9622-9644&author=B.+M.+Dunyakauthor=J.+E.+Gestwicki&title=Peptidyl-proline+isomerases+%28PPlases%29%3A+targets+for+natural+products+and+natural+product-inspired+compounds&doi=10.1021%2Facs.jmedchem.6b00411"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl-Proline Isomerases (PPIases): Targets for Natural Products and Natural Product-Inspired Compounds</span></div><div class="casAuthors">Dunyak, Bryan M.; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9622-9644</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptidyl-proline isomerases (PPIases) are a chaperone superfamily comprising the FK506-binding proteins (FKBPs), cyclophilins, and parvulins.  PPIases catalyze the cis/trans isomerization of proline, acting as a regulatory switch during folding, activation, and/or degrdn. of many proteins.  These "clients" include proteins with key roles in cancer, neurodegeneration, and psychiatric disorders, suggesting that PPIase inhibitors could be important therapeutics.  However, the active site of PPIases is shallow, solvent-exposed, and well conserved between family members, making selective inhibitor design challenging.  Despite these hurdles, macrocyclic natural products, including FK506, rapamycin, and cyclosporin, bind PPIases with nanomolar or better affinity.  De novo attempts to derive new classes of inhibitors have been somewhat less successful, often showcasing the "undruggable" features of PPIases.  Interestingly, the most potent of these next-generation mols. tend to integrate features of the natural products, including macrocyclization or proline mimicry strategies.  Here, we review recent developments and ongoing challenges in the inhibition of PPIases, with a focus on how natural products might inform the creation of potent and selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz44xLeDGfhLVg90H21EOLACvtfcHk0lhtH3otq9rz7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFKms73J&md5=b4c6336096f9558926d965d73fb6b7da</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00411%26sid%3Dliteratum%253Aachs%26aulast%3DDunyak%26aufirst%3DB.%2BM.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DPeptidyl-proline%2520isomerases%2520%2528PPlases%2529%253A%2520targets%2520for%2520natural%2520products%2520and%2520natural%2520product-inspired%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9622%26epage%3D9644%26doi%3D10.1021%2Facs.jmedchem.6b00411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0lhtH3otq9rz7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based macrocycle design in small-molecule drug discovery and simple metrics to identify opportunities for macrocyclization of small-molecule ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-based+macrocycle+design+in+small-molecule+drug+discovery+and+simple+metrics+to+identify+opportunities+for+macrocyclization+of+small-molecule+ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0ljNPghKE6mSxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-based%2520macrocycle%2520design%2520in%2520small-molecule%2520drug%2520discovery%2520and%2520simple%2520metrics%2520to%2520identify%2520opportunities%2520for%2520macrocyclization%2520of%2520small-molecule%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljNPghKE6mSxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The exploration of macrocycles for drug discovery - an underexploited structural class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+exploration+of+macrocycles+for+drug+discovery+-+an+underexploited+structural+class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lgsbIovXcFMNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520exploration%2520of%2520macrocycles%2520for%2520drug%2520discovery%2520-%2520an%2520underexploited%2520structural%2520class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in new drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+new+drug+discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lgsbIovXcFMNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520new%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles are great cycles: applications, opportunities, and challenges of synthetic macrocycles in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+are+great+cycles%3A+applications%2C+opportunities%2C+and+challenges+of+synthetic+macrocycles+in+drug+discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgsbIovXcFMNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520are%2520great%2520cycles%253A%2520applications%252C%2520opportunities%252C%2520and%2520challenges%2520of%2520synthetic%2520macrocycles%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic drugs and clinical candidates: what can medicinal chemists learn from their properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+drugs+and+clinical+candidates%3A+what+can+medicinal+chemists+learn+from+their+properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0ljY7oEnGGs3ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520drugs%2520and%2520clinical%2520candidates%253A%2520what%2520can%2520medicinal%2520chemists%2520learn%2520from%2520their%2520properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leavitt, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent cyclophilin inhibitors based on the structural simplification of sanglifehrin A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01329</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01329" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptlWrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1000-1017&author=V.+A.+Steadmanauthor=S.+B.+Pettitauthor=K.+G.+Poullennecauthor=L.+Lazaridesauthor=A.+J.+Keatsauthor=D.+K.+Deanauthor=S.+J.+Stanwayauthor=C.+A.+Austinauthor=J.+A.+Sanvoisinauthor=G.+M.+Wattauthor=H.+G.+Fliriauthor=A.+C.+Liclicanauthor=D.+Jinauthor=M.+H.+Wongauthor=S.+A.+Leavittauthor=Y.+J.+Leeauthor=Y.+Tianauthor=C.+R.+Freyauthor=T.+C.+Applebyauthor=U.+Schmitzauthor=P.+Jansaauthor=R.+L.+Mackmanauthor=B.+E.+Schultz&title=Discovery+of+potent+cyclophilin+inhibitors+based+on+the+structural+simplification+of+sanglifehrin+A&doi=10.1021%2Facs.jmedchem.6b01329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Cyclophilin Inhibitors Based on the Structural Simplification of Sanglifehrin A</span></div><div class="casAuthors">Steadman, Victoria A.; Pettit, Simon B.; Poullennec, Karine G.; Lazarides, Linos; Keats, Andrew J.; Dean, David K.; Stanway, Steven J.; Austin, Carol A.; Sanvoisin, Jonathan A.; Watt, Gregory M.; Fliri, Hans G.; Liclican, Albert C.; Jin, Debi; Wong, Melanie H.; Leavitt, Stephanie A.; Lee, Yu-Jen; Tian, Yang; Frey, Christian R.; Appleby, Todd C.; Schmitz, Uli; Jansa, Petr; Mackman, Richard L.; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1000-1017</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilin inhibition has been a target for the treatment of hepatitis C and other diseases, but the generation of potent, drug-like mols. through chem. synthesis has been challenging.  In this study, a set of macrocyclic cyclophilin inhibitors was synthesized based on the core structure of the natural product sanglifehrin A.  Initial compd. optimization identified the valine-m-tyrosine-piperazic acid tripeptide (Val-m-Tyr-Pip) in the sanglifehrin core, stereocenters at C14 and C15, and the hydroxyl group of the m-tyrosine (m-Tyr) residue as key contributors to compd. potency.  Replacing the C18-C21 diene unit of sanglifehrin with a styryl group led to potent compds. that displayed a novel binding mode in which the styrene moiety engaged in a π-stacking interaction with Arg55 of cyclophilin A (Cyp A), and the m-Tyr residue was displaced into solvent.  This observation allowed further simplifications of the scaffold to generate new lead compds. in the search for orally bioavailable cyclophilin inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorJXRBXLoC-rVg90H21EOLACvtfcHk0ljY7oEnGGs3ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXptlWrsg%253D%253D&md5=7b3c1ccee76ff6da98064783c0710fd6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01329%26sid%3Dliteratum%253Aachs%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DStanway%26aufirst%3DS.%2BJ.%26aulast%3DAustin%26aufirst%3DC.%2BA.%26aulast%3DSanvoisin%26aufirst%3DJ.%2BA.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DFliri%26aufirst%3DH.%2BG.%26aulast%3DLiclican%26aufirst%3DA.%2BC.%26aulast%3DJin%26aufirst%3DD.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLeavitt%26aufirst%3DS.%2BA.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DFrey%26aufirst%3DC.%2BR.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520potent%2520cyclophilin%2520inhibitors%2520based%2520on%2520the%2520structural%2520simplification%2520of%2520sanglifehrin%2520A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1000%26epage%3D1017%26doi%3D10.1021%2Facs.jmedchem.6b01329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poullennec, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannizzaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiva, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunbar, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fliri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highton, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mish, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoisin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperandio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stepan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watt, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9473</span>– <span class="NLM_lpage">9499</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00802</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00802" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9473-9499&author=R.+L.+Mackmanauthor=V.+A.+Steadmanauthor=D.+K.+Deanauthor=P.+Jansaauthor=K.+G.+Poullennecauthor=T.+Applebyauthor=C.+Austinauthor=C.+A.+Blakemoreauthor=R.+Caiauthor=C.+Cannizzaroauthor=G.+Chinauthor=J.+C.+Chivaauthor=N.+A.+Dunbarauthor=H.+Fliriauthor=A.+J.+Hightonauthor=H.+Huiauthor=M.+Jiauthor=H.+Jinauthor=K.+Karkiauthor=A.+J.+Keatsauthor=L.+Lazaridesauthor=Y.+J.+Leeauthor=A.+Liclicanauthor=M.+Mishauthor=B.+Murrayauthor=S.+B.+Pettitauthor=P.+Pyunauthor=M.+Sangiauthor=R.+Santosauthor=J.+Sanvoisinauthor=U.+Schmitzauthor=A.+Schrierauthor=D.+Siegelauthor=D.+Sperandioauthor=G.+Stepanauthor=Y.+Tianauthor=G.+M.+Wattauthor=H.+Yangauthor=B.+E.+Schultz&title=Discovery+of+a+potent+and+orally+bioavailable+cyclophilin+inhibitor+derived+from+the+sanglifehrin+macrocycle&doi=10.1021%2Facs.jmedchem.8b00802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a potent and orally bioavailable cyclophilin inhibitor derived from the sanglifehrin macrocycle</span></div><div class="casAuthors">Mackman, Richard L.; Steadman, Victoria A.; Dean, David K.; Jansa, Petr; Poullennec, Karine G.; Appleby, Todd; Austin, Carol; Blakemore, Caroline A.; Cai, Ruby; Cannizzaro, Carina; Chin, Gregory; Chiva, Jean-Yves C.; Dunbar, Neil A.; Fliri, Hans; Highton, Adrian J.; Hui, Hon; Ji, Mingzhe; Jin, Haolun; Karki, Kapil; Keats, Andrew J.; Lazarides, Linos; Lee, Yu-Jen; Liclican, Albert; Mish, Michael; Murray, Bernard; Pettit, Simon B.; Pyun, Peter; Sangi, Michael; Santos, Rex; Sanvoisin, Jonathan; Schmitz, Uli; Schrier, Adam; Siegel, Dustin; Sperandio, David; Stepan, George; Tian, Yang; Watt, Gregory M.; Yang, Hai; Schultz, Brian E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9473-9499</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclophilins are a family of peptidyl-prolyl isomerases that are implicated in a wide range of diseases including hepatitis C.  Our aim was to discover through total synthesis an orally bioavailable, non-immunosuppressive cyclophilin (Cyp) inhibitor with potent anti-hepatitis C virus (HCV) activity that could serve as part of an all oral antiviral combination therapy.  An initial lead (I) derived from the sanglifehrin A macrocycle was optimized using structure based design to produce a potent and orally bioavailable inhibitor (II).  The macrocycle ring size was reduced by one atom, and an internal hydrogen bond drove improved permeability and drug-like properties. II demonstrates potent Cyp inhibition (Kd = 5 nM), potent anti-HCV 2a activity (EC50 = 98 nM), and high oral bioavailability in rat (100%) and dog (55%).  The synthetic accessibility and properties of II support its potential as an anti-HCV agent and for interrogating the role of Cyp inhibition in a variety of diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEpEZcwXK8z7Vg90H21EOLACvtfcHk0liv_vBrng-SHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVChtbrI&md5=616cf72e0d2766e159769166c98a9e86</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00802%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DDean%26aufirst%3DD.%2BK.%26aulast%3DJansa%26aufirst%3DP.%26aulast%3DPoullennec%26aufirst%3DK.%2BG.%26aulast%3DAppleby%26aufirst%3DT.%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBlakemore%26aufirst%3DC.%2BA.%26aulast%3DCai%26aufirst%3DR.%26aulast%3DCannizzaro%26aufirst%3DC.%26aulast%3DChin%26aufirst%3DG.%26aulast%3DChiva%26aufirst%3DJ.%2BC.%26aulast%3DDunbar%26aufirst%3DN.%2BA.%26aulast%3DFliri%26aufirst%3DH.%26aulast%3DHighton%26aufirst%3DA.%2BJ.%26aulast%3DHui%26aufirst%3DH.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DKeats%26aufirst%3DA.%2BJ.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DY.%2BJ.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DMish%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DPettit%26aufirst%3DS.%2BB.%26aulast%3DPyun%26aufirst%3DP.%26aulast%3DSangi%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DSanvoisin%26aufirst%3DJ.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DSchrier%26aufirst%3DA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DSperandio%26aufirst%3DD.%26aulast%3DStepan%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWatt%26aufirst%3DG.%2BM.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520orally%2520bioavailable%2520cyclophilin%2520inhibitor%2520derived%2520from%2520the%2520sanglifehrin%2520macrocycle%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9473%26epage%3D9499%26doi%3D10.1021%2Facs.jmedchem.8b00802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girao da Cruz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhazes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marques, M. P. M.</span></span> <span> </span><span class="NLM_article-title">Anticancer activity of phenolic acids of natural or synthetic origin: a structure-activity study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5395</span>– <span class="NLM_lpage">5401</span>, <span class="refDoi"> DOI: 10.1021/jm030956v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030956v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVClt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5395-5401&author=C.+A.+Gomesauthor=T.+Girao+da+Cruzauthor=J.+L.+Andradeauthor=N.+Milhazesauthor=F.+Borgesauthor=M.+P.+M.+Marques&title=Anticancer+activity+of+phenolic+acids+of+natural+or+synthetic+origin%3A+a+structure-activity+study&doi=10.1021%2Fjm030956v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer Activity of Phenolic Acids of Natural or Synthetic Origin: A Structure-Activity Study</span></div><div class="casAuthors">Gomes, Catarina A.; Girao da Cruz, Teresa; Andrade, Jose L.; Milhazes, Nuno; Borges, Fernanda; Marques, M. Paula M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5395-5401</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several phenolic acids-caffeic and gallic acid derivs.-were synthesized and screened for their potential antiproliferative and cytotoxic properties, in different human cancer cell lines: mammary gland and cervix adenocarcinomas and lymphoblastic leukemia.  The selected phenols were structurally related, which allowed us to gather important information regarding the structure-activity relationships underlying the biol. activity of such compds.  This is proposed to be due to a balance between the antioxidant and pro-oxidant properties of this kind of agent.  Distinct effects were found for different cell lines, which points to a significant specificity of action of the drugs tested.  It was verified, for the types of cancer investigated, that the trihydroxylated derivs. yielded better results than the dihydroxylated ones.  Tests in noncancerous cells, human lung fibroblasts, were also undertaken, in view of detg. the toxic side effects of the compds. studied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEnWoIoYj87Vg90H21EOLACvtfcHk0lg5Pyok6s2V2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVClt7g%253D&md5=c079f974dca4ef1c55bfd29bfef10dc9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm030956v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030956v%26sid%3Dliteratum%253Aachs%26aulast%3DGomes%26aufirst%3DC.%2BA.%26aulast%3DGirao%2Bda%2BCruz%26aufirst%3DT.%26aulast%3DAndrade%26aufirst%3DJ.%2BL.%26aulast%3DMilhazes%26aufirst%3DN.%26aulast%3DBorges%26aufirst%3DF.%26aulast%3DMarques%26aufirst%3DM.%2BP.%2BM.%26atitle%3DAnticancer%2520activity%2520of%2520phenolic%2520acids%2520of%2520natural%2520or%2520synthetic%2520origin%253A%2520a%2520structure-activity%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5395%26epage%3D5401%26doi%3D10.1021%2Fjm030956v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span>; <span class="NLM_string-name">Zhang, X.</span></span> <i>Phenylacetic acid derivative and antitumor use thereof</i>. World Patent <span class="NLM_patent">2015/018182 A1</span>, February 2, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=S.+Li&author=X.+Zhang&title=Phenylacetic+acid+derivative+and+antitumor+use+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26jtitle%3DPhenylacetic%2520acid%2520derivative%2520and%2520antitumor%2520use%2520thereof%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bols, M.</span></span> <span> </span><span class="NLM_article-title">An inexpensive and scalable synthesis of Shld</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">6050</span>– <span class="NLM_lpage">6055</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.8b00698</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.8b00698" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A280%3ADC%252BC1MfitFajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=6050-6055&author=F.+P.+Jorgensenauthor=M.+Bols&title=An+inexpensive+and+scalable+synthesis+of+Shld&doi=10.1021%2Facs.joc.8b00698"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">An Inexpensive and Scalable Synthesis of Shld</span></div><div class="casAuthors">Jorgensen Frederik Praestholm; Bols Mikael</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of organic chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6050-6055</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A synthesis of the important FKBP ligand Shld is reported.  The synthesis avoids stoichiometric use of expensive and chiral reagents, maintains enantioselectivity and provides a high overall yield (39%).  The main features in the method are enantioselective alkylation for preparation of the phenyl acetic acid moiety (building block A), catalytic enantioselective reduction for obtaining the chiral diaryl-1-propanol (building block C), and direct acylation of S-pipecolic tartrate rather than use of expensive Fmoc-pipecolic acid.  The assembly of the building blocks A-C is reversed in comparison to previous synthesis, which also eliminates the need for protective groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVlUzHyK4uGrrIbt-tGZmxfW6udTcc2eYZbqLH2-EDz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfitFajsA%253D%253D&md5=e934f7eb001e353f68552fa0bdb22568</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.8b00698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.8b00698%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DF.%2BP.%26aulast%3DBols%26aufirst%3DM.%26atitle%3DAn%2520inexpensive%2520and%2520scalable%2520synthesis%2520of%2520Shld%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2018%26volume%3D83%26spage%3D6050%26epage%3D6055%26doi%3D10.1021%2Facs.joc.8b00698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, E. J.</span></span> <span> </span><span class="NLM_article-title">Enantioselective total synthesis of aspidophytine</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">6771</span>– <span class="NLM_lpage">6772</span>, <span class="refDoi"> DOI: 10.1021/ja9915201</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9915201" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADyaK1MXktVahsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=6771-6772&author=F.+Heauthor=Y.+Boauthor=J.+D.+Altomauthor=E.+J.+Corey&title=Enantioselective+total+synthesis+of+aspidophytine&doi=10.1021%2Fja9915201"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective Total Synthesis of Aspidophytine</span></div><div class="casAuthors">He, Feng; Bo, Yunxin; Altom, Jason D.; Corey, E. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6771-6772</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The alkaloid aspidophytine was prepd. via the coupling of I and II that yielded the pentacyclic synthon III which followed by base catalyzed hydrolysis and oxidative lactonization provided the title compd. in 86% yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5yFwvxG0WhrVg90H21EOLACvtfcHk0lg5Pyok6s2V2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVahsro%253D&md5=1407fccbeb944f8babc7d7b6497aaea1</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja9915201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9915201%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DF.%26aulast%3DBo%26aufirst%3DY.%26aulast%3DAltom%26aufirst%3DJ.%2BD.%26aulast%3DCorey%26aufirst%3DE.%2BJ.%26atitle%3DEnantioselective%2520total%2520synthesis%2520of%2520aspidophytine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D6771%26epage%3D6772%26doi%3D10.1021%2Fja9915201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morandi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickens, Z. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Regioselective Wacker oxidation of internal alkenes: rapid access to functionalized ketones facilitated by cross-metathesis</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">9751</span>– <span class="NLM_lpage">9754</span>, <span class="refDoi"> DOI: 10.1002/anie.201303587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fanie.201303587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhs7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=9751-9754&author=B.+Morandiauthor=Z.+K.+Wickensauthor=R.+H.+Grubbs&title=Regioselective+Wacker+oxidation+of+internal+alkenes%3A+rapid+access+to+functionalized+ketones+facilitated+by+cross-metathesis&doi=10.1002%2Fanie.201303587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective Wacker oxidation of internal alkenes: rapid access to functionalized ketones facilitated by cross-metathesis</span></div><div class="casAuthors">Morandi, Bill; Wickens, Zachary K.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">9751-9754</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A new synthetic strategy to regioselectively access complex ketone products from simple starting materials is described.  A wide range of functionalized alkenes were established as efficient substrates for a regioseνlective Wacker oxidn., thus producing valuable synthetic intermediates (hydroxyketones, aminoketones, ketoesters).  It was demonstrated using both E and Z alkene isomers prepd. by cross-metathesis could be use as the starting materials for this regioselective Wacker oxidn.  An intriguing long-range inductive dependence was shown by competition expts.  The high functional-group tolerance of both the cross-metathesis step and the Wacker oxidn., combined with the predictable regioselectivity of the oxidn. step, holds great promise in the wide adoption of this strategy in org. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroWQSR7iPhhrVg90H21EOLACvtfcHk0li5hRcj6G863Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhs7zN&md5=4d9a66e0d674a7935038f9f45a07cefb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303587%26sid%3Dliteratum%253Aachs%26aulast%3DMorandi%26aufirst%3DB.%26aulast%3DWickens%26aufirst%3DZ.%2BK.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DRegioselective%2520Wacker%2520oxidation%2520of%2520internal%2520alkenes%253A%2520rapid%2520access%2520to%2520functionalized%2520ketones%2520facilitated%2520by%2520cross-metathesis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D9751%26epage%3D9754%26doi%3D10.1002%2Fanie.201303587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span> <span> </span><span class="NLM_article-title">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors as potential antidiabetic agents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">5468</span>– <span class="NLM_lpage">5479</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1016%2Fj.bmc.2011.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21868239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKhsr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=5468-5479&author=M.+J.+Kimauthor=S.+H.+Leeauthor=S.+O.+Parkauthor=H.+Kangauthor=J.+S.+Leeauthor=K.+N.+Leeauthor=M.+E.+Jungauthor=J.+Kimauthor=J.+Lee&title=Novel+macrocyclic+C-aryl+glucoside+SGLT2+inhibitors+as+potential+antidiabetic+agents&doi=10.1016%2Fj.bmc.2011.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors as potential antidiabetic agents</span></div><div class="casAuthors">Kim, Min Ju; Lee, Suk Ho; Park, So Ok; Kang, Hyunku; Lee, Jun Sung; Lee, Ki Nam; Jung, Myung Eun; Kim, Jeongmin; Lee, Jinhwa</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5468-5479</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Novel macrocyclic C-aryl glucoside SGLT2 inhibitors were designed and synthesized.  Two different synthetic routes of macrocyclization were adopted to prep. novel ansa SGLT2 inhibitors.  Among the compds. tested, [1,7]dioxacyclopentadecine macrocycles possessing methylthiophenyl at the distal ring 40 or ethoxyphenyl at the distal ring 23 showed the best in vitro inhibitory activity in this series to date (40, IC50 = 0.778 nM and 23, IC50 = 0.899 nM) against hSGLT2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2MFX8lSt4bVg90H21EOLACvtfcHk0li5hRcj6G863Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKhsr7I&md5=28491049fd76513d3fac62d8215448fc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BO.%26aulast%3DKang%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DLee%26aufirst%3DK.%2BN.%26aulast%3DJung%26aufirst%3DM.%2BE.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DNovel%2520macrocyclic%2520C-aryl%2520glucoside%2520SGLT2%2520inhibitors%2520as%2520potential%2520antidiabetic%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D5468%26epage%3D5479%26doi%3D10.1016%2Fj.bmc.2011.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Fluorescent probes to characterise FK506-binding proteins</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1402</span>– <span class="NLM_lpage">1410</span>, <span class="refDoi"> DOI: 10.1002/cbic.200800806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1002%2Fcbic.200800806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=19418507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtlOlsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=1402-1410&author=C.+Kozanyauthor=A.+Marzauthor=C.+Kressauthor=F.+Hausch&title=Fluorescent+probes+to+characterise+FK506-binding+proteins&doi=10.1002%2Fcbic.200800806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent Probes to Characterise FK506-Binding Proteins</span></div><div class="casAuthors">Kozany, Christian; Maerz, Andreas; Kress, Christoph; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1402-1410</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Talented all-rounders: Fluorescence polarisation assays were developed for members of the FK506-binding protein family by using fluorescent rapamycin analogs (demonstrated in the figure).  These tracers retain medium to high affinity to all tested proteins (FKBP12, -12.6, -13, -25, -51, -52).  They can be used for active-site titrns., competition assays with unlabeled ligands and enable a robust, miniaturized assay adequate for high-throughput screening.  FK506-binding proteins (FKBPs) convey the immunosuppressive action of FK506 and rapamycin and mediate the neuroprotective properties of these compds., and participate in the regulation of calcium channels.  In addn., the larger homologues FKBP51 and FKBP52 act as cochaperones for Hsp90 and regulate the transactivational activity of steroid hormone receptors.  To further characterize these FKBPs, we have synthesized fluorescein-coupled rapamycin analogs.  In fluorescence polarization assays one of these compds. retained high affinity to all tested proteins (Kd: 0.1-20 nM) and could be used for active-site titrns.  To adapt the fluorescence polarization assay for high-throughput purposes, a simplified rapamycin deriv. was synthesized and labeled with fluorescein.  This probe showed moderate affinity for the FK1 domains of FKBP51 (177 nM) and FKBP52 (469 nM) and allowed a highly robust, optimized, miniaturized assay (Z'>0.7) sufficient for high-throughput screening of large compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQIvLuUjAKDbVg90H21EOLACvtfcHk0li5hRcj6G863Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtlOlsro%253D&md5=57f00a9d8116c9bc8007f34c421c7dee</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200800806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200800806%26sid%3Dliteratum%253Aachs%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DMarz%26aufirst%3DA.%26aulast%3DKress%26aufirst%3DC.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DFluorescent%2520probes%2520to%2520characterise%2520FK506-binding%2520proteins%26jtitle%3DChemBioChem%26date%3D2009%26volume%3D10%26spage%3D1402%26epage%3D1410%26doi%3D10.1002%2Fcbic.200800806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pomplun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klages, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unal, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preiser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Chemogenomic profiling of human and microbial FK506-binding proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3660</span>– <span class="NLM_lpage">3673</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00137</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00137" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3660-3673&author=S.+Pomplunauthor=C.+Sippelauthor=A.+Hahleauthor=D.+Tayauthor=K.+Shimaauthor=A.+Klagesauthor=C.+M.+Unalauthor=B.+Riessauthor=H.+T.+Tohauthor=G.+Hansenauthor=H.+S.+Yoonauthor=A.+Bracherauthor=P.+Preiserauthor=J.+Ruppauthor=M.+Steinertauthor=F.+Hausch&title=Chemogenomic+profiling+of+human+and+microbial+FK506-binding+proteins&doi=10.1021%2Facs.jmedchem.8b00137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Chemogenomic Profiling of Human and Microbial FK506-Binding Proteins</span></div><div class="casAuthors">Pomplun, Sebastian; Sippel, Claudia; Haehle, Andreas; Tay, Donald; Shima, Kensuke; Klages, Alina; Uenal, Can Murat; Riess, Benedikt; Toh, Hui Ting; Hansen, Guido; Yoon, Ho Sup; Bracher, Andreas; Preiser, Peter; Rupp, Jan; Steinert, Michael; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3660-3673</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">FK506-binding proteins (FKBPs) are evolutionarily conserved proteins that display peptidyl-prolyl isomerase activities and act as coreceptors for immunosuppressants.  Microbial macrophage-infectivity-potentiator (Mip)-type FKBPs can enhance infectivity.  However, developing druglike ligands for FKBPs or Mips has proven difficult, and many FKBPs and Mips still lack biol. useful ligands.  To explore the scope and potential of C5-substituted [4.3.1]-aza-bicyclic sulfonamides as a broadly applicable class of FKBP inhibitors, we developed a new synthesis method for the bicyclic core scaffold and used it to prep. an FKBP- and Mip-focused library.  This allowed us to perform a systematic structure-activity-relationship anal. across key human FKBPs and microbial Mips, yielding highly improved inhibitors for all the FKBPs studied.  A cocrystal structure confirmed the mol.-binding mode of the core structure and explained the affinity gained as a result of the preferred substituents.  The best FKBP and Mip ligands showed promising antimalarial, antileginonellal, and antichlamydial properties in cellular models of infectivity, suggesting that substituted [4.3.1]-aza-bicyclic sulfonamides could be a novel class of anti-infectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_KYmjlu3wrrVg90H21EOLACvtfcHk0lhNnUKTExOE-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVWru7o%253D&md5=1961bd1a6f7aa868b5dd58319600b840</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00137%26sid%3Dliteratum%253Aachs%26aulast%3DPomplun%26aufirst%3DS.%26aulast%3DSippel%26aufirst%3DC.%26aulast%3DHahle%26aufirst%3DA.%26aulast%3DTay%26aufirst%3DD.%26aulast%3DShima%26aufirst%3DK.%26aulast%3DKlages%26aufirst%3DA.%26aulast%3DUnal%26aufirst%3DC.%2BM.%26aulast%3DRiess%26aufirst%3DB.%26aulast%3DToh%26aufirst%3DH.%2BT.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DPreiser%26aufirst%3DP.%26aulast%3DRupp%26aufirst%3DJ.%26aulast%3DSteinert%26aufirst%3DM.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DChemogenomic%2520profiling%2520of%2520human%2520and%2520microbial%2520FK506-binding%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3660%26epage%3D3673%26doi%3D10.1021%2Facs.jmedchem.8b00137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozany, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thost, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausch, F.</span></span> <span> </span><span class="NLM_article-title">Structural characterization of the PPIase domain of FKBP51, a cochaperone of human Hsp90</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1107/S0907444911013862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS0907444911013862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=21636895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFyrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=549-559&author=A.+Bracherauthor=C.+Kozanyauthor=A.+K.+Thostauthor=F.+Hausch&title=Structural+characterization+of+the+PPIase+domain+of+FKBP51%2C+a+cochaperone+of+human+Hsp90&doi=10.1107%2FS0907444911013862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of the PPIase domain of FKBP51, a cochaperone of human Hsp90</span></div><div class="casAuthors">Bracher, Andreas; Kozany, Christian; Thost, Ann Katrin; Hausch, Felix</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">549-559</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Steroid hormone receptors are key components of mammalian stress and sex hormone systems.  Many of them rely on the Hsp90 chaperone system for full function and are further fine-tuned by Hsp90-assocd. peptidyl-prolyl isomerases such as FK506-binding proteins 51 and 52.  FK506-binding protein 51 (FKBP51) has been shown to reduce glucocorticoid receptor signaling and has been genetically assocd. with human stress resilience and with numerous psychiatric disorders.  The peptidyl-prolyl isomerase domain of FKBP51 contains a high-affinity binding site for the natural products FK506 and rapamycin and has further been shown to convey most of the inhibitory activity on the glucocorticoid receptor.  FKBP51 has therefore become a prime new target for the treatment of stress-related affective disorders that could be amenable to structure-based drug design.  Here, a series of high-resoln. structures of the peptidyl-prolyl isomerase domain of FKBP51 as well as a cocrystal structure with the prototypic ligand FK506 are described.  These structures provide a detailed picture of the drug-binding domain of FKBP51 and the mol. binding mode of its ligand as a starting point for the rational design of improved inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2WGAZjThWGLVg90H21EOLACvtfcHk0lhNnUKTExOE-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFyrsLg%253D&md5=3ec5288a9394ab264678ace99e0fd5f7</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444911013862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911013862%26sid%3Dliteratum%253Aachs%26aulast%3DBracher%26aufirst%3DA.%26aulast%3DKozany%26aufirst%3DC.%26aulast%3DThost%26aufirst%3DA.%2BK.%26aulast%3DHausch%26aufirst%3DF.%26atitle%3DStructural%2520characterization%2520of%2520the%2520PPIase%2520domain%2520of%2520FKBP51%252C%2520a%2520cochaperone%2520of%2520human%2520Hsp90%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D549%26epage%3D559%26doi%3D10.1107%2FS0907444911013862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babine, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S. L.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of protein-ligand complexes: applications to drug design</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1359</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1021/cr960370z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr960370z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADyaK2sXksFSisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=1359-1472&author=R.+E.+Babineauthor=S.+L.+Bender&title=Molecular+recognition+of+protein-ligand+complexes%3A+applications+to+drug+design&doi=10.1021%2Fcr960370z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein-Ligand Complexes: Applications to Drug Design</span></div><div class="casAuthors">Babine, Robert E.; Bender, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1359-1472</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 538 refs.  This article provides a comprehensive discussion of the interactions of small mol. ligands with protein receptors.  It covers the literature of the structures of complexes between small mol. ligands and several enzymes which are relevant as drug targets.  Protein targets which are not enzymes are beyond the scope of this review.  It covers about 20 yr of the literature ending in Sept. 1996; however, most of the relevant work has been published within the last five years.  Whenever possible the relevant Brookhaven Protein Data Bank (PDB) entry has been given.  A major goal of this review is to enhance the qual. understanding of protein-ligand complexes.  A major portion of this review provides an in-depth discussion of protease inhibitors.  Four different mechanistic classes of enzymes that catalyze the same reaction, selective amide hydrolysis within a protein or polypeptide framework, are discussed.  Successful strategies for inhibiting these enzymes vary from class to class and reflect the mechanistic and structural differences of the enzymes.  These enzymes have evolved to recognize proteins and polypeptides.  While inhibition strategies based upon the preferred cleavage sites are now straightforward, they result in inhibitors which are peptide-like in structure.  Peptides and peptidic compds. are usually unsuitable as drug candidates.  This has resulted in the need for compds. which act as peptide mimetics.  Many of the structures described will illustrate the structural basis for peptide mimicry.  There have been major research efforts to develop drug candidates as inhibitors of two different aspartic protease enzymes, renin and HIV-1 protease.  In the search for renin inhibitors the structure of the target protein was not known during most of this research.  As a result, traditional medicinal chem. approaches were used in an attempt to prep. drug candidates.  To date there have been no renin inhibitors which are successful drug candidates.  In contrast, during the search for inhibitors of HIV-1 protease, the structure of the protein was known and many groups have used an iterative structure-based design process.  This has resulted in many structurally distinct classes of inhibitors.  Several successful drug candidates have emerged from these studies and the use of HIV-1 protease structure in drug design represents one of the more impressive success stories in this emerging field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM7ZsftkTf4rVg90H21EOLACvtfcHk0ljoHtKYC94CqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXksFSisLk%253D&md5=b533a1fbdec87724d3ec714f434afd07</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcr960370z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr960370z%26sid%3Dliteratum%253Aachs%26aulast%3DBabine%26aufirst%3DR.%2BE.%26aulast%3DBender%26aufirst%3DS.%2BL.%26atitle%3DMolecular%2520recognition%2520of%2520protein-ligand%2520complexes%253A%2520applications%2520to%2520drug%2520design%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D1359%26epage%3D1472%26doi%3D10.1021%2Fcr960370z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellmig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kastner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malecki, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puhringer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rower, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uhlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">The macromolecular crystallography beamlines at BESSY II of the Helmholtz-Zentrum Berlin: Current status and perspectives</span>. <i>Eur. Phys. J. Plus</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">141</span>, <span class="refDoi"> DOI: 10.1140/epjp/i2015-15141-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1140%2Fepjp%2Fi2015-15141-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2015&pages=141&author=U.+Muellerauthor=R.+Forsterauthor=M.+Hellmigauthor=F.+U.+Huschmannauthor=A.+Kastnerauthor=P.+Maleckiauthor=S.+Puhringerauthor=M.+Rowerauthor=K.+Spartaauthor=M.+Steffienauthor=M.+Uhleinauthor=P.+Wilkauthor=M.+S.+Weiss&title=The+macromolecular+crystallography+beamlines+at+BESSY+II+of+the+Helmholtz-Zentrum+Berlin%3A+Current+status+and+perspectives&doi=10.1140%2Fepjp%2Fi2015-15141-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1140%2Fepjp%2Fi2015-15141-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1140%252Fepjp%252Fi2015-15141-2%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DForster%26aufirst%3DR.%26aulast%3DHellmig%26aufirst%3DM.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DKastner%26aufirst%3DA.%26aulast%3DMalecki%26aufirst%3DP.%26aulast%3DPuhringer%26aufirst%3DS.%26aulast%3DRower%26aufirst%3DM.%26aulast%3DSparta%26aufirst%3DK.%26aulast%3DSteffien%26aufirst%3DM.%26aulast%3DUhlein%26aufirst%3DM.%26aulast%3DWilk%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DThe%2520macromolecular%2520crystallography%2520beamlines%2520at%2520BESSY%2520II%2520of%2520the%2520Helmholtz-Zentrum%2520Berlin%253A%2520Current%2520status%2520and%2520perspectives%26jtitle%3DEur.%2520Phys.%2520J.%2520Plus%26date%3D2015%26volume%3D130%26spage%3D141%26doi%3D10.1140%2Fepjp%2Fi2015-15141-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Briese, C.</span></span> <span> </span><span class="NLM_article-title">Optimal fine phi-slicing for single-photon-counting pixel detectors</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">56</span>, <span class="refDoi"> DOI: 10.1107/S0907444911049833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS0907444911049833" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=22194332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=42-56&author=M.+Muellerauthor=M.+Wangauthor=C.+Schulze-Briese&title=Optimal+fine+phi-slicing+for+single-photon-counting+pixel+detectors&doi=10.1107%2FS0907444911049833"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Optimal fine .vphi.-slicing for single-photon-counting pixel detectors</span></div><div class="casAuthors">Mueller, Marcus; Wang, Meitian; Schulze-Briese, Clemens</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">42-56</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Fine .vphi.-slicing substantially improves scaling statistics and anomalous signal for diffraction data collection with hybrid pixel detectors.  The data-collection parameters used in a macromol. diffraction expt. have a strong impact on data quality.  A careful choice of parameters leads to better data and can make the difference between success and failure in phasing attempts, and will also result in a more accurate at. model.  The selection of parameters has to account for the application of the data in various phasing methods or high-resoln. refinement.  Furthermore, exptl. factors such as crystal characteristics, available expt. time and the properties of the X-ray source and detector have to be considered.  For many years, CCD detectors have been the prevalent type of detectors used in macromol. crystallog.  Recently, hybrid pixel X-ray detectors that operate in single-photon-counting mode have become available.  These detectors have fundamentally different characteristics compared with CCD detectors and different data-collection strategies should be applied.  Fine .vphi.-slicing is a strategy that is particularly well suited to hybrid pixel detectors because of the fast readout time and the absence of readout noise.  A large no. of data sets were systematically collected from crystals of four different proteins in order to investigate the benefit of fine .vphi.-slicing on data quality with a noise-free detector.  The results show that fine .vphi.-slicing can substantially improve scaling statistics and anomalous signal provided that the rotation angle is comparable to half the crystal mosaicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGjNTwtRdWJrVg90H21EOLACvtfcHk0ljoHtKYC94CqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E&md5=03c8d02288ee79a7f41e187df10d872d</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444911049833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911049833%26sid%3Dliteratum%253Aachs%26aulast%3DMueller%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DSchulze-Briese%26aufirst%3DC.%26atitle%3DOptimal%2520fine%2520phi-slicing%2520for%2520single-photon-counting%2520pixel%2520detectors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D42%26epage%3D56%26doi%3D10.1107%2FS0907444911049833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cianci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompidor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karpics, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roessle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cipriani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiedler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span> <span> </span><span class="NLM_article-title">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high-and low-energy phasing with variable beam focusing</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1107/S1600577516016465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=10.1107%2FS1600577516016465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=28009574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=323-332&author=M.+Cianciauthor=G.+Bourenkovauthor=G.+Pompidorauthor=I.+Karpicsauthor=J.+Kallioauthor=I.+Bentoauthor=M.+Roessleauthor=F.+Ciprianiauthor=S.+Fiedlerauthor=T.+R.+Schneider&title=P13%2C+the+EMBL+macromolecular+crystallography+beamline+at+the+low-emittance+PETRA+III+ring+for+high-and+low-energy+phasing+with+variable+beam+focusing&doi=10.1107%2FS1600577516016465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">P13, the EMBL macromolecular crystallography beamline at the low-emittance PETRA III ring for high- and low-energy phasing with variable beam focusing</span></div><div class="casAuthors">Cianci, Michele; Bourenkov, Gleb; Pompidor, Guillaume; Karpics, Ivars; Kallio, Johanna; Bento, Isabel; Roessle, Manfred; Cipriani, Florent; Fiedler, Stefan; Schneider, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. P13 beamline is part of the European Mol. Biol. Lab. Integrated Facility for Structural Biol. at PETRA III (DESY, Hamburg, Germany) and has been in user operation since mid-2013.  P13 is tunable across the energy range from 4 to 17.5 keV to support crystallog. data acquisition exploiting a wide range of elemental absorption edges for exptl. phase detn.  An adaptive Kirkpatrick-Baez focusing system provides an X-ray beam with a high photon flux and tunable focus size to adapt to diverse exptl. situations.  Data collections at energies as low as 4 keV (λ = 3.1 Å) are possible due to a beamline design minimizing background and maximizing photon flux particularly at low energy (up to 1011 photons s-1 at 4 keV), a custom calibration of the PILATUS 6M-F detector for use at low energies, and the availability of a helium path.  At high energies, the high photon flux (5.4 × 1011 photons s-1 at 17.5 keV) combined with a large area detector mounted on a 2θ arm allows data collection to sub-at. resoln. (0.55 Å).  A peak flux of about 8.0 × 1012 photons s-1 is reached at 11 keV.  Automated sample mounting is available by means of the robotic sample changer 'MARVIN' with a dewar capacity of 160 samples.  In close proximity to the beamline, labs. have been set up for sample prepn. and characterization; a lab. specifically equipped for on-site heavy atom derivatization with a library of more than 150 compds. is available to beamline users.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYDoilsi_YbVg90H21EOLACvtfcHk0lgpSmldzvWUJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1ahsA%253D%253D&md5=807e7e497aa2b5f524a4366caa645ee6</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS1600577516016465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577516016465%26sid%3Dliteratum%253Aachs%26aulast%3DCianci%26aufirst%3DM.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DPompidor%26aufirst%3DG.%26aulast%3DKarpics%26aufirst%3DI.%26aulast%3DKallio%26aufirst%3DJ.%26aulast%3DBento%26aufirst%3DI.%26aulast%3DRoessle%26aufirst%3DM.%26aulast%3DCipriani%26aufirst%3DF.%26aulast%3DFiedler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26atitle%3DP13%252C%2520the%2520EMBL%2520macromolecular%2520crystallography%2520beamline%2520at%2520the%2520low-emittance%2520PETRA%2520III%2520ring%2520for%2520high-and%2520low-energy%2520phasing%2520with%2520variable%2520beam%2520focusing%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2017%26volume%3D24%26spage%3D323%26epage%3D332%26doi%3D10.1107%2FS1600577516016465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TXX','PDB','6TXX'); return false;">PDB: 6TXX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW6','PDB','4TW6'); return false;">PDB: 4TW6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TW7','PDB','4TW7'); return false;">PDB: 4TW7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIT','PDB','5DIT'); return false;">PDB: 5DIT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIU','PDB','5DIU'); return false;">PDB: 5DIU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DIV','PDB','5DIV'); return false;">PDB: 5DIV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAF','PDB','6SAF'); return false;">PDB: 6SAF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX','PDB','7AWX'); return false;">PDB: 7AWX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y','PDB','7B9Y'); return false;">PDB: 7B9Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z','PDB','7B9Z'); return false;">PDB: 7B9Z</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6W','PDB','7A6W'); return false;">PDB: 7A6W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6X','PDB','7A6X'); return false;">PDB: 7A6X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BA0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BA0','PDB','7BA0'); return false;">PDB: 7BA0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i93"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02195">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14531"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02195?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02195</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Further experimental details: crystallographic data collection and refinement statistics; X-ray structures of <b>56</b>, <b>33-(</b><i>Z</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, and <b>63</b> in complex with FKBP51FK1; superposition of X-ray structures with SAFit2 and linear SAFit-like ligands; ITC traces of SAFit1 and <b>64a</b>; additional synthetic representations and explanations; analytical data of intermediate compounds; additional information and binding curves from FP assay; and NMR spectra and HPLC traces of final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_001.pdf">jm0c02195_si_001.pdf (15.87 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02195/suppl_file/jm0c02195_si_002.csv">jm0c02195_si_002.csv (4.13 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the X-ray structures of compounds <b>33-(</b><i>Z</i><b>)</b>, <b>35-(</b><i>E</i><b>)</b>, <b>55</b>, <b>56</b>, <b>63</b>, and <b>64a</b> in complex with FKBP51 (14-140)-A19T (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6W">7A6W</a> (<b>33-(</b><i>Z</i><b>)</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Z">7B9Z</a> (<b>35-(</b><i>E</i><b>)</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AWX">7AWX</a> (<b>55</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7A6X">7A6X</a> (<b>56</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7BA0">7BA0</a> (<b>63</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B9Y">7B9Y</a> (<b>64a</b>)) are available from the RCSB Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02195&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02195%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-6%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02195" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799e0249e4e3c5e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
