
<html lang="en"     class="pb-page"  data-request-id="845f5ab9-48d4-40f9-840c-f98ad20fafb2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00161;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure–Activity Relationship of SPOP Inhibitors against Kidney Cancer" /></meta><meta name="dc.Creator" content="Ze  Dong" /></meta><meta name="dc.Creator" content="Zhen  Wang" /></meta><meta name="dc.Creator" content="Zhong-Qiang  Guo" /></meta><meta name="dc.Creator" content="Shouzhe  Gong" /></meta><meta name="dc.Creator" content="Tao  Zhang" /></meta><meta name="dc.Creator" content="Jiang  Liu" /></meta><meta name="dc.Creator" content="Cheng  Luo" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Cai-Guang  Yang" /></meta><meta name="dc.Description" content="Speckle-type POZ protein (SPOP) is overexpressed in the nucleus and misallocated in the cytoplasm in almost all the clear-cell renal cell carcinomas (ccRCCs), which leads to kidney tumorigenesis. P..." /></meta><meta name="Description" content="Speckle-type POZ protein (SPOP) is overexpressed in the nucleus and misallocated in the cytoplasm in almost all the clear-cell renal cell carcinomas (ccRCCs), which leads to kidney tumorigenesis. P..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 16, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00161" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00161" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00161" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00161" /></link>
        
    
    

<title>Structure–Activity Relationship of SPOP Inhibitors against Kidney Cancer | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00161" /></meta><meta property="og:title" content="Structure–Activity Relationship of SPOP Inhibitors against Kidney Cancer" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0018.jpeg" /></meta><meta property="og:description" content="Speckle-type POZ protein (SPOP) is overexpressed in the nucleus and misallocated in the cytoplasm in almost all the clear-cell renal cell carcinomas (ccRCCs), which leads to kidney tumorigenesis. Previously, we elucidated that the oncogenic SPOP-signaling pathway in ccRCC could be suppressed by 6b that inhibits SPOP-mediated protein interactions. Herein, we have established a structure–activity relationship for 6b analogues as SPOP inhibitors. Compound 6lc suppresses the viability and inhibits the colony formation of ccRCC cell lines driven by cytoplasmic SPOP, superior to 6b. Compound 6lc binds to the SPOP protein in vitro and disrupts SPOP binding to phosphatase-and-tensin homologue (PTEN) in HEK293T cells, which causes the observable phenomena: a decline in the ubiquitination of PTEN, elevated levels of both PTEN and dual-specificity phosphatase 7, and decreased levels of phosphorylated AKT and ERK when ccRCC cell lines are exposed to 6lc in a dose–response manner. Taken together, compound 6lc is a potent candidate against kidney tumorigenesis." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00161"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00161">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00161&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00161&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00161&amp;href=/doi/10.1021/acs.jmedchem.0c00161" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4849-4866</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00117" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00202" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.0c02084"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure–Activity Relationship of SPOP Inhibitors against Kidney Cancer</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ze Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ze Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ze++Dong">Ze Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhen Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhen Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, Nanjing 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhen++Wang">Zhen Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhong-Qiang Guo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhong-Qiang Guo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Urology, Zhongnan Hospital of Wuhan University, Hubei 430071, China</div><div class="loa-info-affiliations-info">CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhong-Qiang++Guo">Zhong-Qiang Guo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shouzhe Gong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shouzhe Gong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shouzhe++Gong">Shouzhe Gong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tao Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tao Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tao++Zhang">Tao Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">CAS Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing 100101, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiang++Liu">Jiang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheng Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheng Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, UCAS, Hangzhou 310024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Luo">Cheng Luo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3864-8382" title="Orcid link">http://orcid.org/0000-0003-3864-8382</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hualiang Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hualiang Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, UCAS, Hangzhou 310024, China</div><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, ShanghaiTec University, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Cai-Guang Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cai-Guang Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, UCAS, Hangzhou 310024, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#760f171811151136051f1b1b581715581518"><span class="__cf_email__" data-cfemail="a6dfc7c8c1c5c1e6d5cfcbcb88c7c588c5c8">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cai-Guang++Yang">Cai-Guang Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5778-4422" title="Orcid link">http://orcid.org/0000-0001-5778-4422</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00161&amp;href=/doi/10.1021%2Facs.jmedchem.0c00161" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4849–4866</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 16, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 January 2020</li><li><span class="item_label"><b>Published</b> online</span>16 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00161" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00161</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4849%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZe%2BDong%252C%2BZhen%2BWang%252C%2BZhong-Qiang%2BGuo%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c00161%26title%3DStructure%25E2%2580%2593Activity%2BRelationship%2Bof%2BSPOP%2BInhibitors%2Bagainst%2BKidney%2BCancer%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4866%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00161"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1635</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00161" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure–Activity Relationship of SPOP Inhibitors against Kidney Cancer&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ze&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Zhen&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Zhong-Qiang&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Shouzhe&quot;,&quot;last_name&quot;:&quot;Gong&quot;},{&quot;first_name&quot;:&quot;Tao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Jiang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Cai-Guang&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4849-4866&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00161&quot;},&quot;abstract&quot;:&quot;Speckle-type POZ protein (SPOP) is overexpressed in the nucleus and misallocated in the cytoplasm in almost all the clear-cell renal cell carcinomas (ccRCCs), which leads to kidney tumorigenesis. Previously, we elucidated that the oncogenic SPOP-signaling pathway in ccRCC could be suppressed by 6b that inhibits SPOP-mediated protein interactions. Herein, we have established a structure–activity relationship for 6b analogues as SPOP inhibitors. Compound 6lc suppresses the viability and inhibits the colony formation of ccRCC cell lines driven by cytoplasmic SPOP, superior to 6b. Compound 6lc binds to the SPOP protein in vitro and disrupts SPOP binding to phosphatase-and-tensin homologue (PTEN) in HEK293T cells, which causes the observable phenomena: a decline in the ubiquitination of PTEN, elevated levels of both PTEN and dual-specificity phosphatase 7, and decreased levels of phosphorylated AKT and ERK when ccRCC cell lines are exposed to 6lc in a dose–response manner. Taken together, compound 6lc is a potent &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00161&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00161" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00161&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00161" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00161&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00161" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00161&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00161&amp;href=/doi/10.1021/acs.jmedchem.0c00161" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00161" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00161" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00161%26sid%3Dliteratum%253Aachs%26pmid%3D32297747%26genre%3Darticle%26aulast%3DDong%26date%3D2020%26atitle%3DStructure%25E2%2580%2593Activity%2BRelationship%2Bof%2BSPOP%2BInhibitors%2Bagainst%2BKidney%2BCancer%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4849%26epage%3D4866%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Speckle-type POZ protein (SPOP) is overexpressed in the nucleus and misallocated in the cytoplasm in almost all the clear-cell renal cell carcinomas (ccRCCs), which leads to kidney tumorigenesis. Previously, we elucidated that the oncogenic SPOP-signaling pathway in ccRCC could be suppressed by <b>6b</b> that inhibits SPOP-mediated protein interactions. Herein, we have established a structure–activity relationship for <b>6b</b> analogues as SPOP inhibitors. Compound <b>6lc</b> suppresses the viability and inhibits the colony formation of ccRCC cell lines driven by cytoplasmic SPOP, superior to <b>6b</b>. Compound <b>6lc</b> binds to the SPOP protein in vitro and disrupts SPOP binding to phosphatase-and-tensin homologue (PTEN) in HEK293T cells, which causes the observable phenomena: a decline in the ubiquitination of PTEN, elevated levels of both PTEN and dual-specificity phosphatase 7, and decreased levels of phosphorylated AKT and ERK when ccRCC cell lines are exposed to <b>6lc</b> in a dose–response manner. Taken together, compound <b>6lc</b> is a potent candidate against kidney tumorigenesis.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34780" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34780" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Kidney cancer is resistant to chemotherapy in clinic and usually metastasizes once the disease has already been diagnosed. Several drugs have been approved for the targeted therapy of kidney cancers.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Typically, these drugs are either kinase inhibitors for vascular endothelial growth factor (VEGF) and platelet-derived growth factor (PDGF) receptors or inhibitors for the mammalian target of rapamycin (mTOR). Significantly, the first checkpoint inhibitor nivolumab, a monoclonal antibody targeting PD-1, was approved in clinic for renal cell carcinoma (RCC) therapy.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Bevacizumab, another monoclonal antibody targeting the VEGF receptor, also shows good antitumor activity in patients with metastatic RCC.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Recently, the small-molecule inhibitor PT2399 was shown to successfully disrupt the HIF 2α-Arant interaction and to suppress the progression of clear-cell RCC (ccRCC). PT2385, the analogue of PT2399, has been approved for the clinical research.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Although these antitumor drugs and candidates are available for the treatment of ccRCC, the dual challenges of low response rate and drug resistance demand urgent exploration of untargeted targets and underexplored chemical spaces for the targeted therapy of ccRCC.</div><div class="NLM_p">The nuclear speckle-type POZ protein (SPOP) plays important roles within the ubiquitination machinery in cancers.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> SPOP contains an N-terminal MATH domain (SPOP<sup>MATH</sup>) for selectively binding to a substrate protein and a C-terminal BTB adaptor domain for E3 ubiquitin ligase recognition.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Remarkably, the E3 ligase adaptor SPOP is overexpressed in 85% of kidney cancers; accumulation of dislocated SPOP in the cytoplasm occurs in nearly 100% of the two predominant forms of kidney cancer, namely, primary and metastatic ccRCC.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The cytoplasmic SPOP serves as a regulatory hub in order to promote cancer through the ubiquitination and degradation of multiple cancer proteins that are involved in cellular proliferation and apoptosis.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The two tumor suppressors, phosphatase-and-tensin homologue (PTEN) and dual-specificity phosphatase 7 (DUSP7), are central to the mechanism in the oncogenic SPOP signaling. Binding of SPOP to PTEN and DUSP7 in the cytoplasm degrades these two proteins, which inactivates its downstream AKT/ERK signaling pathways and which eventually promotes ccRCC tumorigenesis. Indeed, the biological rationale is compelling because the inhibition of SPOP-mediated protein interactions and the stabilization of PTEN and DUSP7 proteins would be beneficial for the treatment of ccRCC.</div><div class="NLM_p">We have established that small-molecule targeting of the E3 ligase adaptor SPOP could be a specific approach against ccRCC.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Compounds <b>6a</b> and <b>6b</b> block the interaction of the SPOP ubiquitin ligase with its substrates (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), thus leading to the impaired proliferation of ccRCC cell lines that characteristically have cytoplasmic SPOP. Our discovery of compound <b>6b</b> convincingly demonstrates on a biochemical level in cells the therapeutic potential of blocking SPOP activity to stabilize PTEN and DUSP7 in order to suppress the AKT- and ERK-driven cancer-signaling pathways, which serves as a “proof-of-principle” in order to manipulate the “undruggable” E3 ligase in the ubiquitin proteasome system in ccRCC.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of our previously reported SPOP inhibitors. Compound <b>6c</b> is a negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In this study, we explored more chemical spaces and attempted to establish a structure–activity relationship (SAR) in the aspect of medicinal chemistry for SPOP inhibitors. Several analogues of <b>6b</b> are shown to suppress the proliferation of ccRCC cell lines typically with single-digital micromolar IC<sub>50</sub> values. Among them, compound <b>6lc</b> is active against kidney cancer cell lines through the inhibition of the SPOP-mediated ubiquitination-signaling pathway in ccRCC.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemical Synthesis</h3><div class="NLM_p">We performed structural optimization based on the chemical scaffold of the established SPOP inhibitors in several aspects using medicinal chemistry strategy,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> including (1) N-substituents, (2) various amide side chains, (3) the impact of methyl group position, (4) carboxylic acids and esters, and (5) conjugation shrinking (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of different sets of SPOP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">First, we investigated the effects of various N-substituents attached to cyclic amidine. <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> shows the synthetic route for compounds <b>6d–6z</b>. Condensation of 3-methylpyridin-2-amine <b>1</b> and diethyl malonate at elevated temperature provides 2-hydroxy-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one <b>2</b>. Simultaneous chlorination and formylation of <b>2</b> give the intermediate <b>3</b> under the Vilsmeier–Haack reaction. Various imines <b>4</b> are obtained in good yields by treating the intermediate <b>3</b> with corresponding amines in the presence of anhydrous Na<sub>2</sub>SO<sub>4</sub> in EtOH at room temperature (rt). Last, refluxing imines <b>4</b> with α-cyanoacetamide <b>5a</b> in CHCl<sub>3</sub> provides a series of target compounds <b>6d–6z</b> in high yields.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6d–6z</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 220 °C, diethyl malonate, neat; (b) POCl<sub>3</sub>, DMF; (c) R<sup>1</sup>NH<sub>2</sub>, Na<sub>2</sub>SO<sub>4</sub>, EtOH, rt, overnight; (d) <b>5a</b>, piperidine, CHCl<sub>3</sub>, reflux, overnight; (e) neat, rt, 3 days.</p></p></figure><div class="NLM_p">Next, we introduced various amide side chains into the main scaffold where the R<sup>1</sup> group is held constant as thiophenylethyl because the sulfur-containing heterocycle substituent might make the target compounds more “drug-like” and soluble. When α-cyanoacetamides <b>5b–5k</b> are employed as reagents in the one-step condensation with intermediate <b>4</b>, compounds <b>6la–6lj</b> are obtained in high yields (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6la–6lj</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) R<sup>2</sup>NH<sub>2</sub>, neat, rt, 3 days; (b) piperidine, CHCl<sub>3</sub>, reflux, overnight.</p></p></figure><div class="NLM_p">To investigate the impact of the methyl group attached to the planar tricycle (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), we synthesized 9-methyl-substituted compounds <b>6lk</b> and <b>6ll</b> without methylation in a similar route for the synthesis of compounds <b>6la–6lj</b>.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds with or without the methyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, we prepared a set of esters <b>7a–7j</b> and the corresponding carboxylic acids <b>8a–8j</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Ethyl esters <b>7</b> are synthesized in high yields through the reactions between compound <b>4</b> and ethyl α-cyanoacetate <b>5l</b>. Compounds <b>8a–8j</b> are obtained from alkaline hydrolysis of the corresponding ethyl esters <b>7a–7j</b>.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>7</b> and <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl α-cyanoacetate <b>5l</b>, piperidine, CHCl<sub>3</sub>, reflux, overnight; (b) LiOH, THF/EtOH/H<sub>2</sub>O, 45 °C, overnight.</p></p></figure><div class="NLM_p">Last, we synthesized several bicyclic compounds in order to investigate the effect of polycyclic conjugation on the inhibitory activity of SPOP inhibitors. The synthetic route for the preparation of compounds <b>13</b> and <b>14</b> is depicted in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> Compound <b>9</b> is reacted with diethyl malonate to give ethyl ester <b>10</b>. Alkylation of <b>10</b> with (2-bromoethyl)benzene affords N-alkylated lactam <b>11</b>. Alkaline hydrolysis of <b>11</b> followed by condensation of carboxylic acids and amines provides target compound carboxamides <b>13</b> and <b>14</b> in high yields.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>13</b> and <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperidine, diethyl malonate, neat, reflux, overnight; (b) PhCH<sub>2</sub>CH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF; (c) LiOH, THF/EtOH/H<sub>2</sub>O, 45 °C, 4 h; (d) R<sup>2</sup>NH<sub>2</sub>, HATU, DIEA, DMF.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Nano Differential Scanning Fluorimetry Screening for SPOP Binders</h3><div class="NLM_p">The nano differential scanning fluorimetry (nanoDSF) assay measures the thermal unfolding transitions (<i>T</i><sub>m</sub>) of an interested protein under native conditions, and no extra dye is required.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In order to evaluate whether our newly synthesized compounds bind to the SPOP protein in vitro, we performed a high-throughput screening for the <i>T</i><sub>m</sub> shift (Δ<i>T</i><sub>m</sub>) of SPOP<sup>MATH</sup> induced by compounds (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">S1</a>). Several compounds could stabilize the SPOP<sup>MATH</sup> protein with Δ<i>T</i><sub>m</sub> above 3.0 °C when the 10-fold compound was subjected to the nanoDSF assay, while compound <b>6c</b>, the negative control, had minimal effect on the thermal stability of SPOP<sup>MATH</sup>. To our delight, the Δ<i>T</i><sub>m</sub> could be further shifted up to 4.0 °C when the 20-fold compound <b>6b</b> was assayed. In addition to <b>6b</b>, compounds <b>6e</b>, <b>6g–i</b>, <b>6l</b>, <b>6lc</b>, <b>6ld</b>, <b>7f</b>, and <b>8c</b> as listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> stabilize SPOP<sup>MATH</sup> showing Δ<i>T</i><sub>m</sub> values over 3.5 °C, alongside compounds <b>6r</b>, <b>6lf</b>, <b>6lj</b>, <b>7c</b>, and <b>8b</b>, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">Table S1</a>. Because the SPOP binder <b>6b</b> has been shown to be active against kidney cancer through targeting the SPOP-mediated signaling pathway in ccRCC, the nanoDSF assay could be the first step of screening for small-molecule binders of SPOP that may have the potential to inhibit SPOP-mediated protein interactions in the further biological annotation.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>T</i><sub>m</sub> Shift of SPOP<sup>MATH</sup> in the Presence of Compounds<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">Δ<i>T</i><sub>m</sub> (°C)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">Δ<i>T</i><sub>m</sub> (°C)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">10×</th><th class="colsep0 rowsep0" align="center" char=".">20×</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">10×</th><th class="colsep0 rowsep0" align="center" char=".">20×</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DMSO</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">–1.7</td><td class="colsep0 rowsep0" align="char" char=".">–3.8</td><td class="colsep0 rowsep0" align="left"><b>6la</b></td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">2.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="left"><b>6lb</b></td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="left"><b>6lc</b></td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="left"><b>6ld</b></td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="left"><b>6li</b></td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="left"><b>7e</b></td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="left"><b>7f</b></td><td class="colsep0 rowsep0" align="char" char=".">4.9</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">5.3</td><td class="colsep0 rowsep0" align="left"><b>7j</b></td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="left"><b>8c</b></td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data were calculated using DMSO-treated SPOP<sup>MATH</sup> as the reference.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Data were obtained using the naive SPOP<sup>MATH</sup> sample as the reference.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Antiproliferation of SPOP Binders in ccRCC Cell Lines</h3><div class="NLM_p last">Next, we performed an 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay in order to evaluate the antiproliferative effect of those SPOP binders identified in the nanoDSF screening on kidney cancer cells. The two ccRCC cell lines A498 and OS-RC-2 were assayed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">Table S2</a>). The known SPOP inhibitor <b>6b</b> was assayed as the positive control,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> which inhibits the proliferation of A498 and OS-RC-2 cells up to 88 and 90% at 30 μM, respectively. Compound <b>6c</b>, the negative control, minimally attenuates the proliferation of the two ccRCC cell lines at high concentrations. Several compounds, for example, <b>6d–6i</b>, <b>6k–6l</b>, <b>6n</b>, <b>6lc</b>, and <b>6li</b>, exhibited comparable antiproliferative activities with compound <b>6b</b>. Of note, those compounds are SPOP binders too, which suggest that the nanoDSF is a validated assay for the initial screening of SPOP inhibitors.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Analysis of SARs</h3><div class="NLM_p">With these newly designed <b>6b</b> analogues in hand, we sought to analyze the SAR of SPOP inhibitors. First, we investigated the effects of the R<sup>1</sup> substitutes on the inhibition of cell proliferation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Introduction of methyl (<b>6d</b>), methoxyl (<b>6e</b>), fluoro (<b>6f</b>), chloro (<b>6g</b>), and bromo (<b>6h</b>) substitutes on the para-position of the phenethyl motif has negligible impacts on inhibitory activities, which display antiproliferative rates ranging between 88 and 97% against A498 and OS-RC-2 cells at 30 μM, comparable to compound <b>6b</b>. Interestingly, changing the position of the fluoro group from the para-position (<b>6f</b>) or the meta-position (<b>6i</b>) to the ortho-position (<b>6j</b>) is unfavorable to the cell inhibitory activities because the inhibitory rates of <b>6j</b> are decreased in both ccRCC cell lines. 2-(Diethylamino)ethyl (<b>6k</b>) bearing an acyclic apolar substitute exhibits good cellular activity, with inhibitory rates of between 86 and 89% against the proliferation of A498 and OS-RC-2 cells, respectively. Compared to the thiophenylethyl-substituted compound <b>6l</b>, 2-morpholinoethyl-substituted analogue <b>6m</b> exhibits declined inhibitory potency. 3-Phenylpropyl-substituted compound <b>6n</b> largely suppressed cell proliferation, while an introduction of an oxygen atom into the chain of compound <b>6n</b> causes 2-phenoxyethyl-substituted compound <b>6o</b> to dramatically lose activity. Compound <b>6p</b> bearing 3-morpholinopropyl exhibits declined antiproliferative activities against both A498 and OS-RC-2. Compound <b>6q</b> with the benzyl substituent exhibits declined activities, with 58 and 46% inhibitory rates against the proliferation of A498 and OS-RC-2 cells. A similar phenomenon is observed in compound <b>6r</b> that having pyridin-4-ylmethyl group. Compounds <b>6s</b> and <b>6z</b> bearing aromatic groups exhibit attenuated antiproliferative activities against the two ccRCC cell lines, with inhibitory rates of 12 and 17% in A498 cells and 23 and 38% in OS-RC-2 cells, respectively. Compounds (<b>6t</b>, <b>6u</b>, <b>6v</b>, and <b>6w</b>) with the cyclic or linear R<sup>1</sup>-substituted alkyl group also showed reduced inhibitory activity compared to <b>6b</b>. Last, the introduction of propargyl (<b>6x</b>) or the bulky indole (<b>6y</b>) proves unfavorable to antiproliferation. Taken together, the results indicate that the introduction of polar substituents into the nitrogen is generally unfavorable to overall inhibitory potency in ccRCC cell lines.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SARs of N-Substituents</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0012.gif" alt="" id="GRAPHIC-d7e1190-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="4f262127262d263b2620210f7c7f">[email protected]</a> μM (%)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2a4344424348435e4345446a191a">[email protected]</a> μM (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="left"><b>6o</b></td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="left"><b>6p</b></td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="left"><b>6q</b></td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">88</td><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="char" char=".">35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="left"><b>6s</b></td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">96</td><td class="colsep0 rowsep0" align="left"><b>6t</b></td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="left"><b>6u</b></td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6j</b></td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="left"><b>6v</b></td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6k</b></td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="left"><b>6w</b></td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="left"><b>6x</b></td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6m</b></td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="left"><b>6y</b></td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6n</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="left"><b>6z</b></td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">38</td></tr></tbody></table></div></div><div class="NLM_p">Next, we found that variations of amide side chains in SPOP inhibitors have significant effects on the inhibition of proliferation in ccRCC cell lines (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Changing the 3-morpholinopropyl (<b>6l</b>) to the 3-phenylpropyl (<b>6la</b>) substituent reduces antiproliferative activities, with moderate inhibitory ratios of 62 and 58% for compound <b>6la</b> on A498 and OS-RC-2 cells at 30 μM, respectively. In general, compounds <b>6lb–ld</b> and <b>6lf</b> bearing the short amide side chains largely retained their inhibitory activities against the proliferation of A498 cells. Surprisingly, compound <b>6le</b>, with the 4-fluorophenethyl substituent, exhibits a significantly declined activity against proliferation, with inhibitory ratios of 12 and 21% for A498 and OS-RC-2 cells at 30 μM. The benzyl-amide-substituted compound <b>6lg</b> also becomes less active. Of note, compounds <b>6lh</b> and <b>6li</b> with cyclic substituents largely maintain potency for inhibition, while the introduction of β-aminoalcohol (<b>6lj</b>) exerts only slight influence on antiproliferative activity.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SARs of Amide Side Chains</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0013.gif" alt="" id="GRAPHIC-d7e1450-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="3e575056575c574a5751507e0d0e">[email protected]</a> μM (%)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e9808781808b809d808687a9dad9">[email protected]</a> μM (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6la</b></td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="left"><b>6lf</b></td><td class="colsep0 rowsep0" align="char" char=".">87</td><td class="colsep0 rowsep0" align="char" char=".">75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6lb</b></td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="left"><b>6lg</b></td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6lc</b></td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="left"><b>6lh</b></td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="char" char=".">83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ld</b></td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="left"><b>6li</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6le</b></td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="left"><b>6lj</b></td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">81</td></tr></tbody></table></div></div><div class="NLM_p">Subsequently, we investigated the impact of the methyl substituent on the planar tricycle. We compared the antiproliferative activities of compounds without or with the methyl group on different positions (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>6lk</b> with the 9-methyl substituent and <b>6ll</b> without the methyl group showed attenuated activities against the proliferation compared with compound <b>6l</b> with the 10-methyl substituent, thus indicating that the position of the methyl substituent in the SPOP inhibitor is also important for the antiproliferation in ccRCC cell lines.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Impact of the Methyl Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0014.gif" alt="" id="GRAPHIC-d7e1583-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="573e393f3e353e233e3839176467">[email protected]</a> μM (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">nethyl position</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10-Me</td><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="char" char=".">90</td><td class="colsep0 rowsep0" align="char" char=".">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9-Me</td><td class="colsep0 rowsep0" align="left"><b>6lk</b></td><td class="colsep0 rowsep0" align="char" char=".">75</td><td class="colsep0 rowsep0" align="char" char=".">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">No-Me</td><td class="colsep0 rowsep0" align="left"><b>6ll</b></td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">56</td></tr></tbody></table></div></div><div class="NLM_p">As depicted in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, most esters show reduced activities against the proliferation of A498 and OS-RC-2 cells compared to the corresponding amides (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compound <b>7g</b> is the most active SPOP inhibitor among the 10 esters but only achieves less than 60% antiproliferation in the two ccRCC cell lines at 30 μM. In addition, no carboxylic acids (<b>8a–j</b>), hydrolyzed from the corresponding esters (<b>7a–j</b>) under alkaline condition, show satisfied activities against ccRCC cell lines. In general, the esters or carboxylic acids as SPOP inhibitors are much less active than the amide derivatives.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SARs of Esters and Carboxylic Acids</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0015.gif" alt="" id="GRAPHIC-d7e1667-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e48d8a8c8d868d908d8b8aa4d7d4">[email protected]</a> μM (%)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="."><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="8ee7e0e6e7ece7fae7e1e0cebdbe">[email protected]</a> μM (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">A498</th><th class="colsep0 rowsep0" align="center" char=".">OS-RC-2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="left"><b>8a</b></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="left"><b>8b</b></td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="left"><b>8c</b></td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char=".">48</td><td class="colsep0 rowsep0" align="left"><b>8d</b></td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7e</b></td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="left"><b>8e</b></td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7f</b></td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="left"><b>8f</b></td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7g</b></td><td class="colsep0 rowsep0" align="char" char=".">59</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="left"><b>8g</b></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7h</b></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="left"><b>8h</b></td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7i</b></td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="left"><b>8i</b></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7j</b></td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="left"><b>8j</b></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">33</td></tr></tbody></table></div></div><div class="NLM_p">Last, in order to diversify the chemical scaffold of the SPOP inhibitor, we shrank the tricyclic coplanar conjugation to the bicyclic system. We synthesized several bicyclic compounds with different substituents, including acids, esters, and amides (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Unfortunately, these bicyclic analogues usually display low antiproliferative activity in the two ccRCC cell lines, thus suggesting the important contribution of tricyclic coplanar conjugation on SPOP inhibitors.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SARs of Planar Conjugation</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0016.gif" alt="" id="GRAPHIC-d7e1866-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0017.gif" alt="" id="gr12" /></img><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Quantitation of Antiproliferative Activity of SPOP Inhibitors</h3><div class="NLM_p">Following the SAR analysis of SPOP inhibitors, we quantitatively determined the IC<sub>50</sub> values for those compounds displaying >75% antiproliferative activities in both ccRCC cell lines at a single dose of 30 μM (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). The IC<sub>50</sub> values of compound <b>6b</b> are 2.7 and 5.8 μM against A498 and OS-RC-2 cell lines, respectively, which are similar to our previously reported results. Compound <b>6c</b> is assayed as a negative control. In general, most tested compounds exhibited comparable activities to <b>6b</b> for the inhibition of ccRCC cell proliferation with single-digital micromolar IC<sub>50</sub> values, particularly those compounds containing apolar alkyl groups at the R<sup>1</sup>-position. Compounds <b>6d–6i</b> with phenethyl substitution, compound <b>6k</b> bearing acyclic 2-(diethylamino)ethyl side chain, and compound <b>6l</b> containing a heterocyclic substituent at the R<sup>1</sup> positions show comparable IC<sub>50</sub> values to inhibitor <b>6b</b>. Similarly, compound <b>6n</b> with 3-phenylpropyl displays comparable activity to <b>6b</b>, while <b>6w</b> bearing the propyl group shows a 2-fold drop in IC<sub>50</sub> value for A498 cells. In line with the activities observed in the single dosage MTT assay, the compounds bearing various amide side chains also display potent antiproliferation against the two ccRCC cell lines, including <b>6lc</b>, <b>6lf</b>, and <b>6li</b>. Compound <b>6ld</b> is much less active against the proliferation in OS-RC-2 cells with an unknown mechanism.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Antiproliferative Activity of Selected Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">A498</th><th class="colsep0 rowsep0" align="center">OS-RC-2</th><th class="colsep0 rowsep0" align="center">Compound</th><th class="colsep0 rowsep0" align="center" char="±">A498</th><th class="colsep0 rowsep0" align="center" char="±">OS-RC-2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">2.7 ± 1.0</td><td class="colsep0 rowsep0" align="left">5.8 ± 1.2</td><td class="colsep0 rowsep0" align="left"><b>6k</b></td><td class="colsep0 rowsep0" align="char" char="±">3.5 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">3.9 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">7.6 ± 2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="left">2.1 ± 0.6</td><td class="colsep0 rowsep0" align="left">5.7 ± 1.2</td><td class="colsep0 rowsep0" align="left"><b>6n</b></td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">5.1 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="left">2.8 ± 0.4</td><td class="colsep0 rowsep0" align="left">6.5 ± 1.1</td><td class="colsep0 rowsep0" align="left"><b>6w</b></td><td class="colsep0 rowsep0" align="char" char="±">6.4 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">5.2 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="left">3.1 ± 0.8</td><td class="colsep0 rowsep0" align="left">6.6 ± 1.0</td><td class="colsep0 rowsep0" align="left"><b>6lc</b></td><td class="colsep0 rowsep0" align="char" char="±">2.1 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">3.5 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="left">2.3 ± 0.3</td><td class="colsep0 rowsep0" align="left">4.0 ± 0.8</td><td class="colsep0 rowsep0" align="left"><b>6ld</b></td><td class="colsep0 rowsep0" align="char" char="±">5.0 ± 1.3</td><td class="colsep0 rowsep0" align="char" char="±">24.2 ± 2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="left">3.0 ± 0.5</td><td class="colsep0 rowsep0" align="left">5.0 ± 1.0</td><td class="colsep0 rowsep0" align="left"><b>6lf</b></td><td class="colsep0 rowsep0" align="char" char="±">2.8 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">4.2 ± 0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="left">2.4 ± 0.4</td><td class="colsep0 rowsep0" align="left">4.6 ± 0.6</td><td class="colsep0 rowsep0" align="left"><b>6li</b></td><td class="colsep0 rowsep0" align="char" char="±">6.3 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">5.9 ± 0.8</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Investigation of Antiproliferative Selectivity of <b>6lc</b></h3><div class="NLM_p">Although SPOP localizes in the nucleus and SPOP mutations are common in a variety of cancers, the ccRCC cell lines are unique in that overexpression of SPOP leads to ectopic localization in the cytoplasm. To this end, the SPOP inhibitor could selectively attenuate proliferation in ccRCC cell lines but not in cell lines in which SPOP is absent from the cytoplasm. As expected, <b>6lc</b> displays antiproliferation in the two ccRCC cell lines of A498 and OS-RC-2 with IC<sub>50</sub> of 2.1 and 3.5 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), while <b>6lc</b> minimally inhibits the growth of the nontumor kidney cell line HK-2 (human proximal tubule epithelial cell line). In our SAR exploration, compound <b>6lc</b> is not sharply different from <b>6h</b> and <b>6lf</b> and our previously reported <b>6b</b> in potency against the proliferation of ccRCC cell lines (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). These inhibitors also had similar thermal effects on the stabilization of the SPOP protein in vitro (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, the solubility of <b>6lc</b> is determined to be >90 μM in a cell culture medium in the presence of 0.2% dimethyl sulfoxide (DMSO), which is much more soluble than <b>6b</b> (30–45 μM), <b>6h</b> (5–10 μM), and <b>6lf</b> (2.5–5 μM). In addition, <b>6lc</b> significantly inhibited the colony formation for both A498 and OS-RC-2 cells, superior to <b>6b</b>, <b>6h</b>, and <b>6lf</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Taken these data together, we chose <b>6lc</b> as the representative inhibitor of SPOP-protein interaction for further investigations.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Antiproliferative effect of inhibitor <b>6lc</b>. (A) Quantitation of IC<sub>50</sub> of <b>6lc</b> against the proliferation of ccRCC cell lines and HK-2 cells. (B) Effect of SPOP inhibitors on the colony formation of A498 and OS-RC-2 cell lines. (C) Effect of <b>6lc</b> on the proliferation in HEK293T cells overexpressing SPOP. (D) Effect of <b>6lc</b> on cell proliferation in other cancer cell lines. **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001; unpaired Student’s <i>t</i>-test. Error bar, mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It has been previously shown that HEK293 cells do not express SPOP in the cytoplasm, but the overexpression of SPOP in HEK293 cells promotes tumorigenesis.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> As expected, the recapitulated ccRCC phenotype observed in the HEK293T cells of overexpressing SPOP could be suppressed by the treatment of the SPOP inhibitor <b>6lc</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), thus indicating that compound <b>6lc</b> is specific to the inhibition of the cytoplasmic SPOP. Last, we also determined the effect of <b>6lc</b> on proliferation in other cancer cell lines in which SPOP is absent from the cytoplasm. By over 5-folds, the IC<sub>50</sub> for A498 cell proliferation measured less than that for four other tested cell lines (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). Taken together, the SPOP inhibitor <b>6lc</b> is likely specific to suppress the proliferation of cells driven by cytoplasmic SPOP rather than those cell lines in which SPOP is absent from the cytoplasm.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Characterizations of the Interactions between SPOP and Inhibitors in Vitro</h3><div class="NLM_p">The <i>T</i><sub>m</sub> of SPOP<sup>MATH</sup> is shifted up to 3.5 °C in the presence of the 20-fold compound <b>6lc</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), suggesting that <b>6lc</b> binds to the substrate-binding MATH domain. We further performed the nanoDSF assay under native conditions on the SPOP<sup>23–337</sup> protein that contains both MATH and BTB domains. Compound <b>6lc</b> thermally stabilizes the SPOP<sup>23–337</sup> protein by elevating the <i>T</i><sub>m</sub> up to 2.1 °C even in an equal molar ratio (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Although the negative control compound <b>6c</b> exhibits less effect on the thermal stability of SPOP in nanoDSF measurement (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), it appears that DMSO destabilizes SPOP and that the compound returns the <i>T</i><sub>m</sub> back to its usual state in the absence of DMSO. A similar phenomenon is observed in the conventional DSF screening, in which a fluorescent dye is required, although compound <b>6lc</b> significantly stabilizes SPOP<sup>MATH</sup> by shifting the <i>T</i><sub>m</sub> to 4 °C (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Again, the negative control <b>6c</b> had minimal effect on the stability of SPOP<sup>MATH</sup> in the conventional DSF assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Consistent with our previous conclusion that the SPOP inhibitor <b>6b</b> does not belong to the pan assay interference compound (PAINS),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> neither the conventional DSF performed with a mixture of just the dye and compound <b>6lc</b> yields any changes in fluorescence, nor does the nanoDSF experiment conducted on compound <b>6lc</b> and unrelated proteins yield any changes in fluorescence, for example, BSA and ClpP caused no changes in thermal stability. Instead of a fluorescence-based DSF system, other biophysical methods, such as nuclear magnetic resonance (NMR) titration and surface plasmon resonance (SPR), should also be better options in order to ensure that observed thermal effects are unlikely due to the conjugated nature or the large concentration of compound <b>6lc</b>. Attenuation of signals in Carr–Purcell–Meiboom–Gill (CPMG) NMR titrations alongside positive saturation transfer difference (STD) signals is detected (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">S2A</a>), which indicates that both SPOP<sup>MATH</sup> and SPOP in full length interfere with the state of compound <b>6lc</b>. In addition, SPR measurement has established a moderate affinity for inhibitor <b>6lc</b> binding to the two SPOP proteins, which is around 30 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">S2B</a>). Last, a variety of SPOP inhibitors are observed to thermally stabilize SPOP in the conventional DSF assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G). Collectively, we performed different assays in order to confirm SPOP inhibitor binding to the SPOP protein.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Investigations on the binding of inhibitors to SPOP in vitro. (A) Effect of <b>6lc</b> on the thermal stability of SPOP<sup>23–337</sup> in nanoDSF. (B) Effect of <b>6c</b> on the thermal stability of SPOP<sup>MATH</sup> in nanoDSF. (C) Effect of <b>6lc</b> on the stability of SPOP<sup>MATH</sup> in the conventional DSF assay. (D) Effect of <b>6c</b> on the stability of SPOP<sup>MATH</sup> in the conventional DSF. (E) NMR measurement of <b>6lc</b> binding to SPOP<sup>MATH</sup>. The STD-NMR spectrum for 200 μM <b>6lc</b> is recorded in the presence of SPOP<sup>MATH</sup>. (F) Determination of the affinity of <b>6lc</b> binding to SPOP<sup>MATH</sup> in SPR measurement. Graphs of equilibrium RU responses vs compound concentrations are plotted. (G) Effect of SPOP inhibitors on the stability of SPOP<sup>MATH</sup> in the conventional DSF assay. For nanoDSF and conventional DSF assays, Δ<i>T</i><sub>m</sub> for the DMSO-treated SPOP protein were calculated using a naive SPOP sample as the reference, while Δ<i>T</i><sub>m</sub> data for the compound-treated SPOP were reported using DMSO-treated SPOP as the reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <b>6lc</b> Inhibits SPOP–PTEN Interaction</h3><div class="NLM_p">In order to verify the cellular interaction between <b>6lc</b> and SPOP, we performed a drug affinity responsive target stability (DARTS) assay.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> As expected, the SPOP protein becomes protease-resistant in the presence of <b>6lc</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A), indicating the direct interaction of <b>6lc</b> with SPOP in cell lysates of both A498 and OS-RC-2 cells. Next, we performed the coimmunoprecipitation (Co-IP) experiment in HEK293T cells in order to elucidate whether <b>6lc</b> could disrupt the SPOP-mediated protein interaction. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, the SPOP-binder <b>6lc</b> is able to inhibit SPOP-binding with its substrate protein PTEN in a dose-dependent manner. Consequently, the abundance of the ubiquitinated PTEN protein is steadily reduced when HEK293T cells are exposed to compound <b>6lc</b> in a concentration-dependent way (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effect of <b>6lc</b> on the interaction with SPOP and the disruption of the SPOP–PTEN interaction in HEK293T cells. (A) Representative western blots for SPOP levels in A498 and OS-RC-2 cells in DARTS. (B) Effect of <b>6lc</b> on the SPOP interaction with PTEN in intact HEK293T cells in the Co-IP assay. (C) Effect of <b>6lc</b> on ubiquitination of PTEN in HEK193T cells. The intensity in the western blot was read with ImageJ. All experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 6lc Disrupts the Oncogenic SPOP-Signaling Pathway in Kidney Cancer Cells</h3><div class="NLM_p">Both PTEN and DUSP7 are tumor suppressors that regulate cell proliferation and apoptosis through the cytoplasmic SPOP-signaling pathway in ccRCC.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> We have previously demonstrated that SPOP inhibitors biochemically illuminate in cells the therapeutic potential of blocking SPOP activity to stabilize PTEN and DUSP7, which suppresses the AKT- and ERK-driven cancer-signaling pathways. Indeed, a steady accumulation of PTEN and DUSP7 proteins is observed in a dose-dependent manner when A498 and OS-RC-2 cell lines are exposed to compound <b>6lc</b> for 12 h (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Therefore, the accumulation of PTEN and DUSP7 in the cytoplasm subsequently results in decreases of phosphorylated AKT and ERK in both ccRCC cell lines. The total cellular abundance of AKT and ERK proteins in A498 and OS-RC-2 cells remains constant in the presence of compound <b>6lc</b>. Of note, real-time quantitative PCR (RT-qPCR) experiments showed that changes of PTEN and DUSP6 in A498 and OS-RC-2 ccRCC cell lines are not at the transcription level (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). In addition, several analogues of <b>6lc</b>, namely, compounds <b>6lf</b>, <b>6li</b>, <b>6i</b>, and <b>6l</b>, also lead to the stabilization and accumulation of the PTEN protein in OS-RC-2 cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C).</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Effect of the SPOP inhibitor on the oncogenic SPOP-signaling pathway in kidney cancer cells. (A) Effect of <b>6lc</b> on the protein levels in the SPOP-signaling pathway in A498 and OS-RC-2 cell lines. (B) Effect of <b>6lc</b> on the transcription of <i>PTEN</i> and <i>DUSP7</i> in A498 and OS-RC-2 cell lines. (C) Effect of SPOP inhibitors on the PTEN level in OS-RC-2 cells. All experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85857" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85857" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We performed synthetic optimization based on the previously established chemical scaffold to explore the SAR for potent SPOP inhibitors. We tested five sets of <b>6b</b> analogues to better understand which motifs in the molecule are important for SPOP engagement. A nanoDSF-based high-throughput assay, which measures the thermal stability of an interested protein, is applied as an initial screening of small-molecule binders for the SPOP protein. Then, the antiproliferative activities against A498 and OS-RC-2 cell lines are evaluated, and the SAR is elucidated. Most amide analogues thermally stabilize the SPOP protein and exhibit antiproliferations in ccRCC cells, which suggests that the nanoDSF is an option for the initial screening of SPOP inhibitors. Compound <b>6lc</b> exhibits strong inhibitory potency against both ccRCC cell lines with IC<sub>50</sub> of 2.1 and 3.5 μM, respectively. Additionally, compound <b>6lc</b> significantly inhibits the colony formation for both A498 and OS-RC-2 cell lines, superior to our previously reported <b>6b</b> and other analogues tested. The performance of different assays, including DSF, NMR titration, SPR analysis, and DARTS assay, has confirmed that the inhibitor <b>6lc</b> directly interacts with the SPOP protein in vitro and in cell lysates. Last, an in-depth mechanistic study has revealed that compound <b>6lc</b> disrupts SPOP–substrate protein interaction in both ccRCC cell lines and as a consequence leads to the stabilization and accumulation of tumor suppressors PTEN and DUSP7 and reduced abundances of phosphorylated AKT and ERK in the downstream. Taken together, we have demonstrated that the SPOP inhibitor <b>6lc</b> represents a new lead for further improvement to sufficiently target the cytoplasmic SPOP-signaling pathway in ccRCC, which stands as a promising strategy to combat kidney cancer. Although SPOP has also been shown to be a tumor suppressor in prostate cancer, it is important to note that inhibition of SPOP may not be a strategy for all cancers.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61061" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61061" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Chemistry</h3><div class="NLM_p last">Unless otherwise stated, all solvents and reagents were purchased from commercial sources and used as received. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded using a Varian-Mercury Plus-400, 500, and 600 NMR spectrometer at rt. NMR spectra were calibrated to the solvent signals of DMSO-<i>d</i><sub>6</sub>, TFA<i>-d</i>, MeOD, or CDCl<sub>3</sub>. Chemical shifts are reported in parts per million (δ scale) as referenced to trimethylsilane, and coupling constant (<i>J</i>) values are reported in hertz (Hz). Data are presented as follows: chemical shift, multiplicity (s = singlet, d = doublet, dd = doublet of doublet, t = triplet, q = quartet, m = multiplet, br = broad), <i>J</i> in Hz, and integration. Flash column chromatography was performed using silica gel (230–400 mesh). Analytical thin-layer chromatography (TLC) was performed on silica gel plates and visualized under ultraviolet light (254 nm). The purity and low-resolution mass spectrometry (LRMS) of all the synthesized compounds were determined by a liquid chromatography tandem mass spectrometer (Agilent 1200LC/G6110A, XBridge, 4.6 × 50 mm, 3.5 μm, 50 °C, flow rate = 1.8 mL/min) with aqueous CH<sub>3</sub>CN and water (10 mM ammonium hydrogen carbonate or 0.01% TFA). The purities of all target compounds are >95% in high-performance liquid chromatography (HPLC).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Synthetic Procedure for Compound <b>2</b></h3><div class="NLM_p last">A mixture of 3-methylpyridin-2-amine <b>1</b> and diethyl malonate was stirred at 220 °C for 2 h. The generated ethanol was distilled during the reaction. The residue was recrystallized from ethanol to give product <b>2</b>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthetic Procedure for Compound <b>3</b></h3><div class="NLM_p last">To a cool dimethylformamide (DMF, 144 mL), POCl<sub>3</sub> (44 g, 288 mmol, 4.0 equiv) was added dropwise. After the mixture was stirred for 30 min at 0 °C, compound <b>2</b> (11.8 g, 72 mmol) was added slowly. The reaction was continued at 0 °C for 1 h and at rt for 1 h and was then refluxed for another 1 h. The mixture was poured into ice water and stirred for 1 h. The precipitate was filtered and washed thoroughly with cold water. After a period of vacuum drying, compound <b>3</b> (10.2 g, 63.8% yield) was collected. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.25 (s, 1H), 9.08–8.99 (m, 1H), 8.25 (d, <i>J</i> = 6.8 Hz, 1H), 7.58 (t, <i>J</i> = 7.0 Hz, 1H), 2.53 (s, 3H).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Synthetic Procedure for Compound <b>4</b></h3><div class="NLM_p">A mixture of intermediate <b>3</b> (446 mg, 2 mmol, 1.0 equiv) and corresponding amine (8 mmol, 4.0 equiv) in EtOH (4 mL) was stirred at rt overnight in the presence of anhydrous Na<sub>2</sub>SO<sub>4</sub> (100 mg). The precipitate was filtered and washed with EtOH. The solid was redissolved in CH<sub>2</sub>Cl<sub>2</sub> and filtered in order to remove Na<sub>2</sub>SO<sub>4</sub>. Compound <b>4</b> was obtained after the evaporation of the solvent under reduced pressure in good yields. Compounds <b>4k</b> and <b>4p</b> were directly used in the next step without purification or NMR data collection.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>E</i>)-9-Methyl-2-((4-methylphenethyl)amino)-3-(((4-methylphenethyl)imino)methyl)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4d</b>)</h4><div class="NLM_p last">White solid (400 mg, 46% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.73 (s, 1H), 8.88 (s, 1H), 8.81 (d, <i>J</i> = 6.4 Hz, 1H), 7.49 (d, <i>J</i> = 6.7 Hz, 1H), 7.18 (dd, <i>J</i> = 8.2, 2.3 Hz, 4H), 7.14 (d, <i>J</i> = 6.0 Hz, 4H), 6.79 (t, <i>J</i> = 7.0 Hz, 1H), 3.88–3.72 (m, 4H), 2.94–2.87 (m, 2H), 2.85 (dd, <i>J</i> = 15.5, 8.1 Hz, 2H), 2.49 (s, 3H), 2.35 (s, 3H), 2.34 (s, 3H).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>E</i>)-2-((4-Methoxyphenethyl)amino)-3-(((4-methoxyphenethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4e</b>)</h4><div class="NLM_p last">White solid (452 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.72 (s, 1H), 8.87 (s, 1H), 8.80 (d, <i>J</i> = 6.9 Hz, 1H), 7.48 (d, <i>J</i> = 6.7 Hz, 1H), 7.20 (d, <i>J</i> = 8.6 Hz, 2H), 7.15 (d, <i>J</i> = 8.6 Hz, 2H), 6.93− 6.84 (m, 4H), 6.77 (t, <i>J</i> = 7.0 Hz, 1H), 3.91–3.73 (m, 10H), 2.88 (t, <i>J</i> = 7.4 Hz, 2H), 2.84 (dd, <i>J</i> = 14.7, 7.4 Hz, 2H), 2.48 (s, 3H).</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>E</i>)-2-((4-Fluorophenethyl)amino)-3-(((4-fluorophenethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4f</b>)</h4><div class="NLM_p last">White solid (473 mg, 53% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 10.61 (s, 1H), 8.83 (s, 1H), 8.78 (d, <i>J</i> = 6.7 Hz, 1H), 7.47 (d, <i>J</i> = 6.1 Hz, 1H), 7.24–7.11 (m, 4H), 6.98 (dt, <i>J</i> = 12.8, 8.5 Hz, 4H), 6.77 (t, <i>J</i> = 6.7 Hz, 1H), 3.86–3.67 (m, 4H), 2.87 (t, <i>J</i> = 6.9 Hz, 2H), 2.82 (t, <i>J</i> = 6.7 Hz, 2H), 2.46 (s, 3H).</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>E</i>)-2-((4-Chlorophenethyl)amino)-3-(((4-chlorophenethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4g</b>)</h4><div class="NLM_p last">White solid (556 mg, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.60 (s, 1H), 8.85 (s, 1H), 8.81 (d, <i>J</i> = 6.7 Hz, 1H), 7.50 (d, <i>J</i> = 6.7 Hz, 1H), 7.28 (ddd, <i>J</i> = 10.8, 5.7, 2.1 Hz, 5H), 7.15 (dd, <i>J</i> = 15.3, 8.3 Hz, 4H), 6.80 (t, <i>J</i> = 7.0 Hz, 1H), 3.87–3.71 (m, 4H), 2.88 (t, <i>J</i> = 7.2 Hz, 2H), 2.83 (t, <i>J</i> = 7.1 Hz, 2H), 2.48 (s, 3H).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>E</i>)-2-((4-Bromophenethyl)amino)-3-(((4-bromophenethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4h</b>)</h4><div class="NLM_p last">White solid (466 mg, 41% yield).<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.58 (s, 1H), 8.85 (s, 1H), 8.80 (d, <i>J</i> = 7.0 Hz, 1H), 7.50 (d, <i>J</i> = 6.7 Hz, 1H), 7.43 (dd, <i>J</i> = 12.1, 5.4 Hz, 4H), 7.10 (dd, <i>J</i> = 13.5, 8.3 Hz, 4H), 6.79 (t, <i>J</i> = 7.0 Hz, 1H), 3.78 (dt, <i>J</i> = 14.4, 7.1 Hz, 4H), 2.86 (t, <i>J</i> = 7.2 Hz, 2H), 2.81 (t, <i>J</i> = 7.0 Hz, 2H), 2.47 (s, 3H).</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>E</i>)-2-((3-Fluorophenethyl)amino)-3-(((3-fluorophenethyl)imino)methyl)-9-methyl-4<i>H</i>-quinolizin-4-one (<b>4i</b>)</h4><div class="NLM_p last">White solid (339 mg, 38% yield). <sup>1</sup>H NMR (600 MHz, CDCl3): δ 10.62 (s, 1H), 8.84 (s, 1H), 8.79 (d, <i>J</i> = 7.0 Hz, 1H), 7.48 (d, <i>J</i> = 6.6 Hz, 1H), 7.28–7.20 (m, 2H), 7.01 (d, <i>J</i> = 7.5 Hz, 1H), 6.99–6.94 (m, 2H), 6.94–6.87 (m, 3H), 6.78 (t, <i>J</i> = 6.9 Hz, 1H), 3.80 (m, 4H), 2.90 (t, <i>J</i> = 7.3 Hz, 2H), 2.85 (t, <i>J</i> = 7.0 Hz, 2H), 2.47 (s, 3H).</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>E</i>)-2-((2-Fluorophenethyl)amino)-3-(((2-fluorophenethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4j</b>)</h4><div class="NLM_p last">White solid (552 mg, 62% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.47 (s, 1H), 8.66 (d, <i>J</i> = 1.0 Hz, 1H), 8.58 (d, <i>J</i> = 7.0 Hz, 1H), 7.27 (d, <i>J</i> = 6.7 Hz, 1H), 7.08–6.94 (m, 4H), 6.95–6.79 (m, 4H), 6.56 (t, <i>J</i> = 6.9 Hz, 1H), 3.73–3.52 (m, 4H), 2.76 (dt, <i>J</i> = 19.0, 7.1 Hz, 4H), 2.27 (s, 3H).</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>E</i>)-9-Methyl-2-((2-(thiophen-2-yl)ethyl)amino)-3-(((2-(thiophen-2-yl)ethyl)imino)methyl)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4l</b>)</h4><div class="NLM_p last">Yellow solid (532 mg, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.61 (s, 1H), 8.70 (s, 1H), 8.61 (d, <i>J</i> = 6.9 Hz, 1H), 7.30 (d, <i>J</i> = 6.0 Hz, 1H), 6.98 (dd, <i>J</i> = 11.7, 5.1 Hz, 2H), 6.84−6.74 (m, 2H), 6.71 (s, 1H), 6.65 (s, 1H), 6.60 (t, <i>J</i> = 6.9 Hz, 1H), 3.89–3.52 (m, 4H), 2.97 (dt, <i>J</i> = 28.4, 6.9 Hz, 4H), 2.29 (s, 3H).</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>E</i>)-9-Methyl-2,3-bis((2-morpholinoethyl)amino)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4m</b>)</h4><div class="NLM_p last">Yellow solid (483 mg, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.69 (s, 1H), 8.91 (s, 1H), 8.80 (d, <i>J</i> = 7.1 Hz, 1H), 7.49 (d, <i>J</i> = 6.8 Hz, 1H), 6.79 (t, <i>J</i> = 7.0 Hz, 1H), 3.82–3.71 (m, 12H), 2.74–2.63 (m, 4H), 2.56 (d, <i>J</i> = 3.8 Hz, 8H), 2.45 (s, 3H).</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>E</i>)-9-Methyl-2-((3-phenylpropyl)amino)-3-(((3-phenylpropyl)imino)methyl)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4n</b>)</h4><div class="NLM_p last">Yellow solid (483 mg, 58% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.65 (s, 1H), 8.74 (s, 1H), 8.60 (d, <i>J</i> = 7.1 Hz, 1H), 7.25 (d, <i>J</i> = 6.6 Hz, 1H), 7.22–6.88 (m, 11H), 6.56 (t, <i>J</i> = 6.9 Hz, 1H), 3.71–3.37 (m, 4H), 2.55 (dd, <i>J</i> = 17.2, 9.0 Hz, 5H), 2.20 (s, 4H), 2.01–1.74 (m, 4H).</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>E</i>)-9-Methyl-2-((2-phenoxyethyl)amino)-3-(((2-phenoxyethyl)imino)methyl)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4o</b>)</h4><div class="NLM_p last">White solid (465 mg, 53% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 10.91 (s, 1H), 8.97 (s, 1H), 8.80 (d, <i>J</i> = 6.8 Hz, 1H), 7.48 (d, <i>J</i> = 6.4 Hz, 1H), 7.27–7.22 (m, 4H), 6.91 (ddd, <i>J</i> = 23.7, 15.7, 7.8 Hz, 6H), 6.78 (t, <i>J</i> = 6.9 Hz, 1H), 4.18 (dd, <i>J</i> = 11.3, 5.9 Hz, 4H), 4.02 (dd, <i>J</i> = 10.3, 5.0 Hz, 2H), 3.95 (s, 2H), 2.43 (s, 3H).</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>E</i>)-2-(Benzylamino)-3-((benzylimino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4q</b>)</h4><div class="NLM_p last">Yellow solid (574 mg, 75% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.20 (s, 1H), 9.08 (s, 1H), 8.89–8.82 (m, 1H), 7.47 (d, <i>J</i> = 6.7 Hz, 1H), 7.41–7.23 (m, 10H), 6.79 (t, <i>J</i> = 7.0 Hz, 1H), 4.86 (d, <i>J</i> = 5.7 Hz, 2H), 4.80 (s, 2H), 2.44 (s, 3H).</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>E</i>)-9-Methyl-2,3-bis((pyridin-4-ylmethyl)amino)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4r</b>)</h4><div class="NLM_p last">Yellow solid (358 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.03 (t, <i>J</i> = 5.7 Hz, 1H), 9.08 (s, 1H), 8.83 (d, <i>J</i> = 7.2 Hz, 1H), 8.56–8.52 (m, 4H), 7.52 (dd, <i>J</i> = 6.8, 1.1 Hz, 1H), 7.25 (d, <i>J</i> = 6.0 Hz, 2H), 7.22 (d, <i>J</i> = 6.0 Hz, 2H), 6.86 (t, <i>J</i> = 7.0 Hz, 1H), 4.86 (d, <i>J</i> = 5.9 Hz, 2H), 4.80 (s, 2H), 2.35 (s, 3H).</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>E</i>)-9-Methyl-2-(phenylamino)-3-((phenylimino)methyl)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4s</b>)</h4><div class="NLM_p last">Yellow solid (610 mg, 86% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 13.22 (s, 1H), 9.20 (s, 1H), 8.82 (dd, <i>J</i> = 7.0, 0.6 Hz, 1H), 7.84 (d, <i>J</i> = 7.6 Hz, 2H), 7.50 (d, <i>J</i> = 6.7 Hz, 1H), 7.41 (dd, <i>J</i> = 11.0, 4.6 Hz, 2H), 7.33 (ddd, <i>J</i> = 9.3, 8.2, 4.2 Hz, 4H), 7.23 (dd, <i>J</i> = 13.9, 6.6 Hz, 1H), 7.09 (t, <i>J</i> = 7.4 Hz, 1H), 6.82 (t, <i>J</i> = 6.9 Hz, 1H), 2.48 (s, 3H).</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>E</i>)-2-(Cyclohexylamino)-3-((cyclohexylimino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4t</b>)</h4><div class="NLM_p last">Yellow solid (506 mg, 69% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 11.01 (d, <i>J</i> = 6.5 Hz, 1H), 8.91 (s, 1H), 8.77 (d, <i>J</i> = 7.0 Hz, 1H), 7.42 (d, <i>J</i> = 6.6 Hz, 1H), 6.71 (t, <i>J</i> = 6.9 Hz, 1H), 4.29–4.15 (m, 1H), 3.21 (dd, <i>J</i> = 10.9, 7.7 Hz, 1H), 2.41 (s, 3H), 2.00 (d, <i>J</i> = 7.5 Hz, 2H), 1.87–1.72 (m, 6H), 1.69–1.57 (m, 2H), 1.53–1.37 (m, 9H).</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>E</i>)-9-Methyl-2,3-bis(methylamino)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4u</b>)</h4><div class="NLM_p last">White solid (249 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.41 (s, 1H), 8.89 (d, <i>J</i> = 1.1 Hz, 1H), 8.80 (d, <i>J</i> = 7.2 Hz, 1H), 7.47 (d, <i>J</i> = 6.8 Hz, 1H), 6.77 (t, <i>J</i> = 7.0 Hz, 1H), 3.46 (d, <i>J</i> = 0.9 Hz, 3H), 3.13 (d, <i>J</i> = 3.7 Hz, 3H), 2.45 (s, 3H).</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>E</i>)-2-(Isopropylamino)-3-((isopropylimino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4v</b>)</h4><div class="NLM_p last">White solid (281 mg, 49% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.88 (d, <i>J</i> = 5.7 Hz, 1H), 8.91 (s, 1H), 8.79 (d, <i>J</i> = 7.1 Hz, 1H), 7.45 (d, <i>J</i> = 6.7 Hz, 1H), 6.74 (t, <i>J</i> = 7.0 Hz, 1H), 4.44 (dp, <i>J</i> = 13.2, 6.5 Hz, 1H), 3.51 (dt, <i>J</i> = 12.5, 6.3 Hz, 1H), 2.44 (s, 3H), 1.33 (d, <i>J</i> = 6.5 Hz, 6H), 1.24 (d, <i>J</i> = 6.3 Hz, 6H).</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>E</i>)-9-Methyl-2,3-bis(propylamino)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4w</b>)</h4><div class="NLM_p last">Yellow solid (236 mg, 43% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.80 (s, 1H), 8.90 (s, 1H), 8.80 (d, <i>J</i> = 7.0 Hz, 1H), 7.47 (d, <i>J</i> = 6.6 Hz, 1H), 6.77 (t, <i>J</i> = 6.9 Hz, 1H), 3.68–3.51 (m, 4H), 2.45 (s, 3H), 1.79–1.63 (m, 5H), 1.01 (dt, <i>J</i> = 17.6, 7.4 Hz, 7H).</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>E</i>)-9-Methyl-2,3-bis(prop-2-yn-1-ylamino)-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4x</b>)</h4><div class="NLM_p last">Yellow solid (261 mg, 49% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.56 (s, 1H), 9.13 (t, <i>J</i> = 1.4 Hz, 1H), 8.82 (d, <i>J</i> = 7.1 Hz, 1H), 7.54 (d, <i>J</i> = 6.8 Hz, 1H), 6.85 (t, <i>J</i> = 7.0 Hz, 1H), 4.55–4.32 (m, 4H), 2.49 (s, 4H), 2.24 (t, <i>J</i> = 2.5 Hz, 1H).</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>E</i>)-2-((2-(1<i>H</i>-Indol-3-yl)ethyl)amino)-3-(((2-(1<i>H</i>-indol-3-yl)ethyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4y</b>)</h4><div class="NLM_p last">Yellow solid (616 mg, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.77 (s, 1H), 8.94 (s, 1H), 8.81 (d, <i>J</i> = 7.0 Hz, 1H), 7.89 (s, 1H), 7.68 (d, <i>J</i> = 7.6 Hz, 2H), 7.63 (d, <i>J</i> = 7.7 Hz, 1H), 7.47 (d, <i>J</i> = 6.4 Hz, 1H), 7.41–7.09 (m, 8H), 6.90 (s, 1H), 6.75 (dd, <i>J</i> = 14.6, 7.6 Hz, 2H), 3.91 (dd, <i>J</i> = 12.3, 6.3 Hz, 2H), 3.85 (t, <i>J</i> = 6.6 Hz, 2H), 3.09 (t, <i>J</i> = 6.6 Hz, 2H), 2.98 (t, <i>J</i> = 6.7 Hz, 2H), 2.48 (s, 3H).</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>E</i>)-2-((4-Chlorophenyl)amino)-3-(((4-chlorophenyl)imino)methyl)-9-methyl-4<i>H</i>-pyrido[1,2-<i>a</i>]pyrimidin-4-one (<b>4z</b>)</h4><div class="NLM_p last">Yellow solid (660 mg, 78% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 13.09 (s, 1H), 9.16 (s, 1H), 8.85 (dd, <i>J</i> = 7.0, 0.7 Hz, 1H), 7.80–7.75 (m, 2H), 7.60 (d, <i>J</i> = 6.7 Hz, 1H), 7.38–7.34 (m, 2H), 7.33–7.29 (m, 2H), 7.25–7.20 (m, 2H), 6.91 (t, <i>J</i> = 6.9 Hz, 1H), 2.50 (s, 3H).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Synthetic Procedure for α-Cyanoacetamide <b>5</b></h3><div class="NLM_p">A mixture of ethyl 2-cyanoacetate (1243 mg, 11 mmol, 1.1 equiv) and corresponding amine (10 mmol, 1.0 equiv) was stirred at rt. After the consumption of amine monitored by TLC, the precipitate was filtered and washed thoroughly with ether. Then, the intermediate α-cyanoacetamide <b>5</b> was obtained in high yields.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 2-Cyano-<i>N</i>-(3-morpholinopropyl)acetamide (<b>5a</b>)</h4><div class="NLM_p last">Brown solid (1.86 g, 88% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.90 (s, 1H), 3.82–3.76 (m, 4H), 3.43 (dd, <i>J</i> = 11.6, 5.7 Hz, 2H), 2.57–2.43 (m, 6H), 1.80–1.66 (m, 2H).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 2-Cyano-<i>N</i>-(3-phenylpropyl)acetamide (<b>5b</b>)</h4><div class="NLM_p last">White solid (1.56 g, 77% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 7.28 (t, <i>J</i> = 7.4 Hz, 2H), 7.24–7.14 (m, 3H), 4.91 (s, 3H), 3.23 (t, <i>J</i> = 7.0 Hz, 2H), 2.66 (t, <i>J</i> = 7.7 Hz, 2H), 1.91–1.78 (m, 2H).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-Cyano-<i>N</i>-(2-morpholinoethyl)acetamide (<b>5c</b>)</h4><div class="NLM_p last">White solid (1.49 g, 76% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 4.92 (s, 2H), 3.79–3.66 (m, 2H), 3.39 (t, <i>J</i> = 6.5 Hz, 1H), 3.33 (s, 1H), 2.52 (t, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 2-Cyano-<i>N</i>-(2-(4-methylpiperazin-1-yl)ethyl)acetamide (<b>5d</b>)</h4><div class="NLM_p last">White solid (1.80 g, 86% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 4.92 (s, 3H), 3.38 (t, <i>J</i> = 6.6 Hz, 2H), 2.53 (t, <i>J</i> = 6.6 Hz, 8H), 2.31 (s, 3H).</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-Cyano-<i>N</i>-phenethylacetamide (<b>5e</b>)</h4><div class="NLM_p last">White solid (1.43 g, 76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.34 (m, 2H), 7.30–7.24 (m, 1H), 7.22 (d, <i>J</i> = 7.1 Hz, 2H), 6.48 (s, 1H), 3.62–3.50 (m, 2H), 3.31 (s, 2H), 2.86 (t, <i>J</i> = 7.0 Hz, 2H).</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-Cyano-<i>N</i>-(4-fluorophenethyl)acetamide (<b>5f</b>)</h4><div class="NLM_p last">White solid (1.63 g, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.20–7.11 (m, 2H), 7.00 (t, <i>J</i> = 8.5 Hz, 2H), 6.60 (s, 1H), 3.51 (q, <i>J</i> = 6.7 Hz, 2H), 3.35 (s, 2H), 2.82 (t, <i>J</i> = 7.1 Hz, 2H).</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-Cyano-<i>N</i>-(4-methoxyphenethyl)acetamide (<b>5g</b>)</h4><div class="NLM_p last">White solid (1.58 g, 72% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 7.14 (d, <i>J</i> = 8.6 Hz, 1H), 6.86 (d, <i>J</i> = 8.6 Hz, 1H), 4.92 (s, 2H), 3.77 (s, 3H), 3.41 (t, <i>J</i> = 7.3 Hz, 2H), 2.76 (t, <i>J</i> = 7.3 Hz, 2H).</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-Benzyl-2-cyanoacetamide (<b>5h</b>)</h4><div class="NLM_p last">White solid (1.36 g, 78% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.45–7.23 (m, 5H), 6.72 (m, 1H), 4.43 (s, 2H), 3.35 (s, 2H).</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-Cyano-<i>N</i>-((tetrahydrofuran-2-yl)methyl)acetamide (<b>5i</b>)</h4><div class="NLM_p last">White solid (1.27 g, 76% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 4.91 (s, 3H), 3.99 (s, 1H), 3.88 (s, 1H), 3.76 (s, 1H), 3.31 (d, <i>J</i> = 15.8 Hz, 3H), 1.94 (s, 3H), 1.60 (s, 1H).</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-Cyano-<i>N</i>-cyclohexylacetamide (<b>5j</b>)</h4><div class="NLM_p last">White solid (1.18 g, 71% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 6.19 (s, 1H), 3.85–3.67 (m, 1H), 3.38 (s, 2H), 1.93 (d, <i>J</i> = 12.0 Hz, 2H), 1.74 (d, <i>J</i> = 13.3 Hz, 2H), 1.64 (d, <i>J</i> = 13.0 Hz, 1H), 1.37 (dd, <i>J</i> = 24.7, 12.5 Hz, 2H), 1.31–1.10 (m, 3H).</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2-Cyano-<i>N</i>-(2-hydroxyethyl)acetamide (<b>5k</b>)</h4><div class="NLM_p last">White solid (937 mg, 73% yield). <sup>1</sup>H NMR (400 MHz, MeOD): δ 4.90 (s, 2H), 3.63 (t, <i>J</i> = 5.6 Hz, 2H), 3.34 (t, <i>J</i> = 5.6 Hz, 2H).</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> General Synthetic Procedure for Compound <b>6</b></h3><div class="NLM_p">A mixture containing α-cyanoacetamide <b>5</b> (0.3 mmol, 1.5 equiv) and corresponding imine intermediate <b>4</b> (0.2 mmol, 1.0 equiv) in CHCl<sub>3</sub> was refluxed overnight in the presence of a catalytic amount of piperidine. The yellow mixture was concentrated in vacuum. Product <b>6</b> was obtained by recrystallization from EtOH in high yields.</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-Imino-10-methyl-1-(4-methylphenethyl)-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6d</b>)</h4><div class="NLM_p last">Yellow solid (72 mg, 69% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.2 Hz, 1H), 7.66 (d, <i>J</i> = 6.7 Hz, 1H), 7.23 (d, <i>J</i> = 7.5 Hz, 2H), 7.15 (d, <i>J</i> = 7.7 Hz, 2H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 4.58 (s, 2H), 3.72 (t, <i>J</i> = 4.6 Hz, 4H), 3.53 (dd, <i>J</i> = 12.0, 6.4 Hz, 2H), 3.01 (t, <i>J</i> = 8.1 Hz, 2H), 2.61 (s, 3H), 2.49 (s, 6H), 2.33 (s, 3H), 1.89–1.76 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.1, 150.1, 136.9, 133.9, 129.3, 128.5, 126.1, 114.1, 66.9, 53.7, 43.5, 32.3, 21.0, 18.1. LRMS <i>m</i>/<i>z</i>: 515.2 [M + H]<sup>+</sup>. HPLC purity 97.6%, <i>t</i><sub>r</sub> = 2.04 min.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-Imino-1-(4-methoxyphenethyl)-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6e</b>)</h4><div class="NLM_p last">Yellow solid (75 mg, 71% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.92 (d, <i>J</i> = 6.7 Hz, 1H), 8.66 (s, 1H), 7.69 (d, <i>J</i> = 6.8 Hz, 1H), 7.26 (s, 1H), 7.05 (t, <i>J</i> = 7.0 Hz, 1H), 6.89 (d, <i>J</i> = 8.6 Hz, 1H), 4.63 (s, 1H), 3.81 (s, 2H), 3.75 (t, <i>J</i> = 4.6 Hz, 4H), 3.55 (dd, <i>J</i> = 11.6, 6.3 Hz, 2H), 3.03 (d, <i>J</i> = 8.1 Hz, 2H), 2.63 (s, 3H), 2.59–2.47 (m, 6H), 1.89–1.80 (m, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 158.3, 156.1, 155.4, 152.8, 150.1, 136.9, 133.9, 129.7, 126.1, 114.1, 66.9, 55.2, 53.7, 43.7, 31.8, 18.1. LRMS <i>m</i>/<i>z</i>: 531.2 [M + H]<sup>+</sup>. HPLC purity 96.6%, <i>t</i><sub>r</sub> = 1.93 min.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-(4-Fluorophenethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6f</b>)</h4><div class="NLM_p last">Yellow solid (77 mg, 74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.88 (d, <i>J</i> = 6.9 Hz, 1H), 7.67 (d, <i>J</i> = 6.7 Hz, 1H), 7.29 (t, <i>J</i> = 6.3 Hz, 2H), 7.02 (dd, <i>J</i> = 16.0, 7.9 Hz, 3H), 4.62 (s, 1H), 3.73 (t, <i>J</i> = 4.4 Hz, 6H), 3.54 (dd, <i>J</i> = 11.6, 6.0 Hz, 2H), 3.04 (t, <i>J</i> = 7.9 Hz, 2H), 2.59 (s, 3H), 2.51 (s, 3H), 1.82 (m, <i>J</i> = 6.4 Hz, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 162.5, 160.9, 156.1, 150.2, 137.1, 133.9, 130.3, 126.2, 115.5, 114.3, 67.0, 53.8, 43.4, 32.0, 18.2. LRMS <i>m</i>/<i>z</i>: 519.2 [M + H]<sup>+</sup>. HPLC purity 97.2%, <i>t</i><sub>r</sub> = 1.96 min.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1-(4-Chlorophenethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6g</b>)</h4><div class="NLM_p last">Yellow solid (68 mg, 64% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 7.1 Hz, 1H), 7.69 (d, <i>J</i> = 6.8 Hz, 1H), 7.34–7.25 (m, 5H), 7.04 (t, <i>J</i> = 7.0 Hz, 1H), 4.64 (s, 2H), 3.74 (t, <i>J</i> = 4.5 Hz, 4H), 3.55 (dd, <i>J</i> = 11.8, 6.0 Hz, 2H), 3.05 (t, <i>J</i> = 7.9 Hz, 2H), 2.59 (s, 3H), 2.52 (s, 6H), 1.92–1.76 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.1, 150.2, 137.1, 133.9, 130.2, 128.8, 126.2, 120.3, 114.3, 67.0, 53.8, 43.2, 32.2, 18.2. LRMS <i>m</i>/<i>z</i>: 535.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.05 min.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-(4-Bromophenethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6h</b>)</h4><div class="NLM_p last">Yellow solid (76 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.8 Hz, 1H), 7.69 (d, <i>J</i> = 6.8 Hz, 1H), 7.45 (d, <i>J</i> = 8.2 Hz, 2H), 7.23 (d, <i>J</i> = 8.2 Hz, 2H), 7.04 (t, <i>J</i> = 7.0 Hz, 1H), 4.64 (s, 2H), 3.74 (t, <i>J</i> = 4.6 Hz, 4H), 3.55 (dd, <i>J</i> = 11.8, 6.2 Hz, 2H), 3.04 (t, <i>J</i> = 8.0 Hz, 2H), 2.59 (s, 3H), 2.58–2.47 (m, 6H), 1.84 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.1, 150.2, 137.1, 133.9, 131.7, 130.6, 126.3, 120.3, 114.2, 67.0, 53.8, 43.1, 32.3, 18.2. LRMS <i>m</i>/<i>z</i>: 581.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.07 min.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-(3-Fluorophenethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6i</b>)</h4><div class="NLM_p last">Yellow solid (63 mg, 61% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.89 (d, <i>J</i> = 6.3 Hz, 1H), 7.68 (d, <i>J</i> = 5.8 Hz, 1H), 7.29 (s, 1H), 7.19–7.00 (m, 2H), 6.93 (s, 1H), 4.67 (s, 1H), 3.74 (s, 4H), 3.55 (d, <i>J</i> = 4.5 Hz, 2H), 3.08 (d, <i>J</i> = 7.4 Hz, 2H), 2.60 (s, 3H), 2.52 (s, 6H), 1.84 (d, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 163.9 162.0, 156.1, 155.5, 153.1, 150.2, 137.0, 133.9, 130.1, 126.2, 124.5, 120.3, 115.8, 115.6, 114.2, 113.6, 113.4, 67.0, 53.8, 43.1, 32.6, 18.1. LRMS <i>m</i>/<i>z</i>: 519.3 [M + H]<sup>+</sup>. HPLC purity 96.7%, <i>t</i><sub>r</sub> = 1.97 min.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1-(2-Fluorophenethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6j</b>)</h4><div class="NLM_p last">Yellow solid (59 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.20 (s, 1H), 9.06 (s, 1H), 8.89 (s, 1H), 7.67 (d, <i>J</i> = 5.5 Hz, 1H), 7.30 (s, 1H), 7.27–7.19 (m, 1H), 7.16–6.90 (m, 3H), 4.62 (s, 2H), 3.75 (t, <i>J</i> = 4.5 Hz, 4H), 3.55 (dd, <i>J</i> = 12.0, 6.2 Hz, 2H), 3.12 (s, 2H), 2.60 (s, 3H), 2.51 (s, 6H), 1.84 (dd, <i>J</i> = 13.2, 6.6 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 162.5, 160.5, 156.2, 150.1, 136.9, 134.2, 131.2, 128.7, 126.2, 124.4, 120.2, 115.5, 114.2, 67.0, 53.8, 41.9, 26.5, 17.9. LRMS <i>m</i>/<i>z</i>: 519.3 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.97 min.</div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-(2-(Diethylamino)ethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6k</b>)</h4><div class="NLM_p last">Yellow solid (59 mg, 60% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.4 Hz, 1H), 7.66 (d, <i>J</i> = 6.6 Hz, 1H), 7.03 (t, <i>J</i> = 7.0 Hz, 1H), 4.45 (s, 2H), 3.73 (s, 4H), 3.52 (d, <i>J</i> = 5.9 Hz, 2H), 2.80 (t, <i>J</i> = 6.2 Hz, 2H), 2.68 (dd, <i>J</i> = 13.7, 6.7 Hz, 4H), 2.57 (s, 3H), 2.50 (s, 6H), 1.91–1.76 (m, 2H), 1.10 (t, <i>J</i> = 7.0 Hz, 6H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.2, 155.5, 153.1, 150.1, 136.9, 136.5, 136.1, 133.9, 126.2, 125.6, 120.3, 116.6, 114.1, 67.0, 53.8, 47.8, 18.2, 12.2. LRMS <i>m</i>/<i>z</i>: 496.3 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.84 min.</div></div><div id="sec4_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6l</b>)</h4><div class="NLM_p last">Yellow solid (64 mg, 63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.89 (d, <i>J</i> = 6.6 Hz, 1H), 7.66 (d, <i>J</i> = 6.8 Hz, 1H), 7.17 (s, 1H), 7.02 (t, <i>J</i> = 6.9 Hz, 1H), 6.95 (d, <i>J</i> = 4.8 Hz, 2H), 4.68 (s, 2H), 3.72 (t, <i>J</i> = 4.5 Hz, 4H), 3.53 (dd, <i>J</i> = 11.8, 6.2 Hz, 2H), 3.29 (t, <i>J</i> = 7.9 Hz, 2H), 2.60 (s, 3H), 2.50 (s, 6H), 1.93–1.76 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.2, 155.4, 153.0, 150.2, 137.0, 134.2, 127.1, 126.2, 125.6, 124.0, 120.2, 114.3, 67.0, 53.8, 43.7, 27.0, 18.2. LRMS <i>m</i>/<i>z</i>: 507.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.93 min.</div></div><div id="sec4_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 2-Imino-10-methyl-1-(2-morpholinoethyl)-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6m</b>)</h4><div class="NLM_p last">Yellow solid (73 mg, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.91 (d, <i>J</i> = 7.0 Hz, 1H), 7.68 (d, <i>J</i> = 6.8 Hz, 1H), 7.04 (t, <i>J</i> = 7.0 Hz, 1H), 4.57 (s, 2H), 3.75 (dt, <i>J</i> = 13.9, 4.5 Hz, 8H), 3.53 (dd, <i>J</i> = 12.0, 6.3 Hz, 2H), 2.76 (t, <i>J</i> = 7.0 Hz, 2H), 2.65 (s, 4H), 2.57 (s, 3H), 2.51 (s, 5H), 1.88–1.76 (m, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.2, 150.2, 137.1, 133.9, 126.3, 120.2, 114.3, 67.0, 54.0, 53.8, 18.2. LRMS <i>m</i>/<i>z</i>: 510.3 [M + H]<sup>+</sup>. HPLC purity 97.6%, <i>t</i><sub>r</sub> = 1.66 min.</div></div><div id="sec4_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(3-phenylpropyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6n</b>)</h4><div class="NLM_p last">Yellow solid (69 mg, 67% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.9 Hz, 1H), 8.71 (s, 1H), 7.68 (d, <i>J</i> = 6.7 Hz, 1H), 7.36–7.15 (m, 6H), 7.06 (t, <i>J</i> = 7.0 Hz, 1H), 4.55 (s, 2H), 3.74 (t, <i>J</i> = 4.4 Hz, 4H), 3.53 (s, 2H), 2.87 (t, <i>J</i> = 7.3 Hz, 2H), 2.63–2.47 (m, 6H), 2.42 (s, 3H), 2.21–2.09 (m, 2H), 1.89–1.79 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.1, 155.5, 152.4, 150.1, 140.9, 137.2, 134.2, 128.7, 128.2, 126.1, 114.4, 66.8, 57.5, 53.7, 44.5, 42.2, 33.4, 27.7, 22.4, 17.9. LRMS <i>m</i>/<i>z</i>: 515.2 [M + H]<sup>+</sup>. HPLC purity 97.8%, <i>t</i><sub>r</sub> = 2.00 min.</div></div><div id="sec4_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(2-phenoxyethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6o</b>)</h4><div class="NLM_p last">Yellow solid (67 mg, 65% yield). <sup>1</sup>H NMR (400 MHz, CDCl3): δ 8.90 (s, 1H), 7.67 (d, <i>J</i> = 6.4 Hz, 1H), 7.28 (d, <i>J</i> = 6.5 Hz, 2H), 7.04 (t, <i>J</i> = 7.0 Hz, 0H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 4.77 (s, 1H), 4.43 (t, <i>J</i> = 5.8 Hz, 2H), 3.88–3.64 (m, 4H), 3.55 (dd, <i>J</i> = 11.9, 6.1 Hz, 2H), 2.53 (s, 9H), 1.95–1.76 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.1, 150.1, 137.0, 134.1, 129.4, 126.2, 120.9, 120.4, 115.0, 114.8, 113.9, 67.0, 53.8, 18.0. LRMS <i>m</i>/<i>z</i>: 517.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.94 min.</div></div><div id="sec4_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 2-Imino-10-methyl-<i>N</i>,1-bis(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6p</b>)</h4><div class="NLM_p last">Yellow solid (60 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.89 (d, <i>J</i> = 6.8 Hz, 1H), 7.66 (d, <i>J</i> = 6.7 Hz, 1H), 7.03 (t, <i>J</i> = 7.0 Hz, 1H), 4.43 (s, 2H), 3.75 (dd, <i>J</i> = 14.8, 10.6 Hz, 8H), 3.62–3.39 (m, 2H), 2.56 (s, 4H), 2.49 (s, 12H), 2.12–1.92 (m, 4H), 1.89–1.75 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.2, 153.0, 150.1, 137.0, 133.9, 126.2, 120.0, 114.2, 67.0, 53.7, 40.5, 23.9, 18.1. LRMS <i>m</i>/<i>z</i>: 524.2 [M + H]<sup>+</sup>. HPLC purity 98.2%, <i>t</i><sub>r</sub> = 1.69 min.</div></div><div id="sec4_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 1-Benzyl-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6q</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.6 Hz, 1H), 7.63 (d, <i>J</i> = 6.6 Hz, 1H), 7.31 (s, 4H), 7.02 (t, <i>J</i> = 6.9 Hz, 1H), 5.68 (s, 2H), 3.82–3.64 (m, 4H), 3.53–3.37 (m, 2H), 2.46 (s, 9H), 1.88–1.69 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl3): δ 156.2, 153.3, 150.2, 137.0, 134.3, 128.9, 127.6, 126.5, 125.9, 124.7, 120.4, 114.4, 67.0, 53.8, 45.7, 18.1. LRMS <i>m</i>/<i>z</i>: 487.4 [M + H]<sup>+</sup>. HPLC purity 97.6%, <i>t</i><sub>r</sub> = 2.07 min.</div></div><div id="sec4_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(pyridin-4-ylmethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6r</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.87 (d, <i>J</i> = 7.0 Hz, 1H), 8.51 (d, <i>J</i> = 4.7 Hz, 2H), 7.62 (d, <i>J</i> = 6.8 Hz, 1H), 7.24 (d, <i>J</i> = 4.1 Hz, 2H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 5.70 (s, 2H), 3.70 (t, <i>J</i> = 4.5 Hz, 4H), 3.50 (d, <i>J</i> = 5.4 Hz, 2H), 2.49 (s, 6H), 2.38 (s, 3H), 1.79 (dd, <i>J</i> = 12.8, 6.3 Hz, 2H). LRMS <i>m</i>/<i>z</i>: 488.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.95 min.</div></div><div id="sec4_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-phenyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6s</b>)</h4><div class="NLM_p last">Yellow solid (75 mg, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.96 (s, 1H), 9.11 (s, 1H), 8.87 (d, <i>J</i> = 7.2 Hz, 1H), 7.63 (t, <i>J</i> = 7.5 Hz, 2H), 7.56 (d, <i>J</i> = 7.4 Hz, 1H), 7.54–7.47 (m, 1H), 7.24 (s, 1H), 6.96 (t, <i>J</i> = 7.0 Hz, 1H), 6.65 (s, 1H), 3.92–3.59 (m, 4H), 3.48 (dd, <i>J</i> = 12.7, 6.8 Hz, 2H), 2.42 (t, <i>J</i> = 7.3 Hz, 6H), 1.99 (s, 3H), 1.79 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 163.7, 156.3, 153.8, 150.0, 136.9, 135.6, 134.4, 130.4, 129.4, 126.0, 120.0, 114.3, 96.0, 67.0, 56.6, 53.7, 37.7, 26.5, 17.1. LRMS <i>m</i>/<i>z</i>: 473.2 [M + H]<sup>+</sup>. HPLC purity 98.2%, <i>t</i><sub>r</sub> = 1.92 min.</div></div><div id="sec4_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 1-Cyclohexyl-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6t</b>)</h4><div class="NLM_p last">Yellow solid (63 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.86 (d, <i>J</i> = 6.9 Hz, 1H), 8.46 (s, 1H), 7.64 (d, <i>J</i> = 6.8 Hz, 1H), 7.00 (t, <i>J</i> = 7.0 Hz, 1H), 5.36 (s, 1H), 3.73 (t, <i>J</i> = 4.5 Hz, 4H), 3.52 (dd, <i>J</i> = 11.1, 5.9 Hz, 2H), 2.81 (s, 2H), 2.62 (s, 3H), 2.59–2.46 (m, 5H), 1.95 (d, <i>J</i> = 13.4 Hz, 2H), 1.87–1.77 (m, 5H), 1.52 (m, 2H), 1.31 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.4, 149.4, 136.6, 134.1, 126.0, 120.8, 114.0, 67.0, 53.8, 28.9, 26.8, 25.8, 18.6. LRMS <i>m</i>/<i>z</i>: 479.5 [M + H]<sup>+</sup>. HPLC purity 97.5%, <i>t</i><sub>r</sub> = 2.17 min.</div></div><div id="sec4_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 2-Imino-1,10-dimethyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6u</b>)</h4><div class="NLM_p last">Yellow solid (66 mg, 80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.17 (s, 1H), 9.08 (s, 1H), 8.89 (d, <i>J</i> = 6.4 Hz, 1H), 7.64 (d, <i>J</i> = 6.7 Hz, 1H), 7.00 (dd, <i>J</i> = 9.5, 4.4 Hz, 1H), 4.35 (s, 2H), 3.72 (t, <i>J</i> = 4.5 Hz, 4H), 3.52 (dd, <i>J</i> = 12.1, 6.3 Hz, 2H), 2.56 (s, 3H), 2.48 (s, 5H), 1.90–1.74 (m, 4H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.2, 150.1, 136.8, 134.0, 126.1, 120.1, 114.1, 67.0, 53.8, 43.6, 20.0, 18.0, 11.7. LRMS <i>m</i>/<i>z</i>: 439.1 [M + H]<sup>+</sup>. HPLC purity 96.1%, <i>t</i><sub>r</sub> = 1.81 min.</div></div><div id="sec4_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 2-Imino-1-isopropyl-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6v</b>)</h4><div class="NLM_p last">Yellow solid (71 mg, 81% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.84 (d, <i>J</i> = 7.0 Hz, 1H), 7.61 (d, <i>J</i> = 6.8 Hz, 1H), 6.97 (t, <i>J</i> = 7.0 Hz, 1H), 5.85 (s, 1H), 3.70 (t, <i>J</i> = 4.5 Hz, 4H), 3.50 (dd, <i>J</i> = 11.8, 6.2 Hz, 2H), 2.57 (s, 3H), 2.50 (m, 6H), 1.85–1.75 (m, 2H), 1.69 (d, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.4, 149.6, 136.6, 134.1, 126.0, 120.9, 114.0, 67.0, 53.8, 46.9, 19.5, 18.5. LRMS <i>m</i>/<i>z</i>: 439.1 [M + H]<sup>+</sup>. HPLC purity 97.9%, <i>t</i><sub>r</sub> = 1.80 min.</div></div><div id="sec4_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-propyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6w</b>)</h4><div class="NLM_p last">Yellow solid (74 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.20 (s, 1H), 9.07 (s, 1H), 8.91 (s, 1H), 7.67 (d, <i>J</i> = 6.5 Hz, 1H), 7.03 (s, 2H), 3.74 (s, 4H), 3.54 (d, <i>J</i> = 5.8 Hz, 2H), 2.58 (s, 3H), 2.50 (s, 6H), 1.83 (d, <i>J</i> = 7.5 Hz, 4H), 1.09 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.2, 155.6, 153.0, 150.1, 136.8, 134.0, 126.1, 120.2, 114.1, 67.0, 53.8, 43.6, 20.0, 18.0, 11.7. LRMS <i>m</i>/<i>z</i>: 439.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.82 min.</div></div><div id="sec4_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 2-Imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(prop-2-yn-1-yl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6x</b>)</h4><div class="NLM_p last">Yellow solid (64 mg, 74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.21 (t, <i>J</i> = 4.5 Hz, 1H), 9.07–8.99 (m, 1H), 8.95 (s, 1H), 8.09 (d, <i>J</i> = 1.0 Hz, 1H), 7.77–7.69 (m, 1H), 7.11 (t, <i>J</i> = 7.0 Hz, 1H), 3.80–3.71 (m, 4H), 3.65 (m, 2H), 2.69 (s, 3H), 2.54 (m, 9H), 1.92 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 162.8, 157.3, 149.5, 144.9, 143.6, 142.2, 136.6, 134.5, 125.9, 117.4, 114.6, 108.7, 99.9, 66.9, 56.3, 53.7, 37.6, 26.4, 17.9, 14.3. LRMS <i>m</i>/<i>z</i>: 435.1 [M + H]<sup>+</sup>. HPLC purity 99.4%, <i>t</i><sub>r</sub> = 1.95 min.</div></div><div id="sec4_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 1-(2-(1<i>H</i>-indol-3-yl)ethyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6y</b>)</h4><div class="NLM_p last">Yellow solid (61 mg, 57% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.90 (d, <i>J</i> = 6.9 Hz, 1H), 8.30 (s, 1H), 7.77 (d, <i>J</i> = 7.4 Hz, 1H), 7.64 (d, <i>J</i> = 7.0 Hz, 1H), 7.40 (d, <i>J</i> = 8.1 Hz, 1H), 7.23 (t, <i>J</i> = 7.4 Hz, 1H), 7.17 (t, <i>J</i> = 7.3 Hz, 1H), 7.11 (s, 1H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 4.73 (s, 2H), 3.75 (t, <i>J</i> = 4.5 Hz, 4H), 3.55 (dd, <i>J</i> = 12.0, 6.1 Hz, 2H), 3.25 (t, <i>J</i> = 7.7 Hz, 2H), 2.53 (s, 9H), 1.88–1.81 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 166.8, 156.2, 155.9, 153.3, 150.4, 140.13, 138.3, 136.7, 133.5, 130.4, 127.8, 127.4, 126.3, 123.0, 121.4, 119.6, 119.5, 119.3, 118.7, 116.1, 115.3, 112.3, 111.8, 93.9, 66.7, 56.7, 53.9, 42.8, 38.1, 26.2, 22.7, 17.9. LRMS <i>m</i>/<i>z</i>: 540.3 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.90 min.</div></div><div id="sec4_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 1-(4-Chlorophenyl)-2-imino-10-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6z</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.88 (s, 1H), 9.11 (s, 1H), 8.91 (d, <i>J</i> = 6.8 Hz, 1H), 7.64 (d, <i>J</i> = 8.5 Hz, 2H), 7.58 (d, <i>J</i> = 6.8 Hz, 1H), 7.24 (d, <i>J</i> = 8.5 Hz, 2H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 3.75 (t, <i>J</i> = 4.3 Hz, 4H), 3.51 (q, <i>J</i> = 6.5 Hz, 2H), 2.51 (d, <i>J</i> = 4.9 Hz, 6H), 2.07 (s, 3H), 1.91–1.77 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 163.7, 156.1, 153.7, 150.0, 137.0, 135.4, 134.4, 134.1, 131.0, 130.7, 126.1, 120.0, 114.5, 96.1, 66.7, 56.6, 53.6, 37.7, 26.3, 17.3. LRMS <i>m</i>/<i>z</i>: 507.1 [M + H]<sup>+</sup>. HPLC purity 96.2%, <i>t</i><sub>r</sub> = 2.01 min.</div></div><div id="sec4_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 2-Imino-10-methyl-5-oxo-<i>N</i>-(3-phenylpropyl)-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6la</b>)</h4><div class="NLM_p last">Yellow solid (73 mg, 73% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.18 (s, 1H), 9.06 (s, 1H), 8.92 (d, <i>J</i> = 7.0 Hz, 1H), 7.69 (d, <i>J</i> = 6.6 Hz, 1H), 7.27 (m, 7H), 7.05 (t, <i>J</i> = 6.9 Hz, 1H), 7.02–6.92 (m, 2H), 4.63 (s, 2H), 3.51 (dd, <i>J</i> = 12.7, 6.6 Hz, 2H), 3.30 (t, <i>J</i> = 7.9 Hz, 2H), 2.76 (t, <i>J</i> = 7.6 Hz, 2H), 2.62 (s, 3H), 2.09–1.90 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 156.3, 150.1, 141.8, 137.1, 134.2, 128.5, 127.2, 126.2, 125.9, 125.7, 124.3, 120.1, 114.4, 43.6, 39.2, 33.5, 31.2, 27.0, 18.3. LRMS <i>m</i>/<i>z</i>: 498.1 [M + H]<sup>+</sup>. HPLC purity 99.5%, <i>t</i><sub>r</sub> = 2.25 min.</div></div><div id="sec4_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 2-Imino-10-methyl-<i>N</i>-(2-morpholinoethyl)-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lb</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.13 (s, 1H), 9.08 (s, 1H), 8.93 (s, 1H), 7.69 (s, 1H), 7.13 (s, 1H), 7.03 (m, 1H), 4.63 (m, 2H), 4.05 (s, 1H), 3.05 (t, <i>J</i> = 7.7 Hz, 2H), 2.60 (s, 3H), 2.02 (s, 2H), 1.77 (s, 2H), 1.65 (s, 2H), 1.45 (m, 4H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.3, 155.3, 150.2, 137.1, 134.2, 127.2, 126.2, 125.6, 124.1, 120.3, 114.3, 67.1, 57.3, 53.5, 43.6, 36.6, 27.0, 18.2. LRMS <i>m</i>/<i>z</i>: 493.1 [M + H]<sup>+</sup>. HPLC purity 97.5%, <i>t</i><sub>r</sub> = 1.90 min.</div></div><div id="sec4_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 2-Imino-10-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)ethyl)-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lc</b>)</h4><div class="NLM_p last">Yellow solid (69 mg, 68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.92 (d, <i>J</i> = 6.6 Hz, 1H), 7.69 (d, <i>J</i> = 6.8 Hz, 1H), 7.20 (s, 1H), 7.05 (t, <i>J</i> = 7.0 Hz, 1H), 6.98 (d, <i>J</i> = 6.2 Hz, 2H), 4.69 (s, 2H), 3.60 (d, <i>J</i> = 4.5 Hz, 2H), 3.31 (t, <i>J</i> = 7.9 Hz, 2H), 2.66 (d, <i>J</i> = 6.3 Hz, 3H), 2.62 (s, 5H), 2.60–2.42 (m, 5H), 2.34 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.3, 150.2, 137.0, 134.2, 127.1, 126.1, 125.6, 124.1, 114.3, 56.8, 55.1, 52.9, 46.0, 43.6, 36.8, 27.0, 18.2. LRMS <i>m</i>/<i>z</i>: 506.2 [M + H]<sup>+</sup>. HPLC purity 95.6%, <i>t</i><sub>r</sub> = 1.88 min.</div></div><div id="sec4_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 2-Imino-10-methyl-5-oxo-<i>N</i>-phenethyl-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6ld</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.17 (s, 1H), 8.98 (s, 1H), 8.88 (d, <i>J</i> = 6.1 Hz, 1H), 7.67 (d, <i>J</i> = 6.8 Hz, 1H), 7.44–7.17 (m, 6H), 6.99 (m, 3H), 4.60 (s, 2H), 3.73 (dd, <i>J</i> = 13.3, 6.9 Hz, 2H), 3.27 (t, <i>J</i> = 8.0 Hz, 2H), 2.97 (t, <i>J</i> = 7.3 Hz, 2H), 2.60 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.2, 155.0, 152.7, 150.1, 139.5, 137.1, 134.1, 128.9, 128.5, 127.2, 126.3, 125.6, 124.2, 120.1, 114.4, 43.5, 41.2, 35.9, 26.9, 18.2. LRMS <i>m</i>/<i>z</i>: 502.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.20 min.</div></div><div id="sec4_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> <i>N</i>-(4-Fluorophenethyl)-2-imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6le</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.93 (d, <i>J</i> = 6.7 Hz, 1H), 7.70 (d, <i>J</i> = 6.6 Hz, 1H), 7.28–7.19 (m, 3H), 7.11–6.94 (m, 5H), 4.66 (s, 2H), 3.71 (dd, <i>J</i> = 13.2, 6.8 Hz, 2H), 3.30 (t, <i>J</i> = 7.9 Hz, 2H), 2.94 (t, <i>J</i> = 7.3 Hz, 2H), 2.63 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 164.4, 162.6, 160.6, 156.2, 155.1, 152.7, 150.2, 140.0, 137.2, 135.1, 134.2, 130.2, 128.6, 127.2, 126.2, 125.7, 124.2, 119.8, 115.3, 115.2, 114.4, 43.6, 41.2, 35.0, 26.9, 18.2. LRMS <i>m</i>/<i>z</i>: 484.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.20 min.</div></div><div id="sec4_6_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 2-Imino-<i>N</i>-(4-methoxyphenethyl)-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lf</b>)</h4><div class="NLM_p last">Yellow solid (52 mg, 51% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.16 (s, 1H), 9.06 (s, 1H), 8.91 (s, 1H), 7.68 (d, <i>J</i> = 6.7 Hz, 1H), 7.22 (d, <i>J</i> = 8.3 Hz, 3H), 7.04 (t, <i>J</i> = 7.0 Hz, 1H), 6.99 (s, 1H), 6.96 (d, <i>J</i> = 2.9 Hz, 1H), 6.89 (d, <i>J</i> = 8.5 Hz, 2H), 4.59 (s, 2H), 3.82 (s, 3H), 3.69 (dd, <i>J</i> = 13.2, 6.9 Hz, 2H), 3.28 (t, <i>J</i> = 8.0 Hz, 2H), 2.91 (t, <i>J</i> = 7.3 Hz, 2H), 2.61 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl3): δ 158.2, 156.2, 155.1, 152.8, 150.1, 137.1, 134.1, 131.5, 129.8, 127.2, 126.2, 125.6, 124.2, 120.1, 114.4, 114.0, 55.3, 43.5, 41.4, 35.0, 27.0, 18.2. LRMS <i>m</i>/<i>z</i>: 514.2 [M + H]<sup>+</sup>. HPLC purity 98.1%, <i>t</i><sub>r</sub> = 2.17 min.</div></div><div id="sec4_6_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> <i>N</i>-Benzyl-2-imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lg</b>)</h4><div class="NLM_p last">Yellow solid (64 mg, 68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.56 (s, 1H), 9.10 (s, 1H), 8.91 (s, 1H), 7.69 (d, <i>J</i> = 6.7 Hz, 1H), 7.39 (dt, <i>J</i> = 15.0, 7.5 Hz, 4H), 7.30 (d, <i>J</i> = 7.2 Hz, 1H), 7.21 (d, <i>J</i> = 4.7 Hz, 1H), 7.05 (t, <i>J</i> = 6.9 Hz, 1H), 7.02–6.92 (m, 2H), 4.69 (d, <i>J</i> = 5.2 Hz, 2H), 4.60 (s, 1H), 3.28 (t, <i>J</i> = 8.1 Hz, 2H), 2.62 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 155.8, 154.5, 149.7, 142.7, 136.7, 133.7, 128.2, 127.3, 126.8, 125.7, 125.2, 123.9, 119.4, 114.0, 43.2, 26.47, 17.8. LRMS <i>m</i>/<i>z</i>: 470.0 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.17 min.</div></div><div id="sec4_6_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 2-Imino-10-methyl-5-oxo-<i>N</i>-((tetrahydrofuran-2-yl)methyl)-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lh</b>)</h4><div class="NLM_p last">Yellow solid (73 mg, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.92 (d, <i>J</i> = 6.9 Hz, 1H), 8.81 (s, 1H), 7.69 (d, <i>J</i> = 6.6 Hz, 1H), 7.21 (d, <i>J</i> = 4.7 Hz, 1H), 7.05 (t, <i>J</i> = 6.9 Hz, 1H), 7.02–6.91 (m, 2H), 4.63 (s, 2H), 4.13 (d, <i>J</i> = 3.7 Hz, 1H), 3.96 (dd, <i>J</i> = 14.3, 7.0 Hz, 1H), 3.82 (td, <i>J</i> = 14.1, 5.5 Hz, 2H), 3.47–3.35 (m, 1H), 3.29 (t, <i>J</i> = 8.0 Hz, 2H), 2.62 (s, 3H), 2.15–2.02 (m, 1H), 1.95 (dd, <i>J</i> = 13.8, 6.7 Hz, 2H), 1.75–1.59 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 164.7, 156.2, 155.1, 152.8, 150.2, 140.2, 137.1, 134.2, 127.2, 126.2, 125.6, 124.2, 120.1, 114.4, 95.7, 77.9, 68.3, 43.7, 29.0, 27.0, 26.0, 18.2. LRMS <i>m</i>/<i>z</i>: 464.0 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.99 min.</div></div><div id="sec4_6_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> <i>N</i>-Cyclohexyl-2-imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6li</b>)</h4><div class="NLM_p last">Yellow solid (69 mg, 75% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.92 (s, 1H), 7.69 (d, <i>J</i> = 6.7 Hz, 1H), 7.22 (s, 1H), 7.05 (t, <i>J</i> = 7.0 Hz, 1H), 6.98 (d, <i>J</i> = 8.6 Hz, 1H), 4.63 (s, 2H), 4.02 (s, 1H), 3.29 (t, <i>J</i> = 8.1 Hz, 2H), 2.62 (s, 2H), 2.01 (s, 1H), 1.75 (s, 1H), 1.67–1.59 (m, 1H), 1.41 (m, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.2, 155.3, 152.8, 150.1, 140.3, 137.0, 134.2, 127.2, 126.2, 125.6, 124.1, 120.5, 114.3, 48.3, 43.6, 33.0, 27.0, 25.8, 24.7, 18.2. LRMS <i>m</i>/<i>z</i>: 462.1 [M + H]<sup>+</sup>. HPLC purity 98.3%, <i>t</i><sub>r</sub> = 2.28 min.</div></div><div id="sec4_6_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> <i>N</i>-(2-Hydroxyethyl)-2-imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lj</b>)</h4><div class="NLM_p last">Yellow solid (69 mg, 81% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.86 (d, <i>J</i> = 5.5 Hz, 1H), 7.68 (d, <i>J</i> = 6.7 Hz, 1H), 7.20 (d, <i>J</i> = 4.9 Hz, 1H), 7.04 (t, <i>J</i> = 6.9 Hz, 1H), 7.00–6.95 (m, 1H), 6.93 (d, <i>J</i> = 3.0 Hz, 1H), 4.55 (s, 2H), 3.96–3.82 (m, 2H), 3.64 (d, <i>J</i> = 4.7 Hz, 2H), 3.24 (s, 2H), 2.58 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.1, 154.9, 152.7, 150.1, 137.3, 134.1, 127.2, 126.2, 125.6, 124.2, 114.5, 62.9, 58.5, 43.6, 43.1, 26.9, 18.4, 18.2. LRMS <i>m</i>/<i>z</i>: 424.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.83 min.</div></div><div id="sec4_6_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 2-Imino-9-methyl-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6lk</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.38 (d, <i>J</i> = 6.2 Hz, 1H), 7.65 (d, <i>J</i> = 8.6 Hz, 1H), 7.47 (d, <i>J</i> = 8.9 Hz, 1H), 7.41 (dd, <i>J</i> = 9.1, 2.0 Hz, 1H), 7.20 (d, <i>J</i> = 5.1 Hz, 1H), 7.13 (d, <i>J</i> = 9.1 Hz, 1H), 6.88 (d, <i>J</i> = 3.3 Hz, 1H), 3.83 (q, <i>J</i> = 6.6 Hz, 2H), 3.78–3.67 (m, 4H), 3.59–3.47 (m, 2H), 3.26 (t, <i>J</i> = 6.7 Hz, 4H), 2.50 (s, 5H), 2.43 (s, 3H), 2.27 (s, 2H), 1.81 (d, <i>J</i> = 6.6 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.6, 162.3, 161.0, 155.7, 153.3, 142.0, 141.0, 138.4, 127.2, 126.5, 125.3, 125.9, 124.4, 124.9, 124.7, 121.5, 94.5, 67.0, 53.8, 52.4, 43.2, 30.8, 27.2, 18.2, 17.8. LRMS <i>m</i>/<i>z</i>: 507.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.92 min.</div></div><div id="sec4_6_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 2-Imino-<i>N</i>-(3-morpholinopropyl)-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxamide (<b>6ll</b>)</h4><div class="NLM_p last">Yellow solid (58 mg, 59% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.00 (s, 1H), 7.80 (t, <i>J</i> = 7.3 Hz, 1H), 7.56 (d, <i>J</i> = 8.7 Hz, 1H), 7.19 (s, 1H), 7.12 (s, 1H), 6.96 (s, 2H), 4.68 (s, 2H), 3.74 (t, <i>J</i> = 4.5 Hz, 4H), 3.55 (d, <i>J</i> = 5.6 Hz, 2H), 3.29 (t, <i>J</i> = 7.6 Hz, 2H), 2.52 (s, 6H), 1.94–1.78 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 155.8, 150.7, 138.1, 128.1, 127.0, 126.3, 126.0, 125.6, 124.0, 120.4, 114.7, 67.0, 53.8, 43.3, 27.2. LRMS <i>m</i>/<i>z</i>: 493.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.85 min.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> General Synthetic Procedure for Compound <b>7</b></h3><div class="NLM_p">A mixture of ethyl 2-cyanoacetate (113 mg, 1.0 mmol, 2.0 equiv) and corresponding imine intermediate <b>4</b> (0.5 mmol, 1.0 equiv) was refluxed in CHCl<sub>3</sub> overnight in the presence of a catalytic amount of piperidine. The yellow mixture was concentrated in vacuum. The crude product <b>7</b> was purified by recrystallization from EtOH as a yellow solid.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Ethyl 2-Imino-1-isopropyl-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7a</b>)</h4><div class="NLM_p last">Yellow solid (152 mg, 89% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.69 (s, 1H), 8.85 (dd, <i>J</i> = 7.1, 0.7 Hz, 1H), 8.64 (s, 1H), 7.64 (d, <i>J</i> = 6.8 Hz, 1H), 6.99 (t, <i>J</i> = 7.0 Hz, 1H), 6.09 (dt, <i>J</i> = 13.9, 6.9 Hz, 1H), 4.35 (q, <i>J</i> = 7.1 Hz, 2H), 2.63 (d, <i>J</i> = 12.6 Hz, 3H), 1.73 (t, <i>J</i> = 7.6 Hz, 6H), 1.40 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.7, 157.2, 156.4, 155.0, 150.2, 137.4, 137.0, 134.3, 126.0, 114.1, 114.9, 95.3, 61.0, 58.4, 46.7, 19.5, 18.7, 18.4, 14.3. LRMS <i>m</i>/<i>z</i>: 341.1 [M + H]<sup>+</sup>. HPLC purity 97.7%, <i>t</i><sub>r</sub> = 2.09 min.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> Ethyl 2-Imino-10-methyl-5-oxo-1-propyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7b</b>)</h4><div class="NLM_p last">Yellow solid (148 mg, 87% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.51 (s, 1H), 8.86 (d, <i>J</i> = 7.1 Hz, 1H), 8.66 (s, 1H), 7.66 (d, <i>J</i> = 6.8 Hz, 1H), 7.01 (t, <i>J</i> = 7.0 Hz, 1H), 4.58–4.46 (m, 2H), 4.36 (q, <i>J</i> = 7.1 Hz, 2H), 2.57 (s, 3H), 1.83 (dd, <i>J</i> = 15.2, 7.5 Hz, 2H), 1.41 (t, <i>J</i> = 7.1 Hz, 3H), 1.05 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.1, 155.8, 153.9, 150.4, 136.9, 136.7, 133.7, 125.6, 113.5, 60.5, 43.5, 19.7, 17.5, 13.9, 11.2. LRMS <i>m</i>/<i>z</i>: 341.0 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.08 min.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> Ethyl 2-Imino-10-methyl-5-oxo-1-phenethyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7c</b>)</h4><div class="NLM_p last">Yellow solid (176 mg, 87% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.60 (s, 1H), 8.86 (d, <i>J</i> = 7.1 Hz, 1H), 8.66 (s, 1H), 7.67 (d, <i>J</i> = 6.8 Hz, 1H), 7.38 (d, <i>J</i> = 7.3 Hz, 2H), 7.31 (t, <i>J</i> = 7.5 Hz, 2H), 7.22 (d, <i>J</i> = 7.3 Hz, 1H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 4.84–4.73 (m, 2H), 4.38 (q, <i>J</i> = 7.1 Hz, 2H), 3.14–3.00 (m, 2H), 2.60 (s, 3H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.1, 155.8, 155.3, 153.8, 150.4, 139.0, 136.8, 133.7, 128.5, 127.9, 125.7, 113.6, 113.4, 94.2, 60.6, 43.2, 32.4, 17.7, 13.9. LRMS <i>m</i>/<i>z</i>: 403.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.24 min.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> Ethyl 1-(4-Chlorophenethyl)-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7d</b>)</h4><div class="NLM_p last">Yellow solid (183 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.56 (s, 1H), 8.85 (d, <i>J</i> = 7.1 Hz, 1H), 8.63 (s, 1H), 7.67 (d, <i>J</i> = 6.8 Hz, 1H), 7.36–7.18 (m, 4H), 7.02 (d, <i>J</i> = 7.0 Hz, 1H), 4.77–4.66 (m, 2H), 4.36 (q, <i>J</i> = 7.1 Hz, 2H), 3.03 (dd, <i>J</i> = 9.2, 6.7 Hz, 2H), 2.56 (s, 3H), 1.41 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.5, 156.1, 155.7, 154.2, 150.9, 137.9, 137.5, 137.3, 134.0, 132.0, 130.3, 128.5, 126.2, 114.2, 113.8, 94.6, 61.1, 43.4, 32.2, 18.1, 14.3. LRMS <i>m</i>/<i>z</i>: 437.0 [M + H]<sup>+</sup>. HPLC purity 95.0%, <i>t</i><sub>r</sub> = 2.34 min.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Ethyl 2-Imino-10-methyl-1-(4-methylphenethyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7e</b>)</h4><div class="NLM_p last">Yellow solid (158 mg, 76% yield). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.57 (s, 1H), 8.85 (d, <i>J</i> = 6.7 Hz, 1H), 8.65 (s, 1H), 7.65 (d, <i>J</i> = 6.8 Hz, 1H), 7.30–7.21 (m, 3H), 7.11 (d, <i>J</i> = 7.7 Hz, 2H), 6.99 (t, <i>J</i> = 7.0 Hz, 1H), 4.81–4.68 (m, 2H), 4.36 (q, <i>J</i> = 7.1 Hz, 2H), 3.08–2.97 (m, 2H), 2.59 (s, 3H), 2.30 (s, 3H), 1.40 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.6, 156.3, 155.9, 154.3, 150.9, 137.3, 136.4, 135.7, 134.2, 129.1, 128.9, 126.2, 114.0, 94.7, 61.0, 43.9, 32.5, 21.0, 18.2, 14.3. LRMS <i>m</i>/<i>z</i>: 417.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.37 min.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Ethyl 2-Imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7f</b>)</h4><div class="NLM_p last">Yellow solid (165 mg, 81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.57 (s, 1H), 8.86 (d, <i>J</i> = 7.1 Hz, 1H), 8.66 (s, 1H), 7.67 (d, <i>J</i> = 6.9 Hz, 1H), 7.14 (dd, <i>J</i> = 4.9, 1.3 Hz, 1H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 6.98–6.92 (m, 2H), 4.87–4.74 (m, 2H), 4.37 (q, <i>J</i> = 7.1 Hz, 2H), 3.38–3.25 (m, 2H), 2.58 (d, <i>J</i> = 18.6 Hz, 3H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 171.1, 1654, 156.1, 155.7, 154.1, 150.8, 141.4, 137.3, 134.2, 126.8, 126.0, 125.3, 123.5, 114.1, 113.7, 94.5, 61.0, 60.3, 58.3, 43.6, 26.8, 20.9, 18.3, 18.1, 14.2, 14.1. LRMS <i>m</i>/<i>z</i>: 409.1 [M + H]<sup>+</sup>. HPLC purity 96.8%, <i>t</i><sub>r</sub> = 2.23 min.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Ethyl 1-(Furan-2-ylmethyl)-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7g</b>)</h4><div class="NLM_p last">Yellow solid (158 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 10.31 (s, 1H), 9.28 (s, 1H), 9.03 (d, <i>J</i> = 7.1 Hz, 1H), 8.00 (d, <i>J</i> = 7.0 Hz, 1H), 7.36 (t, <i>J</i> = 7.0 Hz, 1H), 7.25 (s, 1H), 6.99 (d, <i>J</i> = 3.2 Hz, 1H), 6.51 (s, 2H), 6.40–6.30 (m, 1H), 4.46 (q, <i>J</i> = 7.1 Hz, 2H), 2.73 (s, 3H), 1.45 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 164.2, 156.3, 156.1, 145.9, 144.1, 143.0, 140.5, 137.5, 136.0, 126.6, 116.9, 112.6, 110.8, 99.4, 63.0, 40.9, 18.3, 14.2. LRMS <i>m</i>/<i>z</i>: 379.1 [M + H]<sup>+</sup>. HPLC purity 99.0%, <i>t</i><sub>r</sub> = 2.05 min.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Ethyl 1-Benzyl-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7h</b>)</h4><div class="NLM_p last">Yellow solid (165 mg, 85% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.64 (s, 1H), 8.87 (d, <i>J</i> = 7.0 Hz, 1H), 8.71 (s, 1H), 7.63 (d, <i>J</i> = 6.6 Hz, 1H), 7.46 (d, <i>J</i> = 7.3 Hz, 2H), 7.33–7.26 (m, 3H), 7.23 (d, <i>J</i> = 7.2 Hz, 1H), 7.01 (t, <i>J</i> = 7.0 Hz, 1H), 5.83 (s, 2H), 4.37 (q, <i>J</i> = 7.1 Hz, 2H), 2.49 (s, 3H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.0, 156.0, 155.8, 154.0, 150.4, 137.5, 137.1, 136.9, 134.0, 127.8, 127.3, 126.4, 125.7, 113.8, 113.6, 94.3, 60.7, 44.7, 17.8, 13.9. LRMS <i>m</i>/<i>z</i>: 389.0 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.17 min.</div></div><div id="sec4_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Ethyl 2-Imino-10-methyl-5-oxo-1-phenyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7i</b>)</h4><div class="NLM_p last">Yellow solid (166 mg, 89% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.46 (s, 1H), 8.86 (d, <i>J</i> = 7.1 Hz, 1H), 8.77 (s, 1H), 7.57 (t, <i>J</i> = 7.8 Hz, 3H), 7.47 (t, <i>J</i> = 7.4 Hz, 1H), 7.29 (s, 1H), 7.27 (s, 1H), 6.99 (t, <i>J</i> = 7.0 Hz, 1H), 4.38 (q, <i>J</i> = 7.1 Hz, 2H), 2.04 (s, 3H), 1.43 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 164.8, 157.4, 155.8, 155.0, 150.2, 138.0, 137.5, 136.7, 134.2, 128.9, 128.3, 127.6, 125.5, 114.3, 113.9, 94.4, 60.7, 16.7, 13.9. LRMS <i>m</i>/<i>z</i>: 375.1 [M + H]<sup>+</sup>. HPLC purity 98.0%, <i>t</i><sub>r</sub> = 1.97 min.</div></div><div id="sec4_7_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Ethyl 1-(4-Fluorophenyl)-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylate (<b>7j</b>)</h4><div class="NLM_p last">Yellow solid (176 mg, 90% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.87 (d, <i>J</i> = 7.0 Hz, 1H), 8.77 (s, 1H), 7.59 (d, <i>J</i> = 6.8 Hz, 1H), 7.26 (d, <i>J</i> = 6.6 Hz, 4H), 7.02 (t, <i>J</i> = 7.0 Hz, 1H), 4.39 (q, <i>J</i> = 7.1 Hz, 2H), 2.08 (s, 3H), 1.43 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 164.7, 162.6, 160.7, 157.4, 155.7, 154.9, 150.3, 137.7, 137.1, 134.2, 133.6, 130.1, 129.8, 125.6, 116.1, 115.9, 114.1, 94.6, 60.8, 16.8, 13.9. LRMS <i>m</i>/<i>z</i>: 393.1 [M + H]<sup>+</sup>. HPLC purity 99.0%, <i>t</i><sub>r</sub> = 1.99 min.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> General Synthetic Procedure for Compound <b>8</b></h3><div class="NLM_p">To a solution of compound <b>7</b> (0.15 mmol, 1.0 equiv) in tetrahydrofuran (THF)/EtOH (6 mL, 1/2 v/v) was added a solution of LiOH (18 mg, 0.75 mmol, 5.0 equiv) in water (1 mL) at rt. The mixture was stirred at 40 °C overnight. After the complete consumption of compound <b>7</b> monitored in TLC, the mixture was condensed under vacuum. Then, the residue was dissolved in H<sub>2</sub>O. After the solution was adjusted to pH 4.0, a light-yellow solid appeared. A simple filtration provided compound <b>8</b> as a solid.</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 2-Imino-1-isopropyl-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8a</b>)</h4><div class="NLM_p last">Yellow solid (41 mg, 88% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.48 (s, 1H), 8.91 (d, <i>J</i> = 10.3 Hz, 2H), 8.05 (d, <i>J</i> = 6.1 Hz, 1H), 7.37 (t, <i>J</i> = 7.0 Hz, 1H), 2.60 (s, 3H), 1.77 (d, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.4, 156.7, 149.0, 139.1, 138.9, 133.6, 126.1, 117.8, 116.0, 100.0, 17.9. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.46 min.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 2-Imino-10-methyl-5-oxo-1-propyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8b</b>)</h4><div class="NLM_p last">Yellow solid (40 mg, 86% yield). <sup>1</sup>H NMR (500 MHz, TFA-<i>d</i>): δ 9.64 (s, 1H), 9.13 (d, <i>J</i> = 6.7 Hz, 2H), 8.16 (d, <i>J</i> = 6.7 Hz, 2H), 7.49 (t, <i>J</i> = 6.9 Hz, 1H), 4.82 (s, 3H), 2.78 (s, 3H), 2.04 (d, <i>J</i> = 7.4 Hz, 3H), 1.20 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 159.1, 148.9, 147.0, 142.1, 137.6, 132.7, 126.9, 117.1, 108.7, 106.4, 105.5, 104.1, 103.3, 96.6, 91.2, 37.2, 9.5. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.48 min.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 2-Imino-10-methyl-5-oxo-1-phenethyl-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8c</b>)</h4><div class="NLM_p last">Yellow solid (47 mg, 83% yield). <sup>1</sup>H NMR (400 MHz, TFA-<i>d</i>): δ 11.65 (s, 1H), 9.95 (s, 1H), 9.00 (s, 1H), 8.96 (d, <i>J</i> = 7.0 Hz, 1H), 8.16 (d, <i>J</i> = 6.8 Hz, 1H), 7.46 (t, <i>J</i> = 7.1 Hz, 1H), 7.39 (d, <i>J</i> = 7.6 Hz, 2H), 7.31 (t, <i>J</i> = 7.4 Hz, 2H), 7.21 (t, <i>J</i> = 7.3 Hz, 1H), 4.91 (s, 2H), 3.07 (t, <i>J</i> = 7.6 Hz, 3H), 2.57 (s, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 165.7, 156.5, 150.6, 150.3, 141.8, 140.6, 137.3, 133.9, 129.0, 128.4, 126.7, 116.8, 99.7, 44.8, 31.5, 17.4. HPLC purity 96.2%, <i>t</i><sub>r</sub> = 1.59 min. LRMS <i>m</i>/<i>z</i>: 375.2 [M + H]<sup>+</sup>. HPLC purity 95.8%, <i>t</i><sub>r</sub> = 1.57 min.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 1-(4-Chlorophenethyl)-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8d</b>)</h4><div class="NLM_p last">Yellow solid (52 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, TFA-<i>d</i>): δ 9.62 (s, 1H), 9.10 (d, <i>J</i> = 6.8 Hz, 1H), 8.15 (d, <i>J</i> = 7.0 Hz, 1H), 7.48 (t, <i>J</i> = 7.0 Hz, 1H), 7.18 (q, <i>J</i> = 8.3 Hz, 5H), 5.16 (s, 2H), 3.30 (t, <i>J</i> = 6.5 Hz, 2H), 2.74 (s, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 164.3, 154.0, 147.4, 147.2, 142.9, 138.1, 131.9, 129.9, 129.6, 125.4, 124.6, 122.5, 101.8, 96.3, 41.6, 26.8, 12.3. LRMS <i>m</i>/<i>z</i>: 409.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 2-Imino-10-methyl-1-(4-methylphenethyl)-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8e</b>)</h4><div class="NLM_p last">Yellow solid (50 mg, 85% yield). <sup>1</sup>H NMR (400 MHz, TFA-<i>d</i>): δ 9.57 (s, 1H), 9.05 (d, <i>J</i> = 6.8 Hz, 1H), 8.10 (d, <i>J</i> = 6.8 Hz, 1H), 7.43 (t, <i>J</i> = 6.8 Hz, 1H), 7.01 (s, 5H), 5.06 (s, 2H), 3.24 (s, 2H), 2.72 (s, 3H), 2.15 (s, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 164.3, 154.0, 152.5, 147.4, 147.2, 142.9, 137.9, 134.0, 132.1, 128.3, 125.3, 123.8, 122.3, 114.0, 101.7, 96.4, 42.4, 27.2, 14.7, 12.3. LRMS <i>m</i>/<i>z</i>: 389.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 2-Imino-10-methyl-5-oxo-1-(2-(thiophen-2-yl)ethyl)-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8f</b>)</h4><div class="NLM_p last">Yellow solid (50 mg, 88% yield). <sup>1</sup>H NMR (400 MHz, TFA-<i>d</i>): δ 9.60 (s, 1H), 9.09 (d, <i>J</i> = 6.9 Hz, 1H), 8.12 (d, <i>J</i> = 6.7 Hz, 1H), 7.46 (t, <i>J</i> = 7.0 Hz, 1H), 6.85 (s, 1H), 5.11 (s, 2H), 3.55 (d, <i>J</i> = 6.0 Hz, 2H), 2.75 (s, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 164.3, 154.0, 152.7, 147.4, 143.1, 138.1, 132.7, 132.2, 122.9, 122.4, 114.1, 101.7, 96.4, 42.4, 21.7, 12.4. LRMS <i>m</i>/<i>z</i>: 381.1 [M + H]<sup>+</sup>. HPLC purity 95.4%, <i>t</i><sub>r</sub> = 1.49 min.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> 1-(Furan-2-ylmethyl)-2-imino-10-methyl-5-oxo-1,5-dihydro-2<i>H</i>-dipyrido[1,2-<i>a</i>:2′,3′-<i>d</i>]pyrimidine-3-carboxylic Acid (<b>8g</b>)</h4><div class="NLM_p last">Yellow solid (44 mg, 83% yield). <sup>1</sup>H NMR (400 MHz, TFA-<i>d</i>): δ 9.58 (s, 1H), 9.10 (d, <i>J</i> = 6.8 Hz, 1H), 8.16 (d, <i>J</i> = 6.7 Hz, 1H), 7.45 (dd, <i>J</i> = 25.8, 18.7 Hz, 2H), 7.38 (s, 1H), 6.68 (d, <i>J</i> = 2.6 Hz, 1H), 6.39 (s, 1H), 6.04 (s, 2H), 2.81 (s, 3H). <sup>13</sup>C NMR (126 MHz, TFA-<i>d</i>): δ 164.3, 153.9, 153.1, 147.2, 144.0, 143.1, 140.6, 140.2, 139.7, 138.3, 132.1, 128.9, 123.4, 122.5, 114.1, 101.9, 96.4, 36.9, 12.2, 11.6. LRMS <i>m</i>/<i>z</i>: 351.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> General Synthetic Procedure for Compound <b>10</b></h3><div class="NLM_p last">A mixture of aldehyde <b>9</b> (1.2 g, 10 mmol) and diethyl malonate (2.4 g, 15 mmol) was stirred for 24 h in the presence of a catalytic amount of piperidine. After completion of the reaction, ethyl acetate was added to the mixture. Then, a simple filtration provided compound <b>10</b> as a yellow solid, which was directly subjected to the next step without further purification.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> General Synthetic Procedure for Compound <b>11</b></h3><div class="NLM_p">Compound <b>10</b> (3 mmol) and K<sub>2</sub>CO<sub>3</sub> (6 mmol) were dissolved in DMF (15 mL) at rt, and then alkyl halide (4.5 mmol) was added. The resulting mixture was stirred at rt overnight. After completion of the reaction, compound <b>11</b> was purified by silica chromatography as a white solid.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> Ethyl 2-Oxo-1-phenethyl-1,2-dihydro-1,8-naphthyridine-3-carboxylate (<b>11a</b>)</h4><div class="NLM_p last">White solid (41 mg, 64% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.73 (dd, <i>J</i> = 4.7, 1.7 Hz, 1H), 8.40 (s, 1H), 8.00 (dd, <i>J</i> = 7.7, 1.6 Hz, 1H), 7.43 (d, <i>J</i> = 7.3 Hz, 2H), 7.34 (t, <i>J</i> = 7.5 Hz, 2H), 7.25 (dd, <i>J</i> = 7.7, 4.7 Hz, 2H), 4.83–4.72 (m, 2H), 4.47 (q, <i>J</i> = 7.1 Hz, 2H), 3.09–2.99 (m, 2H), 1.46 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 164.6, 159.3, 152.4, 150.4, 141.8, 139.0, 138.0, 129.1, 128.5, 126.4, 124.3, 118.5, 114.1, 61.8, 43.2, 33.9, 14.4. LRMS <i>m</i>/<i>z</i>: 323.0 [M + H]<sup>+</sup>. HPLC purity 95.8%, <i>t</i><sub>r</sub> = 2.05 min.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> Ethyl 2-Oxo-1-phenethyl-1,2-dihydro-1,6-naphthyridine-3-carboxylate (<b>11b</b>)</h4><div class="NLM_p last">White solid (51 mg, 79% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.89 (s, 1H), 8.63 (d, <i>J</i> = 6.1 Hz, 1H), 8.49 (s, 1H), 7.36–7.30 (m, 4H), 7.28–7.24 (m, 1H), 7.14 (d, <i>J</i> = 6.1 Hz, 1H), 4.50–4.39 (m, 4H), 3.08–3.01 (m, 2H), 1.46 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 164.0, 158.5, 152.3, 151.5, 145.3, 141.8, 137.6, 128.8, 127.0, 124.2, 115.2, 108.1, 61.9, 44.5, 33.5, 14.3. LRMS <i>m</i>/<i>z</i>: 323.0 [M + H]<sup>+</sup>. HPLC purity 95.4%, <i>t</i><sub>r</sub> = 1.87 min.</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> General Synthetic Procedure for Compound <b>12</b></h3><div class="NLM_p">To a solution of compound <b>11</b> (1.5 mmol, 1.0 equiv) in THF/EtOH (18 mL, 1/2 v/v) was added a solution of LiOH (7.5 mmol, 5.0 equiv) in water (3 mL) at rt. The mixture was stirred at 40 °C overnight. After complete consumption of compound <b>11</b> monitored by TLC, the mixture was condensed under vacuum, and the residue was dissolved in H<sub>2</sub>O. After the pH was adjusted to 4.0, a light-yellow solid appeared. After filtration, compound <b>12</b> was obtained as a white solid.</div><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> 2-Oxo-1-phenethyl-1,2-dihydro-1,8-naphthyridine-3-carboxylic Acid (<b>12a</b>)</h4><div class="NLM_p last">White solid (36 mg, 61% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.21 (s, 1H), 8.98 (s, 1H), 8.89 (dd, <i>J</i> = 4.7, 1.8 Hz, 1H), 8.56 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.54 (dd, <i>J</i> = 7.8, 4.7 Hz, 1H), 7.35–7.26 (m, 4H), 7.26–7.18 (m, 1H), 4.86–4.63 (m, 2H), 3.10–2.86 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.6, 163.6, 154.0, 149.5, 144.6, 140.5, 138.8, 129.1, 128.9, 126.9, 120.6, 119.9, 115.4, 43.1, 33.6. LRMS <i>m</i>/<i>z</i>: 295.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.57 min.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> 2-Oxo-1-phenethyl-1,2-dihydro-1,6-naphthyridine-3-carboxylic Acid (<b>12b</b>)</h4><div class="NLM_p last">White solid (43 mg, 73% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 14.22 (s, 1H), 8.98 (s, 1H), 8.89 (dd, <i>J</i> = 4.6, 1.8 Hz, 1H), 8.56 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.54 (dd, <i>J</i> = 7.8, 4.7 Hz, 1H), 7.35–7.26 (m, 4H), 7.26–7.19 (m, 1H), 4.73 (dd, <i>J</i> = 9.1, 6.8 Hz, 2H), 3.12–2.86 (m, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.6, 163.6, 154.0, 149.5, 144.6, 140.5, 138.8, 129.1, 128.9, 126.9, 120.6, 119.9, 115.4, 43.1, 33.6. LRMS <i>m</i>/<i>z</i>: 295.1 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 1.58 min.</div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> General Synthetic Procedure for Compounds <b>13</b> and <b>14</b></h3><div class="NLM_p">A mixture of compound <b>12</b> (0.2 mmol, 1.0 equiv) and HATU (0.3 mmol, 1.5 equiv) in DMF (4 mL) was stirred at rt for 5 min, and then DIEA (0.6 mmol, 3.0 equiv) and the corresponding amine (0.3 mmol, 1.5 equiv) were added. The resulting solution was stirred at rt overnight. After completion of the reaction, a white precipitate appeared by the addition of an identical amount of water. Target compounds <b>13</b> and <b>14</b> were obtained by a simple filtration.</div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> <i>N</i>-(4-Fluorophenethyl)-2-oxo-1-phenethyl-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>13a</b>)</h4><div class="NLM_p last">White solid (48 mg, 58% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.71 (t, <i>J</i> = 5.7 Hz, 1H), 8.90 (s, 1H), 8.82 (d, <i>J</i> = 4.7 Hz, 1H), 8.51 (d, <i>J</i> = 7.7 Hz, 1H), 7.46 (dd, <i>J</i> = 7.8, 4.7 Hz, 1H), 7.35–7.26 (m, 3H), 7.23 (t, <i>J</i> = 7.0 Hz, 1H), 7.14 (t, <i>J</i> = 8.8 Hz, 1H), 4.75–4.63 (m, 2H), 3.60 (dd, <i>J</i> = 13.5, 6.8 Hz, 2H), 2.99–2.90 (m, 2H), 2.87 (t, <i>J</i> = 7.4 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.6, 162.1, 160.6, 153.0, 149.5, 142.5, 139.9, 139.1, 135.9, 131.0, 129.1, 129.0, 126.9, 122.7, 119.9, 115.6, 115.5, 115.0, 42.8, 41.0, 34.7, 33.8. LRMS <i>m</i>/<i>z</i>: 404.2 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.22 min.</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> <i>N</i>-(2-(Diethylamino)ethyl)-2-oxo-1-phenethyl-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>13b</b>)</h4><div class="NLM_p last">White solid (46 mg, 59% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.74 (t, <i>J</i> = 5.4 Hz, 1H), 8.91 (s, 1H), 8.81 (dd, <i>J</i> = 4.6, 1.7 Hz, 1H), 8.50 (dd, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.45 (dd, <i>J</i> = 7.8, 4.7 Hz, 1H), 7.35–7.26 (m, 4H), 7.22 (t, <i>J</i> = 6.8 Hz, 1H), 4.86–4.63 (m, 2H), 3.42 (q, <i>J</i> = 6.2 Hz, 2H), 3.03–2.85 (m, 2H), 2.63–2.51 (m, 7H), 1.00 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.5, 162.0, 152.9, 149.6, 142.4, 139.9, 139.1, 129.1, 128.9, 126.9, 122.9, 119.8, 115.0, 51.7, 47.1, 42.7, 37.9, 33.7. LRMS <i>m</i>/<i>z</i>: 393.3 [M + H]<sup>+</sup>. HPLC purity 100%, <i>t</i><sub>r</sub> = 2.20 min.</div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> <i>N</i>-(4-Fluorophenethyl)-2-oxo-1-phenethyl-1,2-dihydro-1,6-naphthyridine-3-carboxamide (<b>14a</b>)</h4><div class="NLM_p last">White solid (69 mg, 85% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.57 (s, 1H), 9.16 (s, 1H), 8.94 (s, 1H), 8.66 (d, <i>J</i> = 6.0 Hz, 1H), 7.65 (d, <i>J</i> = 6.1 Hz, 1H), 7.32 (s, 6H), 7.24 (d, <i>J</i> = 5.9 Hz, 1H), 7.13 (t, <i>J</i> = 8.8 Hz, 2H), 4.54–4.41 (m, 2H), 3.59 (dd, <i>J</i> = 13.4, 6.8 Hz, 2H), 2.99–2.90 (m, 2H), 2.86 (t, <i>J</i> = 7.2 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.4, 161.0, 160.4, 153.3, 151.6, 144.5, 142.2, 138.4, 135.9, 130.9, 129.4, 128.9, 127.1, 123.1, 115.8, 115.6, 115.4, 109.7, 44.1, 41.00, 34.7, 33.3. LRMS <i>m</i>/<i>z</i>: 416.1 [M + H]<sup>+</sup>. HPLC purity 97.5%, <i>t</i><sub>r</sub> = 2.07 min.</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> <i>N</i>-(2-(Diethylamino)ethyl)-2-oxo-1-phenethyl-1,2-dihydro-1,6-naphthyridine-3-carboxamide (<b>14b</b>)</h4><div class="NLM_p last">White solid (71 mg, 90% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (t, <i>J</i> = 5.3 Hz, 1H), 9.16 (s, 1H), 8.95 (s, 1H), 8.65 (d, <i>J</i> = 6.1 Hz, 1H), 7.64 (d, <i>J</i> = 6.2 Hz, 1H), 7.37–7.27 (m, 4H), 7.22 (dd, <i>J</i> = 7.9, 4.9 Hz, 1H), 4.57–4.44 (m, 2H), 3.47–3.37 (m, 2H), 2.99–2.90 (m, 2H), 2.56 (dd, <i>J</i> = 13.5, 6.8 Hz, 4H), 0.99 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.3, 161.7, 153.2, 151.5, 144.5, 142.1, 138.4, 129.3, 128.9, 127.0, 123.2, 115.8, 109.7, 51.7, 47.1, 44.0, 37.9, 33.3, 12.4. LRMS <i>m</i>/<i>z</i>: 393.2 [M + H]<sup>+</sup>. HPLC purity 98.8%, <i>t</i><sub>r</sub> = 1.83 min.</div></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> Protein Expression and Purification</h3><div class="NLM_p">The DNA sequence encoding SPOP<sup>MATH</sup> was subcloned into the BamHI and XhoI restriction sites of the modified pGEX-6P-1 with dual tags of GST and His at the N-terminus. The DNA sequence encoding SPOP (residues 23–337 or full length) was cloned into the <i>BamHI</i> and <i>XhoI</i> restriction sites of the modified pET-23a. The <i>Escherichia coli</i> BL21 (DE3) strain was transformed with the recombinant plasmids and grown in Lysogeny Broth at 37 °C in the presence of 100 μg/mL ampicillin. When the <i>A</i><sub>600</sub> reached 0.6–0.8, protein expression was induced by isopropyl β-d-1-thiogalactopyranoside with a final concentration of 0.5 mM for an additional 16–18 h at 16 °C. The cells were harvested by centrifugation at 5000 g for 10 min and stored at −80 °C.</div><div class="NLM_p last">For protein purification, the cells were suspended in 40 mL of lysis buffer (20 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1 mM DTT), lysed by sonification, and centrifuged at 12,000<i>g</i> for 25 min at 4 °C. The supernatant was passed through 0.45 μm syringe filters and loaded onto the HisTrap HP column (5 mL, GE Healthcare), washed with seven column volumes of 30 mM imidazole, and finally eluted with 300 mM imidazole. Proteins were concentrated via centrifugal filtration with a cutoff of 10 kDa molecular weight. Further purification was conducted through gel filtration (HiLoad 16/60 Superdex 200, GE Healthcare). Fractions were collected and checked by 12% sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE). The SPOP protein was stored at −80 °C.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i131" class="anchor-spacer"></div><h3 class="article-section__title" id="_i131"> nanoDSF Assay</h3><div class="NLM_p last">To a solution of 1 mg/mL of SPOP<sup>MATH</sup> protein in buffer (150 mM NaCl, 100 mM sodium citrate, pH 5.1) or 1 mg/mL of SPOP<sup>23–337</sup> protein in buffer (150 mM NaCl, 20 mM Tris-HCl, pH 7.5) was added a solution of 0.5 μL compound in DMSO to the appropriate concentration. After incubation for 15 min at rt, the mixture was centrifuged for 2 min at 1000<i>g</i> in order to remove any precipitates. Approximately 10 μL of the supernatant fraction was loaded to each capillary, which was then placed on the holder in the sample rack. The thermal denaturation curves were determined by the measurement of the protein intrinsic fluorescence on label-free native nanoDSF (NanoTemper, Prometheus NT.48). The temperature was increased from 20 to 90 °C at a rate of 2°·min<sup>–1</sup>. Tryptophan residues in SPOP were excited at 280 nm, and the fluorescence intensity was recorded at 330 and 350 nm, respectively.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i132" class="anchor-spacer"></div><h3 class="article-section__title" id="_i132"> Conventional DSF Assay</h3><div class="NLM_p last">The conventional DSF assay was performed on the Bio-Rad 96 system (Bio-Red, CFx 96 Touch) using SYPRO orange (Invitrogen) as a dye. A mixture containing 10 μM SPOP<sup>MATH</sup>, 5× SYPRO dye, and different concentrations of the compound was heated from 25 to 95 °C at a 1% ramp rate on the Bio-Rad 96 system (Bio-Red, CFx 96 Touch). The fluorescence intensity was detected at the wavelength of 492 nm for excitation and 610 nm for emission. The melting temperature (<i>T</i><sub>m</sub>) of the protein was calculated by Bio-Rad CFx Manager 3.1 (Bio-Rad) and was further normalized by Origen 8.0.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133"> NMR Titration</h3><div class="NLM_p last">Phosphate-buffered solution (PBS) containing 20 mM sodium phosphate (pH 7.4), 100 mM NaCl, and 5% DMSO was used for NMR. All NMR spectra were acquired on a Bruker AVANCE III-600 MHz spectrometer equipped with a cryogenically cooled probe (Bruker BioSpin, Germany) at 25 °C. Samples contained 200 μM compound and SPOP<sup>MATH</sup> or SPOP<sup>1–374</sup> protein at varying concentrations of 0, 5, 10, and 20 or 0, 1, 2, 4, and 8 μM.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i134" class="anchor-spacer"></div><h3 class="article-section__title" id="_i134"> Surface Plasmon Resonance</h3><div class="NLM_p last">The SPR assay was run on a Biacore T200 instrument (GE Healthcare). The running buffer was 1× PBS-P<sup>+</sup> (10 mM PBS-P<sup>+</sup> (pH 7.4), 137 mM NaCl, 2.7 mM KCl, 0.5% (v/v) surfactant P20, 5% DMSO). SPOP<sup>MATH</sup> or SPOP<sup>1–374</sup> was covalently immobilized onto sensor CM5 chips by a standard amine-coupling procedure in 10 mM of sodium acetate (pH 4.5). A series of compounds were diluted and injected onto a sensor chip at a flow rate of 30 μL/min for 120 s (contact phase), followed by 120 s of buffer flow (dissociation phase). The <i>K</i><sub>D</sub> value was derived through the use of Biacore T200 Evaluation software Version 1.0 (GE Healthcare) and steady-state analysis of data at equilibrium.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i135" class="anchor-spacer"></div><h3 class="article-section__title" id="_i135"> Cell Cultures</h3><div class="NLM_p last">HEK293T, A498, A549, H1299, U251, and HS766T were purchased from ATCC (Rockville, MD, USA) and maintained in Dulbecco’s modified Eagle’s medium supplemented with 10% FBS (Gibco), 100 units/mL streptomycin, and 100 units/mL penicillin. OS-RC-2 was obtained from the National Platform of Experimental Cell Resources for Sci-Tech (Beijing, China) and maintained in a RPMI1640 medium supplemented with 10% FBS (Gibco), 100 units/mL streptomycin, and 100 units/mL penicillin. The culture medium was changed every 2 days. Cells for the subculture were detached and harvested by 0.25% trypsin and 0.02% ethylenediaminetetraacetic acid (EDTA). All cells were cultured at 37 °C in a humidified, 5% CO<sub>2</sub>-containing atmosphere incubator (Thermo Scientific). All cells were regularly checked to be mycoplasma-free.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i136" class="anchor-spacer"></div><h3 class="article-section__title" id="_i136"> Cell Proliferation Assay</h3><div class="NLM_p last">Cell proliferation was determined by an MTT assay. A498 and OS-RC-2 cells (1000–2200/well) were seeded into 96-well plates and cultured at 37 °C for 12 h, and then the compounds were added. After the cells were cultured for an additional 72 h, 10 μL of the MTT reagent was added to each well. The plates were incubated for 4 h. The supernatants were then aspirated, and the formazan crystals in each well were dissolved in 150 μL of DMSO. Absorbance was measured by a microplate reader at a wavelength of 490 nm. The optical density of formazan formed in untreated cells was taken as 100% proliferation. Luminescence was read by Spark 10M (Tecan).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i137" class="anchor-spacer"></div><h3 class="article-section__title" id="_i137"> Colony Formation Assay</h3><div class="NLM_p last">A498 and OS-RC-2 ccRCC cells were seeded into each well of a six-well plate at a density of 1000 cells/well and incubated for 12 days at 37 °C in 5% CO<sub>2</sub>. The culture medium was refreshed every 3 days. The cells were fixed with 10% acetic acid and 10% methanol for 15 min and then stained with 1% crystal violet for 10 min.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i138" class="anchor-spacer"></div><h3 class="article-section__title" id="_i138"> Real-Time Quantitative PCR</h3><div class="NLM_p last">A498 and OS-RC-2 cells were treated with multiple doses of <b>6lc</b>. Total RNA was extracted with Trizol Reagent Kit (Sigma). Then, equivalent amounts of RNA were reversed-transcribed into cDNA using the HiScript III RT SuperMix for qPCR Kit (Vazyme, R323-01). RT-qPCRs were performed using the ChamQ Universal SYBR qPCR Master Mix (Vazyme, Q711-02/03) in the CFX96 Touch Real-Time PCR Detection System (Bio-Rad). Gene expression was calculated through the use of the comparative ΔΔ<i>C</i><sub>T</sub> method with the GAPDH for normalization. The primers used are as follows: DUSP7, tctgactgctccgacggcga (forward), ccttggcgcagccgaggtag (reverse); PTEN, ttgagccgctgtgaggcgag (forward), agccgaggaagaggctgcac (reverse); GAPDH, cgaccactttgtcaagctca (forward), ccctgttgctgtagccaaat (reverse).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i139" class="anchor-spacer"></div><h3 class="article-section__title" id="_i139"> Antibodies</h3><div class="NLM_p last">SPOP (Abcam, ab137537), Flag (Sigma, F1804), Myc (Sigma, M4439), HA (Sigma, H9658), GAPDH (Sigma, G8795), PTEN (Cell Signaling, #9559), DUSP7 (ABGENT, AP8450a), AKT (Cell Signaling, #9272), <i>p</i>-AKT (Thr308) (Cell signaling, #4056), ERK1/2 (Cell Signaling, # 9102), and p-ERK1/2 (Thr202/Tyr204) (Cell Signaling, #4376).</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i140" class="anchor-spacer"></div><h3 class="article-section__title" id="_i140"> Western Blot</h3><div class="NLM_p last">Cells were treated with the SPOP inhibitor at different concentrations for 12 h. The cells were harvested with RIPA lysis buffer (Beyotime, P0013C) and centrifuged at 14,000<i>g</i> for 20 min at 4 °C. The supernatants were centrifuged, and protein concentration was quantified with the bicinchoninic acid (BCA) assay (Thermo Fisher, 23225). The sample was diluted with 5× loading butter and was heated for 5 min at 100 °C and then analyzed on 12% SDS-PAGE. The proteins in SDS-PAGE were transferred onto the nitrocellulose membrane (Millipore, USA), which was blocked with 5% skim milk for 1 h at rt. The membrane was incubated with primary antibody at 4 °C overnight. Horseradish peroxidase-labeled goat anti-mouse or anti-rabbit IgG was used as a secondary antibody. The membrane was visualized using the ECL detection system (GE Healthcare Bioscience, USA). Equivalent amounts of total protein were monitored by GAPDH as the loading control.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i141" class="anchor-spacer"></div><h3 class="article-section__title" id="_i141"> Coimmunoprecipitation</h3><div class="NLM_p last">HEK293T cells were transiently transfected with the indicated plasmids, SPOP-p3×Flag-CMV10 and PTEN-pcDNA4-myc-His, respectively. After 36 h transfection, cells were treated with the SPOP inhibitor at different concentrations for 12 h. The cells were harvested with lysis buffer [50 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1% Triton X 100, 1 mM EDTA, 1 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, and 1× proteinase inhibitor cocktail (Sigma, P8340)] and centrifuged at 14,000<i>g</i> for 20 min at 4 °C. Supernatants were centrifuged, and protein concentration was quantified with the BCA assay (Thermo Fisher, 23225). For immunoprecipitation, the supernatants were immunoprecipitated with anti-Myc-conjugated agarose beads (Sigma, rabbit antibody) for 2 h at 4 °C. Then, the beads were washed 7 times with lysis buffer, and then the sample was diluted with 5× loading buffer. The samples were heated for 5 min at 100 °C and then subjected to western blot analysis. For whole cell lysates, the supernatants were diluted with 5× loading buffer, heated for 5 min at 100 °C, and then subjected to western blot analysis.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i142" class="anchor-spacer"></div><h3 class="article-section__title" id="_i142"> Ubiquitination Assay</h3><div class="NLM_p last">HEK293T cells were transfected in 60 mm dishes with 1 μg each of Myc-PTEN, Flag-SPOP-cyto, and HA-Ub. After 36 h culture, cells were treated with <b>6lc</b> for 8 h and then with 10 μM MG132 (Calbiochem) for another 4 h before harvesting. Cells were harvested in a denaturing buffer (1% SDS, 50 mM Tris-HCl (pH 7.5), 0.5 mM EDTA, 1 mM DTT). The lysate was incubated for 5 min at 100 °C immediately, sonicated, and then diluted 10 times with RIPA lysis buffer. For immunoprecipitation, the lysates were subjected to Co-IP using anti-Myc-conjugated agarose beads as described in the Co-IP assay. For whole cell lysates, the lysates were diluted with 5× loading buffer, heated for 5 min at 100 °C, and then subjected to western blot analysis.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i143" class="anchor-spacer"></div><h3 class="article-section__title" id="_i143"> DARTS Assay</h3><div class="NLM_p last">The DARTS assay was performed on the basis of the reported protocol.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Briefly, 1 × 10<sup>6</sup> A498 or OS-RC-2 cells were collected and washed with ice-cold PBS. Then, the cells were lysed with lysis buffer using the M-PER reagent (Thermo Fisher, 78501) with a protease inhibitor cocktail (Thermo Fisher, 78430) and a phosphatase inhibitor cocktail (Thermo Fisher, 78420). Cell lysis was incubated on ice for 10 min and centrifuged for 10 min at 18,000<i>g</i> at 4 °C. The supernatant was removed and transferred into TNC buffer [50 mM Tris-HCl (pH 8.0), 50 mM NaCl, and 10 mM CaCl<sub>2</sub>]. Protein concentration was quantified with the BCA assay (Thermo Fisher, 23225). Cell suspension was distributed into different 1.5 mL PCR tubes (120 μg), incubated with diffident concentrations of the compound (0, 1, 5, 10, and 50 μM) for 20 min at rt, and then subsequently digested with pronase (1:1000 for A498 and 1:2000 for OS-RC-2) for 10 min at rt. The reaction was stopped by adding a protease inhibitor cocktail for 10 min on ice. Finally, the supernatants were diluted with 5× loading buffer, heated for 5 min at 70 °C, and then subjected to western blot analysis.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i144"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00161" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34677" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34677" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00161?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00161</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Properties of SPOP inhibitors and synthetic methods and analytical data for target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf">jm0c00161_si_001.pdf (4.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_002.csv">jm0c00161_si_002.csv (5.23 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00161" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13273" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13273" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cai-Guang Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, UCAS, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5778-4422" title="Orcid link">http://orcid.org/0000-0001-5778-4422</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2d544c434a4e4a6d5e444040034c4e034e43"><span class="__cf_email__" data-cfemail="add4ccc3cacecaeddec4c0c083ccce83cec3">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ze Dong</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhen Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, Nanjing 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhong-Qiang Guo</span> - <span class="hlFld-Affiliation affiliation">Department
of Urology, Zhongnan Hospital of Wuhan University, Hubei 430071, China</span>; 
    <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Genome Sciences and Information, Beijing Institute
of Genomics, Chinese Academy of Sciences, Beijing 100101, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shouzhe Gong</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tao Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiang Liu</span> - <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Genome Sciences and Information, Beijing Institute
of Genomics, Chinese Academy of Sciences, Beijing 100101, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Luo</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, UCAS, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3864-8382" title="Orcid link">http://orcid.org/0000-0003-3864-8382</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, UCAS, Hangzhou 310024, China</span>; 
    <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, ShanghaiTec University, Shanghai 201210, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Z.D. and Z.W. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i146">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by grants from the National Natural Science Foundation of China (21807103 to Z.D. and 21725801 to C.-G.Y.) and the National Science & Technology Major Project “Key New Drug Creation and Manufacturing Program”, China (2018ZX09711002 to Z.D.). We thank S. F. Reichard, MA for editing the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ccRCCs</td><td class="NLM_def"><p class="first last">clear-cell renal cell carcinomas</p></td></tr><tr><td class="NLM_term">CPMG</td><td class="NLM_def"><p class="first last">Carr–Purcell–Meiboom–Gill</p></td></tr><tr><td class="NLM_term">Co-IP</td><td class="NLM_def"><p class="first last">coimmunoprecipitation</p></td></tr><tr><td class="NLM_term">DARTS</td><td class="NLM_def"><p class="first last">drug affinity responsive target stability</p></td></tr><tr><td class="NLM_term">DUSP7</td><td class="NLM_def"><p class="first last">dual-specificity phosphatase 7</p></td></tr><tr><td class="NLM_term">mTOR</td><td class="NLM_def"><p class="first last">mammalian target of rapamycin</p></td></tr><tr><td class="NLM_term">nanoDSF</td><td class="NLM_def"><p class="first last">nano differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compound</p></td></tr><tr><td class="NLM_term">PDGF</td><td class="NLM_def"><p class="first last">platelet-derived growth factor</p></td></tr><tr><td class="NLM_term">PTEN</td><td class="NLM_def"><p class="first last">phosphatase-and-tensin homologue</p></td></tr><tr><td class="NLM_term">RCC</td><td class="NLM_def"><p class="first last">renal cell carcinomas</p></td></tr><tr><td class="NLM_term">RT-qPCR</td><td class="NLM_def"><p class="first last">real-time quantitative PCR</p></td></tr><tr><td class="NLM_term">SPOP</td><td class="NLM_def"><p class="first last">speckle-type POZ protein</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation transfer difference</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i148">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60751" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60751" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricketts, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crooks, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span> <span> </span><span class="NLM_article-title">Targeting HIF2α in clear-cell renal cell carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.09.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccell.2016.09.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27728802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=515-517&author=C.+J.+Rickettsauthor=D.+R.+Crooksauthor=W.+M.+Linehan&title=Targeting+HIF2%CE%B1+in+clear-cell+renal+cell+carcinoma&doi=10.1016%2Fj.ccell.2016.09.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting HIF2α in Clear-Cell Renal Cell Carcinoma</span></div><div class="casAuthors">Ricketts, Christopher J.; Crooks, Daniel R.; Linehan, W. Marston</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">515-517</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In two recent studies, the HIF2α antagonist, PT2399, decreased HIF2α-dependent transcription and tumor growth in selected VHL-deficient clear-cell renal cell carcinoma (ccRCC) models.  These studies validate HIF2α as a therapeutic target in ccRCC, reveal variable sensitivity to HIF2α antagonism, and provide the foundation for predictive biomarker-driven clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2XxxvzJRkQbVg90H21EOLACvtfcHk0lgdMWfgzSIFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtLfM&md5=e77f71e456917b156a2d47e038c069b4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.09.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.09.016%26sid%3Dliteratum%253Aachs%26aulast%3DRicketts%26aufirst%3DC.%2BJ.%26aulast%3DCrooks%26aufirst%3DD.%2BR.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26atitle%3DTargeting%2520HIF2%25CE%25B1%2520in%2520clear-cell%2520renal%2520cell%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D515%26epage%3D517%26doi%3D10.1016%2Fj.ccell.2016.09.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rini, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escudier, B.</span></span> <span> </span><span class="NLM_article-title">Renal cell carcinoma</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1132</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(09)60229-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2FS0140-6736%2809%2960229-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19269025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslOltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=1119-1132&author=B.+I.+Riniauthor=S.+C.+Campbellauthor=B.+Escudier&title=Renal+cell+carcinoma&doi=10.1016%2Fs0140-6736%2809%2960229-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Renal cell carcinoma</span></div><div class="casAuthors">Rini, Brian I.; Campbell, Steven C.; Escudier, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">9669</span>),
    <span class="NLM_cas:pages">1119-1132</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Considerable progress has been made in the treatment of patients with renal cell carcinoma, with innovative surgical and systemic strategies revolutionizing the management of this disease.  In localized disease, partial nephrectomy for small tumors and radical nephrectomy for large tumors continue to be the gold-std. treatments, with emphasis on approaches that have reduced invasiveness and preserve renal function.  Addnl., cytoreductive nephrectomy is often indicated before the start of systemic treatment in patients with metastatic disease as part of integrated management strategy.  The effectiveness of immunotherapy, although previously widely used for treatment of metastatic renal cell carcinoma, is still controversial, and is mainly reserved for patients with good prognostic factors.  Development of treatments that have specific targets in relevant biol. pathways has been the main advance in treatment.  Targeted drugs, including inhibitors of the vascular endothelial growth factor and mammalian target of rapamycin pathways, have shown robust effectiveness and offer new therapeutic options for the patients with metastatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2JoMfrWyvd7Vg90H21EOLACvtfcHk0lgdMWfgzSIFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslOltrs%253D&md5=e1239eb0ddaeb1cac5ad986daaf381b7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2809%2960229-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252809%252960229-4%26sid%3Dliteratum%253Aachs%26aulast%3DRini%26aufirst%3DB.%2BI.%26aulast%3DCampbell%26aufirst%3DS.%2BC.%26aulast%3DEscudier%26aufirst%3DB.%26atitle%3DRenal%2520cell%2520carcinoma%26jtitle%3DLancet%26date%3D2009%26volume%3D373%26spage%3D1119%26epage%3D1132%26doi%3D10.1016%2Fs0140-6736%2809%2960229-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Presta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisholm, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krummen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, N.</span></span> <span> </span><span class="NLM_article-title">Humanization of an anti-vascular endothelial growth factor monoclonal antibody for the therapy of solid tumors and other disorders</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4593</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=9377574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4593&author=L.+G.+Prestaauthor=H.+Chenauthor=S.+J.+O%E2%80%99Connorauthor=V.+Chisholmauthor=Y.+G.+Mengauthor=L.+Krummenauthor=M.+Winklerauthor=N.+Ferrara&title=Humanization+of+an+anti-vascular+endothelial+growth+factor+monoclonal+antibody+for+the+therapy+of+solid+tumors+and+other+disorders"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Humanization of an anti-vascular endothelial growth factor monoclonal antibody for the therapy of solid tumors and other disorders</span></div><div class="casAuthors">Presta, Leonard G.; Chen, Helen; O'connor, Shane J.; Chisholm, Vanessa; Meng, Y. Gloria; Krummen, Lynne; Winkler, Marjorie; Ferrara, Napoleone</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4593-4599</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is a major mediator of angiogenesis assocd. with tumors and other pathol. conditions, including proliferative diabetic retinopathy and age-related macular degeneration.  The murine anti-human VEGF monoclonal antibody (muMAb VEGF) A.4.6.1 has been shown to potently suppress angiogenesis and growth in a variety of human tumor cells lines transplanted in nude mice and also to inhibit neovascularization in a primate model of ischemic retinal disease.  In this report, we describe the humanization of muMAb VEGF A.4.6.1. by site-directed mutagenesis of a human framework.  Not only the residues involved in the six complementarity-detg. regions but also several framework residues were changed from human to murine.  Humanized anti-VEGF F(ab) and IgG1 variants bind VEGF with affinity very similar to that of the original murine antibody.  Furthermore, recombinant humanized MAb VEGF inhibits VEGF-induced proliferation of endothelial cells in vitro and tumor growth in vivo with potency and efficacy very similar to those of muMAb VEGF A.4.6.1.  Therefore, recombinant humanized MAb VEGF is suitable to test the hypothesis that inhibition of VEGF-induced angiogenesis is a valid strategy for the treatment of solid tumors and other disorders in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooXq2l9auk17Vg90H21EOLACvtfcHk0lj2l3zSFIl4rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKqtLg%253D&md5=9fab64ed63a178629d4b0c45eabc05ff</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPresta%26aufirst%3DL.%2BG.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DS.%2BJ.%26aulast%3DChisholm%26aufirst%3DV.%26aulast%3DMeng%26aufirst%3DY.%2BG.%26aulast%3DKrummen%26aufirst%3DL.%26aulast%3DWinkler%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DN.%26atitle%3DHumanization%2520of%2520an%2520anti-vascular%2520endothelial%2520growth%2520factor%2520monoclonal%2520antibody%2520for%2520the%2520therapy%2520of%2520solid%2520tumors%2520and%2520other%2520disorders%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavia-Jimenez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homayoun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yousuf, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedrosa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zojwalla, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugarolas, J.</span></span> <span> </span><span class="NLM_article-title">Targeting renal cell carcinoma with a HIF-2 antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>539</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1038/nature19796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnature19796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27595394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyis7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=539&publication_year=2016&pages=112-117&author=W.+Chenauthor=H.+Hillauthor=A.+Christieauthor=M.+S.+Kimauthor=E.+Hollomanauthor=A.+Pavia-Jimenezauthor=F.+Homayounauthor=Y.+Maauthor=N.+Patelauthor=P.+Yellauthor=G.+Haoauthor=Q.+Yousufauthor=A.+Joyceauthor=I.+Pedrosaauthor=H.+Geigerauthor=H.+Zhangauthor=J.+Changauthor=K.+H.+Gardnerauthor=R.+K.+Bruickauthor=C.+Reevesauthor=T.+H.+Hwangauthor=K.+Courtneyauthor=E.+Frenkelauthor=X.+Sunauthor=N.+Zojwallaauthor=T.+Wongauthor=J.+P.+Rizziauthor=E.+M.+Wallaceauthor=J.+A.+Joseyauthor=Y.+Xieauthor=X.-J.+Xieauthor=P.+Kapurauthor=R.+M.+McKayauthor=J.+Brugarolas&title=Targeting+renal+cell+carcinoma+with+a+HIF-2+antagonist&doi=10.1038%2Fnature19796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting renal cell carcinoma with a HIF-2 antagonist</span></div><div class="casAuthors">Chen, Wenfang; Hill, Haley; Christie, Alana; Kim, Min Soo; Holloman, Eboni; Pavia-Jimenez, Andrea; Homayoun, Farrah; Ma, Yuanqing; Patel, Nirav; Yell, Paul; Hao, Guiyang; Yousuf, Qurratulain; Joyce, Allison; Pedrosa, Ivan; Geiger, Heather; Zhang, He; Chang, Jenny; Gardner, Kevin H.; Bruick, Richard K.; Reeves, Catherine; Hwang, Tae Hyun; Courtney, Kevin; Frenkel, Eugene; Sun, Xiankai; Zojwalla, Naseem; Wong, Tai; Rizzi, James P.; Wallace, Eli M.; Josey, John A.; Xie, Yang; Xie, Xian-Jin; Kapur, Payal; McKay, Renee M.; Brugarolas, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">539</span>
        (<span class="NLM_cas:issue">7627</span>),
    <span class="NLM_cas:pages">112-117</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC) is characterized by inactivation of the von Hippel-Lindau tumor suppressor gene (VHL).  Because no other gene is mutated as frequently in ccRCC and VHL mutations are truncal, VHL inactivation is regarded as the governing event.  VHL loss activates the HIF-2 transcription factor, and constitutive HIF-2 activity restores tumorigenesis in VHL-reconstituted ccRCC cells.  HIF-2 has been implicated in angiogenesis and multiple other processes, but angiogenesis is the main target of drugs such as the tyrosine kinase inhibitor sunitinib.  HIF-2 has been regarded as undruggable.  Here we use a tumorgraft/patient-derived xenograft platform to evaluate PT2399, a selective HIF-2 antagonist that was identified using a structure-based design approach.  PT2399 dissocd. HIF-2 (an obligatory heterodimer of HIF-2α-HIF-1β) in human ccRCC cells and suppressed tumorigenesis in 56% (10 out of 18) of such lines.  PT2399 had greater activity than sunitinib, was active in sunitinib-progressing tumors, and was better tolerated.  Unexpectedly, some VHL-mutant ccRCCs were resistant to PT2399.  Resistance occurred despite HIF-2 dissocn. in tumors and evidence of Hif-2 inhibition in the mouse, as detd. by suppression of circulating erythropoietin, a HIF-2 target and possible pharmacodynamic marker.  We identified a HIF-2-dependent gene signature in sensitive tumors.  Gene expression was largely unaffected by PT2399 in resistant tumors, illustrating the specificity of the drug.  Sensitive tumors exhibited a distinguishing gene expression signature and generally higher levels of HIF-2α.  Prolonged PT2399 treatment led to resistance.  We identified binding site and second site suppressor mutations in HIF-2α and HIF-1β, resp.  Both mutations preserved HIF-2 dimers despite treatment with PT2399.  Finally, an extensively pretreated patient whose tumor had given rise to a sensitive tumorgraft showed disease control for more than 11 mo when treated with a close analog of PT2399, PT2385.  We validate HIF-2 as a target in ccRCC, show that some ccRCCs are HIF-2 independent, and set the stage for biomarker-driven clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnxhUY1OeFy7Vg90H21EOLACvtfcHk0lj2l3zSFIl4rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyis7rL&md5=006f9c774419f6226e22681286f43b6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature19796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19796%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DHill%26aufirst%3DH.%26aulast%3DChristie%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DHolloman%26aufirst%3DE.%26aulast%3DPavia-Jimenez%26aufirst%3DA.%26aulast%3DHomayoun%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DYell%26aufirst%3DP.%26aulast%3DHao%26aufirst%3DG.%26aulast%3DYousuf%26aufirst%3DQ.%26aulast%3DJoyce%26aufirst%3DA.%26aulast%3DPedrosa%26aufirst%3DI.%26aulast%3DGeiger%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGardner%26aufirst%3DK.%2BH.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26aulast%3DReeves%26aufirst%3DC.%26aulast%3DHwang%26aufirst%3DT.%2BH.%26aulast%3DCourtney%26aufirst%3DK.%26aulast%3DFrenkel%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZojwalla%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DT.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.-J.%26aulast%3DKapur%26aufirst%3DP.%26aulast%3DMcKay%26aufirst%3DR.%2BM.%26aulast%3DBrugarolas%26aufirst%3DJ.%26atitle%3DTargeting%2520renal%2520cell%2520carcinoma%2520with%2520a%2520HIF-2%2520antagonist%26jtitle%3DNature%26date%3D2016%26volume%3D539%26spage%3D112%26epage%3D117%26doi%3D10.1038%2Fnature19796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>539</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1038/nature19795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnature19795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27595393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyis73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=539&publication_year=2016&pages=107-111&author=H.+Choauthor=X.+Duauthor=J.+P.+Rizziauthor=E.+Liberzonauthor=A.+A.+Chakrabortyauthor=W.+Gaoauthor=I.+Carvoauthor=S.+Signorettiauthor=R.+K.+Bruickauthor=J.+A.+Joseyauthor=E.+M.+Wallaceauthor=W.+G.+Kaelin&title=On-target+efficacy+of+a+HIF-2%CE%B1+antagonist+in+preclinical+kidney+cancer+models&doi=10.1038%2Fnature19795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models</span></div><div class="casAuthors">Cho, Hyejin; Du, Xinlin; Rizzi, James P.; Liberzon, Ella; Chakraborty, Abhishek A.; Gao, Wenhua; Carvo, Ingrid; Signoretti, Sabina; Bruick, Richard K.; Josey, John A.; Wallace, Eli M.; Kaelin, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">539</span>
        (<span class="NLM_cas:issue">7627</span>),
    <span class="NLM_cas:pages">107-111</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma, the most common form of kidney cancer, is usually linked to inactivation of the pVHL tumor suppressor protein and consequent accumulation of the HIF-2α transcription factor (also known as EPAS1).  Here we show that a small mol. (PT2399) (I) that directly inhibits HIF-2α causes tumor regression in preclin. mouse models of primary and metastatic pVHL-defective clear cell renal cell carcinoma in an on-target fashion. pVHL-defective clear cell renal cell carcinoma cell lines display unexpectedly variable sensitivity to PT2399, however, suggesting the need for predictive biomarkers to be developed to use this approach optimally in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXNjIx5YKNerVg90H21EOLACvtfcHk0lhulFyhPBKItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyis73F&md5=f8dc173e758b6a6a17630b791c0c318c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature19795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19795%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26aulast%3DLiberzon%26aufirst%3DE.%26aulast%3DChakraborty%26aufirst%3DA.%2BA.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DCarvo%26aufirst%3DI.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DOn-target%2520efficacy%2520of%2520a%2520HIF-2%25CE%25B1%2520antagonist%2520in%2520preclinical%2520kidney%2520cancer%2520models%26jtitle%3DNature%26date%3D2016%26volume%3D539%26spage%3D107%26epage%3D111%26doi%3D10.1038%2Fnature19795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuneo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittag, T.</span></span> <span> </span><span class="NLM_article-title">The ubiquitin ligase adaptor SPOP in cancer</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">3946</span>– <span class="NLM_lpage">3958</span>, <span class="refDoi"> DOI: 10.1111/febs.15056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1111%2Ffebs.15056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=31495053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslyns7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2019&pages=3946-3958&author=M.+J.+Cuneoauthor=T.+Mittag&title=The+ubiquitin+ligase+adaptor+SPOP+in+cancer&doi=10.1111%2Ffebs.15056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin ligase adaptor SPOP in cancer</span></div><div class="casAuthors">Cuneo, Matthew J.; Mittag, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3946-3958</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The dysregulation of ubiquitin-mediated proteasomal degrdn. has emerged as an important mechanism of pathogenesis in several cancers.  The speckle-type POZ protein (SPOP) functions as a substrate adaptor for the cullin3-RING ubiquitin ligase and controls the cellular persistence of a diverse array of protein substrates in hormone signalling, epigenetic control and cell cycle regulation, to name a few.  Mutations in SPOP and the resulting dysregulation of this proteostatic pathway play causative roles in the pathogenesis of prostate and endometrial cancers, whereas overexpression and mislocalization are assocd. with kidney cancer.  Understanding the mol. mechanism of the normal function of SPOP as well as the cause of SPOP-mediated oncogenesis is thus crit. for eventual therapeutic targeting of SPOP and other related pathways.  Here, we will review SPOP structure, function and the mol. mechanism of how this function is achieved.  We will then review how mutations and protein mislocalization contribute to cancer pathogenesis and will provide a perspective on how SPOP may be targeted therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom0drBAEwmobVg90H21EOLACvtfcHk0lhulFyhPBKItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslyns7%252FP&md5=5157444927d173c40edd7b8e1fda696b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Ffebs.15056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.15056%26sid%3Dliteratum%253Aachs%26aulast%3DCuneo%26aufirst%3DM.%2BJ.%26aulast%3DMittag%26aufirst%3DT.%26atitle%3DThe%2520ubiquitin%2520ligase%2520adaptor%2520SPOP%2520in%2520cancer%26jtitle%3DFEBS%2520J.%26date%3D2019%26volume%3D286%26spage%3D3946%26epage%3D3958%26doi%3D10.1111%2Ffebs.15056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Functional roles of speckle-type Poz (SPOP) Protein in Genomic stability</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3262</span>, <span class="refDoi"> DOI: 10.7150/jca.25930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.7150%2Fjca.25930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=30271484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A280%3ADC%252BB3czktl2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3257-3262&author=X.+Weiauthor=J.+Friedauthor=Y.+Liauthor=L.+Huauthor=M.+Gaoauthor=S.+Zhangauthor=B.+Xu&title=Functional+roles+of+speckle-type+Poz+%28SPOP%29+Protein+in+Genomic+stability&doi=10.7150%2Fjca.25930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Functional roles of Speckle-Type Poz (SPOP) Protein in Genomic stability</span></div><div class="casAuthors">Wei Xi; Zhang Sheng; Fried Joshua; Xu Bo; Fried Joshua; Xu Bo; Li Ying; Hu Linfei; Gao Ming; Xu Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3257-3262</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Understanding the functional significance of the essential elements in maintaining genomic stability provides insights into the process of tumor initiation and progression, and predicts therapeutic responses.  One such element that has recently attracted significant attention is the Speckle-Type Poz Protein (SPOP), an E3 ubiquitin ligase adaptor protein.  SPOP is frequently mutated or has altered expression in various cancers, including prostate, renal and endometrial.  SPOP is involved in the regulation of proteasome-mediated degradation of several oncoproteins.  Moreover, recent data also indicate SPOP's direct involvement in the DNA damage response.  SPOP mutants induce alternations in the DNA damage repair pathway by promoting the error-prone Non-homologous end joining (NHEJ) pathway.  SPOP has been linked with significant functions in cellular signaling pathways and cancer suppression.  This mini-review will discuss recent findings regarding SPOP's role in genomic stability in the pathological setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOhFLms-L2x8AKB7Fa-xrUfW6udTcc2eauA9X7c5bW4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czktl2ktA%253D%253D&md5=05d065f59219aa6cac4c098b984f8e2d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7150%2Fjca.25930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.25930%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DX.%26aulast%3DFried%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DFunctional%2520roles%2520of%2520speckle-type%2520Poz%2520%2528SPOP%2529%2520Protein%2520in%2520Genomic%2520stability%26jtitle%3DJ.%2520Cancer%26date%3D2018%26volume%3D9%26spage%3D3257%26epage%3D3262%26doi%3D10.7150%2Fjca.25930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mani, R.-S.</span></span> <span> </span><span class="NLM_article-title">The emerging role of speckle-type POZ protein (SPOP) in cancer development</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1498</span>– <span class="NLM_lpage">1502</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.drudis.2014.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=25058385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWktbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1498-1502&author=R.-S.+Mani&title=The+emerging+role+of+speckle-type+POZ+protein+%28SPOP%29+in+cancer+development&doi=10.1016%2Fj.drudis.2014.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of speckle-type POZ protein (SPOP) in cancer development</span></div><div class="casAuthors">Mani, Ram-Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1498-1502</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Speckle-type POZ (pox virus and zinc finger protein) protein (SPOP) is an E3 ubiquitin ligase adaptor protein that is frequently mutated in prostate and endometrial cancers.  All the cancer-assocd. SPOP mutations reported to date are clustered in the meprin and TRAF (Tumor necrosis factor receptor-assocd. factor) homol. (MATH) domain, presumably affecting substrate binding.  SPOP mutations in prostate cancer are mutually exclusive with the ETS (Erythroblast transformation-specific) family gene rearrangements and define a distinct mol. subclass of prostate cancer.  SPOP mutations contribute to prostate cancer development by altering the steady-state levels of key components in the androgen-signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_7-WBSfpBirVg90H21EOLACvtfcHk0lhulFyhPBKItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWktbfJ&md5=da5d4ba48b523fa5d6bd8d06e5b61a78</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DR.-S.%26atitle%3DThe%2520emerging%2520role%2520of%2520speckle-type%2520POZ%2520protein%2520%2528SPOP%2529%2520in%2520cancer%2520development%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1498%26epage%3D1502%26doi%3D10.1016%2Fj.drudis.2014.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joe, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. H.</span></span> <span> </span><span class="NLM_article-title">BTB domain-containing speckle-type POZ protein (SPOP) serves as an adaptor of Daxx for ubiquitination by Cul3-based ubiquitin ligase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">12664</span>– <span class="NLM_lpage">12672</span>, <span class="refDoi"> DOI: 10.1074/jbc.m600204200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1074%2Fjbc.M600204200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=16524876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFKktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=12664-12672&author=J.+E.+Kwonauthor=M.+Laauthor=K.+H.+Ohauthor=Y.+M.+Ohauthor=G.+R.+Kimauthor=J.+H.+Seolauthor=S.+H.+Baekauthor=T.+Chibaauthor=K.+Tanakaauthor=O.+S.+Bangauthor=C.+O.+Joeauthor=C.+H.+Chung&title=BTB+domain-containing+speckle-type+POZ+protein+%28SPOP%29+serves+as+an+adaptor+of+Daxx+for+ubiquitination+by+Cul3-based+ubiquitin+ligase&doi=10.1074%2Fjbc.m600204200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BTB Domain-containing Speckle-type POZ Protein (SPOP) Serves as an Adaptor of Daxx for Ubiquitination by Cul3-based Ubiquitin Ligase</span></div><div class="casAuthors">Kwon, Jeong Eun; La, Muhnho; Oh, Kyu Hee; Oh, Young Mi; Kim, Gi Ryang; Seol, Jae Hong; Baek, Sung Hee; Chiba, Tomoki; Tanaka, Keiji; Bang, Ok Sun; Joe, Cheol O.; Chung, Chin Ha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12664-12672</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Daxx is a multifunctional protein that regulates a variety of cellular processes, including transcription, cell cycle, and apoptosis.  SPOP is a BTB (Bric-a-brac/Tramtrack/Broad complex) protein that constitutes Cul3-based ubiquitin ligases.  Here we show that SPOP serves as an adaptor of Daxx for the ubiquitination by Cul3-based ubiquitin ligase and subsequent degrdn. by the proteasome.  Expression of SPOP with Cul3 markedly reduced Daxx level, and this degrdn. was blocked by SPOP-specific short hairpin RNAs.  Inhibition of the proteasome by MG132 caused the prevention of Daxx degrdn. in parallel with the accumulation of ubiquitinated Daxx.  Expression of SPOP with Cul3 reversed Daxx-mediated repression of ETS1- and p53-dependent transcription, and short hairpin RNA-mediated knock down of SPOP blocked the recovery of their transcriptional activation.  Furthermore, Daxx degrdn. led to the cleavage of poly(ADP-ribose) polymerase and the increase in the no. of terminal deoxynucleotidyltransferase-mediated dUTP-fluorescein nick end-labeling-pos. apoptotic cells.  These results suggest that SPOP/Cul3-ubiquitin ligase plays an essential role in the control of Daxx level and, thus, in the regulation of Daxx-mediated cellular processes, including transcriptional regulation and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovP7jb4-r9HrVg90H21EOLACvtfcHk0lgnTRLEqLuHCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFKktrY%253D&md5=ab2133948dc79c942ff1b7a794cf423d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M600204200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M600204200%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DJ.%2BE.%26aulast%3DLa%26aufirst%3DM.%26aulast%3DOh%26aufirst%3DK.%2BH.%26aulast%3DOh%26aufirst%3DY.%2BM.%26aulast%3DKim%26aufirst%3DG.%2BR.%26aulast%3DSeol%26aufirst%3DJ.%2BH.%26aulast%3DBaek%26aufirst%3DS.%2BH.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DBang%26aufirst%3DO.%2BS.%26aulast%3DJoe%26aufirst%3DC.%2BO.%26aulast%3DChung%26aufirst%3DC.%2BH.%26atitle%3DBTB%2520domain-containing%2520speckle-type%2520POZ%2520protein%2520%2528SPOP%2529%2520serves%2520as%2520an%2520adaptor%2520of%2520Daxx%2520for%2520ubiquitination%2520by%2520Cul3-based%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D12664%26epage%3D12672%26doi%3D10.1074%2Fjbc.m600204200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Errington, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bueler, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinstein, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabartty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span> <span> </span><span class="NLM_article-title">Adaptor protein self-assembly drives the control of a cullin-RING ubiquitin ligase</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1153</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.str.2012.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=22632832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1WksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1141-1153&author=W.+J.+Erringtonauthor=M.+Q.+Khanauthor=S.+A.+Buelerauthor=J.+L.+Rubinsteinauthor=A.+Chakrabarttyauthor=G.+G.+Priv%C3%A9&title=Adaptor+protein+self-assembly+drives+the+control+of+a+cullin-RING+ubiquitin+ligase&doi=10.1016%2Fj.str.2012.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptor Protein Self-Assembly Drives the Control of a Cullin-RING Ubiquitin Ligase</span></div><div class="casAuthors">Errington, Wesley J.; Khan, M. Qasim; Bueler, Stephanie A.; Rubinstein, John L.; Chakrabartty, Avijit; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1141-1153</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The E3 ligases recruit substrate proteins for targeted ubiquitylation.  Recent insights into the mechanisms of ubiquitylation demonstrate that E3 ligases can possess active regulatory properties beyond those of a simple assembly scaffold.  Here, the dimeric structure of the E3 ligase adaptor protein SPOP (speckle-type POZ protein) in complex with the N-terminal domain of Cul3 is described at 2.4 Å resoln.  It was found that SPOP forms large oligomers that can form heteromeric species with the closely related paralog SPOPL.  In combination, SPOP and SPOPL (SPOP-like) form a mol. rheostat that can fine-tune E3 ubiquitin ligase activity by affecting the oligomeric state of the E3 complex.  It is proposed that adaptor protein self-assembly provides a graded level of regulation of the SPOP/Cul3 E3 ligase toward its multiple protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCM69_zYDo17Vg90H21EOLACvtfcHk0lgnTRLEqLuHCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1WksLY%253D&md5=854dbfe8ed84bbc87d6726b3bee50fcc</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DErrington%26aufirst%3DW.%2BJ.%26aulast%3DKhan%26aufirst%3DM.%2BQ.%26aulast%3DBueler%26aufirst%3DS.%2BA.%26aulast%3DRubinstein%26aufirst%3DJ.%2BL.%26aulast%3DChakrabartty%26aufirst%3DA.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26atitle%3DAdaptor%2520protein%2520self-assembly%2520drives%2520the%2520control%2520of%2520a%2520cullin-RING%2520ubiquitin%2520ligase%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1141%26epage%3D1153%26doi%3D10.1016%2Fj.str.2012.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nourse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walden, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seyedin, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoggard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.molcel.2009.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19818708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2009&pages=39-50&author=M.+Zhuangauthor=M.+F.+Calabreseauthor=J.+Liuauthor=M.+B.+Waddellauthor=A.+Nourseauthor=M.+Hammelauthor=D.+J.+Millerauthor=H.+Waldenauthor=D.+M.+Dudaauthor=S.+N.+Seyedinauthor=T.+Hoggardauthor=J.+W.+Harperauthor=K.+P.+Whiteauthor=B.+A.+Schulman&title=Structures+of+SPOP-substrate+complexes%3A+insights+into+molecular+architectures+of+BTB-Cul3+ubiquitin+ligases&doi=10.1016%2Fj.molcel.2009.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases</span></div><div class="casAuthors">Zhuang, Min; Calabrese, Matthew F.; Liu, Jiang; Waddell, M. Brett; Nourse, Amanda; Hammel, Michal; Miller, Darcie J.; Walden, Helen; Duda, David M.; Seyedin, Steven N.; Hoggard, Timothy; Harper, J. Wade; White, Kevin P.; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In the largest E3 ligase subfamily, Cul3 binds a BTB domain, and an assocd. protein-interaction domain such as MATH recruits substrates for ubiquitination.  Here, we present biochem. and structural analyses of the MATH-BTB protein, SPOP.  We define a SPOP-binding consensus (SBC) and det. structures revealing recognition of SBCs from the phosphatase Puc (Puckered), the transcriptional regulator Ci (Cubitus interruptus), and the chromatin component histone variant MacroH2A.  We identify a dimeric SPOP-Cul3 assembly involving a conserved helical structure C-terminal of BTB domains, which we call "3-box" due to its ability to facilitate Cul3 binding and its resemblance to F-/SOCS-boxes in other cullin-based E3s.  Structural flexibility between the substrate-binding MATH and Cul3-binding BTB/3-box domains potentially allows a SPOP dimer to engage multiple SBCs found within a single substrate, such as Puc.  These studies provide a mol. understanding of how MATH-BTB proteins recruit substrates to Cul3 and how their dimerization and conformational variability may facilitate avid interactions with diverse substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWn7mZNy7FLVg90H21EOLACvtfcHk0lhDRwnw1ZR9gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbN&md5=65bb062fd7e17ce46bab56624f892e8e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWaddell%26aufirst%3DM.%2BB.%26aulast%3DNourse%26aufirst%3DA.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DD.%2BJ.%26aulast%3DWalden%26aufirst%3DH.%26aulast%3DDuda%26aufirst%3DD.%2BM.%26aulast%3DSeyedin%26aufirst%3DS.%2BN.%26aulast%3DHoggard%26aufirst%3DT.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructures%2520of%2520SPOP-substrate%2520complexes%253A%2520insights%2520into%2520molecular%2520architectures%2520of%2520BTB-Cul3%2520ubiquitin%2520ligases%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D36%26spage%3D39%26epage%3D50%26doi%3D10.1016%2Fj.molcel.2009.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iossifov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeletti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negre, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stricker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ahmadie, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tretiakova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera-Alberto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimm, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzhetsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span> <span> </span><span class="NLM_article-title">Analysis of Drosophila segmentation network identifies a JNK pathway factor overexpressed in kidney cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>323</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1126/science.1157669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1126%2Fscience.1157669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19164706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1elurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2009&pages=1218-1222&author=J.+Liuauthor=M.+Ghanimauthor=L.+Xueauthor=C.+D.+Brownauthor=I.+Iossifovauthor=C.+Angelettiauthor=S.+Huaauthor=N.+Negreauthor=M.+Ludwigauthor=T.+Strickerauthor=H.+A.+Al-Ahmadieauthor=M.+Tretiakovaauthor=R.+L.+Campauthor=M.+Perera-Albertoauthor=D.+L.+Rimmauthor=T.+Xuauthor=A.+Rzhetskyauthor=K.+P.+White&title=Analysis+of+Drosophila+segmentation+network+identifies+a+JNK+pathway+factor+overexpressed+in+kidney+cancer&doi=10.1126%2Fscience.1157669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Drosophila Segmentation Network Identifies a JNK Pathway Factor Overexpressed in Kidney Cancer</span></div><div class="casAuthors">Liu, Jiang; Ghanim, Murad; Xue, Lei; Brown, Christopher D.; Iossifov, Ivan; Angeletti, Cesar; Hua, Sujun; Negre, Nicolas; Ludwig, Michael; Stricker, Thomas; Al-Ahmadie, Hikmat A.; Tretiakova, Maria; Camp, Robert L.; Perera-Alberto, Montse; Rimm, David L.; Xu, Tian; Rzhetsky, Andrey; White, Kevin P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">5918</span>),
    <span class="NLM_cas:pages">1218-1222</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The authors constructed a large-scale functional network model in Drosophila melanogaster built around two key transcription factors involved in the process of embryonic segmentation.  Anal. of the model allowed the identification of a new role for the ubiquitin E3 ligase complex factor SPOP.  In Drosophila, the gene encoding SPOP is a target of segmentation transcription factors.  Drosophila SPOP mediates degrdn. of the Jun kinase phosphatase Puckered, thereby inducing tumor necrosis factor (TNF)/Eiger-dependent apoptosis.  In humans, it was found that SPOP plays a conserved role in TNF-mediated JNK signaling and was highly expressed in 99% of clear cell renal cell carcinomas (RCCs), the most prevalent form of kidney cancer.  SPOP expression distinguished histol. subtypes of RCC and facilitated identification of clear cell RCC as the primary tumor for metastatic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHL_AT2Jqj7Vg90H21EOLACvtfcHk0lhDRwnw1ZR9gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1elurw%253D&md5=1a9d3a462740157b10d522610ebaea0c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.1157669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1157669%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DGhanim%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DC.%2BD.%26aulast%3DIossifov%26aufirst%3DI.%26aulast%3DAngeletti%26aufirst%3DC.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DNegre%26aufirst%3DN.%26aulast%3DLudwig%26aufirst%3DM.%26aulast%3DStricker%26aufirst%3DT.%26aulast%3DAl-Ahmadie%26aufirst%3DH.%2BA.%26aulast%3DTretiakova%26aufirst%3DM.%26aulast%3DCamp%26aufirst%3DR.%2BL.%26aulast%3DPerera-Alberto%26aufirst%3DM.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DRzhetsky%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26atitle%3DAnalysis%2520of%2520Drosophila%2520segmentation%2520network%2520identifies%2520a%2520JNK%2520pathway%2520factor%2520overexpressed%2520in%2520kidney%2520cancer%26jtitle%3DScience%26date%3D2009%26volume%3D323%26spage%3D1218%26epage%3D1222%26doi%3D10.1126%2Fscience.1157669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ci, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karmakar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinker-Schaeffer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggener, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stricker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span> <span> </span><span class="NLM_article-title">SPOP promotes tumorigenesis by acting as a key regulatory hub in kidney cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccr.2014.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=24656772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1OmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=455-468&author=G.+Liauthor=W.+Ciauthor=S.+Karmakarauthor=K.+Chenauthor=R.+Dharauthor=Z.+Fanauthor=Z.+Guoauthor=J.+Zhangauthor=Y.+Keauthor=L.+Wangauthor=M.+Zhuangauthor=S.+Huauthor=X.+Liauthor=L.+Zhouauthor=X.+Liauthor=M.+F.+Calabreseauthor=E.+R.+Watsonauthor=S.+M.+Prasadauthor=C.+Rinker-Schaefferauthor=S.+E.+Eggenerauthor=T.+Strickerauthor=Y.+Tianauthor=B.+A.+Schulmanauthor=J.+Liuauthor=K.+P.+White&title=SPOP+promotes+tumorigenesis+by+acting+as+a+key+regulatory+hub+in+kidney+cancer&doi=10.1016%2Fj.ccr.2014.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SPOP Promotes Tumorigenesis by Acting as a Key Regulatory Hub in Kidney Cancer</span></div><div class="casAuthors">Li, Guoqiang; Ci, Weimin; Karmakar, Subhradip; Chen, Ke; Dhar, Ruby; Fan, Zhixiang; Guo, Zhongqiang; Zhang, Jing; Ke, Yuwen; Wang, Lu; Zhuang, Min; Hu, Shengdi; Li, Xuesong; Zhou, Liqun; Li, Xianghong; Calabrese, Matthew F.; Watson, Edmond R.; Prasad, Sandip M.; Rinker-Schaeffer, Carrie; Eggener, Scott E.; Stricker, Thomas; Tian, Yong; Schulman, Brenda A.; Liu, Jiang; White, Kevin P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">455-468</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Hypoxic stress and hypoxia-inducible factors (HIFs) play important roles in a wide range of tumors.  We demonstrate that SPOP, which encodes an E3 ubiquitin ligase component, is a direct transcriptional target of HIFs in clear cell renal cell carcinoma (ccRCC).  Furthermore, hypoxia results in cytoplasmic accumulation of SPOP, which is sufficient to induce tumorigenesis.  This tumorigenic activity occurs through the ubiquitination and degrdn. of multiple regulators of cellular proliferation and apoptosis, including the tumor suppressor PTEN, ERK phosphatases, the proapoptotic mol. Daxx, and the Hedgehog pathway transcription factor Gli2.  Knockdown of SPOP specifically kills ccRCC cells, indicating that it may be a promising therapeutic target.  Collectively, our results indicate that SPOP serves as a regulatory hub to promote ccRCC tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrscPybRIwJzLVg90H21EOLACvtfcHk0lhDRwnw1ZR9gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1OmsLw%253D&md5=6f271d0450f77f8e2474b2b324b4b555</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.%26aulast%3DCi%26aufirst%3DW.%26aulast%3DKarmakar%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DDhar%26aufirst%3DR.%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhuang%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DWatson%26aufirst%3DE.%2BR.%26aulast%3DPrasad%26aufirst%3DS.%2BM.%26aulast%3DRinker-Schaeffer%26aufirst%3DC.%26aulast%3DEggener%26aufirst%3DS.%2BE.%26aulast%3DStricker%26aufirst%3DT.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26atitle%3DSPOP%2520promotes%2520tumorigenesis%2520by%2520acting%2520as%2520a%2520key%2520regulatory%2520hub%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D455%26epage%3D468%26doi%3D10.1016%2Fj.ccr.2014.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeting of E3 ligase adaptor SPOP in kidney cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccell.2016.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27622336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=474-484&author=Z.-Q.+Guoauthor=T.+Zhengauthor=B.+Chenauthor=C.+Luoauthor=S.+Ouyangauthor=S.+Gongauthor=J.+Liauthor=L.-L.+Maoauthor=F.+Lianauthor=Y.+Yangauthor=Y.+Huangauthor=L.+Liauthor=J.+Luauthor=B.+Zhangauthor=L.+Zhouauthor=H.+Dingauthor=Z.+Gaoauthor=L.+Zhouauthor=G.+Liauthor=R.+Zhouauthor=K.+Chenauthor=J.+Liuauthor=Y.+Wenauthor=L.+Gongauthor=Y.+Keauthor=S.-D.+Yangauthor=X.-B.+Qiuauthor=N.+Zhangauthor=J.+Renauthor=D.+Zhongauthor=C.-G.+Yangauthor=J.+Liuauthor=H.+Jiang&title=Small-molecule+targeting+of+E3+ligase+adaptor+SPOP+in+kidney+cancer&doi=10.1016%2Fj.ccell.2016.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Targeting of E3 Ligase Adaptor SPOP in Kidney Cancer</span></div><div class="casAuthors">Guo, Zhong-Qiang; Zheng, Tong; Chen, Baoen; Luo, Cheng; Ouyang, Sisheng; Gong, Shouzhe; Li, Jiafei; Mao, Liu-Liang; Lian, Fulin; Yang, Yong; Huang, Yue; Li, Li; Lu, Jing; Zhang, Bidong; Zhou, Luming; Ding, Hong; Gao, Zhiwei; Zhou, Liqun; Li, Guoqiang; Zhou, Ran; Chen, Ke; Liu, Jingqiu; Wen, Yi; Gong, Likun; Ke, Yuwen; Yang, Shang-Dong; Qiu, Xiao-Bo; Zhang, Naixia; Ren, Jin; Zhong, Dafang; Yang, Cai-Guang; Liu, Jiang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-484</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In the cytoplasm of virtually all clear-cell renal cell carcinoma (ccRCC), speckle-type POZ protein (SPOP) is overexpressed and misallocated, which may induce proliferation and promote kidney tumorigenesis.  In normal cells, however, SPOP is located in the nucleus and induces apoptosis.  Here we show that a structure-based design and subsequent hit optimization yield small mols. that can inhibit the SPOP-substrate protein interaction and can suppress oncogenic SPOP-signaling pathways.  These inhibitors kill human ccRCC cells that are dependent on oncogenic cytoplasmic SPOP.  Notably, these inhibitors minimally affect the viability of other cells in which SPOP is not accumulated in the cytoplasm.  Our findings validate the SPOP-substrate protein interaction as an attractive target specific to ccRCC that may yield novel drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSo7OKJgIr7Vg90H21EOLACvtfcHk0lgF9YCmqVQB7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK&md5=cd349f46b9ae34c0158fb268666c4c9e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DZ.-Q.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.-L.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-D.%26aulast%3DQiu%26aufirst%3DX.-B.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-G.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSmall-molecule%2520targeting%2520of%2520E3%2520ligase%2520adaptor%2520SPOP%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D474%26epage%3D484%26doi%3D10.1016%2Fj.ccell.2016.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, L.</span></span> <span> </span><span class="NLM_article-title">Kidney cancer: On target - inhibiting SPOP in ccRCC</span>. <i>Nat. Rev. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">630</span>, <span class="refDoi"> DOI: 10.1038/nrurol.2016.192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnrurol.2016.192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1CgsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=630&author=L.+Stone&title=Kidney+cancer%3A+On+target+-+inhibiting+SPOP+in+ccRCC&doi=10.1038%2Fnrurol.2016.192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Kidney cancer On target - inhibiting SPOP in ccRCC</span></div><div class="casAuthors">Stone, Louise</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Urology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">630</span>CODEN:
                <span class="NLM_cas:coden">NRNADQ</span>;
        ISSN:<span class="NLM_cas:issn">1759-4812</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kidney cancer On target - inhibiting SPOP in ccRCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4Pw-GS9Hk5rVg90H21EOLACvtfcHk0lgF9YCmqVQB7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1CgsbfK&md5=302e1d2a0424b728d86b078b707ea5d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrurol.2016.192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrurol.2016.192%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DL.%26atitle%3DKidney%2520cancer%253A%2520On%2520target%2520-%2520inhibiting%2520SPOP%2520in%2520ccRCC%26jtitle%3DNat.%2520Rev.%2520Urol.%26date%3D2016%26volume%3D13%26spage%3D630%26doi%3D10.1038%2Fnrurol.2016.192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-G.</span></span> <span> </span><span class="NLM_article-title">Targeting SPOP with small molecules provides a novel strategy for kidney cancer therapy</span>. <i>Sci. China: Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1007/s11427-016-0297-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1007%2Fs11427-016-0297-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27888385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A280%3ADC%252BC2sjgsFemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=91-93&author=T.+Zhengauthor=C.-G.+Yang&title=Targeting+SPOP+with+small+molecules+provides+a+novel+strategy+for+kidney+cancer+therapy&doi=10.1007%2Fs11427-016-0297-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting SPOP with small molecules provides a novel strategy for kidney cancer therapy</span></div><div class="casAuthors">Zheng Tong; Yang Cai-Guang; Zheng Tong</div><div class="citationInfo"><span class="NLM_cas:title">Science China. Life sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTKDDAd1lUQJGhHGZei4OW9fW6udTcc2ea8uaC1fIgfOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjgsFemsg%253D%253D&md5=a1b798136386d3c7e171142817f99e97</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs11427-016-0297-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11427-016-0297-2%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DC.-G.%26atitle%3DTargeting%2520SPOP%2520with%2520small%2520molecules%2520provides%2520a%2520novel%2520strategy%2520for%2520kidney%2520cancer%2520therapy%26jtitle%3DSci.%2520China%253A%2520Life%2520Sci.%26date%3D2017%26volume%3D60%26spage%3D91%26epage%3D93%26doi%3D10.1007%2Fs11427-016-0297-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2529</span>– <span class="NLM_lpage">2591</span>, <span class="refDoi"> DOI: 10.1021/jm1013693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design&doi=10.1021%2Fjm1013693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0lht7q9iZa122Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591%26doi%3D10.1021%2Fjm1013693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">St
Jean, D. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotsch, C.</span></span> <span> </span><span class="NLM_article-title">Mitigating heterocycle metabolism in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6002</span>– <span class="NLM_lpage">6020</span>, <span class="refDoi"> DOI: 10.1021/jm300343m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300343m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFWhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6002-6020&author=D.+J.+St%0AJeanauthor=C.+Fotsch&title=Mitigating+heterocycle+metabolism+in+drug+discovery&doi=10.1021%2Fjm300343m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mitigating Heterocycle Metabolism in Drug Discovery</span></div><div class="casAuthors">St. Jean, David J.; Fotsch, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6002-6020</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Extensive data from metab. studies have allowed medicinal chemists to develop general principles for reducing compd. metab.  These methods include, but are not limited to, reducing lipophilicity, altering sterics and electronics, introducing a conformational constraint, and altering the stereochem. of their compds.  While no single method is able to solve every metabolic problem, these principles do give medicinal chemists guidance on how to improve the metabolic liabilities of their compds.  If the specific site of metab. is known, medicinal chemists block the site, typically with a fluorine, or replace the metabolically labile group with a bioisostere.  While several authors have reviewed these techniques for reducing metab., there is no review that summarizes different approaches to improving them metabolic stability of heterocycles.  In this review, we summarize examples where changes were made at or near the heterocycle to improve metabolic stability.  By summarizing these examples, we hope to provide a useful guide to medicinal chemists as they attempt to improve the metabolic profile of their own heterocyclic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovQjdBBgXtPbVg90H21EOLACvtfcHk0lht7q9iZa122Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFWhsL0%253D&md5=48c262556b8a2cc7dc5539ff91a53a71</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm300343m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300343m%26sid%3Dliteratum%253Aachs%26aulast%3DSt%2BJean%26aufirst%3DD.%2BJ.%26aulast%3DFotsch%26aufirst%3DC.%26atitle%3DMitigating%2520heterocycle%2520metabolism%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6002%26epage%3D6020%26doi%3D10.1021%2Fjm300343m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jindal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devadoss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and pharmacological evaluation of novel 2-(4-substituted piperazin-1-yl)1, 8 naphthyridine 3-carboxylic acids as 5-HT3 receptor antagonists for the management of depression</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1111%2Fcbdd.12370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=24903617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSqur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2014&pages=721-731&author=A.+K.+Dharauthor=R.+Maheshauthor=A.+Jindalauthor=T.+Devadossauthor=S.+Bhatt&title=Design%2C+synthesis%2C+and+pharmacological+evaluation+of+novel+2-%284-substituted+piperazin-1-yl%291%2C+8+naphthyridine+3-carboxylic+acids+as+5-HT3+receptor+antagonists+for+the+management+of+depression&doi=10.1111%2Fcbdd.12370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Pharmacological Evaluation of Novel 2-(4-substituted piperazin-1-yl)-1,8-Naphthyridine-3-Carboxylic Acids as 5-HT3 Receptor Antagonists for the Management of Depression</span></div><div class="casAuthors">Dhar, Arghya K.; Mahesh, Radhakrishnan; Jindal, Ankur; Devadoss, Thangaraj; Bhatt, Shvetank</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-731</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">1,8-Naphthyridine-3-carboxylic acid analogs I (R = Me, Ph, Bn, 4-ClC6H4, etc.) were synthesized and found to possess potential 5-HT3 receptor antagonism as well as antidepressant-like activity.  Initially, 5-HT3 receptor antagonism of all the compds. was detd. in the form of pA2 value against agonist 2-Me 5-HT in longitudinal muscle-myenteric plexus prepn. from guinea-pig ileum.  Among all the compds. tested, compd. I (R = Ph) demonstrated most promising pA2 value of 7.6.  Subsequently, all compds. were evaluated for antidepressant activity using forced swim test and tail suspension test in mice.  Compds. I (R = Ph, 4-O2NC6H4, 4-ClC6H4, 2-MeOC6H4, 4-MeOC6H4) exhibited significant (p < 0.05) antidepressant-like activity as compd. to vehicle-treated group.  Importantly, none of the tested compd. affected locomotor activity of mice at tested dose levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIEfpC3IQVLVg90H21EOLACvtfcHk0liWNev0nLMYSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSqur%252FJ&md5=b4f1e51a923fa9ced4e98389d50cc228</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12370%26sid%3Dliteratum%253Aachs%26aulast%3DDhar%26aufirst%3DA.%2BK.%26aulast%3DMahesh%26aufirst%3DR.%26aulast%3DJindal%26aufirst%3DA.%26aulast%3DDevadoss%26aufirst%3DT.%26aulast%3DBhatt%26aufirst%3DS.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520pharmacological%2520evaluation%2520of%2520novel%25202-%25284-substituted%2520piperazin-1-yl%25291%252C%25208%2520naphthyridine%25203-carboxylic%2520acids%2520as%25205-HT3%2520receptor%2520antagonists%2520for%2520the%2520management%2520of%2520depression%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2014%26volume%3D84%26spage%3D721%26epage%3D731%26doi%3D10.1111%2Fcbdd.12370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saccomanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adinolfi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligresti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrarini, P. L.</span></span> <span> </span><span class="NLM_article-title">Rational design, synthesis, and pharmacological properties of new 1,8-naphthyridin-2(1H)-on-3-carboxamide derivatives as highly selective cannabinoid-2 receptor agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3644</span>– <span class="NLM_lpage">3651</span>, <span class="refDoi"> DOI: 10.1021/jm801563d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801563d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlslOqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3644-3651&author=C.+Maneraauthor=G.+Saccomanniauthor=B.+Adinolfiauthor=V.+Benettiauthor=A.+Ligrestiauthor=M.+G.+Cascioauthor=T.+Tuccinardiauthor=V.+Lucchesiauthor=A.+Martinelliauthor=P.+Nieriauthor=E.+Masiniauthor=V.+Di+Marzoauthor=P.+L.+Ferrarini&title=Rational+design%2C+synthesis%2C+and+pharmacological+properties+of+new+1%2C8-naphthyridin-2%281H%29-on-3-carboxamide+derivatives+as+highly+selective+cannabinoid-2+receptor+agonists&doi=10.1021%2Fjm801563d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design, Synthesis, and Pharmacological Properties of New 1,8-Naphthyridin-2(1H)-on-3-carboxamide Derivatives as Highly Selective Cannabinoid-2 Receptor Agonists</span></div><div class="casAuthors">Manera, Clementina; Saccomanni, Giuseppe; Adinolfi, Barbara; Benetti, Veronica; Ligresti, Alessia; Cascio, Maria Grazia; Tuccinardi, Tiziano; Lucchesi, Valentina; Martinelli, Adriano; Nieri, Paola; Masini, Emanuela; Di Marzo, Vincenzo; Ferrarini, Pier Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3644-3651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The CB2 receptor activation can be exploited for the treatment of diseases such as chronic pain and tumors of immune origin, devoid of psychotropic activity.  On the basis of our already reported 1,8-naphthyridin-4(1H)-on-3-carboxamide derivs., new 1,8-naphthyridin-2(1H)-on-3-carboxamide derivs. were designed, synthesized, and tested for their affinities toward the human CB1 and CB2 cannabinoid receptors.  Some of the reported compds. showed a subnanomolar CB2 affinity with a CB1/CB2 selectivity ratio greater than 200.  Further studies revealed that the compd. which presented benzyl and carboxy-4-methylcyclohexylamide substituents bound in the 1 and 3 positions exerted a CB2-mediated inhibitory action on immunol. human basophil activation.  On the human T cell leukemia line Jurkat the same deriv. induced a concn.-dependent decrease of cell viability.  The obtained results suggest that 1,8-naphthyridin-2(1H)-on-3-carboxamides represent a new scaffold very suitable for the development of new promising CB2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcHJZnJ6fZcbVg90H21EOLACvtfcHk0liWNev0nLMYSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlslOqsLY%253D&md5=ca24de2d70f151057cf749ee9ff5151f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm801563d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801563d%26sid%3Dliteratum%253Aachs%26aulast%3DManera%26aufirst%3DC.%26aulast%3DSaccomanni%26aufirst%3DG.%26aulast%3DAdinolfi%26aufirst%3DB.%26aulast%3DBenetti%26aufirst%3DV.%26aulast%3DLigresti%26aufirst%3DA.%26aulast%3DCascio%26aufirst%3DM.%2BG.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DLucchesi%26aufirst%3DV.%26aulast%3DMartinelli%26aufirst%3DA.%26aulast%3DNieri%26aufirst%3DP.%26aulast%3DMasini%26aufirst%3DE.%26aulast%3DDi%2BMarzo%26aufirst%3DV.%26aulast%3DFerrarini%26aufirst%3DP.%2BL.%26atitle%3DRational%2520design%252C%2520synthesis%252C%2520and%2520pharmacological%2520properties%2520of%2520new%25201%252C8-naphthyridin-2%25281H%2529-on-3-carboxamide%2520derivatives%2520as%2520highly%2520selective%2520cannabinoid-2%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3644%26epage%3D3651%26doi%3D10.1021%2Fjm801563d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardew, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitag-Pohl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, E.</span></span> <span> </span><span class="NLM_article-title">How to stabilize protein: stability screens for thermal shift assays and nano differential scanning fluorimetry in the virus-X project</span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">58666</span>, <span class="refDoi"> DOI: 10.3791/58666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.3791%2F58666" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=58666&author=D.+Bruceauthor=E.+Cardewauthor=S.+Freitag-Pohlauthor=E.+Pohl&title=How+to+stabilize+protein%3A+stability+screens+for+thermal+shift+assays+and+nano+differential+scanning+fluorimetry+in+the+virus-X+project&doi=10.3791%2F58666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3791%2F58666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F58666%26sid%3Dliteratum%253Aachs%26aulast%3DBruce%26aufirst%3DD.%26aulast%3DCardew%26aufirst%3DE.%26aulast%3DFreitag-Pohl%26aufirst%3DS.%26aulast%3DPohl%26aufirst%3DE.%26atitle%3DHow%2520to%2520stabilize%2520protein%253A%2520stability%2520screens%2520for%2520thermal%2520shift%2520assays%2520and%2520nano%2520differential%2520scanning%2520fluorimetry%2520in%2520the%2520virus-X%2520project%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2019%26volume%3D144%26spage%3D58666%26doi%3D10.3791%2F58666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pai, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomenick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiestl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span> <span> </span><span class="NLM_article-title">Drug affinity responsive target stability (DARTS) for small-molecule target identification</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1263</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2269-7_22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1007%2F978-1-4939-2269-7_22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=25618353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls12murk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1263&publication_year=2015&pages=287-298&author=M.+Y.+Paiauthor=B.+Lomenickauthor=H.+Hwangauthor=R.+Schiestlauthor=W.+McBrideauthor=J.+A.+Looauthor=J.+Huang&title=Drug+affinity+responsive+target+stability+%28DARTS%29+for+small-molecule+target+identification&doi=10.1007%2F978-1-4939-2269-7_22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Affinity Responsive Target Stability (DARTS) for Small-Molecule Target Identification</span></div><div class="casAuthors">Pai, Melody Y.; Lomenick, Brett; Hwang, Heejun; Schiestl, Robert; McBride, William; Loo, Joseph A.; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1263</span>
        (<span class="NLM_cas:issue">Chemical Biology</span>),
    <span class="NLM_cas:pages">287-298</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug affinity responsive target stability (DARTS) is a relatively quick and straightforward approach to identify potential protein targets for small mols.  It relies on the protection against proteolysis conferred on the target protein by interaction with a small mol.  The greatest advantage of this method is being able to use the native small mol. without having to immobilize or modify it (e.g., by incorporation of biotin, fluorescent, radioisotope, or photoaffinity labels).  Here we describe in detail the protocol for performing unbiased DARTS with complex protein lysates to identify binding targets of small mols. and for using DARTS-Western blotting to test, screen, or validate potential small-mol. targets.  Although the ideas have mainly been developed from studying mols. in areas of biol. that are currently of interest to us and our collaborators, the general principles should be applicable to the anal. of all mols. in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZkyptImVfKbVg90H21EOLACvtfcHk0liWNev0nLMYSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls12murk%253D&md5=17d9d8199173653f814d80e0945c94ad</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2269-7_22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2269-7_22%26sid%3Dliteratum%253Aachs%26aulast%3DPai%26aufirst%3DM.%2BY.%26aulast%3DLomenick%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DSchiestl%26aufirst%3DR.%26aulast%3DMcBride%26aufirst%3DW.%26aulast%3DLoo%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DDrug%2520affinity%2520responsive%2520target%2520stability%2520%2528DARTS%2529%2520for%2520small-molecule%2520target%2520identification%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1263%26spage%3D287%26epage%3D298%26doi%3D10.1007%2F978-1-4939-2269-7_22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong  Yang</span>, <span class="hlFld-ContribAuthor ">Jingqi  Shi</span>, <span class="hlFld-ContribAuthor ">Jiaming  Chen</span>, <span class="hlFld-ContribAuthor ">Xiaoqi  Jia</span>, <span class="hlFld-ContribAuthor ">Cuiying  Shi</span>, <span class="hlFld-ContribAuthor ">Lifang  Ma</span>, <span class="hlFld-ContribAuthor ">Ziyuan  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Visible-light enabled room-temperature dealkylative imidation of secondary and tertiary amines promoted by aerobic ruthenium catalysis. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (31)
                                     , 18966-18973. <a href="https://doi.org/10.1039/D0RA10517A" title="DOI URL">https://doi.org/10.1039/D0RA10517A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA10517A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA10517A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DVisible-light%252Benabled%252Broom-temperature%252Bdealkylative%252Bimidation%252Bof%252Bsecondary%252Band%252Btertiary%252Bamines%252Bpromoted%252Bby%252Baerobic%252Bruthenium%252Bcatalysis%26aulast%3DYang%26aufirst%3DDong%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D31%26spage%3D18966%26epage%3D18973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhen  Wang</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Baoen  Chen</span>, <span class="hlFld-ContribAuthor ">Sisheng  Ouyang</span>, <span class="hlFld-ContribAuthor ">Tong  Zheng</span>, <span class="hlFld-ContribAuthor ">Ran  Zhou</span>, <span class="hlFld-ContribAuthor ">Ze  Dong</span>, <span class="hlFld-ContribAuthor ">Yue  Huang</span>, <span class="hlFld-ContribAuthor ">Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Jianhua  Gan</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Cai‐Guang  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">A Peptide Binder of
              E3
              Ligase Adaptor
              SPOP
              Disrupts Oncogenic
              SPOP‐Protein
              Interactions in Kidney Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Chemistry</span><span> <strong>2021,</strong> <em>39 </em>
                                    (2)
                                     , 274-280. <a href="https://doi.org/10.1002/cjoc.202000462" title="DOI URL">https://doi.org/10.1002/cjoc.202000462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cjoc.202000462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcjoc.202000462%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Chemistry%26atitle%3DA%252BPeptide%252BBinder%252Bof%252BE3%252BLigase%252BAdaptor%252BSPOP%252BDisrupts%252BOncogenic%252BSPOP%2525E2%252580%252590Protein%252BInteractions%252Bin%252BKidney%252BCancer%252BCells%26aulast%3DWang%26aufirst%3DZhen%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D2%26spage%3D274%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of our previously reported SPOP inhibitors. Compound <b>6c</b> is a negative control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of different sets of SPOP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>6d–6z</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 220 °C, diethyl malonate, neat; (b) POCl<sub>3</sub>, DMF; (c) R<sup>1</sup>NH<sub>2</sub>, Na<sub>2</sub>SO<sub>4</sub>, EtOH, rt, overnight; (d) <b>5a</b>, piperidine, CHCl<sub>3</sub>, reflux, overnight; (e) neat, rt, 3 days.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>6la–6lj</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) R<sup>2</sup>NH<sub>2</sub>, neat, rt, 3 days; (b) piperidine, CHCl<sub>3</sub>, reflux, overnight.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds with or without the methyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>7</b> and <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl α-cyanoacetate <b>5l</b>, piperidine, CHCl<sub>3</sub>, reflux, overnight; (b) LiOH, THF/EtOH/H<sub>2</sub>O, 45 °C, overnight.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>13</b> and <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) piperidine, diethyl malonate, neat, reflux, overnight; (b) PhCH<sub>2</sub>CH<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF; (c) LiOH, THF/EtOH/H<sub>2</sub>O, 45 °C, 4 h; (d) R<sup>2</sup>NH<sub>2</sub>, HATU, DIEA, DMF.</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Antiproliferative effect of inhibitor <b>6lc</b>. (A) Quantitation of IC<sub>50</sub> of <b>6lc</b> against the proliferation of ccRCC cell lines and HK-2 cells. (B) Effect of SPOP inhibitors on the colony formation of A498 and OS-RC-2 cell lines. (C) Effect of <b>6lc</b> on the proliferation in HEK293T cells overexpressing SPOP. (D) Effect of <b>6lc</b> on cell proliferation in other cancer cell lines. **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001; unpaired Student’s <i>t</i>-test. Error bar, mean ± SD, <i>n</i> = 3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Investigations on the binding of inhibitors to SPOP in vitro. (A) Effect of <b>6lc</b> on the thermal stability of SPOP<sup>23–337</sup> in nanoDSF. (B) Effect of <b>6c</b> on the thermal stability of SPOP<sup>MATH</sup> in nanoDSF. (C) Effect of <b>6lc</b> on the stability of SPOP<sup>MATH</sup> in the conventional DSF assay. (D) Effect of <b>6c</b> on the stability of SPOP<sup>MATH</sup> in the conventional DSF. (E) NMR measurement of <b>6lc</b> binding to SPOP<sup>MATH</sup>. The STD-NMR spectrum for 200 μM <b>6lc</b> is recorded in the presence of SPOP<sup>MATH</sup>. (F) Determination of the affinity of <b>6lc</b> binding to SPOP<sup>MATH</sup> in SPR measurement. Graphs of equilibrium RU responses vs compound concentrations are plotted. (G) Effect of SPOP inhibitors on the stability of SPOP<sup>MATH</sup> in the conventional DSF assay. For nanoDSF and conventional DSF assays, Δ<i>T</i><sub>m</sub> for the DMSO-treated SPOP protein were calculated using a naive SPOP sample as the reference, while Δ<i>T</i><sub>m</sub> data for the compound-treated SPOP were reported using DMSO-treated SPOP as the reference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effect of <b>6lc</b> on the interaction with SPOP and the disruption of the SPOP–PTEN interaction in HEK293T cells. (A) Representative western blots for SPOP levels in A498 and OS-RC-2 cells in DARTS. (B) Effect of <b>6lc</b> on the SPOP interaction with PTEN in intact HEK293T cells in the Co-IP assay. (C) Effect of <b>6lc</b> on ubiquitination of PTEN in HEK193T cells. The intensity in the western blot was read with ImageJ. All experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/medium/jm0c00161_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Effect of the SPOP inhibitor on the oncogenic SPOP-signaling pathway in kidney cancer cells. (A) Effect of <b>6lc</b> on the protein levels in the SPOP-signaling pathway in A498 and OS-RC-2 cell lines. (B) Effect of <b>6lc</b> on the transcription of <i>PTEN</i> and <i>DUSP7</i> in A498 and OS-RC-2 cell lines. (C) Effect of SPOP inhibitors on the PTEN level in OS-RC-2 cells. All experiments were performed in triplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.0c00161/20200507/images/large/jm0c00161_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00161&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i148">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51726" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51726" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricketts, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crooks, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span> <span> </span><span class="NLM_article-title">Targeting HIF2α in clear-cell renal cell carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.09.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccell.2016.09.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27728802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=515-517&author=C.+J.+Rickettsauthor=D.+R.+Crooksauthor=W.+M.+Linehan&title=Targeting+HIF2%CE%B1+in+clear-cell+renal+cell+carcinoma&doi=10.1016%2Fj.ccell.2016.09.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting HIF2α in Clear-Cell Renal Cell Carcinoma</span></div><div class="casAuthors">Ricketts, Christopher J.; Crooks, Daniel R.; Linehan, W. Marston</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">515-517</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In two recent studies, the HIF2α antagonist, PT2399, decreased HIF2α-dependent transcription and tumor growth in selected VHL-deficient clear-cell renal cell carcinoma (ccRCC) models.  These studies validate HIF2α as a therapeutic target in ccRCC, reveal variable sensitivity to HIF2α antagonism, and provide the foundation for predictive biomarker-driven clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2XxxvzJRkQbVg90H21EOLACvtfcHk0lgrHwZWmT504w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GqtLfM&md5=e77f71e456917b156a2d47e038c069b4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.09.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.09.016%26sid%3Dliteratum%253Aachs%26aulast%3DRicketts%26aufirst%3DC.%2BJ.%26aulast%3DCrooks%26aufirst%3DD.%2BR.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26atitle%3DTargeting%2520HIF2%25CE%25B1%2520in%2520clear-cell%2520renal%2520cell%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D515%26epage%3D517%26doi%3D10.1016%2Fj.ccell.2016.09.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rini, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escudier, B.</span></span> <span> </span><span class="NLM_article-title">Renal cell carcinoma</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1132</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(09)60229-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2FS0140-6736%2809%2960229-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19269025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjslOltrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=1119-1132&author=B.+I.+Riniauthor=S.+C.+Campbellauthor=B.+Escudier&title=Renal+cell+carcinoma&doi=10.1016%2Fs0140-6736%2809%2960229-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Renal cell carcinoma</span></div><div class="casAuthors">Rini, Brian I.; Campbell, Steven C.; Escudier, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">9669</span>),
    <span class="NLM_cas:pages">1119-1132</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Considerable progress has been made in the treatment of patients with renal cell carcinoma, with innovative surgical and systemic strategies revolutionizing the management of this disease.  In localized disease, partial nephrectomy for small tumors and radical nephrectomy for large tumors continue to be the gold-std. treatments, with emphasis on approaches that have reduced invasiveness and preserve renal function.  Addnl., cytoreductive nephrectomy is often indicated before the start of systemic treatment in patients with metastatic disease as part of integrated management strategy.  The effectiveness of immunotherapy, although previously widely used for treatment of metastatic renal cell carcinoma, is still controversial, and is mainly reserved for patients with good prognostic factors.  Development of treatments that have specific targets in relevant biol. pathways has been the main advance in treatment.  Targeted drugs, including inhibitors of the vascular endothelial growth factor and mammalian target of rapamycin pathways, have shown robust effectiveness and offer new therapeutic options for the patients with metastatic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2JoMfrWyvd7Vg90H21EOLACvtfcHk0lgrHwZWmT504w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjslOltrs%253D&md5=e1239eb0ddaeb1cac5ad986daaf381b7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2809%2960229-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252809%252960229-4%26sid%3Dliteratum%253Aachs%26aulast%3DRini%26aufirst%3DB.%2BI.%26aulast%3DCampbell%26aufirst%3DS.%2BC.%26aulast%3DEscudier%26aufirst%3DB.%26atitle%3DRenal%2520cell%2520carcinoma%26jtitle%3DLancet%26date%3D2009%26volume%3D373%26spage%3D1119%26epage%3D1132%26doi%3D10.1016%2Fs0140-6736%2809%2960229-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Presta, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisholm, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krummen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrara, N.</span></span> <span> </span><span class="NLM_article-title">Humanization of an anti-vascular endothelial growth factor monoclonal antibody for the therapy of solid tumors and other disorders</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4593</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=9377574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=4593&author=L.+G.+Prestaauthor=H.+Chenauthor=S.+J.+O%E2%80%99Connorauthor=V.+Chisholmauthor=Y.+G.+Mengauthor=L.+Krummenauthor=M.+Winklerauthor=N.+Ferrara&title=Humanization+of+an+anti-vascular+endothelial+growth+factor+monoclonal+antibody+for+the+therapy+of+solid+tumors+and+other+disorders"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Humanization of an anti-vascular endothelial growth factor monoclonal antibody for the therapy of solid tumors and other disorders</span></div><div class="casAuthors">Presta, Leonard G.; Chen, Helen; O'connor, Shane J.; Chisholm, Vanessa; Meng, Y. Gloria; Krummen, Lynne; Winkler, Marjorie; Ferrara, Napoleone</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4593-4599</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) is a major mediator of angiogenesis assocd. with tumors and other pathol. conditions, including proliferative diabetic retinopathy and age-related macular degeneration.  The murine anti-human VEGF monoclonal antibody (muMAb VEGF) A.4.6.1 has been shown to potently suppress angiogenesis and growth in a variety of human tumor cells lines transplanted in nude mice and also to inhibit neovascularization in a primate model of ischemic retinal disease.  In this report, we describe the humanization of muMAb VEGF A.4.6.1. by site-directed mutagenesis of a human framework.  Not only the residues involved in the six complementarity-detg. regions but also several framework residues were changed from human to murine.  Humanized anti-VEGF F(ab) and IgG1 variants bind VEGF with affinity very similar to that of the original murine antibody.  Furthermore, recombinant humanized MAb VEGF inhibits VEGF-induced proliferation of endothelial cells in vitro and tumor growth in vivo with potency and efficacy very similar to those of muMAb VEGF A.4.6.1.  Therefore, recombinant humanized MAb VEGF is suitable to test the hypothesis that inhibition of VEGF-induced angiogenesis is a valid strategy for the treatment of solid tumors and other disorders in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooXq2l9auk17Vg90H21EOLACvtfcHk0lgrHwZWmT504w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKqtLg%253D&md5=9fab64ed63a178629d4b0c45eabc05ff</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPresta%26aufirst%3DL.%2BG.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DS.%2BJ.%26aulast%3DChisholm%26aufirst%3DV.%26aulast%3DMeng%26aufirst%3DY.%2BG.%26aulast%3DKrummen%26aufirst%3DL.%26aulast%3DWinkler%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DN.%26atitle%3DHumanization%2520of%2520an%2520anti-vascular%2520endothelial%2520growth%2520factor%2520monoclonal%2520antibody%2520for%2520the%2520therapy%2520of%2520solid%2520tumors%2520and%2520other%2520disorders%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D4593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavia-Jimenez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homayoun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yell, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yousuf, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedrosa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Courtney, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zojwalla, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugarolas, J.</span></span> <span> </span><span class="NLM_article-title">Targeting renal cell carcinoma with a HIF-2 antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>539</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1038/nature19796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnature19796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27595394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyis7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=539&publication_year=2016&pages=112-117&author=W.+Chenauthor=H.+Hillauthor=A.+Christieauthor=M.+S.+Kimauthor=E.+Hollomanauthor=A.+Pavia-Jimenezauthor=F.+Homayounauthor=Y.+Maauthor=N.+Patelauthor=P.+Yellauthor=G.+Haoauthor=Q.+Yousufauthor=A.+Joyceauthor=I.+Pedrosaauthor=H.+Geigerauthor=H.+Zhangauthor=J.+Changauthor=K.+H.+Gardnerauthor=R.+K.+Bruickauthor=C.+Reevesauthor=T.+H.+Hwangauthor=K.+Courtneyauthor=E.+Frenkelauthor=X.+Sunauthor=N.+Zojwallaauthor=T.+Wongauthor=J.+P.+Rizziauthor=E.+M.+Wallaceauthor=J.+A.+Joseyauthor=Y.+Xieauthor=X.-J.+Xieauthor=P.+Kapurauthor=R.+M.+McKayauthor=J.+Brugarolas&title=Targeting+renal+cell+carcinoma+with+a+HIF-2+antagonist&doi=10.1038%2Fnature19796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting renal cell carcinoma with a HIF-2 antagonist</span></div><div class="casAuthors">Chen, Wenfang; Hill, Haley; Christie, Alana; Kim, Min Soo; Holloman, Eboni; Pavia-Jimenez, Andrea; Homayoun, Farrah; Ma, Yuanqing; Patel, Nirav; Yell, Paul; Hao, Guiyang; Yousuf, Qurratulain; Joyce, Allison; Pedrosa, Ivan; Geiger, Heather; Zhang, He; Chang, Jenny; Gardner, Kevin H.; Bruick, Richard K.; Reeves, Catherine; Hwang, Tae Hyun; Courtney, Kevin; Frenkel, Eugene; Sun, Xiankai; Zojwalla, Naseem; Wong, Tai; Rizzi, James P.; Wallace, Eli M.; Josey, John A.; Xie, Yang; Xie, Xian-Jin; Kapur, Payal; McKay, Renee M.; Brugarolas, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">539</span>
        (<span class="NLM_cas:issue">7627</span>),
    <span class="NLM_cas:pages">112-117</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC) is characterized by inactivation of the von Hippel-Lindau tumor suppressor gene (VHL).  Because no other gene is mutated as frequently in ccRCC and VHL mutations are truncal, VHL inactivation is regarded as the governing event.  VHL loss activates the HIF-2 transcription factor, and constitutive HIF-2 activity restores tumorigenesis in VHL-reconstituted ccRCC cells.  HIF-2 has been implicated in angiogenesis and multiple other processes, but angiogenesis is the main target of drugs such as the tyrosine kinase inhibitor sunitinib.  HIF-2 has been regarded as undruggable.  Here we use a tumorgraft/patient-derived xenograft platform to evaluate PT2399, a selective HIF-2 antagonist that was identified using a structure-based design approach.  PT2399 dissocd. HIF-2 (an obligatory heterodimer of HIF-2α-HIF-1β) in human ccRCC cells and suppressed tumorigenesis in 56% (10 out of 18) of such lines.  PT2399 had greater activity than sunitinib, was active in sunitinib-progressing tumors, and was better tolerated.  Unexpectedly, some VHL-mutant ccRCCs were resistant to PT2399.  Resistance occurred despite HIF-2 dissocn. in tumors and evidence of Hif-2 inhibition in the mouse, as detd. by suppression of circulating erythropoietin, a HIF-2 target and possible pharmacodynamic marker.  We identified a HIF-2-dependent gene signature in sensitive tumors.  Gene expression was largely unaffected by PT2399 in resistant tumors, illustrating the specificity of the drug.  Sensitive tumors exhibited a distinguishing gene expression signature and generally higher levels of HIF-2α.  Prolonged PT2399 treatment led to resistance.  We identified binding site and second site suppressor mutations in HIF-2α and HIF-1β, resp.  Both mutations preserved HIF-2 dimers despite treatment with PT2399.  Finally, an extensively pretreated patient whose tumor had given rise to a sensitive tumorgraft showed disease control for more than 11 mo when treated with a close analog of PT2399, PT2385.  We validate HIF-2 as a target in ccRCC, show that some ccRCCs are HIF-2 independent, and set the stage for biomarker-driven clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnxhUY1OeFy7Vg90H21EOLACvtfcHk0lhXe4ERANAmXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyis7rL&md5=006f9c774419f6226e22681286f43b6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature19796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19796%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DHill%26aufirst%3DH.%26aulast%3DChristie%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DM.%2BS.%26aulast%3DHolloman%26aufirst%3DE.%26aulast%3DPavia-Jimenez%26aufirst%3DA.%26aulast%3DHomayoun%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DYell%26aufirst%3DP.%26aulast%3DHao%26aufirst%3DG.%26aulast%3DYousuf%26aufirst%3DQ.%26aulast%3DJoyce%26aufirst%3DA.%26aulast%3DPedrosa%26aufirst%3DI.%26aulast%3DGeiger%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGardner%26aufirst%3DK.%2BH.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26aulast%3DReeves%26aufirst%3DC.%26aulast%3DHwang%26aufirst%3DT.%2BH.%26aulast%3DCourtney%26aufirst%3DK.%26aulast%3DFrenkel%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZojwalla%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DT.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.-J.%26aulast%3DKapur%26aufirst%3DP.%26aulast%3DMcKay%26aufirst%3DR.%2BM.%26aulast%3DBrugarolas%26aufirst%3DJ.%26atitle%3DTargeting%2520renal%2520cell%2520carcinoma%2520with%2520a%2520HIF-2%2520antagonist%26jtitle%3DNature%26date%3D2016%26volume%3D539%26spage%3D112%26epage%3D117%26doi%3D10.1038%2Fnature19796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.</span></span> <span> </span><span class="NLM_article-title">On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>539</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1038/nature19795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnature19795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27595393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyis73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=539&publication_year=2016&pages=107-111&author=H.+Choauthor=X.+Duauthor=J.+P.+Rizziauthor=E.+Liberzonauthor=A.+A.+Chakrabortyauthor=W.+Gaoauthor=I.+Carvoauthor=S.+Signorettiauthor=R.+K.+Bruickauthor=J.+A.+Joseyauthor=E.+M.+Wallaceauthor=W.+G.+Kaelin&title=On-target+efficacy+of+a+HIF-2%CE%B1+antagonist+in+preclinical+kidney+cancer+models&doi=10.1038%2Fnature19795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models</span></div><div class="casAuthors">Cho, Hyejin; Du, Xinlin; Rizzi, James P.; Liberzon, Ella; Chakraborty, Abhishek A.; Gao, Wenhua; Carvo, Ingrid; Signoretti, Sabina; Bruick, Richard K.; Josey, John A.; Wallace, Eli M.; Kaelin, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">539</span>
        (<span class="NLM_cas:issue">7627</span>),
    <span class="NLM_cas:pages">107-111</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma, the most common form of kidney cancer, is usually linked to inactivation of the pVHL tumor suppressor protein and consequent accumulation of the HIF-2α transcription factor (also known as EPAS1).  Here we show that a small mol. (PT2399) (I) that directly inhibits HIF-2α causes tumor regression in preclin. mouse models of primary and metastatic pVHL-defective clear cell renal cell carcinoma in an on-target fashion. pVHL-defective clear cell renal cell carcinoma cell lines display unexpectedly variable sensitivity to PT2399, however, suggesting the need for predictive biomarkers to be developed to use this approach optimally in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXNjIx5YKNerVg90H21EOLACvtfcHk0lhXe4ERANAmXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyis73F&md5=f8dc173e758b6a6a17630b791c0c318c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature19795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19795%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26aulast%3DLiberzon%26aufirst%3DE.%26aulast%3DChakraborty%26aufirst%3DA.%2BA.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DCarvo%26aufirst%3DI.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26aulast%3DJosey%26aufirst%3DJ.%2BA.%26aulast%3DWallace%26aufirst%3DE.%2BM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DOn-target%2520efficacy%2520of%2520a%2520HIF-2%25CE%25B1%2520antagonist%2520in%2520preclinical%2520kidney%2520cancer%2520models%26jtitle%3DNature%26date%3D2016%26volume%3D539%26spage%3D107%26epage%3D111%26doi%3D10.1038%2Fnature19795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuneo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittag, T.</span></span> <span> </span><span class="NLM_article-title">The ubiquitin ligase adaptor SPOP in cancer</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">3946</span>– <span class="NLM_lpage">3958</span>, <span class="refDoi"> DOI: 10.1111/febs.15056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1111%2Ffebs.15056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=31495053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslyns7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2019&pages=3946-3958&author=M.+J.+Cuneoauthor=T.+Mittag&title=The+ubiquitin+ligase+adaptor+SPOP+in+cancer&doi=10.1111%2Ffebs.15056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin ligase adaptor SPOP in cancer</span></div><div class="casAuthors">Cuneo, Matthew J.; Mittag, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3946-3958</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The dysregulation of ubiquitin-mediated proteasomal degrdn. has emerged as an important mechanism of pathogenesis in several cancers.  The speckle-type POZ protein (SPOP) functions as a substrate adaptor for the cullin3-RING ubiquitin ligase and controls the cellular persistence of a diverse array of protein substrates in hormone signalling, epigenetic control and cell cycle regulation, to name a few.  Mutations in SPOP and the resulting dysregulation of this proteostatic pathway play causative roles in the pathogenesis of prostate and endometrial cancers, whereas overexpression and mislocalization are assocd. with kidney cancer.  Understanding the mol. mechanism of the normal function of SPOP as well as the cause of SPOP-mediated oncogenesis is thus crit. for eventual therapeutic targeting of SPOP and other related pathways.  Here, we will review SPOP structure, function and the mol. mechanism of how this function is achieved.  We will then review how mutations and protein mislocalization contribute to cancer pathogenesis and will provide a perspective on how SPOP may be targeted therapeutically.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom0drBAEwmobVg90H21EOLACvtfcHk0lhXe4ERANAmXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslyns7%252FP&md5=5157444927d173c40edd7b8e1fda696b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1111%2Ffebs.15056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.15056%26sid%3Dliteratum%253Aachs%26aulast%3DCuneo%26aufirst%3DM.%2BJ.%26aulast%3DMittag%26aufirst%3DT.%26atitle%3DThe%2520ubiquitin%2520ligase%2520adaptor%2520SPOP%2520in%2520cancer%26jtitle%3DFEBS%2520J.%26date%3D2019%26volume%3D286%26spage%3D3946%26epage%3D3958%26doi%3D10.1111%2Ffebs.15056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Functional roles of speckle-type Poz (SPOP) Protein in Genomic stability</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3262</span>, <span class="refDoi"> DOI: 10.7150/jca.25930</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.7150%2Fjca.25930" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=30271484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A280%3ADC%252BB3czktl2ktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3257-3262&author=X.+Weiauthor=J.+Friedauthor=Y.+Liauthor=L.+Huauthor=M.+Gaoauthor=S.+Zhangauthor=B.+Xu&title=Functional+roles+of+speckle-type+Poz+%28SPOP%29+Protein+in+Genomic+stability&doi=10.7150%2Fjca.25930"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Functional roles of Speckle-Type Poz (SPOP) Protein in Genomic stability</span></div><div class="casAuthors">Wei Xi; Zhang Sheng; Fried Joshua; Xu Bo; Fried Joshua; Xu Bo; Li Ying; Hu Linfei; Gao Ming; Xu Bo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3257-3262</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Understanding the functional significance of the essential elements in maintaining genomic stability provides insights into the process of tumor initiation and progression, and predicts therapeutic responses.  One such element that has recently attracted significant attention is the Speckle-Type Poz Protein (SPOP), an E3 ubiquitin ligase adaptor protein.  SPOP is frequently mutated or has altered expression in various cancers, including prostate, renal and endometrial.  SPOP is involved in the regulation of proteasome-mediated degradation of several oncoproteins.  Moreover, recent data also indicate SPOP's direct involvement in the DNA damage response.  SPOP mutants induce alternations in the DNA damage repair pathway by promoting the error-prone Non-homologous end joining (NHEJ) pathway.  SPOP has been linked with significant functions in cellular signaling pathways and cancer suppression.  This mini-review will discuss recent findings regarding SPOP's role in genomic stability in the pathological setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOhFLms-L2x8AKB7Fa-xrUfW6udTcc2eYS7LEUmNGCb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czktl2ktA%253D%253D&md5=05d065f59219aa6cac4c098b984f8e2d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7150%2Fjca.25930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.25930%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DX.%26aulast%3DFried%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DFunctional%2520roles%2520of%2520speckle-type%2520Poz%2520%2528SPOP%2529%2520Protein%2520in%2520Genomic%2520stability%26jtitle%3DJ.%2520Cancer%26date%3D2018%26volume%3D9%26spage%3D3257%26epage%3D3262%26doi%3D10.7150%2Fjca.25930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mani, R.-S.</span></span> <span> </span><span class="NLM_article-title">The emerging role of speckle-type POZ protein (SPOP) in cancer development</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1498</span>– <span class="NLM_lpage">1502</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.drudis.2014.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=25058385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlWktbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1498-1502&author=R.-S.+Mani&title=The+emerging+role+of+speckle-type+POZ+protein+%28SPOP%29+in+cancer+development&doi=10.1016%2Fj.drudis.2014.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of speckle-type POZ protein (SPOP) in cancer development</span></div><div class="casAuthors">Mani, Ram-Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1498-1502</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Speckle-type POZ (pox virus and zinc finger protein) protein (SPOP) is an E3 ubiquitin ligase adaptor protein that is frequently mutated in prostate and endometrial cancers.  All the cancer-assocd. SPOP mutations reported to date are clustered in the meprin and TRAF (Tumor necrosis factor receptor-assocd. factor) homol. (MATH) domain, presumably affecting substrate binding.  SPOP mutations in prostate cancer are mutually exclusive with the ETS (Erythroblast transformation-specific) family gene rearrangements and define a distinct mol. subclass of prostate cancer.  SPOP mutations contribute to prostate cancer development by altering the steady-state levels of key components in the androgen-signaling pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_7-WBSfpBirVg90H21EOLACvtfcHk0ljK1VQaEzMIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWktbfJ&md5=da5d4ba48b523fa5d6bd8d06e5b61a78</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DR.-S.%26atitle%3DThe%2520emerging%2520role%2520of%2520speckle-type%2520POZ%2520protein%2520%2528SPOP%2529%2520in%2520cancer%2520development%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1498%26epage%3D1502%26doi%3D10.1016%2Fj.drudis.2014.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seol, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joe, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. H.</span></span> <span> </span><span class="NLM_article-title">BTB domain-containing speckle-type POZ protein (SPOP) serves as an adaptor of Daxx for ubiquitination by Cul3-based ubiquitin ligase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">12664</span>– <span class="NLM_lpage">12672</span>, <span class="refDoi"> DOI: 10.1074/jbc.m600204200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1074%2Fjbc.M600204200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=16524876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFKktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=12664-12672&author=J.+E.+Kwonauthor=M.+Laauthor=K.+H.+Ohauthor=Y.+M.+Ohauthor=G.+R.+Kimauthor=J.+H.+Seolauthor=S.+H.+Baekauthor=T.+Chibaauthor=K.+Tanakaauthor=O.+S.+Bangauthor=C.+O.+Joeauthor=C.+H.+Chung&title=BTB+domain-containing+speckle-type+POZ+protein+%28SPOP%29+serves+as+an+adaptor+of+Daxx+for+ubiquitination+by+Cul3-based+ubiquitin+ligase&doi=10.1074%2Fjbc.m600204200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BTB Domain-containing Speckle-type POZ Protein (SPOP) Serves as an Adaptor of Daxx for Ubiquitination by Cul3-based Ubiquitin Ligase</span></div><div class="casAuthors">Kwon, Jeong Eun; La, Muhnho; Oh, Kyu Hee; Oh, Young Mi; Kim, Gi Ryang; Seol, Jae Hong; Baek, Sung Hee; Chiba, Tomoki; Tanaka, Keiji; Bang, Ok Sun; Joe, Cheol O.; Chung, Chin Ha</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">12664-12672</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Daxx is a multifunctional protein that regulates a variety of cellular processes, including transcription, cell cycle, and apoptosis.  SPOP is a BTB (Bric-a-brac/Tramtrack/Broad complex) protein that constitutes Cul3-based ubiquitin ligases.  Here we show that SPOP serves as an adaptor of Daxx for the ubiquitination by Cul3-based ubiquitin ligase and subsequent degrdn. by the proteasome.  Expression of SPOP with Cul3 markedly reduced Daxx level, and this degrdn. was blocked by SPOP-specific short hairpin RNAs.  Inhibition of the proteasome by MG132 caused the prevention of Daxx degrdn. in parallel with the accumulation of ubiquitinated Daxx.  Expression of SPOP with Cul3 reversed Daxx-mediated repression of ETS1- and p53-dependent transcription, and short hairpin RNA-mediated knock down of SPOP blocked the recovery of their transcriptional activation.  Furthermore, Daxx degrdn. led to the cleavage of poly(ADP-ribose) polymerase and the increase in the no. of terminal deoxynucleotidyltransferase-mediated dUTP-fluorescein nick end-labeling-pos. apoptotic cells.  These results suggest that SPOP/Cul3-ubiquitin ligase plays an essential role in the control of Daxx level and, thus, in the regulation of Daxx-mediated cellular processes, including transcriptional regulation and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovP7jb4-r9HrVg90H21EOLACvtfcHk0ljK1VQaEzMIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFKktrY%253D&md5=ab2133948dc79c942ff1b7a794cf423d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M600204200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M600204200%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DJ.%2BE.%26aulast%3DLa%26aufirst%3DM.%26aulast%3DOh%26aufirst%3DK.%2BH.%26aulast%3DOh%26aufirst%3DY.%2BM.%26aulast%3DKim%26aufirst%3DG.%2BR.%26aulast%3DSeol%26aufirst%3DJ.%2BH.%26aulast%3DBaek%26aufirst%3DS.%2BH.%26aulast%3DChiba%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DK.%26aulast%3DBang%26aufirst%3DO.%2BS.%26aulast%3DJoe%26aufirst%3DC.%2BO.%26aulast%3DChung%26aufirst%3DC.%2BH.%26atitle%3DBTB%2520domain-containing%2520speckle-type%2520POZ%2520protein%2520%2528SPOP%2529%2520serves%2520as%2520an%2520adaptor%2520of%2520Daxx%2520for%2520ubiquitination%2520by%2520Cul3-based%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D12664%26epage%3D12672%26doi%3D10.1074%2Fjbc.m600204200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Errington, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bueler, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinstein, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabartty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Privé, G. G.</span></span> <span> </span><span class="NLM_article-title">Adaptor protein self-assembly drives the control of a cullin-RING ubiquitin ligase</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1153</span>, <span class="refDoi"> DOI: 10.1016/j.str.2012.04.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.str.2012.04.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=22632832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1WksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=1141-1153&author=W.+J.+Erringtonauthor=M.+Q.+Khanauthor=S.+A.+Buelerauthor=J.+L.+Rubinsteinauthor=A.+Chakrabarttyauthor=G.+G.+Priv%C3%A9&title=Adaptor+protein+self-assembly+drives+the+control+of+a+cullin-RING+ubiquitin+ligase&doi=10.1016%2Fj.str.2012.04.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Adaptor Protein Self-Assembly Drives the Control of a Cullin-RING Ubiquitin Ligase</span></div><div class="casAuthors">Errington, Wesley J.; Khan, M. Qasim; Bueler, Stephanie A.; Rubinstein, John L.; Chakrabartty, Avijit; Prive, Gilbert G.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1141-1153</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The E3 ligases recruit substrate proteins for targeted ubiquitylation.  Recent insights into the mechanisms of ubiquitylation demonstrate that E3 ligases can possess active regulatory properties beyond those of a simple assembly scaffold.  Here, the dimeric structure of the E3 ligase adaptor protein SPOP (speckle-type POZ protein) in complex with the N-terminal domain of Cul3 is described at 2.4 Å resoln.  It was found that SPOP forms large oligomers that can form heteromeric species with the closely related paralog SPOPL.  In combination, SPOP and SPOPL (SPOP-like) form a mol. rheostat that can fine-tune E3 ubiquitin ligase activity by affecting the oligomeric state of the E3 complex.  It is proposed that adaptor protein self-assembly provides a graded level of regulation of the SPOP/Cul3 E3 ligase toward its multiple protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCM69_zYDo17Vg90H21EOLACvtfcHk0lh3ioiUih-HGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1WksLY%253D&md5=854dbfe8ed84bbc87d6726b3bee50fcc</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2012.04.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2012.04.009%26sid%3Dliteratum%253Aachs%26aulast%3DErrington%26aufirst%3DW.%2BJ.%26aulast%3DKhan%26aufirst%3DM.%2BQ.%26aulast%3DBueler%26aufirst%3DS.%2BA.%26aulast%3DRubinstein%26aufirst%3DJ.%2BL.%26aulast%3DChakrabartty%26aufirst%3DA.%26aulast%3DPriv%25C3%25A9%26aufirst%3DG.%2BG.%26atitle%3DAdaptor%2520protein%2520self-assembly%2520drives%2520the%2520control%2520of%2520a%2520cullin-RING%2520ubiquitin%2520ligase%26jtitle%3DStructure%26date%3D2012%26volume%3D20%26spage%3D1141%26epage%3D1153%26doi%3D10.1016%2Fj.str.2012.04.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nourse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walden, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duda, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seyedin, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoggard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2009.09.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.molcel.2009.09.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19818708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2009&pages=39-50&author=M.+Zhuangauthor=M.+F.+Calabreseauthor=J.+Liuauthor=M.+B.+Waddellauthor=A.+Nourseauthor=M.+Hammelauthor=D.+J.+Millerauthor=H.+Waldenauthor=D.+M.+Dudaauthor=S.+N.+Seyedinauthor=T.+Hoggardauthor=J.+W.+Harperauthor=K.+P.+Whiteauthor=B.+A.+Schulman&title=Structures+of+SPOP-substrate+complexes%3A+insights+into+molecular+architectures+of+BTB-Cul3+ubiquitin+ligases&doi=10.1016%2Fj.molcel.2009.09.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases</span></div><div class="casAuthors">Zhuang, Min; Calabrese, Matthew F.; Liu, Jiang; Waddell, M. Brett; Nourse, Amanda; Hammel, Michal; Miller, Darcie J.; Walden, Helen; Duda, David M.; Seyedin, Steven N.; Hoggard, Timothy; Harper, J. Wade; White, Kevin P.; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-50</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">In the largest E3 ligase subfamily, Cul3 binds a BTB domain, and an assocd. protein-interaction domain such as MATH recruits substrates for ubiquitination.  Here, we present biochem. and structural analyses of the MATH-BTB protein, SPOP.  We define a SPOP-binding consensus (SBC) and det. structures revealing recognition of SBCs from the phosphatase Puc (Puckered), the transcriptional regulator Ci (Cubitus interruptus), and the chromatin component histone variant MacroH2A.  We identify a dimeric SPOP-Cul3 assembly involving a conserved helical structure C-terminal of BTB domains, which we call "3-box" due to its ability to facilitate Cul3 binding and its resemblance to F-/SOCS-boxes in other cullin-based E3s.  Structural flexibility between the substrate-binding MATH and Cul3-binding BTB/3-box domains potentially allows a SPOP dimer to engage multiple SBCs found within a single substrate, such as Puc.  These studies provide a mol. understanding of how MATH-BTB proteins recruit substrates to Cul3 and how their dimerization and conformational variability may facilitate avid interactions with diverse substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWn7mZNy7FLVg90H21EOLACvtfcHk0lh3ioiUih-HGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChtLbN&md5=65bb062fd7e17ce46bab56624f892e8e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2009.09.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2009.09.022%26sid%3Dliteratum%253Aachs%26aulast%3DZhuang%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWaddell%26aufirst%3DM.%2BB.%26aulast%3DNourse%26aufirst%3DA.%26aulast%3DHammel%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DD.%2BJ.%26aulast%3DWalden%26aufirst%3DH.%26aulast%3DDuda%26aufirst%3DD.%2BM.%26aulast%3DSeyedin%26aufirst%3DS.%2BN.%26aulast%3DHoggard%26aufirst%3DT.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructures%2520of%2520SPOP-substrate%2520complexes%253A%2520insights%2520into%2520molecular%2520architectures%2520of%2520BTB-Cul3%2520ubiquitin%2520ligases%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D36%26spage%3D39%26epage%3D50%26doi%3D10.1016%2Fj.molcel.2009.09.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghanim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iossifov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeletti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negre, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludwig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stricker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ahmadie, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tretiakova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camp, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera-Alberto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimm, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzhetsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span> <span> </span><span class="NLM_article-title">Analysis of Drosophila segmentation network identifies a JNK pathway factor overexpressed in kidney cancer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>323</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1222</span>, <span class="refDoi"> DOI: 10.1126/science.1157669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1126%2Fscience.1157669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=19164706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1elurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2009&pages=1218-1222&author=J.+Liuauthor=M.+Ghanimauthor=L.+Xueauthor=C.+D.+Brownauthor=I.+Iossifovauthor=C.+Angelettiauthor=S.+Huaauthor=N.+Negreauthor=M.+Ludwigauthor=T.+Strickerauthor=H.+A.+Al-Ahmadieauthor=M.+Tretiakovaauthor=R.+L.+Campauthor=M.+Perera-Albertoauthor=D.+L.+Rimmauthor=T.+Xuauthor=A.+Rzhetskyauthor=K.+P.+White&title=Analysis+of+Drosophila+segmentation+network+identifies+a+JNK+pathway+factor+overexpressed+in+kidney+cancer&doi=10.1126%2Fscience.1157669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Drosophila Segmentation Network Identifies a JNK Pathway Factor Overexpressed in Kidney Cancer</span></div><div class="casAuthors">Liu, Jiang; Ghanim, Murad; Xue, Lei; Brown, Christopher D.; Iossifov, Ivan; Angeletti, Cesar; Hua, Sujun; Negre, Nicolas; Ludwig, Michael; Stricker, Thomas; Al-Ahmadie, Hikmat A.; Tretiakova, Maria; Camp, Robert L.; Perera-Alberto, Montse; Rimm, David L.; Xu, Tian; Rzhetsky, Andrey; White, Kevin P.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">5918</span>),
    <span class="NLM_cas:pages">1218-1222</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The authors constructed a large-scale functional network model in Drosophila melanogaster built around two key transcription factors involved in the process of embryonic segmentation.  Anal. of the model allowed the identification of a new role for the ubiquitin E3 ligase complex factor SPOP.  In Drosophila, the gene encoding SPOP is a target of segmentation transcription factors.  Drosophila SPOP mediates degrdn. of the Jun kinase phosphatase Puckered, thereby inducing tumor necrosis factor (TNF)/Eiger-dependent apoptosis.  In humans, it was found that SPOP plays a conserved role in TNF-mediated JNK signaling and was highly expressed in 99% of clear cell renal cell carcinomas (RCCs), the most prevalent form of kidney cancer.  SPOP expression distinguished histol. subtypes of RCC and facilitated identification of clear cell RCC as the primary tumor for metastatic lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWHL_AT2Jqj7Vg90H21EOLACvtfcHk0lh3ioiUih-HGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1elurw%253D&md5=1a9d3a462740157b10d522610ebaea0c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.1157669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1157669%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DGhanim%26aufirst%3DM.%26aulast%3DXue%26aufirst%3DL.%26aulast%3DBrown%26aufirst%3DC.%2BD.%26aulast%3DIossifov%26aufirst%3DI.%26aulast%3DAngeletti%26aufirst%3DC.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DNegre%26aufirst%3DN.%26aulast%3DLudwig%26aufirst%3DM.%26aulast%3DStricker%26aufirst%3DT.%26aulast%3DAl-Ahmadie%26aufirst%3DH.%2BA.%26aulast%3DTretiakova%26aufirst%3DM.%26aulast%3DCamp%26aufirst%3DR.%2BL.%26aulast%3DPerera-Alberto%26aufirst%3DM.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DRzhetsky%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26atitle%3DAnalysis%2520of%2520Drosophila%2520segmentation%2520network%2520identifies%2520a%2520JNK%2520pathway%2520factor%2520overexpressed%2520in%2520kidney%2520cancer%26jtitle%3DScience%26date%3D2009%26volume%3D323%26spage%3D1218%26epage%3D1222%26doi%3D10.1126%2Fscience.1157669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ci, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karmakar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinker-Schaeffer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eggener, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stricker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. P.</span></span> <span> </span><span class="NLM_article-title">SPOP promotes tumorigenesis by acting as a key regulatory hub in kidney cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccr.2014.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=24656772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXks1OmsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=455-468&author=G.+Liauthor=W.+Ciauthor=S.+Karmakarauthor=K.+Chenauthor=R.+Dharauthor=Z.+Fanauthor=Z.+Guoauthor=J.+Zhangauthor=Y.+Keauthor=L.+Wangauthor=M.+Zhuangauthor=S.+Huauthor=X.+Liauthor=L.+Zhouauthor=X.+Liauthor=M.+F.+Calabreseauthor=E.+R.+Watsonauthor=S.+M.+Prasadauthor=C.+Rinker-Schaefferauthor=S.+E.+Eggenerauthor=T.+Strickerauthor=Y.+Tianauthor=B.+A.+Schulmanauthor=J.+Liuauthor=K.+P.+White&title=SPOP+promotes+tumorigenesis+by+acting+as+a+key+regulatory+hub+in+kidney+cancer&doi=10.1016%2Fj.ccr.2014.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SPOP Promotes Tumorigenesis by Acting as a Key Regulatory Hub in Kidney Cancer</span></div><div class="casAuthors">Li, Guoqiang; Ci, Weimin; Karmakar, Subhradip; Chen, Ke; Dhar, Ruby; Fan, Zhixiang; Guo, Zhongqiang; Zhang, Jing; Ke, Yuwen; Wang, Lu; Zhuang, Min; Hu, Shengdi; Li, Xuesong; Zhou, Liqun; Li, Xianghong; Calabrese, Matthew F.; Watson, Edmond R.; Prasad, Sandip M.; Rinker-Schaeffer, Carrie; Eggener, Scott E.; Stricker, Thomas; Tian, Yong; Schulman, Brenda A.; Liu, Jiang; White, Kevin P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">455-468</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Hypoxic stress and hypoxia-inducible factors (HIFs) play important roles in a wide range of tumors.  We demonstrate that SPOP, which encodes an E3 ubiquitin ligase component, is a direct transcriptional target of HIFs in clear cell renal cell carcinoma (ccRCC).  Furthermore, hypoxia results in cytoplasmic accumulation of SPOP, which is sufficient to induce tumorigenesis.  This tumorigenic activity occurs through the ubiquitination and degrdn. of multiple regulators of cellular proliferation and apoptosis, including the tumor suppressor PTEN, ERK phosphatases, the proapoptotic mol. Daxx, and the Hedgehog pathway transcription factor Gli2.  Knockdown of SPOP specifically kills ccRCC cells, indicating that it may be a promising therapeutic target.  Collectively, our results indicate that SPOP serves as a regulatory hub to promote ccRCC tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrscPybRIwJzLVg90H21EOLACvtfcHk0lgLn_EyNa-d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXks1OmsLw%253D&md5=6f271d0450f77f8e2474b2b324b4b555</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.%26aulast%3DCi%26aufirst%3DW.%26aulast%3DKarmakar%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DDhar%26aufirst%3DR.%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhuang%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DWatson%26aufirst%3DE.%2BR.%26aulast%3DPrasad%26aufirst%3DS.%2BM.%26aulast%3DRinker-Schaeffer%26aufirst%3DC.%26aulast%3DEggener%26aufirst%3DS.%2BE.%26aulast%3DStricker%26aufirst%3DT.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DK.%2BP.%26atitle%3DSPOP%2520promotes%2520tumorigenesis%2520by%2520acting%2520as%2520a%2520key%2520regulatory%2520hub%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D455%26epage%3D468%26doi%3D10.1016%2Fj.ccr.2014.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule targeting of E3 ligase adaptor SPOP in kidney cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1016%2Fj.ccell.2016.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27622336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=474-484&author=Z.-Q.+Guoauthor=T.+Zhengauthor=B.+Chenauthor=C.+Luoauthor=S.+Ouyangauthor=S.+Gongauthor=J.+Liauthor=L.-L.+Maoauthor=F.+Lianauthor=Y.+Yangauthor=Y.+Huangauthor=L.+Liauthor=J.+Luauthor=B.+Zhangauthor=L.+Zhouauthor=H.+Dingauthor=Z.+Gaoauthor=L.+Zhouauthor=G.+Liauthor=R.+Zhouauthor=K.+Chenauthor=J.+Liuauthor=Y.+Wenauthor=L.+Gongauthor=Y.+Keauthor=S.-D.+Yangauthor=X.-B.+Qiuauthor=N.+Zhangauthor=J.+Renauthor=D.+Zhongauthor=C.-G.+Yangauthor=J.+Liuauthor=H.+Jiang&title=Small-molecule+targeting+of+E3+ligase+adaptor+SPOP+in+kidney+cancer&doi=10.1016%2Fj.ccell.2016.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Targeting of E3 Ligase Adaptor SPOP in Kidney Cancer</span></div><div class="casAuthors">Guo, Zhong-Qiang; Zheng, Tong; Chen, Baoen; Luo, Cheng; Ouyang, Sisheng; Gong, Shouzhe; Li, Jiafei; Mao, Liu-Liang; Lian, Fulin; Yang, Yong; Huang, Yue; Li, Li; Lu, Jing; Zhang, Bidong; Zhou, Luming; Ding, Hong; Gao, Zhiwei; Zhou, Liqun; Li, Guoqiang; Zhou, Ran; Chen, Ke; Liu, Jingqiu; Wen, Yi; Gong, Likun; Ke, Yuwen; Yang, Shang-Dong; Qiu, Xiao-Bo; Zhang, Naixia; Ren, Jin; Zhong, Dafang; Yang, Cai-Guang; Liu, Jiang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-484</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In the cytoplasm of virtually all clear-cell renal cell carcinoma (ccRCC), speckle-type POZ protein (SPOP) is overexpressed and misallocated, which may induce proliferation and promote kidney tumorigenesis.  In normal cells, however, SPOP is located in the nucleus and induces apoptosis.  Here we show that a structure-based design and subsequent hit optimization yield small mols. that can inhibit the SPOP-substrate protein interaction and can suppress oncogenic SPOP-signaling pathways.  These inhibitors kill human ccRCC cells that are dependent on oncogenic cytoplasmic SPOP.  Notably, these inhibitors minimally affect the viability of other cells in which SPOP is not accumulated in the cytoplasm.  Our findings validate the SPOP-substrate protein interaction as an attractive target specific to ccRCC that may yield novel drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSo7OKJgIr7Vg90H21EOLACvtfcHk0lgLn_EyNa-d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK&md5=cd349f46b9ae34c0158fb268666c4c9e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DZ.-Q.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.-L.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-D.%26aulast%3DQiu%26aufirst%3DX.-B.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.-G.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSmall-molecule%2520targeting%2520of%2520E3%2520ligase%2520adaptor%2520SPOP%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D474%26epage%3D484%26doi%3D10.1016%2Fj.ccell.2016.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, L.</span></span> <span> </span><span class="NLM_article-title">Kidney cancer: On target - inhibiting SPOP in ccRCC</span>. <i>Nat. Rev. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">630</span>, <span class="refDoi"> DOI: 10.1038/nrurol.2016.192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1038%2Fnrurol.2016.192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1CgsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=630&author=L.+Stone&title=Kidney+cancer%3A+On+target+-+inhibiting+SPOP+in+ccRCC&doi=10.1038%2Fnrurol.2016.192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Kidney cancer On target - inhibiting SPOP in ccRCC</span></div><div class="casAuthors">Stone, Louise</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Urology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">630</span>CODEN:
                <span class="NLM_cas:coden">NRNADQ</span>;
        ISSN:<span class="NLM_cas:issn">1759-4812</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kidney cancer On target - inhibiting SPOP in ccRCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4Pw-GS9Hk5rVg90H21EOLACvtfcHk0lgLn_EyNa-d3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1CgsbfK&md5=302e1d2a0424b728d86b078b707ea5d3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrurol.2016.192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrurol.2016.192%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DL.%26atitle%3DKidney%2520cancer%253A%2520On%2520target%2520-%2520inhibiting%2520SPOP%2520in%2520ccRCC%26jtitle%3DNat.%2520Rev.%2520Urol.%26date%3D2016%26volume%3D13%26spage%3D630%26doi%3D10.1038%2Fnrurol.2016.192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-G.</span></span> <span> </span><span class="NLM_article-title">Targeting SPOP with small molecules provides a novel strategy for kidney cancer therapy</span>. <i>Sci. China: Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1007/s11427-016-0297-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1007%2Fs11427-016-0297-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=27888385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A280%3ADC%252BC2sjgsFemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=91-93&author=T.+Zhengauthor=C.-G.+Yang&title=Targeting+SPOP+with+small+molecules+provides+a+novel+strategy+for+kidney+cancer+therapy&doi=10.1007%2Fs11427-016-0297-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting SPOP with small molecules provides a novel strategy for kidney cancer therapy</span></div><div class="casAuthors">Zheng Tong; Yang Cai-Guang; Zheng Tong</div><div class="citationInfo"><span class="NLM_cas:title">Science China. Life sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTKDDAd1lUQJGhHGZei4OW9fW6udTcc2eaPqnncTlOkXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjgsFemsg%253D%253D&md5=a1b798136386d3c7e171142817f99e97</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs11427-016-0297-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11427-016-0297-2%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DC.-G.%26atitle%3DTargeting%2520SPOP%2520with%2520small%2520molecules%2520provides%2520a%2520novel%2520strategy%2520for%2520kidney%2520cancer%2520therapy%26jtitle%3DSci.%2520China%253A%2520Life%2520Sci.%26date%3D2017%26volume%3D60%26spage%3D91%26epage%3D93%26doi%3D10.1007%2Fs11427-016-0297-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2529</span>– <span class="NLM_lpage">2591</span>, <span class="refDoi"> DOI: 10.1021/jm1013693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design&doi=10.1021%2Fjm1013693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0lig6a2ORT51dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591%26doi%3D10.1021%2Fjm1013693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">St
Jean, D. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotsch, C.</span></span> <span> </span><span class="NLM_article-title">Mitigating heterocycle metabolism in drug discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6002</span>– <span class="NLM_lpage">6020</span>, <span class="refDoi"> DOI: 10.1021/jm300343m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300343m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFWhsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6002-6020&author=D.+J.+St%0AJeanauthor=C.+Fotsch&title=Mitigating+heterocycle+metabolism+in+drug+discovery&doi=10.1021%2Fjm300343m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mitigating Heterocycle Metabolism in Drug Discovery</span></div><div class="casAuthors">St. Jean, David J.; Fotsch, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6002-6020</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Extensive data from metab. studies have allowed medicinal chemists to develop general principles for reducing compd. metab.  These methods include, but are not limited to, reducing lipophilicity, altering sterics and electronics, introducing a conformational constraint, and altering the stereochem. of their compds.  While no single method is able to solve every metabolic problem, these principles do give medicinal chemists guidance on how to improve the metabolic liabilities of their compds.  If the specific site of metab. is known, medicinal chemists block the site, typically with a fluorine, or replace the metabolically labile group with a bioisostere.  While several authors have reviewed these techniques for reducing metab., there is no review that summarizes different approaches to improving them metabolic stability of heterocycles.  In this review, we summarize examples where changes were made at or near the heterocycle to improve metabolic stability.  By summarizing these examples, we hope to provide a useful guide to medicinal chemists as they attempt to improve the metabolic profile of their own heterocyclic compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovQjdBBgXtPbVg90H21EOLACvtfcHk0lig6a2ORT51dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFWhsL0%253D&md5=48c262556b8a2cc7dc5539ff91a53a71</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm300343m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300343m%26sid%3Dliteratum%253Aachs%26aulast%3DSt%2BJean%26aufirst%3DD.%2BJ.%26aulast%3DFotsch%26aufirst%3DC.%26atitle%3DMitigating%2520heterocycle%2520metabolism%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6002%26epage%3D6020%26doi%3D10.1021%2Fjm300343m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jindal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devadoss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and pharmacological evaluation of novel 2-(4-substituted piperazin-1-yl)1, 8 naphthyridine 3-carboxylic acids as 5-HT3 receptor antagonists for the management of depression</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1111/cbdd.12370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1111%2Fcbdd.12370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=24903617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSqur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2014&pages=721-731&author=A.+K.+Dharauthor=R.+Maheshauthor=A.+Jindalauthor=T.+Devadossauthor=S.+Bhatt&title=Design%2C+synthesis%2C+and+pharmacological+evaluation+of+novel+2-%284-substituted+piperazin-1-yl%291%2C+8+naphthyridine+3-carboxylic+acids+as+5-HT3+receptor+antagonists+for+the+management+of+depression&doi=10.1111%2Fcbdd.12370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Pharmacological Evaluation of Novel 2-(4-substituted piperazin-1-yl)-1,8-Naphthyridine-3-Carboxylic Acids as 5-HT3 Receptor Antagonists for the Management of Depression</span></div><div class="casAuthors">Dhar, Arghya K.; Mahesh, Radhakrishnan; Jindal, Ankur; Devadoss, Thangaraj; Bhatt, Shvetank</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-731</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">1,8-Naphthyridine-3-carboxylic acid analogs I (R = Me, Ph, Bn, 4-ClC6H4, etc.) were synthesized and found to possess potential 5-HT3 receptor antagonism as well as antidepressant-like activity.  Initially, 5-HT3 receptor antagonism of all the compds. was detd. in the form of pA2 value against agonist 2-Me 5-HT in longitudinal muscle-myenteric plexus prepn. from guinea-pig ileum.  Among all the compds. tested, compd. I (R = Ph) demonstrated most promising pA2 value of 7.6.  Subsequently, all compds. were evaluated for antidepressant activity using forced swim test and tail suspension test in mice.  Compds. I (R = Ph, 4-O2NC6H4, 4-ClC6H4, 2-MeOC6H4, 4-MeOC6H4) exhibited significant (p < 0.05) antidepressant-like activity as compd. to vehicle-treated group.  Importantly, none of the tested compd. affected locomotor activity of mice at tested dose levels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaIEfpC3IQVLVg90H21EOLACvtfcHk0ljGLECD4LrFdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSqur%252FJ&md5=b4f1e51a923fa9ced4e98389d50cc228</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12370%26sid%3Dliteratum%253Aachs%26aulast%3DDhar%26aufirst%3DA.%2BK.%26aulast%3DMahesh%26aufirst%3DR.%26aulast%3DJindal%26aufirst%3DA.%26aulast%3DDevadoss%26aufirst%3DT.%26aulast%3DBhatt%26aufirst%3DS.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520pharmacological%2520evaluation%2520of%2520novel%25202-%25284-substituted%2520piperazin-1-yl%25291%252C%25208%2520naphthyridine%25203-carboxylic%2520acids%2520as%25205-HT3%2520receptor%2520antagonists%2520for%2520the%2520management%2520of%2520depression%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2014%26volume%3D84%26spage%3D721%26epage%3D731%26doi%3D10.1111%2Fcbdd.12370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saccomanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adinolfi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benetti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligresti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascio, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuccinardi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Marzo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrarini, P. L.</span></span> <span> </span><span class="NLM_article-title">Rational design, synthesis, and pharmacological properties of new 1,8-naphthyridin-2(1H)-on-3-carboxamide derivatives as highly selective cannabinoid-2 receptor agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">3644</span>– <span class="NLM_lpage">3651</span>, <span class="refDoi"> DOI: 10.1021/jm801563d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801563d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlslOqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=3644-3651&author=C.+Maneraauthor=G.+Saccomanniauthor=B.+Adinolfiauthor=V.+Benettiauthor=A.+Ligrestiauthor=M.+G.+Cascioauthor=T.+Tuccinardiauthor=V.+Lucchesiauthor=A.+Martinelliauthor=P.+Nieriauthor=E.+Masiniauthor=V.+Di+Marzoauthor=P.+L.+Ferrarini&title=Rational+design%2C+synthesis%2C+and+pharmacological+properties+of+new+1%2C8-naphthyridin-2%281H%29-on-3-carboxamide+derivatives+as+highly+selective+cannabinoid-2+receptor+agonists&doi=10.1021%2Fjm801563d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design, Synthesis, and Pharmacological Properties of New 1,8-Naphthyridin-2(1H)-on-3-carboxamide Derivatives as Highly Selective Cannabinoid-2 Receptor Agonists</span></div><div class="casAuthors">Manera, Clementina; Saccomanni, Giuseppe; Adinolfi, Barbara; Benetti, Veronica; Ligresti, Alessia; Cascio, Maria Grazia; Tuccinardi, Tiziano; Lucchesi, Valentina; Martinelli, Adriano; Nieri, Paola; Masini, Emanuela; Di Marzo, Vincenzo; Ferrarini, Pier Luigi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3644-3651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The CB2 receptor activation can be exploited for the treatment of diseases such as chronic pain and tumors of immune origin, devoid of psychotropic activity.  On the basis of our already reported 1,8-naphthyridin-4(1H)-on-3-carboxamide derivs., new 1,8-naphthyridin-2(1H)-on-3-carboxamide derivs. were designed, synthesized, and tested for their affinities toward the human CB1 and CB2 cannabinoid receptors.  Some of the reported compds. showed a subnanomolar CB2 affinity with a CB1/CB2 selectivity ratio greater than 200.  Further studies revealed that the compd. which presented benzyl and carboxy-4-methylcyclohexylamide substituents bound in the 1 and 3 positions exerted a CB2-mediated inhibitory action on immunol. human basophil activation.  On the human T cell leukemia line Jurkat the same deriv. induced a concn.-dependent decrease of cell viability.  The obtained results suggest that 1,8-naphthyridin-2(1H)-on-3-carboxamides represent a new scaffold very suitable for the development of new promising CB2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcHJZnJ6fZcbVg90H21EOLACvtfcHk0ljGLECD4LrFdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlslOqsLY%253D&md5=ca24de2d70f151057cf749ee9ff5151f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm801563d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801563d%26sid%3Dliteratum%253Aachs%26aulast%3DManera%26aufirst%3DC.%26aulast%3DSaccomanni%26aufirst%3DG.%26aulast%3DAdinolfi%26aufirst%3DB.%26aulast%3DBenetti%26aufirst%3DV.%26aulast%3DLigresti%26aufirst%3DA.%26aulast%3DCascio%26aufirst%3DM.%2BG.%26aulast%3DTuccinardi%26aufirst%3DT.%26aulast%3DLucchesi%26aufirst%3DV.%26aulast%3DMartinelli%26aufirst%3DA.%26aulast%3DNieri%26aufirst%3DP.%26aulast%3DMasini%26aufirst%3DE.%26aulast%3DDi%2BMarzo%26aufirst%3DV.%26aulast%3DFerrarini%26aufirst%3DP.%2BL.%26atitle%3DRational%2520design%252C%2520synthesis%252C%2520and%2520pharmacological%2520properties%2520of%2520new%25201%252C8-naphthyridin-2%25281H%2529-on-3-carboxamide%2520derivatives%2520as%2520highly%2520selective%2520cannabinoid-2%2520receptor%2520agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D3644%26epage%3D3651%26doi%3D10.1021%2Fjm801563d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruce, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardew, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freitag-Pohl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, E.</span></span> <span> </span><span class="NLM_article-title">How to stabilize protein: stability screens for thermal shift assays and nano differential scanning fluorimetry in the virus-X project</span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">58666</span>, <span class="refDoi"> DOI: 10.3791/58666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.3791%2F58666" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2019&pages=58666&author=D.+Bruceauthor=E.+Cardewauthor=S.+Freitag-Pohlauthor=E.+Pohl&title=How+to+stabilize+protein%3A+stability+screens+for+thermal+shift+assays+and+nano+differential+scanning+fluorimetry+in+the+virus-X+project&doi=10.3791%2F58666"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3791%2F58666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F58666%26sid%3Dliteratum%253Aachs%26aulast%3DBruce%26aufirst%3DD.%26aulast%3DCardew%26aufirst%3DE.%26aulast%3DFreitag-Pohl%26aufirst%3DS.%26aulast%3DPohl%26aufirst%3DE.%26atitle%3DHow%2520to%2520stabilize%2520protein%253A%2520stability%2520screens%2520for%2520thermal%2520shift%2520assays%2520and%2520nano%2520differential%2520scanning%2520fluorimetry%2520in%2520the%2520virus-X%2520project%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2019%26volume%3D144%26spage%3D58666%26doi%3D10.3791%2F58666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pai, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomenick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiestl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span> <span> </span><span class="NLM_article-title">Drug affinity responsive target stability (DARTS) for small-molecule target identification</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1263</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2269-7_22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=10.1007%2F978-1-4939-2269-7_22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=25618353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls12murk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1263&publication_year=2015&pages=287-298&author=M.+Y.+Paiauthor=B.+Lomenickauthor=H.+Hwangauthor=R.+Schiestlauthor=W.+McBrideauthor=J.+A.+Looauthor=J.+Huang&title=Drug+affinity+responsive+target+stability+%28DARTS%29+for+small-molecule+target+identification&doi=10.1007%2F978-1-4939-2269-7_22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Affinity Responsive Target Stability (DARTS) for Small-Molecule Target Identification</span></div><div class="casAuthors">Pai, Melody Y.; Lomenick, Brett; Hwang, Heejun; Schiestl, Robert; McBride, William; Loo, Joseph A.; Huang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1263</span>
        (<span class="NLM_cas:issue">Chemical Biology</span>),
    <span class="NLM_cas:pages">287-298</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Drug affinity responsive target stability (DARTS) is a relatively quick and straightforward approach to identify potential protein targets for small mols.  It relies on the protection against proteolysis conferred on the target protein by interaction with a small mol.  The greatest advantage of this method is being able to use the native small mol. without having to immobilize or modify it (e.g., by incorporation of biotin, fluorescent, radioisotope, or photoaffinity labels).  Here we describe in detail the protocol for performing unbiased DARTS with complex protein lysates to identify binding targets of small mols. and for using DARTS-Western blotting to test, screen, or validate potential small-mol. targets.  Although the ideas have mainly been developed from studying mols. in areas of biol. that are currently of interest to us and our collaborators, the general principles should be applicable to the anal. of all mols. in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZkyptImVfKbVg90H21EOLACvtfcHk0ljGLECD4LrFdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls12murk%253D&md5=17d9d8199173653f814d80e0945c94ad</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2269-7_22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2269-7_22%26sid%3Dliteratum%253Aachs%26aulast%3DPai%26aufirst%3DM.%2BY.%26aulast%3DLomenick%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DH.%26aulast%3DSchiestl%26aufirst%3DR.%26aulast%3DMcBride%26aufirst%3DW.%26aulast%3DLoo%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DJ.%26atitle%3DDrug%2520affinity%2520responsive%2520target%2520stability%2520%2528DARTS%2529%2520for%2520small-molecule%2520target%2520identification%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1263%26spage%3D287%26epage%3D298%26doi%3D10.1007%2F978-1-4939-2269-7_22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i144"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00161">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55742"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00161?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00161</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Properties of SPOP inhibitors and synthetic methods and analytical data for target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_001.pdf">jm0c00161_si_001.pdf (4.12 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00161/suppl_file/jm0c00161_si_002.csv">jm0c00161_si_002.csv (5.23 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00161&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00161%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00161" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679abb8d9fc13c70","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
