
<html lang="en"     class="pb-page"  data-request-id="e27a7dcc-6334-4204-866c-eddac0dd93fd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-7;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b00261;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A" /></meta><meta name="dc.Creator" content="John R.  Horton" /></meta><meta name="dc.Creator" content="Xu  Liu" /></meta><meta name="dc.Creator" content="Lizhen  Wu" /></meta><meta name="dc.Creator" content="Kai  Zhang" /></meta><meta name="dc.Creator" content="John  Shanks" /></meta><meta name="dc.Creator" content="Xing  Zhang" /></meta><meta name="dc.Creator" content="Ganesha  Rai" /></meta><meta name="dc.Creator" content="Bryan T.  Mott" /></meta><meta name="dc.Creator" content="Daniel J.  Jansen" /></meta><meta name="dc.Creator" content="Stephen C.  Kales" /></meta><meta name="dc.Creator" content="Mark J.  Henderson" /></meta><meta name="dc.Creator" content="Katherine  Pohida" /></meta><meta name="dc.Creator" content="Yuhong  Fang" /></meta><meta name="dc.Creator" content="Xin  Hu" /></meta><meta name="dc.Creator" content="Ajit  Jadhav" /></meta><meta name="dc.Creator" content="David J.  Maloney" /></meta><meta name="dc.Creator" content="Matthew D.  Hall" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Haian  Fu" /></meta><meta name="dc.Creator" content="Paula M.  Vertino" /></meta><meta name="dc.Creator" content="Qin  Yan" /></meta><meta name="dc.Creator" content="Xiaodong  Cheng" /></meta><meta name="dc.Description" content="Isomers of chiral drugs can exhibit marked differences in biological activities. We studied the binding and inhibitory activities of 12 compounds against KDM5A. Among them are two pairs of enantiom..." /></meta><meta name="Description" content="Isomers of chiral drugs can exhibit marked differences in biological activities. We studied the binding and inhibitory activities of 12 compounds against KDM5A. Among them are two pairs of enantiom..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 14, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00261" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00261" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00261" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00261" /></link>
        
    
    

<title>Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00261" /></meta><meta property="og:title" content="Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0009.jpeg" /></meta><meta property="og:description" content="Isomers of chiral drugs can exhibit marked differences in biological activities. We studied the binding and inhibitory activities of 12 compounds against KDM5A. Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compounds N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compounds N54 and N55). In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity. The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity. We determined structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity. These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00261"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00261">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00261&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00261&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00261&amp;href=/doi/10.1021/acs.jmedchem.8b00261" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3193-3208</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00209" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00343" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">John R. Horton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John R. Horton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+R.++Horton">John R. Horton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xu++Liu">Xu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lizhen Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lizhen Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lizhen++Wu">Lizhen Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kai Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kai Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kai++Zhang">Kai Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Shanks</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Shanks</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Shanks">John Shanks</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xing Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xing Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xing++Zhang">Xing Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ganesha Rai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ganesha Rai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ganesha++Rai">Ganesha Rai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bryan T. Mott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bryan T. Mott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bryan+T.++Mott">Bryan T. Mott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel J. Jansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel J. Jansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.++Jansen">Daniel J. Jansen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen C. Kales</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen C. Kales</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+C.++Kales">Stephen C. Kales</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark J. Henderson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark J. Henderson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+J.++Henderson">Mark J. Henderson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine Pohida</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine Pohida</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Pohida">Katherine Pohida</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuhong Fang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuhong Fang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuhong++Fang">Yuhong Fang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Hu">Xin Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ajit Jadhav</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajit Jadhav</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajit++Jadhav">Ajit Jadhav</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David J. Maloney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Maloney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Maloney">David J. Maloney</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5607-4178" title="Orcid link">http://orcid.org/0000-0002-5607-4178</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Hall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Hall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Hall">Matthew D. Hall</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Simeonov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Simeonov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haian Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haian Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology,  Department of Hematology and Medical Oncology,  Emory Chemical Biology Discovery Center,  The Winship Cancer Institute, Emory University, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haian++Fu">Haian Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paula M. Vertino</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paula M. Vertino</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Winship Cancer Institute,  Department of Radiation Oncology, Emory University, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paula+M.++Vertino">Paula M. Vertino</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qin Yan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qin Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Q.Y.: E-mail, <a href="/cdn-cgi/l/email-protection#e4958d8aca9d858aa49d858881ca818091"><span class="__cf_email__" data-cfemail="f7869e99d98e9699b78e969b92d9929382">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qin++Yan">Qin Yan</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xiaodong Cheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaodong Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><span class="conrtib-corresp"><strong>*</strong>X.C.: E-mail, <a href="/cdn-cgi/l/email-protection#225a414a474c4517624f46434c464750514d4c0c4d5045"><span class="__cf_email__" data-cfemail="bbc3d8d3ded5dc8efbd6dfdad5dfdec9c8d4d595d4c9dc">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaodong++Cheng">Xiaodong Cheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6967-6362" title="Orcid link">http://orcid.org/0000-0002-6967-6362</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00261&amp;href=/doi/10.1021%2Facs.jmedchem.8b00261" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3193–3208</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 14, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 February 2018</li><li><span class="item_label"><b>Published</b> online</span>14 March 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 April 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00261" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00261</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3193%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BR.%2BHorton%252C%2BXu%2BLiu%252C%2BLizhen%2BWu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D7%26contentID%3Dacs.jmedchem.8b00261%26title%3DInsights%2Binto%2Bthe%2BAction%2Bof%2BInhibitor%2BEnantiomers%2Bagainst%2BHistone%2BLysine%2BDemethylase%2B5A%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3208%26publicationDate%3DApril%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00261"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1670</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00261" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;R. Horton&quot;},{&quot;first_name&quot;:&quot;Xu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Lizhen&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Kai&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Shanks&quot;},{&quot;first_name&quot;:&quot;Xing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Ganesha&quot;,&quot;last_name&quot;:&quot;Rai&quot;},{&quot;first_name&quot;:&quot;Bryan&quot;,&quot;last_name&quot;:&quot;T. Mott&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. Jansen&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;C. Kales&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;J. Henderson&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Pohida&quot;},{&quot;first_name&quot;:&quot;Yuhong&quot;,&quot;last_name&quot;:&quot;Fang&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Ajit&quot;,&quot;last_name&quot;:&quot;Jadhav&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Maloney&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Hall&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Haian&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Paula&quot;,&quot;last_name&quot;:&quot;M. Vertino&quot;},{&quot;first_name&quot;:&quot;Qin&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Xiaodong&quot;,&quot;last_name&quot;:&quot;Cheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;3193-3208&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00261&quot;},&quot;abstract&quot;:&quot;Isomers of chiral drugs can exhibit marked differences in biological activities. We studied the binding and inhibitory activities of 12 compounds against KDM5A. Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compounds N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compounds N54 and N55). In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity. The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity. We determined structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity. These res&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00261&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00261" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00261&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00261" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00261&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00261" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00261&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00261&amp;href=/doi/10.1021/acs.jmedchem.8b00261" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00261" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00261" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (11 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00261%26sid%3Dliteratum%253Aachs%26pmid%3D29537847%26genre%3Darticle%26aulast%3DHorton%26date%3D2018%26atitle%3DInsights%2Binto%2Bthe%2BAction%2Bof%2BInhibitor%2BEnantiomers%2Bagainst%2BHistone%2BLysine%2BDemethylase%2B5A%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D7%26spage%3D3193%26epage%3D3208%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/jmcmar.2018.61.issue-7/20180412/jmcmar.2018.61.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Isomers of chiral drugs can exhibit marked differences in biological activities. We studied the binding and inhibitory activities of 12 compounds against KDM5A. Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (<i>R</i>)- and (<i>S</i>)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid (compounds <b>N51</b> and <b>N52</b>) and (<i>R</i>)<i>-</i> and (<i>S</i>)<i>-N</i>-(1-(3-isopropyl-1<i>H</i>-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compounds <b>N54</b> and <b>N55</b>). In vitro, the <i>S</i> enantiomer of the <b>N51</b>/<b>N52</b> pair (<b>N52</b>) and the <i>R</i> enantiomer of the <b>N54</b>/<b>N55</b> pair (<b>N54</b>) exhibited about 4- to 5-fold greater binding affinity. The more potent enzyme inhibition of KDM5A by the <i>R</i>-isoform for the cell-permeable <b>N54</b>/<b>N55</b> pair translated to differences in growth inhibitory activity. We determined structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity. These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17200" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17200" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone lysine methylation is involved in a wide range of biological processes including gene expression, chromatin organization, dosage compensation, and epigenetic memory and is often compromised in cancers (reviewed in ref <a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a>). Histone methylation can be transcriptionally repressive or activating, depending on the position and identity (lysine or arginine) of the methylated residue and the extent of its methylation (i.e., monomethylated, dimethylated, or trimethylated).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Methylation of lysine residue 4, 36, or 79 of histone H3 is typically associated with active transcription, while methylation of lysine 9 or 27 of histone H3 and lysine 20 of histone H4 contributes to repression.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Along with other histone modifications such as acetylation, phosphorylation, and ubiquitination, other regulatory factors, and DNA cytosine modifications, these modifications ultimately determine chromatin conformation and the local potential for gene expression.</div><div class="NLM_p">To enable changes in chromatin structure and gene expression, histone modifications have to be dynamic. Although protein extracts containing lysine demethylase activity were identified and partially purified as early as the 1960s (reviewed in ref <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a>), the first protein lysine-specific demethylase 1 (LSD1, also known as KDM1A), a flavin-dependent amine oxidase, was identified nearly a half century later.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This was immediately followed by the discovery of a second and larger class of Jumonji-domain containing demethylases:<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> a family of >20 Fe(II) and α-ketoglutarate (αKG)-dependent dioxygenases (KDM2–8). Histone demethylases have since become important therapeutic targets, particularly in human cancers where these enzymes are frequently mutated and/or misregulated.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">Histone H3 lysine 4 (H3K4) methylation is typically associated with regions of accessible chromatin, including gene promoters and enhancers. There are two different enzyme families that act on methylated H3K4; LSD1/KDM1 removes methyl groups from low-degree (mono- or di-) methylated H3K4<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (reviewed in ref <a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a>), whereas JARID1/KDM5 removes methyl groups from the higher-degree (tri- or di-) methylated H3K4 forms<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (reviewed in ref<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a>). The KDM5 family is unique among histone demethylases in that each member contains an atypical split catalytic domain wherein the insertion of a DNA-binding ARID and a histone-interacting PHD1 domain separates the Jumonji domain into two segments, JmjN and JmjC (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1A in Supporting Information</a>). Although all four members of the KDM5 family catalyze the demethylation of the same histone mark, they have different tissue expression profiles and are present in distinct protein complexes, suggesting different cellular functions.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a></div><div class="NLM_p">The KDM5 family has been implicated in the pathogenesis of several types of cancer and plays a role in drug resistance and in metastasis.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> For example, KDM5A is amplified or overexpressed in breast cancer,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> lung cancer,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> hepatocellular carcinoma,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and gastric cancers.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> KDM5A has been linked to the control of proliferation and senescence by antagonizing the functions of retinoblastoma protein<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> and through direct repression of cyclin-dependent kinase inhibitor genes.<a onclick="showRef(event, 'ref15 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref17 ref18">(15,17,18)</a> Knockout of KDM5A significantly prolonged survival in genetically engineered mouse tumor models.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Moreover, KDM5A plays a role in the epithelial–mesenchymal transition<a onclick="showRef(event, 'ref16 ref24'); return false;" href="javascript:void(0);" class="ref ref16 ref24">(16,24)</a> and the suppression of invasion<a onclick="showRef(event, 'ref15 ref23'); return false;" href="javascript:void(0);" class="ref ref15 ref23">(15,23)</a> and metastasis.<a onclick="showRef(event, 'ref23 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref25">(23,25)</a> Increased levels of KDM5A have been shown to confer anticancer drug resistance in lung cancer,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> breast cancer,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and glioblastoma.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> KDM5B is similarly overexpressed in a variety of cancers, including breast cancer and others,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> and is associated with a poor prognosis, chemoresistance, and metastasis.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34 ref35">(30−35)</a></div><div class="NLM_p">Given the mounting evidence from human tumors and model systems supporting a role for the KDM5A and KDM5B as oncogenic drivers, extensive efforts have been devoted to develop KDM5 family specific inhibitors. Several cell-permeable inhibitors have been identified (<b>KDM5-C70</b> (patent WO2014053491), <b>CPI-455</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and <b>KDOAM-25</b><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>) and have been shown to alter global levels of H3K4me3 and to inhibit cancer cell growth, thus serving as the basis for further development of therapeutic agents that combat KDM5 oncogenic potential. We and others have studied the binding modes of chemically diverse KDM5 inhibitors in complex with KDM5A and KDM5B demethylases.<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a> We noted that some of the compounds that inhibit enzyme activity in vitro were racemic mixtures, and thus we were intrigued by the opportunity to identify stereospecific inhibitors of KDM5 and to understand the structural basis for that selectivity.</div><div class="NLM_p">The individual enantiomers of chiral drugs can exhibit marked differences in biological activity, and thus detailed characterization of the individual isomers is critical.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> With regard to αKG-dependent dioxygenases in particular, cancer-associated mutations in the isocitrate dehydrogenases IDH1 and IDH2 alter the function of these enzymes such that instead of catalyzing the oxidative decarboxylation of isocitrate to αKG, they go on to produce the (<i>R</i>)-enantiomer of 2-hydroxyglutarate (2HG),<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> a weak inhibitor of many αKG-dependent dioxygenases.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> Interestingly, whereas the (<i>S</i>)-isomer is a more potent in vitro inhibitor of most of the αKG-dependent enzymes tested, only (<i>R</i>)-2HG exhibits cellular activity, inducing cytokine independence and blocking differentiation of hematopoietic cells<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and targeting FTO N6-methyladenine RNA demethylase in leukemic cells,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> suggesting that enantiomeric inhibitors of the broader class of αKG-dependent dioxygenases may have distinct in vitro versus in vivo activities.</div><div class="NLM_p last">Here we study the structural features and growth inhibitory properties of two pairs of enantiomeric KDM5 inhibitors, namely, (<i>R</i>)- and (<i>S</i>)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid (compounds <b>N51</b> and <b>N52</b>) and (<i>R</i>)<i>-</i> and (<i>S</i>)<i>-N</i>-(1-(3-isopropyl-1<i>H</i>-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compounds <b>N54</b> and <b>N55</b>). We find that, in vitro, the <i>S</i> isoform of the <b>N51</b>/<b>N52</b> pair (<b>N52</b>) and the <i>R</i> isoform of the <b>N54</b>/<b>N55</b> pair (<b>N54</b>) exhibited greater binding and more potent inhibition of KDM5A. For the cell-permeable <b>N54/N55</b> pair, this difference in potency translated to a difference in cells wherein the (<i>R</i>)-N54 showed enhanced accumulation of H3K4me3 and greater growth inhibition.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We previously defined the minimal requirements for in vitro enzymatic activity of KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical Zn-binding domain.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Deletion of internal ARID and PHD1 domains (ΔAP) between JmjN and JmjC as well as the region C-terminal to the Zn-binding domain generated KDM5A(1–739)ΔAP, which was used for enzyme activity studies (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1C</a>). The protein produced without the Zn-binding domain, KDM5A(1–588)ΔAP, was used for compound binding assays and structural studies (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1D</a>). Recombinant proteins were affinity-purified from <i>Escherichia coli</i> using an N-terminal His-SUMO-tag, which was subsequently removed as described.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Structural comparisons with a longer construct of KDM5A,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> which comprises the entire N-terminal half (including ARID, PHD1, and the C-terminal Zn helical domain) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1B</a>), showed that the deletion of the ARID-PHD1 domains does not interfere with either the folding of the linked JmjN-JmjC domain or the conformation of its active site. The chemically related compounds <b>CPI-455</b> and <b>N8</b> bind in the active site of the two KDM5A constructs in exactly the same manner (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1E</a>). The shorter construct with the linked JmjN-JmjC domain provides an opportunity for studying numerous KDM5A demethylase inhibitors at near atomic resolution by X-ray crystallography.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Structure–Function Relationships among KDM5-Directed Inhibitors</h3><div class="NLM_p">Among the 12 compounds we examined, 9 contain a 1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid (pyridine-7-carboxylic acid) moiety and 2-chlorophenyl moiety connected to a variable length of alkoxyether (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B). Compounds <b>N9</b>, <b>N40</b>, <b>N41</b>, <b>N42</b>, <b>N46</b>, <b>N47</b>, and <b>N48</b> are racemic mixtures containing the same chiral secondary carbon atom connected by the three different substituents and a hydrogen. The racemic mixture of compound <b>N40</b> was separated by chiral chromatography (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Supporting Information methods</a>), yielding two enantiomers <b>N51</b> and <b>N52</b> (the assignment of <i>R</i>- and <i>S</i>-enantiomers is based on the observed electron densities in the final refined structures). Using the purified minimal catalytic domain KDM5A(1–588)ΔAP, we measured the dissociation constants (<i>K</i><sub>D</sub>) by isothermal titration calorimetry (ITC) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S2</a>). Among the mixtures, <b>N9</b> (patent WO2014164708), which lacks a piperidine ring, has an approximately 3-fold weaker binding affinity relative to <b>N40</b>. Compounds with piperidine (<b>N40</b> or <b>N48</b>), difluoropiperidine (<b>N41</b>), or pyrrolidine (<b>N46</b>) substitutions have similar binding affinities in the range of 0.15–0.25 μM. This indicates that the identity of the basic substituents at the end of alkoxyether and the length of alkyl linker has a relatively small impact on binding affinity and is tolerant of structural modification. The substitution of pyrrolidine ring in <b>N46</b> with an imidazole ring in <b>N47</b> reduced the binding affinity by ∼3-fold. Shortening the alkoxyether extension from two-carbon to one-carbon reduced the binding affinity by ∼2-fold (<b>N41</b> and <b>N42</b>), whereas an increase in the alkyl chain length to three carbons did not affect binding affinity (<b>N40</b> and <b>N48</b>). Importantly, among the compounds examined, the pure <i>S</i>-enantiomer <b>N52</b> exhibited the highest binding affinity with a <i>K</i><sub>D</sub> of 60 nM, approximately 4-fold better than that of the <i>R-</i>enantiomer <b>N51</b>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding of KDM5A by compound N9 and its derivatives. (A) Chemical structures of <b>N9</b> related compounds containing a constant pyridine-7-carboxylic acid moiety and 2-chlorophenyl moiety connected to a variable hydroxymethyl extension. (B) Chemical structures of (<i>R</i>)- and (<i>S</i>)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid (compounds <b>N51</b> and <b>N52</b>). (C) Summary of isothermal titration calorimetry measurement of dissociation constants (<i>K</i><sub>D</sub>) of various compounds to KDM5A(1–588)ΔAP (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S2</a> for original binding data). (D) Summary of the IC<sub>50</sub> values of inhibition of demethylation for KDM5A (residues 1–1090) by AlphaLISA (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S3</a> for original inhibition data). (E) Summary of inhibitor-bound X-ray structures of KDM5A(1–588)ΔAP (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a> for detailed statistics of X-ray diffraction and refinement).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We noted that the <b>N9</b>-based compounds (with a pyridine-7-carboxylate moiety) are intrinsically fluorescent which interferes with the fluorescence signal detected in the formaldehyde dehydrogenase (FDH)-coupled demethylase assay. We thus employed an AlphaLISA method to assess potency of these compounds on enzyme inhibition. This assay utilizes an antibody against methylated H3K4me3 to measure the conversion (disappearance) of trimethylated lysine 4 of histone H3 peptide substrate (biotinylated H3(1–21)K4me3) and a commercially available recombinant human KDM5A large fragment (residues 1–1090) with a C-terminal FLAG-tag (BPS Bioscience). The inhibitory activities in this assay (IC<sub>50</sub> values) correlated reasonably well with the relative binding affinities for the KDM5A linked Jumonji domain construct, as revealed by the pairwise comparisons between <b>N9</b> and <b>N40</b>, <b>N41</b> and <b>N42</b>, <b>N46</b> and <b>N47</b>, and <b>N54</b> and <b>N55</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). However, there are some significant differences between the two assays. For example, the ∼4-fold difference in dissociation constants between the two enantiomers <b>N51</b> and <b>N52</b> was not reflected in the enzyme inhibitory assays, where the two enantiomers displayed similar IC<sub>50</sub> values (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). The similar <i>K</i><sub>D</sub> values observed among <b>N40</b> (<i>K</i><sub>D</sub> = 0.18 μM), <b>N51</b> (0.22), <b>N41</b> (0.21), <b>N46</b> (0.15), and <b>N48</b> (0.25) corresponded well with the nearly equivalent IC<sub>50</sub> values with the exception of <b>N41</b>, which demonstrated reduced inhibitory activity (by approximately 6- to 15-fold) compared to other compounds: <b>N40</b> (IC<sub>50</sub> = 0.69 μM), <b>N51</b> (0.67), <b>N41</b> (4.5), <b>N46</b> (0.29), and <b>N48</b> (0.43). The inconsistency between the binding affinity and inhibitory constants might be due to the different KDM5A constructs used in the respective assays. The deleted internal domains (ARID, PHD1, and C-terminal Zn helical domain) might contribute to the interaction with the inhibitors in the active site that were not observed in the binding assays, although alternative explanations exist (see <a class="ref internalNav" href="#sec3" aria-label="Discussion">Discussion</a>). In addition, potent compounds (with IC<sub>50</sub> values of <1 μM) against KDM5A exhibited significant selectivity against the KDM4A catalytic domain by a factor of 20–50 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S3B</a>), whereas compound <b>N42</b>, with the least inhibition against KDM5A (IC<sub>50</sub> = 22 μM), demonstrated no selectivity between the two enzymes. We note that more comprehensive selectivity screening against other histone demethylases (and the larger family of dioxygenases) will be performed in the near future.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Structures of KDM5A Linked Jumonji Domain in Complex with Nine Inhibitors Containing Pyridinecarboxylate Scaffold</h3><div class="NLM_p">We determined the structures of nine different inhibitors in ternary complex with KDM5A(1–588)ΔAP and Mn(II)  (instead of Fe(II)) to the resolution range of 1.22–1.99 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a>). Compounds were soaked into preformed crystals of KDM5A complexed with αKG and thus had to displace bound αKG to be visible in the active site. The inhibitor-bound KDM5A Jumonji domains are similar in overall structure to the αKG-bound form; for example, there was a root-mean-squared deviation of 0.2 Å across 289 pairs of Cα atoms between the <b>N46</b>-bound structure of KDM5A (at the highest resolution of 1.22 Å) and that of αKG-bound at 1.39 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVB">5IVB</a>). Like other structurally characterized αKG-dependent dioxygenases,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> the metal ion was bound by six ligands in an octahedral coordination. The side chains of His483, Glu485, and His571 occupy three of the coordination sites (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), while the pyridine ring nitrogen atom and two water molecules provide the other three ligands (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of ternary complex of <b>N46</b>-KDM5A-Mn(II) at 1.22 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>). (A) Six metal ligands involving three protein residues, two water molecules, and the ring nitrogen atom of pyridine. The omit electron densities, contoured at 10σ and 5σ above the mean, are shown for Mn(II) (magenta mesh) and compound <b>N46</b> (gray mesh), respectively. (B) The carboxylate group of pyridine-7-carboxylic acid moiety forms ionic and hydrogen bonding interactions with Lys501 and Tyr409. (C) Extensive van der Waals contacts formed between pyridine ring and 2-chlorophenyl ring of <b>N46</b> and aromatic residues of KDM5A in the active site. (D) The chlorine atom of 2-chlorophenyl ring binds in a hydrophobic pocket formed by Ala411, Tyr409, and Tyr472.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The pyridinecarboxylic acid portion of <b>N46</b> occupies the αKG binding site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), with the carboxylate group forming an extensive H-bond network with Lys501, Tyr409, and water-mediated interactions similar to those normally engaged by αKG.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The carboxylic acids of the other eight pyridine-based compounds (<b>N9</b>, <b>N40</b>, <b>N51</b>, <b>N52</b>, <b>N41</b>, <b>N42</b>, <b>N47</b>, and <b>N48</b>) exhibited nearly identical interactions with the protein residues (Tyr409, Asn575, Lys501, Asn493, and Ser491) involved in direct and indirect contacts (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–I). This finding underscores the critical nature of these interactions for defining the binding mode for this class of inhibitors and clearly illustrates why any modification of the carboxylic acid reduces inhibitory potency significantly, as was observed previously when comparing the structures of the catalytic domain bound by cofactor αKG or inhibitors containing an isonicotinic acid moiety.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In addition to these polar interactions with Lys501 and Tyr409, the aromatic pyridine ring is engaged extensively in face-to-face π–π interactions with Tyr472, Phe480 and an edge-to-edge interaction with Trp503 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural snapshots of <b>N9</b> related compounds bound into KDM5A active site. (A–I) The pyridine-7-carboxylic acid moiety involves in conserved interactions with Lys501, Asn575, and Tyr409 (top left corner of each panel). The omit electron densities, contoured at 5σ above the mean, are shown for each inhibitor (gray mesh). (A) In compound <b>N9</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU">6BGU</a>), the relative positions of 2-chlorophenyl moiety and the propoxymethyl group are switched in comparison with other compounds containing larger chemical part attached to the end of hydroxymethyl extension. (B) In the racemic mixture of compound <b>N40</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>), two possible conformations of 2-chlorophenyl moiety can be modeled into the electron density. (C, D) In the purified enantiomers (<i>R</i>)-<b>N51</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>) and (<i>S</i>)-<b>N52</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), the chlorine atom of the chlorophenyl ring points to opposite directions. (E–I) For the other five racemic mixtures of compounds, <b>N41</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW">6BGW</a>), <b>N42</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX">6BGX</a>), <b>N46</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>), <b>N47</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ">6BGZ</a>), and <b>N48</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5">6BH5</a>), the (<i>R</i>)-<b>N52</b>-like conformation dominates. (J) Superimposition of compounds <b>N9</b> and <b>N46</b>. (K) Superimposition of compounds <b>N41</b> and <b>N42</b>. Note the inhibitor-specific interaction with Trp470 (compound <b>N41</b>) and Asp486 (compound <b>N42</b>). (L) Superimposition of compounds <b>N46</b> and <b>N47</b>. Note the conformational change of Asp412.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Moving away from the αKG binding region (occupied by the pyridinecarboxylic acid), the 2-chlorophenyl in compound <b>N9</b> (in <i>R</i> conformation) points to an open solvent channel with the phenyl ring apparently not well-ordered (as indicated by the poor electron density). Meanwhile the propoxyl group forms van der Waals contacts (interatomic distance varying between 3.4 and 3.8 Å) with the guanidine group of Arg73, Tyr409, and Ala411 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). This binding site appears relatively tight, suggesting that additional substitutions on the propoxyl would not be tolerated, and explains why the remaining eight pyridine-based analogs, all of which have a larger alkoxyether extension, adopt a different orientation in this binding site. For these compounds, we observe that these two substituents on the chiral carbon (the 2-chlorophenyl and the alkoxyether extension) have switched positions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B–J).</div><div class="NLM_p">Knowing that compound <b>N40</b> is a racemic mixture, we modeled the 2-chlorophenyl into the electron density in two conformations (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). With the structures of purified enantiomers <b>N51</b> and <b>N52</b>, the electron density is well-defined, allowing us to unambiguously position the chlorine atom of the chlorophenyl ring pointing either to solvent (<b>N51</b>) or into the protein (<b>N52</b>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). The chlorine atom makes extensive van der Waals contacts with Ala411, Tyr409, and Tyr472 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). These Cl interactions with the protein presumably contribute to the 4-fold stronger binding by <b>N52</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Superimposition of the three structures (<b>N40</b>, <b>N51</b>, and <b>N52</b>) shows the density for <b>N40</b> is consistent with being a mixture of the two enantiomers (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). And the dissociation constant of <b>N40</b> is between that measured for each of the individual enantiomers (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). For the other five racemic mixtures of chlorophenyl-containing compounds (<b>N41</b>, <b>N42</b>, <b>N46</b>, <b>N47</b>, and <b>N48</b>), the electron density mapped well with the (<i>S</i>)-<b>N52</b>-like conformation with the chlorine atom interacting with Ala411 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F–J), suggesting the preferential binding of this enantiomer to the protein.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chiral pairs of (<i>R</i>)-<b>N51</b> and (<i>S</i>)-<b>N52</b> bind in the KDM5A active site. (A) Superimposition of racemic mixture of compound <b>N40</b> (magenta; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>) and the purified enantiomers (<i>R</i>)-<b>N51</b> (cyan; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>) and (<i>S</i>)-<b>N52</b> (gold; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>). Note the pyridine-7-carboxylate moiety is superimposable. (B, C) In (<i>R</i>)-<b>N51</b>, Y472 forms two hydrogen bonds with the compound. In (<i>S</i>)-<b>N52</b>, Asp412 forms an ionic interaction with the compound. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron densities, contoured at 1.2σ, are shown for the inhibitor and immediate neighboring residues. (D) Superimposition of (<i>R</i>)-<b>N51</b> and (<i>S</i>)-<b>N52</b>. Note the conformational change of Asp412.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, we observed two different side-chain conformations of Asp412, as well as a shift of ∼2 Å in the main-chain Cα atom when comparing the <b>N46</b>-bound structure with that of <b>N47</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>L). In the pyrrolidine-containing <b>N46</b> structure, Asp412 bridges between Arg73 and Tyr472, whereas in the imidazole-containing <b>N47</b> structure, this Asp412-Tyr472 interaction breaks down. These observations may explain the different <i>K</i><sub>D</sub> values measured for <b>N46</b> and <b>N47</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).</div><div class="NLM_p last">In summary, each of the three substitutions on the chiral carbon has a different impact on the interactions with KDM5A; the planar ring of 1<i>H</i>-pyrrolopyridine-7-carboxylic acid occupies the highly conserved, αKG-binding region of the catalytic site, linking metal binding to the Lys501-Asn575-Tyr409 network. These interactions are essential for the potency of the inhibitors and are likely the primary determinants that anchor the compound’s orientation. The 2-chlorophenyl moiety, particularly the chlorine atom, makes extensive van der Waals contacts with Ala411, Tyr409, and Tyr472. Finally, the alkoxyether extension allows a variety of different interactions specific for each inhibitor, with Asp412 being a particularly interesting site.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structures of Chiral Pair (<i>R</i>)-<b>N51</b> and (<i>S</i>)-<b>N52</b></h3><div class="NLM_p last">We overlaid the two structures of <b>N51</b> and <b>N52</b> in the KDM5A binding pocket (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). While the 2-chlorophenyl moiety and ethoxypiperidine branch are nonsuperimposable, the 1<i>H</i>-pyrrolopyridine-7-carboxylate moiety is. We find that, besides the 1<i>H</i>-pyrrolopyridine-7-carboxylate moiety, the other two components of the enantiomers have different interactions with KDM5A. As noted above, there are extensive hydrophobic interactions with the chlorine atom in <b>N52</b> and that are absent in <b>N51</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). In addition, the nitrogen atom of the piperidine ring in <b>N51</b> is within H-bond distance to the hydroxyl oxygen atom of Tyr472 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In contrast, the corresponding nitrogen atom in <b>N52</b> interacts with one of the carboxylate oxygen atoms of Asp412 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The six-membered piperidine ring in <b>N52</b> could carry a formal charge if the ring nitrogen is protonated such that the close contact with the negatively charged Asp412 may enhance the binding affinity of <b>N52</b> with KDM5A (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). These differences in the protein–ligand interaction are likely responsible for the ∼4-fold difference in binding affinity between <b>N51</b> and <b>N52</b>. Interestingly, we observed a conformational change of Asp412 (involving both main-chain and side-chain atoms) upon binding of <b>N51</b> versus <b>N52</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), which is analogous to the difference observed between compounds <b>N47</b> and <b>N46</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>L). In both cases, interaction with Asp412 (Asp412-<b>N52</b> or Asp412-<b>N46</b>) seems to correlate with stronger binding affinity (lower <i>K</i><sub>D</sub> values).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> The Methyl Ester Derivatives of Pyridinecarboxylic Acid Are Cell Inactive</h3><div class="NLM_p last">Some previously identified KDM inhibitors, including <b>GSK-J1</b><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and <b>KDM5-C49</b> (patent WO2014053491), are acidic, displayed a competitive mechanism of inhibition relative to cofactor αKG in vitro, and are cell impermeable.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> Here we observe a similar binding mode for the compounds containing the 1<i>H</i>-pyrrolopyridine-7-carboxylic acid, wherein the distal carboxylic acid displays strong ionic and hydrogen bonding interactions with Lys501 and Tyr409 in the active site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). In the cases of <b>GSK-J1</b> and <b>KDM5-C49</b>, the respective ethyl ester derivatives, <b>GSK-J4</b> and <b>KDM5-C70</b>, were designed as cell-permeable prodrugs that are hydrolyzed by unknown esterase(s) within the cell to generate active <b>GSK-J1</b> and <b>KDM5-C49</b>, respectively. By analogy, we generated methyl ester derivatives of <b>N40</b>, <b>N41</b>, and <b>N46</b> to test their activity in cells. However, exposure of BT474 breast cancer cells to any of these at concentrations up to 38 μM had no impact on cellular H3K4me3 methylation levels, whereas <b>KDM5-C70</b> did (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S4</a>). The lack of cellular activity is probably a result of the combination of potentially reduced cellular penetration and less efficient hydrolysis of these methyl ester derivatives in the cell.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Enantiomers (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b> Adopt Similar Binding Modes</h3><div class="NLM_p">Compounds <b>N54</b> and <b>N55</b> represent a second pair of chiral KDM5 inhibitors based on a completely different molecular scaffold. The compounds are notable for being uncharged, without a carboxylate group (patent WO2016057924).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The chiral center at the C3-position on the pyrrolidine ring is connected to the nitrogen of a cyclopropylamide (C3–N bond in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). ITC data showed that the dissociation constant (<i>K</i><sub>D</sub>) for binding to KDM5A is more than 5-fold lower for <b>N54</b> than <b>N55</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S2</a>). The difference in binding affinity between the two enantiomers correlated well with the relative inhibitory activity (IC<sub>50</sub> values), and the potency of inhibition was dependent on αKG concentration (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S5A</a>), suggesting that the inhibitors compete with αKG for binding. We note that the approximately 5- to 7-fold difference in IC<sub>50</sub> values between <b>N54</b> and <b>N55</b> against KDM5A from two different sources, i.e., our purified KDM5A(1–739)ΔAP and the commercially available KDM5A(1–1090)-Flag, was similar and independent of the method used to determine inhibitory activity (FDH-coupled and AlphaLISA) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b> bind highly similar in the KDM5A active site. (A) Chemical structures of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b>. (B) Inhibition of demethylation of KDM5A(1–739)ΔAP by (<i>R</i>)-<b>N54</b> via FDH-coupled assay at three concentrations of αKG (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5A</a> for inhibition data by (<i>S</i>)-<b>N55</b>). (C) Structure of <b>N54</b>-KDM5A-Mn(II) showing the binding of <b>N54</b> compound in the active site involving two metal ligands (upper right corner), a direct hydrogen bond with Asn575 (left), and aromatic interactions via Phe480 (top), Tyr472 (bottom), and Trp503 (in the background away from the viewer). See <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5B</a> for structure of (<i>S</i>)-<b>N55</b> bound in the active site of KDM5A. (D) The terminal cyclopropyl group forms a stacking interaction with Tyr409 (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5C</a> for corresponding interaction in (<i>S</i>)-<b>N55</b>). (E) The terminal isopropyl methyl groups form van der Waals interactions with Trp470, Asn585, and Val584 (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5C</a> for corresponding interaction in (<i>S</i>)-<b>N55</b>). (F) Superimposition of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b>. (G, H) Superimposition of compounds <b>N52</b> (dark magenta) and <b>N54</b> (yellow) illustrates conformational changes of Tyr409 (G) and Trp780 (H). (I) An ordered glycerol molecule was observed near the inhibitor, forming direct H-bond interaction with the inhibitor. (J) A surface representation of KDM5A active site showing the binding of <b>N54</b> compound together with a glycerol molecule. (K) A surface representation of KDM5A showing the binding of compound <b>N52</b> occupying a much larger space in the active site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We determined structures of <b>N54</b>- and <b>N55</b>-bound KDM5A, at resolutions of 1.45 and 1.70 Å, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a>). The unambiguous electron density helped define the absolute stereochemistry of the two enantiomers. Both compounds occupy the αKG-binding site as expected and provide two ligands to the metal via a pyrazole ring nitrogen and an adjacent amide carbonyl oxygen atom (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S5B,C</a>). The second pyrazole ring nitrogen (NH group) forms a H-bond with Glu485 (one of the three protein metal ligands). Like a peptide bond, the partial double bond character makes the amide carbonyl and pyrazole ring coplanar. The terminal isopropyl groups form van der Waals interactions with side chains of Trp470 and Asn585 and main-chain carbon of Val584 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S5D</a>). On the other end of the molecule, the carbonyl oxygen of the cyclopropylcarboxamide forms a H-bond network with Lys501 and Asn575 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S5C</a>). The cyclopropyl group forms a stacking interaction with Tyr409. Again, the six atoms associated with the peptide-like bond (amide nitrogen −NH and carbonyl oxygen −C═O) are planar. The central, five-membered pyrrolidine ring containing the chiral center at C3 is surrounded by aromatic residues forming a three-sided cage, with Phe480 on the ceiling, Tyr472 on the floor, and Trp503 on the back wall (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S5B</a>).</div><div class="NLM_p">Superimposition of the bound conformations of the two enantiomers revealed that the pyrrolidine ring rotates so that the chiral carbon C3 of the second enantiomer now occupies the position of ring carbon C4 in the structure of the first enantiomer (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). During the course of our study, Liang et al.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> published a structure of (<i>R</i>)-<b>N54</b> bound with KDM5A(12–800) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V9T">5V9T</a>), in which the entire N-terminal half of KDM5A was used for cocrystallization (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S6A</a>). Superimposition of the two (<i>R</i>)-<b>N54</b>-bound KDM5A structures demonstrates that positioning of (<i>R</i>)-<b>N54</b> is nearly identically in the two structures, being unaffected by the absence or presence of the internal ARID-PHD1 (PHD1 is disordered in the structure) or the helical Zn-domain (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S6B,C</a>); no additional interactions with residues outside of the linked JmjN-JmjC domain were observed.</div><div class="NLM_p">Two key interacting aromatic residues (Tyr409 and Trp780) were found to undergo substantial inhibitor-induced conformational changes. In the 1<i>H</i>-pyrrolopyridine-7-carboxylate based inhibitor<b>s</b>, the side chain Asn575 bridges between Lys501 and Tyr409, the latter forming a hydrogen bond with the inhibitor carboxylate (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–I). By contrast, in the <b>N54</b> complex, the side chain of Tyr409 is pushed away by the cyclopropyl group, rotating nearly 90° as compared to its position in the <b>N52</b>-bound structure (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G), and forms a van der Waals contact with the cyclopropyl group (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Similarly, the indole ring of Trp780 rotates approximately 90° from the piperidine-interacting conformation (<b>N52</b>) to adopt an orientation that enables interaction with the isopropyl group (<b>N54</b>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H). In both cases, the alternative positioning involves the π-stacking forces, which could be exploited to design inhibitor specific interactions.</div><div class="NLM_p last">A highly ordered glycerol molecule, used as a cryoprotectant during crystal freezing, is observed near the inhibitor in both the <b>N54</b> and <b>N55</b> structures (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I). The glycerol molecule forms a direct H-bond interaction with the amide nitrogen that is bound to the chiral C3 carbon and side chain of Asp412, as well as an indirect interaction with Arg73 and a network of water-mediated interactions with Tyr409 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I). Design of new inhibitors that expand to occupy this solvent-filled space, and pick up increased protein–ligand interactions, is an avenue for potentially increasing the potency of KDM5 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>J). Indeed, the most potent 1<i>H</i>-pyrrolopyridine-7-carboxylate based compound <b>N52</b> occupies a greater volume (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>K) and exhibited stronger binding than <b>N54</b> by a factor of 40 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Superimposition of <b>N52</b>-bound and <b>N54</b>-bound structures revealed that compound <b>N52</b> occupies one-third larger volume in the active site (390 Å<sup>3</sup>) than that of <b>N54</b> (290 Å<sup>3</sup>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Cellular Effect of Compounds (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b></h3><div class="NLM_p">We next examined whether the difference in biochemical activities of enantiomers <b>N54</b> and <b>N55</b> is also reflected in a cellular setting. We treated several cell lines representing different subtypes of breast cancer, including BT474 and SKBR3 (HER2+), MCF7 (ER+), MDA-MB231 (triple negative for ER, PR, and HER2) cells with <b>N54</b> and <b>N55</b> at concentrations ranging from 0.1 to 10 μM. Global levels of trimethylated lysine 4 of histone H3 (H3K4me3), the KDM5 substrate, were elevated to varying degrees across the cell lines in response to both <b>N54</b> and <b>N55</b>, with <b>N54</b> being ∼10-fold more potent than <b>N55</b> at driving similar levels of H3K4me3 accumulation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Consistent with the idea that <b>N54</b> and <b>N55</b> are specific for KDM5,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a><b>N54</b> or <b>N55</b> treatment showed minimal effects on other histone methylation marks, including H3K9me3 (substrates for the KDM4 family) and H3K27me3 (substrate for the KDM6 family) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). To further compare the effects of <b>N54</b> and <b>N55</b> on the growth of breast cancer cells, we performed colony formation assays using MCF7 cells. Consistent with our previous studies,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a><b>KDM5-C70</b> at 1 μM significantly inhibited the growth of MCF7 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C,D). <b>N54</b> at 10 μM showed similar growth inhibitory activity to 1 μM <b>KDM5-C70</b>, whereas <b>N55</b> was ineffective even at 10 μM, consistent with a relative difference in potency between <b>N54</b> and <b>N55</b> in inducing global H4K4me3 levels.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (<i>R</i>)-Enantiomer <b>N54</b> is more active than (<i>S</i>)-enantiomer <b>N55</b> in cellular assays. (A, B) Western blot analyses of (A) SKBR3 and BT474 cells, (B) MCF7 and MDA-MB231 cells treated with the indicated concentration of <b>N54</b>, <b>N55</b>, <b>KDM5-C70</b>, or DMSO control for 3 days. Representative data from biological duplicate experiments performed in duplicate are shown. (C, D) Colony-formation assays of MCF-7 cells treated with DMSO, <b>KDM5-C70</b> (1 μM), and <b>N54</b> and <b>N55</b> (1 or 10 μM). Shown are (C) representative images and (D) quantification of biological triplicate experiments. “Intensity %” indicates the measured intensity of cell confluency using ImageJ. Error bars denote SEM: (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The impact of KDM5 inhibitors on breast cancer cell proliferation was further assessed by live-cell imaging in the presence of 0.1–50 μM compound. Both <b>N54</b> and <b>N55</b> inhibited BT474 growth but had no effect on MCF7, SKBR3, or MDA-MB231 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A–D). Notably, the growth rates for the four lines were variable, with doubling times ranging from 32 h (MDA-MB231) to 149 h (BT474). Interestingly, <b>N54</b>/<b>N55</b> showed the greatest effects on the growth of BT474 cells with <b>N54</b> being somewhat (∼2-fold) more potent than <b>N55</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E,F). Consistent with our previous studies,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> the ability of a KDM5 inhibitor to affect global H3K4me3 levels correlated poorly with its growth inhibitory activity; treatment with both <b>N54</b>, and to a lesser degree <b>N55</b>, resulted in an increase in H3K4me3 levels in all cell lines while the impact on growth varied considerably. The discordance in MCF7 growth inhibitory response as measured in bulk cultures over 4 days (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) versus in colony formation assays (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) may arise from the latter being a more sensitive measure of inhibition, requiring that cells survive plating at low density and then proliferate from a single cell over 12 days (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Taken together, these data are consistent with the in vitro biochemical data showing that (<i>R</i>)-<b>N54</b> is the more potent than its enantiomer (<i>S</i>)-<b>N55</b> in cells and further underscore the necessity for both biochemical and cell-based assays in the development of KDM5 inhibitors.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Proliferation of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b> treated breast cancer lines. <b>N54</b> and <b>N55</b> inhibit proliferation of (A) BT474 but not (B) MCF7, (C) SKBR3, or (D) MDA- MB231. Growth curves represent percent confluence (mean ± SD, <i>n</i> = 3). (E) <b>N54</b> is a more potent inhibitor of BT474 proliferation than <b>N55</b>. AUC for growth (% confluence over 288 h) is plotted versus inhibitor concentration. (F) Micrographs at 240 h post-treatment, with the confluence mask shown in yellow. Scale bar is 300 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02388" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">KDM5 family enzymes (particularly KDM5A and KDM5B) have been studied in several types of cancer and cancer-related processes including cell proliferation, drug resistance, and metastasis.<a onclick="showRef(event, 'ref11 ref12 ref53'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref53">(11,12,53)</a> Knockout of KDM5A significantly prolonged survival in genetically engineered mouse tumor models.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> These studies set the foundation that KDM5A inhibition could be exploited for cancer treatment. Our extensive efforts to characterize the molecular interactions of KDM5A with structurally diverse inhibitors provide important insights into structure–activity relationships for KDM5 inhibitors<a onclick="showRef(event, 'ref38 ref49'); return false;" href="javascript:void(0);" class="ref ref38 ref49">(38,49)</a> (this study).</div><div class="NLM_p">Here we present two pairs of enantiomers representing two distinct inhibitor chemotypes, compounds <b>N51</b> and <b>N52</b> [(<i>R</i>)- and (<i>S</i>)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid] and compounds <b>N54</b> and <b>N55</b> [(<i>R</i>)<i>-</i> and (<i>S</i>)<i>-N</i>-(1-(3-isopropyl-1<i>H</i>-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide]. In vitro, the <i>S</i> enantiomer of <b>N51</b>/<b>N52</b> and the <i>R</i> isoform of <b>N54</b>/<b>N55</b> exhibited more potent binding by approximately a factor of 4–5. As noted above, there was a discrepancy for compounds <b>N51</b> and <b>N52</b> between the binding affinity <i>K</i><sub>D</sub> values measured by using the linked JmjN-JmjC domain of KDM5A(1–588)ΔAP wherein there was a ∼4-fold difference in the binding affinities (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), yet the enzyme inhibitory activities (IC<sub>50</sub> values) measured using the commercially available KDM5A(1–1090)-Flag were comparable (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). To further investigate, we examined the crystal structures of a larger fragment of KDM5A(12–800) published by Vinogradova et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and Liang et al.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> for additional interactions that might explain the observed difference. Like (<i>R</i>)-<b>N54</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S6</a>), two related compounds, <b>CPI-455</b>, in complex with KDM5A(12–800),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and <b>N8</b>, in complex with KDM5A(1–588)ΔAP,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> showed no additional interactions with residues outside of the linked JmjN-JmjC domain (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S1E</a>). Another explanation may be variation between batches of commercial enzymes used in AlphaLISA, as we noted in our previous study<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (comparing Flag-KDM5B-FL in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">S4B</a> in Horton et al., 2016).</div><div class="NLM_p">For the cell-permeable <b>N54/N55</b> pair the differences observed in binding affinity and in vitro enzyme inhibitory activity translated to a difference in cellular potency, with the <i>R</i>-isoform (<b>N54</b>) exhibiting a greater ability to inhibit KDM5, as indicated by the accumulation of intracellular H3K4me3 levels, and a somewhat enhanced growth inhibitory activity relative to the <i>S</i>-isoform (<b>N55</b>). Interestingly, while there was some concordance between the relative ability of the two isoforms to influence intracellular H3K4me3 levels and to impact cell growth within individual cell lines (e.g., <b>N54</b> more potent than <b>N55</b>), there appeared to be little relationship between treatment-induced H3K4me3 accumulation and growth inhibitory effects of the two compounds across cell lines. Indeed, consistent with our previous studies on <b>KDM5-C70</b>,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> treatment with <b>N54</b>, or to a lesser extent <b>N55</b>, resulted in the accumulation of H3K4me3 across all cell lines, an indication that it reaches its intracellular target, but the dose-dependence of this effect had little relationship to growth inhibitory activity. These data suggest that there are additional cellular factors beyond H3K4me3 levels that contribute to differential effects of KDM5 inhibition on cell growth.</div><div class="NLM_p">While our study should aid in the successful design of selective and potent epigenetic inhibitors of KDM5 demethylases in vitro (for example, (<i>S</i>)-<b>N52</b> exhibited approximately 40 fold stronger binding than (<i>R</i>)-<b>N54</b>), a remaining challenge is to make these compounds cell permeable. As mentioned above, <b>KDM5-C70</b> was designed as a cell-permeable prodrug that is hydrolyzed by intracellular esterase(s) to generate <b>KDM5-C49</b>, which contains an isonicotinic acid moiety with a carboxylic acid (PCT WO 2014053491).<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> Like (<i>S</i>)-<b>N52</b> and the related compounds examined in this study, the terminal carboxylic acid group forms an extensive hydrogen bonding network with Lys501, Tyr409, and Asn454, similar to those normally engaged by αKG (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A–C). The hydroxyl of Tyr409 forms two hydrogen bonds, accepting a proton from the amide group of Asn454 and donating a proton to the carboxylate group of the inhibitor. The low permeability is likely to be due to the strong acidic nature of the carboxylate moiety, or overall compound polarity, and certain modifications (for example, an ethyl ester derivative) were needed to improve cell permeability of these compounds. Subsequently, several cell-permeable compounds were discovered that lack a carboxylate moiety, including <b>CPI-455</b>,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (<i>R</i>)-<b>N54</b>,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and <b>KDOAM-25</b><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> which simply replaces carboxylate of <b>KDM5-C49</b> with a carboxyamide (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D–F). We also crystallized the KDM5A-linked Jumonji domain in complex with <b>KDOAM-25</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a> and <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>E), which has a similar dissociation constant (<i>K</i><sub>D</sub> = 2.7 μM) and half maximal inhibitory concentration at 1 mM αKG (IC<sub>50</sub> = 18 μM) as that of (<i>R</i>)-<b>N54</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S7</a>). The common feature of these neutral, carboxylate-absent compounds, like (<i>R</i>)-<b>N54</b>, is that Tyr409 undergoes a series of conformational changes upon the binding of inhibitor (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>G,H). For the carboxylic acid containing inhibitors or αKG, the side chain Asn575 bridges between Lys501 and Tyr409, which form hydrogen bonds with the carboxylic group. By contrast, in the <b>KDOAM-25</b> complex, the side chain of Tyr409 is rotated away approximately 30° from that of the <b>KDM5-C49</b>-bound form (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>G). An ordered water molecule occupies the position of the hydroxyl oxygen atom of Tyr409, thus maintaining H-bond interaction between two amide groups of Asn575 and <b>KDOAM-25</b>, both of which function as proton donors. In the <b>CPI-455</b> and (<i>R</i>)-<b>N54</b> bound conformation, the side chain of Tyr409 is rotated nearly 90°, resulting in a van der Waals contact with the isopropyl moiety (<b>CPI-455</b>) or the cyclopropyl group (<b>N54</b>), respectively. It seems that the conformation of Tyr409 is the largest difference observed in KDM5A after binding of these two inhibitor scaffolds, likely explained by the presence or absence of the carboxylate group, and thus, a common feature among these three cell permeable compounds that lack a carboxylate moiety. These results suggest that further development of KDM5 inhibitors of less acidity could benefit from taking advantage of the inhibitor-induced flexibility of Tyr409. Furthermore, to make more cell penetrant compounds, one could potentially take advantage of adding moieties such as a furyl ring or thienyl moiety.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitor specific Tyr409 conformations. (A–C) Tyr409 adopts a nearly identical conformation via the H-bond interaction with the carboxylate moiety of (<i>S</i>)-<b>N52</b> (panel A; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), <b>KDM5-C49</b> (panel B; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ISL">5ISL</a>), and αKG (panel C; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVB">5IVB</a>). (D–F) Tyr409 adopts different conformations in the complex with (<i>R</i>)-<b>N54</b> (panel D; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2">6BH2</a>), <b>KDOAM-25</b> (panel E; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF">5IWF</a>), and <b>CPI-455</b> (panel F; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH">5CEH</a>). (G) Superimposition of <b>KDM5-C49</b> and <b>KDOAM-25</b> bound KDM5A linked JmjN-JmjC domain structures. (H) Superimposition of (<i>R</i>)-<b>N54</b>, <b>KDM5-C49</b>, and <b>KDOAM-25</b> bound KDM5A linked JmjN-JmjC domain structures, illustrating conformational changes of Tyr409 upon inhibitor binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65157" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Individual enantiomers of chiral drugs are known to exhibit marked differences in biological activities. For example, the (<i>R</i>)- and (<i>S</i>)-enantiomers of 2HG are structurally similar to αKG but demonstrate distinct differences in potency and inhibitory activity against various αKG-dependent dioxygenase enzymes. Here we show that binding of <b>N54</b>/<b>N55</b> induced conformational changes in KDM5A, particularly Tyr409, which interacts with the acidic carboxylate moiety of <b>N51</b>/<b>N52</b>, as well as the cofactor αKG. In addition, Asp412 makes specific interactions with some inhibitors but not with others. These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Compound Synthesis</h3><div class="NLM_p last">Compounds were synthesized in the National Center for Advancing Translational Sciences (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). Purity determination was performed using an Agilent diode array detector for both method 1 and method 2 (below). Mass determination was performed using an Agilent 6130 mass spectrometer with electrospray ionization in the positive mode. Chiral separations were performed by normal phase chromatography using a CHIRALPAKAD column (5 cm × 50 cm, 20 μm) on an Agilent 1200 HPLC. <sup>1</sup>H NMR spectra were recorded on Varian 400 MHz spectrometers. Chemical shifts are reported in ppm with undeuterated solvent (DMSO-<i>d</i><sub>6</sub> at 2.49 ppm) as internal standard for DMSO-<i>d</i><sub>6</sub> solutions. All of the analogs tested in the biological assays have purity greater than 95%, based on both analytical methods. High resolution mass spectrometry was recorded on Agilent 6210 time-of-flight LC/MS system. Confirmation of molecular formula was accomplished using electrospray ionization in the positive mode with the Agilent Masshunter software (version B.02).</div></div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Method 1</h3><div class="NLM_p last">A 7 min gradient of 4–100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with an 8 min run time at a flow rate of 1 mL/min. A Phenomenex Luna C18 column (3 μm, 3 mm × 75 mm) was used at a temperature of 50 °C.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Method 2</h3><div class="NLM_p">A 3 min gradient of 4–100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with a 4.5 min run time at a flow rate of 1 mL/min. A Phenomenex Gemini Phenyl column (3 μm, 3 mm × 100 mm) was used at a temperature of 50 °C.</div><div class="NLM_p">All air or moisture sensitive reactions were performed under positive pressure of nitrogen or argon with oven-dried glassware. Anhydrous solvents and bases such as dichloromethane, <i>N</i>,<i>N</i>-dimethylforamide (DMF), acetonitrile, ethanol, DMSO, dioxane, Hunig’s base, and triethylamine were purchased from Sigma-Aldrich. Palladium catalysts were purchased from Strem chemicals and used as such. SiliaCat heterogeneous catalyst DPP-Pd (catalog no. R390-100) was purchased from SiliCycle Inc. Preparative purification was performed on a Waters semipreparative HPLC system using a Phenomenex Luna C18 column (5 μm, 30 mm × 75 mm) at a flow rate of 45 mL/min. The mobile phase consisted of acetonitrile and water (each containing 0.1% trifluoroacetic acid). A gradient of 10–50% acetonitrile over 8 min was used during the purification. Fraction collection was triggered by UV detection (220 nm). Analytical analysis was performed on an Agilent LC/MS (Agilent Technologies, Santa Clara, CA). General procedure for the syntheses of analogs is shown below:</div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0010.gif" alt="" id="dgr1" /></img></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis of 1-(2-Chlorophenyl)-3-(trimethylsilyl)prop-2-yn-1-ol</h3><div class="NLM_p last">To a solution of trimethylsilylacetylene (13.07 mL, 92 mmol, 1.3 equiv) in THF (50 mL) was added 2.5 M solution of <i>n</i>-butyllithium in THF (37.0 mL, 92 mmol, 1.3 equiv) dropwise upon cooling in an ice bath at −5 °C. The reaction was stirred for 1 h at 0 °C, and then 2-chlorobenzaldehyde (8.01 mL, 71.1 mmol, 1 equiv) was added in dropwise manner upon cooling in ice bath. After stirring the reaction for 1 h at room temperature, it was quenched with ammonium chloride upon cooling. The reaction was concentrated and extracted with diethyl ether. The organic layer was subsequently washed with water and brine, dried over sodium sulfate. The crude product after evaporating the ether was purified on a Isco normal phase flash system using a 120 g gold column eluting with 0–30% ethyl acetate in hexanes over 20 column volumes to obtain 16.9 g (99% yield) of colorless oil.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of 2-((1-(2-Chlorophenyl)prop-2-yn-1-yl)oxy)tetrahydro-2<i>H</i>-pyran</h3><div class="NLM_p last">To a mixture of 1-(2-chlorophenyl)-3-(trimethylsilyl)prop-2-yn-1-ol (17 g, 71.2 mmol, 1 equiv) and dihydropyran (8.12 mL, 89 mmol, 1.25 equiv) in DCM (150 mL) was added <i>p</i>-toluenesulfonic acid monohydrate (0.677 g, 3.56 mmol, 5 mol %) upon cooling in an ice bath. After stirring the reaction for 2 h at room temperature, the reaction mixture was diluted with DCM and subsequently washed with saturated bicarbonate, water, and brine. The organic layer was dried over sodium sulfate and concentrated to obtain crude product which was taken in 100 mL of methanol and then treated with K<sub>2</sub>CO<sub>3</sub> (9.84 g, 71.2 mmol, 1 equiv). After stirring for 1 h at room temperature excess methanol was removed. The residue obtained was suspended in DCM and filtered through a pad of silica. The filtrate was concentrated and the crude product was purified on an Isco normal phase flash system (an automated chromatography instrument) using a 330 g silica column eluting with 0–20% ethyl acetate in hexanes over 16 column volumes to obtain 14 g (78% yield) of oil.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Methyl 3-Amino-2-(3-(2-chlorophenyl)-3-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)prop-1-yn-1-yl)isonicotinate</h3><div class="NLM_p last">A mixture of methyl 3-amino-2-chloroisonicotinate (1.202 g, 6.44 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (2.226 g, 16.11 mmol, 2.5 equiv), XPhosPd(crotyl)Cl (<a href="http://jmcct.com/products-services/product_p578.html" class="extLink">http://jmcct.com/products-services/product_p578.html</a>, catalog no. Pd-170) (0.217 g, 0.322 mmol, 5 mol %), and methyl 3-amino-2-(3-(2-chlorophenyl)-3-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)prop-1-yn-1-yl)isonicotinate (2.21 g, 5.51 mmol, 1.3 equiv) in acetonitrile (12 mL) was bubbled with argon for 5 min and then capped and stirred at 75 °C overnight. The reaction was stirred with silica thiol for 15 min and then diluted with ethyl acetate and filtered through a pad of silica. The crude product obtained after removing the solvent was purified on an Isco flash system using a 120 g gold column eluting with 0–40% ethyl acetate (contains 0.1% TEA) in hexanes over 25 column volumes to obtain a yellow solid in 86% yield (2.21 g).</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of Methyl 2-((2-chlorophenyl)(hydroxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate (U.S. Patent 9,738,637; Example 71)</h3><div class="NLM_p last">A solution of methyl 3-amino-2-(3-(2-chlorophenyl)-3-((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)prop-1-yn-1-yl)isonicotinate (5 g, 12.47 mmol, 1 equiv) in DMF (12 mL) was bubbled with argon 15 min, then copper(I) iodide (0.475 g, 2.495 mmol, 0.2 equiv) was added and the mixture bubbled again for 15 min. The reaction was capped and stirred at 110 °C for 7 h. The reaction was cooled and stirred with thiol resin and again stirred for 1 h at room temperature. The reaction mixture was diluted with ethyl acetate and filtered through a pad of silica. Further, the filtrate was subsequently washed with water, brine and dried over MgSO<sub>4</sub>. The crude product obtained after evaporating the solvent was purified on an Isco flash system using a 120 g silica column eluting with 10–60% ethyl acetate in hexanes over 25 column volumes to obtain pure product 72% yield (3.6 g). A mixture of the pure methyl 2-((2-chlorophenyl)((tetrahydro-2<i>H</i>-pyran-2-yl)oxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate (0.7 g, 1.746 mmol, 1 equiv) and <i>p</i>-toluenesulfonic acid monohydrate (0.332 g, 1.746 mmol, 5 mol %, 1 equiv) in methanol (25 mL) was stirred at 65 °C until the starting material completely disappeared. The solvent was removed and the crude product was washed with water and dried to get pure solid product.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedure for the Syntheses of Analogs</h3><div class="NLM_p last">A mixture of chlorophenyl)(hydroxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate (0.1 g, 0.316 mmol, 1 equiv), <i>p</i>-toluenesulfonic acid monohydrate (0.180 g, 0.947 mmol, 3 equiv), and appropriate alcohol (1.5–2 equiv) in an open vial was stirred neat at 130–135 °C for 1 h. The product formed was a mixture of acid, ester, and trans ester. The crude reaction mass was taken in water and directly injected to an Isco reverse phase column to purify on a flash system. Eluting the column with a gradient of acetonitrile in water (0.1% TFA) over 16 column volumes provided fractions containing a mixture of all three products. The residue obtained after removing the solvent was treated with 1.5 M LiOH in THF/MeOH/water. The solvent was removed and the residue was taken in DMSO and purified on a preparative HPLC to obtain pure products.</div></div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-((2-Chlorophenyl)(propoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N9</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 4.822 min and <i>t</i><sub>2</sub> = 3.185 min. NMR data were consistent with previously reported data for this compound in patent WO2014164708 A1 (PCT/US2014/023273; Histone demethylase inhibitors; Quanticel Pharmaceuticals, Inc.).</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-((2-Chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid·TFA (<b>N40</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 3.049 min and <i>t</i><sub>2</sub> = 2.448 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.52 (s, 1H), 9.07 (s, 1H), 8.53–8.44 (m, 1H), 7.67–7.47 (m, 3H), 7.49–7.27 (m, 3H), 6.46 (s, 1H), 6.22 (s, 1H), 3.92–3.35 (m, 6H), 2.92 (q, <i>J</i> = 11.2 Hz, 2H), 1.78 (d, <i>J</i> = 14.2 Hz, 2H), 1.66 (d, <i>J</i> = 12.8 Hz, 4H); HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>22</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>3</sub>; 414.1579 found 414.1587.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 2-((2-Chlorophenyl)(2-(4,4-difluoropiperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N41</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 3.069 min and <i>t</i><sub>2</sub> = 2.421 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.49 (s, 1H), 9.75 (s, 1H), 8.47 (d, <i>J</i> = 4.8 Hz, 1H), 7.59 (dd, <i>J</i> = 4.9, 1.3 Hz, 1H), 7.54 (m, 2H), 7.50–7.42 (m, 2H), 6.28 (s, 1H), 6.24 (s, 1H), 3.93 (m, 1H), 3.83 (m, 1H), 3.72–3.24 (m, 4H), 2.77–2.63 (m, 1H), 2.56–2.48 (m, 1H), 2.42–2.18 (m, 4H). HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>22</sub>H<sub>23</sub>ClF<sub>2</sub>N<sub>3</sub>O<sub>3</sub>; 450.1391 found 450.1393.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 2-((2-Chlorophenyl)((4,4-difluorocyclohexyl)methoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N42</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 4.742 min and <i>t</i><sub>2</sub> = 3.148 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.89 (s, 1H), 8.57 (d, <i>J</i> = 5.3 Hz, 1H), 7.72 (d, <i>J</i> = 5.3 Hz, 1H), 7.51 (tt, <i>J</i> = 4.3, 2.7 Hz, 1H), 7.41 (q, <i>J</i> = 3.0 Hz, 3H), 6.38 (d, <i>J</i> = 1.9 Hz, 1H), 6.16 (s, 1H), 3.53 (dd, <i>J</i> = 8.8, 5.9 Hz, 1H), 3.41 (dd, <i>J</i> = 8.8, 6.2 Hz, 1H), 2.05–1.92 (m, 2H), 1.79 (tdd, <i>J</i> = 17.3, 12.8, 6.5 Hz, 5H), 1.24 (t, <i>J</i> = 10.5 Hz, 2H); HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>22</sub>H<sub>22</sub>ClF<sub>2</sub>N<sub>2</sub>O<sub>3</sub>; 435.1282 found 435.1273.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-((2-Chlorophenyl)(2-(1-methylpyrrolidin-2-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N46</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 2.935 min and <i>t</i><sub>2</sub> = 2.468 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 8.31 (dd, <i>J</i> = 4.8, 3.5 Hz, 1H), 7.58 (dd, <i>J</i> = 7.5, 1.9 Hz, 1H), 7.53–7.32 (m, 4H), 6.17 (s, 1H), 6.11 (d, <i>J</i> = 4.1 Hz, 1H), 3.86–3.33 (m, 4H), 2.60–2.56 (m, 3H), 2.25–1.53 (m, 4H); HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>22</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>3</sub>; 414.1579 found 414.1570.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-((2-Chlorophenyl)(2-(1-methyl-1<i>H</i>-imidazol-2-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N47</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 2.807 min and <i>t</i><sub>2</sub> = 2.338 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.70 (s, 1H), 8.41 (d, <i>J</i> = 4.9 Hz, 1H), 7.55–7.46 (m, 2H), 7.46–7.35 (m, 4H), 7.05 (d, <i>J</i> = 1.3 Hz, 1H), 6.86 (d, <i>J</i> = 1.3 Hz, 1H), 6.17 (s, 1H), 6.11 (d, <i>J</i> = 0.6 Hz, 1H), 3.85 (ddt, <i>J</i> = 32.8, 9.2, 6.6 Hz, 2H), 3.55 (s, 3H), 3.01 (td, <i>J</i> = 6.7, 2.1 Hz, 2H); HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>21</sub>H<sub>20</sub>ClN<sub>4</sub>O<sub>3</sub>; 411.1218 found 411.1218.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-((2-Chlorophenyl)(3-(piperidin-1-yl)propoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N48</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 3.009 min and <i>t</i><sub>2</sub> = 2.351 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.95 (s, 1H), 8.28 (d, <i>J</i> = 4.8 Hz, 1H), 7.59–7.52 (m, 1H), 7.49 (d, <i>J</i> = 4.7 Hz, 1H), 7.45–7.36 (m, 1H), 7.40–7.30 (m, 2H), 5.85 (s, 1H), 5.75 (d, <i>J</i> = 9.2 Hz, 1H), 3.87–3.41 (m, 4H), 3.3–3.13 (m, 4H), 2.15–1.31 (m, 8H); HRMS (ESI) <i>m</i>/<i>z</i> (M + H)<sup>+</sup> calculated for C<sub>23</sub>H<sub>27</sub>ClN<sub>3</sub>O<sub>3</sub>; 428.1735 found 428.1741.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (<i>R</i>)- and (<i>S</i>)-2-((2-Chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N51</b> and <b>N52</b>)</h3><div class="NLM_p last">Racemic mixture of ester precursor was separated by normal phase chromatography using a CHIRALPAKAD column (5 cm × 50 cm, 20 μm) on an Agilent 1200 HPLC with a hexane/isopropanol/diethylamine (85:15:0.1) isocratic mobile phase eluting at 35 mL/min and detecting at 230 nm. Separation of 70 mg of the racemic mixture provided 20 mg of enantiomer 1 (with negative rotation), with enantiomeric excess greater than 98%, and 20 mg of enantiomer 2 (with positive rotation) with enantiomeric excess 95.9%. Absolute configuration was determined by cocrystallization with KDM5A enzyme.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (<i>R</i>)<i>-</i> and (<i>S</i>)<i>-N</i>-(1-(3-Isopropyl-1<i>H</i>-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (<b>N54</b> and <b>N55</b>)</h3><div class="NLM_p last">LC–MS retention time: <i>t</i><sub>1</sub> = 3.530 min and <i>t</i><sub>2</sub> = 2.590 min. The title compounds were prepared as individual enantiomers from enantiopure starting materials according to previously reported protocol, and NMR data were consistent with previously reported data for this compound in patent WO2016057924 A1 (PCT/US2015/054949; Pyrrolidine amide compounds as histone demethylase inhibitors; Genetech, Inc., Constellation Pharmaceuticals, Inc.). Absolute configuration was determined by cocrystallization with KDM5A enzyme.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Crystallography</h3><div class="NLM_p last">The N-terminal fragments of human KDM5A and the internal deletion constructs (deleting ARID and PhD1 domains (ΔAP)) were prepared in a pET28 plasmid containing an N-terminal His-SUMO tag sequence and purified by three-column chromatography utilizing affinity, anion exchange, and sizing exclusion as previously described in detail.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The KDM5A(1–588)ΔAP, in 20 mM Hepes, pH 8.0, 300 mM NaCl, 5% glycerol, 0.5 mM tris(2-carboxyethyl)phosphine (TCEP), was mixed with MnCl<sub>2</sub> and αKG at an approximate molar ratio of 1:5 and concentrated to ∼50 mg/mL (∼1.4 mM) and then diluted to ∼10 mg/mL (280 μM) for cocrystallization. Experiments utilized the sitting drop technique and were conducted at 16 °C by mixing 0.2 μL of the complex with an equal volume of a well solution. Typically, clusters of physically twinned crystals grew with αKG with a well solution containing 1.5 M ammonium sulfate, 12% glycerol, and 0.1 M Tris-HCl (pH 8.5). However, these crystals could be crushed to produce microseeds and utilized to produce some large and single crystals from a 96-well screen containing 1.2–1.35 M (NH4)<sub>2</sub>SO<sub>4</sub>, 0.1 M Tris-HCl (pH 8.6–9.2), 0–20% glycerol, and 25 mM (Na/K) dibasic/monobasic phosphate. Inhibitors are soaked into these preformed crystals of KDM5A-αKG-Mn(II) complexes by transferring a crystal into a new drop containing mother liquor and an inhibitor (∼500 μM), allowing the crystal to remain in this drop for a few hours to 1 day for the inhibitor to exchange with αKG. These were then mounted into nylon cryoloops (Hampton Research, Inc.) and frozen in liquid nitrogen after the addition of more glycerol (up to ∼30% total) to the mother liquor as a cryoprotectant. X-ray diffraction data were collected on either the SER-CAT beamline 22-ID or 22-BM at the Advanced Photon Source at Argonne National Laboratory at 100 K with 1° oscillation images and were processed and merged with HKL2000<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a>). The structures were determined by molecular replacement using our initial KDM5A structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E6H">5E6H</a>). Repeated rounds of manual refitting and crystallographic refinement were performed using COOT<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> and PHENIX.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p">ITC experiments were carried out as described.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Briefly, enzyme was dialyzed extensively against 20 mM HEPES, pH 8.0, 300 mM NaCl, 5% glycerol, 0.5 mM tris(2-carboxyethyl)phosphine) supplemented with 500 μM MnCl<sub>2</sub> and 10% (v/v) DMSO, and the dialysate was used to prepare fresh working solutions of various inhibitors. The manganese ion was utilized instead of iron since it would not oxidize during the time frame of these experiments. Titration was carried out at 0.42–1.1 mM compound [I] concentration and enzyme [E] concentration of 25–80 μM. Compounds were injected into protein or buffer (to measure the heat of dilution) in 25 steps of 4 μL volume using a MicroCal Auto-iTC<sub>200</sub>. Binding affinity (<i>K</i><sub>D</sub>), stoichiometry (<i>N</i>), and binding enthalpy (Δ<i>H</i>) were determined by fitting the data using the ITC data analysis module of Origin version 7.0 (OriginLab Corp.).</div><div class="NLM_p last">For each ITC experiment, we performed a test run to figure out the optimal enzyme and inhibitor concentrations to be used and range of titrations. Each curve is fitted for 20 data points. Due to the small differences in the <i>K</i><sub>D</sub> values of very similar chemical compounds, the data presented resulted from experiments done at the same time using the same batch of enzymes, which is essential to avoid the variation from batches of enzyme preparation.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Formaldehyde Dehydrogenase (FDH)-Coupled Demethylase Assay</h3><div class="NLM_p last">FDH assays were performed as described.<a onclick="showRef(event, 'ref38 ref49'); return false;" href="javascript:void(0);" class="ref ref38 ref49">(38,49)</a> In brief, inhibitors <b>N54</b> and <b>N55</b> with the concentrations ranging from 1.25 mM to 10 nM in a half-log serial dilution were preincubated with 0.5 μM enzyme [E] at room temperature (21 °C) in the reaction buffer, containing 50 mM MES (pH 6.8), 50 μM (NH<sub>4</sub>)<sub>2</sub>Fe(SO<sub>4</sub>)<sub>2</sub>, 2 mM ascorbic acid, 10 μg of FDH, and 10% DMSO. After 15 min, 15 μM histone peptide H3(1–24)K4me3 [S] and 0.6 mM APAD<sup>+</sup> (3-acetylpyridine adenine dinucleotide, a more stable analogue of NAD<sup>+</sup>; Sigma-Aldrich A5251) were added to initiate the reaction for 15 min, with the fluorescence signal recorded in a BioTek Synergy 4 hybrid microplate reader using 380 nm (20 nm bandwidth) and 460 nM (40 nm bandwidth) as the excitation and emission filters, respectively. The initial velocity was calculated, and the dose–response curves were fitted by a four-parameter logistic equation using GraphPad Prism 5.0: activity (%) = <i>B</i> + (<i>M</i> – <i>B</i>)/[1 + 10<sup>((logIC<sub>50</sub>-log[I])·Hill slope)</sup>], where <i>B</i> is the baseline (usually 0 for no activity or maximum inhibition), <i>M</i> is maximum activity or no inhibition (usually 1), [I] is inhibitor concentration, and the Hill slope refers to the steepness of the curve (usually −1).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> AlphaLISA Assay</h3><div class="NLM_p">Alpha streptavidin donor beads (catalog no. 6760002), anti-methyl-histone H3 lysine 4 (H3K4me2/1) AlphaLISA acceptor beads (catalog AL116), AlphaLISA 5× epigenetics buffer (catalog AL008-C1), and 30× epigenetics buffer supplement (AL008-C2) were purchased from PerkinElmer (Waltham, MA). C-terminal Flag-tagged KDM5A (residues 1–1090; catalog 50110) was purchased from BPS Bioscience (San Diego, CA). The biotinylated H3(1–21)K4me3 (catalog AS-64192) peptide was purchased from AnaSpec (Freemont, CA) and bovine serum albumin (BSA; catalog A8412) was purchased from Sigma (St. Louis, MO).</div><div class="NLM_p last">All assays were performed in 1536-well, white, solid-bottom assay plates (catalog no. 789175) obtained from Greiner Bio-One (Monroe, NC) and read using an EnVision multilabel plate reader from PerkinElmer (Waltham, MA). Reactions were performed in 50 mM HEPES, pH 7.25, containing 0.01% BSA and 0.01% Tween-20. To each well, 3 μL of enzyme mix, containing KDM5A(1–1090)-Flag (final 10 nM), was added using a BioRAPTR flying reagent dispenser (FRD; Beckman Coulter, Fullerton, CA) while buffer alone served as a negative control. A Kalypsys pin-tool was then employed to transfer 23 nL of compound solutions in DMSO to each well, yielding a final concentration range up to 57 μM. Following a 10 min incubation at room temperature, 1 μL of 4× substrate mix containing biotinylated-H3K4(me3) peptide substrate (final 100 nM), αKG (final 25 μM), (NH<sub>4</sub>)<sub>2</sub>Fe(SO<sub>4</sub>)<sub>2</sub> (final 50 μM), and ascorbic acid (final 10 μM) was added to initiate the reaction. Reactions were allowed to proceed at room temperature for 45 min prior to detection. To detect demethylation of biotinylated-H3K4(me3) peptide product, reactions were incubated with 20 μg/mL anti-methyl-histone H3K4 (me2/1) acceptor beads in 1× epigenetics buffer for 1 h, then followed with a 30 min incubation with 20 μg/mL streptavidin-coated donor AlphaScreen beads. Beads were added in 1.5 and 3 μL of FRD dispense, respectively, for a final volume of 7.5 μL, and plates were protected from the light at room temperature during donor bead incubation. Microplates were read on an EnVision multilabel plate reader using the 1536-well plate HTS AlphaScreen aperture (excitation time 80 ms, measurement time 240 ms). Data were normalized to the mean of the no-enzyme and uninhibited controls, and IC<sub>50</sub> values were determined by nonlinear regression fits using GraphPad Prism4 software.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Cell Culture</h3><div class="NLM_p last">MCF7, BT474, and MDA-MB231 cells were cultured in RPMI1640 supplemented with 10% fetal bovine serum and 1% penicillin and streptomycin. SKBR3 cells were cultured in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum and 1% penicillin and streptomycin. All cells were grown at 37 °C with 5% CO<sub>2</sub>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Western Blot Analysis</h3><div class="NLM_p last">SKBR3, BT474, MDA-MB231, and MCF7 cells were treated with DMSO, <b>KDM5-C70</b>, <b>N54</b>, or <b>N55</b> for 3 days. Histone extraction and Western blot analysis were performed as described previously.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The following antibodies were obtained commercially: rabbit anti-histone H3 (ab1791) (Abcam); rabbit anti-H3K4me3 (C42D8), H3K9me3 (D4W1U), H3K27me3 (C36B11) (Cell Signaling Technology).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Colony Formation Assay</h3><div class="NLM_p last">MCF7 cells were seeded at 2000 cells/well and treated with 0.1% DMSO, 1 or 10 μM <b>N54</b>, 1 or 10 μM <b>N55</b>, or 1 μM <b>KDM5-C70</b> the following day. Cells were treated for 12 days with media replaced every 3 days. Cells were fixed for 5 min in 4% paraformaldehyde in phosphate-buffered saline, stained in 0.05% crystal violet in double-distilled water for 45 min, gently washed with water, and allowed to dry overnight before photographing. ImageJ plug-in ColonyArea was used to quantify crystal violet staining as described previously.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> IncuCyte Experiments</h3><div class="NLM_p last">Cells were seeded in 384-well plates (Cell Carrier Ultra, PerkinElmer) at 2.5 × 10<sup>3</sup> cells per well (50 μL volume) using a Multidrop Combi Reagent Dispenser (Thermo Fisher) and returned to a humidified incubator (5% CO<sub>2</sub>) overnight. Culture medium was RPMI (Life Technologies no. 11835) + 10% fetal bovine serum + 50 U/mL penicillin + 50 μg/mL streptomycin. Compounds (190 nL in DMSO) were transferred using a pin tool. Plates were covered with a gas permeable sealing membrane (Breathe Easy, Diversified Biotech) and imaged every 4 h using an IncuCyte Zoom (Essen Bioscience) equipped with a 10× objective (Nikon). Cell growth (phase object confluence) was calculated using IncuCyte ZOOM software.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Immunocytochemistry</h3><div class="NLM_p last">BT474 cells were seeded in 384-well plates (Cell Carrier Ultra, PerkinElmer) at 2.5 × 10<sup>3</sup> cells per well (50 μL volume). After overnight incubation, 190 nL of compound (in DMSO) was added by pin transfer ([final] = 62 nM to 63 μM). After 48 h, the cells were fixed (fresh 4% paraformaldehyde in 1× PBS, pH 7.4), permeabilized (PBS + 0.1% Triton X-100 + 0.2% BSA), and blocked (PBS + 0.1% Triton X-100 + 2% BSA + 5% normal goat serum). Primary antibodies against H3K4me3 (Abcam 1012) and H3 (Cell Signaling CS4499) were incubated overnight at 4 °C. Secondary antibodies conjugated to AlexaFluor488 and 568 dyes were added, and nuclei were stained with Hoechst. Cells were imaged on an IN Cell Analyzer 2200. H3K4Me3 and total H3 intensity was measured on a per cell basis (in the nuclear region, defined as overlapping with Hoechst staining), and H3K4me3/total H3 ratio was calculated.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00261" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00261" class="ext-link">10.1021/acs.jmedchem.8b00261</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figures S1–S7, and Tables S1–S2 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf">jm8b00261_si_001.pdf (4.08 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_002.csv">jm8b00261_si_002.csv (0.66 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors of KDM5A in complex with <b>N9</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU">6BGU</a>), <b>N40</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>), <b>N41</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW">6BGW</a>), <b>N42</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX">6BGX</a>), <b>N46</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>), <b>N47</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ">6BGZ</a>), <b>N48</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5">6BH5</a>), <b>N51</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>), <b>N52</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), <b>N54</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2">6BH2</a>), <b>N55</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH3">6BH3</a>), and <b>KDOAM-25</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF">5IWF</a>) have been deposited in the Protein Data Bank, and authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00261" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71540" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71540" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qin Yan</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6011090e4e19010e2019010c054e050415"><span class="__cf_email__" data-cfemail="b4c5ddda9acdd5daf4cdd5d8d19ad1d0c1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaodong Cheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United
States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6967-6362" title="Orcid link">http://orcid.org/0000-0002-6967-6362</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8df5eee5e8e3eab8cde0e9ece3e9e8fffee2e3a3e2ffea"><span class="__cf_email__" data-cfemail="acd4cfc4c9c2cb99ecc1c8cdc2c8c9dedfc3c282c3decb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John R. Horton</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xu Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lizhen Wu</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kai Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Pathology, Yale School of Medicine, New Haven, Connecticut 06520, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Shanks</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xing Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ganesha Rai</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bryan T. Mott</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. Jansen</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen C. Kales</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark J. Henderson</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Pohida</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuhong Fang</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Hu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajit Jadhav</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Maloney</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5607-4178" title="Orcid link">http://orcid.org/0000-0002-5607-4178</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Hall</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of
Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haian Fu</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacology, 
    Department of Hematology
and Medical Oncology, 
    Emory Chemical Biology
Discovery Center, 
    The Winship Cancer Institute, 
    , , , , and , Emory University, Atlanta, Georgia 30322, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paula M. Vertino</span> - <span class="hlFld-Affiliation affiliation">The Winship Cancer Institute, 
    Department of Radiation Oncology, 
    , , , , and , Emory University, Atlanta, Georgia 30322, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.R.H. performed all crystallographic experiments. X.L. generated ITC and FDH data and chemical drawings. L.W. and K.Z. performed Western blot analysis and colony formation assays. J.S. performed protein purification, and X.Z. participated in experimental design and discussions throughout. G.R., B.T.M., D.J.J., and K.P. synthesized compounds. S.C.K performed AlphaLISA experiments. M.J.H. performed immunocytochemistry and IncuCyte experiments. Y.F. performed chiral separation of molecules. X.H. performed data processing. A.J. and M.D.H. designed experiments and discussed the study. D.J.M. and A.S. planned synthesis of compounds. H.F. participated in organization and discussion of the study. J.R.H., P.M.V., Q.Y., and X.C. wrote the initial manuscript, and all were involved in analyzing data and discussion. The X-ray crystallographic component of the study was initially carried out at Emory University (as dated in Table S1), and the structural analyses and the manuscript were completed at the University of Texas MD Anderson Cancer Center.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part with Federal funds from the National Cancer Institute (NCI), National Institutes of Health (NIH), under NCI Chemical Biology Consortium Contract HHSN261200800001E (to H.F.), NIH Grant GM114306 and Cancer Prevention Research Institute of Texas Grant RR160029 (to X.C.), American Cancer Society Research Scholar Grant RSG-13-384-01-DMC and DoD Breast Cancer Research Program Award W81XWH-14-1-0308 (to Q.Y.), and Leslie H. Warner Postdoctoral Fellowship (to L.W.). The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. The Department of Biochemistry of Emory University School of Medicine supported the use of the Southeast Regional Collaborative Access Team (SERCAT) synchrotron beamlines at the Advanced Photon Source of Argonne National Laboratory. We thank Dr. Paul E. Brennan of Structural Genomics Consortium for KDOAM-25. We thank Anthony Welch, Andrew Flint, and Dane Liston of Frederick National Laboratory for Cancer Research for administrative support, and Andrew Flint for extensive comments and editing the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">αKG</td><td class="NLM_def"><p class="first last">α-ketoglutarate</p></td></tr><tr><td class="NLM_term">2HG</td><td class="NLM_def"><p class="first last">2-hydroxyglutarate</p></td></tr><tr><td class="NLM_term">KDM5A</td><td class="NLM_def"><p class="first last">histone lysine demethylase 5A</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 60 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, S. M.</span>, <span class="NLM_string-name">Li, E.</span></span>, Eds.  <i>Epigenetics and Disease: Pharmaceutical Opportunities</i>; Progress in Drug Research, Vol. 67; <span class="NLM_publisher-name">Springer Basel AG</span>, <span class="NLM_year">2011</span>; DOI: <span class="refDoi"> DOI: 10.1007/978-3-7643-8989-5</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2F978-3-7643-8989-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+M.+Gasser&author=E.+Li&title=Epigenetics+and+Disease%3A+Pharmaceutical+Opportunities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F978-3-7643-8989-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-7643-8989-5%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DS.%2BM.%26btitle%3DEpigenetics%2520and%2520Disease%253A%2520Pharmaceutical%2520Opportunities%26pub%3DSpringer%2520Basel%2520AG%26date%3D2011%26doi%3D10.1007%2F978-3-7643-8989-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of chromatin by histone modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+chromatin+by+histone+modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0ljWDfPc2h1o_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520by%2520histone%2520modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herz, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garruss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilatifard, A.</span></span> <span> </span><span class="NLM_article-title">SET for life: biochemical activities and biological functions of SET domain-containing proteins</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2013.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.tibs.2013.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=24148750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs12ltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=621-639&author=H.+M.+Herzauthor=A.+Garrussauthor=A.+Shilatifard&title=SET+for+life%3A+biochemical+activities+and+biological+functions+of+SET+domain-containing+proteins&doi=10.1016%2Fj.tibs.2013.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SET for life: biochemical activities and biological functions of SET domain-containing proteins</span></div><div class="casAuthors">Herz, Hans-Martin; Garruss, Alexander; Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">621-639</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  SET domain-contg. proteins belong to a group of enzymes named after a common domain that utilizes the cofactor S-adenosyl-L-methionine (SAM) to achieve methylation of its substrates.  Many SET domain-contg. proteins were shown to display catalytic activity towards particular lysine residues on histones,but emerging evidence also indicates that various nonhistone proteins are specifically targeted by this clade of enzymes.  Here, the authors summarize the most recent findings on the biol. functions of the major families of SET domain-contg. proteins catalyzing the methylation of histones 3 on lysines 4, 9, 27, and 36 (H3K4, H3K9,H3K27, and H3K36) and histone 4 on lysine 20 (H4K20), as well as enzymes that were reported to modify nonhistone substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVOb-qpf8h3LVg90H21EOLACvtfcHk0liysD7TQDtydQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs12ltrjK&md5=1214f35b69249e0d0dcd2cdfe992e9bc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DHerz%26aufirst%3DH.%2BM.%26aulast%3DGarruss%26aufirst%3DA.%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DSET%2520for%2520life%253A%2520biochemical%2520activities%2520and%2520biological%2520functions%2520of%2520SET%2520domain-containing%2520proteins%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D621%26epage%3D639%26doi%3D10.1016%2Fj.tibs.2013.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paik, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Historical review: the field of protein methylation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2007.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.tibs.2007.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=17291768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisleisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2007&pages=146-152&author=W.+K.+Paikauthor=D.+C.+Paikauthor=S.+Kim&title=Historical+review%3A+the+field+of+protein+methylation&doi=10.1016%2Fj.tibs.2007.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Historical review: the field of protein methylation</span></div><div class="casAuthors">Paik, Woon Ki; Paik, David C.; Kim, Sangduk</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-152</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although the field of protein methylation enjoys widespread interest in the scientific literature of today, this is a recent phenomenon.  Papers on 'protein methylation' were first published in the 1960s.  By the early 1980s, it was known that lysine, arginine, histidine and dicarboxylic amino acids were post-translationally methylated by highly specific methyltransferases.  However, despite these early advances, the biol. importance of these reactions remained largely unproven.  With the introduction of modern mol. biol. techniques in the mid-1990s, an enormous surge of interest in protein methylation occurred.  It is now clear that protein methylation carries many important biol. functions, including gene regulation and signal transduction.  Thus, the story of protein-methylation research is a testament to both modern mol. biol. and the importance of continuing to pursue lines of research in which the precise biol. function might not be currently known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr258CjMUdF2bVg90H21EOLACvtfcHk0liysD7TQDtydQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisleisb4%253D&md5=e32da6065de5f6fe7d92fe6f0ffc3d97</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2007.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2007.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DPaik%26aufirst%3DW.%2BK.%26aulast%3DPaik%26aufirst%3DD.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DHistorical%2520review%253A%2520the%2520field%2520of%2520protein%2520methylation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2007%26volume%3D32%26spage%3D146%26epage%3D152%26doi%3D10.1016%2Fj.tibs.2007.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liysD7TQDtydQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchers, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation by a family of JmjC domain-containing proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">816</span>, <span class="refDoi"> DOI: 10.1038/nature04433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature04433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=16362057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=811-816&author=Y.+Tsukadaauthor=J.+Fangauthor=H.+Erdjument-Bromageauthor=M.+E.+Warrenauthor=C.+H.+Borchersauthor=P.+Tempstauthor=Y.+Zhang&title=Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins&doi=10.1038%2Fnature04433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation by a family of JmjC domain-containing proteins</span></div><div class="casAuthors">Tsukada, Yu-ichi; Fang, Jia; Erdjument-Bromage, Hediye; Warren, Maria E.; Borchers, Christoph H.; Tempst, Paul; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7078</span>),
    <span class="NLM_cas:pages">811-816</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Covalent modification of histones has an important role in regulating chromatin dynamics and transcription.  Whereas most covalent histone modifications are reversible, until recently it was unknown whether Me groups could be actively removed from histones.  Using a biochem. assay coupled with chromatog., we have purified a novel JmjC domain-contg. protein, JHDM1 (JmjC domain-contg. histone demethylase 1), that specifically demethylates histone H3 at lysine 36 (H3-K36).  In the presence of Fe(II) and α-ketoglutarate (αKG), JHDM1 demethylates H3-methyl-K36 and generates formaldehyde and succinate.  Overexpression of JHDM1 reduced the level of dimethyl-H3-K36 (H3K36me2) in vivo.  The demethylase activity of the JmjC domain-contg. proteins is conserved, as a JHDM1 homolog in Saccharomyces cerevisiae also has H3-K36 demethylase activity.  Thus, we identify the JmjC domain as a novel demethylase signature motif and uncover a protein demethylation mechanism that is conserved from yeast to human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKQSpeu3QBl7Vg90H21EOLACvtfcHk0liysD7TQDtydQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D&md5=08f672cc5f81ca88288496f5d8069e3e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature04433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04433%26sid%3Dliteratum%253Aachs%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWarren%26aufirst%3DM.%2BE.%26aulast%3DBorchers%26aufirst%3DC.%2BH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520by%2520a%2520family%2520of%2520JmjC%2520domain-containing%2520proteins%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D811%26epage%3D816%26doi%3D10.1038%2Fnature04433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span> <span> </span><span class="NLM_article-title">The discovery of histone demethylases</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a017947</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a017947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1101%2Fcshperspect.a017947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=a017947&author=Y.+G.+Shiauthor=Y.+Tsukada&title=The+discovery+of+histone+demethylases&doi=10.1101%2Fcshperspect.a017947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of histone demethylases</span></div><div class="casAuthors">Shi, Yujiang Geno; Tsukada, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">A017947/1-A017947/3</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Histone methylation is a key element of the eukaryotic epigenome.  Since the discovery of the first histone demethylase (HDM) in 2004, more than 20 demethylases have been identified and characterized.  They belong to either the LSD family or the JmjC family, demonstrating the reversibility of all methylation states at almost all major histone lysine methylation sites.  These findings ended decades of debate about the reversibility of histone methylation, representing a major breakthrough that shifts our understanding of epigenetic inheritance and regulation of genome function.  Here, we summarize the discovery of HDMs and more recent advances, challenges, and future prospects of HDM research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbFLxtuYl3LVg90H21EOLACvtfcHk0li_eHKDh051kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGhsb%252FF&md5=ed2c88e4ec1ba5745617fae942c59c11</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a017947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a017947%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BG.%26aulast%3DTsukada%26aufirst%3DY.%26atitle%3DThe%2520discovery%2520of%2520histone%2520demethylases%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3Da017947%26doi%3D10.1101%2Fcshperspect.a017947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jambhekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastas, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylase inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026484</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1101%2Fcshperspect.a026484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28049654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026484&author=A.+Jambhekarauthor=J.+N.+Anastasauthor=Y.+Shi&title=Histone+lysine+demethylase+inhibitors&doi=10.1101%2Fcshperspect.a026484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase inhibitors</span></div><div class="casAuthors">Jambhekar, Ashwini; Anastas, Jamie N.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">a026484/1-a026484/26</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The dynamic regulation of covalent modifications to histones is essential for maintaining genomic integrity and cell identity and is often compromised in cancer.  Aberrant expression of histone lysine demethylases has been documented in many types of blood and solid tumors, and thus demethylases represent promising therapeutic targets.  Recent advances in high-throughput chem. screening, structure-based drug design, and structure-activity relationship studies have improved both the specificity and the in vivo efficacy of demethylase inhibitors.  This review will briefly outline the connection between demethylases and cancer and will provide a comprehensive overview of the structure, specificity, and utility of currently available demethylase inhibitors.  To date, a select group of demethylase inhibitors is being evaluated in clin. trials, and addnl. compds. may soon follow from the bench to the bedside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqMUZw4KjjDLVg90H21EOLACvtfcHk0li_eHKDh051kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF&md5=22ba67d04750ab8d410baee59a64d7e6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26aulast%3DJambhekar%26aufirst%3DA.%26aulast%3DAnastas%26aufirst%3DJ.%2BN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026484%26doi%3D10.1101%2Fcshperspect.a026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0li_eHKDh051kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sennels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappsilber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcini, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri- and dimethylated lysine 4 on histone 3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1076</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-+and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-%2520and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0lg8s84vVGldVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0ljfI2GccyTJjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase lysine demethylase 5B in development and cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">8980</span>– <span class="NLM_lpage">8991</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.13858" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=8980-8991&author=M.+Hanauthor=W.+Xuauthor=P.+Chengauthor=H.+Jinauthor=X.+Wang&title=Histone+demethylase+lysine+demethylase+5B+in+development+and+cancer&doi=10.18632%2Foncotarget.13858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13858%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DCheng%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DHistone%2520demethylase%2520lysine%2520demethylase%25205B%2520in%2520development%2520and%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D8980%26epage%3D8991%26doi%3D10.18632%2Foncotarget.13858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombkowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holowatyj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Q.</span></span> <span> </span><span class="NLM_article-title">Genomic amplification and a role in drug-resistance for the KDM5A histone demethylase in breast cancer</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">256</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=22937203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Snsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=247-256&author=J.+Houauthor=J.+Wuauthor=A.+Dombkowskiauthor=K.+Zhangauthor=A.+Holowatyjauthor=J.+L.+Boernerauthor=Z.+Q.+Yang&title=Genomic+amplification+and+a+role+in+drug-resistance+for+the+KDM5A+histone+demethylase+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic amplification and a role in drug-resistance for the KDM5A histone demethylase in breast cancer</span></div><div class="casAuthors">Hou, Jinling; Wu, Jack; Dombkowski, Alan; Zhang, Kezhong; Holowatyj, Andreana; Boerner, Julie L.; Yang, Zeng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">247-256</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 5A (KDM5A), an enzyme that removes activating H3K4 di- and tri-methylation marks, plays crit. roles in controlling transcription and chromatin architecture, yet its biol. functions largely remain uncharacterized, particularly in the context of human cancer.  In the present study, we found that the KDM5A gene was significantly amplified and over-expressed in various human tumors, including breast cancer.  Reducing the expression of KDM5A by shRNA knockdown inhibited proliferation of KDM5A-amplified breast cancer cells.  More importantly, we demonstrated that KDM5A over-expression was assocd. with breast cancer drug resistance.  Furthermore, knockdown of KDM5A gene expression altered H3K4 methylation and induced upregulation of CDK inhibitors as well as genes mediating apoptotic cell death.  Taken together, our study strongly links KDM5A histone demethylase activity to breast cancer proliferation and drug resistance, and suggests KDM5A is a potential target for breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPVl0slRNT5LVg90H21EOLACvtfcHk0ljfI2GccyTJjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Snsb3O&md5=493b21eebbe762fcb8238434fa623faa</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDombkowski%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DHolowatyj%26aufirst%3DA.%26aulast%3DBoerner%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DZ.%2BQ.%26atitle%3DGenomic%2520amplification%2520and%2520a%2520role%2520in%2520drug-resistance%2520for%2520the%2520KDM5A%2520histone%2520demethylase%2520in%2520breast%2520cancer%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2012%26volume%3D4%26spage%3D247%26epage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeng, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, L. J.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 promotes lung tumorigenesis and cancer metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">4711</span>– <span class="NLM_lpage">4721</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1158%2F0008-5472.CAN-12-3165" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=4711-4721&author=Y.+C.+Tengauthor=C.+F.+Leeauthor=Y.+S.+Liauthor=Y.+R.+Chenauthor=P.+W.+Hsiaoauthor=M.+Y.+Chanauthor=F.+M.+Linauthor=H.+D.+Huangauthor=Y.+T.+Chenauthor=Y.+M.+Jengauthor=C.+H.+Hsuauthor=Q.+Yanauthor=M.+D.+Tsaiauthor=L.+J.+Juan&title=Histone+demethylase+RBP2+promotes+lung+tumorigenesis+and+cancer+metastasis&doi=10.1158%2F0008-5472.CAN-12-3165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3165%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DY.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BF.%26aulast%3DLi%26aufirst%3DY.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BR.%26aulast%3DHsiao%26aufirst%3DP.%2BW.%26aulast%3DChan%26aufirst%3DM.%2BY.%26aulast%3DLin%26aufirst%3DF.%2BM.%26aulast%3DHuang%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DY.%2BT.%26aulast%3DJeng%26aufirst%3DY.%2BM.%26aulast%3DHsu%26aufirst%3DC.%2BH.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DTsai%26aufirst%3DM.%2BD.%26aulast%3DJuan%26aufirst%3DL.%2BJ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520promotes%2520lung%2520tumorigenesis%2520and%2520cancer%2520metastasis%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D4711%26epage%3D4721%26doi%3D10.1158%2F0008-5472.CAN-12-3165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">RBP2 induces epithelial-mesenchymal transition in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e84735</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0084735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0084735" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e84735&author=S.+Wangauthor=Y.+Wangauthor=H.+Wuauthor=L.+Hu&title=RBP2+induces+epithelial-mesenchymal+transition+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0084735"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084735%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DRBP2%2520induces%2520epithelial-mesenchymal%2520transition%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De84735%26doi%3D10.1371%2Fjournal.pone.0084735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase retinoblastoma binding protein 2 is overexpressed in hepatocellular carcinoma and negatively regulated by hsa-miR-212</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e69784</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0069784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0069784" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69784&author=X.+Liangauthor=J.+Zengauthor=L.+Wangauthor=M.+Fangauthor=Q.+Wangauthor=M.+Zhaoauthor=X.+Xuauthor=Z.+Liuauthor=W.+Liauthor=S.+Liuauthor=H.+Yuauthor=J.+Jiaauthor=C.+Chen&title=Histone+demethylase+retinoblastoma+binding+protein+2+is+overexpressed+in+hepatocellular+carcinoma+and+negatively+regulated+by+hsa-miR-212&doi=10.1371%2Fjournal.pone.0069784"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069784%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%26atitle%3DHistone%2520demethylase%2520retinoblastoma%2520binding%2520protein%25202%2520is%2520overexpressed%2520in%2520hepatocellular%2520carcinoma%2520and%2520negatively%2520regulated%2520by%2520hsa-miR-212%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69784%26doi%3D10.1371%2Fjournal.pone.0069784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorkholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span> <span> </span><span class="NLM_article-title">The histone demethylase RBP2 Is overexpressed in gastric cancer and its inhibition triggers senescence of cancer cells</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2009.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1053%2Fj.gastro.2009.10.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2010&pages=981-992&author=J.+Zengauthor=Z.+Geauthor=L.+Wangauthor=Q.+Liauthor=N.+Wangauthor=M.+Bjorkholmauthor=J.+Jiaauthor=D.+Xu&title=The+histone+demethylase+RBP2+Is+overexpressed+in+gastric+cancer+and+its+inhibition+triggers+senescence+of+cancer+cells&doi=10.1053%2Fj.gastro.2009.10.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2009.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2009.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DBjorkholm%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26atitle%3DThe%2520histone%2520demethylase%2520RBP2%2520Is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520its%2520inhibition%2520triggers%2520senescence%2520of%2520cancer%2520cells%26jtitle%3DGastroenterology%26date%3D2010%26volume%3D138%26spage%3D981%26epage%3D992%26doi%3D10.1053%2Fj.gastro.2009.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiping, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lixiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiuming, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuqun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhifang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yundong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shili, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chunyan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jihui, J.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-212 inhibits proliferation of gastric cancer by directly repressing retinoblastoma binding protein 2</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2672</span>, <span class="refDoi"> DOI: 10.1002/jcb.24613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1002%2Fjcb.24613" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2013&pages=2666-2672&author=Z.+Jipingauthor=F.+Mingauthor=W.+Lixiangauthor=L.+Xiumingauthor=S.+Yuqunauthor=Y.+Hanauthor=L.+Zhifangauthor=S.+Yundongauthor=L.+Shiliauthor=C.+Chunyanauthor=J.+Jihui&title=MicroRNA-212+inhibits+proliferation+of+gastric+cancer+by+directly+repressing+retinoblastoma+binding+protein+2&doi=10.1002%2Fjcb.24613"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcb.24613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.24613%26sid%3Dliteratum%253Aachs%26aulast%3DJiping%26aufirst%3DZ.%26aulast%3DMing%26aufirst%3DF.%26aulast%3DLixiang%26aufirst%3DW.%26aulast%3DXiuming%26aufirst%3DL.%26aulast%3DYuqun%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhifang%26aufirst%3DL.%26aulast%3DYundong%26aufirst%3DS.%26aulast%3DShili%26aufirst%3DL.%26aulast%3DChunyan%26aufirst%3DC.%26aulast%3DJihui%26aufirst%3DJ.%26atitle%3DMicroRNA-212%2520inhibits%2520proliferation%2520of%2520gastric%2520cancer%2520by%2520directly%2520repressing%2520retinoblastoma%2520binding%2520protein%25202%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2013%26volume%3D114%26spage%3D2666%26epage%3D2672%26doi%3D10.1002%2Fjcb.24613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fattaey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dembski, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuocolo, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanobik, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Defeo-Jones, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of the retinoblastoma binding proteins RBP1 and RBP2</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3149</span>– <span class="NLM_lpage">3156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1993&pages=3149-3156&author=A.+R.+Fattaeyauthor=K.+Helinauthor=M.+S.+Dembskiauthor=N.+Dysonauthor=E.+Harlowauthor=G.+A.+Vuocoloauthor=M.+G.+Hanobikauthor=K.+M.+Haskellauthor=A.+Oliffauthor=D.+Defeo-Jones&title=Characterization+of+the+retinoblastoma+binding+proteins+RBP1+and+RBP2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFattaey%26aufirst%3DA.%2BR.%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDembski%26aufirst%3DM.%2BS.%26aulast%3DDyson%26aufirst%3DN.%26aulast%3DHarlow%26aufirst%3DE.%26aulast%3DVuocolo%26aufirst%3DG.%2BA.%26aulast%3DHanobik%26aufirst%3DM.%2BG.%26aulast%3DHaskell%26aufirst%3DK.%2BM.%26aulast%3DOliff%26aufirst%3DA.%26aulast%3DDefeo-Jones%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520the%2520retinoblastoma%2520binding%2520proteins%2520RBP1%2520and%2520RBP2%26jtitle%3DOncogene%26date%3D1993%26volume%3D8%26spage%3D3149%26epage%3D3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">Binding of pRB to the PHD protein RBP2 promotes cellular differentiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.molcel.2005.05.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=623-635&author=E.+V.+Benevolenskayaauthor=H.+L.+Murrayauthor=P.+Brantonauthor=R.+A.+Youngauthor=W.+G.+Kaelin&title=Binding+of+pRB+to+the+PHD+protein+RBP2+promotes+cellular+differentiation&doi=10.1016%2Fj.molcel.2005.05.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMurray%26aufirst%3DH.%2BL.%26aulast%3DBranton%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DBinding%2520of%2520pRB%2520to%2520the%2520PHD%2520protein%2520RBP2%2520promotes%2520cellular%2520differentiation%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D623%26epage%3D635%26doi%3D10.1016%2Fj.molcel.2005.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beshiri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisanzio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamishima, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Loss of the retinoblastoma binding protein 2 (RBP2) histone demethylase suppresses tumorigenesis in mice lacking Rb1 or Men1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13379</span>– <span class="NLM_lpage">13386</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110104108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1073%2Fpnas.1110104108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13379-13386&author=W.+Linauthor=J.+Caoauthor=J.+Liuauthor=M.+L.+Beshiriauthor=Y.+Fujiwaraauthor=J.+Francisauthor=A.+D.+Cherniackauthor=C.+Geisenauthor=L.+P.+Blairauthor=M.+R.+Zouauthor=X.+Shenauthor=D.+Kawamoriauthor=Z.+Liuauthor=C.+Grisanzioauthor=H.+Watanabeauthor=Y.+A.+Minamishimaauthor=Q.+Zhangauthor=R.+N.+Kulkarniauthor=S.+Signorettiauthor=S.+J.+Rodigauthor=R.+T.+Bronsonauthor=S.+H.+Orkinauthor=D.+P.+Tuckauthor=E.+V.+Benevolenskayaauthor=M.+Meyersonauthor=W.+G.+Kaelinauthor=Q.+Yan&title=Loss+of+the+retinoblastoma+binding+protein+2+%28RBP2%29+histone+demethylase+suppresses+tumorigenesis+in+mice+lacking+Rb1+or+Men1&doi=10.1073%2Fpnas.1110104108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110104108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110104108%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBeshiri%26aufirst%3DM.%2BL.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DGeisen%26aufirst%3DC.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DKawamori%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DGrisanzio%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DMinamishima%26aufirst%3DY.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DLoss%2520of%2520the%2520retinoblastoma%2520binding%2520protein%25202%2520%2528RBP2%2529%2520histone%2520demethylase%2520suppresses%2520tumorigenesis%2520in%2520mice%2520lacking%2520Rb1%2520or%2520Men1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13379%26epage%3D13386%26doi%3D10.1073%2Fpnas.1110104108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 is critical for breast cancer progression and metastasis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.celrep.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=24582965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=868-877&author=J.+Caoauthor=Z.+Liuauthor=W.+K.+Cheungauthor=M.+Zhaoauthor=S.+Y.+Chenauthor=S.+W.+Chanauthor=C.+J.+Boothauthor=D.+X.+Nguyenauthor=Q.+Yan&title=Histone+demethylase+RBP2+is+critical+for+breast+cancer+progression+and+metastasis&doi=10.1016%2Fj.celrep.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Demethylase RBP2 Is Critical for Breast Cancer Progression and Metastasis</span></div><div class="casAuthors">Cao, Jian; Liu, Zongzhi; Cheung, William K. C.; Zhao, Minghui; Chen, Sophia Y.; Chan, Siew Wee; Booth, Carmen J.; Nguyen, Don X.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-877</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metastasis is a major clin. challenge for cancer treatment.  Emerging evidence suggests that aberrant epigenetic modifications contribute significantly to tumor formation and progression.  However, the drivers and roles of such epigenetic changes in tumor metastasis are still poorly understood.  Using bioinformatic anal. of human breast cancer gene-expression data sets, we identified histone demethylase RBP2 as a putative mediator of metastatic progression.  By using both human breast cancer cells and genetically engineered mice, we demonstrated that RBP2 is crit. for breast cancer metastasis to the lung in multiple in vivo models.  Mechanistically, RBP2 promotes metastasis as a pleiotropic pos. regulator of many metastasis genes, including TNC.  In addn., RBP2 loss suppresses tumor formation in MMTV-neu transgenic mice.  These results suggest that therapeutic targeting of RBP2 is a potential strategy for inhibition of tumor progression and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXZwTcPjZTYLVg90H21EOLACvtfcHk0lgWe3t6ZkX4ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D&md5=ecc62cbf9e2bcab60746534aa536b87d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DCheung%26aufirst%3DW.%2BK.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%2BY.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DBooth%26aufirst%3DC.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BX.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520is%2520critical%2520for%2520breast%2520cancer%2520progression%2520and%2520metastasis%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D868%26epage%3D877%26doi%3D10.1016%2Fj.celrep.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannappan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, S.</span></span> <span> </span><span class="NLM_article-title">RBP2 induces stem-like cancer cells by promoting EMT and is a prognostic marker for renal cell carcinoma</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">e238</span>, <span class="refDoi"> DOI: 10.1038/emm.2016.37</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Femm.2016.37" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27282106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XptV2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=e238&author=D.+Zhouauthor=V.+Kannappanauthor=X.+Chenauthor=J.+Liauthor=X.+Lengauthor=J.+Zhangauthor=S.+Xuan&title=RBP2+induces+stem-like+cancer+cells+by+promoting+EMT+and+is+a+prognostic+marker+for+renal+cell+carcinoma&doi=10.1038%2Femm.2016.37"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 induces stem-like cancer cells by promoting EMT and is a prognostic marker for renal cell carcinoma</span></div><div class="casAuthors">Zhou, Dahai; Kannappan, Vinodh; Chen, Xiang; Li, Jingqin; Leng, Xuefeng; Zhang, Jinping; Xuan, Shiying</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e238</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Renal cell carcinoma (RCC), one of the most common kidney cancers, has a poor prognosis.  Epithelial to mesenchymal transition (EMT) is a hallmark of carcinoma invasion and metastasis.  Several studies have examd. the mol. regulation of EMT, but the relationship between histone demethylases and EMT is little understood.  In this study, we investigated the role of retinoblastoma-binding protein-2 (RBP2), a histone demethylase that is highly expressed in RCC and is pos. correlated with poor RCC prognosis in the regulation of EMT.  We found that ectopic overexpression of RBP2 can induce cancer stem cell-like (CSC) phenotypes through EMT in RCC cells by converting them to a more mesenchymal phenotype.  This results in increased resistance to apoptosis, which leads to enhanced tumor growth in xenograft models.  Together, our data show that RBP2 is an epigenetic regulator that has an important role in the initiation of CSC phenotypes through EMT, leading to tumor progression.  RBP2 is also a novel biomol. for RCC diagnosis, and prognosis and may be a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9Gdi_fboErVg90H21EOLACvtfcHk0ljIKbVmdp7TkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptV2is7o%253D&md5=beced570b811a36b2f35328e58c6ae37</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Femm.2016.37&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2016.37%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DKannappan%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DXuan%26aufirst%3DS.%26atitle%3DRBP2%2520induces%2520stem-like%2520cancer%2520cells%2520by%2520promoting%2520EMT%2520and%2520is%2520a%2520prognostic%2520marker%2520for%2520renal%2520cell%2520carcinoma%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2016%26volume%3D48%26spage%3De238%26doi%3D10.1038%2Femm.2016.37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ci, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 promotes malignant progression of gastric cancer through TGF-beta1-(p-Smad3)-RBP2-E-cadherin-Smad3 feedback circuit</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17661</span>– <span class="NLM_lpage">17674</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.3756" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17661-17674&author=X.+Liangauthor=J.+Zengauthor=L.+Wangauthor=L.+Shenauthor=X.+Maauthor=S.+Liauthor=Y.+Wuauthor=L.+Maauthor=X.+Ciauthor=Q.+Guoauthor=M.+Jiaauthor=H.+Shenauthor=Y.+Sunauthor=Z.+Liuauthor=S.+Liuauthor=W.+Liauthor=H.+Yuauthor=C.+Chenauthor=J.+Jia&title=Histone+demethylase+RBP2+promotes+malignant+progression+of+gastric+cancer+through+TGF-beta1-%28p-Smad3%29-RBP2-E-cadherin-Smad3+feedback+circuit&doi=10.18632%2Foncotarget.3756"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3756%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DCi%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DJ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520promotes%2520malignant%2520progression%2520of%2520gastric%2520cancer%2520through%2520TGF-beta1-%2528p-Smad3%2529-RBP2-E-cadherin-Smad3%2520feedback%2520circuit%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17661%26epage%3D17674%26doi%3D10.18632%2Foncotarget.3756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azizian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2010.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20371346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=69-80&author=S.+V.+Sharmaauthor=D.+Y.+Leeauthor=B.+Liauthor=M.+P.+Quinlanauthor=F.+Takahashiauthor=S.+Maheswaranauthor=U.+McDermottauthor=N.+Azizianauthor=L.+Zouauthor=M.+A.+Fischbachauthor=K.+K.+Wongauthor=K.+Brandstetterauthor=B.+Wittnerauthor=S.+Ramaswamyauthor=M.+Classonauthor=J.+Settleman&title=A+chromatin-mediated+reversible+drug-tolerant+state+in+cancer+cell+subpopulations&doi=10.1016%2Fj.cell.2010.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span></div><div class="casAuthors">Sharma, Sreenath V.; Lee, Diana Y.; Li, Bihua; Quinlan, Margaret P.; Takahashi, Fumiyuki; Maheswaran, Shyamala; McDermott, Ultan; Azizian, Nancy; Zou, Lee; Fischbach, Michael A.; Wong, Kwok-Kin; Brandstetter, Kathleyn; Wittner, Ben; Ramaswamy, Sridhar; Classon, Marie; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-80</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Accumulating evidence implicates heterogeneity within cancer cell populations in the response to stressful exposures, including drug treatments.  While modeling the acute response to various anticancer agents in drug-sensitive human tumor cell lines, we consistently detected a small subpopulation of reversibly "drug-tolerant" cells.  These cells demonstrate >100-fold reduced drug sensitivity and maintain viability via engagement of IGF-1 receptor signaling and an altered chromatin state that requires the histone demethylase RBP2/KDM5A/Jarid1A.  This drug-tolerant phenotype is transiently acquired and relinquished at low frequency by individual cells within the population, implicating the dynamic regulation of phenotypic heterogeneity in drug tolerance.  The drug-tolerant subpopulation can be selectively ablated by treatment with IGF-1 receptor inhibitors or chromatin-modifying agents, potentially yielding a therapeutic opportunity.  Together, these findings suggest that cancer cell populations employ a dynamic survival strategy in which individual cells transiently assume a reversibly drug-tolerant state to protect the population from eradication by potentially lethal exposures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK0DpI9mbR_bVg90H21EOLACvtfcHk0ljIKbVmdp7TkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D&md5=2070b7cc4385a4f98db15e5ebcd4589d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DLee%26aufirst%3DD.%2BY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQuinlan%26aufirst%3DM.%2BP.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DAzizian%26aufirst%3DN.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DBrandstetter%26aufirst%3DK.%26aulast%3DWittner%26aufirst%3DB.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520chromatin-mediated%2520reversible%2520drug-tolerant%2520state%2520in%2520cancer%2520cell%2520subpopulations%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.cell.2010.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carra, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parodi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattarozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carosio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forlani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marubbi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romani, M.</span></span> <span> </span><span class="NLM_article-title">The histone demethylase KDM5A is a key factor for the resistance to Temozolomide in glioblastoma</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3418</span>– <span class="NLM_lpage">3429</span>, <span class="refDoi"> DOI: 10.1080/15384101.2015.1090063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1080%2F15384101.2015.1090063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=3418-3429&author=B.+Banelliauthor=E.+Carraauthor=F.+Barbieriauthor=R.+Wurthauthor=F.+Parodiauthor=A.+Pattarozziauthor=R.+Carosioauthor=A.+Forlaniauthor=G.+Allemanniauthor=D.+Marubbiauthor=T.+Florioauthor=A.+Dagaauthor=M.+Romani&title=The+histone+demethylase+KDM5A+is+a+key+factor+for+the+resistance+to+Temozolomide+in+glioblastoma&doi=10.1080%2F15384101.2015.1090063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1080%2F15384101.2015.1090063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2015.1090063%26sid%3Dliteratum%253Aachs%26aulast%3DBanelli%26aufirst%3DB.%26aulast%3DCarra%26aufirst%3DE.%26aulast%3DBarbieri%26aufirst%3DF.%26aulast%3DWurth%26aufirst%3DR.%26aulast%3DParodi%26aufirst%3DF.%26aulast%3DPattarozzi%26aufirst%3DA.%26aulast%3DCarosio%26aufirst%3DR.%26aulast%3DForlani%26aufirst%3DA.%26aulast%3DAllemanni%26aufirst%3DG.%26aulast%3DMarubbi%26aufirst%3DD.%26aulast%3DFlorio%26aufirst%3DT.%26aulast%3DDaga%26aufirst%3DA.%26aulast%3DRomani%26aufirst%3DM.%26atitle%3DThe%2520histone%2520demethylase%2520KDM5A%2520is%2520a%2520key%2520factor%2520for%2520the%2520resistance%2520to%2520Temozolomide%2520in%2520glioblastoma%26jtitle%3DCell%2520Cycle%26date%3D2015%26volume%3D14%26spage%3D3418%26epage%3D3429%26doi%3D10.1080%2F15384101.2015.1090063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tateishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klose, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabrizio, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor-Papadimitriou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">PLU-1 is an H3K4 demethylase involved in transcriptional repression and breast cancer cell proliferation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.molcel.2007.03.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=801-812&author=K.+Yamaneauthor=K.+Tateishiauthor=R.+J.+Kloseauthor=J.+Fangauthor=L.+A.+Fabrizioauthor=H.+Erdjument-Bromageauthor=J.+Taylor-Papadimitriouauthor=P.+Tempstauthor=Y.+Zhang&title=PLU-1+is+an+H3K4+demethylase+involved+in+transcriptional+repression+and+breast+cancer+cell+proliferation&doi=10.1016%2Fj.molcel.2007.03.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DTateishi%26aufirst%3DK.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DFabrizio%26aufirst%3DL.%2BA.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTaylor-Papadimitriou%26aufirst%3DJ.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPLU-1%2520is%2520an%2520H3K4%2520demethylase%2520involved%2520in%2520transcriptional%2520repression%2520and%2520breast%2520cancer%2520cell%2520proliferation%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D801%26epage%3D812%26doi%3D10.1016%2Fj.molcel.2007.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Network</span> <span> </span><span class="NLM_article-title">Comprehensive
molecular portraits of human breast tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1038/nature11412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature11412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23000897" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=61-70&author=Cancer%0AGenome+Atlas+Network&title=Comprehensive%0Amolecular+portraits+of+human+breast+tumours&doi=10.1038%2Fnature11412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature11412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11412%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%250Amolecular%2520portraits%2520of%2520human%2520breast%2520tumours%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D61%26epage%3D70%26doi%3D10.1038%2Fnature11412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span> <span> </span><span class="NLM_article-title">Overexpression of JARID1B is associated with poor prognosis and chemotherapy resistance in epithelial ovarian cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2859-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2Fs13277-014-2859-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisleiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=2465-2472&author=L.+Wangauthor=Y.+Maoauthor=G.+Duauthor=C.+Heauthor=S.+Han&title=Overexpression+of+JARID1B+is+associated+with+poor+prognosis+and+chemotherapy+resistance+in+epithelial+ovarian+cancer&doi=10.1007%2Fs13277-014-2859-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of JARID1B is associated with poor prognosis and chemotherapy resistance in epithelial ovarian cancer</span></div><div class="casAuthors">Wang, Lishuang; Mao, Yuanfu; Du, Guiqin; He, Chunbo; Han, Shiyu</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2465-2472</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">JARID1B, a histone demethylase, has been reported to be highly expressed in various human cancers.  In the present study, we investigated the assocn. of JARID1B level with epithelial ovarian cancer (EOC) and prognosis of patients with EOC.  We analyzed JARID1B expression in 20 normal ovaries, 20 benign ovarian tumor (BOT) samples, and 45 epithelial ovarian carcinoma specimens by quant. PCR (qRT-PCR) and western blotting analyses.  JARID1B was further examd. in 120 EOC specimens from patients with different histol. stages via immunohistochem.  Possible correlations between JARID1B levels and prognosis as well as chemotherapy resistance of EOC patients were detd. by univariate and multivariate analyses.  JARID1B level was significantly increased in EOC, as compared to normal ovaries and BOT.  Among 120 EOC cases examd., the 5-yr progression-free survival (PFS) rates were 17 and 85 % in patients with high and low JARID1B expression, resp. (hazard ratio = 17.85, 95 % confidence interval (CI) 6.31-50.51, P < 0.001).  Similarly, the 5-yr overall survival (OS) rates for patients with high and low JARID1B expression were 28 and 92 % resp. (hazard ratio = 21.8, 95 % CI 5.92-71.81, P < 0.001).  Pos. correlation between JARID1B level and chemotherapy resistance was obsd. in patients with EOC (odds ratio (OR) 36.81, 95 % CI 4.84-280.11, P < 0.001).  JARID1B could serve as an important biomarker for prognosis and chemotherapy resistance of EOC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCGJs4I05U0LVg90H21EOLACvtfcHk0liiR5bFrPY9lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisleiurw%253D&md5=454837068da870bda633be1e4ba80a27</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2859-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2859-z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DS.%26atitle%3DOverexpression%2520of%2520JARID1B%2520is%2520associated%2520with%2520poor%2520prognosis%2520and%2520chemotherapy%2520resistance%2520in%2520epithelial%2520ovarian%2520cancer%26jtitle%3DTumor%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D2465%26epage%3D2472%26doi%3D10.1007%2Fs13277-014-2859-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span> <span> </span><span class="NLM_article-title">JARID1B promotes metastasis and epithelial-mesenchymal transition via PTEN/AKT signaling in hepatocellular carcinoma cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12723</span>– <span class="NLM_lpage">12739</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.3713" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12723-12739&author=B.+Tangauthor=G.+Qiauthor=F.+Tangauthor=S.+Yuanauthor=Z.+Wangauthor=X.+Liangauthor=B.+Liauthor=S.+Yuauthor=J.+Liuauthor=Q.+Huangauthor=Y.+Weiauthor=R.+Zhaiauthor=B.+Leiauthor=H.+Yuauthor=X.+Jiaoauthor=S.+He&title=JARID1B+promotes+metastasis+and+epithelial-mesenchymal+transition+via+PTEN%2FAKT+signaling+in+hepatocellular+carcinoma+cells&doi=10.18632%2Foncotarget.3713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3713%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DB.%26aulast%3DQi%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DZhai%26aufirst%3DR.%26aulast%3DLei%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26atitle%3DJARID1B%2520promotes%2520metastasis%2520and%2520epithelial-mesenchymal%2520transition%2520via%2520PTEN%252FAKT%2520signaling%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D12723%26epage%3D12739%26doi%3D10.18632%2Foncotarget.3713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span> <span> </span><span class="NLM_article-title">KDM5B is overexpressed in gastric cancer and is required for gastric cancer cell proliferation and metastasis</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25628922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=87-100&author=Z.+Wangauthor=F.+Tangauthor=G.+Qiauthor=S.+Yuanauthor=G.+Zhangauthor=B.+Tangauthor=S.+He&title=KDM5B+is+overexpressed+in+gastric+cancer+and+is+required+for+gastric+cancer+cell+proliferation+and+metastasis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">KDM5B is overexpressed in gastric cancer and is required for gastric cancer cell proliferation and metastasis</span></div><div class="casAuthors">Wang, Zhenran; Tang, Fang; Qi, Guangying; Yuan, Shengguang; Zhang, Guangyu; Tang, Bo; He, Songqing</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-100</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Epigenetic alterations such as aberrant expression of histone-modifying enzymes have been implicated in tumorigenesis.  KDM5B (also known as JARID1B) is a newly identified histone demethylase that regulates chromatin structure or gene expression by removing Me residues from trimethylated lysine 4 on histone H3.  Recent observations have shown oncogenic activity of KDM5B.  However, the role of KDM5B in gastric cancer carcinogenesis remains unclear.  In this study, we aimed to investigate the role of KDM5B in gastric cancer.  Immunohistochem. anal., western blotting, and qRT-PCR were used to measure the levels of KDM5B in gastric cancer cell lines, 45 pairs of gastric cancer tissues and the adjacent nonneoplastic tissues.  KDM5B and shKDM5B were transfected into gastric cancer cells to investigate its role on regulating cell proliferation which was measured by MTT and colony formation assay.  Cell's migration and invasion were measured by Transwell and Matrigel anal. in vitro.  PCNA expression was measured by immunofluorescence staining and immunohistochem. anal.  The in vivo tumorigenesis and metastasis assays were performed in SCID mice.  In clin. gastric cancer samples, we found that KDM5B expression was significantly up-regulated in cancer lesions compared with paired normal gastric tissues.  By silencing or overexpressing KDM5B in gastric cancer cells, we found that KDM5B could promote cell growth and metastasis in vitro.  An in vivo assay showed that KDM5B not only dramatically promoted gastric cancer cell xenograft formation and growth but also promoted gastric cancer cell metastasis in a liver metastasis model.  Moreover, we demonstrated that KDM5B promoted gastric cancer metastasis via regulation of the Akt pathway.  Our study provided evidence that KDM5B functions as a novel tumor oncogene in gastric cancer and may be a potential therapeutic target for gastric cancer management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWbphoPhfOT7Vg90H21EOLACvtfcHk0ljsVHhrtLLH5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFOrtL4%253D&md5=3efc8f5ca6162a10712837e0c0b8155b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DS.%26atitle%3DKDM5B%2520is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520is%2520required%2520for%2520gastric%2520cancer%2520cell%2520proliferation%2520and%2520metastasis%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2015%26volume%3D5%26spage%3D87%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adebayo, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span> <span> </span><span class="NLM_article-title">Silencing JARID1B suppresses oncogenicity, stemness and increases radiation sensitivity in human oral carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2015.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.canlet.2015.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26184998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOkt73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2015&pages=36-45&author=C.+S.+Linauthor=Y.+C.+Linauthor=B.+O.+Adebayoauthor=A.+Wuauthor=J.+H.+Chenauthor=Y.+J.+Pengauthor=M.+F.+Chengauthor=W.+H.+Leeauthor=M.+Hsiaoauthor=T.+Y.+Chaoauthor=C.+T.+Yeh&title=Silencing+JARID1B+suppresses+oncogenicity%2C+stemness+and+increases+radiation+sensitivity+in+human+oral+carcinoma&doi=10.1016%2Fj.canlet.2015.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing JARID1B suppresses oncogenicity, stemness and increases radiation sensitivity in human oral carcinoma</span></div><div class="casAuthors">Lin, Chun-Shu; Lin, Ying-Chin; Adebayo, Bamodu Oluwaseun; Wu, Alexander; Chen, Jia-Hong; Peng, Yi-Jen; Cheng, Ming-Fang; Lee, Wei-Hwa; Hsiao, Michael; Chao, Tsu-Yi; Yeh, Chi-Tai</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-45</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Oral squamous cell carcinoma (OSCC) is a major cause of human mortality globally and radiotherapy is one of the main treatment modalities, however its therapeutic effect is often limited by radioresistance.  JARID1B is an epigenetic factor with reported oncogenic potential in various cancer types.  We investigated the effect of JARID1B inhibition on migration and invasion of human OSCC cell lines, as well as on clin. patients' outcome.  Wound healing, matrigel invasion, Sulforhodamine B, and spheroid formation assays were used to characterize the signaling pathways of shJARID1B in response to radiation treatment.  We evaluated the prognostic relevance of Jarid1b expression in a cohort of 81 OSCC patients.  Human OSCC cell lines, including SAS, HSC3, Cal27, TW2.6 and SCC4 cells, were used.  ShJARID1B cells significantly inhibited migration and invasion ability compared to their vector or wild type counterparts.  Silencing shJARID1B significantly inhibited oral cancer stem cell activity and potentiated the tumor-inhibitory activity of radiation therapy in OSCC.  Radiotherapy coupled with shJARID1B knockdown reduced mRNA levels of NQO1, KEAP1, NRF2, FOXO1, FOXO3, KLF4, OCT4, CD133, and Nanog in malignant OSCC cells.  OSCC spheroid formation ability was markedly reduced in the shJARID1B cells.  JARID1B overexpression is a dependent prognostic factor in OSCC patients.  Silencing shJARID1B inhibits migration and invasion of human OSCC, reduces cancer stem cell activities and potentiates tumor-inhibiting radiotherapeutic effects.  JARID1B knockdown prior to radiotherapy is a potential effective therapeutic strategy for the treatment of OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIgWCI5f2rwLVg90H21EOLACvtfcHk0lhVYqAXEQ6fYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOkt73J&md5=9ada7ae0e6817152339cada752a18c3d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2015.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2015.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BS.%26aulast%3DLin%26aufirst%3DY.%2BC.%26aulast%3DAdebayo%26aufirst%3DB.%2BO.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DPeng%26aufirst%3DY.%2BJ.%26aulast%3DCheng%26aufirst%3DM.%2BF.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DChao%26aufirst%3DT.%2BY.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26atitle%3DSilencing%2520JARID1B%2520suppresses%2520oncogenicity%252C%2520stemness%2520and%2520increases%2520radiation%2520sensitivity%2520in%2520human%2520oral%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D368%26spage%3D36%26epage%3D45%26doi%3D10.1016%2Fj.canlet.2015.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adebayo, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. M.</span></span> <span> </span><span class="NLM_article-title">JARID1B expression plays a critical role in chemoresistance and stem cell-like phenotype of neuroblastoma cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0125343</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0125343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0125343" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0125343&author=Y.+T.+Kuoauthor=Y.+L.+Liuauthor=B.+O.+Adebayoauthor=P.+H.+Shihauthor=W.+H.+Leeauthor=L.+S.+Wangauthor=Y.+F.+Liaoauthor=W.+M.+Hsuauthor=C.+T.+Yehauthor=C.+M.+Lin&title=JARID1B+expression+plays+a+critical+role+in+chemoresistance+and+stem+cell-like+phenotype+of+neuroblastoma+cells&doi=10.1371%2Fjournal.pone.0125343"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0125343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0125343%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DY.%2BT.%26aulast%3DLiu%26aufirst%3DY.%2BL.%26aulast%3DAdebayo%26aufirst%3DB.%2BO.%26aulast%3DShih%26aufirst%3DP.%2BH.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DL.%2BS.%26aulast%3DLiao%26aufirst%3DY.%2BF.%26aulast%3DHsu%26aufirst%3DW.%2BM.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26aulast%3DLin%26aufirst%3DC.%2BM.%26atitle%3DJARID1B%2520expression%2520plays%2520a%2520critical%2520role%2520in%2520chemoresistance%2520and%2520stem%2520cell-like%2520phenotype%2520of%2520neuroblastoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0125343%26doi%3D10.1371%2Fjournal.pone.0125343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamodu, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, T. Y.</span></span> <span> </span><span class="NLM_article-title">Aberrant KDM5B expression promotes aggressive breast cancer through MALAT1 overexpression and downregulation of hsa-miR-448</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">160</span>, <span class="refDoi"> DOI: 10.1186/s12885-016-2108-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1186%2Fs12885-016-2108-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26917489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=160&author=O.+A.+Bamoduauthor=W.+C.+Huangauthor=W.+H.+Leeauthor=A.+Wuauthor=L.+S.+Wangauthor=M.+Hsiaoauthor=C.+T.+Yehauthor=T.+Y.+Chao&title=Aberrant+KDM5B+expression+promotes+aggressive+breast+cancer+through+MALAT1+overexpression+and+downregulation+of+hsa-miR-448&doi=10.1186%2Fs12885-016-2108-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant KDM5B expression promotes aggressive breast cancer through MALAT1 overexpression and downregulation of hsa-miR-448</span></div><div class="casAuthors">Bamodu, Oluwaseun Adebayo; Huang, Wen-Chien; Lee, Wei-Hwa; Wu, Alexander; Wang, Liang Shun; Hsiao, Michael; Yeh, Chi-Tai; Chao, Tsu-Yi</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">160/1-160/18</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Triple neg. breast cancers (TNBC) possess cell dedifferentiation characteristics, carry out activities connate to those of cancer stem cells (CSCs) and are assocd. with increased metastasis, as well as, poor clin. prognosis.  The regulatory mechanism of this highly malignant phenotype is still poorly characterized.  Accruing evidence support the role of non-coding RNAs (ncRNAs) as potent regulators of CSC and metastatic gene expression, with their dysregulation implicated in tumorigenesis and disease progression.  Methods: In this study, we investigated TNBC metastasis, metastasis-assocd. genes and potential inhibitory mechanisms using bioinformatics, tissue microarray analyses, immunoblotting, polymerase chain reaction, loss and gain of gene function assays and comparative analyses of data obtained.  Results: Compared with other breast cancer types, the highly metastatic MDA-MB-231 cells concurrently exhibited increased expression levels of Lysine-specific demethylase 5B protein (KDM5B) and long non-coding RNA (lncRNA), MALAT1, suggesting their functional assocn.  KDM5B-silencing in the TNBC cells correlated with the upregulation of hsa-miR-448 and led to suppression of MALAT1 expression with decreased migration, invasion and clonogenic capacity in vitro, as well as, poor survival in vivo.  This projects MALAT1 as a mediator of KDM5B oncogenic potential and highlights the crit. role of this microRNA, lncRNA and histone demethylase in cancer cell motility and metastatic colonization.  Increased expression of KDM5B correlating with disease progression and poor clin. outcome in breast cancer was reversed by hsa-miR-448.  Conclusions: Our findings demonstrate the crit. role of KDM5B and its neg. regulator hsa-miR-448 in TNBC metastasis and progression.  Hsa-miR-448 disrupting KDM5B-MALAT1 signalling axis and assocd. activities in TNBC cells, projects it as a putative therapeutic factor for selective eradication of TNBC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHXcNra2EJ3rVg90H21EOLACvtfcHk0lhVYqAXEQ6fYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWmtb8%253D&md5=74b8458814f7349adef040c96dcb7914</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2Fs12885-016-2108-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-016-2108-5%26sid%3Dliteratum%253Aachs%26aulast%3DBamodu%26aufirst%3DO.%2BA.%26aulast%3DHuang%26aufirst%3DW.%2BC.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%2BS.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26aulast%3DChao%26aufirst%3DT.%2BY.%26atitle%3DAberrant%2520KDM5B%2520expression%2520promotes%2520aggressive%2520breast%2520cancer%2520through%2520MALAT1%2520overexpression%2520and%2520downregulation%2520of%2520hsa-miR-448%26jtitle%3DBMC%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D160%26doi%3D10.1186%2Fs12885-016-2108-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0lg03QokfzOE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of mm1s myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+Mullerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+mm1s+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of ∼50 μM and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0lg03QokfzOE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520mm1s%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for KDM5A histone lysine demethylase inhibition by diverse compounds</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.chembiol.2016.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27427228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=769-781&author=J.+R.+Hortonauthor=X.+Liuauthor=M.+Galeauthor=L.+Wuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=P.+J.+Webberauthor=J.+S.+Bellauthor=S.+C.+Kalesauthor=B.+T.+Mottauthor=G.+Raiauthor=D.+J.+Jansenauthor=M.+J.+Hendersonauthor=D.+J.+Urbanauthor=M.+D.+Hallauthor=A.+Simeonovauthor=D.+J.+Maloneyauthor=M.+A.+Johnsauthor=H.+Fuauthor=A.+Jadhavauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Structural+basis+for+KDM5A+histone+lysine+demethylase+inhibition+by+diverse+compounds&doi=10.1016%2Fj.chembiol.2016.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for KDM5A Histone Lysine Demethylase Inhibition by Diverse Compounds</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Gale, Molly; Wu, Lizhen; Shanks, John R.; Zhang, Xing; Webber, Philip J.; Bell, Joshua S. K.; Kales, Stephen C.; Mott, Bryan T.; Rai, Ganesha; Jansen, Daniel J.; Henderson, Mark J.; Urban, Daniel J.; Hall, Matthew D.; Simeonov, Anton; Maloney, David J.; Johns, Margaret A.; Fu, Haian; Jadhav, Ajit; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">769-781</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases removes Me groups from methylated lysine 4 of histone H3.  Accumulating evidence supports a role for KDM5 family members as oncogenic drivers.  We compare the in vitro inhibitory properties and binding affinity of ten diverse compds. with all four family members, and present the crystal structures of the KDM5A-linked Jumonji domain in complex with eight of these inhibitors in the presence of Mn(II).  All eight inhibitors structurally examd. occupy the binding site of α-ketoglutarate, but differ in their specific binding interactions, including the no. of ligands involved in metal coordination.  We also obsd. inhibitor-induced conformational changes in KDM5A, particularly those residues involved in the binding of α-ketoglutarate, the anticipated peptide substrate, and intramol. interactions.  We discuss how particular chem. moieties contribute to inhibitor potency and suggest strategies that might be utilized in the successful design of selective and potent epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstbuqHdktk7Vg90H21EOLACvtfcHk0lg03QokfzOE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L&md5=0dc780604904ac406211a09577df5bf5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGale%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWebber%26aufirst%3DP.%2BJ.%26aulast%3DBell%26aufirst%3DJ.%2BS.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520KDM5A%2520histone%2520lysine%2520demethylase%2520inhibition%2520by%2520diverse%2520compounds%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D769%26epage%3D781%26doi%3D10.1016%2Fj.chembiol.2016.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaThangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnchembio.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27214403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=539-545&author=C.+Johanssonauthor=S.+Velupillaiauthor=A.+Tumberauthor=A.+Szykowskaauthor=E.+S.+Hookwayauthor=R.+P.+Nowakauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=M.+Philpottauthor=N.+Burgess-Brownauthor=N.+Wuauthor=J.+Kopecauthor=A.+Nuzziauthor=H.+Steuberauthor=U.+Egnerauthor=V.+Badockauthor=S.+Munroauthor=N.+B.+LaThangueauthor=S.+Westawayauthor=J.+Brownauthor=N.+Athanasouauthor=R.+Prinjhaauthor=P.+E.+Brennanauthor=U.+Oppermann&title=Structural+analysis+of+human+KDM5B+guides+histone+demethylase+inhibitor+development&doi=10.1038%2Fnchembio.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span></div><div class="casAuthors">Johansson, Catrine; Velupillai, Srikannathasan; Tumber, Anthony; Szykowska, Aleksandra; Hookway, Edward S.; Nowak, Radoslaw P.; Strain-Damerell, Claire; Gileadi, Carina; Philpott, Martin; Burgess-Brown, Nicola; Wu, Na; Kopec, Jola; Nuzzi, Andrea; Steuber, Holger; Egner, Ursula; Badock, Volker; Munro, Shonagh; LaThangue, Nicholas B.; Westaway, Sue; Brown, Jack; Athanasou, Nick; Prinjha, Rab; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-545</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the KDM5 (also known as JARID1) family are 2-oxoglutarate- and Fe2+-dependent oxygenases that act as histone H3K4 demethylases, thereby regulating cell proliferation and stem cell self-renewal and differentiation.  Here, the authors report crystal structures of the catalytic core of the human KDM5B enzyme in complex with 3 inhibitor chemotypes.  These scaffolds exploitet several aspects of the KDM5 active site, and their selectivity profiles reflected their hybrid features with respect to the KDM4 and KDM6 families.  Whereas GSK-J1, a previously identified KDM6 inhibitor, showed ∼7-fold less inhibitory activity toward KDM5B than toward KDM6 proteins, KDM5-C49 displayed 25-100-fold selectivity between KDM5B and KDM6B.  The cell-permeable deriv. KDM5-C70 had an antiproliferative effect in myeloma cells, leading to genome-wide elevation of H3K4me3 levels.  The selective inhibitor GSK467 exploited unique binding modes, but it lacked cellular potency in the myeloma system.  Taken together, these structural leads deliver multiple starting points for further rational and selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp-2vDMpTjHbVg90H21EOLACvtfcHk0liDvP4V_oG87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D&md5=2c4b6e5779ba43ed3584fc6f76b987cd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2087%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DKopec%26aufirst%3DJ.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLaThangue%26aufirst%3DN.%2BB.%26aulast%3DWestaway%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520analysis%2520of%2520human%2520KDM5B%2520guides%2520histone%2520demethylase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D539%26epage%3D545%26doi%3D10.1038%2Fnchembio.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham-Huy, C.</span></span> <span> </span><span class="NLM_article-title">Chiral drugs: an overview</span>. <i>Int. J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23674971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=85-100&author=L.+A.+Nguyenauthor=H.+Heauthor=C.+Pham-Huy&title=Chiral+drugs%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral drugs. An overview</span></div><div class="casAuthors">Nguyen, Lien Ai; He, Hua; Chuong, Pham-Huy</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biomedical Science (Monterey Park, CA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>CODEN:
                <span class="NLM_cas:coden">IJBSDB</span>;
        ISSN:<span class="NLM_cas:issn">1550-9702</span>.
    
            (<span class="NLM_cas:orgname">Master Publishing Group</span>)
        </div><div class="casAbstract">A review.  About more than half of the drugs currently in use are chiral compds. and near 90% of the last ones are marketed as racemates consisting of an equimolar mixt. of two enantiomers.  Although they have the same chem. structure, most isomers of chiral drugs exhibit marked differences in biol. activities such as pharmacol., toxicol., pharmacokinetics, metab. etc.  Some mechanisms of these properties are also explained.  Therefore, it is important to promote the chiral sepn. and anal. of racemic drugs in pharmaceutical industry as well as in clinic in order to eliminate the unwanted isomer from the prepn. and to find an optimal treatment and a right therapeutic control for the patient.  In this article, we review the nomenclature, pharmacol., toxicol., pharmacokinetics, metab. etc of some usual chiral drugs as well as their mechanisms.  Different techniques used for the chiral sepn. in pharmaceutical industry as well as in clin. analyses are also examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovoqsLM_3bxLVg90H21EOLACvtfcHk0liDvP4V_oG87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ&md5=254cbca060bb4cfd04eac365488f5b62</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DL.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DPham-Huy%26aufirst%3DC.%26atitle%3DChiral%2520drugs%253A%2520an%2520overview%26jtitle%3DInt.%2520J.%2520Biomed.%2520Sci.%26date%3D2006%26volume%3D2%26spage%3D85%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chhabra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aseri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, D.</span></span> <span> </span><span class="NLM_article-title">A review of drug isomerism and its significance</span>. <i>Int. J. Appl. Basic. Med. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.4103/2229-516X.112233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.4103%2F2229-516X.112233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23776834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A280%3ADC%252BC3sjjsVymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=16-18&author=N.+Chhabraauthor=M.+L.+Aseriauthor=D.+Padmanabhan&title=A+review+of+drug+isomerism+and+its+significance&doi=10.4103%2F2229-516X.112233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A review of drug isomerism and its significance</span></div><div class="casAuthors">Chhabra Naveen; Aseri Madan L; Padmanabhan Deepak</div><div class="citationInfo"><span class="NLM_cas:title">International journal of applied & basic medical research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-8</span>
        ISSN:<span class="NLM_cas:issn">2229-516X</span>.
    </div><div class="casAbstract">Isomerism finds its importance in the field of clinical pharmacology and pharmacotherapeutics, as isomers differ in their pharmacokinetic and pharmacodyanmic properties.  Drug isomerism has opened a new era of drug development.  Currently, knowledge of isomerism has helped us in introducing safer and more effective drug alternatives of the newer as well as existing drugs.  Many existing drugs have gone chiral switch i.e., switching from racemic mixture to one of its isomers.  Cetrizine to levocetrizine is one of such examples, where effective and safer drug has been made available.  In this article, we have attempted to review the basic concepts of stereochemistry and chirality and their significance in pharmacotherapeutics.  Various pharmacological aspects such as pharmacokinetic and pharmacodynamic variations resulting out of chirality has been discussed in detail in this article.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMuPJrm3KbPqNc3HjJRoFAfW6udTcc2eZEnopCQ2y4z7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjjsVymtA%253D%253D&md5=a27d4e2765ceb4c3a86bc79ade0544a1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.4103%2F2229-516X.112233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F2229-516X.112233%26sid%3Dliteratum%253Aachs%26aulast%3DChhabra%26aufirst%3DN.%26aulast%3DAseri%26aufirst%3DM.%2BL.%26aulast%3DPadmanabhan%26aufirst%3DD.%26atitle%3DA%2520review%2520of%2520drug%2520isomerism%2520and%2520its%2520significance%26jtitle%3DInt.%2520J.%2520Appl.%2520Basic.%2520Med.%2520Res.%26date%3D2013%26volume%3D3%26spage%3D16%26epage%3D18%26doi%3D10.4103%2F2229-516X.112233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittinger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keenan, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liau, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span> <span> </span><span class="NLM_article-title">Cancer-associated IDH1 mutations produce 2-hydroxyglutarate</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature08617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature08617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=19935646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=739-744&author=L.+Dangauthor=D.+W.+Whiteauthor=S.+Grossauthor=B.+D.+Bennettauthor=M.+A.+Bittingerauthor=E.+M.+Driggersauthor=V.+R.+Fantinauthor=H.+G.+Jangauthor=S.+Jinauthor=M.+C.+Keenanauthor=K.+M.+Marksauthor=R.+M.+Prinsauthor=P.+S.+Wardauthor=K.+E.+Yenauthor=L.+M.+Liauauthor=J.+D.+Rabinowitzauthor=L.+C.+Cantleyauthor=C.+B.+Thompsonauthor=M.+G.+Vander%0AHeidenauthor=S.+M.+Su&title=Cancer-associated+IDH1+mutations+produce+2-hydroxyglutarate&doi=10.1038%2Fnature08617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated IDH1 mutations produce 2-hydroxyglutarate</span></div><div class="casAuthors">Dang, Lenny; White, David W.; Gross, Stefan; Bennett, Bryson D.; Bittinger, Mark A.; Driggers, Edward M.; Fantin, Valeria R.; Jang, Hyun-Gyung; Jin, Shengfang; Keenan, Marie C.; Marks, Kevin M.; Prins, Robert M.; Ward, Patrick S.; Yen, Katharine E.; Liau, Linda M.; Rabinowitz, Joshua D.; Cantley, Lewis C.; Thompson, Craig B.; Vander Heiden, Matthew G.; Su, Shinsan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7274</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in the enzyme cytosolic isocitrate dehydrogenase 1 (IDH1) are a common feature of a major subset of primary human brain cancers.  These mutations occur at a single amino acid residue of the IDH1 active site, resulting in loss of the enzyme's ability to catalyze conversion of isocitrate to α-ketoglutarate.  However, only a single copy of the gene is mutated in tumors, raising the possibility that the mutations do not result in a simple loss of function.  Here we show that cancer-assocd. IDH1 mutations result in a new ability of the enzyme to catalyze the NADPH-dependent redn. of α-ketoglutarate to R(-)-2-hydroxyglutarate (2HG).  Structural studies demonstrate that when arginine 132 is mutated to histidine, residues in the active site are shifted to produce structural changes consistent with reduced oxidative decarboxylation of isocitrate and acquisition of the ability to convert α-ketoglutarate to 2HG.  Excess accumulation of 2HG has been shown to lead to an elevated risk of malignant brain tumors in patients with inborn errors of 2HG metab.  Similarly, in human malignant gliomas harbouring IDH1 mutations, we find markedly elevated levels of 2HG.  These data demonstrate that the IDH1 mutations result in prodn. of the onco-metabolite 2HG, and indicate that the excess 2HG which accumulates in vivo contributes to the formation and malignant progression of gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkIgPaD-aavrVg90H21EOLACvtfcHk0lgsQyaKW7GIAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlurzE&md5=b8661605276da5cbd92641e45c5b8c87</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature08617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08617%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DL.%26aulast%3DWhite%26aufirst%3DD.%2BW.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DBittinger%26aufirst%3DM.%2BA.%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKeenan%26aufirst%3DM.%2BC.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DPrins%26aufirst%3DR.%2BM.%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DYen%26aufirst%3DK.%2BE.%26aulast%3DLiau%26aufirst%3DL.%2BM.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DSu%26aufirst%3DS.%2BM.%26atitle%3DCancer-associated%2520IDH1%2520mutations%2520produce%25202-hydroxyglutarate%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnature08617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coller, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedvat, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perl, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">The common feature of leukemia-associated IDH1 and IDH2 mutations is a neomorphic enzyme activity converting alpha-ketoglutarate to 2-hydroxyglutarate</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.01.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.ccr.2010.01.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20171147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=225-234&author=P.+S.+Wardauthor=J.+Patelauthor=D.+R.+Wiseauthor=O.+Abdel-Wahabauthor=B.+D.+Bennettauthor=H.+A.+Collerauthor=J.+R.+Crossauthor=V.+R.+Fantinauthor=C.+V.+Hedvatauthor=A.+E.+Perlauthor=J.+D.+Rabinowitzauthor=M.+Carrollauthor=S.+M.+Suauthor=K.+A.+Sharpauthor=R.+L.+Levineauthor=C.+B.+Thompson&title=The+common+feature+of+leukemia-associated+IDH1+and+IDH2+mutations+is+a+neomorphic+enzyme+activity+converting+alpha-ketoglutarate+to+2-hydroxyglutarate&doi=10.1016%2Fj.ccr.2010.01.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The common feature of leukemia-associated IDH1 and IDH2 mutations is a neomorphic enzyme activity converting α-ketoglutarate to 2-hydroxyglutarate</span></div><div class="casAuthors">Ward, Patrick S.; Patel, Jay; Wise, David R.; Abdel-Wahab, Omar; Bennett, Bryson D.; Coller, Hilary A.; Cross, Justin R.; Fantin, Valeria R.; Hedvat, Cyrus V.; Perl, Alexander E.; Rabinowitz, Joshua D.; Carroll, Martin; Su, Shinsan M.; Sharp, Kim A.; Levine, Ross L.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-234</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The somatic mutations in cytosolic isocitrate dehydrogenase 1 (IDH1) obsd. in gliomas can lead to the prodn. of 2-hydroxyglutarate (2HG).  Here, the authors report that tumor 2HG is elevated in a high percentage of patients with cytogenetically normal acute myeloid leukemia (AML).  Surprisingly, less than half of cases with elevated 2HG possessed IDH1 mutations.  The remaining cases with elevated 2HG had mutations in IDH2, the mitochondrial homolog of IDH1.  These data demonstrate that a shared feature of all cancer-assocd. IDH mutations is prodn. of the oncometabolite 2HG.  Furthermore, AML patients with IDH mutations display a significantly reduced no. of other well characterized AML-assocd. mutations and/or assocd. chromosomal abnormalities, potentially implicating IDH mutation in a distinct mechanism of AML pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppKH_IGRymibVg90H21EOLACvtfcHk0lgsQyaKW7GIAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWqtLg%253D&md5=cf235f63ccd050ff3b121b94d3c49b9e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.01.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.01.020%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DWise%26aufirst%3DD.%2BR.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DColler%26aufirst%3DH.%2BA.%26aulast%3DCross%26aufirst%3DJ.%2BR.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DHedvat%26aufirst%3DC.%2BV.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DS.%2BM.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DThe%2520common%2520feature%2520of%2520leukemia-associated%2520IDH1%2520and%2520IDH2%2520mutations%2520is%2520a%2520neomorphic%2520enzyme%2520activity%2520converting%2520alpha-ketoglutarate%2520to%25202-hydroxyglutarate%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D225%26epage%3D234%26doi%3D10.1016%2Fj.ccr.2010.01.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span> <span> </span><span class="NLM_article-title">Oncometabolite 2-hydroxyglutarate is a competitive inhibitor of alpha-ketoglutarate-dependent dioxygenases</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.ccr.2010.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21251613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvValsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=17-30&author=W.+Xuauthor=H.+Yangauthor=Y.+Liuauthor=Y.+Yangauthor=P.+Wangauthor=S.+H.+Kimauthor=S.+Itoauthor=C.+Yangauthor=P.+Wangauthor=M.+T.+Xiaoauthor=L.+X.+Liuauthor=W.+Q.+Jiangauthor=J.+Liuauthor=J.+Y.+Zhangauthor=B.+Wangauthor=S.+Fryeauthor=Y.+Zhangauthor=Y.+H.+Xuauthor=Q.+Y.+Leiauthor=K.+L.+Guanauthor=S.+M.+Zhaoauthor=Y.+Xiong&title=Oncometabolite+2-hydroxyglutarate+is+a+competitive+inhibitor+of+alpha-ketoglutarate-dependent+dioxygenases&doi=10.1016%2Fj.ccr.2010.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Oncometabolite 2-Hydroxyglutarate Is a Competitive Inhibitor of α-Ketoglutarate-Dependent Dioxygenases</span></div><div class="casAuthors">Xu, Wei; Yang, Hui; Liu, Ying; Yang, Ying; Wang, Ping; Kim, Se-Hee; Ito, Shinsuke; Yang, Chen; Wang, Pu; Xiao, Meng-Tao; Liu, Li-xia; Jiang, Wen-qing; Liu, Jing; Zhang, Jin-ye; Wang, Bin; Frye, Stephen; Zhang, Yi; Xu, Yan-hui; Lei, Qun-ying; Guan, Kun-Liang; Zhao, Shi-min; Xiong, Yue</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-30</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: IDH1 and IDH2 mutations occur frequently in gliomas and acute myeloid leukemia, leading to simultaneous loss and gain of activities in the prodn. of α-ketoglutarate (α-KG) and 2-hydroxyglutarate (2-HG), resp.  Here we demonstrate that 2-HG is a competitive inhibitor of multiple α-KG-dependent dioxygenases, including histone demethylases and the TET family of 5-methylcytosine (5mC) hydroxylases. 2-HG occupies the same space as α-KG does in the active site of histone demethylases.  Ectopic expression of tumor-derived IDH1 and IDH2 mutants inhibits histone demethylation and 5mC hydroxylation.  In glioma, IDH1 mutations are assocd. with increased histone methylation and decreased 5-hydroxylmethylcytosine (5hmC).  Hence, tumor-derived IDH1 and IDH2 mutations reduce α-KG and accumulate an α-KG antagonist, 2-HG, leading to genome-wide histone and DNA methylation alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpry-WZlrav7Vg90H21EOLACvtfcHk0lg_ADaCfpIU1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvValsQ%253D%253D&md5=27f1e2d50b9dc1dd30bcc576d6318662</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DM.%2BT.%26aulast%3DLiu%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DW.%2BQ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DFrye%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%2BH.%26aulast%3DLei%26aufirst%3DQ.%2BY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26aulast%3DZhao%26aufirst%3DS.%2BM.%26aulast%3DXiong%26aufirst%3DY.%26atitle%3DOncometabolite%25202-hydroxyglutarate%2520is%2520a%2520competitive%2520inhibitor%2520of%2520alpha-ketoglutarate-dependent%2520dioxygenases%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D17%26epage%3D30%26doi%3D10.1016%2Fj.ccr.2010.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillringhaus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagg, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woon, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifton, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klose, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claridge, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span> <span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1038/embor.2011.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fembor.2011.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21460794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-469&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases&doi=10.1038%2Fembor.2011.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span></div><div class="casAuthors">Chowdhury, Rasheduzzaman; Yeoh, Kar-Kheng; Tian, Ya-Min; Hillringhaus, Lars; Bagg, Eleanor A.; Rose, Nathan R.; Leung, Ivanhoe K.-H.; Li, Xuan-S.; Woon, Esther C.-Y.; Yang, Ming; McDonough, Michael A.; King, Oliver N.; Clifton, Ian J.; Klose, Robert J.; Claridge, Timothy D. W.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenases (IDHs) have a gain-of-function effect leading to R(-)-2-hydroxyglutarate (R-2HG) accumulation.  By using biochem., structural and cellular assays, we show that either or both R- and S-2HG inhibit 2-oxoglutarate (2OG)-dependent oxygenases with varying potencies.  Half-maximal inhibitory concn. (IC50) values for the R-form of 2HG varied from approx. 25 μM for the histone N.vepsiln.-lysine demethylase JMJD2A to more than 5 mM for the hypoxia-inducible factor (HIF) prolyl hydroxylase.  The results indicate that candidate oncogenic pathways in IDH-assocd. malignancy should include those that are regulated by other 2OG oxygenases than HIF hydroxylases, in particular those involving the regulation of histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHxuB6sKkTbVg90H21EOLACvtfcHk0lg_ADaCfpIU1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D&md5=1b862b1337346af6b796093cea34056f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.43%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fembor.2011.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramkissoon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhaak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">Transformation by the (R)-enantiomer of 2-hydroxyglutarate linked to EGLN activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1038/nature10898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature10898" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=484-488&author=P.+Koivunenauthor=S.+Leeauthor=C.+G.+Duncanauthor=G.+Lopezauthor=G.+Luauthor=S.+Ramkissoonauthor=J.+A.+Losmanauthor=P.+Joensuuauthor=U.+Bergmannauthor=S.+Grossauthor=J.+Travinsauthor=S.+Weissauthor=R.+Looperauthor=K.+L.+Ligonauthor=R.+G.+Verhaakauthor=H.+Yanauthor=W.+G.+Kaelin&title=Transformation+by+the+%28R%29-enantiomer+of+2-hydroxyglutarate+linked+to+EGLN+activation&doi=10.1038%2Fnature10898"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature10898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10898%26sid%3Dliteratum%253Aachs%26aulast%3DKoivunen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DDuncan%26aufirst%3DC.%2BG.%26aulast%3DLopez%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DRamkissoon%26aufirst%3DS.%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DJoensuu%26aufirst%3DP.%26aulast%3DBergmann%26aufirst%3DU.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DLooper%26aufirst%3DR.%26aulast%3DLigon%26aufirst%3DK.%2BL.%26aulast%3DVerhaak%26aufirst%3DR.%2BG.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DTransformation%2520by%2520the%2520%2528R%2529-enantiomer%2520of%25202-hydroxyglutarate%2520linked%2520to%2520EGLN%2520activation%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D484%26epage%3D488%26doi%3D10.1038%2Fnature10898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">(R)-2-hydroxyglutarate is sufficient to promote leukemogenesis and its effects are reversible</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">1621</span>– <span class="NLM_lpage">1625</span>, <span class="refDoi"> DOI: 10.1126/science.1231677</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1126%2Fscience.1231677" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23393090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1621-1625&author=J.+A.+Losmanauthor=R.+E.+Looperauthor=P.+Koivunenauthor=S.+Leeauthor=R.+K.+Schneiderauthor=C.+McMahonauthor=G.+S.+Cowleyauthor=D.+E.+Rootauthor=B.+L.+Ebertauthor=W.+G.+Kaelin&title=%28R%29-2-hydroxyglutarate+is+sufficient+to+promote+leukemogenesis+and+its+effects+are+reversible&doi=10.1126%2Fscience.1231677"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-2-Hydroxyglutarate is sufficient to promote leukemogenesis and its effects are reversible</span></div><div class="casAuthors">Losman, Julie-Aurore; Looper, Ryan E.; Koivunen, Peppi; Lee, Sungwoo; Schneider, Rebekka K.; McMahon, Christine; Cowley, Glenn S.; Root, David E.; Ebert, Benjamin L.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6127</span>),
    <span class="NLM_cas:pages">1621-1625</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mutations in IDH1 and IDH2, the genes coding for isocitrate dehydrogenases 1 and 2, are common in several human cancers, including leukemias, and result in overprodn. of the (R)-enantiomer of 2-hydroxyglutarate [(R)-2HG].  Elucidation of the role of IDH mutations and (R)-2HG in leukemogenesis has been hampered by a lack of appropriate cell-based models.  Here, we show that a canonical IDH1 mutant, IDH1 R132H, promotes cytokine independence and blocks differentiation in hematopoietic cells.  These effects can be recapitulated by (R)-2HG, but not (S)-2HG, despite the fact that (S)-2HG more potently inhibits enzymes, such as the 5'-methylcytosine hydroxylase TET2, that have previously been linked to the pathogenesis of IDH mutant tumors.  We provide evidence that this paradox relates to the ability of (S)-2HG, but not (R)-2HG, to inhibit the EglN prolyl hydroxylases.  Addnl., we show that transformation by (R)-2HG is reversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTDGHR9SE0P7Vg90H21EOLACvtfcHk0liiuowJyRifiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOhsLc%253D&md5=1bb23bea1550fd47e479f6d3833741ee</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1126%2Fscience.1231677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1231677%26sid%3Dliteratum%253Aachs%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DLooper%26aufirst%3DR.%2BE.%26aulast%3DKoivunen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DR.%2BK.%26aulast%3DMcMahon%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3D%2528R%2529-2-hydroxyglutarate%2520is%2520sufficient%2520to%2520promote%2520leukemogenesis%2520and%2520its%2520effects%2520are%2520reversible%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D1621%26epage%3D1625%26doi%3D10.1126%2Fscience.1231677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachtergaele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibbe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferchen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plas, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">R-2HG exhibits anti-tumor activity by targeting FTO/m(6)A/MYC/CEBPA signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2017.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=29249359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=90-105&author=R.+Suauthor=L.+Dongauthor=C.+Liauthor=S.+Nachtergaeleauthor=M.+Wunderlichauthor=Y.+Qingauthor=X.+Dengauthor=Y.+Wangauthor=X.+Wengauthor=C.+Huauthor=M.+Yuauthor=J.+Skibbeauthor=Q.+Daiauthor=D.+Zouauthor=T.+Wuauthor=K.+Yuauthor=H.+Wengauthor=H.+Huangauthor=K.+Ferchenauthor=X.+Qinauthor=B.+Zhangauthor=J.+Qiauthor=A.+T.+Sasakiauthor=D.+R.+Plasauthor=J.+E.+Bradnerauthor=M.+Weiauthor=G.+Marcucciauthor=X.+Jiangauthor=J.+C.+Mulloyauthor=J.+Jinauthor=C.+Heauthor=J.+Chen&title=R-2HG+exhibits+anti-tumor+activity+by+targeting+FTO%2Fm%286%29A%2FMYC%2FCEBPA+signaling&doi=10.1016%2Fj.cell.2017.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">R-2HG Exhibits Anti-tumor Activity by Targeting FTO/m6A/MYC/CEBPA Signaling</span></div><div class="casAuthors">Su, Rui; Dong, Lei; Li, Chenying; Nachtergaele, Sigrid; Wunderlich, Mark; Qing, Ying; Deng, Xiaolan; Wang, Yungui; Weng, Xiaocheng; Hu, Chao; Yu, Mengxia; Skibbe, Jennifer; Dai, Qing; Zou, Dongling; Wu, Tong; Yu, Kangkang; Weng, Hengyou; Huang, Huilin; Ferchen, Kyle; Qin, Xi; Zhang, Bin; Qi, Jun; Sasaki, Atsuo T.; Plas, David R.; Bradner, James E.; Wei, Minjie; Marcucci, Guido; Jiang, Xi; Mulloy, James C.; Jin, Jie; He, Chuan; Chen, Jianjun</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">90-105.e23</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">R-2-hydroxyglutarate (R-2HG), produced at high levels by mutant isocitrate dehydrogenase 1/2 (IDH1/2) enzymes, was reported as an oncometabolite.  We show here that R-2HG also exerts a broad anti-leukemic activity in vitro and in vivo by inhibiting leukemia cell proliferation/viability and by promoting cell-cycle arrest and apoptosis.  Mechanistically, R-2HG inhibits fat mass and obesity-assocd. protein (FTO) activity, thereby increasing global N6-methyladenosine (m6A) RNA modification in R-2HG-sensitive leukemia cells, which in turn decreases the stability of MYC/CEBPA transcripts, leading to the suppression of relevant pathways.  Ectopically expressed mutant IDH1 and S-2HG recapitulate the effects of R-2HG.  High levels of FTO sensitize leukemic cells to R-2HG, whereas hyperactivation of MYC signaling confers resistance that can be reversed by the inhibition of MYC signaling.  R-2HG also displays anti-tumor activity in glioma.  Collectively, while R-2HG accumulated in IDH1/2 mutant cancers contributes to cancer initiation, our work demonstrates anti-tumor effects of 2HG in inhibiting proliferation/survival of FTO-high cancer cells via targeting FTO/m6A/MYC/CEBPA signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMervd4vttCLVg90H21EOLACvtfcHk0liiuowJyRifiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7rI&md5=ffb994f5c202903fd5714c042adeb869</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DR.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DNachtergaele%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DQing%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DSkibbe%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DFerchen%26aufirst%3DK.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DPlas%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DR-2HG%2520exhibits%2520anti-tumor%2520activity%2520by%2520targeting%2520FTO%252Fm%25286%2529A%252FMYC%252FCEBPA%2520signaling%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D90%26epage%3D105%26doi%3D10.1016%2Fj.cell.2017.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and α-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm α-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was ∼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a ∼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0ljkaOcgDkApzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausinger, R. P.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanisms of Fe(II)- and 2-oxoglutarate-dependent oxygenases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">20702</span>– <span class="NLM_lpage">20711</span>, <span class="refDoi"> DOI: 10.1074/jbc.R115.648691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.R115.648691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26152721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20702-20711&author=S.+Martinezauthor=R.+P.+Hausinger&title=Catalytic+mechanisms+of+Fe%28II%29-+and+2-oxoglutarate-dependent+oxygenases&doi=10.1074%2Fjbc.R115.648691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Mechanisms of Fe(II)- and 2-Oxoglutarate-dependent Oxygenases</span></div><div class="casAuthors">Martinez, Salette; Hausinger, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">20702-20711</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Mononuclear non-heme Fe(II)- and 2-oxoglutarate (2OG)-dependent oxygenases comprise a large family of enzymes that utilize an Fe(IV)-oxo intermediate to initiate diverse oxidative transformations with important biol. roles.  Here, four of the major types of Fe(II)/2OG-dependent reactions are detailed: hydroxylation, halogenation, ring formation, and desatn.  In addn., an atypical epimerization reaction is described.  Studies identifying several key intermediates in catalysis are concisely summarized, and the proposed mechanisms are explained.  In addn., a variety of other transformations catalyzed by selected family members are briefly described to further highlight the chem. versatility of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-tW8bZZFBybVg90H21EOLACvtfcHk0ljkaOcgDkApzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrt77E&md5=a31238fa27dcb8d88adb19292ad44a4d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R115.648691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R115.648691%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DS.%26aulast%3DHausinger%26aufirst%3DR.%2BP.%26atitle%3DCatalytic%2520mechanisms%2520of%2520Fe%2528II%2529-%2520and%25202-oxoglutarate-dependent%2520oxygenases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20702%26epage%3D20711%26doi%3D10.1074%2Fjbc.R115.648691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudlebusch, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">E1</span>– <span class="NLM_lpage">2</span>, <span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3D2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2974</span>– <span class="NLM_lpage">2981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.bmcl.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28512031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2974-2981&author=J.+Liangauthor=S.+Labadieauthor=B.+Zhangauthor=D.+F.+Ortwineauthor=S.+Patelauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=T.+Mauerauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=L.+Anauthor=Y.+Ranauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=From+a+novel+HTS+hit+to+potent%2C+selective%2C+and+orally+bioavailable+KDM5+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span></div><div class="casAuthors">Liang, Jun; Labadie, Sharada; Zhang, Birong; Ortwine, Daniel F.; Patel, Snahel; Vinogradova, Maia; Kiefer, James R.; Mauer, Till; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; An, Le; Ran, Yingqing; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2974-2981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screening (HTS) of the Genentech/Roche library identified a novel, uncharged scaffold as a KDM5A inhibitor.  Lacking insight into the binding mode, initial attempts to improve inhibitor potency failed to improve potency, and synthesis of analogs was further hampered by the presence of a C-C bond between the pyrrolidine and pyridine.  Replacing this with a C-N bond significantly simplified synthesis, yielding pyrazole analog, [3-(4-bromo-1H-pyrazol-1-yl)-1-pyrrolidinyl][5-(1-methylethyl)-1H-pyrazol-3-yl]methanone (35), of which the authors obtained a co-crystal structure with KDM5A.  Using structure-based design approach, the authors identified, N-[(3R)-1-[[5-(1-methylethyl)-1H-pyrazol-3-yl]carbonyl]-3-pyrrolidinyl]cyclopropanecarboxamide (50), with improved biochem., cell potency and reduced MW and lower lipophilicity (Log D) compared with the original hit.  Furthermore, 50 showed lower clearance than [5-(1-methylethyl)-1H-pyrazol-3-yl][(3S)-3-[6-methyl-4-(1-methyl-1H-pyrazol-4-yl)-2-pyridinyl]-1-pyrrolidinyl]methanone (9) in mice.  In combination with its remarkably low plasma protein binding (PPB) in mice (40%), oral dosing of 50 at 5 mg/kg resulted in unbound Cmax ∼2-fold of its cell potency (PC9 H3K4Me3 0.96 μM), meeting the authors' criteria for an in vivo tool compd. from a new scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpksjXQBrVYsrVg90H21EOLACvtfcHk0ljkaOcgDkApzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D&md5=04bee4e29775202fb1882adeccb8e722</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMauer%26aufirst%3DT.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DFrom%2520a%2520novel%2520HTS%2520hit%2520to%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2974%26epage%3D2981%26doi%3D10.1016%2Fj.bmcl.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span>; <span class="NLM_string-name">Gale, M.</span>; <span class="NLM_string-name">Yan, Q.</span>; <span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">The molecular basis of histone demethylation</span>. In  <i>DNA and Histone Methylation As Cancer Targets</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span>, <span class="NLM_string-name">Tsukada, Y.-i.</span></span>, Eds.; Cancer Drug Discovery and Development; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">219</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-3-319-59786-7_7</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2F978-3-319-59786-7_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=151-219&author=J.+R.+Horton&author=M.+Gale&author=Q.+Yan&author=X.+Chengauthor=A.+Kaneda&author=Y.-i.+Tsukada&title=DNA+and+Histone+Methylation+As+Cancer+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520molecular%2520basis%2520of%2520histone%2520demethylation%26btitle%3DDNA%2520and%2520Histone%2520Methylation%2520As%2520Cancer%2520Targets%26aulast%3DKaneda%26aufirst%3DA.%26pub%3DSpringer%26date%3D2017%26spage%3D151%26epage%3D219%26doi%3D10.1007%2F978-3-319-59786-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 1. 3-amino-4-pyridine carboxylate derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1357-1369&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=H.+Dialloauthor=C.+Douaultauthor=G.+Drewesauthor=R.+Eagleauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=J.+Liddleauthor=J.+Mosleyauthor=M.+Muelbaierauthor=R.+Randleauthor=I.+Riojaauthor=A.+Ruegerauthor=G.+A.+Sealauthor=R.+J.+Sheppardauthor=O.+Singhauthor=J.+Taylorauthor=P.+Thomasauthor=D.+Thomsonauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+1.+3-amino-4-pyridine+carboxylate+derivatives&doi=10.1021%2Facs.jmedchem.5b01537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01537%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRandle%26aufirst%3DR.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThomson%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25201.%25203-amino-4-pyridine%2520carboxylate%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1357%26epage%3D1369%26doi%3D10.1021%2Facs.jmedchem.5b01537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Multiparametric scaling of diffraction intensities</span>. <i>Acta Crystallogr., Sect. A: Found. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1107/S0108767303005488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0108767303005488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=12714773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFChtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=228-234&author=Z.+Otwinowskiauthor=D.+Borekauthor=W.+Majewskiauthor=W.+Minor&title=Multiparametric+scaling+of+diffraction+intensities&doi=10.1107%2FS0108767303005488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Multiparametric scaling of diffraction intensities</span></div><div class="casAuthors">Otwinowski, Zbyszek; Borek, Dominika; Majewski, Wladyslaw; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">A59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">228-234</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">A novel and general approach to scaling diffraction intensities is presented.  The method minimizes the disagreement among multiple measurements of symmetry-related reflections using a stable refinement procedure.  The scale factors are described by a flexible exponential function that allows different scaling corrections to be chosen and combined according to the needs of the expt.  The scaling model presented here includes: scale and temp. factor per batch of data; temp. factor as a continuous function of the radiation dose; absorption in the crystal; uneven exposure within a single diffraction image; and corrections for phenomena that depend on the diffraction peak position on the detector.  This scaling model can be extended to include addnl. corrections for various instrumental and data-collection problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobnTSFjuXFu7Vg90H21EOLACvtfcHk0ljxyKf6bh5A4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFChtbg%253D&md5=81ef008e4c8fa87b2863552558ed18ad</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0108767303005488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767303005488%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DMajewski%26aufirst%3DW.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DMultiparametric%2520scaling%2520of%2520diffraction%2520intensities%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D228%26epage%3D234%26doi%3D10.1107%2FS0108767303005488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljxyKf6bh5A4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljxyKf6bh5A4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0ljxyKf6bh5A4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhJRUldytkL_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tackett, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule inhibitors of Jumonji AT-rich interactive domain 1B (JARID1B) histone demethylase by a sensitive high throughput screen</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">9408</span>– <span class="NLM_lpage">9417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.419861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.M112.419861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23408432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=9408-9417&author=J.+Sayeghauthor=J.+Caoauthor=M.+R.+Zouauthor=A.+Moralesauthor=L.+P.+Blairauthor=M.+Norciaauthor=D.+Hoyerauthor=A.+J.+Tackettauthor=J.+S.+Merkelauthor=Q.+Yan&title=Identification+of+small+molecule+inhibitors+of+Jumonji+AT-rich+interactive+domain+1B+%28JARID1B%29+histone+demethylase+by+a+sensitive+high+throughput+screen&doi=10.1074%2Fjbc.M112.419861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Small Molecule Inhibitors of Jumonji AT-rich Interactive Domain 1B (JARID1B) Histone Demethylase by a Sensitive High Throughput Screen</span></div><div class="casAuthors">Sayegh, Joyce; Cao, Jian; Zou, Mike Ran; Morales, Alfonso; Blair, Lauren P.; Norcia, Michael; Hoyer, Denton; Tackett, Alan J.; Merkel, Jane S.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9408-9417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">JARID1B (also known as KDM5B or PLU1) is a member of the JARID1 family of histone lysine demethylases responsible for the demethylation of trimethylated lysine 27 in histone H3 (H3K4me3), a mark for actively transcribed genes.  JARID1B is overexpressed in several cancers, including breast cancer, prostate cancer, and lung cancer.  In addn., JARID1B is required for mammary tumor formation in syngeneic or xenograft mouse models.  JARID1B-expressing melanoma cells are assocd. with increased self-renewal character.  Therefore, JARID1B represents an attractive target for cancer therapy.  Here we characterized JARID1B using a homogeneous luminescence-based demethylase assay.  We then conducted a high throughput screen of over 15,000 small mols. to identify inhibitors of JARID1B.  From this screen, we identified several known JmjC histone demethylase inhibitors, including 2,4-pyridinedicarboxylic acid and catechols.  More importantly, we identified several novel inhibitors, including 2-4(4-methylphenyl)-1,2-benzisothiazol-3(2H)-one (PBIT), which inhibits JARID1B with an IC50 of about 3 μmin vitro.  Consistent with this, PBIT treatment inhibited removal of H3K4me3 by JARID1B in cells.  Furthermore, this compd. inhibited proliferation of cells expressing higher levels of JARID1B.  These results suggest that this novel small mol. inhibitor is a lead compd. that can be further optimized for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiVU6VegMXeLVg90H21EOLACvtfcHk0lhJRUldytkL_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvF2qsLw%253D&md5=f7d20eaea77373475d630456828defc1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.419861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.419861%26sid%3Dliteratum%253Aachs%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DMorales%26aufirst%3DA.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DNorcia%26aufirst%3DM.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DMerkel%26aufirst%3DJ.%2BS.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520inhibitors%2520of%2520Jumonji%2520AT-rich%2520interactive%2520domain%25201B%2520%2528JARID1B%2529%2520histone%2520demethylase%2520by%2520a%2520sensitive%2520high%2520throughput%2520screen%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D9408%26epage%3D9417%26doi%3D10.1074%2Fjbc.M112.419861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">John R. Horton, Clayton B. Woodcock, Qin Chen, Xu Liu, Xing Zhang, John Shanks, Ganesha Rai, Bryan T. Mott, Daniel J. Jansen, Stephen C. Kales, Mark J. Henderson, Matthew Cyr, Katherine Pohida, Xin Hu, Pranav Shah, Xin Xu, Ajit Jadhav, David J. Maloney, Matthew D. Hall, Anton Simeonov, Haian Fu, Paula M. Vertino, <span class="NLM_string-name hlFld-ContribAuthor">Xiaodong Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (23)
                                     , 10588-10601. <a href="https://doi.org/10.1021/acs.jmedchem.8b01219" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01219</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01219%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BEngineering%252Bof%252BIrreversible%252BInhibitors%252Bagainst%252BHistone%252BLysine%252BDemethylase%252BKDM5A%26aulast%3DHorton%26aufirst%3DJohn%2BR.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D02082018%26date%3D15112018%26date%3D03112018%26volume%3D61%26issue%3D23%26spage%3D10588%26epage%3D10601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuka  Miyake</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Atsushi  Hatanaka</span>, <span class="hlFld-ContribAuthor ">Yoshinori  Suzuma</span>, <span class="hlFld-ContribAuthor ">Miki  Suzuki</span>, <span class="hlFld-ContribAuthor ">Hidehiko  Kodama</span>, <span class="hlFld-ContribAuthor ">Yoshinobu  Arai</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel lysine demethylase 5-selective inhibitors by inhibitor-based fragment merging strategy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (6)
                                     , 1119-1129. <a href="https://doi.org/10.1016/j.bmc.2019.02.006" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252Blysine%252Bdemethylase%252B5-selective%252Binhibitors%252Bby%252Binhibitor-based%252Bfragment%252Bmerging%252Bstrategy%26aulast%3DMiyake%26aufirst%3DYuka%26date%3D2019%26volume%3D27%26issue%3D6%26spage%3D1119%26epage%3D1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miranda  Wright</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of JmjC-Containing Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-253. <a href="https://doi.org/10.1007/7355_2019_75" title="DOI URL">https://doi.org/10.1007/7355_2019_75</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_75%26sid%3Dliteratum%253Aachs%26atitle%3DInhibitors%252Bof%252BJmjC-Containing%252BHistone%252BDemethylases%26aulast%3DWright%26aufirst%3DMiranda%26date%3D2019%26date%3D2020%26spage%3D221%26epage%3D253%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizhen  Wu</span>, <span class="hlFld-ContribAuthor ">Jian  Cao</span>, <span class="hlFld-ContribAuthor ">Wesley L.  Cai</span>, <span class="hlFld-ContribAuthor ">Sabine M.  Lang</span>, <span class="hlFld-ContribAuthor ">John R.  Horton</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Jansen</span>, <span class="hlFld-ContribAuthor ">Zongzhi Z.  Liu</span>, <span class="hlFld-ContribAuthor ">Jocelyn F.  Chen</span>, <span class="hlFld-ContribAuthor ">Meiling  Zhang</span>, <span class="hlFld-ContribAuthor ">Bryan T.  Mott</span>, <span class="hlFld-ContribAuthor ">Katherine  Pohida</span>, <span class="hlFld-ContribAuthor ">Ganesha  Rai</span>, <span class="hlFld-ContribAuthor ">Stephen C.  Kales</span>, <span class="hlFld-ContribAuthor ">Mark J.  Henderson</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">Shu  Zhu</span>, <span class="hlFld-ContribAuthor ">Akiko  Iwasaki</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Hall</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Cheng</span>, <span class="hlFld-ContribAuthor ">Gerald S.  Shadel</span>, <span class="hlFld-ContribAuthor ">Qin  Yan</span>, . </span><span class="cited-content_cbyCitation_article-title">KDM5 histone demethylases repress immune response via suppression of STING. </span><span class="cited-content_cbyCitation_journal-name">PLOS Biology</span><span> <strong>2018,</strong> <em>16 </em>
                                    (8)
                                     , e2006134. <a href="https://doi.org/10.1371/journal.pbio.2006134" title="DOI URL">https://doi.org/10.1371/journal.pbio.2006134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pbio.2006134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pbio.2006134%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Biology%26atitle%3DKDM5%252Bhistone%252Bdemethylases%252Brepress%252Bimmune%252Bresponse%252Bvia%252Bsuppression%252Bof%252BSTING%26aulast%3DWu%26aufirst%3DLizhen%26date%3D2018%26date%3D2018%26volume%3D16%26issue%3D8%26spage%3De2006134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding of KDM5A by compound N9 and its derivatives. (A) Chemical structures of <b>N9</b> related compounds containing a constant pyridine-7-carboxylic acid moiety and 2-chlorophenyl moiety connected to a variable hydroxymethyl extension. (B) Chemical structures of (<i>R</i>)- and (<i>S</i>)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylic acid (compounds <b>N51</b> and <b>N52</b>). (C) Summary of isothermal titration calorimetry measurement of dissociation constants (<i>K</i><sub>D</sub>) of various compounds to KDM5A(1–588)ΔAP (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S2</a> for original binding data). (D) Summary of the IC<sub>50</sub> values of inhibition of demethylation for KDM5A (residues 1–1090) by AlphaLISA (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S3</a> for original inhibition data). (E) Summary of inhibitor-bound X-ray structures of KDM5A(1–588)ΔAP (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Table S1</a> for detailed statistics of X-ray diffraction and refinement).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of ternary complex of <b>N46</b>-KDM5A-Mn(II) at 1.22 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>). (A) Six metal ligands involving three protein residues, two water molecules, and the ring nitrogen atom of pyridine. The omit electron densities, contoured at 10σ and 5σ above the mean, are shown for Mn(II) (magenta mesh) and compound <b>N46</b> (gray mesh), respectively. (B) The carboxylate group of pyridine-7-carboxylic acid moiety forms ionic and hydrogen bonding interactions with Lys501 and Tyr409. (C) Extensive van der Waals contacts formed between pyridine ring and 2-chlorophenyl ring of <b>N46</b> and aromatic residues of KDM5A in the active site. (D) The chlorine atom of 2-chlorophenyl ring binds in a hydrophobic pocket formed by Ala411, Tyr409, and Tyr472.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural snapshots of <b>N9</b> related compounds bound into KDM5A active site. (A–I) The pyridine-7-carboxylic acid moiety involves in conserved interactions with Lys501, Asn575, and Tyr409 (top left corner of each panel). The omit electron densities, contoured at 5σ above the mean, are shown for each inhibitor (gray mesh). (A) In compound <b>N9</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU">6BGU</a>), the relative positions of 2-chlorophenyl moiety and the propoxymethyl group are switched in comparison with other compounds containing larger chemical part attached to the end of hydroxymethyl extension. (B) In the racemic mixture of compound <b>N40</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>), two possible conformations of 2-chlorophenyl moiety can be modeled into the electron density. (C, D) In the purified enantiomers (<i>R</i>)-<b>N51</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>) and (<i>S</i>)-<b>N52</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), the chlorine atom of the chlorophenyl ring points to opposite directions. (E–I) For the other five racemic mixtures of compounds, <b>N41</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW">6BGW</a>), <b>N42</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX">6BGX</a>), <b>N46</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>), <b>N47</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ">6BGZ</a>), and <b>N48</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5">6BH5</a>), the (<i>R</i>)-<b>N52</b>-like conformation dominates. (J) Superimposition of compounds <b>N9</b> and <b>N46</b>. (K) Superimposition of compounds <b>N41</b> and <b>N42</b>. Note the inhibitor-specific interaction with Trp470 (compound <b>N41</b>) and Asp486 (compound <b>N42</b>). (L) Superimposition of compounds <b>N46</b> and <b>N47</b>. Note the conformational change of Asp412.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chiral pairs of (<i>R</i>)-<b>N51</b> and (<i>S</i>)-<b>N52</b> bind in the KDM5A active site. (A) Superimposition of racemic mixture of compound <b>N40</b> (magenta; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>) and the purified enantiomers (<i>R</i>)-<b>N51</b> (cyan; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>) and (<i>S</i>)-<b>N52</b> (gold; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>). Note the pyridine-7-carboxylate moiety is superimposable. (B, C) In (<i>R</i>)-<b>N51</b>, Y472 forms two hydrogen bonds with the compound. In (<i>S</i>)-<b>N52</b>, Asp412 forms an ionic interaction with the compound. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron densities, contoured at 1.2σ, are shown for the inhibitor and immediate neighboring residues. (D) Superimposition of (<i>R</i>)-<b>N51</b> and (<i>S</i>)-<b>N52</b>. Note the conformational change of Asp412.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b> bind highly similar in the KDM5A active site. (A) Chemical structures of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b>. (B) Inhibition of demethylation of KDM5A(1–739)ΔAP by (<i>R</i>)-<b>N54</b> via FDH-coupled assay at three concentrations of αKG (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5A</a> for inhibition data by (<i>S</i>)-<b>N55</b>). (C) Structure of <b>N54</b>-KDM5A-Mn(II) showing the binding of <b>N54</b> compound in the active site involving two metal ligands (upper right corner), a direct hydrogen bond with Asn575 (left), and aromatic interactions via Phe480 (top), Tyr472 (bottom), and Trp503 (in the background away from the viewer). See <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5B</a> for structure of (<i>S</i>)-<b>N55</b> bound in the active site of KDM5A. (D) The terminal cyclopropyl group forms a stacking interaction with Tyr409 (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5C</a> for corresponding interaction in (<i>S</i>)-<b>N55</b>). (E) The terminal isopropyl methyl groups form van der Waals interactions with Trp470, Asn585, and Val584 (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf" class="ext-link">Figure S5C</a> for corresponding interaction in (<i>S</i>)-<b>N55</b>). (F) Superimposition of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b>. (G, H) Superimposition of compounds <b>N52</b> (dark magenta) and <b>N54</b> (yellow) illustrates conformational changes of Tyr409 (G) and Trp780 (H). (I) An ordered glycerol molecule was observed near the inhibitor, forming direct H-bond interaction with the inhibitor. (J) A surface representation of KDM5A active site showing the binding of <b>N54</b> compound together with a glycerol molecule. (K) A surface representation of KDM5A showing the binding of compound <b>N52</b> occupying a much larger space in the active site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (<i>R</i>)-Enantiomer <b>N54</b> is more active than (<i>S</i>)-enantiomer <b>N55</b> in cellular assays. (A, B) Western blot analyses of (A) SKBR3 and BT474 cells, (B) MCF7 and MDA-MB231 cells treated with the indicated concentration of <b>N54</b>, <b>N55</b>, <b>KDM5-C70</b>, or DMSO control for 3 days. Representative data from biological duplicate experiments performed in duplicate are shown. (C, D) Colony-formation assays of MCF-7 cells treated with DMSO, <b>KDM5-C70</b> (1 μM), and <b>N54</b> and <b>N55</b> (1 or 10 μM). Shown are (C) representative images and (D) quantification of biological triplicate experiments. “Intensity %” indicates the measured intensity of cell confluency using ImageJ. Error bars denote SEM: (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Proliferation of (<i>R</i>)-<b>N54</b> and (<i>S</i>)-<b>N55</b> treated breast cancer lines. <b>N54</b> and <b>N55</b> inhibit proliferation of (A) BT474 but not (B) MCF7, (C) SKBR3, or (D) MDA- MB231. Growth curves represent percent confluence (mean ± SD, <i>n</i> = 3). (E) <b>N54</b> is a more potent inhibitor of BT474 proliferation than <b>N55</b>. AUC for growth (% confluence over 288 h) is plotted versus inhibitor concentration. (F) Micrographs at 240 h post-treatment, with the confluence mask shown in yellow. Scale bar is 300 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/medium/jm-2018-002612_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Inhibitor specific Tyr409 conformations. (A–C) Tyr409 adopts a nearly identical conformation via the H-bond interaction with the carboxylate moiety of (<i>S</i>)-<b>N52</b> (panel A; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), <b>KDM5-C49</b> (panel B; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ISL">5ISL</a>), and αKG (panel C; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVB">5IVB</a>). (D–F) Tyr409 adopts different conformations in the complex with (<i>R</i>)-<b>N54</b> (panel D; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2">6BH2</a>), <b>KDOAM-25</b> (panel E; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF">5IWF</a>), and <b>CPI-455</b> (panel F; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH">5CEH</a>). (G) Superimposition of <b>KDM5-C49</b> and <b>KDOAM-25</b> bound KDM5A linked JmjN-JmjC domain structures. (H) Superimposition of (<i>R</i>)-<b>N54</b>, <b>KDM5-C49</b>, and <b>KDOAM-25</b> bound KDM5A linked JmjN-JmjC domain structures, illustrating conformational changes of Tyr409 upon inhibitor binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-7/acs.jmedchem.8b00261/20180406/images/large/jm-2018-002612_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00261&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74344" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74344" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 60 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, S. M.</span>, <span class="NLM_string-name">Li, E.</span></span>, Eds.  <i>Epigenetics and Disease: Pharmaceutical Opportunities</i>; Progress in Drug Research, Vol. 67; <span class="NLM_publisher-name">Springer Basel AG</span>, <span class="NLM_year">2011</span>; DOI: <span class="refDoi"> DOI: 10.1007/978-3-7643-8989-5</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2F978-3-7643-8989-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=S.+M.+Gasser&author=E.+Li&title=Epigenetics+and+Disease%3A+Pharmaceutical+Opportunities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F978-3-7643-8989-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-7643-8989-5%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DS.%2BM.%26btitle%3DEpigenetics%2520and%2520Disease%253A%2520Pharmaceutical%2520Opportunities%26pub%3DSpringer%2520Basel%2520AG%26date%3D2011%26doi%3D10.1007%2F978-3-7643-8989-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Regulation of chromatin by histone modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1038/cr.2011.22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fcr.2011.22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21321607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=381-395&author=A.+J.+Bannisterauthor=T.+Kouzarides&title=Regulation+of+chromatin+by+histone+modifications&doi=10.1038%2Fcr.2011.22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of chromatin by histone modifications</span></div><div class="casAuthors">Bannister, Andrew J.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-395</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Chromatin is not an inert structure, but rather an instructive DNA scaffold that can respond to external cues to regulate the many uses of DNA.  A principal component of chromatin that plays a key role in this regulation is the modification of histones.  There is an ever-growing list of these modifications and the complexity of their action is only just beginning to be understood.  However, it is clear that histone modifications play fundamental roles in most biol. processes that are involved in the manipulation and expression of DNA.  Here, we describe the known histone modifications, define where they are found genomically and discuss some of their functional consequences, concg. mostly on transcription where the majority of characterization has taken place.  Cell Research (2011) 21:381-395. doi:10.1038/cr.2011.22; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ObWaVBHPorVg90H21EOLACvtfcHk0lgpMz_QEa_gXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurc%253D&md5=bbdf55be25985810b90ba2efe6399f1b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.22%26sid%3Dliteratum%253Aachs%26aulast%3DBannister%26aufirst%3DA.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DRegulation%2520of%2520chromatin%2520by%2520histone%2520modifications%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D381%26epage%3D395%26doi%3D10.1038%2Fcr.2011.22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herz, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garruss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shilatifard, A.</span></span> <span> </span><span class="NLM_article-title">SET for life: biochemical activities and biological functions of SET domain-containing proteins</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2013.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.tibs.2013.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=24148750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs12ltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=621-639&author=H.+M.+Herzauthor=A.+Garrussauthor=A.+Shilatifard&title=SET+for+life%3A+biochemical+activities+and+biological+functions+of+SET+domain-containing+proteins&doi=10.1016%2Fj.tibs.2013.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SET for life: biochemical activities and biological functions of SET domain-containing proteins</span></div><div class="casAuthors">Herz, Hans-Martin; Garruss, Alexander; Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">621-639</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  SET domain-contg. proteins belong to a group of enzymes named after a common domain that utilizes the cofactor S-adenosyl-L-methionine (SAM) to achieve methylation of its substrates.  Many SET domain-contg. proteins were shown to display catalytic activity towards particular lysine residues on histones,but emerging evidence also indicates that various nonhistone proteins are specifically targeted by this clade of enzymes.  Here, the authors summarize the most recent findings on the biol. functions of the major families of SET domain-contg. proteins catalyzing the methylation of histones 3 on lysines 4, 9, 27, and 36 (H3K4, H3K9,H3K27, and H3K36) and histone 4 on lysine 20 (H4K20), as well as enzymes that were reported to modify nonhistone substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVOb-qpf8h3LVg90H21EOLACvtfcHk0lgpMz_QEa_gXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs12ltrjK&md5=1214f35b69249e0d0dcd2cdfe992e9bc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DHerz%26aufirst%3DH.%2BM.%26aulast%3DGarruss%26aufirst%3DA.%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DSET%2520for%2520life%253A%2520biochemical%2520activities%2520and%2520biological%2520functions%2520of%2520SET%2520domain-containing%2520proteins%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D621%26epage%3D639%26doi%3D10.1016%2Fj.tibs.2013.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paik, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Historical review: the field of protein methylation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2007.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.tibs.2007.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=17291768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisleisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2007&pages=146-152&author=W.+K.+Paikauthor=D.+C.+Paikauthor=S.+Kim&title=Historical+review%3A+the+field+of+protein+methylation&doi=10.1016%2Fj.tibs.2007.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Historical review: the field of protein methylation</span></div><div class="casAuthors">Paik, Woon Ki; Paik, David C.; Kim, Sangduk</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-152</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although the field of protein methylation enjoys widespread interest in the scientific literature of today, this is a recent phenomenon.  Papers on 'protein methylation' were first published in the 1960s.  By the early 1980s, it was known that lysine, arginine, histidine and dicarboxylic amino acids were post-translationally methylated by highly specific methyltransferases.  However, despite these early advances, the biol. importance of these reactions remained largely unproven.  With the introduction of modern mol. biol. techniques in the mid-1990s, an enormous surge of interest in protein methylation occurred.  It is now clear that protein methylation carries many important biol. functions, including gene regulation and signal transduction.  Thus, the story of protein-methylation research is a testament to both modern mol. biol. and the importance of continuing to pursue lines of research in which the precise biol. function might not be currently known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr258CjMUdF2bVg90H21EOLACvtfcHk0li_v9iM4OgX3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXisleisb4%253D&md5=e32da6065de5f6fe7d92fe6f0ffc3d97</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2007.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2007.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DPaik%26aufirst%3DW.%2BK.%26aulast%3DPaik%26aufirst%3DD.%2BC.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DHistorical%2520review%253A%2520the%2520field%2520of%2520protein%2520methylation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2007%26volume%3D32%26spage%3D146%26epage%3D152%26doi%3D10.1016%2Fj.tibs.2007.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0li_v9iM4OgX3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchers, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation by a family of JmjC domain-containing proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>439</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">816</span>, <span class="refDoi"> DOI: 10.1038/nature04433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature04433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=16362057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=811-816&author=Y.+Tsukadaauthor=J.+Fangauthor=H.+Erdjument-Bromageauthor=M.+E.+Warrenauthor=C.+H.+Borchersauthor=P.+Tempstauthor=Y.+Zhang&title=Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins&doi=10.1038%2Fnature04433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation by a family of JmjC domain-containing proteins</span></div><div class="casAuthors">Tsukada, Yu-ichi; Fang, Jia; Erdjument-Bromage, Hediye; Warren, Maria E.; Borchers, Christoph H.; Tempst, Paul; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7078</span>),
    <span class="NLM_cas:pages">811-816</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Covalent modification of histones has an important role in regulating chromatin dynamics and transcription.  Whereas most covalent histone modifications are reversible, until recently it was unknown whether Me groups could be actively removed from histones.  Using a biochem. assay coupled with chromatog., we have purified a novel JmjC domain-contg. protein, JHDM1 (JmjC domain-contg. histone demethylase 1), that specifically demethylates histone H3 at lysine 36 (H3-K36).  In the presence of Fe(II) and α-ketoglutarate (αKG), JHDM1 demethylates H3-methyl-K36 and generates formaldehyde and succinate.  Overexpression of JHDM1 reduced the level of dimethyl-H3-K36 (H3K36me2) in vivo.  The demethylase activity of the JmjC domain-contg. proteins is conserved, as a JHDM1 homolog in Saccharomyces cerevisiae also has H3-K36 demethylase activity.  Thus, we identify the JmjC domain as a novel demethylase signature motif and uncover a protein demethylation mechanism that is conserved from yeast to human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKQSpeu3QBl7Vg90H21EOLACvtfcHk0li_v9iM4OgX3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D&md5=08f672cc5f81ca88288496f5d8069e3e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature04433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04433%26sid%3Dliteratum%253Aachs%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWarren%26aufirst%3DM.%2BE.%26aulast%3DBorchers%26aufirst%3DC.%2BH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520by%2520a%2520family%2520of%2520JmjC%2520domain-containing%2520proteins%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D811%26epage%3D816%26doi%3D10.1038%2Fnature04433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span> <span> </span><span class="NLM_article-title">The discovery of histone demethylases</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a017947</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a017947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1101%2Fcshperspect.a017947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslGhsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=a017947&author=Y.+G.+Shiauthor=Y.+Tsukada&title=The+discovery+of+histone+demethylases&doi=10.1101%2Fcshperspect.a017947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of histone demethylases</span></div><div class="casAuthors">Shi, Yujiang Geno; Tsukada, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">A017947/1-A017947/3</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Histone methylation is a key element of the eukaryotic epigenome.  Since the discovery of the first histone demethylase (HDM) in 2004, more than 20 demethylases have been identified and characterized.  They belong to either the LSD family or the JmjC family, demonstrating the reversibility of all methylation states at almost all major histone lysine methylation sites.  These findings ended decades of debate about the reversibility of histone methylation, representing a major breakthrough that shifts our understanding of epigenetic inheritance and regulation of genome function.  Here, we summarize the discovery of HDMs and more recent advances, challenges, and future prospects of HDM research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLbFLxtuYl3LVg90H21EOLACvtfcHk0ljm8c6R6Ztawg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslGhsb%252FF&md5=ed2c88e4ec1ba5745617fae942c59c11</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a017947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a017947%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BG.%26aulast%3DTsukada%26aufirst%3DY.%26atitle%3DThe%2520discovery%2520of%2520histone%2520demethylases%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3Da017947%26doi%3D10.1101%2Fcshperspect.a017947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jambhekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastas, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylase inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026484</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1101%2Fcshperspect.a026484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28049654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026484&author=A.+Jambhekarauthor=J.+N.+Anastasauthor=Y.+Shi&title=Histone+lysine+demethylase+inhibitors&doi=10.1101%2Fcshperspect.a026484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase inhibitors</span></div><div class="casAuthors">Jambhekar, Ashwini; Anastas, Jamie N.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">a026484/1-a026484/26</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The dynamic regulation of covalent modifications to histones is essential for maintaining genomic integrity and cell identity and is often compromised in cancer.  Aberrant expression of histone lysine demethylases has been documented in many types of blood and solid tumors, and thus demethylases represent promising therapeutic targets.  Recent advances in high-throughput chem. screening, structure-based drug design, and structure-activity relationship studies have improved both the specificity and the in vivo efficacy of demethylase inhibitors.  This review will briefly outline the connection between demethylases and cancer and will provide a comprehensive overview of the structure, specificity, and utility of currently available demethylase inhibitors.  To date, a select group of demethylase inhibitors is being evaluated in clin. trials, and addnl. compds. may soon follow from the bench to the bedside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqMUZw4KjjDLVg90H21EOLACvtfcHk0ljm8c6R6Ztawg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF&md5=22ba67d04750ab8d410baee59a64d7e6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26aulast%3DJambhekar%26aufirst%3DA.%26aulast%3DAnastas%26aufirst%3DJ.%2BN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026484%26doi%3D10.1101%2Fcshperspect.a026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0ljm8c6R6Ztawg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sennels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappsilber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcini, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri- and dimethylated lysine 4 on histone 3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1076</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-+and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-%2520and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0ljT6tlyBeWfJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0ljT6tlyBeWfJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase lysine demethylase 5B in development and cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">8980</span>– <span class="NLM_lpage">8991</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.13858" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=8980-8991&author=M.+Hanauthor=W.+Xuauthor=P.+Chengauthor=H.+Jinauthor=X.+Wang&title=Histone+demethylase+lysine+demethylase+5B+in+development+and+cancer&doi=10.18632%2Foncotarget.13858"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13858%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DCheng%26aufirst%3DP.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DHistone%2520demethylase%2520lysine%2520demethylase%25205B%2520in%2520development%2520and%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D8980%26epage%3D8991%26doi%3D10.18632%2Foncotarget.13858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombkowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holowatyj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z. Q.</span></span> <span> </span><span class="NLM_article-title">Genomic amplification and a role in drug-resistance for the KDM5A histone demethylase in breast cancer</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">256</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=22937203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Snsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=247-256&author=J.+Houauthor=J.+Wuauthor=A.+Dombkowskiauthor=K.+Zhangauthor=A.+Holowatyjauthor=J.+L.+Boernerauthor=Z.+Q.+Yang&title=Genomic+amplification+and+a+role+in+drug-resistance+for+the+KDM5A+histone+demethylase+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic amplification and a role in drug-resistance for the KDM5A histone demethylase in breast cancer</span></div><div class="casAuthors">Hou, Jinling; Wu, Jack; Dombkowski, Alan; Zhang, Kezhong; Holowatyj, Andreana; Boerner, Julie L.; Yang, Zeng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">247-256</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 5A (KDM5A), an enzyme that removes activating H3K4 di- and tri-methylation marks, plays crit. roles in controlling transcription and chromatin architecture, yet its biol. functions largely remain uncharacterized, particularly in the context of human cancer.  In the present study, we found that the KDM5A gene was significantly amplified and over-expressed in various human tumors, including breast cancer.  Reducing the expression of KDM5A by shRNA knockdown inhibited proliferation of KDM5A-amplified breast cancer cells.  More importantly, we demonstrated that KDM5A over-expression was assocd. with breast cancer drug resistance.  Furthermore, knockdown of KDM5A gene expression altered H3K4 methylation and induced upregulation of CDK inhibitors as well as genes mediating apoptotic cell death.  Taken together, our study strongly links KDM5A histone demethylase activity to breast cancer proliferation and drug resistance, and suggests KDM5A is a potential target for breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPVl0slRNT5LVg90H21EOLACvtfcHk0ljT6tlyBeWfJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Snsb3O&md5=493b21eebbe762fcb8238434fa623faa</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DDombkowski%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DHolowatyj%26aufirst%3DA.%26aulast%3DBoerner%26aufirst%3DJ.%2BL.%26aulast%3DYang%26aufirst%3DZ.%2BQ.%26atitle%3DGenomic%2520amplification%2520and%2520a%2520role%2520in%2520drug-resistance%2520for%2520the%2520KDM5A%2520histone%2520demethylase%2520in%2520breast%2520cancer%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2012%26volume%3D4%26spage%3D247%26epage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeng, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, L. J.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 promotes lung tumorigenesis and cancer metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">4711</span>– <span class="NLM_lpage">4721</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1158%2F0008-5472.CAN-12-3165" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=4711-4721&author=Y.+C.+Tengauthor=C.+F.+Leeauthor=Y.+S.+Liauthor=Y.+R.+Chenauthor=P.+W.+Hsiaoauthor=M.+Y.+Chanauthor=F.+M.+Linauthor=H.+D.+Huangauthor=Y.+T.+Chenauthor=Y.+M.+Jengauthor=C.+H.+Hsuauthor=Q.+Yanauthor=M.+D.+Tsaiauthor=L.+J.+Juan&title=Histone+demethylase+RBP2+promotes+lung+tumorigenesis+and+cancer+metastasis&doi=10.1158%2F0008-5472.CAN-12-3165"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3165%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DY.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BF.%26aulast%3DLi%26aufirst%3DY.%2BS.%26aulast%3DChen%26aufirst%3DY.%2BR.%26aulast%3DHsiao%26aufirst%3DP.%2BW.%26aulast%3DChan%26aufirst%3DM.%2BY.%26aulast%3DLin%26aufirst%3DF.%2BM.%26aulast%3DHuang%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DY.%2BT.%26aulast%3DJeng%26aufirst%3DY.%2BM.%26aulast%3DHsu%26aufirst%3DC.%2BH.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DTsai%26aufirst%3DM.%2BD.%26aulast%3DJuan%26aufirst%3DL.%2BJ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520promotes%2520lung%2520tumorigenesis%2520and%2520cancer%2520metastasis%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D4711%26epage%3D4721%26doi%3D10.1158%2F0008-5472.CAN-12-3165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span> <span> </span><span class="NLM_article-title">RBP2 induces epithelial-mesenchymal transition in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e84735</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0084735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0084735" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e84735&author=S.+Wangauthor=Y.+Wangauthor=H.+Wuauthor=L.+Hu&title=RBP2+induces+epithelial-mesenchymal+transition+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0084735"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084735%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DL.%26atitle%3DRBP2%2520induces%2520epithelial-mesenchymal%2520transition%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De84735%26doi%3D10.1371%2Fjournal.pone.0084735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase retinoblastoma binding protein 2 is overexpressed in hepatocellular carcinoma and negatively regulated by hsa-miR-212</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e69784</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0069784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0069784" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e69784&author=X.+Liangauthor=J.+Zengauthor=L.+Wangauthor=M.+Fangauthor=Q.+Wangauthor=M.+Zhaoauthor=X.+Xuauthor=Z.+Liuauthor=W.+Liauthor=S.+Liuauthor=H.+Yuauthor=J.+Jiaauthor=C.+Chen&title=Histone+demethylase+retinoblastoma+binding+protein+2+is+overexpressed+in+hepatocellular+carcinoma+and+negatively+regulated+by+hsa-miR-212&doi=10.1371%2Fjournal.pone.0069784"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0069784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0069784%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%26atitle%3DHistone%2520demethylase%2520retinoblastoma%2520binding%2520protein%25202%2520is%2520overexpressed%2520in%2520hepatocellular%2520carcinoma%2520and%2520negatively%2520regulated%2520by%2520hsa-miR-212%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De69784%26doi%3D10.1371%2Fjournal.pone.0069784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorkholm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span> <span> </span><span class="NLM_article-title">The histone demethylase RBP2 Is overexpressed in gastric cancer and its inhibition triggers senescence of cancer cells</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2009.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1053%2Fj.gastro.2009.10.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2010&pages=981-992&author=J.+Zengauthor=Z.+Geauthor=L.+Wangauthor=Q.+Liauthor=N.+Wangauthor=M.+Bjorkholmauthor=J.+Jiaauthor=D.+Xu&title=The+histone+demethylase+RBP2+Is+overexpressed+in+gastric+cancer+and+its+inhibition+triggers+senescence+of+cancer+cells&doi=10.1053%2Fj.gastro.2009.10.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2009.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2009.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DBjorkholm%26aufirst%3DM.%26aulast%3DJia%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26atitle%3DThe%2520histone%2520demethylase%2520RBP2%2520Is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520its%2520inhibition%2520triggers%2520senescence%2520of%2520cancer%2520cells%26jtitle%3DGastroenterology%26date%3D2010%26volume%3D138%26spage%3D981%26epage%3D992%26doi%3D10.1053%2Fj.gastro.2009.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiping, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lixiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiuming, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuqun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhifang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yundong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shili, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chunyan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jihui, J.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-212 inhibits proliferation of gastric cancer by directly repressing retinoblastoma binding protein 2</span>. <i>J. Cell. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2666</span>– <span class="NLM_lpage">2672</span>, <span class="refDoi"> DOI: 10.1002/jcb.24613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1002%2Fjcb.24613" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2013&pages=2666-2672&author=Z.+Jipingauthor=F.+Mingauthor=W.+Lixiangauthor=L.+Xiumingauthor=S.+Yuqunauthor=Y.+Hanauthor=L.+Zhifangauthor=S.+Yundongauthor=L.+Shiliauthor=C.+Chunyanauthor=J.+Jihui&title=MicroRNA-212+inhibits+proliferation+of+gastric+cancer+by+directly+repressing+retinoblastoma+binding+protein+2&doi=10.1002%2Fjcb.24613"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcb.24613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.24613%26sid%3Dliteratum%253Aachs%26aulast%3DJiping%26aufirst%3DZ.%26aulast%3DMing%26aufirst%3DF.%26aulast%3DLixiang%26aufirst%3DW.%26aulast%3DXiuming%26aufirst%3DL.%26aulast%3DYuqun%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DZhifang%26aufirst%3DL.%26aulast%3DYundong%26aufirst%3DS.%26aulast%3DShili%26aufirst%3DL.%26aulast%3DChunyan%26aufirst%3DC.%26aulast%3DJihui%26aufirst%3DJ.%26atitle%3DMicroRNA-212%2520inhibits%2520proliferation%2520of%2520gastric%2520cancer%2520by%2520directly%2520repressing%2520retinoblastoma%2520binding%2520protein%25202%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2013%26volume%3D114%26spage%3D2666%26epage%3D2672%26doi%3D10.1002%2Fjcb.24613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fattaey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dembski, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuocolo, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanobik, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Defeo-Jones, D.</span></span> <span> </span><span class="NLM_article-title">Characterization of the retinoblastoma binding proteins RBP1 and RBP2</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3149</span>– <span class="NLM_lpage">3156</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1993&pages=3149-3156&author=A.+R.+Fattaeyauthor=K.+Helinauthor=M.+S.+Dembskiauthor=N.+Dysonauthor=E.+Harlowauthor=G.+A.+Vuocoloauthor=M.+G.+Hanobikauthor=K.+M.+Haskellauthor=A.+Oliffauthor=D.+Defeo-Jones&title=Characterization+of+the+retinoblastoma+binding+proteins+RBP1+and+RBP2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFattaey%26aufirst%3DA.%2BR.%26aulast%3DHelin%26aufirst%3DK.%26aulast%3DDembski%26aufirst%3DM.%2BS.%26aulast%3DDyson%26aufirst%3DN.%26aulast%3DHarlow%26aufirst%3DE.%26aulast%3DVuocolo%26aufirst%3DG.%2BA.%26aulast%3DHanobik%26aufirst%3DM.%2BG.%26aulast%3DHaskell%26aufirst%3DK.%2BM.%26aulast%3DOliff%26aufirst%3DA.%26aulast%3DDefeo-Jones%26aufirst%3DD.%26atitle%3DCharacterization%2520of%2520the%2520retinoblastoma%2520binding%2520proteins%2520RBP1%2520and%2520RBP2%26jtitle%3DOncogene%26date%3D1993%26volume%3D8%26spage%3D3149%26epage%3D3156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">Binding of pRB to the PHD protein RBP2 promotes cellular differentiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2005.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.molcel.2005.05.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2005&pages=623-635&author=E.+V.+Benevolenskayaauthor=H.+L.+Murrayauthor=P.+Brantonauthor=R.+A.+Youngauthor=W.+G.+Kaelin&title=Binding+of+pRB+to+the+PHD+protein+RBP2+promotes+cellular+differentiation&doi=10.1016%2Fj.molcel.2005.05.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMurray%26aufirst%3DH.%2BL.%26aulast%3DBranton%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DBinding%2520of%2520pRB%2520to%2520the%2520PHD%2520protein%2520RBP2%2520promotes%2520cellular%2520differentiation%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D18%26spage%3D623%26epage%3D635%26doi%3D10.1016%2Fj.molcel.2005.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beshiri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisanzio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamishima, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Loss of the retinoblastoma binding protein 2 (RBP2) histone demethylase suppresses tumorigenesis in mice lacking Rb1 or Men1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13379</span>– <span class="NLM_lpage">13386</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110104108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1073%2Fpnas.1110104108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13379-13386&author=W.+Linauthor=J.+Caoauthor=J.+Liuauthor=M.+L.+Beshiriauthor=Y.+Fujiwaraauthor=J.+Francisauthor=A.+D.+Cherniackauthor=C.+Geisenauthor=L.+P.+Blairauthor=M.+R.+Zouauthor=X.+Shenauthor=D.+Kawamoriauthor=Z.+Liuauthor=C.+Grisanzioauthor=H.+Watanabeauthor=Y.+A.+Minamishimaauthor=Q.+Zhangauthor=R.+N.+Kulkarniauthor=S.+Signorettiauthor=S.+J.+Rodigauthor=R.+T.+Bronsonauthor=S.+H.+Orkinauthor=D.+P.+Tuckauthor=E.+V.+Benevolenskayaauthor=M.+Meyersonauthor=W.+G.+Kaelinauthor=Q.+Yan&title=Loss+of+the+retinoblastoma+binding+protein+2+%28RBP2%29+histone+demethylase+suppresses+tumorigenesis+in+mice+lacking+Rb1+or+Men1&doi=10.1073%2Fpnas.1110104108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110104108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110104108%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBeshiri%26aufirst%3DM.%2BL.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DGeisen%26aufirst%3DC.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DKawamori%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DGrisanzio%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DMinamishima%26aufirst%3DY.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DLoss%2520of%2520the%2520retinoblastoma%2520binding%2520protein%25202%2520%2528RBP2%2529%2520histone%2520demethylase%2520suppresses%2520tumorigenesis%2520in%2520mice%2520lacking%2520Rb1%2520or%2520Men1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13379%26epage%3D13386%26doi%3D10.1073%2Fpnas.1110104108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 is critical for breast cancer progression and metastasis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.celrep.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=24582965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=868-877&author=J.+Caoauthor=Z.+Liuauthor=W.+K.+Cheungauthor=M.+Zhaoauthor=S.+Y.+Chenauthor=S.+W.+Chanauthor=C.+J.+Boothauthor=D.+X.+Nguyenauthor=Q.+Yan&title=Histone+demethylase+RBP2+is+critical+for+breast+cancer+progression+and+metastasis&doi=10.1016%2Fj.celrep.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Demethylase RBP2 Is Critical for Breast Cancer Progression and Metastasis</span></div><div class="casAuthors">Cao, Jian; Liu, Zongzhi; Cheung, William K. C.; Zhao, Minghui; Chen, Sophia Y.; Chan, Siew Wee; Booth, Carmen J.; Nguyen, Don X.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-877</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metastasis is a major clin. challenge for cancer treatment.  Emerging evidence suggests that aberrant epigenetic modifications contribute significantly to tumor formation and progression.  However, the drivers and roles of such epigenetic changes in tumor metastasis are still poorly understood.  Using bioinformatic anal. of human breast cancer gene-expression data sets, we identified histone demethylase RBP2 as a putative mediator of metastatic progression.  By using both human breast cancer cells and genetically engineered mice, we demonstrated that RBP2 is crit. for breast cancer metastasis to the lung in multiple in vivo models.  Mechanistically, RBP2 promotes metastasis as a pleiotropic pos. regulator of many metastasis genes, including TNC.  In addn., RBP2 loss suppresses tumor formation in MMTV-neu transgenic mice.  These results suggest that therapeutic targeting of RBP2 is a potential strategy for inhibition of tumor progression and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXZwTcPjZTYLVg90H21EOLACvtfcHk0li-3cqeleeHwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D&md5=ecc62cbf9e2bcab60746534aa536b87d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DCheung%26aufirst%3DW.%2BK.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%2BY.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DBooth%26aufirst%3DC.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BX.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520is%2520critical%2520for%2520breast%2520cancer%2520progression%2520and%2520metastasis%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D868%26epage%3D877%26doi%3D10.1016%2Fj.celrep.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannappan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuan, S.</span></span> <span> </span><span class="NLM_article-title">RBP2 induces stem-like cancer cells by promoting EMT and is a prognostic marker for renal cell carcinoma</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">e238</span>, <span class="refDoi"> DOI: 10.1038/emm.2016.37</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Femm.2016.37" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27282106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XptV2is7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=e238&author=D.+Zhouauthor=V.+Kannappanauthor=X.+Chenauthor=J.+Liauthor=X.+Lengauthor=J.+Zhangauthor=S.+Xuan&title=RBP2+induces+stem-like+cancer+cells+by+promoting+EMT+and+is+a+prognostic+marker+for+renal+cell+carcinoma&doi=10.1038%2Femm.2016.37"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 induces stem-like cancer cells by promoting EMT and is a prognostic marker for renal cell carcinoma</span></div><div class="casAuthors">Zhou, Dahai; Kannappan, Vinodh; Chen, Xiang; Li, Jingqin; Leng, Xuefeng; Zhang, Jinping; Xuan, Shiying</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e238</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Renal cell carcinoma (RCC), one of the most common kidney cancers, has a poor prognosis.  Epithelial to mesenchymal transition (EMT) is a hallmark of carcinoma invasion and metastasis.  Several studies have examd. the mol. regulation of EMT, but the relationship between histone demethylases and EMT is little understood.  In this study, we investigated the role of retinoblastoma-binding protein-2 (RBP2), a histone demethylase that is highly expressed in RCC and is pos. correlated with poor RCC prognosis in the regulation of EMT.  We found that ectopic overexpression of RBP2 can induce cancer stem cell-like (CSC) phenotypes through EMT in RCC cells by converting them to a more mesenchymal phenotype.  This results in increased resistance to apoptosis, which leads to enhanced tumor growth in xenograft models.  Together, our data show that RBP2 is an epigenetic regulator that has an important role in the initiation of CSC phenotypes through EMT, leading to tumor progression.  RBP2 is also a novel biomol. for RCC diagnosis, and prognosis and may be a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9Gdi_fboErVg90H21EOLACvtfcHk0li-3cqeleeHwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptV2is7o%253D&md5=beced570b811a36b2f35328e58c6ae37</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Femm.2016.37&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femm.2016.37%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DKannappan%26aufirst%3DV.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DXuan%26aufirst%3DS.%26atitle%3DRBP2%2520induces%2520stem-like%2520cancer%2520cells%2520by%2520promoting%2520EMT%2520and%2520is%2520a%2520prognostic%2520marker%2520for%2520renal%2520cell%2520carcinoma%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2016%26volume%3D48%26spage%3De238%26doi%3D10.1038%2Femm.2016.37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ci, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, J.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 promotes malignant progression of gastric cancer through TGF-beta1-(p-Smad3)-RBP2-E-cadherin-Smad3 feedback circuit</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17661</span>– <span class="NLM_lpage">17674</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.3756" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17661-17674&author=X.+Liangauthor=J.+Zengauthor=L.+Wangauthor=L.+Shenauthor=X.+Maauthor=S.+Liauthor=Y.+Wuauthor=L.+Maauthor=X.+Ciauthor=Q.+Guoauthor=M.+Jiaauthor=H.+Shenauthor=Y.+Sunauthor=Z.+Liuauthor=S.+Liuauthor=W.+Liauthor=H.+Yuauthor=C.+Chenauthor=J.+Jia&title=Histone+demethylase+RBP2+promotes+malignant+progression+of+gastric+cancer+through+TGF-beta1-%28p-Smad3%29-RBP2-E-cadherin-Smad3+feedback+circuit&doi=10.18632%2Foncotarget.3756"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3756%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DCi%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DJia%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DJia%26aufirst%3DJ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520promotes%2520malignant%2520progression%2520of%2520gastric%2520cancer%2520through%2520TGF-beta1-%2528p-Smad3%2529-RBP2-E-cadherin-Smad3%2520feedback%2520circuit%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17661%26epage%3D17674%26doi%3D10.18632%2Foncotarget.3756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azizian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2010.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20371346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=69-80&author=S.+V.+Sharmaauthor=D.+Y.+Leeauthor=B.+Liauthor=M.+P.+Quinlanauthor=F.+Takahashiauthor=S.+Maheswaranauthor=U.+McDermottauthor=N.+Azizianauthor=L.+Zouauthor=M.+A.+Fischbachauthor=K.+K.+Wongauthor=K.+Brandstetterauthor=B.+Wittnerauthor=S.+Ramaswamyauthor=M.+Classonauthor=J.+Settleman&title=A+chromatin-mediated+reversible+drug-tolerant+state+in+cancer+cell+subpopulations&doi=10.1016%2Fj.cell.2010.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span></div><div class="casAuthors">Sharma, Sreenath V.; Lee, Diana Y.; Li, Bihua; Quinlan, Margaret P.; Takahashi, Fumiyuki; Maheswaran, Shyamala; McDermott, Ultan; Azizian, Nancy; Zou, Lee; Fischbach, Michael A.; Wong, Kwok-Kin; Brandstetter, Kathleyn; Wittner, Ben; Ramaswamy, Sridhar; Classon, Marie; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-80</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Accumulating evidence implicates heterogeneity within cancer cell populations in the response to stressful exposures, including drug treatments.  While modeling the acute response to various anticancer agents in drug-sensitive human tumor cell lines, we consistently detected a small subpopulation of reversibly "drug-tolerant" cells.  These cells demonstrate >100-fold reduced drug sensitivity and maintain viability via engagement of IGF-1 receptor signaling and an altered chromatin state that requires the histone demethylase RBP2/KDM5A/Jarid1A.  This drug-tolerant phenotype is transiently acquired and relinquished at low frequency by individual cells within the population, implicating the dynamic regulation of phenotypic heterogeneity in drug tolerance.  The drug-tolerant subpopulation can be selectively ablated by treatment with IGF-1 receptor inhibitors or chromatin-modifying agents, potentially yielding a therapeutic opportunity.  Together, these findings suggest that cancer cell populations employ a dynamic survival strategy in which individual cells transiently assume a reversibly drug-tolerant state to protect the population from eradication by potentially lethal exposures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK0DpI9mbR_bVg90H21EOLACvtfcHk0lg52_GvIWmvmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D&md5=2070b7cc4385a4f98db15e5ebcd4589d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DLee%26aufirst%3DD.%2BY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQuinlan%26aufirst%3DM.%2BP.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DAzizian%26aufirst%3DN.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DBrandstetter%26aufirst%3DK.%26aulast%3DWittner%26aufirst%3DB.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520chromatin-mediated%2520reversible%2520drug-tolerant%2520state%2520in%2520cancer%2520cell%2520subpopulations%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.cell.2010.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carra, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parodi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattarozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carosio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forlani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allemanni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marubbi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Florio, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romani, M.</span></span> <span> </span><span class="NLM_article-title">The histone demethylase KDM5A is a key factor for the resistance to Temozolomide in glioblastoma</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">3418</span>– <span class="NLM_lpage">3429</span>, <span class="refDoi"> DOI: 10.1080/15384101.2015.1090063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1080%2F15384101.2015.1090063" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=3418-3429&author=B.+Banelliauthor=E.+Carraauthor=F.+Barbieriauthor=R.+Wurthauthor=F.+Parodiauthor=A.+Pattarozziauthor=R.+Carosioauthor=A.+Forlaniauthor=G.+Allemanniauthor=D.+Marubbiauthor=T.+Florioauthor=A.+Dagaauthor=M.+Romani&title=The+histone+demethylase+KDM5A+is+a+key+factor+for+the+resistance+to+Temozolomide+in+glioblastoma&doi=10.1080%2F15384101.2015.1090063"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1080%2F15384101.2015.1090063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15384101.2015.1090063%26sid%3Dliteratum%253Aachs%26aulast%3DBanelli%26aufirst%3DB.%26aulast%3DCarra%26aufirst%3DE.%26aulast%3DBarbieri%26aufirst%3DF.%26aulast%3DWurth%26aufirst%3DR.%26aulast%3DParodi%26aufirst%3DF.%26aulast%3DPattarozzi%26aufirst%3DA.%26aulast%3DCarosio%26aufirst%3DR.%26aulast%3DForlani%26aufirst%3DA.%26aulast%3DAllemanni%26aufirst%3DG.%26aulast%3DMarubbi%26aufirst%3DD.%26aulast%3DFlorio%26aufirst%3DT.%26aulast%3DDaga%26aufirst%3DA.%26aulast%3DRomani%26aufirst%3DM.%26atitle%3DThe%2520histone%2520demethylase%2520KDM5A%2520is%2520a%2520key%2520factor%2520for%2520the%2520resistance%2520to%2520Temozolomide%2520in%2520glioblastoma%26jtitle%3DCell%2520Cycle%26date%3D2015%26volume%3D14%26spage%3D3418%26epage%3D3429%26doi%3D10.1080%2F15384101.2015.1090063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamane, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tateishi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klose, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabrizio, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdjument-Bromage, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor-Papadimitriou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">PLU-1 is an H3K4 demethylase involved in transcriptional repression and breast cancer cell proliferation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.molcel.2007.03.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=801-812&author=K.+Yamaneauthor=K.+Tateishiauthor=R.+J.+Kloseauthor=J.+Fangauthor=L.+A.+Fabrizioauthor=H.+Erdjument-Bromageauthor=J.+Taylor-Papadimitriouauthor=P.+Tempstauthor=Y.+Zhang&title=PLU-1+is+an+H3K4+demethylase+involved+in+transcriptional+repression+and+breast+cancer+cell+proliferation&doi=10.1016%2Fj.molcel.2007.03.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DTateishi%26aufirst%3DK.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DFabrizio%26aufirst%3DL.%2BA.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTaylor-Papadimitriou%26aufirst%3DJ.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPLU-1%2520is%2520an%2520H3K4%2520demethylase%2520involved%2520in%2520transcriptional%2520repression%2520and%2520breast%2520cancer%2520cell%2520proliferation%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D801%26epage%3D812%26doi%3D10.1016%2Fj.molcel.2007.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Cancer
Genome Atlas Network</span> <span> </span><span class="NLM_article-title">Comprehensive
molecular portraits of human breast tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">70</span>, <span class="refDoi"> DOI: 10.1038/nature11412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature11412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23000897" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=61-70&author=Cancer%0AGenome+Atlas+Network&title=Comprehensive%0Amolecular+portraits+of+human+breast+tumours&doi=10.1038%2Fnature11412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature11412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11412%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%250Amolecular%2520portraits%2520of%2520human%2520breast%2520tumours%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D61%26epage%3D70%26doi%3D10.1038%2Fnature11412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span> <span> </span><span class="NLM_article-title">Overexpression of JARID1B is associated with poor prognosis and chemotherapy resistance in epithelial ovarian cancer</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2472</span>, <span class="refDoi"> DOI: 10.1007/s13277-014-2859-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2Fs13277-014-2859-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisleiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=2465-2472&author=L.+Wangauthor=Y.+Maoauthor=G.+Duauthor=C.+Heauthor=S.+Han&title=Overexpression+of+JARID1B+is+associated+with+poor+prognosis+and+chemotherapy+resistance+in+epithelial+ovarian+cancer&doi=10.1007%2Fs13277-014-2859-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of JARID1B is associated with poor prognosis and chemotherapy resistance in epithelial ovarian cancer</span></div><div class="casAuthors">Wang, Lishuang; Mao, Yuanfu; Du, Guiqin; He, Chunbo; Han, Shiyu</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2465-2472</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">JARID1B, a histone demethylase, has been reported to be highly expressed in various human cancers.  In the present study, we investigated the assocn. of JARID1B level with epithelial ovarian cancer (EOC) and prognosis of patients with EOC.  We analyzed JARID1B expression in 20 normal ovaries, 20 benign ovarian tumor (BOT) samples, and 45 epithelial ovarian carcinoma specimens by quant. PCR (qRT-PCR) and western blotting analyses.  JARID1B was further examd. in 120 EOC specimens from patients with different histol. stages via immunohistochem.  Possible correlations between JARID1B levels and prognosis as well as chemotherapy resistance of EOC patients were detd. by univariate and multivariate analyses.  JARID1B level was significantly increased in EOC, as compared to normal ovaries and BOT.  Among 120 EOC cases examd., the 5-yr progression-free survival (PFS) rates were 17 and 85 % in patients with high and low JARID1B expression, resp. (hazard ratio = 17.85, 95 % confidence interval (CI) 6.31-50.51, P < 0.001).  Similarly, the 5-yr overall survival (OS) rates for patients with high and low JARID1B expression were 28 and 92 % resp. (hazard ratio = 21.8, 95 % CI 5.92-71.81, P < 0.001).  Pos. correlation between JARID1B level and chemotherapy resistance was obsd. in patients with EOC (odds ratio (OR) 36.81, 95 % CI 4.84-280.11, P < 0.001).  JARID1B could serve as an important biomarker for prognosis and chemotherapy resistance of EOC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCGJs4I05U0LVg90H21EOLACvtfcHk0lha_tUiYUVF8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisleiurw%253D&md5=454837068da870bda633be1e4ba80a27</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs13277-014-2859-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-014-2859-z%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DS.%26atitle%3DOverexpression%2520of%2520JARID1B%2520is%2520associated%2520with%2520poor%2520prognosis%2520and%2520chemotherapy%2520resistance%2520in%2520epithelial%2520ovarian%2520cancer%26jtitle%3DTumor%2520Biol.%26date%3D2015%26volume%3D36%26spage%3D2465%26epage%3D2472%26doi%3D10.1007%2Fs13277-014-2859-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span> <span> </span><span class="NLM_article-title">JARID1B promotes metastasis and epithelial-mesenchymal transition via PTEN/AKT signaling in hepatocellular carcinoma cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12723</span>– <span class="NLM_lpage">12739</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.18632%2Foncotarget.3713" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12723-12739&author=B.+Tangauthor=G.+Qiauthor=F.+Tangauthor=S.+Yuanauthor=Z.+Wangauthor=X.+Liangauthor=B.+Liauthor=S.+Yuauthor=J.+Liuauthor=Q.+Huangauthor=Y.+Weiauthor=R.+Zhaiauthor=B.+Leiauthor=H.+Yuauthor=X.+Jiaoauthor=S.+He&title=JARID1B+promotes+metastasis+and+epithelial-mesenchymal+transition+via+PTEN%2FAKT+signaling+in+hepatocellular+carcinoma+cells&doi=10.18632%2Foncotarget.3713"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3713%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DB.%26aulast%3DQi%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DZhai%26aufirst%3DR.%26aulast%3DLei%26aufirst%3DB.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DJiao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DS.%26atitle%3DJARID1B%2520promotes%2520metastasis%2520and%2520epithelial-mesenchymal%2520transition%2520via%2520PTEN%252FAKT%2520signaling%2520in%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D12723%26epage%3D12739%26doi%3D10.18632%2Foncotarget.3713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span> <span> </span><span class="NLM_article-title">KDM5B is overexpressed in gastric cancer and is required for gastric cancer cell proliferation and metastasis</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=25628922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=87-100&author=Z.+Wangauthor=F.+Tangauthor=G.+Qiauthor=S.+Yuanauthor=G.+Zhangauthor=B.+Tangauthor=S.+He&title=KDM5B+is+overexpressed+in+gastric+cancer+and+is+required+for+gastric+cancer+cell+proliferation+and+metastasis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">KDM5B is overexpressed in gastric cancer and is required for gastric cancer cell proliferation and metastasis</span></div><div class="casAuthors">Wang, Zhenran; Tang, Fang; Qi, Guangying; Yuan, Shengguang; Zhang, Guangyu; Tang, Bo; He, Songqing</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-100</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Epigenetic alterations such as aberrant expression of histone-modifying enzymes have been implicated in tumorigenesis.  KDM5B (also known as JARID1B) is a newly identified histone demethylase that regulates chromatin structure or gene expression by removing Me residues from trimethylated lysine 4 on histone H3.  Recent observations have shown oncogenic activity of KDM5B.  However, the role of KDM5B in gastric cancer carcinogenesis remains unclear.  In this study, we aimed to investigate the role of KDM5B in gastric cancer.  Immunohistochem. anal., western blotting, and qRT-PCR were used to measure the levels of KDM5B in gastric cancer cell lines, 45 pairs of gastric cancer tissues and the adjacent nonneoplastic tissues.  KDM5B and shKDM5B were transfected into gastric cancer cells to investigate its role on regulating cell proliferation which was measured by MTT and colony formation assay.  Cell's migration and invasion were measured by Transwell and Matrigel anal. in vitro.  PCNA expression was measured by immunofluorescence staining and immunohistochem. anal.  The in vivo tumorigenesis and metastasis assays were performed in SCID mice.  In clin. gastric cancer samples, we found that KDM5B expression was significantly up-regulated in cancer lesions compared with paired normal gastric tissues.  By silencing or overexpressing KDM5B in gastric cancer cells, we found that KDM5B could promote cell growth and metastasis in vitro.  An in vivo assay showed that KDM5B not only dramatically promoted gastric cancer cell xenograft formation and growth but also promoted gastric cancer cell metastasis in a liver metastasis model.  Moreover, we demonstrated that KDM5B promoted gastric cancer metastasis via regulation of the Akt pathway.  Our study provided evidence that KDM5B functions as a novel tumor oncogene in gastric cancer and may be a potential therapeutic target for gastric cancer management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWbphoPhfOT7Vg90H21EOLACvtfcHk0lgqSfGHffnswA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFOrtL4%253D&md5=3efc8f5ca6162a10712837e0c0b8155b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DTang%26aufirst%3DF.%26aulast%3DQi%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DTang%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DS.%26atitle%3DKDM5B%2520is%2520overexpressed%2520in%2520gastric%2520cancer%2520and%2520is%2520required%2520for%2520gastric%2520cancer%2520cell%2520proliferation%2520and%2520metastasis%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2015%26volume%3D5%26spage%3D87%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adebayo, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span> <span> </span><span class="NLM_article-title">Silencing JARID1B suppresses oncogenicity, stemness and increases radiation sensitivity in human oral carcinoma</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2015.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.canlet.2015.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26184998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOkt73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2015&pages=36-45&author=C.+S.+Linauthor=Y.+C.+Linauthor=B.+O.+Adebayoauthor=A.+Wuauthor=J.+H.+Chenauthor=Y.+J.+Pengauthor=M.+F.+Chengauthor=W.+H.+Leeauthor=M.+Hsiaoauthor=T.+Y.+Chaoauthor=C.+T.+Yeh&title=Silencing+JARID1B+suppresses+oncogenicity%2C+stemness+and+increases+radiation+sensitivity+in+human+oral+carcinoma&doi=10.1016%2Fj.canlet.2015.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing JARID1B suppresses oncogenicity, stemness and increases radiation sensitivity in human oral carcinoma</span></div><div class="casAuthors">Lin, Chun-Shu; Lin, Ying-Chin; Adebayo, Bamodu Oluwaseun; Wu, Alexander; Chen, Jia-Hong; Peng, Yi-Jen; Cheng, Ming-Fang; Lee, Wei-Hwa; Hsiao, Michael; Chao, Tsu-Yi; Yeh, Chi-Tai</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-45</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Oral squamous cell carcinoma (OSCC) is a major cause of human mortality globally and radiotherapy is one of the main treatment modalities, however its therapeutic effect is often limited by radioresistance.  JARID1B is an epigenetic factor with reported oncogenic potential in various cancer types.  We investigated the effect of JARID1B inhibition on migration and invasion of human OSCC cell lines, as well as on clin. patients' outcome.  Wound healing, matrigel invasion, Sulforhodamine B, and spheroid formation assays were used to characterize the signaling pathways of shJARID1B in response to radiation treatment.  We evaluated the prognostic relevance of Jarid1b expression in a cohort of 81 OSCC patients.  Human OSCC cell lines, including SAS, HSC3, Cal27, TW2.6 and SCC4 cells, were used.  ShJARID1B cells significantly inhibited migration and invasion ability compared to their vector or wild type counterparts.  Silencing shJARID1B significantly inhibited oral cancer stem cell activity and potentiated the tumor-inhibitory activity of radiation therapy in OSCC.  Radiotherapy coupled with shJARID1B knockdown reduced mRNA levels of NQO1, KEAP1, NRF2, FOXO1, FOXO3, KLF4, OCT4, CD133, and Nanog in malignant OSCC cells.  OSCC spheroid formation ability was markedly reduced in the shJARID1B cells.  JARID1B overexpression is a dependent prognostic factor in OSCC patients.  Silencing shJARID1B inhibits migration and invasion of human OSCC, reduces cancer stem cell activities and potentiates tumor-inhibiting radiotherapeutic effects.  JARID1B knockdown prior to radiotherapy is a potential effective therapeutic strategy for the treatment of OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIgWCI5f2rwLVg90H21EOLACvtfcHk0lgqSfGHffnswA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOkt73J&md5=9ada7ae0e6817152339cada752a18c3d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2015.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2015.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.%2BS.%26aulast%3DLin%26aufirst%3DY.%2BC.%26aulast%3DAdebayo%26aufirst%3DB.%2BO.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DPeng%26aufirst%3DY.%2BJ.%26aulast%3DCheng%26aufirst%3DM.%2BF.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DChao%26aufirst%3DT.%2BY.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26atitle%3DSilencing%2520JARID1B%2520suppresses%2520oncogenicity%252C%2520stemness%2520and%2520increases%2520radiation%2520sensitivity%2520in%2520human%2520oral%2520carcinoma%26jtitle%3DCancer%2520Lett.%26date%3D2015%26volume%3D368%26spage%3D36%26epage%3D45%26doi%3D10.1016%2Fj.canlet.2015.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adebayo, B. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. M.</span></span> <span> </span><span class="NLM_article-title">JARID1B expression plays a critical role in chemoresistance and stem cell-like phenotype of neuroblastoma cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0125343</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0125343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1371%2Fjournal.pone.0125343" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0125343&author=Y.+T.+Kuoauthor=Y.+L.+Liuauthor=B.+O.+Adebayoauthor=P.+H.+Shihauthor=W.+H.+Leeauthor=L.+S.+Wangauthor=Y.+F.+Liaoauthor=W.+M.+Hsuauthor=C.+T.+Yehauthor=C.+M.+Lin&title=JARID1B+expression+plays+a+critical+role+in+chemoresistance+and+stem+cell-like+phenotype+of+neuroblastoma+cells&doi=10.1371%2Fjournal.pone.0125343"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0125343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0125343%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DY.%2BT.%26aulast%3DLiu%26aufirst%3DY.%2BL.%26aulast%3DAdebayo%26aufirst%3DB.%2BO.%26aulast%3DShih%26aufirst%3DP.%2BH.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DL.%2BS.%26aulast%3DLiao%26aufirst%3DY.%2BF.%26aulast%3DHsu%26aufirst%3DW.%2BM.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26aulast%3DLin%26aufirst%3DC.%2BM.%26atitle%3DJARID1B%2520expression%2520plays%2520a%2520critical%2520role%2520in%2520chemoresistance%2520and%2520stem%2520cell-like%2520phenotype%2520of%2520neuroblastoma%2520cells%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0125343%26doi%3D10.1371%2Fjournal.pone.0125343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamodu, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, T. Y.</span></span> <span> </span><span class="NLM_article-title">Aberrant KDM5B expression promotes aggressive breast cancer through MALAT1 overexpression and downregulation of hsa-miR-448</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">160</span>, <span class="refDoi"> DOI: 10.1186/s12885-016-2108-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1186%2Fs12885-016-2108-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26917489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWmtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=160&author=O.+A.+Bamoduauthor=W.+C.+Huangauthor=W.+H.+Leeauthor=A.+Wuauthor=L.+S.+Wangauthor=M.+Hsiaoauthor=C.+T.+Yehauthor=T.+Y.+Chao&title=Aberrant+KDM5B+expression+promotes+aggressive+breast+cancer+through+MALAT1+overexpression+and+downregulation+of+hsa-miR-448&doi=10.1186%2Fs12885-016-2108-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant KDM5B expression promotes aggressive breast cancer through MALAT1 overexpression and downregulation of hsa-miR-448</span></div><div class="casAuthors">Bamodu, Oluwaseun Adebayo; Huang, Wen-Chien; Lee, Wei-Hwa; Wu, Alexander; Wang, Liang Shun; Hsiao, Michael; Yeh, Chi-Tai; Chao, Tsu-Yi</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">160/1-160/18</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Triple neg. breast cancers (TNBC) possess cell dedifferentiation characteristics, carry out activities connate to those of cancer stem cells (CSCs) and are assocd. with increased metastasis, as well as, poor clin. prognosis.  The regulatory mechanism of this highly malignant phenotype is still poorly characterized.  Accruing evidence support the role of non-coding RNAs (ncRNAs) as potent regulators of CSC and metastatic gene expression, with their dysregulation implicated in tumorigenesis and disease progression.  Methods: In this study, we investigated TNBC metastasis, metastasis-assocd. genes and potential inhibitory mechanisms using bioinformatics, tissue microarray analyses, immunoblotting, polymerase chain reaction, loss and gain of gene function assays and comparative analyses of data obtained.  Results: Compared with other breast cancer types, the highly metastatic MDA-MB-231 cells concurrently exhibited increased expression levels of Lysine-specific demethylase 5B protein (KDM5B) and long non-coding RNA (lncRNA), MALAT1, suggesting their functional assocn.  KDM5B-silencing in the TNBC cells correlated with the upregulation of hsa-miR-448 and led to suppression of MALAT1 expression with decreased migration, invasion and clonogenic capacity in vitro, as well as, poor survival in vivo.  This projects MALAT1 as a mediator of KDM5B oncogenic potential and highlights the crit. role of this microRNA, lncRNA and histone demethylase in cancer cell motility and metastatic colonization.  Increased expression of KDM5B correlating with disease progression and poor clin. outcome in breast cancer was reversed by hsa-miR-448.  Conclusions: Our findings demonstrate the crit. role of KDM5B and its neg. regulator hsa-miR-448 in TNBC metastasis and progression.  Hsa-miR-448 disrupting KDM5B-MALAT1 signalling axis and assocd. activities in TNBC cells, projects it as a putative therapeutic factor for selective eradication of TNBC cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHXcNra2EJ3rVg90H21EOLACvtfcHk0lgsDjDdmkL-dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWmtb8%253D&md5=74b8458814f7349adef040c96dcb7914</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2Fs12885-016-2108-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-016-2108-5%26sid%3Dliteratum%253Aachs%26aulast%3DBamodu%26aufirst%3DO.%2BA.%26aulast%3DHuang%26aufirst%3DW.%2BC.%26aulast%3DLee%26aufirst%3DW.%2BH.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%2BS.%26aulast%3DHsiao%26aufirst%3DM.%26aulast%3DYeh%26aufirst%3DC.%2BT.%26aulast%3DChao%26aufirst%3DT.%2BY.%26atitle%3DAberrant%2520KDM5B%2520expression%2520promotes%2520aggressive%2520breast%2520cancer%2520through%2520MALAT1%2520overexpression%2520and%2520downregulation%2520of%2520hsa-miR-448%26jtitle%3DBMC%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D160%26doi%3D10.1186%2Fs12885-016-2108-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0lgsDjDdmkL-dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of mm1s myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+Mullerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+mm1s+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of ∼50 μM and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0li_ysITgVNgeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520mm1s%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for KDM5A histone lysine demethylase inhibition by diverse compounds</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.chembiol.2016.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27427228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=769-781&author=J.+R.+Hortonauthor=X.+Liuauthor=M.+Galeauthor=L.+Wuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=P.+J.+Webberauthor=J.+S.+Bellauthor=S.+C.+Kalesauthor=B.+T.+Mottauthor=G.+Raiauthor=D.+J.+Jansenauthor=M.+J.+Hendersonauthor=D.+J.+Urbanauthor=M.+D.+Hallauthor=A.+Simeonovauthor=D.+J.+Maloneyauthor=M.+A.+Johnsauthor=H.+Fuauthor=A.+Jadhavauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Structural+basis+for+KDM5A+histone+lysine+demethylase+inhibition+by+diverse+compounds&doi=10.1016%2Fj.chembiol.2016.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for KDM5A Histone Lysine Demethylase Inhibition by Diverse Compounds</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Gale, Molly; Wu, Lizhen; Shanks, John R.; Zhang, Xing; Webber, Philip J.; Bell, Joshua S. K.; Kales, Stephen C.; Mott, Bryan T.; Rai, Ganesha; Jansen, Daniel J.; Henderson, Mark J.; Urban, Daniel J.; Hall, Matthew D.; Simeonov, Anton; Maloney, David J.; Johns, Margaret A.; Fu, Haian; Jadhav, Ajit; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">769-781</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases removes Me groups from methylated lysine 4 of histone H3.  Accumulating evidence supports a role for KDM5 family members as oncogenic drivers.  We compare the in vitro inhibitory properties and binding affinity of ten diverse compds. with all four family members, and present the crystal structures of the KDM5A-linked Jumonji domain in complex with eight of these inhibitors in the presence of Mn(II).  All eight inhibitors structurally examd. occupy the binding site of α-ketoglutarate, but differ in their specific binding interactions, including the no. of ligands involved in metal coordination.  We also obsd. inhibitor-induced conformational changes in KDM5A, particularly those residues involved in the binding of α-ketoglutarate, the anticipated peptide substrate, and intramol. interactions.  We discuss how particular chem. moieties contribute to inhibitor potency and suggest strategies that might be utilized in the successful design of selective and potent epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstbuqHdktk7Vg90H21EOLACvtfcHk0li_ysITgVNgeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L&md5=0dc780604904ac406211a09577df5bf5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGale%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWebber%26aufirst%3DP.%2BJ.%26aulast%3DBell%26aufirst%3DJ.%2BS.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520KDM5A%2520histone%2520lysine%2520demethylase%2520inhibition%2520by%2520diverse%2520compounds%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D769%26epage%3D781%26doi%3D10.1016%2Fj.chembiol.2016.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaThangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnchembio.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=27214403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=539-545&author=C.+Johanssonauthor=S.+Velupillaiauthor=A.+Tumberauthor=A.+Szykowskaauthor=E.+S.+Hookwayauthor=R.+P.+Nowakauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=M.+Philpottauthor=N.+Burgess-Brownauthor=N.+Wuauthor=J.+Kopecauthor=A.+Nuzziauthor=H.+Steuberauthor=U.+Egnerauthor=V.+Badockauthor=S.+Munroauthor=N.+B.+LaThangueauthor=S.+Westawayauthor=J.+Brownauthor=N.+Athanasouauthor=R.+Prinjhaauthor=P.+E.+Brennanauthor=U.+Oppermann&title=Structural+analysis+of+human+KDM5B+guides+histone+demethylase+inhibitor+development&doi=10.1038%2Fnchembio.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span></div><div class="casAuthors">Johansson, Catrine; Velupillai, Srikannathasan; Tumber, Anthony; Szykowska, Aleksandra; Hookway, Edward S.; Nowak, Radoslaw P.; Strain-Damerell, Claire; Gileadi, Carina; Philpott, Martin; Burgess-Brown, Nicola; Wu, Na; Kopec, Jola; Nuzzi, Andrea; Steuber, Holger; Egner, Ursula; Badock, Volker; Munro, Shonagh; LaThangue, Nicholas B.; Westaway, Sue; Brown, Jack; Athanasou, Nick; Prinjha, Rab; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-545</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the KDM5 (also known as JARID1) family are 2-oxoglutarate- and Fe2+-dependent oxygenases that act as histone H3K4 demethylases, thereby regulating cell proliferation and stem cell self-renewal and differentiation.  Here, the authors report crystal structures of the catalytic core of the human KDM5B enzyme in complex with 3 inhibitor chemotypes.  These scaffolds exploitet several aspects of the KDM5 active site, and their selectivity profiles reflected their hybrid features with respect to the KDM4 and KDM6 families.  Whereas GSK-J1, a previously identified KDM6 inhibitor, showed ∼7-fold less inhibitory activity toward KDM5B than toward KDM6 proteins, KDM5-C49 displayed 25-100-fold selectivity between KDM5B and KDM6B.  The cell-permeable deriv. KDM5-C70 had an antiproliferative effect in myeloma cells, leading to genome-wide elevation of H3K4me3 levels.  The selective inhibitor GSK467 exploited unique binding modes, but it lacked cellular potency in the myeloma system.  Taken together, these structural leads deliver multiple starting points for further rational and selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp-2vDMpTjHbVg90H21EOLACvtfcHk0lhE53DcVJdhag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D&md5=2c4b6e5779ba43ed3584fc6f76b987cd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2087%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DKopec%26aufirst%3DJ.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLaThangue%26aufirst%3DN.%2BB.%26aulast%3DWestaway%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520analysis%2520of%2520human%2520KDM5B%2520guides%2520histone%2520demethylase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D539%26epage%3D545%26doi%3D10.1038%2Fnchembio.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham-Huy, C.</span></span> <span> </span><span class="NLM_article-title">Chiral drugs: an overview</span>. <i>Int. J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">100</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23674971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=85-100&author=L.+A.+Nguyenauthor=H.+Heauthor=C.+Pham-Huy&title=Chiral+drugs%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Chiral drugs. An overview</span></div><div class="casAuthors">Nguyen, Lien Ai; He, Hua; Chuong, Pham-Huy</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biomedical Science (Monterey Park, CA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-100</span>CODEN:
                <span class="NLM_cas:coden">IJBSDB</span>;
        ISSN:<span class="NLM_cas:issn">1550-9702</span>.
    
            (<span class="NLM_cas:orgname">Master Publishing Group</span>)
        </div><div class="casAbstract">A review.  About more than half of the drugs currently in use are chiral compds. and near 90% of the last ones are marketed as racemates consisting of an equimolar mixt. of two enantiomers.  Although they have the same chem. structure, most isomers of chiral drugs exhibit marked differences in biol. activities such as pharmacol., toxicol., pharmacokinetics, metab. etc.  Some mechanisms of these properties are also explained.  Therefore, it is important to promote the chiral sepn. and anal. of racemic drugs in pharmaceutical industry as well as in clinic in order to eliminate the unwanted isomer from the prepn. and to find an optimal treatment and a right therapeutic control for the patient.  In this article, we review the nomenclature, pharmacol., toxicol., pharmacokinetics, metab. etc of some usual chiral drugs as well as their mechanisms.  Different techniques used for the chiral sepn. in pharmaceutical industry as well as in clin. analyses are also examd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovoqsLM_3bxLVg90H21EOLACvtfcHk0lhE53DcVJdhag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSltrnJ&md5=254cbca060bb4cfd04eac365488f5b62</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DL.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DPham-Huy%26aufirst%3DC.%26atitle%3DChiral%2520drugs%253A%2520an%2520overview%26jtitle%3DInt.%2520J.%2520Biomed.%2520Sci.%26date%3D2006%26volume%3D2%26spage%3D85%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chhabra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aseri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, D.</span></span> <span> </span><span class="NLM_article-title">A review of drug isomerism and its significance</span>. <i>Int. J. Appl. Basic. Med. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.4103/2229-516X.112233</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.4103%2F2229-516X.112233" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23776834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A280%3ADC%252BC3sjjsVymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=16-18&author=N.+Chhabraauthor=M.+L.+Aseriauthor=D.+Padmanabhan&title=A+review+of+drug+isomerism+and+its+significance&doi=10.4103%2F2229-516X.112233"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A review of drug isomerism and its significance</span></div><div class="casAuthors">Chhabra Naveen; Aseri Madan L; Padmanabhan Deepak</div><div class="citationInfo"><span class="NLM_cas:title">International journal of applied & basic medical research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-8</span>
        ISSN:<span class="NLM_cas:issn">2229-516X</span>.
    </div><div class="casAbstract">Isomerism finds its importance in the field of clinical pharmacology and pharmacotherapeutics, as isomers differ in their pharmacokinetic and pharmacodyanmic properties.  Drug isomerism has opened a new era of drug development.  Currently, knowledge of isomerism has helped us in introducing safer and more effective drug alternatives of the newer as well as existing drugs.  Many existing drugs have gone chiral switch i.e., switching from racemic mixture to one of its isomers.  Cetrizine to levocetrizine is one of such examples, where effective and safer drug has been made available.  In this article, we have attempted to review the basic concepts of stereochemistry and chirality and their significance in pharmacotherapeutics.  Various pharmacological aspects such as pharmacokinetic and pharmacodynamic variations resulting out of chirality has been discussed in detail in this article.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMuPJrm3KbPqNc3HjJRoFAfW6udTcc2ebqctnaWeArvrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjjsVymtA%253D%253D&md5=a27d4e2765ceb4c3a86bc79ade0544a1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.4103%2F2229-516X.112233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F2229-516X.112233%26sid%3Dliteratum%253Aachs%26aulast%3DChhabra%26aufirst%3DN.%26aulast%3DAseri%26aufirst%3DM.%2BL.%26aulast%3DPadmanabhan%26aufirst%3DD.%26atitle%3DA%2520review%2520of%2520drug%2520isomerism%2520and%2520its%2520significance%26jtitle%3DInt.%2520J.%2520Appl.%2520Basic.%2520Med.%2520Res.%26date%3D2013%26volume%3D3%26spage%3D16%26epage%3D18%26doi%3D10.4103%2F2229-516X.112233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittinger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keenan, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liau, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span> <span> </span><span class="NLM_article-title">Cancer-associated IDH1 mutations produce 2-hydroxyglutarate</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nature08617</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature08617" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=19935646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlurzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=739-744&author=L.+Dangauthor=D.+W.+Whiteauthor=S.+Grossauthor=B.+D.+Bennettauthor=M.+A.+Bittingerauthor=E.+M.+Driggersauthor=V.+R.+Fantinauthor=H.+G.+Jangauthor=S.+Jinauthor=M.+C.+Keenanauthor=K.+M.+Marksauthor=R.+M.+Prinsauthor=P.+S.+Wardauthor=K.+E.+Yenauthor=L.+M.+Liauauthor=J.+D.+Rabinowitzauthor=L.+C.+Cantleyauthor=C.+B.+Thompsonauthor=M.+G.+Vander%0AHeidenauthor=S.+M.+Su&title=Cancer-associated+IDH1+mutations+produce+2-hydroxyglutarate&doi=10.1038%2Fnature08617"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer-associated IDH1 mutations produce 2-hydroxyglutarate</span></div><div class="casAuthors">Dang, Lenny; White, David W.; Gross, Stefan; Bennett, Bryson D.; Bittinger, Mark A.; Driggers, Edward M.; Fantin, Valeria R.; Jang, Hyun-Gyung; Jin, Shengfang; Keenan, Marie C.; Marks, Kevin M.; Prins, Robert M.; Ward, Patrick S.; Yen, Katharine E.; Liau, Linda M.; Rabinowitz, Joshua D.; Cantley, Lewis C.; Thompson, Craig B.; Vander Heiden, Matthew G.; Su, Shinsan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7274</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in the enzyme cytosolic isocitrate dehydrogenase 1 (IDH1) are a common feature of a major subset of primary human brain cancers.  These mutations occur at a single amino acid residue of the IDH1 active site, resulting in loss of the enzyme's ability to catalyze conversion of isocitrate to α-ketoglutarate.  However, only a single copy of the gene is mutated in tumors, raising the possibility that the mutations do not result in a simple loss of function.  Here we show that cancer-assocd. IDH1 mutations result in a new ability of the enzyme to catalyze the NADPH-dependent redn. of α-ketoglutarate to R(-)-2-hydroxyglutarate (2HG).  Structural studies demonstrate that when arginine 132 is mutated to histidine, residues in the active site are shifted to produce structural changes consistent with reduced oxidative decarboxylation of isocitrate and acquisition of the ability to convert α-ketoglutarate to 2HG.  Excess accumulation of 2HG has been shown to lead to an elevated risk of malignant brain tumors in patients with inborn errors of 2HG metab.  Similarly, in human malignant gliomas harbouring IDH1 mutations, we find markedly elevated levels of 2HG.  These data demonstrate that the IDH1 mutations result in prodn. of the onco-metabolite 2HG, and indicate that the excess 2HG which accumulates in vivo contributes to the formation and malignant progression of gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkIgPaD-aavrVg90H21EOLACvtfcHk0lhyEK5tbOseDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlurzE&md5=b8661605276da5cbd92641e45c5b8c87</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fnature08617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08617%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DL.%26aulast%3DWhite%26aufirst%3DD.%2BW.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DBittinger%26aufirst%3DM.%2BA.%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKeenan%26aufirst%3DM.%2BC.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DPrins%26aufirst%3DR.%2BM.%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DYen%26aufirst%3DK.%2BE.%26aulast%3DLiau%26aufirst%3DL.%2BM.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DSu%26aufirst%3DS.%2BM.%26atitle%3DCancer-associated%2520IDH1%2520mutations%2520produce%25202-hydroxyglutarate%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnature08617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wise, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coller, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedvat, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perl, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span> <span> </span><span class="NLM_article-title">The common feature of leukemia-associated IDH1 and IDH2 mutations is a neomorphic enzyme activity converting alpha-ketoglutarate to 2-hydroxyglutarate</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">225</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.01.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.ccr.2010.01.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20171147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=225-234&author=P.+S.+Wardauthor=J.+Patelauthor=D.+R.+Wiseauthor=O.+Abdel-Wahabauthor=B.+D.+Bennettauthor=H.+A.+Collerauthor=J.+R.+Crossauthor=V.+R.+Fantinauthor=C.+V.+Hedvatauthor=A.+E.+Perlauthor=J.+D.+Rabinowitzauthor=M.+Carrollauthor=S.+M.+Suauthor=K.+A.+Sharpauthor=R.+L.+Levineauthor=C.+B.+Thompson&title=The+common+feature+of+leukemia-associated+IDH1+and+IDH2+mutations+is+a+neomorphic+enzyme+activity+converting+alpha-ketoglutarate+to+2-hydroxyglutarate&doi=10.1016%2Fj.ccr.2010.01.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The common feature of leukemia-associated IDH1 and IDH2 mutations is a neomorphic enzyme activity converting α-ketoglutarate to 2-hydroxyglutarate</span></div><div class="casAuthors">Ward, Patrick S.; Patel, Jay; Wise, David R.; Abdel-Wahab, Omar; Bennett, Bryson D.; Coller, Hilary A.; Cross, Justin R.; Fantin, Valeria R.; Hedvat, Cyrus V.; Perl, Alexander E.; Rabinowitz, Joshua D.; Carroll, Martin; Su, Shinsan M.; Sharp, Kim A.; Levine, Ross L.; Thompson, Craig B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-234</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The somatic mutations in cytosolic isocitrate dehydrogenase 1 (IDH1) obsd. in gliomas can lead to the prodn. of 2-hydroxyglutarate (2HG).  Here, the authors report that tumor 2HG is elevated in a high percentage of patients with cytogenetically normal acute myeloid leukemia (AML).  Surprisingly, less than half of cases with elevated 2HG possessed IDH1 mutations.  The remaining cases with elevated 2HG had mutations in IDH2, the mitochondrial homolog of IDH1.  These data demonstrate that a shared feature of all cancer-assocd. IDH mutations is prodn. of the oncometabolite 2HG.  Furthermore, AML patients with IDH mutations display a significantly reduced no. of other well characterized AML-assocd. mutations and/or assocd. chromosomal abnormalities, potentially implicating IDH mutation in a distinct mechanism of AML pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppKH_IGRymibVg90H21EOLACvtfcHk0lhyEK5tbOseDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWqtLg%253D&md5=cf235f63ccd050ff3b121b94d3c49b9e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.01.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.01.020%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DWise%26aufirst%3DD.%2BR.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DBennett%26aufirst%3DB.%2BD.%26aulast%3DColler%26aufirst%3DH.%2BA.%26aulast%3DCross%26aufirst%3DJ.%2BR.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DHedvat%26aufirst%3DC.%2BV.%26aulast%3DPerl%26aufirst%3DA.%2BE.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26aulast%3DCarroll%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DS.%2BM.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26atitle%3DThe%2520common%2520feature%2520of%2520leukemia-associated%2520IDH1%2520and%2520IDH2%2520mutations%2520is%2520a%2520neomorphic%2520enzyme%2520activity%2520converting%2520alpha-ketoglutarate%2520to%25202-hydroxyglutarate%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D225%26epage%3D234%26doi%3D10.1016%2Fj.ccr.2010.01.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span> <span> </span><span class="NLM_article-title">Oncometabolite 2-hydroxyglutarate is a competitive inhibitor of alpha-ketoglutarate-dependent dioxygenases</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">30</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.ccr.2010.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21251613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpvValsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=17-30&author=W.+Xuauthor=H.+Yangauthor=Y.+Liuauthor=Y.+Yangauthor=P.+Wangauthor=S.+H.+Kimauthor=S.+Itoauthor=C.+Yangauthor=P.+Wangauthor=M.+T.+Xiaoauthor=L.+X.+Liuauthor=W.+Q.+Jiangauthor=J.+Liuauthor=J.+Y.+Zhangauthor=B.+Wangauthor=S.+Fryeauthor=Y.+Zhangauthor=Y.+H.+Xuauthor=Q.+Y.+Leiauthor=K.+L.+Guanauthor=S.+M.+Zhaoauthor=Y.+Xiong&title=Oncometabolite+2-hydroxyglutarate+is+a+competitive+inhibitor+of+alpha-ketoglutarate-dependent+dioxygenases&doi=10.1016%2Fj.ccr.2010.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Oncometabolite 2-Hydroxyglutarate Is a Competitive Inhibitor of α-Ketoglutarate-Dependent Dioxygenases</span></div><div class="casAuthors">Xu, Wei; Yang, Hui; Liu, Ying; Yang, Ying; Wang, Ping; Kim, Se-Hee; Ito, Shinsuke; Yang, Chen; Wang, Pu; Xiao, Meng-Tao; Liu, Li-xia; Jiang, Wen-qing; Liu, Jing; Zhang, Jin-ye; Wang, Bin; Frye, Stephen; Zhang, Yi; Xu, Yan-hui; Lei, Qun-ying; Guan, Kun-Liang; Zhao, Shi-min; Xiong, Yue</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-30</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: IDH1 and IDH2 mutations occur frequently in gliomas and acute myeloid leukemia, leading to simultaneous loss and gain of activities in the prodn. of α-ketoglutarate (α-KG) and 2-hydroxyglutarate (2-HG), resp.  Here we demonstrate that 2-HG is a competitive inhibitor of multiple α-KG-dependent dioxygenases, including histone demethylases and the TET family of 5-methylcytosine (5mC) hydroxylases. 2-HG occupies the same space as α-KG does in the active site of histone demethylases.  Ectopic expression of tumor-derived IDH1 and IDH2 mutants inhibits histone demethylation and 5mC hydroxylation.  In glioma, IDH1 mutations are assocd. with increased histone methylation and decreased 5-hydroxylmethylcytosine (5hmC).  Hence, tumor-derived IDH1 and IDH2 mutations reduce α-KG and accumulate an α-KG antagonist, 2-HG, leading to genome-wide histone and DNA methylation alterations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpry-WZlrav7Vg90H21EOLACvtfcHk0lhyEK5tbOseDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpvValsQ%253D%253D&md5=27f1e2d50b9dc1dd30bcc576d6318662</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DM.%2BT.%26aulast%3DLiu%26aufirst%3DL.%2BX.%26aulast%3DJiang%26aufirst%3DW.%2BQ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DFrye%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%2BH.%26aulast%3DLei%26aufirst%3DQ.%2BY.%26aulast%3DGuan%26aufirst%3DK.%2BL.%26aulast%3DZhao%26aufirst%3DS.%2BM.%26aulast%3DXiong%26aufirst%3DY.%26atitle%3DOncometabolite%25202-hydroxyglutarate%2520is%2520a%2520competitive%2520inhibitor%2520of%2520alpha-ketoglutarate-dependent%2520dioxygenases%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D19%26spage%3D17%26epage%3D30%26doi%3D10.1016%2Fj.ccr.2010.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeoh, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillringhaus, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagg, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woon, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifton, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klose, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claridge, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span> <span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1038/embor.2011.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fembor.2011.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=21460794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-469&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases&doi=10.1038%2Fembor.2011.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span></div><div class="casAuthors">Chowdhury, Rasheduzzaman; Yeoh, Kar-Kheng; Tian, Ya-Min; Hillringhaus, Lars; Bagg, Eleanor A.; Rose, Nathan R.; Leung, Ivanhoe K.-H.; Li, Xuan-S.; Woon, Esther C.-Y.; Yang, Ming; McDonough, Michael A.; King, Oliver N.; Clifton, Ian J.; Klose, Robert J.; Claridge, Timothy D. W.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenases (IDHs) have a gain-of-function effect leading to R(-)-2-hydroxyglutarate (R-2HG) accumulation.  By using biochem., structural and cellular assays, we show that either or both R- and S-2HG inhibit 2-oxoglutarate (2OG)-dependent oxygenases with varying potencies.  Half-maximal inhibitory concn. (IC50) values for the R-form of 2HG varied from approx. 25 μM for the histone N.vepsiln.-lysine demethylase JMJD2A to more than 5 mM for the hypoxia-inducible factor (HIF) prolyl hydroxylase.  The results indicate that candidate oncogenic pathways in IDH-assocd. malignancy should include those that are regulated by other 2OG oxygenases than HIF hydroxylases, in particular those involving the regulation of histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHxuB6sKkTbVg90H21EOLACvtfcHk0lgaGd1feOqWGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D&md5=1b862b1337346af6b796093cea34056f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.43%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fembor.2011.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramkissoon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhaak, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">Transformation by the (R)-enantiomer of 2-hydroxyglutarate linked to EGLN activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1038/nature10898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature10898" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=484-488&author=P.+Koivunenauthor=S.+Leeauthor=C.+G.+Duncanauthor=G.+Lopezauthor=G.+Luauthor=S.+Ramkissoonauthor=J.+A.+Losmanauthor=P.+Joensuuauthor=U.+Bergmannauthor=S.+Grossauthor=J.+Travinsauthor=S.+Weissauthor=R.+Looperauthor=K.+L.+Ligonauthor=R.+G.+Verhaakauthor=H.+Yanauthor=W.+G.+Kaelin&title=Transformation+by+the+%28R%29-enantiomer+of+2-hydroxyglutarate+linked+to+EGLN+activation&doi=10.1038%2Fnature10898"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnature10898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10898%26sid%3Dliteratum%253Aachs%26aulast%3DKoivunen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DDuncan%26aufirst%3DC.%2BG.%26aulast%3DLopez%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DRamkissoon%26aufirst%3DS.%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DJoensuu%26aufirst%3DP.%26aulast%3DBergmann%26aufirst%3DU.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DS.%26aulast%3DLooper%26aufirst%3DR.%26aulast%3DLigon%26aufirst%3DK.%2BL.%26aulast%3DVerhaak%26aufirst%3DR.%2BG.%26aulast%3DYan%26aufirst%3DH.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DTransformation%2520by%2520the%2520%2528R%2529-enantiomer%2520of%25202-hydroxyglutarate%2520linked%2520to%2520EGLN%2520activation%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D484%26epage%3D488%26doi%3D10.1038%2Fnature10898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Losman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr</span></span> <span> </span><span class="NLM_article-title">(R)-2-hydroxyglutarate is sufficient to promote leukemogenesis and its effects are reversible</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>339</i></span>,  <span class="NLM_fpage">1621</span>– <span class="NLM_lpage">1625</span>, <span class="refDoi"> DOI: 10.1126/science.1231677</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1126%2Fscience.1231677" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23393090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1621-1625&author=J.+A.+Losmanauthor=R.+E.+Looperauthor=P.+Koivunenauthor=S.+Leeauthor=R.+K.+Schneiderauthor=C.+McMahonauthor=G.+S.+Cowleyauthor=D.+E.+Rootauthor=B.+L.+Ebertauthor=W.+G.+Kaelin&title=%28R%29-2-hydroxyglutarate+is+sufficient+to+promote+leukemogenesis+and+its+effects+are+reversible&doi=10.1126%2Fscience.1231677"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-2-Hydroxyglutarate is sufficient to promote leukemogenesis and its effects are reversible</span></div><div class="casAuthors">Losman, Julie-Aurore; Looper, Ryan E.; Koivunen, Peppi; Lee, Sungwoo; Schneider, Rebekka K.; McMahon, Christine; Cowley, Glenn S.; Root, David E.; Ebert, Benjamin L.; Kaelin, William G., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6127</span>),
    <span class="NLM_cas:pages">1621-1625</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mutations in IDH1 and IDH2, the genes coding for isocitrate dehydrogenases 1 and 2, are common in several human cancers, including leukemias, and result in overprodn. of the (R)-enantiomer of 2-hydroxyglutarate [(R)-2HG].  Elucidation of the role of IDH mutations and (R)-2HG in leukemogenesis has been hampered by a lack of appropriate cell-based models.  Here, we show that a canonical IDH1 mutant, IDH1 R132H, promotes cytokine independence and blocks differentiation in hematopoietic cells.  These effects can be recapitulated by (R)-2HG, but not (S)-2HG, despite the fact that (S)-2HG more potently inhibits enzymes, such as the 5'-methylcytosine hydroxylase TET2, that have previously been linked to the pathogenesis of IDH mutant tumors.  We provide evidence that this paradox relates to the ability of (S)-2HG, but not (R)-2HG, to inhibit the EglN prolyl hydroxylases.  Addnl., we show that transformation by (R)-2HG is reversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTDGHR9SE0P7Vg90H21EOLACvtfcHk0lgaGd1feOqWGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOhsLc%253D&md5=1bb23bea1550fd47e479f6d3833741ee</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1126%2Fscience.1231677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1231677%26sid%3Dliteratum%253Aachs%26aulast%3DLosman%26aufirst%3DJ.%2BA.%26aulast%3DLooper%26aufirst%3DR.%2BE.%26aulast%3DKoivunen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DR.%2BK.%26aulast%3DMcMahon%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3D%2528R%2529-2-hydroxyglutarate%2520is%2520sufficient%2520to%2520promote%2520leukemogenesis%2520and%2520its%2520effects%2520are%2520reversible%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D1621%26epage%3D1625%26doi%3D10.1126%2Fscience.1231677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nachtergaele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibbe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferchen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plas, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">R-2HG exhibits anti-tumor activity by targeting FTO/m(6)A/MYC/CEBPA signaling</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.cell.2017.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=29249359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=90-105&author=R.+Suauthor=L.+Dongauthor=C.+Liauthor=S.+Nachtergaeleauthor=M.+Wunderlichauthor=Y.+Qingauthor=X.+Dengauthor=Y.+Wangauthor=X.+Wengauthor=C.+Huauthor=M.+Yuauthor=J.+Skibbeauthor=Q.+Daiauthor=D.+Zouauthor=T.+Wuauthor=K.+Yuauthor=H.+Wengauthor=H.+Huangauthor=K.+Ferchenauthor=X.+Qinauthor=B.+Zhangauthor=J.+Qiauthor=A.+T.+Sasakiauthor=D.+R.+Plasauthor=J.+E.+Bradnerauthor=M.+Weiauthor=G.+Marcucciauthor=X.+Jiangauthor=J.+C.+Mulloyauthor=J.+Jinauthor=C.+Heauthor=J.+Chen&title=R-2HG+exhibits+anti-tumor+activity+by+targeting+FTO%2Fm%286%29A%2FMYC%2FCEBPA+signaling&doi=10.1016%2Fj.cell.2017.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">R-2HG Exhibits Anti-tumor Activity by Targeting FTO/m6A/MYC/CEBPA Signaling</span></div><div class="casAuthors">Su, Rui; Dong, Lei; Li, Chenying; Nachtergaele, Sigrid; Wunderlich, Mark; Qing, Ying; Deng, Xiaolan; Wang, Yungui; Weng, Xiaocheng; Hu, Chao; Yu, Mengxia; Skibbe, Jennifer; Dai, Qing; Zou, Dongling; Wu, Tong; Yu, Kangkang; Weng, Hengyou; Huang, Huilin; Ferchen, Kyle; Qin, Xi; Zhang, Bin; Qi, Jun; Sasaki, Atsuo T.; Plas, David R.; Bradner, James E.; Wei, Minjie; Marcucci, Guido; Jiang, Xi; Mulloy, James C.; Jin, Jie; He, Chuan; Chen, Jianjun</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">90-105.e23</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">R-2-hydroxyglutarate (R-2HG), produced at high levels by mutant isocitrate dehydrogenase 1/2 (IDH1/2) enzymes, was reported as an oncometabolite.  We show here that R-2HG also exerts a broad anti-leukemic activity in vitro and in vivo by inhibiting leukemia cell proliferation/viability and by promoting cell-cycle arrest and apoptosis.  Mechanistically, R-2HG inhibits fat mass and obesity-assocd. protein (FTO) activity, thereby increasing global N6-methyladenosine (m6A) RNA modification in R-2HG-sensitive leukemia cells, which in turn decreases the stability of MYC/CEBPA transcripts, leading to the suppression of relevant pathways.  Ectopically expressed mutant IDH1 and S-2HG recapitulate the effects of R-2HG.  High levels of FTO sensitize leukemic cells to R-2HG, whereas hyperactivation of MYC signaling confers resistance that can be reversed by the inhibition of MYC signaling.  R-2HG also displays anti-tumor activity in glioma.  Collectively, while R-2HG accumulated in IDH1/2 mutant cancers contributes to cancer initiation, our work demonstrates anti-tumor effects of 2HG in inhibiting proliferation/survival of FTO-high cancer cells via targeting FTO/m6A/MYC/CEBPA signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMervd4vttCLVg90H21EOLACvtfcHk0liAj2WN-nB6jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2js7rI&md5=ffb994f5c202903fd5714c042adeb869</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DR.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DNachtergaele%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DQing%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWeng%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DSkibbe%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZou%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DFerchen%26aufirst%3DK.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DPlas%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DR-2HG%2520exhibits%2520anti-tumor%2520activity%2520by%2520targeting%2520FTO%252Fm%25286%2529A%252FMYC%252FCEBPA%2520signaling%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D90%26epage%3D105%26doi%3D10.1016%2Fj.cell.2017.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and α-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm α-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was ∼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a ∼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0liAj2WN-nB6jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausinger, R. P.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanisms of Fe(II)- and 2-oxoglutarate-dependent oxygenases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">20702</span>– <span class="NLM_lpage">20711</span>, <span class="refDoi"> DOI: 10.1074/jbc.R115.648691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.R115.648691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=26152721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOrt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20702-20711&author=S.+Martinezauthor=R.+P.+Hausinger&title=Catalytic+mechanisms+of+Fe%28II%29-+and+2-oxoglutarate-dependent+oxygenases&doi=10.1074%2Fjbc.R115.648691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic Mechanisms of Fe(II)- and 2-Oxoglutarate-dependent Oxygenases</span></div><div class="casAuthors">Martinez, Salette; Hausinger, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">20702-20711</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Mononuclear non-heme Fe(II)- and 2-oxoglutarate (2OG)-dependent oxygenases comprise a large family of enzymes that utilize an Fe(IV)-oxo intermediate to initiate diverse oxidative transformations with important biol. roles.  Here, four of the major types of Fe(II)/2OG-dependent reactions are detailed: hydroxylation, halogenation, ring formation, and desatn.  In addn., an atypical epimerization reaction is described.  Studies identifying several key intermediates in catalysis are concisely summarized, and the proposed mechanisms are explained.  In addn., a variety of other transformations catalyzed by selected family members are briefly described to further highlight the chem. versatility of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-tW8bZZFBybVg90H21EOLACvtfcHk0liAj2WN-nB6jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOrt77E&md5=a31238fa27dcb8d88adb19292ad44a4d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R115.648691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R115.648691%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DS.%26aulast%3DHausinger%26aufirst%3DR.%2BP.%26atitle%3DCatalytic%2520mechanisms%2520of%2520Fe%2528II%2529-%2520and%25202-oxoglutarate-dependent%2520oxygenases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20702%26epage%3D20711%26doi%3D10.1074%2Fjbc.R115.648691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudlebusch, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">E1</span>– <span class="NLM_lpage">2</span>, <span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3D2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2974</span>– <span class="NLM_lpage">2981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1016%2Fj.bmcl.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=28512031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2974-2981&author=J.+Liangauthor=S.+Labadieauthor=B.+Zhangauthor=D.+F.+Ortwineauthor=S.+Patelauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=T.+Mauerauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=L.+Anauthor=Y.+Ranauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=From+a+novel+HTS+hit+to+potent%2C+selective%2C+and+orally+bioavailable+KDM5+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span></div><div class="casAuthors">Liang, Jun; Labadie, Sharada; Zhang, Birong; Ortwine, Daniel F.; Patel, Snahel; Vinogradova, Maia; Kiefer, James R.; Mauer, Till; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; An, Le; Ran, Yingqing; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2974-2981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screening (HTS) of the Genentech/Roche library identified a novel, uncharged scaffold as a KDM5A inhibitor.  Lacking insight into the binding mode, initial attempts to improve inhibitor potency failed to improve potency, and synthesis of analogs was further hampered by the presence of a C-C bond between the pyrrolidine and pyridine.  Replacing this with a C-N bond significantly simplified synthesis, yielding pyrazole analog, [3-(4-bromo-1H-pyrazol-1-yl)-1-pyrrolidinyl][5-(1-methylethyl)-1H-pyrazol-3-yl]methanone (35), of which the authors obtained a co-crystal structure with KDM5A.  Using structure-based design approach, the authors identified, N-[(3R)-1-[[5-(1-methylethyl)-1H-pyrazol-3-yl]carbonyl]-3-pyrrolidinyl]cyclopropanecarboxamide (50), with improved biochem., cell potency and reduced MW and lower lipophilicity (Log D) compared with the original hit.  Furthermore, 50 showed lower clearance than [5-(1-methylethyl)-1H-pyrazol-3-yl][(3S)-3-[6-methyl-4-(1-methyl-1H-pyrazol-4-yl)-2-pyridinyl]-1-pyrrolidinyl]methanone (9) in mice.  In combination with its remarkably low plasma protein binding (PPB) in mice (40%), oral dosing of 50 at 5 mg/kg resulted in unbound Cmax ∼2-fold of its cell potency (PC9 H3K4Me3 0.96 μM), meeting the authors' criteria for an in vivo tool compd. from a new scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpksjXQBrVYsrVg90H21EOLACvtfcHk0ljNbewrccJEFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D&md5=04bee4e29775202fb1882adeccb8e722</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMauer%26aufirst%3DT.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DFrom%2520a%2520novel%2520HTS%2520hit%2520to%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2974%26epage%3D2981%26doi%3D10.1016%2Fj.bmcl.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span>; <span class="NLM_string-name">Gale, M.</span>; <span class="NLM_string-name">Yan, Q.</span>; <span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">The molecular basis of histone demethylation</span>. In  <i>DNA and Histone Methylation As Cancer Targets</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span>, <span class="NLM_string-name">Tsukada, Y.-i.</span></span>, Eds.; Cancer Drug Discovery and Development; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">219</span>, DOI: <span class="refDoi"> DOI: 10.1007/978-3-319-59786-7_7</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1007%2F978-3-319-59786-7_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=151-219&author=J.+R.+Horton&author=M.+Gale&author=Q.+Yan&author=X.+Chengauthor=A.+Kaneda&author=Y.-i.+Tsukada&title=DNA+and+Histone+Methylation+As+Cancer+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520molecular%2520basis%2520of%2520histone%2520demethylation%26btitle%3DDNA%2520and%2520Histone%2520Methylation%2520As%2520Cancer%2520Targets%26aulast%3DKaneda%26aufirst%3DA.%26pub%3DSpringer%26date%3D2017%26spage%3D151%26epage%3D219%26doi%3D10.1007%2F978-3-319-59786-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 1. 3-amino-4-pyridine carboxylate derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1357-1369&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=H.+Dialloauthor=C.+Douaultauthor=G.+Drewesauthor=R.+Eagleauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=J.+Liddleauthor=J.+Mosleyauthor=M.+Muelbaierauthor=R.+Randleauthor=I.+Riojaauthor=A.+Ruegerauthor=G.+A.+Sealauthor=R.+J.+Sheppardauthor=O.+Singhauthor=J.+Taylorauthor=P.+Thomasauthor=D.+Thomsonauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+1.+3-amino-4-pyridine+carboxylate+derivatives&doi=10.1021%2Facs.jmedchem.5b01537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01537%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRandle%26aufirst%3DR.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThomson%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25201.%25203-amino-4-pyridine%2520carboxylate%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1357%26epage%3D1369%26doi%3D10.1021%2Facs.jmedchem.5b01537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewski, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Multiparametric scaling of diffraction intensities</span>. <i>Acta Crystallogr., Sect. A: Found. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">234</span>, <span class="refDoi"> DOI: 10.1107/S0108767303005488</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0108767303005488" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=12714773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFChtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=228-234&author=Z.+Otwinowskiauthor=D.+Borekauthor=W.+Majewskiauthor=W.+Minor&title=Multiparametric+scaling+of+diffraction+intensities&doi=10.1107%2FS0108767303005488"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Multiparametric scaling of diffraction intensities</span></div><div class="casAuthors">Otwinowski, Zbyszek; Borek, Dominika; Majewski, Wladyslaw; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section A: Foundations of Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">A59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">228-234</span>CODEN:
                <span class="NLM_cas:coden">ACACEQ</span>;
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">A novel and general approach to scaling diffraction intensities is presented.  The method minimizes the disagreement among multiple measurements of symmetry-related reflections using a stable refinement procedure.  The scale factors are described by a flexible exponential function that allows different scaling corrections to be chosen and combined according to the needs of the expt.  The scaling model presented here includes: scale and temp. factor per batch of data; temp. factor as a continuous function of the radiation dose; absorption in the crystal; uneven exposure within a single diffraction image; and corrections for phenomena that depend on the diffraction peak position on the detector.  This scaling model can be extended to include addnl. corrections for various instrumental and data-collection problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobnTSFjuXFu7Vg90H21EOLACvtfcHk0lgTn6RhOg8O_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFChtbg%253D&md5=81ef008e4c8fa87b2863552558ed18ad</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0108767303005488&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767303005488%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DMajewski%26aufirst%3DW.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DMultiparametric%2520scaling%2520of%2520diffraction%2520intensities%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D228%26epage%3D234%26doi%3D10.1107%2FS0108767303005488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgTn6RhOg8O_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgTn6RhOg8O_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0liGUfV5C_6LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liGUfV5C_6LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tackett, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Identification of small molecule inhibitors of Jumonji AT-rich interactive domain 1B (JARID1B) histone demethylase by a sensitive high throughput screen</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">9408</span>– <span class="NLM_lpage">9417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.419861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=10.1074%2Fjbc.M112.419861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=23408432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;key=1%3ACAS%3A528%3ADC%252BC3sXkvF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=9408-9417&author=J.+Sayeghauthor=J.+Caoauthor=M.+R.+Zouauthor=A.+Moralesauthor=L.+P.+Blairauthor=M.+Norciaauthor=D.+Hoyerauthor=A.+J.+Tackettauthor=J.+S.+Merkelauthor=Q.+Yan&title=Identification+of+small+molecule+inhibitors+of+Jumonji+AT-rich+interactive+domain+1B+%28JARID1B%29+histone+demethylase+by+a+sensitive+high+throughput+screen&doi=10.1074%2Fjbc.M112.419861"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Small Molecule Inhibitors of Jumonji AT-rich Interactive Domain 1B (JARID1B) Histone Demethylase by a Sensitive High Throughput Screen</span></div><div class="casAuthors">Sayegh, Joyce; Cao, Jian; Zou, Mike Ran; Morales, Alfonso; Blair, Lauren P.; Norcia, Michael; Hoyer, Denton; Tackett, Alan J.; Merkel, Jane S.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9408-9417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">JARID1B (also known as KDM5B or PLU1) is a member of the JARID1 family of histone lysine demethylases responsible for the demethylation of trimethylated lysine 27 in histone H3 (H3K4me3), a mark for actively transcribed genes.  JARID1B is overexpressed in several cancers, including breast cancer, prostate cancer, and lung cancer.  In addn., JARID1B is required for mammary tumor formation in syngeneic or xenograft mouse models.  JARID1B-expressing melanoma cells are assocd. with increased self-renewal character.  Therefore, JARID1B represents an attractive target for cancer therapy.  Here we characterized JARID1B using a homogeneous luminescence-based demethylase assay.  We then conducted a high throughput screen of over 15,000 small mols. to identify inhibitors of JARID1B.  From this screen, we identified several known JmjC histone demethylase inhibitors, including 2,4-pyridinedicarboxylic acid and catechols.  More importantly, we identified several novel inhibitors, including 2-4(4-methylphenyl)-1,2-benzisothiazol-3(2H)-one (PBIT), which inhibits JARID1B with an IC50 of about 3 μmin vitro.  Consistent with this, PBIT treatment inhibited removal of H3K4me3 by JARID1B in cells.  Furthermore, this compd. inhibited proliferation of cells expressing higher levels of JARID1B.  These results suggest that this novel small mol. inhibitor is a lead compd. that can be further optimized for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiVU6VegMXeLVg90H21EOLACvtfcHk0liGUfV5C_6LUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXkvF2qsLw%253D&md5=f7d20eaea77373475d630456828defc1</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.419861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.419861%26sid%3Dliteratum%253Aachs%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DMorales%26aufirst%3DA.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DNorcia%26aufirst%3DM.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DMerkel%26aufirst%3DJ.%2BS.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DIdentification%2520of%2520small%2520molecule%2520inhibitors%2520of%2520Jumonji%2520AT-rich%2520interactive%2520domain%25201B%2520%2528JARID1B%2529%2520histone%2520demethylase%2520by%2520a%2520sensitive%2520high%2520throughput%2520screen%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D9408%26epage%3D9417%26doi%3D10.1074%2Fjbc.M112.419861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IVB','PDB','5IVB'); return false;">PDB: 5IVB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY','PDB','6BGY'); return false;">PDB: 6BGY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU','PDB','6BGU'); return false;">PDB: 6BGU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV','PDB','6BGV'); return false;">PDB: 6BGV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0','PDB','6BH0'); return false;">PDB: 6BH0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1','PDB','6BH1'); return false;">PDB: 6BH1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW','PDB','6BGW'); return false;">PDB: 6BGW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX','PDB','6BGX'); return false;">PDB: 6BGX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ','PDB','6BGZ'); return false;">PDB: 6BGZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5','PDB','6BH5'); return false;">PDB: 6BH5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V9T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5V9T','PDB','5V9T'); return false;">PDB: 5V9T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ISL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ISL','PDB','5ISL'); return false;">PDB: 5ISL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2','PDB','6BH2'); return false;">PDB: 6BH2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF','PDB','5IWF'); return false;">PDB: 5IWF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CEH','PDB','5CEH'); return false;">PDB: 5CEH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E6H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E6H','PDB','5E6H'); return false;">PDB: 5E6H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH3','PDB','6BH3'); return false;">PDB: 6BH3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00261">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71264"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00261">10.1021/acs.jmedchem.8b00261</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figures S1–S7, and Tables S1–S2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_001.pdf">jm8b00261_si_001.pdf (4.08 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00261/suppl_file/jm8b00261_si_002.csv">jm8b00261_si_002.csv (0.66 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors of KDM5A in complex with <b>N9</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGU">6BGU</a>), <b>N40</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGV">6BGV</a>), <b>N41</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGW">6BGW</a>), <b>N42</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGX">6BGX</a>), <b>N46</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGY">6BGY</a>), <b>N47</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BGZ">6BGZ</a>), <b>N48</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH5">6BH5</a>), <b>N51</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH0">6BH0</a>), <b>N52</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH1">6BH1</a>), <b>N54</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH2">6BH2</a>), <b>N55</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BH3">6BH3</a>), and <b>KDOAM-25</b> (code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IWF">5IWF</a>) have been deposited in the Protein Data Bank, and authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00261%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00261" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                11MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bb4bc9283ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
