
<html lang="en"     class="pb-page"  data-request-id="de2eeff1-dca8-4074-b1cf-1d22594aea64"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01019;issue:issue:10.1021/jmcmar.2017.60.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship" /></meta><meta name="dc.Creator" content="Paola  Vianello" /></meta><meta name="dc.Creator" content="Luca  Sartori" /></meta><meta name="dc.Creator" content="Federica  Amigoni" /></meta><meta name="dc.Creator" content="Anna  Cappa" /></meta><meta name="dc.Creator" content="Giovanni  Fagá" /></meta><meta name="dc.Creator" content="Raimondo  Fattori" /></meta><meta name="dc.Creator" content="Elena  Legnaghi" /></meta><meta name="dc.Creator" content="Giuseppe  Ciossani" /></meta><meta name="dc.Creator" content="Andrea  Mattevi" /></meta><meta name="dc.Creator" content="Giuseppe  Meroni" /></meta><meta name="dc.Creator" content="Loris  Moretti" /></meta><meta name="dc.Creator" content="Valentina  Cecatiello" /></meta><meta name="dc.Creator" content="Sebastiano  Pasqualato" /></meta><meta name="dc.Creator" content="Alessia  Romussi" /></meta><meta name="dc.Creator" content="Florian  Thaler" /></meta><meta name="dc.Creator" content="Paolo  Trifiró" /></meta><meta name="dc.Creator" content="Manuela  Villa" /></meta><meta name="dc.Creator" content="Oronza A.  Botrugno" /></meta><meta name="dc.Creator" content="Paola  Dessanti" /></meta><meta name="dc.Creator" content="Saverio  Minucci" /></meta><meta name="dc.Creator" content="Stefania  Vultaggio" /></meta><meta name="dc.Creator" content="Elisa  Zagarrí" /></meta><meta name="dc.Creator" content="Mario  Varasi" /></meta><meta name="dc.Creator" content="Ciro  Mercurio" /></meta><meta name="dc.Description" content="The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1). KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development..." /></meta><meta name="Description" content="The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1). KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 27, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01019" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01019" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01019" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01019" /></link>
        
    
    

<title>Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01019" /></meta><meta property="og:title" content="Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0023.jpeg" /></meta><meta property="og:description" content="The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1). KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development of novel cancer therapeutics. We have previously described (Part 1, DOI 10.1021.acs.jmedchem.6b01018) the identification of thieno[3,2-b]pyrrole-5-carboxamides as novel reversible inhibitors of KDM1A, whose preliminary exploration resulted in compound 2 with biochemical IC50 = 160 nM. We now report the structure-guided optimization of this chemical series based on multiple ligand/KDM1A-CoRest cocrystal structures, which led to several extremely potent inhibitors. In particular, compounds 46, 49, and 50 showed single-digit nanomolar IC50 values for in vitro inhibition of KDM1A, with high selectivity in secondary assays. In THP-1 cells, these compounds transcriptionally affected the expression of genes regulated by KDM1A such as CD14, CD11b, and CD86. Moreover, 49 and 50 showed a remarkable anticlonogenic cell growth effect on MLL-AF9 human leukemia cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01019"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01019">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01019&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01019&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01019&amp;href=/doi/10.1021/acs.jmedchem.6b01019" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1693-1715</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01018" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01115" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Vianello">Paola Vianello</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9100-2711" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luca++Sartori">Luca Sartori</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Federica++Amigoni">Federica Amigoni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Cappa">Anna Cappa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giovanni++Fag%C3%A1">Giovanni Fagá</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Raimondo++Fattori">Raimondo Fattori</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Legnaghi">Elena Legnaghi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Ciossani">Giuseppe Ciossani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Mattevi">Andrea Mattevi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9523-7128" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Meroni">Giuseppe Meroni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Loris++Moretti">Loris Moretti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9851-2965" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Valentina++Cecatiello">Valentina Cecatiello</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sebastiano++Pasqualato">Sebastiano Pasqualato</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessia++Romussi">Alessia Romussi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florian++Thaler">Florian Thaler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Trifir%C3%B3">Paolo Trifiró</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Manuela++Villa">Manuela Villa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oronza+A.++Botrugno">Oronza A. Botrugno</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Dessanti">Paola Dessanti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Saverio++Minucci">Saverio Minucci</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Vultaggio">Stefania Vultaggio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elisa++Zagarr%C3%AD">Elisa Zagarrí</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mario++Varasi">Mario Varasi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ciro++Mercurio">Ciro Mercurio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Biology and Biotechnology, University of Pavia, Via Ferrata 1, 27100 Pavia, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Crystallography Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">IFOM- The FIRC Institute of Molecular Oncology Foundation, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biosciences, University of Milan, Via Celoria 26, 20133 Milano, Italy</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Genextra Group, DAC s.r.l., Via Adamello 16, 20139 Milano, Italy</span></div><div class="corresp-info"><strong>*</strong>For P.V.: phone, +39 02 9437 5030; e-mail, <a href="/cdn-cgi/l/email-protection#e99988868588c79f8088878c858586a9808f8684c78c9c"><span class="__cf_email__" data-cfemail="c8b8a9a7a4a9e6bea1a9a6ada4a4a788a1aea7a5e6adbd">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For L.S.: phone, +39 02 9437 5121; e-mail, <a href="/cdn-cgi/l/email-protection#573b223436792436252338253e173e31383a793222"><span class="__cf_email__" data-cfemail="e488918785ca978596908b968da48d828b89ca8191">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01019&amp;href=/doi/10.1021%2Facs.jmedchem.6b01019" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1693–1715</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 10, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 July 2016</li><li><span class="item_label"><b>Published</b> online</span>27 February 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 March 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01019" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01019</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1693%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPaola%2BVianello%252C%2BLuca%2BSartori%252C%2BFederica%2BAmigoni%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D5%26contentID%3Dacs.jmedchem.6b01019%26title%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2Bas%2BNew%2BReversible%2BInhibitors%2Bof%2BHistone%2BLysine%2BDemethylase%2BKDM1A%252FLSD1.%2BPart%2B2%253A%2BStructure-Based%2BDrug%2BDesign%2Band%2BStructure%25E2%2580%2593Activity%2BRelationship%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1715%26publicationDate%3DMarch%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01019"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2585</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">37</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01019" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Vianello&quot;},{&quot;first_name&quot;:&quot;Luca&quot;,&quot;last_name&quot;:&quot;Sartori&quot;},{&quot;first_name&quot;:&quot;Federica&quot;,&quot;last_name&quot;:&quot;Amigoni&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Cappa&quot;},{&quot;first_name&quot;:&quot;Giovanni&quot;,&quot;last_name&quot;:&quot;Fagá&quot;},{&quot;first_name&quot;:&quot;Raimondo&quot;,&quot;last_name&quot;:&quot;Fattori&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Legnaghi&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Ciossani&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Mattevi&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Meroni&quot;},{&quot;first_name&quot;:&quot;Loris&quot;,&quot;last_name&quot;:&quot;Moretti&quot;},{&quot;first_name&quot;:&quot;Valentina&quot;,&quot;last_name&quot;:&quot;Cecatiello&quot;},{&quot;first_name&quot;:&quot;Sebastiano&quot;,&quot;last_name&quot;:&quot;Pasqualato&quot;},{&quot;first_name&quot;:&quot;Alessia&quot;,&quot;last_name&quot;:&quot;Romussi&quot;},{&quot;first_name&quot;:&quot;Florian&quot;,&quot;last_name&quot;:&quot;Thaler&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Trifiró&quot;},{&quot;first_name&quot;:&quot;Manuela&quot;,&quot;last_name&quot;:&quot;Villa&quot;},{&quot;first_name&quot;:&quot;Oronza&quot;,&quot;last_name&quot;:&quot;A. Botrugno&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Dessanti&quot;},{&quot;first_name&quot;:&quot;Saverio&quot;,&quot;last_name&quot;:&quot;Minucci&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Vultaggio&quot;},{&quot;first_name&quot;:&quot;Elisa&quot;,&quot;last_name&quot;:&quot;Zagarrí&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;Varasi&quot;},{&quot;first_name&quot;:&quot;Ciro&quot;,&quot;last_name&quot;:&quot;Mercurio&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;1693-1715&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01019&quot;},&quot;abstract&quot;:&quot;The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1). KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development of novel cancer therapeutics. We have previously described (Part 1, DOI 10.1021.acs.jmedchem.6b01018) the identification of thieno[3,2-b]pyrrole-5-carboxamides as novel reversible inhibitors of KDM1A, whose preliminary exploration resulted in compound 2 with biochemical IC50 = 160 nM. We now report the structure-guided optimization of this chemical series based on multiple ligand/KDM1A-CoRest cocrystal structures, which led to several extremely potent inhibitors. In particular, compounds 46, 49, and 50 showed single-digit nanomolar IC50 values for in vitro inhibition of KDM1A, with high selectivity in secondary assays. In THP-1 cells, these compounds transcriptionally affected the expression of genes regulated by KDM1A such as CD14, CD11b, and CD86. Moreover, 49 and 50 showed a remarkable anticlonogenic cell growth&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01019&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01019" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01019&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01019" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01019&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01019" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01019&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01019&amp;href=/doi/10.1021/acs.jmedchem.6b01019" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01019" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01019" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26pmid%3D28186757%26genre%3Darticle%26aulast%3DVianello%26date%3D2017%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2Bas%2BNew%2BReversible%2BInhibitors%2Bof%2BHistone%2BLysine%2BDemethylase%2BKDM1A%252FLSD1.%2BPart%2B2%253A%2BStructure-Based%2BDrug%2BDesign%2Band%2BStructure%25E2%2580%2593Activity%2BRelationship%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D5%26spage%3D1693%26epage%3D1715%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/jmcmar.2017.60.issue-5/20170309/jmcmar.2017.60.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1). KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development of novel cancer therapeutics. We have previously described (<a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1, DOI 10.1021.acs.jmedchem.6b01018</a>) the identification of thieno[3,2-<i>b</i>]pyrrole-5-carboxamides as novel reversible inhibitors of KDM1A, whose preliminary exploration resulted in compound <b>2</b> with biochemical IC<sub>50</sub> = 160 nM. We now report the structure-guided optimization of this chemical series based on multiple ligand/KDM1A-CoRest cocrystal structures, which led to several extremely potent inhibitors. In particular, compounds <b>46</b>, <b>49</b>, and <b>50</b> showed single-digit nanomolar IC<sub>50</sub> values for in vitro inhibition of KDM1A, with high selectivity in secondary assays. In THP-1 cells, these compounds transcriptionally affected the expression of genes regulated by KDM1A such as CD14, CD11b, and CD86. Moreover, <b>49</b> and <b>50</b> showed a remarkable anticlonogenic cell growth effect on MLL-AF9 human leukemia cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44647" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44647" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aberrant post-translational modifications of chromatin have been associated with the onset and progression of several human diseases, including cancer. In particular, the <i>N</i>-methylation status of histones is one of the crucial factors that regulate transcription and eventually control enhancement or repression of gene expression. Histones can be methylated on lysine and arginine residues by histone methyl transferase proteins. Lysines demethylation is specifically catalyzed by two families of enzymes: (i) lysine specific demethylases (KDM1). which are flavine adenine dinucleotide (FAD) dependent proteins related to monoamine oxidases (MAOs) and catalyze methyl removal from mono- and dimethylated histones H3K9 and H3K4, and (ii) JmjC-domain containing histone demethylases (KDM2, KDM3, KDM4, KDM5, JARID2, KDM6, KDM7, KDM8), which utilize α-ketoglutarate (2OG) and iron (Fe<sup>(II)</sup>) as a cofactor, to catalyze demethylation of mono-, di-, and trimethylated lysines at multiple sites.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Two human FAD-dependent demethylases have been identified: LSD1, also known as KDM1A,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and LSD2, also known as KDM1B.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> KDM1A is highly expressed in different cancer types, including acute myeloid leukemia,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> prostate cancer,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> ER-negative breast cancer,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and neuroblastoma<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and often its overexpression correlates with poor prognosis.</div><div class="NLM_p">After the discovery of tranylcypromine as a weak irreversible KDM1A inhibitor,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> an extensive effort has been dedicated to the field.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(10-18)</a> Three irreversible tranylcypromine derivatives from GSK, Oryzon (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>: GSK2879552,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> ORY1001,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> depicted as <b>55</b>) and Incyte (INC059872<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> undisclosed structure) are now in clinical stages. Conversely, reports on the preclinical development of low molecular weight reversible inhibitors have been much more limited.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24-26)</a> In particular, SP-2509 was reported by Salarius to be active in xenograft models,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> although the compound itself did not demonstrate a robust correlation between biochemical activity against KDM1A and inhibition of cell proliferation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative KDM1A inhibitors described in the literature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As reported in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>, (<a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">DOI 10.1021.acs.jmedchem.6b01018</a>),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> from an high throughput screening campaign (HTS), we identified a promising chemical series, namely 4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (represented by compound <b>1</b>, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>) with inhibitory activity against KDM1A. Initial SAR studies indicated that substitutions at the phenyl ring of <b>1</b> could increase the inhibitory potency, resulting in compound <b>2</b>, with an IC<sub>50</sub> of 162 nM for KDM1A inhibition.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hit <b>1</b> emerged from HTS, subsequent optimization led to compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Docking experiments of compound <b>2</b> allowed dual pose orientations (see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) directing the positively charged piperidine tail toward two negatively charged regions. We synthesized a series of derivatives able to reach different portions of the protein to solve the ambiguity of docking results and to increase inhibitory potency. The new compounds prepared gave us a clear indication of the relevant substituents for inhibitory activity and their interactions with protein active site of this novel class of noncovalent KDM1A inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40378" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40378" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Chemistry</h3><div class="NLM_p">A general synthesis for the derivatives reported is depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> (method A). Ethyl 4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>3a</b>)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> was methylated<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> at the pyrrole nitrogen and hydrolyzed to the acid intermediate <b>3c</b>. Reaction of the corresponding acyl chloride with the appropriate aniline (<b>3e</b>–<b>28e</b>) gave final amides <b>3-21</b> and <b>22f</b>–<b>28f</b>, in which the piperidine nitrogen is BOC-protected. Finally, compounds <b>22f</b>–<b>28f</b> were deprotected to <b>22</b>–<b>28</b> by acidic treatment and compound <b>22</b> was treated with the appropriate aldehydes under reductive amination conditions to give <b>29</b> or <b>30</b> or with acetyl chloride to give <b>31</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Thieno[3,2-<i>b</i>]pyrrole 5-carboxamides (Method A)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (1) NaH, DMF, rt, 30 min, (2) MeI, rt, 1 h; (b) LiOH, H<sub>2</sub>O, EtOH, reflux, 30 min; (c) (1) SOCl<sub>2</sub>, DMF cat., THF, reflux, 2 h, (2) pyridine, rt, 90 min; (d) 4 N HCl in 1.4-dioxane, 1,4-dioxane, rt, 8–24 h; (e) (i) CH<sub>3</sub>CHO or PhCHO, 1,2-DCE, rt, 2 h, then NaBH(OAc)<sub>3</sub> overnight or (ii) CH<sub>3</sub>COCl, DMAP, THF, rt, 2 h.</p></p></figure><div class="NLM_p">The general synthesis of the anilino derivatives is represented in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>: (bromomethyl)-nitro-benzenes (<b>15g</b>–<b>17g</b>) were coupled with the appropriate biphenols to give <b>15i</b>–<b>19i</b>. Mitsunobu reaction of <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate with the desired nitrophenols allowed the preparation of derivatives <b>16j</b>–<b>18j</b>, which were transformed into the <i>N</i>-methyl derivatives <b>16d</b>–<b>18d</b>, by Boc removal after acidic treatment, and reductive amination with formaldehyde and NaBH<sub>3</sub>CN (<b>15d</b> and <b>19d</b> were instead prepared by reaction of <b>15i</b> and <b>19i</b> with 1-methylpiperidin-4-ol). Nitro reduction originated desired anilines <b>15e</b>–<b>19e</b> and <b>22e</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Anilino Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 4.5 h; (b) <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate or 1-methylpiperidin-4-ol, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, overnight; (c) (1) 4 N HCl in 1,4-dioxane, 1,4-dioxane, rt, 2 h; or TFA, DCM, overnight; (2) HCHO, NaBH<sub>3</sub>CN, MeOH, rt, 1–2 h; (d) PtO<sub>2</sub>, H<sub>2</sub> (H–CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH. Final compounds have been numbered with numeric terms. Intermediates have been numbered with alpha numeric terms, referring to the first final compound generated by the intermediate itself, and a letter, referring to the functional group present in the molecule: −NO<sub>2</sub>═, <b>d</b> or <b>j</b>; −NH<sub>2</sub>═, <b>e</b>; −final derivatives BOC protected═, <b>f</b>; (−NO<sub>2</sub>, −Br)═ <b>g</b>; (−OH, −OH)═ <b>h</b>; (−NO<sub>2</sub>, −OH)═ <b>i</b>; (−NH<sub>2</sub>, −OH)═ <b>k</b>; −PhOH═, <b>l</b>.</p></p></figure><div class="NLM_p">Different methods used to prepare anilino derivatives (<b>20e</b>, <b>21e</b>, <b>23e</b>–<b>28e</b>) are reported in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">Alternatively, a general procedure for the synthesis of the compounds is depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The basic moiety BOC protected was inserted at the last step followed by usual protective group removal. The nitro group in compound <b>15i</b> was reduced via catalytic hydrogenation to give the aniline <b>33k</b>, which was condensed to compound <b>3c</b> previously transformed into acyl chloride. The thus obtained <b>33l</b> through Mitsunobu reaction with alcohols R<sub>5</sub>-OH generated <b>33f</b>–<b>43f</b> that were further deprotected to <b>33</b>–<b>43</b> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0012.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides (Method B)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH; (b) (1) <b>3c</b>, SOCl<sub>2</sub>, DMF cat., THF, 2 h, reflux, (2) then <b>33k</b>, pyridine, rt, 10 min; (c) R<sub>4</sub>OH, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, overnight; (d) 4 N HCl in 1,4-dioxane 1,4-dioxane/MeOH, 3 h, rt. R<sub>4</sub>═R<sub>5</sub>-BOC</p></p></figure><div class="NLM_p"><i>Ortho</i>,<i>meta</i>-disubstituted compounds (<a class="ref internalNav" href="#tbl5" aria-label="Tables 5">Tables 5</a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>) were prepared according to <a class="ref internalNav" href="#sch4" aria-label="Schemes 4">Schemes 4</a> or <a class="ref internalNav" href="#sch5" aria-label="5">5</a>.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, 2-(hydroxymethyl)-6-nitro-benzoate <b>45n</b>, obtained from 7-nitro-3<i>H</i>-isobenzofuran-1-one<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> by heating in 1 M sodium hydroxide and THF, was treated with NaH and an excess of the required alkyl halides, generating the <b>45v</b> and <b>50v</b>. Subsequent reduction of the ester gave the alcohols <b>45w</b> and <b>50w</b>, which were converted into their bromo derivatives <b>45g</b> and <b>50g</b>. Reaction with hydroquinone furnished the phenols <b>45l</b> and <b>50l</b>. Phenolic function was thus accessible to Mitsunobu reaction to give <b>45d</b>–<b>50d</b>, their nitro groups were transformed into anilines <b>45e</b>–<b>50e</b>, useful building blocks for the benzamide preparation <b>45f</b>–<b>50f</b>, obtained by reacting with <b>3c</b> or <b>49c</b>. Boc-protecting groups were finally removed to give the compounds <b>45</b>–<b>50</b>.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides <i>o</i>,<i>m</i>-Disubstituted<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1 M NaOH, THF, water, 50 °C, 1 h, quantitative; (b) (1) NaH, DMA 0 °C to rt, 30 min, (2) CH<sub>3</sub>I or CH<sub>3</sub>CH<sub>2</sub>I, 0 °C, then rt overnight; (b) LiBH<sub>4</sub>, THF, 0 °C, then rt, 3 h or 60 °C, 30 min; (d) CBr<sub>4</sub>, PPh<sub>3</sub>, CH<sub>3</sub>CN, 2 h, rt; (e) hydroquinone, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 2 h; (f) R<sub>4</sub>OH, PPh<sub>3</sub>, DIAD, THF, 0 °C, then rt, 12 h; (g) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH; (h) (1) <b>3c</b> or <b>49c</b>, SOCl<sub>2</sub>, DMF cat., THF, reflux 3 h, (2) <b>45e</b>–<b>50e</b> pyridine, rt, 1 h; (i) 4 M HCl in 1.4-dioxane, 1,4-dioxane, rt, 24 h. R<sub>4</sub>═R<sub>5</sub>-BOC.</p></p></figure><div class="NLM_p">Alternatively, disubstituted derivatives were prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>: <b>45n</b> was alkylated with isopropyl iodide, providing <b>52o</b>, which was further converted into <b>52p</b> by protecting the alcoholic group with TBDSCl. The carboxylic function was reduced to <b>52q</b> and the thus formed alcohol transformed into its mesylate <b>52r</b> to facilitate the reaction with hydroquinone and the ether formation in <b>52l</b>. Free phenolic oxygen reacted with <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate to give <b>52d</b>, which, after reduction to the anilino derivative <b>52e</b>, reacted with the carboxylic acid <b>3c</b> to give intermediate <b>52s</b>. Removal of the protective group (<b>52t</b>), transformation of the alcoholic function into a mesylate group (<b>52u</b>) and subsequent reaction with morpholine or an appropriate alcohol gave the BOC intermediates <b>52f</b>–<b>54f</b>, which were finally hydrolyzed to give <b>52</b>–<b>54</b>.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0014.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides <i>o</i>,<i>m</i>-Disubstituted<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) isopropyl iodide, DMA, rt, 48 h; (b) TBDSCl, imidazole, DCM, 0 °C, 15 min; (c) LiBH<sub>4</sub>, THF, 60 °C, 30 min; (d) mesyl chloride, TEA, DCM, −15 to 0 °C, 20 min; (e) hydroquinone, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 45 min; (f) <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C then rt, overnight; (g) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 75 °C, atm pressure), EtOH; (h) (1) <b>3c</b>, SOCl<sub>2</sub>, DMF cat., THF, reflux, 3 h, (2) <b>52e</b>, pyridine, rt 1 h; (i) TBAF, THF, 45 min, rt; (j) mesyl chloride, TEA, DCM, −15 °C, 30 min; (k) morpholine, TEA, DCM, rt, 12 h, or NaH, ethanol or 2-propanol, 0 °C to rt, 1 h; (l) 4 N HCl in 1,4-dioxane, 0 °C or rt, 1h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR Investigation</h3><div class="NLM_p">The evidence that docking runs allowed dual placement of piperidine moiety and the lack of unambiguous data from X-ray crystallography for compound <b>2</b> prompted us to perform an electrostatic potential analysis of the protein active site (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), identifying two regions of potential interactions for positively charged piperidine tail, both justifying such an increase in potency of compound <b>2</b> with respect to <b>1</b>. One region is located around Asp375, above the thienopyrrole plane. The other is located around Asp555 and Asp556, which are located below the thienopyrrole plane. The compound (green in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) would adopt an extended conformation to interact with Asp375, whereas the interaction with Asp556 would force the inhibitor in a U-shaped conformation (pink in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). The two poses of <b>2</b>, obtained with different docking programs and submitted to molecular dynamics, showed complexes stability sufficient to be both considered as starting points for SAR exploration (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a> for details).</div><figure id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking poses for compound <b>2</b>. In terms of energies, the interaction with Asp375 (green) is favored, while the U-shaped conformation (pink) points toward an interesting cluster of negatively charged residues (Asp555, Asp556).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Taking into consideration the results obtained when <i>meta</i> position of the hit was modified, as described in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> we planned a careful SAR exploration, initially directed to the <i>ortho</i>, <i>meta</i>, and <i>para</i> positions on the benzamide ring and subsequently to the optimal residue identified. Different spacers, hydroquinone vs resorcinol or cathecol, and different terminal basic moieties, were also investigated and to conclude the best results were combined to generate <i>ortho</i>–<i>meta</i> disubstituted derivatives (see <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SAR investigation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> <i>Ortho</i>, <i>meta</i>, and <i>para</i> Benzamide Investigation</h4><div class="NLM_p">The investigation on the benzamide was designed with the following aims:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">1.</td><td colspan="5" valign="top" class="paragraph"><p class="last">to explore the area occupied by the substituents on benzamide function,</p></td></tr><tr class="li1"><td valign="top">2.</td><td colspan="5" valign="top" class="paragraph"><p class="last">to modulate the substituents’ contribution to the inhibitory activity,</p></td></tr><tr class="li1"><td valign="top">3.</td><td colspan="5" valign="top" class="paragraph"><p class="last">to verify whether the structural modifications of the ligand could provide X-ray structures.</p></td></tr></table></div><div class="NLM_p">As reported in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, the substituents were selected from those already tested in the <i>meta</i> series.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The expansion was fully explored to gain a complete SAR and to direct the substituents toward the two negatively charged regions. We used piperidine as a basic moiety to take advantage of activity increase shown in the <i>meta</i> series reported in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Whereas methoxy, methoxymethyl, aryloxy methyl, and arylmethoxy substituents in the benzamide moiety maintained high micromolar activity when applied to the <i>meta</i> position (<b>1</b>, <b>7</b>, <b>10</b>, <b>13</b>), they brought instead to almost inactive compounds when placed in <i>ortho</i> and <i>para</i> position (<b>6</b>, <b>8</b>, <b>9</b>, <b>11</b>, <b>12</b>, <b>14</b>). When the chain bearing a piperidinoxyphenoxymethyl group was moved from <i>meta</i> to <i>ortho</i> and <i>para</i>, the potency of the unsubstituted term <b>3</b> was however improved, although while <i>para</i> derivative <b>17</b> resulted in the micromolar range, <i>ortho</i> derivatives <b>15</b> and <b>22</b> resulted in being slightly more active than the corresponding <i>meta</i> ones, <b>16</b> and <b>2</b>. We report that several attempts to place unsubstituted piperidine in position <i>para</i> failed for synthetic hurdles during BOC deprotection of the intermediate.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>Ortho</i>, <i>meta</i>, and <i>para</i> Investigation on the Benzamide:KDM1A Inhibitory Activity (IC<sub>50</sub> μM<a class="ref internalNav" href="#t1fn2" aria-label="a">a</a>)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0015.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn2"><div class="footnote" id="t1fn2"><sup>Table a</sup><p class="last">All the compounds reported in table have been prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div><div></div></div><div class="NLM_p">This preliminary exploration resulted in being quite satisfactory. One of the most potent <i>ortho</i> derivatives <b>15</b> easily crystallized, and the analysis of X-ray structure provided an explanation for potency improvement (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). <i>Ortho</i> derivative <b>15</b> accommodates the benzamide moiety, favoring aromatic–aromatic interactions, and directs the basic terminal residue toward one of the two negatively charged regions (Asp555). Compound <b>15</b> unambiguously follows one of the two hypothesized positions, pointing downward with respect to the thienopyrrole plane. The ligand occupies the catalytic site of KDM1A, and the size and the shape of the active site renders the packed “U shaped” conformation as the most preferred one. Main hydrophobic interactions have been evidenced with no visible polar contacts. The thienopyrrole moiety is comprised between residues Val333, Thr335, Leu663, Tyr765, and Thr814 into a cage that is delimitated by flavin. The second ring system is packed between residues Val333, Phe543, and Trp699.</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray representations of <b>15</b> and <b>17</b> shows as the ortho <b>15</b> (A) accommodates the benzamide moiety in a way to favor aromatic–aromatic interactions and directs the basic terminal residue toward one of the two negatively charged regions (Asp555). Conversely, in the para derivative <b>17</b> (B), the benzamide moiety is forced out of the plane, possibly explaining the lower potency, while the terminal basic moiety located in proximity to Glu379 (2.9 A) could contribute to the residual micromolar potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>17</b> (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B), a <i>para</i> substituted derivative that showed to inhibit KDM1A, albeit with considerably less potency (2.5 μM), has also been crystallized. The benzamide moiety of <b>17</b> is forced out of the plane, altering the geometry of its aromatic–aromatic interaction with flavin, a possible explanation for the lower potency, while the terminal basic moiety located in proximity to Glu379 (2.9 A) could contribute to the residual activity.</div><div class="NLM_p">To conclude activity and structural comparisons between <i>ortho</i>, <i>meta</i>, and <i>para</i> derivatives, we speculate that <i>meta</i> derivative <b>2</b> could adopt a conformation similar to <b>15</b> but having interaction with Asp556 instead of Asp555. The <i>para</i> derivative <b>17</b> interacts in the upward negative region (Glu379), reachable also by obtained docking pose of <b>2</b>, but the 5–5 system planarity distortion and the lesser biochemical potency suggest that this conformation is unlikely the one adopted by <b>2</b>. The possible explanation for the inactivity of shorter derivatives (<b>4</b>–<b>6</b>, <b>8</b>–<b>14</b>) is that only <i>meta</i> derivatives can reach interactions with Gln358 (docking results, see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p">Finally, compound <b>15</b> was also evaluated to define its biochemical mechanism of inhibition. According to <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A, Lineweaver–Burk plot indicated a nearly competitive type inhibition for compound <b>15</b>, with a <i>K</i><sub>i</sub> = 2.3 μM. Accordingly, in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B, we report the overlap of X-ray structures of <b>15</b> with <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> clearly showing that the <i>ortho</i> substituent on the benzamide moiety occupies the same region of Met-Thr-Arg-Ala residues of the substrate.</div><figure id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Lineweaver–Burk plot for <b>15</b>. (B) X-ray structure of <b>15</b> superimposed to <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>. The tail of compound <b>15</b> occupies part of the region normally reserved to the substrate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> <i>Ortho</i>, <i>meta</i>, and <i>para</i> Substitution of the Hydroquinone</h4><div class="NLM_p">The influence of the <i>ortho</i>, <i>meta</i>, and <i>para</i> position of the biphenol moiety was investigated (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), the <i>meta</i> being detrimental (<b>18</b> resulted in being 3-fold less active than <b>15</b>) while the <i>ortho</i> being barely active (<b>19</b> IC<sub>50</sub>= 68.2 μM). This trend in potency is justified by observing that the exit vector orientation for substituents on <i>para</i> biphenol moiety is directed toward Asp555.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. <i>Ortho</i>, <i>meta</i>, and <i>para</i> Substitution of the Hydroquinone<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0016.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn2"><div class="footnote" id="t2fn2"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t2fn1"><sup>Table b</sup><p class="last">All the compounds reported in Table have been prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1.">Scheme 1.</a></p></div></div><div></div></div><div class="NLM_p last">We continued our investigation by further modifying the lateral chain as depicted in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>: specifically, only <i>ortho</i> substituted derivatives were prepared, both for the higher potency of the analogues and for the advantage of having a support from X-ray crystallography.</div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Modifications on the Spacer</h4><div class="NLM_p">Several compounds were designed and prepared to alter the hydroquinonic feature of <b>22</b> (see <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The phenyl ring has been replaced with piperidinomethyl moiety <b>44</b>, resulting in a much less potent compound. Also amide in place of ether (<b>25</b>) or nitrogen in place of oxygen (<b>24</b> and <b>26</b>) caused a reduction of potency. The position of oxygen seemed irrelevant, <b>16</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) being equipotent to <b>20</b>, its removal instead barely influenced the activity (<b>23</b> vs <b>22</b>) and was of a much greater impact when it occurred at the distal oxygen as in <b>26</b> or <b>21</b>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Modifications on the Spacer<a class="ref internalNav" href="#t3fn1" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0017.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table c</sup><p class="last">All the compounds reported in Table have been prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</p></div><div class="footnote" id="t3fn2"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t3fn3"><sup>Table b</sup><p class="last">Prepared according to <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></div><div></div></div><div class="NLM_p">To explain these trends various aspects have been investigated:<ul class="NLM_list-list_type-bullet"><li><p class="inline">Conformational analysis,</p></li><li><p class="inline">HOMO–LUMO energies and localization,</p></li><li><p class="inline">partial charges on NH (N-Me).</p></li></ul></div><div class="NLM_p last">While for <b>44</b> and for <b>25</b> the conformational accessible space can quite easily explain the activity decrease, when examining the couples <b>22</b>/<b>26</b> and <b>15</b>/<b>21</b>, there are neither differences in conformational space nor in docking results or in nitrogen partial charges. Assuming that electronic configuration might influence the affinity of the ligand and considering the aromatic–aromatic interaction between FAD and 5–5 thienopyrrole system, we investigated HOMO–LUMO localization and energies (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a> for details). We have noticed that the decrease in activity is associated with the HOMO shifting toward piperidino ring, away from the 5–5 system. The hypothesis that this shift may decrease the strength of the electronic interaction between FAD and 5–5 system is under further investigations based on more accurate ab initio models.</div></div><div id="sec2_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Different Basic Moieties (<i>ortho</i>)</h4><div class="NLM_p">The inspection of the ring bearing the basic nitrogen was also extensively performed as reported in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>: when 4-piperidinol was converted to 3-piperidinol as in <b>41</b> or shortened to 3-pirrolidinole (<b>42</b>) and azetidin-3yl-methanol (<b>27</b>), or when the cycle was opened to <i>N</i>-methylaminepropanol (<b>43</b>) or guanidinomethyl (<b>28</b>), the activity significantly dropped from 2- to 10-fold. Equipotent derivatives were obtained with 4-aminocyclohexanol, where the <i>cis</i> isomer <b>36</b> resulted in being 2-fold more potent than the <i>trans</i> isomer (<b>40</b>) and (1<i>S,</i>5<i>R</i>)-8-azabicyclo[3.2.1]octan-3-ol (<b>37</b>). The activity significantly improved with 4-azepanole (<b>35</b>) and pyrrolidin-3-ylmethanol (<b>34</b>), in particular its 3<i>R</i> enantiomers (<b>33</b>) that showed an IC<sub>50</sub> of 18 nM.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Different Basic Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0018.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">Prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">Prepared according to <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>.</p></div></div><div></div></div><div class="NLM_p">A small set of residues was prepared to alter the basicity of the terminal nitrogen and the space occupancy of <b>22</b>. Ethyl derivative <b>29</b> resulted in being slightly less active, while enlarging substituent as in <b>30</b> or removing the basicity (<b>31</b>), as expected, resulted in complete inactivity.</div><div class="NLM_p"><b>34</b> was crystallized (see <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>) and showed the same orientation and interaction pattern with protein residues as the parent <b>15</b>. The length of the chain and the protein residue flexibility allow interactions of nitrogen atoms with Asp555, where (3<i>R</i>)-pyrrolidine group in <b>33</b> shows the best activity among the series.</div><figure id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0007.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray of <b>34</b>: The compound showed the same orientation and interaction pattern with protein residues as the parent <b>15</b>. The length of the chain and the protein residue flexibility allow interactions of nitrogen atoms with Asp555, where (3<i>R</i>)-pyrrolidine group in <b>33</b> shows the best activity among the series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>Ortho</i>–<i>meta</i> Disubstituted Derivatives</h4><div class="NLM_p">By observing X-ray crystal structure of <b>1</b> and <b>34</b> superimposed (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>), it was evident that <i>ortho</i> and <i>meta</i> substituents could occupy different pockets of the KDM1A active site, thus suggesting that their copresence could facilitate the interaction with the enzyme. With this in mind, we approached the <i>ortho</i>–<i>meta</i> double substitution, expanding separately the two positions according to <a class="ref internalNav" href="#tbl5" aria-label="Tables 5">Tables 5</a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>. We observed a further improvement in the activity: <b>46</b> in fact gave a single-digit nanomolar IC<sub>50</sub>.</div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0008.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Superimposition of X-ray structures obtained with <b>1</b> (green) and <b>34</b> (cyan), showing the different regions of interaction for <i>ortho</i> and <i>meta</i> substituents.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. <i>Ortho</i>–<i>meta</i> Disubstituted Derivatives: SAR on Piperidinol</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0019.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t5fn2"><sup>Table b</sup><p class="last">Prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>.</p></div><div class="footnote" id="t5fn3"><sup>Table c</sup><p class="last">Prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. <i>Ortho</i>–<i>meta</i> Disubstituted Derivatives: SAR on O-Me<a class="ref internalNav" href="#t6fn1" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0020.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn2"><div class="footnote" id="t6fn2"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t6fn1"><sup>Table b</sup><p class="last">All the compounds reported in table have been prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>.</p></div></div><div></div></div></div><div id="sec2_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Combining Best Features</h4><div class="NLM_p">The last three compounds reported in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a> were prepared by combining all those features that had demonstrated positive impact in terms of biochemical potency. <b>49</b> was obtained by applying the best substitution on pyrrolic nitrogen (ethyl group identified in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>) and compound <b>50</b> by adding an ethyloxymethyl in <i>meta</i>: their IC<sub>50</sub>s were more or less identical as well as their selectivity versus KDM1B and MAOs.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Combining Best Features<a class="ref internalNav" href="#t7fn1" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0021.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn2"><div class="footnote" id="t7fn2"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t7fn3"><sup>Table b</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) at the fixed dose of 1 μM for <b>46</b>, <b>49</b>, and <b>50</b> and 0.5 μM for <b>55</b>, measured after 24 h of compound treatment.</p></div><div class="footnote" id="t7fn1"><sup>Table c</sup><p class="last">All the compounds reported in Table have been prepared according to <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>.</p></div></div><div></div></div><div class="NLM_p"><b>49</b> and <b>50</b> have been crystallized, confirming the working hypothesis (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>). The increased potency of disubstituted compounds can be related to additional interactions between Gln358 and ethyloxymethyl chain, which is also responsible of conformational restraints on the longer chain, thus with entropic gain that further induces the U-shaped conformation.</div><figure id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0009.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray of <b>49</b> (cyano) and <b>50</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Biology</h3><div class="NLM_p">The ability of the best compounds to inhibit KDM1A in human acute myeloid THP-1 cells was evaluated. THP-1 cells, characterized by the MLL-AF9 translocation, were selected for the strong experimental evidence, demonstrating the relevance of KDM1A in sustaining the oncogenic potential of MLL-AF9 leukemia stem cells.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Moreover, the absence of evident effect on global level of histone H3K4 methylation after genetic ablation of KDM1A both in cancer cells<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and hematopoietc cells<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> let us follow KDM1A inhibition in cells, looking at the transcriptional effect on known KDM1A regulating genes (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Figure S2</a>). To this purpose, THP-1 cells were incubated with the selected inhibitors at the fixed concentration of 1 μM, and after 24 h, CD14, CD86, and CD11b mRNA levels were measured by quantitative RT-PCR. As reported in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>, all the tested compounds, similarly to <b>55</b>, were able to induce an increase of mRNA expression of the selected genes. The observed effects being similar to what has been obtained by KDM1A down-regulation by shRNA demonstrated their ability to block KDM1A activity in cells inducing differentiation. To additionally confirm induction of differentiation in AML cells mediated by our compounds, we evaluated the expression of CD86 and CD11b by flow cytometry in THP-1 cells treated for 48 h. As reported in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Figure S2</a>, all the three compounds induced a clear increase of the expression of the two differentiation markers with respect to vehicle treated cells. The antiproliferative activity of <b>49</b>, <b>46</b>, and <b>50</b> was determined on three different leukemia cell lines: THP-1, NB4, and MV4-11, representative of different subtypes of AMLs. THP-1 and MV4-11 are both characterized by MLL translocation and MV4-11 by a concurrent presence of FLT3 mutation. As shown in <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>, the compounds tested are demonstrated to exert antiproliferative effects on the evaluated cell lines at a micromolar level.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Antiproliferation Data</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0022.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div><div></div></div><div class="NLM_p">To verify the ability of <b>49</b> and <b>50</b> to induce apoptosis or to affect cell cycle progression, annexin V–propidium iodide (PI) double staining was used on THP-1 cells. Data (as reported in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Figure S3</a>) showed that 72 h treatment at 1 μM of <b>49</b> and <b>50</b> and 0.5 μM of <b>55</b> did not induce cell death, in agreement with the known pro-differentiation effects of KDM1A inhibition in AML cells.,<a onclick="showRef(event, 'ref34 ref45'); return false;" href="javascript:void(0);" class="ref ref34 ref45">(34, 45)</a> Finally, the anticlonogenic potential of <b>49</b> and <b>50</b> on human acute myeloid leukemia THP-1 was determined. As reported in the <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a> both of the compounds strongly inhibited the colony growth of THP-1 cells and, similarly to <b>55</b>, induced differentiation, as demonstrated by the increased expression of CD86 marker.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Anticlonogenic Potential</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">THP-1<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a> methylcellulose colony formation % inhibition</th><th class="colsep0 rowsep0" align="center" char="±">CD86<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a>,<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="char" char="±">63.5 ± 2.1</td><td class="colsep0 rowsep0" align="char" char="±">31.1 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="char" char="±">70.5 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">39.6 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="char" char="±">70 ± 2.8</td><td class="colsep0 rowsep0" align="char" char="±">55 ± 8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup>a</sup><p class="last">Colonies were counted after 13 days. Percentage of inhibition is referenced to the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2).</p></div><div class="footnote" id="t9fn2"><sup>b</sup><p class="last">Compounds <b>49</b> and <b>50</b> were tested at the dose of 1 μM, <b>55</b> at the dose of 0.5 μM.</p></div><div class="footnote" id="t9fn3"><sup>c</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) measured a the end of experiment.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27172" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27172" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have reported the development of thieno[3,2-<i>b</i>]pyrrole KDM1A inhibitors, a novel chemical series identified by an HTS campaign, which potently and reversibly inhibit KDM1A. Several different synthetic pathways allowed an extensive exploration of various portions of the molecule and led to the identification of the main structural requirements for the inhibitory activity. This work has been supported by multiple ligand/KDM1A-CoRest cocrystal structures, which rationalized the gain in potency due to polar interaction with two negatively charged areas of the catalytic site. We identified critical molecular features for potency: ortho substitution in the benzamide portion brought to more stable complex with the enzyme, giving optimal inhibitory activity and allowing the solution of multiple crystal structures. 3D information thus obtained guided subsequent optimization. Para substitution of the hydroquinone moiety was mandatory for the potency. The lack of substituents on the terminal nitrogen facilitates the interaction with the enzyme and the pyrrolidin-3-yl-methanol residue <b>34</b>, in particular, its 3<i>R</i> enantiomer <b>33</b>, resulted as the most efficacious basic moiety among those explored. Concurrent bisubstitution of the benzamide at positions <i>ortho</i> and <i>meta</i> brought to a clear improvement of the potency, generating single-digit nanomolar inhibitors <b>46</b>, <b>49</b>, and <b>50</b>. These compounds were characterized for their cellular activity: they clearly demonstrated their ability to inhibit KDM1A in THP-1 cells and to affect the proliferation in different leukemia cell lines. Moreover, both <b>49</b> and <b>50</b> remarkably inhibit colony formation in THP-1 human leukemia cells. The interesting biological data of this new series of reversible inhibitors prompted us to bring thienopyrroles toward further development.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87462" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87462" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedures</h3><div class="NLM_p">Melting points were determined on a BUCHI melting point B-540 and are uncorrected. Reagents and solvents used, unless stated otherwise, were of commercially available reagent grade quality and were used without further purification. Reaction solvents were used anhydrous, purchased in Sure/Seal bottles stored over molecular sieves. Silica gel flash chromatography purifications were performed on Merck silica gel 60 (0.04–0.063 mm) or Biotage Isolera One system using prepacked columns from Grace and Biotage. Nuclear magnetic resonance spectra (<sup>1</sup>H NMR) were recorded on an Agilent VNMRS 500 MHz spectrometer at 300 K and are referenced in ppm (δ) relative to TMS. Coupling constants (<i>J</i>) are expressed in hertz (Hz). HPLC-MS experiments were performed on an Acquity UPLC apparatus, equipped with a diode array and a Micromass SQD single quadrupole (Waters). The compounds used for biological tests were monitored at 220 and 254 nM with two different methods, and their purity was found to be at least 95%. Semipreparative HPLC was performed on a Waters apparatus, eluting the sample through a BEH C 18 column (5 μm, 19 mm × 100 mm) using decreasingly polar mixtures of water (containing 5% of MeCN and 0.1% of trifluoroacetic acid) and MeCN (containing 0.1% of trifluoroacetic acid).</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>3c</b>)</h4><div class="NLM_p last">First, 0.92 g (38 mmol) of LiOH in 11 mL of H<sub>2</sub>O was added at rt to a solution of 1.6 g (7.7 mmol) of ethyl 4-methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>3b</b>, see compound <b>105a</b> in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>) in 11 mL of EtOH. The mixture was stirred for 30 min at reflux. The solvent was evaporated then H<sub>2</sub>O was added and the solution was brought to a pH value of about 2 with 2 M HCl. The mixture was extracted with EtOAc, the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated to give 1.39 g of <b>3c</b> (99%) as a beige solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 12.45 (bs, 1 H), 7.54 (d, <i>J</i> = 5.4 Hz, 1 H), 7.20 (d, <i>J</i> = 5.4 Hz, 1 H), 7.12 (s, 1 H), 3.99 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 182 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Procedure for the Synthesis of Compounds <b>3</b>–<b>21</b> and the Intermediates <b>22f</b>–<b>28f</b></h3><div class="NLM_p">First, 0.24 g (2.0 mmol, 0.15 mL) of SOCl<sub>2</sub> and 3 drops of dry DMF were added at rt to a solution of 0.277 g (1.53 mmol) of 4-methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic acid (<b>3c</b>) in 9 mL of dry THF. Then the mixture was stirred for 2 h under reflux. The reaction mixture was then cooled down to rt and added to a solution of 1.3 mmol of aniline in 7 mL of pyridine. The mixture was stirred for 30 min at rt and for a further 45 min at 60 °C. Then the mixture was diluted with DCM, washed with a saturated NaHCO<sub>3</sub> solution and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give an oil which was purified by flash chromatography.</div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Similarly Prepared</h4><div id="sec4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 4-Methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>3</b>)</h5><div class="NLM_p last">See compound <b>23</b> in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> <i>N</i>-[2-(Methoxymethyl)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>4</b>) (from <b>3c</b> and 2-(Methoxymethyl)aniline)</h5><div class="NLM_p last">Yield 57%, pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.63 (s, 1 H), 7.69 <i>-</i>7.65 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.42<i>-</i>7.37 (m, 1 H), 7.36–7.30 (m, 1 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.22–7.17 (m, 2 H), 4.51 (s, 2 H), 4.01 (s, 3 H), 3.33 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 301 [M + H]<sup>+</sup>; mp 96–98 °C.</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> <i>N</i>-[4-(Methoxymethyl)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>5</b>) (from <b>3c</b> and 4-(Methoxymethyl)aniline)</h5><div class="NLM_p last">Yield 45%, orange solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.01 (s, 1 H) 7.73<i>-</i>7.69 (m, 2 H) 7.51 (d, <i>J</i> = 5.4 Hz, 1 H) 7.32 (s, 1 H) 7.29<i>-</i>7.26 (m, 2 H) 7.24 (d, <i>J</i> = 5.4 Hz, 1 H) 4.36 (s, 2 H) 4.01 (s, 3 H) 3.27 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 301 [M + H]<sup>+</sup>; mp 147–150 °C.</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> <i>N</i>-(4-Methoxyphenyl)-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>8</b>) (from <b>3c</b> and 4-Methoxyaniline)</h5><div class="NLM_p last">Yield 73%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.88 (s, 1 H), 7.67–7.59 (m, 2 H), 7.49 (d, <i>J</i> = 5.4 Hz, 1 H), 7.27 (s, 1 H), 7.23 (d, <i>J</i> = 5.4 Hz, 1 H), 6.95–6.85 (m, 2 H), 4.01 (s, 3 H), 3.74 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 287 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 4-Methyl-<i>N</i>-[2-(4-pyridyloxymethyl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>9</b>) (from <b>3c</b> and 2-(4-Pyridyloxymethyl)aniline)</h5><div class="NLM_p last">Yield 42%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.92 (s, 1 H), 7.59–7.55 (m, 2 H), 7.53 (d, <i>J</i> = 5.4 Hz, 1 H), 7.42–7.34 (m, 2 H), 7.33–7.28 (m, 2 H), 7.25 (d, <i>J</i> = 5.4 Hz, 1 H), 7.18–7.12 (m, 1 H), 6.10–5.95 (m, 2 H), 5.09 (s, 2 H), 3.99 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 142–146 °C dec.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 4-Methyl-<i>N</i>-[3-(4-pyridyloxymethyl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>10</b>)</h5><div class="NLM_p last">See compound <b>23</b> in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 4-Methyl-<i>N</i>-[4-(4-pyridyloxymethyl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>11</b>) (from <b>3c</b> and 4-(4-Pyridyloxymethyl)aniline)</h5><div class="NLM_p last">Yield 16%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.06 (s, 1 H), 7.78–7.71 (m, 4 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.33–7.28 (m, 3 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 6.12–6.04 (m, 2 H), 5.01 (s, 2 H), 4.00 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 295 °C dec.</div></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 4-Methyl-<i>N</i>-[2-(4-pyridylmethoxy)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>12</b>) (from <b>3c</b> and 2-(4-Pyridylmethoxy)aniline)</h5><div class="NLM_p last">Yield 19%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.32 (s, 1 H) 8.58–8.49 (m, 2 H) 7.75–7.70 (m, 1 H) 7.54–7.48 (m, 3 H) 7.30 (s, 1 H) 7.24 (d, <i>J</i> = 5.4 Hz, 1 H) 7.17–7.11 (m, 1 H) 7.10–7.06 (m, 1 H), 7.02–6.94 (m, 1 H) 5.27 (s, 2 H) 4.01 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 141–145 °C.</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 4-Methyl-<i>N</i>-[3-(4-pyridylmethoxy)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>13</b>) (from <b>3c</b> and 3-(4-Pyridylmethoxy)aniline)</h5><div class="NLM_p last">Yield 14%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.99 (s, 1 H), 8.62–8.54 (m, 2 H), 7.57–7.54 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.47–7.43 (m, 2 H), 7.35–7.30 (m, 2 H), 7.27–7.21 (m, 2 H), 6.75–6.70 (m, 1 H), 5.18 (s, 2 H), 4.01 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 4-Methyl-<i>N</i>-[4-(4-pyridylmethoxy)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>14</b>) (from <b>3c</b> and 4-(4-Pyridylmethoxy)aniline)</h5><div class="NLM_p last">Yield 17%, pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.91 (s, 1 H), 8.60–8.55 (m, 2 H), 7.66–7.61 (m, 2 H), 7.50 (d, <i>J</i> = 5.4 Hz, 1 H), 7.46–7.41 (m, 2 H), 7.27 (s, 1 H), 7.23 (d, <i>J</i> = 5.4 Hz, 1 H), 7.03–6.97 (m, 2 H), 5.17 (s, 2 H), 4.00 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 190–193 °C.</div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 4-Methyl-<i>N</i>-[2-[[4-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>15</b>) (from <b>3c</b> and 2-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>15e</b>)</h5><div class="NLM_p last">Yield 12%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.55–7.48 (m, 3 H), 7.39–7.33 (m, 1 H), 7.28–7.21 (m, 3 H), 6.91–6.80 (m, 4 H), 5.09 (s, 2 H), 4.22–4.11 (m, 1 H), 4.00 (s, 3 H), 2.61–2.52 (m, 2 H), 2.14 (s, 3 H), 2.12–2.05 (m, 2 H), 1.88–1.79 (m, 2 H), 1.61–1.48 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 144–147 °C.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 4-Methyl-<i>N</i>-[3-[[4-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>16</b>) (from <b>3c</b> and 3-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>16e</b>)</h5><div class="NLM_p last">See also <b>91</b> in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 4-Methyl-<i>N</i>-[4-[[4-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>17</b>) (from <b>3c</b> and 4-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>17e</b>)</h5><div class="NLM_p last">Yield 15%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.03 (s, 1 H), 7.78–7.70 (m, 2 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.42–7.35 (m, 2 H), 7.32 (s, 1 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 6.94–6.82 (m, 4 H), 4.97 (s, 2 H), 4.25–4.13 (m, 1 H), 4.02 (s, 3 H), 2.64–2.54 (m, 2 H), 2.20–2.04 (m, 5 H), 1.92–1.80 (m, 2 H), 1.63–1.50 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 164–168 °C dec.</div></div><div id="sec4_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 4-Methyl-<i>N</i>-[2-[[3-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>18</b>) (from <b>3c</b> and 2-[[3-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>18e</b>)</h5><div class="NLM_p last">Yield 54%, white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.56–7.47 (m, 3 H), 7.40–7.33 (m, 1 H), 7.29–7.21 (m, 3 H), 7.16–7.09 (m, 1 H), 6.56–6.46 (m, 3 H), 5.15 (s, 2 H), 4.30–4.21 (m, 1 H), 4.00 (s, 3 H), 2.60–2.51 (m, 2 H), 2.13 (s, 3 H), 2.11–2.03 (m, 2 H), 1.88–1.78 (m, 2 H), 1.61–1.47 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 147–150 °C.</div></div><div id="sec4_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 4-Methyl-<i>N</i>-[2-[[2-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>19</b>) (from <b>3c</b> and 2-[[2-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>19e</b>)</h5><div class="NLM_p last">Yield 14%, off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.68 (s, 1 H), 7.66–7.62 (m, 1 H), 7.61–7.56 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.39–7.33 (m, 1 H), 7.27–7.21 (m, 3 H), 7.05–6.98 (m, 2 H), 6.91–6.83 (m, 2 H), 5.18 (s, 2 H), 4.29–4.19 (m, 1 H), 4.01 (s, 3 H), 2.57–2.51 (m, 2 H), 2.09 (s, 3 H), 2.06–1.97 (m, 2 H), 1.83–1.74 (m, 2 H), 1.61–1.48 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 4-Methyl-<i>N</i>-[2-[[4-[(1-methyl-4-piperidyl)oxy]phenyl]methoxy]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>20</b>) (from <b>3c</b> and 2-[[4-[(1-Methyl-4-piperidyl)oxy]phenyl]methoxy]aniline, <b>20e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 67%, white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.08 (s, 1 H) 7.85–7.73 (m, 1 H) 7.56–7.47 (m, 1 H) 7.45–7.35 (m, 2 H) 7.28–7.20 (m, 1 H) 7.19–7.08 (m, 3 H) 7.00–6.84 (m, 3 H) 5.10 (s, 2 H) 4.41–4.27 (m, 1 H) 4.00 (s, 3 H) 2.60 (br s, 2 H) 2.24–2.07 (m, 5 H) 1.98–1.47 (m, 4 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 137–140 °C.</div></div><div id="sec4_2_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 4-Methyl-<i>N</i>-[2-[[4-[(4-methylpiperazin-1-yl)methyl]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>21</b>) (from <b>3c</b> and 2-[[4-[(4-Methylpiperazin-1-yl)methyl]phenoxy]methyl]aniline (<b>21e</b>), See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.75 (s, 1 H), 7.56–7.49 (m, 3 H), 7.40–7.33 (m, 1 H), 7.29–7.21 (m, 3 H), 7.19–7.13 (m, 2 H), 6.94–6.88 (m, 2 H), 5.14 (s, 2 H), 3.99 (s, 3 H), 3.35 (s, 2 H), 2.47–2.06 (m, 11 H). MS (ESI): <i>m</i>/<i>z</i> 475 [M + H]<sup>+</sup>; mp 143–147 °C.</div></div></div><div id="sec4_2_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>22f</b>) (from <b>3c</b> and Aniline <b>22e</b>)</h4><div class="NLM_p last">Yield 58%, white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.55–7.47 (m, 3 H), 7.40–7.32 (m, 1 H), 7.29–7.19 (m, 3 H), 6.94–6.83 (m, 4 H), 5.10 (s, 2 H), 4.42–4.32 (m, 1 H), 4.00 (s, 3 H), 3.68–3.55 (m, 2 H), 3.22–3.05 (m, 2 H), 1.88–1.75 (m, 2 H), 1.49–1.40 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 584 [M + Na]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Similarly Prepared</h4><div id="sec4_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> <i>tert</i>-Butyl 4-[4-[2-[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]ethyl]phenoxy]piperidine-1-carboxylate (<b>23f</b>) (from <b>3c</b> and <i>tert</i>-Butyl 4-[4-[2-(2-Aminophenyl)ethyl]phenoxy]piperidine-1-carboxylate, <b>23e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 25%, off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.76–7.71 (m, 1 H), 7.31 (d, <i>J</i> = 5.4 Hz, 1 H), 7.30–7.25 (m, 2 H), 7.22–7.17 (m, 1 H), 7.02–6.96 (m, 4 H), 6.84–6.80 (m, 2 H), 6.56 (s, 1 H), 4.44–4.34 (m, 1 H), 4.06 (s, 3 H), 3.74–3.64 (m, 2 H), 3.32–3.23 (m, 2 H), 2.95–2.86 (m, 4 H), 1.94–1.83 (m, 2 H), 1.77–1.65 (m, 2 H), 1.47 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 560 [M + H]<sup>+</sup>.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenyl]piperazine-1-carboxylate (<b>24f</b>) (from <b>3c</b> and <i>tert</i>-Butyl 4-[4-[(2-Aminophenyl)methoxy]phenyl]piperazine-1-carboxylate, <b>24e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 83%, white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.05 (br s, 1 H), 8.30–8.21 (m, 1 H), 7.46–7.39 (m, 1 H), 7.36–7.28 (m, 2 H), 7.17–7.11 (m, 1 H), 7.08–6.87 (m, 5 H), 6.81 (s, 1 H), 5.15 (s, 2 H), 4.12 (s, 3 H), 3.71–3.47 (m, 4 H), 3.17–2.95 (m, 4 H), 1.50 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 547 [M + H]<sup>+</sup>.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]benzoyl]amino]phenoxy]piperidine-1-carboxylate (<b>25f</b>) (from <b>3c</b> and 4-[4-[(2-Aminobenzoyl)amino]phenoxy]piperidine-1-carboxylate, <b>25e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 57%, white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.74 (s, 1 H) 10.44 (s, 1 H) 8.43–8.56 (m, 1 H)7.85–7.99 (m, 1 H) 7.47–7.68 (m, 4 H) 6.90–7.31 (m, 5 H) 4.42–4.60 (m, 1 H) 4.04 (s, 3 H) 3.59–3.72 (m, 2 H) 3.09–3.27 (m, 2 H) 1.77–1.95 (m, 2 H) 1.45–1.60 (m, 2 H) 1.41 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 575 [M + H]<sup>+</sup>.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> <i>tert</i>-Butyl 4-[<i>N</i>-<i>tert</i>-Butoxycarbonyl-4-[[2-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]anilino]piperidine-1-carboxylate (<b>26f</b>) (from <b>3c</b> and <i>tert</i>-Butyl 4-[4-[(2-Aminophenyl)methoxy]-<i>N</i>-<i>tert</i>-butoxycarbonyl-anilino]piperidine-1-carboxylate, <b>26e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 56%, off-white solid. MS (ESI): <i>m</i>/<i>z</i> 661 [M + H]<sup>+</sup>.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> <i>tert</i>-Butyl 3-[[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]methyl]azetidine-1-carboxylate (<b>27f</b>) (from <b>3c</b> and <i>tert</i>-Butyl 3-[[4-[(2-Aminophenyl)methoxy]phenoxy]methyl]azetidine-1-carboxylate, <b>27e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>))</h5><div class="NLM_p last">Yield 31%, pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.05 (s, 1 H), 8.33–8.18 (m, 1 H), 7.47–7.40 (m, 1 H), 7.35–7.29 (m, 2 H), 7.16–7.10 (m, 1 H), 7.02–6.95 (m, 3 H), 6.90–6.85 (m, 2 H), 6.82 (s, 1 H), 5.15 (s, 2 H), 4.13 (s, 3 H), 4.12–4.03 (m, 4 H), 3.83–3.76 (m, 2 H), 3.01–2.90 (m, 1 H), 1.46 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 548 [M + H]<sup>+</sup>.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> <i>tert</i>-Butyl <i>N</i>-[(<i>Z</i>)-<i>N</i>,<i>N</i>′-Bis(<i>tert</i>-butoxycarbonyl)carbamimidoyl]-<i>N</i>-[[4-[[2-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenyl]methyl]carbamate (<b>28f</b>) (from <b>3c</b> and <i>tert</i>-Butyl <i>N</i>-[[4-[(2-Aminophenyl)methoxy]phenyl]methyl]-<i>N</i>-[<i>N</i>,<i>N</i>′-Bis(<i>tert</i>-butoxycarbonyl) carbamimidoyl]carbamate, <b>28e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 72%, white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.25 (br s, 1 H), 9.78 (br s, 1 H), 7.53–7.46 (m, 3 H), 7.38–7.32 (m, 1 H), 7.28–7.21 (m, 3 H), 7.20–7.15 (m, 1 H), 6.94–6.87 (m, 1 H), 5.14 (s, 2 H), 4.62 (s, 2 H), 4.00 (s, 3 H), 1.49–1.29 (m, 27 H). MS (ESI): <i>m</i>/<i>z</i> 734 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-[(2-Nitrophenyl)methoxy]phenol (<b>15i</b>)</h4><div class="NLM_p last">A mixture of 6.6 g (30 mmol) of 1-(bromomethyl)-2-nitro-benzene <b>15g</b> 16.5 g (150 mmol) of hydroquinone (<b>15h</b>) and 6.2 g (45 mmol) of K<sub>2</sub>CO<sub>3</sub> in 110 mL acetone was heated at reflux until disappearance of the starting material. After 7 h, the mixture was cooled down to RT, K<sub>2</sub>CO<sub>3</sub> was filtered off, and the solvent was concentrated. The crude mixture was taken up in DCM triturated, and the filtrate was concentrated and purified on a silica pad eluting with DCM to give 4.87 g (66%) of <b>15i</b> as a beige solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.20–8.14 (m, 1 H), 7.95–7.88 (m, 1 H), 7.72–7.66 (m, 1 H), 7.53–7.45 (m, 1 H), 6.92–6.76 (m, 4 H), 5.44 (s, 2 H), 4.44 (s, 1 H). MS (ESI): <i>m</i>/<i>z</i> 268 [M + Na]<sup>+</sup>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Similarly Prepared</h4><div id="sec4_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 4-[(3-Nitrophenyl)methoxy]phenol (<b>16i</b>) (from 1-(Bromomethyl)-3-nitro-benzene, <b>16g</b>, and Benzene-1,4-diol, <b>15h</b>)</h5><div class="NLM_p last">Yield 71% (1.7 g), pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.99 (s, 1 H), 8.27 (s, 1 H), 8.21–8.14 (m, 1 H), 7.92–7.85 (m, 1 H), 7.73–7.65 (m, 1 H), 6.90–6.80 (m, 2 H), 6.72–6.63 (m, 2 H), 5.15 (s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 244 [M – H]<sup>−</sup>.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 4-[(4-Nitrophenyl)methoxy]phenol (<b>17i</b>) (from 1-(Bromomethyl)-4-nitro-benzene, <b>17g</b>, and Benzene-1,4-diol, <b>15h</b>)</h5><div class="NLM_p last">Yield 61% (1.5 g), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.29–8.21 (m, 2 H), 7.65–7.56 (m, 2 H), 6.90–6.75 (m, 4 H), 5.13 (s, 2 H), 4.45 (s, 1 H). MS (ESI): <i>m</i>/<i>z</i> 244 [M – H]<sup>−</sup>.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 3-[(2-Nitrophenyl)methoxy]phenol (<b>18i</b>) (from 1-(Bromomethyl)-2-nitro-benzene, <b>15g</b>, and Benzene-1,3-diol, <b>18h</b>)</h5><div class="NLM_p last">Yield 77% (4.7 g), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.10 (dd, <i>J</i> = 1.0, 8.3 Hz, 1 H), 7.81 (dd, <i>J</i> = 1.0, 7.8 Hz, 1 H), 7.65–7.59 (m, 1 H), 7.45–7.39 (m, 1 H), 7.1–7.07 (m, 1 H), 6.54–6.49 (m, 1 H), 6.45–6.41 (m, 1 H), 6.42–6.38 (m, 1 H), 5.41 (s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 244 [M – H]<sup>−</sup>.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 2-[(2-Nitrophenyl)methoxy]phenol (<b>19i</b>) (from 1-(Bromomethyl)-2-nitro-benzene, <b>15g</b>, and Benzene-1,2-diol, <b>19h</b>)</h5><div class="NLM_p last">Yield 91% (1.3 g), yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.12 (s, 1 H), 8.17–8.12 (m, 1 H), 7.93–7.89 (m, 1 H), 7.85–7.77 (m, 1 H), 7.65–7.57 (m, 2 H), 6.95–6.93 (m Hz, 2 H), 6.87–6.77 (m, 2 H), 6.75–6.65 (m, 1 H), 5.43 (s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 244 [M – H]<sup>−</sup>.</div></div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>tert</i>-Butyl 4-[4-[(2-Nitrophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>15j</b>)</h4><div class="NLM_p last">First, 0.85 g (4.0 mmol) of diisopropyl azodicarboxylate was added dropwise to a solution of 0.49 g (2.0 mmol) of 4-[(2-nitrophenyl)methoxy]phenol <b>15i</b>, 0.83 g (4.0 mmol) of <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate, and 1.1 g (4.0 mmol) of triphenylphosphine in 20 mL of dry THF cooled down to 0 °C and under a N<sub>2</sub> atmosphere. The mixture was allowed to reach rt and stirred overnight. The solvent was evaporated and the crude mixture purified by flash chromatography (eluent <i>n</i>-hexane/acetone, 0–10% of acetone) to give 0.71 g (83%) of <b>15j</b> as a yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.22–8.13 (m, 1 H), 7.96–7.87 (m, 1 H), 7.74–7.65 (m, 1 H), 7.54–7.44 (m, 1 H), 6.99–6.82 (m, 4 H), 5.45 (s, 2 H), 4.41–4.28 (m, 1 H), 3.78–3.65 (m, 2 H), 3.37–3.24 (m, 2 H), 1.96–1.85 (m, 2 H), 1.79–1.68 (m, 2 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 373 [M + H – 56]<sup>+</sup>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Similarly Prepared</h4><div id="sec4_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 1-Methyl-4-[4-[(2-nitrophenyl)methoxy]phenoxy]piperidine (<b>15d</b>) (from <b>15i</b> and 1-Methyl-4-piperidinol)</h5><div class="NLM_p last">Yield 70% (0.72 g), purified by flash chromatography (eluent DCM:MeOH 98:2 until the elution of Ph<sub>3</sub>PO and then DCM/MeOH/NH<sub>3</sub>, 96:4:0.4 v:v:v) as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.15–8.06 (m, 1 H), 7.84–7.72 (m, 2 H), 7.67–7.55 (m, 1 H), 6.96–6.82 (m, 4 H), 5.37 (s, 2 H), 4.26–4.14 (m, 1 H), 2.60 (br s, 2 H), 2.24–2.05 (m, 5 H), 1.93–1.80 (m, 2 H), 1.66–1.50 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 343 [M + H]<sup>+</sup>.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> <i>tert</i>-Butyl 4-[4-[(3-Nitrophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>16j</b>) (from <b>16i</b> and <i>tert</i>-Butyl 4-Hydroxypiperidine-1-carboxylate)</h5><div class="NLM_p last">Yield 78% (1.34 g), pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.29 (s, 1 H), 8.22–8.15 (m, 1 H), 7.93–7.86 (m, 1 H), 7.75–7.66 (m, 1 H), 7.01–6.86 (m, 4 H), 5.19 (s, 2 H), 4.45–4.34 (m, 1 H), 3.70–3.57 (m, 2 H), 3.21–3.01 (m, 2 H), 1.89–1.77 (m, 2 H), 1.55–1.42 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 373 [M + H – 56]<sup>+</sup>.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>tert</i>-Butyl 4-[4-[(4-Nitrophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>17j</b>) (from <b>17i</b> and <i>tert</i>-Butyl 4-Hydroxypiperidine-1-carboxylate)</h5><div class="NLM_p last">Yield 62% (0.53 g), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.29–8.22 (m, 2 H), 7.65–7.57 (m, 2 H), 6.93–6.82 (m, 4 H), 5.14 (s, 2 H), 4.39–4.29 (m, 1 H), 3.77–3.65 (m, 2 H), 3.37–3.25 (m, 2 H), 1.96–1.84 (m, 2 H), 1.79–1.66 (m, 2 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 373 [M + H – 56]<sup>+</sup>.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> <i>tert</i>-Butyl 4-[3-[(2-Nitrophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>18j</b>) (from <b>18i</b> and <i>tert</i>-Butyl 4-Hydroxypiperidine-1-carboxylate)</h5><div class="NLM_p last">Yield 70% (0.6 g), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.21–8.16 (m, 1 H), 7.93–7.88 (m, 1 H), 7.72–7.67 (m, 1 H), 7.53–7.47 (m, 1 H), 7.24–7.17 (m, 1 H), 6.61–6.55 (m, 3 H), 5.48 (s, 2 H), 4.50–4.41 (m, 1 H), 3.76–3.67 (m, 2 H), 3.39–3.28 (m, 2 H), 1.97–1.86 (m, 2 H), 1.80–1.70 (m, 2 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 451 [M + Na]<sup>+</sup>.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 1-Methyl-4-[2-[(2-nitrophenyl)methoxy]phenoxy]piperidine (<b>19d</b>) (from <b>19i</b>)</h5><div class="NLM_p last">Yield 71% (1.29 g), yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.15–8.11 (m,1 H), 7.91–7.87 (m, 1 H), 7.85–7.77 (m, 2 H), 7.68–7.60 (m, 2 H), 7.09–7.01 (m, 2 H), 6.95–6.85 (m, 2 H), 5.44 (s, 2 H), 4.28 (br s, 1 H), 2.60 (br s, 2 H), 2.16 (br s, 5 H), 1.86 (br s, 2 H), 1.71–1.58 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 343 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1-Methyl-4-[4-[(3-nitrophenyl)methoxy]phenoxy]piperidine (<b>16d</b>)</h4><div class="NLM_p last">Five mL of 4 M HCl in 1,4-dioxane was added to a solution of 0.655 g (1.53 mmol) of <i>tert</i>-butyl 4-[4-[(3-nitrophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>16j</b>) in 4 mL of dry 1,4-dioxane, and the mixture was stirred at rt for 90 min. The reaction mixture was then concentrated, and the residue was taken up in saturated NaHCO<sub>3</sub> solution and extracted with EtOAc. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to afford 0.49 g (98%) of 4-[4-[(3-nitrophenyl)methoxy]phenoxy]piperidine as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.29 (s, 1 H), 8.22–8.14 (m, 1 H), 7.92–7.83 (m, 1 H), 7.73–7.66 (m, 1 H), 6.99–6.82 (m, 4 H), 5.19 (s, 2 H), 4.29–4.16 (m, 1 H), 2.97–2.84 (m, 2 H), 2.57–2.50 (m, 2 H), 1.90–1.79 (m, 2 H), 1.47–1.33 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 329 [M + H]<sup>+</sup>. First, 0.13 mL (1.788 mmol) of formaldehyde (37% in H<sub>2</sub>O), 0.085 mL (1.5 mmol) of glacial acetic acid, and 0.118 g (1.788 mmol) of NaBH<sub>3</sub>CN were added to 0.49 g (1.492 mmol) of 4-[4-[(3-nitrophenyl)methoxy]phenoxy]piperidine in 15 mL of MeOH. The reaction mixture was stirred at rt until disappearance of the starting material. Then the solvent was removed and the crude mixture was partitioned between H<sub>2</sub>O and DCM. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered off, and concentrated to afford a yellow oil. The crude product was purified by flash chromatography (eluent DCM/MeOH/NH<sub>3</sub>, 96:4:0.4 v:v:v) to afford 0.407 g (80%) of <b>16d</b> as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.29 (s, 1 H), 8.22–8.13 (m, 1 H), 7.94–7.83 (m, 1 H), 7.74–7.65 (m, 1 H), 7.03–6.80 (m, 4 H), 5.20 (s, 2 H), 4.38 (br s, 1 H), 3.11–2.64 (m, 4 H), 2.55 (br s, 3 H), 2.04–1.65 (m, 4 H). MS (ESI): <i>m</i>/<i>z</i> 343 [M + H]<sup>+</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Similarly Prepared</h4><div id="sec4_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 1-Methyl-4-[4-[(4-nitrophenyl)methoxy]phenoxy]piperidine (<b>17d</b>) (from <b>17j</b>)</h5><div class="NLM_p last">According to the procedure for <b>16d</b>, except for nitrogen deprotection, which was carried out with TFA in DCM; yield 79% (0.073 g), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.34–8.19 (m, 2 H), 7.68–7.52 (m, 2 H), 6.98–6.79 (m, 4 H), 5.14 (s, 2 H), 4.42–4.17 (m, 1 H), 2.93–2.76 (m, 2 H), 2.58–2.35 (m, 5 H), 2.13–2.02 (m, 2 H), 1.97–1.83 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 343 [M + H]<sup>+</sup>.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 1-Methyl-4-[3-[(2-nitrophenyl)methoxy]phenoxy]piperidine (<b>18d</b>) (from <b>18j</b>)</h5><div class="NLM_p last">Yield 75% (117 mg), yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.16–8.06 (m, 1 H), 7.81–7.75 (m, 2 H), 7.67–7.56 (m, 1 H), 7.23–7.10 (m, 1 H), 6.62–6.51 (m, 3 H), 5.42 (s, 2 H), 4.46–4.30 (m, 1 H), 2.82–2.64 (m, 2 H), 2.38–2.18 (m, 5 H), 1.97–1.83 (m, 2 H), 1.72–1.54 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 343 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> General Procedure to Reduce Nitro Group to Aniline</h3><div class="NLM_p">First, 1.50 mmol of nitro derivative was dissolved in 60 mL of EtOH. Then the solution was hydrogenated in an H-Cube apparatus (PtO<sub>2</sub> cartridge, atmospheric pressure, 70 °C, flow 0.5 mL/min). The solvent was evaporated to give the desired aniline.</div><div id="sec4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Similarly Prepared</h4><div id="sec4_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 2-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline (<b>15e</b>) (from <b>15d</b>)</h5><div class="NLM_p last">Yield 83% (218 mg), off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.19–7.10 (m, 1 H), 7.06–6.98 (m, 1 H), 6.95–6.82 (m, 4 H), 6.71–6.63 (m, 1 H), 6.58–6.48 (m, 1 H), 4.88 (s, 2 H), 4.25–4.09 (m, 1 H), 2.62–2.52 (m, 2 H), 2.21–2.05 (m, 5 H), 1.92–1.80 (m, 2 H), 1.64–1.48 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 3-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline (<b>16e</b>) (from <b>16d</b>)</h5><div class="NLM_p last">Yield 63% (147 mg), pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.03–6.95 (m, 1 H), 6.94–6.83 (m, 4 H), 6.60 (s, 1 H), 6.55–6.44 (m, 2 H), 5.09 (br s, 2 H), 4.86 (s, 2 H), 4.31 (br s, 1 H), 2.91 (br s, 2 H), 2.62 (brs, 2 H), 2.45 (br s, 3 H), 1.92 (br s, 2 H), 1.71 (br s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 4-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline (<b>17e</b>) (from <b>17d</b>)</h5><div class="NLM_p last">Yield 90% (58 mg), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.25–7.18 (m, 2 H), 6.93–6.82 (m, 4 H), 6.74–6.66 (m, 2 H), 4.89 (s, 2 H), 4.22 (br s, 1 H), 3.72 (br s, 2 H), 2.81–2.71 (m, 2 H), 2.43–2.27 (m, 5 H), 2.10–1.96 (m, 2 H), 1.92–1.79 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 2-[[3-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline (<b>18e</b>) (from <b>18d</b>)</h5><div class="NLM_p last">Yield 91% (118 mg), pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.20–7.11 (m, 2 H), 7.05–6.97 (m, 1 H), 6.70–6.64 (m, 1 H), 6.61–6.48 (m, 4 H), 5.04 (br s, 2 H), 4.92 (s, 2 H), 4.37–4.26 (m, 1 H), 2.67–2.55 (m, 2 H), 2.21–2.09 (m, 5 H), 1.94–1.84 (m, 2 H), 1.67–1.51 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 2-[[2-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline (<b>19e</b>) (from <b>19d</b>)</h5><div class="NLM_p last">Yield 75% (175 mg), pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.28–7.24 (m, 1 H), 7.22–7.17 (m, 1 H), 7.12–7.06 (m, 2 H), 7.01–6.93 (m, 2 H), 6.77–6.74 (m, 1 H), 6.64–6.58 (m, 1 H), 5.23 (br s, 2 H), 5.00 (s, 2 H), 4.41–4.30 (m, 1 H), 2.78–2.65 (m, 2 H), 2.25 (br s, 5 H), 2.01–1.89 (m, 2 H), 1.78–1.61 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 313 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> <i>tert</i>-Butyl 4-[4-[(2-Aminophenyl)methoxy]phenoxy]piperidine-1-carboxylate (<b>22e</b>) (from <b>15j</b>)</h5><div class="NLM_p last">Yield 98% (0.59 g), yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.22–7.15 (m, 2 H), 6.97–6.83 (m, 4 H), 6.82–6.75 (m, 2 H), 5.00 (s, 2 H), 4.39–4.26 (m, 1 H), 3.78–3.62 (m, 2 H), 3.36–3.21 (m, 2 H), 1.97–1.82 (m, 2 H), 1.79–1.67 (m, 2 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 399 [M + H]<sup>+</sup>.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 4-[(2-Aminophenyl)methoxy]phenol (<b>33k</b>) (from <b>15i</b>)</h5><div class="NLM_p last">Yield 98% (1.1 g), pale-yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.22–7.14 (m, 2 H), 6.93–6.85 (m, 2 H), 6.82–6.72 (m, 4 H), 4.99 (s, 2 H), 4.32 (bs, 3 H). MS (ESI): <i>m</i>/<i>z</i> 238 [M + Na]<sup>+</sup>.</div></div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> General Procedure for the Synthesis of <b>22</b>–<b>28</b> (BOC-Deprotection)</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 4-Methyl-<i>N</i>-[2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phen-oxy]piperidine-1-carboxylate (<b>22</b>)</h4><div class="NLM_p last">First, 0.407 g (0.725 mmol) of <i>tert</i>-butyl 4-[4-[[2-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>22f</b>) was dissolved in a mixture of 3 mL of dry 1,4-dioxane and 3 mL of dry MeOH. Then 4.17 mL of 4 M HCl in 1,4-dioxane was added and the mixture was stirred at rt for 3 h. The reaction mixture was then concentrated and the residue was taken up in a saturated NaHCO<sub>3</sub> solution and extracted with EtOAc. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated, and triturated in Et<sub>2</sub>O give 0.309 g (92%) of <b>22</b> as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.57–7.46 (m, 3 H), 7.40–7.31 (m, 1 H), 7.29–7.17 (m, 3 H), 6.93–6.79 (m, 4 H), 5.09 (s, 2 H), 4.27–4.13 (m, 1 H), 4.00 (s, 3 H), 2.94–2.84 (m, 2 H), 2.49–2.42 (m, 2 H), 1.89–1.73 (m, 2 H), 1.43–1.30 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Similarly Prepared</h4><div id="sec4_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 4-Methyl-<i>N</i>-[2-[2-[4-(4-piperidyloxy)phenyl]ethyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide, Hydrochloride (<b>23</b>) (from <b>23f</b>)</h5><div class="NLM_p last">The alkaline workup reported for compound <b>22</b> was not performed. Yield 77% (26 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.67 (s, 1 H), 8.51 (br s, 2 H), 7.51 (d, <i>J</i> = 5.4 Hz, 1 H), 7.33–7.27 (m, 3 H), 7.26–7.17 (m, 3 H), 7.10–7.05 (m, 2 H), 6.87–6.82 (m, 2 H), 4.59–4.49 (m, 1 H), 4.00 (s, 3 H), 3.24–3.15 (m, 2 H), 3.10–2.99 (m, 2 H), 2.89–2.72 (m, 4 H), 2.07–1.96 (m, 2 H), 1.81–1.70 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 460 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 4-Methyl-<i>N</i>-[2-[(4-piperazin-1-ylphenoxy)methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>24</b>) (from <b>24f</b>)</h5><div class="NLM_p last">The compound was purified as a free base with silica gel flash chromatography, obtaining 36 mg (yield 47%) pale-orange solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.06 (s, 1 H), 8.29–8.25 (m, 1 H), 7.45–7.39 (m, 1 H), 7.35–7.28 (m, 2 H), 7.16–7.11 (m, 1 H), 7.03–6.90 (m, 5 H), 6.81 (s, 1 H), 5.14 (s, 2 H), 4.13 (s, 3 H), 3.19–3.10 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 447 [M + H]<sup>+</sup>; mp 195–200 °C.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 4-Methyl-<i>N</i>-[2-[[4-(4-piperidyloxy)phenyl]carbamoyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>25</b>) (from <b>25f</b>)</h5><div class="NLM_p last">The compound was purified as a free base with silica gel flash chromatography, yielding 20 mg (48%) of a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.76 (br s, 1 H), 10.43 (br s, 1 H), 8.60–8.38 (m, 1 H), 8.01–7.80 (m, 1 H), 7.71–7.51 (m, 4 H), 7.31–6.87 (m, 5 H), 4.36 (br s, 1 H), 4.04 (s, 3 H), 3.02–2.88 (m, 2 H), 2.64–2.52 (m, 3 H), 1.98–1.82 (m, 2 H), 1.53–1.36 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 475 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> 4-Methyl-<i>N</i>-[2-[[4-(4-piperidylamino)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Trifluoroacetate (<b>26</b>) (from <b>26f</b>)</h5><div class="NLM_p last">The alkaline workup reported for compound <b>22</b> was not performed, and the compound was purified as trifluoroacetate preparative HPLC, yielding 7 mg (35%) of brownish oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.36 (s, 1 H), 9.28–8.79 (m, 2 H), 8.53 (d, <i>J</i> = 9.8 Hz, 1 H), 8.15–7.95 (m, 1 H), 7.80–7.68 (m, 1 H), 7.54 (d, <i>J</i> = 5.4 Hz, 1 H), 7.32–7.15 (m, 4 H), 7.07–7.00 (m, 1 H), 6.90–6.76 (m, 2 H), 6.64–6.53 (m, 2 H), 4.33 (br s, 2 H), 4.03 (s, 3 H), 3.65–3.54 (m, 1 H), 3.32–3.22 (m, 2 H), 2.98–2.85 (m, 2 H), 2.01–1.88 (m, 2 H), 1.61–1.46 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 461 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>N</i>-[2-[[4-(Azetidin-3-ylmethoxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Trifluoroacetate (<b>27</b>) (from <b>27f</b>)</h5><div class="NLM_p last">The alkaline workup reported for compound <b>22</b> was not performed. The compound was purified as trifluoroacetate preparative HPLC, yielding 53 mg (65%) of beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.74 (s, 1 H), 8.52 (br s, 2 H), 7.56–7.47 (m, 3 H), 7.40–7.31 (m, 1 H), 7.29–7.21 (m, 3 H), 6.97–6.83 (m, 4 H), 5.11 (s, 2 H), 4.09–3.96 (m, 7 H), 3.85–3.76 (m, 2 H), 3.21–3.07 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 448 [M + H]<sup>+</sup>; mp 165–168 °C.</div></div><div id="sec4_10_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> <i>N</i>-[2-[[4-(Guanidinomethyl)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Trifluoroacetate (<b>28</b>) (from <b>28f</b>)</h5><div class="NLM_p last">TFA was used instead of HCl in dioxane, and the compound was purified as trifluoroacetate by preparative HPLC, yielding 27 mg (60%) of beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.80 (s, 1 H), 7.77 (t, <i>J</i> = 5.8 Hz, 1 H), 7.55–7.45 (m, 3 H), 7.39–7.33 (m, 1 H), 7.30–7.23 (m, 3 H), 7.22–7.17 (m, 2 H), 6.98 (d, <i>J</i> = 8.2 Hz, 2 H), 5.16 (s, 2 H), 4.24 (d, <i>J</i> = 6.0 Hz, 2 H), 4.00 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 434 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_10_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Synthesis of <i>N</i>-[2-[[4-[(1-Ethyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>29</b>)</h4><div class="NLM_p last">First, 0.009 g (0.20 mmol) of acetaldehyde was added under inert atmosphere to a solution of 0.078 g (0.17 mmol) of 4-methyl-<i>N</i>-[2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>22</b>) in 1.3 mL of 1,2-dichloroethane. Then the mixture was stirred at rt for 2 h, then 0.060 g (0.27 mmol) of NaBH(OAc)<sub>3</sub> was added and the mixture was stirred overnight. The reaction was stopped by addition of a saturated NaHCO<sub>3</sub> solution, and the product was extracted with DCM. The organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and filtered, and the solvent was removed in vacuo. The crude mixture was purified by flash chromatography (eluent DCM/MeOH/NH<sub>3</sub> 96:4:0.4, v:v:v) to afford 21 mg (25%) of <b>29</b> as a beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.99 (s, 1 H), 8.36–8.22 (m, 1 H), 7.47–7.39 (m, 1 H), 7.36–7.29 (m, 2 H), 7.18–7.11 (m, 1 H), 7.04–6.96 (m, 3 H), 6.91–6.85 (m, 2 H), 6.81 (s, 1 H), 5.14 (s, 2 H), 4.60 (bs, 1 H), 4.13 (s, 3 H), 3.41 (bs, 2 H), 3.09 (bs, 4 H), 2.60 (bs, 2 H), 2.19 (bs, 2 H), 1.47 (bs, 3 H). MS (ESI): <i>m</i>/<i>z</i> 490 [M + H]<sup>+</sup>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Similarly Prepared</h4><div id="sec4_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> <i>N</i>-[2-[[4-[(1-Benzyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>30</b>) (from <b>22</b> and Benzaldehyde)</h5><div class="NLM_p last">Yield 57% (41 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.71 (s, 1 H), 7.56–7.47 (m, 3 H), 7.40–7.16 (m, 9 H), 6.93–6.75 (m, 4 H), 5.09 (s, 2 H), 4.24–4.15 (m, 1 H), 3.99 (s, 3 H), 3.45 (s, 2 H), 2.70–2.57 (m, 2 H), 2.23–2.10 (m, 2 H), 1.93–1.79 (m, 2 H), 1.64–1.49 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 552 [M + H]<sup>+</sup>; mp 193–197 °C.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> N-[2-[[4-[(1-Acetyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>31</b>)</h5><div class="NLM_p last">First, 0.046 g (0.37 mmol) of a solution of <i>N</i>,<i>N</i>-dimethylpyridin-4-amine in dry THF (1 mL) was added dropwise to a solution of 0.069 g (0.1495 mmol) of 4-methyl-<i>N</i>-[2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>22</b>) and 0.02347 g (0.2990 mmol0 of acetyl chloride in THF (2 mL). Then the reaction was stirred at room temperature for 2 h. The mixture was concentrated in vacuo, and the residue was purified by flash chromatography, eluting with DCM/MeOH 98.5/1.5 to afford 71 mg (94%) of <i>N</i>-[2-[[4-[(1-acetyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.56–7.46 (m, 3 H), 7.40–7.32 (m, 1 H), 7.28–7.17 (m, 3 H), 6.95–6.82 (m, 4 H), 5.10 (s, 2 H), 4.48–4.37 (m, 1 H), 4.00 (s, 3 H), 3.85–3.73 (m, 1 H), 3.69–3.57 (m, 1 H), 3.31–3.22 (m, 1 H), 3.21–3.12 (m, 1 H), 1.99 (s, 3 H), 1.93–1.75 (m, 2 H), 1.60–1.50 (m, 1 H), 1.48–1.37 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 504 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>N</i>-[2-[(4-Hydroxyphenoxy)methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>33l</b>)</h5><div class="NLM_p last">First, 0.31 g (2.6 mmol, 0.19 mL) of SOCl<sub>2</sub> and 3 drops of dry DMF were added at rt to a solution of 0.360 g (2.00 mmol) of 4-methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic acid <b>3c</b> in 8 mL of dry THF. Then the mixture was stirred for 2 h at reflux. The reaction mixture was then cooled down to rt and poured into a solution of 0.384 g (1.78 mmol) of 4-[(2-aminophenyl)methoxy]phenol (<b>33k</b>) in 6 mL of pyridine. The mixture was stirred for 10 min at RT, then quenched with water and concentrated. The residue was dissolved in DCM, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent evaporated to give an oil which was purified by flash chromatography (<i>n</i>-hexane/acetone, 0–20% of acetone) to give 416 mg (62%) <b>33l</b> as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 8.93 (s, 1 H), 7.57–7.47 (m, 3 H), 7.39–7.31 (m, 1 H), 7.27–7.20 (m, 3 H), 6.82–6.76 (m, 2 H), 6.67–6.59 (m, 2 H), 5.05 (s, 2 H), 4.00 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 379 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> General Procedure for the Synthesis of <b>33f</b>–<b>43f</b> (Mitzunobu Reaction)</h3><div class="NLM_p">First, 0.075 g (0.35 mmol) of diisopropyl azodicarboxylate was added dropwise to a solution of 0.089 g (0.24 mmol) of <i>N</i>-[2-[(4-hydroxyphenoxy)methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>33l</b>), 0.35 mmol of the required alchol, and 0.094 g (0.35 mmol) of triphenylphosphine in 3 mL of dry THF cooled down to 0 °C and under N<sub>2</sub> atmosphere. Then the reaction mixture was allowed to reach rt and was stirred overnight. The solvent was evaporated and the crude mixture was purified by flash chromatography (eluent <i>n</i>-hexane/acetone, from 0% to 11% of acetone). Trituration in Et<sub>2</sub>O provided the desired compound.</div><div id="sec4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Similarly Prepared</h4><div id="sec4_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> <i>tert</i>-Butyl (3<i>R</i>)-3-[[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>33f</b>) (from <b>33l</b> and <i>tert</i>-Butyl (3<i>R</i>)-3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 73% (79 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.03 (s, 1 H), 8.32–8.17 (m, 1 H), 7.48–7.38 (m, 1 H), 7.34–7.29 (m, 2 H), 7.17–7.09 (m, 1 H), 7.01–6.95 (m, 3 H), 6.89–6.84 (m, 1 H), 6.82 (s, 1 H), 5.14 (s, 2 H), 4.13 (s, 3 H), 3.92–3.84 (m, 2 H), 3.66–3.56 (m, 1 H), 3.42–3.32 (m, 2 H), 3.25–3.17 (m, 1 H), 3.16–3.08 (m, 1 H), 2.73–2.61 (m, 1 H), 2.13–2.03 (m, 1 H), 1.88–1.75 (m, 1 H), 1.47 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 562 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> <i>tert</i>-Butyl 3-[[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>34f</b>) (from <b>33l</b> and <i>tert</i>-Butyl 3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 34% (123 mg), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.05 (s, 1 H), 8.34–8.17 (m, 1 H), 7.45–7.40 (m, 1 H), 7.34–7.29 (m, 2 H), 7.16–7.11 (m, 1 H), 7.02–6.95 (m, 3 H), 6.89–6.84 (m, 2 H), 6.82 (s, 1 H), 5.14 (s, 2 H), 4.13 (s, 3 H), 3.94–3.82 (m, 2 H), 3.65–3.55 (m, 1 H), 3.54–3.43 (m, 1 H), 3.42–3.32 (m, 1 H), 3.27–3.16 (m, 1 H), 2.74–2.60 (m, 1 H), 2.13–2.02 (m, 1 H), 1.86–1.74 (m, 1 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 562 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]azepane-1-carboxylate, (<b>35f</b>) (from <b>33l</b> and <i>tert</i>-Butyl 4-Hydroxyazepane-1-carboxylate)</h5><div class="NLM_p last">Yield 25% (37 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.07 (s, 1 H), 8.30–8.25 (m, 1 H), 7.46–7.39 (m, 1 H), 7.34–7.29 (m, 2 H), 7.16–7.11 (m, 1 H), 7.00–6.96 (m, 3 H), 6.88–6.82 (m, 3 H), 5.14 (s, 2 H), 4.39–4.31 (m, 1 H), 4.13 (s, 3 H), 3.64–3.25 (m, 4 H), 2.10–1.84 (m, 5 H), 1.70–1.59 (m, 1 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 598 [M + Na]<sup>+</sup>.</div></div><div id="sec4_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> <i>tert</i>-Butyl <i>N</i>-[4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]cyclohexyl]carbamate (<b>36f</b>) (from <b>33l</b> and <i>tert</i>-Butyl <i>N</i>-(4-Hydroxycyclohexyl)carbamate)</h5><div class="NLM_p last">Yield 52% (70 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.04 (s, 1 H), 8.32–8.20 (m, 1 H), 7.47–7.39 (m, 1 H), 7.34–7.28 (m, 2 H), 7.17–7.10 (m, 1 H), 7.01–6.95 (m, 3 H), 6.90–6.85 (m, 2 H), 6.83 (s, 1 H), 5.14 (s, 2 H), 4.40–4.32 (m, 1 H), 4.13 (s, 3 H), 3.50–3.49 (m, 1 H), 2.04–1.91 (m, 2 H), 1.82–1.72 (m, 2 H), 1.70–1.59 (m, 6 H), 1.46 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 576 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> <i>tert</i>-Butyl (1<i>S</i>,5<i>R</i>)-3-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>37f</b>) (from <b>33l</b> and <i>tert</i>-Butyl (1<i>R</i>,5<i>S</i>)-3-Hydroxy-8-azabicyclo[3.2.1]octane-8-carboxylate)</h5><div class="NLM_p last">Yield 60% (82 mg), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.05 (s, 1 H), 8.31–8.22 (m, 1 H), 7.46–7.39 (m, 1 H), 7.35–7.29 (m, 2 H), 7.16–7.11 (m, 1 H), 7.00–6.93 (m, 3 H), 6.92–6.86 (m, 2 H), 6.83 (s, 1 H), 5.14 (s, 2 H), 4.60–4.49 (m, 1 H), 4.13 (s, 3 H), 2.13–1.97 (m, 4 H), 1.86–1.63 (m, 4 H), 1.62–1.58 (m, 2 H), 1.49 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 588 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> <i>tert</i>-Butyl 4-[[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]-phenoxy]methyl]piperidine-1-carboxylate (<b>38f</b>) (from <b>33l</b> and <i>tert</i>-Butyl 4-(Hydroxymethyl)piperidine-1-carboxylate)</h5><div class="NLM_p last">Yield 59% (80 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.07 (s, 1 H), 8.34–8.18 (m, 1 H), 7.46–7.39 (m, 1 H), 7.34–7.28 (m, 2 H), 7.16–7.10 (m, 1 H), 7.01–6.95 (m, 3 H), 6.88–6.80 (m, 3 H), 5.14 (s, 2 H), 4.21–4.10 (m, 5 H), 3.77 (d, <i>J</i> = 6.4 Hz, 2 H), 2.81–2.70 (m, 2 H), 2.01–1.89 (m, 1 H), 1.87–1.79 (m, 2 H), 1.47 (s, 9 H), 1.26–1.19 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 576 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> <i>tert</i>-Butyl (3<i>S</i>)-3-[[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>39f</b>) (from <b>33l</b> and <i>tert</i>-Butyl (3<i>S</i>)-3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 58% (63 mg), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.04 (s, 1 H), 8.35–8.19 (m, 1 H), 7.45–7.39 (m, 1 H), 7.34–7.28 (m, 2 H), 7.17–7.10 (m, 1 H), 7.02–6.94 (m, 3 H), 6.90–6.84 (m, 2 H), 6.82 (s, 1 H), 5.14 (s, 2 H), 4.13 (s, 3 H), 3.93–3.83 (m, 2 H), 3.53–3.41 (m, 1 H), 3.40–3.28 (m, 1 H), 3.26–3.18 (m, 1 H), 3.16–3.08 (m, 1 H), 2.71–2.62 (m, 1 H), 2.12–2.03 (m, 1 H), 1.86–1.74 (m, 1 H), 1.47 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 562 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> <i>tert</i>-Butyl <i>N</i>-[4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]cyclohexyl]carbamate (<b>40f</b>) (from <b>33l</b> and <i>tert</i>-Butyl <i>N</i>-(4-Hydroxycyclohexyl)carbamate)</h5><div class="NLM_p last">Yield 60% (81 mg), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.06 (s, 1 H), 8.34–8.18 (m, 1 H), 7.45–7.40 (m, 1 H), 7.35–7.29 (m, 2 H), 7.16–7.10 (m, 1 H), 7.00–6.94 (m, 3 H), 6.90–6.85 (m, 2 H), 6.82 (s, 1 H), 5.13 (s, 2 H), 4.13 (s, 3 H), 4.11–4.03 (m, 1 H), 2.16–2.04 (m, 4 H), 1.73–1.58 (m, 6 H), 1.46 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 576 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> <i>tert</i>-Butyl 3-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>41f</b>) (from <b>33l</b> and <i>tert</i>-Butyl 3-Hydroxypiperidine-1-carboxylate)</h5><div class="NLM_p last">Yield 89% (117 mg), pale-yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.08 (s, 1 H), 8.34–8.16 (m, 1 H), 7.45–7.39 (m, 1 H), 7.34–7.28 (m, 2 H), 7.16–7.10 (m, 1 H), 7.00–6.95 (m, 3 H), 6.92–6.87 (m, 2 H), 6.85 (s, 1 H), 5.14 (s, 2 H), 4.19–4.10 (m, 4 H), 3.24–3.02 (m, 2 H), 1.93–1.67 (m, 4 H), 1.56–1.44 (m, 11 H). MS (ESI): <i>m</i>/<i>z</i> 584 [M + Na]<sup>+</sup>.</div></div><div id="sec4_13_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> <i>tert</i>-Butyl 3-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]pyrrolidine-1-carboxylate (<b>42f</b>) (from <b>33l</b> and <i>tert</i>-Butyl 3-Hydroxypyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 76% (99 mg), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.05 (s, 1 H), 8.33–8.14 (m, 1 H), 7.47–7.39 (m, 1 H), 7.35–7.28 (m, 2 H), 7.17–7.11 (m, 1 H), 7.02–6.95 (m, 3 H), 6.89–6.80 (m, 3 H), 5.14 (s, 2 H), 4.86–4.77 (m, 1 H), 4.13 (s, 3 H), 3.67–3.41 (m, 4 H), 2.23–1.87 (m, 2 H), 1.48 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 548 [M + H]<sup>+</sup>.</div></div><div id="sec4_13_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> <i>tert</i>-Butyl <i>N</i>-methyl-<i>N</i>-[3-[4-[[2-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]propyl]carbamate (<b>43f</b>) (from <b>33l</b> and <i>tert</i>-Butyl <i>N</i>-(3-Hydroxypropyl)-<i>N</i>-methyl-carbamate)</h5><div class="NLM_p last">Yield 58% (47 mg), ochre solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.09 (s, 1 H), 8.31–8.25 (m, 1 H), 7.45–7.38 (m, 1 H), 7.34–7.28 (m, 2 H), 7.16–7.10 (m, 1 H), 7.02–6.95 (m, 3 H), 6.88–6.81 (m, 3 H), 5.14 (s, 2 H), 4.13 (s, 3 H), 3.94 (t, <i>J</i> = 6.1 Hz, 2 H), 3.41 (t, <i>J</i> = 7.1 Hz, 2 H), 2.89 (s, 3 H), 2.04–1.95 (m, 2 H), 1.44 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 550 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> General Procedure for the Synthesis of <b>33</b>–<b>43</b> (BOC-Deprotection)</h3><div id="sec4_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 4- Methyl-<i>N</i>-[2-[[4-(4-piperidylmethoxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>38</b>)</h4><div class="NLM_p last">First, 0.14 mmol of the boc-protected derivative was dissolved in 1 mL of dry 1,4-dioxane. Then 0.80 mL of 4 M HCl in 1,4-dioxane was added and the mixture was stirred at rt for 2 h. The mixture was then concentrated, and the residue was taken up in a saturated NaHCO<sub>3</sub> solution and extracted with DCM. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, the solvent removed, and the product was purified by silica gel flash chromatography (eluent DCM/MeOH/NH<sub>3</sub>, 94:6:0.6, v:v:v). Trituration in Et<sub>2</sub>O provided the desired compounds</div></div><div id="sec4_14_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Similarly Prepared</h4><div id="sec4_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> 4-Methyl-<i>N</i>-[2-[[4-[[(3<i>R</i>)-pyrrolidin-3-yl]methoxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>33</b>) (from <b>33f</b>)</h5><div class="NLM_p last">Yield 62% (40 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.56–7.46 (m, 3 H), 7.39–7.33 (m, 1 H), 7.28–7.20 (m, 3 H), 6.94–6.78 (m, 4 H), 5.09 (s, 2 H), 4.00 (s, 3 H), 3.82–3.68 (m, 2 H), 2.90–2.77 (m, 2 H), 2.73–2.66 (m, 1 H), 2.60–2.54 (m, 1 H), 2.41–2.33 (m, 1 H), 1.85–1.75 (m, 1 H), 1.43–1.30 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>.</div></div><div id="sec4_14_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 4-Methyl-<i>N</i>-[2-[[4-(pyrrolidin-3-ylmethoxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>34</b>) (from <b>34f</b>)</h5><div class="NLM_p last">Yield 40% (26 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.57–7.46 (m, 3 H), 7.40–7.31 (m, 1 H), 7.29–7.18 (m, 3 H), 6.95–6.75 (m, 4 H), 5.09 (s, 2 H), 4.00 (s, 3 H), 3.81–3.68 (m, 2 H), 2.88–2.74 (m, 2 H), 2.72–2.61 (m, 1 H), 2.59–2.53 (m, 1 H), 2.43–2.30 (m, 1 H), 1.85–1.73 (m, 1 H), 1.42–1.29 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>.</div></div><div id="sec4_14_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> <i>N</i>-[2-[[4-(Azepan-4-yloxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>35</b>) (from <b>35f</b>)</h5><div class="NLM_p last">The alkaline workup reported for compound <b>38f</b> was not performed. Yield 51% (33 mg), off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.75 (s, 1 H), 8.86 (br s, 2 H), 7.21–7.55 (m, 7 H), 6.82–6.94 (m, 4 H), 5.10 (s, 2 H), 4.55–4.48 (m, 1 H), 4.00 (s, 3 H), 3.01–3.23 (m, 4 H), 1.63–2.14 (m, 6 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>.</div></div><div id="sec4_14_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> <i>N</i>-[2-[[4-(4-Aminocyclohexoxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>36</b>) (from <b>36f</b>)</h5><div class="NLM_p last">Yield 26% (15 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.59–7.47 (m, 3 H), 7.42–7.32 (m, 1 H), 7.29–7.15 (m, 3 H), 6.95–6.76 (m, 4 H), 5.09 (s, 2 H), 4.29 (br s, 1 H), 4.00 (s, 3 H), 2.67 (bs, 1 H), 1.93–1.30 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 147–150 °C.</div></div><div id="sec4_14_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> <i>N</i>-[2-[[4-[[(1<i>S</i>,5<i>R</i>)-8-Azabicyclo[3.2.1]octan-3-yl]oxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>37</b>) (from <b>37f</b>)</h5><div class="NLM_p last">Yield 80% (50 mg), off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.56–7.47 (m, 3 H), 7.40–7.32 (m, 1 H), 7.29–7.19 (m, 3 H), 6.91–6.78 (m, 4 H), 5.09 (s, 2 H), 4.45–4.33 (m, 1 H), 4.00 (s, 3 H), 3.45–3.39 (m, 2 H), 1.98–1.34 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 488 [M + H]<sup>+</sup>; mp 163–167 °C.</div></div><div id="sec4_14_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> 4-Methyl-<i>N</i>-[2-[[4-(4-piperidylmethoxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>38</b>) (from <b>38f</b>)</h5><div class="NLM_p last">Yield 63% (41 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.56–7.46 (m, 3 H), 7.39–7.31 (m, 1 H), 7.29–7.18 (m, 3 H), 6.93–6.77 (m, 4 H), 5.09 (s, 2 H), 4.00 (s, 3 H), 3.68 (d, <i>J</i> = 6.4 Hz, 2 H), 2.96–2.85 (m, 2 H), 2.46–2.39 (m, 2 H), 1.80–1.68 (m, 1 H), 1.67–1.58 (m, 2 H), 1.16–1.01 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 153–156 °C.</div></div><div id="sec4_14_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> 4-Methyl-<i>N</i>-[2-[[4-(4-piperidylmethoxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>39</b>) (from <b>39f</b>)</h5><div class="NLM_p last">Yield 37% (19 mg), off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.55–7.47 (m, 3 H), 7.39–7.32 (m, 1 H), 7.28–7.20 (m, 3 H), 6.93–6.78 (m, 4 H), 5.10 (s, 2 H), 4.02–3.97 (m, 3 H), 3.83–3.71 (m, 2 H), 2.96–2.83 (m, 2 H), 2.80–2.72 (m, 1 H), 2.67–2.59 (m, 1 H), 2.46–2.38 (m, 1 H), 1.89–1.78 (m, 1 H), 1.48–1.36 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>; mp 115–120 °C dec.</div></div><div id="sec4_14_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> <i>N</i>-[2-[[4-(4-Aminocyclohexoxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>40</b>) (from <b>40f</b>)</h5><div class="NLM_p last">Yield 45% (30 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.57–7.46 (m, 3 H), 7.40–7.32 (m, 1 H), 7.28–7.19 (m, 3 H), 6.92–6.74 (m, 4 H), 5.09 (s, 2 H), 4.12–4.02 (m, 1 H), 4.00 (s, 3 H), 2.67–2.54 (m, 1 H), 2.11–1.02 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>.</div></div><div id="sec4_14_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i121" class="anchor-spacer"></div><h5 class="article-section__title" id="_i121"> 4-Methyl-<i>N</i>-[2-[[4-(3-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>41</b>) (from <b>41f</b>)</h5><div class="NLM_p last">Yield 59% (55 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.72 (s, 1 H), 7.56–7.47 (m, 3 H), 7.40–7.32 (m, 1 H), 7.29–7.19 (m, 3 H), 6.93–6.78 (m, 4 H), 5.09 (s, 2 H), 4.09–4.02 (m, 1 H), 4.00 (s, 3 H), 3.07–2.97 (m, 1 H), 2.77–2.67 (m, 1 H), 2.46–2.38 (m, 2 H), 2.01–1.91 (m, 1 H), 1.67–1.57 (m, 1 H), 1.46–1.33 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>; mp 154–157 °C.</div></div><div id="sec4_14_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> 4-Methyl-<i>N</i>-[2-[(4-pyrrolidin-3-yloxyphenoxy)methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>42</b>) (from <b>42f</b>)</h5><div class="NLM_p last">Yield 43% (33 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.73 (s, 1 H), 7.56–7.46 (m, 3 H), 7.40–7.31 (m, 1 H), 7.29–7.18 (m, 3 H), 6.93–6.74 (m, 4 H), 5.09 (s, 2 H), 4.79–4.66 (m, 1 H), 4.00 (s, 3 H), 3.04–2.94 (m, 1 H), 2.92–2.69 (m, 3 H), 1.99–1.85 (m, 1 H), 1.76–1.63 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 448 [M + H]<sup>+</sup>; mp 158–162 °C.</div></div><div id="sec4_14_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> 4-Methyl-<i>N</i>-[2-[[4-[3-(methylamino)propoxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide hydrochloride (<b>43</b>) (from <b>43f</b>)</h5><div class="NLM_p last">The alkaline workup reported for compound <b>38f</b> was not performed. Yield 58% (47 mg), ochre solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.65 (br s, 2 H), 9.01 (s, 1 H), 8.29–8.22 (m, 1 H), 7.45–7.38 (m, 1 H), 7.32–7.28 (m, 2 H), 7.15–7.09 (m, 1 H), 7.00–6.92 (m, 3 H), 6.89–6.84 (m, 2 H), 6.80 (s, 1 H), 5.10 (s, 2 H), 4.12 (s, 3 H), 4.07–4.01 (m, 2 H), 3.26–3.16 (m, 2 H), 2.74 (s, 3 H), 2.43–2.32 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 450 [M + H]<sup>+</sup>; mp 184–188 °C.</div></div></div><div id="sec4_14_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> Methyl 2-(Methoxymethyl)-6-nitro-benzoate (<b>45v</b>)</h4><div class="NLM_p last">First, 1.621 g (9.05 mmol) of 7-nitro-3<i>H</i>-isobenzofuran-1-one was suspended in 9.5 mL of 1 M NaOH and 5 mL of THF. Then the mixture was heated at 50 °C for 30 min and the solution evaporated under vacuum to give sodium 2-(hydroxymethyl)-6-nitro-benzoate (<b>45n</b>) as a white powder. Then 0.43 g (11 mmol) of 60% NaH (mineral oil) were added to a suspension of <b>45n</b> in DMA at 0 °C, and the mixture was allowed to spontaneously warm to rt. After 30 min, 3.85 g (27.2 mmol) of iodomethane were added at 0 °C and the mixture was stirred at rt overnight. Water was added, and the mixture extracted with EtOAc, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to give 2.25 g of crude which was purified by silica gel column chromatography (eluent <i>n</i>-hexane/EtOAc) to yield 971 mg (48%) of <b>45v</b> as a white powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.15–7.71 (m, 3 H), 4.51 (s, 2 H), 3.84 (s, 3 H), 3.26 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 248 [M + 23]<sup>+</sup>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> Similarly Prepared</h4><div id="sec4_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> Ethyl 2-(Ethoxymethyl)-6-nitro-benzoate (<b>50v</b>) (from <b>45n</b> and Iodoethane)</h5><div class="NLM_p last">Yield 46% (131 mg), pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.14–7.70 (m, 3 H), 4.55 (s, 2 H), 4.34–4.27 (m, 2 H), 3.46–3.40 (m, 2 H), 1.31–1.06 (m, 6 H).</div></div></div><div id="sec4_15_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> [2-(Methoxymethyl)-6-nitro-phenyl]methanol (<b>45w</b>)</h4><div class="NLM_p last">A solution of 0.84 g (3.7 mmol) of methyl 2-(methoxymethyl)-6-nitro-benzoate (<b>45v</b>) in 3 mL of dry THF was added to a stirred suspension of 9.3 mL (18.6 mmol) of 2 M lithium borohydride in THF at 0 °C under nitrogen, followed by 2 mL of dry MeOH, and the mixture was stirred 3 h at rt. The reaction was cooled to 0 °C, treated with a saturated aqueous NH<sub>4</sub>Cl solution, and extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to give the crude product which was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc) to yield 589 mg (80%) of <b>45w</b> as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.75–7.63 (m, 2 H), 7.52–7.45 (m, 1 H), 5.29–5.24 (m, 1 H), 4.64–4.61 (m, 2 H), 4.58 (s, 2 H), 3.33 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 198 [M + H]<sup>+</sup>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> Similarly Prepared</h4><div id="sec4_16_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> [2-(Ethoxymethyl)-6-nitro-phenyl]methanol (<b>50w</b>) (from <b>50v</b>)</h5><div class="NLM_p last">Yield 45% (48 mg), pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.75–7.45 (m, 3 H), 5.28–3.23 (m, 1 H), 4.67–4.57 (m, 4 H), 3.55–3.49 (m, 2 H), 1.19–1.13 (m, 3 H).</div></div></div><div id="sec4_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> 2-(Bromomethyl)-1-(methoxymethyl)-3-nitro-benzene (<b>45g</b>)</h4><div class="NLM_p last">To a stirred solution of 556 mg (2.82 mmol) of [2-(methoxymethyl)-6-nitro-phenyl]methanol (<b>45w</b>) in 1.3 mL of dry acetonitrile, 1.4 g (4.2 mmol) of carbon tetrabromide and 1.3 g (5.1 mmol) of triphenylphosphine were added and the mixture stirred at rt for 1 h. Water was added, solvent evaporated, and the aqueous mixture extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to give the crude which was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc) to yield 623 mg (85%) of <b>45g</b> as a pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.98–7.75 (2m, 2 H), 7.65–7.59 (m, 1 H), 4.81 (s, 2 H), 4.66 (s, 2 H), 3.33 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 261 [M + H]<sup>+</sup>.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> Similarly Prepared</h4><div id="sec4_17_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> 2-(Bromomethyl)-1-(ethoxymethyl)-3-nitro-benzene (<b>50g</b>) (from <b>50w</b>)</h5><div class="NLM_p last">Yield 67% (41 mg), yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.96–7.56 (m, 3 H), 4.81 (s, 2 H), 4.69 (s, 2 H), 3.59–3.52 (m, 2 H), 1.21–1.16 (m, 3 H).</div></div></div><div id="sec4_17_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenol (<b>45l</b>)</h4><div class="NLM_p last">First, 0.620 g (2.39 mmol) of 2-(bromomethyl)-1-(methoxymethyl)-3-nitro-benzene (<b>45g</b>), 1.31 g (11.93 mmol) of hydroquinone, and 0.49 g (3.6 mmol) of K<sub>2</sub>CO<sub>3</sub> were suspended in 10 mL of acetone and refluxed in a sealed vial for 2.5 h. Solvent was removed under vacuum, and the residue was portioned between DCM and water. The aqueous phase was extracted with DCM, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to give the crude (700 mg) which was purified by flash chromatography (eluent DCM/MeOH 99:1) to yield 519 mg (79%) of <b>45l</b> as a dark oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.01 (s, 1 H), 7.87–7.71 (m, 2 H), 7.65–7.54 (m, 1 H), 6.81–6.62 (2m, 4 H), 5.11 (s, 2 H), 4.60 (s, 2 H), 3.32 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 290 [M + H]<sup>+</sup>.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> Similarly Prepared</h4><div id="sec4_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i135" class="anchor-spacer"></div><h5 class="article-section__title" id="_i135"> 4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenol (<b>50l</b>) (from <b>50g</b>)</h5><div class="NLM_p last">Yield 55% (24 mg), yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.01 (s, 1 H), 7.88–7.53 (m, 3 H), 6.79–6.61 (m, 4 H), 5.11 (s, 2 H), 4.63 (s, 2 H), 3.46–3.54 (m, 2 H), 1.17–1.11 (m, 3 H).</div></div></div><div id="sec4_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> <i>tert</i>-Butyl 4-[4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>45d</b>)</h4><div class="NLM_p last">First, 0.14 g (0.52 mmol) of triphenyl phosphine were added to a stirred solution of 0.1 g (0.346 mmol) of 4-[[2-(methoxymethyl)-6-nitro-phenyl]methoxy]phenol (<b>45l</b>) and 0.10 g (0.52 mmol) of <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate in 2 mL of dry THF under nitrogen, at rt. The mixture was cooled to 0 °C, 0.10 g (0.52 mmol) of diisopropyl azodicarboxylate were added, and the reaction stirred at rt overnight. Solvent was removed, and the residue was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc) to give 237 mg (80%) of <b>45d</b> as a colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.88–7.59 (3m, 3 H), 6.96–6.81 (m, 4 H), 5.15 (s, 2 H), 4.62 (s, 2 H), 4.46–4.37 (m, 1 H), 3.69–3.57 (m, 2 H), 3.29 (s, 3 H), 3.22–3.09 (m, 2 H), 1.92–1.44 (2m, 4 H), 1.40 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 473 [M + H]<sup>+</sup>.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> Similarly Prepared</h4><div id="sec4_19_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> <i>tert</i>-Butyl (3<i>R</i>)-3-[[4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>46d</b>) (from <b>45l</b> and <i>tert</i>-Butyl (3<i>R</i>)-3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">The compound was obtained as a 4:5 mixture with isopropyl <i>N</i>-(isopropoxycarbonylamino)carbamate (65% purity, 61% calculated yield) and used in the next step without any further purification. MS (ESI): <i>m</i>/<i>z</i> 474 [M + H]<sup>+</sup>.</div></div><div id="sec4_19_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> <i>tert</i>-Butyl 4-[4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]azepane-1-carboxylate (<b>47d</b>) (from <b>45l</b> and <i>tert</i>-Butyl 4-Hydroxyazepane-1-carboxylate)</h5><div class="NLM_p last">The compound was used in the next step without any further purification (quantitative yield), sticky yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.88–7.52 (m, 3 H), 6.90–6.82 (m, 4 H), 5.15 (s, 2 H), 4.61 (s, 2 H), 4.38 (m, 1 H), 3.44–3.20 (m, 4 H), 3.33 (s, 3H), 1.99–1.50 (m, 6 H), 1.40 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 487 [M + H]<sup>+</sup>.</div></div><div id="sec4_19_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i140" class="anchor-spacer"></div><h5 class="article-section__title" id="_i140"> <i>tert</i>-Butyl (3<i>S</i>)-3-[[4-[[2-(Methoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>48d</b>) (from <b>45l</b><i>tert</i>-Butyl (3<i>S</i>)-3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 49% (80 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.87–7.55 (m, 3 H), 6.91–6.84 (m, 4 H), 5.15 (s, 2 H), 4.61 (s, 2 H), 3.93–3.82 (m, 2 H), 3.49–2.98 (m, 4 H), 3.31 (s, 3H), 2.66–2.51 (m, 1 H), 2.03–1.60 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 473 [M + H]<sup>+</sup>.</div></div><div id="sec4_19_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i141" class="anchor-spacer"></div><h5 class="article-section__title" id="_i141"> <i>tert</i>-Butyl (3<i>R</i>)-3-[[4-[[2-(Ethoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>50d</b>) (from <b>50l</b><i>tert</i>-Butyl (3<i>R</i>)-3-(Hydroxymethyl)pyrrolidine-1-carboxylate)</h5><div class="NLM_p last">Yield 55% (21 mg), yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.86–7.54 (m, 3 H), 6.90–6.84 (m, 4 H), 5.16 (s, 2 H), 4.64 (s, 2 H), 3.93–3.82 (m, 2 H), 3.53–3.47 (m, 2 H), 3.47–3.17 (m, 4 H), 2.65–2.52 (m, 1 H), 2.03–1.61 (m, 2 H), 1.38 (s, 9 H), 1.16–1.11 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 457 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_19_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> <i>tert</i>-Butyl 4-[4-[[2-Amino-6-(methoxymethyl)phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>45e</b>)</h4><div class="NLM_p last">First, 0.163 g (0.345 mmol) of <i>tert</i>-butyl 4-[4-[[2-(methoxymethyl)-6-nitro-phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>45d</b>) were dissolved in 163 mL of EtOH, and then the solution was hydrogenated in a H-Cube apparatus (PtO<sub>2</sub> cartridge, atmospheric pressure, 75 °C, flow 0.5 mL/min). After a single run, the conversion was almost complete and solvent was evaporated, affording 122 mg (80%) of <b>45e</b> as a colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.05–6.98 (m, 1 H), 6.98–6.85 (m, 4 H), 6.70–6.53 (m, 2 H), 5.07 (s, 2 H), 4.93 (s, 2 H), 4.45–4.33 (m,s, 3 H), 3.68–3.55 (m, 2 H), 3.22 (s, 3 H), 3.19–3.09 (m, 2 H), 1.91–1.79 (m, 2 H), 1.54–1.43 (m, 2 H), 1.40 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 443 [M + H]<sup>+</sup>.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> Similarly Prepared</h4><div id="sec4_20_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i144" class="anchor-spacer"></div><h5 class="article-section__title" id="_i144"> <i>tert</i>-Butyl (3<i>R</i>)-3-[[4-[[2-Amino-6-(methoxymethyl)phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>46e</b>) (from <b>46d</b>)</h5><div class="NLM_p last">Quantitative yield (94 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.06–6.52 (m, 7 H), 5.07 (s, 2 H), 4.93 (s, 2 H), 4.36 (s, 2 H), 3.94–3.79 (m, 2 H), 3.50–2.81 (m, 7 H), 2.40–1.46 (m, 3 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 443 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i145" class="anchor-spacer"></div><h5 class="article-section__title" id="_i145"> <i>tert</i>-Butyl 4-[4-[[2-Amino-6-(methoxymethyl)phenyl]methoxy]phenoxy]azepane-1-carboxylate (<b>47e</b>) (from <b>47d</b>)</h5><div class="NLM_p last">Yield 91% (187 mg); colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.04–6.54 (m, 7 H), 5.06 (s, 2 H), 4.92 (s, 2 H),4.41–4.35 (m, 1H), 4.36 (s, 3 H), 3.43–3.24 (m, 4 H), 3.21 (s, 3 H), 2.03–1.49 (m, 6 H), 1.40 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 457 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i146" class="anchor-spacer"></div><h5 class="article-section__title" id="_i146"> <i>tert</i>-Butyl (3<i>S</i>)-3-[[4-[[2-Amino-6-(methoxymethyl)phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>48e</b>) (from <b>48d</b>)</h5><div class="NLM_p last">Yield 95% (70 mg); colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.05–6.52 (m, 7 H), 5.07 (s, 2 H), 4.92 (s, 2 H), 4.35 (s, 2 H), 3.93–3.81 (m, 2 H), 3.50–3.00 (m, 4 H), 3.21 (s, 3H), 2.66–2.52 (m, 1 H), 2.05–1.60 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 443 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i147" class="anchor-spacer"></div><h5 class="article-section__title" id="_i147"> <i>tert</i>-Butyl (3<i>R</i>)-3-[[4-[[2-Amino-6-(ethoxymethyl)phenyl]methoxy]phenoxy]methyl]pyrrolidine-1-carboxylate (<b>50e</b>) (from <b>50d</b>)</h5><div class="NLM_p">Yield 99% (18 mg), colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.06–6.52 (m, 7 H), 5.11–5.01 (br s, 2 H), 4.93 (s, 2 H), 4.39 (s, 2 H), 3.93–3.81 (m, 2 H), 3.47–3.02 (m, 6 H), 2.66–2.52 (m, 1 H), 2.04–1.59 (m, 2 H), 1.39 (s, 9 H), 1.09–1.04 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 457 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Compounds <b>45</b>–<b>50</b> were prepared according to the procedure described for intermediates <b>22f</b>–<b>28f</b> and subsequent BOC deprotection under acidic treatment conducted as illustrated for compounds <b>22</b>–<b>28</b>.</div></div><div id="sec4_20_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i148" class="anchor-spacer"></div><h5 class="article-section__title" id="_i148"> <i>N</i>-[3-(Methoxymethyl)-2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>45</b>) (from <b>45e</b> and <b>3c</b>)</h5><div class="NLM_p last">Yield 44% over two steps (65 mg), white solid. Boc intermediate: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.79 (s, 1 H), 7.52–7.48 (m, 1 H), 7.47–7.30 (3m, 3 H), 7.25–7.20 (m, 1 H), 7.17 (s, 1 H), 6.92–6.84 (m, 4 H), 5.06 (s, 2 H), 4.56 (s, 2 H), 4.46–4.21 (m, 1 H), 3.98 (s, 3 H), 3.70–3.57 (m, 2 H), 3.30 (s, 3 H), 3.21–3.02 (m, 2 H), 1.88–1.41 (2m, 4 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 606 [M + H]<sup>+</sup>. <b>45</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81 (s, 1 H), 8.63–8.34 (m, 2 H), 7.52–7.49 (m, 1 H), 7.47–7.29 (3m, 3 H), 7.24–7.21 (m, 1 H), 7.19 (s, 1 H), 6.95–6.97 (m, 4 H), 5.07 (s, 2 H), 4.56 (s, 2 H), 4.51–4.40 (m, 1 H), 3.98 (s, 3 H), 3.30 (s, 3 H), 3.25–2.96 (2m, 4 H), 2.07–1.69 (2m, 4 H). MS (ESI): <i>m</i>/<i>z</i> 506 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i149" class="anchor-spacer"></div><h5 class="article-section__title" id="_i149"> <i>N</i>-[3-(Methoxymethyl)-2-[[4-[[(3<i>R</i>)-pyrrolidin-3-yl]methoxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>46</b>) (from <b>46e</b> and <b>3c</b>)</h5><div class="NLM_p last">Yield 12% over two steps (14 mg), off-white solid. Boc intermediate: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.79 (s, 1 H), 7.50–7.48 (m, 1 H), 7.46–7.29 (m, 3 H), 7.23–7.20 (m, 1 H), 7.17 (s, 1 H), 6.91–6.80 (m, 4 H), 5.06 (s, 2 H), 4.55 (s, 2 H), 3.98 (s, 3 H), 3.89–3.74 (m, 2 H), 3.47–2.98 (m, 7 H), 2.62–2.50 (m, 1 H), 2.00–1.60 (m, 2 H), 1.38 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 606 [M + H]<sup>+</sup>. <b>46</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81 (s, 1 H), 8.92–8.49 (br s, 2 H), 7.51–7.48 (m, 1 H), 7.46–7.29 (m, 3 H), 7.24–7.21 (m, 1 H), 7.19 (s, 1 H), 6.94–6.81 (m, 4 H), 5.06 (s, 2 H), 4.55 (s, 2 H), 3.98 (s, 3 H), 3.94–3.79 (m, 2 H), 3.36–3.30 (m, 1 H), 3.29 (s, 3 H), 3.26–3.09 (m, 2 H), 3.00–2.93 (m, 1 H), 2.71–2.60 (m, 1 H), 2.10–2.00 (m, 1H), 1.65–1.75 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 506 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i150" class="anchor-spacer"></div><h5 class="article-section__title" id="_i150"> <i>N</i>-[2-[[4-(Azepan-4-yloxy)phenoxy]methyl]-3-(methoxymethyl)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>47</b>) (from <b>47e</b> and <b>3c</b>)</h5><div class="NLM_p last">Yield 37% over two steps (81 mg), off-white solid. Boc intermediate: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.89 (s, 1 H), 7.53–7.48 (m, 1 H), 7.48–7.29 (m, 3 H), 7.24–7.20 (m, 1 H), 7.16 (s, 1 H), 6.91–6.78 (m, 4 H), 5.06 (s, 2 H), 4.56 (s, 2 H), 4.37–4.26 (m, 1 H), 3.97 (s, 3 H), 3.29 (s, 3H), 3.43–3.18 (m, 4 H), 1.97–1.46 (m, 6 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 620 [M + H]<sup>+</sup>. <b>47</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.80 (s, 1 H), 8.66–8.32 (m, 2 H), 7.53–7.48 (m, 1 H), 7.46–7.29 (m, 3 H), 7.25–7.21 (m, 1 H), 7.18 (s, 1 H), 6.82–6.92 (m, 4 H), 5.06 (s, 2 H), 4.56 (s, 2 H), 4.52–4.47 (m, 1 H), 3.98 (s, 3 H), 3.29 (s, 3 H), 3.23–2.99 (m, 4 H), 2.15–1.59 (m, 6 H). MS (ESI): <i>m</i>/<i>z</i> 520 [M + H]<sup>+</sup>; mp 168–171 °C.</div></div><div id="sec4_20_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i151" class="anchor-spacer"></div><h5 class="article-section__title" id="_i151"> <i>N</i>-[3-(Methoxymethyl)-2-[[4-[[(3<i>S</i>)-pyrrolidin-3-yl]methoxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>48</b>) (from <b>48e</b> and <b>3c</b>)</h5><div class="NLM_p last">Yield 35% over two steps (29 mg), off-white solid. Boc intermediate: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.83–9.75 (m, 1 H), 7.52–7.47 (m, 1 H), 7.46–7.29 (m, 3 H), 7.24–7.21 (m, 1 H), 7.17 (s, 1 H), 6.92–6.78 (m, 4 H), 5.06 (s, 2 H), 4.55 (s, 2 H), 3.98 (s, 3 H), 3.90–3.76 (m, 2 H), 3.46–2.96 (m, 4 H), 3.29 (s, 3H), 2.61–2.51 (m, 1 H), 1.98–1.60 (m, 2 H), 1.42–1.35 (m, 9 H). MS (ESI): <i>m</i>/<i>z</i> 606 [M + H]<sup>+</sup>. <b>48</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81 (br s, 1 H), 8.88–8.57 (br s, 2 H), 7.52–7.49 (m, 1 H), 7.47–7.29 (m, 3 H), 7.25–7.22 (m, 1 H), 7.19 (s, 1 H), 6.95–6.79 (m, 4 H), 5.06 (s, 2 H), 4.55 (s, 2 H), 3.98 (s, 3 H), 3.93–3.82 (m, 2H), 3.34–2.93 (m, 4 H), 3.29 (s, 3H), 2.72–2.60 (m, 1 H), 2.12–1.65 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 506 [M + H]<sup>+</sup>; mp 118 °C dec.</div></div><div id="sec4_20_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i152" class="anchor-spacer"></div><h5 class="article-section__title" id="_i152"> 4-Ethyl-<i>N</i>-[3-(methoxymethyl)-2-[[4-[[(3<i>R</i>)-pyrrolidin-3-yl]methoxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>49</b>) (from <b>46e</b> and Ethyl 6-Methylthieno [2,3-<i>b</i>]pyrrole-5-carboxylate, See <b>105b</b> in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01018">Part 1</a><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>)</h5><div class="NLM_p last"><b>49</b> was purified by preparative HPLC and characterized as trifluoroacetate. Yield 19% over two steps (84 mg), white foam. Boc intermediate: <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.10 (s, 1 H), 8.23 (m, 1 H), 7.39 (m, 1 H), 7.29 (d, <i>J</i> = 5.4 Hz, 1 H), 7.13 (m, 1 H), 7.05–6.99 (m, 2 H), 6.97 (d, <i>J</i> = 5.4 Hz, 1 H), 6.90–6.84 (m, 2 H), 6.73 (s, 1 H), 5.23 (s, 2 H), 4.61 (q, <i>J</i> = 7.3 Hz, 2 H), 4.52 (s, 2 H), 3.95–3.85 (m, 2 H), 3.64–3.57 (m, 1 H), 3.53–3.46 (m, 1 H), 3.42–3.36 (m, 1 H), 3.34 (s, 3 H), 3.25–3.19 (m, 1 H), 2.72–2.64 (m, 1 H), 2.12–2.05 (m, 1 H), 1.85–1.76 (m, 1 H), 1.50–1.43 (m, 12 H). MS (ESI): <i>m</i>/<i>z</i> 620 [M + H]<sup>+</sup>. <b>49</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81 (s, 1 H), 8.64 (br s, 2 H), 7.50 (d, <i>J</i> = 5.5 Hz, 1 H), 7.47–7.43 (m, 1 H), 7.39 (m, 1 H), 7.35–7.29 (m, 1 H), 7.24 (d, <i>J</i> = 5.5 Hz, 1 H), 7.19 (s, 1 H), 6.94–6.87 (m, 2 H), 6.86–6.80 (m, 2 H), 5.07 (s, 2 H), 4.58–4.45 (m, 4 H), 3.93–3.88 (m, 1 H), 3.87–3.80 (m, 1 H), 3.36–3.27 (m, 4 H), 3.26–3.20 (m, 1 H), 3.19–3.11 (m, 1 H), 3.02–2.93 (m, 1 H), 2.71–2.61 (m, 1 H), 2.11–2.01 (m, 1 H), 1.75–1.65 (m, 1 H), 1.27 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 520 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i153" class="anchor-spacer"></div><h5 class="article-section__title" id="_i153"> <i>N</i>-[3-(Ethoxymethyl)-2-[[4-[[(3<i>R</i>)-pyrrolidin-3-yl]methoxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>50</b>) (from <b>50e</b> and <b>3c</b>)</h5><div class="NLM_p last">Yield 39% over two steps (11 mg), off-white solid. Boc intermediate: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81–9.75 (br s, 1 H), 7.52–7.47 (m, 1 H), 7.46–7.29 (m, 3 H), 7.25–7.20 (m, 1 H), 7.18 (s, 1 H), 6.91–6.79 (m, 4 H), 5.07 (s, 2 H), 4.59 (s, 2 H), 3.98 (s, 3 H), 3.88–3.75 (m, 2 H), 3.54–2.97 (m, 6 H), 2.60–2.45 (m., 1 H), 2.02–1.91 (m, 1 H), 1.74–1.58 (m, 1 H), 1.38 (s, 9 H), 1.24–1.04 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 620 [M + H]<sup>+</sup>. <b>50</b>: <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.8 (s, 1 H), 8.84–8.56 (br s, 2 H), 7.53–7.48 (m, 1 H), 7.46–7.27 (m, 3 H), 7.26–7.22 (m, 1 H), 7.19 (s, 1 H), 6.93–6.81 (m, 4 H), 5.07 (s, 2 H), 4.59 (s, 2 H), 3.98 (s, 3 H), 3.94–3.80 (m, 2 H), 3.52–3.42 (m, 2 H), 3.33–2.93 (m, 4 H), 2.71–2.59 (m, 1 H), 2.12–1.64 (m, 2 H), 1.16–1.10 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 520 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i154" class="anchor-spacer"></div><h5 class="article-section__title" id="_i154"> Isopropyl 2-(Hydroxymethyl)-6-nitrobenzoate (<b>52o</b>)</h5><div class="NLM_p last">To a stirred solution of 0.734 g (3.35 mmol) of sodium 2-(hydroxymethyl)-6-nitrobenzoate (<b>45n</b>) in 3 mL of DMA, 0.598 g (3.52 mmol) of 2-iodopropane was added and the mixture stirred at rt for 12 h. A further portion of 0.067 mL of 2-iodopropane (0.67 mmol) was added, and stirring was continued at rt for 16 h. Water was added to the mixture and the aqueous phase extracted with EtOAc; the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The residue was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc) to give 388 mg (48%) of <b>52o</b> as a pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.11–7.67 (m, 3 H), 5.59 (br s, 1 H), 5.22–5.09 (m, 1 H), 4.58 (m, 2 H), 1.28 (m, 6 H).</div></div><div id="sec4_20_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i155" class="anchor-spacer"></div><h5 class="article-section__title" id="_i155"> Isopropyl 2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-nitrobenzoate (<b>52p</b>)</h5><div class="NLM_p last">First, 0.27 g (1.8 mmol) of <i>tert</i>-butyl-chloro-dimethylsilane dissolved in 1 mL of DCM was added under nitrogen at 0 °C to a solution of 0.387 g (1.62 mmol) of isopropyl 2-(hydroxymethyl)-6-nitro-benzoate (<b>52o</b>) and 0.14 g (2.1 mmol) of imidazole in 1 mL of dry DCM. Then the mixture was stirred for 15 min; the white precipitate was filtered off and rinsed with DCM providing 572 mg (quantitative) of <b>52p</b> as a pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.11–7.73 (m, 3 H), 5.21–5.11 (m, 1 H), 4.77 (s, 2 H), 1.31–1.27 (m, 6 H), 0.87 (s, 9 H), 0.06 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 354 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i156" class="anchor-spacer"></div><h5 class="article-section__title" id="_i156"> [2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methanol (<b>52q</b>)</h5><div class="NLM_p last">A solution of 0.594 g (1.68 mmol) of isopropyl 2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-nitrobenzoate (<b>52p</b>) in 1.7 mL of THF was added to 2.7 mL (5.4 mmol) of 2 M LiBH<sub>4</sub> in THF at 0 °C under nitrogen. Then, 0.24 mL of dry MeOH were added and the mixture was stirred at 60 °C for 30 min, cooled down to 0 °C, quenched with a saturated aqueous NH<sub>4</sub>Cl solution, and extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a residue, which was purified by flash chromatography (eluent <i>n</i>-hexane EtOAc from 1% to 10% of EtOAc), providing 303 mg (61%) of <b>52q</b> as a pale-yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.77–7.44 (m, 3 H), 5.28–5.23 (m, 1 H), 4.88 (s, 2 H), 4.63–4.59 (m, 2 H), 0.90 (s, 9 H), 0.09 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 298 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i157" class="anchor-spacer"></div><h5 class="article-section__title" id="_i157"> [2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methylmethanesulfonate (<b>52r</b>)</h5><div class="NLM_p last">First, 0.17 mL (1.2 mmol) of TEA were added under nitrogen to a solution of 0.28 g (0.94 mmol) of [2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methanol (<b>52q</b>) in 3 mL of DCM at −15 °C, followed by 0.087 mL (1.1 mmol) of methanesulfonyl chloride. The mixture was stirred for 20 min at −15 °C, quenched with a saturated aqueous NaHCO<sub>3</sub> solution, and extracted with DCM. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford 350 mg (quantitative) of <b>52r</b> as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.95–7.65 (m, 3 H), 5.43 (s, 2 H), 4.93 (s, 2 H), 3.23 (s, 3 H), 0.90 (s, 9 H), 0.1 (s, 6 H).</div></div><div id="sec4_20_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i158" class="anchor-spacer"></div><h5 class="article-section__title" id="_i158"> 4-[[2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methoxy]phenol (<b>52l</b>)</h5><div class="NLM_p last">A solution of 0.354 g (0.94 mmol) of [2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methylmethanesulfonate (<b>52r</b>) in 4 mL of acetone was added dropwise to a suspension of 0.519 g (4.71 mmol) of benzene-1,4-diol and 0.33 g (2.4 mmol) of K<sub>2</sub>CO<sub>3</sub> in 1 mL of acetone. The mixture was stirred overnight at rt and then for 45 min at 60 °C. Solvent was removed, and the residue was extracted with DCM. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a residue, which was purified by flash chromatography (eluent <i>n</i>-hexane EtOAc from 3% to 30% of EtOAc), providing 218 mg (59%) of <b>52l</b> as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.02 (s, 1 H), 7.82–7.55 (m, 3 H), 6.78–6.62 (m, 4 H), 5.10 (s, 2 H), 4.88 (s, 2 H), 0.87 (s, 9 H), 0.06 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 390 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i159" class="anchor-spacer"></div><h5 class="article-section__title" id="_i159"> <i>tert</i>-Butyl 4-[4-[[2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methoxy]phenoxy]-piperidine-1-carboxylate (<b>52d</b>)</h5><div class="NLM_p last">First, 0.04 g (0.15 mmol) of triphenylphosphine were added to a solution of 0.04 g (0.10 mmol) 4-[[2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methoxy]phenol (<b>52l</b>) and 0.031 g (0.15 mmol) of <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate in 0.6 mL of THF at 0 °C and under nitrogen. Then 0.031 g (0.15 mmol) of diisopropyl azodicarboxylate were slowly added and the mixture was stirred overnight at rt. Solvent was removed and the residue was purified by flash chromatography (eluent <i>n</i>-hexane EtOAc from 2% to 20% of EtOAc), providing 35.5 mg (60%) of <b>52d</b> as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.85–7.55 (m, 3 H), 6.94–6.81 (m, 4 H), 5.14 (s, 2 H), 4.89 (s, 2 H), 4.45–4.34 (m, 1 H), 3.68–3.55 (m, 2 H), 3.21–3.02 (m, 2 H), 1.89–1.43 (m, 4 H), 1.39 (s, 9 H), 0.87 (s, 9 H), 0.06 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 573 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i160" class="anchor-spacer"></div><h5 class="article-section__title" id="_i160"> <i>tert</i>-Butyl 4-[4-[[2-Amino-6-[(1-methyl-1-trimethylsilyl-ethoxy)methyl]phenyl]methoxy]-phenoxy]piperidine-1-carboxylate (<b>52e</b>)</h5><div class="NLM_p last">A 0.025 M solution of 0.223 g (0.37 mmol) of <i>tert</i>-butyl 4-[4-[[2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-nitro-phenyl]methoxy]phenoxy]-piperidine-1-carboxylate (<b>52d</b>) in 23 mL of EtOAc was hydrogenated in a H-cube apparatus (PtO<sub>2</sub>, atmospheric pressure, 75 °C, flow 0.5 mL/min) for 4 h. Solvent was evaporated under vacuum to afford 209 mg (quant) of <b>52e</b> as a colorless sticky oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.04–6.56 (m, 7 H), 5.05 (br s, 2 H), 4.93 (s, 2 H), 4.65 (s, 2 H), 4.44–4.36 (m, 1 H), 3.67–3.54 (m, 2 H), 3.24–3.03 (m, 2 H), 1.89–1.43 (m, 4 H), 1.39 (s, 9 H), 0.83 (s, 9 H), 0.01 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 543 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i161" class="anchor-spacer"></div><h5 class="article-section__title" id="_i161"> <i>tert</i>-Butyl 4-[4-[[2-[[<i>tert</i>-Butyl(dimethyl)silyl]oxymethyl]-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52s</b>)</h5><div class="NLM_p last">The compound was prepared according to the procedure reported for compounds <b>3-21</b>. Then 31 mg (67%) of <b>52s</b> were obtained starting from 0.012 g (0.066 mmol) of 4-methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic acid (<b>3c</b>) and 0.030 g (0.060 mmol) of <i>tert</i>-butyl 4-[4-[[2-amino-6-[(1-methyl-1-trimethylsilyl-ethoxy)methyl]phenyl]methoxy]phenoxy]-piperidine-1-carboxylate <b>52e</b>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.80 (s, 1 H), 7.84–7.50 (m, 1 H), 7.41–7.36 (m, 3 H), 7.23–7.20 (m, 1 H), 7.19 (s, 1 H), 6.87–6.85 (m, 4 H), 5.05 (s, 2 H), 4.85 (s, 2 H), 4.43–4.29 (m, 1 H), 3.98 (s, 3 H), 3.65–3.54 (m, 2 H), 3.20–3.02 (m, 2 H), 1.87–1.74 (m, 2 H), 1.51–1.41 (m, 2 H), 1.38 (s, 9 H), 0.88 (s, 9 H), 0.05 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 707 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i162" class="anchor-spacer"></div><h5 class="article-section__title" id="_i162"> <i>tert</i>-Butyl 4-[4-[[2-(Hydroxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52t</b>)</h5><div class="NLM_p last">First, 0.047 mL (0.047 mmol) of a 1 M solution of TBAF in THF were added to a solution of 0.03 g (0.042 mmol) of <i>tert</i>-butyl 4-[4-[[2-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52s</b>) in 0.2 mL of THF. Then the mixture was stirred 45 min at rt, solvent was removed, and the residue was extracted with EtOAc; the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a residue, which was purified by flash chromatography (eluent <i>n</i>-hexaneEtOAc from 12% to 100% of EtOAc), providing 19 mg (76%) of <b>52t</b> as a white sticky solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.78 (s, 1 H), 7.50–7.48 (m, 1 H), 7.40–7.35 (m, 3 H), 7.23–6.20 (m, 1 H), 7.17 (s, 1 H), 6.91–6.83 (m, 4 H), 5.24 (m, 1 H), 5.04 (s, 2 H), 4.66–4.62 (m, 2 H), 4.41–4.32 (m, 1 H), 3.97 (s, 3 H), 3.66–3.56 (m, 2 H), 3.20–3.04 (m, 2 H), 1.86–1.75 (m, 2 H), 1.49–1.43 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 592 [M + H]<sup>+</sup>.</div></div><div id="sec4_20_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i163" class="anchor-spacer"></div><h5 class="article-section__title" id="_i163"> <i>tert</i>-Butyl 4-[4-[[2-(Methylsulfonyloxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52u</b>)</h5><div class="NLM_p last">First, 0.016 mL (0.11 mmol) of TEA were added to a solution of 0.040 g (0.068 mmol) of <i>tert</i>-butyl 4-[4-[[2-(hydroxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52t</b>) in 0.4 mL of DCM at −15 °C under nitrogen, followed by 0.011 g (0.095 mmol) of methanesulfonyl chloride. Then the reaction mixture was stirred for 25 min at −15 °C, quenched with a saturated aqueous NaHCO<sub>3</sub> solution, and extracted with DCM. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give 49.7 mg (quantitative) of <b>52u</b> as a yellow oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.91 (s, 1 H), 7.55–7.13 (m, 6 H), 6.93–6.78 (m, 4 H), 5.44 (s, 2 H), 5.09 (s, 2 H), 4.41–4.31 (m, 1 H), 3.98 (s, 3 H), 3.66–3.55 (m, 2 H), 3.20 (s, 3 H), 3.14–3.06 (m, 2 H), 1.87–1.75 (m, 2 H), 1.51–1.40 (m, 2 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 670 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_20_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i164" class="anchor-spacer"></div><h4 class="article-section__title" id="_i164"> <i>tert</i>-Butyl 4-[4-[[2-(Ethoxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52f</b>)</h4><div class="NLM_p last">First, 0.0541 g (1.35 mmol) of 60% NaH (mineral oil) were added to 5.7 mL of cooled EtOH, and the mixture stirred 1 h at rt; then 0.0226 g (0.0338 mmol) of <i>tert</i>-butyl 4-[4-[[2-(methylsulfonyloxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>52u</b>) were added and stirring was continued at rt for 1 h. Solvent was removed, and the residue was partitioned between EtOAc and brine. The aqueous phase was extracted with EtOAc; the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc from 6% to 50% of EtOAc) to yield 16.2 mg (77%) of <b>52f</b> as an orange oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.78 (s, 1 H), 7.53–7.48 (m, 1 H), 7.46–7.29 (m, 3 H), 7.24–7.20 (m, 1 H), 7.17 (s, 1 H), 6.92–6.84 (m, 4 H), 5.07 (s, 2 H), 4.59 (s, 2 H), 4.40–4.30 (m, 1 H), 3.98 (s, 3 H), 3.67–3.57 (m, 2 H), 3.52–3.43 (m, 2 H), 3.20–3.01 (m, 2 H), 1.87–1.40 (m, 4H), 1.39 (s, 9 H), 1.15–1.10 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 620 [M + H]<sup>+</sup>.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i165" class="anchor-spacer"></div><h4 class="article-section__title" id="_i165"> Similarly Prepared</h4><div id="sec4_21_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i166" class="anchor-spacer"></div><h5 class="article-section__title" id="_i166"> <i>tert</i>-Butyl 4-[4-[[2-[(4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]-6-(morpholinomethyl)phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>53f</b>) (from <b>52u</b> and Morpholine)</h5><div class="NLM_p last">Yield 50% (10.5 mg), white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.70 (s, 1 H), 7.52–7.19 (m, 5 H), 7.15 (s, 1 H), 6.94–6.83 (m, 4 H), 5.19 (s, 2 H), 4.42–4.29 (m, 1 H), 3.97 (s, 3 H), 3.68–3.60 (m, 2 H), 3.57 (s, 2 H), 3.53–3.46 (m, 2 H), 3.18–3.01 (m, 4 H), 2.38–2.22 (m, 4 H), 1.86–1.41 (m, 4 H), 1.39 (s, 9 H). MS (ESI): <i>m</i>/<i>z</i> 661 [M + H]<sup>+</sup>.</div></div><div id="sec4_21_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i167" class="anchor-spacer"></div><h5 class="article-section__title" id="_i167"> <i>tert</i>-Butyl 4-[4-[[2-(Isopropoxymethyl)-6-[(4-methylthieno[3,2-<i>b</i>]pyrrole-5-carbonyl)amino]phenyl]methoxy]phenoxy]piperidine-1-carboxylate (<b>54f</b>) (from <b>52u</b> and 2-propanol)</h5><div class="NLM_p">Yield 40% (16 mg), colorless oil. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.77 (s, 1 H), 7.53–7.47 (m, 1 H), 7.46–7.30 (m, 3 H), 7.24–7.20 (m, 1 H), 7.16 (s, 1 H), 6.91–6.84 (m, 4 H), 5.07 (s, 2 H), 4.59 (s, 2 H), 4.39–4.30 (m, 1 H), 3.98 (s, 3 H), 3.66–3.55 (m, 3 H), 3.20–3.03 (m, 2 H), 1.87–1.40 (m, 4H), 1.39 (s, 9 H), 1.13–1.09 (m, 6 H). MS (ESI): <i>m</i>/<i>z</i> 634 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Boc deprotection of <b>52f</b>–<b>54f</b> was perfomed according to the general procedure reported for compounds <b>22</b>–<b>28</b>.</div></div><div id="sec4_21_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i168" class="anchor-spacer"></div><h5 class="article-section__title" id="_i168"> <i>N</i>-[3-(Ethoxymethyl)-2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Hydrochloride (<b>52</b>) (from <b>52f</b>)</h5><div class="NLM_p last">Yield 76% (10.5 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.80 (s, 1 H), 8.30 (br s, 2 H), 7.51–7.48 (m, 1 H), 7.47–7.30 (m, 3 H), 7.25–7.21 (m, 1 H), 7.19 (s, 1 H), 6.93–6.87 (m, 4 H), 5.07 (s, 2 H), 4.59 (s, 2 H), 4.48–4.40 (m, 1 H), 3.98 (s, 3 H), 3.50–3.44 (m, 2 H), 3.23–2.92 (m, 4 H), 2.04–1.66 (m, 4 H), 1.15–1.10 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 520 [M + H]<sup>+</sup>; mp 223–226 °C.</div></div><div id="sec4_21_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i169" class="anchor-spacer"></div><h5 class="article-section__title" id="_i169"> 4-Methyl-<i>N</i>-[3-(morpholinomethyl)-2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide Dihydrochloride (<b>53</b>) (from <b>53f</b>)</h5><div class="NLM_p last">Yield 80% (7.5 mg), white solid. <sup>1</sup>H NMR (D<sub>2</sub>O) δ (ppm): 7.52–7.35 (m, 3 H), 7.28 (d, <i>J</i> = 5.4 Hz, 1 H), 6.92 (d, <i>J</i> = 5.4 Hz, 1 H), 6.81–6.67 (m, 5 H), 5.10 (s, 2 H), 4.48 (s, 2 H), 4.24 (br s, 1 H), 3.99–3.85 (m, 2 H), 3.65 (s, 5 H), 3.66–3.56 (m, 2H), 3.46–3.32 (m, 2 H), 3.27–3.11 (m, 4 H), 2.99–2.89 (m, 2 H), 1.87–1.61 (m, 4 H). MS (ESI): <i>m</i>/<i>z</i> 561 [M + H]<sup>+</sup>.</div></div><div id="sec4_21_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i170" class="anchor-spacer"></div><h5 class="article-section__title" id="_i170"> <i>N</i>-[3-(Isopropoxymethyl)-2-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>54</b>) (from <b>54f</b>)</h5><div class="NLM_p last">Yield 40% (5.6 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.12 (s, 1 H), 8.23–8.17 (m, 1 H), 7.41–7.34 (m, 1 H), 7.32–7.25 (m, 1 H), 7.16–7.11 (m, 1 H), 7.04–6.99 (m, 2 H), 6.98–6.94 (m, 1 H), 6.93–6.88 (m, 2 H), 6.75 (s, 1 H), 5.25 (s, 2 H), 4.56 (s, 2 H), 4.36–4.27 (m, 1 H), 4.12 (s, 3 H), 3.71–3.61 (m, 1 H), 3.25–2.73 (m, 4 H), 2.10–1.64 (m, 4 H), 1.20–1.15 (m, 6 H). MS (ESI): <i>m</i>/<i>z</i> 534 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i171" class="anchor-spacer"></div><h3 class="article-section__title" id="_i171"> Protein Preparation, Crystallization, and Structure Determination</h3><div class="NLM_p last">The KDM1A/CoREST complex was expressed and purified as previously reported.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The complex, concentrated at approximately 7 mg/mL, was crystallized in hanging drops in 24-well plates at 20 °C, mixing 1 μL of protein complex with 1 μL of reservoir precipitant solution (1.1–1.2 M Na/K tartrate, 0.1 M ADA, pH 6.0). Crystals were soaked with the different compounds by adding to the drops 1 μL of 1 mM inhibitor solution, prepared by diluting the 20 mM stock of inhibitor in 100% DMSO in 1.6 M Na/K tartrate, 10% glycerol, 0.1 M ADA, pH 6.0. After ca. 1–2 h, crystals were transferred into 1 mM solution of inhibitor in 1.6 M Na/K tartrate, 10% glycerol, 0.1 M ADA, pH 6.0, and after few seconds or some minutes they were flash frozen in liquid nitrogen. Diffraction patterns from crystals soaked with different compounds were collected at beamline X06DA/PXIII of the Swiss Light Source (SLS, Paul Sherrer Institut, Villigen, Switzerland) or ID30A-1/Massif-1 of the European Synchrotron Radiation Facility (ESRF, Grenoble, France) at 100 K. Diffraction data were processed with XDS/XSCALE<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> within the automated data reduction system xia2.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The structures were solved by molecular replacement with Phaser<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> within the Phenix suite,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> using the protein chains from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a> as search model. The models were improved by iterative cycles of manual building in Coot<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and refinement with phenix.refine.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Inhibitors were automatically placed with the phenix.ligandfit procedure<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> in Phenix, and the structures were then further refined. Crystallographic statistics are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i172" class="anchor-spacer"></div><h3 class="article-section__title" id="_i172"> Molecular Modeling Activities</h3><div class="NLM_p last">Pose production, clustering, and scoring were implemented within the architecture described in our recent work.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Protein targets have been prepared and used according to the procedures described in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i173" class="anchor-spacer"></div><h3 class="article-section__title" id="_i173"> Biology</h3><div class="NLM_p last">Material and methods for biochemical and cellular assays have been included in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i174"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01019">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01019" class="ext-link">10.1021/acs.jmedchem.6b01019</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthetic procedures for the preparation of compounds <b>20e</b>, <b>21e</b>, <b>23e</b>–<b>28e</b>, and <b>44</b>; material and methods for biochemical and cellular assays; data collection and refinement statistics for compounds <b>15</b>, <b>17</b>, <b>34</b>, <b>49</b>, and <b>50</b>; HOMO values and localization; HOMO pictures for compounds <b>15</b>, <b>20</b>–<b>26</b>, and <b>44</b>; calculated physicochemical properties (molecular surface area, AlogP, and Log <i>D</i>) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Protein Target (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Compound <b>2</b>, pose A (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Compound <b>2</b>, pose B (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf">jm6b01019_si_001.pdf (1.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_002.csv">jm6b01019_si_002.csv (10.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_003.pdb">jm6b01019_si_003.pdb (510.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_004.pdb">jm6b01019_si_004.pdb (11.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_005.pdb">jm6b01019_si_005.pdb (11.84 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the crystal structures of KDM1A with compounds <b>15</b>, <b>17</b>, <b>34</b>, <b>49</b>, and <b>50</b> can be accessed using PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGT">5LGT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHG">5LHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGU">5LGU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHH">5LHH</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI">5LHI</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01019" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52120" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52120" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Vianello</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9100-2711" title="Orcid link">http://orcid.org/0000-0001-9100-2711</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1464757b78753a627d757a7178787b547d727b793a7161"><span class="__cf_email__" data-cfemail="89f9e8e6e5e8a7ffe0e8e7ece5e5e6c9e0efe6e4a7ecfc">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luca Sartori</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#305c4553511e435142445f42597059565f5d1e5545"><span class="__cf_email__" data-cfemail="35594056541b465447415a475c755c535a581b5040">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federica Amigoni</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Cappa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giovanni Fagá</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raimondo Fattori</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Legnaghi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Ciossani</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
1, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Mattevi</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
1, 27100 Pavia, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9523-7128" title="Orcid link">http://orcid.org/0000-0002-9523-7128</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Meroni</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loris Moretti</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9851-2965" title="Orcid link">http://orcid.org/0000-0002-9851-2965</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Valentina Cecatiello</span> - <span class="hlFld-Affiliation affiliation">IFOM-
The FIRC Institute of Molecular Oncology Foundation, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Crystallography
Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastiano Pasqualato</span> - <span class="hlFld-Affiliation affiliation">Crystallography
Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessia Romussi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florian Thaler</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Trifiró</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuela Villa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oronza A. Botrugno</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Dessanti</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saverio Minucci</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biosciences, University of Milan, Via Celoria 26, 20133 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Vultaggio</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisa Zagarrí</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario Varasi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ciro Mercurio</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Genextra
Group, DAC s.r.l., Via
Adamello 16, 20139 Milano, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7635-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i179">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Fondazione Cariplo (2010.0778), AIRC (15208), MIUR (progetto Epigen), FIRB RBFR10ZJQT and RF-2010-2318330 projects by MIUR, Sapienza Ateneo project 2013, IIT-Sapienza project, FP7 projects BLUEPRINT/282510 and A-PARADDISE/602080, and Regione Lombardia, Fondo per la Promozione di Accordi Istituzionale, Bando di Invito di cui al Decreto no. 4779 del 14/05/2009, progetto DiVA, BioStruct-X (FP7/2007-2013, grant agreement 283570), and Rasna Theraupeutics Inc. We thank scientists at the X06DA/PXIII beamline at the Swiss Light Source and the ID30A-1/Massif-1 beamline at the European Synchrotron Radiation Facility for valuable help with data collection.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i180" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i180"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i181" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i181"> Abbreviations Used</h2><tr><td class="NLM_term">LSD</td><td class="NLM_def"><p class="first last">lysine-specific histone demethylase</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">APL</td><td class="NLM_def"><p class="first last">acute promyelocytic leukemia</p></td></tr><tr><td class="NLM_term">br s</td><td class="NLM_def"><p class="first last">broad signal</p></td></tr><tr><td class="NLM_term"><i>n</i>-BuLi</td><td class="NLM_def"><p class="first last"><i>n</i>-butyllithium</p></td></tr><tr><td class="NLM_term">CH<sub>2</sub>Cl<sub>2</sub></td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">DMEDA</td><td class="NLM_def"><p class="first last">2-(dimethylamino)ethylamine</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tripyrrolidino-phosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butanol</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i182">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18477" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18477" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 45 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Hoffmann, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roatsch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">The Role of Histone Demethylases in Cancer Therapy</span> <span class="citation_source-journal">Mol. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1016/j.molonc.2012.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.molonc.2012.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22902149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12gt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=683-703&author=I.+Hoffmannauthor=M.+Roatschauthor=M.+L.+Schmittauthor=L.+Carlinoauthor=M.+Pippelauthor=W.+Sipplauthor=M.+Jung&title=The+Role+of+Histone+Demethylases+in+Cancer+Therapy&doi=10.1016%2Fj.molonc.2012.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of histone demethylases in cancer therapy</span></div><div class="casAuthors">Hoffmann, Inga; Roatsch, Martin; Schmitt, Martin L.; Carlino, Luca; Pippel, Martin; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">683-703</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Reversible histone methylation has emerged in the last few years as an important mechanism of epigenetic regulation.  Histone methyltransferases and demethylases have been identified as contributing factors in the development of several diseases, esp. cancer.  Therefore, they have been postulated to be new drug targets with high therapeutic potential.  Here, we review histone demethylases with a special focus on their potential role in oncol. drug discovery.  We present an overview over the different classes of enzymes, their biochem., selected data on their role in physiol. and already available inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHdjKp6f5Og7Vg90H21EOLACvtfcHk0ljtKhCBBRo5DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12gt7zO&md5=9830a57c650e94b2b9a38ea71b0d1a4b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2012.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2012.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DI.%26aulast%3DRoatsch%26aufirst%3DM.%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520Role%2520of%2520Histone%2520Demethylases%2520in%2520Cancer%2520Therapy%26jtitle%3DMol.%2520Oncol.%26date%3D2012%26volume%3D6%26spage%3D683%26epage%3D703%26doi%3D10.1016%2Fj.molonc.2012.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljtKhCBBRo5DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Profumo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A Novel Mammalian Flavin-Dependent Histone Demethylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">17775</span><span class="NLM_x">–</span> <span class="NLM_lpage">17782</span><span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+Novel+Mammalian+Flavin-Dependent+Histone+Demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0ljtKhCBBRo5DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Mammalian%2520Flavin-Dependent%2520Histone%2520Demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">LSD1 Inhibition: A Therapeutic Strategy in Cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+P.+Somervaille&title=LSD1+Inhibition%3A+A+Therapeutic+Strategy+in+Cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0licjXgV4KZq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DLSD1%2520Inhibition%253A%2520A%2520Therapeutic%2520Strategy%2520in%2520Cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 Demethylates Repressive Histone Marks to Promote Androgen-Receptor-Dependent Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+Demethylates+Repressive+Histone+Marks+to+Promote+Androgen-Receptor-Dependent+Transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0licjXgV4KZq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520Demethylates%2520Repressive%2520Histone%2520Marks%2520to%2520Promote%2520Androgen-Receptor-Dependent%2520Transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) Is Highly Expressed in ER-Negative Breast Cancers and a Biomarker Predicting Aggressive Biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+%28LSD1%29+Is+Highly+Expressed+in+ER-Negative+Breast+Cancers+and+a+Biomarker+Predicting+Aggressive+Biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0licjXgV4KZq3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520Is%2520Highly%2520Expressed%2520in%2520ER-Negative%2520Breast%2520Cancers%2520and%2520a%2520Biomarker%2520Predicting%2520Aggressive%2520Biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lgb5lJiGhLw2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+Lysine+4+Demethylation+Is+a+Target+of+Nonselective+Antidepressive+Medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lgb5lJiGhLw2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520Lysine%25204%2520Demethylation%2520Is%2520a%2520Target%2520of%2520Nonselective%2520Antidepressive%2520Medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Benelkebir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgkinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duriez, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulleid, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Enantioselective Synthesis of Tranylcypromine Analogues as Lysine Demethylase (LSD1) Inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3709</span><span class="NLM_x">–</span> <span class="NLM_lpage">3716</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.bmc.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=21382717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3709-3716&author=H.+Benelkebirauthor=C.+Hodgkinsonauthor=P.+J.+Duriezauthor=A.+L.+Haydenauthor=R.+A.+Bulleidauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Ganesan&title=Enantioselective+Synthesis+of+Tranylcypromine+Analogues+as+Lysine+Demethylase+%28LSD1%29+Inhibitors&doi=10.1016%2Fj.bmc.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span></div><div class="casAuthors">Benelkebir, Hanae; Hodgkinson, Christopher; Duriez, Patrick J.; Hayden, Annette L.; Bulleid, Rosemary A.; Crabb, Simon J.; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3709-3716</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Asym. cyclopropanation of styrenes by tert-Bu diazoacetate followed by ester hydrolysis and Curtius rearrangement gave a series of tranylcypromine analogs as single enantiomers.  The o-, m-, and p-bromo analogs were all more active than tranylcypromine in a LSD1 enzyme assay.  The m- and p-bromo analogs were micromolar growth inhibitors of the LNCaP prostate cancer cell line as were the corresponding biphenyl analogs prepd. from the bromide by Suzuki crosscoupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHGveNGE6AbVg90H21EOLACvtfcHk0lh4eWMgG4sROQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D&md5=a8517cf8753c76b1403b6354977fc2a2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DBenelkebir%26aufirst%3DH.%26aulast%3DHodgkinson%26aufirst%3DC.%26aulast%3DDuriez%26aufirst%3DP.%2BJ.%26aulast%3DHayden%26aufirst%3DA.%2BL.%26aulast%3DBulleid%26aufirst%3DR.%2BA.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEnantioselective%2520Synthesis%2520of%2520Tranylcypromine%2520Analogues%2520as%2520Lysine%2520Demethylase%2520%2528LSD1%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3709%26epage%3D3716%26doi%3D10.1016%2Fj.bmc.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile Synthesis of Substituted Trans-2-Arylcyclopropylamine Inhibitors of the Human Histone Demethylase LSD1 and Monoamine Oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+Synthesis+of+Substituted+Trans-2-Arylcyclopropylamine+Inhibitors+of+the+Human+Histone+Demethylase+LSD1+and+Monoamine+Oxidases+A+and+B&doi=10.1016%2Fj.bmcl.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0lh4eWMgG4sROQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520Synthesis%2520of%2520Substituted%2520Trans-2-Arylcyclopropylamine%2520Inhibitors%2520of%2520the%2520Human%2520Histone%2520Demethylase%2520LSD1%2520and%2520Monoamine%2520Oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051%26doi%3D10.1016%2Fj.bmcl.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Hojfeldt, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Histone Lysine Demethylases as Targets for Anticancer Therapy</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">930</span><span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+Hojfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+Lysine+Demethylases+as+Targets+for+Anticancer+Therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0lh4eWMgG4sROQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DHojfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520Lysine%2520Demethylases%2520as%2520Targets%2520for%2520Anticancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+Overview+of+Phenylcyclopropylamine+Derivatives%3A+Biochemical+and+Biological+Significance+and+Recent+Developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0lhs5daq804qQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520Overview%2520of%2520Phenylcyclopropylamine%2520Derivatives%253A%2520Biochemical%2520and%2520Biological%2520Significance%2520and%2520Recent%2520Developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally Designed Trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+Designed+Trans-2-Phenylcyclopropylamine+Derivatives+Potently+Inhibit+Histone+Demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lhs5daq804qQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520Designed%2520Trans-2-Phenylcyclopropylamine%2520Derivatives%2520Potently%2520Inhibit%2520Histone%2520Demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1-Selective Inactivators: Protein-Targeted Drug Delivery Mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">–</span> <span class="NLM_lpage">8624</span><span class="refDoi"> DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-Specific+Demethylase+1-Selective+Inactivators%3A+Protein-Targeted+Drug+Delivery+Mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0lhs5daq804qQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-Specific%2520Demethylase%25201-Selective%2520Inactivators%253A%2520Protein-Targeted%2520Drug%2520Delivery%2520Mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Pieroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annunziato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzali, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiró, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beato, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span> </span><span class="NLM_article-title">Further Insights into the SAR of α-Substituted Cyclopropylamine Derivatives as Inhibitors of Histone Demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2014.12.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=377-386&author=M.+Pieroniauthor=G.+Annunziatoauthor=E.+Azzaliauthor=P.+Dessantiauthor=C.+Mercurioauthor=G.+Meroniauthor=P.+Trifir%C3%B3author=P.+Vianelloauthor=M.+Villaauthor=C.+Beatoauthor=M.+Varasiauthor=G.+Costantino&title=Further+Insights+into+the+SAR+of+%CE%B1-Substituted+Cyclopropylamine+Derivatives+as+Inhibitors+of+Histone+Demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.12.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DAnnunziato%26aufirst%3DG.%26aulast%3DAzzali%26aufirst%3DE.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBeato%26aufirst%3DC.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DFurther%2520Insights%2520into%2520the%2520SAR%2520of%2520%25CE%25B1-Substituted%2520Cyclopropylamine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D92%26spage%3D377%26epage%3D386%26doi%3D10.1016%2Fj.ejmech.2014.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Synthesis, Biological Activity and Mechanistic Insights of 1-Substituted Cyclopropylamine Derivatives: A Novel Class of Irreversible Inhibitors of Histone Demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.08.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2014.08.068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=25173853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=352-363&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=G.+Ciossaniauthor=P.+Dessantiauthor=A.+Maiauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=F.+Thalerauthor=M.+Tortoriciauthor=P.+Trifiroauthor=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Synthesis%2C+Biological+Activity+and+Mechanistic+Insights+of+1-Substituted+Cyclopropylamine+Derivatives%3A+A+Novel+Class+of+Irreversible+Inhibitors+of+Histone+Demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.08.068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: A novel class of irreversible inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Ciossani, Giuseppe; Dessanti, Paola; Mai, Antonello; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Thaler, Florian; Tortorici, Marcello; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">352-363</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone demethylase KDM1A [i.e., gene LSD1, histone H3 lysine-4 demethylase LSD1] has become an attractive therapeutic target for the treatment of cancer as well as other disorders such as viral infections.  The authors report on the synthesis of compds. derived from the expansion of tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  These compds., which are substituted on the cyclopropyl core moiety, were evaluated for their ability to inhibit KDM1A in vitro as well as to function in cells by modulating the expression of Gfi-1b, a well recognized KDM1A target gene.  The mols. were all found to covalently inhibit KDM1A and to become increasingly selective against human monoamine oxidase MAO A and MAO B through the introduction of bulkier substituents on the cyclopropylamine ring.  Structural and biochem. anal. of selected trans-isomers showed that the two stereoisomers are endowed with similar inhibitory activities against KDM1A, but form different covalent adducts with FAD co-enzyme [i.e., riboflavin 5'-(trihydrogen diphosphate) P'→5'-ester with adenosine, FAD].  The synthesis of the target compds. was achieved using (2S)-2-(phenyl)oxirane, (2R)-2-(phenyl)oxirane, racemic (phenyl)oxirane as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ-thHP_b6rVg90H21EOLACvtfcHk0lhVNvkMd5zj4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP&md5=6285108e113c25939e3aa0b1ee5390c6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.08.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.08.068%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DSynthesis%252C%2520Biological%2520Activity%2520and%2520Mechanistic%2520Insights%2520of%25201-Substituted%2520Cyclopropylamine%2520Derivatives%253A%2520A%2520Novel%2520Class%2520of%2520Irreversible%2520Inhibitors%2520of%2520Histone%2520Demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D352%26epage%3D363%26doi%3D10.1016%2Fj.ejmech.2014.08.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saponara, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monaldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruoppolo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilset, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Pure Enantiomers of Benzoylamino-Tranylcypromine: LSD1 Inhibition, Gene Modulation in Human Leukemia Cells and Effects on Clonogenic Potential of Murine Promyelocytic Blasts</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">174</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2015.02.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=163-174&author=S.+Valenteauthor=V.+Rodriguezauthor=C.+Mercurioauthor=P.+Vianelloauthor=B.+Saponaraauthor=R.+Cirilliauthor=G.+Ciossaniauthor=D.+Labellaauthor=B.+Marroccoauthor=D.+Monaldiauthor=G.+Ruoppoloauthor=M.+Tilsetauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=A.+Matteviauthor=M.+Varasiauthor=A.+Mai&title=Pure+Enantiomers+of+Benzoylamino-Tranylcypromine%3A+LSD1+Inhibition%2C+Gene+Modulation+in+Human+Leukemia+Cells+and+Effects+on+Clonogenic+Potential+of+Murine+Promyelocytic+Blasts&doi=10.1016%2Fj.ejmech.2015.02.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.060%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DV.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSaponara%26aufirst%3DB.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMonaldi%26aufirst%3DD.%26aulast%3DRuoppolo%26aufirst%3DG.%26aulast%3DTilset%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DPure%2520Enantiomers%2520of%2520Benzoylamino-Tranylcypromine%253A%2520LSD1%2520Inhibition%252C%2520Gene%2520Modulation%2520in%2520Human%2520Leukemia%2520Cells%2520and%2520Effects%2520on%2520Clonogenic%2520Potential%2520of%2520Murine%2520Promyelocytic%2520Blasts%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D163%26epage%3D174%26doi%3D10.1016%2Fj.ejmech.2015.02.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span> </span><span class="NLM_article-title">Phase I Dose Escalation Study for GSK2879552 in Subjects with Acute Myeloid Leukemia (AML)</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&amp;rank=2" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&rank=2</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Phase+I+Dose+Escalation+Study+for+GSK2879552+in+Subjects+with+Acute+Myeloid+Leukemia+%28AML%29.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2014%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02177812%3Fterm%3DGSK2879552%26rank%3D2+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DPhase%2520I%2520Dose%2520Escalation%2520Study%2520for%2520GSK2879552%2520in%2520Subjects%2520with%2520Acute%2520Myeloid%2520Leukemia%2520%2528AML%2529%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span> </span><span class="NLM_article-title">Investigation of GSK2879552 in Subjects with Relapsed/Refractory Small Cell Lung Carcinoma</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+with+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2013%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DInvestigation%2520of%2520GSK2879552%2520in%2520Subjects%2520with%2520Relapsed%252FRefractory%2520Small%2520Cell%2520Lung%2520Carcinoma%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carceller, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buesa, C.</span><span> </span><span class="NLM_article-title">Advances in the Development of Histone Lysine Demethylase Inhibitors</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+Development+of+Histone+Lysine+Demethylase+Inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0lgy0Rv_5_DnaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520Development%2520of%2520Histone%2520Lysine%2520Demethylase%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span> </span><span class="NLM_article-title">A phase I study of Human Pharmacokinetics and Safety of ORY-1001, an LSD1 inhibitor, in relapsed or refractory acute leukaemia (AL)</span>. EU Clinical Trials Register; <span class="NLM_publisher-name">European Medicines Agency</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES" class="extLink">https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES</a> (Accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=A+phase+I+study+of+Human+Pharmacokinetics+and+Safety+of+ORY-1001%2C+an+LSD1+inhibitor%2C+in+relapsed+or+refractory+acute+leukaemia+%28AL%29.+EU+Clinical+Trials+Register%3B+European+Medicines+Agency%3A+London%2C+2013%3B+https%3A%2F%2Fwww.clinicaltrialsregister.eu%2Fctr-search%2Ftrial%2F2013-002447-29%2FES+%28Accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DA%2520phase%2520I%2520study%2520of%2520Human%2520Pharmacokinetics%2520and%2520Safety%2520of%2520ORY-1001%252C%2520an%2520LSD1%2520inhibitor%252C%2520in%2520relapsed%2520or%2520refractory%2520acute%2520leukaemia%2520%2528AL%2529%26jtitle%3DEU%2520Clinical%2520Trials%2520Register%26pub%3DEuropean%2520Medicines%2520Agency%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span> </span><span class="NLM_article-title">An Open-Label, Dose-Escalation/Dose-Expansion Safety Study of INCB059872 in Subjects With Advanced Malignancies</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/study/NCT02712905" class="extLink">https://clinicaltrials.gov/ct2/show/study/NCT02712905</a> (Accessed October 7, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=An+Open-Label%2C+Dose-Escalation%2FDose-Expansion+Safety+Study+of+INCB059872+in+Subjects+With+Advanced+Malignancies.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2016%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2Fstudy%2FNCT02712905+%28Accessed+October+7%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DAn%2520Open-Label%252C%2520Dose-Escalation%252FDose-Expansion%2520Safety%2520Study%2520of%2520INCB059872%2520in%2520Subjects%2520With%2520Advanced%2520Malignancies%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lgy0Rv_5_DnaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Wu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">3-(Piperidin-4-Ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+Weiauthor=Z.+Fengauthor=L.+Dengauthor=Y.+Song&title=3-%28Piperidin-4-Ylmethoxy%29pyridine+Containing+Compounds+Are+Potent+Inhibitors+of+Lysine+Specific+Demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0ljzUVqCE0epwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26atitle%3D3-%2528Piperidin-4-Ylmethoxy%2529pyridine%2520Containing%2520Compounds%2520Are%2520Potent%2520Inhibitors%2520of%2520Lysine%2520Specific%2520Demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-Benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-Throughput+Virtual+Screening+Identifies+Novel+N%E2%80%B2-%281-Phenylethylidene%29-Benzohydrazides+as+Potent%2C+Specific%2C+and+Reversible+LSD1+Inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0ljzUVqCE0epwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-Throughput%2520Virtual%2520Screening%2520Identifies%2520Novel%2520N%25E2%2580%25B2-%25281-Phenylethylidene%2529-Benzohydrazides%2520as%2520Potent%252C%2520Specific%252C%2520and%2520Reversible%2520LSD1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Sankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulvihill, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckerle, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lessnick, S. L.</span><span> </span><span class="NLM_article-title">Reversible LSD1 Inhibition Interferes with Global EWS/ETS Transcriptional Activity and Impedes Ewing Sarcoma Tumor Growth</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">4584</span><span class="NLM_x">–</span> <span class="NLM_lpage">4597</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-0072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F1078-0432.CCR-14-0072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=24963049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyku7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=4584-4597&author=S.+Sankarauthor=E.+R.+Theisenauthor=J.+Bearssauthor=T.+Mulvihillauthor=L.+M.+Hoffmanauthor=V.+Sornaauthor=M.+C.+Beckerleauthor=S.+Sharmaauthor=S.+L.+Lessnick&title=Reversible+LSD1+Inhibition+Interferes+with+Global+EWS%2FETS+Transcriptional+Activity+and+Impedes+Ewing+Sarcoma+Tumor+Growth&doi=10.1158%2F1078-0432.CCR-14-0072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible LSD1 Inhibition Interferes with Global EWS/ETS Transcriptional Activity and Impedes Ewing Sarcoma Tumor Growth</span></div><div class="casAuthors">Sankar, Savita; Theisen, Emily R.; Bearss, Jared; Mulvihill, Timothy; Hoffman, Laura M.; Sorna, Venkataswamy; Beckerle, Mary C.; Sharma, Sunil; Lessnick, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4584-4597</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Ewing sarcoma is a pediatric bone tumor that absolutely relies on the transcriptional activity of the EWS/ETS family of fusion oncoproteins.  While the most common fusion, EWS/FLI, utilizes lysine-specific demethylase 1 (LSD1) to repress crit. tumor suppressors, small-mol. blockade of LSD1 has not yet been thoroughly explored as a therapeutic approach for Ewing sarcoma.  We therefore evaluated the translational potential of potent and specific LSD1 inhibition with HCI2509 on the transcriptional program of both EWS/FLI and EWS/ERG as well as the downstream oncogenic phenotypes driven by EWS/ETS fusions in both in vitro and in vivo models of Ewing sarcoma.  Exptl. Design: RNA-seq was used to compare the transcriptional profiles of EWS/FLI, EWS/ERG, and treatment with HCI2509 in both EWS/FLI- and EWS/ERG-contg. cell lines.  We then evaluated morphol. phenotypes of treated cells with immunofluorescence.  The induction of apoptosis was evaluated using caspase-3/7 activation and TUNEL staining.  Colony forming assays were used to test oncogenic transformation and xenograft studies with patient-derived cell lines were used to evaluate the effects of HCI2509 on tumorigenesis.  Results: HCI2509 caused a dramatic reversal of both the up- and downregulated transcriptional profiles of EWS/FLI and EWS/ERG accompanied by the induction of apoptosis and disruption of morphol. and oncogenic phenotypes modulated by EWS/FLI.  Importantly, HCI2509 displayed single-agent efficacy in multiple xenograft models.  Conclusions: These data support epigenetic modulation with HCI2509 as a therapeutic strategy for Ewing sarcoma, and highlight a crit. dual role for LSD1 in the oncogenic transcriptional activity of EWS/ETS proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-MyK6B0OhLVg90H21EOLACvtfcHk0ljzUVqCE0epwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyku7zP&md5=1ee7f6fff6279302104d0482759f5d26</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0072%26sid%3Dliteratum%253Aachs%26aulast%3DSankar%26aufirst%3DS.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DBearss%26aufirst%3DJ.%26aulast%3DMulvihill%26aufirst%3DT.%26aulast%3DHoffman%26aufirst%3DL.%2BM.%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DBeckerle%26aufirst%3DM.%2BC.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLessnick%26aufirst%3DS.%2BL.%26atitle%3DReversible%2520LSD1%2520Inhibition%2520Interferes%2520with%2520Global%2520EWS%252FETS%2520Transcriptional%2520Activity%2520and%2520Impedes%2520Ewing%2520Sarcoma%2520Tumor%2520Growth%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D4584%26epage%3D4597%26doi%3D10.1158%2F1078-0432.CCR-14-0072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajiwala, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janat-Amsbury, M.</span><span> </span><span class="NLM_article-title">Reversible Inhibition of Lysine Specific Demethylase 1 Is a Novel Anti-Tumor Strategy for Poorly Differentiated Endometrial Carcinoma</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">752</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1186%2F1471-2407-14-752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=752&author=E.+R.+Theisenauthor=S.+Gajiwalaauthor=J.+Bearssauthor=V.+Sornaauthor=S.+Sharmaauthor=M.+Janat-Amsbury&title=Reversible+Inhibition+of+Lysine+Specific+Demethylase+1+Is+a+Novel+Anti-Tumor+Strategy+for+Poorly+Differentiated+Endometrial+Carcinoma&doi=10.1186%2F1471-2407-14-752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-752%26sid%3Dliteratum%253Aachs%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DGajiwala%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DJ.%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DJanat-Amsbury%26aufirst%3DM.%26atitle%3DReversible%2520Inhibition%2520of%2520Lysine%2520Specific%2520Demethylase%25201%2520Is%2520a%2520Novel%2520Anti-Tumor%2520Strategy%2520for%2520Poorly%2520Differentiated%2520Endometrial%2520Carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D752%26doi%3D10.1186%2F1471-2407-14-752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sartori, L.; Mercurio, C.; Amigoni, F.; Cappa, A.; Fagá, G.; Fattori, R.; Legnaghi, E.; Ciossani, G.; Mattevi, A.; Meroni, G.; Moretti, L.; Cecatiello, V.; Pasqualato, S.; Romussi, A.; Thaler, F.; Trifiró, P.; Villa, M.; Vultaggio, S.; Botrugno, O. A.; Dessanti, P.; Minucci, S.; Zagarrí, E.; Carettoni, D.; Iuzzolino, L.; Varasi, M.; Vianello, P.</span><span> </span><span class="NLM_article-title">Thieno[3,2-B]pyrrole-5-Carboxamides as Novel Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High Throughput Screening and Preliminary Exploration</span>.  <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01018</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=L.+Sartori&author=C.+Mercurio&author=F.+Amigoni&author=A.+Cappa&author=G.+Fag%C3%A1&author=R.+Fattori&author=E.+Legnaghi&author=G.+Ciossani&author=A.+Mattevi&author=G.+Meroni&author=L.+Moretti&author=V.+Cecatiello&author=S.+Pasqualato&author=A.+Romussi&author=F.+Thaler&author=P.+Trifir%C3%B3&author=M.+Villa&author=S.+Vultaggio&author=O.+A.+Botrugno&author=P.+Dessanti&author=S.+Minucci&author=E.+Zagarr%C3%AD&author=D.+Carettoni&author=L.+Iuzzolino&author=M.+Varasi&author=P.+Vianello&title=Thieno%5B3%2C2-B%5Dpyrrole-5-Carboxamides+as+Novel+Reversible+Inhibitors+of+Histone+Lysine+Demethylase+KDM1A%2FLSD1.+Part+1%3A+High+Throughput+Screening+and+Preliminary+Exploration&doi=10.1021%2Facs.jmedchem.6b01018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DThieno%255B3%252C2-B%255Dpyrrole-5-Carboxamides%2520as%2520Novel%2520Reversible%2520Inhibitors%2520of%2520Histone%2520Lysine%2520Demethylase%2520KDM1A%252FLSD1.%2520Part%25201%253A%2520High%2520Throughput%2520Screening%2520and%2520Preliminary%2520Exploration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Ching, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merits, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, C. L. L.</span><span> </span><span class="NLM_article-title">Trisubstituted thieno[3,2-B]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9196</span><span class="NLM_x">–</span> <span class="NLM_lpage">9213</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9196-9213&author=K.+C.+Chingauthor=Y.+W.+Kamauthor=A.+Meritsauthor=L.+F.+P.+Ngauthor=C.+L.+L.+Chai&title=Trisubstituted+thieno%5B3%2C2-B%5Dpyrrole+5-Carboxamides+as+Potent+Inhibitors+of+Alphaviruses&doi=10.1021%2Facs.jmedchem.5b01047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Thieno[3,2-b]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span></div><div class="casAuthors">Ching, Kuan-Chieh; Kam, Yiu-Wing; Merits, Andres; Ng, Lisa F. P.; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9196-9213</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chikungunya virus (CHIKV) is a re-emerging vector-borne alphavirus and is transmitted to humans by Aedes mosquitoes.  Despite the re-emergence of CHIKV as an epidemic threat, there is no approved effective antiviral treatment currently available for CHIKV.  Herein, we report the synthesis and structure-activity relationship studies of a class of thieno[3,2-b]pyrroles and the discovery of a trisubstituted thieno[3,2-b]pyrrole 5-carboxamide 15c (I) that exhibits potent inhibitory activity against in vitro CHIKV infection.  Compd. 15c displayed low micromolar activity (EC50 value of ca. 2 μM) and limited cytotoxic liability (CC50 > 100 μM) therefore furnishing a selectivity index of greater than 32.  Notably, 15c not only controlled viral RNA prodn., but efficiently inhibited the expression of CHIKV nsP1, nsP3, capsid, and E2 proteins at a concn. as low as 2.5 μM.  More importantly, 15c also demonstrated broad spectrum antiviral activity against other clin. important alphaviruses such as O'nyong-nyong virus and Sindbis virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC7wEIPTmwQ7Vg90H21EOLACvtfcHk0lg9QctiS2BcxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI&md5=dc938296fd5fd7628b7de37590b6918a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01047%26sid%3Dliteratum%253Aachs%26aulast%3DChing%26aufirst%3DK.%2BC.%26aulast%3DKam%26aufirst%3DY.%2BW.%26aulast%3DMerits%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DL.%2BF.%2BP.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DTrisubstituted%2520thieno%255B3%252C2-B%255Dpyrrole%25205-Carboxamides%2520as%2520Potent%2520Inhibitors%2520of%2520Alphaviruses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9196%26epage%3D9213%26doi%3D10.1021%2Facs.jmedchem.5b01047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Venable, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvorak, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grice, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jablonowski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pio, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurmond, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovenberg, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carruthers, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, J. P.</span><span> </span><span class="NLM_article-title">Preparation and Biological Evaluation of Indole, Benzimidazole, and Thienopyrrole Piperazine Carboxamides: Potent Human Histamine H4 Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8289</span><span class="NLM_x">–</span> <span class="NLM_lpage">8298</span><span class="refDoi"> DOI: 10.1021/jm0502081</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0502081" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=8289-8298&author=J.+D.+Venableauthor=H.+Caiauthor=W.+Chaiauthor=C.+A.+Dvorakauthor=C.+A.+Griceauthor=J.+A.+Jablonowskiauthor=C.+R.+Shahauthor=A.+K.+Kwokauthor=K.+S.+Lyauthor=B.+Pioauthor=J.+Weiauthor=P.+J.+Desaiauthor=W.+Jiangauthor=S.+Nguyenauthor=P.+Lingauthor=S.+J.+Wilsonauthor=P.+J.+Dunfordauthor=R.+L.+Thurmondauthor=T.+W.+Lovenbergauthor=L.+Karlssonauthor=N.+I.+Carruthersauthor=J.+P.+Edwards&title=Preparation+and+Biological+Evaluation+of+Indole%2C+Benzimidazole%2C+and+Thienopyrrole+Piperazine+Carboxamides%3A+Potent+Human+Histamine+H4+Antagonists&doi=10.1021%2Fjm0502081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0502081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0502081%26sid%3Dliteratum%253Aachs%26aulast%3DVenable%26aufirst%3DJ.%2BD.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DChai%26aufirst%3DW.%26aulast%3DDvorak%26aufirst%3DC.%2BA.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26aulast%3DJablonowski%26aufirst%3DJ.%2BA.%26aulast%3DShah%26aufirst%3DC.%2BR.%26aulast%3DKwok%26aufirst%3DA.%2BK.%26aulast%3DLy%26aufirst%3DK.%2BS.%26aulast%3DPio%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DDesai%26aufirst%3DP.%2BJ.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DLing%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DS.%2BJ.%26aulast%3DDunford%26aufirst%3DP.%2BJ.%26aulast%3DThurmond%26aufirst%3DR.%2BL.%26aulast%3DLovenberg%26aufirst%3DT.%2BW.%26aulast%3DKarlsson%26aufirst%3DL.%26aulast%3DCarruthers%26aufirst%3DN.%2BI.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26atitle%3DPreparation%2520and%2520Biological%2520Evaluation%2520of%2520Indole%252C%2520Benzimidazole%252C%2520and%2520Thienopyrrole%2520Piperazine%2520Carboxamides%253A%2520Potent%2520Human%2520Histamine%2520H4%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D8289%26epage%3D8298%26doi%3D10.1021%2Fjm0502081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Ilyin, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dmitrieva, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kustova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manaev, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivachtchenko, A. V.</span><span> </span><span class="NLM_article-title">Synthesis of Heterocyclic Compounds Possessing the 4H-thieno[3,2-B]pyrrole Moiety</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1021/cc060091h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc060091h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWlsL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=96-106&author=A.+P.+Ilyinauthor=I.+G.+Dmitrievaauthor=V.+A.+Kustovaauthor=A.+V.+Manaevauthor=A.+V.+Ivachtchenko&title=Synthesis+of+Heterocyclic+Compounds+Possessing+the+4H-thieno%5B3%2C2-B%5Dpyrrole+Moiety&doi=10.1021%2Fcc060091h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Heterocyclic Compounds Possessing the 4H-Thieno[3,2-b]Pyrrole Moiety</span></div><div class="casAuthors">Ilyin, Alexey P.; Dmitrieva, Irina G.; Kustova, Veronika A.; Manaev, Alexandre V.; Ivachtchenko, Alexandre V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-106</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of novel heterocyclic combinatorial libraries contg. 4H-thieno[3,2-b]pyrrole, thieno[2',3':4,5]pyrrolo[1,2-d][1,2,4]triazine and thieno[2',3':4,5]pyrrolo[1,2-a]pyrazine heterocyclic moieties were obtained by parallel soln.-phase synthesis.  Key steps include different reactions of initial alkyl 4H-thieno[3,2-b]pyrrole-5-carboxylates, such as alkylation with alkylating agents; transformation of the carboxylate group into different reactive functionalities, followed by reactions with electrophilic species; intramol. cyclizations; and amide bond formation.  Simple manual techniques for parallel reactions were coupled with easy purifn. procedures to give high-purity final products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF9Lw2eRWTmLVg90H21EOLACvtfcHk0lg9QctiS2BcxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWlsL%252FF&md5=2e66224e7fdb6d92537b551ed16ce82b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcc060091h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc060091h%26sid%3Dliteratum%253Aachs%26aulast%3DIlyin%26aufirst%3DA.%2BP.%26aulast%3DDmitrieva%26aufirst%3DI.%2BG.%26aulast%3DKustova%26aufirst%3DV.%2BA.%26aulast%3DManaev%26aufirst%3DA.%2BV.%26aulast%3DIvachtchenko%26aufirst%3DA.%2BV.%26atitle%3DSynthesis%2520of%2520Heterocyclic%2520Compounds%2520Possessing%2520the%25204H-thieno%255B3%252C2-B%255Dpyrrole%2520Moiety%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2007%26volume%3D9%26spage%3D96%26epage%3D106%26doi%3D10.1021%2Fcc060091h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Soucy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favreau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayser, M. M.</span><span> </span><span class="NLM_article-title">The Regioselectivity of Metal Hydride Reductions of 3-Substituted Phthalic Anhydrides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1021/jo00377a023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00377a023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADyaL2sXmslSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1987&pages=129-134&author=C.+Soucyauthor=D.+Favreauauthor=M.+M.+Kayser&title=The+Regioselectivity+of+Metal+Hydride+Reductions+of+3-Substituted+Phthalic+Anhydrides&doi=10.1021%2Fjo00377a023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The regioselectivity of metal hydride reductions of 3-substituted phthalic anhydrides</span></div><div class="casAuthors">Soucy, C.; Favreau, D.; Kayser, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-34</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The redn. of 3-methoxyphthalide by metal hydrides was reinvestigated.  Various effects controlling selectivity of redns. in 3-substituted phthalides were studied, and a qual. interpretation of the results was proposed.  Methods for obtaining enhanced yields of one or the other lactonic product were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU_DHjeYOELrVg90H21EOLACvtfcHk0li6wjPQtWuuBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmslSgsg%253D%253D&md5=3993790041c50d330cffaa9c6235bd21</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjo00377a023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00377a023%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DC.%26aulast%3DFavreau%26aufirst%3DD.%26aulast%3DKayser%26aufirst%3DM.%2BM.%26atitle%3DThe%2520Regioselectivity%2520of%2520Metal%2520Hydride%2520Reductions%2520of%25203-Substituted%2520Phthalic%2520Anhydrides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1987%26volume%3D52%26spage%3D129%26epage%3D134%26doi%3D10.1021%2Fjo00377a023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+Basis+of+LSD1-CoREST+Selectivity+in+Histone+H3+Recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0li6wjPQtWuuBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520LSD1-CoREST%2520Selectivity%2520in%2520Histone%2520H3%2520Recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+P.+Somervaille&title=The+Histone+Demethylase+KDM1A+Sustains+the+Oncogenic+Potential+of+MLL-AF9+Leukemia+Stem+Cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0li6wjPQtWuuBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DThe%2520Histone%2520Demethylase%2520KDM1A%2520Sustains%2520the%2520Oncogenic%2520Potential%2520of%2520MLL-AF9%2520Leukemia%2520Stem%2520Cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanigan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Loss of LSD1 (Lysine-Specific Demethylase 1) Suppresses Growth and Alters Gene Expression of Human Colon Cancer Cells in a p53- and DNMT1(DNA Methyltransferase 1)-Independent Manner</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">459</span><span class="NLM_x">–</span> <span class="NLM_lpage">468</span><span class="refDoi"> DOI: 10.1042/BJ20121360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1042%2FBJ20121360" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=459-468&author=L.+Jinauthor=C.+L.+Haniganauthor=Y.+Wuauthor=W.+Wangauthor=B.+H.+Parkauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Loss+of+LSD1+%28Lysine-Specific+Demethylase+1%29+Suppresses+Growth+and+Alters+Gene+Expression+of+Human+Colon+Cancer+Cells+in+a+p53-+and+DNMT1%28DNA+Methyltransferase+1%29-Independent+Manner&doi=10.1042%2FBJ20121360"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1042%2FBJ20121360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121360%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHanigan%26aufirst%3DC.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DLoss%2520of%2520LSD1%2520%2528Lysine-Specific%2520Demethylase%25201%2529%2520Suppresses%2520Growth%2520and%2520Alters%2520Gene%2520Expression%2520of%2520Human%2520Colon%2520Cancer%2520Cells%2520in%2520a%2520p53-%2520and%2520DNMT1%2528DNA%2520Methyltransferase%25201%2529-Independent%2520Manner%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D459%26epage%3D468%26doi%3D10.1042%2FBJ20121360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone Demethylase Lsd1 Represses Hematopoietic Stem and Progenitor Cell Signatures during Blood Cell Maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.-J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+Demethylase+Lsd1+Represses+Hematopoietic+Stem+and+Progenitor+Cell+Signatures+during+Blood+Cell+Maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0lgHwPwGXoPQSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520Demethylase%2520Lsd1%2520Represses%2520Hematopoietic%2520Stem%2520and%2520Progenitor%2520Cell%2520Signatures%2520during%2520Blood%2520Cell%2520Maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgHwPwGXoPQSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Winter, G.</span><span> </span><span class="NLM_article-title">Xia2: An Expert System for Macromolecular Crystallography Data Reduction</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span><span class="refDoi"> DOI: 10.1107/S0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=Xia2%3A+An+Expert+System+for+Macromolecular+Crystallography+Data+Reduction&doi=10.1107%2FS0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0lj6EL5X3EPlkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DXia2%253A%2520An%2520Expert%2520System%2520for%2520Macromolecular%2520Crystallography%2520Data%2520Reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2FS0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lj6EL5X3EPlkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lj6EL5X3EPlkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgZWTvZorzZ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lgZWTvZorzZ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohn, J. D.</span><span> </span><span class="NLM_article-title">Automated Ligand Fitting by Core-Fragment Fitting and Extension into Density</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1107/S0907444906017161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444906017161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16855309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=915-922&author=T.+C.+Terwilligerauthor=H.+Kleiauthor=P.+D.+Adamsauthor=N.+W.+Moriartyauthor=J.+D.+Cohn&title=Automated+Ligand+Fitting+by+Core-Fragment+Fitting+and+Extension+into+Density&doi=10.1107%2FS0907444906017161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Automated ligand fitting by core-fragment fitting and extension into density</span></div><div class="casAuthors">Terwilliger, Thomas C.; Klei, Herbert; Adams, Paul D.; Moriarty, Nigel W.; Cohn, Judith D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">915-922</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A procedure for fitting of ligands to electron-d. maps by first fitting a core fragment of the ligand to d. and then extending the remainder of the ligand into d. is presented.  The approach was tested by fitting 9327 ligands over a wide range of resolns. (most are in the range 0.8-4.8 Å) from the Protein Data Bank (PDB) into (Fo - Fc)exp(iφc) difference d. calcd. using entries from the PDB without these ligands.  The procedure was able to place 58% of these 9327 ligands within 2 Å (r.m.s.d.) of the coordinates of the atoms in the original PDB entry for that ligand.  The success of the fitting procedure was relatively insensitive to the size of the ligand in the range 10-100 non-H atoms and was only moderately sensitive to resoln., with the percentage of ligands placed near the coordinates of the original PDB entry for fits in the range 58-73% over all resoln. ranges tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZHd3GwwvtWLVg90H21EOLACvtfcHk0lgZWTvZorzZ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksLc%253D&md5=dc3efbc9f8249c27ea9fa6658d31e316</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444906017161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906017161%26sid%3Dliteratum%253Aachs%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DKlei%26aufirst%3DH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DCohn%26aufirst%3DJ.%2BD.%26atitle%3DAutomated%2520Ligand%2520Fitting%2520by%2520Core-Fragment%2520Fitting%2520and%2520Extension%2520into%2520Density%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D915%26epage%3D922%26doi%3D10.1107%2FS0907444906017161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Moretti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, L.</span><span> </span><span class="NLM_article-title">Software Infrastructure for Computer-Aided Drug Discovery and Development, a Practical Example with Guidelines</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span><span class="refDoi"> DOI: 10.1002/minf.201501037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fminf.201501037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=27546042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=382-390&author=L.+Morettiauthor=L.+Sartori&title=Software+Infrastructure+for+Computer-Aided+Drug+Discovery+and+Development%2C+a+Practical+Example+with+Guidelines&doi=10.1002%2Fminf.201501037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Software Infrastructure for Computer-aided Drug Discovery and Development, a Practical Example with Guidelines</span></div><div class="casAuthors">Moretti, Loris; Sartori, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">382-390</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the field of Computer-Aided Drug Discovery and Development (CADDD) the proper software infrastructure is essential for everyday investigations.  The creation of such an environment should be carefully planned and implemented with certain features in order to be productive and efficient.  Here, we describe a soln. to integrate std. computational services into a functional unit that empowers modeling applications for drug discovery.  This system allows users with various level of expertise to run in silico expts. automatically and without the burden of file formatting for different software, managing the actual computation, keeping track of the activities and graphical rendering of the structural outcomes.  To showcase the potential of this approach, performances of five different docking programs on an Hiv-1 protease test set are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUtVBT1Wj49LVg90H21EOLACvtfcHk0lgZWTvZorzZ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL&md5=c9db5ce783d44c6d15f889cbec5ca3f7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201501037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201501037%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DSoftware%2520Infrastructure%2520for%2520Computer-Aided%2520Drug%2520Discovery%2520and%2520Development%252C%2520a%2520Practical%2520Example%2520with%2520Guidelines%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D382%26epage%3D390%26doi%3D10.1002%2Fminf.201501037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 37 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alessia Romussi, Anna Cappa, Paola Vianello, Silvia Brambillasca, Maria Rosaria Cera, Roberto Dal Zuffo, Giovanni Fagà, Raimondo Fattori, Loris Moretti, Paolo Trifirò, Manuela Villa, Stefania Vultaggio, Valentina Cecatiello, Sebastiano Pasqualato, Giulio Dondio, Chi Wai Eric So, Saverio Minucci, Luca Sartori, Mario Varasi, <span class="NLM_string-name hlFld-ContribAuthor">Ciro Mercurio</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible Inhibitors of KDM1A Efficacious in Acute Myeloid Leukemia Models. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 754-759. <a href="https://doi.org/10.1021/acsmedchemlett.9b00604" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00604</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00604%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BReversible%252BInhibitors%252Bof%252BKDM1A%252BEfficacious%252Bin%252BAcute%252BMyeloid%252BLeukemia%252BModels%26aulast%3DRomussi%26aufirst%3DAlessia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D16122019%26date%3D13022020%26date%3D20022020%26date%3D13022020%26volume%3D11%26issue%3D5%26spage%3D754%26epage%3D759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel P.  Mould</span>, <span class="hlFld-ContribAuthor ">Cristina  Alli</span>, <span class="hlFld-ContribAuthor ">Ulf  Bremberg</span>, <span class="hlFld-ContribAuthor ">Sharon  Cartic</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>, <span class="hlFld-ContribAuthor ">Matthis  Geitmann</span>, <span class="hlFld-ContribAuthor ">Alba  Maiques-Diaz</span>, <span class="hlFld-ContribAuthor ">Alison E.  McGonagle</span>, <span class="hlFld-ContribAuthor ">Tim C. P.  Somervaille</span>, <span class="hlFld-ContribAuthor ">Gary J.  Spencer</span>, <span class="hlFld-ContribAuthor ">Fabrice  Turlais</span>, and <span class="hlFld-ContribAuthor ">Donald  Ogilvie</span>  . </span><span class="cited-content_cbyCitation_article-title">Development of (4-Cyanophenyl)glycine Derivatives as Reversible Inhibitors of Lysine Specific Demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (19)
                                     , 7984-7999. <a href="https://doi.org/10.1021/acs.jmedchem.7b00462" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00462%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B%2525284-Cyanophenyl%252529glycine%252BDerivatives%252Bas%252BReversible%252BInhibitors%252Bof%252BLysine%252BSpecific%252BDemethylase%252B1%26aulast%3DMould%26aufirst%3DDaniel%2BP.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28032017%26date%3D21092017%26date%3D12102017%26date%3D11092017%26volume%3D60%26issue%3D19%26spage%3D7984%26epage%3D7999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Sartori</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Giovanni  Fagá</span>, <span class="hlFld-ContribAuthor ">Raimondo  Fattori</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Loris  Moretti</span>, <span class="hlFld-ContribAuthor ">Valentina  Cecatiello</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Pasqualato</span>, <span class="hlFld-ContribAuthor ">Alessia  Romussi</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Paolo  Trifiró</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Elisa  Zagarrí</span>, <span class="hlFld-ContribAuthor ">Daniele  Carettoni</span>, <span class="hlFld-ContribAuthor ">Lucia  Iuzzolino</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Paola  Vianello</span>  . </span><span class="cited-content_cbyCitation_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1673-1692. <a href="https://doi.org/10.1021/acs.jmedchem.6b01018" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%252Bas%252BNew%252BReversible%252BInhibitors%252Bof%252BHistone%252BLysine%252BDemethylase%252BKDM1A%25252FLSD1.%252BPart%252B1%25253A%252BHigh-Throughput%252BScreening%252Band%252BPreliminary%252BExploration%26aulast%3DSartori%26aufirst%3DLuca%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D25072016%26date%3D27022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D1673%26epage%3D1692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Avineesh  Singh</span>, <span class="hlFld-ContribAuthor ">Vijay K.  Patel</span>, <span class="hlFld-ContribAuthor ">Harish  Rajak</span>. </span><span class="cited-content_cbyCitation_article-title">Appraisal of pyrrole as connecting unit in hydroxamic acid based histone deacetylase inhibitors: Synthesis, anticancer evaluation and molecular docking studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1240 </em>, 130590. <a href="https://doi.org/10.1016/j.molstruc.2021.130590" title="DOI URL">https://doi.org/10.1016/j.molstruc.2021.130590</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2021.130590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2021.130590%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DAppraisal%252Bof%252Bpyrrole%252Bas%252Bconnecting%252Bunit%252Bin%252Bhydroxamic%252Bacid%252Bbased%252Bhistone%252Bdeacetylase%252Binhibitors%25253A%252BSynthesis%25252C%252Banticancer%252Bevaluation%252Band%252Bmolecular%252Bdocking%252Bstudies%26aulast%3DSingh%26aufirst%3DAvineesh%26date%3D2021%26volume%3D1240%26spage%3D130590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Hong-Mei  Ren</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Yue-Jiao  Liu</span>, <span class="hlFld-ContribAuthor ">Yong-Tao  Xu</span>, <span class="hlFld-ContribAuthor ">Zi-Hao  He</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Hang  Yuan</span>, <span class="hlFld-ContribAuthor ">Shu-Hui  Chen</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of novel dual inhibitors targeting lysine specific demethylase 1 (LSD1) and histone deacetylases (HDAC) for treatment of gastric cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113453. <a href="https://doi.org/10.1016/j.ejmech.2021.113453" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113453</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113453%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bdual%252Binhibitors%252Btargeting%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Band%252Bhistone%252Bdeacetylases%252B%252528HDAC%252529%252Bfor%252Btreatment%252Bof%252Bgastric%252Bcancer%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D220%26spage%3D113453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong  Shao</span>, <span class="hlFld-ContribAuthor ">Fang  Wang</span>, <span class="hlFld-ContribAuthor ">Jingqi  Shi</span>, <span class="hlFld-ContribAuthor ">Lifang  Ma</span>, <span class="hlFld-ContribAuthor ">Ziyuan  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Synergetic copper/TEMPO-catalysed benzylic C–H imidation with
              N
              -fluorobenzenesulfonimide at room temperature and tandem conversions with alcohols or arenes. </span><span class="cited-content_cbyCitation_journal-name">Organic Chemistry Frontiers</span><span> <strong>2021,</strong> <em>8 </em>
                                    (13)
                                     , 3298-3307. <a href="https://doi.org/10.1039/D1QO00340B" title="DOI URL">https://doi.org/10.1039/D1QO00340B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1QO00340B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1QO00340B%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Chemistry%2520Frontiers%26atitle%3DSynergetic%252Bcopper%25252FTEMPO-catalysed%252Bbenzylic%252BC%2525E2%252580%252593H%252Bimidation%252Bwith%252BN%252B-fluorobenzenesulfonimide%252Bat%252Broom%252Btemperature%252Band%252Btandem%252Bconversions%252Bwith%252Balcohols%252Bor%252Barenes%26aulast%3DShao%26aufirst%3DZhong%26date%3D2021%26date%3D2021%26volume%3D8%26issue%3D13%26spage%3D3298%26epage%3D3307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Fang</span>, <span class="hlFld-ContribAuthor ">Chunsheng  Hu</span>, <span class="hlFld-ContribAuthor ">Yong  Ding</span>, <span class="hlFld-ContribAuthor ">Hongxia  Qin</span>, <span class="hlFld-ContribAuthor ">Yafei  Luo</span>, <span class="hlFld-ContribAuthor ">Zhigang  Xu</span>, <span class="hlFld-ContribAuthor ">Jiangping  Meng</span>, <span class="hlFld-ContribAuthor ">Zhongzhu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of
              
                4
                H
              
              ‐thieno[3,2‐
              b
              ]pyrrole derivatives as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2021,</strong> <em>15 </em><a href="https://doi.org/10.1002/jhet.4285" title="DOI URL">https://doi.org/10.1002/jhet.4285</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.4285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.4285%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DDiscovery%252Bof%252B4%252BH%252B%2525E2%252580%252590thieno%25255B3%25252C2%2525E2%252580%252590%252Bb%252B%25255Dpyrrole%252Bderivatives%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DFang%26aufirst%3DBo%26date%3D2021%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena Yu.  Schmidt</span>, <span class="hlFld-ContribAuthor ">Nadezhda V.  Semenova</span>, <span class="hlFld-ContribAuthor ">Inna V.  Tatarinova</span>, <span class="hlFld-ContribAuthor ">Igor A.  Ushakov</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Vashchenko</span>, <span class="hlFld-ContribAuthor ">Boris A.  Trofimov</span>. </span><span class="cited-content_cbyCitation_article-title">Functionalized Thieno[3,2‐
              b
              ]pyrroles from Acylthiophenes, Acetylene Gas and Hydrazines in Two Steps. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>2021 </em>
                                    (19)
                                     , 2802-2806. <a href="https://doi.org/10.1002/ejoc.202100377" title="DOI URL">https://doi.org/10.1002/ejoc.202100377</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100377%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DFunctionalized%252BThieno%25255B3%25252C2%2525E2%252580%252590%252Bb%252B%25255Dpyrroles%252Bfrom%252BAcylthiophenes%25252C%252BAcetylene%252BGas%252Band%252BHydrazines%252Bin%252BTwo%252BSteps%26aulast%3DSchmidt%26aufirst%3DElena%2BYu.%26date%3D2021%26date%3D2021%26volume%3D2021%26issue%3D19%26spage%3D2802%26epage%3D2806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking, 3D-QSAR, and molecular dynamics simulations of thieno[3,2-b]pyrrole derivatives against anticancer targets of KDM1A/LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>39 </em>
                                    (4)
                                     , 1189-1202. <a href="https://doi.org/10.1080/07391102.2020.1726819" title="DOI URL">https://doi.org/10.1080/07391102.2020.1726819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1726819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1726819%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%25252C%252B3D-QSAR%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bof%252Bthieno%25255B3%25252C2-b%25255Dpyrrole%252Bderivatives%252Bagainst%252Banticancer%252Btargets%252Bof%252BKDM1A%25252FLSD1%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1189%26epage%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Jun  Fu</span>, <span class="hlFld-ContribAuthor ">Jun  Li</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Annual review of LSD1/KDM1A inhibitors in 2020. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113254. <a href="https://doi.org/10.1016/j.ejmech.2021.113254" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113254%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAnnual%252Breview%252Bof%252BLSD1%25252FKDM1A%252Binhibitors%252Bin%252B2020%26aulast%3DFu%26aufirst%3DDong-Jun%26date%3D2021%26volume%3D214%26spage%3D113254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Z. F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">New Carboxamides of the Thieno[3,2-b]pyrrole Series. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>56 </em>
                                    (10)
                                     , 1850-1853. <a href="https://doi.org/10.1134/S1070428020100309" title="DOI URL">https://doi.org/10.1134/S1070428020100309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428020100309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428020100309%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DNew%252BCarboxamides%252Bof%252Bthe%252BThieno%25255B3%25252C2-b%25255Dpyrrole%252BSeries%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2020%26date%3D2020%26volume%3D56%26issue%3D10%26spage%3D1850%26epage%3D1853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samir  Mehndiratta</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 969-981. <a href="https://doi.org/10.1039/D0MD00141D" title="DOI URL">https://doi.org/10.1039/D0MD00141D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00141D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00141D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DMehndiratta%26aufirst%3DSamir%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D969%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Jiming  Wang</span>, <span class="hlFld-ContribAuthor ">Qinwen  Jiang</span>, <span class="hlFld-ContribAuthor ">Liyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>194 </em>, 112243. <a href="https://doi.org/10.1016/j.ejmech.2020.112243" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Btetrahydroquinoline-based%252Breversible%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXinran%26date%3D2020%26volume%3D194%26spage%3D112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Hongyi  Liu</span>, <span class="hlFld-ContribAuthor ">Yifan  Chen</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yi  Yu</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR, molecular docking, and molecular dynamics simulation study of thieno[3,2-
              b
              ]pyrrole-5-carboxamide derivatives as LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 6927-6943. <a href="https://doi.org/10.1039/C9RA10085G" title="DOI URL">https://doi.org/10.1039/C9RA10085G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA10085G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA10085G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3D3D-QSAR%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudy%252Bof%252Bthieno%25255B3%25252C2-%252Bb%252B%25255Dpyrrole-5-carboxamide%252Bderivatives%252Bas%252BLSD1%252Binhibitors%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D6927%26epage%3D6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Z. F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">4H-Thieno[3,2-b]pyrrole-5-carboxylate Conjugates with Taurine and Its Tetrabutylammonium Salt. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>55 </em>
                                    (12)
                                     , 1902-1906. <a href="https://doi.org/10.1134/S1070428019120157" title="DOI URL">https://doi.org/10.1134/S1070428019120157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428019120157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428019120157%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3D4H-Thieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxylate%252BConjugates%252Bwith%252BTaurine%252Band%252BIts%252BTetrabutylammonium%252BSalt%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2019%26date%3D2020%26volume%3D55%26issue%3D12%26spage%3D1902%26epage%3D1906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0811-9" title="DOI URL">https://doi.org/10.1186/s13045-019-0811-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DLSD1%25252FKDM1A%252Binhibitors%252Bin%252Bclinical%252Btrials%25253A%252Badvances%252Band%252Bprospects%26aulast%3DFang%26aufirst%3DYuan%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Linfeng  Jin</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Qinghe  Gao</span>, <span class="hlFld-ContribAuthor ">Yingchao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Binding Mode of Reversible LSD1 Inhibitors Derived from Stilbene Derivatives by 3D-QSAR, Molecular Docking, and Molecular Dynamics Simulation. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4479. <a href="https://doi.org/10.3390/molecules24244479" title="DOI URL">https://doi.org/10.3390/molecules24244479</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244479%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DInvestigating%252Bthe%252BBinding%252BMode%252Bof%252BReversible%252BLSD1%252BInhibitors%252BDerived%252Bfrom%252BStilbene%252BDerivatives%252Bby%252B3D-QSAR%25252C%252BMolecular%252BDocking%25252C%252Band%252BMolecular%252BDynamics%252BSimulation%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Qi-Bing  Gao</span>, <span class="hlFld-ContribAuthor ">Ying  Yan</span>, <span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Li</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying the novel inhibitors of lysine-specific demethylase 1 (LSD1) combining pharmacophore-based and structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (16)
                                     , 4200-4214. <a href="https://doi.org/10.1080/07391102.2018.1538903" title="DOI URL">https://doi.org/10.1080/07391102.2018.1538903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1538903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1538903%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DIdentifying%252Bthe%252Bnovel%252Binhibitors%252Bof%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Bcombining%252Bpharmacophore-based%252Band%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DSun%26aufirst%3DXu-Dong%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D16%26spage%3D4200%26epage%3D4214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 388-401. <a href="https://doi.org/10.1016/j.ejmech.2019.02.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bstructure-activity%252Brelationship%252Bstudies%252Band%252Bbiological%252Bcharacterization%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine-based%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacoepigenetics of LSD1 Inhibitors in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 523-530. <a href="https://doi.org/10.1016/B978-0-12-813939-4.00014-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-813939-4.00014-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813939-4.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813939-4.00014-0%26sid%3Dliteratum%253Aachs%26atitle%3DPharmacoepigenetics%252Bof%252BLSD1%252BInhibitors%252Bin%252BCancer%26aulast%3DYu%26aufirst%3DBin%26date%3D2019%26spage%3D523%26epage%3D530%26pub%3DElsevier%26atitle%3DPharmacoepigenetics%26date%3D2019%26volume%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soraya  Alnabulsi</span>, <span class="hlFld-ContribAuthor ">Enas A.  Al-Hurani</span>, <span class="hlFld-ContribAuthor ">Nizar A.  Al-shar'i</span>, <span class="hlFld-ContribAuthor ">Tamam  El-Elimat</span>. </span><span class="cited-content_cbyCitation_article-title">. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2019,</strong>,, 107440. <a href="https://doi.org/10.1016/j.jmgm.2019.107440" title="DOI URL">https://doi.org/10.1016/j.jmgm.2019.107440</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2019.107440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2019.107440%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26aulast%3DAlnabulsi%26aufirst%3DSoraya%26date%3D2019%26volume%3D93%26spage%3D107440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mode and unbinding mechanism of LSD1 inhibitors by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (47)
                                     , 29833-29846. <a href="https://doi.org/10.1039/C8CP03090A" title="DOI URL">https://doi.org/10.1039/C8CP03090A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03090A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03090A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmode%252Band%252Bunbinding%252Bmechanism%252Bof%252BLSD1%252Binhibitors%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DSun%26aufirst%3DXudong%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D47%26spage%3D29833%26epage%3D29846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Chen  Zhou</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Qihua  Zhu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (17)
                                     , 4871-4880. <a href="https://doi.org/10.1016/j.bmc.2018.08.026" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.08.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.08.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252B5-arylidene%252Bbarbiturates%252Bas%252Bpotent%25252C%252Bselective%25252C%252Breversible%252BLSD1%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bacute%252Bpromyelocytic%252Bleukemia%26aulast%3DXu%26aufirst%3DSiyuan%26date%3D2018%26volume%3D26%26issue%3D17%26spage%3D4871%26epage%3D4880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seda A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Zoya F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">Vadim V.  Zagitov</span>, <span class="hlFld-ContribAuthor ">Fanusa A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">Mansur S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">New 4H-thieno[3,2-b]pyrrole-5-carboxamides. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2018,</strong> <em>54 </em>
                                    (8)
                                     , 819-822. <a href="https://doi.org/10.1007/s10593-018-2355-2" title="DOI URL">https://doi.org/10.1007/s10593-018-2355-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-018-2355-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-018-2355-2%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DNew%252B4H-thieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%26aulast%3DTorosyan%26aufirst%3DSeda%2BA.%26date%3D2018%26date%3D2018%26volume%3D54%26issue%3D8%26spage%3D819%26epage%3D822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xueyuan  Bi</span>, <span class="hlFld-ContribAuthor ">Yanshen  Ren</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>78 </em>, 7-16. <a href="https://doi.org/10.1016/j.bioorg.2018.02.016" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.02.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B4-%2525284-benzyloxy%252529phenoxypiperidines%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Shin  Sato</span>, <span class="hlFld-ContribAuthor ">Tomoko  Hashimoto</span>, <span class="hlFld-ContribAuthor ">Kenji  Matsuno</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure of LSD1 in Complex with 4-[5-(Piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (7)
                                     , 1538. <a href="https://doi.org/10.3390/molecules23071538" title="DOI URL">https://doi.org/10.3390/molecules23071538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23071538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23071538%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCrystal%252BStructure%252Bof%252BLSD1%252Bin%252BComplex%252Bwith%252B4-%25255B5-%252528Piperidin-4-ylmethoxy%252529-2-%252528p-tolyl%252529pyridin-3-yl%25255Dbenzonitrile%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D7%26spage%3D1538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic drug discovery: a success story for cofactor interference. </span><span class="cited-content_cbyCitation_journal-name">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span> <strong>2018,</strong> <em>373 </em>
                                    (1748)
                                     , 20170069. <a href="https://doi.org/10.1098/rstb.2017.0069" title="DOI URL">https://doi.org/10.1098/rstb.2017.0069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1098/rstb.2017.0069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1098%2Frstb.2017.0069%26sid%3Dliteratum%253Aachs%26jtitle%3DPhilosophical%2520Transactions%2520of%2520the%2520Royal%2520Society%2520B%253A%2520Biological%2520Sciences%26atitle%3DEpigenetic%252Bdrug%252Bdiscovery%25253A%252Ba%252Bsuccess%252Bstory%252Bfor%252Bcofactor%252Binterference%26aulast%3DGanesan%26aufirst%3DA.%26date%3D2018%26date%3D2018%26volume%3D373%26issue%3D1748%26spage%3D20170069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Priya  Chacko</span>, <span class="hlFld-ContribAuthor ">Kalegowda  Shivashankar</span>. </span><span class="cited-content_cbyCitation_article-title">I 2 -catalyzed one-pot synthesis of benzofuro/thieno[2,3- b ]pyrrole motifs. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2018,</strong> <em>74 </em>
                                    (13)
                                     , 1520-1526. <a href="https://doi.org/10.1016/j.tet.2018.02.013" title="DOI URL">https://doi.org/10.1016/j.tet.2018.02.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2018.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2018.02.013%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DI%252B2%252B-catalyzed%252Bone-pot%252Bsynthesis%252Bof%252Bbenzofuro%25252Fthieno%25255B2%25252C3-%252Bb%252B%25255Dpyrrole%252Bmotifs%26aulast%3DChacko%26aufirst%3DPriya%26date%3D2018%26volume%3D74%26issue%3D13%26spage%3D1520%26epage%3D1526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel P.  Mould</span>, <span class="hlFld-ContribAuthor ">Ulf  Bremberg</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>, <span class="hlFld-ContribAuthor ">Matthis  Geitmann</span>, <span class="hlFld-ContribAuthor ">Alison E.  McGonagle</span>, <span class="hlFld-ContribAuthor ">Tim C.P.  Somervaille</span>, <span class="hlFld-ContribAuthor ">Gary J.  Spencer</span>, <span class="hlFld-ContribAuthor ">Donald J.  Ogilvie</span>. </span><span class="cited-content_cbyCitation_article-title">Development and evaluation of 4-(pyrrolidin-3-yl)benzonitrile derivatives as inhibitors of lysine specific demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (20)
                                     , 4755-4759. <a href="https://doi.org/10.1016/j.bmcl.2017.08.052" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.08.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.08.052%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDevelopment%252Band%252Bevaluation%252Bof%252B4-%252528pyrrolidin-3-yl%252529benzonitrile%252Bderivatives%252Bas%252Binhibitors%252Bof%252Blysine%252Bspecific%252Bdemethylase%252B1%26aulast%3DMould%26aufirst%3DDaniel%2BP.%26date%3D2017%26volume%3D27%26issue%3D20%26spage%3D4755%26epage%3D4759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoli  Fu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1227-1242. <a href="https://doi.org/10.4155/fmc-2017-0068" title="DOI URL">https://doi.org/10.4155/fmc-2017-0068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Btoward%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DFu%26aufirst%3DXiaoli%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1227%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative KDM1A inhibitors described in the literature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Hit <b>1</b> emerged from HTS, subsequent optimization led to compound <b>2</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Thieno[3,2-<i>b</i>]pyrrole 5-carboxamides (Method A)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (1) NaH, DMF, rt, 30 min, (2) MeI, rt, 1 h; (b) LiOH, H<sub>2</sub>O, EtOH, reflux, 30 min; (c) (1) SOCl<sub>2</sub>, DMF cat., THF, reflux, 2 h, (2) pyridine, rt, 90 min; (d) 4 N HCl in 1.4-dioxane, 1,4-dioxane, rt, 8–24 h; (e) (i) CH<sub>3</sub>CHO or PhCHO, 1,2-DCE, rt, 2 h, then NaBH(OAc)<sub>3</sub> overnight or (ii) CH<sub>3</sub>COCl, DMAP, THF, rt, 2 h.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Anilino Intermediates<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 4.5 h; (b) <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate or 1-methylpiperidin-4-ol, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, overnight; (c) (1) 4 N HCl in 1,4-dioxane, 1,4-dioxane, rt, 2 h; or TFA, DCM, overnight; (2) HCHO, NaBH<sub>3</sub>CN, MeOH, rt, 1–2 h; (d) PtO<sub>2</sub>, H<sub>2</sub> (H–CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH. Final compounds have been numbered with numeric terms. Intermediates have been numbered with alpha numeric terms, referring to the first final compound generated by the intermediate itself, and a letter, referring to the functional group present in the molecule: −NO<sub>2</sub>═, <b>d</b> or <b>j</b>; −NH<sub>2</sub>═, <b>e</b>; −final derivatives BOC protected═, <b>f</b>; (−NO<sub>2</sub>, −Br)═ <b>g</b>; (−OH, −OH)═ <b>h</b>; (−NO<sub>2</sub>, −OH)═ <b>i</b>; (−NH<sub>2</sub>, −OH)═ <b>k</b>; −PhOH═, <b>l</b>.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0012.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides (Method B)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH; (b) (1) <b>3c</b>, SOCl<sub>2</sub>, DMF cat., THF, 2 h, reflux, (2) then <b>33k</b>, pyridine, rt, 10 min; (c) R<sub>4</sub>OH, PPh<sub>3</sub>, DIAD, THF, 0 °C to rt, overnight; (d) 4 N HCl in 1,4-dioxane 1,4-dioxane/MeOH, 3 h, rt. R<sub>4</sub>═R<sub>5</sub>-BOC</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides <i>o</i>,<i>m</i>-Disubstituted<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1 M NaOH, THF, water, 50 °C, 1 h, quantitative; (b) (1) NaH, DMA 0 °C to rt, 30 min, (2) CH<sub>3</sub>I or CH<sub>3</sub>CH<sub>2</sub>I, 0 °C, then rt overnight; (b) LiBH<sub>4</sub>, THF, 0 °C, then rt, 3 h or 60 °C, 30 min; (d) CBr<sub>4</sub>, PPh<sub>3</sub>, CH<sub>3</sub>CN, 2 h, rt; (e) hydroquinone, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 2 h; (f) R<sub>4</sub>OH, PPh<sub>3</sub>, DIAD, THF, 0 °C, then rt, 12 h; (g) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 70 °C, atm pressure), EtOH; (h) (1) <b>3c</b> or <b>49c</b>, SOCl<sub>2</sub>, DMF cat., THF, reflux 3 h, (2) <b>45e</b>–<b>50e</b> pyridine, rt, 1 h; (i) 4 M HCl in 1.4-dioxane, 1,4-dioxane, rt, 24 h. R<sub>4</sub>═R<sub>5</sub>-BOC.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0014.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides <i>o</i>,<i>m</i>-Disubstituted<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) isopropyl iodide, DMA, rt, 48 h; (b) TBDSCl, imidazole, DCM, 0 °C, 15 min; (c) LiBH<sub>4</sub>, THF, 60 °C, 30 min; (d) mesyl chloride, TEA, DCM, −15 to 0 °C, 20 min; (e) hydroquinone, K<sub>2</sub>CO<sub>3</sub>, acetone, reflux, 45 min; (f) <i>tert</i>-butyl 4-hydroxypiperidine-1-carboxylate, PPh<sub>3</sub>, DIAD, THF, 0 °C then rt, overnight; (g) PtO<sub>2</sub>, H<sub>2</sub> (H-CUBE, 0.5 mL/min, 0.025 M, 75 °C, atm pressure), EtOH; (h) (1) <b>3c</b>, SOCl<sub>2</sub>, DMF cat., THF, reflux, 3 h, (2) <b>52e</b>, pyridine, rt 1 h; (i) TBAF, THF, 45 min, rt; (j) mesyl chloride, TEA, DCM, −15 °C, 30 min; (k) morpholine, TEA, DCM, rt, 12 h, or NaH, ethanol or 2-propanol, 0 °C to rt, 1 h; (l) 4 N HCl in 1,4-dioxane, 0 °C or rt, 1h.</p></p></figure><figure data-id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking poses for compound <b>2</b>. In terms of energies, the interaction with Asp375 (green) is favored, while the U-shaped conformation (pink) points toward an interesting cluster of negatively charged residues (Asp555, Asp556).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SAR investigation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray representations of <b>15</b> and <b>17</b> shows as the ortho <b>15</b> (A) accommodates the benzamide moiety in a way to favor aromatic–aromatic interactions and directs the basic terminal residue toward one of the two negatively charged regions (Asp555). Conversely, in the para derivative <b>17</b> (B), the benzamide moiety is forced out of the plane, possibly explaining the lower potency, while the terminal basic moiety located in proximity to Glu379 (2.9 A) could contribute to the residual micromolar potency.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Lineweaver–Burk plot for <b>15</b>. (B) X-ray structure of <b>15</b> superimposed to <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>. The tail of compound <b>15</b> occupies part of the region normally reserved to the substrate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0007.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. X-ray of <b>34</b>: The compound showed the same orientation and interaction pattern with protein residues as the parent <b>15</b>. The length of the chain and the protein residue flexibility allow interactions of nitrogen atoms with Asp555, where (3<i>R</i>)-pyrrolidine group in <b>33</b> shows the best activity among the series.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0008.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Superimposition of X-ray structures obtained with <b>1</b> (green) and <b>34</b> (cyan), showing the different regions of interaction for <i>ortho</i> and <i>meta</i> substituents.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/medium/jm-2016-01019n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0009.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray of <b>49</b> (cyano) and <b>50</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01019/20170303/images/large/jm-2016-01019n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01019&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i182">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 45 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Hoffmann, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roatsch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">The Role of Histone Demethylases in Cancer Therapy</span> <span class="citation_source-journal">Mol. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1016/j.molonc.2012.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.molonc.2012.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22902149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12gt7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=683-703&author=I.+Hoffmannauthor=M.+Roatschauthor=M.+L.+Schmittauthor=L.+Carlinoauthor=M.+Pippelauthor=W.+Sipplauthor=M.+Jung&title=The+Role+of+Histone+Demethylases+in+Cancer+Therapy&doi=10.1016%2Fj.molonc.2012.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The role of histone demethylases in cancer therapy</span></div><div class="casAuthors">Hoffmann, Inga; Roatsch, Martin; Schmitt, Martin L.; Carlino, Luca; Pippel, Martin; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">683-703</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Reversible histone methylation has emerged in the last few years as an important mechanism of epigenetic regulation.  Histone methyltransferases and demethylases have been identified as contributing factors in the development of several diseases, esp. cancer.  Therefore, they have been postulated to be new drug targets with high therapeutic potential.  Here, we review histone demethylases with a special focus on their potential role in oncol. drug discovery.  We present an overview over the different classes of enzymes, their biochem., selected data on their role in physiol. and already available inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHdjKp6f5Og7Vg90H21EOLACvtfcHk0lhfYZxvMPTW0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12gt7zO&md5=9830a57c650e94b2b9a38ea71b0d1a4b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2012.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2012.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DI.%26aulast%3DRoatsch%26aufirst%3DM.%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520Role%2520of%2520Histone%2520Demethylases%2520in%2520Cancer%2520Therapy%26jtitle%3DMol.%2520Oncol.%26date%3D2012%26volume%3D6%26spage%3D683%26epage%3D703%26doi%3D10.1016%2Fj.molonc.2012.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0li6p1gO2bnJ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Profumo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A Novel Mammalian Flavin-Dependent Histone Demethylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">17775</span><span class="NLM_x">–</span> <span class="NLM_lpage">17782</span><span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+Novel+Mammalian+Flavin-Dependent+Histone+Demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0li6p1gO2bnJ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Mammalian%2520Flavin-Dependent%2520Histone%2520Demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">LSD1 Inhibition: A Therapeutic Strategy in Cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+P.+Somervaille&title=LSD1+Inhibition%3A+A+Therapeutic+Strategy+in+Cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0li6p1gO2bnJ3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DLSD1%2520Inhibition%253A%2520A%2520Therapeutic%2520Strategy%2520in%2520Cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 Demethylates Repressive Histone Marks to Promote Androgen-Receptor-Dependent Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+Demethylates+Repressive+Histone+Marks+to+Promote+Androgen-Receptor-Dependent+Transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0lgP5-JyzDKKaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520Demethylates%2520Repressive%2520Histone%2520Marks%2520to%2520Promote%2520Androgen-Receptor-Dependent%2520Transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) Is Highly Expressed in ER-Negative Breast Cancers and a Biomarker Predicting Aggressive Biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+%28LSD1%29+Is+Highly+Expressed+in+ER-Negative+Breast+Cancers+and+a+Biomarker+Predicting+Aggressive+Biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0lgP5-JyzDKKaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520Is%2520Highly%2520Expressed%2520in%2520ER-Negative%2520Breast%2520Cancers%2520and%2520a%2520Biomarker%2520Predicting%2520Aggressive%2520Biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lgP5-JyzDKKaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+Lysine+4+Demethylation+Is+a+Target+of+Nonselective+Antidepressive+Medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0ljztOJtU8kcYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520Lysine%25204%2520Demethylation%2520Is%2520a%2520Target%2520of%2520Nonselective%2520Antidepressive%2520Medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Benelkebir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgkinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duriez, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bulleid, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Enantioselective Synthesis of Tranylcypromine Analogues as Lysine Demethylase (LSD1) Inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3709</span><span class="NLM_x">–</span> <span class="NLM_lpage">3716</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.bmc.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=21382717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3709-3716&author=H.+Benelkebirauthor=C.+Hodgkinsonauthor=P.+J.+Duriezauthor=A.+L.+Haydenauthor=R.+A.+Bulleidauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Ganesan&title=Enantioselective+Synthesis+of+Tranylcypromine+Analogues+as+Lysine+Demethylase+%28LSD1%29+Inhibitors&doi=10.1016%2Fj.bmc.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective synthesis of tranylcypromine analogues as lysine demethylase (LSD1) inhibitors</span></div><div class="casAuthors">Benelkebir, Hanae; Hodgkinson, Christopher; Duriez, Patrick J.; Hayden, Annette L.; Bulleid, Rosemary A.; Crabb, Simon J.; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3709-3716</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Asym. cyclopropanation of styrenes by tert-Bu diazoacetate followed by ester hydrolysis and Curtius rearrangement gave a series of tranylcypromine analogs as single enantiomers.  The o-, m-, and p-bromo analogs were all more active than tranylcypromine in a LSD1 enzyme assay.  The m- and p-bromo analogs were micromolar growth inhibitors of the LNCaP prostate cancer cell line as were the corresponding biphenyl analogs prepd. from the bromide by Suzuki crosscoupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuHGveNGE6AbVg90H21EOLACvtfcHk0ljztOJtU8kcYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVyju7s%253D&md5=a8517cf8753c76b1403b6354977fc2a2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DBenelkebir%26aufirst%3DH.%26aulast%3DHodgkinson%26aufirst%3DC.%26aulast%3DDuriez%26aufirst%3DP.%2BJ.%26aulast%3DHayden%26aufirst%3DA.%2BL.%26aulast%3DBulleid%26aufirst%3DR.%2BA.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEnantioselective%2520Synthesis%2520of%2520Tranylcypromine%2520Analogues%2520as%2520Lysine%2520Demethylase%2520%2528LSD1%2529%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3709%26epage%3D3716%26doi%3D10.1016%2Fj.bmc.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile Synthesis of Substituted Trans-2-Arylcyclopropylamine Inhibitors of the Human Histone Demethylase LSD1 and Monoamine Oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+Synthesis+of+Substituted+Trans-2-Arylcyclopropylamine+Inhibitors+of+the+Human+Histone+Demethylase+LSD1+and+Monoamine+Oxidases+A+and+B&doi=10.1016%2Fj.bmcl.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0ljztOJtU8kcYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520Synthesis%2520of%2520Substituted%2520Trans-2-Arylcyclopropylamine%2520Inhibitors%2520of%2520the%2520Human%2520Histone%2520Demethylase%2520LSD1%2520and%2520Monoamine%2520Oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051%26doi%3D10.1016%2Fj.bmcl.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Hojfeldt, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Histone Lysine Demethylases as Targets for Anticancer Therapy</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">930</span><span class="refDoi"> DOI: 10.1038/nrd4154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1038%2Fnrd4154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=24232376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=917-930&author=J.+W.+Hojfeldtauthor=K.+Aggerauthor=K.+Helin&title=Histone+Lysine+Demethylases+as+Targets+for+Anticancer+Therapy&doi=10.1038%2Fnrd4154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylases as targets for anticancer therapy</span></div><div class="casAuthors">Hojfeldt, Jonas W.; Agger, Karl; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">917-930</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  It has recently been demonstrated that the genes controlling the epigenetic programs that are required for maintaining chromatin structure and cell identity include genes that drive human cancer.  This observation has led to an increased awareness of chromatin-assocd. proteins as potentially interesting drug targets.  The successful introduction of DNA methylation and histone deacetylase (HDAC) inhibitors for the treatment of specific subtypes of cancer has paved the way for the use of epigenetic therapy.  Here, we highlight key biol. findings demonstrating the roles of members of the histone lysine demethylase class of enzymes in the development of cancers, discuss the potential and challenges of therapeutically targeting them, and highlight emerging small-mol. inhibitors of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoThyggRtM7qrVg90H21EOLACvtfcHk0ljFEBKwvgMb2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslygtrrP&md5=b120d43a0353078b016c8dc8f7cac644</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd4154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4154%26sid%3Dliteratum%253Aachs%26aulast%3DHojfeldt%26aufirst%3DJ.%2BW.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DHistone%2520Lysine%2520Demethylases%2520as%2520Targets%2520for%2520Anticancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D917%26epage%3D930%26doi%3D10.1038%2Fnrd4154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+Overview+of+Phenylcyclopropylamine+Derivatives%3A+Biochemical+and+Biological+Significance+and+Recent+Developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0ljFEBKwvgMb2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520Overview%2520of%2520Phenylcyclopropylamine%2520Derivatives%253A%2520Biochemical%2520and%2520Biological%2520Significance%2520and%2520Recent%2520Developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally Designed Trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+Designed+Trans-2-Phenylcyclopropylamine+Derivatives+Potently+Inhibit+Histone+Demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0ljFEBKwvgMb2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520Designed%2520Trans-2-Phenylcyclopropylamine%2520Derivatives%2520Potently%2520Inhibit%2520Histone%2520Demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kakizawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuhara, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1-Selective Inactivators: Protein-Targeted Drug Delivery Mechanism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">8620</span><span class="NLM_x">–</span> <span class="NLM_lpage">8624</span><span class="refDoi"> DOI: 10.1002/anie.201303999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fanie.201303999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=8620-8624&author=D.+Ogasawaraauthor=Y.+Itohauthor=H.+Tsumotoauthor=T.+Kakizawaauthor=K.+Minoauthor=K.+Fukuharaauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyataauthor=T.+Suzuki&title=Lysine-Specific+Demethylase+1-Selective+Inactivators%3A+Protein-Targeted+Drug+Delivery+Mechanism&doi=10.1002%2Fanie.201303999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1-selective inactivators: Protein-targeted drug delivery mechanism</span></div><div class="casAuthors">Ogasawara, Daisuke; Itoh, Yukihiro; Tsumoto, Hiroki; Kakizawa, Taeko; Mino, Koshiki; Fukuhara, Kiyoshi; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">8620-8624</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of novel LSD1 (LSD1 = lysine-specific demethylase) inactivators based on new concept of LSD1-targeted PCPA (PCPA = trans-2-phenylcyclopropylamine) delivery have been designed.  This concept was tested with peptide 1a [1a = (ART-X-QTARKSTGGKAPRKQLA)] X = (I) and confirmed that 1a does indeed inhibit LSD1 selectively and efficiently by delivering PCPA directly to the LSD1 active site.  The enzyme is then inactivated by FAD-PCPA adduct formation, as in the case of PCPA itself.  This strategy could be applied to the design of nonpeptide, small-mol. LSD1 inactivators.  Mechanistic studies and biol. evaluations revealed that these small-mol. compds. are potent LSD1-selective inactivators (comparable to peptide 1a) in enzyme assays, and they also exhibit potent cell growth inhibitory activities against cancer cell lines.  This is the first time that a drug-carrier-type system has been employed to target a specific mol.  These novel LSD1-selective inactivators are considered to be candidates for anticancer agents, as well as bioprobes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqQOLhLwmv9rVg90H21EOLACvtfcHk0ljFEBKwvgMb2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsbnF&md5=71bd05a50aeb73431444f2588fba2dac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303999%26sid%3Dliteratum%253Aachs%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DKakizawa%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DLysine-Specific%2520Demethylase%25201-Selective%2520Inactivators%253A%2520Protein-Targeted%2520Drug%2520Delivery%2520Mechanism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D8620%26epage%3D8624%26doi%3D10.1002%2Fanie.201303999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Pieroni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annunziato, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzali, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiró, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beato, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, G.</span><span> </span><span class="NLM_article-title">Further Insights into the SAR of α-Substituted Cyclopropylamine Derivatives as Inhibitors of Histone Demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">386</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2014.12.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=377-386&author=M.+Pieroniauthor=G.+Annunziatoauthor=E.+Azzaliauthor=P.+Dessantiauthor=C.+Mercurioauthor=G.+Meroniauthor=P.+Trifir%C3%B3author=P.+Vianelloauthor=M.+Villaauthor=C.+Beatoauthor=M.+Varasiauthor=G.+Costantino&title=Further+Insights+into+the+SAR+of+%CE%B1-Substituted+Cyclopropylamine+Derivatives+as+Inhibitors+of+Histone+Demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.12.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DAnnunziato%26aufirst%3DG.%26aulast%3DAzzali%26aufirst%3DE.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBeato%26aufirst%3DC.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DCostantino%26aufirst%3DG.%26atitle%3DFurther%2520Insights%2520into%2520the%2520SAR%2520of%2520%25CE%25B1-Substituted%2520Cyclopropylamine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D92%26spage%3D377%26epage%3D386%26doi%3D10.1016%2Fj.ejmech.2014.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Synthesis, Biological Activity and Mechanistic Insights of 1-Substituted Cyclopropylamine Derivatives: A Novel Class of Irreversible Inhibitors of Histone Demethylase KDM1A</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">363</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2014.08.068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2014.08.068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=25173853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2014&pages=352-363&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=G.+Ciossaniauthor=P.+Dessantiauthor=A.+Maiauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=F.+Thalerauthor=M.+Tortoriciauthor=P.+Trifiroauthor=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Synthesis%2C+Biological+Activity+and+Mechanistic+Insights+of+1-Substituted+Cyclopropylamine+Derivatives%3A+A+Novel+Class+of+Irreversible+Inhibitors+of+Histone+Demethylase+KDM1A&doi=10.1016%2Fj.ejmech.2014.08.068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and mechanistic insights of 1-substituted cyclopropylamine derivatives: A novel class of irreversible inhibitors of histone demethylase KDM1A</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Ciossani, Giuseppe; Dessanti, Paola; Mai, Antonello; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Thaler, Florian; Tortorici, Marcello; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">352-363</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone demethylase KDM1A [i.e., gene LSD1, histone H3 lysine-4 demethylase LSD1] has become an attractive therapeutic target for the treatment of cancer as well as other disorders such as viral infections.  The authors report on the synthesis of compds. derived from the expansion of tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  These compds., which are substituted on the cyclopropyl core moiety, were evaluated for their ability to inhibit KDM1A in vitro as well as to function in cells by modulating the expression of Gfi-1b, a well recognized KDM1A target gene.  The mols. were all found to covalently inhibit KDM1A and to become increasingly selective against human monoamine oxidase MAO A and MAO B through the introduction of bulkier substituents on the cyclopropylamine ring.  Structural and biochem. anal. of selected trans-isomers showed that the two stereoisomers are endowed with similar inhibitory activities against KDM1A, but form different covalent adducts with FAD co-enzyme [i.e., riboflavin 5'-(trihydrogen diphosphate) P'→5'-ester with adenosine, FAD].  The synthesis of the target compds. was achieved using (2S)-2-(phenyl)oxirane, (2R)-2-(phenyl)oxirane, racemic (phenyl)oxirane as starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEJ-thHP_b6rVg90H21EOLACvtfcHk0ljJWJgEGveX6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGnsrvP&md5=6285108e113c25939e3aa0b1ee5390c6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.08.068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.08.068%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DSynthesis%252C%2520Biological%2520Activity%2520and%2520Mechanistic%2520Insights%2520of%25201-Substituted%2520Cyclopropylamine%2520Derivatives%253A%2520A%2520Novel%2520Class%2520of%2520Irreversible%2520Inhibitors%2520of%2520Histone%2520Demethylase%2520KDM1A%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D86%26spage%3D352%26epage%3D363%26doi%3D10.1016%2Fj.ejmech.2014.08.068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saponara, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monaldi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruoppolo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tilset, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Pure Enantiomers of Benzoylamino-Tranylcypromine: LSD1 Inhibition, Gene Modulation in Human Leukemia Cells and Effects on Clonogenic Potential of Murine Promyelocytic Blasts</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">163</span><span class="NLM_x">–</span> <span class="NLM_lpage">174</span><span class="refDoi"> DOI: 10.1016/j.ejmech.2015.02.060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ejmech.2015.02.060" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2015&pages=163-174&author=S.+Valenteauthor=V.+Rodriguezauthor=C.+Mercurioauthor=P.+Vianelloauthor=B.+Saponaraauthor=R.+Cirilliauthor=G.+Ciossaniauthor=D.+Labellaauthor=B.+Marroccoauthor=D.+Monaldiauthor=G.+Ruoppoloauthor=M.+Tilsetauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=A.+Matteviauthor=M.+Varasiauthor=A.+Mai&title=Pure+Enantiomers+of+Benzoylamino-Tranylcypromine%3A+LSD1+Inhibition%2C+Gene+Modulation+in+Human+Leukemia+Cells+and+Effects+on+Clonogenic+Potential+of+Murine+Promyelocytic+Blasts&doi=10.1016%2Fj.ejmech.2015.02.060"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.02.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.02.060%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRodriguez%26aufirst%3DV.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSaponara%26aufirst%3DB.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMonaldi%26aufirst%3DD.%26aulast%3DRuoppolo%26aufirst%3DG.%26aulast%3DTilset%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DPure%2520Enantiomers%2520of%2520Benzoylamino-Tranylcypromine%253A%2520LSD1%2520Inhibition%252C%2520Gene%2520Modulation%2520in%2520Human%2520Leukemia%2520Cells%2520and%2520Effects%2520on%2520Clonogenic%2520Potential%2520of%2520Murine%2520Promyelocytic%2520Blasts%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D94%26spage%3D163%26epage%3D174%26doi%3D10.1016%2Fj.ejmech.2015.02.060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span> </span><span class="NLM_article-title">Phase I Dose Escalation Study for GSK2879552 in Subjects with Acute Myeloid Leukemia (AML)</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2014</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&amp;rank=2" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02177812?term=GSK2879552&rank=2</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Phase+I+Dose+Escalation+Study+for+GSK2879552+in+Subjects+with+Acute+Myeloid+Leukemia+%28AML%29.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2014%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02177812%3Fterm%3DGSK2879552%26rank%3D2+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DPhase%2520I%2520Dose%2520Escalation%2520Study%2520for%2520GSK2879552%2520in%2520Subjects%2520with%2520Acute%2520Myeloid%2520Leukemia%2520%2528AML%2529%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span> </span><span class="NLM_article-title">Investigation of GSK2879552 in Subjects with Relapsed/Refractory Small Cell Lung Carcinoma</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+with+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2013%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DInvestigation%2520of%2520GSK2879552%2520in%2520Subjects%2520with%2520Relapsed%252FRefractory%2520Small%2520Cell%2520Lung%2520Carcinoma%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Maes, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carceller, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buesa, C.</span><span> </span><span class="NLM_article-title">Advances in the Development of Histone Lysine Demethylase Inhibitors</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.coph.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.coph.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=26057211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=52-60&author=T.+Maesauthor=E.+Carcellerauthor=J.+Salasauthor=A.+Ortegaauthor=C.+Buesa&title=Advances+in+the+Development+of+Histone+Lysine+Demethylase+Inhibitors&doi=10.1016%2Fj.coph.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of histone lysine demethylase inhibitors</span></div><div class="casAuthors">Maes, Tamara; Carceller, Elena; Salas, Jordi; Ortega, Alberto; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52-60</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The covalent modification of histones is closely assocd. with regulation of gene transcription.  Chromatin modifications have been suggested to represent an epigenetic code that is dynamically 'written' and 'erased' by specialized proteins, and 'read', or interpreted, by proteins that translate the code into gene expression changes.  Initially thought to be an irreversible process, histone methylation is now known to be reversed by demethylases, FAD dependent amineoxidases and by iron(II)-alpha-ketoglutarate dependent deoxygenases of the Jumonji family.  Altered histone demethylase activities have been assocd. with human disease, including cancer.  The first wave of novel investigational drugs directed against KDM1A has recently entered the clinic, and the first specific inhibitor targeting a Jumonji KDM is advancing in preclin. regulatory studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHCr97IlCYFrVg90H21EOLACvtfcHk0ljHRMnKJZVtWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Slt7o%253D&md5=02e8a5f8d426743be1eb209c329bd3ed</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DCarceller%26aufirst%3DE.%26aulast%3DSalas%26aufirst%3DJ.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DAdvances%2520in%2520the%2520Development%2520of%2520Histone%2520Lysine%2520Demethylase%2520Inhibitors%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60%26doi%3D10.1016%2Fj.coph.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span> </span><span class="NLM_article-title">A phase I study of Human Pharmacokinetics and Safety of ORY-1001, an LSD1 inhibitor, in relapsed or refractory acute leukaemia (AL)</span>. EU Clinical Trials Register; <span class="NLM_publisher-name">European Medicines Agency</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES" class="extLink">https://www.clinicaltrialsregister.eu/ctr-search/trial/2013-002447-29/ES</a> (Accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=A+phase+I+study+of+Human+Pharmacokinetics+and+Safety+of+ORY-1001%2C+an+LSD1+inhibitor%2C+in+relapsed+or+refractory+acute+leukaemia+%28AL%29.+EU+Clinical+Trials+Register%3B+European+Medicines+Agency%3A+London%2C+2013%3B+https%3A%2F%2Fwww.clinicaltrialsregister.eu%2Fctr-search%2Ftrial%2F2013-002447-29%2FES+%28Accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DA%2520phase%2520I%2520study%2520of%2520Human%2520Pharmacokinetics%2520and%2520Safety%2520of%2520ORY-1001%252C%2520an%2520LSD1%2520inhibitor%252C%2520in%2520relapsed%2520or%2520refractory%2520acute%2520leukaemia%2520%2528AL%2529%26jtitle%3DEU%2520Clinical%2520Trials%2520Register%26pub%3DEuropean%2520Medicines%2520Agency%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span> </span><span class="NLM_article-title">An Open-Label, Dose-Escalation/Dose-Expansion Safety Study of INCB059872 in Subjects With Advanced Malignancies</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/study/NCT02712905" class="extLink">https://clinicaltrials.gov/ct2/show/study/NCT02712905</a> (Accessed October 7, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=An+Open-Label%2C+Dose-Escalation%2FDose-Expansion+Safety+Study+of+INCB059872+in+Subjects+With+Advanced+Malignancies.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2016%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2Fstudy%2FNCT02712905+%28Accessed+October+7%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DAn%2520Open-Label%252C%2520Dose-Escalation%252FDose-Expansion%2520Safety%2520Study%2520of%2520INCB059872%2520in%2520Subjects%2520With%2520Advanced%2520Malignancies%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0ljHRMnKJZVtWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Wu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">3-(Piperidin-4-Ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">253</span><span class="NLM_x">–</span> <span class="NLM_lpage">263</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+Weiauthor=Z.+Fengauthor=L.+Dengauthor=Y.+Song&title=3-%28Piperidin-4-Ylmethoxy%29pyridine+Containing+Compounds+Are+Potent+Inhibitors+of+Lysine+Specific+Demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0ljHRMnKJZVtWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26atitle%3D3-%2528Piperidin-4-Ylmethoxy%2529pyridine%2520Containing%2520Compounds%2520Are%2520Potent%2520Inhibitors%2520of%2520Lysine%2520Specific%2520Demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-Benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-Throughput+Virtual+Screening+Identifies+Novel+N%E2%80%B2-%281-Phenylethylidene%29-Benzohydrazides+as+Potent%2C+Specific%2C+and+Reversible+LSD1+Inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0ljydfaqRe8GHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-Throughput%2520Virtual%2520Screening%2520Identifies%2520Novel%2520N%25E2%2580%25B2-%25281-Phenylethylidene%2529-Benzohydrazides%2520as%2520Potent%252C%2520Specific%252C%2520and%2520Reversible%2520LSD1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Sankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulvihill, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beckerle, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lessnick, S. L.</span><span> </span><span class="NLM_article-title">Reversible LSD1 Inhibition Interferes with Global EWS/ETS Transcriptional Activity and Impedes Ewing Sarcoma Tumor Growth</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">4584</span><span class="NLM_x">–</span> <span class="NLM_lpage">4597</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-0072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1158%2F1078-0432.CCR-14-0072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=24963049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyku7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=4584-4597&author=S.+Sankarauthor=E.+R.+Theisenauthor=J.+Bearssauthor=T.+Mulvihillauthor=L.+M.+Hoffmanauthor=V.+Sornaauthor=M.+C.+Beckerleauthor=S.+Sharmaauthor=S.+L.+Lessnick&title=Reversible+LSD1+Inhibition+Interferes+with+Global+EWS%2FETS+Transcriptional+Activity+and+Impedes+Ewing+Sarcoma+Tumor+Growth&doi=10.1158%2F1078-0432.CCR-14-0072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible LSD1 Inhibition Interferes with Global EWS/ETS Transcriptional Activity and Impedes Ewing Sarcoma Tumor Growth</span></div><div class="casAuthors">Sankar, Savita; Theisen, Emily R.; Bearss, Jared; Mulvihill, Timothy; Hoffman, Laura M.; Sorna, Venkataswamy; Beckerle, Mary C.; Sharma, Sunil; Lessnick, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4584-4597</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Ewing sarcoma is a pediatric bone tumor that absolutely relies on the transcriptional activity of the EWS/ETS family of fusion oncoproteins.  While the most common fusion, EWS/FLI, utilizes lysine-specific demethylase 1 (LSD1) to repress crit. tumor suppressors, small-mol. blockade of LSD1 has not yet been thoroughly explored as a therapeutic approach for Ewing sarcoma.  We therefore evaluated the translational potential of potent and specific LSD1 inhibition with HCI2509 on the transcriptional program of both EWS/FLI and EWS/ERG as well as the downstream oncogenic phenotypes driven by EWS/ETS fusions in both in vitro and in vivo models of Ewing sarcoma.  Exptl. Design: RNA-seq was used to compare the transcriptional profiles of EWS/FLI, EWS/ERG, and treatment with HCI2509 in both EWS/FLI- and EWS/ERG-contg. cell lines.  We then evaluated morphol. phenotypes of treated cells with immunofluorescence.  The induction of apoptosis was evaluated using caspase-3/7 activation and TUNEL staining.  Colony forming assays were used to test oncogenic transformation and xenograft studies with patient-derived cell lines were used to evaluate the effects of HCI2509 on tumorigenesis.  Results: HCI2509 caused a dramatic reversal of both the up- and downregulated transcriptional profiles of EWS/FLI and EWS/ERG accompanied by the induction of apoptosis and disruption of morphol. and oncogenic phenotypes modulated by EWS/FLI.  Importantly, HCI2509 displayed single-agent efficacy in multiple xenograft models.  Conclusions: These data support epigenetic modulation with HCI2509 as a therapeutic strategy for Ewing sarcoma, and highlight a crit. dual role for LSD1 in the oncogenic transcriptional activity of EWS/ETS proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-MyK6B0OhLVg90H21EOLACvtfcHk0ljydfaqRe8GHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyku7zP&md5=1ee7f6fff6279302104d0482759f5d26</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0072%26sid%3Dliteratum%253Aachs%26aulast%3DSankar%26aufirst%3DS.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DBearss%26aufirst%3DJ.%26aulast%3DMulvihill%26aufirst%3DT.%26aulast%3DHoffman%26aufirst%3DL.%2BM.%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DBeckerle%26aufirst%3DM.%2BC.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLessnick%26aufirst%3DS.%2BL.%26atitle%3DReversible%2520LSD1%2520Inhibition%2520Interferes%2520with%2520Global%2520EWS%252FETS%2520Transcriptional%2520Activity%2520and%2520Impedes%2520Ewing%2520Sarcoma%2520Tumor%2520Growth%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2014%26volume%3D20%26spage%3D4584%26epage%3D4597%26doi%3D10.1158%2F1078-0432.CCR-14-0072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajiwala, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janat-Amsbury, M.</span><span> </span><span class="NLM_article-title">Reversible Inhibition of Lysine Specific Demethylase 1 Is a Novel Anti-Tumor Strategy for Poorly Differentiated Endometrial Carcinoma</span> <span class="citation_source-journal">BMC Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">752</span><span class="refDoi"> DOI: 10.1186/1471-2407-14-752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1186%2F1471-2407-14-752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=752&author=E.+R.+Theisenauthor=S.+Gajiwalaauthor=J.+Bearssauthor=V.+Sornaauthor=S.+Sharmaauthor=M.+Janat-Amsbury&title=Reversible+Inhibition+of+Lysine+Specific+Demethylase+1+Is+a+Novel+Anti-Tumor+Strategy+for+Poorly+Differentiated+Endometrial+Carcinoma&doi=10.1186%2F1471-2407-14-752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-752%26sid%3Dliteratum%253Aachs%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DGajiwala%26aufirst%3DS.%26aulast%3DBearss%26aufirst%3DJ.%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DJanat-Amsbury%26aufirst%3DM.%26atitle%3DReversible%2520Inhibition%2520of%2520Lysine%2520Specific%2520Demethylase%25201%2520Is%2520a%2520Novel%2520Anti-Tumor%2520Strategy%2520for%2520Poorly%2520Differentiated%2520Endometrial%2520Carcinoma%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D752%26doi%3D10.1186%2F1471-2407-14-752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sartori, L.; Mercurio, C.; Amigoni, F.; Cappa, A.; Fagá, G.; Fattori, R.; Legnaghi, E.; Ciossani, G.; Mattevi, A.; Meroni, G.; Moretti, L.; Cecatiello, V.; Pasqualato, S.; Romussi, A.; Thaler, F.; Trifiró, P.; Villa, M.; Vultaggio, S.; Botrugno, O. A.; Dessanti, P.; Minucci, S.; Zagarrí, E.; Carettoni, D.; Iuzzolino, L.; Varasi, M.; Vianello, P.</span><span> </span><span class="NLM_article-title">Thieno[3,2-B]pyrrole-5-Carboxamides as Novel Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High Throughput Screening and Preliminary Exploration</span>.  <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01018</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=L.+Sartori&author=C.+Mercurio&author=F.+Amigoni&author=A.+Cappa&author=G.+Fag%C3%A1&author=R.+Fattori&author=E.+Legnaghi&author=G.+Ciossani&author=A.+Mattevi&author=G.+Meroni&author=L.+Moretti&author=V.+Cecatiello&author=S.+Pasqualato&author=A.+Romussi&author=F.+Thaler&author=P.+Trifir%C3%B3&author=M.+Villa&author=S.+Vultaggio&author=O.+A.+Botrugno&author=P.+Dessanti&author=S.+Minucci&author=E.+Zagarr%C3%AD&author=D.+Carettoni&author=L.+Iuzzolino&author=M.+Varasi&author=P.+Vianello&title=Thieno%5B3%2C2-B%5Dpyrrole-5-Carboxamides+as+Novel+Reversible+Inhibitors+of+Histone+Lysine+Demethylase+KDM1A%2FLSD1.+Part+1%3A+High+Throughput+Screening+and+Preliminary+Exploration&doi=10.1021%2Facs.jmedchem.6b01018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DThieno%255B3%252C2-B%255Dpyrrole-5-Carboxamides%2520as%2520Novel%2520Reversible%2520Inhibitors%2520of%2520Histone%2520Lysine%2520Demethylase%2520KDM1A%252FLSD1.%2520Part%25201%253A%2520High%2520Throughput%2520Screening%2520and%2520Preliminary%2520Exploration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26doi%3D10.1021%2Facs.jmedchem.6b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Ching, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merits, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, C. L. L.</span><span> </span><span class="NLM_article-title">Trisubstituted thieno[3,2-B]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9196</span><span class="NLM_x">–</span> <span class="NLM_lpage">9213</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9196-9213&author=K.+C.+Chingauthor=Y.+W.+Kamauthor=A.+Meritsauthor=L.+F.+P.+Ngauthor=C.+L.+L.+Chai&title=Trisubstituted+thieno%5B3%2C2-B%5Dpyrrole+5-Carboxamides+as+Potent+Inhibitors+of+Alphaviruses&doi=10.1021%2Facs.jmedchem.5b01047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Thieno[3,2-b]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span></div><div class="casAuthors">Ching, Kuan-Chieh; Kam, Yiu-Wing; Merits, Andres; Ng, Lisa F. P.; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9196-9213</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chikungunya virus (CHIKV) is a re-emerging vector-borne alphavirus and is transmitted to humans by Aedes mosquitoes.  Despite the re-emergence of CHIKV as an epidemic threat, there is no approved effective antiviral treatment currently available for CHIKV.  Herein, we report the synthesis and structure-activity relationship studies of a class of thieno[3,2-b]pyrroles and the discovery of a trisubstituted thieno[3,2-b]pyrrole 5-carboxamide 15c (I) that exhibits potent inhibitory activity against in vitro CHIKV infection.  Compd. 15c displayed low micromolar activity (EC50 value of ca. 2 μM) and limited cytotoxic liability (CC50 > 100 μM) therefore furnishing a selectivity index of greater than 32.  Notably, 15c not only controlled viral RNA prodn., but efficiently inhibited the expression of CHIKV nsP1, nsP3, capsid, and E2 proteins at a concn. as low as 2.5 μM.  More importantly, 15c also demonstrated broad spectrum antiviral activity against other clin. important alphaviruses such as O'nyong-nyong virus and Sindbis virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC7wEIPTmwQ7Vg90H21EOLACvtfcHk0lgDGbpwI0tBFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI&md5=dc938296fd5fd7628b7de37590b6918a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01047%26sid%3Dliteratum%253Aachs%26aulast%3DChing%26aufirst%3DK.%2BC.%26aulast%3DKam%26aufirst%3DY.%2BW.%26aulast%3DMerits%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DL.%2BF.%2BP.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DTrisubstituted%2520thieno%255B3%252C2-B%255Dpyrrole%25205-Carboxamides%2520as%2520Potent%2520Inhibitors%2520of%2520Alphaviruses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9196%26epage%3D9213%26doi%3D10.1021%2Facs.jmedchem.5b01047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Venable, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvorak, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grice, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jablonowski, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwok, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pio, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurmond, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovenberg, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlsson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carruthers, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, J. P.</span><span> </span><span class="NLM_article-title">Preparation and Biological Evaluation of Indole, Benzimidazole, and Thienopyrrole Piperazine Carboxamides: Potent Human Histamine H4 Antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">8289</span><span class="NLM_x">–</span> <span class="NLM_lpage">8298</span><span class="refDoi"> DOI: 10.1021/jm0502081</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0502081" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=8289-8298&author=J.+D.+Venableauthor=H.+Caiauthor=W.+Chaiauthor=C.+A.+Dvorakauthor=C.+A.+Griceauthor=J.+A.+Jablonowskiauthor=C.+R.+Shahauthor=A.+K.+Kwokauthor=K.+S.+Lyauthor=B.+Pioauthor=J.+Weiauthor=P.+J.+Desaiauthor=W.+Jiangauthor=S.+Nguyenauthor=P.+Lingauthor=S.+J.+Wilsonauthor=P.+J.+Dunfordauthor=R.+L.+Thurmondauthor=T.+W.+Lovenbergauthor=L.+Karlssonauthor=N.+I.+Carruthersauthor=J.+P.+Edwards&title=Preparation+and+Biological+Evaluation+of+Indole%2C+Benzimidazole%2C+and+Thienopyrrole+Piperazine+Carboxamides%3A+Potent+Human+Histamine+H4+Antagonists&doi=10.1021%2Fjm0502081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0502081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0502081%26sid%3Dliteratum%253Aachs%26aulast%3DVenable%26aufirst%3DJ.%2BD.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DChai%26aufirst%3DW.%26aulast%3DDvorak%26aufirst%3DC.%2BA.%26aulast%3DGrice%26aufirst%3DC.%2BA.%26aulast%3DJablonowski%26aufirst%3DJ.%2BA.%26aulast%3DShah%26aufirst%3DC.%2BR.%26aulast%3DKwok%26aufirst%3DA.%2BK.%26aulast%3DLy%26aufirst%3DK.%2BS.%26aulast%3DPio%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DDesai%26aufirst%3DP.%2BJ.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DLing%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DS.%2BJ.%26aulast%3DDunford%26aufirst%3DP.%2BJ.%26aulast%3DThurmond%26aufirst%3DR.%2BL.%26aulast%3DLovenberg%26aufirst%3DT.%2BW.%26aulast%3DKarlsson%26aufirst%3DL.%26aulast%3DCarruthers%26aufirst%3DN.%2BI.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26atitle%3DPreparation%2520and%2520Biological%2520Evaluation%2520of%2520Indole%252C%2520Benzimidazole%252C%2520and%2520Thienopyrrole%2520Piperazine%2520Carboxamides%253A%2520Potent%2520Human%2520Histamine%2520H4%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D8289%26epage%3D8298%26doi%3D10.1021%2Fjm0502081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Ilyin, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dmitrieva, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kustova, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manaev, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivachtchenko, A. V.</span><span> </span><span class="NLM_article-title">Synthesis of Heterocyclic Compounds Possessing the 4H-thieno[3,2-B]pyrrole Moiety</span> <span class="citation_source-journal">J. Comb. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1021/cc060091h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc060091h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWlsL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=96-106&author=A.+P.+Ilyinauthor=I.+G.+Dmitrievaauthor=V.+A.+Kustovaauthor=A.+V.+Manaevauthor=A.+V.+Ivachtchenko&title=Synthesis+of+Heterocyclic+Compounds+Possessing+the+4H-thieno%5B3%2C2-B%5Dpyrrole+Moiety&doi=10.1021%2Fcc060091h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Heterocyclic Compounds Possessing the 4H-Thieno[3,2-b]Pyrrole Moiety</span></div><div class="casAuthors">Ilyin, Alexey P.; Dmitrieva, Irina G.; Kustova, Veronika A.; Manaev, Alexandre V.; Ivachtchenko, Alexandre V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">96-106</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of novel heterocyclic combinatorial libraries contg. 4H-thieno[3,2-b]pyrrole, thieno[2',3':4,5]pyrrolo[1,2-d][1,2,4]triazine and thieno[2',3':4,5]pyrrolo[1,2-a]pyrazine heterocyclic moieties were obtained by parallel soln.-phase synthesis.  Key steps include different reactions of initial alkyl 4H-thieno[3,2-b]pyrrole-5-carboxylates, such as alkylation with alkylating agents; transformation of the carboxylate group into different reactive functionalities, followed by reactions with electrophilic species; intramol. cyclizations; and amide bond formation.  Simple manual techniques for parallel reactions were coupled with easy purifn. procedures to give high-purity final products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF9Lw2eRWTmLVg90H21EOLACvtfcHk0lgDGbpwI0tBFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWlsL%252FF&md5=2e66224e7fdb6d92537b551ed16ce82b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcc060091h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc060091h%26sid%3Dliteratum%253Aachs%26aulast%3DIlyin%26aufirst%3DA.%2BP.%26aulast%3DDmitrieva%26aufirst%3DI.%2BG.%26aulast%3DKustova%26aufirst%3DV.%2BA.%26aulast%3DManaev%26aufirst%3DA.%2BV.%26aulast%3DIvachtchenko%26aufirst%3DA.%2BV.%26atitle%3DSynthesis%2520of%2520Heterocyclic%2520Compounds%2520Possessing%2520the%25204H-thieno%255B3%252C2-B%255Dpyrrole%2520Moiety%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2007%26volume%3D9%26spage%3D96%26epage%3D106%26doi%3D10.1021%2Fcc060091h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Soucy, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Favreau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kayser, M. M.</span><span> </span><span class="NLM_article-title">The Regioselectivity of Metal Hydride Reductions of 3-Substituted Phthalic Anhydrides</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1021/jo00377a023</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00377a023" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADyaL2sXmslSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1987&pages=129-134&author=C.+Soucyauthor=D.+Favreauauthor=M.+M.+Kayser&title=The+Regioselectivity+of+Metal+Hydride+Reductions+of+3-Substituted+Phthalic+Anhydrides&doi=10.1021%2Fjo00377a023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The regioselectivity of metal hydride reductions of 3-substituted phthalic anhydrides</span></div><div class="casAuthors">Soucy, C.; Favreau, D.; Kayser, M. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-34</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The redn. of 3-methoxyphthalide by metal hydrides was reinvestigated.  Various effects controlling selectivity of redns. in 3-substituted phthalides were studied, and a qual. interpretation of the results was proposed.  Methods for obtaining enhanced yields of one or the other lactonic product were developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqU_DHjeYOELrVg90H21EOLACvtfcHk0lgDGbpwI0tBFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXmslSgsg%253D%253D&md5=3993790041c50d330cffaa9c6235bd21</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjo00377a023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00377a023%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DC.%26aulast%3DFavreau%26aufirst%3DD.%26aulast%3DKayser%26aufirst%3DM.%2BM.%26atitle%3DThe%2520Regioselectivity%2520of%2520Metal%2520Hydride%2520Reductions%2520of%25203-Substituted%2520Phthalic%2520Anhydrides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1987%26volume%3D52%26spage%3D129%26epage%3D134%26doi%3D10.1021%2Fjo00377a023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+Basis+of+LSD1-CoREST+Selectivity+in+Histone+H3+Recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lj8Iy26VVoSPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520LSD1-CoREST%2520Selectivity%2520in%2520Histone%2520H3%2520Recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+P.+Somervaille&title=The+Histone+Demethylase+KDM1A+Sustains+the+Oncogenic+Potential+of+MLL-AF9+Leukemia+Stem+Cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lj8Iy26VVoSPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DThe%2520Histone%2520Demethylase%2520KDM1A%2520Sustains%2520the%2520Oncogenic%2520Potential%2520of%2520MLL-AF9%2520Leukemia%2520Stem%2520Cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanigan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Loss of LSD1 (Lysine-Specific Demethylase 1) Suppresses Growth and Alters Gene Expression of Human Colon Cancer Cells in a p53- and DNMT1(DNA Methyltransferase 1)-Independent Manner</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">459</span><span class="NLM_x">–</span> <span class="NLM_lpage">468</span><span class="refDoi"> DOI: 10.1042/BJ20121360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1042%2FBJ20121360" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=459-468&author=L.+Jinauthor=C.+L.+Haniganauthor=Y.+Wuauthor=W.+Wangauthor=B.+H.+Parkauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Loss+of+LSD1+%28Lysine-Specific+Demethylase+1%29+Suppresses+Growth+and+Alters+Gene+Expression+of+Human+Colon+Cancer+Cells+in+a+p53-+and+DNMT1%28DNA+Methyltransferase+1%29-Independent+Manner&doi=10.1042%2FBJ20121360"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1042%2FBJ20121360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121360%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHanigan%26aufirst%3DC.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DLoss%2520of%2520LSD1%2520%2528Lysine-Specific%2520Demethylase%25201%2529%2520Suppresses%2520Growth%2520and%2520Alters%2520Gene%2520Expression%2520of%2520Human%2520Colon%2520Cancer%2520Cells%2520in%2520a%2520p53-%2520and%2520DNMT1%2528DNA%2520Methyltransferase%25201%2529-Independent%2520Manner%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D459%26epage%3D468%26doi%3D10.1042%2FBJ20121360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone Demethylase Lsd1 Represses Hematopoietic Stem and Progenitor Cell Signatures during Blood Cell Maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.-J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+Demethylase+Lsd1+Represses+Hematopoietic+Stem+and+Progenitor+Cell+Signatures+during+Blood+Cell+Maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0liYboo8zsMU3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520Demethylase%2520Lsd1%2520Represses%2520Hematopoietic%2520Stem%2520and%2520Progenitor%2520Cell%2520Signatures%2520during%2520Blood%2520Cell%2520Maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liYboo8zsMU3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Winter, G.</span><span> </span><span class="NLM_article-title">Xia2: An Expert System for Macromolecular Crystallography Data Reduction</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span><span class="refDoi"> DOI: 10.1107/S0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=Xia2%3A+An+Expert+System+for+Macromolecular+Crystallography+Data+Reduction&doi=10.1107%2FS0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0liYboo8zsMU3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DXia2%253A%2520An%2520Expert%2520System%2520for%2520Macromolecular%2520Crystallography%2520Data%2520Reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2FS0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0limiFSj4XptQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0limiFSj4XptQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0limiFSj4XptQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0limiFSj4XptQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohn, J. D.</span><span> </span><span class="NLM_article-title">Automated Ligand Fitting by Core-Fragment Fitting and Extension into Density</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1107/S0907444906017161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1107%2FS0907444906017161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=16855309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=915-922&author=T.+C.+Terwilligerauthor=H.+Kleiauthor=P.+D.+Adamsauthor=N.+W.+Moriartyauthor=J.+D.+Cohn&title=Automated+Ligand+Fitting+by+Core-Fragment+Fitting+and+Extension+into+Density&doi=10.1107%2FS0907444906017161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Automated ligand fitting by core-fragment fitting and extension into density</span></div><div class="casAuthors">Terwilliger, Thomas C.; Klei, Herbert; Adams, Paul D.; Moriarty, Nigel W.; Cohn, Judith D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">915-922</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A procedure for fitting of ligands to electron-d. maps by first fitting a core fragment of the ligand to d. and then extending the remainder of the ligand into d. is presented.  The approach was tested by fitting 9327 ligands over a wide range of resolns. (most are in the range 0.8-4.8 Å) from the Protein Data Bank (PDB) into (Fo - Fc)exp(iφc) difference d. calcd. using entries from the PDB without these ligands.  The procedure was able to place 58% of these 9327 ligands within 2 Å (r.m.s.d.) of the coordinates of the atoms in the original PDB entry for that ligand.  The success of the fitting procedure was relatively insensitive to the size of the ligand in the range 10-100 non-H atoms and was only moderately sensitive to resoln., with the percentage of ligands placed near the coordinates of the original PDB entry for fits in the range 58-73% over all resoln. ranges tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZHd3GwwvtWLVg90H21EOLACvtfcHk0ljr-82HiJWWXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksLc%253D&md5=dc3efbc9f8249c27ea9fa6658d31e316</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444906017161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906017161%26sid%3Dliteratum%253Aachs%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DKlei%26aufirst%3DH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DCohn%26aufirst%3DJ.%2BD.%26atitle%3DAutomated%2520Ligand%2520Fitting%2520by%2520Core-Fragment%2520Fitting%2520and%2520Extension%2520into%2520Density%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D915%26epage%3D922%26doi%3D10.1107%2FS0907444906017161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Moretti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, L.</span><span> </span><span class="NLM_article-title">Software Infrastructure for Computer-Aided Drug Discovery and Development, a Practical Example with Guidelines</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span><span class="refDoi"> DOI: 10.1002/minf.201501037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=10.1002%2Fminf.201501037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=27546042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=382-390&author=L.+Morettiauthor=L.+Sartori&title=Software+Infrastructure+for+Computer-Aided+Drug+Discovery+and+Development%2C+a+Practical+Example+with+Guidelines&doi=10.1002%2Fminf.201501037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Software Infrastructure for Computer-aided Drug Discovery and Development, a Practical Example with Guidelines</span></div><div class="casAuthors">Moretti, Loris; Sartori, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">382-390</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the field of Computer-Aided Drug Discovery and Development (CADDD) the proper software infrastructure is essential for everyday investigations.  The creation of such an environment should be carefully planned and implemented with certain features in order to be productive and efficient.  Here, we describe a soln. to integrate std. computational services into a functional unit that empowers modeling applications for drug discovery.  This system allows users with various level of expertise to run in silico expts. automatically and without the burden of file formatting for different software, managing the actual computation, keeping track of the activities and graphical rendering of the structural outcomes.  To showcase the potential of this approach, performances of five different docking programs on an Hiv-1 protease test set are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUtVBT1Wj49LVg90H21EOLACvtfcHk0ljr-82HiJWWXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL&md5=c9db5ce783d44c6d15f889cbec5ca3f7</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201501037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201501037%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DSoftware%2520Infrastructure%2520for%2520Computer-Aided%2520Drug%2520Discovery%2520and%2520Development%252C%2520a%2520Practical%2520Example%2520with%2520Guidelines%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D382%26epage%3D390%26doi%3D10.1002%2Fminf.201501037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D','PDB','2V1D'); return false;">PDB: 2V1D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGT','PDB','5LGT'); return false;">PDB: 5LGT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHG','PDB','5LHG'); return false;">PDB: 5LHG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGU','PDB','5LGU'); return false;">PDB: 5LGU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHH','PDB','5LHH'); return false;">PDB: 5LHH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI','PDB','5LHI'); return false;">PDB: 5LHI</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i174"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01019">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65748"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01019">10.1021/acs.jmedchem.6b01019</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthetic procedures for the preparation of compounds <b>20e</b>, <b>21e</b>, <b>23e</b>–<b>28e</b>, and <b>44</b>; material and methods for biochemical and cellular assays; data collection and refinement statistics for compounds <b>15</b>, <b>17</b>, <b>34</b>, <b>49</b>, and <b>50</b>; HOMO values and localization; HOMO pictures for compounds <b>15</b>, <b>20</b>–<b>26</b>, and <b>44</b>; calculated physicochemical properties (molecular surface area, AlogP, and Log <i>D</i>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_002.csv">CSV</a>)</p></li><li><p class="inline">Protein Target (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Compound <b>2</b>, pose A (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Compound <b>2</b>, pose B (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_001.pdf">jm6b01019_si_001.pdf (1.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_002.csv">jm6b01019_si_002.csv (10.81 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_003.pdb">jm6b01019_si_003.pdb (510.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_004.pdb">jm6b01019_si_004.pdb (11.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01019/suppl_file/jm6b01019_si_005.pdb">jm6b01019_si_005.pdb (11.84 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the crystal structures of KDM1A with compounds <b>15</b>, <b>17</b>, <b>34</b>, <b>49</b>, and <b>50</b> can be accessed using PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGT">5LGT</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHG">5LHG</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGU">5LGU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHH">5LHH</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI">5LHI</a>, respectively. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01019%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01019" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996c63ad7125a5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
