
<html lang="en"     class="pb-page"  data-request-id="6125e638-b54e-4ba2-85ac-67d93608948d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01937;issue:issue:10.1021/jmcmar.2021.64.issue-7;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Evaluation of (2-Aminocyclopropyl)phenyl Derivatives as Novel Positron Emission Tomography Imaging Agents for Lysine-Specific Demethylase 1 in the Brain" /></meta><meta name="dc.Creator" content="Yasushi  Hattori" /></meta><meta name="dc.Creator" content="Satoru  Matsuda" /></meta><meta name="dc.Creator" content="Rina  Baba" /></meta><meta name="dc.Creator" content="Kouta  Matsumiya" /></meta><meta name="dc.Creator" content="Shinji  Iwasaki" /></meta><meta name="dc.Creator" content="Cristian C.  Constantinescu" /></meta><meta name="dc.Creator" content="Thomas J.  Morley" /></meta><meta name="dc.Creator" content="Vincent M.  Carroll" /></meta><meta name="dc.Creator" content="Caroline  Papin" /></meta><meta name="dc.Creator" content="Alexandra  Gouasmat" /></meta><meta name="dc.Creator" content="David  Alagille" /></meta><meta name="dc.Creator" content="Gilles  Tamagnan" /></meta><meta name="dc.Creator" content="Tatsuki  Koike" /></meta><meta name="dc.Description" content="Dysregulation of histone H3 lysine 4 (H3K4) methylation is implicated in the pathogenesis of neurodevelopmental disorders. Lysine-specific demethylase 1 (LSD1) determines the methylation status of ..." /></meta><meta name="Description" content="Dysregulation of histone H3 lysine 4 (H3K4) methylation is implicated in the pathogenesis of neurodevelopmental disorders. Lysine-specific demethylase 1 (LSD1) determines the methylation status of ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 17, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01937" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01937" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01937" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01937" /></link>
        
    
    

<title>Design, Synthesis, and Evaluation of (2-Aminocyclopropyl)phenyl Derivatives as Novel Positron Emission Tomography Imaging Agents for Lysine-Specific Demethylase 1 in the Brain | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01937" /></meta><meta property="og:title" content="Design, Synthesis, and Evaluation of (2-Aminocyclopropyl)phenyl Derivatives as Novel Positron Emission Tomography Imaging Agents for Lysine-Specific Demethylase 1 in the Brain" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0015.jpeg" /></meta><meta property="og:description" content="Dysregulation of histone H3 lysine 4 (H3K4) methylation is implicated in the pathogenesis of neurodevelopmental disorders. Lysine-specific demethylase 1 (LSD1) determines the methylation status of H3K4 through flavin adenine dinucleotide (FAD)-mediated histone demethylation. Therefore, LSD1 inhibition in the brain can be a novel therapeutic option for treating these disorders. Positron emission tomography (PET) imaging of LSD1 allows for investigating LSD1 expression levels under normal and disease conditions and validating target engagement of therapeutic LSD1 inhibitors. This study designed and synthesized (2-aminocyclopropyl)phenyl derivatives with irreversible binding to LSD1 as PET imaging agents for LSD1 in the brain. We optimized lipophilicity of the lead compound to minimize the risk of nonspecific binding and identified 1e with high selectivity over monoamine oxidase A and B, which are a family of FAD-dependent enzymes homologous to LSD1. PET imaging in a monkey showed a high uptake of [18F]1e to regions enriched with LSD1, indicating its specific binding to LSD1." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01937"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01937">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01937&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01937&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01937&amp;href=/doi/10.1021/acs.jmedchem.0c01937" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3780-3793</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01898" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01940" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Evaluation of (2-Aminocyclopropyl)phenyl Derivatives as Novel Positron Emission Tomography Imaging Agents for Lysine-Specific Demethylase 1 in the Brain</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yasushi Hattori</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yasushi Hattori</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yasushi++Hattori">Yasushi Hattori</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4054-3619" title="Orcid link">http://orcid.org/0000-0003-4054-3619</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Satoru Matsuda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Satoru Matsuda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Satoru++Matsuda">Satoru Matsuda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rina Baba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rina Baba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rina++Baba">Rina Baba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kouta Matsumiya</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kouta Matsumiya</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kouta++Matsumiya">Kouta Matsumiya</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shinji Iwasaki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shinji Iwasaki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shinji++Iwasaki">Shinji Iwasaki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cristian C. Constantinescu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cristian C. Constantinescu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cristian+C.++Constantinescu">Cristian C. Constantinescu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas J. Morley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas J. Morley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+J.++Morley">Thomas J. Morley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vincent M. Carroll</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vincent M. Carroll</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vincent+M.++Carroll">Vincent M. Carroll</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Caroline Papin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Caroline Papin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Caroline++Papin">Caroline Papin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexandra Gouasmat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexandra Gouasmat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Invicro LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexandra++Gouasmat">Alexandra Gouasmat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David Alagille</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Alagille</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">XingImaging LLC, 760 Temple Street, New Haven, Connecticut 06510, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Alagille">David Alagille</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gilles Tamagnan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gilles Tamagnan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">XingImaging LLC, 760 Temple Street, New Haven, Connecticut 06510, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gilles++Tamagnan">Gilles Tamagnan</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Tatsuki Koike</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tatsuki Koike</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#abdfcadfd8dec0c285c0c4c2c0ceebdfcac0cecfca85c8c4c6"><span class="__cf_email__" data-cfemail="0175607572746a682f6a6e686a644175606a6465602f626e6c">[email protected]</span></a>. Phone: +81-466-32-1132.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tatsuki++Koike">Tatsuki Koike</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3264-1406" title="Orcid link">http://orcid.org/0000-0002-3264-1406</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01937&amp;href=/doi/10.1021%2Facs.jmedchem.0c01937" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3780–3793</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 17, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 November 2020</li><li><span class="item_label"><b>Published</b> online</span>17 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01937" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01937</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3780%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYasushi%2BHattori%252C%2BSatoru%2BMatsuda%252C%2BRina%2BBaba%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D7%26contentID%3Dacs.jmedchem.0c01937%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2B%25282-Aminocyclopropyl%2529phenyl%2BDerivatives%2Bas%2BNovel%2BPositron%2BEmission%2BTomography%2BImaging%2BAgents%2Bfor%2BLysine-Specific%2BDemethylase%2B1%2Bin%2Bthe%2BBrain%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3793%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01937"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">764</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01937" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Evaluation of (2-Aminocyclopropyl)phenyl Derivatives as Novel Positron Emission Tomography Imaging Agents for Lysine-Specific Demethylase 1 in the Brain&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yasushi&quot;,&quot;last_name&quot;:&quot;Hattori&quot;},{&quot;first_name&quot;:&quot;Satoru&quot;,&quot;last_name&quot;:&quot;Matsuda&quot;},{&quot;first_name&quot;:&quot;Rina&quot;,&quot;last_name&quot;:&quot;Baba&quot;},{&quot;first_name&quot;:&quot;Kouta&quot;,&quot;last_name&quot;:&quot;Matsumiya&quot;},{&quot;first_name&quot;:&quot;Shinji&quot;,&quot;last_name&quot;:&quot;Iwasaki&quot;},{&quot;first_name&quot;:&quot;Cristian&quot;,&quot;last_name&quot;:&quot;C. Constantinescu&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;J. Morley&quot;},{&quot;first_name&quot;:&quot;Vincent&quot;,&quot;last_name&quot;:&quot;M. Carroll&quot;},{&quot;first_name&quot;:&quot;Caroline&quot;,&quot;last_name&quot;:&quot;Papin&quot;},{&quot;first_name&quot;:&quot;Alexandra&quot;,&quot;last_name&quot;:&quot;Gouasmat&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Alagille&quot;},{&quot;first_name&quot;:&quot;Gilles&quot;,&quot;last_name&quot;:&quot;Tamagnan&quot;},{&quot;first_name&quot;:&quot;Tatsuki&quot;,&quot;last_name&quot;:&quot;Koike&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3780-3793&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01937&quot;},&quot;abstract&quot;:&quot;Dysregulation of histone H3 lysine 4 (H3K4) methylation is implicated in the pathogenesis of neurodevelopmental disorders. Lysine-specific demethylase 1 (LSD1) determines the methylation status of H3K4 through flavin adenine dinucleotide (FAD)-mediated histone demethylation. Therefore, LSD1 inhibition in the brain can be a novel therapeutic option for treating these disorders. Positron emission tomography (PET) imaging of LSD1 allows for investigating LSD1 expression levels under normal and disease conditions and validating target engagement of therapeutic LSD1 inhibitors. This study designed and synthesized (2-aminocyclopropyl)phenyl derivatives with irreversible binding to LSD1 as PET imaging agents for LSD1 in the brain. We optimized lipophilicity of the lead compound to minimize the risk of nonspecific binding and identified 1e with high selectivity over monoamine oxidase A and B, which are a family of FAD-dependent enzymes homologous to LSD1. PET imaging in a monkey showed a high uptake of [18F]1e to reg&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01937&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01937" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01937&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01937" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01937&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01937" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01937&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01937&amp;href=/doi/10.1021/acs.jmedchem.0c01937" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01937" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01937" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01937%26sid%3Dliteratum%253Aachs%26pmid%3D33729758%26genre%3Darticle%26aulast%3DHattori%26date%3D2021%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2B%25282-Aminocyclopropyl%2529phenyl%2BDerivatives%2Bas%2BNovel%2BPositron%2BEmission%2BTomography%2BImaging%2BAgents%2Bfor%2BLysine-Specific%2BDemethylase%2B1%2Bin%2Bthe%2BBrain%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D7%26spage%3D3780%26epage%3D3793%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292462" title="Central nervous system">Central nervous system</a>,</li><li><a href="/action/doSearch?ConceptID=292200" title="Layers">Layers</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/jmcmar.2021.64.issue-7/20210408/jmcmar.2021.64.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Dysregulation of histone H3 lysine 4 (H3K4) methylation is implicated in the pathogenesis of neurodevelopmental disorders. Lysine-specific demethylase 1 (LSD1) determines the methylation status of H3K4 through flavin adenine dinucleotide (FAD)-mediated histone demethylation. Therefore, LSD1 inhibition in the brain can be a novel therapeutic option for treating these disorders. Positron emission tomography (PET) imaging of LSD1 allows for investigating LSD1 expression levels under normal and disease conditions and validating target engagement of therapeutic LSD1 inhibitors. This study designed and synthesized (2-aminocyclopropyl)phenyl derivatives with irreversible binding to LSD1 as PET imaging agents for LSD1 in the brain. We optimized lipophilicity of the lead compound to minimize the risk of nonspecific binding and identified <b>1e</b> with high selectivity over monoamine oxidase A and B, which are a family of FAD-dependent enzymes homologous to LSD1. PET imaging in a monkey showed a high uptake of [<sup>18</sup>F]<b>1e</b> to regions enriched with LSD1, indicating its specific binding to LSD1.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71474" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71474" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic control by chromatin remodeling through the modifications of histones (e.g., acetylation, methylation, and phosphorylation) is essential in the regulation of gene transcription.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Specifically, the methylation of histone H3 lysine 4 (H3K4) is involved in various neural functions (e.g., learning and memory) through the regulation of synaptic plasticity.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Recent studies have revealed that the reduced methylation of H3K4 and the consequent reduction of gene transcription via the alteration of the chromatin structure is implicated in the pathogenesis of various psychiatric and neurological disorders including schizophrenia,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> autism spectrum disorders (ASD),<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> intellectual disability disorders,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and Huntington’s disease.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Therefore, the recovery of H3K4 methylation levels can be a promising approach to treat patients with epigenetic dysfunction.</div><div class="NLM_p">Lysine-specific demethylase 1 (LSD1, also known as KDM1A) determines the methylation status of H3K4 by catalyzing the removal of methyl groups from mono- and di-methylated H3K4 through flavin adenine dinucleotide (FAD)-dependent enzymatic oxidation.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Therefore, the recovery of H3K4 methylation levels by the inhibition of LSD1 enzyme activity in the brain may offer novel therapeutic options for treating central nervous system diseases with epigenetic dysregulation. For example, it is reported that some of the brain-penetrating LSD inhibitors increased H3K4 methylation levels and improved the learning function in a mouse model of schizophrenia and ASD.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a></div><div class="NLM_p">Positron emission tomography (PET) is a non-invasive imaging technology that allows us to quantify physiological and pathological processes in living subjects. PET imaging may potentially allow us to determine the occupancy of therapeutic LSD1 inhibitors in LSD1; this technology will help to assess the target engagement of LSD1-targeting drugs. In addition, the expression levels of LSD1 and their alterations in homeostatic and disease conditions can be assessed by <i>in vivo</i> PET scans. However, no PET imaging agents for LSD1 have been published thus far.</div><div class="NLM_p">A number of LSD1 inhibitors as antitumor agents like (±)<b>1</b> have been reported thus far.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> X-ray crystal analyses revealed that <b>1</b> (eutomer) produces a <b>1</b>-FAD adduct in the binding pocket of LSD1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> However, our group identified irreversible-type selective LSD1 inhibitors such as T-448 which uniquely form a compact formylated FAD adduct and do not show steric hindrance with LSD1 cofactors such as GFI1B, which is a critical regulator of hematopoiesis (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Hence, for the design of LSD1 PET imaging agents, it is essential to carefully consider the structures of resultant FAD adducts. Radiolabeling positions in the LSD1 inhibitors which generate formylated FAD adducts are very limited; thus, we selected (±)<b>1</b> as a lead compound for the development of LSD1 PET imaging agents (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). In this report, we describe the design and synthesis of novel PET imaging agents for LSD1 as well as their <i>in vivo</i> characterization by a monkey brain PET scan.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known LSD1 inhibitors as antitumor agents, FAD and <b>1</b>-FAD adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of T-448 and the formylated-FAD adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Novel PET imaging agents for LSD1 in the brain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26051" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26051" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">The <i>N</i>-(4-(2-aminocyclopropyl)phenyl)amide derivative <b>1a</b> was synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The protection of aniline <b>2</b><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> by the Troc group produced compound <b>3</b>. The deprotection of <b>3</b> by hydrogen chloride yielded compound <b>4</b>, which was then converted to <b>5</b> via reductive amination with cyclopropanealdehyde and the subsequent protection of the cyclopropylamine group by treatment with Boc<sub>2</sub>O. The removal of the Troc group of <b>5</b> through zinc treatment under acidic conditions yielded the aniline derivative <b>6</b>. The condensation of <b>6</b> with benzoic acid and the subsequent deprotection produced the desired product <b>1a</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the <i>N</i>-(4-(2-Aminocyclopropyl)phenyl)amide Derivative <b>1a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2,2-trichloroethyl chloroformate, TEA, THF, 66%; (b) HCl, CPME, quant; (c) (1) cyclopropanecarbaldehyde, NaHCO<sub>3</sub>, THF/MeOH, (2) NaBH<sub>4</sub>, and (3) Boc<sub>2</sub>O, 74%; (d) Zn, AcOH, THF, 68%; (e) benzoyl chloride, TEA, THF, 99%; and (f) HCl, CPME, 77%.</p></p></figure><div class="NLM_p">The 4-(2-aminocyclopropyl)benzamide derivatives <b>1b–d</b> were synthesized according to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The Corey–Chaykovsky reaction of <b>8</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> followed by deprotection of the <i>tert</i>-butyl moiety afforded the carboxylic acid derivative <b>10</b>, which was then subjected to Curtius rearrangement to furnish <b>11</b>. The optical resolution of <b>11</b> by HPLC yielded the optically pure intermediate <b>12</b>. Intermediates <b>11</b> and <b>12</b> were converted to carboxylic acid derivatives <b>13a</b> and <b>13b</b> by hydrolysis. Desired products <b>1b</b>, <b>1c,</b> and <b>1d</b> were prepared from <b>13a</b> and <b>13b</b> via condensation with the corresponding amine, deprotection by acid, and subsequent reductive amination with cyclopropanealdehyde, respectively.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4-(2-aminocyclopropyl)benzamide Derivatives <b>1b–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) trimethylsulfoxonium iodide, KO<i>t</i>Bu, DMSO, 57%; (b) TFA, 87%; (c) DPPA, TEA, <i>t</i>BuOH, toluene, 85%; (d) optical resolution by HPLC (Column: CHIRALPAK AD, desired isomer is tR2), 35%; (e) NaOH, MeOH, <b>13a</b>: 95%, <b>13b</b>: 96%; (f) corresponding amine, EDCI, HOBt, TEA, DMF for <b>14a</b>, 61%; (g) corresponding amine, HATU, TEA, DMF for <b>14b</b> and <b>14c</b>, <b>14b</b>: 67%, <b>14c</b>: 96%; (h) HCl, EtOAc, <b>15a</b>: quant, <b>15b</b>: quant, <b>15c</b>: 99%; and (i) (1) cyclopropanecarbaldehyde, NaHCO<sub>3</sub>, THF/MeOH and (2) NaBH<sub>4</sub>, <b>1b</b>: 33%, <b>1c</b>: 34%, <b>1d</b>: 32%.</p></p></figure><div class="NLM_p">The synthesis of <b>1e</b> is shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The alkylation of compound <b>16</b><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> afforded <b>17</b>, which was then converted to carboxylic acid <b>19</b> via the carbonylation reaction and subsequent saponification. The optical resolution of <b>19</b> by HPLC yielded the optically pure intermediate <b>20</b>. The condensation of <b>20</b> with 6-fluoro-<i>N</i>-methylpyridin-3-amine and the subsequent deprotection produced desired product <b>1e</b>.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>1e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (bromomethyl)cyclopropane, NaH, DMF, 87%; (b) CO (3 atm), PdCl<sub>2</sub> (dppf), TEA, MeOH, 87%; (c) NaOH, water/MeOH, 90%; (d) optical resolution by HPLC (Column: CHIRALPAK AD-H, desired isomer is tR2), 46%; (e) (COCl)<sub>2</sub>, DMF, THF, and then 6-fluoro-<i>N</i>-methylpyridin-3-amine, DMAP, pyridine, THF, quant; and (f) HCl, EtOAc, 76%.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Designed Compounds and <i>in Vitro</i> Characterization</h3><div class="NLM_p">Central nervous system PET imaging agents not only require high affinity and target selectivity for sufficient <i>in vivo</i> binding potential but also a desirable chemical structure, physicochemical and pharmacokinetic properties that are appropriate for radiolabeling, efficient blood–brain barrier (BBB) penetration, and low levels of undesirable nonspecific binding (NSB).<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> (±)<b>1</b> irreversibly inhibits LSD1 enzyme activity with 29 nM concentration for 50% inhibition (IC<sub>50</sub>). This compound is unlikely to be a substrate for P-glycoprotein [P-gp, also known as multidrug resistance 1 (MDR1)] because its efflux BA/AB ratio is 0.8, which indicates its good brain permeability.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, (±)<b>1</b> lacks a site for <sup>11</sup>C and <sup>18</sup>F radiolabeling, and its NSB may not be sufficiently low, judging from its low unbound fraction (Fu) in rat plasma (<1%).<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> This will be attributed to the relatively high lipophilicity of the compound (log <i>D</i> = 3.29).<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> Thus, to overcome these concerns, we decided to modify the biphenyl amide moiety of (±)<b>1</b> to reduce the lipophilicity by introducing a chemical group that can be labeled with <sup>18</sup>F.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">The synthesized compounds were first evaluated by the <i>in vitro</i> parameters of LSD1, IC<sub>50</sub>, log <i>D</i>, MDR1 efflux ratio, and Fu, in rat plasma.</div><div class="NLM_p">Our initial efforts focused on reducing the lipophilicity of (±)<b>1</b> by removing the distal benzene ring. The <i>N</i>-(4-(2-Aminocyclopropyl)phenyl)amide derivative <b>1a</b> and 4-(2-aminocyclopropyl)benzamide derivative <b>1b</b> exhibit lower log <i>D</i> values (<b>1a</b>:1.88 and <b>1b</b>:2.26) and higher Fu values (<b>1a</b>:24% and <b>1b</b>:40%) than those of (±)<b>1</b>. This is in line with our hypothetical view that the NSB of a compound can be reduced by lowering its lipophilicity. These compounds also show increased inhibitory activity compared with that of (±)<b>1</b>. Specifically, the 4-(2-aminocyclopropyl)benzamide derivative <b>1b</b> exhibits an approximately sevenfold improvement in the activity (IC<sub>50</sub> = 4.1 nM). Because <b>1b</b> also exhibits an MDR1 efflux ratio (0.5) that is suitable for BBB penetration, which is similar to (±)<b>1</b>, we further modified <b>1b</b> to introduce a chemical group that can be labeled with <sup>18</sup>F. The 3-Cyano-4-fluorophenyl derivative <b>1c</b> and 6-fluoropyridine derivative <b>1d</b>, which are designed to introduce a fluorine atom as the labeling part, exhibit a single-digit nanomolar activity for the inhibition of LSD1. On the basis of these findings and the measured log <i>D</i> values, we selected the pyridine derivative <b>1d</b> with a relatively lower lipophilicity compared with <b>1c</b> as the first candidate for radiolabeling (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>In Vitro</i> Characterization of Compounds (±)<b>1</b> and <b>1a–e</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0013.gif" alt="" id="GRAPHIC-d7e811-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0014.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values and 95% confidence intervals (given in parentheses) were calculated by nonlinear regression analysis of percent inhibition data (<i>n</i> = 2). All values are rounded to two significant digits.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">pH <sub>=</sub> 7.4.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">MDR1 efflux ratios in P-gp-overexpressing cells at the 1 μM substrate.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Unbound fraction in rat plasma.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Racemic compound.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Optically pure compound.</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p">The nitro-precursor <b>22</b> was prepared by the condensation of <b>20</b> with 6-nitro-3-pyridinamine (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>), and the chloro-precursor <b>26</b> was prepared from <b>13</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf" class="ext-link">Scheme S1</a>). The radiolabeling of [<sup>18</sup>F]<b>1d</b> was attempted from the precursors using various basic conditions (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf" class="ext-link">Scheme S2 and Table S1</a>). However, no radiolabeled [<sup>18</sup>F]<b>1d</b> was identified in any of the attempted conditions. A possible reason may be the presence of NH of the amide group at the para-position in the leaving groups on the aromatic ring (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf" class="ext-link">Figure S1</a>). This will increase the electron density at the carbon of the C–NO<sub>2</sub> bond in <b>22</b> or the C–Cl bond in <b>26</b> in the basic radiolabeling condition and will deactivate this center to a nucleophilic attack by [<sup>18</sup>F]fluoride. From this hypothetical view, the pyridinyl <i>N</i>-methyl amide derivative <b>1e</b> was designed to remove the active proton in the precursor. Because compound <b>1e</b> shows a potent inhibitory activity for LSD1 as well as a high Fu value, we selected <b>1e</b> as the second candidate for radiolabeling and further characterization.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of [<sup>18</sup>F]<b>1e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (COCl)<sub>2</sub>, DMF, THF, and then 6-nitro-3-pyridinamine, DMAP, pyridine, THF, 33%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 89%; and (c) K[<sup>18</sup>F]F, 2,2,2-cryptand, K<sub>2</sub>CO<sub>3</sub>, DMSO, 160 °C, 10 min, and then 1 M HCl solution (0.5 mL), 90 °C, 4 min; radiochemical purity, >98%; molar activity, 268 ± 206 GBq/μmol (<i>n</i> = 5).</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Radiolabeling of [<sup>18</sup>F]<b>1e</b></h3><div class="NLM_p last">The synthesis of [<sup>18</sup>F]<b>1e</b> is illustrated in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The nitro-precursor <b>23</b> was prepared by the <i>N</i>-methylation of <b>22</b>. Nucleophilic [<sup>18</sup>F]fluorination with the K[<sup>18</sup>F]F/2,2,2-cryptand of <b>23</b>, followed by the acidic removal of the Boc-group successfully afforded [<sup>18</sup>F]<b>1e</b> (29.6 ± 6.6% (<i>n</i> = 3) radiochemical yield, decay-corrected for cyclotron-produced [<sup>18</sup>F]fluoride) with high molar activity (>100 GBq/μmol at the end of synthesis) and radiochemical purity (>98%).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> PET Imaging of a Monkey Brain with [<sup>18</sup>F]<b>1e</b></h3><div class="NLM_p">The PET imaging of a monkey brain with [<sup>18</sup>F]<b>1e</b> was conducted. Average [<sup>18</sup>F]<b>1e</b> PET images are presented in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> to illustrate the tracer uptake and distribution in the brain over early 0–30 min and late 30–240 min postinjection intervals. [<sup>18</sup>F]<b>1e</b> penetrated into the brain with the maximum whole-brain percentage of injected dose (%ID) of ∼2.2%. The brain distribution of [<sup>18</sup>F]<b>1e</b> revealed that the highest uptake occurred in the cerebellar lobes and the anterior cingulate, followed by the insular cortex, caudate, putamen, and amygdala; the lowest radioactivity uptake was in the corpus callosum and pons. Average standard uptake values (SUVs) between 60 and 240 min after tracer injection are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. The time activity curves (TAC) in each brain region in SUV (g/mL) units are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. [<sup>18</sup>F]<b>1e</b> showed a rapid uptake up to ∼5 min, followed by the washout up to ∼60 min, and a steady state up to the end of the scan (240 min), indicating irreversible binding.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Average [<sup>18</sup>F]<b>1e</b> PET images of a monkey brain between 0 and 30 min (middle row) and between 30 and 240 min postinjection (bottom row) in SUV units. Horizontal sections containing the corona radiate (left), sagittal sections near the midline (middle), and coronal sections through the orbits (right) are superimposed on the corresponding individual MRI data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. TAC (SUV; g/mL) of [<sup>18</sup>F]<b>1e</b> in a monkey brain between 0 and 240 min postinjection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of the [<sup>18</sup>F]<b>1e</b> Brain Uptake Evaluated as Average SUV between 60 and 240 min After Tracer Injection</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">region</th><th class="colsep0 rowsep0" align="center" char=".">average SUV</th><th class="colsep0 rowsep0" align="center">region</th><th class="colsep0 rowsep0" align="center" char=".">average SUV</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">whole brain</td><td class="colsep0 rowsep0" align="char" char=".">1.04</td><td class="colsep0 rowsep0" align="left">insula</td><td class="colsep0 rowsep0" align="char" char=".">1.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">frontal lobe</td><td class="colsep0 rowsep0" align="char" char=".">1.09</td><td class="colsep0 rowsep0" align="left">caudate nucleus</td><td class="colsep0 rowsep0" align="char" char=".">1.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">temporal lobe</td><td class="colsep0 rowsep0" align="char" char=".">1.06</td><td class="colsep0 rowsep0" align="left">putamen</td><td class="colsep0 rowsep0" align="char" char=".">1.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hippocampus</td><td class="colsep0 rowsep0" align="char" char=".">1.01</td><td class="colsep0 rowsep0" align="left">thalamus</td><td class="colsep0 rowsep0" align="char" char=".">1.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">amygdala</td><td class="colsep0 rowsep0" align="char" char=".">1.10</td><td class="colsep0 rowsep0" align="left">cerebellar lobes</td><td class="colsep0 rowsep0" align="char" char=".">1.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">parietal lobe</td><td class="colsep0 rowsep0" align="char" char=".">1.04</td><td class="colsep0 rowsep0" align="left">corpus Callosum</td><td class="colsep0 rowsep0" align="char" char=".">0.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">occipital lobe</td><td class="colsep0 rowsep0" align="char" char=".">1.04</td><td class="colsep0 rowsep0" align="left">pons</td><td class="colsep0 rowsep0" align="char" char=".">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">anterior Cingulate</td><td class="colsep0 rowsep0" align="char" char=".">1.31</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> <b>1e</b>-FAD Adduct</h3><div class="NLM_p">To study whether the radiolabeled positions can remain in the FAD adduct after the inactivation of FAD in LSD1, we performed the LC/MS analysis using human recombinant LSD1 reacted with <b>1e</b>. <b>1e</b> generated a peak with <i>m</i>/<i>z</i> 534.6246, which corresponded to the molecular weight of the <b>1e</b>-FAD adduct (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). This result confirmed that <b>1e</b> irreversibly inhibited LSD1, and the radiolabeled position remained in the resultant FAD adduct.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. LC/MS spectra of FAD and <b>1a</b>-FAD adduct in the human recombinant LSD1 treated with <b>1e</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> LSD1 Distribution in the Brain</h3><div class="NLM_p">LSD1 protein biodistribution in the brain has not been reported so far, although the information is useful to assess the specific binding of [<sup>18</sup>F]<b>1e</b> to LSD1. Hence, precise distribution patterns of LSD1 in the human brain, monkey brain, and rat brain were evaluated by the western blot (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). In the human brain, the LSD1 expression level in the cerebellum was the highest, and that in the medulla was the lowest among the tested regions. Similar distribution patterns were observed in monkeys and rats; the expression levels were relatively higher in the cerebellum, and lower in the brain stem regions (pons and medulla). This distribution pattern of LSD1 was consistent with the radioactivity uptake of [<sup>18</sup>F]<b>1e</b>, which suggested the specificity of [<sup>18</sup>F]<b>1e</b> for LSD1.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Distribution patterns of LSD1 in the regions of human, monkey, and rat brains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> <i>In Vitro</i> Binding of <b>1e</b> and Its Selectivity for MAOA and MAOB</h3><div class="NLM_p">Given that the <i>in vivo</i> PET results indicate that [<sup>18</sup>F]<b>1e</b> may be a potential PET imaging agent that reflects LSD1 expression in the brain, the binding properties of <b>1e</b> were further assessed by <i>in vitro</i> measurements. The specificity constant of irreversible inhibition (<i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>) for <b>1e</b> was determined as 5.5 × 10<sup>5</sup> by a peroxidase-coupled assay. The specificity of compound <b>1e</b> for LSD1 was also assessed using an <i>in vitro</i> enzyme assay for MAOA and MAOB, and with FAD-dependent enzymes with a high amino acid sequence homology for LSD1. Compound <b>1e</b> showed a very weak inhibitory activity for MAOA/B with 19,000 nM and >100,000 nM IC<sub>50</sub>, respectively. These results support the high selectivity of <b>1e</b> for LSD1 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>In vitro</i> Profiles of <b>1e</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">LSD1</th><th class="rowsep1 colsep0" align="center">MAOA/MAOB</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">5.5.E+05</td><td class="colsep0 rowsep0" align="left">19,000/>100,000</td></tr></tbody></table></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Starting from the irreversible LSD1 inhibitor (±)<b>1</b>, we designed and synthesized a series of novel (2-aminocyclopropyl)phenyl derivatives to identify PET imaging agents for LSD1. By tailoring the lipophilicity of the molecule to optimize NSB and BBB permeability, we successfully identified <b>1e</b> with a potent inhibitory activity for LSD1, an MDR1 efflux ratio suitable for BBB penetration, and a high Fu value. The <i>in vivo</i> PET analysis of [<sup>18</sup>F]<b>1e</b> in a monkey brain and the supplemental <i>in vitro</i> evaluation indicated the specific uptake of [<sup>18</sup>F]<b>1e</b> in regions with high LSD1 expression. Further validation, including blocking studies with pretreatment of LSD1 inhibitors and detailed radiometabolite analysis, is underway to assess the suitability of [<sup>18</sup>F]<b>1e</b> as a potential lead for clinical translation.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General</h3><div class="NLM_p">Proton and carbon nuclear magnetic resonance (<sup>1</sup>H NMR and <sup>13</sup>C NMR) spectra were recorded on Bruker Ultra Shield-300 (300 MHz) or Varian INOVA-400 (400 MHz) instruments. Chemical shifts are given in parts per million (ppm) with tetramethylsilane as an internal standard. Abbreviations used are as follows: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublets of doublet, and brs = broad singlet. Coupling constants (<i>J</i> values) are given in hertz (Hz). The acidic protons of diketones, carboxylic acids, alcohols, or anilines were not frequently observed in <sup>1</sup>H NMR spectra. Elemental analyses were performed by Takeda Analytical Laboratories Ltd., Toray Research Center, Inc., or Sumika Chemical analysis Service, Ltd. Purity data were collected by a HPLC system with a Corona CAD (Charged Aerosol Detector) or a photo diode array detector. The column was a Capcell Pak C18AQ (50 mm × 3.0 mm I.D., Shiseido, Japan) or L-column 2 ODS (30 mm × 2.0 mm I.D., CERI, Japan) with a temperature of 50 °C and a flow rate of 0.5 mL/min. Mobile phases A and B under a neutral condition were a mixture of 50 mmol/L ammonium acetate, water, and acetonitrile (1:8:1, v/v/v) and a mixture of 50 mmol/L ammonium acetate and acetonitrile (1:9, v/v), respectively. The ratio of mobile phase B was increased linearly from 5 to 95% over 3 min and kept at 95% over the next 1 min. Mobile phases A and B under an acidic condition were a mixture of 0.2% formic acid in 10 mmol/L ammonium formate and 0.2% formic acid in acetonitrile, respectively. The ratio of mobile phase B was increased linearly from 14 to 86% over 3 min and kept at 86% over the next 1 min. Chemical intermediates were characterized by <sup>1</sup>H NMR and/or <sup>13</sup>C NMR and/or mass spectral data. The HPLC analyses were performed using a Shimadzu UFLC instrument, equipped with a L-column 2 ODS (3.0 × 50 mm, 2 μm) column, eluting with a gradient of 5–90% solvent B in solvent A (solvent A was 0.1% TFA in water, and solvent B was 0.1% TFA in acetonitrile), at a flow rate of 1.2 mL/min, with UV detection at 220 nm. Reagents and solvents were obtained from commercial sources and used without further purification. Reaction progress was determined by thin layer chromatography analysis on Merck Kieselgel 60 F254 plates or Fuji Silysia NH plates. Chromatographic purification was performed on silica gel columns [(Merck Kieselgel 60, 70–230 mesh size or 230–400 mesh size, Merck) or (Chromatorex NH-DM 1020, 100–200 mesh size)] or on Purif-Pack (SI or NH, particle size: 60 μm, Fuji Silysia Chemical, Ltd.). The purities of all tested compounds in biological systems were confirmed to be more than 95% pure, as determined by analytical HPLC.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>rac</i>-((1<i>R</i>,2<i>S</i>)-2-(4-(((2,2,2-Trichloroethoxy)carbonyl)amino)phenyl)cyclopropyl)carbamate (<b>3</b>)</h4><div class="NLM_p last">To a solution of <b>2</b> (17 g, 68 mmol) and Et<sub>3</sub>N (11 mL, 81 mmol) in THF (340 mL) was added 2,2,2-trichloroethyl chloroformate (11 mL, 81 mmol). After being stirred at room temperature overnight, the mixture was poured into saturated NH<sub>4</sub>Cl aqueous solution and extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 0–25% EtOAc in hexane) to give the title compound (19.0 g, 45 mmol, 66%) as a white powder.<sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 1.07–1.19 (2H, m), 1.45 (9H, s), 1.95–2.08 (1H, m), 2.68 (1H, br s), 4.81 (3H, br s), 6.84 (1H, br s), 7.11 (2H, d, <i>J</i> = 7.8 Hz), 7.31 (2H, d, <i>J</i> = 7.8 Hz).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>rac</i>-2,2,2-Trichloroethyl (4-((1<i>S</i>,2<i>R</i>)-2-Aminocyclopropyl)phenyl)carbamate Hydrochloride (<b>4</b>)</h4><div class="NLM_p last">A mixture of <b>3</b> (19.0 g, 44.8 mmol) and 4 M HCl/CPME solution (188 mL, 752 mmol) was stirred at room temperature for 3 h. The mixture was concentrated <i>in vacuo</i> to give the title compound (16.2 g, 45.0 mmol, 100%) as a pale yellow powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.09–1.22 (1H, m), 1.24–1.37 (1H, m), 2.16–2.30 (1H, m), 2.69–2.81 (1H, m), 4.93 (2H, s), 7.11 (2H, d, <i>J</i> = 8.3 Hz), 7.43 (2H, d, <i>J</i> = 8.3 Hz), 8.21 (3H, br s), 10.11 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 324.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>rac</i>-<i>tert</i>-Butyl (Cyclopropylmethyl)((1<i>R</i>,2<i>S</i>)-2-(4-(((2,2,2-trichloroethoxy)carbonyl)amino)phenyl)cyclopropyl)carbamate (<b>5</b>)</h4><div class="NLM_p last">To a solution of <b>4</b> (16.2 g, 45.0 mmol) and NaHCO<sub>3</sub> (7.56 g, 90.0 mmol) in THF (112 mL)/MeOH (112 mL) was added cyclopropanecarbaldehyde (4.37 mL, 58.5 mmol). After being stirred at 60 °C for 2 h, the reaction mixture was cooled to 0 °C. To the mixture was added NaBH<sub>4</sub> (3.40 g, 90.0 mmol) portionwise. After being stirred at room temperature for 1 h, Boc<sub>2</sub>O (14.7 g, 67.5 mmol) was added. After being stirred at room temperature overnight, the mixture was poured into water and extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 0–20% EtOAc in hexane) to give the title compound (15.9 g, 33.3 mmol, 74%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.05–0.15 (1H, m), 0.16–0.27 (1H, m), 0.32–0.51 (2H, m), 0.89–1.04 (1H, m), 1.12–1.25 (2H, m), 1.35 (9H, s), 2.00–2.08 (1H, m), 2.62–2.75 (1H, m), 2.99 (1H, dd, <i>J</i> = 14.2, 6.9 Hz), 3.17 (1H, dd, <i>J</i> = 14.2, 6.9 Hz), 4.93 (2H, s), 7.08 (2H, d, <i>J</i> = 8.7 Hz), 7.40 (2H, d, <i>J</i> = 8.1 Hz), 10.06 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 378.9 [M + H-Boc]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>rac</i>-<i>tert</i>-Butyl ((1<i>R</i>,2<i>S</i>)-2-(4-Aminophenyl)cyclopropyl)(cyclopropylmethyl)carbamate (<b>6</b>)</h4><div class="NLM_p last">To a solution of <b>5</b> (16 g, 33 mmol) in THF (170 mL) were added zinc dust (33 g, 500 mmol) and acetic acid (5.0 mL, 87 mmol). After being stirred at room temperature for 5 h, the mixture was basified with 1 M NaOH aqueous solution (100 mL), and EtOAc (500 mL) was added. The mixture was filtered through a celite pad, and the organic layer of the filtrate was separated. The organic layer was washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (NH silica gel, eluted with 0–30% EtOAc in hexane) to give the title compound (6.8 g, 23 mmol, 68%) as a white powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.06–0.16 (1H, m), 0.16–0.26 (1H, m), 0.33–0.48 (2H, m), 0.89–1.12 (3H, m), 1.36 (9H, s), 1.85–1.95 (1H, m), 2.53–2.60 (1H, m), 2.97 (1H, dd, <i>J</i> = 14.2, 6.8 Hz), 3.15 (1H, dd, <i>J</i> = 14.2, 6.8 Hz), 4.83 (2H, s), 6.46 (2H, d, <i>J</i> = 7.9 Hz), 6.80 (2H, d, <i>J</i> = 7.9 Hz). MS (ESI/APCI) <i>m</i>/<i>z</i> 203.0 [M + H-Boc]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>rac</i>-<i>tert</i>-Butyl ((1<i>R</i>,2<i>S</i>)-2-(4-Benzamidophenyl)cyclopropyl)(cyclopropylmethyl)carbamate (<b>7</b>)</h4><div class="NLM_p last">To a solution of <b>6</b> (92 mg, 0.30 mmol) and Et<sub>3</sub>N (51 μL, 0.36 mmol) in THF (1.5 mL) was added benzoyl chloride (42 mL, 0.36 mmol). After being stirred at room temperature overnight, the mixture was quenched with saturated NH4Cl aqueous solution and extracted with EtOAc. The organic layer was separated, washed with saturated NaHCO<sub>3</sub> aqueous solution and brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give the title compound (120 mg, 0.30 mmol, 99%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.06–0.17 (1H, m), 0.18–0.28 (1H, m), 0.33–0.50 (2H, m), 0.89–1.03 (1H, m), 1.12–1.29 (2H, m), 1.37 (9H, s), 2.01–2.11 (1H, m), 2.64–2.77 (1H, m), 2.91–3.07 (1H, m), 3.14–3.26 (1H, m), 7.12 (2H, d, <i>J</i> = 8.6 Hz), 7.47–7.61 (3H, m), 7.67 (2H, d, <i>J</i> = 8.6 Hz), 7.91–7.98 (2H, m), 10.18 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 405.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>rac</i>-<i>N</i>-(4-((1<i>S</i>,2<i>R</i>)-2-((Cyclopropylmethyl)amino)cyclopropyl)phenyl)benzamide Hydrogen Chloride (<b>1a</b>)</h4><div class="NLM_p last">A mixture of <b>7</b> (123 mg, 0.30 mmol) and 4 M HCl/CPME solution (1.6 mL, 6.40 mmol) was stirred at room temperature for 3 h. The mixture was concentrated <i>in vacuo</i>, and the residue was purified by recrystallization from MeOH/IPE to give the title compound (79 mg, 0.231 mmol, 77%) as a colorless powder. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.22–0.38 (2H, m), 0.46–0.63 (2H, m), 0.90–1.11 (1H, m), 1.13–1.30 (1H, m), 1.31–1.49 (1H, m), 2.28–2.46 (1H, m), 2.78–2.97 (3H, m), 7.15 (2H, d, <i>J</i> = 8.7 Hz), 7.47–7.63 (3H, m), 7.64–7.75 (2H, m), 7.90–7.98 (2H, m), 10.22 (1H, br s). mp 193–200 °C. MS (ESI/APCI) <i>m</i>/<i>z</i> 307.2 [M + H]<sup>+</sup>. HPLC purity: 99.9%. Anal. Calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>2</sub>O: C, 70.06; H, 6.76; N, 8.17. Found: C, 69.96; H, 6.88; N, 8.21.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>rac</i>-Methyl 4-((1<i>R</i>,2<i>R</i>)2-(<i>tert</i>-Butoxycarbonyl)cyclopropyl)benzoate (<b>9</b>)</h4><div class="NLM_p last">To a solution of trimethylsulfoxonium iodide (35.1 g, 1560 mmol) in DMSO (600 mL) was added KO<i>t</i>Bu (16.4 g, 146 mmol) at room temperature. After being stirred at the same temperature for 30 min, a solution of <b>8</b> (34.9 g, 133 mmol) was added to the reaction mixture. The mixture was stirred at room temperature for 2 days. The mixture was quenched with water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 2–10% EtOAc in hexane) to give the title compound (20.8 g, 75 mmol, 57%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.32–1.53 (2H, m), 1.90–1.98 (1H, m), 2.40–2.47 (1H, m), 3.83 (3H, s), 7.28–7.35 (2H, m), 7.81–7.88 (2H, m). MS (ESI/APCI) <i>m</i>/<i>z</i> 219.1 [M – H – <i>t</i>Bu]<sup>−</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>rac</i>-(1<i>R</i>,2<i>R</i>)-2-(4-(Methoxycarbonyl)phenyl)cyclopropane-1-carboxylic Acid (<b>10</b>)</h4><div class="NLM_p last">TFA (58.0 mL, 753 mmol) was added to <b>9</b> (20.8 g, 75.3 mmol) at 0 °C. The mixture was stirred at room temperature under a dry atmosphere for 2 h. The mixture was concentrated <i>in vacuo</i>. The mixture was poured into water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give the title compound (14.5 g, 65.8 mmol, 87%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.36–1.54 (2H, m), 1.87–1.95 (1H, m), 2.43–2.49 (1H, m), 3.83 (3H, s), 7.28–7.35 (2H, m), 7.81–7.91 (2H, m), 12.39 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 219.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>rac</i>-Methyl 4-((1<i>S</i>,2<i>R</i>)-2-((<i>tert</i>-Butoxycarbonyl)amino)cyclopropyl)benzoate (<b>11</b>)</h4><div class="NLM_p last">To a mixture of <b>10</b> (14.5g, 65.8 mmol) in toluene (dry) (150 mL) and <i>tert</i>-butyl alcohol (150 mL, 1590 mmol) were added DPPA (21.2 mL, 98.8 mmol) and TEA (13.8 mL, 98.8 mmol) in one portion at 25 °C under nitrogen atmosphere. The mixture was stirred at 90 °C for 16 h. The mixture was concentrated in reduced pressure at 45 °C. The residue was purified by column chromatography (silica gel, eluted with 5–20% EtOAc in hexane) to give the title compound (16.4 g, 56.3 mmol, 85%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.14–1.26 (2H, m), 1.37 (9H, s), 1.91–2.04 (1H, m), 2.62–2.78 (1H, m), 3.83 (3H, s), 7.18–7.25 (2H, m), 7.26–7.36 (1H, m), 7.80–7.88 (2H, m).</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>rac</i>-4-((1<i>S</i>,2<i>R</i>)-2-((<i>tert</i>-Butoxycarbonyl)amino)cyclopropyl)benzoic Acid (<b>13a</b>)</h4><div class="NLM_p last">8 M NaOH aqueous solution (0.686 mL, 5.49 mmol) was added to a mixture of <b>11</b> (160 mg, 0.55 mmol) in MeOH (5 mL) at room temperature. The mixture was stirred at room temperature for 1 day. The reaction mixture was partitioned between EtOAc and water, and then, the aqueous layer was acidified with 6 M HCl aqueous solution to pH = 2–3. The mixture was extracted with EtOAc. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give the title compound (144 mg, 0.520 mmol, 95%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ1.07–1.26 (2H, m), 1.37 (9H, s), 1.92–2.04 (1H, m), 2.61–2.72 (1H, m), 7.18 (2H, d, <i>J</i> = 8.3 Hz), 7.28 (1H, br s), 7.81 (2H, d, <i>J</i> = 8.3 Hz), 12.72 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 276.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>rac</i>-<i>tert</i>-Butyl ((1<i>R</i>,2<i>S</i>)-2-(4-(Phenylcarbamoyl)phenyl)cyclopropyl)carbamate (<b>14a</b>)</h4><div class="NLM_p last">To a mixture of <b>13a</b> (300 mg, 1.08 mmol), EDCI (250 mg, 1.30 mmol) and HOBt (176 mg, 1.30 mmol) in anhydrous DMF (5 mL) were added aniline (121 mg, 1.30 mmol) and TEA (218 mg, 2.16 mmol) at 24 °C. The resulting mixture was stirred at 24 °C for 4 h. The reaction mixture was partitioned between EtOAc and water. The organic layer was separated, and the aqueous layer was extracted with EtOAc. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated to dryness under reduced pressure. The residue was purified by column chromatography (silica gel, eluted with 2–50% EtOAc in PE) and then washed with PE/EtOAc (40 mL/2 mL) to afford the title compound (233 mg, 0.66 mmol, 61%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.12–1.22 (2H, m), 1.36 (9H, s), 1.90–2.02 (1H, m), 2.65–2.74 (1H, m), 7.07 (1H, t, <i>J</i> = 7.2 Hz), 7.20 (2H, d, <i>J</i> = 8.4 Hz), 7.24–7.37 (3H, m), 7.74 (2H, d, <i>J</i> = 8.0 Hz), 7.84 (2H, d, <i>J</i> = 8.0 Hz), 10.12 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 353.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>rac</i>-4-((1<i>S</i>,2<i>R</i>)-2-Aminocyclopropyl)-<i>N</i>-phenylbenzamide Hydrochloride (<b>15a</b>)</h4><div class="NLM_p last"><b>14a</b> (100 mg, 0.28 mmol) was suspended in 10% HCl/MeOH solution (10.0 mL, 24.1 mmol) at room temperature, and the mixture was stirred at room temperature for 7 h. The reaction mixture was concentrated <i>in vacuo</i> and dried <i>in vacuo</i> to give the title compound (83 mg, 0.286 mmol, quant) as a white powder. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD): δ1.14–1.93 (2H, m), 2.49 (1H, br s), 2.95 (1H, br s), 6.80–8.26 (9H, m). MS (ESI/APCI) <i>m</i>/<i>z</i> 253.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>rac</i>-4-((1<i>S</i>,2<i>R</i>)-2-((Cyclopropylmethyl)amino)cyclopropyl)-<i>N</i>-phenylbenzamide Hydrochloride (<b>1b</b>)</h4><div class="NLM_p last">To a solution of <b>15a</b> (61.9 mg, 0.21 mmol) and NaHCO<sub>3</sub> (36.0 mg, 0.43 mmol) in THF (536 μL)/MeOH (536 μL) was added cyclopropanecarbaldehyde (20.8 μL, 0.28 mmol). After being stirred at 60 °C for 2 h, the reaction mixture was cooled to 0 °C, and NaBH<sub>4</sub> (16.2 mg, 0.43 mmol) was added portionwise. After being stirred at room temperature for 1 h, the mixture was poured into saturated NaHCO<sub>3</sub> aqueous solution and extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (NH silica gel, eluted with 50–100% EtOAc in hexane). The combined fraction was acidified with 4 M HCl/CPME (536 μL, 2.14 mmol) and concentrated <i>in vacuo</i>. The residue was recrystallized from MeOH/IPE to give the title compound (24.0 mg, 0.070 mmol, 33%) as colorless crystals. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ0.28–0.41 (2H, m), 0.51–0.63 (2H, m), 0.97–1.12 (1H, m), 1.28–1.45 (1H, m), 1.46–1.67 (1H, m), 2.52–2.62 (1H, m), 2.87–3.09 (3H, m), 7.02–7.14 (1H, m), 7.27–7.40 (4H, m), 7.77 (2H, d, <i>J</i> = 8.2 Hz), 7.91 (2H, d, <i>J</i> = 8.2 Hz), 9.14 (2H, br s), 10.19 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 307.3 [M + H]<sup>+</sup>. HPLC purity: 97.3%. Anal. Calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>2</sub>O: C, 69.7; H, 6.78; N, 8.13. Found: C, 69.70; H, 6.81; N, 8.03.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Methyl 4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-((<i>tert</i>-Butoxycarbonyl)amino)cyclopropyl)benzoate (<b>12</b>)</h4><div class="NLM_p last"><b>11</b> (3.6 g, 12.4 mmol) was purified by preparative HPLC (column: CHIRALPAK AD, 50 mmID × 500 mmL, mobile phase: hexane/ethanol = 900/100, pressure: 0.2 MPa, flow rate: 80 mL/min, temperature: 30 °C, detection: UV 220 nm), and the obtained fraction was concentrated under reduced pressure to give the title compound (1.25 g, 12.4 mmol, 35% yield, > 99% ee) having a longer retention time. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.09–1.26 (2H, m), 1.37 (9H, s), 1.92–2.03 (1H, m), 2.64–2.78 (1H, m), 3.83 (3H, s), 7.21 (2H, d, <i>J</i> = 8.3 Hz), 7.27–7.35 (1H, m), 7.84 (2H, d, <i>J</i> = 8.3 Hz).</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-((<i>tert</i>-Butoxycarbonyl)amino)cyclopropyl)benzoic Acid (<b>13b</b>)</h4><div class="NLM_p last">NaOH (7.55 mL, 15.1 mmol) was added to a solution of <b>12</b> (1.1g, 3.78 mmol) in MeOH (11 mL, 3.78 mmol)/THF (11 mL, 3.78 mmol) at room temperature. The mixture was stirred at room temperature for 3 h. The mixture was neutralized with 1 M HCl aqueous solution at 0 °C and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i> to give the title compound (1.00 g, 3.61 mmol, 96%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.09–1.26 (2H, m), 1.37 (9H, s), 1.90–2.03 (1H, m), 2.62–2.77 (1H, m), 7.18 (2H, d, <i>J</i> = 8.3 Hz), 7.23–7.34 (1H, m), 7.81(2H, d, <i>J</i> = 8.3 Hz), 12.77 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 276.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>tert</i>-Butyl ((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-(4-((3-Cyano-4-fluorophenyl)carbamoyl)phenyl)cyclopropyl)carbamate (<b>14b</b>)</h4><div class="NLM_p last">HATU (329 mg, 0.87 mmol) was added to a mixture of <b>13</b> (200 mg, 0.72 mmol), <i>m</i>-anisidine (118 mg, 0.87 mmol), and TEA (0.302 mL, 2.16 mmol) in DMF(dry) (4.0 mL) at room temperature. The mixture was stirred at room temperature under a dry atmosphere for 18 h. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give the title compound (190 mg, 0.48 mmol, 67%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 1.20–1.31 (2H, m), 1.46 (9H, s), 2.01–2.25 (1H, m), 2.78 (1H, br s), 4.85 (1H, br s), 7.23 (2H, s), 7.75 (2H, d, <i>J</i> = 8.3 Hz), 7.84(2H, t, <i>J</i> = 3.8 Hz), 8.02 (1H, dd, <i>J</i> = 5.3, 2.6 Hz). MS (ESI/APCI) <i>m</i>/<i>z</i> 394.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-Aminocyclopropyl)-<i>N</i>-(3-Cyano-4-fluorophenyl)benzamide Hydrochloride (<b>15b</b>)</h4><div class="NLM_p last">The mixture of <b>14b</b> (190 mg, 0.48 mmol) and 4 M HCl/EtOAc solution (4 mL, 16.0 mmol) was stirred at room temperature under a dry atmosphere overnight. The precipitate was collected by filtration to give the title compound (160 mg, 0.482 mmol, quant). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.24–1.40 (1H, m), 1.42–1.55 (1H, m), 2.36–2.48 (1H, m), 2.83–2.99 (1H, m), 7.34 (2H, d, <i>J</i> = 8.3 Hz), 7.55 (1H, t, <i>J</i> = 9.0Hz), 7.94 (2H, d, <i>J</i> = 8.3 Hz), 8.10 (1H, ddd, <i>J</i> = 9.3, 5.0, 2.6 Hz), 8.30 (1H, dd, <i>J</i> = 5.7, 2.6 Hz), 8.25–8.36 (1H, m), 8.50 (3H, d, <i>J</i> = 3.0 Hz), 10.60 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 295.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>N</i>-(3-Cyano-4-fluorophenyl)-4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-((Cyclopropylmethyl)amino)cyclopropyl)benzamide Hydrochloride (<b>1c</b>)</h4><div class="NLM_p last">Cyclopropanecarboxaldehyde (0.014 mL, 0.18 mmol) was added to a solution of <b>15b</b> (50 mg, 0.15 mmol) and TEA (0.084 mL, 0.60 mmol) in MeOH (1 mL))/THF (1 mL) at room temperature. The mixture was stirred at 60 °C under a dry atmosphere for 2 h. NaBH<sub>4</sub> (6.84 mg, 0.18 mmol) was added to the reaction mixture at 0 °C. The mixture was stirred at 0 °C under a dry atmosphere for 10 min. The mixture was poured into saturated NaHCO<sub>3</sub> aqueous solution at 0 °C, and MeOH and THF were removed <i>in vacuo</i>. The obtained mixture was extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The obtained residue was purified by column chromatography (NH silica gel, eluted with 50–100% EtOAc in Hexane). The desired fraction was acidified with 4 M HCl/EtOAc solution, and the whole was concentrated <i>in vacuo</i> to give the title compound (20.0 mg, 0.052 mmol, 34%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.28–0.43 (2H, m), 0.52–0.65 (2H, m), 1.05 (1H, t, <i>J</i> = 7.2 Hz), 1.34–1.46 (1H, m), 1.52–1.66 (1H, m), 2.56 (1H, br s), 2.91–3.12 (3H, m), 7.36 (2H, d, <i>J</i> = 8.3 Hz), 7.56 (1H, t, <i>J</i> = 9.0 Hz), 7.93 (2H, d, <i>J</i> = 8.3 Hz), 8.09 (1H, ddd, <i>J</i> = 9.3, 5.0, 2.6 Hz), 8.29 (1H, dd, <i>J</i> = 5.7, 2.6 Hz), 9.15 (2H, br s), 10.56 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 350.1 [M + H]<sup>+</sup>. [α]<sub class="stack">D</sub><sup class="stack">20</sup> −67.2 (<i>c</i> 0.5, MeOH). HPLC purity: 100%.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-((Cyclopropylmethyl)amino)cyclopropyl)-<i>N</i>-(6-fluoropyridin-3-yl)benzamide (<b>1d</b>)</h4><div class="NLM_p last">HATU (293 mg, 0.77 mmol) was added to a mixture of <b>13</b> (178 mg, 0.64 mmol), 6-fluoropyridin-3-amine (86 mg, 0.77 mmol) and TEA (0.268 mL, 1.93 mmol) in DMF (dry) (2.0 mL) at room temperature. The mixture was stirred at room temperature under a dry atmosphere for 18 h. The mixture was quenched with water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i> to give <b>14c</b> (230 mg, 0.619 mmol, 96%). The mixture of <b>14c</b> (230 mg, 0.62 mmol) and 4 M HCl/EtOAc solution (4 mL, 16.0 mmol) was stirred at room temperature under N<sub>2</sub> overnight. The precipitate was collected by filtration to give <b>15c</b> (210 mg, 0.610 mmol, 99%). Cyclopropanecarboxaldehyde (0.055 mL, 0.73 mmol) was added to a solution of <b>15c</b> (210 mg, 0.61 mmol) and TEA (0.340 mL, 2.44 mmol) in MeOH (2 mL) at room temperature. The mixture was stirred at 50 °C under a dry atmosphere for 2 h. NaBH<sub>4</sub> (27.7 mg, 0.73 mmol) was added to the reaction mixture at 0 °C. The mixture was stirred at 0 °C under a dry atmosphere for 10 min. The mixture was poured into saturated NaHCO<sub>3</sub> aqueous solution at 0 °C and extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 0–50% MeOH in EtOAc). The desired fraction was acidified with 4 M HCl/EtOAc solution, and the whole was concentrated <i>in vacuo</i>. The solid was crystallized from EtOH–heptane to give the title compound (70.0 mg, 0.193 mmol, 32%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.36 (2H, d, <i>J</i> = 4.5 Hz), 0.52–0.63 (2H, m), 0.98–1.14 (1H, m), 1.34–1.46 (1H, m), 1.54–1.66 (1H, m), 2.53–2.63 (1H, m), 2.91–3.13 (3H, m), 7.21 (1H, dd, <i>J</i> = 9.0, 3.4 Hz), 7.36 (2H, d, <i>J</i> = 8.7 Hz), 7.94 (2H, d, <i>J</i> = 8.3 Hz), 8.28–8.38 (1H, m), 8.59 (1H, s), 9.16 (2H, br s), 10.51 (1H, s). MS (ESI/APCI) <i>m</i>/<i>z</i> 326.1 [M + H]<sup>+</sup>. [α]<sub class="stack">D</sub><sup class="stack">20</sup> −77.2 (<i>c</i> 0.5, MeOH). HPLC purity: 100%.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> <i>rac-tert</i>-Butyl ((1<i>S</i>,2<i>R</i>)-2-(4-Bromophenyl)cyclopropyl)(cyclopropylmethyl)carbamate (<b>17</b>)</h4><div class="NLM_p last">NaH (243 mg, 6.09 mmol) was added to a mixture of <b>16</b> (950 mg, 3.04 mmol) in DMF (15 mL) at 0 °C. The mixture was stirred at 0 °C for 10 min. (Bromomethyl)cyclopropane (0.443 mL, 4.56 mmol) in DMF (1 mL) was added dropwise to the mixture at 0 °C, and the whole was stirred at 0 °C to room temperature for 2.5 h. The mixture was quenched with water at 0 °C and extracted with EtOAc. The organic layer was separated, washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 1–20% EtOAc in hexane) to give the title compound (970 mg, 2.65 mmol, 87%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 0.09–0.31 (2H, m), 0.36–0.54 (2H, m), 0.92–1.09 (1H, m), 1.13–1.21 (1H, m), 1.23–1.31 (1H, m), 1.39 (9H, s), 2.00–2.12 (1H, m), 2.75–2.84 (1H, m), 3.02 (1H, dd, <i>J</i> = 14.4, 6.8 Hz), 3.26 (1H, dd, <i>J</i> = 14.4, 6.8 Hz), 6.96–7.06 (2H, m), 7.33–7.44 (2H, m). MS (ESI/APCI) <i>m</i>/<i>z</i> 310.1 [M – <i>t</i>Bu + H]<sup>+</sup>.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>rac</i>-Methyl 4-((1<i>S</i>,2<i>R</i>)-2-((<i>tert</i>-Butoxycarbonyl)(cyclopropylmethyl)amino)cyclopropyl)benzoate (<b>18</b>)</h4><div class="NLM_p last">TEA (0.444 mL, 3.18 mmol) was added to a mixture of <b>17</b> (1060 mg, 2.89 mmol) and PdCl<sub>2</sub>(dppf) (212 mg, 0.29 mmol) in MeOH (30 mL). The mixture was stirred at 90 °C under CO pressure (3 atm) for 2 days. The insoluble material was removed by filtration through a celite pad, and the filtrate was concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 1–25% EtOAc in hexane) to give the title compound (875 mg, 2.53 mmol, 87%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 0.08–0.18 (1H, m), 0.19–0.29 (1H, m), 0.37–0.54 (2H, m), 0.93–1.07 (1H, m), 1.29–1.39 (2H, m), 1.41 (9H, s), 2.08–2.19 (1H, m), 2.83–2.93 (1H, m), 3.05 (1H, dd, <i>J</i> = 14.2, 7.0 Hz), 3.26 (1H, dd, <i>J</i> = 14.2, 7.0 Hz), 3.90 (3H, s), 7.10–7.19 (2H, m), 7.90–7.98 (2H, m). MS (ESI/APCI) <i>m</i>/<i>z</i> 290.2 [M – <i>t</i>Bu + H]<sup>+</sup>.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>rac</i>-4-((1<i>S</i>,2<i>R</i>)-2-((<i>tert</i>-Butoxycarbonyl)(cyclopropylmethyl)amino)cyclopropyl)benzoic Acid (<b>19</b>)</h4><div class="NLM_p last">2 M NaOH aqueous solution (75 mL, 151 mmol) was added to a mixture of <b>18</b> (13.0 g, 37.6 mmol) in MeOH (75 mL) at room temperature. The mixture was stirred at room temperature overnight. The reaction mixture was concentrated <i>in vacuo</i>, and the aqueous layer was acidified with 2 M HCl aqueous solution to pH = 4–5. The mixture was extracted with EtOAc. The combined organic phase was washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give the title compound (11.2 g, 33.8 mmol, 90%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.05–0.15 (1H, m), 0.18–0.28 (1H, m), 0.33–0.51 (2H, m), 0.89–1.05 (1H, m), 1.25–1.39 (11H, m), 2.09–2.23 (1H, m), 2.77–2.85 (1H, m), 2.92–3.03 (1H, m), 3.16–3.27 (1H, m), 7.19–7.29 (2H, m), 7.78–7.88 (2H, m), 12.60 (1H, br s). MS (ESI/APCI) <i>m</i>/<i>z</i> 330.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-((1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>))-2-((<i>tert-</i>Butoxycarbonyl)(cyclopropylmethyl)amino)cyclopropyl)benzoic Acid (<b>20</b>)</h4><div class="NLM_p last"><b>19</b> (5.00 g, 17.8 mmol) was fractionated by SFC (column: CHIRALPAK ADH, 20 mmID × 250 mmL, 5 μm, mobile phase: CO<sub>2</sub>/MeOH = 900/100, pressure: 100 bar, flow rate: 75 mL/min, temperature: 35 °C, detection: UV 220 nm), and the obtained fraction was concentrated under reduced pressure to give the title compound (5.2 g, 15.7 mmol, 46% yield, > 99% ee) having a long retention time.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>tert</i>-Butyl (Cyclopropylmethyl)[(1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>)]-2-{4-[(6-Fluoropyridin-3-yl)(methyl)carbamoyl]phenyl}cyclopropyl]carbamate (<b>21</b>)</h4><div class="NLM_p last">To a solution of <b>20</b> (1.05 g, 3.17 mmol) in THF (40 mL) were added (COCl)<sub>2</sub> (0.545 mL, 6.34 mmol) and DMF (1 drop) at 0 °C. The mixture was stirred at 0 °C under a dry atmosphere for 1 h. The mixture was concentrated <i>in vacuo</i>. To a solution of 6-fluoro-<i>N</i>-methylpyridin-3-amine (0.480 g, 3.80 mmol) and DMAP (0.194 g, 1.58 mmol) in pyridine (15 mL) was added a solution of acid chloride derivative in THF (1 mL) at room temperature. The mixture was stirred at room temperature for 10 min. The mixture was neutralized with aqueous saturated NaHCO<sub>3</sub> solution and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 30–50% EtOAc in hexane) to give the title compound (1.39 g, 3.16 mmol, quant). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.02–0.11 (1H, m), 0.15–0.26 (1H, m), 0.30–0.49 (2H, m), 0.87–1.00 (1H, m), 1.12–1.37 (11H, m), 1.94–2.05 (1H, m), 2.64–2.75 (1H, m), 2.85–2.97 (1H, m), 3.11–3.25 (1H, m), 3.36 (3H, s), 6.96–7.06 (2H, m), 7.14–7.20 (3H, m), 7.85–7.94 (1H, m), 7.98–8.05 (1H, m). MS (ESI/APCI) <i>m</i>/<i>z</i> 330.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-{[(1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>)]-2-[(Cyclopropylmethyl)amino]cyclopropyl}-<i>N</i>-(6-fluoropyridin-3-yl)-<i>N</i>-methylbenzamide Hydrogen Chloride (<b>1e</b>)</h4><div class="NLM_p last">The mixture of <b>21</b> (1.39 g, 3.16 mmol) and 4 M HCl/EtOAc solution (6.33 mL, 25.3 mmol) in EtOAc (8 mL) was stirred at room temperature under a dry atmosphere for 3 h. The mixture was concentrated <i>in vacuo</i>. The precipitate was recrystallized from EtOH–IPE to give the title compound (0.900 g, 2.39 mmol, 76%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.25–0.36 (2H, m), 0.48–0.59 (2H, m), 0.91–1.08 (1H, m), 1.20–1.32 (1H, m), 1.43–1.55 (1H, m), 2.34–2.45 (1H, m), 2.85–2.99 (3H, m), 3.36 (3H, s), 7.03–7.12 (2H, m), 7.13–7.26 (3H, m), 7.90–7.99 (1H, m), 8.00–8.05 (1H, m), 9.07 (2H, br s). <sup>13</sup>C NMR (101MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.83, 162.02, 159.67, 146.49, 141.34, 139.99, 134.10, 128.94, 126.29, 110.46, 52.15, 38.33, 37.80, 20.83, 13.52, 7.38, 4.51. MS (ESI/APCI) <i>m</i>/<i>z</i> 340.3 [M + H]<sup>+</sup>. [α]<sub class="stack">D</sub><sup class="stack">20</sup> −58.8 (<i>c</i> 0.5, MeOH). HPLC purity: 100%. Anal. Calcd for C<sub>20</sub>H<sub>22</sub>FNO HCl: C, 63.91; H, 6.17; N, 11.18. Found: C, 63.88; H, 6.13; N, 11.23.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>tert</i>-Butyl (Cyclopropylmethyl)((1<i>R</i>,2<i>S</i>) or (1<i>S</i>,2<i>R</i>))-2-(4-((6-Nitropyridin-3-yl)carbamoyl)phenyl)cyclopropyl)carbamate (<b>22</b>)</h4><div class="NLM_p last">To a solution of <b>20</b> (150 mg, 0.45 mmol) in THF (5 mL) were added (COCl)<sub>2</sub> (0.078 mL, 0.91 mmol) and DMF (1 drop) at 0 °C. The mixture was stirred at room temperature under a dry atmosphere for 60 min. The mixture was concentrated <i>in vacuo</i>. To a solution of 6-nitropyridin-3-amine (94 mg, 0.68 mmol) and DMAP (27.6 mg, 0.23 mmol) in pyridine (1 mL) was added a solution of acid chloride derivative in THF (1 mL) at 60 °C. The mixture was stirred at 60 °C for 2 h. The mixture was neutralized with saturated NaHCO<sub>3</sub> aqueous solution and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 10–50% EtOAc in hexane) to give the title compound (68.0 mg, 0.150 mmol, 33%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 0.10–0.32 (2H, m), 0.38–0.59 (2H, m), 1.02 (1H, br s), 1.30 (1H, d, <i>J</i> = 6.0 Hz), 1.36–1.49 (10H, m), 2.18 (1H, br s), 2.91 (1H, br s), 2.99–3.12 (1H, m), 3.29 (1H, dd, <i>J</i> = 14.3, 6.8 Hz), 7.22–7.26 (2H, m), 7.81 (2H, d, <i>J</i> = 7.9 Hz), 8.23 (1H, br s), 8.34 (1H, d, <i>J</i> = 9.0 Hz), 8.67 (1H, d, <i>J</i> = 2.6 Hz), 8.73 (1H, d, <i>J</i> = 8.7 Hz). MS (ESI/APCI) <i>m</i>/<i>z</i> 397.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>tert</i>-Butyl (Cyclopropylmethyl)((1<i>R</i>,2<i>S</i>) or (1<i>S</i>,2<i>R</i>))-2-(4-(Methyl(6-nitropyridin-3-yl)carbamoyl)phenyl)cyclopropyl)carbamate (<b>23</b>)</h4><div class="NLM_p last">MeI (0.041 mL, 0.66 mmol) was added to a solution of <b>22</b> (60 mg, 0.13 mmol) and K<sub>2</sub>CO<sub>3</sub> (36.7 mg, 0.27 mmol) in DMF(dry) (1 mL) at room temperature. The mixture was stirred at room temperature under a dry atmosphere for 2 h. The mixture was quenched with saturated NH<sub>4</sub>Cl aqueous solution at room temperature and extracted with EtOAc. The organic layer was separated, washed with brine, dried over MgSO<sub>4</sub> and concentrated <i>in vacuo</i>. The residue was purified by column chromatography (silica gel, eluted with 40–60% EtOAc in hexane) to give the title compound (55.0 mg, 0.118 mmol, 89%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 0.02–0.13 (1H, m), 0.14–0.24 (1H, m), 0.29–0.49 (2H, m), 0.84–1.01 (1H, m), 1.21 (2H, br s), 1.27 (9H, s), 1.98–2.08 (1H, m), 2.64–2.76 (1H, m), 2.86–2.99 (1H, m), 3.12–3.25 (1H, m), 3.46 (3H, s), 7.07 (2H, d, <i>J</i> = 8.3 Hz), 7.25 (2H, d, <i>J</i> = 8.3 Hz), 8.05 (1H, dd, <i>J</i> = 8.9, 2.6 Hz), 8.25 (1H, d, <i>J</i> = 8.9 Hz), 8.40 (1H, d, <i>J</i> = 2.5 Hz). MS (ESI/APCI) <i>m</i>/<i>z</i> 411.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-{[(1<i>S</i>,2<i>R</i>) or (1<i>R</i>,2<i>S</i>)]-2-[(Cyclopropylmethyl)amino]cyclopropyl}-<i>N</i>-(6-[<sup>18</sup>F]fluoropyridin-3-yl)-<i>N</i>-methylbenzamide Hydrogen Chloride ([<sup>18</sup>F]<b>1e</b>)</h4><div class="NLM_p">[<sup>18</sup>F]Fluoride, as a solution in O-18 water, was purchased from Cardinal Health Nuclear Pharmacy, Location no. 869 (Hartford, CT). Pre-conditioned Waters QMA cartridges and Kryptofix 2.2.2 were obtained from ABX (Radeberg, Germany). Potassium carbonate, anhydrous DMSO, DMF, acetonitrile, and <i>tert</i>-Amyl alcohol were purchased from Sigma-Aldrich. Ethanol was purchased from Pharmco (Brookfield, CT). HPLC solvents were obtained from VWR. Solid phase extraction cartridges (Sep-Pak) were acquired from Waters. Sterile vials, sterile water for injection, and sterile 0.9% sodium chloride injection were purchased from Hospira (Lake Forest, IL). Ascorbic acid injection (Ascor L 500) was obtained from McGuff Pharmaceuticals, Inc (Santa Ana, CA). Ultra-high purity helium and air were acquired from ABCO (Waterford, CT).</div><div class="NLM_p">Radiochemical experiments were performed using a GE TRACERlab FXF-N synthesizer module. Radioactivity was measured in a calibrated Capintec CRC-15R dose calibrator. Analytical HPLC was performed using a Waters 600 pump with a Waters 2487 UV detector and a Bioscan flow-count unit with a B-FC-3200 probe. Analytical chromatograms were recorded and processed using Waters Empower software.</div><div class="NLM_p last">The following sequence for the TRACERlab FxN, previously optimized by Invicro, was used for the radiosynthesis. [<sup>18</sup>F]Fluoride was trapped on a pre-conditioned Waters QMA cartridge before elution from the QMA cartridge into reaction vessel 1 (RV1) with K222 and K<sub>2</sub>CO<sub>3</sub> (10 mg/1.2 mg) in a mixture of acetonitrile/water (4/1, 1 mL). The contents of RV1 were evaporated at 95 °C for 4 min with helium flow under vacuum before acetonitrile (1 mL) was added to RV1 and evaporated at 95 °C for 2 min with helium flow under vacuum. This azeotrope was repeated once more. A solution of the precursor <b>23</b> (1 mg) in DMSO (1 mL) was added to RV1 and heated to 160 °C for 10 min, after which it was cooled to 90 °C, and 1 M hydrochloric acid (0.5 mL) was added and heated for a further period of 4 min. On completion of the radiolabeling and deprotections, the reactor contents were cooled to 40 °C and diluted with the HPLC mobile phase (2 mL) and water (2 mL). The resulting mixture was purified by semi-preparative HPLC using a Phenomenex Luna C18(2) column (250 × 9.4 mm), eluting with a mixture of acetonitrile/water/triethylamine (45/55/0.1 v/v/v) at 4 mL/min. The product peak was collected and diluted in water for injection (WFI, 15 mL), and the diluted product fraction was loaded onto a Waters tC18 light solid phase extraction (SPE) cartridge (Sep-Pak). The SPE cartridge was rinsed with WFI (10 mL), before the product was eluted with ethanol (1 mL) into the formulation flask. The SPE cartridge was rinsed with 4 mL of formulation, and the rinse was also collected into the formulation flask. The resulting formulated product solution was filtered through a 0.2 μm sterilizing filter into the final product vial. The final product formulation consisted of a mixture of ethanol (0.05 mL/mL) and ascorbic acid (0.5 mg/mL) in normal saline (20 mL total volume).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>In Vitro</i> LSD1 Demethylase Assay</h3><div class="NLM_p last">An HTRF detection system (Cisbio) was used to estimate <i>in vitro</i> LSD1 demethylation activity. The reaction buffer consisted of 50 mM Tris–HCl (pH 8.0), 1 mM DTT, and 0.01% BSA. The demethylation reaction was started by adding 1 μM biotinylated mono-methylated Lys 4 histone H3 peptide [ART(mK)QTARKSTGGKAPRKQLAGGK-Biotin] (Scrum Inc.) to the enzyme (25 nM recombinant LSD1/CoREST) and then terminated by adding 1 mM <i>trans</i>-2-phenylcyclopropylamine after 15 min. The demethylated peptide was detected by the cryptate-labeled anti-histone H3 antibody and streptavidin-XLent! (Cisbio) diluted with the detection buffer consisting of 800 mM KF and 0.1% BSA. After incubation at room temperature for 1 h, the plates were read using an EnVision 2102 multilabel reader (PerkinElmer). The total reaction was set as 0% inhibitory activity, and the reaction without the enzyme was set as 100% inhibitory activity. Curve fittings and calculations of IC<sub>50</sub> values with 95% CI were performed with the program XLfit ver. 5 (ID Business Solutions Ltd.), with the maximum and minimum of the curve constrained to 100 and 0, respectively.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Measurement of log <i>D</i><sub>7.4</sub></h3><div class="NLM_p last">log D<sub>7.4</sub>, which is a partition coefficient between 1-octanol and aqueous buffer, pH 7.4, of the compounds was measured on the chromatographic procedure whose condition was developed based on a published method.<a onclick="showRef(event, 'ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref32">(30,32)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Measurement of MDR1 Membrane Permeability</h3><div class="NLM_p last">Human MDR1-expressing LLC-PK1 cells were cultured, and the transcellular transport study was performed with minor modifications to the method reported previously.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> In brief, the cells were grown in a Transwell 96-well permeable support (pore size 0.4 μm, 0.143 cm<sup>2</sup> surface area) with a polycarbonate membrane (Corning Life Sciences, Lowell, MA, USA). The cells were preincubated with M199 at 37 °C. Subsequently, transcellular transport was initiated by the addition of M199 either to apical compartments (75 μL) or to basolateral compartments (250 μL) containing 1 μM test compounds. The assay was terminated by the removal of each assay plate after 1 h. Aliquots (25 μL) from the opposite compartments were mixed with acetonitrile and then centrifuged. The compound concentrations in the supernatant were measured by LC–MS/MS. The apparent permeability (<i>P</i><sub>app</sub>) of test compounds in the receiver wells was determined, and the efflux ratio (ER) for the MDR1 membrane permeability test was calculated using the following equation<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_m001.gif" alt="" id="_i52" /></img></span>where <i>P</i><sub>app,AtoB</sub> and <i>P</i><sub>app,BtoA</sub> represent the apparent permeability in the apical-to-basal direction and the basal-to-apical direction, respectively.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Unbound Fraction in Rat Plasma</h3><div class="NLM_p last">The plasma binding of each compound was determined by the equilibrium dialysis method with HTDialysis Teflon dialysis chambers and cellulose membranes (MWCO 6–8 kDa). The plasma was mixed with the compound solution for a final concentration of 1 μM. Dialysis was conducted against phosphate buffered saline (PBS) at 37 °C for 16–20 h. The concentrations of compounds in both biological sample and PBS sides were determined by liquid chromatography/tandem mass spectrometry (LC–MS/MS). The unbound fraction in the biological sample (Fu) was calculated as the ratio of the peak area of compounds from the PBS side to those from the plasma side of the dialysis apparatus.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <i>In Vitro</i> MAOA/B Enzyme Assay</h3><div class="NLM_p last">MAOA/B enzyme activity was measured with the MAO-Glo assay (Promega) by following the manufacturer’s instructions. MAOA and MAOB were purchased from Sigma-Aldrich. Briefly, a test compound was mixed with a reaction solution containing the MAOA or MAOB enzyme for 10 min, and then, the reaction was started by adding the MAO substrate. After the reaction at room temperature for 60 min, the Luciferine detection reagent (Promega) was added to terminate the reaction. The luminescence was measured by EnVision. Luminescent signals of the complete reaction mixture were defined as 0% inhibitory activity, and those of the reaction without the enzyme were defined as 100% inhibitory activity. Curve fittings and calculations of IC<sub>50</sub> values were performed with the program XLfit ver. 5 with the maximum and minimum of the curve constrained to 100 and 0, respectively.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Analysis of the FAD Adduct with <b>1e</b> in LSD1 Protein by High Resolution Mass Spectrometry</h3><div class="NLM_p">Recombinant LSD1 protein (125 μM) was treated with 1 mM <b>1e</b> at room temperature for 1, 2, and 4 h in 50 mM Tris–HCl (pH 8.0) to form compound FAD adducts in LSD1. Subsequently, the protein samples were denatured by adding 8 M urea solution to a final concentration of 6.4 M for 1 h on ice, and the compound FAD adducts were extracted on ice with acetonitrile for 15 min. The precipitates were removed by centrifugation. The supernatants were added with 500 mM ammonium acetate to a final concentration of 5 mM.</div><div class="NLM_p last">The sample solution diluted 10-fold with 5 mM ammonium acetate in 5% acetonitrile was injected to an LC/MS system consisting of a high-resolution mass spectrometer (HR-MS, Orbitrap Elite, Thermo Fisher Scientific) and a Prominence UFLC system (Shimadzu). Chromatographic separation was achieved at 40 °C using a CAPCELL CORE AQ analytical column (2.1 × 100 mm, 2.7 μm, Shiseido) in linear gradient elution with the mobile phases A and B consisted of 5 mM ammonium acetate in 98% water/2% acetonitrile (v/v) and 5 mM ammonium acetate in 5% water/95% acetonitrile (v/v), respectively. The flow rate was set at 0.5 mL/min. The mass spectrometer was operated in the negative ionization with the FT full scan mode. Xcalibur 2.2 software (Thermo Fisher Scientific) was used for data acquisition and processing.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Western Blotting</h3><div class="NLM_p last">LSD1 proteins in human, monkey, and rat brain tissues were analyzed by western blotting using the e-PAGEL (ATTO, #E-R520L), Tris/Glycine/SDS running buffer (Bio-Rad, #161–0732), iBlot gel transfer sevice, (Invitrogen, #IB1001), iBlot PVDF transfer stack, Regular (Invitrogen, #IB4010-01), PVDF blocking reagent for Can Get Signal (TOYOBO, RNYPBR01), Can Get Signal immunoreaction enhancer solution (TOYOBO, NKB-101), anti-LSD1 antibody (Cell Signaling Technology, #2139), and anti-pan actin antibody (Cell Signaling Technology, #4968). The blots were developed using an Immobilon Western Chemiluminescent HRP Substrate (Millipore, #WBKLS0100).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>In Vitro</i> LSD1 Enzyme Kinetic Assay for Determination of <i>K</i><sub>inact</sub>/<i>K</i><sub>I</sub> Values</h3><div class="NLM_p last">Kinetics of <i>in vitro</i> LSD1 enzyme activity was measured with a peroxidase-coupled assay. Various concentrations of the compounds were incubated with 500 μM mono-methylated Lys 4 histone H3 peptide that was synthesized by Scrum Inc. in assay buffer (50 mM Tris–HCl (pH 8.0) and 0.1% BSA). After the addition of 20 mg/mL horseradish peroxidase and 50 μM AmplexRed reagent (Life Technologies), the enzyme reaction was initiated by incubation with 20 nM recombinant LSD1/CoREST. The hydrogen peroxide production during the amine oxidase reaction by LSD1 was monitored by the AmplexRed reagent. The fluorescence signals of the resultant product resorufin were detected at 585 nm, with the excitation at 570 nm, by using Spectra Max (Molecular Devices). Because the compounds were irreversible inhibitors, their progress curve of fluorescent signals with background subtraction was fitted with the following <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>, using GraphPad Prism 5 Software (GraphPad Software Inc.)<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_m002.gif" alt="" id="_i58" /></img><span class="labelSpan">(1)</span></span>where <i>v</i><sub>i</sub> represents the initial velocity and <i>k</i><sub>obs</sub> and <i>t</i> demonstrate the pseudo-first order rate constant and time, respectively. <i>y</i><sub>0</sub> is the product when <i>t</i> is 0. The estimated <i>k</i><sub>obs</sub> values were replotted as a function of inhibitor concentration, and <i>k</i><sub>inact</sub> or <i>K</i><sub>I</sub> values were calculated from the following <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a><span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_m003.gif" alt="" id="_i59" /></img><span class="labelSpan">(2)</span></span>where <i>K</i><sub>m</sub> represents the concentration of the histone H3 peptide substrate, evaluated at 20 μM.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Animals</h3><div class="NLM_p last">One male rhesus monkey (<i>Macaca mulatta</i>) housed at the Yale University School of Medicine in New Haven, CT, U.S.A., was used as the research subject. PET imaging was carried out under institutional animal care protocols complying with the United States federal regulations. Animal care approval and oversight for this study was provided by the Yale University Institutional Animal Care and Use Committee. The animals fasted for 18 to 24 h before the study. At 2 h prior to injection of the radiotracer, the animal was anesthetized with intramuscular (i. m.) ketamine (10 mg/kg), given glycopyrrolate (0.005 to 0.01 mg/kg, i. m.), transferred to the PET camera, and immediately intubated with an endotracheal tube for continued anesthesia with isoflurane administered, to effect, through a rebreathing circuit. An intravenous line (i. v.) was placed and used for administration of the radiotracers. Body temperature was kept at approximately 37 °C using a heated water blanket. Vital signs, including heart rate, blood pressure, respiration rate, oxygen saturation, and body temperature, were monitored at least every 10 to 20 min during the study.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> PET Studies in the Monkey Brain</h3><div class="NLM_p last">The radiotracer [<sup>18</sup>F]<b>1e</b> was administered via iv bolus over 3 min. Following radiotracer administration, list mode emission data were collected over 240 min on the microPET FOCUS 220 camera (Siemens Medical Solutions, Inc.) and reconstructed into a series of 57 dynamic 3D frames as follows: 6 × 0.5, 3 × 1, 2 × 2 and 46 × 5 min. The dynamic series were subsequently reconstructed using filtered back projection with corrections for random, scatter, and attenuation provided by the camera manufacturer. Reconstructed PET image data volumes were transferred to the image processing PMOD software package (PMOD Technologies, Zurich, Switzerland), where the images were realigned with the rhesus’ individual MRI to apply a volume of interest (VOI) template<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> comprising the following regions: amygdala, hippocampus, frontal lobe, temporal lobe, parietal lobe, occipital lobe, anterior cingulate, insula, caudate, putamen, thalamus, cerebellum, corpus callosum, and pons. Average activity concentration (kBq/cc) within each VOI was determined, and TAC were generated for each study, depicting the regional brain activity concentration over time and reflecting the total uptake (specific plus non displaceable). TAC were also expressed in SUV units (g/mL) by normalizing by the weight of the animal and the injected dose.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i62"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01937" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01937?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01937</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis of chloro-precursor <b>26</b>; radiolabeling of [<sup>18</sup>F]<b>1d</b>; chromatograms of chiral separation; HPLC traces of <b>1a</b>–<b>e</b>; and chromatograms for purification and analysis of [<sup>18</sup>F]<b>1e</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (CSV) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf">jm0c01937_si_001.pdf (373.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_002.csv">jm0c01937_si_002.csv (0.32 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01937" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06558" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06558" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tatsuki Koike</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3264-1406" title="Orcid link">http://orcid.org/0000-0002-3264-1406</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0c786d787f7967652267636567694c786d6769686d226f6361"><span class="__cf_email__" data-cfemail="add9ccd9ded8c6c483c6c2c4c6c8edd9ccc6c8c9cc83cec2c0">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasushi Hattori</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4054-3619" title="Orcid link">http://orcid.org/0000-0003-4054-3619</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satoru Matsuda</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rina Baba</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kouta Matsumiya</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shinji Iwasaki</span> - <span class="hlFld-Affiliation affiliation">Research, Takeda Pharmaceutical Company Limited, 26-1 Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cristian C. Constantinescu</span> - <span class="hlFld-Affiliation affiliation">Invicro
LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas J. Morley</span> - <span class="hlFld-Affiliation affiliation">Invicro
LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincent M. Carroll</span> - <span class="hlFld-Affiliation affiliation">Invicro
LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Caroline Papin</span> - <span class="hlFld-Affiliation affiliation">Invicro
LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexandra Gouasmat</span> - <span class="hlFld-Affiliation affiliation">Invicro
LLC, 60 Temple Street, Suite 8A, New Haven, Connecticut 06518, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Alagille</span> - <span class="hlFld-Affiliation affiliation">XingImaging
LLC, 760 Temple Street, New Haven, Connecticut 06510, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gilles Tamagnan</span> - <span class="hlFld-Affiliation affiliation">XingImaging
LLC, 760 Temple Street, New Haven, Connecticut 06510, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Misa Iwatani for performing enzyme assay, Yusuke Tominari for the synthesis of compounds, and Mitsuhiro Ito and Shigemitsu Matsumoto for fruitful discussion.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain barrier</p></td></tr><tr><td class="NLM_term">Boc<sub>2</sub>O</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl dicarbonate</p></td></tr><tr><td class="NLM_term">CH<sub>3</sub>CN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">CPME</td><td class="NLM_def"><p class="first last">cyclopentyl methyl ether</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethyl-4-aminopyridine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">IPE</td><td class="NLM_def"><p class="first last">diisopropyl ether</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">petroleum ether</p></td></tr><tr><td class="NLM_term">PET</td><td class="NLM_def"><p class="first last">positron emission tomography</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">PdCl<sub>2</sub> (dppf)</td><td class="NLM_def"><p class="first last">[1,1′-bis(diphenylphosphino)ferrocene]palladium(II) dichloride dichloromethane adduct</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lgpd190gnRntA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varier, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmers, H. T. M.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1815</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2010.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.bbcan.2010.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=20951770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+M.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer&doi=10.1016%2Fj.bbcan.2010.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine methylation and demethylation pathways in cancer</span></div><div class="casAuthors">Varier, Radhika A.; Timmers, H. T. Marc</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1815</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-89</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The genetic changes leading to the development of human cancer are accompanied by alterations in the structure and modification status of chromatin, which represent powerful regulatory mechanisms for gene expression and genome stability.  These epigenetic alterations have sparked interest into deciphering the regulatory pathways and function of post-translational modifications of histones during the initiation and progression of cancer.  In this review we describe and summarize the current knowledge of several histone lysine methyltransferase and demethylase pathways relevant to cancer.  Mechanistic insight into histone modifications will pave the way for the development and therapeutic application of "epidrugs" in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MyfXQs2YR7Vg90H21EOLACvtfcHk0lgpd190gnRntA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D&md5=3331b133c4f1b15beab600420bb859f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%2BM.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89%26doi%3D10.1016%2Fj.bbcan.2010.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1038/nrm3327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases&doi=10.1038%2Fnrm3327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0lh6WuN4OX-AwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311%26doi%3D10.1038%2Fnrm3327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emily, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melissa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debasis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephen, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ian, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, M.</span></span> <span> </span><span class="NLM_article-title">Hooker Brain-penetrant LSD1 inhibitors can block memory consolidation</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">128</span></span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=N.+Rameshauthor=L.+R.+Emilyauthor=M.+Melissaauthor=P.+Debasisauthor=N.+Stephanieauthor=M.+C.+Stephenauthor=T.+H.+Ianauthor=A.+W.+Marceloauthor=J.+H.+Stephenauthor=M.+Jacob&title=Hooker+Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DRamesh%26aufirst%3DN.%26aulast%3DEmily%26aufirst%3DL.%2BR.%26aulast%3DMelissa%26aufirst%3DM.%26aulast%3DDebasis%26aufirst%3DP.%26aulast%3DStephanie%26aufirst%3DN.%26aulast%3DStephen%26aufirst%3DM.%2BC.%26aulast%3DIan%26aufirst%3DT.%2BH.%26aulast%3DMarcelo%26aufirst%3DA.%2BW.%26aulast%3DStephen%26aufirst%3DJ.%2BH.%26aulast%3DJacob%26aufirst%3DM.%26atitle%3DHooker%2520Brain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Network
& Pathway Analysis Subgroup of Psychiatric Genomics Consortium</span> <span> </span><span class="NLM_article-title">Psychiatric genome-wide association
study analyses implicate neuronal, immune and histone pathways</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">209</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fnn.3922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=25599223" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=199-209&author=Network%0A%26+Pathway+Analysis+Subgroup+of+Psychiatric+Genomics+Consortium&title=Psychiatric+genome-wide+association%0Astudy+analyses+implicate+neuronal%2C+immune+and+histone+pathways"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnn.3922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3922%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPsychiatric%2520genome-wide%2520association%250Astudy%2520analyses%2520implicate%2520neuronal%252C%2520immune%2520and%2520histone%2520pathways%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D199%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matevossian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarian, S.</span></span> <span> </span><span class="NLM_article-title">Prefrontal dysfunction in schizophrenia involves mixed-lineage leukemia 1-regulated histone methylation at GABAergic gene promoters</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">11254</span>– <span class="NLM_lpage">11262</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.3272-07.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1523%2FJNEUROSCI.3272-07.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17942719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ekur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=11254-11262&author=H.-S.+Huangauthor=A.+Matevossianauthor=C.+Whittleauthor=S.+Y.+Kimauthor=A.+Schumacherauthor=S.+P.+Bakerauthor=S.+Akbarian&title=Prefrontal+dysfunction+in+schizophrenia+involves+mixed-lineage+leukemia+1-regulated+histone+methylation+at+GABAergic+gene+promoters&doi=10.1523%2Fjneurosci.3272-07.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prefrontal dysfunction in schizophrenia involves mixed-lineage leukemia 1-regulated histone methylation at GABAergic gene promoters</span></div><div class="casAuthors">Huang, Hsien-Sung; Matevossian, Anouch; Whittle, Catheryne; Kim, Se Young; Schumacher, Armin; Baker, Stephen P.; Akbarian, Schahram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11254-11262</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Alterations in GABAergic mRNA expression play a key role for prefrontal dysfunction in schizophrenia and other neurodevelopmental disease.  Here, we show that histone H3-lysine 4 methylation, a chromatin mark assocd. with the transcriptional process, progressively increased at GAD1 and other GABAergic gene promoters (GAD2, NPY, SST) in human prefrontal cortex (PFC) from prenatal to peripubertal ages and throughout adulthood.  Alterations in schizophrenia included decreased GAD1 expression and H3K4-trimethylation, predominantly in females and in conjunction with a risk haplotype at the 5' end of GAD1.  Heterozygosity for a truncated, lacZ knock-in allele of mixed-lineage leukemia 1 (Mll1), a histone methyltransferase expressed in GABAergic and other cortical neurons, resulted in decreased H3K4 methylation at GABAergic gene promoters.  In contrast, Gad1 H3K4 (tri)methylation and Mll1 occupancy was increased in cerebral cortex of mice after treatment with the atypical antipsychotic, clozapine.  These effects were not mimicked by haloperidol or genetic ablation of dopamine D2 and D3 receptors, suggesting that blockade of D2-like signaling is not sufficient for clozapine-induced histone methylation.  Therefore, chromatin remodeling mechanisms at GABAergic gene promoters, including MLL1-mediated histone methylation, operate throughout an extended period of normal human PFC development and play a role in the neurobiol. of schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK8Vzt1G1AL7Vg90H21EOLACvtfcHk0lgjk64rxraEjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ekur3J&md5=be3431b2d2b456e9d6d5a9d21460dda6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3272-07.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3272-07.2007%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-S.%26aulast%3DMatevossian%26aufirst%3DA.%26aulast%3DWhittle%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DSchumacher%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DS.%2BP.%26aulast%3DAkbarian%26aufirst%3DS.%26atitle%3DPrefrontal%2520dysfunction%2520in%2520schizophrenia%2520involves%2520mixed-lineage%2520leukemia%25201-regulated%2520histone%2520methylation%2520at%2520GABAergic%2520gene%2520promoters%26jtitle%3DJ.%2520Neurosci.%26date%3D2007%26volume%3D27%26spage%3D11254%26epage%3D11262%26doi%3D10.1523%2Fjneurosci.3272-07.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shulha, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virgil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Z.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Signatures of Autism</span>. <i>Arch. Gen. Psychiatr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1001/archgenpsychiatry.2011.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1001%2Farchgenpsychiatry.2011.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22065254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFSkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2012&pages=314-324&author=H.+P.+Shulhaauthor=I.+Cheungauthor=C.+Whittleauthor=J.+Wangauthor=D.+Virgilauthor=C.+L.+Linauthor=Y.+Guoauthor=A.+Lessardauthor=S.+Akbarianauthor=Z.+Weng&title=Epigenetic+Signatures+of+Autism&doi=10.1001%2Farchgenpsychiatry.2011.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic signatures of autism: trimethylated H3K4 landscapes in prefrontal neurons</span></div><div class="casAuthors">Shulha, Hennady P.; Cheung, Iris; Whittle, Catheryne; Wang, Jie; Virgil, Daniel; Lin, Cong L.; Guo, Yin; Lessard, Andree; Akbarian, Schahram; Weng, Zhiping</div><div class="citationInfo"><span class="NLM_cas:title">Archives of General Psychiatry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">314-324</span>CODEN:
                <span class="NLM_cas:coden">ARGPAQ</span>;
        ISSN:<span class="NLM_cas:issn">0003-990X</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Neuronal dysfunction in cerebral cortex and other brain regions could contribute to the cognitive and behavioral defects in autism.  The objective of the study was to characterize epigenetic signatures of autism in prefrontal cortex neurons.  We performed fluorescence-activated sorting and sepn. of neuronal and nonneuronal nuclei from postmortem prefrontal cortex, digested the chromatin with micrococcal nuclease, and deeply sequenced the DNA from the mononucleosomes with trimethylated H3K4 (H3K4me3), a histone mark assocd. with transcriptional regulation.  Approx. 15 billion base pairs of H3K4me3-enriched sequences were collected from 32 brains.  Participants: A total of 16 subjects diagnosed as having autism and 16 control subjects ranging in age from 0.5 to 70 years participated in the study at the academic medical center.  Identification of genomic loci showing autism-assocd. H3K4me3 changes in prefrontal cortex neurons was carried out.  Subjects with autism showed no evidence for generalized disruption of the developmentally regulated remodeling of the H3K4me3 landscape that defines normal prefrontal cortex neurons in early infancy.  However, excess spreading of H3K4me3 from the transcription start sites into downstream gene bodies and upstream promoters was obsd. specifically in neuronal chromatin from 4 of 16 autism cases but not in controls.  Variable subsets of autism cases exhibit altered H3K4me3 peaks at numerous genes regulating neuronal connectivity, social behaviors, and cognition, often in conjunction with altered expression of the corresponding transcripts.  Autism-assocd. H3K4me3 peaks were significantly enriched in genes and loci implicated in neurodevelopmental diseases.  Prefrontal cortex neurons from subjects with autism show changes in chromatin structures at hundreds of loci genome-wide, revealing considerable overlap between genetic and epigenetic risk maps of developmental brain disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRlm7JvRmMurVg90H21EOLACvtfcHk0lhBUxJu5HJp6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFSkt7Y%253D&md5=5d60b2308b7cbfb610f2fd4f12c1d96f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1001%2Farchgenpsychiatry.2011.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchgenpsychiatry.2011.151%26sid%3Dliteratum%253Aachs%26aulast%3DShulha%26aufirst%3DH.%2BP.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DWhittle%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVirgil%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DC.%2BL.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DLessard%26aufirst%3DA.%26aulast%3DAkbarian%26aufirst%3DS.%26aulast%3DWeng%26aufirst%3DZ.%26atitle%3DEpigenetic%2520Signatures%2520of%2520Autism%26jtitle%3DArch.%2520Gen.%2520Psychiatr.%26date%3D2012%26volume%3D69%26spage%3D314%26epage%3D324%26doi%3D10.1001%2Farchgenpsychiatry.2011.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wynde, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doughty, M. L.</span></span> <span> </span><span class="NLM_article-title">Role of H3K4 demethylases in complex neurodevelopmental diseases</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.2217/epi.10.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.2217%2Fepi.10.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22121901" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=407-418&author=C.+Wyndeauthor=L.+Stalkerauthor=M.+L.+Doughty&title=Role+of+H3K4+demethylases+in+complex+neurodevelopmental+diseases&doi=10.2217%2Fepi.10.12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2217%2Fepi.10.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.10.12%26sid%3Dliteratum%253Aachs%26aulast%3DWynde%26aufirst%3DC.%26aulast%3DStalker%26aufirst%3DL.%26aulast%3DDoughty%26aufirst%3DM.%2BL.%26atitle%3DRole%2520of%2520H3K4%2520demethylases%2520in%2520complex%2520neurodevelopmental%2520diseases%26jtitle%3DEpigenomics%26date%3D2010%26volume%3D2%26spage%3D407%26epage%3D418%26doi%3D10.2217%2Fepi.10.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Atalaya, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barco, A.</span></span> <span> </span><span class="NLM_article-title">Histone H3 lysine methylation in cognition and intellectual disability disorders</span>. <i>Learn. Mem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">579</span>, <span class="refDoi"> DOI: 10.1101/lm.029363.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1101%2Flm.029363.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24045506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFeis77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=570-579&author=S.+Parkelauthor=J.+P.+Lopez-Atalayaauthor=A.+Barco&title=Histone+H3+lysine+methylation+in+cognition+and+intellectual+disability+disorders&doi=10.1101%2Flm.029363.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 lysine methylation in cognition and intellectual disability disorders</span></div><div class="casAuthors">Parkel, Sven; Lopez-Atalaya, Jose P.; Barco, Angel</div><div class="citationInfo"><span class="NLM_cas:title">Learning & Memory</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">570-579</span>CODEN:
                <span class="NLM_cas:coden">LEMEFO</span>;
        ISSN:<span class="NLM_cas:issn">1072-0502</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Recent research indicates that epigenetic mechanisms and, in particular, the post-translational modification (PTM) of histories may contribute to memory encoding and storage.  Among the dozens of possible histone PTMs, the methylation/demethylation of lysines in the N-terminal tall of histone H3 exhibits particularly strong links with cognitive abilities.  First, the persistence and tight assocn. with distinct transcriptional states of the gene make these modifications particularly suitable for being part of the mol. underpinnings of memory storage.  Second, correlative evidence indicates that the methylation/demethylation of lysines in histone H3 is actively regulated during memory processes.  Third, several enzymes regulating these PTMs are assocd. with intellectual disability disorders.  We review here these three lines of evidence and discuss the potential role of epigenetic mechanisms centered on the methylation of lysine residues on histone H3 in neuroplasticity and neurodevelopmental disorders assocd. with intellectual disability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroEcFk0SSlPrVg90H21EOLACvtfcHk0lhBUxJu5HJp6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFeis77K&md5=a4d679f8e3bf2459bacd051669b1f3cc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Flm.029363.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Flm.029363.112%26sid%3Dliteratum%253Aachs%26aulast%3DParkel%26aufirst%3DS.%26aulast%3DLopez-Atalaya%26aufirst%3DJ.%2BP.%26aulast%3DBarco%26aufirst%3DA.%26atitle%3DHistone%2520H3%2520lysine%2520methylation%2520in%2520cognition%2520and%2520intellectual%2520disability%2520disorders%26jtitle%3DLearn.%2520Mem.%26date%3D2013%26volume%3D20%26spage%3D570%26epage%3D579%26doi%3D10.1101%2Flm.029363.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vashishtha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gipson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratter, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadorf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel-Ciernia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troncosco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadri-Vakili, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span> <span> </span><span class="NLM_article-title">Targeting H3K4 trimethylation in Huntington disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">E3027</span>– <span class="NLM_lpage">E3036</span>, <span class="refDoi"> DOI: 10.1073/pnas.1311323110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1073%2Fpnas.1311323110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=23872847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVShsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E3027-E3036&author=M.+Vashishthaauthor=C.+W.+Ngauthor=F.+Yildirimauthor=T.+A.+Gipsonauthor=I.+H.+Kratterauthor=L.+Bodaiauthor=W.+Songauthor=A.+Lauauthor=A.+Labadorfauthor=A.+Vogel-Cierniaauthor=J.+Troncoscoauthor=C.+A.+Rossauthor=G.+P.+Batesauthor=D.+Kraincauthor=G.+Sadri-Vakiliauthor=S.+Finkbeinerauthor=J.+L.+Marshauthor=D.+E.+Housmanauthor=E.+Fraenkelauthor=L.+M.+Thompson&title=Targeting+H3K4+trimethylation+in+Huntington+disease&doi=10.1073%2Fpnas.1311323110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting H3K4 trimethylation in Huntington disease</span></div><div class="casAuthors">Vashishtha, Malini; Ng, Christopher W.; Yildirim, Ferah; Gipson, Theresa A.; Kratter, Ian H.; Bodai, Laszlo; Song, Wan; Lau, Alice; Labadorf, Adam; Vogel-Ciernia, Annie; Troncosco, Juan; Ross, Christopher A.; Bates, Gillian P.; Krainc, Dimitri; Sadri-Vakili, Ghazaleh; Finkbeiner, Steven; Marsh, J. Lawrence; Housman, David E.; Fraenkel, Ernest; Thompson, Leslie M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">E3027-E3036, SE3027/1-SE3027/44</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transcriptional dysregulation is an early feature of Huntington disease (HD).  We obsd. gene-specific changes in histone H3 lysine 4 trimethylation (H3K4me3) at transcriptionally repressed promoters in R6/2 mouse and human HD brain.  Genome-wide anal. showed a chromatin signature for this mark.  Reducing the levels of the H3K4 demethylase SMCX/Jarid1c in primary neurons reversed down-regulation of key neuronal genes caused by mutant Huntingtin expression.  Finally, redn. of SMCX/Jarid1c in primary neurons from BACHD mice or the single Jarid1 in a Drosophila HD model was protective.  Therefore, targeting this epigenetic signature may be an effective strategy to ameliorate the consequences of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAKMx94fdqS7Vg90H21EOLACvtfcHk0lgRsuiMA4J9AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVShsL%252FN&md5=aa922cf9c891efa8873679deec877230</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1311323110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1311323110%26sid%3Dliteratum%253Aachs%26aulast%3DVashishtha%26aufirst%3DM.%26aulast%3DNg%26aufirst%3DC.%2BW.%26aulast%3DYildirim%26aufirst%3DF.%26aulast%3DGipson%26aufirst%3DT.%2BA.%26aulast%3DKratter%26aufirst%3DI.%2BH.%26aulast%3DBodai%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DLabadorf%26aufirst%3DA.%26aulast%3DVogel-Ciernia%26aufirst%3DA.%26aulast%3DTroncosco%26aufirst%3DJ.%26aulast%3DRoss%26aufirst%3DC.%2BA.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DKrainc%26aufirst%3DD.%26aulast%3DSadri-Vakili%26aufirst%3DG.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DMarsh%26aufirst%3DJ.%2BL.%26aulast%3DHousman%26aufirst%3DD.%2BE.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26atitle%3DTargeting%2520H3K4%2520trimethylation%2520in%2520Huntington%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3DE3027%26epage%3DE3036%26doi%3D10.1073%2Fpnas.1311323110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lgWkz1qtQ1m1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philip, A. C.</span></span> <span> </span><span class="NLM_article-title">LSD1 and the chemistry of histone demethylation</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">568</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2007.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cbpa.2007.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17851108" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=561-568&author=C.+C.+Jeffreyauthor=A.+C.+Philip&title=LSD1+and+the+chemistry+of+histone+demethylation&doi=10.1016%2Fj.cbpa.2007.07.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DJeffrey%26aufirst%3DC.%2BC.%26aulast%3DPhilip%26aufirst%3DA.%2BC.%26atitle%3DLSD1%2520and%2520the%2520chemistry%2520of%2520histone%2520demethylation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D561%26epage%3D568%26doi%3D10.1016%2Fj.cbpa.2007.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halene, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlichman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christian, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonak, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gur, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blendy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodkin, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gur, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, S. J.</span></span> <span> </span><span class="NLM_article-title">Assessment of NMDA receptor NR1 subunit hypofunction in mice as a model for schizophrenia</span>. <i>Gene Brain Behav.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1111/j.1601-183x.2009.00504.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1111%2Fj.1601-183X.2009.00504.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19563516" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=661-675&author=T.+B.+Haleneauthor=R.+S.+Ehrlichmanauthor=Y.+Liangauthor=E.+P.+Christianauthor=G.+J.+Jonakauthor=T.+L.+Gurauthor=J.+A.+Blendyauthor=H.+C.+Dowauthor=E.+S.+Brodkinauthor=F.+Schneiderauthor=R.+C.+Gurauthor=S.+J.+Siegel&title=Assessment+of+NMDA+receptor+NR1+subunit+hypofunction+in+mice+as+a+model+for+schizophrenia&doi=10.1111%2Fj.1601-183x.2009.00504.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1601-183X.2009.00504.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1601-183X.2009.00504.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalene%26aufirst%3DT.%2BB.%26aulast%3DEhrlichman%26aufirst%3DR.%2BS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DChristian%26aufirst%3DE.%2BP.%26aulast%3DJonak%26aufirst%3DG.%2BJ.%26aulast%3DGur%26aufirst%3DT.%2BL.%26aulast%3DBlendy%26aufirst%3DJ.%2BA.%26aulast%3DDow%26aufirst%3DH.%2BC.%26aulast%3DBrodkin%26aufirst%3DE.%2BS.%26aulast%3DSchneider%26aufirst%3DF.%26aulast%3DGur%26aufirst%3DR.%2BC.%26aulast%3DSiegel%26aufirst%3DS.%2BJ.%26atitle%3DAssessment%2520of%2520NMDA%2520receptor%2520NR1%2520subunit%2520hypofunction%2520in%2520mice%2520as%2520a%2520model%2520for%2520schizophrenia%26jtitle%3DGene%2520Brain%2520Behav.%26date%3D2009%26volume%3D8%26spage%3D661%26epage%3D675%26doi%3D10.1111%2Fj.1601-183x.2009.00504.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohn, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gainetdinov, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, B. H.</span></span> <span> </span><span class="NLM_article-title">Mice with reduced NMDA receptor expression display behaviors related to schizophrenia</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">436</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)81972-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2FS0092-8674%2800%2981972-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10481908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADyaK1MXlslWmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=427-436&author=A.+R.+Mohnauthor=R.+R.+Gainetdinovauthor=M.+G.+Caronauthor=B.+H.+Koller&title=Mice+with+reduced+NMDA+receptor+expression+display+behaviors+related+to+schizophrenia&doi=10.1016%2Fs0092-8674%2800%2981972-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mice with reduced NMDA receptor expression display behaviors related to schizophrenia</span></div><div class="casAuthors">Mohn, Amy R.; Gainetdinov, Raul R.; Caron, Marc G.; Koller, Beverly H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">427-436</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">N-methyl-D-aspartate receptors (NMDARs) represent a subclass of glutamate receptors that play a crit. role in neuronal development and physiol.  We report here the generation of mice expressing only 5% of normal levels of the essential NMDAR1 (NR1) subunit.  Unlike NR1 null mice, these mice survive to adulthood and display behavioral abnormalities, including increased motor activity and stereotypy and deficits in social and sexual interactions.  These behavioral alterations are similar to those obsd. in pharmacol. induced animal models of schizophrenia and can be ameliorated by treatment with haloperidol or clozapine, antipsychotic drugs that antagonize dopaminergic and serotonergic receptors.  These findings support a model in which reduced NMDA receptor activity results in schizophrenic-like behavior and reveals how pharmacol. manipulation of monoaminergic pathways can affect this phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouZjUiHREyAbVg90H21EOLACvtfcHk0livnL7qKo6s_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlslWmsb0%253D&md5=39d97aaab71d2d9848210eec9e7faa1c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981972-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981972-8%26sid%3Dliteratum%253Aachs%26aulast%3DMohn%26aufirst%3DA.%2BR.%26aulast%3DGainetdinov%26aufirst%3DR.%2BR.%26aulast%3DCaron%26aufirst%3DM.%2BG.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26atitle%3DMice%2520with%2520reduced%2520NMDA%2520receptor%2520expression%2520display%2520behaviors%2520related%2520to%2520schizophrenia%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D427%26epage%3D436%26doi%3D10.1016%2Fs0092-8674%2800%2981972-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wesseling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guest, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmoune, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahn, S.</span></span> <span> </span><span class="NLM_article-title">Integrative proteomic analysis of the NMDA NR1 knockdown mouse model reveals effects on central and peripheral pathways associated with schizophrenia and autism spectrum disorders</span>. <i>Mol. Autism.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">38</span>, <span class="refDoi"> DOI: 10.1186/2040-2392-5-38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1186%2F2040-2392-5-38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=25061506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVChsbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=38&author=H.+Wesselingauthor=P.+C.+Guestauthor=C.-M.+Leeauthor=E.+H.+Wongauthor=H.+Rahmouneauthor=S.+Bahn&title=Integrative+proteomic+analysis+of+the+NMDA+NR1+knockdown+mouse+model+reveals+effects+on+central+and+peripheral+pathways+associated+with+schizophrenia+and+autism+spectrum+disorders&doi=10.1186%2F2040-2392-5-38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative proteomic analysis of the NMDA NR1 knockdown mouse model reveals effects on central and peripheral pathways associated with schizophrenia and autism spectrum disorders</span></div><div class="casAuthors">Wesseling, Hendrik; Guest, Paul C.; Lee, Chi-Ming; Wong, Erik H. F.; Rahmoune, Hassan; Bahn, Sabine</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Autism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38/1-38/32, 32 pp.</span>CODEN:
                <span class="NLM_cas:coden">MAOUBR</span>;
        ISSN:<span class="NLM_cas:issn">2040-2392</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Over the last decade, the transgenic N-methyl-d-aspartate receptor (NMDAR) NR1-knockdown mouse (NR1neo-/-) has been investigated as a glutamate hypofunction model for schizophrenia.  Recent research has now revealed that the model also recapitulates cognitive and neg. symptoms in the continuum of other psychiatric diseases, particularly autism spectrum disorders (ASD).  As previous studies have mostly focussed on behavioral readouts, a mol. characterization of this model will help to identify novel biomarkers or potential drug targets.  Methods: Here, we have used multiplex immunoassay analyses to investigate peripheral analyte alterations in serum of NR1neo-/- mice, as well as a combination of shotgun label-free liq. chromatog. mass spectrometry, bioinformatic pathway analyses, and a shotgun-based 40-plex selected reaction monitoring (SRM) assay to investigate altered mol. pathways in the frontal cortex and hippocampus.  All findings were cross compared to identify translatable findings between the brain and periphery.  Results: Multiplex immunoassay profiling led to identification of 29 analytes that were significantly altered in sera of NR1neo-/- mice.  The highest magnitude changes were found for neurotrophic factors (VEGFA, EGF, IGF-1), apolipoprotein A1, and fibrinogen.  We also found decreased levels of several chemokines.  Following this, LC-MSE profiling led to identification of 48 significantly changed proteins in the frontal cortex and 41 in the hippocampus.  In particular, MARCS, the mitochondrial pyruvate kinase, and CamKII-alpha were affected.  Based on the combination of protein set enrichment and bioinformatic pathway anal., we designed orthogonal SRM-assays which validated the abnormalities of proteins involved in synaptic long-term potentiation, myelination, and the ERK-signalling pathway in both brain regions.  In contrast, increased levels of proteins involved in neurotransmitter metab. and release were found only in the frontal cortex and abnormalities of proteins involved in the purinergic system were found exclusively in the hippocampus.  Conclusions: Taken together, this multi-platform profiling study has identified peripheral changes which are potentially linked to central alterations in synaptic plasticity and neuronal function assocd. with NMDAR-NR1 hypofunction.  Therefore, the reported proteomic changes may be useful as translational biomarkers in human and rodent model drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_PWLtT7GYpLVg90H21EOLACvtfcHk0liEhfuEaVAqbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVChsbnM&md5=bdef8cecd164673870cb7537d0feb447</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F2040-2392-5-38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2040-2392-5-38%26sid%3Dliteratum%253Aachs%26aulast%3DWesseling%26aufirst%3DH.%26aulast%3DGuest%26aufirst%3DP.%2BC.%26aulast%3DLee%26aufirst%3DC.-M.%26aulast%3DWong%26aufirst%3DE.%2BH.%26aulast%3DRahmoune%26aufirst%3DH.%26aulast%3DBahn%26aufirst%3DS.%26atitle%3DIntegrative%2520proteomic%2520analysis%2520of%2520the%2520NMDA%2520NR1%2520knockdown%2520mouse%2520model%2520reveals%2520effects%2520on%2520central%2520and%2520peripheral%2520pathways%2520associated%2520with%2520schizophrenia%2520and%2520autism%2520spectrum%2520disorders%26jtitle%3DMol.%2520Autism.%26date%3D2014%26volume%3D5%26spage%3D38%26doi%3D10.1186%2F2040-2392-5-38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lg999A1oG3XDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallipoli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulators as promising therapeutic targets in acute myeloid leukemia</span>. <i>Ther. Adv. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1177/2040620715577614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1177%2F2040620715577614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=26137202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGhsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=103-119&author=P.+Gallipoliauthor=G.+Giotopoulosauthor=B.+J.+P.+Huntly&title=Epigenetic+regulators+as+promising+therapeutic+targets+in+acute+myeloid+leukemia&doi=10.1177%2F2040620715577614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulators as promising therapeutic targets in acute myeloid leukemia</span></div><div class="casAuthors">Gallipoli, Paolo; Giotopoulos, George; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Hematology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">103-119</span>CODEN:
                <span class="NLM_cas:coden">TAHHAT</span>;
        ISSN:<span class="NLM_cas:issn">2040-6207</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Acute myeloid leukemia (AML) the most prevalent acute leukemia in adults, is an aggressive hematol. malignancy arising in hematopoietic stem and progenitor cells.  With the exception of a few specific AML subtypes, the mainstays of treatment have not significantly changed over the last 20 years, and are still based on std. cytotoxic chemotherapy.  As a result, clin. outcome remains poor for the majority of patients, with overall long-term survival in the region of 20-30%.  Recent successes in characterizing the genetic Landscape of AML have highlighted that, despite its heterogeneity, many cases of AML carry recurrent mutations in genes encoding epigenetic regulators.  Transcriptional dysregulation and altered epigenetic function have therefore emerged as exciting areas in AML research and it is becoming increasingly clear that epigenetic dysfunction is central to leukemogenesis in AML.  This has subsequently paved the way for the development of epigenetically targeted therapies.  In this review, we will discuss the most recent advances in our understanding of the role of epigenetic dysregulation in AML pathobiol.  We will particularly focus on those altered epigenetic programs that have been shown to be central to the development and maintenance of AML in prectinical models.  We will discuss the recent development of therapeutics specifically targeting these key epigenetic programs in AML, describe their mechanism of action and present their current clin. development.  Finally, we will discuss the opportunities presented by epigenetically targeted therapy in AML and will highlight future challenges ahead for the AML community, to ensure that these novel therapeutics are optimally translated into clin. practice and result in clin. improvement for AML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4US2Qy98czLVg90H21EOLACvtfcHk0liQz-j1xZtNXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGhsbfJ&md5=4cb560fcfb7712b0cd19302eda963782</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1177%2F2040620715577614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040620715577614%26sid%3Dliteratum%253Aachs%26aulast%3DGallipoli%26aufirst%3DP.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DEpigenetic%2520regulators%2520as%2520promising%2520therapeutic%2520targets%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DTher.%2520Adv.%2520Hematol.%26date%3D2015%26volume%3D6%26spage%3D103%26epage%3D119%26doi%3D10.1177%2F2040620715577614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairiki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hikichi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">A Novel LSD1 Inhibitor T-3775440 disrupts GFI1B-containing complex leading to transdifferentiation and impaired growth of AML cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1158%2F1535-7163.MCT-16-0471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=27903753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=273-284&author=Y.+Ishikawaauthor=K.+Gamoauthor=M.+Yabukiauthor=S.+Takagiauthor=K.+Toyoshimaauthor=K.+Nakayamaauthor=A.+Nakayamaauthor=M.+Morimotoauthor=H.+Miyashitaauthor=R.+Dairikiauthor=Y.+Hikichiauthor=N.+Tomitaauthor=D.+Tomitaauthor=S.+Imamuraauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Matsumotoauthor=R.+Haraauthor=T.+Nomuraauthor=K.+Tsuchidaauthor=K.+Nakamura&title=A+Novel+LSD1+Inhibitor+T-3775440+disrupts+GFI1B-containing+complex+leading+to+transdifferentiation+and+impaired+growth+of+AML+cells&doi=10.1158%2F1535-7163.MCT-16-0471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel LSD1 Inhibitor T-3775440 Disrupts GFI1B-Containing Complex Leading to Transdifferentiation and Impaired Growth of AML Cells</span></div><div class="casAuthors">Ishikawa, Yoshinori; Gamo, Kanae; Yabuki, Masato; Takagi, Shinji; Toyoshima, Kosei; Nakayama, Kazuhide; Nakayama, Akiko; Morimoto, Megumi; Miyashita, Hitoshi; Dairiki, Ryo; Hikichi, Yukiko; Tomita, Naoki; Tomita, Daisuke; Imamura, Shinichi; Iwatani, Misa; Kamada, Yusuke; Matsumoto, Satoru; Hara, Ryujiro; Nomura, Toshiyuki; Tsuchida, Ken; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-284</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Dysregulation of lysine (K)-specific demethylase 1A (LSD1), also known as KDM1A, has been implicated in the development of various cancers, including leukemia.  Here, we describe the antileukemic activity and mechanism of action of T-3775440, a novel irreversible LSD1 inhibitor.  Cell growth anal. of leukemia cell lines revealed that acute erythroid leukemia (AEL) and acute megakaryoblastic leukemia cells (AMKL) were highly sensitive to this compd.  T-3775440 treatment enforced transdifferentiation of erythroid/megakaryocytic lineages into granulomonocytic-like lineage cells.  Mechanistically, T-3775440 disrupted the interaction between LSD1 and growth factor-independent 1B (GFI1B), a transcription factor crit. for the differentiation processes of erythroid and megakaryocytic lineage cells.  Knockdown of LSD1 and GFI1B recapitulated T-3775440-induced transdifferentiation and cell growth suppression, highlighting the significance of LSD1-GFI1B axis inhibition with regard to the anti-AML effects of T-3775440.  Moreover, T-3775440 exhibited significant antitumor efficacy in AEL and AMKL xenograft models.  Our findings provide a rationale for evaluating LSD1 inhibitors as potential treatments and indicate a novel mechanism of action against AML, particularly AEL and AMKL.  Mol Cancer Ther; 16(2); 273-84. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomcxpsluqnf7Vg90H21EOLACvtfcHk0liQz-j1xZtNXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D&md5=2335343b4d5e7bbc197a1d5aec54e3af</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0471%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DGamo%26aufirst%3DK.%26aulast%3DYabuki%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DToyoshima%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DDairiki%26aufirst%3DR.%26aulast%3DHikichi%26aufirst%3DY.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DImamura%26aufirst%3DS.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DA%2520Novel%2520LSD1%2520Inhibitor%2520T-3775440%2520disrupts%2520GFI1B-containing%2520complex%2520leading%2520to%2520transdifferentiation%2520and%2520impaired%2520growth%2520of%2520AML%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D273%26epage%3D284%26doi%3D10.1158%2F1535-7163.MCT-16-0471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyofuku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, H.</span></span> <span> </span><span class="NLM_article-title">T-448, a specific inhibitor of LSD1 enzyme activity, improves learning function without causing thrombocytopenia in mice</span>. <i>Neuropsychopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1512</span>, <span class="refDoi"> DOI: 10.1038/s41386-018-0300-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fs41386-018-0300-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=30580376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyrtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2019&pages=1505-1512&author=S.+Matsudaauthor=R.+Babaauthor=H.+Okiauthor=S.+Morimotoauthor=M.+Toyofukuauthor=S.+Igakiauthor=Y.+Kamadaauthor=S.+Iwasakiauthor=K.+Matsumiyaauthor=R.+Hibinoauthor=H.+Kamadaauthor=T.+Hirakawaauthor=M.+Iwataniauthor=K.+Tsuchidaauthor=R.+Haraauthor=M.+Itoauthor=H.+Kimura&title=T-448%2C+a+specific+inhibitor+of+LSD1+enzyme+activity%2C+improves+learning+function+without+causing+thrombocytopenia+in+mice&doi=10.1038%2Fs41386-018-0300-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">T-448, a specific inhibitor of LSD1 enzyme activity, improves learning function without causing thrombocytopenia in mice</span></div><div class="casAuthors">Matsuda, Satoru; Baba, Rina; Oki, Hideyuki; Morimoto, Shinji; Toyofuku, Masashi; Igaki, Shigeru; Kamada, Yusuke; Iwasaki, Shinji; Matsumiya, Kota; Hibino, Ryosuke; Kamada, Hiroko; Hirakawa, Takeshi; Iwatani, Misa; Tsuchida, Ken; Hara, Ryujiro; Ito, Mitsuhiro; Kimura, Haruhide</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychopharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1505-1512</span>CODEN:
                <span class="NLM_cas:coden">NEROEW</span>;
        ISSN:<span class="NLM_cas:issn">0893-133X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dysregulation of histone H3 lysine 4 (H3K4) methylation has been implicated in the pathogenesis of several neurodevelopmental disorders.  Targeting lysine-specific demethylase 1 (LSD1), an H3K4 demethylase, is therefore a promising approach to treat these disorders.  However, LSD1 forms complexes with cofactors including growth factor independent 1B (GFI1B), a crit. regulator of hematopoietic differentiation.  Known tranylcypromine-based irreversible LSD1 inhibitors bind to coenzyme FAD (FAD) and disrupt the LSD1-GFI1B complex, which is assocd. with hematotoxicity such as thrombocytopenia, representing a major hurdle in the development of LSD1 inhibitors as therapeutic agents.  To discover LSD1 inhibitors with potent epigenetic modulation and lower risk of hematotoxicity, we screened small mols. that enhance H3K4 methylation by the inhibition of LSD1 enzyme activity in primary cultured rat neurons but have little impact on LSD1-GFI1B complex in human TF-1a erythroblasts.  Here we report the discovery of a specific inhibitor of LSD1 enzyme activity, T-448 (3-((1S,2R)-2-(cyclobutylamino)cyclopropyl)-N-(5-methyl-1,3,4-thiadiazol-2-yl)benzamide fumarate).  T-448 has minimal impact on the LSD1-GFI1B complex and a superior hematol. safety profile in mice via the generation of a compact formyl-FAD adduct.  T-448 increased brain H3K4 methylation and partially restored learning function in mice with NMDA receptor hypofunction.  T-448-type LSD1 inhibitors with improved safety profiles may provide unique therapeutic approaches for central nervous system disorders assocd. with epigenetic dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZnyoBIF0Nr7Vg90H21EOLACvtfcHk0liQz-j1xZtNXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyrtr7I&md5=a3a94d000e5461bcdf0924c3c3f9cb65</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41386-018-0300-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41386-018-0300-9%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DR.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DMorimoto%26aufirst%3DS.%26aulast%3DToyofuku%26aufirst%3DM.%26aulast%3DIgaki%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DIwasaki%26aufirst%3DS.%26aulast%3DMatsumiya%26aufirst%3DK.%26aulast%3DHibino%26aufirst%3DR.%26aulast%3DKamada%26aufirst%3DH.%26aulast%3DHirakawa%26aufirst%3DT.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DH.%26atitle%3DT-448%252C%2520a%2520specific%2520inhibitor%2520of%2520LSD1%2520enzyme%2520activity%252C%2520improves%2520learning%2520function%2520without%2520causing%2520thrombocytopenia%2520in%2520mice%26jtitle%3DNeuropsychopharmacology%26date%3D2019%26volume%3D44%26spage%3D1505%26epage%3D1512%26doi%3D10.1038%2Fs41386-018-0300-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanenghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirilli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karytinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">6827</span>– <span class="NLM_lpage">6833</span>, <span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0liy396R6HzwGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelamegam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricq, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malvaez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span> <span> </span><span class="NLM_article-title">Brain-penetrant LSD1 inhibitors can block memory consolidation</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1021/cn200104y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=R.+Neelamegamauthor=E.+L.+Ricqauthor=M.+Malvaezauthor=D.+Patnaikauthor=S.+Nortonauthor=S.+M.+Carlinauthor=I.+T.+Hillauthor=M.+A.+Woodauthor=S.+J.+Haggartyauthor=J.+M.+Hooker&title=Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation&doi=10.1021%2Fcn200104y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-Penetrant LSD1 Inhibitors Can Block Memory Consolidation</span></div><div class="casAuthors">Neelamegam, Ramesh; Ricq, Emily L.; Malvaez, Melissa; Patnaik, Debasis; Norton, Stephanie; Carlin, Stephen M.; Hill, Ian T.; Wood, Marcelo A.; Haggarty, Stephen J.; Hooker, Jacob M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-128</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of histone modifications in the brain may represent a new mechanism for brain disorder therapy.  Post-translational modifications of histones regulate gene expression, affecting major cellular processes such as proliferation, differentiation, and function.  An important enzyme involved in one of these histone modifications is lysine specific demethylase 1 (LSD1).  This enzyme is flavin-dependent and exhibits homol. to amine oxidases.  Parnate (2-phenylcyclopropylamine (2-PCPA); tranylcypromine) is a potent inhibitor of monoamine oxidases, and derivs. of 2-PCPA have been used for development of selective LSD1 inhibitors based on the ability to form covalent adducts with FAD.  Here we report the synthesis and in vitro characterization of LSD1 inhibitors that bond covalently to FAD.  The two most potent and selective inhibitors were used to demonstrate brain penetration when administered systemically to rodents.  First, radiosynthesis of a positron-emitting analog was used to obtain preliminary biodistribution data and whole brain time-activity curves.  Second, we demonstrate that this series of LSD1 inhibitors is capable of producing a cognitive effect in a mouse model.  By using a memory formation paradigm, novel object recognition, we show that LSD1 inhibition can abolish long-term memory formation without affecting short-term memory, providing further evidence for the importance of reversible histone methylation in the function of the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRTx9XGRzjgrVg90H21EOLACvtfcHk0liy396R6HzwGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E&md5=cd365810b397d0ece732ea853a3e93da</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DNeelamegam%26aufirst%3DR.%26aulast%3DRicq%26aufirst%3DE.%2BL.%26aulast%3DMalvaez%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DNorton%26aufirst%3DS.%26aulast%3DCarlin%26aufirst%3DS.%2BM.%26aulast%3DHill%26aufirst%3DI.%2BT.%26aulast%3DWood%26aufirst%3DM.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26atitle%3DBrain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128%26doi%3D10.1021%2Fcn200104y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Structure activity relationship and modeling studies of inhibitors of lysine specific demethylase 1</span>. <i>PloS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0170301</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0170301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1371%2Fjournal.pone.0170301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=28158205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0170301&author=C.+Zhouauthor=F.+Wuauthor=L.+Luauthor=L.+Weiauthor=E.+Paiauthor=Y.+Yaoauthor=Y.+Song&title=Structure+activity+relationship+and+modeling+studies+of+inhibitors+of+lysine+specific+demethylase+1&doi=10.1371%2Fjournal.pone.0170301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure activity relationship and modeling studies of inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Zhou, Chao; Wu, Fangrui; Lu, Lianghao; Wei, Liping; Pai, Eric; Yao, Yuan; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0170301/1-e0170301/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Post-translational modifications of histone play important roles in gene transcription.  Aberrant methylation of histone lysine sidechains have been often found in cancer.  Lysine specific demethylase 1 (LSD1), which can demethylate histone H3 lysine 4 (H3K4) and other proteins, has recently been found to be a drug target for acute myeloid leukemia.  To understand structure activity/selectivity relationships of LSD1 inhibitors, several series of cyclopropylamine and related compds. were synthesized and tested for their activities against LSD1 and related monoamine oxidase (MAO) A and B.  Several cyclopropylamine contg. compds. were found to be highly potent and selective inhibitors of LSD1.  A novel series cyclopropylimine compds. also exhibited strong inhibitory activity against LSD1.  Structure activity relationships (SAR) of these compds. are discussed.  Docking studies were performed to provide possible binding models of a representative compd. in LSD1 and MAO-A.  Moreover, these modeling studies can rationalize the obsd. SARs and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFKcJwJZQw7Vg90H21EOLACvtfcHk0liy396R6HzwGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyqs7Y%253D&md5=4d6a733dd6ef4a4b4f34a77c2d08a104</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0170301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0170301%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DPai%26aufirst%3DE.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DStructure%2520activity%2520relationship%2520and%2520modeling%2520studies%2520of%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DPloS%2520One%26date%3D2017%26volume%3D12%26spage%3De0170301%26doi%3D10.1371%2Fjournal.pone.0170301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, R.</span></span> <span> </span><span class="NLM_article-title">In vivo site-directed radiotracers: a mini-review</span>. <i>Nucl. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/j.nucmedbio.2008.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.nucmedbio.2008.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19026942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaisLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2008&pages=805-815&author=S.+Patelauthor=R.+Gibson&title=In+vivo+site-directed+radiotracers%3A+a+mini-review&doi=10.1016%2Fj.nucmedbio.2008.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo site-directed radiotracers: a mini-review</span></div><div class="casAuthors">Patel, Shil; Gibson, Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Nuclear Medicine and Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NMBIEO</span>;
        ISSN:<span class="NLM_cas:issn">0969-8051</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review on the process for developing reversible, monovalent site-specific radiotracers in drug design.  It is concerned with the characteristics of radiotracers such as binding, lipophilicity, and binding specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYUEhMFA7d7bVg90H21EOLACvtfcHk0li1ft3JHXSB9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaisLvL&md5=869985e0741d52a76cb546972097e2a8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.nucmedbio.2008.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nucmedbio.2008.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DGibson%26aufirst%3DR.%26atitle%3DIn%2520vivo%2520site-directed%2520radiotracers%253A%2520a%2520mini-review%26jtitle%3DNucl.%2520Med.%2520Biol.%26date%3D2008%26volume%3D35%26spage%3D805%26epage%3D815%26doi%3D10.1016%2Fj.nucmedbio.2008.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laruelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slifstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Positron emission tomography: imaging and quantification of neurotransporter availability</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/s1046-2023(02)00085-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2FS1046-2023%2802%2900085-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=12183117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtFWhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2002&pages=287-299&author=M.+Laruelleauthor=M.+Slifsteinauthor=Y.+Huang&title=Positron+emission+tomography%3A+imaging+and+quantification+of+neurotransporter+availability&doi=10.1016%2Fs1046-2023%2802%2900085-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Positron emission tomography: imaging and quantification of neurotransporter availability</span></div><div class="casAuthors">Laruelle, Marc; Slifstein, Mark; Huang, Yiyun</div><div class="citationInfo"><span class="NLM_cas:title">Methods (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">287-299</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">A review.  Over the last decade, a large no. of radiotracers have been developed to image and quantify transporter availability with positron emission tomog. (PET) or single-photon emission computed tomog. (SPECT).  Radiotracers suitable to image dopamine transporters (DATs) and serotonin transporters (SERTs) have been the object of most efforts.  Following a brief overview of DAT and SERT radiotracers that have been demonstrated to be suitable for quant. anal. in vivo, this article describes the principal methods that have been used for the anal. of these data.  Kinetic modeling is the most direct implementation of the compartment models, but with some tracers accurate input function measurement and good compartment configuration identification can be difficult to obtain.  Other methods were designed to overcome some particular vulnerability to error of classic kinetic modeling, but introduced new vulnerabilities in the process.  Ref. region methods obviate the need for arterial plasma measurement, but are not as robust to violations of the underlying modeling assumptions as methods using the arterial input function.  Graphical methods give ests. of distribution vols. without the requirement of compartment model specification, but provide a biased estimator in the presence of statistical noise.  True equil. methods are quite robust, but their use is limited to expts. with tracers that are suitable for const. infusion.  In conclusion, no universally "best" method is applicable to all neurotransporter imaging studies, and careful evaluation of model-based methods is required for each radiotracer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3jdsXbliaMLVg90H21EOLACvtfcHk0li1ft3JHXSB9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtFWhs74%253D&md5=1ec923e982f1769d80fd74b68d7b05ed</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1046-2023%2802%2900085-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1046-2023%252802%252900085-3%26sid%3Dliteratum%253Aachs%26aulast%3DLaruelle%26aufirst%3DM.%26aulast%3DSlifstein%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DPositron%2520emission%2520tomography%253A%2520imaging%2520and%2520quantification%2520of%2520neurotransporter%2520availability%26jtitle%3DMethods%26date%3D2002%26volume%3D27%26spage%3D287%26epage%3D299%26doi%3D10.1016%2Fs1046-2023%2802%2900085-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, V. W.</span></span> <span> </span><span class="NLM_article-title">PET radiotracers: crossing the blood-brain barrier and surviving metabolism</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2009.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.tips.2009.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19616318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1KgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=431-440&author=V.+W.+Pike&title=PET+radiotracers%3A+crossing+the+blood-brain+barrier+and+surviving+metabolism&doi=10.1016%2Fj.tips.2009.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">PET radiotracers: crossing the blood-brain barrier and surviving metabolism</span></div><div class="casAuthors">Pike, Victor W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">431-440</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Radiotracers for imaging protein targets in the living human brain with positron emission tomog. (PET) are increasingly useful in clin. research and in drug development.  Such radiotracers must fulfill many criteria, among which an ability to enter brain adequately and reversibly without contamination by troublesome radiometabolites is desirable for accurate measurement of the d. of a target protein (e.g. neuroreceptor, transporter, enzyme or plaque).  Candidate radiotracers can fail as a result of poor passive brain entry, rejection from brain by efflux transporters or undesirable metab.  These issues are reviewed.  Emerging PET radiotracers for measuring efflux transporter function and new strategies for ameliorating radiotracer metab. are discussed.  A growing understanding of the mol. features affecting the brain penetration, metab. and efflux transporter sensitivity of prospective radiotracers should ultimately lead to their more rational and efficient design, and also to their greater efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6ftMCpkOSmrVg90H21EOLACvtfcHk0li1ft3JHXSB9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1KgsLg%253D&md5=97704995fecd968baf1e0892bf70324e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2009.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2009.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DV.%2BW.%26atitle%3DPET%2520radiotracers%253A%2520crossing%2520the%2520blood-brain%2520barrier%2520and%2520surviving%2520metabolism%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2009%26volume%3D30%26spage%3D431%26epage%3D440%26doi%3D10.1016%2Fj.tips.2009.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mireles, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Morais, S. M.</span></span> <span> </span><span class="NLM_article-title">In V<i>itro</i> P-glycoprotein assays to predict the <i>in Vivo</i> interactions of P-glycoprotein with drugs in the central nervous system</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1124/dmd.107.017434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1124%2Fdmd.107.017434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17962372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1yktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=268-275&author=B.+Fengauthor=J.+B.+Millsauthor=R.+E.+Davidsonauthor=R.+J.+Mirelesauthor=J.+S.+Janiszewskiauthor=M.+D.+Troutmanauthor=S.+M.+de+Morais&title=In+Vitro+P-glycoprotein+assays+to+predict+the+in+Vivo+interactions+of+P-glycoprotein+with+drugs+in+the+central+nervous+system&doi=10.1124%2Fdmd.107.017434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro P-glycoprotein assays to predict the in vivo interactions of P-glycoprotein with drugs in the central nervous system</span></div><div class="casAuthors">Feng, Bo; Mills, Jessica B.; Davidson, Ralph E.; Mireles, Rouchelle J.; Janiszewski, John S.; Troutman, Matthew D.; de Morais, Sonia M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">268-275</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Thirty-one structurally diverse marketed central nervous system (CNS)-active drugs, one active metabolite, and seven non-CNS-active compds. were tested in three P-glycoprotein (P-gp) in vitro assays: transwell assays using MDCK, human MDR1-MDCK, and mouse Mdr1a-MDCK cells, ATPase, and calcein AM inhibition.  Addnl., the permeability for these compds. was measured in two in vitro models: parallel artificial membrane permeation assay and apical-to-basolateral apparent permeability in MDCK.  The exposure of the same set of compds. in brain and plasma was measured in P-gp knockout (KO) and wild-type (WT) mice after s.c. administration.  One drug and its metabolite, risperidone and 9-hydroxyrisperidone, of the 32 CNS compds., and 6 of the 7 non-CNS drugs were detd. to have pos. efflux using ratio of ratios in MDR1-MDCK vs. MDCK transwell assays.  Data from transwell studies correlated well with the brain-to-plasma area under the curve ratios between P-gp KO and WT mice for the 32 CNS compds.  In addn., 3300 Pfizer compds. were tested in MDR1-MDCK and Mdr1a-MDCK transwell assays, with a good correlation (R2 = 0.92) between the efflux ratios in human MDR1-MDCK and mouse Mdr1a-MDCK cells.  Permeability data showed that the majority of the 32 CNS compds. have moderate to high passive permeability.  This work has demonstrated that in vitro transporter assays help in understanding the role of P-gp-mediated efflux activity in detg. the disposition of CNS drugs in vivo, and the transwell assay is a valuable in vitro assay to evaluate human P-gp interaction with compds. for assessing brain penetration of new chem. entities to treat CNS disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsQyRSOySyGLVg90H21EOLACvtfcHk0lgyJLUSHLYLvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1yktbc%253D&md5=bbf8b6969bd6d6b3d741b6ef9e7853f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fdmd.107.017434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.107.017434%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DMills%26aufirst%3DJ.%2BB.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DMireles%26aufirst%3DR.%2BJ.%26aulast%3DJaniszewski%26aufirst%3DJ.%2BS.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3Dde%2BMorais%26aufirst%3DS.%2BM.%26atitle%3DIn%2520Vitro%2520P-glycoprotein%2520assays%2520to%2520predict%2520the%2520in%2520Vivo%2520interactions%2520of%2520P-glycoprotein%2520with%2520drugs%2520in%2520the%2520central%2520nervous%2520system%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2008%26volume%3D36%26spage%3D268%26epage%3D275%26doi%3D10.1124%2Fdmd.107.017434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liston, T. E.</span></span> <span> </span><span class="NLM_article-title">Species independence in brain tissue binding using brain homogenates</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1270</span>– <span class="NLM_lpage">1277</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.038778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1124%2Fdmd.111.038778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=21474681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFWktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1270-1277&author=L.+Diauthor=J.+P.+Umlandauthor=G.+Changauthor=Y.+Huangauthor=Z.+Linauthor=D.+O.+Scottauthor=M.+D.+Troutmanauthor=T.+E.+Liston&title=Species+independence+in+brain+tissue+binding+using+brain+homogenates&doi=10.1124%2Fdmd.111.038778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Species independence in brain tissue binding using brain homogenates</span></div><div class="casAuthors">Di, Li; Umland, John P.; Chang, George; Huang, Youping; Lin, Zhen; Scott, Dennis O.; Troutman, Matthew D.; Liston, Theodore E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1270-1277</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Species independence of brain tissue binding was assessed with a large no. of structurally diverse compds. using equil. dialysis with brain homogenates of seven species and strains (Wistar Han rat, Sprague-Dawley rat, CD-1 mouse, Hartley guinea pig, beagle dog, cynomolgus monkey, and human).  The results showed that the fractions unbound of the seven species and strains were strongly correlated with correlation coeffs. ranging from 0.93 to 0.99.  The cross-species/strain correlations were not significantly different from the interassay correlation with the same species.  The linear correlation between Wistar Han and other species had a slope close to 1 and an intercept near 0.  Based on orthogonal statistical anal., no correction is needed for extrapolation of fraction unbound from Wistar Han rat to the other species or strains.  Hence, brain tissue binding of Wistar Han rat can be used to obtain binding of other species and strains in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoub4FiLWkAFrVg90H21EOLACvtfcHk0lgyJLUSHLYLvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFWktrg%253D&md5=93a51addedb22de4648e6e5ee39c6c81</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.038778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.038778%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DUmland%26aufirst%3DJ.%2BP.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DScott%26aufirst%3DD.%2BO.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DListon%26aufirst%3DT.%2BE.%26atitle%3DSpecies%2520independence%2520in%2520brain%2520tissue%2520binding%2520using%2520brain%2520homogenates%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1270%26epage%3D1277%26doi%3D10.1124%2Fdmd.111.038778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaddan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zasadny, K.</span></span> <span> </span><span class="NLM_article-title">Design and selection parameters to accelerate the discovery of novel central nervous system positron emission tomography (PET) ligands and their application in the development of a novel phosphodiesterase 2A PET ligand</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4568</span>– <span class="NLM_lpage">4579</span>, <span class="refDoi"> DOI: 10.1021/jm400312y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400312y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4568-4579&author=L.+Zhangauthor=A.+Villalobosauthor=E.+M.+Beckauthor=T.+Bocanauthor=T.+A.+Chappieauthor=L.+Chenauthor=S.+Grimwoodauthor=S.+D.+Heckauthor=C.+J.+Helalauthor=X.+Houauthor=J.+M.+Humphreyauthor=J.+Luauthor=M.+B.+Skaddanauthor=T.+J.+McCarthyauthor=P.+R.+Verhoestauthor=T.+T.+Wagerauthor=K.+Zasadny&title=Design+and+selection+parameters+to+accelerate+the+discovery+of+novel+central+nervous+system+positron+emission+tomography+%28PET%29+ligands+and+their+application+in+the+development+of+a+novel+phosphodiesterase+2A+PET+ligand&doi=10.1021%2Fjm400312y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Selection Parameters to Accelerate the Discovery of Novel Central Nervous System Positron Emission Tomography (PET) Ligands and Their Application in the Development of a Novel Phosphodiesterase 2A PET Ligand</span></div><div class="casAuthors">Zhang, Lei; Villalobos, Anabella; Beck, Elizabeth M.; Bocan, Thomas; Chappie, Thomas A.; Chen, Laigao; Grimwood, Sarah; Heck, Steven D.; Helal, Christopher J.; Hou, Xinjun; Humphrey, John M.; Lu, Jiemin; Skaddan, Marc B.; McCarthy, Timothy J.; Verhoest, Patrick R.; Wager, Travis T.; Zasadny, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4568-4579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To accelerate the discovery of novel small mol. central nervous system (CNS) positron emission tomog. (PET) ligands, we aimed to define a property space that would facilitate ligand design and prioritization, thereby providing a higher probability of success for novel PET ligand development.  Toward this end, we built a database consisting of 62 PET ligands that have successfully reached the clinic and 15 radioligands that failed in late-stage development as neg. controls.  A systematic anal. of these ligands identified a set of preferred parameters for physicochem. properties, brain permeability, and nonspecific binding (NSB).  These preferred parameters have subsequently been applied to several programs and have led to the successful development of novel PET ligands with reduced resources and timelines.  This strategy is illustrated here by the discovery of the novel phosphodiesterase 2A (PDE2A) PET ligand 4-(3-[18F]fluoroazetidin-1-yl)-7-methyl-5-{1-methyl-5-[4-(trifluoromethyl)phenyl]-1H-pyrazol-4-yl}imidazo[5,1-f][1,2,4]triazine, [18F]PF-05270430 (5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTCG9TZE0X-bVg90H21EOLACvtfcHk0lgEEUuF3VR0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D&md5=ee7ce04829d7456a632c02c331c4edfa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm400312y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400312y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DE.%2BM.%26aulast%3DBocan%26aufirst%3DT.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DGrimwood%26aufirst%3DS.%26aulast%3DHeck%26aufirst%3DS.%2BD.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DSkaddan%26aufirst%3DM.%2BB.%26aulast%3DMcCarthy%26aufirst%3DT.%2BJ.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DZasadny%26aufirst%3DK.%26atitle%3DDesign%2520and%2520selection%2520parameters%2520to%2520accelerate%2520the%2520discovery%2520of%2520novel%2520central%2520nervous%2520system%2520positron%2520emission%2520tomography%2520%2528PET%2529%2520ligands%2520and%2520their%2520application%2520in%2520the%2520development%2520of%2520a%2520novel%2520phosphodiesterase%25202A%2520PET%2520ligand%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4568%26epage%3D4579%26doi%3D10.1021%2Fjm400312y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span> <span> </span><span class="NLM_article-title">Impact of physicochemical profiling for rational approach on drug discovery</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1228</span>, <span class="refDoi"> DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+physicochemical+profiling+for+rational+approach+on+drug+discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of π-π stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 μg/mL and 10 μg/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lgEEUuF3VR0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520physicochemical%2520profiling%2520for%2520rational%2520approach%2520on%2520drug%2520discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeyasu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oizumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Studies of Novel 1, 4-Dihydropyridine Ca Antagonist CS-905. I. Measurement of Partition Coefficient (logP) by High Performance Liquid Chromatography (HPLC)</span>. <i>Yakugaku Zasshi</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">213</span>, <span class="refDoi"> DOI: 10.1248/yakushi1947.115.3_213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fyakushi1947.115.3_213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=7738779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADyaK2MXmvFWksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1995&pages=213&author=K.+Masumotoauthor=A.+Takeyasuauthor=K.+Oizumiauthor=T.+Kobayashi&title=Studies+of+Novel+1%2C+4-Dihydropyridine+Ca+Antagonist+CS-905.+I.+Measurement+of+Partition+Coefficient+%28logP%29+by+High+Performance+Liquid+Chromatography+%28HPLC%29&doi=10.1248%2Fyakushi1947.115.3_213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 1,4-dihydropyridine Ca antagonist CS-905. I. Measurement of partition coefficient (logP) by high performance liquid chromatography (HPLC)</span></div><div class="casAuthors">Masumoto, Kazue; Takeyasu, Akira; Oizumi, Kiyoshi; Kobayashi, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Yakugaku Zasshi</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">213-20</span>CODEN:
                <span class="NLM_cas:coden">YKKZAJ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6903</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A new 1,4-dihydropyridine deriv., CS-905, currently under development as a Ca antagonist, showed a gradual onset and long duration in its antihypertensive effect upon single oral administration.  The partition coeff. for CS-905 was measured along with 11 known dihydropyridine Ca antagonists by the HPLC method to clarify its mode of action.  The log PHPLC for CS-905 was 5.18, almost the same as that of manidipine and this value is the highest among the measured drugs.  Its pharmacokinetic profile in SHR was also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSNcDUIuEYoLVg90H21EOLACvtfcHk0lgEEUuF3VR0rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvFWksL4%253D&md5=41fdae8c51d78f930ae1ce053d6cf512</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1248%2Fyakushi1947.115.3_213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi1947.115.3_213%26sid%3Dliteratum%253Aachs%26aulast%3DMasumoto%26aufirst%3DK.%26aulast%3DTakeyasu%26aufirst%3DA.%26aulast%3DOizumi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStudies%2520of%2520Novel%25201%252C%25204-Dihydropyridine%2520Ca%2520Antagonist%2520CS-905.%2520I.%2520Measurement%2520of%2520Partition%2520Coefficient%2520%2528logP%2529%2520by%2520High%2520Performance%2520Liquid%2520Chromatography%2520%2528HPLC%2529%26jtitle%3DYakugaku%2520Zasshi%26date%3D1995%26volume%3D115%26spage%3D213%26doi%3D10.1248%2Fyakushi1947.115.3_213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, T.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of (4R)-1-{3-[2-(18F)Fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one ([18F]T-401) as a Novel Positron-Emission Tomography Imaging Agent for Monoacylglycerol Lipase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2362</span>– <span class="NLM_lpage">2375</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2362-2375&author=Y.+Hattoriauthor=K.+Aoyamaauthor=J.+Maedaauthor=N.+Arimuraauthor=Y.+Takahashiauthor=M.+Sasakiauthor=M.+Fujinagaauthor=C.+Sekiauthor=Y.+Nagaiauthor=K.+Kawamuraauthor=T.+Yamasakiauthor=M.-R.+Zhangauthor=M.+Higuchiauthor=T.+Koike&title=Design%2C+Synthesis%2C+and+Evaluation+of+%284R%29-1-%7B3-%5B2-%2818F%29Fluoro-4-methylpyridin-3-yl%5Dphenyl%7D-4-%5B4-%281%2C3-thiazol-2-ylcarbonyl%29piperazin-1-yl%5Dpyrrolidin-2-one+%28%5B18F%5DT-401%29+as+a+Novel+Positron-Emission+Tomography+Imaging+Agent+for+Monoacylglycerol+Lipase&doi=10.1021%2Facs.jmedchem.8b01576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of (4R)-1-{3-[2-(18F)Fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one ([18F]T-401) as a Novel Positron-Emission Tomography Imaging Agent for Monoacylglycerol Lipase</span></div><div class="casAuthors">Hattori, Yasushi; Aoyama, Kazunobu; Maeda, Jun; Arimura, Naoto; Takahashi, Yasuko; Sasaki, Masako; Fujinaga, Masayuki; Seki, Chie; Nagai, Yuji; Kawamura, Kazunori; Yamasaki, Tomoteru; Zhang, Ming-Rong; Higuchi, Makoto; Koike, Tatsuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2362-2375</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) is a cytosolic serine hydrolase involved in endocannabinoid and inflammatory signaling.  Positron-emission tomog. (PET) imaging of MAGL serves to validate target engagement of therapeutic MAGL inhibitors as well as to investigate MAGL levels under normal and disease conditions.  However, PET radioligands with reversible binding kinetics for MAGL, which allow quant. assessment of MAGL, are hitherto unavailable.  In this study, we designed and synthesized fluoro-contg. PET probes starting from a recently identified piperazinyl pyrrolidine-2-one deriv. with reversible binding to MAGL.  By tailoring the lipophilicity of the mol. to optimize nonspecific binding and blood-brain barrier permeability, we successfully identified two compds. that show high uptake to regions enriched with MAGL.  PET imaging of wild-type and MAGL-deficient mice as well as a macaque monkey indicated that [18F]I ((4R)-1-{3-[2-(18F)fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one, [18F]T-401) specifically binds to MAGL with adequate reversibility, yielding a high contrast for MAGL within an appropriate imaging time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppgW6fA1E8WLVg90H21EOLACvtfcHk0lgubLjbLByrUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSht7k%253D&md5=4848fa8fce411e960b580b5bd007e0c9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01576%26sid%3Dliteratum%253Aachs%26aulast%3DHattori%26aufirst%3DY.%26aulast%3DAoyama%26aufirst%3DK.%26aulast%3DMaeda%26aufirst%3DJ.%26aulast%3DArimura%26aufirst%3DN.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DC.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DKawamura%26aufirst%3DK.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DHiguchi%26aufirst%3DM.%26aulast%3DKoike%26aufirst%3DT.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520%25284R%2529-1-%257B3-%255B2-%252818F%2529Fluoro-4-methylpyridin-3-yl%255Dphenyl%257D-4-%255B4-%25281%252C3-thiazol-2-ylcarbonyl%2529piperazin-1-yl%255Dpyrrolidin-2-one%2520%2528%255B18F%255DT-401%2529%2520as%2520a%2520Novel%2520Positron-Emission%2520Tomography%2520Imaging%2520Agent%2520for%2520Monoacylglycerol%2520Lipase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2362%26epage%3D2375%26doi%3D10.1021%2Facs.jmedchem.8b01576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span> <span> </span><span class="NLM_article-title">Impact of Physicochemical Profiling for Rational Approach on Drug Discovery</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1238</span>, <span class="refDoi"> DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228-1238&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+Physicochemical+Profiling+for+Rational+Approach+on+Drug+Discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of π-π stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 μg/mL and 10 μg/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lgubLjbLByrUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520Physicochemical%2520Profiling%2520for%2520Rational%2520Approach%2520on%2520Drug%2520Discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26epage%3D1238%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebihara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, S.</span></span> <span> </span><span class="NLM_article-title">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1007/s11095-006-0285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1007%2Fs11095-006-0285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=16779700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=1460-1472&author=T.+Takeuchiauthor=S.+Yoshitomiauthor=T.+Higuchiauthor=K.+Ikemotoauthor=S.-I.+Niwaauthor=T.+Ebiharaauthor=M.+Katohauthor=T.+Yokoiauthor=S.+Asahi&title=Establishment+and+Characterization+of+the+Transformants+Stably-Expressing+MDR1+Derived+from+Various+Animal+Species+in+LLC-PK1&doi=10.1007%2Fs11095-006-0285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span></div><div class="casAuthors">Takeuchi, Toshiyuki; Yoshitomi, Sumie; Higuchi, Tomoaki; Ikemoto, Keiko; Niwa, Shin-Ichi; Ebihara, Takuya; Katoh, Miki; Yokoi, Tsuyoshi; Asahi, Satoru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Stable transformants expressing human multidrug resistance 1 (MDR1), monkey MDR1, canine MDR1, rat MDR1a, rat MDR1b, mouse mdr1a, and mouse mdr1b in LLC-PK1 were established to investigate species differences in P-glycoprotein (P-gp, ABCB1) mediated efflux activity.  Methods: The seven cDNAs of MDR1 from five animals were cloned, and their transformants stably expressing the series of MDR1 in LLC-PK1 were established.  Transport studies of clarithromycin, daunorubicin, digoxin, erythromycin, etoposide, paclitaxel, propranolol, quinidine, ritonavir, saquinavir, verapamil, and vinblastine were performed by using these cells, and efflux activity was compared among the species.  Results: Except for propranolol, all compds. showed efflux activity in all transformants, and were judged to be substrates of P-gp.  There were slight interspecies and interisoforms differences in the substrate recognition.  However, the efflux ratio among the series of the MDR1 stably expressing cells showed good correlation as represented between human and monkey MDR1, and poor correlation as represented between human MDR1 and mouse mdr1a, and human and canine MDR1.  Conclusions: Results in the present study indicate that all MDR1 stably expressing cells have efflux activity for various P-gp substrates, and that interspecies differences and similarities of the P-gp substrate efflux activity may exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28IPlZqwtHLVg90H21EOLACvtfcHk0lgubLjbLByrUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D&md5=a21044c202d7ff2eb8ab27c4623909de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-0285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-0285-7%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DIkemoto%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DS.-I.%26aulast%3DEbihara%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DT.%26aulast%3DAsahi%26aufirst%3DS.%26atitle%3DEstablishment%2520and%2520Characterization%2520of%2520the%2520Transformants%2520Stably-Expressing%2520MDR1%2520Derived%2520from%2520Various%2520Animal%2520Species%2520in%2520LLC-PK1%26jtitle%3DPharm.%2520Res.%26date%3D2006%26volume%3D23%26spage%3D1460%26epage%3D1472%26doi%3D10.1007%2Fs11095-006-0285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rohlfing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroenke, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowden, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfefferbaum, A.</span></span> <span> </span><span class="NLM_article-title">The INIA19 Template and NeuroMaps Atlas for Primate Brain Image Parcellation and Spatial Normalization</span>. <i>Front. Neuroinf.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">27</span>, <span class="refDoi"> DOI: 10.3389/fninf.2012.00027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.3389%2Ffninf.2012.00027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=23230398" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=27&author=T.+Rohlfingauthor=C.+D.+Kroenkeauthor=E.+V.+Sullivanauthor=M.+F.+Dubachauthor=D.+M.+Bowdenauthor=K.+A.+Grantauthor=A.+Pfefferbaum&title=The+INIA19+Template+and+NeuroMaps+Atlas+for+Primate+Brain+Image+Parcellation+and+Spatial+Normalization&doi=10.3389%2Ffninf.2012.00027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3389%2Ffninf.2012.00027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffninf.2012.00027%26sid%3Dliteratum%253Aachs%26aulast%3DRohlfing%26aufirst%3DT.%26aulast%3DKroenke%26aufirst%3DC.%2BD.%26aulast%3DSullivan%26aufirst%3DE.%2BV.%26aulast%3DDubach%26aufirst%3DM.%2BF.%26aulast%3DBowden%26aufirst%3DD.%2BM.%26aulast%3DGrant%26aufirst%3DK.%2BA.%26aulast%3DPfefferbaum%26aufirst%3DA.%26atitle%3DThe%2520INIA19%2520Template%2520and%2520NeuroMaps%2520Atlas%2520for%2520Primate%2520Brain%2520Image%2520Parcellation%2520and%2520Spatial%2520Normalization%26jtitle%3DFront.%2520Neuroinf.%26date%3D2012%26volume%3D6%26spage%3D27%26doi%3D10.3389%2Ffninf.2012.00027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of known LSD1 inhibitors as antitumor agents, FAD and <b>1</b>-FAD adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structures of T-448 and the formylated-FAD adduct.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Novel PET imaging agents for LSD1 in the brain.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the <i>N</i>-(4-(2-Aminocyclopropyl)phenyl)amide Derivative <b>1a</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2,2-trichloroethyl chloroformate, TEA, THF, 66%; (b) HCl, CPME, quant; (c) (1) cyclopropanecarbaldehyde, NaHCO<sub>3</sub>, THF/MeOH, (2) NaBH<sub>4</sub>, and (3) Boc<sub>2</sub>O, 74%; (d) Zn, AcOH, THF, 68%; (e) benzoyl chloride, TEA, THF, 99%; and (f) HCl, CPME, 77%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4-(2-aminocyclopropyl)benzamide Derivatives <b>1b–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) trimethylsulfoxonium iodide, KO<i>t</i>Bu, DMSO, 57%; (b) TFA, 87%; (c) DPPA, TEA, <i>t</i>BuOH, toluene, 85%; (d) optical resolution by HPLC (Column: CHIRALPAK AD, desired isomer is tR2), 35%; (e) NaOH, MeOH, <b>13a</b>: 95%, <b>13b</b>: 96%; (f) corresponding amine, EDCI, HOBt, TEA, DMF for <b>14a</b>, 61%; (g) corresponding amine, HATU, TEA, DMF for <b>14b</b> and <b>14c</b>, <b>14b</b>: 67%, <b>14c</b>: 96%; (h) HCl, EtOAc, <b>15a</b>: quant, <b>15b</b>: quant, <b>15c</b>: 99%; and (i) (1) cyclopropanecarbaldehyde, NaHCO<sub>3</sub>, THF/MeOH and (2) NaBH<sub>4</sub>, <b>1b</b>: 33%, <b>1c</b>: 34%, <b>1d</b>: 32%.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>1e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (bromomethyl)cyclopropane, NaH, DMF, 87%; (b) CO (3 atm), PdCl<sub>2</sub> (dppf), TEA, MeOH, 87%; (c) NaOH, water/MeOH, 90%; (d) optical resolution by HPLC (Column: CHIRALPAK AD-H, desired isomer is tR2), 46%; (e) (COCl)<sub>2</sub>, DMF, THF, and then 6-fluoro-<i>N</i>-methylpyridin-3-amine, DMAP, pyridine, THF, quant; and (f) HCl, EtOAc, 76%.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of [<sup>18</sup>F]<b>1e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (COCl)<sub>2</sub>, DMF, THF, and then 6-nitro-3-pyridinamine, DMAP, pyridine, THF, 33%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 89%; and (c) K[<sup>18</sup>F]F, 2,2,2-cryptand, K<sub>2</sub>CO<sub>3</sub>, DMSO, 160 °C, 10 min, and then 1 M HCl solution (0.5 mL), 90 °C, 4 min; radiochemical purity, >98%; molar activity, 268 ± 206 GBq/μmol (<i>n</i> = 5).</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Average [<sup>18</sup>F]<b>1e</b> PET images of a monkey brain between 0 and 30 min (middle row) and between 30 and 240 min postinjection (bottom row) in SUV units. Horizontal sections containing the corona radiate (left), sagittal sections near the midline (middle), and coronal sections through the orbits (right) are superimposed on the corresponding individual MRI data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. TAC (SUV; g/mL) of [<sup>18</sup>F]<b>1e</b> in a monkey brain between 0 and 240 min postinjection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. LC/MS spectra of FAD and <b>1a</b>-FAD adduct in the human recombinant LSD1 treated with <b>1e</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/medium/jm0c01937_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Distribution patterns of LSD1 in the regions of human, monkey, and rat brains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-7/acs.jmedchem.0c01937/20210331/images/large/jm0c01937_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01937&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76581" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76581" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Chromatin modifications and their function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0liBWoh0V-cmeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varier, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmers, H. T. M.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>1815</i></span>,  <span class="NLM_fpage">75</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.bbcan.2010.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.bbcan.2010.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=20951770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+M.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer&doi=10.1016%2Fj.bbcan.2010.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine methylation and demethylation pathways in cancer</span></div><div class="casAuthors">Varier, Radhika A.; Timmers, H. T. Marc</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1815</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-89</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The genetic changes leading to the development of human cancer are accompanied by alterations in the structure and modification status of chromatin, which represent powerful regulatory mechanisms for gene expression and genome stability.  These epigenetic alterations have sparked interest into deciphering the regulatory pathways and function of post-translational modifications of histones during the initiation and progression of cancer.  In this review we describe and summarize the current knowledge of several histone lysine methyltransferase and demethylase pathways relevant to cancer.  Mechanistic insight into histone modifications will pave the way for the development and therapeutic application of "epidrugs" in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MyfXQs2YR7Vg90H21EOLACvtfcHk0liGWBHPxt5iJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D&md5=3331b133c4f1b15beab600420bb859f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%2BM.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89%26doi%3D10.1016%2Fj.bbcan.2010.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kooistra, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">297</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1038/nrm3327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases&doi=10.1038%2Fnrm3327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0liGWBHPxt5iJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311%26doi%3D10.1038%2Fnrm3327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emily, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melissa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debasis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephen, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ian, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcelo, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, M.</span></span> <span> </span><span class="NLM_article-title">Hooker Brain-penetrant LSD1 inhibitors can block memory consolidation</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">128</span></span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=N.+Rameshauthor=L.+R.+Emilyauthor=M.+Melissaauthor=P.+Debasisauthor=N.+Stephanieauthor=M.+C.+Stephenauthor=T.+H.+Ianauthor=A.+W.+Marceloauthor=J.+H.+Stephenauthor=M.+Jacob&title=Hooker+Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DRamesh%26aufirst%3DN.%26aulast%3DEmily%26aufirst%3DL.%2BR.%26aulast%3DMelissa%26aufirst%3DM.%26aulast%3DDebasis%26aufirst%3DP.%26aulast%3DStephanie%26aufirst%3DN.%26aulast%3DStephen%26aufirst%3DM.%2BC.%26aulast%3DIan%26aufirst%3DT.%2BH.%26aulast%3DMarcelo%26aufirst%3DA.%2BW.%26aulast%3DStephen%26aufirst%3DJ.%2BH.%26aulast%3DJacob%26aufirst%3DM.%26atitle%3DHooker%2520Brain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Network
& Pathway Analysis Subgroup of Psychiatric Genomics Consortium</span> <span> </span><span class="NLM_article-title">Psychiatric genome-wide association
study analyses implicate neuronal, immune and histone pathways</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">209</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fnn.3922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=25599223" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=199-209&author=Network%0A%26+Pathway+Analysis+Subgroup+of+Psychiatric+Genomics+Consortium&title=Psychiatric+genome-wide+association%0Astudy+analyses+implicate+neuronal%2C+immune+and+histone+pathways"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnn.3922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3922%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DPsychiatric%2520genome-wide%2520association%250Astudy%2520analyses%2520implicate%2520neuronal%252C%2520immune%2520and%2520histone%2520pathways%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D199%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matevossian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarian, S.</span></span> <span> </span><span class="NLM_article-title">Prefrontal dysfunction in schizophrenia involves mixed-lineage leukemia 1-regulated histone methylation at GABAergic gene promoters</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">11254</span>– <span class="NLM_lpage">11262</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.3272-07.2007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1523%2FJNEUROSCI.3272-07.2007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17942719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1ekur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=11254-11262&author=H.-S.+Huangauthor=A.+Matevossianauthor=C.+Whittleauthor=S.+Y.+Kimauthor=A.+Schumacherauthor=S.+P.+Bakerauthor=S.+Akbarian&title=Prefrontal+dysfunction+in+schizophrenia+involves+mixed-lineage+leukemia+1-regulated+histone+methylation+at+GABAergic+gene+promoters&doi=10.1523%2Fjneurosci.3272-07.2007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Prefrontal dysfunction in schizophrenia involves mixed-lineage leukemia 1-regulated histone methylation at GABAergic gene promoters</span></div><div class="casAuthors">Huang, Hsien-Sung; Matevossian, Anouch; Whittle, Catheryne; Kim, Se Young; Schumacher, Armin; Baker, Stephen P.; Akbarian, Schahram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11254-11262</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Alterations in GABAergic mRNA expression play a key role for prefrontal dysfunction in schizophrenia and other neurodevelopmental disease.  Here, we show that histone H3-lysine 4 methylation, a chromatin mark assocd. with the transcriptional process, progressively increased at GAD1 and other GABAergic gene promoters (GAD2, NPY, SST) in human prefrontal cortex (PFC) from prenatal to peripubertal ages and throughout adulthood.  Alterations in schizophrenia included decreased GAD1 expression and H3K4-trimethylation, predominantly in females and in conjunction with a risk haplotype at the 5' end of GAD1.  Heterozygosity for a truncated, lacZ knock-in allele of mixed-lineage leukemia 1 (Mll1), a histone methyltransferase expressed in GABAergic and other cortical neurons, resulted in decreased H3K4 methylation at GABAergic gene promoters.  In contrast, Gad1 H3K4 (tri)methylation and Mll1 occupancy was increased in cerebral cortex of mice after treatment with the atypical antipsychotic, clozapine.  These effects were not mimicked by haloperidol or genetic ablation of dopamine D2 and D3 receptors, suggesting that blockade of D2-like signaling is not sufficient for clozapine-induced histone methylation.  Therefore, chromatin remodeling mechanisms at GABAergic gene promoters, including MLL1-mediated histone methylation, operate throughout an extended period of normal human PFC development and play a role in the neurobiol. of schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK8Vzt1G1AL7Vg90H21EOLACvtfcHk0ljQts6O5nZ8pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1ekur3J&md5=be3431b2d2b456e9d6d5a9d21460dda6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3272-07.2007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3272-07.2007%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-S.%26aulast%3DMatevossian%26aufirst%3DA.%26aulast%3DWhittle%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DSchumacher%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DS.%2BP.%26aulast%3DAkbarian%26aufirst%3DS.%26atitle%3DPrefrontal%2520dysfunction%2520in%2520schizophrenia%2520involves%2520mixed-lineage%2520leukemia%25201-regulated%2520histone%2520methylation%2520at%2520GABAergic%2520gene%2520promoters%26jtitle%3DJ.%2520Neurosci.%26date%3D2007%26volume%3D27%26spage%3D11254%26epage%3D11262%26doi%3D10.1523%2Fjneurosci.3272-07.2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shulha, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virgil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Z.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Signatures of Autism</span>. <i>Arch. Gen. Psychiatr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1001/archgenpsychiatry.2011.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1001%2Farchgenpsychiatry.2011.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22065254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFSkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2012&pages=314-324&author=H.+P.+Shulhaauthor=I.+Cheungauthor=C.+Whittleauthor=J.+Wangauthor=D.+Virgilauthor=C.+L.+Linauthor=Y.+Guoauthor=A.+Lessardauthor=S.+Akbarianauthor=Z.+Weng&title=Epigenetic+Signatures+of+Autism&doi=10.1001%2Farchgenpsychiatry.2011.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic signatures of autism: trimethylated H3K4 landscapes in prefrontal neurons</span></div><div class="casAuthors">Shulha, Hennady P.; Cheung, Iris; Whittle, Catheryne; Wang, Jie; Virgil, Daniel; Lin, Cong L.; Guo, Yin; Lessard, Andree; Akbarian, Schahram; Weng, Zhiping</div><div class="citationInfo"><span class="NLM_cas:title">Archives of General Psychiatry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">314-324</span>CODEN:
                <span class="NLM_cas:coden">ARGPAQ</span>;
        ISSN:<span class="NLM_cas:issn">0003-990X</span>.
    
            (<span class="NLM_cas:orgname">American Medical Association</span>)
        </div><div class="casAbstract">Neuronal dysfunction in cerebral cortex and other brain regions could contribute to the cognitive and behavioral defects in autism.  The objective of the study was to characterize epigenetic signatures of autism in prefrontal cortex neurons.  We performed fluorescence-activated sorting and sepn. of neuronal and nonneuronal nuclei from postmortem prefrontal cortex, digested the chromatin with micrococcal nuclease, and deeply sequenced the DNA from the mononucleosomes with trimethylated H3K4 (H3K4me3), a histone mark assocd. with transcriptional regulation.  Approx. 15 billion base pairs of H3K4me3-enriched sequences were collected from 32 brains.  Participants: A total of 16 subjects diagnosed as having autism and 16 control subjects ranging in age from 0.5 to 70 years participated in the study at the academic medical center.  Identification of genomic loci showing autism-assocd. H3K4me3 changes in prefrontal cortex neurons was carried out.  Subjects with autism showed no evidence for generalized disruption of the developmentally regulated remodeling of the H3K4me3 landscape that defines normal prefrontal cortex neurons in early infancy.  However, excess spreading of H3K4me3 from the transcription start sites into downstream gene bodies and upstream promoters was obsd. specifically in neuronal chromatin from 4 of 16 autism cases but not in controls.  Variable subsets of autism cases exhibit altered H3K4me3 peaks at numerous genes regulating neuronal connectivity, social behaviors, and cognition, often in conjunction with altered expression of the corresponding transcripts.  Autism-assocd. H3K4me3 peaks were significantly enriched in genes and loci implicated in neurodevelopmental diseases.  Prefrontal cortex neurons from subjects with autism show changes in chromatin structures at hundreds of loci genome-wide, revealing considerable overlap between genetic and epigenetic risk maps of developmental brain disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRlm7JvRmMurVg90H21EOLACvtfcHk0ljQts6O5nZ8pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFSkt7Y%253D&md5=5d60b2308b7cbfb610f2fd4f12c1d96f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1001%2Farchgenpsychiatry.2011.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchgenpsychiatry.2011.151%26sid%3Dliteratum%253Aachs%26aulast%3DShulha%26aufirst%3DH.%2BP.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DWhittle%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVirgil%26aufirst%3DD.%26aulast%3DLin%26aufirst%3DC.%2BL.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DLessard%26aufirst%3DA.%26aulast%3DAkbarian%26aufirst%3DS.%26aulast%3DWeng%26aufirst%3DZ.%26atitle%3DEpigenetic%2520Signatures%2520of%2520Autism%26jtitle%3DArch.%2520Gen.%2520Psychiatr.%26date%3D2012%26volume%3D69%26spage%3D314%26epage%3D324%26doi%3D10.1001%2Farchgenpsychiatry.2011.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wynde, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doughty, M. L.</span></span> <span> </span><span class="NLM_article-title">Role of H3K4 demethylases in complex neurodevelopmental diseases</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.2217/epi.10.12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.2217%2Fepi.10.12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=22121901" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=407-418&author=C.+Wyndeauthor=L.+Stalkerauthor=M.+L.+Doughty&title=Role+of+H3K4+demethylases+in+complex+neurodevelopmental+diseases&doi=10.2217%2Fepi.10.12"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2217%2Fepi.10.12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.10.12%26sid%3Dliteratum%253Aachs%26aulast%3DWynde%26aufirst%3DC.%26aulast%3DStalker%26aufirst%3DL.%26aulast%3DDoughty%26aufirst%3DM.%2BL.%26atitle%3DRole%2520of%2520H3K4%2520demethylases%2520in%2520complex%2520neurodevelopmental%2520diseases%26jtitle%3DEpigenomics%26date%3D2010%26volume%3D2%26spage%3D407%26epage%3D418%26doi%3D10.2217%2Fepi.10.12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parkel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Atalaya, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barco, A.</span></span> <span> </span><span class="NLM_article-title">Histone H3 lysine methylation in cognition and intellectual disability disorders</span>. <i>Learn. Mem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">570</span>– <span class="NLM_lpage">579</span>, <span class="refDoi"> DOI: 10.1101/lm.029363.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1101%2Flm.029363.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24045506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFeis77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=570-579&author=S.+Parkelauthor=J.+P.+Lopez-Atalayaauthor=A.+Barco&title=Histone+H3+lysine+methylation+in+cognition+and+intellectual+disability+disorders&doi=10.1101%2Flm.029363.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 lysine methylation in cognition and intellectual disability disorders</span></div><div class="casAuthors">Parkel, Sven; Lopez-Atalaya, Jose P.; Barco, Angel</div><div class="citationInfo"><span class="NLM_cas:title">Learning & Memory</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">570-579</span>CODEN:
                <span class="NLM_cas:coden">LEMEFO</span>;
        ISSN:<span class="NLM_cas:issn">1072-0502</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Recent research indicates that epigenetic mechanisms and, in particular, the post-translational modification (PTM) of histories may contribute to memory encoding and storage.  Among the dozens of possible histone PTMs, the methylation/demethylation of lysines in the N-terminal tall of histone H3 exhibits particularly strong links with cognitive abilities.  First, the persistence and tight assocn. with distinct transcriptional states of the gene make these modifications particularly suitable for being part of the mol. underpinnings of memory storage.  Second, correlative evidence indicates that the methylation/demethylation of lysines in histone H3 is actively regulated during memory processes.  Third, several enzymes regulating these PTMs are assocd. with intellectual disability disorders.  We review here these three lines of evidence and discuss the potential role of epigenetic mechanisms centered on the methylation of lysine residues on histone H3 in neuroplasticity and neurodevelopmental disorders assocd. with intellectual disability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroEcFk0SSlPrVg90H21EOLACvtfcHk0ljQts6O5nZ8pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFeis77K&md5=a4d679f8e3bf2459bacd051669b1f3cc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Flm.029363.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Flm.029363.112%26sid%3Dliteratum%253Aachs%26aulast%3DParkel%26aufirst%3DS.%26aulast%3DLopez-Atalaya%26aufirst%3DJ.%2BP.%26aulast%3DBarco%26aufirst%3DA.%26atitle%3DHistone%2520H3%2520lysine%2520methylation%2520in%2520cognition%2520and%2520intellectual%2520disability%2520disorders%26jtitle%3DLearn.%2520Mem.%26date%3D2013%26volume%3D20%26spage%3D570%26epage%3D579%26doi%3D10.1101%2Flm.029363.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vashishtha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gipson, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratter, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadorf, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel-Ciernia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troncosco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krainc, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadri-Vakili, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraenkel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span> <span> </span><span class="NLM_article-title">Targeting H3K4 trimethylation in Huntington disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">E3027</span>– <span class="NLM_lpage">E3036</span>, <span class="refDoi"> DOI: 10.1073/pnas.1311323110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1073%2Fpnas.1311323110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=23872847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVShsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=E3027-E3036&author=M.+Vashishthaauthor=C.+W.+Ngauthor=F.+Yildirimauthor=T.+A.+Gipsonauthor=I.+H.+Kratterauthor=L.+Bodaiauthor=W.+Songauthor=A.+Lauauthor=A.+Labadorfauthor=A.+Vogel-Cierniaauthor=J.+Troncoscoauthor=C.+A.+Rossauthor=G.+P.+Batesauthor=D.+Kraincauthor=G.+Sadri-Vakiliauthor=S.+Finkbeinerauthor=J.+L.+Marshauthor=D.+E.+Housmanauthor=E.+Fraenkelauthor=L.+M.+Thompson&title=Targeting+H3K4+trimethylation+in+Huntington+disease&doi=10.1073%2Fpnas.1311323110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting H3K4 trimethylation in Huntington disease</span></div><div class="casAuthors">Vashishtha, Malini; Ng, Christopher W.; Yildirim, Ferah; Gipson, Theresa A.; Kratter, Ian H.; Bodai, Laszlo; Song, Wan; Lau, Alice; Labadorf, Adam; Vogel-Ciernia, Annie; Troncosco, Juan; Ross, Christopher A.; Bates, Gillian P.; Krainc, Dimitri; Sadri-Vakili, Ghazaleh; Finkbeiner, Steven; Marsh, J. Lawrence; Housman, David E.; Fraenkel, Ernest; Thompson, Leslie M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">E3027-E3036, SE3027/1-SE3027/44</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transcriptional dysregulation is an early feature of Huntington disease (HD).  We obsd. gene-specific changes in histone H3 lysine 4 trimethylation (H3K4me3) at transcriptionally repressed promoters in R6/2 mouse and human HD brain.  Genome-wide anal. showed a chromatin signature for this mark.  Reducing the levels of the H3K4 demethylase SMCX/Jarid1c in primary neurons reversed down-regulation of key neuronal genes caused by mutant Huntingtin expression.  Finally, redn. of SMCX/Jarid1c in primary neurons from BACHD mice or the single Jarid1 in a Drosophila HD model was protective.  Therefore, targeting this epigenetic signature may be an effective strategy to ameliorate the consequences of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAKMx94fdqS7Vg90H21EOLACvtfcHk0lhIlEm5Q4r3_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVShsL%252FN&md5=aa922cf9c891efa8873679deec877230</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1311323110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1311323110%26sid%3Dliteratum%253Aachs%26aulast%3DVashishtha%26aufirst%3DM.%26aulast%3DNg%26aufirst%3DC.%2BW.%26aulast%3DYildirim%26aufirst%3DF.%26aulast%3DGipson%26aufirst%3DT.%2BA.%26aulast%3DKratter%26aufirst%3DI.%2BH.%26aulast%3DBodai%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DLabadorf%26aufirst%3DA.%26aulast%3DVogel-Ciernia%26aufirst%3DA.%26aulast%3DTroncosco%26aufirst%3DJ.%26aulast%3DRoss%26aufirst%3DC.%2BA.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DKrainc%26aufirst%3DD.%26aulast%3DSadri-Vakili%26aufirst%3DG.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DMarsh%26aufirst%3DJ.%2BL.%26aulast%3DHousman%26aufirst%3DD.%2BE.%26aulast%3DFraenkel%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26atitle%3DTargeting%2520H3K4%2520trimethylation%2520in%2520Huntington%2520disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3DE3027%26epage%3DE3036%26doi%3D10.1073%2Fpnas.1311323110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lhIlEm5Q4r3_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philip, A. C.</span></span> <span> </span><span class="NLM_article-title">LSD1 and the chemistry of histone demethylation</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">561</span>– <span class="NLM_lpage">568</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2007.07.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.cbpa.2007.07.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17851108" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=561-568&author=C.+C.+Jeffreyauthor=A.+C.+Philip&title=LSD1+and+the+chemistry+of+histone+demethylation&doi=10.1016%2Fj.cbpa.2007.07.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.014%26sid%3Dliteratum%253Aachs%26aulast%3DJeffrey%26aufirst%3DC.%2BC.%26aulast%3DPhilip%26aufirst%3DA.%2BC.%26atitle%3DLSD1%2520and%2520the%2520chemistry%2520of%2520histone%2520demethylation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D561%26epage%3D568%26doi%3D10.1016%2Fj.cbpa.2007.07.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halene, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlichman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christian, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonak, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gur, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blendy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodkin, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gur, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, S. J.</span></span> <span> </span><span class="NLM_article-title">Assessment of NMDA receptor NR1 subunit hypofunction in mice as a model for schizophrenia</span>. <i>Gene Brain Behav.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1111/j.1601-183x.2009.00504.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1111%2Fj.1601-183X.2009.00504.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19563516" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=661-675&author=T.+B.+Haleneauthor=R.+S.+Ehrlichmanauthor=Y.+Liangauthor=E.+P.+Christianauthor=G.+J.+Jonakauthor=T.+L.+Gurauthor=J.+A.+Blendyauthor=H.+C.+Dowauthor=E.+S.+Brodkinauthor=F.+Schneiderauthor=R.+C.+Gurauthor=S.+J.+Siegel&title=Assessment+of+NMDA+receptor+NR1+subunit+hypofunction+in+mice+as+a+model+for+schizophrenia&doi=10.1111%2Fj.1601-183x.2009.00504.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1111%2Fj.1601-183X.2009.00504.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1601-183X.2009.00504.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalene%26aufirst%3DT.%2BB.%26aulast%3DEhrlichman%26aufirst%3DR.%2BS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DChristian%26aufirst%3DE.%2BP.%26aulast%3DJonak%26aufirst%3DG.%2BJ.%26aulast%3DGur%26aufirst%3DT.%2BL.%26aulast%3DBlendy%26aufirst%3DJ.%2BA.%26aulast%3DDow%26aufirst%3DH.%2BC.%26aulast%3DBrodkin%26aufirst%3DE.%2BS.%26aulast%3DSchneider%26aufirst%3DF.%26aulast%3DGur%26aufirst%3DR.%2BC.%26aulast%3DSiegel%26aufirst%3DS.%2BJ.%26atitle%3DAssessment%2520of%2520NMDA%2520receptor%2520NR1%2520subunit%2520hypofunction%2520in%2520mice%2520as%2520a%2520model%2520for%2520schizophrenia%26jtitle%3DGene%2520Brain%2520Behav.%26date%3D2009%26volume%3D8%26spage%3D661%26epage%3D675%26doi%3D10.1111%2Fj.1601-183x.2009.00504.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohn, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gainetdinov, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, B. H.</span></span> <span> </span><span class="NLM_article-title">Mice with reduced NMDA receptor expression display behaviors related to schizophrenia</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">427</span>– <span class="NLM_lpage">436</span>, <span class="refDoi"> DOI: 10.1016/s0092-8674(00)81972-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2FS0092-8674%2800%2981972-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10481908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADyaK1MXlslWmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1999&pages=427-436&author=A.+R.+Mohnauthor=R.+R.+Gainetdinovauthor=M.+G.+Caronauthor=B.+H.+Koller&title=Mice+with+reduced+NMDA+receptor+expression+display+behaviors+related+to+schizophrenia&doi=10.1016%2Fs0092-8674%2800%2981972-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mice with reduced NMDA receptor expression display behaviors related to schizophrenia</span></div><div class="casAuthors">Mohn, Amy R.; Gainetdinov, Raul R.; Caron, Marc G.; Koller, Beverly H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">427-436</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">N-methyl-D-aspartate receptors (NMDARs) represent a subclass of glutamate receptors that play a crit. role in neuronal development and physiol.  We report here the generation of mice expressing only 5% of normal levels of the essential NMDAR1 (NR1) subunit.  Unlike NR1 null mice, these mice survive to adulthood and display behavioral abnormalities, including increased motor activity and stereotypy and deficits in social and sexual interactions.  These behavioral alterations are similar to those obsd. in pharmacol. induced animal models of schizophrenia and can be ameliorated by treatment with haloperidol or clozapine, antipsychotic drugs that antagonize dopaminergic and serotonergic receptors.  These findings support a model in which reduced NMDA receptor activity results in schizophrenic-like behavior and reveals how pharmacol. manipulation of monoaminergic pathways can affect this phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouZjUiHREyAbVg90H21EOLACvtfcHk0lgoIQ9-NMc-Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlslWmsb0%253D&md5=39d97aaab71d2d9848210eec9e7faa1c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981972-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981972-8%26sid%3Dliteratum%253Aachs%26aulast%3DMohn%26aufirst%3DA.%2BR.%26aulast%3DGainetdinov%26aufirst%3DR.%2BR.%26aulast%3DCaron%26aufirst%3DM.%2BG.%26aulast%3DKoller%26aufirst%3DB.%2BH.%26atitle%3DMice%2520with%2520reduced%2520NMDA%2520receptor%2520expression%2520display%2520behaviors%2520related%2520to%2520schizophrenia%26jtitle%3DCell%26date%3D1999%26volume%3D98%26spage%3D427%26epage%3D436%26doi%3D10.1016%2Fs0092-8674%2800%2981972-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wesseling, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guest, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmoune, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahn, S.</span></span> <span> </span><span class="NLM_article-title">Integrative proteomic analysis of the NMDA NR1 knockdown mouse model reveals effects on central and peripheral pathways associated with schizophrenia and autism spectrum disorders</span>. <i>Mol. Autism.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">38</span>, <span class="refDoi"> DOI: 10.1186/2040-2392-5-38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1186%2F2040-2392-5-38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=25061506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVChsbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=38&author=H.+Wesselingauthor=P.+C.+Guestauthor=C.-M.+Leeauthor=E.+H.+Wongauthor=H.+Rahmouneauthor=S.+Bahn&title=Integrative+proteomic+analysis+of+the+NMDA+NR1+knockdown+mouse+model+reveals+effects+on+central+and+peripheral+pathways+associated+with+schizophrenia+and+autism+spectrum+disorders&doi=10.1186%2F2040-2392-5-38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative proteomic analysis of the NMDA NR1 knockdown mouse model reveals effects on central and peripheral pathways associated with schizophrenia and autism spectrum disorders</span></div><div class="casAuthors">Wesseling, Hendrik; Guest, Paul C.; Lee, Chi-Ming; Wong, Erik H. F.; Rahmoune, Hassan; Bahn, Sabine</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Autism</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">38/1-38/32, 32 pp.</span>CODEN:
                <span class="NLM_cas:coden">MAOUBR</span>;
        ISSN:<span class="NLM_cas:issn">2040-2392</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Over the last decade, the transgenic N-methyl-d-aspartate receptor (NMDAR) NR1-knockdown mouse (NR1neo-/-) has been investigated as a glutamate hypofunction model for schizophrenia.  Recent research has now revealed that the model also recapitulates cognitive and neg. symptoms in the continuum of other psychiatric diseases, particularly autism spectrum disorders (ASD).  As previous studies have mostly focussed on behavioral readouts, a mol. characterization of this model will help to identify novel biomarkers or potential drug targets.  Methods: Here, we have used multiplex immunoassay analyses to investigate peripheral analyte alterations in serum of NR1neo-/- mice, as well as a combination of shotgun label-free liq. chromatog. mass spectrometry, bioinformatic pathway analyses, and a shotgun-based 40-plex selected reaction monitoring (SRM) assay to investigate altered mol. pathways in the frontal cortex and hippocampus.  All findings were cross compared to identify translatable findings between the brain and periphery.  Results: Multiplex immunoassay profiling led to identification of 29 analytes that were significantly altered in sera of NR1neo-/- mice.  The highest magnitude changes were found for neurotrophic factors (VEGFA, EGF, IGF-1), apolipoprotein A1, and fibrinogen.  We also found decreased levels of several chemokines.  Following this, LC-MSE profiling led to identification of 48 significantly changed proteins in the frontal cortex and 41 in the hippocampus.  In particular, MARCS, the mitochondrial pyruvate kinase, and CamKII-alpha were affected.  Based on the combination of protein set enrichment and bioinformatic pathway anal., we designed orthogonal SRM-assays which validated the abnormalities of proteins involved in synaptic long-term potentiation, myelination, and the ERK-signalling pathway in both brain regions.  In contrast, increased levels of proteins involved in neurotransmitter metab. and release were found only in the frontal cortex and abnormalities of proteins involved in the purinergic system were found exclusively in the hippocampus.  Conclusions: Taken together, this multi-platform profiling study has identified peripheral changes which are potentially linked to central alterations in synaptic plasticity and neuronal function assocd. with NMDAR-NR1 hypofunction.  Therefore, the reported proteomic changes may be useful as translational biomarkers in human and rodent model drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_PWLtT7GYpLVg90H21EOLACvtfcHk0lgoIQ9-NMc-Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVChsbnM&md5=bdef8cecd164673870cb7537d0feb447</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1186%2F2040-2392-5-38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F2040-2392-5-38%26sid%3Dliteratum%253Aachs%26aulast%3DWesseling%26aufirst%3DH.%26aulast%3DGuest%26aufirst%3DP.%2BC.%26aulast%3DLee%26aufirst%3DC.-M.%26aulast%3DWong%26aufirst%3DE.%2BH.%26aulast%3DRahmoune%26aufirst%3DH.%26aulast%3DBahn%26aufirst%3DS.%26atitle%3DIntegrative%2520proteomic%2520analysis%2520of%2520the%2520NMDA%2520NR1%2520knockdown%2520mouse%2520model%2520reveals%2520effects%2520on%2520central%2520and%2520peripheral%2520pathways%2520associated%2520with%2520schizophrenia%2520and%2520autism%2520spectrum%2520disorders%26jtitle%3DMol.%2520Autism.%26date%3D2014%26volume%3D5%26spage%3D38%26doi%3D10.1186%2F2040-2392-5-38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smitheman, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamat, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federowicz, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butticello, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnette, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degenhardt, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNulty, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rouse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDevitt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crouthamel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Concha, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span> <span> </span><span class="NLM_article-title">A DNA hypomethylation signature predicts antitumor activity of LSD1 inhibitors in SCLC</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+hypomethylation+signature+predicts+antitumor+activity+of+LSD1+inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhmZinqRiH_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520hypomethylation%2520signature%2520predicts%2520antitumor%2520activity%2520of%2520LSD1%2520inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallipoli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J. P.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulators as promising therapeutic targets in acute myeloid leukemia</span>. <i>Ther. Adv. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1177/2040620715577614</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1177%2F2040620715577614" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=26137202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGhsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=103-119&author=P.+Gallipoliauthor=G.+Giotopoulosauthor=B.+J.+P.+Huntly&title=Epigenetic+regulators+as+promising+therapeutic+targets+in+acute+myeloid+leukemia&doi=10.1177%2F2040620715577614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulators as promising therapeutic targets in acute myeloid leukemia</span></div><div class="casAuthors">Gallipoli, Paolo; Giotopoulos, George; Huntly, Brian J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Therapeutic Advances in Hematology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">103-119</span>CODEN:
                <span class="NLM_cas:coden">TAHHAT</span>;
        ISSN:<span class="NLM_cas:issn">2040-6207</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications Ltd.</span>)
        </div><div class="casAbstract">A review.  Acute myeloid leukemia (AML) the most prevalent acute leukemia in adults, is an aggressive hematol. malignancy arising in hematopoietic stem and progenitor cells.  With the exception of a few specific AML subtypes, the mainstays of treatment have not significantly changed over the last 20 years, and are still based on std. cytotoxic chemotherapy.  As a result, clin. outcome remains poor for the majority of patients, with overall long-term survival in the region of 20-30%.  Recent successes in characterizing the genetic Landscape of AML have highlighted that, despite its heterogeneity, many cases of AML carry recurrent mutations in genes encoding epigenetic regulators.  Transcriptional dysregulation and altered epigenetic function have therefore emerged as exciting areas in AML research and it is becoming increasingly clear that epigenetic dysfunction is central to leukemogenesis in AML.  This has subsequently paved the way for the development of epigenetically targeted therapies.  In this review, we will discuss the most recent advances in our understanding of the role of epigenetic dysregulation in AML pathobiol.  We will particularly focus on those altered epigenetic programs that have been shown to be central to the development and maintenance of AML in prectinical models.  We will discuss the recent development of therapeutics specifically targeting these key epigenetic programs in AML, describe their mechanism of action and present their current clin. development.  Finally, we will discuss the opportunities presented by epigenetically targeted therapy in AML and will highlight future challenges ahead for the AML community, to ensure that these novel therapeutics are optimally translated into clin. practice and result in clin. improvement for AML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4US2Qy98czLVg90H21EOLACvtfcHk0lhmZinqRiH_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGhsbfJ&md5=4cb560fcfb7712b0cd19302eda963782</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1177%2F2040620715577614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040620715577614%26sid%3Dliteratum%253Aachs%26aulast%3DGallipoli%26aufirst%3DP.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26atitle%3DEpigenetic%2520regulators%2520as%2520promising%2520therapeutic%2520targets%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DTher.%2520Adv.%2520Hematol.%26date%3D2015%26volume%3D6%26spage%3D103%26epage%3D119%26doi%3D10.1177%2F2040620715577614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takagi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyoshima, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dairiki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hikichi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, K.</span></span> <span> </span><span class="NLM_article-title">A Novel LSD1 Inhibitor T-3775440 disrupts GFI1B-containing complex leading to transdifferentiation and impaired growth of AML cells</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">284</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1158%2F1535-7163.MCT-16-0471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=27903753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=273-284&author=Y.+Ishikawaauthor=K.+Gamoauthor=M.+Yabukiauthor=S.+Takagiauthor=K.+Toyoshimaauthor=K.+Nakayamaauthor=A.+Nakayamaauthor=M.+Morimotoauthor=H.+Miyashitaauthor=R.+Dairikiauthor=Y.+Hikichiauthor=N.+Tomitaauthor=D.+Tomitaauthor=S.+Imamuraauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Matsumotoauthor=R.+Haraauthor=T.+Nomuraauthor=K.+Tsuchidaauthor=K.+Nakamura&title=A+Novel+LSD1+Inhibitor+T-3775440+disrupts+GFI1B-containing+complex+leading+to+transdifferentiation+and+impaired+growth+of+AML+cells&doi=10.1158%2F1535-7163.MCT-16-0471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel LSD1 Inhibitor T-3775440 Disrupts GFI1B-Containing Complex Leading to Transdifferentiation and Impaired Growth of AML Cells</span></div><div class="casAuthors">Ishikawa, Yoshinori; Gamo, Kanae; Yabuki, Masato; Takagi, Shinji; Toyoshima, Kosei; Nakayama, Kazuhide; Nakayama, Akiko; Morimoto, Megumi; Miyashita, Hitoshi; Dairiki, Ryo; Hikichi, Yukiko; Tomita, Naoki; Tomita, Daisuke; Imamura, Shinichi; Iwatani, Misa; Kamada, Yusuke; Matsumoto, Satoru; Hara, Ryujiro; Nomura, Toshiyuki; Tsuchida, Ken; Nakamura, Kazuhide</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-284</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Dysregulation of lysine (K)-specific demethylase 1A (LSD1), also known as KDM1A, has been implicated in the development of various cancers, including leukemia.  Here, we describe the antileukemic activity and mechanism of action of T-3775440, a novel irreversible LSD1 inhibitor.  Cell growth anal. of leukemia cell lines revealed that acute erythroid leukemia (AEL) and acute megakaryoblastic leukemia cells (AMKL) were highly sensitive to this compd.  T-3775440 treatment enforced transdifferentiation of erythroid/megakaryocytic lineages into granulomonocytic-like lineage cells.  Mechanistically, T-3775440 disrupted the interaction between LSD1 and growth factor-independent 1B (GFI1B), a transcription factor crit. for the differentiation processes of erythroid and megakaryocytic lineage cells.  Knockdown of LSD1 and GFI1B recapitulated T-3775440-induced transdifferentiation and cell growth suppression, highlighting the significance of LSD1-GFI1B axis inhibition with regard to the anti-AML effects of T-3775440.  Moreover, T-3775440 exhibited significant antitumor efficacy in AEL and AMKL xenograft models.  Our findings provide a rationale for evaluating LSD1 inhibitors as potential treatments and indicate a novel mechanism of action against AML, particularly AEL and AMKL.  Mol Cancer Ther; 16(2); 273-84. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomcxpsluqnf7Vg90H21EOLACvtfcHk0lhmZinqRiH_bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertr4%253D&md5=2335343b4d5e7bbc197a1d5aec54e3af</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0471%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DY.%26aulast%3DGamo%26aufirst%3DK.%26aulast%3DYabuki%26aufirst%3DM.%26aulast%3DTakagi%26aufirst%3DS.%26aulast%3DToyoshima%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DK.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMorimoto%26aufirst%3DM.%26aulast%3DMiyashita%26aufirst%3DH.%26aulast%3DDairiki%26aufirst%3DR.%26aulast%3DHikichi%26aufirst%3DY.%26aulast%3DTomita%26aufirst%3DN.%26aulast%3DTomita%26aufirst%3DD.%26aulast%3DImamura%26aufirst%3DS.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DNomura%26aufirst%3DT.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DK.%26atitle%3DA%2520Novel%2520LSD1%2520Inhibitor%2520T-3775440%2520disrupts%2520GFI1B-containing%2520complex%2520leading%2520to%2520transdifferentiation%2520and%2520impaired%2520growth%2520of%2520AML%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D273%26epage%3D284%26doi%3D10.1158%2F1535-7163.MCT-16-0471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baba, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyofuku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hibino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirakawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, H.</span></span> <span> </span><span class="NLM_article-title">T-448, a specific inhibitor of LSD1 enzyme activity, improves learning function without causing thrombocytopenia in mice</span>. <i>Neuropsychopharmacology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1505</span>– <span class="NLM_lpage">1512</span>, <span class="refDoi"> DOI: 10.1038/s41386-018-0300-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1038%2Fs41386-018-0300-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=30580376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyrtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2019&pages=1505-1512&author=S.+Matsudaauthor=R.+Babaauthor=H.+Okiauthor=S.+Morimotoauthor=M.+Toyofukuauthor=S.+Igakiauthor=Y.+Kamadaauthor=S.+Iwasakiauthor=K.+Matsumiyaauthor=R.+Hibinoauthor=H.+Kamadaauthor=T.+Hirakawaauthor=M.+Iwataniauthor=K.+Tsuchidaauthor=R.+Haraauthor=M.+Itoauthor=H.+Kimura&title=T-448%2C+a+specific+inhibitor+of+LSD1+enzyme+activity%2C+improves+learning+function+without+causing+thrombocytopenia+in+mice&doi=10.1038%2Fs41386-018-0300-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">T-448, a specific inhibitor of LSD1 enzyme activity, improves learning function without causing thrombocytopenia in mice</span></div><div class="casAuthors">Matsuda, Satoru; Baba, Rina; Oki, Hideyuki; Morimoto, Shinji; Toyofuku, Masashi; Igaki, Shigeru; Kamada, Yusuke; Iwasaki, Shinji; Matsumiya, Kota; Hibino, Ryosuke; Kamada, Hiroko; Hirakawa, Takeshi; Iwatani, Misa; Tsuchida, Ken; Hara, Ryujiro; Ito, Mitsuhiro; Kimura, Haruhide</div><div class="citationInfo"><span class="NLM_cas:title">Neuropsychopharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1505-1512</span>CODEN:
                <span class="NLM_cas:coden">NEROEW</span>;
        ISSN:<span class="NLM_cas:issn">0893-133X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Dysregulation of histone H3 lysine 4 (H3K4) methylation has been implicated in the pathogenesis of several neurodevelopmental disorders.  Targeting lysine-specific demethylase 1 (LSD1), an H3K4 demethylase, is therefore a promising approach to treat these disorders.  However, LSD1 forms complexes with cofactors including growth factor independent 1B (GFI1B), a crit. regulator of hematopoietic differentiation.  Known tranylcypromine-based irreversible LSD1 inhibitors bind to coenzyme FAD (FAD) and disrupt the LSD1-GFI1B complex, which is assocd. with hematotoxicity such as thrombocytopenia, representing a major hurdle in the development of LSD1 inhibitors as therapeutic agents.  To discover LSD1 inhibitors with potent epigenetic modulation and lower risk of hematotoxicity, we screened small mols. that enhance H3K4 methylation by the inhibition of LSD1 enzyme activity in primary cultured rat neurons but have little impact on LSD1-GFI1B complex in human TF-1a erythroblasts.  Here we report the discovery of a specific inhibitor of LSD1 enzyme activity, T-448 (3-((1S,2R)-2-(cyclobutylamino)cyclopropyl)-N-(5-methyl-1,3,4-thiadiazol-2-yl)benzamide fumarate).  T-448 has minimal impact on the LSD1-GFI1B complex and a superior hematol. safety profile in mice via the generation of a compact formyl-FAD adduct.  T-448 increased brain H3K4 methylation and partially restored learning function in mice with NMDA receptor hypofunction.  T-448-type LSD1 inhibitors with improved safety profiles may provide unique therapeutic approaches for central nervous system disorders assocd. with epigenetic dysregulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZnyoBIF0Nr7Vg90H21EOLACvtfcHk0liFVlovyuTVrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyrtr7I&md5=a3a94d000e5461bcdf0924c3c3f9cb65</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41386-018-0300-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41386-018-0300-9%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuda%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DR.%26aulast%3DOki%26aufirst%3DH.%26aulast%3DMorimoto%26aufirst%3DS.%26aulast%3DToyofuku%26aufirst%3DM.%26aulast%3DIgaki%26aufirst%3DS.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DIwasaki%26aufirst%3DS.%26aulast%3DMatsumiya%26aufirst%3DK.%26aulast%3DHibino%26aufirst%3DR.%26aulast%3DKamada%26aufirst%3DH.%26aulast%3DHirakawa%26aufirst%3DT.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DTsuchida%26aufirst%3DK.%26aulast%3DHara%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DKimura%26aufirst%3DH.%26atitle%3DT-448%252C%2520a%2520specific%2520inhibitor%2520of%2520LSD1%2520enzyme%2520activity%252C%2520improves%2520learning%2520function%2520without%2520causing%2520thrombocytopenia%2520in%2520mice%26jtitle%3DNeuropsychopharmacology%26date%3D2019%26volume%3D44%26spage%3D1505%26epage%3D1512%26doi%3D10.1038%2Fs41386-018-0300-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanenghi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirilli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karytinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">6827</span>– <span class="NLM_lpage">6833</span>, <span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0liFVlovyuTVrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelamegam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricq, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malvaez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooker, J. M.</span></span> <span> </span><span class="NLM_article-title">Brain-penetrant LSD1 inhibitors can block memory consolidation</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">120</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1021/cn200104y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=R.+Neelamegamauthor=E.+L.+Ricqauthor=M.+Malvaezauthor=D.+Patnaikauthor=S.+Nortonauthor=S.+M.+Carlinauthor=I.+T.+Hillauthor=M.+A.+Woodauthor=S.+J.+Haggartyauthor=J.+M.+Hooker&title=Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation&doi=10.1021%2Fcn200104y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-Penetrant LSD1 Inhibitors Can Block Memory Consolidation</span></div><div class="casAuthors">Neelamegam, Ramesh; Ricq, Emily L.; Malvaez, Melissa; Patnaik, Debasis; Norton, Stephanie; Carlin, Stephen M.; Hill, Ian T.; Wood, Marcelo A.; Haggarty, Stephen J.; Hooker, Jacob M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-128</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of histone modifications in the brain may represent a new mechanism for brain disorder therapy.  Post-translational modifications of histones regulate gene expression, affecting major cellular processes such as proliferation, differentiation, and function.  An important enzyme involved in one of these histone modifications is lysine specific demethylase 1 (LSD1).  This enzyme is flavin-dependent and exhibits homol. to amine oxidases.  Parnate (2-phenylcyclopropylamine (2-PCPA); tranylcypromine) is a potent inhibitor of monoamine oxidases, and derivs. of 2-PCPA have been used for development of selective LSD1 inhibitors based on the ability to form covalent adducts with FAD.  Here we report the synthesis and in vitro characterization of LSD1 inhibitors that bond covalently to FAD.  The two most potent and selective inhibitors were used to demonstrate brain penetration when administered systemically to rodents.  First, radiosynthesis of a positron-emitting analog was used to obtain preliminary biodistribution data and whole brain time-activity curves.  Second, we demonstrate that this series of LSD1 inhibitors is capable of producing a cognitive effect in a mouse model.  By using a memory formation paradigm, novel object recognition, we show that LSD1 inhibition can abolish long-term memory formation without affecting short-term memory, providing further evidence for the importance of reversible histone methylation in the function of the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRTx9XGRzjgrVg90H21EOLACvtfcHk0lh-tregu24_Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E&md5=cd365810b397d0ece732ea853a3e93da</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DNeelamegam%26aufirst%3DR.%26aulast%3DRicq%26aufirst%3DE.%2BL.%26aulast%3DMalvaez%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DNorton%26aufirst%3DS.%26aulast%3DCarlin%26aufirst%3DS.%2BM.%26aulast%3DHill%26aufirst%3DI.%2BT.%26aulast%3DWood%26aufirst%3DM.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26atitle%3DBrain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128%26doi%3D10.1021%2Fcn200104y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pai, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span> <span> </span><span class="NLM_article-title">Structure activity relationship and modeling studies of inhibitors of lysine specific demethylase 1</span>. <i>PloS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0170301</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0170301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1371%2Fjournal.pone.0170301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=28158205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0170301&author=C.+Zhouauthor=F.+Wuauthor=L.+Luauthor=L.+Weiauthor=E.+Paiauthor=Y.+Yaoauthor=Y.+Song&title=Structure+activity+relationship+and+modeling+studies+of+inhibitors+of+lysine+specific+demethylase+1&doi=10.1371%2Fjournal.pone.0170301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure activity relationship and modeling studies of inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Zhou, Chao; Wu, Fangrui; Lu, Lianghao; Wei, Liping; Pai, Eric; Yao, Yuan; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0170301/1-e0170301/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Post-translational modifications of histone play important roles in gene transcription.  Aberrant methylation of histone lysine sidechains have been often found in cancer.  Lysine specific demethylase 1 (LSD1), which can demethylate histone H3 lysine 4 (H3K4) and other proteins, has recently been found to be a drug target for acute myeloid leukemia.  To understand structure activity/selectivity relationships of LSD1 inhibitors, several series of cyclopropylamine and related compds. were synthesized and tested for their activities against LSD1 and related monoamine oxidase (MAO) A and B.  Several cyclopropylamine contg. compds. were found to be highly potent and selective inhibitors of LSD1.  A novel series cyclopropylimine compds. also exhibited strong inhibitory activity against LSD1.  Structure activity relationships (SAR) of these compds. are discussed.  Docking studies were performed to provide possible binding models of a representative compd. in LSD1 and MAO-A.  Moreover, these modeling studies can rationalize the obsd. SARs and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplFKcJwJZQw7Vg90H21EOLACvtfcHk0lh-tregu24_Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyqs7Y%253D&md5=4d6a733dd6ef4a4b4f34a77c2d08a104</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0170301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0170301%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DPai%26aufirst%3DE.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DStructure%2520activity%2520relationship%2520and%2520modeling%2520studies%2520of%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DPloS%2520One%26date%3D2017%26volume%3D12%26spage%3De0170301%26doi%3D10.1371%2Fjournal.pone.0170301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, R.</span></span> <span> </span><span class="NLM_article-title">In vivo site-directed radiotracers: a mini-review</span>. <i>Nucl. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1016/j.nucmedbio.2008.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.nucmedbio.2008.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19026942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaisLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2008&pages=805-815&author=S.+Patelauthor=R.+Gibson&title=In+vivo+site-directed+radiotracers%3A+a+mini-review&doi=10.1016%2Fj.nucmedbio.2008.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo site-directed radiotracers: a mini-review</span></div><div class="casAuthors">Patel, Shil; Gibson, Raymond</div><div class="citationInfo"><span class="NLM_cas:title">Nuclear Medicine and Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NMBIEO</span>;
        ISSN:<span class="NLM_cas:issn">0969-8051</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review on the process for developing reversible, monovalent site-specific radiotracers in drug design.  It is concerned with the characteristics of radiotracers such as binding, lipophilicity, and binding specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYUEhMFA7d7bVg90H21EOLACvtfcHk0lh-tregu24_Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaisLvL&md5=869985e0741d52a76cb546972097e2a8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.nucmedbio.2008.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nucmedbio.2008.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DGibson%26aufirst%3DR.%26atitle%3DIn%2520vivo%2520site-directed%2520radiotracers%253A%2520a%2520mini-review%26jtitle%3DNucl.%2520Med.%2520Biol.%26date%3D2008%26volume%3D35%26spage%3D805%26epage%3D815%26doi%3D10.1016%2Fj.nucmedbio.2008.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laruelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slifstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span> <span> </span><span class="NLM_article-title">Positron emission tomography: imaging and quantification of neurotransporter availability</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/s1046-2023(02)00085-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2FS1046-2023%2802%2900085-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=12183117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtFWhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2002&pages=287-299&author=M.+Laruelleauthor=M.+Slifsteinauthor=Y.+Huang&title=Positron+emission+tomography%3A+imaging+and+quantification+of+neurotransporter+availability&doi=10.1016%2Fs1046-2023%2802%2900085-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Positron emission tomography: imaging and quantification of neurotransporter availability</span></div><div class="casAuthors">Laruelle, Marc; Slifstein, Mark; Huang, Yiyun</div><div class="citationInfo"><span class="NLM_cas:title">Methods (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">287-299</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">A review.  Over the last decade, a large no. of radiotracers have been developed to image and quantify transporter availability with positron emission tomog. (PET) or single-photon emission computed tomog. (SPECT).  Radiotracers suitable to image dopamine transporters (DATs) and serotonin transporters (SERTs) have been the object of most efforts.  Following a brief overview of DAT and SERT radiotracers that have been demonstrated to be suitable for quant. anal. in vivo, this article describes the principal methods that have been used for the anal. of these data.  Kinetic modeling is the most direct implementation of the compartment models, but with some tracers accurate input function measurement and good compartment configuration identification can be difficult to obtain.  Other methods were designed to overcome some particular vulnerability to error of classic kinetic modeling, but introduced new vulnerabilities in the process.  Ref. region methods obviate the need for arterial plasma measurement, but are not as robust to violations of the underlying modeling assumptions as methods using the arterial input function.  Graphical methods give ests. of distribution vols. without the requirement of compartment model specification, but provide a biased estimator in the presence of statistical noise.  True equil. methods are quite robust, but their use is limited to expts. with tracers that are suitable for const. infusion.  In conclusion, no universally "best" method is applicable to all neurotransporter imaging studies, and careful evaluation of model-based methods is required for each radiotracer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3jdsXbliaMLVg90H21EOLACvtfcHk0ljoBaTB-MkcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtFWhs74%253D&md5=1ec923e982f1769d80fd74b68d7b05ed</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2FS1046-2023%2802%2900085-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1046-2023%252802%252900085-3%26sid%3Dliteratum%253Aachs%26aulast%3DLaruelle%26aufirst%3DM.%26aulast%3DSlifstein%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26atitle%3DPositron%2520emission%2520tomography%253A%2520imaging%2520and%2520quantification%2520of%2520neurotransporter%2520availability%26jtitle%3DMethods%26date%3D2002%26volume%3D27%26spage%3D287%26epage%3D299%26doi%3D10.1016%2Fs1046-2023%2802%2900085-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, V. W.</span></span> <span> </span><span class="NLM_article-title">PET radiotracers: crossing the blood-brain barrier and surviving metabolism</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">431</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2009.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1016%2Fj.tips.2009.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=19616318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1KgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=431-440&author=V.+W.+Pike&title=PET+radiotracers%3A+crossing+the+blood-brain+barrier+and+surviving+metabolism&doi=10.1016%2Fj.tips.2009.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">PET radiotracers: crossing the blood-brain barrier and surviving metabolism</span></div><div class="casAuthors">Pike, Victor W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">431-440</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Radiotracers for imaging protein targets in the living human brain with positron emission tomog. (PET) are increasingly useful in clin. research and in drug development.  Such radiotracers must fulfill many criteria, among which an ability to enter brain adequately and reversibly without contamination by troublesome radiometabolites is desirable for accurate measurement of the d. of a target protein (e.g. neuroreceptor, transporter, enzyme or plaque).  Candidate radiotracers can fail as a result of poor passive brain entry, rejection from brain by efflux transporters or undesirable metab.  These issues are reviewed.  Emerging PET radiotracers for measuring efflux transporter function and new strategies for ameliorating radiotracer metab. are discussed.  A growing understanding of the mol. features affecting the brain penetration, metab. and efflux transporter sensitivity of prospective radiotracers should ultimately lead to their more rational and efficient design, and also to their greater efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6ftMCpkOSmrVg90H21EOLACvtfcHk0ljoBaTB-MkcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1KgsLg%253D&md5=97704995fecd968baf1e0892bf70324e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2009.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2009.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DV.%2BW.%26atitle%3DPET%2520radiotracers%253A%2520crossing%2520the%2520blood-brain%2520barrier%2520and%2520surviving%2520metabolism%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2009%26volume%3D30%26spage%3D431%26epage%3D440%26doi%3D10.1016%2Fj.tips.2009.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mireles, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Morais, S. M.</span></span> <span> </span><span class="NLM_article-title">In V<i>itro</i> P-glycoprotein assays to predict the <i>in Vivo</i> interactions of P-glycoprotein with drugs in the central nervous system</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">268</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1124/dmd.107.017434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1124%2Fdmd.107.017434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=17962372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1yktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=268-275&author=B.+Fengauthor=J.+B.+Millsauthor=R.+E.+Davidsonauthor=R.+J.+Mirelesauthor=J.+S.+Janiszewskiauthor=M.+D.+Troutmanauthor=S.+M.+de+Morais&title=In+Vitro+P-glycoprotein+assays+to+predict+the+in+Vivo+interactions+of+P-glycoprotein+with+drugs+in+the+central+nervous+system&doi=10.1124%2Fdmd.107.017434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro P-glycoprotein assays to predict the in vivo interactions of P-glycoprotein with drugs in the central nervous system</span></div><div class="casAuthors">Feng, Bo; Mills, Jessica B.; Davidson, Ralph E.; Mireles, Rouchelle J.; Janiszewski, John S.; Troutman, Matthew D.; de Morais, Sonia M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">268-275</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Thirty-one structurally diverse marketed central nervous system (CNS)-active drugs, one active metabolite, and seven non-CNS-active compds. were tested in three P-glycoprotein (P-gp) in vitro assays: transwell assays using MDCK, human MDR1-MDCK, and mouse Mdr1a-MDCK cells, ATPase, and calcein AM inhibition.  Addnl., the permeability for these compds. was measured in two in vitro models: parallel artificial membrane permeation assay and apical-to-basolateral apparent permeability in MDCK.  The exposure of the same set of compds. in brain and plasma was measured in P-gp knockout (KO) and wild-type (WT) mice after s.c. administration.  One drug and its metabolite, risperidone and 9-hydroxyrisperidone, of the 32 CNS compds., and 6 of the 7 non-CNS drugs were detd. to have pos. efflux using ratio of ratios in MDR1-MDCK vs. MDCK transwell assays.  Data from transwell studies correlated well with the brain-to-plasma area under the curve ratios between P-gp KO and WT mice for the 32 CNS compds.  In addn., 3300 Pfizer compds. were tested in MDR1-MDCK and Mdr1a-MDCK transwell assays, with a good correlation (R2 = 0.92) between the efflux ratios in human MDR1-MDCK and mouse Mdr1a-MDCK cells.  Permeability data showed that the majority of the 32 CNS compds. have moderate to high passive permeability.  This work has demonstrated that in vitro transporter assays help in understanding the role of P-gp-mediated efflux activity in detg. the disposition of CNS drugs in vivo, and the transwell assay is a valuable in vitro assay to evaluate human P-gp interaction with compds. for assessing brain penetration of new chem. entities to treat CNS disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsQyRSOySyGLVg90H21EOLACvtfcHk0ljoBaTB-MkcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1yktbc%253D&md5=bbf8b6969bd6d6b3d741b6ef9e7853f3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fdmd.107.017434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.107.017434%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DMills%26aufirst%3DJ.%2BB.%26aulast%3DDavidson%26aufirst%3DR.%2BE.%26aulast%3DMireles%26aufirst%3DR.%2BJ.%26aulast%3DJaniszewski%26aufirst%3DJ.%2BS.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3Dde%2BMorais%26aufirst%3DS.%2BM.%26atitle%3DIn%2520Vitro%2520P-glycoprotein%2520assays%2520to%2520predict%2520the%2520in%2520Vivo%2520interactions%2520of%2520P-glycoprotein%2520with%2520drugs%2520in%2520the%2520central%2520nervous%2520system%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2008%26volume%3D36%26spage%3D268%26epage%3D275%26doi%3D10.1124%2Fdmd.107.017434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umland, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troutman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liston, T. E.</span></span> <span> </span><span class="NLM_article-title">Species independence in brain tissue binding using brain homogenates</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">1270</span>– <span class="NLM_lpage">1277</span>, <span class="refDoi"> DOI: 10.1124/dmd.111.038778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1124%2Fdmd.111.038778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=21474681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFWktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=1270-1277&author=L.+Diauthor=J.+P.+Umlandauthor=G.+Changauthor=Y.+Huangauthor=Z.+Linauthor=D.+O.+Scottauthor=M.+D.+Troutmanauthor=T.+E.+Liston&title=Species+independence+in+brain+tissue+binding+using+brain+homogenates&doi=10.1124%2Fdmd.111.038778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Species independence in brain tissue binding using brain homogenates</span></div><div class="casAuthors">Di, Li; Umland, John P.; Chang, George; Huang, Youping; Lin, Zhen; Scott, Dennis O.; Troutman, Matthew D.; Liston, Theodore E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1270-1277</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Species independence of brain tissue binding was assessed with a large no. of structurally diverse compds. using equil. dialysis with brain homogenates of seven species and strains (Wistar Han rat, Sprague-Dawley rat, CD-1 mouse, Hartley guinea pig, beagle dog, cynomolgus monkey, and human).  The results showed that the fractions unbound of the seven species and strains were strongly correlated with correlation coeffs. ranging from 0.93 to 0.99.  The cross-species/strain correlations were not significantly different from the interassay correlation with the same species.  The linear correlation between Wistar Han and other species had a slope close to 1 and an intercept near 0.  Based on orthogonal statistical anal., no correction is needed for extrapolation of fraction unbound from Wistar Han rat to the other species or strains.  Hence, brain tissue binding of Wistar Han rat can be used to obtain binding of other species and strains in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoub4FiLWkAFrVg90H21EOLACvtfcHk0lgol06nFWC4wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFWktrg%253D&md5=93a51addedb22de4648e6e5ee39c6c81</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fdmd.111.038778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.111.038778%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26aulast%3DUmland%26aufirst%3DJ.%2BP.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DScott%26aufirst%3DD.%2BO.%26aulast%3DTroutman%26aufirst%3DM.%2BD.%26aulast%3DListon%26aufirst%3DT.%2BE.%26atitle%3DSpecies%2520independence%2520in%2520brain%2520tissue%2520binding%2520using%2520brain%2520homogenates%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2011%26volume%3D39%26spage%3D1270%26epage%3D1277%26doi%3D10.1124%2Fdmd.111.038778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimwood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphrey, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skaddan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zasadny, K.</span></span> <span> </span><span class="NLM_article-title">Design and selection parameters to accelerate the discovery of novel central nervous system positron emission tomography (PET) ligands and their application in the development of a novel phosphodiesterase 2A PET ligand</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4568</span>– <span class="NLM_lpage">4579</span>, <span class="refDoi"> DOI: 10.1021/jm400312y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400312y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4568-4579&author=L.+Zhangauthor=A.+Villalobosauthor=E.+M.+Beckauthor=T.+Bocanauthor=T.+A.+Chappieauthor=L.+Chenauthor=S.+Grimwoodauthor=S.+D.+Heckauthor=C.+J.+Helalauthor=X.+Houauthor=J.+M.+Humphreyauthor=J.+Luauthor=M.+B.+Skaddanauthor=T.+J.+McCarthyauthor=P.+R.+Verhoestauthor=T.+T.+Wagerauthor=K.+Zasadny&title=Design+and+selection+parameters+to+accelerate+the+discovery+of+novel+central+nervous+system+positron+emission+tomography+%28PET%29+ligands+and+their+application+in+the+development+of+a+novel+phosphodiesterase+2A+PET+ligand&doi=10.1021%2Fjm400312y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Selection Parameters to Accelerate the Discovery of Novel Central Nervous System Positron Emission Tomography (PET) Ligands and Their Application in the Development of a Novel Phosphodiesterase 2A PET Ligand</span></div><div class="casAuthors">Zhang, Lei; Villalobos, Anabella; Beck, Elizabeth M.; Bocan, Thomas; Chappie, Thomas A.; Chen, Laigao; Grimwood, Sarah; Heck, Steven D.; Helal, Christopher J.; Hou, Xinjun; Humphrey, John M.; Lu, Jiemin; Skaddan, Marc B.; McCarthy, Timothy J.; Verhoest, Patrick R.; Wager, Travis T.; Zasadny, Kenneth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4568-4579</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To accelerate the discovery of novel small mol. central nervous system (CNS) positron emission tomog. (PET) ligands, we aimed to define a property space that would facilitate ligand design and prioritization, thereby providing a higher probability of success for novel PET ligand development.  Toward this end, we built a database consisting of 62 PET ligands that have successfully reached the clinic and 15 radioligands that failed in late-stage development as neg. controls.  A systematic anal. of these ligands identified a set of preferred parameters for physicochem. properties, brain permeability, and nonspecific binding (NSB).  These preferred parameters have subsequently been applied to several programs and have led to the successful development of novel PET ligands with reduced resources and timelines.  This strategy is illustrated here by the discovery of the novel phosphodiesterase 2A (PDE2A) PET ligand 4-(3-[18F]fluoroazetidin-1-yl)-7-methyl-5-{1-methyl-5-[4-(trifluoromethyl)phenyl]-1H-pyrazol-4-yl}imidazo[5,1-f][1,2,4]triazine, [18F]PF-05270430 (5).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTCG9TZE0X-bVg90H21EOLACvtfcHk0lgol06nFWC4wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyhtro%253D&md5=ee7ce04829d7456a632c02c331c4edfa</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm400312y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400312y%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DVillalobos%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DE.%2BM.%26aulast%3DBocan%26aufirst%3DT.%26aulast%3DChappie%26aufirst%3DT.%2BA.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DGrimwood%26aufirst%3DS.%26aulast%3DHeck%26aufirst%3DS.%2BD.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DHumphrey%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DSkaddan%26aufirst%3DM.%2BB.%26aulast%3DMcCarthy%26aufirst%3DT.%2BJ.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DZasadny%26aufirst%3DK.%26atitle%3DDesign%2520and%2520selection%2520parameters%2520to%2520accelerate%2520the%2520discovery%2520of%2520novel%2520central%2520nervous%2520system%2520positron%2520emission%2520tomography%2520%2528PET%2529%2520ligands%2520and%2520their%2520application%2520in%2520the%2520development%2520of%2520a%2520novel%2520phosphodiesterase%25202A%2520PET%2520ligand%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4568%26epage%3D4579%26doi%3D10.1021%2Fjm400312y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span> <span> </span><span class="NLM_article-title">Impact of physicochemical profiling for rational approach on drug discovery</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1228</span>, <span class="refDoi"> DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+physicochemical+profiling+for+rational+approach+on+drug+discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of π-π stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 μg/mL and 10 μg/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lh8oM64KrNjYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520physicochemical%2520profiling%2520for%2520rational%2520approach%2520on%2520drug%2520discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeyasu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oizumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span> <span> </span><span class="NLM_article-title">Studies of Novel 1, 4-Dihydropyridine Ca Antagonist CS-905. I. Measurement of Partition Coefficient (logP) by High Performance Liquid Chromatography (HPLC)</span>. <i>Yakugaku Zasshi</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">213</span>, <span class="refDoi"> DOI: 10.1248/yakushi1947.115.3_213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fyakushi1947.115.3_213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=7738779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADyaK2MXmvFWksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=1995&pages=213&author=K.+Masumotoauthor=A.+Takeyasuauthor=K.+Oizumiauthor=T.+Kobayashi&title=Studies+of+Novel+1%2C+4-Dihydropyridine+Ca+Antagonist+CS-905.+I.+Measurement+of+Partition+Coefficient+%28logP%29+by+High+Performance+Liquid+Chromatography+%28HPLC%29&doi=10.1248%2Fyakushi1947.115.3_213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 1,4-dihydropyridine Ca antagonist CS-905. I. Measurement of partition coefficient (logP) by high performance liquid chromatography (HPLC)</span></div><div class="casAuthors">Masumoto, Kazue; Takeyasu, Akira; Oizumi, Kiyoshi; Kobayashi, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Yakugaku Zasshi</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">213-20</span>CODEN:
                <span class="NLM_cas:coden">YKKZAJ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6903</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">A new 1,4-dihydropyridine deriv., CS-905, currently under development as a Ca antagonist, showed a gradual onset and long duration in its antihypertensive effect upon single oral administration.  The partition coeff. for CS-905 was measured along with 11 known dihydropyridine Ca antagonists by the HPLC method to clarify its mode of action.  The log PHPLC for CS-905 was 5.18, almost the same as that of manidipine and this value is the highest among the measured drugs.  Its pharmacokinetic profile in SHR was also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSNcDUIuEYoLVg90H21EOLACvtfcHk0lh8oM64KrNjYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvFWksL4%253D&md5=41fdae8c51d78f930ae1ce053d6cf512</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1248%2Fyakushi1947.115.3_213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fyakushi1947.115.3_213%26sid%3Dliteratum%253Aachs%26aulast%3DMasumoto%26aufirst%3DK.%26aulast%3DTakeyasu%26aufirst%3DA.%26aulast%3DOizumi%26aufirst%3DK.%26aulast%3DKobayashi%26aufirst%3DT.%26atitle%3DStudies%2520of%2520Novel%25201%252C%25204-Dihydropyridine%2520Ca%2520Antagonist%2520CS-905.%2520I.%2520Measurement%2520of%2520Partition%2520Coefficient%2520%2528logP%2529%2520by%2520High%2520Performance%2520Liquid%2520Chromatography%2520%2528HPLC%2529%26jtitle%3DYakugaku%2520Zasshi%26date%3D1995%26volume%3D115%26spage%3D213%26doi%3D10.1248%2Fyakushi1947.115.3_213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, T.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of (4R)-1-{3-[2-(18F)Fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one ([18F]T-401) as a Novel Positron-Emission Tomography Imaging Agent for Monoacylglycerol Lipase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2362</span>– <span class="NLM_lpage">2375</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01576</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01576" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVSht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2362-2375&author=Y.+Hattoriauthor=K.+Aoyamaauthor=J.+Maedaauthor=N.+Arimuraauthor=Y.+Takahashiauthor=M.+Sasakiauthor=M.+Fujinagaauthor=C.+Sekiauthor=Y.+Nagaiauthor=K.+Kawamuraauthor=T.+Yamasakiauthor=M.-R.+Zhangauthor=M.+Higuchiauthor=T.+Koike&title=Design%2C+Synthesis%2C+and+Evaluation+of+%284R%29-1-%7B3-%5B2-%2818F%29Fluoro-4-methylpyridin-3-yl%5Dphenyl%7D-4-%5B4-%281%2C3-thiazol-2-ylcarbonyl%29piperazin-1-yl%5Dpyrrolidin-2-one+%28%5B18F%5DT-401%29+as+a+Novel+Positron-Emission+Tomography+Imaging+Agent+for+Monoacylglycerol+Lipase&doi=10.1021%2Facs.jmedchem.8b01576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of (4R)-1-{3-[2-(18F)Fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one ([18F]T-401) as a Novel Positron-Emission Tomography Imaging Agent for Monoacylglycerol Lipase</span></div><div class="casAuthors">Hattori, Yasushi; Aoyama, Kazunobu; Maeda, Jun; Arimura, Naoto; Takahashi, Yasuko; Sasaki, Masako; Fujinaga, Masayuki; Seki, Chie; Nagai, Yuji; Kawamura, Kazunori; Yamasaki, Tomoteru; Zhang, Ming-Rong; Higuchi, Makoto; Koike, Tatsuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2362-2375</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Monoacylglycerol lipase (MAGL) is a cytosolic serine hydrolase involved in endocannabinoid and inflammatory signaling.  Positron-emission tomog. (PET) imaging of MAGL serves to validate target engagement of therapeutic MAGL inhibitors as well as to investigate MAGL levels under normal and disease conditions.  However, PET radioligands with reversible binding kinetics for MAGL, which allow quant. assessment of MAGL, are hitherto unavailable.  In this study, we designed and synthesized fluoro-contg. PET probes starting from a recently identified piperazinyl pyrrolidine-2-one deriv. with reversible binding to MAGL.  By tailoring the lipophilicity of the mol. to optimize nonspecific binding and blood-brain barrier permeability, we successfully identified two compds. that show high uptake to regions enriched with MAGL.  PET imaging of wild-type and MAGL-deficient mice as well as a macaque monkey indicated that [18F]I ((4R)-1-{3-[2-(18F)fluoro-4-methylpyridin-3-yl]phenyl}-4-[4-(1,3-thiazol-2-ylcarbonyl)piperazin-1-yl]pyrrolidin-2-one, [18F]T-401) specifically binds to MAGL with adequate reversibility, yielding a high contrast for MAGL within an appropriate imaging time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppgW6fA1E8WLVg90H21EOLACvtfcHk0lh8oM64KrNjYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVSht7k%253D&md5=4848fa8fce411e960b580b5bd007e0c9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01576%26sid%3Dliteratum%253Aachs%26aulast%3DHattori%26aufirst%3DY.%26aulast%3DAoyama%26aufirst%3DK.%26aulast%3DMaeda%26aufirst%3DJ.%26aulast%3DArimura%26aufirst%3DN.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DC.%26aulast%3DNagai%26aufirst%3DY.%26aulast%3DKawamura%26aufirst%3DK.%26aulast%3DYamasaki%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DM.-R.%26aulast%3DHiguchi%26aufirst%3DM.%26aulast%3DKoike%26aufirst%3DT.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520%25284R%2529-1-%257B3-%255B2-%252818F%2529Fluoro-4-methylpyridin-3-yl%255Dphenyl%257D-4-%255B4-%25281%252C3-thiazol-2-ylcarbonyl%2529piperazin-1-yl%255Dpyrrolidin-2-one%2520%2528%255B18F%255DT-401%2529%2520as%2520a%2520Novel%2520Positron-Emission%2520Tomography%2520Imaging%2520Agent%2520for%2520Monoacylglycerol%2520Lipase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2362%26epage%3D2375%26doi%3D10.1021%2Facs.jmedchem.8b01576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakashima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, Y.</span></span> <span> </span><span class="NLM_article-title">Impact of Physicochemical Profiling for Rational Approach on Drug Discovery</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1238</span>, <span class="refDoi"> DOI: 10.1248/cpb.c13-00436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1248%2Fcpb.c13-00436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=24292785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=1228-1238&author=S.+Nakashimaauthor=K.+Yamamotoauthor=Y.+Araiauthor=Y.+Ikeda&title=Impact+of+Physicochemical+Profiling+for+Rational+Approach+on+Drug+Discovery&doi=10.1248%2Fcpb.c13-00436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of physicochemical profiling for rational approach on drug discovery</span></div><div class="casAuthors">Nakashima, Shoko; Yamamoto, Katsuhiko; Arai, Yuta; Ikeda, Yukihiro</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1228-1238</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Solid-state characterization plays a vital role in lead optimization and candidate selection with the appropriate physicochem. properties for proper oral dosage formulation.  Aq. soly. is an important parameter in the successful development of oral dosage formulation since poor aq. soly. limits absorption.  In this study, we summarized an efficient approach using a small amt. of sample for solid-state characterization, including thermodn. soly., which is defined as physicochem. profiling.  By using the physicochem. profiling results of 75 anti-cancer drugs and clin. candidates, we examd. the relationship between thermodn. soly. and mol. structural parameters and assessed the effects of thermodn. soly. on pharmacokinetic profile for rational sol. drug design.  The Log DpH 7.4, arom. ring count, and hydrogen bond count were good indicators for predicting sparingly sol. compds. that increase the lattice energy because of π-π stacking and hydrogen bonds, resulting in lowered thermodn. soly.  The level of thermodn. soly. in simulated intestinal fluid (pH 6.8) in the presence and absence of bile acid, which is required for minimal acceptable bioavailability (>30%), was 1 μg/mL and 10 μg/mL, resp.  Physicochem. profiling, which includes thermodn. soly. considering its solid-state properties, contributes to rational lead optimization and efficient candidate selection for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXTGzqqcJSbVg90H21EOLACvtfcHk0lgTQ5q9tvzYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFaiur8%253D&md5=c1291e838e8509c2fe2a1edfaf509ed4</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1248%2Fcpb.c13-00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.c13-00436%26sid%3Dliteratum%253Aachs%26aulast%3DNakashima%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DArai%26aufirst%3DY.%26aulast%3DIkeda%26aufirst%3DY.%26atitle%3DImpact%2520of%2520Physicochemical%2520Profiling%2520for%2520Rational%2520Approach%2520on%2520Drug%2520Discovery%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2013%26volume%3D61%26spage%3D1228%26epage%3D1238%26doi%3D10.1248%2Fcpb.c13-00436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshitomi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikemoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebihara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asahi, S.</span></span> <span> </span><span class="NLM_article-title">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1007/s11095-006-0285-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.1007%2Fs11095-006-0285-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=16779700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=1460-1472&author=T.+Takeuchiauthor=S.+Yoshitomiauthor=T.+Higuchiauthor=K.+Ikemotoauthor=S.-I.+Niwaauthor=T.+Ebiharaauthor=M.+Katohauthor=T.+Yokoiauthor=S.+Asahi&title=Establishment+and+Characterization+of+the+Transformants+Stably-Expressing+MDR1+Derived+from+Various+Animal+Species+in+LLC-PK1&doi=10.1007%2Fs11095-006-0285-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment and Characterization of the Transformants Stably-Expressing MDR1 Derived from Various Animal Species in LLC-PK1</span></div><div class="casAuthors">Takeuchi, Toshiyuki; Yoshitomi, Sumie; Higuchi, Tomoaki; Ikemoto, Keiko; Niwa, Shin-Ichi; Ebihara, Takuya; Katoh, Miki; Yokoi, Tsuyoshi; Asahi, Satoru</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1460-1472</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Stable transformants expressing human multidrug resistance 1 (MDR1), monkey MDR1, canine MDR1, rat MDR1a, rat MDR1b, mouse mdr1a, and mouse mdr1b in LLC-PK1 were established to investigate species differences in P-glycoprotein (P-gp, ABCB1) mediated efflux activity.  Methods: The seven cDNAs of MDR1 from five animals were cloned, and their transformants stably expressing the series of MDR1 in LLC-PK1 were established.  Transport studies of clarithromycin, daunorubicin, digoxin, erythromycin, etoposide, paclitaxel, propranolol, quinidine, ritonavir, saquinavir, verapamil, and vinblastine were performed by using these cells, and efflux activity was compared among the species.  Results: Except for propranolol, all compds. showed efflux activity in all transformants, and were judged to be substrates of P-gp.  There were slight interspecies and interisoforms differences in the substrate recognition.  However, the efflux ratio among the series of the MDR1 stably expressing cells showed good correlation as represented between human and monkey MDR1, and poor correlation as represented between human MDR1 and mouse mdr1a, and human and canine MDR1.  Conclusions: Results in the present study indicate that all MDR1 stably expressing cells have efflux activity for various P-gp substrates, and that interspecies differences and similarities of the P-gp substrate efflux activity may exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28IPlZqwtHLVg90H21EOLACvtfcHk0lgTQ5q9tvzYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFOmur0%253D&md5=a21044c202d7ff2eb8ab27c4623909de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs11095-006-0285-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-006-0285-7%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DYoshitomi%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DIkemoto%26aufirst%3DK.%26aulast%3DNiwa%26aufirst%3DS.-I.%26aulast%3DEbihara%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DYokoi%26aufirst%3DT.%26aulast%3DAsahi%26aufirst%3DS.%26atitle%3DEstablishment%2520and%2520Characterization%2520of%2520the%2520Transformants%2520Stably-Expressing%2520MDR1%2520Derived%2520from%2520Various%2520Animal%2520Species%2520in%2520LLC-PK1%26jtitle%3DPharm.%2520Res.%26date%3D2006%26volume%3D23%26spage%3D1460%26epage%3D1472%26doi%3D10.1007%2Fs11095-006-0285-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rohlfing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroenke, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubach, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowden, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfefferbaum, A.</span></span> <span> </span><span class="NLM_article-title">The INIA19 Template and NeuroMaps Atlas for Primate Brain Image Parcellation and Spatial Normalization</span>. <i>Front. Neuroinf.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">27</span>, <span class="refDoi"> DOI: 10.3389/fninf.2012.00027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=10.3389%2Ffninf.2012.00027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;key=23230398" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=27&author=T.+Rohlfingauthor=C.+D.+Kroenkeauthor=E.+V.+Sullivanauthor=M.+F.+Dubachauthor=D.+M.+Bowdenauthor=K.+A.+Grantauthor=A.+Pfefferbaum&title=The+INIA19+Template+and+NeuroMaps+Atlas+for+Primate+Brain+Image+Parcellation+and+Spatial+Normalization&doi=10.3389%2Ffninf.2012.00027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3389%2Ffninf.2012.00027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffninf.2012.00027%26sid%3Dliteratum%253Aachs%26aulast%3DRohlfing%26aufirst%3DT.%26aulast%3DKroenke%26aufirst%3DC.%2BD.%26aulast%3DSullivan%26aufirst%3DE.%2BV.%26aulast%3DDubach%26aufirst%3DM.%2BF.%26aulast%3DBowden%26aufirst%3DD.%2BM.%26aulast%3DGrant%26aufirst%3DK.%2BA.%26aulast%3DPfefferbaum%26aufirst%3DA.%26atitle%3DThe%2520INIA19%2520Template%2520and%2520NeuroMaps%2520Atlas%2520for%2520Primate%2520Brain%2520Image%2520Parcellation%2520and%2520Spatial%2520Normalization%26jtitle%3DFront.%2520Neuroinf.%26date%3D2012%26volume%3D6%26spage%3D27%26doi%3D10.3389%2Ffninf.2012.00027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i62"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01937">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_74607"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01937?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01937</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis of chloro-precursor <b>26</b>; radiolabeling of [<sup>18</sup>F]<b>1d</b>; chromatograms of chiral separation; HPLC traces of <b>1a</b>–<b>e</b>; and chromatograms for purification and analysis of [<sup>18</sup>F]<b>1e</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (CSV) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_001.pdf">jm0c01937_si_001.pdf (373.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01937/suppl_file/jm0c01937_si_002.csv">jm0c01937_si_002.csv (0.32 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01937&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01937%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-7%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01937" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799939729073d6c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
