
<html lang="en"     class="pb-page"  data-request-id="6db4416f-41d6-41cb-8111-db48053a4648"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01747;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-1;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Tenofovir-Amino Acid Conjugates Act as Polymerase Substrates—Implications for Avoiding Cellular Phosphorylation in the Discovery of Nucleotide Analogues" /></meta><meta name="dc.Creator" content="Weijie  Gu" /></meta><meta name="dc.Creator" content="Sergio  Martinez" /></meta><meta name="dc.Creator" content="Hoai  Nguyen" /></meta><meta name="dc.Creator" content="Hongtao  Xu" /></meta><meta name="dc.Creator" content="Piet  Herdewijn" /></meta><meta name="dc.Creator" content="Steven  De Jonghe" /></meta><meta name="dc.Creator" content="Kalyan  Das" /></meta><meta name="dc.Description" content="Nucleotide analogues are used for treating viral infections such as HIV, hepatitis B, hepatitis C, influenza, and SARS-CoV-2. To become polymerase substrates, a nucleotide analogue must be phosphor..." /></meta><meta name="Description" content="Nucleotide analogues are used for treating viral infections such as HIV, hepatitis B, hepatitis C, influenza, and SARS-CoV-2. To become polymerase substrates, a nucleotide analogue must be phosphor..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 23, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01747" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01747" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01747" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01747" /></link>
        
    
    

<title>Tenofovir-Amino Acid Conjugates Act as Polymerase Substrates—Implications for Avoiding Cellular Phosphorylation in the Discovery of Nucleotide Analogues | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01747" /></meta><meta property="og:title" content="Tenofovir-Amino Acid Conjugates Act as Polymerase Substrates—Implications for Avoiding Cellular Phosphorylation in the Discovery of Nucleotide Analogues" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0009.jpeg" /></meta><meta property="og:description" content="Nucleotide analogues are used for treating viral infections such as HIV, hepatitis B, hepatitis C, influenza, and SARS-CoV-2. To become polymerase substrates, a nucleotide analogue must be phosphorylated by cellular kinases which is rate-limiting. The goal of this study is to develop dNTP/NTP analogues directly from nucleotides. Tenofovir (TFV) analogues were synthesized by conjugating with amino acids. We demonstrate that some conjugates act as dNTP analogues and HIV-1 reverse transcriptase (RT) catalytically incorporates the TFV part as the chain terminator. X-ray structures in complex with HIV-1 RT/dsDNA showed binding of the conjugates at the polymerase active site, however, in different modes in the presence of Mg2+ versus Mn2+ ions. The adaptability of the compounds is seemingly essential for catalytic incorporation of TFV by RT. 4d with a carboxyl sidechain demonstrated the highest incorporation. 4e showed weak incorporation and rather behaved as a dNTP-competitive inhibitor. This result advocates the feasibility of designing NTP/dNTP analogues by chemical substitutions to nucleotide analogues." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01747"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01747">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01747&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01747&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01747&amp;href=/doi/10.1021/acs.jmedchem.0c01747" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 782-796</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01743" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01762" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Tenofovir-Amino Acid Conjugates Act as Polymerase Substrates—Implications for Avoiding Cellular Phosphorylation in the Discovery of Nucleotide Analogues</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Weijie Gu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weijie Gu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology, Immunology and Transplantation, Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div><div class="loa-info-affiliations-info">KU Leuven, Department of Pharmaceutical and Pharmacological Sciences, Laboratory of Medicinal Chemistry, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weijie++Gu">Weijie Gu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sergio Martinez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sergio Martinez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology, Immunology and Transplantation, Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sergio++Martinez">Sergio Martinez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hoai Nguyen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hoai Nguyen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Pharmaceutical and Pharmacological Sciences, Laboratory of Medicinal Chemistry, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hoai++Nguyen">Hoai Nguyen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongtao Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongtao Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology, Immunology and Transplantation, Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongtao++Xu">Hongtao Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Piet Herdewijn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Piet Herdewijn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Pharmaceutical and Pharmacological Sciences, Laboratory of Medicinal Chemistry, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Piet++Herdewijn">Piet Herdewijn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3589-8503" title="Orcid link">http://orcid.org/0000-0003-3589-8503</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steven De Jonghe</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven De Jonghe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology, Immunology and Transplantation, Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2457504152414a0a40414e4b4a434c41644f5148415152414a0a4641"><span class="__cf_email__" data-cfemail="d2a1a6b7a4b7bcfcb6b7b8bdbcb5bab792b9a7beb7a7a4b7bcfcb0b7">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven++De+Jonghe">Steven De Jonghe</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3872-6558" title="Orcid link">http://orcid.org/0000-0002-3872-6558</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Kalyan Das</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kalyan Das</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology, Immunology and Transplantation, Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f79c969b8e9699d9939684b79c829b9282819299d99592"><span class="__cf_email__" data-cfemail="f79c969b8e9699d9939684b79c829b9282819299d99592">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kalyan++Das">Kalyan Das</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8897-324X" title="Orcid link">http://orcid.org/0000-0002-8897-324X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01747&amp;href=/doi/10.1021%2Facs.jmedchem.0c01747" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 782–796</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 23, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 October 2020</li><li><span class="item_label"><b>Published</b> online</span>23 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01747" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01747</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D782%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DWeijie%2BGu%252C%2BSergio%2BMartinez%252C%2BHoai%2BNguyen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D1%26contentID%3Dacs.jmedchem.0c01747%26title%3DTenofovir-Amino%2BAcid%2BConjugates%2BAct%2Bas%2BPolymerase%2BSubstrates%25E2%2580%2594Implications%2Bfor%2BAvoiding%2BCellular%2BPhosphorylation%2Bin%2Bthe%2BDiscovery%2Bof%2BNucleotide%2BAnalogues%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D796%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01747"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">777</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01747" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Tenofovir-Amino Acid Conjugates Act as Polymerase Substrates—Implications for Avoiding Cellular Phosphorylation in the Discovery of Nucleotide Analogues&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Weijie&quot;,&quot;last_name&quot;:&quot;Gu&quot;},{&quot;first_name&quot;:&quot;Sergio&quot;,&quot;last_name&quot;:&quot;Martinez&quot;},{&quot;first_name&quot;:&quot;Hoai&quot;,&quot;last_name&quot;:&quot;Nguyen&quot;},{&quot;first_name&quot;:&quot;Hongtao&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Piet&quot;,&quot;last_name&quot;:&quot;Herdewijn&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;De Jonghe&quot;},{&quot;first_name&quot;:&quot;Kalyan&quot;,&quot;last_name&quot;:&quot;Das&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;782-796&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01747&quot;},&quot;abstract&quot;:&quot;Nucleotide analogues are used for treating viral infections such as HIV, hepatitis B, hepatitis C, influenza, and SARS-CoV-2. To become polymerase substrates, a nucleotide analogue must be phosphorylated by cellular kinases which is rate-limiting. The goal of this study is to develop dNTP/NTP analogues directly from nucleotides. Tenofovir (TFV) analogues were synthesized by conjugating with amino acids. We demonstrate that some conjugates act as dNTP analogues and HIV-1 reverse transcriptase (RT) catalytically incorporates the TFV part as the chain terminator. X-ray structures in complex with HIV-1 RT/dsDNA showed binding of the conjugates at the polymerase active site, however, in different modes in the presence of Mg2+ versus Mn2+ ions. The adaptability of the compounds is seemingly essential for catalytic incorporation of TFV by RT. 4d with a carboxyl sidechain demonstrated the highest incorporation. 4e showed weak incorporation and rather behaved as a dNTP-competitive inhibitor. This result advocates the &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01747&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01747" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01747&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01747" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01747&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01747" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01747&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01747&amp;href=/doi/10.1021/acs.jmedchem.0c01747" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01747" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01747" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01747%26sid%3Dliteratum%253Aachs%26pmid%3D33356231%26genre%3Darticle%26aulast%3DGu%26date%3D2021%26atitle%3DTenofovir-Amino%2BAcid%2BConjugates%2BAct%2Bas%2BPolymerase%2BSubstrates%25E2%2580%2594Implications%2Bfor%2BAvoiding%2BCellular%2BPhosphorylation%2Bin%2Bthe%2BDiscovery%2Bof%2BNucleotide%2BAnalogues%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D1%26spage%3D782%26epage%3D796%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292559" title="Chelation">Chelation</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=290667" title="Ions">Ions</a>,</li><li><a href="/action/doSearch?ConceptID=292316" title="Transition metals">Transition metals</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/jmcmar.2021.64.issue-1/20210114/jmcmar.2021.64.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Nucleotide analogues are used for treating viral infections such as HIV, hepatitis B, hepatitis C, influenza, and SARS-CoV-2. To become polymerase substrates, a nucleotide analogue must be phosphorylated by cellular kinases which is rate-limiting. The goal of this study is to develop dNTP/NTP analogues directly from nucleotides. Tenofovir (TFV) analogues were synthesized by conjugating with amino acids. We demonstrate that some conjugates act as dNTP analogues and HIV-1 reverse transcriptase (RT) catalytically incorporates the TFV part as the chain terminator. X-ray structures in complex with HIV-1 RT/dsDNA showed binding of the conjugates at the polymerase active site, however, in different modes in the presence of Mg<sup>2+</sup> versus Mn<sup>2+</sup> ions. The adaptability of the compounds is seemingly essential for catalytic incorporation of TFV by RT. <b>4d</b> with a carboxyl sidechain demonstrated the highest incorporation. <b>4e</b> showed weak incorporation and rather behaved as a dNTP-competitive inhibitor. This result advocates the feasibility of designing NTP/dNTP analogues by chemical substitutions to nucleotide analogues.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58317" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58317" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Nucleoside and nucleotide inhibitors are among the leading drugs used for the treatment of human immunodeficiency virus (HIV)-infected patients.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Zidovudine (3′-azidothymidine, AZT) is the first HIV drug approved in 1987, and currently, the nucleoside analogues abacavir (ABC), emtricitabine (FTC), lamivudine (3TC), and AZT and nucleotide analogue tenofovir (TFV) are active components (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) of the drug cocktails used in treating HIV infection under antiretroviral therapy (ART).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Nucleos(t)ide analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Ribonucleotide analogues are used for treating infections caused by the viruses which use RNA-dependent RNA polymerases (RdRp) for replication/transcription of viral genes–such as hepatitis C (HCV), influenza, and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). A nucleoside analogue requires three-step phosphorylation, and a nucleotide such as TFV, the HCV drug sofosbuvir (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), or the SARS-CoV-2 drug remdesivir requires the final two phosphorylation steps; a nucleotide analogue is generally delivered as a prodrug. Several NTP/dNTP analogues have been synthesized and have demonstrated efficacy <i>in vitro</i>.<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8">(2−8)</a> However, cellular phosphorylation is a key hurdle in the successful development of most nucleotide analogues as drugs. Chemical substitutions that can convert a nucleotide to an NTP/dNTP mimic have broad significance in designing polymerase inhibitors.</div><div class="NLM_p">The transcription and replication of viral genes are carried out by RNA or DNA polymerases, and the polymerization reaction is accomplished by a conserved catalytic mechanism analogous to that described for <i>Escherichia coli</i> DNA polymerase I.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Polymerase active sites contain highly conserved catalytic aspartate or glutamate residues that chelate two Mg<sup>2+</sup> ions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The Mg<sup>2+</sup> ion occupying site A deprotonates the primer 3′-hydroxyl, and the second Mg<sup>2+</sup> ion at site B positions the triphosphate group appropriately for catalysis by chelating all three phosphates. The nucleophilic attack by 3′-OH at the α-phosphate of incoming dNTP forms a phosphodiester bond between the two groups, and β–γ pyrophosphate is released as a byproduct. Activation of a nucleophile by Mg<sup>2+</sup> can also be found in other enzymes that cleave the phosphodiester bonds in a DNA backbone, such as some nucleases.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The chelation of the triphosphate moiety to the catalytic metal ions at the DNA and RNA polymerase active sites can be exploited for drug design analogous to the design of HIV integrase inhibitors from the diketo acid scaffold.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Metal chelation at the active site of HIV-1 RT. (A) Octahedral coordination of Mg<sup>2+</sup> ions in the HIV-1 RT/DNA catalytic complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PQU">4PQU</a>); the primer strand was terminated with dideoxyguanosine to prevent incorporation. (B) Schematic representation of Mg<sup>2+</sup> ion chelation in the polymerase active site of HIV-1 RT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The enzyme reverse transcriptase (RT) that copies HIV-1 genomic RNA to a dsDNA at the early stage of the viral infection has been well studied biochemically and structurally. RT has been the target of almost half of the approved antiviral drugs for treating HIV infection. The non-nucleoside RT inhibiting drugs (NNRTIs) allosterically inhibit DNA polymerization. The nucleoside/nucleotide RT inhibitors (NRTIs) are incorporated by RT to the growing primer strand and act as chain terminators (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The polymerase active site of HIV-1 RT is also an attractive target for designing dNTP-competitive inhibitors. Small molecule inhibitors such as INDOPY-1<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> and α-carboxy nucleoside phosphonates (α-CNPs)<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18 ref19 ref20">(14−20)</a> have been discovered as nucleoside-competing RT inhibitors (NcRTIs). INDOPY-1 binding does not involve metal chelation,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> whereas α-CNPs chelate the active-site metal ions.<a onclick="showRef(event, 'ref18 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref18 ref21 ref22">(18,21,22)</a> A series of α-CNPs, such as T-α-CNP, (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) were synthesized as NcRTIs.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18 ref19 ref20">(14−20)</a> α-CNPs keep the natural nucleobases, but the 2′-deoxyribose moiety is substituted with a cyclopentyl ring. On this latter ring, a 3′ oxygen is linked to the α-methyl carbon, which is linked to a carboxylic acid and a phosphonate group. Both groups chelate catalytic Mg<sup>2+</sup> ions in the polymerase active site, and the chelation of α-CNP mimics that of the triphosphate moiety of a dNTP.</div><div class="NLM_p">The alanine substitution has been used in the prodrug formulation of TFV, Sofosbuvir, and Remdesivir. In contrast, the amino-acid substitutions in the current study are introduced as the pyrophosphate  analogs that would act as the leaving groups in the nucleic acid polymerization reaction. The Herdewijn laboratory previously synthesized phosphoramidate conjugates of 2′-deoxynucleotides and natural amino acids (e.g., <span class="smallcaps smallerCapital">l</span>-aspartic acid, <span class="smallcaps smallerCapital">l</span>-histidine),<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> unnatural amino acids (e.g. <span class="smallcaps smallerCapital">d</span>-aspartic acid, iminodiacetate (IDA), iminodipropionate (IDP), and 3-phosphono-<span class="smallcaps smallerCapital">l</span>-alanine),<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> and dipeptides.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> It was demonstrated that several congeners within this family (e.g. 5′-<span class="smallcaps smallerCapital">l</span>-aspartic- and 5′-<span class="smallcaps smallerCapital">l</span>-histidinyl-phosphoramidate analogues of 2′-dAMP) were active in a primer extension assay. For example, <span class="smallcaps smallerCapital">l</span>-Asp-dAMP was able to mimic the natural nucleoside triphosphate dATP, and the 2′-deoxy-adenosine part is incorporated into a growing DNA strand by HIV-1 RT. However, for <span class="smallcaps smallerCapital">l</span>-Asp-dAMP, the catalytic efficiency (<i>V</i><sub>max</sub>/<i>K</i><sub>m</sub>) is decreased 1300-fold compared to dATP primarily due to an increase in <i>K</i><sub>m</sub>.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Only one example of a nucleotide analogue conjugated to an amino acid has been described. <span class="smallcaps smallerCapital">l</span>-Asp-PMEA (PhosphonoMethoxyEthylAdenine) has been shown to act as a chain terminator in a primer extension assay.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> As a general principle, the conjugates can convert a nucleoside (or analogue) into an active form for direct incorporation by RT or other polymerases, without needing the metabolic conversion. However, up to now, there is no structural information available for visualizing how these alternative leaving groups of the conjugates bind and what drives the catalytic incorporation at the polymerase active site.</div><div class="NLM_p">Our current study follows the earlier chemical synthesis work to generate a series of conjugates of the amino acids <span class="smallcaps smallerCapital">l</span>-Asp, <span class="smallcaps smallerCapital">d</span>-Asp, <span class="smallcaps smallerCapital">l</span>-His, <span class="smallcaps smallerCapital">l</span>-Glu, <span class="smallcaps smallerCapital">l</span>-Met, <span class="smallcaps smallerCapital">l</span>-Arg, and <span class="smallcaps smallerCapital">l</span>-Phe with the nucleotide dAMP and TFV (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). All compounds were tested for catalytic incorporation by RT and for RT inhibition. Only <b>3a</b> demonstrated multiple nucleotide incorporation among the dAMP conjugates, whereas several TFV conjugates demonstrated incorporation of TFV to the primer strand as a chain terminator. Crystal structures of TFV conjugates were determined in complex with HIV-1 RT/dsDNA in the catalytic competent state. The structures were analyzed in the context of their binding and modes of inhibition.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and Conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) amino acid ester, DCC, <i>t</i>-BuOH, H<sub>2</sub>O, reflux, 3 h; (ii) 0.4 M NaOH, MeOH, rt, 3 h.</p></p></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56086" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56086" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemical Synthesis</h3><div class="NLM_p last">The synthesis of the amino acid conjugates of dAMP and TFV was achieved according to a previously reported procedure<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Briefly, a <i>N,N</i>-dicyclohexylcarbodiimide (DCC) mediated coupling of an appropriate amino acid ester with dAMP (compound <b>1</b>) or TFV (compound <b>2</b>), followed by alkaline deprotection of the ester moiety that furnished the desired dAMP amino acid conjugates <b>3a–g</b> as well as the TFV conjugates <b>4a–g</b>. The following amino acids were selected for coupling: <span class="smallcaps smallerCapital">l</span>-aspartic acid (series <b>a</b>), <span class="smallcaps smallerCapital">d</span>-aspartic acid (series <b>b</b>), <span class="smallcaps smallerCapital">l</span>-histidine (series <b>c</b>), <span class="smallcaps smallerCapital">l</span>-glutamic acid (series <b>d</b>), <span class="smallcaps smallerCapital">l</span>-methionine (series <b>e</b>), <span class="smallcaps smallerCapital">l</span>-arginine (series <b>f</b>), and <span class="smallcaps smallerCapital">l</span>-phenylalanine (series <b>g</b>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Primer Extension Assay by HIV-1 RT</h3><div class="NLM_p">The ability of HIV-1 RT to incorporate the amino acid conjugates <b>3e–g</b> and <b>4a–g</b> into a growing DNA strand was investigated by a gel-based nucleotide incorporation assay with the primer (P) and template (T) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–E). The primer-template duplex has 7 overhanging dT nucleotides at the 5′ end of the template, and hence, this template/primer combination allows seven primer extensions. Compounds <b>3a</b>–<b>d</b> have been evaluated earlier in this assay, and it was demonstrated that compound <b>3a</b> was best at incorporation among dAMP analogues.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> The <i>V</i><sub>max</sub> (rate attained when substrate concentration is much higher than <i>K</i><sub>m</sub>) of compound <b>3a</b> was only one-third of that of the natural substrate dATP, but the specificity of incorporation (<i>V</i><sub>max</sub>/<i>K</i><sub>m</sub>) was 1,300 times lower compared to dATP.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a><b>3c</b> was close behind at 1.5 times less efficient than <b>3a</b>. The remaining amino acids (<span class="smallcaps smallerCapital">d</span>-aspartic acid: compound <b>3b</b>; <span class="smallcaps smallerCapital">l</span>-glutamic acid: compound <b>3d</b>) behaved significantly worse as substrates for RT. This indicates that the stereochemistry and the chemical composition of the amino acids are important for the amino acid-dAMP conjugates to function as substrates of HIV-1 RT. Compounds <b>3e</b>, <b>3f</b>, and <b>3g</b> (carrying a <span class="smallcaps smallerCapital">l</span>-methionine, <span class="smallcaps smallerCapital">l</span>-arginine, and <span class="smallcaps smallerCapital">l</span>-phenylalanine side chain, respectively, instead of the pyrophosphate moiety) are novel and were assessed in the current study for their ability to be incorporated in a growing DNA strand, catalyzed by HIV-1 RT. Compound <b>3a</b> was selected as a reference compound, whereas dATP was included as the positive control. Compound <b>3a</b> was tested at a concentration of 500 μM with 3 mM MgCl<sub>2</sub> as the metal ion source. After 1 h of polymerase reaction, most of the primer extended by 2 or 3 bases, which is in agreement with previous data.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> None of the novel amino acid dAMP conjugates (compounds <b>3e</b>–<b>g</b>) showed noticeable incorporation when tested at concentrations of 500 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) up to 1 mM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), either in the presence of Mg<sup>2+</sup>, or with added Mn<sup>2+</sup> ions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A–E).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Primer extension and polymerase inhibition assays. (A–C) Primer extension with amino acid conjugates of 2′-dAMP and TFV by HIV-1 RT. Reaction conditions: 125 nM primer/template; [dATP] = 50 μM; [amino acid conjugate] = 500 μM; [HIV-1 RT] = 0.5 μg/μL, 3 mM MgCl<sub>2</sub>. (D,E) With 2 mM MnCl<sub>2</sub> added. In (E), [amino acid conjugate] = 1 mM. (F) Dose–response curve showing inhibition of HIV-1 RT in the presence of Mg<sup>2+</sup> ions. The inhibitory effect of compound <b>4e</b> on dsDNA synthesis by recombinant HIV-1 RT was determined by PicoGreen-based spectrophotometric assay. Relative levels of dsDNA synthesis were calculated as a percentage of levels in reaction in the absence of the compound. Data points represent the means of representative experiments run in triplicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The TFV conjugates with amino acids (compounds <b>4a</b>–<b>g</b>) were also evaluated in the incorporation assay. When tested at a concentration of 500 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B–C), P + 1 incorporation was observed for compounds <b>4b</b> and <b>4d</b>, especially compound <b>4d</b> leading to substantially more incorporation (26.81% of P to P + 1 conversion, Table S1 <a href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf" class="ext-link">Supporting Information</a>) compared to the remaining conjugate analogues. Unlike dAMP, TFV acts as a chain terminator, and therefore, no additional nucleotide addition was observed.</div><div class="NLM_p last">The presence of 2 mM MnCl<sub>2</sub> improved the incorporation for compounds <b>4a–b</b> and <b>4d–e</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D) with the percentages of P to P+1 conversion of compound <b>4d</b> being the highest (40.78% after 1 h of reaction). When the concentrations of compounds <b>4a</b>, <b>4c,</b> and <b>4g</b> were increased from 500 μM to 1 mM, a small amount of the P+1 product was also observed (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), whereas for compound <b>4f,</b> no incorporation was observed. Overall, among the TFV conjugates, compound <b>4d</b> was the most efficiently incorporated, either with Mg<sup>2+</sup> or in combination with Mn<sup>2+</sup>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Inhibition of HIV-1 RT</h3><div class="NLM_p last">The inhibitory effect of the compounds <b>3a–g</b> and <b>4a–g</b> on the dsDNA synthesis, catalyzed by recombinant HIV-1 RT, was determined by a PicoGreen-based spectrophotometric assay.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> At the highest tested concentration of 2.5 mM, only compound <b>4e</b> demonstrated RT inhibition. Therefore, this compound was subjected to a full dose–response curve analysis using various concentrations of compound <b>4e</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The IC<sub>50</sub> of compound <b>4e</b> was 146 μM in the presence of magnesium.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-ray Crystallography Using RT Cross-Linked to DNA</h3><div class="NLM_p last">For this study, six crystal structures of <b>4b</b>, <b>4d</b>, and <b>4e</b> compounds are determined in complex with HIV-1 RT/DNA either in the presence of MgCl<sub>2</sub>, or MgCl<sub>2</sub> + MnCl<sub>2</sub>. Crystals of RT cross-linked to a dsDNA template/primer were grown following a published protocol.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The crystals were soaked to form complexes with individual compounds. The diffraction data for soaked complexes were collected at synchrotron beamlines, and the structures were determined at resolutions ranging between 2.6 and 3.15 Å. The RT/DNA complex crystallizes with P2<sub>1</sub> space group symmetry and contains two copies of RT/DNA in the asymmetric unit. This crystal form has been used to study several different classes of RT inhibitors, as well as resistance mutations: namely, the complexes with α-CNP,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> dATP,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the allosteric inhibitor nevirapine<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> and the chain terminator AZT-TP before and after primer incorporation.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> In general, the electron density for the inhibitor in one of the two copies is better than the other. Therefore, the copy with the better inhibitor density is used for the analysis and discussion unless otherwise stated.  RT can use either Mg<sup>2+</sup> or Mn<sup>2+</sup> ions for catalysis of DNA polymerization; however, catalysis using Mg<sup>2+</sup> ion requires high precision chelation groups and geometry. In contrast, Mn<sup>2+</sup> ions are more forgiving. Therefore, we prepared the complexes of inhibitors in the presence of either Mg<sup>2+</sup> alone or Mn<sup>2+</sup> and Mg<sup>2+</sup> ions for a comprehensive understanding of the metal chelation and adaptivity of these compounds at the polymerase active site.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structures of RT/DNA/<span class="smallcaps smallerCapital">l</span>-Glu-TFV (Compound <b>4d</b>)</h3><div class="NLM_p">Our incorporation assay showed that the TFV moiety of <b>4d</b> is incorporated into the DNA primer by RT in the presence of either Mg<sup>2+</sup> or Mn<sup>2+</sup> ions. TFV incorporation from <b>4d</b> is the best of all TFV conjugates in this study (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,D), and the incorporation is significantly enhanced with Mn<sup>2+</sup>. RT/DNA crystals were soaked in the crystallization solution containing 1 mM <b>4d</b> for 1 h; the crystallization solution had 5 mM MgCl<sub>2</sub>. For obtaining the structure in the presence of Mn<sup>2+</sup> ions, 2 mM MnCl<sub>2</sub> was added to the above soaking protocol. The crystal structures of the RT/DNA/<b>4d</b> complex in the presence of Mg<sup>2+</sup> or Mg<sup>2+</sup> + Mn<sup>2+</sup> ions were determined at 2.95 and 2.75 Å, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). In the RT/DNA/<b>4d</b>/Mg<sup>2+</sup> structure, the binding of <b>4d</b> at the polymerase active site of RT does not involve metal chelation, and in fact, no Mg<sup>2+</sup> ion is present in the active site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). However, in the presence of the Mn<sup>2+</sup> ion, the compound chelates a metal ion at the active site in the RT/DNA/<b>4d</b>/Mn<sup>2+</sup> structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) 2Fo-Fc electron density map at 2.95 Å resolution defines the binding of <b>4d</b> at the active site of the RT/DNA/<b>4d</b> complex in the presence of MgCl<sub>2</sub>; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG">7AIG</a>. No Mg<sup>2+</sup> ion is present at the polymerase active site. (B) 2Fo-Fc density map at 2.75 Å resolution shows the binding of <b>4d</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF">7AIF</a>. (C) <b>4d</b> binds at the dNTP-binding pocket. Template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxy-guanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4d</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) top view showing the binding of <b>4d</b> to the RT/DNA complex either in the presence of Mg<sup>2+</sup> (orange bonds) or added Mn<sup>2+</sup> ions (green bonds). The Mn<sup>2+</sup> binds at the site B. The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site and repositions the Glu side-chain. Structures were aligned in the palm domain (chain C) using the alpha carbons of residues 85–119, 151–242, and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the structure that does not involve metal chelation, the adenine base of <b>4d</b> base pairs with the thymine of the first base overhangs. However, base pairing and base stacking are not optimal; adenine is tilted approximately 23° away from the plane of the thymine causing a loss of the stacking interactions with the dsDNA nucleotide guanine base at the primer terminus (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The sidechain of the conjugated <span class="smallcaps smallerCapital">l</span>-Glu points toward the triphosphate-binding region of dNTP (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A); however, <span class="smallcaps smallerCapital">l</span>-Glu is less ordered due to lack of significant interaction primarily due to the lack of metal chelation. The backbone carboxyl group of the <span class="smallcaps smallerCapital">l</span>-Glu moiety interacts with the conserved residue Arg72; both oxygens form 3.1 or 2.7 Å salt bridges with the Nε atom and the Nη2 atoms of Arg72, respectively. Arg72 is a part of the fingers β3−β4 hairpin, and Arg72 plays a critical role in the binding and positioning of dNTP at the polymerase active site.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> In dNTP-bound structures, Arg72 side-chain stacks with the dNTP base and forms polar interaction with α-phosphate. In the current structures, however, the orientation of Arg72 is altered when compared to the dNTP-bound RT/DNA structures. The phosphonoamidate group forms a 3.2 Å H-bond with the Nε2 atom of Gln151 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><div class="NLM_p">Despite weak binding of the compound at the active site and without Mg<sup>2+</sup> ion chelation in the structure, <b>4d</b> demonstrates primer extension by RT (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) in the presence of Mg<sup>2+</sup> ions. However, the compound at 2 mM concentration did not show any noticeable RT inhibition in the PicoGreen assay. The inhibition data in agreement with the structures suggests significantly weak binding of the compound <b>4d</b>, and dATP outcompetes <b>4d</b> in the inhibition assay.</div><div class="NLM_p last">The mode of binding of <b>4d</b> in the presence of Mn<sup>2+</sup> ions has a significant difference from that in the presence of Mg<sup>2+</sup>. The electron density reveals the presence of one Mn<sup>2+</sup> ion positioned at the metal B site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) and chelates the catalytic aspartates Asp110 and Asp185, the backbone carbonyl of Val111, and a phosphonoamidate oxygen of <b>4d</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). This positional equivalent of the phosphonoamidate chelation mimics the chelation of the β-phosphate of TFV-DP.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> For the TFV part in <b>4d</b>, the base pairing, base stacking, and the interaction of the acyclic linker are reminiscent of the binding of TFV-DP to RT.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In the structure of RT/DNA/<b>4d</b> in the presence of Mn<sup>2+</sup>, when compared with the structure without a metal ion at the active site, the phosphonate group is pulled toward the active side by ∼1 Å to chelate the Mn<sup>2+</sup> ion (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E,F). Consequently, the binding of the TFV moiety of <b>4d</b> is improved as the adenine base now stacks with the guanine base at the primer end. The conjugated glutamate sidechain and backbone carboxyl groups interact with the guanidinium group of the conserved residue Arg72, and the interactions improved the stability and electron density for the Glu sidechain. The side chain forms a 2.5 Å salt bridge with Nη2, and the backbone carboxylate of Glu forms a 3.0 Å interaction with Nε of Arg72. Comparison of both structures of <b>4d</b> shows that the Glu part is reorganized yet maintains the polar interactions with Arg72 using a different set of atoms. The structure and primer extension data suggest that binding of <b>4d</b> is improved in the presence of Mn<sup>2+</sup> via metal chelation; however, the chelation is not optimal for catalytic incorporation. The suboptimal binding and loss of chelation of <b>4d</b> compared to dATP or TFV-DP are primarily responsible for the weak binding. Two different modes with suboptimal binding of <b>4d</b> support the conformational flexibility, which would help orient the compound favorably for catalytic incorporation by RT. <b>4d</b> is flexible, and its two carboxyl groups and phosphonamidate are mobile in the structure. This flexibility aided with the additional flexibility of the acyclic TFV backbone helps <b>4d</b> attain the transient chelation events with Mg<sup>2+</sup> for TFV incorporation – a structural explanation for the observed incorporation in our assay. Lacking the flexibility, <b>3d</b> did not show dAMP incorporation.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Structures of RT/DNA/<span class="smallcaps smallerCapital">d</span>-Asp-TFV (Compound <b>4b</b>)</h3><div class="NLM_p">The TFV moiety of <b>4b</b> is incorporated into the DNA primer by RT in the presence of either Mg<sup>2+</sup> or Mn<sup>2+</sup> ions; TFV incorporation is enhanced in the presence of Mn<sup>2+</sup> ions. TFV incorporation from <b>4b</b>, however, is about half of that from <b>4d</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,D). In the same fashion as for <b>4d</b>, we soaked the crystals of HIV-1 RT/dsDNA with <b>4b</b> either in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup> and Mg<sup>2+</sup>. The crystal structures of RT/DNA/<b>4b</b>/Mg<sup>2+</sup> and RT/DNA/<b>4b</b>/Mn<sup>2+</sup> complexes were obtained at 3.15 and 2.73 Å, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). Analogous to the binding of <b>4d</b>, no metal chelation is involved in the binding of <b>4b</b> in the structure obtained only in the presence of Mg<sup>2+</sup> ions (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). In the second structure, an Mn<sup>2+</sup> ion is chelating the compound <b>4b</b> at the active site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) 2Fo-Fc electron density map at 3.15 Å resolution showing the binding of <b>4b</b> at the active site of the RT/DNA/<b>4b</b> complex in the presence of Mg<sup>2+</sup>; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID">7AID</a>. (B) 2Fo-Fc density map at 2.73 Å resolution showing the binding of <b>4b</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX">7AHX</a>. (C) <b>4b</b> sits in the dNTP binding site in the presence of Mg<sup>2+</sup>; no Mg<sup>2+</sup>ion is present in the polymerase active site. The template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxyguanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4b</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) Top view of <b>4b</b> in complex with RT in the presence of either Mg<sup>2+</sup> (orange bonds) or Mg<sup>2+</sup> + Mn<sup>2+</sup> (green bonds). Mn<sup>2+</sup> binds at the metal ion site B. The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site, and repositions the <span class="smallcaps smallerCapital">d</span>-Asp side-chain. Structures were aligned on the palm domain (chain C) using the alpha carbons of residues 85–119 and 151–242 and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In either case, the adenine base-pairs with the thymine of the template first base overhang. However, base pairing and base stacking are not optimal when <b>4b</b> is not involved in metal chelation, as the case for <b>4d</b>. The adenine base of <b>4b</b> is tilted by approximately 19°, nearly the same amount for <b>4d</b>, away from the plane of the template thymine. The sidechain of the conjugated <span class="smallcaps smallerCapital">d</span>-Asp points toward the triphosphate-binding site; however, unlike the triphosphate moiety of a dNTP, <span class="smallcaps smallerCapital">d</span>-Asp is not involved in metal chelation and has poor density due to the absence of significant interactions (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,C). The backbone carboxyl group of the <span class="smallcaps smallerCapital">d</span>-Asp moiety interacts with the conserved residue Arg72. One oxygen forms a 2.7 Å salt bridge with the Nε atom of Arg72. The phosphonoamidate group of <b>4b</b>, unlike that of <b>4d</b>, is not involved in any polar interaction.</div><div class="NLM_p">In the presence of Mg<sup>2+</sup>, <b>4b</b> demonstrated primer extension by RT despite the lack of Mg<sup>2+</sup> ion chelation in our structure; however, the efficiency of primer extension from <b>4b</b> is less than that from <b>4d</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The modes of binding of <b>4b</b>, which is similar to that of <b>4d</b>, could explain the TVF incorporation from <b>4b</b> by a transient chelation event with Mg<sup>2+</sup>, which is again aided by two carboxyl groups and the flexible acyclic TFV backbone.</div><div class="NLM_p last"><b>4b</b> also has a significantly different mode of binding in the presence of Mn<sup>2+</sup> ions compared to Mg<sup>2+</sup>, which is analogous to the binding of <b>4d</b>. For the TFV moiety of <b>4b</b>, the base pairing and base stacking is improved, and the acyclic linker is repositioned to orient TFV moiety binding in a way similar to that of TFV-DP to RT<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). The conjugated <span class="smallcaps smallerCapital">d</span>-Asp sidechain and backbone interact with the guanidinium group of Arg72, and the interaction improved the stability and electron density for the <span class="smallcaps smallerCapital">d</span>-Asp side chain (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). As for <b>4d</b>, the two carboxyl groups of <b>4b</b> interact with the guanidinium of Arg72. The side chain forms a 2.6 Å salt bridge with Nη2, and the backbone carboxylate forms one of 2.4 Å to Nε. The phosphonoamidate group is moved about 0.8 Å closer toward the Mn<sup>2+</sup> ion at metal site B for chelation compared to its position in a nonchelated state (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E,F). As a consequence, both carboxyl groups of the <span class="smallcaps smallerCapital">d</span>-Asp moiety now interact with Arg72. The TFV incorporation is very similar for <span class="smallcaps smallerCapital">l</span>- or <span class="smallcaps smallerCapital">d</span>-Asp conjugates in the presence of Mn<sup>2+</sup> ions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D).The longer and more flexible side-chain of Glu in <b>4d</b> appears to be more adaptive for the carboxyl side-chain compared to that of Asp in <b>4b</b> and is possibly better able to interact as a leaving group; Arg72 interacts with the pyrophosphate leaving group of a dNTP in the process of the polymerization reaction.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> There is no RT inhibition at 2 mM of either compounds.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structure of RT/DNA/<span class="smallcaps smallerCapital">l</span>-Met-TFV (Compound <b>4e</b>)</h3><div class="NLM_p">The incorporation of the TFV moiety from <b>4e</b> by RT is negligible either in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup> ions, when compared with TFV incorporation from <b>4d</b> or <b>4b</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,D). The crystal structures of RT/DNA/<b>4e</b> complexes in the absence and presence of Mn<sup>2+</sup> were obtained at 2.95 and 2.62 Å, respectively (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Analogous to <b>4d</b>, the binding of <b>4e</b> at the polymerase active site does not involve metal chelation when the complex was formed in the presence of Mg<sup>2+</sup> ions only (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C), whereas the complex in the presence of Mn<sup>2+</sup> ions involves metal chelation at the active site (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). With Mg<sup>2+</sup>, there is much less tilting of the adenine base when compared with the binding of <b>4d</b> or <b>4b</b>. Adenine is tilted only by 9° out of the plane of its complementary thymine base of the template overhang. However, the adenine base still loses stacking interaction with the guanine base at the primer end analogous to that observed for <b>4d</b> and <b>4b</b> in the presence of Mg<sup>2+</sup> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Unlike the previous conjugates with carboxyl-containing side chains, a methionine can have hydrophobic interactions and cannot participate in metal chelation. The sidechain of the conjugated <span class="smallcaps smallerCapital">l</span>-Met only has clear density for the sulfur atom, which is weakly interacting with Arg72; the distance between the <span class="smallcaps smallerCapital">l</span>-Met methyl group and Nη2 of Arg72 is ∼3.4 Å (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C,D). The backbone carboxyl group of the <span class="smallcaps smallerCapital">l</span>-Met moiety interacts with the guanidinium group of Arg72, forming a 2.3 Å salt bridge with Nη2. Two phosphonoamidate oxygens form H-bonds—a 2.7 Å H-bond with the backbone amide of Ala114 and a 2.7 Å H-bond with Nε2 of Gln151, the latter is also observed for <b>4d</b>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) 2Fo-Fc electron density map at 3.15 Å resolution defines the position of <b>4e</b> in the active site of the RT/DNA/4e complex in the presence of Mg<sup>2+</sup> ions; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ">7AIJ</a>. (B) 2Fo-Fc density map at 2.73 Å resolution defines the binding of <b>4e</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII">7AII</a>. (C) <b>4e</b> sits in the dNTP-binding site in the presence of MgCl<sub>2</sub>; no Mg<sup>2+</sup> ion is present at the active site. The template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxyguanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4e</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) Top view of <b>4e</b> in complex with RT in the presence of either Mg<sup>2+</sup> (orange bonds) or Mg<sup>2+</sup>+ Mn<sup>2+</sup> (green bonds). The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site, and repositions the Met side-chain. Structures were aligned in the palm domain (chain C) using the alpha carbons of residues 85–119 and 151–242 and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the presence of Mg<sup>2+</sup>, 500 μM <b>4e</b> demonstrated relatively little primer extension by RT (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The PicoGreen assay gave an IC<sub>50</sub> of 146 μM with Mg<sup>2+</sup>, the only noticeable inhibition by any of the compounds covered in this study (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). <b>4e</b> appears to be a competitive inhibitor and not a good substrate for TFV incorporation.</div><div class="NLM_p last"><b>4e</b> also has a significantly different mode of binding in the presence of Mn<sup>2+</sup> ions compared to Mg<sup>2+</sup>. For the TFV moiety, the base pairing, base stacking, and interaction of the acyclic linker are also similar to those of <b>4b</b> and <b>4d</b> and the binding of TFV-DP to RT<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). The sulfur and the Cε methyl groups of the L-Met sidechain are at a distance of ∼3.4 Å from the Nη2 of Arg72 in both structures of <b>4e</b>. The backbone carboxyl forms a 2.7 Å salt bridge with Nε of Arg72. An indication of improved binding is that the β3−β4 finger loop, has a better density in the Mn<sup>2+</sup> structure; β3−β4 finger loop is closed down and better ordered in the RT structures with a bound dNTP. The electron density reveals the presence of one Mn<sup>2+</sup> ion positioned at the metal B site and chelates the catalytic aspartates Asp110 and Asp185, the backbone carbonyl of Val111, and a phosphonoamidate oxygen of <b>4e</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). The phosphonoamidate group is moved about 1.9 Å, highest among all three compounds, closer toward the Mn<sup>2+</sup> ion for chelation compared to its position in the nonchelated state (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E,F). The key structural differences of <b>4e</b> from <b>4d</b> and <b>4b</b>, as discussed below, may be contributing to the better RT inhibition by <b>4e</b> compared to the remaining analogues. <b>4e</b> has less tilting of its adenine than <span class="smallcaps smallerCapital">l</span>-Glu or <span class="smallcaps smallerCapital">d</span>-Asp conjugates, improving the base stacking. The phosphonoamidate group requires a large shift (1.9 Å) while going from free to the metal chelation state making the free state less favorable for catalysis. Finally, the phosphonoamidate group is more stabilized in its nonchelating state by forming two hydrogen bonds to the protein, compared to none for <b>4b</b> and one for <b>4d</b>. The TFV part of <b>4e</b> binds in an optimized way even in the absence of metal chelation which is contrary to that observed for the other two compounds. The structure shows a weak hydrophobic interaction of the Met side-chain with the guanidinium group of Arg72. The H-bonds of <b>4b</b> and <b>4d</b> with the Arg72 side-chain are solvent-exposed and can be replaced for dNTP binding with almost no energy cost. Apparently, a hydrophobic interaction of the guanidinium group of Arg72 may cost a small resistance to the adaptability of Arg72, which may contribute to the observed resistance by <b>4e</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our results show the incorporation of TFV from TFV-amino acid conjugates by RT as a chain terminator, and different incorporation efficiency is observed for different amino acid conjugates. The <span class="smallcaps smallerCapital">l</span>-Glu conjugate that contains a flexible carboxyl sidechain provides the highest TFV incorporation, followed by <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-Asp conjugates. The amino acid conjugate Met that lacks the carboxyl side chain does not help TFV incorporation. It appears that the main-chain and side-chain carboxyl groups of Glu and Asp have the ability to chelate the metal ions at the active site and help to achieve catalytic incorporation of TFV. The structures show that the carboxyl groups form salt-bridges with the guanidinium side chain of Arg72. Arg72 is a highly conserved residue that plays an important role in the binding and positioning of dNTP for catalysis and pyrophosphate release.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a></div><div class="NLM_p">The crystal structures of TFV derivatives in complexes with RT show that the compounds are not bound in catalytic competent mode (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>–<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). However, structures indicate weaker and flexible binding of the compounds at the polymerase active site. We did not observe any Mg<sup>2+</sup> ion involved in the binding of the inhibitors at the active site. Structures of the complexes in the presence of MnCl<sub>2</sub> reveal the presence of one Mn<sup>2+</sup> ion that approximately takes the position of metal B, and the ion is chelating only one phosphonate oxygen. Compared to the binding mode of the compounds with no metal chelation, the phosphonoamidate groups of the compounds are shifted toward the active site for chelating the Mn<sup>2+</sup> ion. Consequently, the binding mode of the TFV part is aligned with the binding mode of TFV of TFV-DP,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and the amino acid parts of our compounds are repositioned compared to the respective binding modes in the absence of a metal ion.</div><div class="NLM_p">The catalytic incorporation efficiency of the TFV from <b>4d</b> and <b>4b</b> is much less than that from TFV-DP. In the crystal structure, TFV-DP is bound in the catalytic mode.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In contrast, the low incorporation suggests that the catalytic-relevant conformation of our compounds in the active site is a rather high-energy conformation, and our structures do not capture that conformation. Weaker binding of our compounds would support faster association (<i>k</i><sub>on</sub>) and dissociation (<i>k</i><sub>off</sub>) rates, which in combination with the internal flexibility appears to increase the probability of attaining the favorable conformation for catalytic incorporation of TFV. Despite no Mg<sup>2+</sup> ion is present at the active site of the complexes in crystal structures, RT catalytically incorporates TFV from Asp/Glu conjugates suggesting that Mg<sup>2+</sup> ions get involved in the dynamic process of binding of the compounds and catalytic incorporation of TFV. In fact, the bound Mn<sup>2+</sup> ion present in the structures does not substantially enhance the rate of incorporation compared to Mg<sup>2+</sup> ions because the observed mode of chelation with Mn<sup>2+</sup> is not catalytically competent, and the binding mode would take conformational rearrangement for catalytic incorporation.</div><div class="NLM_p">As discussed above, the compounds are weak binders compared to a dATP substrate and their H-bond interactions with Arg72 are solvent-exposed. Therefore, the compounds fail to compete with dATP binding, and consequently, we did not observe noticeable RT inhibition. One of the major differences in the modes of binding is that a dNTP binding involves three phosphate oxygens forming four chelations with both catalytic metal ions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). In contrast, our compounds only have one chelation when the Mn<sup>2+</sup> ion is present. α-CNPs, a class of NcRTIs that show comparable metal chelation at the active site as dNTPs, have comparable RT binding affinity to that of dNTP. A crystal structure bound to RT/dsDNA shows that T-α-CNP binding replaces all four chelations of a dNTP with both Mg<sup>2+</sup> ions at the active site.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The carboxyl group of T-α-CNP mimics the chelation of the α-phosphate oxygen of a dNTP, and two phosphonate oxygens mimic the chelation of β- and γ-phosphates. T-α-CNP in the <span class="smallcaps smallerCapital">l</span>-enantiomer form was found to inhibit several viral polymerases, with the best IC<sub>50</sub> of 0.41 μM for HIV-1 RT. The notion that octahedral coordination correlates with tighter inhibition is also supported by the study in which the phosphonate group of α-CNP is substituted with a carboxyl group. A 2.95 Å crystal structure in complex with HIV-1 RT/dsDNA show weaker metal chelation.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This reduced chelation was reflected in a 50-fold higher IC<sub>50</sub> to 20 μM for two carboxyl-containing compounds.</div><div class="NLM_p">The residue Arg72 and the structural element β3−β4 finger loop that contains Arg72 are solvent-exposed and highly flexible. This flexibility is critical for dNTP binding. Thereby, the polar interactions between Arg72 and carboxyl groups of Glu/Asp conjugates are solvent-exposed and can be easily dissociated when a dNTP competes for binding, which could be a possible explanation for not observing noticeable inhibition by the compounds. We observed measurable but weak inhibition of polymerization only by <b>4e</b> that lacks a carboxyl side chain.</div><div class="NLM_p">In comparison to TFV analogues, most dAMP-amino acid conjugates do not show incorporation. The increased flexibility of the TFV linker compared to a cyclic deoxyribose ring of dAMP appears to play a role in facilitating catalytic incorporation. However, <b>3a</b> demonstrated multiple incorporations of dAMP molecules (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) suggesting that the binding mode of <b>3a</b> is somewhat favorable for attaining the catalytic-competent mode for dAMP incorporation. The <i>V</i><sub>max</sub> of <b>3a</b> is 1/3 that of dATP, but the <i>K</i><sub>m</sub> is 403-fold higher.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Thus, the specificity of incorporation (<i>V</i><sub>max</sub>/<i>K</i><sub>m</sub>) is 1300-fold lower than for dATP. Given that the difference is almost entirely due to the higher <i>K</i><sub>m</sub>, the binding mode of <b>3a</b> is not optimal for catalytic incorporation by RT. The higher <i>K</i><sub>m</sub> values of our compounds also correlate with their nonproductive modes of binding. No significant RT inhibition by any dAMP conjugate suggests that <b>3a</b> binding is also not optimum for the compound to act as a competitive inhibitor.</div><div class="NLM_p last">Our study provides the first structural perspective of nucleotide-amino acid conjugates functioning as chain terminators or competitive inhibitors. It appears that the binding of a conjugate in a catalytically competent mode is not energetically favorable and the process of association/dissociation of the molecules helps attain the chelating mode right for incorporation. We expect that the compounds with a right substitutions will generate dNTP analogues for efficient incorporation. Such studies on viral RNA polymerases may help in developing NTP analogues to avoid cellular phosphorylation as a relatively small number of nucleotide analogues are approved for the treatment of RNA viruses. Compared to the open and highly flexible dNTP-binding pocket of RT, the NTP binding pocket of viral RdRp, such as that of SARS-CoV-2,<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> are better formed, suggesting that the current approach may be valuable for designing RdRp inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07874" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07874" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have solved, at 2.6–3.15 Å, the structures of HIV-1 RT/DNA in complex with analogues of TFV diphosphate that have been modified by replacing the β and γ phosphate groups with an <span class="smallcaps smallerCapital">l</span>- or <span class="smallcaps smallerCapital">d</span>-amino acid conjugated through its amino group to the α phosphate or phosphonate moiety. <b>4d</b> was found to work as intended, generating one primer extension in our assay. Metal chelation and electron density were seen for Mn<sup>2+</sup>, but not for Mg<sup>2+</sup>. However, the loss of chelation causes the phosphonoamidate group to move away from metal site B and generate new H-bond interactions with key residues or a backbone amide in the active site. A large variation in the modes of binding and chelation of the compounds provides opportunities for developing target specific inhibitors of various RNA and DNA polymerases. Structural understanding will help design metal chelators as improved (d)NTP-competing inhibitors or (d)NTP analogues that will be efficiently incorporated by a polymerase without requiring cellular phosphorylation. The broader implication of this study is to explore how the conjugates exhibit different modes of inhibition. Currently, the compounds are weakly active in the enzymatic (incorporation and competition) assays and, therefore, are not further profiled in antiviral assays. In addition, the chemical and metabolic stability of these types of compounds are problematic for further biological profiling, as previously shown by our group.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> However, the structural understanding of the binding modes of these compounds will allow us to improve further their enzymatic potency.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemical Synthesis</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> General Information</h4><div class="NLM_p last">All reagents and solvents were purchased from commercial sources and were used as obtained. Moisture-sensitive reactions were carried out in an oven-dried glassware under a nitrogen or argon atmosphere. <sup>1</sup>H, <sup>13</sup>C, and <sup>31</sup>P NMR spectra were recorded on Bruker Avance 300 or 500 MHz spectrometer using tetramethylsilane as the internal standard or referenced to the residual solvent signal and 85% H<sub>3</sub>PO<sub>4</sub> for <sup>31</sup>P NMR experiments. Splitting patterns are designated as s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), br(broad), and dd (doublet of doublets). Coupling constants are expressed in hertz (Hz). High-resolution mass spectra (HRMS) were obtained on a quadruple orthogonal acceleration time-of-flight mass spectrometer (Synapt G2 HDMS, Waters, Milford, MA). Samples were infused at 3 μL/min, and spectra were obtained in positive (or negative) ionization mode with a resolution of 15,000 (fwhm) using leucine enkephalin as lock mass. Precoated aluminum sheets (254 nm) were used for TLC. Compounds were visualized with UV light (λ = 254 nm). Products were purified by flash chromatography on silica gel (35–70 μm, 60 Å). All final compounds were purified by preparative RP-HPLC (C18 Phenomenex Gemini column, 110 Å, 10 μm, 21.2 mm × 250 mm) using an eluent gradient of CH<sub>3</sub>CN with 50 mmol TEAB as the eluent buffer. Final compounds were isolated as their triethylammonium salts. Prior to biological evaluation, compounds were converted into the corresponding sodium salts via a sodium ion exchange resin. The exact concentration of the compounds was determined by the Lambert–Beer law. The purity of all tested compounds was more than 95%, as determined by HPLC analysis, using the analytical method mentioned before.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-aspartic Acid Triethylammonium Salt (<b>3a</b>)</h4><div class="NLM_p">In a two-neck round bottom flask, 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-aspartic acid dimethyl ester hydrochloride (417 mg, 2.11 mmol) were dissolved in a mixture of <i>t</i>-BuOH and water (5:1, 12 mL). Et<sub>3</sub>N (0.7 mL) was added, followed by the dropwise addition of a freshly prepared solution of DCC (626 mg, 3.02 mmol) in <i>t</i>-BuOH (1.2 mL). The reaction mixture was heated for 2–3 h at 85 °C while stirring under argon. The progress of the reaction was monitored by TLC (<i>i</i>-PrOH/H<sub>2</sub>O/NH<sub>3</sub>, 7:2:2). Upon completion of the reaction, the reaction mixture was cooled down to room temperature and the solvent was removed by rotary evaporation keeping the temperature of the water bath below 30 °C. The resulting crude residue was purified by silica gel flash chromatography, the mobile phase being a gradient of CH<sub>2</sub>Cl<sub>2</sub>/MeOH/H<sub>2</sub>O (v/v/v) in a ratio gradually ranging from 5:1:0 to 5:2:0.25 and 5:3:0.25, affording the intermediate dimethylester as a white solid. This product was dissolved in 0.4 M NaOH in a MeOH/H<sub>2</sub>O mixture (4:1, 10 mL) and stirred at room temperature under argon for 2–3 h. The reaction course was monitored by TLC (<i>i</i>-PrOH/H<sub>2</sub>O/NH<sub>3</sub>, 6:2:2) until disappearance of the starting material. The reaction mixture was neutralized by addition of triethylammonium acetate (1 M, pH ∼ 7.5). The solvent was removed under reduced pressure, and the resulting residue was purified by silica gel flash chromatography, the mobile phase being a gradient of <i>i</i>-PrOH/H<sub>2</sub>O/NH<sub>3</sub> in a ratio ranging from 10:0:0 to 10:1:1 and 7:1:1 (v/v/v). The product was finally purified by preparative HPLC with a gradient of CH<sub>3</sub>CN in a 50 mM TEAB buffer (pH = 7.4) to yield the target compound as a colorless solid (71 mg, 53% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.46 (s, 1H), 8.18 (s, 1H), 6.46 (t, <i>J</i> = 6.9 Hz, 1H), 4.68 (s, 1H), 4.24 (s, 1H), 4.08–3.83 (m, 2H), 3.75 (dd, <i>J</i> = 13.6, 7.8 Hz, 1H), 2.87–2.68 (m, 1H), 2.64–2.28 (m, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.54 (d, <i>J</i> = 4.7 Hz), 178.60 (s), 155.21(s), 152.47 (s), 148.57 (s), 139.89 (s), 118.37 (s), 86.01 (d, <i>J</i> = 9.2 Hz), 83.44 (s), 71.15 (s), 63.82 (d, <i>J</i> = 5.1 Hz), 54.39 (s), 42.60 (d, <i>J</i> = 6.2 Hz), 38.91 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 7.09 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>14</sub>H<sub>19</sub>N<sub>6</sub>O<sub>9</sub>P, 445.0878; found, 445.0881.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (((((<i>R</i>)-1-(6-Amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-aspartic Acid Triethylammonium Salt (<b>4a</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-aspartic acid dimethyl ester hydrochloride (453 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (49 mg, 36.9% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.16 (s, 1H), 8.06 (s, 1H), 4.21 (ddd, <i>J</i> = 20.0, 14.9, 4.6 Hz, 2H), 3.86 (dd, <i>J</i> = 10.1, 5.6 Hz, 1H), 3.73 (dd, <i>J</i> = 13.4, 7.8 Hz, 1H), 3.48 (dt, <i>J</i> = 21.2, 12.6 Hz, 2H), 2.39 (ddd, <i>J</i> = 22.0, 14.3, 6.1 Hz, 2H), 1.01 (d, <i>J</i> = 6.1 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 181.45 (d, <i>J</i> = 2.9 Hz), 179.54 (s), 155.22 (s), 152.06 (s), 148.95 (s), 143.41 (s), 117.77 (s), 75.54 (d, <i>J</i> = 11.7 Hz), 66.19 (d, <i>J</i> = 145.5 Hz), 54.81 (s), 47.42 (s), 44.11 (d, <i>J</i> = 5.4 Hz), 16.01 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.41 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>13</sub>H<sub>19</sub>N<sub>6</sub>O<sub>7</sub>P, 401.0980; found, 401.0981.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">d</span>-aspartic Acid Triethylammonium Salt (<b>3b</b>)</h4><div class="NLM_p">This compound was prepared from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">d</span>-aspartic acid dimethyl ester hydrochloride (453 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (68 mg, 50.8% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.47 (s, 1H), 8.19 (s, 1H), 6.46 (t, <i>J</i> = 6.8 Hz, 1H), 4.70 (d, <i>J</i> = 2.5 Hz, 1H), 4.31–4.14 (m, 1H), 3.93 (s, 2H), 3.78–3.70 (m, 1H), 2.86–2.70 (m, 1H), 2.63–2.33 (m, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 181.12 (d, <i>J</i> = 3.2 Hz), 179.36 (s), 155.27 (s), 152.47 (s), 148.45 (s), 139.84 (s), 118.30 (s), 85.97 (d, <i>J</i> = 9.2 Hz), 83.33 (s), 71.00 (s), 63.73 (d, <i>J</i> = 4.8 Hz), 55.05 (s), 43.50 (d, <i>J</i> = 7.1 Hz), 38.78 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 6.77 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>14</sub>H<sub>19</sub>N<sub>6</sub>O<sub>9</sub>P, 445.0878; found, 445.0919.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (((((<i>R</i>)-1-(6-Amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">d</span>-aspartic Acid Triethylammonium Salt (<b>4b</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">d</span>-aspartic acid dimethyl ester hydrochloride (453 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (45 mg, 34% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.19 (s, 1H), 8.12 (s, 1H), 4.33–4.15 (m, 2H), 3.95–3.82 (m, 1H), 3.72 (dd, <i>J</i> = 13.8, 7.9 Hz, 1H), 3.60–3.36 (m, 2H), 2.45–2.22 (m, 2H), 1.04 (d, <i>J</i> = 6.3 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 181.27 (d, <i>J</i> = 3.1 Hz), 179.38 (s), 155.27 (s), 152.18 (s), 149.12 (s), 143.46 (s), 117.91 (s), 75.45 (d, <i>J</i> = 11.6 Hz), 66.06 (d, <i>J</i> = 144.0 Hz), 54.73 (s), 47.46 (s), 44.16 (d, <i>J</i> = 5.0 Hz), 15.88 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.34 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>13</sub>H<sub>19</sub>N<sub>6</sub>O<sub>7</sub>P, 401.0980; found, 401.0971.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-histidine Triethylammonium Salt (<b>3c</b>)</h4><div class="NLM_p">This compound was prepared from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-histidine methyl ester dihydrochloride (511 mg, 2.11 mmol) according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (70 mg, 50% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.23 (s, 2H), 7.93 (s, 1H), 6.93 (s, 1H), 6.23 (t, <i>J</i> = 11.9 Hz, 1H), 4.58–4.54 (m, 1H), 4.13 (s, 1H), 3.84 (s, 2H), 3.69–3.62 (m, 1H), 2.81–2.63 (m, 3H), 2.52–2.44 (m, 1H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 178.18 (d, <i>J</i> = 8.0 Hz), 154.68 (s), 152.09 (s), 147.99 (s), 139.22 (s), 132.30 (s), 129.21 (s), 117.76 (s), 116.16 (s), 85.45 (d, <i>J</i> = 9.1 Hz), 83.01 (s), 70.73 (s), 63.42 (d, <i>J</i> = 4.6 Hz), 55.72 (s), 38.24 (s), 29.31 (d, <i>J</i> = 2.8 Hz).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 6.41 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>16</sub>H<sub>21</sub>N<sub>8</sub>O<sub>7</sub>P, 467.1198; found, 467.1195.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (((((<i>R</i>)-1-(6-Amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-histidine Triethylammonium Salt (<b>4c</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy) propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-histidine methyl ester dihydrochloride (554 mg, 2.29 mmol) according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (60 mg, 43% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.18 (s, 1H), 8.13 (s, 1H), 7.53 (s, 1H), 6.73 (s, 1H), 4.32–4.17 (m, 2H), 3.96–3.91 (m, 1H), 3.79–3.72 (m, 1H), 3.53–3.32 (m, 2H), 2.77 (dd, <i>J</i> = 14.0 Hz, 7.9 Hz, 2H), 1.11 (d, <i>J</i> = 6.2 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.10 (d, <i>J</i> = 4.8 Hz), 154.96 (s), 151.83 (s), 148.66 (s), 142.86 (s), 134.93 (s), 119.90 (s), 119.79 (s), 117.62 (s), 75.07 (d, <i>J</i> = 12.4 Hz), 65.46 (d, <i>J</i> = 145.5 Hz), 56.18 (s), 47.39 (s), 31.72 (s), 15.57 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.26 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>15</sub>H<sub>21</sub>N<sub>8</sub>O<sub>5</sub>P, 423.1300; found, 423.1298.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-glutamic Acid Triethylammonium Salt (<b>3d</b>)</h4><div class="NLM_p">This compound was prepared from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-glutamic acid dimethyl ester hydrochloride (447 mg, 2.11 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (46 mg, 33% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.39 (s, 1H), 8.08 (s, 1H), 6.37 (t, <i>J</i> = 6.9 Hz, 1H), 4.59 (t, <i>J</i> = 3.0 Hz, 1H), 4.16 (s, 1H), 3.92–3.79 (m, 2H), 3.42 (dd, <i>J</i> = 15.1, 6.5 Hz, 1H), 2.78–2.66 (m, 1H), 2.53–2.45 (m, 1H), 2.12 (t, <i>J</i> = 8.4 Hz, 2H), 1.87–1.69 (m, 2H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 182.83 (s), 181.53 (d, <i>J</i> = 4.4 Hz), 155.83 (s), 152.86 (s), 148.86 (s), 140.15 (s), 118.81 (s), 86.30 (d, <i>J</i> = 9.3 Hz), 83.75 (s), 71.56 (s), 64.12 (d, <i>J</i> = 4.9 Hz), 56.95 (s), 39.32 (s), 34.05 (s), 32.22 (d, <i>J</i> = 5.8 Hz).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 7.03 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>15</sub>H<sub>21</sub>N<sub>6</sub>O<sub>9</sub>P, 459.1035; found, 459.1042.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (((((<i>R</i>)-1-(6-Amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-glutamic Acid Triethylammonium Salt (<b>4d</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-glutamic acid dimethyl ester hydrochloride (485 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (43 mg, 31% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.23 (s, 1H), 8.19 (s, 1H), 4.35–4.19 (m, 2H), 3.95–3.87 (m, 1H), 3.59–3.39 (m, 3H), 2.17 (t, <i>J</i> = 8.3 Hz, 2H), 1.87–1.65 (m, 2H), 1.07 (d, <i>J</i> = 6.1 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 182.61 (s), 181.24 (d, <i>J</i> = 3.6 Hz), 155.20 (s), 152.08 (s), 148.96 (s), 143.30 (s), 117.76 (s), 75.39 (d, <i>J</i> = 11.7 Hz), 65.90 (d, <i>J</i> = 165.0 Hz), 56.32 (s), 47.44 (s), 33.69 (s), 32.36 (d, <i>J</i> = 4.4 Hz), 15.83 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.42 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>14</sub>H<sub>21</sub>N<sub>6</sub>O<sub>7</sub>P, 415.1136; found, 415.1136.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-methionine Triethylammonium Salt (<b>3e</b>)</h4><div class="NLM_p">This compound was prepared from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-methionine methyl ester hydrochloride (421 mg, 2.11 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (51 mg, 37% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.36 (s, 1H), 8.01 (s, 1H), 6.31 (t, <i>J</i> = 6.8 Hz, 1H), 4.59 (t, <i>J</i> = 3.0 Hz, 1H), 4.13 (s, 1H), 3.83 (d, <i>J</i> = 3.3 Hz, 2H), 3.49–3.42 (m, 1H), 2.68 (d, <i>J</i> = 5.9 Hz, 1H), 2.51–2.44 (m, 1H), 2.26 (dd, <i>J</i> = 15.5, 7.6 Hz, 2H), 1.83 (s, 3H), 1.65–1.58 (m, 2H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.81 (d, <i>J</i> = 5.8 Hz), 155.50 (s), 152.72 (s), 148.73 (s), 140.02 (s), 118.60 (s), 86.26 (d, <i>J</i> = 9.4 Hz), 83.72 (s), 71.49 (s), 64.02 (d, <i>J</i> = 4.9 Hz), 56.47 (s), 39.35 (s), 34.86 (d, <i>J</i> = 4.0 Hz), 29.38 (s), 14.20 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 6.73 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>15</sub>H<sub>23</sub>N<sub>6</sub>O<sub>7</sub>PS, 461.1014; found, 461.1019.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (((((<i>R</i>)-1-(6-Amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-methionine Triethylammonium Salt (<b>4e</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-methionine methyl ester hydrochloride (457 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (42 mg, 30% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.11 (s, 1H), 8.01 (s, 1H), 4.27–4.08 (m, 2H), 3.83 (dd, <i>J</i> = 9.7, 5.5 Hz, 1H), 3.62–3.41 (m, 2H), 3.34 (dd, <i>J</i> = 12.6, 9.5 Hz, 1H), 2.28 (t, <i>J</i> = 7.2 Hz, 2H), 1.89 (s, 3H), 1.75–1.50 (m, 2H), 0.99 (d, <i>J</i> = 6.2 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.87 (d, <i>J</i> = 4.6 Hz), 155.13 (s), 152.02 (s), 148.89 (s), 143.20 (s), 117.74 (s), 75.10 (d, <i>J</i> = 12.3 Hz), 65.77 (d, <i>J</i> = 144.75 Hz), 55.77 (s), 47.61 (s), 34.91 (d, <i>J</i> = 3.3 Hz), 29.04 (s), 15.66 (s), 13.90 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.31 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>14</sub>H<sub>23</sub>N<sub>6</sub>O<sub>5</sub>PS, 417.1115; found, 417.1113.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>)-2-(((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)amino)-5-((diaminomethylene)amino)pentanoic Acid Triethylammonium Salt (<b>3f</b>)</h4><div class="NLM_p">This compound was synthesized from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-arginine methyl ester dihydrochloride (551 mg, 2.11 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (47 mg, 32% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.37 (s, 1H), 8.01 (s, 1H), 6.32 (t, <i>J</i> = 7.0 Hz, 1H), 4.64–4.56 (m, 1H), 4.16 (t, <i>J</i> = 3.0 Hz, 1H), 3.85 (dd, <i>J</i> = 6.5, 3.3 Hz, 2H), 3.42 (dt, <i>J</i> = 8.6, 5.3 Hz, 1H), 2.74–2.64 (m, 3H), 2.53–2.46 (m, 1H), 1.43–1.25 (m, 4H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.95 (d, <i>J</i> = 6.0 Hz), 156.28 (s), 155.12 (s), 152.43 (s), 148.35 (s), 139.65 (s), 118.11 (s), 85.96 (d, <i>J</i> = 9.3 Hz), 83.30 (s), 71.26 (s), 63.75 (d, <i>J</i> = 4.6 Hz), 56.03 (s), 40.66 (s), 38.95 (s), 31.57 (d, <i>J</i> = 4.2 Hz), 23.66 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 6.83 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>16</sub>H<sub>26</sub>N<sub>9</sub>O<sub>7</sub>P, 486.1620; found, 486.1610.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (((((<i>R</i>)-1-(6-amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-arginine Triethylammonium Salt (<b>4f</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-arginine methyl ester dihydrochloride (598 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (35 mg, 24% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.05 (s, 1H), 7.92 (s, 1H), 4.16–4.01 (m, 2H), 3.90–3.60 (m, 1H), 3.53–3.22 (m, 3H), 2.92 (s, 2H), 1.40 (d, <i>J</i> = 5.5 Hz, 4H), 0.94 (t, <i>J</i> = 6.0 Hz, 3H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 181.13 (d, <i>J</i> = 4.4 Hz), 156.38 (s), 154.99 (s), 151.94 (s), 148.74 (s), 143.03 (s), 117.55 (s), 75.16 (d, <i>J</i> = 12.1 Hz), 65.81 (d, <i>J</i> = 143.3 Hz), 55.77 (s), 47.42 (s), 40.73 (s), 32.24 (d, <i>J</i> = 2.7 Hz), 23.77 (s), 15.68 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 17.36 (s).</div><div class="NLM_p last">HRMS: [M – H]<sup>−</sup> calcd for C<sub>15</sub>H<sub>26</sub>N<sub>9</sub>O<sub>5</sub>P, 442.1722; found, 442.1715.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> ((((2<i>R</i>,3<i>S</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3-hydroxytetrahydrofuran-2-yl)methoxy)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine Triethylammonium Salt (<b>3g</b>)</h4><div class="NLM_p">This compound was prepared from 2′-deoxyadenosine 5′-monophosphate (100 mg, 0.30 mmol) and <span class="smallcaps smallerCapital">l</span>-phenylalanine methyl ester hydrochloride (455 mg, 2.11 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (47 mg, 33% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.32 (s, 1H), 8.02 (s, 1H), 7.17–6.95 (m, 5H), 6.30 (t, <i>J</i> = 6.8 Hz, 1H), 4.62–4.51 (m, 1H), 4.18–4.08 (m, 1H), 3.81–3.67 (m, 3H), 2.74–2.64 (m, 3H)), 2.54–2.46 (m, 1H).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 180.52 (d, <i>J</i> = 5.7 Hz), 155.16 (s), 152.38 (s), 148.34 (s), 139.60 (s), 138.00 (s), 129.21 (s), 127.88 (s), 125.98 (s), 118.30 (s), 85.88 (d, <i>J</i> = 9.2 Hz), 83.38 (s), 71.10 (s), 63.59 (d, <i>J</i> = 5.1 Hz), 58.18 (s), 40.70 (d, <i>J</i> = 4.8 Hz), 38.94 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 6.61 (s).</div><div class="NLM_p last">HRMS: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>23</sub>N<sub>6</sub>O<sub>7</sub>P, 479.1438; found, 479.1439.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (((((<i>R</i>)-1-(6-amino-9<i>H</i>-purin-9-yl)propan-2-yl)oxy)methyl)(hydroxy)phosphoryl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine Triethylammonium Salt (<b>4g</b>)</h4><div class="NLM_p">This compound was prepared from 9-[(<i>R</i>)-2-(phosphonomethoxy)propyl]adenine monohydrate (100 mg, 0.33 mmol) and <span class="smallcaps smallerCapital">l</span>-phenylalanine methyl ester hydrochloride (494 mg, 2.29 mmol), according to the procedure used for the synthesis of compound <b>3a</b>, yielding the title compound as a colorless solid (57 mg, 40% over two steps).</div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 8.03 (s, 1H), 7.95 (s, 1H), 7.10–6.96 (m, 5H), 4.10–4.05 (m, 2H), 3.78–3.61 (m, 2H), 3.27 (dd, <i>J</i> = 12.6, 9.2 Hz, 1H), 3.04 (dd, <i>J</i> = 12.6, 9.2 Hz, 1H), 2.61 (d, <i>J</i> = 6.1 Hz, 2H), 1.00 (d, <i>J</i> = 6.2 Hz).</div><div class="NLM_p"><sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 181.03 (d, <i>J</i> = 4.4 Hz), 155.33 (s), 152.24 (s), 149.04 (s), 143.28 (s), 138.45 (s), 129.65 (s), 128.19 (s), 126.29 (s), 118.03 (s), 75.46 (d, <i>J</i> = 12.3 Hz), 66.44 (d, <i>J</i> = 147.0 Hz), 58.14 (s), 47.96 (s), 41.59 (d, <i>J</i> = 3.1 Hz), 16.23 (s).</div><div class="NLM_p"><sup>31</sup>P NMR (121 MHz, D<sub>2</sub>O): δ 16.75 (s).</div><div class="NLM_p last">HRMS: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>23</sub>N<sub>6</sub>O<sub>5</sub>P, 435.1540; found, 435.1538.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> RT Expression, RT/DNA Cross-Linking, Purification</h3><div class="NLM_p">RT construct RT127A used for this study was derived from a previously reported construct RT13A.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The mutation Asp498Asn in the RT127A construct was introduced using the methods described therein; Asp498 is a part of the RNase H active site. The Asp498Asn mutant RT blocks RNase H activity, but this mutant has polymerase activity comparable to wild-type RT.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The Asp498Asn mutant RT was expressed and purified as previously reported.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Briefly, the RT was expressed in BL21-CodonPlus-RIL cells, induced with 1 mM IPTG at an OD<sub>600</sub> of 0.9, followed by expression at 37 °C for 3 h. The cells were sonicated at a power output of ∼45 W with a Branson Sonifier SFX250 sonicator. The samples were purified using a nickel-nitrilotriacetic acid (Ni-NTA) column according to the manufacturer’s recommendations (Qiagen). The final purification step was carried out using a Mono Q column, and the purified RT samples were buffer exchanged into 10 mM Tris, pH 8.0, and 75 mM NaCl.</div><div class="NLM_p last">The 27-mer DNA template (5′-ATGGAAGGCGCCCGAACAGGGACTG TG-3′), containing the PBS sequence from the HIV genome, was synthesized by Integrated DNA Technologies. The 20-mer primer (5′-ACAGTCCCTGTTCGG<b>G</b>CGCC-3′) bearing a cross-linkable thioalkyl tether (on <b>G</b> in the primer strand)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was custom synthesized by Midland Certified Reagent Company, using phosphoramidite that was custom synthesized by Chemgenes, and the primer was annealed to the template. The 27:20-mer dsDNA was cross-linked to RT127A at the mutated Q258C site of p66, and the cross-linked primer was extended with ddG at the 3′-end through RT polymerization.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The cross-linked RT127A–DNA (AZT-terminated primer) complex was purified using Ni-NTA and heparin columns in tandem as previously described.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The His tag was removed by adding human rhinovirus 14 3C protease at a 1:10 ratio to the RT–DNA complex at 4 °C for 48 h. Ni-NTA beads were added to remove the cleaved His tag and His-tagged protease. An Amicon Ultra-4 Ultracel unit (30-KDa cutoff) was used to exchange the buffer into 75 mM NaCl, 10 mM Tris–HCl, pH 8.0. The protein was concentrated to 12 mg/mL using a mass extinction coefficient of OD<sub>280</sub> = 5.4 for 1 mg/mL.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Crystallization and Structure Determination</h3><div class="NLM_p last">Sitting drops of 0.1 μL of protein plus 0.1 μL of well solution were set up at room temperature on Swiss CI UVXPO three drop (midi) or two drop plates. Each well contained 40 μL (midi) or 85 μL of 16–19.5% v/v PEG Smear Broad (Molecular Dimensions, MD2-250-261), 0.2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 0.1 M Tris–HCl, pH 8.3 or 8.5. The protein solution was 10 mg/mL in 75 mM NaCl, 10 mM Tris, pH 8.0. Trays were incubated at 4 °C. Crystals began appearing in 2 or 3 days and were done growing by 2 weeks. Crystals chosen for data collection were approximately 150 mm × 100 mm × 60 mm in size. To soak compounds into crystals, compounds were first dissolved in DMSO to 100 mM. Two cryosoaking solutions were prepared: 10 or 20% v/v glycerol, 20% v/v PEG Smear Broad, 0.2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.1 M Tris–HCl pH 8.0, 5 mM MgCl<sub>2</sub> +/– 2 mM MnCl<sub>2</sub>, 2 mM compound. For freezing crystals, a nine-well depression plate was used and sealed with a plastic film. Crystals were soaked in 50 μl of the 10% glycerol solution for 1–3 h, then soaked in the same volume of 20% glycerol solution for 1–2 min, and then flash frozen in liquid nitrogen. X-ray diffraction data were collected at ESRF beamlines ID29 and ID30B and at SOLEIL beamline PX2A. Images were integrated using iMosflm<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and scaled using AIMLESS<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> in the CCP4 v7.0 package.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The complexes of RT/DNA with each compound were solved starting with rigid body refinement in PHENIX,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> using a starting model of RT/DNA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V4I">3V4I</a>). The crystals grew in space group P2<sub>1</sub> with two RT heterodimers in the asymmetric unit. The atomic models were rebuilt by manual model building in COOT,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> guided by structure validation with MolProbity.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The models were refined in PHENIX.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Data and refinement statistics are in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf" class="ext-link">Table S2</a>. Figures were prepared with PyMOL.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Incorporation Assay</h3><div class="NLM_p last">The primer labeled with Cy5, 5′- Cy5 -CAGGAAACAGCTATGAC, was annealed with the template sequence 3′- GTCCTTTGTCGATACTGTTTTTTT-5′. This DNA template/primer was used for tests of primer extension by HIV-RT with the various compounds in this study. Reactions were carried out at 37 °C with 125 nM primer/template, 500 μM compound or 50 μM dATP, and 0.5 μg/μL reaction of RT. The buffer was 50 mM Tris–HCl, 3 mM MgCl<sub>2</sub> with or without 2 mM MnCl<sub>2</sub>, and 10 mM DTT, pH 8.3. The time periods were 10 min, 1 h, and 24 h. Aliquots were taken at 10 and 60 min and quenched by adding a double volume of quenching buffer (90% formamide, 50 mM EDTA, and 0.05% orange G) and heated at 95 °C for 5 min. The samples were separated on a 1 mm 20% denaturing polyacrylamide gel, and gel bands were visualized using the Typhoon FLA 9500 imaging system (GE Healthcare Life Sciences). The images were processed using ImageQuant TL v8.1.0.0 (GE Healthcare). The expression and purification of the RT127A construct has been described.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Inhibition of HIV-1 Reverse Transcriptase Activity</h3><div class="NLM_p last">Enzyme assays using recombinant HIV-1 RT were carried out in 96-well black plates by measuring the conversion of double-strand DNA from the Td100/Pd18 template-primer using a PicoGreen-based spectrophotometric assay in the presence of increasing concentrations of compounds as described previously.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Briefly, reactions containing purified recombinant HIV-1 RT, Td100/Pd18 DNA substrate in a buffer containing 50 mm Tris, pH 7.8, 6 mM MgCl<sub>2</sub> or 2 mM MnCl<sub>2</sub>, and 50 mM NaCl were initiated by the addition of 10 μM dNTPs. DNA synthesis was carried out for 30 min at 37 °C followed by quenching with the addition of 100 mM EDTA. The Quant-iT PicoGreen reagent (Thermo Fisher Scientific Inc.) was added to quantify the amount of formed double-stranded DNA in TE buffer. The reaction mixtures were then excited at 480 nm, and fluorescence was monitored at 520 nm using a 96-well CLARIOstar microplate reader (BMG LABTECH). Dose–response curves of triplicate samples were plotted using GraphPad Prism version 5.0 (GraphPad Software, San Diego, CA) to determine the 50% inhibitory concentration (IC<sub>50</sub>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01747" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01747?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01747</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table of summary of incorporation, resolution, and inhibition data for <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-amino acid phosphoramidates, and phosphonoamidates in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup>; table of X-ray data and refinement statistics; ligplots of <b>4d</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; ligplots of <b>4b</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; ligplots of <b>4e</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; and <sup>1</sup>H, <sup>13</sup>C, and <sup>31</sup>P NMR spectra of compounds <b>3a</b>–<b>g</b> and <b>4a–g</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf">jm0c01747_si_001.pdf (1.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_002.csv">jm0c01747_si_002.csv (1.02 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates for RT/DNA plus compounds were deposited in the Protein Data Bank as follows: compound <b>4b</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX">7AHX</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID">7AID</a>), compound <b>4d</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF">7AIF</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG">7AIG</a>), and compound <b>4e</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII">7AII</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ">7AIJ</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c01747" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25511" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25511" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven De Jonghe</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology, Immunology and Transplantation,
Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3872-6558" title="Orcid link">http://orcid.org/0000-0002-3872-6558</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3f4c4b5a495a51115b5a55505158575a7f544a535a4a495a51115d5a"><span class="__cf_email__" data-cfemail="265552435043480842434c4948414e43664d534a4353504348084443">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kalyan Das</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology, Immunology and Transplantation,
Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8897-324X" title="Orcid link">http://orcid.org/0000-0002-8897-324X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#274c464b5e464909434654674c524b4252514249094542"><span class="__cf_email__" data-cfemail="204b414c59414e0e444153604b554c455556454e0e4245">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weijie Gu</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology, Immunology and Transplantation,
Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Pharmaceutical and Pharmacological Sciences,
Laboratory of Medicinal Chemistry, Rega
Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sergio Martinez</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology, Immunology and Transplantation,
Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hoai Nguyen</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Pharmaceutical and Pharmacological Sciences,
Laboratory of Medicinal Chemistry, Rega
Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongtao Xu</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology, Immunology and Transplantation,
Laboratory of Virology and Chemotherapy, Rega Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Piet Herdewijn</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Pharmaceutical and Pharmacological Sciences,
Laboratory of Medicinal Chemistry, Rega
Institute for Medical Research, Herestraat 49, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3589-8503" title="Orcid link">http://orcid.org/0000-0003-3589-8503</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Compounds were synthesized and characterized by W.G. Preparation of RT crystals; collection of X-ray data, refinement, and model building were done by S.M. and K.D. The gel incorporation assay was done by H.N. The PicoGreen inhibition assay was done by H.X. and W.G. The manuscript was written by K.D., S.M., S.D.J., and W.G. The project was designed and supervised by K.D., S.D.J., and P.H. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to staff for support and access to data collection at the following beamlines: SOLEIL PX1 and PX2A, Diamond I04, and ESRF ID29, ID30D, ID30A-3. W.G. acknowledges the China Scholarship Council (CSC) for funding (grant no. 201707060007). We thank Eddy Arnold (Rutgers University) for the plasmid and cross-linkable DNA. The study was supported by KU Leuven start-up and Rega Virology and Chemotherapy internal grants to K.D.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">3TC</td><td class="NLM_def"><p class="first last">(−)-<span class="smallcaps smallerCapital">l</span>-2′,3′-dideoxy-3′-thiacytidine (lamivudine)</p></td></tr><tr><td class="NLM_term">α-CNP</td><td class="NLM_def"><p class="first last">alpha-carboxy nucleoside phosphonate</p></td></tr><tr><td class="NLM_term">T-α-CNP</td><td class="NLM_def"><p class="first last">thymine alpha-carboxy nucleoside phosphonate</p></td></tr><tr><td class="NLM_term">G-α-CNP</td><td class="NLM_def"><p class="first last">guanine alpha-carboxy nucleoside phosphonate</p></td></tr><tr><td class="NLM_term">ABC</td><td class="NLM_def"><p class="first last">(1S,cis)-4-[2-amino-6-(cyclopropylamino)-9<i>H</i>-purin-9-yl]-2-cyclopentene-1-methanol (abacavir)</p></td></tr><tr><td class="NLM_term">AZT</td><td class="NLM_def"><p class="first last">3′-azido-3′-deoxythymidine</p></td></tr><tr><td class="NLM_term">ART</td><td class="NLM_def"><p class="first last">anti-retroviral therapy</p></td></tr><tr><td class="NLM_term">dATP</td><td class="NLM_def"><p class="first last">2′ deoxyadenosine triphosphate</p></td></tr><tr><td class="NLM_term">dAMP</td><td class="NLM_def"><p class="first last">2′ deoxyadenosine monophosphate</p></td></tr><tr><td class="NLM_term">dNTP</td><td class="NLM_def"><p class="first last">2′deoxynucleoside triphosphate</p></td></tr><tr><td class="NLM_term">dTTP</td><td class="NLM_def"><p class="first last">2′ deoxythymidine triphosphate</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">d</span>-Asp</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">d</span>-aspartate</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">dsDNA</td><td class="NLM_def"><p class="first last">double-stranded DNA</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">FTC</td><td class="NLM_def"><p class="first last">2′,3′-dideoxy-5-fluoro-3′-thiacytidine (emtricitabine)</p></td></tr><tr><td class="NLM_term">Gly</td><td class="NLM_def"><p class="first last">glycine</p></td></tr><tr><td class="NLM_term">HIV-1</td><td class="NLM_def"><p class="first last">human immunodeficiency virus 1</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-azabenzotriazole</p></td></tr><tr><td class="NLM_term"><i>i</i>-PrOH</td><td class="NLM_def"><p class="first last">isopropyl alcohol</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Ala</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-alanine</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Glu</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-glutamate</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-His</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-histidine</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Met</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Pro</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-proline</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Ser</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-serine</p></td></tr><tr><td class="NLM_term"><span class="smallcaps smallerCapital">l</span>-Tyr</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">l</span>-tyrosine</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">NcRTIs</td><td class="NLM_def"><p class="first last">nucleoside-competing RT inhibitors</p></td></tr><tr><td class="NLM_term">PFV</td><td class="NLM_def"><p class="first last">prototype foamy virus</p></td></tr><tr><td class="NLM_term">RNAP</td><td class="NLM_def"><p class="first last">RNA polymerase</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reverse phase high pressure liquid chromatography</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">reverse transcriptase</p></td></tr><tr><td class="NLM_term">SARS-CoV-2</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus 2</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl alcohol</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">TEAA</td><td class="NLM_def"><p class="first last">triethylammonium acetate</p></td></tr><tr><td class="NLM_term">TFV</td><td class="NLM_def"><p class="first last">(((1<i>R</i>)-2-(6-amino-9<i>H</i>-purin-9-yl)-1-methylethoxy)methyl)phosphonic acid (tenofovir)</p></td></tr><tr><td class="NLM_term">TFV-DP</td><td class="NLM_def"><p class="first last">tenofovir diphosphate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">Approved antiviral drugs over the past 50 years</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1128/cmr.00102-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1128%2FCMR.00102-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=27281742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252Fos12htA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=695-747&author=E.+De%0AClercqauthor=G.+Li&title=Approved+antiviral+drugs+over+the+past+50+years&doi=10.1128%2Fcmr.00102-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Approved Antiviral Drugs over the Past 50 Years</span></div><div class="casAuthors">De Clercq Erik; Li Guangdi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical microbiology reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">695-747</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the first antiviral drug, idoxuridine, was approved in 1963, 90 antiviral drugs categorized into 13 functional groups have been formally approved for the treatment of the following 9 human infectious diseases: (i) HIV infections (protease inhibitors, integrase inhibitors, entry inhibitors, nucleoside reverse transcriptase inhibitors, nonnucleoside reverse transcriptase inhibitors, and acyclic nucleoside phosphonate analogues), (ii) hepatitis B virus (HBV) infections (lamivudine, interferons, nucleoside analogues, and acyclic nucleoside phosphonate analogues), (iii) hepatitis C virus (HCV) infections (ribavirin, interferons, NS3/4A protease inhibitors, NS5A inhibitors, and NS5B polymerase inhibitors), (iv) herpesvirus infections (5-substituted 2'-deoxyuridine analogues, entry inhibitors, nucleoside analogues, pyrophosphate analogues, and acyclic guanosine analogues), (v) influenza virus infections (ribavirin, matrix 2 protein inhibitors, RNA polymerase inhibitors, and neuraminidase inhibitors), (vi) human cytomegalovirus infections (acyclic guanosine analogues, acyclic nucleoside phosphonate analogues, pyrophosphate analogues, and oligonucleotides), (vii) varicella-zoster virus infections (acyclic guanosine analogues, nucleoside analogues, 5-substituted 2'-deoxyuridine analogues, and antibodies), (viii) respiratory syncytial virus infections (ribavirin and antibodies), and (ix) external anogenital warts caused by human papillomavirus infections (imiquimod, sinecatechins, and podofilox).  Here, we present for the first time a comprehensive overview of antiviral drugs approved over the past 50 years, shedding light on the development of effective antiviral treatments against current and emerging infectious diseases worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSbPpNpCY_qYTEC-75aqa5AfW6udTcc2eYjreZZ7-37Q7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252Fos12htA%253D%253D&md5=1d25d961c1c6a72d6bc4634041bbb6fe</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1128%2FCMR.00102-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00102-15%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DG.%26atitle%3DApproved%2520antiviral%2520drugs%2520over%2520the%2520past%252050%2520years%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2016%26volume%3D29%26spage%3D695%26epage%3D747%26doi%3D10.1128%2Fcmr.00102-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gollnest, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Lipophilic prodrugs of nucleoside triphosphates as biochemical probes and potential antivirals</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8716</span>, <span class="refDoi"> DOI: 10.1038/ncomms9716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fncomms9716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=26503889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8716&author=T.+Gollnestauthor=T.+D.+de+Oliveiraauthor=D.+Scholsauthor=J.+Balzariniauthor=C.+Meier&title=Lipophilic+prodrugs+of+nucleoside+triphosphates+as+biochemical+probes+and+potential+antivirals&doi=10.1038%2Fncomms9716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic prodrugs of nucleoside triphosphates as biochemical probes and potential antivirals</span></div><div class="casAuthors">Gollnest, Tristan; Dinis de Oliveira, Thiago; Schols, Dominique; Balzarini, Jan; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8716</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The antiviral activity of nucleoside reverse transcriptase inhibitors is often limited by ineffective phosphorylation.  We report on a nucleoside triphosphate (NTP) prodrug approach in which the γ-phosphate of NTPs is bioreversibly modified.  A series of TriPPPro-compds. bearing two lipophilic masking units at the γ-phosphate and d4T as a nucleoside analog are synthesized.  Successful delivery of d4TTP is demonstrated in human CD4+ T-lymphocyte cell exts. by an enzyme-triggered mechanism with high selectivity.  In antiviral assays, the compds. are potent inhibitors of HIV-1 and HIV-2 in CD4+ T-cell (CEM) cultures.  Highly lipophilic acyl residues lead to higher membrane permeability that results in intracellular delivery of phosphorylated metabolites in thymidine kinase-deficient CEM/TK- cells with higher antiviral activity than the parent nucleoside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr69l3mBVsk_7Vg90H21EOLACvtfcHk0lhxWpb3Kb1uzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnsL7N&md5=d245295f8ad5ca9ab4c548f62a2a938f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fncomms9716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9716%26sid%3Dliteratum%253Aachs%26aulast%3DGollnest%26aufirst%3DT.%26aulast%3Dde%2BOliveira%26aufirst%3DT.%2BD.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DLipophilic%2520prodrugs%2520of%2520nucleoside%2520triphosphates%2520as%2520biochemical%2520probes%2520and%2520potential%2520antivirals%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8716%26doi%3D10.1038%2Fncomms9716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gollnest, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinis de Oliveira, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauber, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Membrane-permeable triphosphate prodrugs of nucleoside analogues</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5255</span>– <span class="NLM_lpage">5258</span>, <span class="refDoi"> DOI: 10.1002/anie.201511808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fanie.201511808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=27008042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC28fivFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5255-5258&author=T.+Gollnestauthor=T.+Dinis+de+Oliveiraauthor=A.+Rathauthor=I.+Hauberauthor=D.+Scholsauthor=J.+Balzariniauthor=C.+Meier&title=Membrane-permeable+triphosphate+prodrugs+of+nucleoside+analogues&doi=10.1002%2Fanie.201511808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane-permeable Triphosphate Prodrugs of Nucleoside Analogues</span></div><div class="casAuthors">Gollnest Tristan; Dinis de Oliveira Thiago; Rath Anna; Meier Chris; Hauber Ilona; Schols Dominique; Balzarini Jan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5255-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The metabolic conversion of nucleoside analogues into their triphosphates often proceeds insufficiently.  Rate-limitations can be at the mono-, but also at the di- and triphosphorylation steps.  We developed a nucleoside triphosphate (NTP) delivery system (TriPPPro-approach).  In this approach, NTPs are masked by two bioreversible units at the γ-phosphate.  Using a procedure involving H-phosphonate chemistry, a series of derivatives bearing approved, as well as potentially antivirally active, nucleoside analogues was synthesized.  The enzyme-triggered delivery of NTPs was demonstrated by pig liver esterase, in human T-lymphocyte cell extracts and by a polymerase chain reaction using a prodrug of thymidine triphosphate.  The TriPPPro-compounds of some HIV-inactive nucleoside analogues showed marked anti-HIV activity.  For cellular uptake studies, a fluorescent TriPPPro-compound was prepared that delivered the triphosphorylated metabolite to intact CEM cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWWCDsvsG6Ru2qxNLjzXe8fW6udTcc2eYJsJfsMnIiYrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fivFGmtQ%253D%253D&md5=4bea432418c452ed82633ee0ab6bf274</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fanie.201511808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201511808%26sid%3Dliteratum%253Aachs%26aulast%3DGollnest%26aufirst%3DT.%26aulast%3DDinis%2Bde%2BOliveira%26aufirst%3DT.%26aulast%3DRath%26aufirst%3DA.%26aulast%3DHauber%26aufirst%3DI.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DMembrane-permeable%2520triphosphate%2520prodrugs%2520of%2520nucleoside%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D5255%26epage%3D5258%26doi%3D10.1002%2Fanie.201511808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Anti-HIV-active nucleoside triphosphate prodrugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6003</span>– <span class="NLM_lpage">6027</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00271</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00271" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1Krurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6003-6027&author=X.+Jiaauthor=D.+Scholsauthor=C.+Meier&title=Anti-HIV-active+nucleoside+triphosphate+prodrugs&doi=10.1021%2Facs.jmedchem.0c00271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV-Active Nucleoside Triphosphate Prodrugs</span></div><div class="casAuthors">Jia, Xiao; Schols, Dominique; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6003-6027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We disclose a study on nucleoside triphosphate (NTP) analogs in which the γ-phosphate is covalently modified by two different biodegradable masking units and d4T as nucleoside analog that enable the delivery of d4TTP with high selectivity in phosphate buffer (pH 7.3) and by enzyme-triggered reactions in human CD4+ T-lymphocyte CEM cell exts.  This allows the bypass of all steps normally needed in the intracellular phosphorylation.  These TriPPPro-nucleotides comprising an acyloxybenzyl (AB; ester) or an alkoxycarbonyloxybenzyl (ACB; carbonate) in combination with an ACB moiety are described as NTP delivery systems.  The introduction of these two different groups led to the selective formation of γ-(ACB)-d4TTPs by chem. hydrolysis and in particular by cell ext. enzymes. γ-(AB)-d4TTPs are faster cleaved than γ-(ACB)-d4TTPs.  In antiviral assays, the compds. are highly active against HIV-1 and HIV-2 in wild-type CEM/O cells and more importantly in thymidine kinase-deficient CD4+ T-cells (CEM/TK-).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopqqrhtKobqbVg90H21EOLACvtfcHk0lhxWpb3Kb1uzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1Krurg%253D&md5=ec2e2cca45554cf6776d4db2f76a566b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00271%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DX.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DAnti-HIV-active%2520nucleoside%2520triphosphate%2520prodrugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6003%26epage%3D6027%26doi%3D10.1021%2Facs.jmedchem.0c00271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Nucleoside diphosphate and triphosphate prodrugs - an unsolvable task?</span>. <i>Antivir. Chem. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1177/2040206617738656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1177%2F2040206617738656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29096525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ntFShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=69-82&author=C.+Meier&title=Nucleoside+diphosphate+and+triphosphate+prodrugs+-+an+unsolvable+task%3F&doi=10.1177%2F2040206617738656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside diphosphate and triphosphate prodrugs - An unsolvable task?</span></div><div class="casAuthors">Meier Chris</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral chemistry & chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">69-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this review, our recent advances in the development of nucleoside di- and nucleoside triphosphate prodrugs is summarized.  Previously, we had developed a successful membrane-permeable pronucleotide system for the intracellular delivery of nucleoside monophosphates as well, the so-called cycloSal-approach.  In contrast to that work in which the delivery is initiated by a chemically driven hydrolysis reaction, for the di- and triphosphate delivery, an enzymatic trigger mechanism involving (carboxy)esterases had to be used.  The other features of the new pronucleotide approaches are: (i) lipophilic modification was restricted to the terminal phosphate group leaving charges at the internal phosphate moieties and (ii) appropriate lipophilicity is introduced by long aliphatic residues within the bipartite prodrug moiety.  The conceptional design of the di- and triphosphate prodrug systems will be described and the chemical synthesis, the hydrolysis properties, a structure-activity relationship and antiviral activity data will be discussed as well.  The advantage of these new approaches is that all phosphorylation steps from the nucleoside analogue into the bioactive nucleoside triphosphate form can be bypassed in the case of the triphosphate prodrugs.  Moreover, enzymatic processes like the deamination of nucleosides or nucleoside monophosphates which lead to catabolic clearance of the potential antivirally active compound can be avoided by the delivery of the higher phosphorylated nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW7N2U7Gnno0NHe_gFldTbfW6udTcc2eYV-25TRGxETLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ntFShuw%253D%253D&md5=f061bae9d87e96c0cd99e3a872a5c00b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F2040206617738656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040206617738656%26sid%3Dliteratum%253Aachs%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DNucleoside%2520diphosphate%2520and%2520triphosphate%2520prodrugs%2520-%2520an%2520unsolvable%2520task%253F%26jtitle%3DAntivir.%2520Chem.%2520Chemother.%26date%3D2017%26volume%3D25%26spage%3D69%26epage%3D82%26doi%3D10.1177%2F2040206617738656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weising, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterrenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Antiviral Evaluation of TriPPPro-AbacavirTP, TriPPPro-CarbovirTP, and Their 1′,2′-cis-Disubstituted Analogues</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1771</span>– <span class="NLM_lpage">1778</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcmdc.201800361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29943432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOrt7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1771-1778&author=S.+Weisingauthor=V.+Sterrenbergauthor=D.+Scholsauthor=C.+Meier&title=Synthesis+and+Antiviral+Evaluation+of+TriPPPro-AbacavirTP%2C+TriPPPro-CarbovirTP%2C+and+Their+1%E2%80%B2%2C2%E2%80%B2-cis-Disubstituted+Analogues&doi=10.1002%2Fcmdc.201800361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Antiviral Evaluation of TriPPPro-AbacavirTP, TriPPPro-CarbovirTP, and Their 1',2'-cis-Disubstituted Analogs</span></div><div class="casAuthors">Weising, Simon; Sterrenberg, Vincente; Schols, Dominique; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1771-1778</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we describe the synthesis of lipophilic triphosphate prodrugs of abacavir, carbovir, and their 1',2'-cis-substituted carbocyclic analogs.  The 1',2'-cis-carbocyclic nucleosides were prepd. by starting from enantiomerically pure (1R,2S)-2-((benzyloxy)methyl)cyclopent-3-en-1-ol by a microwave-assisted Mitsunobu-type reaction with 2-amino-6-chloropurine.  All four nucleoside analogs were prepd. from their 2-amino-6-chloropurine precursors.  The nucleosides were converted into their corresponding nucleoside triphosphate prodrugs (TriPPPro approach) by application of the H-phosphonate route.  The TriPPPro compds. were hydrolyzed in different media, in which the formation of nucleoside triphosphates was proven.  While the TriPPPro compds. of abacavir and carbovir showed increased antiviral activity over their parent nucleoside, the TriPPPro compds. of the 1',2'-cis-substituted analogs as well as their parent nucleosides proved to be inactive against HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPFvFXL33ISrVg90H21EOLACvtfcHk0lg1_0DENFxS3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOrt7%252FO&md5=25cb2870ae43ac07d8b21944039bcadd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800361%26sid%3Dliteratum%253Aachs%26aulast%3DWeising%26aufirst%3DS.%26aulast%3DSterrenberg%26aufirst%3DV.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Antiviral%2520Evaluation%2520of%2520TriPPPro-AbacavirTP%252C%2520TriPPPro-CarbovirTP%252C%2520and%2520Their%25201%25E2%2580%25B2%252C2%25E2%2580%25B2-cis-Disubstituted%2520Analogues%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D1771%26epage%3D1778%26doi%3D10.1002%2Fcmdc.201800361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kandil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuigan, C.</span></span> <span> </span><span class="NLM_article-title">Polyfluoroaromatic stavudine (d4T) ProTides exhibit enhanced anti-HIV activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126721</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.bmcl.2019.126721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=31679972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2gsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126721&author=S.+Kandilauthor=C.+Pannecouqueauthor=F.+M.+Chapmanauthor=A.+D.+Westwellauthor=C.+McGuigan&title=Polyfluoroaromatic+stavudine+%28d4T%29+ProTides+exhibit+enhanced+anti-HIV+activity&doi=10.1016%2Fj.bmcl.2019.126721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Polyfluoroaromatic stavudine (d4T) ProTides exhibit enhanced anti-HIV activity</span></div><div class="casAuthors">Kandil, Sahar; Pannecouque, Christophe; Chapman, Fiona M.; Westwell, Andrew D.; McGuigan, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">126721</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human Immunodeficiency Virus (HIV) damages the immune system and leads to the life-threatening acquired immunodeficiency syndrome (AIDS).  Despite the advances in the field of antiretroviral treatment, HIV remains a major public health challenge.  Nucleosides represent a prominent chemotherapeutic class for treating viruses, however their cellular uptake, kinase-mediated activation and catabolism are limiting factors.  Herein, we report the synthesis and in vitro evaluation of stavudine (d4T) ProTides contg. polyfluorinated aryl groups against two strains; HIV-1 (IIIB) and HIV-2 (ROD).  ProTide I contg. a meta-substituted pentafluorosulfanyl (3-SF5) aryl group showed superior antiviral activity over the parent d4T and the nonfluorinated analog.  ProTide Ihas low nanomolar antiviral activity; (IC50 = 30 nM, HIV-1) and (IC50 = 36 nM, HIV-2) which is over tenfold more potent than d4T.  Interestingly, ProTide I showed a significantly high selectivity indexes with SI = 1753 (HIV-1) and 1461 (HIV-2) which is more than twice that of the d4T.  All ProTides were screened in wild type as well as thymidine kinase deficient (TK-) cells.  Enzymic activation of ProTide 5d using carboxypeptidase Y enzyme and monitored using both 31P and 19F NMR is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYpry6VggorrVg90H21EOLACvtfcHk0lg1_0DENFxS3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2gsrfJ&md5=52ee5ca71aec4cb8323eb5c39a0cc3d7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126721%26sid%3Dliteratum%253Aachs%26aulast%3DKandil%26aufirst%3DS.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DChapman%26aufirst%3DF.%2BM.%26aulast%3DWestwell%26aufirst%3DA.%2BD.%26aulast%3DMcGuigan%26aufirst%3DC.%26atitle%3DPolyfluoroaromatic%2520stavudine%2520%2528d4T%2529%2520ProTides%2520exhibit%2520enhanced%2520anti-HIV%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126721%26doi%3D10.1016%2Fj.bmcl.2019.126721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seley-Radtke, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, M. K.</span></span> <span> </span><span class="NLM_article-title">The evolution of nucleoside analogue antivirals: a review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.antiviral.2018.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29649496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslOntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2018&pages=66-86&author=K.+L.+Seley-Radtkeauthor=M.+K.+Yates&title=The+evolution+of+nucleoside+analogue+antivirals%3A+a+review+for+chemists+and+non-chemists.+Part+1%3A+Early+structural+modifications+to+the+nucleoside+scaffold&doi=10.1016%2Fj.antiviral.2018.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of nucleoside analogue antivirals: A review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</span></div><div class="casAuthors">Seley-Radtke, Katherine L.; Yates, Mary K.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-86</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This is the first of two invited articles reviewing the development of nucleoside-analog antiviral drugs, written for a target audience of virologists and other non-chemists, as well as chemists who may not be familiar with the field.  Rather than providing a simple chronol. account, we have examd. and attempted to explain the thought processes, advances in synthetic chem. and lessons learned from antiviral testing that led to a few mols. being moved forward to eventual approval for human therapies, while others were discarded.  The present paper focuses on early, relatively simplistic changes made to the nucleoside scaffold, beginning with modifications of the nucleoside sugars of Ara-C and other arabinose-derived nucleoside analogs in the 1960's.  A future paper will review more recent developments, focusing esp. on more complex modifications, particularly those involving multiple changes to the nucleoside scaffold.  We hope that these articles will help virologists and others outside the field of medicinal chem. to understand why certain drugs were successfully developed, while the majority of candidate compds. encountered barriers due to low-yielding synthetic routes, toxicity or other problems that led to their abandonment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTNbV8qi9P4bVg90H21EOLACvtfcHk0lgjfHTbfpg3pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslOntrc%253D&md5=ccd9ad86ef8a1baf078e21747edef73d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DSeley-Radtke%26aufirst%3DK.%2BL.%26aulast%3DYates%26aufirst%3DM.%2BK.%26atitle%3DThe%2520evolution%2520of%2520nucleoside%2520analogue%2520antivirals%253A%2520a%2520review%2520for%2520chemists%2520and%2520non-chemists.%2520Part%25201%253A%2520Early%2520structural%2520modifications%2520to%2520the%2520nucleoside%2520scaffold%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D154%26spage%3D66%26epage%3D86%26doi%3D10.1016%2Fj.antiviral.2018.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, J. A.</span></span> <span> </span><span class="NLM_article-title">A general two-metal-ion mechanism for catalytic RNA</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">6498</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1073/pnas.90.14.6498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.90.14.6498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=8341661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK3sXms1Ghsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=6498-6502&author=T.+A.+Steitzauthor=J.+A.+Steitz&title=A+general+two-metal-ion+mechanism+for+catalytic+RNA&doi=10.1073%2Fpnas.90.14.6498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A general two-metal-ion mechanism for catalytic RNA</span></div><div class="casAuthors">Steitz, Thomas A.; Steitz, Joan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6498-502</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A mechanism is proposed for the RNA-catalyzed reactions involved in RNA splicing and RNase P hydrolysis of precursor tRNA.  The mechanism postulates that chem. catalysis is facilitated by 2 divalent metal ions 3.9 Å apart, as in phosphoryl transfer reactions catalyzed by protein enzymes, such as the 3',5'-exonuclease of Escherichia coli DNA polymerase I.  One metal ion activates the attacking water or sugar OH group, while the other coordinates and stabilizes the oxyanion leaving group.  Both ions act as Lewis acids and stabilize the expected pentacovalent transition state.  The symmetry of a 2-metal ion catalytic site fits well with the known reaction pathway of group I self-splicing introns and can also be reconciled with emerging data on group II self-splicing introns, the spliceosome, and RNase P.  The role of the RNA is to position the 2 catalytic metal ions and properly orient the substrates via 3 specific binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0QHemnv83hrVg90H21EOLACvtfcHk0lgjfHTbfpg3pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXms1Ghsb4%253D&md5=a785aaf05d7c5ecdaa688e7ebb90a12b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.14.6498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.14.6498%26sid%3Dliteratum%253Aachs%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26aulast%3DSteitz%26aufirst%3DJ.%2BA.%26atitle%3DA%2520general%2520two-metal-ion%2520mechanism%2520for%2520catalytic%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D6498%26epage%3D6502%26doi%3D10.1073%2Fpnas.90.14.6498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span> <span> </span><span class="NLM_article-title">Nucleases: diversity of structure, function and mechanism</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1017/s0033583510000181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1017%2FS0033583510000181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20854710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1-93&author=W.+Yang&title=Nucleases%3A+diversity+of+structure%2C+function+and+mechanism&doi=10.1017%2Fs0033583510000181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleases: diversity of structure, function and mechanism</span></div><div class="casAuthors">Yang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-93</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Nucleases cleave the phosphodiester bonds of nucleic acids and may be endo or exo, DNase or RNase, topoisomerases, recombinases, ribozymes, or RNA splicing enzymes.  In this review, I survey nuclease activities with known structures and catalytic machinery and classify them by reaction mechanism and metal-ion dependence and by their biol. function ranging from DNA replication, recombination, repair, RNA maturation, processing, interference, to defense, nutrient regeneration or cell death.  Several general principles emerge from this anal.  There is little correlation between catalytic mechanism and biol. function.  A single catalytic mechanism can be adapted in a variety of reactions and biol. pathways.  Conversely, a single biol. process can often be accomplished by multiple tertiary and quaternary folds and by more than one catalytic mechanism.  Two-metal-ion-dependent nucleases comprise the largest no. of different tertiary folds and mediate the most diverse set of biol. functions.  Metal-ion-dependent cleavage is exclusively assocd. with exonucleases producing mononucleotides and endonucleases that cleave double- or single-stranded substrates in helical and base-stacked conformations.  All metal-ion-independent RNases generate 2'.3'-cyclic phosphate products, and all metal-ion-independent DNases form phospho-protein intermediates.  I also find several previously unnoted relationships between different nucleases and shared catalytic configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7b0a8_b1YbVg90H21EOLACvtfcHk0lgjfHTbfpg3pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D&md5=4cd0bd5cec91b4572adf183ba2d90e7a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000181%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26atitle%3DNucleases%253A%2520diversity%2520of%2520structure%252C%2520function%2520and%2520mechanism%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2011%26volume%3D44%26spage%3D1%26epage%3D93%26doi%3D10.1017%2Fs0033583510000181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grobler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillmock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felock, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espeseth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbertson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgeois, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melamed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazuda, D. J.</span></span> <span> </span><span class="NLM_article-title">Diketo acid inhibitor mechanism and HIV-1 integrase: implications for metal binding in the active site of phosphotransferase enzymes</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">6661</span>– <span class="NLM_lpage">6666</span>, <span class="refDoi"> DOI: 10.1073/pnas.092056199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.092056199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=11997448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFCrs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=6661-6666&author=J.+A.+Groblerauthor=K.+Stillmockauthor=B.+Huauthor=M.+Witmerauthor=P.+Felockauthor=A.+S.+Espesethauthor=A.+Wolfeauthor=M.+Egbertsonauthor=M.+Bourgeoisauthor=J.+Melamedauthor=J.+S.+Waiauthor=S.+Youngauthor=J.+Vaccaauthor=D.+J.+Hazuda&title=Diketo+acid+inhibitor+mechanism+and+HIV-1+integrase%3A+implications+for+metal+binding+in+the+active+site+of+phosphotransferase+enzymes&doi=10.1073%2Fpnas.092056199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Diketo acid inhibitor mechanism and HIV-1 integrase: implications for metal binding in the active site of phosphotransferase enzymes</span></div><div class="casAuthors">Grobler, Jay A.; Stillmock, Kara; Hu, Binghua; Witmer, Marc; Felock, Peter; Espeseth, Amy S.; Wolfe, Abigail; Egbertson, Melissa; Bourgeois, Michele; Melamed, Jeffrey; Wai, John S.; Young, Steve; Vacca, Joseph; Hazuda, Daria J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6661-6666</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The process of integrating the reverse-transcribed HIV-1 DNA into the host chromosomal DNA is catalyzed by the virally encoded enzyme integrase (IN).  Integration requires two metal-dependent reactions, 3' end processing and strand transfer.  Compds. that contain a diketo acid moiety have been shown to selectively inhibit the strand transfer reaction of IN in vitro and in infected cells and are effective as inhibitors of HIV-1 replication.  To characterize the mol. basis of inhibition, we used functional assays and binding assays to evaluate a series of structurally related analogs.  These studies focused on investigating the role of the conserved carboxylate and metal binding.  We demonstrate that an acidic moiety such as a carboxylate or isosteric heterocycle is not required for binding to the enzyme complex but is essential for inhibition and confers distinct metal-dependent properties on the inhibitor.  Binding requires divalent metal and resistance is metal dependent with active site mutants displaying resistance only when the enzymes are evaluated in the context of Mg2+.  The mechanism of action of these inhibitors is therefore likely a consequence of the interaction between the acid moiety and metal ion(s) in the IN active site, resulting in a functional sequestration of the crit. metal cofactor(s).  These studies thus have implications for modeling active site inhibitors of IN, designing and evaluating analogs with improved efficacy, and identifying inhibitors of other metal-dependent phosphotransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkSyCLCLHKbrVg90H21EOLACvtfcHk0lgjfHTbfpg3pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFCrs78%253D&md5=a5d2ccbecd621eaeb1901416e5154fc1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.092056199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.092056199%26sid%3Dliteratum%253Aachs%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DEspeseth%26aufirst%3DA.%2BS.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DEgbertson%26aufirst%3DM.%26aulast%3DBourgeois%26aufirst%3DM.%26aulast%3DMelamed%26aufirst%3DJ.%26aulast%3DWai%26aufirst%3DJ.%2BS.%26aulast%3DYoung%26aufirst%3DS.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DDiketo%2520acid%2520inhibitor%2520mechanism%2520and%2520HIV-1%2520integrase%253A%2520implications%2520for%2520metal%2520binding%2520in%2520the%2520active%2520site%2520of%2520phosphotransferase%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D6661%26epage%3D6666%26doi%3D10.1073%2Fpnas.092056199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structural basis of HIV-1 inhibition by nucleotide-competing reverse transcriptase inhibitor INDOPY-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9996</span>– <span class="NLM_lpage">10002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01289</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFGqur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9996-10002&author=F.+X.+Ruizauthor=A.+Hoangauthor=K.+Dasauthor=E.+Arnold&title=Structural+basis+of+HIV-1+inhibition+by+nucleotide-competing+reverse+transcriptase+inhibitor+INDOPY-1&doi=10.1021%2Facs.jmedchem.9b01289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of HIV-1 Inhibition by Nucleotide-Competing Reverse Transcriptase Inhibitor INDOPY-1</span></div><div class="casAuthors">Ruiz, F. Xavier; Hoang, Anthony; Das, Kalyan; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9996-10002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) is an essential enzyme, targeting half of approved anti-AIDS drugs.  While nucleoside RT inhibitors (NRTIs) are DNA chain terminators, the nucleotide-competing RT inhibitor (NcRTI) INDOPY-1 blocks dNTP binding to RT.  Lack of structural information hindered INDOPY-1 improvement.  Here we report the HIV-1 RT/DNA/INDOPY-1 crystal structure, revealing a unique mode of inhibitor binding at the polymerase active site without involving catalytic metal ions.  The structure may enable new strategies for developing NcRTIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy__3eTkJVObVg90H21EOLACvtfcHk0li5BvYrgej7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGqur%252FL&md5=ccf2bf84e6905e96856eb1586f32ec4c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01289%26sid%3Dliteratum%253Aachs%26aulast%3DRuiz%26aufirst%3DF.%2BX.%26aulast%3DHoang%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520of%2520HIV-1%2520inhibition%2520by%2520nucleotide-competing%2520reverse%2520transcriptase%2520inhibitor%2520INDOPY-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9996%26epage%3D10002%26doi%3D10.1021%2Facs.jmedchem.9b01289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesteleyn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Marck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettens, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Baere, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivens, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Ginderen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schoubroeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehteshami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigerinck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertogs, K.</span></span> <span> </span><span class="NLM_article-title">Indolopyridones inhibit human immunodeficiency virus reverse transcriptase with a novel mechanism of action</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">12283</span>– <span class="NLM_lpage">12292</span>, <span class="refDoi"> DOI: 10.1128/jvi.00889-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1128%2FJVI.00889-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17020946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWksb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2006&pages=12283-12292&author=D.+Jochmansauthor=J.+Devalauthor=B.+Kesteleynauthor=H.+Van+Marckauthor=E.+Bettensauthor=I.+De+Baereauthor=P.+Dehertoghauthor=T.+Ivensauthor=M.+Van+Ginderenauthor=B.+Van+Schoubroeckauthor=M.+Ehteshamiauthor=P.+Wigerinckauthor=M.+G%C3%B6tteauthor=K.+Hertogs&title=Indolopyridones+inhibit+human+immunodeficiency+virus+reverse+transcriptase+with+a+novel+mechanism+of+action&doi=10.1128%2Fjvi.00889-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Indolopyridones inhibit human immunodeficiency virus reverse transcriptase with a novel mechanism of action</span></div><div class="casAuthors">Jochmans, Dirk; Deval, Jerome; Kesteleyn, Bart; Van Marck, Herwig; Bettens, Eva; De Baere, Inky; Dehertogh, Pascale; Ivens, Tania; Van Ginderen, Marcia; Van Schoubroeck, Bertrand; Ehteshami, Maryam; Wigerinck, Piet; Gotte, Matthias; Hertogs, Kurt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12283-12292</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We have discovered a novel class of human immunodeficiency virus (HIV) reverse transcriptase (RT) inhibitors that block the polymn. reaction in a mode distinct from those of the nucleoside or nucleotide RT inhibitors (NRTIs) and nonnucleoside RT inhibitors (NNRTIs).  For this class of indolopyridone compds., steady-state kinetics revealed competitive inhibition with respect to the nucleotide substrate.  Despite substantial structural differences with classical chain terminators or natural nucleotides, these data suggest that the nucleotide binding site of HIV RT may accommodate this novel class of RT inhibitors.  To test this hypothesis, we have studied the mechanism of action of the prototype compd. indolopyridone-1 (INDOPY-1) using a variety of complementary biochem. tools.  Time course expts. with heteropolymeric templates showed "hot spots" for inhibition following the incorporation of pyrimidines (T>C).  Moreover, binding studies and site-specific footprinting expts. revealed that INDOPY-1 traps the complex in the posttranslocational state, preventing binding and incorporation of the next complementary nucleotide.  The novel mode of action translates into a unique resistance profile.  While INDOPY-1 susceptibility is unaffected by mutations assocd. with NNRTI or multidrug NRTI resistance, mutations M184V and Y115F are assocd. with decreased susceptibility, and mutation K65R confers hypersusceptibility to INDOPY-1.  This resistance profile provides addnl. evidence for active site binding.  In conclusion, this class of indolopyridones can occupy the nucleotide binding site of HIV RT by forming a stable ternary complex whose stability is mainly dependent on the nature of the primer 3' end.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_zFvcKcwnfbVg90H21EOLACvtfcHk0li5BvYrgej7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWksb%252FO&md5=8477ac7cd7f82746870c7a6b2c89c1d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.00889-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00889-06%26sid%3Dliteratum%253Aachs%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DKesteleyn%26aufirst%3DB.%26aulast%3DVan%2BMarck%26aufirst%3DH.%26aulast%3DBettens%26aufirst%3DE.%26aulast%3DDe%2BBaere%26aufirst%3DI.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DIvens%26aufirst%3DT.%26aulast%3DVan%2BGinderen%26aufirst%3DM.%26aulast%3DVan%2BSchoubroeck%26aufirst%3DB.%26aulast%3DEhteshami%26aufirst%3DM.%26aulast%3DWigerinck%26aufirst%3DP.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DHertogs%26aufirst%3DK.%26atitle%3DIndolopyridones%2520inhibit%2520human%2520immunodeficiency%2520virus%2520reverse%2520transcriptase%2520with%2520a%2520novel%2520mechanism%2520of%2520action%26jtitle%3DJ.%2520Virol.%26date%3D2006%26volume%3D80%26spage%3D12283%26epage%3D12292%26doi%3D10.1128%2Fjvi.00889-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debarge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of α-Carboxy Phosphononucleosides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1021/jo101738e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo101738e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFShtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=105-126&author=S.+Debargeauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Design+and+Synthesis+of+%CE%B1-Carboxy+Phosphononucleosides&doi=10.1021%2Fjo101738e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of α-Carboxy Phosphono-Nucleosides</span></div><div class="casAuthors">Debarge, Sebastien; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-126</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rhodium catalyzed O-H insertion reactions employing α-diazo-phosphonate with appropriately protected thymidine, uridine, cytosine, adenosine and guanosine derivs. leads to novel 5'-phosphono-nucleoside derivs.  Deprotection led to a novel series of phosphono derivs. bearing a carboxylic acid moiety adjacent to the phosphonate group with potential antiviral and/or anticancer activity.  The phosphono-nucleosides bearing an α-carboxylic acid group are envisaged as potential diphosphate mimics.  Conversion to mono- and diphosphorylated phosphono-nucleosides has been effected for evaluation as nucleoside triphosphate mimics.  Most of the novel phosphono-nucleosides proved to be inactive against a variety of DNA and RNA viruses.  Only the phosphono AZT derivs. showed weak activity against HIV-1 and HIV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9bW35PpIOvrVg90H21EOLACvtfcHk0li5BvYrgej7Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFShtbvK&md5=66461aaaf0a248f92117aa5797bf081e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo101738e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo101738e%26sid%3Dliteratum%253Aachs%26aulast%3DDebarge%26aufirst%3DS.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520%25CE%25B1-Carboxy%2520Phosphononucleosides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D105%26epage%3D126%26doi%3D10.1021%2Fjo101738e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span> <span> </span><span class="NLM_article-title">Pronounced Inhibition Shift from HIV Reverse Transcriptase to Herpetic DNA Polymerases by Increasing the Flexibility of α-Carboxy Nucleoside Phosphonates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8110</span>– <span class="NLM_lpage">8127</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kit7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8110-8127&author=J.+Johnauthor=Y.+Kimauthor=N.+Bennettauthor=K.+Dasauthor=S.+Liekensauthor=L.+Naesensauthor=E.+Arnoldauthor=A.+R.+Maguireauthor=M.+G%C3%B6tteauthor=W.+Dehaenauthor=J.+Balzarini&title=Pronounced+Inhibition+Shift+from+HIV+Reverse+Transcriptase+to+Herpetic+DNA+Polymerases+by+Increasing+the+Flexibility+of+%CE%B1-Carboxy+Nucleoside+Phosphonates&doi=10.1021%2Facs.jmedchem.5b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pronounced Inhibition Shift from HIV Reverse Transcriptase to Herpetic DNA Polymerases by Increasing the Flexibility of α-Carboxy Nucleoside Phosphonates</span></div><div class="casAuthors">John, Jubi; Kim, Youngju; Bennett, Nicholas; Das, Kalyan; Liekens, Sandra; Naesens, Lieve; Arnold, Eddy; Maguire, Anita R.; Gotte, Matthias; Dehaen, Wim; Balzarini, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8110-8127</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">α-Carboxynucleoside phosphonates (α-CNPs) are novel viral DNA polymerase inhibitors that do not need metabolic conversion for enzyme inhibition.  The prototype contains a cyclopentyl linker between nucleobase and α-carboxyphosphonate and preferentially (50- to 100-fold) inhibits HIV-1 RT compared with herpetic DNA polymerases.  A synthesis methodol. involving three steps has been developed for the synthesis of a series of novel α-CNPs, including a Rh(II)-catalyzed O-H insertion that connects the carboxyphosphonate group to a linker moiety and an attachment of a nucleobase to the other end of the linker by a Mitsunobu reaction followed by final deprotection.  Replacing the cyclopentyl moiety in the prototype α-CNPs by a more flexible entity results in a selectivity shift of ∼100-fold in favor of the herpetic DNA polymerases when compared to selectivity for HIV-1 RT.  The nature of the kinetic interaction of the acyclic α-CNPs against the herpetic DNA polymerases differs from the nature of the nucleobase-specific kinetic interaction of the cyclopentyl α-CNPs against HIV RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGootBqwSKGJwrVg90H21EOLACvtfcHk0lhcouTLUjwfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kit7fO&md5=870e3e13a71997e19138133dbb3eacf2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01180%26sid%3Dliteratum%253Aachs%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DDehaen%26aufirst%3DW.%26aulast%3DBalzarini%26aufirst%3DJ.%26atitle%3DPronounced%2520Inhibition%2520Shift%2520from%2520HIV%2520Reverse%2520Transcriptase%2520to%2520Herpetic%2520DNA%2520Polymerases%2520by%2520Increasing%2520the%2520Flexibility%2520of%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8110%26epage%3D8127%26doi%3D10.1021%2Facs.jmedchem.5b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keane, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legigan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of α-Carboxy Nucleoside Phosphonate Analogues and Evaluation as HIV-1 Reverse Transcriptase-Targeting Agents</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2479</span>– <span class="NLM_lpage">2493</span>, <span class="refDoi"> DOI: 10.1021/jo502549y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502549y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=2479-2493&author=S.+J.+Keaneauthor=A.+Fordauthor=N.+D.+Mullinsauthor=N.+M.+Maguireauthor=T.+Legiganauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Design+and+Synthesis+of+%CE%B1-Carboxy+Nucleoside+Phosphonate+Analogues+and+Evaluation+as+HIV-1+Reverse+Transcriptase-Targeting+Agents&doi=10.1021%2Fjo502549y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of α-Carboxy Nucleoside Phosphonate Analogues and Evaluation as HIV-1 Reverse Transcriptase-Targeting Agents</span></div><div class="casAuthors">Keane, Sarah J.; Ford, Alan; Mullins, Nicholas D.; Maguire, Nuala M.; Legigan, Thibaut; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2479-2493</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of the first series of a new class of nucleoside phosphonate analogs, e.g. I, is described.  Addn. of a carboxyl group at the α position of carbocyclic nucleoside phosphonate analogs leads to a novel class of potent HIV reverse transcriptase (RT) inhibitors, α-carboxy nucleoside phosphonates (α-CNPs).  Key steps in the synthesis of the compds. are Rh-catalyzed O-H insertion and Pd-catalyzed allylation reactions.  In cell-free assays, the final products are markedly inhibitory against HIV RT and do not require phosphorylation to exhibit anti-RT activity, which indicates that the α-carboxy-phosphonate function is efficiently recognized by HIV RT as a triphosphate entity, an unprecedented property of nucleoside mono-phosphates and mono-phosphonates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPoDSAEHJHR7Vg90H21EOLACvtfcHk0lhcouTLUjwfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7nF&md5=23a5cebed6ac346fdd7ddff18fa92474</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjo502549y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502549y%26sid%3Dliteratum%253Aachs%26aulast%3DKeane%26aufirst%3DS.%2BJ.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DN.%2BD.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DLegigan%26aufirst%3DT.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonate%2520Analogues%2520and%2520Evaluation%2520as%2520HIV-1%2520Reverse%2520Transcriptase-Targeting%2520Agents%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2479%26epage%3D2493%26doi%3D10.1021%2Fjo502549y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Guanine α-Carboxy Nucleoside Phosphonate (G-α-CNP), a Direct Inhibitor of Multiple Viral DNA Polymerases</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">10510</span>– <span class="NLM_lpage">10517</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.8b01124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.8b01124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVCqtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=10510-10517&author=N.+M.+Maguireauthor=A.+Fordauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Synthesis+of+Guanine+%CE%B1-Carboxy+Nucleoside+Phosphonate+%28G-%CE%B1-CNP%29%2C+a+Direct+Inhibitor+of+Multiple+Viral+DNA+Polymerases&doi=10.1021%2Facs.joc.8b01124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Guanine α-Carboxy Nucleoside Phosphonate (G-α-CNP), a Direct Inhibitor of Multiple Viral DNA Polymerases</span></div><div class="casAuthors">Maguire, Nuala M.; Ford, Alan; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10510-10517</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of guanine α-carboxy nucleoside phosphonate (G-α-CNP) is described.  Two routes provide access to racemic G-α-CNP I, one via base construction and the other utilizing Tsuji-Trost allylic substitution.  The latter methodol. was also applied to the enantiopure synthesis of both antipodes of G-α-CNP, each of which shows interesting antiviral DNA polymerase activity.  Addnl., we report an improved multi-gram scale prepn. of the cyclopentene building block 10, starting material for the preferred Tsuji-Trost route to I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBENxgoUKQ07Vg90H21EOLACvtfcHk0lhcouTLUjwfjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVCqtrvN&md5=22abf2613902b9b0765d513c119fa49e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.8b01124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.8b01124%26sid%3Dliteratum%253Aachs%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DSynthesis%2520of%2520Guanine%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonate%2520%2528G-%25CE%25B1-CNP%2529%252C%2520a%2520Direct%2520Inhibitor%2520of%2520Multiple%2520Viral%2520DNA%2520Polymerases%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2018%26volume%3D83%26spage%3D10510%26epage%3D10517%26doi%3D10.1021%2Facs.joc.8b01124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Exploring the role of the α-carboxyphosphonate moiety in the HIV-RT activity of α-carboxy nucleoside phosphonates</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2454</span>– <span class="NLM_lpage">2465</span>, <span class="refDoi"> DOI: 10.1039/c5ob02507a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1039%2FC5OB02507A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=26813581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtF2isL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2454-2465&author=N.+D.+Mullinsauthor=N.+M.+Maguireauthor=A.+Fordauthor=K.+Dasauthor=E.+Arnoldauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Exploring+the+role+of+the+%CE%B1-carboxyphosphonate+moiety+in+the+HIV-RT+activity+of+%CE%B1-carboxy+nucleoside+phosphonates&doi=10.1039%2Fc5ob02507a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the role of the α-carboxyphosphonate moiety in the HIV-RT activity of α-carboxy nucleoside phosphonates</span></div><div class="casAuthors">Mullins, Nicholas D.; Maguire, Nuala M.; Ford, Alan; Das, Kalyan; Arnold, Eddy; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2454-2465</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As α-carboxy nucleoside phosphonates (α-CNPs) have demonstrated a novel mode of action of HIV-1 reverse transcriptase inhibition, structurally related derivs. were synthesized, namely the malonate I, two unsatd. and satd. bisphosphonates and amide.  These compds. were evaluated for inhibition of HIV-1 reverse transcriptase in cell-free assays.  The importance of the α-carboxy phosphonoacetic acid moiety for achieving reverse transcriptase inhibition, without the need for prior phosphorylation, was confirmed.  The malonate deriv. I was less active by two orders of magnitude than the original α-CNPs, while displaying the same pattern of kinetic behavior; interestingly the activity resides in the "L"-enantiomer of I, as seen with the earlier series of α-CNPs.  A crystal structure with an RT/DNA complex at 2.95 Å resoln. revealed the binding of the "L"-enantiomer of I, at the polymerase active site with a weaker metal ion chelation environment compared to 1a (T-α-CNP) which may explain the lower inhibitory activity of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodQ95f7BrdvLVg90H21EOLACvtfcHk0ljqfvG_qv9TPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtF2isL8%253D&md5=732f00d94b631696fe4bf29a26f4af08</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2FC5OB02507A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB02507A%26sid%3Dliteratum%253Aachs%26aulast%3DMullins%26aufirst%3DN.%2BD.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DExploring%2520the%2520role%2520of%2520the%2520%25CE%25B1-carboxyphosphonate%2520moiety%2520in%2520the%2520HIV-RT%2520activity%2520of%2520%25CE%25B1-carboxy%2520nucleoside%2520phosphonates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D2454%26epage%3D2465%26doi%3D10.1039%2Fc5ob02507a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hladezuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chastagner, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debarge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Development of O-H insertion for the attachment of phosphonates to nucleosides; synthesis of α-carboxy phosphononucleosides</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1894</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2011.12.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.tet.2011.12.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=1894-1909&author=I.+Hladezukauthor=V.+Chastagnerauthor=S.+G.+Collinsauthor=S.+J.+Plunkettauthor=A.+Fordauthor=S.+Debargeauthor=A.+R.+Maguire&title=Development+of+O-H+insertion+for+the+attachment+of+phosphonates+to+nucleosides%3B+synthesis+of+%CE%B1-carboxy+phosphononucleosides&doi=10.1016%2Fj.tet.2011.12.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of O-H insertion for the attachment of phosphonates to nucleosides; synthesis of α-carboxy phosphononucleosides</span></div><div class="casAuthors">Hladezuk, Isabelle; Chastagner, Veronique; Collins, Stuart G.; Plunkett, Stephen J.; Ford, Alan; Debarge, Sebastien; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1894-1909</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Development of rhodium catalyzed O-H insertion reactions employing α-diazophosphonates with appropriately protected adenosine, uridine and thymidine derivs. is described.  This synthetic methodol. leads, following deprotection, to novel phosphononucleoside derivs. bearing a carboxylic acid moiety adjacent to the phosphonate, e.g. I.  Protection strategies are crit. to the success of the key O-H insertion.  There are two important aspects: avoiding competing insertion pathways or catalyst poisoning, and being able to achieve deprotection without degrdn. of the phosphononucleosides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3LL9mtO-XL7Vg90H21EOLACvtfcHk0ljqfvG_qv9TPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGqsL4%253D&md5=9ef4af19fc5834b7158277bb6ceda5ac</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2011.12.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2011.12.077%26sid%3Dliteratum%253Aachs%26aulast%3DHladezuk%26aufirst%3DI.%26aulast%3DChastagner%26aufirst%3DV.%26aulast%3DCollins%26aufirst%3DS.%2BG.%26aulast%3DPlunkett%26aufirst%3DS.%2BJ.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DDebarge%26aufirst%3DS.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDevelopment%2520of%2520O-H%2520insertion%2520for%2520the%2520attachment%2520of%2520phosphonates%2520to%2520nucleosides%253B%2520synthesis%2520of%2520%25CE%25B1-carboxy%2520phosphononucleosides%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D1894%26epage%3D1909%26doi%3D10.1016%2Fj.tet.2011.12.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Berckelaer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmer, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Guanine α-carboxy nucleoside phosphonate (G-α-CNP) shows a different inhibitory kinetic profile against the DNA polymerases of human immunodeficiency virus (HIV) and herpes viruses</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.bcp.2017.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=28390939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFWgsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=51-61&author=J.+Balzariniauthor=M.+Menniauthor=K.+Dasauthor=L.+van+Berckelaerauthor=A.+Fordauthor=N.+M.+Maguireauthor=S.+Liekensauthor=P.+E.+Boehmerauthor=E.+Arnoldauthor=M.+G%C3%B6tteauthor=A.+R.+Maguire&title=Guanine+%CE%B1-carboxy+nucleoside+phosphonate+%28G-%CE%B1-CNP%29+shows+a+different+inhibitory+kinetic+profile+against+the+DNA+polymerases+of+human+immunodeficiency+virus+%28HIV%29+and+herpes+viruses&doi=10.1016%2Fj.bcp.2017.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Guanine α-carboxy nucleoside phosphonate (G-α-CNP) shows a different inhibitory kinetic profile against the DNA polymerases of human immunodeficiency virus (HIV) and herpes viruses</span></div><div class="casAuthors">Balzarini, Jan; Menni, Michael; Das, Kalyan; van Berckelaer, Lizette; Ford, Alan; Maguire, Nuala M.; Liekens, Sandra; Boehmer, Paul E.; Arnold, Eddy; Gotte, Matthias; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">α-Carboxy nucleoside phosphonates (α-CNPs) are modified nucleotides that represent a novel class of nucleotide-competing reverse transcriptase (RT) inhibitors (NcRTIs).  They were designed to act directly against HIV-1 RT without the need for prior activation (phosphorylation).  In this respect, they differ from the nucleoside or nucleotide RTIs [N(t)RTIs] that require conversion to their triphosphate forms before being inhibitory to HIV-1 RT.  The guanine deriv. (G-α-CNP) has now been synthesized and investigated for the first time.  The (L)-(+)-enantiomer of G-α-CNP directly and competitively inhibits HIV-1 RT by interacting with the substrate active site of the enzyme.  The (D)-(-)-enantiomer proved inactive against HIV-1 RT.  In contrast, the (+)- and (-)-enantiomers of G-α-CNP inhibited herpes (i.e. HSV-1, HCMV) DNA polymerases in a non- or uncompetitive manner, strongly indicating interaction of the (L)-(+)- and the (D)-(-)-G-α-CNPs at a location different from the polymerase substrate active site of the herpes enzymes.  Such entirely different inhibition profile of viral polymerases is unprecedented for a single antiviral drug mol.  Moreover, within the class of α-CNPs, subtle differences in their sensitivity to mutant HIV-1 RT enzymes were obsd. depending on the nature of the nucleobase in the α-CNP mols.  The unique properties of the α-CNPs make this class of compds., including G-α-CNP, direct acting inhibitors of multiple viral DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru34nHOlZAWLVg90H21EOLACvtfcHk0ljqfvG_qv9TPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFWgsbc%253D&md5=6f725c42e63f8fc28e5dc9fa9feb13ec</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMenni%26aufirst%3DM.%26aulast%3DDas%26aufirst%3DK.%26aulast%3Dvan%2BBerckelaer%26aufirst%3DL.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DBoehmer%26aufirst%3DP.%2BE.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DGuanine%2520%25CE%25B1-carboxy%2520nucleoside%2520phosphonate%2520%2528G-%25CE%25B1-CNP%2529%2520shows%2520a%2520different%2520inhibitory%2520kinetic%2520profile%2520against%2520the%2520DNA%2520polymerases%2520of%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520and%2520herpes%2520viruses%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D136%26spage%3D51%26epage%3D61%26doi%3D10.1016%2Fj.bcp.2017.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernatchez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vande Voorde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Alpha-carboxy nucleoside phosphonates as universal nucleoside triphosphate mimics</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">3475</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1073/pnas.1420233112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.1420233112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=25733891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Grtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=3475-3480&author=J.+Balzariniauthor=K.+Dasauthor=J.+A.+Bernatchezauthor=S.+E.+Martinezauthor=M.+Ngureauthor=S.+Keaneauthor=A.+Fordauthor=N.+Maguireauthor=N.+Mullinsauthor=J.+Johnauthor=Y.+Kimauthor=W.+Dehaenauthor=J.+Vande+Voordeauthor=S.+Liekensauthor=L.+Naesensauthor=M.+G%C3%B6tteauthor=A.+R.+Maguireauthor=E.+Arnold&title=Alpha-carboxy+nucleoside+phosphonates+as+universal+nucleoside+triphosphate+mimics&doi=10.1073%2Fpnas.1420233112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha-carboxy nucleoside phosphonates as universal nucleoside triphosphate mimics</span></div><div class="casAuthors">Balzarini, Jan; Das, Kalyan; Bernatchez, Jean A.; Martinez, Sergio E.; Ngure, Marianne; Keane, Sarah; Ford, Alan; Maguire, Nuala; Mullins, Niki; John, Jubi; Kim, Youngju; Dehaen, Wim; Voorde, Johan Vande; Liekens, Sandra; Naesens, Lieve; Gotte, Matthias; Maguire, Anita R.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3475-3480</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polymerases have a structurally highly conserved neg. charged amino acid motif that is strictly required for Mg2+ cation-dependent catalytic incorporation of (d)NTP nucleotides into nucleic acids.  Based on these characteristics, a nucleoside monophosphonate scaffold, α-carboxy nucleoside phosphonate (α-CNP), was designed that is recognized by a variety of polymerases.  Kinetic, biochem., and crystallog. studies with HIV-1 reverse transcriptase revealed that α-CNPs mimic the dNTP binding through a carboxylate oxygen, two phosphonate oxygens, and base-pairing with the template.  In particular, the carboxyl oxygen of the α-CNP acts as the potential equiv. of the α-phosphate oxygen of dNTPs and two oxygens of the phosphonate group of the α-CNP chelate Mg2+, mimicking the chelation by the β- and γ-phosphate oxygens of dNTPs. α-CNPs (i) do not require metabolic activation (phosphorylation), (ii) bind directly to the substrate-binding site, (iii) chelate one of the two active site Mg2+ ions, and (iv) reversibly inhibit the polymerase catalytic activity without being incorporated into nucleic acids.  In addn., α-CNPs were also found to selectively interact with regulatory (i.e., allosteric) Mg2+-dNTP-binding sites of nucleos(t)ide-metabolizing enzymes susceptible to metabolic regulation. α-CNPs represent an entirely novel and broad technol. platform for the development of specific substrate active- or regulatory-site inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz-g4rWbNDV7Vg90H21EOLACvtfcHk0liK-oku3gYmWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Grtro%253D&md5=0cff60a295b9eb86a591cce8997c6613</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1420233112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1420233112%26sid%3Dliteratum%253Aachs%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBernatchez%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DNgure%26aufirst%3DM.%26aulast%3DKeane%26aufirst%3DS.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMaguire%26aufirst%3DN.%26aulast%3DMullins%26aufirst%3DN.%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DDehaen%26aufirst%3DW.%26aulast%3DVande%2BVoorde%26aufirst%3DJ.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DAlpha-carboxy%2520nucleoside%2520phosphonates%2520as%2520universal%2520nucleoside%2520triphosphate%2520mimics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D3475%26epage%3D3480%26doi%3D10.1073%2Fpnas.1420233112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Conformational states of HIV-1 reverse transcriptase for nucleotide incorporation vs pyrophosphorolysis-binding of foscarnet</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2158</span>– <span class="NLM_lpage">2164</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00187</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00187" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2158-2164&author=K.+Dasauthor=J.+Balzariniauthor=M.+T.+Millerauthor=A.+R.+Maguireauthor=J.+J.+DeStefanoauthor=E.+Arnold&title=Conformational+states+of+HIV-1+reverse+transcriptase+for+nucleotide+incorporation+vs+pyrophosphorolysis-binding+of+foscarnet&doi=10.1021%2Facschembio.6b00187"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational States of HIV-1 Reverse Transcriptase for Nucleotide Incorporation vs Pyrophosphorolysis-Binding of Foscarnet</span></div><div class="casAuthors">Das, Kalyan; Balzarini, Jan; Miller, Matthew T.; Maguire, Anita R.; DeStefano, Jeffrey J.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2158-2164</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) catalytically incorporates individual nucleotides into a viral DNA strand complementing an RNA or DNA template strand; the polymerase active site of RT adopts multiple conformational and structural states while performing this task.  The states assocd. are dNTP binding at the N site, catalytic incorporation of a nucleotide, release of a pyrophosphate, and translocation of the primer 3'-end to the P site.  Structural characterization of each of these states may help in understanding the mol. mechanisms of drug activity and resistance and in developing new RT inhibitors.  Using a 38-mer DNA template-primer aptamer as the substrate mimic, we crystd. an RT/dsDNA complex that is catalytically active, yet translocation-incompetent in crystals.  The ability of RT to perform dNTP binding and incorporation in crystals permitted obtaining a series of structures: (I) RT/DNA (P-site), (II) RT/DNA/AZTTP ternary, (III) RT/AZT-terminated DNA (N-site), and (IV) RT/AZT-terminated DNA (N-site)/foscarnet complexes.  The stable N-site complex permitted the binding of foscarnet as a pyrophosphate mimic.  The Mg2+ ions dissocd. after catalytic addn. of AZTMP in the pretranslocated structure III, whereas ions A and B had re-entered the active site to bind foscarnet in structure IV.  The binding of foscarnet involves chelation with the Mg2+ (B) ion and interactions with K65 and R72.  The anal. of interactions of foscarnet and the recently discovered nucleotide-competing RT inhibitor (NcRTI) α-T-CNP in two different conformational states of the enzyme provides insights for developing new classes of polymerase active site RT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPnYWlNCZhubVg90H21EOLACvtfcHk0liK-oku3gYmWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVSksLk%253D&md5=aa013aea155b60e18620a5b5d378aa18</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00187%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DM.%2BT.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DDeStefano%26aufirst%3DJ.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DConformational%2520states%2520of%2520HIV-1%2520reverse%2520transcriptase%2520for%2520nucleotide%2520incorporation%2520vs%2520pyrophosphorolysis-binding%2520of%2520foscarnet%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2158%26epage%3D2164%26doi%3D10.1021%2Facschembio.6b00187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adelfinskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Amino acid phosphoramidate nucleotides as alternative substrates for HIV-1 reverse transcriptase</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4356</span>– <span class="NLM_lpage">4358</span>, <span class="refDoi"> DOI: 10.1002/anie.200605016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fanie.200605016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17443759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BD2svgtlKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4356-4358&author=O.+Adelfinskayaauthor=P.+Herdewijn&title=Amino+acid+phosphoramidate+nucleotides+as+alternative+substrates+for+HIV-1+reverse+transcriptase&doi=10.1002%2Fanie.200605016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid phosphoramidate nucleotides as alternative substrates for HIV-1 reverse transcriptase</span></div><div class="casAuthors">Adelfinskaya Olga; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4356-8</span>
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSu-rO0KHaB_rJzI8owOhpDfW6udTcc2eaa4OJ1-Rmyubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svgtlKnsw%253D%253D&md5=3ac775f81677aadc732603bd4dd45f96</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.200605016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200605016%26sid%3Dliteratum%253Aachs%26aulast%3DAdelfinskaya%26aufirst%3DO.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DAmino%2520acid%2520phosphoramidate%2520nucleotides%2520as%2520alternative%2520substrates%2520for%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2007%26volume%3D46%26spage%3D4356%26epage%3D4358%26doi%3D10.1002%2Fanie.200605016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adelfinskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marliere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauwelaerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Polymerase-catalyzed synthesis of DNA from phosphoramidate conjugates of deoxynucleotides and amino acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">5060</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkm498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17652326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFWgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=5060-5072&author=O.+Adelfinskayaauthor=M.+Terrazasauthor=M.+Froeyenauthor=P.+Marliereauthor=K.+Nauwelaertsauthor=P.+Herdewijn&title=Polymerase-catalyzed+synthesis+of+DNA+from+phosphoramidate+conjugates+of+deoxynucleotides+and+amino+acids&doi=10.1093%2Fnar%2Fgkm498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Polymerase-catalyzed synthesis of DNA from phosphoramidate conjugates of deoxynucleotides and amino acids</span></div><div class="casAuthors">Adelfinskaya, O.; Terrazas, M.; Froeyen, M.; Marliere, P.; Nauwelaerts, K.; Herdewijn, P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5060-5072</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Some selected amino acids, in particular L-aspartic acid (L-Asp) and L-histidine (L-His), can function as leaving group during polymerase-catalyzed incorporation of deoxyadenosine monophosphate (dAMP) in DNA.  Although L-Asp-dAMP and L-His-dAMP bind, most probably, in a different way in the active site of the enzyme, aspartic acid and histidine can be considered as mimics of the pyrophosphate moiety of deoxyadenosine triphosphate.  L-Aspartic acid is more efficient than D-aspartic acid as leaving group.  Such P-N conjugates of amino acids and deoxynucleotides provide a novel exptl. ground for diversifying nucleic acid metab. in the field of synthetic biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqe7h6NI-V7Vg90H21EOLACvtfcHk0lh-ptGeL69p0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFWgsLs%253D&md5=9bad158cc983934910ca44c87b77a573</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm498%26sid%3Dliteratum%253Aachs%26aulast%3DAdelfinskaya%26aufirst%3DO.%26aulast%3DTerrazas%26aufirst%3DM.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DMarliere%26aufirst%3DP.%26aulast%3DNauwelaerts%26aufirst%3DK.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DPolymerase-catalyzed%2520synthesis%2520of%2520DNA%2520from%2520phosphoramidate%2520conjugates%2520of%2520deoxynucleotides%2520and%2520amino%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D5060%26epage%3D5072%26doi%3D10.1093%2Fnar%2Fgkm498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Terrazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Enzymatically Catalyzed DNA Synthesis Using L-Asp-dGMP, L-Asp-dCMP, and L-Asp-dTMP</span>. <i>Chem. Biodivers.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1002/cbdv.200890013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbdv.200890013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=18205125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252Fnt12nsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=31-39&author=M.+Terrazasauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=Enzymatically+Catalyzed+DNA+Synthesis+Using+L-Asp-dGMP%2C+L-Asp-dCMP%2C+and+L-Asp-dTMP&doi=10.1002%2Fcbdv.200890013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymatically catalyzed DNA synthesis using L-Asp-dGMP, L-Asp-dCMP, and L-Asp-dTMP</span></div><div class="casAuthors">Terrazas Montserrat; Marliere Philippe; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & biodiversity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The replacement of the pyrophosphate moiety of deoxynucleoside triphosphates by L-aspartic acid allows incorporation of natural deoxynucleosides into DNA using HIV reverse transcriptase (RT) as enzyme, while retaining the canonical base-pair selectivity.  N-Methylation of the L-aspartic acid leaving group results in a reduced fidelity of incorporation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXucBseKJbyZggzHGM8X5CfW6udTcc2eYhSjdDGhaH1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252Fnt12nsA%253D%253D&md5=7a3af442dc2967b128d4acefca0f8f64</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200890013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200890013%26sid%3Dliteratum%253Aachs%26aulast%3DTerrazas%26aufirst%3DM.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DEnzymatically%2520Catalyzed%2520DNA%2520Synthesis%2520Using%2520L-Asp-dGMP%252C%2520L-Asp-dCMP%252C%2520and%2520L-Asp-dTMP%26jtitle%3DChem.%2520Biodivers.%26date%3D2008%26volume%3D5%26spage%3D31%26epage%3D39%26doi%3D10.1002%2Fcbdv.200890013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giraut, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Iminodiacetic-phosphoramidates as metabolic prototypes for diversifying nucleic acid polymerization in vivo</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2541</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp1246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkp1246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20097909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFGnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2541-2550&author=A.+Girautauthor=X.-p.+Songauthor=M.+Froeyenauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=Iminodiacetic-phosphoramidates+as+metabolic+prototypes+for+diversifying+nucleic+acid+polymerization+in+vivo&doi=10.1093%2Fnar%2Fgkp1246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Iminodiacetic-phosphoramidates as metabolic prototypes for diversifying nucleic acid polymerization in vivo</span></div><div class="casAuthors">Giraut, Anne; Song, Xiao-Ping; Froeyen, Matheus; Marliere, Philippe; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2541-2550</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Previous studies in our lab. proved that certain functional groups are able to mimic the pyrophosphate moiety and act as leaving groups in the enzymic polymn. of DNAs by HIV-1 reverse transcriptase.  When the potential leaving group possesses two carboxylic acid moieties linked to the nucleoside via a phosphoramidate bond, it is efficiently recognized by this error-prone enzyme, resulting in nucleotide incorporation into DNA.  Here, we present a new efficient alternative leaving group, iminodiacetic acid, which displays enhanced kinetics and an enhanced elongation capacity compared to previous results obtained with amino acid deoxyadenosine phosphoramidates.  Iminodiacetic acid phosphoramidate of deoxyadenosine monophosphate (IDA-dAMP) is processed by HIV-1 RT as a substrate for single nucleotide incorporation and displays a typical Michaelis-Menten kinetic profile.  This novel substrate also proved to be successful in primer strand elongation of a seven-base template overhang.  Modeling of this new substrate in the active site of the enzyme revealed that the interactions formed between the triphosphate moiety, magnesium ions and enzyme's residues could be different from those of the natural triphosphate substrate and is likely to involve addnl. amino acid residues.  Preliminary testing for a potential metabolic accessibility lets us to envision its possible use in an orthogonal system for nucleic acid synthesis that would not influence or be influenced by genetic information from the outside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobcJNqBOdKGbVg90H21EOLACvtfcHk0lhZQCMNwzen7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFGnu74%253D&md5=5b632d1c2cb21440cf6ead8bf69aa494</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1246%26sid%3Dliteratum%253Aachs%26aulast%3DGiraut%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DX.-p.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DIminodiacetic-phosphoramidates%2520as%2520metabolic%2520prototypes%2520for%2520diversifying%2520nucleic%2520acid%2520polymerization%2520in%2520vivo%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2541%26epage%3D2550%26doi%3D10.1093%2Fnar%2Fgkp1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Iminodipropionic acid as the leaving group for DNA polymerization by HIV-1 reverse transcriptase</span>. <i>Chembiochem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1868</span>– <span class="NLM_lpage">1880</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbic.201100160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21714056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVCmu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1868-1880&author=X.-P.+Songauthor=C.+Bouillonauthor=E.+Lescrinierauthor=P.+Herdewijn&title=Iminodipropionic+acid+as+the+leaving+group+for+DNA+polymerization+by+HIV-1+reverse+transcriptase&doi=10.1002%2Fcbic.201100160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Iminodipropionic Acid as the Leaving Group for DNA Polymerization by HIV-1 Reverse Transcriptase</span></div><div class="casAuthors">Song, Xiao-Ping; Bouillon, Camille; Lescrinier, Eveline; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1868-1880</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Previous studies have demonstrated that some selected amino monoacids and amino diacids can function as leaving groups in the polymerase-catalyzed incorporation of deoxynucleotides into DNA.  Among these, the iminodiacetic acid phosphoramidate of deoxyadenosine monophosphate (IDA-dAMP) represents an interesting example, as it could overcome some of the problems obsd. when using L-aspartic acid as the leaving group, i.e., poor chain elongation.  We have now synthesized and evaluated a series of IDA-dAMP analogs that bear either an extended aliph. chain in the amino acid function, or a phosphonic acid moiety (substituting for the carboxylic acid function).  Among these compds., the nucleotide with an iminodipropionic acid leaving group (IDP-dAMP) was identified as the best substrate; the excellent single incorporation (91% conversion to a P+1 strand at 50 μM) was at a substrate concn. ten times lower than that used for IDA-dAMP.  This nucleotide also presented improved kinetics and elongation capability compared to IDA-dAMP.  The analogs with T, G, and C base moieties were also investigated for their incorporation ability with HIV-1 RT.  The incorporation efficiency was found to decrease in the order A > T > G > C.  The properties of the iminodipropionic acid as the leaving group surpass those of previously evaluated leaving groups; this acid will be a prime candidate for in vivo testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjsahfrtqUgLVg90H21EOLACvtfcHk0lhZQCMNwzen7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVCmu78%253D&md5=eea9ba713fb278dd1ab7589500d9dc57</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100160%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.-P.%26aulast%3DBouillon%26aufirst%3DC.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DIminodipropionic%2520acid%2520as%2520the%2520leaving%2520group%2520for%2520DNA%2520polymerization%2520by%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DChembiochem%26date%3D2011%26volume%3D12%26spage%3D1868%26epage%3D1880%26doi%3D10.1002%2Fcbic.201100160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">3-Phosphono-L-alanine as pyrophosphate mimic for DNA synthesis using HIV-1 reverse transcriptase</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/c0ob00554a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1039%2FC0OB00554A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21103490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=111-119&author=S.+Yangauthor=M.+Froeyenauthor=E.+Lescrinierauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=3-Phosphono-L-alanine+as+pyrophosphate+mimic+for+DNA+synthesis+using+HIV-1+reverse+transcriptase&doi=10.1039%2Fc0ob00554a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">3-Phosphono-L-alanine as pyrophosphate mimic for DNA synthesis using HIV-1 reverse transcriptase</span></div><div class="casAuthors">Yang, Shiqiong; Froeyen, Mathy; Lescrinier, Eveline; Marliere, Philippe; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of sulf(on)ate and phosph(on)ate amino acid phosphoramidate analogs of deoxynucleotides were synthesized as potential substrates for HIV-1 reverse transcriptase.  Taurine, L-cysteic acid, 3-phosphono-L-alanine, O-sulfonato-L-serine, and O-phospho-l-serine were investigated as leaving groups in an enzyme catalyzed DNA synthesis protocol.  Among these analogs, the phosphonate congener performed best and 3-phosphono-L-alanine can be considered as an excellent mimic of the pyrophosphate (PPi) moiety of deoxyadenosine triphosphate, to be used in enzymic synthesis of nucleic acids.  During a single nucleotide incorporation assay the use of 3-phosphono-L-Ala-dAMP as substrate resulted in 95% conversion to a P + 1 strand in 60 min at 50 μM (a concn. 10 times less than found for L-Asp-dAMP) and with improved incorporation kinetics and less stalling.  For the sequences investigated, the efficiency of the incorporation is base dependent and decreases in the order (A ≥ T = G > C).  In all cases, the incorporation follows Watson-Crick rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeYzCNJ85TSbVg90H21EOLACvtfcHk0lhZQCMNwzen7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FO&md5=c9ab71936ae04d10cd5a787f06feb556</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC0OB00554A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00554A%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3D3-Phosphono-L-alanine%2520as%2520pyrophosphate%2520mimic%2520for%2520DNA%2520synthesis%2520using%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D111%26epage%3D119%26doi%3D10.1039%2Fc0ob00554a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Dipeptides as leaving group in the enzyme-catalyzed DNA synthesis</span>. <i>Chem. Biodivers.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2685</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1002/cbdv.201200261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbdv.201200261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=23255441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jsVOrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=2685-2700&author=X.-P.+Songauthor=C.+Bouillonauthor=E.+Lescrinierauthor=P.+Herdewijn&title=Dipeptides+as+leaving+group+in+the+enzyme-catalyzed+DNA+synthesis&doi=10.1002%2Fcbdv.201200261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dipeptides as leaving group in the enzyme-catalyzed DNA synthesis</span></div><div class="casAuthors">Song Xiao-Ping; Bouillon Camille; Lescrinier Eveline; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & biodiversity</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2685-700</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Conjugates of 2'-deoxyadenosine monophosphate with dipeptides have been synthesized and tested as substrates for several polymerases.  Although the incorporation efficiency is not very high, it demonstrates that some of these dipeptides can be accommodated in the active site of polymerases and function as leaving groups in the enzymatic synthesis of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8LuHMtpFx6CA3UY9EwTNrfW6udTcc2ea5pzA-NxVe_bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jsVOrtA%253D%253D&md5=7497c0debe23bc82146910349a07e2f9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.201200261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.201200261%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.-P.%26aulast%3DBouillon%26aufirst%3DC.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DDipeptides%2520as%2520leaving%2520group%2520in%2520the%2520enzyme-catalyzed%2520DNA%2520synthesis%26jtitle%3DChem.%2520Biodivers.%26date%3D2012%26volume%3D9%26spage%3D2685%26epage%3D2700%26doi%3D10.1002%2Fcbdv.201200261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalman, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntee, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Biological Activity of Aromatic Amino Acid Phosphoramidates of 5-Fluoro-2’-deoxyuridine and 1-β-Arabinofuranosylcytosine: Evidence of Phosphoramidase Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">4569</span>– <span class="NLM_lpage">4575</span>, <span class="refDoi"> DOI: 10.1021/jm9603680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9603680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=4569-4575&author=T.+W.+Abrahamauthor=T.+I.+Kalmanauthor=E.+J.+McInteeauthor=C.+R.+Wagner&title=Synthesis+and+Biological+Activity+of+Aromatic+Amino+Acid+Phosphoramidates+of+5-Fluoro-2%E2%80%99-deoxyuridine+and+1-%CE%B2-Arabinofuranosylcytosine%3A+Evidence+of+Phosphoramidase+Activity&doi=10.1021%2Fjm9603680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Activity of Aromatic Amino Acid Phosphoramidates of 5-Fluoro-2'-deoxyuridine and 1-β-Arabinofuranosylcytosine: Evidence of Phosphoramidase Activity</span></div><div class="casAuthors">Abraham, Timothy W.; Kalman, Thomas I.; McIntee, Edward J.; Wagner, Carston R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4569-4575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The amino acid phosphoramidate diesters of FUdR and Ara-C, fluorodeoxyuridyl N-(carbomethoxyphenylethyl)phosphoramidate I, 5-fluoro-2'-deoxy-5'-uridyl N-(1-carbomethoxy-2-indolylethyl)phosphoramidate, 1-β-arabinofuranosylcytosine 5'-N-(1-carbomethoxy-2-phenylethyl)phosphoramidate, and 1-β-arabinofuranosylcytosine 5'-N-(1-carbomethoxy-2-indolylethyl)phosphoramidate, were synthesized and tested for their antitumor activity against L1210 mouse lymphocytic leukemia cells and CCRF-CEM human T-cell lymphoblastic leukemia cells.  Stability studies revealed that >99% of the phosphoramidates remained intact after incubation in 20% calf or 20% human serum.  Intracellular thymidylate synthase (TS) inhibition studies revealed that treatment of L1210 and CCRF-CEM cells with I resulted in significant inhibition of TS in intact and permeabilized cells, while treatment of L929 TK- cells with these compds. did not result in inhibition of TS activity in intact cells.  However, permeabilization of L929 TK- cells enhanced the activity of 5a,b toward intracellular TS by 900- and 1500-fold, resp.  Consequently, it is proposed that the metab. of the phosphoramidate diesters of FUdR in proliferating tissue proceeds through two sep. enzymic steps involving P-N bond cleavage by an unknown phosphoramidase followed by P-O bond cleavage by phosphatases such as 5'-nucleotidase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmqZMoJ5CQFrVg90H21EOLACvtfcHk0ljkzwh-Qca3pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1ylsL8%253D&md5=2698a91970ae6f18d34c68b6e885997c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm9603680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9603680%26sid%3Dliteratum%253Aachs%26aulast%3DAbraham%26aufirst%3DT.%2BW.%26aulast%3DKalman%26aufirst%3DT.%2BI.%26aulast%3DMcIntee%26aufirst%3DE.%2BJ.%26aulast%3DWagner%26aufirst%3DC.%2BR.%26atitle%3DSynthesis%2520and%2520Biological%2520Activity%2520of%2520Aromatic%2520Amino%2520Acid%2520Phosphoramidates%2520of%25205-Fluoro-2%25E2%2580%2599-deoxyuridine%2520and%25201-%25CE%25B2-Arabinofuranosylcytosine%253A%2520Evidence%2520of%2520Phosphoramidase%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D4569%26epage%3D4575%26doi%3D10.1021%2Fjm9603680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singer, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haugland, R. P.</span></span> <span> </span><span class="NLM_article-title">Characterization of PicoGreen reagent and development of a fluorescence-based solution assay for double-stranded DNA quantitation</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>249</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1006/abio.1997.2177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1006%2Fabio.1997.2177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=9212875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2sXktF2rsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1997&pages=228-238&author=V.+L.+Singerauthor=L.+J.+Jonesauthor=S.+T.+Yueauthor=R.+P.+Haugland&title=Characterization+of+PicoGreen+reagent+and+development+of+a+fluorescence-based+solution+assay+for+double-stranded+DNA+quantitation&doi=10.1006%2Fabio.1997.2177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of PicoGreen reagent and development of a fluorescence-based solution assay for double-stranded DNA quantitation</span></div><div class="casAuthors">Singer, Victoria L.; Jones, Laurie J.; Yue, Stephen T.; Haugland, Richard P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">228-238</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A sensitive assay for detecting double-stranded (ds) DNA in soln. is described.  This assay employs a new dye, PicoGreen dsDNA quantitation reagent, which becomes intensely fluorescent upon binding nucleic acids.  The brightness of this reagent is due to its high quantum yield (∼0.5, bound to ds calf thymus DNA) and large molar extinction coeff. (∼70,000 cm-1 M-1).  The fluorescence enhancement of this dye upon binding dsDNA is >1000-fold, with excitation and emission maxima near those of fluorescein.  Unlike Hoechst 33258, PicoGreen reagent fluorescence intensity was the same upon binding to poly(dA)·poly(dT) and poly(dG)·poly(dC) homopolymers.  The PicoGreen assay allowed the detection of 25 pg/mL dsDNA, surpassing the sensitivity achieved with Hoechst 33258 by 400-fold.  The linear concn. range for DNA quantitation extended over four orders of magnitude-25 pg/mL to 1 μg/m-with a single dye concn.  Assay linearity was maintained even in the presence of salts, proteins, poly(ethylene glycol), urea, chloroform, ethanol, and agarose; some ionic detergents and heparin interfered.  Linear DNAs yielded slightly brighter signals than supercoiled plasmids.  Finally, the assay showed greater dsDNA:RNA selectivity than Hoechst 33258 in low ionic strength buffer and better dsDNA:single-stranded DNA selectivity in 1 M NaCl.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomOWcQ1NrPjLVg90H21EOLACvtfcHk0ljkzwh-Qca3pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktF2rsbo%253D&md5=d331528a1da7ed5b724022a0e53139d5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fabio.1997.2177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1997.2177%26sid%3Dliteratum%253Aachs%26aulast%3DSinger%26aufirst%3DV.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BJ.%26aulast%3DYue%26aufirst%3DS.%2BT.%26aulast%3DHaugland%26aufirst%3DR.%2BP.%26atitle%3DCharacterization%2520of%2520PicoGreen%2520reagent%2520and%2520development%2520of%2520a%2520fluorescence-based%2520solution%2520assay%2520for%2520double-stranded%2520DNA%2520quantitation%26jtitle%3DAnal.%2520Biochem.%26date%3D1997%26volume%3D249%26spage%3D228%26epage%3D238%26doi%3D10.1006%2Fabio.1997.2177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">HIV-1 reverse transcriptase complex with DNA and nevirapine reveals non-nucleoside inhibition mechanism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnsmb.2223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22266819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=253-259&author=K.+Dasauthor=S.+E.+Martinezauthor=J.+D.+Baumanauthor=E.+Arnold&title=HIV-1+reverse+transcriptase+complex+with+DNA+and+nevirapine+reveals+non-nucleoside+inhibition+mechanism&doi=10.1038%2Fnsmb.2223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 reverse transcriptase complex with DNA and nevirapine reveals non-nucleoside inhibition mechanism</span></div><div class="casAuthors">Das, Kalyan; Martinez, Sergio E.; Bauman, Joseph D.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-259</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Combinations of nucleoside and non-nucleoside inhibitors (NNRTIs) of HIV-1 reverse transcriptase (RT) are widely used in anti-AIDS therapies.  Five NNRTIs, including nevirapine, are clin. drugs; however, the mol. mechanism of inhibition by NNRTIs is not clear.  We detd. the crystal structures of RT-DNA-nevirapine, RT-DNA, and RT-DNA-AZT-triphosphate complexes at 2.85-, 2.70-, and 2.80-Å resoln., resp.  The RT-DNA complex in the crystal could bind nevirapine or AZT-triphosphate but not both.  Binding of nevirapine led to opening of the NNRTI-binding pocket.  The pocket formation caused shifting of the 3' end of the DNA primer by ∼5.5 Å away from its polymerase active site position.  Nucleic acid interactions with fingers and palm subdomains were reduced, the dNTP-binding pocket was distorted and the thumb opened up.  The structures elucidate complementary roles of nucleoside and non-nucleoside inhibitors in inhibiting RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxsRKzSWU5S7Vg90H21EOLACvtfcHk0ljkzwh-Qca3pA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGqtrg%253D&md5=8acb0ebef279d7fe999080d12298792c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2223%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHIV-1%2520reverse%2520transcriptase%2520complex%2520with%2520DNA%2520and%2520nevirapine%2520reveals%2520non-nucleoside%2520inhibition%2520mechanism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D253%26epage%3D259%26doi%3D10.1038%2Fnsmb.2223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandwar, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structures of HIV-1 RT-RNA/DNA ternary complexes with dATP and nevirapine reveal conformational flexibility of RNA/DNA: insights into requirements for RNase H cleavage</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">8125</span>– <span class="NLM_lpage">8137</span>, <span class="refDoi"> DOI: 10.1093/nar/gku487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgku487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=24880687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=8125-8137&author=K.+Dasauthor=S.+E.+Martinezauthor=R.+P.+Bandwarauthor=E.+Arnold&title=Structures+of+HIV-1+RT-RNA%2FDNA+ternary+complexes+with+dATP+and+nevirapine+reveal+conformational+flexibility+of+RNA%2FDNA%3A+insights+into+requirements+for+RNase+H+cleavage&doi=10.1093%2Fnar%2Fgku487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of HIV-1 RT-RNA/DNA ternary complexes with dATP and nevirapine reveal conformational flexibility of RNA/DNA: insights into requirements for RNase H cleavage</span></div><div class="casAuthors">Das, Kalyan; Martinez, Sergio E.; Bandwar, Rajiv P.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8125-8137</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In synthesizing a double-stranded DNA from viral RNA, HIV-1 reverse transcriptase (RT) generates an RNA/DNA intermediate.  RT also degrades the RNA strand and synthesizes the second DNA strand.  The RNase H active site of RT functions as a nuclease to cleave the RNA strand; however, the structural basis for endonucleolytic cleavage of the RNA strand remains elusive.  Here we report crystal structures of RT-RNA/DNA-dATP and RT-RNA/DNA-nevirapine (NVP) ternary complexes at 2.5 and 2.9 Å resoln., resp.  The polymerase region of RT-RNA/DNA-dATP complex resembles DNA/DNA ternary complexes apart from addnl. interactions of 2'-OH groups of the RNA strand.  The conformation and binding of RNA/DNA deviates significantly after the seventh nucleotide vs. a DNA/DNA substrate.  Binding of NVP slides the RNA/DNA non-uniformly over RT, and the RNA strand moves closer to the RNase H active site.  Two addnl. structures, one contg. a gapped RNA and another a bulged RNA, reveal that conformational changes of an RNA/DNA and increased interactions with the RNase H domain, including the interaction of a 2'-OH with N474, help to position the RNA nearer to the active site.  The structures and existing biochem. data suggest a nucleic acid conformation-induced mechanism for guiding cleavage of the RNA strand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt_GxtIkYGgLVg90H21EOLACvtfcHk0liGx3MNGv_mMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnO&md5=3f7b7e2daa6511407dc1d292420c275d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku487%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DBandwar%26aufirst%3DR.%2BP.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructures%2520of%2520HIV-1%2520RT-RNA%252FDNA%2520ternary%2520complexes%2520with%2520dATP%2520and%2520nevirapine%2520reveal%2520conformational%2520flexibility%2520of%2520RNA%252FDNA%253A%2520insights%2520into%2520requirements%2520for%2520RNase%2520H%2520cleavage%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8125%26epage%3D8137%26doi%3D10.1093%2Fnar%2Fgku487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandwar, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the role of the K65R mutation in HIV-1 reverse transcriptase polymerization, excision antagonism, and tenofovir resistance</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">35092</span>– <span class="NLM_lpage">35100</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.022525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M109.022525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=19812032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWitbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35092-35100&author=K.+Dasauthor=R.+P.+Bandwarauthor=K.+L.+Whiteauthor=J.+Y.+Fengauthor=S.+G.+Sarafianosauthor=S.+Tuskeauthor=X.+Tuauthor=A.+D.+Clarkauthor=P.+L.+Boyerauthor=X.+Houauthor=B.+L.+Gaffneyauthor=R.+A.+Jonesauthor=M.+D.+Millerauthor=S.+H.+Hughesauthor=E.+Arnold&title=Structural+basis+for+the+role+of+the+K65R+mutation+in+HIV-1+reverse+transcriptase+polymerization%2C+excision+antagonism%2C+and+tenofovir+resistance&doi=10.1074%2Fjbc.m109.022525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Role of the K65R Mutation in HIV-1 Reverse Transcriptase Polymerization, Excision Antagonism, and Tenofovir Resistance</span></div><div class="casAuthors">Das, Kalyan; Bandwar, Rajiv P.; White, Kirsten L.; Feng, Joy Y.; Sarafianos, Stefan G.; Tuske, Steven; Tu, Xiong-Ying; Clark, Arthur D., Jr.; Boyer, Paul L.; Hou, Xiao-Rong; Gaffney, Barbara L.; Jones, Roger A.; Miller, Michael D.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">35092-35100</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">K65R is a primary reverse transcriptase (RT) mutation selected in human immunodeficiency virus type 1-infected patients taking antiretroviral regimens contg. tenofovir disoproxil fumarate or other nucleoside analog RT drugs.  The authors detd. the crystal structures of K65R mutant RT crosslinked to double-stranded DNA and in complexes with tenofovir diphosphate (TFV-DP) or dATP.  The crystals permit substitution of TFV-DP with dATP at the dNTP-binding site.  The guanidinium planes of the arginines K65R and Arg72 were stacked to form a mol. platform that restricts the conformational adaptability of both of the residues, which explains the neg. effects of the K65R mutation on nucleotide incorporation and on excision.  Furthermore, the guanidinium planes of K65R and Arg72 were stacked in two different rotameric conformations in TFV-DP- and dATP-bound structures that may help explain how K65R RT discriminates the drug from substrates.  These K65R-mediated effects on RT structure and function help the authors to visualize the complex interaction with other key nucleotide RT drug resistance mutations, such as M184V, L74V, and thymidine analog resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVMu34Ijj-abVg90H21EOLACvtfcHk0liGx3MNGv_mMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWitbzF&md5=3bda12aca3dbce964898817070c306c1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.022525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.022525%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBandwar%26aufirst%3DR.%2BP.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DTu%26aufirst%3DX.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520for%2520the%2520role%2520of%2520the%2520K65R%2520mutation%2520in%2520HIV-1%2520reverse%2520transcriptase%2520polymerization%252C%2520excision%2520antagonism%252C%2520and%2520tenofovir%2520resistance%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35092%26epage%3D35100%26doi%3D10.1074%2Fjbc.m109.022525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modak, M. J.</span></span> <span> </span><span class="NLM_article-title">Site-directed Mutagenesis of Arginine 72 of HIV-1 Reverse Transcriptase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">19729</span>– <span class="NLM_lpage">19735</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.34.19729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.270.34.19729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=7544345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2MXnvVCmtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=19729-19735&author=S.+G.+Sarafianosauthor=V.+N.+Pandeyauthor=N.+Kaushikauthor=M.+J.+Modak&title=Site-directed+Mutagenesis+of+Arginine+72+of+HIV-1+Reverse+Transcriptase&doi=10.1074%2Fjbc.270.34.19729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed mutagenesis of arginine 72 of HIV-1 reverse transcriptase. Catalytic role and inhibitor sensitivity</span></div><div class="casAuthors">Sarafianos, Stefanos g.; Pandey, Virendra N.; Kaushik, Neerja; Modak, Mukund J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">19729-35</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">To det. the catalytic role of Arg72 of HIV-1 reverse transcriptase (RT), the authors carried out site-directed mutagenesis at codon 72.  Two mutant proteins (R72A and R72K) were purified and characterized.  With Arg to Ala substitution the kcat of the polymerase reaction was reduced by nearly 100-fold with poly(rA) template, but only about 5-15-fold with poly(rC) and poly(dC) templates.  The Arg to Lys substitution exhibited a qual. similar pattern, although the overall redn. in kcat was less severe.  Most interestingly, the authors noted a large difference in the rate const. of the first and second nucleotide incorporation by R72A, suggesting that Arg72 participates in the reaction after the formation of the first phosphodiester bond.  The authors propose this step to be the pyrophosphate binding and removal step following the nucleotidyltransferase reaction.  Support for this proposal is obtained from the observation that the R72A mutant (i) exhibited a pronounced translocation defect in the processivity anal., (ii) lacked the ability to catalyze pyrophosphorolysis, and (iii) showed complete resistance to phosphonoformate, an analog of PPi.  Arg72 is the first residue of HIV-1 RT proposed to be involved in the pyrophosphate binding/removal function of RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGUwF5rYC_-LVg90H21EOLACvtfcHk0liGx3MNGv_mMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvVCmtLg%253D&md5=2ebbd8aa30c11149c7bec11e944c1c8f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.34.19729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.34.19729%26sid%3Dliteratum%253Aachs%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DPandey%26aufirst%3DV.%2BN.%26aulast%3DKaushik%26aufirst%3DN.%26aulast%3DModak%26aufirst%3DM.%2BJ.%26atitle%3DSite-directed%2520Mutagenesis%2520of%2520Arginine%252072%2520of%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D19729%26epage%3D19735%26doi%3D10.1074%2Fjbc.270.34.19729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naeger, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structures of HIV-1 RT-DNA complexes before and after incorporation of the anti-AIDS drug tenofovir</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1038/nsmb760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnsmb760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=15107837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVCqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=469-474&author=S.+Tuskeauthor=S.+G.+Sarafianosauthor=A.+D.+Clarkauthor=J.+Dingauthor=L.+K.+Naegerauthor=K.+L.+Whiteauthor=M.+D.+Millerauthor=C.+S.+Gibbsauthor=P.+L.+Boyerauthor=P.+Clarkauthor=G.+Wangauthor=B.+L.+Gaffneyauthor=R.+A.+Jonesauthor=D.+M.+Jerinaauthor=S.+H.+Hughesauthor=E.+Arnold&title=Structures+of+HIV-1+RT-DNA+complexes+before+and+after+incorporation+of+the+anti-AIDS+drug+tenofovir&doi=10.1038%2Fnsmb760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of HIV-1 RT-DNA complexes before and after incorporation of the anti-AIDS drug tenofovir</span></div><div class="casAuthors">Tuske, Steve; Sarafianos, Stefan G.; Clark, Arthur D.; Ding, Jianping; Naeger, Lisa K.; White, Kirsten L.; Miller, Michael D.; Gibbs, Craig S.; Boyer, Paul L.; Clark, Patrick; Wang, Gang; Gaffney, Barbara L.; Jones, Roger A.; Jerina, Donald M.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">469-474</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tenofovir, also known as PMPA, R-9-2-(phosphonomethoxypropyl)adenine, is a nucleotide reverse transcriptase (RT) inhibitor.  We have detd. the crystal structures of two related complexes of HIV-1 RT with template primer and tenofovir: (i) a ternary complex at a resoln. of 3.0 Å of RT crosslinked to a dideoxy-terminated DNA with tenofovir-diphosphate bound as the incoming substrate; and (ii) a RT-DNA complex at a resoln. of 3.1 Å with tenofovir at the 3' primer terminus.  The tenofovir nucleotide in the tenofovir-terminated structure seems to adopt multiple conformations.  Some nucleoside reverse transcriptase inhibitors, including 3TC and AZT, have elements ('handles') that project beyond the corresponding elements on normal dNTPs (the 'substrate envelope').  HIV-1 RT resistance mechanisms to AZT and 3TC take advantage of these handles; tenofovir's structure lacks handles that could protrude through the substrate envelope to cause resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi1ei68A2R6bVg90H21EOLACvtfcHk0lifY0Cu1tGgyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVCqsL8%253D&md5=da428445f0276f07fd66dbde46732fbf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnsmb760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb760%26sid%3Dliteratum%253Aachs%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DNaeger%26aufirst%3DL.%2BK.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DGibbs%26aufirst%3DC.%2BS.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DClark%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DJerina%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructures%2520of%2520HIV-1%2520RT-DNA%2520complexes%2520before%2520and%2520after%2520incorporation%2520of%2520the%2520anti-AIDS%2520drug%2520tenofovir%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D469%26epage%3D474%26doi%3D10.1038%2Fnsmb760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1499</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1126/science.abc1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1126%2Fscience.abc1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=32358203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1499-1504&author=W.+Yinauthor=C.+Maoauthor=X.+Luanauthor=D.-D.+Shenauthor=Q.+Shenauthor=H.+Suauthor=X.+Wangauthor=F.+Zhouauthor=W.+Zhaoauthor=M.+Gaoauthor=S.+Changauthor=Y.-C.+Xieauthor=G.+Tianauthor=H.-W.+Jiangauthor=S.-C.+Taoauthor=J.+Shenauthor=Y.+Jiangauthor=H.+Jiangauthor=Y.+Xuauthor=S.+Zhangauthor=Y.+Zhangauthor=H.+E.+Xu&title=Structural+basis+for+inhibition+of+the+RNA-dependent+RNA+polymerase+from+SARS-CoV-2+by+remdesivir&doi=10.1126%2Fscience.abc1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span></div><div class="casAuthors">Yin, Wanchao; Mao, Chunyou; Luan, Xiaodong; Shen, Dan-Dan; Shen, Qingya; Su, Haixia; Wang, Xiaoxi; Zhou, Fulai; Zhao, Wenfeng; Gao, Minqi; Chang, Shenghai; Xie, Yuan-Chao; Tian, Guanghui; Jiang, He-Wei; Tao, Sheng-Ce; Shen, Jingshan; Jiang, Yi; Jiang, Hualiang; Xu, Yechun; Zhang, Shuyang; Zhang, Yan; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">1499-1504</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has become a global crisis.  Replication of SARS-CoV-2 requires the viral RNA-dependent RNA polymerase (RdRp) enzyme, a target of the antiviral drug remdesivir.  Here, we report the cryo-electron microscopy structure of the SARS-CoV-2 RdRp, both in the apo form at 2.8-angstrom resoln. and in complex with a 50-base template-primer RNA and remdesivir at 2.5-angstrom resoln.  The complex structure reveals that the partial double-stranded RNA template is inserted into the central channel of the RdRp, where remdesivir is covalently incorporated into the primer strand at the first replicated base pair, and terminates chain elongation.  Our structures provide insights into the mechanism of viral RNA replication and a rational template for drug design to combat the viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlizem_f4e1LVg90H21EOLACvtfcHk0lifY0Cu1tGgyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK&md5=e5d7c8a1c27a02fdbfa9381482a0c3e8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc1560%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLuan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DD.-D.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.-C.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.-W.%26aulast%3DTao%26aufirst%3DS.-C.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520RNA-dependent%2520RNA%2520polymerase%2520from%2520SARS-CoV-2%2520by%2520remdesivir%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1499%26epage%3D1504%26doi%3D10.1126%2Fscience.abc1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hillen, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokic, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dienemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegunov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, P.</span></span> <span> </span><span class="NLM_article-title">Structure of replicating SARS-CoV-2 polymerase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2368-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fs41586-020-2368-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=32438371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=154-156&author=H.+S.+Hillenauthor=G.+Kokicauthor=L.+Farnungauthor=C.+Dienemannauthor=D.+Tegunovauthor=P.+Cramer&title=Structure+of+replicating+SARS-CoV-2+polymerase&doi=10.1038%2Fs41586-020-2368-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of replicating SARS-CoV-2 polymerase</span></div><div class="casAuthors">Hillen, Hauke S.; Kokic, Goran; Farnung, Lucas; Dienemann, Christian; Tegunov, Dimitry; Cramer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7819</span>),
    <span class="NLM_cas:pages">154-156</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The new coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) uses an RNA-dependent RNA polymerase (RdRp) for the replication of its genome and the transcription of its genes1-3.  Here we present a cryo-electron microscopy structure of the SARS-CoV-2 RdRp in an active form that mimics the replicating enzyme.  The structure comprises the viral proteins non-structural protein 12 (nsp12), nsp8 and nsp7, and more than two turns of RNA template-product duplex.  The active-site cleft of nsp12 binds to the first turn of RNA and mediates RdRp activity with conserved residues.  Two copies of nsp8 bind to opposite sides of the cleft and position the second turn of RNA.  Long helical extensions in nsp8 protrude along exiting RNA, forming pos. charged 'sliding poles'.  These sliding poles can account for the known processivity of RdRp that is required for replicating the long genome of coronaviruses3.  Our results enable a detailed anal. of the inhibitory mechanisms that underlie the antiviral activity of substances such as remdesivir, a drug for the treatment of coronavirus disease 2019 (COVID-19)4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvvV9anYxFpbVg90H21EOLACvtfcHk0lglwLglhv46ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLjN&md5=a6c1595799a31219d6a54228f5df82c9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2368-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2368-8%26sid%3Dliteratum%253Aachs%26aulast%3DHillen%26aufirst%3DH.%2BS.%26aulast%3DKokic%26aufirst%3DG.%26aulast%3DFarnung%26aufirst%3DL.%26aulast%3DDienemann%26aufirst%3DC.%26aulast%3DTegunov%26aufirst%3DD.%26aulast%3DCramer%26aufirst%3DP.%26atitle%3DStructure%2520of%2520replicating%2520SARS-CoV-2%2520polymerase%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D154%26epage%3D156%26doi%3D10.1038%2Fs41586-020-2368-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielssens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyubankova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceulemans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Influence of the nucleobase and anchimeric assistance of the carboxyl acid groups in the hydrolysis of amino acid nucleoside phosphoramidates</span>. <i>Chemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1002/chem.201102279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fchem.201102279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22173724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1elur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=857-868&author=M.+Maitiauthor=S.+Michielssensauthor=N.+Dyubankovaauthor=M.+Maitiauthor=E.+Lescrinierauthor=A.+Ceulemansauthor=P.+Herdewijn&title=Influence+of+the+nucleobase+and+anchimeric+assistance+of+the+carboxyl+acid+groups+in+the+hydrolysis+of+amino+acid+nucleoside+phosphoramidates&doi=10.1002%2Fchem.201102279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Influence of the Nucleobase and Anchimeric Assistance of the Carboxyl Acid Groups in the Hydrolysis of Amino Acid Nucleoside Phosphoramidates</span></div><div class="casAuthors">Maiti, Munmun; Michielssens, Servaas; Dyubankova, Natalia; Maiti, Mohitosh; Lescrinier, Eveline; Ceulemans, Arnout; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">857-868, S857/1-S857/35</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Nucleoside phosphoramidates (NPs) are a class of nucleotide analogs that has been developed as potential antiviral/antitumor prodrugs.  Recently, we have shown that some amino acid nucleoside phosphoramidates (aaNPs) can act as substrates for viral polymerases like HIV-1 RT.  Herein, we report the synthesis and hydrolysis of a series of new aaNPs, contg. either natural or modified nucleobases to define the basis for their differential reactivity.  Aq. stability, kinetics, and hydrolysis pathways were studied by NMR spectroscopy at different soln. pD values (5-7) and temps.  It was obsd. that the kinetics and mechanism (P-N and/or P-O bond cleavage) of the hydrolysis reaction largely depend on the nature of the nucleobase and amino acid moieties.  Aspartyl NPs were found to be more reactive than Gly or β-Ala NPs.  For aspartyl NPs, the order of reactivity of the nucleobase was 1-deazaadenine>7-deazaadenine>adenine>thymine≥3-deazaadenine.  Notably, neutral aq. solns. of Asp-1-deaza-dAMP degraded spontaneously even at 4 °C through exclusive P-O bond hydrolysis (a 50-fold reactivity difference for Asp-1-deaza-dAMP vs. Asp-3-deaza-dAMP at pD 5 and 70 °C).  Conformational studies by NMR spectroscopy and mol. modeling suggest the involvement of the protonated N3 atom in adenine and 1- and 7-deazaadenine in the intramol. catalysis of the hydrolysis reaction through the rare syn conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaANmdiC86rVg90H21EOLACvtfcHk0lglwLglhv46ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1elur7J&md5=097824e68615f4ba50cb55c6f24c56df</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fchem.201102279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201102279%26sid%3Dliteratum%253Aachs%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DMichielssens%26aufirst%3DS.%26aulast%3DDyubankova%26aufirst%3DN.%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DCeulemans%26aufirst%3DA.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DInfluence%2520of%2520the%2520nucleobase%2520and%2520anchimeric%2520assistance%2520of%2520the%2520carboxyl%2520acid%2520groups%2520in%2520the%2520hydrolysis%2520of%2520amino%2520acid%2520nucleoside%2520phosphoramidates%26jtitle%3DChemistry%26date%3D2012%26volume%3D18%26spage%3D857%26epage%3D868%26doi%3D10.1002%2Fchem.201102279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baweja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shatkin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Crystal engineering of HIV-1 reverse transcriptase for structure-based drug design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5092</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkn464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=18676450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGkt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=5083-5092&author=J.+D.+Baumanauthor=K.+Dasauthor=W.+C.+Hoauthor=M.+Bawejaauthor=D.+M.+Himmelauthor=A.+D.+Clarkauthor=D.+A.+Orenauthor=P.+L.+Boyerauthor=S.+H.+Hughesauthor=A.+J.+Shatkinauthor=E.+Arnold&title=Crystal+engineering+of+HIV-1+reverse+transcriptase+for+structure-based+drug+design&doi=10.1093%2Fnar%2Fgkn464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal engineering of HIV-1 reverse transcriptase for structure-based drug design</span></div><div class="casAuthors">Bauman, Joseph D.; Das, Kalyan; Ho, William C.; Baweja, Mukta; Himmel, Daniel M.; Clark, Arthur D., Jr.; Oren, Deena A.; Boyer, Paul L.; Hughes, Stephen H.; Shatkin, Aaron J.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5083-5092</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) is a primary target for anti-AIDS drugs.  Structures of HIV-1 RT, usually detd. at ∼2.5-3.0 Å resoln., are important for understanding enzyme function and mechanisms of drug resistance in addn. to being helpful in the design of RT inhibitors.  Despite hundreds of attempts, it was not possible to obtain the structure of a complex of HIV-1 RT with TMC278, a nonnucleoside RT inhibitor (NNRTI) in advanced clin. trials.  A systematic and iterative protein crystal engineering approach was developed to optimize RT for obtaining crystals in complexes with TMC278 and other NNRTIs that diffract X-rays to 1.8 Å resoln.  Another form of engineered RT was optimized to produce a high-resoln. apo-RT crystal form, reported here at 1.85 Å resoln., with a distinct RT conformation.  Engineered RTs were mutagenized using a new, flexible and cost effective method called methylated overlap-extension ligation independent cloning.  Our anal. suggests that reducing the solvent content, increasing lattice contacts, and stabilizing the internal low-energy conformations of RT are crit. for the growth of crystals that diffract to high resoln.  The new RTs enable rapid crystn. and yield high-resoln. structures that are useful in designing/developing new anti-AIDS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8sHD_iC_LsLVg90H21EOLACvtfcHk0lglwLglhv46ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGkt7%252FI&md5=c7957e67c3a017b289e03de94a78cbbd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn464%26sid%3Dliteratum%253Aachs%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DHo%26aufirst%3DW.%2BC.%26aulast%3DBaweja%26aufirst%3DM.%26aulast%3DHimmel%26aufirst%3DD.%2BM.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DOren%26aufirst%3DD.%2BA.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DCrystal%2520engineering%2520of%2520HIV-1%2520reverse%2520transcriptase%2520for%2520structure-based%2520drug%2520design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D5083%26epage%3D5092%26doi%3D10.1093%2Fnar%2Fgkn464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seehra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bambara, R. A.</span></span> <span> </span><span class="NLM_article-title">Characterization of an RNase H deficient mutant of human immunodeficiency virus-1 reverse transcriptase having an aspartate to asparagine change at position 498</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>1219</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1016/0167-4781(94)90062-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2F0167-4781%2894%2990062-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=7522572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2MXhsVSksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1219&publication_year=1994&pages=380-388&author=J.+J.+DeStefanoauthor=W.+Wuauthor=J.+Seehraauthor=J.+McCoyauthor=D.+Lastonauthor=E.+Alboneauthor=P.+J.+Fayauthor=R.+A.+Bambara&title=Characterization+of+an+RNase+H+deficient+mutant+of+human+immunodeficiency+virus-1+reverse+transcriptase+having+an+aspartate+to+asparagine+change+at+position+498&doi=10.1016%2F0167-4781%2894%2990062-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of an RNase H deficient mutant of human immunodeficiency virus-1 reverse transcriptase having an aspartate to asparagine change at position 498</span></div><div class="casAuthors">DeStefano, Jeffrey J.; Wu, Weimin; Seehra, Jasbir; McCoy, John; Laston, David; Albone, Earl; Fay, Philip J.; Bambara, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Structure and Expression</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">1219</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">380-8</span>CODEN:
                <span class="NLM_cas:coden">BBGSD5</span>;
        ISSN:<span class="NLM_cas:issn">0167-4781</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">It has been proposed that Asp-443, Glu-478, and Asp-498 are important in RNase H mediated catalysis by human immunodeficiency virus-1 reverse transcriptase (Davies J.F., Hostomska, Z. Hostomsky, Z., Jordan, S.R. and Matthews, D.A. (1991) Science 251, 88-95; Mizrahi, V., Usdin, M.T., Harington, A. and Dudding, L.R. (1990) Nucleic Acids Res. 18, 5359-5363).  Single point mutations at either position 443 (Mizrahi, V., Usdin, M.T., Harington, A. and Dudding, L.R. (1990) Nucleic Acids Res. 18, 5359-5363) or 478 (Schatz, O., Cromme, F.V., Greuninger-Leitch, F. and Le Grice, S.F.J. (1989) FEBS Lett. 257, 311-314) severely inhibit RNase H activity but have only small effects on polymerase activity.  The authors show here that a single mutation at position 498 of Asp to Asn (mutant D498N) results in a stable enzyme with a 20-fold redn. in the ratio of RNase H to polymerase activity.  The mutant and wild type enzymes were equally processive, paused in the same locations, and extended primers at the same rate during DNA synthesis on a heteropolymeric RNA template.  The rate of elongation on the homopolymeric template poly(rA) was also the same.  The results indicate that the mutation does not affect normally measured catalytic parameters of the polymerase function of the enzyme.  D498N catalyzed strand transfer synthesis on homopolymeric, but not heteropolymeric templates, indicating that RNase H activity is not required for the former activity, but is for the latter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvnmPl3U2jbVg90H21EOLACvtfcHk0liYyR94F1TNIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhsVSksr4%253D&md5=bdd9b7a2287839ee73c9f4c459b7ec69</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0167-4781%2894%2990062-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0167-4781%252894%252990062-0%26sid%3Dliteratum%253Aachs%26aulast%3DDeStefano%26aufirst%3DJ.%2BJ.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DSeehra%26aufirst%3DJ.%26aulast%3DMcCoy%26aufirst%3DJ.%26aulast%3DLaston%26aufirst%3DD.%26aulast%3DAlbone%26aufirst%3DE.%26aulast%3DFay%26aufirst%3DP.%2BJ.%26aulast%3DBambara%26aufirst%3DR.%2BA.%26atitle%3DCharacterization%2520of%2520an%2520RNase%2520H%2520deficient%2520mutant%2520of%2520human%2520immunodeficiency%2520virus-1%2520reverse%2520transcriptase%2520having%2520an%2520aspartate%2520to%2520asparagine%2520change%2520at%2520position%2520498%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1994%26volume%3D1219%26spage%3D380%26epage%3D388%26doi%3D10.1016%2F0167-4781%2894%2990062-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of guanosine and deoxyguanosine phosphoramidites with cross-linkable thioalkyl tethers for direct incorporation into RNA and DNA</span>. <i>Nucleos Nucleot. Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1076</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1080/15257770903368385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1080%2F15257770903368385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20183575" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1076-1094&author=X.+Houauthor=G.+Wangauthor=B.+L.+Gaffneyauthor=R.+A.+Jones&title=Synthesis+of+guanosine+and+deoxyguanosine+phosphoramidites+with+cross-linkable+thioalkyl+tethers+for+direct+incorporation+into+RNA+and+DNA&doi=10.1080%2F15257770903368385"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1080%2F15257770903368385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15257770903368385%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26atitle%3DSynthesis%2520of%2520guanosine%2520and%2520deoxyguanosine%2520phosphoramidites%2520with%2520cross-linkable%2520thioalkyl%2520tethers%2520for%2520direct%2520incorporation%2520into%2520RNA%2520and%2520DNA%26jtitle%3DNucleos%2520Nucleot.%2520Nucleic%2520Acids%26date%3D2009%26volume%3D28%26spage%3D1076%26epage%3D1094%26doi%3D10.1080%2F15257770903368385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilankumaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroth, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Trapping HIV-1 reverse transcriptase before and after translocation on DNA</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">16280</span>– <span class="NLM_lpage">16288</span>, <span class="refDoi"> DOI: 10.1074/jbc.m212911200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M212911200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=12554739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt1ehtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=16280-16288&author=S.+G.+Sarafianosauthor=A.+D.+Clarkauthor=S.+Tuskeauthor=C.+J.+Squireauthor=K.+Dasauthor=D.+Shengauthor=P.+Ilankumaranauthor=A.+R.+Rameshaauthor=H.+Krothauthor=J.+M.+Sayerauthor=D.+M.+Jerinaauthor=P.+L.+Boyerauthor=S.+H.+Hughesauthor=E.+Arnold&title=Trapping+HIV-1+reverse+transcriptase+before+and+after+translocation+on+DNA&doi=10.1074%2Fjbc.m212911200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping HIV-1 Reverse Transcriptase Before and After Translocation on DNA</span></div><div class="casAuthors">Sarafianos, Stefan G.; Clark, Arthur D., Jr.; Tuske, Steve; Squire, Christopher J.; Das, Kalyan; Sheng, Dequan; Ilankumaran, Palanichamy; Ramesha, Andagar R.; Kroth, Heiko; Sayer, Jane M.; Jerina, Donald M.; Boyer, Paul L.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">16280-16288</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A disulfide crosslinking strategy was used to covalently trap as a stable complex (complex N) a short-lived, kinetic intermediate in DNA polymn.  This intermediate corresponds to the product of polymn. prior to translocation.  We also prepd. the trapped complex that corresponds to the product of polymn. after translocation (complex P).  The crosslinking method that we used is a variation of a technique developed by the Verdine and Harrison labs.  It involves disulfide interchange between an engineered sulfhydryl group of the protein (Q258C mutation) and a disulfide-contg. tether attached at the N2 amino group of a modified dG in either the template or the primer strand of the nucleic acid.  We report here a highly efficient synthesis of the precursor, bis(3-aminopropyl)disulfide dihydrochloride, used to introduce this substituent into the oligonucleotide.  Efficient crosslinking takes place when the base pair contg. the substituent is positioned seven registers from the dNTP-binding site (N site) and the N site is occupied.  Complex N, but not complex P, is a substrate for the ATP-based excision reaction that unblocks nucleoside reverse transcriptase inhibitor (NRTI)-terminated primers and causes resistance to several NRTIs, confirming predictions that the excision reaction takes place only when the 3'-end of the primer is bound at the N site.  These techniques can be used for biochem. and structural studies of the mechanism of DNA polymn., translocation, and excision-based resistance of RT to NRTIs.  They may also be useful in studying other DNA or RNA polymerases or other enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7jRh1ft9lT7Vg90H21EOLACvtfcHk0liYyR94F1TNIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt1ehtLk%253D&md5=8fe58e8d0b1c383fe2552880bb2925ae</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M212911200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M212911200%26sid%3Dliteratum%253Aachs%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DSquire%26aufirst%3DC.%2BJ.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DSheng%26aufirst%3DD.%26aulast%3DIlankumaran%26aufirst%3DP.%26aulast%3DRamesha%26aufirst%3DA.%2BR.%26aulast%3DKroth%26aufirst%3DH.%26aulast%3DSayer%26aufirst%3DJ.%2BM.%26aulast%3DJerina%26aufirst%3DD.%2BM.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DTrapping%2520HIV-1%2520reverse%2520transcriptase%2520before%2520and%2520after%2520translocation%2520on%2520DNA%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D16280%26epage%3D16288%26doi%3D10.1074%2Fjbc.m212911200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">Integrating macromolecular X-ray diffraction data with the graphical user interface iMosflm</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1310</span>– <span class="NLM_lpage">1325</span>, <span class="refDoi"> DOI: 10.1038/nprot.2017.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnprot.2017.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=28569763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovFCqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1310-1325&author=H.+R.+Powellauthor=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=A.+G.+W.+Leslie&title=Integrating+macromolecular+X-ray+diffraction+data+with+the+graphical+user+interface+iMosflm&doi=10.1038%2Fnprot.2017.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating macromolecular X-ray diffraction data with the graphical user interface iMosflm</span></div><div class="casAuthors">Powell, Harold R.; Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1310-1325</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">X-ray crystallog. is the predominant source of structural information for biol. macromols., providing fundamental insights into biol. function.  The availability of robust and user-friendly software to process the collected X-ray diffraction images makes the technique accessible to a wider range of scientists. iMosflm/MOSFLM (http://www.mrc-lmb.cam.ac.uk/harry/imosflm) is a software package designed to achieve this goal.  The graphical user interface (GUI) version of MOSFLM (called iMosflm) is designed to guide inexperienced users through the steps of data integration, while retaining powerful features for more experienced users.  Images from almost all com. available X-ray detectors can be handled using this software.  Although the program uses only 2D profile fitting, it can readily integrate data collected in the 'fine phi-slicing' mode (in which the rotation angle per image is less than the crystal mosaic spread by a factor of at least 2), which is commonly used with modern very fast readout detectors.  The GUI provides real-time feedback on the success of the indexing step and the progress of data processing.  This feedback includes the ability to monitor detector and crystal parameter refinement and to display the av. spot shape in different regions of the detector.  Data scaling and merging tasks can be initiated directly from the interface.  Using this protocol, a data set of 360 images with ∼2,000 reflections per image can be processed in ∼4 min.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUd2z_blS8nrVg90H21EOLACvtfcHk0liYyR94F1TNIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovFCqu7w%253D&md5=eb357d1354d93138e2100e0693ce0e0f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2017.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2017.037%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DIntegrating%2520macromolecular%2520X-ray%2520diffraction%2520data%2520with%2520the%2520graphical%2520user%2520interface%2520iMosflm%26jtitle%3DNat.%2520Protoc.%26date%3D2017%26volume%3D12%26spage%3D1310%26epage%3D1325%26doi%3D10.1038%2Fnprot.2017.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgYzuKJhnfk7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgYzuKJhnfk7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgYzuKJhnfk7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhVsSPFXuZMDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhVsSPFXuZMDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span> <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%2C+LLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michailidis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzek, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norden, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parniak, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span> <span> </span><span class="NLM_article-title">Biochemical mechanism of HIV-1 resistance to rilpivirine</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">38110</span>– <span class="NLM_lpage">38123</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.398180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M112.398180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22955279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WjtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=38110-38123&author=K.+Singhauthor=B.+Marchandauthor=D.+K.+Raiauthor=B.+Sharmaauthor=E.+Michailidisauthor=E.+M.+Ryanauthor=K.+B.+Matzekauthor=M.+D.+Leslieauthor=A.+N.+Hagedornauthor=Z.+Liauthor=P.+R.+Nordenauthor=A.+Hachiyaauthor=M.+A.+Parniakauthor=H.-T.+Xuauthor=M.+A.+Wainbergauthor=S.+G.+Sarafianos&title=Biochemical+mechanism+of+HIV-1+resistance+to+rilpivirine&doi=10.1074%2Fjbc.m112.398180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical Mechanism of HIV-1 Resistance to Rilpivirine</span></div><div class="casAuthors">Singh, Kamalendra; Marchand, Bruno; Rai, Devendra K.; Sharma, Bechan; Michailidis, Eleftherios; Ryan, Emily M.; Matzek, Kayla B.; Leslie, Maxwell D.; Hagedorn, Ariel N.; Li, Zhe; Norden, Pieter R.; Hachiya, Atsuko; Parniak, Michael A.; Xu, Hong-Tao; Wainberg, Mark A.; Sarafianos, Stefan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">38110-38123</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Rilpivirine (RPV) is a second generation nonnucleoside reverse transcriptase (RT) inhibitor (NNRTI) that efficiently inhibits HIV-1 resistant to first generation NNRTIs.  Virol. failure during therapy with RPV and emtricitabine is assocd. with the appearance of E138K and M184I mutations in RT.  Here we investigate the biochem. mechanism of RT inhibition and resistance to RPV.  We used two transient kinetics approaches (quench-flow and stopped-flow) to det. how subunit-specific mutations in RT p66 or p51 affect assocn. and dissocn. of RPV to RT as well as their impact on binding of dNTP and DNA and the catalytic incorporation of nucleotide.  We compared WT with four subunit-specific RT mutants, p66M184I/p51WT, p66E138K/p51E138K, p66E138K/M184I/p51E138K, and p66M184I/p51E138K.  Ile-184 in p66 (p66184I) decreased the catalytic efficiency of RT (kpol/Kd.dNTP), primarily through a decrease in dNTP binding (Kd.dNTP).  Lys-138 either in both subunits or in p51 alone abrogated the neg. effect of p66184I by restoring dNTP binding.  Furthermore, p51138K reduced RPV susceptibility by altering the ratio of RPV dissocn. to RPV assocn., resulting in a net redn. in RPV equil. binding affinity (Kd.RPV = koff.RPV/kon.RPV).  Quantum mechanics/mol. mechanics hybrid mol. modeling revealed that p51E138K affects access to the RPV binding site by disrupting the salt bridge between p51E138 and p66K101. p66184I caused repositioning of the Tyr-183 active site residue and decreased the efficiency of RT, whereas the addn. of p51138K restored Tyr-183 to a WT-like conformation, thus abrogating the Ile-184-induced functional defects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_d23tgjqi-7Vg90H21EOLACvtfcHk0lhzGtTW0ZSC8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WjtLfJ&md5=2f7ed4760eaa646e49d181e75d16ee94</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.398180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.398180%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DB.%26aulast%3DRai%26aufirst%3DD.%2BK.%26aulast%3DSharma%26aufirst%3DB.%26aulast%3DMichailidis%26aufirst%3DE.%26aulast%3DRyan%26aufirst%3DE.%2BM.%26aulast%3DMatzek%26aufirst%3DK.%2BB.%26aulast%3DLeslie%26aufirst%3DM.%2BD.%26aulast%3DHagedorn%26aufirst%3DA.%2BN.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DNorden%26aufirst%3DP.%2BR.%26aulast%3DHachiya%26aufirst%3DA.%26aulast%3DParniak%26aufirst%3DM.%2BA.%26aulast%3DXu%26aufirst%3DH.-T.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26atitle%3DBiochemical%2520mechanism%2520of%2520HIV-1%2520resistance%2520to%2520rilpivirine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D38110%26epage%3D38123%26doi%3D10.1074%2Fjbc.m112.398180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Nucleos(t)ide analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Metal chelation at the active site of HIV-1 RT. (A) Octahedral coordination of Mg<sup>2+</sup> ions in the HIV-1 RT/DNA catalytic complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PQU">4PQU</a>); the primer strand was terminated with dideoxyguanosine to prevent incorporation. (B) Schematic representation of Mg<sup>2+</sup> ion chelation in the polymerase active site of HIV-1 RT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and Conditions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) amino acid ester, DCC, <i>t</i>-BuOH, H<sub>2</sub>O, reflux, 3 h; (ii) 0.4 M NaOH, MeOH, rt, 3 h.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Primer extension and polymerase inhibition assays. (A–C) Primer extension with amino acid conjugates of 2′-dAMP and TFV by HIV-1 RT. Reaction conditions: 125 nM primer/template; [dATP] = 50 μM; [amino acid conjugate] = 500 μM; [HIV-1 RT] = 0.5 μg/μL, 3 mM MgCl<sub>2</sub>. (D,E) With 2 mM MnCl<sub>2</sub> added. In (E), [amino acid conjugate] = 1 mM. (F) Dose–response curve showing inhibition of HIV-1 RT in the presence of Mg<sup>2+</sup> ions. The inhibitory effect of compound <b>4e</b> on dsDNA synthesis by recombinant HIV-1 RT was determined by PicoGreen-based spectrophotometric assay. Relative levels of dsDNA synthesis were calculated as a percentage of levels in reaction in the absence of the compound. Data points represent the means of representative experiments run in triplicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) 2Fo-Fc electron density map at 2.95 Å resolution defines the binding of <b>4d</b> at the active site of the RT/DNA/<b>4d</b> complex in the presence of MgCl<sub>2</sub>; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG">7AIG</a>. No Mg<sup>2+</sup> ion is present at the polymerase active site. (B) 2Fo-Fc density map at 2.75 Å resolution shows the binding of <b>4d</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF">7AIF</a>. (C) <b>4d</b> binds at the dNTP-binding pocket. Template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxy-guanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4d</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) top view showing the binding of <b>4d</b> to the RT/DNA complex either in the presence of Mg<sup>2+</sup> (orange bonds) or added Mn<sup>2+</sup> ions (green bonds). The Mn<sup>2+</sup> binds at the site B. The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site and repositions the Glu side-chain. Structures were aligned in the palm domain (chain C) using the alpha carbons of residues 85–119, 151–242, and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) 2Fo-Fc electron density map at 3.15 Å resolution showing the binding of <b>4b</b> at the active site of the RT/DNA/<b>4b</b> complex in the presence of Mg<sup>2+</sup>; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID">7AID</a>. (B) 2Fo-Fc density map at 2.73 Å resolution showing the binding of <b>4b</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX">7AHX</a>. (C) <b>4b</b> sits in the dNTP binding site in the presence of Mg<sup>2+</sup>; no Mg<sup>2+</sup>ion is present in the polymerase active site. The template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxyguanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4b</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) Top view of <b>4b</b> in complex with RT in the presence of either Mg<sup>2+</sup> (orange bonds) or Mg<sup>2+</sup> + Mn<sup>2+</sup> (green bonds). Mn<sup>2+</sup> binds at the metal ion site B. The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site, and repositions the <span class="smallcaps smallerCapital">d</span>-Asp side-chain. Structures were aligned on the palm domain (chain C) using the alpha carbons of residues 85–119 and 151–242 and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/medium/jm0c01747_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) 2Fo-Fc electron density map at 3.15 Å resolution defines the position of <b>4e</b> in the active site of the RT/DNA/4e complex in the presence of Mg<sup>2+</sup> ions; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ">7AIJ</a>. (B) 2Fo-Fc density map at 2.73 Å resolution defines the binding of <b>4e</b> in the presence of Mn<sup>2+</sup> (pink sphere); PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII">7AII</a>. (C) <b>4e</b> sits in the dNTP-binding site in the presence of MgCl<sub>2</sub>; no Mg<sup>2+</sup> ion is present at the active site. The template strand is magenta, and the primer 3′ terminus is cyan. The primer strand is terminated with dideoxyguanosine monophosphate. (D) Mn<sup>2+</sup> metal ion at site B chelates a phosphonoamidate oxygen of <b>4e</b>, catalytic residues Asp110 and Asp185, and the backbone carbonyl of V111. (E) Side view and (F) Top view of <b>4e</b> in complex with RT in the presence of either Mg<sup>2+</sup> (orange bonds) or Mg<sup>2+</sup>+ Mn<sup>2+</sup> (green bonds). The chelation with Mn<sup>2+</sup> causes better stacking of the adenine base with the DNA, brings the phosphonoamidate group closer toward the active site, and repositions the Met side-chain. Structures were aligned in the palm domain (chain C) using the alpha carbons of residues 85–119 and 151–242 and the CCP4 program SUPERPOSE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-1/acs.jmedchem.0c01747/20210107/images/large/jm0c01747_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01747&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86825" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86825" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span> <span> </span><span class="NLM_article-title">Approved antiviral drugs over the past 50 years</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1128/cmr.00102-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1128%2FCMR.00102-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=27281742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252Fos12htA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=695-747&author=E.+De%0AClercqauthor=G.+Li&title=Approved+antiviral+drugs+over+the+past+50+years&doi=10.1128%2Fcmr.00102-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Approved Antiviral Drugs over the Past 50 Years</span></div><div class="casAuthors">De Clercq Erik; Li Guangdi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical microbiology reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">695-747</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the first antiviral drug, idoxuridine, was approved in 1963, 90 antiviral drugs categorized into 13 functional groups have been formally approved for the treatment of the following 9 human infectious diseases: (i) HIV infections (protease inhibitors, integrase inhibitors, entry inhibitors, nucleoside reverse transcriptase inhibitors, nonnucleoside reverse transcriptase inhibitors, and acyclic nucleoside phosphonate analogues), (ii) hepatitis B virus (HBV) infections (lamivudine, interferons, nucleoside analogues, and acyclic nucleoside phosphonate analogues), (iii) hepatitis C virus (HCV) infections (ribavirin, interferons, NS3/4A protease inhibitors, NS5A inhibitors, and NS5B polymerase inhibitors), (iv) herpesvirus infections (5-substituted 2'-deoxyuridine analogues, entry inhibitors, nucleoside analogues, pyrophosphate analogues, and acyclic guanosine analogues), (v) influenza virus infections (ribavirin, matrix 2 protein inhibitors, RNA polymerase inhibitors, and neuraminidase inhibitors), (vi) human cytomegalovirus infections (acyclic guanosine analogues, acyclic nucleoside phosphonate analogues, pyrophosphate analogues, and oligonucleotides), (vii) varicella-zoster virus infections (acyclic guanosine analogues, nucleoside analogues, 5-substituted 2'-deoxyuridine analogues, and antibodies), (viii) respiratory syncytial virus infections (ribavirin and antibodies), and (ix) external anogenital warts caused by human papillomavirus infections (imiquimod, sinecatechins, and podofilox).  Here, we present for the first time a comprehensive overview of antiviral drugs approved over the past 50 years, shedding light on the development of effective antiviral treatments against current and emerging infectious diseases worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSbPpNpCY_qYTEC-75aqa5AfW6udTcc2eb4iauIYyRy1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252Fos12htA%253D%253D&md5=1d25d961c1c6a72d6bc4634041bbb6fe</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1128%2FCMR.00102-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00102-15%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DG.%26atitle%3DApproved%2520antiviral%2520drugs%2520over%2520the%2520past%252050%2520years%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2016%26volume%3D29%26spage%3D695%26epage%3D747%26doi%3D10.1128%2Fcmr.00102-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gollnest, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Oliveira, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Lipophilic prodrugs of nucleoside triphosphates as biochemical probes and potential antivirals</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8716</span>, <span class="refDoi"> DOI: 10.1038/ncomms9716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fncomms9716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=26503889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8716&author=T.+Gollnestauthor=T.+D.+de+Oliveiraauthor=D.+Scholsauthor=J.+Balzariniauthor=C.+Meier&title=Lipophilic+prodrugs+of+nucleoside+triphosphates+as+biochemical+probes+and+potential+antivirals&doi=10.1038%2Fncomms9716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic prodrugs of nucleoside triphosphates as biochemical probes and potential antivirals</span></div><div class="casAuthors">Gollnest, Tristan; Dinis de Oliveira, Thiago; Schols, Dominique; Balzarini, Jan; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8716</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The antiviral activity of nucleoside reverse transcriptase inhibitors is often limited by ineffective phosphorylation.  We report on a nucleoside triphosphate (NTP) prodrug approach in which the γ-phosphate of NTPs is bioreversibly modified.  A series of TriPPPro-compds. bearing two lipophilic masking units at the γ-phosphate and d4T as a nucleoside analog are synthesized.  Successful delivery of d4TTP is demonstrated in human CD4+ T-lymphocyte cell exts. by an enzyme-triggered mechanism with high selectivity.  In antiviral assays, the compds. are potent inhibitors of HIV-1 and HIV-2 in CD4+ T-cell (CEM) cultures.  Highly lipophilic acyl residues lead to higher membrane permeability that results in intracellular delivery of phosphorylated metabolites in thymidine kinase-deficient CEM/TK- cells with higher antiviral activity than the parent nucleoside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr69l3mBVsk_7Vg90H21EOLACvtfcHk0liyNsXjjxKcMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnsL7N&md5=d245295f8ad5ca9ab4c548f62a2a938f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fncomms9716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9716%26sid%3Dliteratum%253Aachs%26aulast%3DGollnest%26aufirst%3DT.%26aulast%3Dde%2BOliveira%26aufirst%3DT.%2BD.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DLipophilic%2520prodrugs%2520of%2520nucleoside%2520triphosphates%2520as%2520biochemical%2520probes%2520and%2520potential%2520antivirals%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8716%26doi%3D10.1038%2Fncomms9716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gollnest, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinis de Oliveira, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauber, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Membrane-permeable triphosphate prodrugs of nucleoside analogues</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5255</span>– <span class="NLM_lpage">5258</span>, <span class="refDoi"> DOI: 10.1002/anie.201511808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fanie.201511808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=27008042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC28fivFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5255-5258&author=T.+Gollnestauthor=T.+Dinis+de+Oliveiraauthor=A.+Rathauthor=I.+Hauberauthor=D.+Scholsauthor=J.+Balzariniauthor=C.+Meier&title=Membrane-permeable+triphosphate+prodrugs+of+nucleoside+analogues&doi=10.1002%2Fanie.201511808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane-permeable Triphosphate Prodrugs of Nucleoside Analogues</span></div><div class="casAuthors">Gollnest Tristan; Dinis de Oliveira Thiago; Rath Anna; Meier Chris; Hauber Ilona; Schols Dominique; Balzarini Jan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5255-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The metabolic conversion of nucleoside analogues into their triphosphates often proceeds insufficiently.  Rate-limitations can be at the mono-, but also at the di- and triphosphorylation steps.  We developed a nucleoside triphosphate (NTP) delivery system (TriPPPro-approach).  In this approach, NTPs are masked by two bioreversible units at the γ-phosphate.  Using a procedure involving H-phosphonate chemistry, a series of derivatives bearing approved, as well as potentially antivirally active, nucleoside analogues was synthesized.  The enzyme-triggered delivery of NTPs was demonstrated by pig liver esterase, in human T-lymphocyte cell extracts and by a polymerase chain reaction using a prodrug of thymidine triphosphate.  The TriPPPro-compounds of some HIV-inactive nucleoside analogues showed marked anti-HIV activity.  For cellular uptake studies, a fluorescent TriPPPro-compound was prepared that delivered the triphosphorylated metabolite to intact CEM cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWWCDsvsG6Ru2qxNLjzXe8fW6udTcc2eb4iauIYyRy1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fivFGmtQ%253D%253D&md5=4bea432418c452ed82633ee0ab6bf274</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fanie.201511808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201511808%26sid%3Dliteratum%253Aachs%26aulast%3DGollnest%26aufirst%3DT.%26aulast%3DDinis%2Bde%2BOliveira%26aufirst%3DT.%26aulast%3DRath%26aufirst%3DA.%26aulast%3DHauber%26aufirst%3DI.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DMembrane-permeable%2520triphosphate%2520prodrugs%2520of%2520nucleoside%2520analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D5255%26epage%3D5258%26doi%3D10.1002%2Fanie.201511808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Anti-HIV-active nucleoside triphosphate prodrugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6003</span>– <span class="NLM_lpage">6027</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00271</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00271" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXps1Krurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6003-6027&author=X.+Jiaauthor=D.+Scholsauthor=C.+Meier&title=Anti-HIV-active+nucleoside+triphosphate+prodrugs&doi=10.1021%2Facs.jmedchem.0c00271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV-Active Nucleoside Triphosphate Prodrugs</span></div><div class="casAuthors">Jia, Xiao; Schols, Dominique; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6003-6027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We disclose a study on nucleoside triphosphate (NTP) analogs in which the γ-phosphate is covalently modified by two different biodegradable masking units and d4T as nucleoside analog that enable the delivery of d4TTP with high selectivity in phosphate buffer (pH 7.3) and by enzyme-triggered reactions in human CD4+ T-lymphocyte CEM cell exts.  This allows the bypass of all steps normally needed in the intracellular phosphorylation.  These TriPPPro-nucleotides comprising an acyloxybenzyl (AB; ester) or an alkoxycarbonyloxybenzyl (ACB; carbonate) in combination with an ACB moiety are described as NTP delivery systems.  The introduction of these two different groups led to the selective formation of γ-(ACB)-d4TTPs by chem. hydrolysis and in particular by cell ext. enzymes. γ-(AB)-d4TTPs are faster cleaved than γ-(ACB)-d4TTPs.  In antiviral assays, the compds. are highly active against HIV-1 and HIV-2 in wild-type CEM/O cells and more importantly in thymidine kinase-deficient CD4+ T-cells (CEM/TK-).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopqqrhtKobqbVg90H21EOLACvtfcHk0lgYRd-BWLGKew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXps1Krurg%253D&md5=ec2e2cca45554cf6776d4db2f76a566b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00271%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DX.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DAnti-HIV-active%2520nucleoside%2520triphosphate%2520prodrugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6003%26epage%3D6027%26doi%3D10.1021%2Facs.jmedchem.0c00271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Nucleoside diphosphate and triphosphate prodrugs - an unsolvable task?</span>. <i>Antivir. Chem. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1177/2040206617738656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1177%2F2040206617738656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29096525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ntFShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=69-82&author=C.+Meier&title=Nucleoside+diphosphate+and+triphosphate+prodrugs+-+an+unsolvable+task%3F&doi=10.1177%2F2040206617738656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleoside diphosphate and triphosphate prodrugs - An unsolvable task?</span></div><div class="casAuthors">Meier Chris</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral chemistry & chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">69-82</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this review, our recent advances in the development of nucleoside di- and nucleoside triphosphate prodrugs is summarized.  Previously, we had developed a successful membrane-permeable pronucleotide system for the intracellular delivery of nucleoside monophosphates as well, the so-called cycloSal-approach.  In contrast to that work in which the delivery is initiated by a chemically driven hydrolysis reaction, for the di- and triphosphate delivery, an enzymatic trigger mechanism involving (carboxy)esterases had to be used.  The other features of the new pronucleotide approaches are: (i) lipophilic modification was restricted to the terminal phosphate group leaving charges at the internal phosphate moieties and (ii) appropriate lipophilicity is introduced by long aliphatic residues within the bipartite prodrug moiety.  The conceptional design of the di- and triphosphate prodrug systems will be described and the chemical synthesis, the hydrolysis properties, a structure-activity relationship and antiviral activity data will be discussed as well.  The advantage of these new approaches is that all phosphorylation steps from the nucleoside analogue into the bioactive nucleoside triphosphate form can be bypassed in the case of the triphosphate prodrugs.  Moreover, enzymatic processes like the deamination of nucleosides or nucleoside monophosphates which lead to catabolic clearance of the potential antivirally active compound can be avoided by the delivery of the higher phosphorylated nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW7N2U7Gnno0NHe_gFldTbfW6udTcc2eZw-h4VQYtiJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ntFShuw%253D%253D&md5=f061bae9d87e96c0cd99e3a872a5c00b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F2040206617738656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040206617738656%26sid%3Dliteratum%253Aachs%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DNucleoside%2520diphosphate%2520and%2520triphosphate%2520prodrugs%2520-%2520an%2520unsolvable%2520task%253F%26jtitle%3DAntivir.%2520Chem.%2520Chemother.%26date%3D2017%26volume%3D25%26spage%3D69%26epage%3D82%26doi%3D10.1177%2F2040206617738656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weising, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterrenberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schols, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, C.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Antiviral Evaluation of TriPPPro-AbacavirTP, TriPPPro-CarbovirTP, and Their 1′,2′-cis-Disubstituted Analogues</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1771</span>– <span class="NLM_lpage">1778</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcmdc.201800361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29943432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOrt7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1771-1778&author=S.+Weisingauthor=V.+Sterrenbergauthor=D.+Scholsauthor=C.+Meier&title=Synthesis+and+Antiviral+Evaluation+of+TriPPPro-AbacavirTP%2C+TriPPPro-CarbovirTP%2C+and+Their+1%E2%80%B2%2C2%E2%80%B2-cis-Disubstituted+Analogues&doi=10.1002%2Fcmdc.201800361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Antiviral Evaluation of TriPPPro-AbacavirTP, TriPPPro-CarbovirTP, and Their 1',2'-cis-Disubstituted Analogs</span></div><div class="casAuthors">Weising, Simon; Sterrenberg, Vincente; Schols, Dominique; Meier, Chris</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1771-1778</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we describe the synthesis of lipophilic triphosphate prodrugs of abacavir, carbovir, and their 1',2'-cis-substituted carbocyclic analogs.  The 1',2'-cis-carbocyclic nucleosides were prepd. by starting from enantiomerically pure (1R,2S)-2-((benzyloxy)methyl)cyclopent-3-en-1-ol by a microwave-assisted Mitsunobu-type reaction with 2-amino-6-chloropurine.  All four nucleoside analogs were prepd. from their 2-amino-6-chloropurine precursors.  The nucleosides were converted into their corresponding nucleoside triphosphate prodrugs (TriPPPro approach) by application of the H-phosphonate route.  The TriPPPro compds. were hydrolyzed in different media, in which the formation of nucleoside triphosphates was proven.  While the TriPPPro compds. of abacavir and carbovir showed increased antiviral activity over their parent nucleoside, the TriPPPro compds. of the 1',2'-cis-substituted analogs as well as their parent nucleosides proved to be inactive against HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPFvFXL33ISrVg90H21EOLACvtfcHk0lgYRd-BWLGKew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOrt7%252FO&md5=25cb2870ae43ac07d8b21944039bcadd</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800361%26sid%3Dliteratum%253Aachs%26aulast%3DWeising%26aufirst%3DS.%26aulast%3DSterrenberg%26aufirst%3DV.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Antiviral%2520Evaluation%2520of%2520TriPPPro-AbacavirTP%252C%2520TriPPPro-CarbovirTP%252C%2520and%2520Their%25201%25E2%2580%25B2%252C2%25E2%2580%25B2-cis-Disubstituted%2520Analogues%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D1771%26epage%3D1778%26doi%3D10.1002%2Fcmdc.201800361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kandil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannecouque, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuigan, C.</span></span> <span> </span><span class="NLM_article-title">Polyfluoroaromatic stavudine (d4T) ProTides exhibit enhanced anti-HIV activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126721</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.bmcl.2019.126721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=31679972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2gsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126721&author=S.+Kandilauthor=C.+Pannecouqueauthor=F.+M.+Chapmanauthor=A.+D.+Westwellauthor=C.+McGuigan&title=Polyfluoroaromatic+stavudine+%28d4T%29+ProTides+exhibit+enhanced+anti-HIV+activity&doi=10.1016%2Fj.bmcl.2019.126721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Polyfluoroaromatic stavudine (d4T) ProTides exhibit enhanced anti-HIV activity</span></div><div class="casAuthors">Kandil, Sahar; Pannecouque, Christophe; Chapman, Fiona M.; Westwell, Andrew D.; McGuigan, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">126721</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human Immunodeficiency Virus (HIV) damages the immune system and leads to the life-threatening acquired immunodeficiency syndrome (AIDS).  Despite the advances in the field of antiretroviral treatment, HIV remains a major public health challenge.  Nucleosides represent a prominent chemotherapeutic class for treating viruses, however their cellular uptake, kinase-mediated activation and catabolism are limiting factors.  Herein, we report the synthesis and in vitro evaluation of stavudine (d4T) ProTides contg. polyfluorinated aryl groups against two strains; HIV-1 (IIIB) and HIV-2 (ROD).  ProTide I contg. a meta-substituted pentafluorosulfanyl (3-SF5) aryl group showed superior antiviral activity over the parent d4T and the nonfluorinated analog.  ProTide Ihas low nanomolar antiviral activity; (IC50 = 30 nM, HIV-1) and (IC50 = 36 nM, HIV-2) which is over tenfold more potent than d4T.  Interestingly, ProTide I showed a significantly high selectivity indexes with SI = 1753 (HIV-1) and 1461 (HIV-2) which is more than twice that of the d4T.  All ProTides were screened in wild type as well as thymidine kinase deficient (TK-) cells.  Enzymic activation of ProTide 5d using carboxypeptidase Y enzyme and monitored using both 31P and 19F NMR is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYpry6VggorrVg90H21EOLACvtfcHk0lg94m77lh9D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2gsrfJ&md5=52ee5ca71aec4cb8323eb5c39a0cc3d7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126721%26sid%3Dliteratum%253Aachs%26aulast%3DKandil%26aufirst%3DS.%26aulast%3DPannecouque%26aufirst%3DC.%26aulast%3DChapman%26aufirst%3DF.%2BM.%26aulast%3DWestwell%26aufirst%3DA.%2BD.%26aulast%3DMcGuigan%26aufirst%3DC.%26atitle%3DPolyfluoroaromatic%2520stavudine%2520%2528d4T%2529%2520ProTides%2520exhibit%2520enhanced%2520anti-HIV%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126721%26doi%3D10.1016%2Fj.bmcl.2019.126721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seley-Radtke, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, M. K.</span></span> <span> </span><span class="NLM_article-title">The evolution of nucleoside analogue antivirals: a review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.antiviral.2018.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=29649496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslOntrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2018&pages=66-86&author=K.+L.+Seley-Radtkeauthor=M.+K.+Yates&title=The+evolution+of+nucleoside+analogue+antivirals%3A+a+review+for+chemists+and+non-chemists.+Part+1%3A+Early+structural+modifications+to+the+nucleoside+scaffold&doi=10.1016%2Fj.antiviral.2018.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of nucleoside analogue antivirals: A review for chemists and non-chemists. Part 1: Early structural modifications to the nucleoside scaffold</span></div><div class="casAuthors">Seley-Radtke, Katherine L.; Yates, Mary K.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-86</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  This is the first of two invited articles reviewing the development of nucleoside-analog antiviral drugs, written for a target audience of virologists and other non-chemists, as well as chemists who may not be familiar with the field.  Rather than providing a simple chronol. account, we have examd. and attempted to explain the thought processes, advances in synthetic chem. and lessons learned from antiviral testing that led to a few mols. being moved forward to eventual approval for human therapies, while others were discarded.  The present paper focuses on early, relatively simplistic changes made to the nucleoside scaffold, beginning with modifications of the nucleoside sugars of Ara-C and other arabinose-derived nucleoside analogs in the 1960's.  A future paper will review more recent developments, focusing esp. on more complex modifications, particularly those involving multiple changes to the nucleoside scaffold.  We hope that these articles will help virologists and others outside the field of medicinal chem. to understand why certain drugs were successfully developed, while the majority of candidate compds. encountered barriers due to low-yielding synthetic routes, toxicity or other problems that led to their abandonment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTNbV8qi9P4bVg90H21EOLACvtfcHk0lg94m77lh9D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslOntrc%253D&md5=ccd9ad86ef8a1baf078e21747edef73d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DSeley-Radtke%26aufirst%3DK.%2BL.%26aulast%3DYates%26aufirst%3DM.%2BK.%26atitle%3DThe%2520evolution%2520of%2520nucleoside%2520analogue%2520antivirals%253A%2520a%2520review%2520for%2520chemists%2520and%2520non-chemists.%2520Part%25201%253A%2520Early%2520structural%2520modifications%2520to%2520the%2520nucleoside%2520scaffold%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D154%26spage%3D66%26epage%3D86%26doi%3D10.1016%2Fj.antiviral.2018.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, J. A.</span></span> <span> </span><span class="NLM_article-title">A general two-metal-ion mechanism for catalytic RNA</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">6498</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1073/pnas.90.14.6498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.90.14.6498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=8341661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK3sXms1Ghsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=6498-6502&author=T.+A.+Steitzauthor=J.+A.+Steitz&title=A+general+two-metal-ion+mechanism+for+catalytic+RNA&doi=10.1073%2Fpnas.90.14.6498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A general two-metal-ion mechanism for catalytic RNA</span></div><div class="casAuthors">Steitz, Thomas A.; Steitz, Joan A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6498-502</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">A mechanism is proposed for the RNA-catalyzed reactions involved in RNA splicing and RNase P hydrolysis of precursor tRNA.  The mechanism postulates that chem. catalysis is facilitated by 2 divalent metal ions 3.9 Å apart, as in phosphoryl transfer reactions catalyzed by protein enzymes, such as the 3',5'-exonuclease of Escherichia coli DNA polymerase I.  One metal ion activates the attacking water or sugar OH group, while the other coordinates and stabilizes the oxyanion leaving group.  Both ions act as Lewis acids and stabilize the expected pentacovalent transition state.  The symmetry of a 2-metal ion catalytic site fits well with the known reaction pathway of group I self-splicing introns and can also be reconciled with emerging data on group II self-splicing introns, the spliceosome, and RNase P.  The role of the RNA is to position the 2 catalytic metal ions and properly orient the substrates via 3 specific binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0QHemnv83hrVg90H21EOLACvtfcHk0lg94m77lh9D5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXms1Ghsb4%253D&md5=a785aaf05d7c5ecdaa688e7ebb90a12b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.14.6498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.14.6498%26sid%3Dliteratum%253Aachs%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26aulast%3DSteitz%26aufirst%3DJ.%2BA.%26atitle%3DA%2520general%2520two-metal-ion%2520mechanism%2520for%2520catalytic%2520RNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D6498%26epage%3D6502%26doi%3D10.1073%2Fpnas.90.14.6498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span> <span> </span><span class="NLM_article-title">Nucleases: diversity of structure, function and mechanism</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1017/s0033583510000181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1017%2FS0033583510000181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20854710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1-93&author=W.+Yang&title=Nucleases%3A+diversity+of+structure%2C+function+and+mechanism&doi=10.1017%2Fs0033583510000181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleases: diversity of structure, function and mechanism</span></div><div class="casAuthors">Yang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-93</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Nucleases cleave the phosphodiester bonds of nucleic acids and may be endo or exo, DNase or RNase, topoisomerases, recombinases, ribozymes, or RNA splicing enzymes.  In this review, I survey nuclease activities with known structures and catalytic machinery and classify them by reaction mechanism and metal-ion dependence and by their biol. function ranging from DNA replication, recombination, repair, RNA maturation, processing, interference, to defense, nutrient regeneration or cell death.  Several general principles emerge from this anal.  There is little correlation between catalytic mechanism and biol. function.  A single catalytic mechanism can be adapted in a variety of reactions and biol. pathways.  Conversely, a single biol. process can often be accomplished by multiple tertiary and quaternary folds and by more than one catalytic mechanism.  Two-metal-ion-dependent nucleases comprise the largest no. of different tertiary folds and mediate the most diverse set of biol. functions.  Metal-ion-dependent cleavage is exclusively assocd. with exonucleases producing mononucleotides and endonucleases that cleave double- or single-stranded substrates in helical and base-stacked conformations.  All metal-ion-independent RNases generate 2'.3'-cyclic phosphate products, and all metal-ion-independent DNases form phospho-protein intermediates.  I also find several previously unnoted relationships between different nucleases and shared catalytic configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7b0a8_b1YbVg90H21EOLACvtfcHk0lhds57Vb7Yrew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D&md5=4cd0bd5cec91b4572adf183ba2d90e7a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000181%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26atitle%3DNucleases%253A%2520diversity%2520of%2520structure%252C%2520function%2520and%2520mechanism%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2011%26volume%3D44%26spage%3D1%26epage%3D93%26doi%3D10.1017%2Fs0033583510000181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grobler, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stillmock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felock, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espeseth, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egbertson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourgeois, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melamed, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazuda, D. J.</span></span> <span> </span><span class="NLM_article-title">Diketo acid inhibitor mechanism and HIV-1 integrase: implications for metal binding in the active site of phosphotransferase enzymes</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">6661</span>– <span class="NLM_lpage">6666</span>, <span class="refDoi"> DOI: 10.1073/pnas.092056199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.092056199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=11997448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD38XjvFCrs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=6661-6666&author=J.+A.+Groblerauthor=K.+Stillmockauthor=B.+Huauthor=M.+Witmerauthor=P.+Felockauthor=A.+S.+Espesethauthor=A.+Wolfeauthor=M.+Egbertsonauthor=M.+Bourgeoisauthor=J.+Melamedauthor=J.+S.+Waiauthor=S.+Youngauthor=J.+Vaccaauthor=D.+J.+Hazuda&title=Diketo+acid+inhibitor+mechanism+and+HIV-1+integrase%3A+implications+for+metal+binding+in+the+active+site+of+phosphotransferase+enzymes&doi=10.1073%2Fpnas.092056199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Diketo acid inhibitor mechanism and HIV-1 integrase: implications for metal binding in the active site of phosphotransferase enzymes</span></div><div class="casAuthors">Grobler, Jay A.; Stillmock, Kara; Hu, Binghua; Witmer, Marc; Felock, Peter; Espeseth, Amy S.; Wolfe, Abigail; Egbertson, Melissa; Bourgeois, Michele; Melamed, Jeffrey; Wai, John S.; Young, Steve; Vacca, Joseph; Hazuda, Daria J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">6661-6666</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The process of integrating the reverse-transcribed HIV-1 DNA into the host chromosomal DNA is catalyzed by the virally encoded enzyme integrase (IN).  Integration requires two metal-dependent reactions, 3' end processing and strand transfer.  Compds. that contain a diketo acid moiety have been shown to selectively inhibit the strand transfer reaction of IN in vitro and in infected cells and are effective as inhibitors of HIV-1 replication.  To characterize the mol. basis of inhibition, we used functional assays and binding assays to evaluate a series of structurally related analogs.  These studies focused on investigating the role of the conserved carboxylate and metal binding.  We demonstrate that an acidic moiety such as a carboxylate or isosteric heterocycle is not required for binding to the enzyme complex but is essential for inhibition and confers distinct metal-dependent properties on the inhibitor.  Binding requires divalent metal and resistance is metal dependent with active site mutants displaying resistance only when the enzymes are evaluated in the context of Mg2+.  The mechanism of action of these inhibitors is therefore likely a consequence of the interaction between the acid moiety and metal ion(s) in the IN active site, resulting in a functional sequestration of the crit. metal cofactor(s).  These studies thus have implications for modeling active site inhibitors of IN, designing and evaluating analogs with improved efficacy, and identifying inhibitors of other metal-dependent phosphotransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkSyCLCLHKbrVg90H21EOLACvtfcHk0lhds57Vb7Yrew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjvFCrs78%253D&md5=a5d2ccbecd621eaeb1901416e5154fc1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.092056199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.092056199%26sid%3Dliteratum%253Aachs%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DEspeseth%26aufirst%3DA.%2BS.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DEgbertson%26aufirst%3DM.%26aulast%3DBourgeois%26aufirst%3DM.%26aulast%3DMelamed%26aufirst%3DJ.%26aulast%3DWai%26aufirst%3DJ.%2BS.%26aulast%3DYoung%26aufirst%3DS.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DDiketo%2520acid%2520inhibitor%2520mechanism%2520and%2520HIV-1%2520integrase%253A%2520implications%2520for%2520metal%2520binding%2520in%2520the%2520active%2520site%2520of%2520phosphotransferase%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2002%26volume%3D99%26spage%3D6661%26epage%3D6666%26doi%3D10.1073%2Fpnas.092056199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structural basis of HIV-1 inhibition by nucleotide-competing reverse transcriptase inhibitor INDOPY-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9996</span>– <span class="NLM_lpage">10002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01289</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01289" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFGqur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9996-10002&author=F.+X.+Ruizauthor=A.+Hoangauthor=K.+Dasauthor=E.+Arnold&title=Structural+basis+of+HIV-1+inhibition+by+nucleotide-competing+reverse+transcriptase+inhibitor+INDOPY-1&doi=10.1021%2Facs.jmedchem.9b01289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of HIV-1 Inhibition by Nucleotide-Competing Reverse Transcriptase Inhibitor INDOPY-1</span></div><div class="casAuthors">Ruiz, F. Xavier; Hoang, Anthony; Das, Kalyan; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9996-10002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) is an essential enzyme, targeting half of approved anti-AIDS drugs.  While nucleoside RT inhibitors (NRTIs) are DNA chain terminators, the nucleotide-competing RT inhibitor (NcRTI) INDOPY-1 blocks dNTP binding to RT.  Lack of structural information hindered INDOPY-1 improvement.  Here we report the HIV-1 RT/DNA/INDOPY-1 crystal structure, revealing a unique mode of inhibitor binding at the polymerase active site without involving catalytic metal ions.  The structure may enable new strategies for developing NcRTIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy__3eTkJVObVg90H21EOLACvtfcHk0ljaLkAmvCApoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGqur%252FL&md5=ccf2bf84e6905e96856eb1586f32ec4c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01289%26sid%3Dliteratum%253Aachs%26aulast%3DRuiz%26aufirst%3DF.%2BX.%26aulast%3DHoang%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520of%2520HIV-1%2520inhibition%2520by%2520nucleotide-competing%2520reverse%2520transcriptase%2520inhibitor%2520INDOPY-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9996%26epage%3D10002%26doi%3D10.1021%2Facs.jmedchem.9b01289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deval, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kesteleyn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Marck, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettens, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Baere, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivens, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Ginderen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schoubroeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehteshami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigerinck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertogs, K.</span></span> <span> </span><span class="NLM_article-title">Indolopyridones inhibit human immunodeficiency virus reverse transcriptase with a novel mechanism of action</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">12283</span>– <span class="NLM_lpage">12292</span>, <span class="refDoi"> DOI: 10.1128/jvi.00889-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1128%2FJVI.00889-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17020946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlWksb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2006&pages=12283-12292&author=D.+Jochmansauthor=J.+Devalauthor=B.+Kesteleynauthor=H.+Van+Marckauthor=E.+Bettensauthor=I.+De+Baereauthor=P.+Dehertoghauthor=T.+Ivensauthor=M.+Van+Ginderenauthor=B.+Van+Schoubroeckauthor=M.+Ehteshamiauthor=P.+Wigerinckauthor=M.+G%C3%B6tteauthor=K.+Hertogs&title=Indolopyridones+inhibit+human+immunodeficiency+virus+reverse+transcriptase+with+a+novel+mechanism+of+action&doi=10.1128%2Fjvi.00889-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Indolopyridones inhibit human immunodeficiency virus reverse transcriptase with a novel mechanism of action</span></div><div class="casAuthors">Jochmans, Dirk; Deval, Jerome; Kesteleyn, Bart; Van Marck, Herwig; Bettens, Eva; De Baere, Inky; Dehertogh, Pascale; Ivens, Tania; Van Ginderen, Marcia; Van Schoubroeck, Bertrand; Ehteshami, Maryam; Wigerinck, Piet; Gotte, Matthias; Hertogs, Kurt</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12283-12292</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We have discovered a novel class of human immunodeficiency virus (HIV) reverse transcriptase (RT) inhibitors that block the polymn. reaction in a mode distinct from those of the nucleoside or nucleotide RT inhibitors (NRTIs) and nonnucleoside RT inhibitors (NNRTIs).  For this class of indolopyridone compds., steady-state kinetics revealed competitive inhibition with respect to the nucleotide substrate.  Despite substantial structural differences with classical chain terminators or natural nucleotides, these data suggest that the nucleotide binding site of HIV RT may accommodate this novel class of RT inhibitors.  To test this hypothesis, we have studied the mechanism of action of the prototype compd. indolopyridone-1 (INDOPY-1) using a variety of complementary biochem. tools.  Time course expts. with heteropolymeric templates showed "hot spots" for inhibition following the incorporation of pyrimidines (T>C).  Moreover, binding studies and site-specific footprinting expts. revealed that INDOPY-1 traps the complex in the posttranslocational state, preventing binding and incorporation of the next complementary nucleotide.  The novel mode of action translates into a unique resistance profile.  While INDOPY-1 susceptibility is unaffected by mutations assocd. with NNRTI or multidrug NRTI resistance, mutations M184V and Y115F are assocd. with decreased susceptibility, and mutation K65R confers hypersusceptibility to INDOPY-1.  This resistance profile provides addnl. evidence for active site binding.  In conclusion, this class of indolopyridones can occupy the nucleotide binding site of HIV RT by forming a stable ternary complex whose stability is mainly dependent on the nature of the primer 3' end.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_zFvcKcwnfbVg90H21EOLACvtfcHk0ljaLkAmvCApoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlWksb%252FO&md5=8477ac7cd7f82746870c7a6b2c89c1d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FJVI.00889-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00889-06%26sid%3Dliteratum%253Aachs%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DDeval%26aufirst%3DJ.%26aulast%3DKesteleyn%26aufirst%3DB.%26aulast%3DVan%2BMarck%26aufirst%3DH.%26aulast%3DBettens%26aufirst%3DE.%26aulast%3DDe%2BBaere%26aufirst%3DI.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DIvens%26aufirst%3DT.%26aulast%3DVan%2BGinderen%26aufirst%3DM.%26aulast%3DVan%2BSchoubroeck%26aufirst%3DB.%26aulast%3DEhteshami%26aufirst%3DM.%26aulast%3DWigerinck%26aufirst%3DP.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DHertogs%26aufirst%3DK.%26atitle%3DIndolopyridones%2520inhibit%2520human%2520immunodeficiency%2520virus%2520reverse%2520transcriptase%2520with%2520a%2520novel%2520mechanism%2520of%2520action%26jtitle%3DJ.%2520Virol.%26date%3D2006%26volume%3D80%26spage%3D12283%26epage%3D12292%26doi%3D10.1128%2Fjvi.00889-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debarge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of α-Carboxy Phosphononucleosides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1021/jo101738e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo101738e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFShtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=105-126&author=S.+Debargeauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Design+and+Synthesis+of+%CE%B1-Carboxy+Phosphononucleosides&doi=10.1021%2Fjo101738e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of α-Carboxy Phosphono-Nucleosides</span></div><div class="casAuthors">Debarge, Sebastien; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">105-126</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rhodium catalyzed O-H insertion reactions employing α-diazo-phosphonate with appropriately protected thymidine, uridine, cytosine, adenosine and guanosine derivs. leads to novel 5'-phosphono-nucleoside derivs.  Deprotection led to a novel series of phosphono derivs. bearing a carboxylic acid moiety adjacent to the phosphonate group with potential antiviral and/or anticancer activity.  The phosphono-nucleosides bearing an α-carboxylic acid group are envisaged as potential diphosphate mimics.  Conversion to mono- and diphosphorylated phosphono-nucleosides has been effected for evaluation as nucleoside triphosphate mimics.  Most of the novel phosphono-nucleosides proved to be inactive against a variety of DNA and RNA viruses.  Only the phosphono AZT derivs. showed weak activity against HIV-1 and HIV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9bW35PpIOvrVg90H21EOLACvtfcHk0ljaLkAmvCApoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFShtbvK&md5=66461aaaf0a248f92117aa5797bf081e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo101738e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo101738e%26sid%3Dliteratum%253Aachs%26aulast%3DDebarge%26aufirst%3DS.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520%25CE%25B1-Carboxy%2520Phosphononucleosides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2011%26volume%3D76%26spage%3D105%26epage%3D126%26doi%3D10.1021%2Fjo101738e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span> <span> </span><span class="NLM_article-title">Pronounced Inhibition Shift from HIV Reverse Transcriptase to Herpetic DNA Polymerases by Increasing the Flexibility of α-Carboxy Nucleoside Phosphonates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8110</span>– <span class="NLM_lpage">8127</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kit7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8110-8127&author=J.+Johnauthor=Y.+Kimauthor=N.+Bennettauthor=K.+Dasauthor=S.+Liekensauthor=L.+Naesensauthor=E.+Arnoldauthor=A.+R.+Maguireauthor=M.+G%C3%B6tteauthor=W.+Dehaenauthor=J.+Balzarini&title=Pronounced+Inhibition+Shift+from+HIV+Reverse+Transcriptase+to+Herpetic+DNA+Polymerases+by+Increasing+the+Flexibility+of+%CE%B1-Carboxy+Nucleoside+Phosphonates&doi=10.1021%2Facs.jmedchem.5b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Pronounced Inhibition Shift from HIV Reverse Transcriptase to Herpetic DNA Polymerases by Increasing the Flexibility of α-Carboxy Nucleoside Phosphonates</span></div><div class="casAuthors">John, Jubi; Kim, Youngju; Bennett, Nicholas; Das, Kalyan; Liekens, Sandra; Naesens, Lieve; Arnold, Eddy; Maguire, Anita R.; Gotte, Matthias; Dehaen, Wim; Balzarini, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8110-8127</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">α-Carboxynucleoside phosphonates (α-CNPs) are novel viral DNA polymerase inhibitors that do not need metabolic conversion for enzyme inhibition.  The prototype contains a cyclopentyl linker between nucleobase and α-carboxyphosphonate and preferentially (50- to 100-fold) inhibits HIV-1 RT compared with herpetic DNA polymerases.  A synthesis methodol. involving three steps has been developed for the synthesis of a series of novel α-CNPs, including a Rh(II)-catalyzed O-H insertion that connects the carboxyphosphonate group to a linker moiety and an attachment of a nucleobase to the other end of the linker by a Mitsunobu reaction followed by final deprotection.  Replacing the cyclopentyl moiety in the prototype α-CNPs by a more flexible entity results in a selectivity shift of ∼100-fold in favor of the herpetic DNA polymerases when compared to selectivity for HIV-1 RT.  The nature of the kinetic interaction of the acyclic α-CNPs against the herpetic DNA polymerases differs from the nature of the nucleobase-specific kinetic interaction of the cyclopentyl α-CNPs against HIV RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGootBqwSKGJwrVg90H21EOLACvtfcHk0liEKryBvxVByQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kit7fO&md5=870e3e13a71997e19138133dbb3eacf2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01180%26sid%3Dliteratum%253Aachs%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DBennett%26aufirst%3DN.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DDehaen%26aufirst%3DW.%26aulast%3DBalzarini%26aufirst%3DJ.%26atitle%3DPronounced%2520Inhibition%2520Shift%2520from%2520HIV%2520Reverse%2520Transcriptase%2520to%2520Herpetic%2520DNA%2520Polymerases%2520by%2520Increasing%2520the%2520Flexibility%2520of%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8110%26epage%3D8127%26doi%3D10.1021%2Facs.jmedchem.5b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keane, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legigan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of α-Carboxy Nucleoside Phosphonate Analogues and Evaluation as HIV-1 Reverse Transcriptase-Targeting Agents</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">2479</span>– <span class="NLM_lpage">2493</span>, <span class="refDoi"> DOI: 10.1021/jo502549y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo502549y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=2479-2493&author=S.+J.+Keaneauthor=A.+Fordauthor=N.+D.+Mullinsauthor=N.+M.+Maguireauthor=T.+Legiganauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Design+and+Synthesis+of+%CE%B1-Carboxy+Nucleoside+Phosphonate+Analogues+and+Evaluation+as+HIV-1+Reverse+Transcriptase-Targeting+Agents&doi=10.1021%2Fjo502549y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of α-Carboxy Nucleoside Phosphonate Analogues and Evaluation as HIV-1 Reverse Transcriptase-Targeting Agents</span></div><div class="casAuthors">Keane, Sarah J.; Ford, Alan; Mullins, Nicholas D.; Maguire, Nuala M.; Legigan, Thibaut; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2479-2493</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of the first series of a new class of nucleoside phosphonate analogs, e.g. I, is described.  Addn. of a carboxyl group at the α position of carbocyclic nucleoside phosphonate analogs leads to a novel class of potent HIV reverse transcriptase (RT) inhibitors, α-carboxy nucleoside phosphonates (α-CNPs).  Key steps in the synthesis of the compds. are Rh-catalyzed O-H insertion and Pd-catalyzed allylation reactions.  In cell-free assays, the final products are markedly inhibitory against HIV RT and do not require phosphorylation to exhibit anti-RT activity, which indicates that the α-carboxy-phosphonate function is efficiently recognized by HIV RT as a triphosphate entity, an unprecedented property of nucleoside mono-phosphates and mono-phosphonates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPoDSAEHJHR7Vg90H21EOLACvtfcHk0liEKryBvxVByQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7nF&md5=23a5cebed6ac346fdd7ddff18fa92474</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjo502549y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo502549y%26sid%3Dliteratum%253Aachs%26aulast%3DKeane%26aufirst%3DS.%2BJ.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMullins%26aufirst%3DN.%2BD.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DLegigan%26aufirst%3DT.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonate%2520Analogues%2520and%2520Evaluation%2520as%2520HIV-1%2520Reverse%2520Transcriptase-Targeting%2520Agents%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D2479%26epage%3D2493%26doi%3D10.1021%2Fjo502549y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Guanine α-Carboxy Nucleoside Phosphonate (G-α-CNP), a Direct Inhibitor of Multiple Viral DNA Polymerases</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">10510</span>– <span class="NLM_lpage">10517</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.8b01124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.8b01124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVCqtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2018&pages=10510-10517&author=N.+M.+Maguireauthor=A.+Fordauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Synthesis+of+Guanine+%CE%B1-Carboxy+Nucleoside+Phosphonate+%28G-%CE%B1-CNP%29%2C+a+Direct+Inhibitor+of+Multiple+Viral+DNA+Polymerases&doi=10.1021%2Facs.joc.8b01124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Guanine α-Carboxy Nucleoside Phosphonate (G-α-CNP), a Direct Inhibitor of Multiple Viral DNA Polymerases</span></div><div class="casAuthors">Maguire, Nuala M.; Ford, Alan; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10510-10517</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis of guanine α-carboxy nucleoside phosphonate (G-α-CNP) is described.  Two routes provide access to racemic G-α-CNP I, one via base construction and the other utilizing Tsuji-Trost allylic substitution.  The latter methodol. was also applied to the enantiopure synthesis of both antipodes of G-α-CNP, each of which shows interesting antiviral DNA polymerase activity.  Addnl., we report an improved multi-gram scale prepn. of the cyclopentene building block 10, starting material for the preferred Tsuji-Trost route to I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBENxgoUKQ07Vg90H21EOLACvtfcHk0ljic1oniaUdMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVCqtrvN&md5=22abf2613902b9b0765d513c119fa49e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.8b01124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.8b01124%26sid%3Dliteratum%253Aachs%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DSynthesis%2520of%2520Guanine%2520%25CE%25B1-Carboxy%2520Nucleoside%2520Phosphonate%2520%2528G-%25CE%25B1-CNP%2529%252C%2520a%2520Direct%2520Inhibitor%2520of%2520Multiple%2520Viral%2520DNA%2520Polymerases%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2018%26volume%3D83%26spage%3D10510%26epage%3D10517%26doi%3D10.1021%2Facs.joc.8b01124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Exploring the role of the α-carboxyphosphonate moiety in the HIV-RT activity of α-carboxy nucleoside phosphonates</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2454</span>– <span class="NLM_lpage">2465</span>, <span class="refDoi"> DOI: 10.1039/c5ob02507a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1039%2FC5OB02507A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=26813581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtF2isL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2454-2465&author=N.+D.+Mullinsauthor=N.+M.+Maguireauthor=A.+Fordauthor=K.+Dasauthor=E.+Arnoldauthor=J.+Balzariniauthor=A.+R.+Maguire&title=Exploring+the+role+of+the+%CE%B1-carboxyphosphonate+moiety+in+the+HIV-RT+activity+of+%CE%B1-carboxy+nucleoside+phosphonates&doi=10.1039%2Fc5ob02507a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the role of the α-carboxyphosphonate moiety in the HIV-RT activity of α-carboxy nucleoside phosphonates</span></div><div class="casAuthors">Mullins, Nicholas D.; Maguire, Nuala M.; Ford, Alan; Das, Kalyan; Arnold, Eddy; Balzarini, Jan; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2454-2465</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As α-carboxy nucleoside phosphonates (α-CNPs) have demonstrated a novel mode of action of HIV-1 reverse transcriptase inhibition, structurally related derivs. were synthesized, namely the malonate I, two unsatd. and satd. bisphosphonates and amide.  These compds. were evaluated for inhibition of HIV-1 reverse transcriptase in cell-free assays.  The importance of the α-carboxy phosphonoacetic acid moiety for achieving reverse transcriptase inhibition, without the need for prior phosphorylation, was confirmed.  The malonate deriv. I was less active by two orders of magnitude than the original α-CNPs, while displaying the same pattern of kinetic behavior; interestingly the activity resides in the "L"-enantiomer of I, as seen with the earlier series of α-CNPs.  A crystal structure with an RT/DNA complex at 2.95 Å resoln. revealed the binding of the "L"-enantiomer of I, at the polymerase active site with a weaker metal ion chelation environment compared to 1a (T-α-CNP) which may explain the lower inhibitory activity of I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodQ95f7BrdvLVg90H21EOLACvtfcHk0ljic1oniaUdMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtF2isL8%253D&md5=732f00d94b631696fe4bf29a26f4af08</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2FC5OB02507A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB02507A%26sid%3Dliteratum%253Aachs%26aulast%3DMullins%26aufirst%3DN.%2BD.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DExploring%2520the%2520role%2520of%2520the%2520%25CE%25B1-carboxyphosphonate%2520moiety%2520in%2520the%2520HIV-RT%2520activity%2520of%2520%25CE%25B1-carboxy%2520nucleoside%2520phosphonates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D2454%26epage%3D2465%26doi%3D10.1039%2Fc5ob02507a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hladezuk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chastagner, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plunkett, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debarge, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Development of O-H insertion for the attachment of phosphonates to nucleosides; synthesis of α-carboxy phosphononucleosides</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">1894</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2011.12.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.tet.2011.12.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=1894-1909&author=I.+Hladezukauthor=V.+Chastagnerauthor=S.+G.+Collinsauthor=S.+J.+Plunkettauthor=A.+Fordauthor=S.+Debargeauthor=A.+R.+Maguire&title=Development+of+O-H+insertion+for+the+attachment+of+phosphonates+to+nucleosides%3B+synthesis+of+%CE%B1-carboxy+phosphononucleosides&doi=10.1016%2Fj.tet.2011.12.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of O-H insertion for the attachment of phosphonates to nucleosides; synthesis of α-carboxy phosphononucleosides</span></div><div class="casAuthors">Hladezuk, Isabelle; Chastagner, Veronique; Collins, Stuart G.; Plunkett, Stephen J.; Ford, Alan; Debarge, Sebastien; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1894-1909</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Development of rhodium catalyzed O-H insertion reactions employing α-diazophosphonates with appropriately protected adenosine, uridine and thymidine derivs. is described.  This synthetic methodol. leads, following deprotection, to novel phosphononucleoside derivs. bearing a carboxylic acid moiety adjacent to the phosphonate, e.g. I.  Protection strategies are crit. to the success of the key O-H insertion.  There are two important aspects: avoiding competing insertion pathways or catalyst poisoning, and being able to achieve deprotection without degrdn. of the phosphononucleosides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3LL9mtO-XL7Vg90H21EOLACvtfcHk0ljic1oniaUdMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGqsL4%253D&md5=9ef4af19fc5834b7158277bb6ceda5ac</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2011.12.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2011.12.077%26sid%3Dliteratum%253Aachs%26aulast%3DHladezuk%26aufirst%3DI.%26aulast%3DChastagner%26aufirst%3DV.%26aulast%3DCollins%26aufirst%3DS.%2BG.%26aulast%3DPlunkett%26aufirst%3DS.%2BJ.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DDebarge%26aufirst%3DS.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DDevelopment%2520of%2520O-H%2520insertion%2520for%2520the%2520attachment%2520of%2520phosphonates%2520to%2520nucleosides%253B%2520synthesis%2520of%2520%25CE%25B1-carboxy%2520phosphononucleosides%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D1894%26epage%3D1909%26doi%3D10.1016%2Fj.tet.2011.12.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Berckelaer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmer, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span> <span> </span><span class="NLM_article-title">Guanine α-carboxy nucleoside phosphonate (G-α-CNP) shows a different inhibitory kinetic profile against the DNA polymerases of human immunodeficiency virus (HIV) and herpes viruses</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2Fj.bcp.2017.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=28390939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFWgsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=51-61&author=J.+Balzariniauthor=M.+Menniauthor=K.+Dasauthor=L.+van+Berckelaerauthor=A.+Fordauthor=N.+M.+Maguireauthor=S.+Liekensauthor=P.+E.+Boehmerauthor=E.+Arnoldauthor=M.+G%C3%B6tteauthor=A.+R.+Maguire&title=Guanine+%CE%B1-carboxy+nucleoside+phosphonate+%28G-%CE%B1-CNP%29+shows+a+different+inhibitory+kinetic+profile+against+the+DNA+polymerases+of+human+immunodeficiency+virus+%28HIV%29+and+herpes+viruses&doi=10.1016%2Fj.bcp.2017.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Guanine α-carboxy nucleoside phosphonate (G-α-CNP) shows a different inhibitory kinetic profile against the DNA polymerases of human immunodeficiency virus (HIV) and herpes viruses</span></div><div class="casAuthors">Balzarini, Jan; Menni, Michael; Das, Kalyan; van Berckelaer, Lizette; Ford, Alan; Maguire, Nuala M.; Liekens, Sandra; Boehmer, Paul E.; Arnold, Eddy; Gotte, Matthias; Maguire, Anita R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">α-Carboxy nucleoside phosphonates (α-CNPs) are modified nucleotides that represent a novel class of nucleotide-competing reverse transcriptase (RT) inhibitors (NcRTIs).  They were designed to act directly against HIV-1 RT without the need for prior activation (phosphorylation).  In this respect, they differ from the nucleoside or nucleotide RTIs [N(t)RTIs] that require conversion to their triphosphate forms before being inhibitory to HIV-1 RT.  The guanine deriv. (G-α-CNP) has now been synthesized and investigated for the first time.  The (L)-(+)-enantiomer of G-α-CNP directly and competitively inhibits HIV-1 RT by interacting with the substrate active site of the enzyme.  The (D)-(-)-enantiomer proved inactive against HIV-1 RT.  In contrast, the (+)- and (-)-enantiomers of G-α-CNP inhibited herpes (i.e. HSV-1, HCMV) DNA polymerases in a non- or uncompetitive manner, strongly indicating interaction of the (L)-(+)- and the (D)-(-)-G-α-CNPs at a location different from the polymerase substrate active site of the herpes enzymes.  Such entirely different inhibition profile of viral polymerases is unprecedented for a single antiviral drug mol.  Moreover, within the class of α-CNPs, subtle differences in their sensitivity to mutant HIV-1 RT enzymes were obsd. depending on the nature of the nucleobase in the α-CNP mols.  The unique properties of the α-CNPs make this class of compds., including G-α-CNP, direct acting inhibitors of multiple viral DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru34nHOlZAWLVg90H21EOLACvtfcHk0ljoDTKnbhp8Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFWgsbc%253D&md5=6f725c42e63f8fc28e5dc9fa9feb13ec</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMenni%26aufirst%3DM.%26aulast%3DDas%26aufirst%3DK.%26aulast%3Dvan%2BBerckelaer%26aufirst%3DL.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMaguire%26aufirst%3DN.%2BM.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DBoehmer%26aufirst%3DP.%2BE.%26aulast%3DArnold%26aufirst%3DE.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26atitle%3DGuanine%2520%25CE%25B1-carboxy%2520nucleoside%2520phosphonate%2520%2528G-%25CE%25B1-CNP%2529%2520shows%2520a%2520different%2520inhibitory%2520kinetic%2520profile%2520against%2520the%2520DNA%2520polymerases%2520of%2520human%2520immunodeficiency%2520virus%2520%2528HIV%2529%2520and%2520herpes%2520viruses%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D136%26spage%3D51%26epage%3D61%26doi%3D10.1016%2Fj.bcp.2017.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernatchez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullins, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vande Voorde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Alpha-carboxy nucleoside phosphonates as universal nucleoside triphosphate mimics</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">3475</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1073/pnas.1420233112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1073%2Fpnas.1420233112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=25733891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1Grtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=3475-3480&author=J.+Balzariniauthor=K.+Dasauthor=J.+A.+Bernatchezauthor=S.+E.+Martinezauthor=M.+Ngureauthor=S.+Keaneauthor=A.+Fordauthor=N.+Maguireauthor=N.+Mullinsauthor=J.+Johnauthor=Y.+Kimauthor=W.+Dehaenauthor=J.+Vande+Voordeauthor=S.+Liekensauthor=L.+Naesensauthor=M.+G%C3%B6tteauthor=A.+R.+Maguireauthor=E.+Arnold&title=Alpha-carboxy+nucleoside+phosphonates+as+universal+nucleoside+triphosphate+mimics&doi=10.1073%2Fpnas.1420233112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Alpha-carboxy nucleoside phosphonates as universal nucleoside triphosphate mimics</span></div><div class="casAuthors">Balzarini, Jan; Das, Kalyan; Bernatchez, Jean A.; Martinez, Sergio E.; Ngure, Marianne; Keane, Sarah; Ford, Alan; Maguire, Nuala; Mullins, Niki; John, Jubi; Kim, Youngju; Dehaen, Wim; Voorde, Johan Vande; Liekens, Sandra; Naesens, Lieve; Gotte, Matthias; Maguire, Anita R.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3475-3480</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polymerases have a structurally highly conserved neg. charged amino acid motif that is strictly required for Mg2+ cation-dependent catalytic incorporation of (d)NTP nucleotides into nucleic acids.  Based on these characteristics, a nucleoside monophosphonate scaffold, α-carboxy nucleoside phosphonate (α-CNP), was designed that is recognized by a variety of polymerases.  Kinetic, biochem., and crystallog. studies with HIV-1 reverse transcriptase revealed that α-CNPs mimic the dNTP binding through a carboxylate oxygen, two phosphonate oxygens, and base-pairing with the template.  In particular, the carboxyl oxygen of the α-CNP acts as the potential equiv. of the α-phosphate oxygen of dNTPs and two oxygens of the phosphonate group of the α-CNP chelate Mg2+, mimicking the chelation by the β- and γ-phosphate oxygens of dNTPs. α-CNPs (i) do not require metabolic activation (phosphorylation), (ii) bind directly to the substrate-binding site, (iii) chelate one of the two active site Mg2+ ions, and (iv) reversibly inhibit the polymerase catalytic activity without being incorporated into nucleic acids.  In addn., α-CNPs were also found to selectively interact with regulatory (i.e., allosteric) Mg2+-dNTP-binding sites of nucleos(t)ide-metabolizing enzymes susceptible to metabolic regulation. α-CNPs represent an entirely novel and broad technol. platform for the development of specific substrate active- or regulatory-site inhibitors with therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz-g4rWbNDV7Vg90H21EOLACvtfcHk0lgsmmcGaG-eNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1Grtro%253D&md5=0cff60a295b9eb86a591cce8997c6613</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1420233112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1420233112%26sid%3Dliteratum%253Aachs%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBernatchez%26aufirst%3DJ.%2BA.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DNgure%26aufirst%3DM.%26aulast%3DKeane%26aufirst%3DS.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DMaguire%26aufirst%3DN.%26aulast%3DMullins%26aufirst%3DN.%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DDehaen%26aufirst%3DW.%26aulast%3DVande%2BVoorde%26aufirst%3DJ.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DG%25C3%25B6tte%26aufirst%3DM.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DAlpha-carboxy%2520nucleoside%2520phosphonates%2520as%2520universal%2520nucleoside%2520triphosphate%2520mimics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D3475%26epage%3D3480%26doi%3D10.1073%2Fpnas.1420233112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Conformational states of HIV-1 reverse transcriptase for nucleotide incorporation vs pyrophosphorolysis-binding of foscarnet</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2158</span>– <span class="NLM_lpage">2164</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00187</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00187" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVSksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2158-2164&author=K.+Dasauthor=J.+Balzariniauthor=M.+T.+Millerauthor=A.+R.+Maguireauthor=J.+J.+DeStefanoauthor=E.+Arnold&title=Conformational+states+of+HIV-1+reverse+transcriptase+for+nucleotide+incorporation+vs+pyrophosphorolysis-binding+of+foscarnet&doi=10.1021%2Facschembio.6b00187"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational States of HIV-1 Reverse Transcriptase for Nucleotide Incorporation vs Pyrophosphorolysis-Binding of Foscarnet</span></div><div class="casAuthors">Das, Kalyan; Balzarini, Jan; Miller, Matthew T.; Maguire, Anita R.; DeStefano, Jeffrey J.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2158-2164</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) catalytically incorporates individual nucleotides into a viral DNA strand complementing an RNA or DNA template strand; the polymerase active site of RT adopts multiple conformational and structural states while performing this task.  The states assocd. are dNTP binding at the N site, catalytic incorporation of a nucleotide, release of a pyrophosphate, and translocation of the primer 3'-end to the P site.  Structural characterization of each of these states may help in understanding the mol. mechanisms of drug activity and resistance and in developing new RT inhibitors.  Using a 38-mer DNA template-primer aptamer as the substrate mimic, we crystd. an RT/dsDNA complex that is catalytically active, yet translocation-incompetent in crystals.  The ability of RT to perform dNTP binding and incorporation in crystals permitted obtaining a series of structures: (I) RT/DNA (P-site), (II) RT/DNA/AZTTP ternary, (III) RT/AZT-terminated DNA (N-site), and (IV) RT/AZT-terminated DNA (N-site)/foscarnet complexes.  The stable N-site complex permitted the binding of foscarnet as a pyrophosphate mimic.  The Mg2+ ions dissocd. after catalytic addn. of AZTMP in the pretranslocated structure III, whereas ions A and B had re-entered the active site to bind foscarnet in structure IV.  The binding of foscarnet involves chelation with the Mg2+ (B) ion and interactions with K65 and R72.  The anal. of interactions of foscarnet and the recently discovered nucleotide-competing RT inhibitor (NcRTI) α-T-CNP in two different conformational states of the enzyme provides insights for developing new classes of polymerase active site RT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPnYWlNCZhubVg90H21EOLACvtfcHk0lgsmmcGaG-eNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVSksLk%253D&md5=aa013aea155b60e18620a5b5d378aa18</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00187%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DM.%2BT.%26aulast%3DMaguire%26aufirst%3DA.%2BR.%26aulast%3DDeStefano%26aufirst%3DJ.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DConformational%2520states%2520of%2520HIV-1%2520reverse%2520transcriptase%2520for%2520nucleotide%2520incorporation%2520vs%2520pyrophosphorolysis-binding%2520of%2520foscarnet%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2158%26epage%3D2164%26doi%3D10.1021%2Facschembio.6b00187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adelfinskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Amino acid phosphoramidate nucleotides as alternative substrates for HIV-1 reverse transcriptase</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4356</span>– <span class="NLM_lpage">4358</span>, <span class="refDoi"> DOI: 10.1002/anie.200605016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fanie.200605016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17443759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BD2svgtlKnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4356-4358&author=O.+Adelfinskayaauthor=P.+Herdewijn&title=Amino+acid+phosphoramidate+nucleotides+as+alternative+substrates+for+HIV-1+reverse+transcriptase&doi=10.1002%2Fanie.200605016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid phosphoramidate nucleotides as alternative substrates for HIV-1 reverse transcriptase</span></div><div class="casAuthors">Adelfinskaya Olga; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4356-8</span>
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSu-rO0KHaB_rJzI8owOhpDfW6udTcc2eZHA417iJXFm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svgtlKnsw%253D%253D&md5=3ac775f81677aadc732603bd4dd45f96</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.200605016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200605016%26sid%3Dliteratum%253Aachs%26aulast%3DAdelfinskaya%26aufirst%3DO.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DAmino%2520acid%2520phosphoramidate%2520nucleotides%2520as%2520alternative%2520substrates%2520for%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2007%26volume%3D46%26spage%3D4356%26epage%3D4358%26doi%3D10.1002%2Fanie.200605016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adelfinskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marliere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauwelaerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Polymerase-catalyzed synthesis of DNA from phosphoramidate conjugates of deoxynucleotides and amino acids</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">5060</span>– <span class="NLM_lpage">5072</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkm498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=17652326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFWgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=5060-5072&author=O.+Adelfinskayaauthor=M.+Terrazasauthor=M.+Froeyenauthor=P.+Marliereauthor=K.+Nauwelaertsauthor=P.+Herdewijn&title=Polymerase-catalyzed+synthesis+of+DNA+from+phosphoramidate+conjugates+of+deoxynucleotides+and+amino+acids&doi=10.1093%2Fnar%2Fgkm498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Polymerase-catalyzed synthesis of DNA from phosphoramidate conjugates of deoxynucleotides and amino acids</span></div><div class="casAuthors">Adelfinskaya, O.; Terrazas, M.; Froeyen, M.; Marliere, P.; Nauwelaerts, K.; Herdewijn, P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5060-5072</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Some selected amino acids, in particular L-aspartic acid (L-Asp) and L-histidine (L-His), can function as leaving group during polymerase-catalyzed incorporation of deoxyadenosine monophosphate (dAMP) in DNA.  Although L-Asp-dAMP and L-His-dAMP bind, most probably, in a different way in the active site of the enzyme, aspartic acid and histidine can be considered as mimics of the pyrophosphate moiety of deoxyadenosine triphosphate.  L-Aspartic acid is more efficient than D-aspartic acid as leaving group.  Such P-N conjugates of amino acids and deoxynucleotides provide a novel exptl. ground for diversifying nucleic acid metab. in the field of synthetic biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquqe7h6NI-V7Vg90H21EOLACvtfcHk0liVZ7jE_eo0yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFWgsLs%253D&md5=9bad158cc983934910ca44c87b77a573</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm498%26sid%3Dliteratum%253Aachs%26aulast%3DAdelfinskaya%26aufirst%3DO.%26aulast%3DTerrazas%26aufirst%3DM.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DMarliere%26aufirst%3DP.%26aulast%3DNauwelaerts%26aufirst%3DK.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DPolymerase-catalyzed%2520synthesis%2520of%2520DNA%2520from%2520phosphoramidate%2520conjugates%2520of%2520deoxynucleotides%2520and%2520amino%2520acids%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D5060%26epage%3D5072%26doi%3D10.1093%2Fnar%2Fgkm498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Terrazas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Enzymatically Catalyzed DNA Synthesis Using L-Asp-dGMP, L-Asp-dCMP, and L-Asp-dTMP</span>. <i>Chem. Biodivers.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1002/cbdv.200890013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbdv.200890013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=18205125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BD1c%252Fnt12nsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=31-39&author=M.+Terrazasauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=Enzymatically+Catalyzed+DNA+Synthesis+Using+L-Asp-dGMP%2C+L-Asp-dCMP%2C+and+L-Asp-dTMP&doi=10.1002%2Fcbdv.200890013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymatically catalyzed DNA synthesis using L-Asp-dGMP, L-Asp-dCMP, and L-Asp-dTMP</span></div><div class="casAuthors">Terrazas Montserrat; Marliere Philippe; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & biodiversity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The replacement of the pyrophosphate moiety of deoxynucleoside triphosphates by L-aspartic acid allows incorporation of natural deoxynucleosides into DNA using HIV reverse transcriptase (RT) as enzyme, while retaining the canonical base-pair selectivity.  N-Methylation of the L-aspartic acid leaving group results in a reduced fidelity of incorporation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXucBseKJbyZggzHGM8X5CfW6udTcc2eZHA417iJXFm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c%252Fnt12nsA%253D%253D&md5=7a3af442dc2967b128d4acefca0f8f64</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.200890013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.200890013%26sid%3Dliteratum%253Aachs%26aulast%3DTerrazas%26aufirst%3DM.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DEnzymatically%2520Catalyzed%2520DNA%2520Synthesis%2520Using%2520L-Asp-dGMP%252C%2520L-Asp-dCMP%252C%2520and%2520L-Asp-dTMP%26jtitle%3DChem.%2520Biodivers.%26date%3D2008%26volume%3D5%26spage%3D31%26epage%3D39%26doi%3D10.1002%2Fcbdv.200890013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giraut, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-p.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Iminodiacetic-phosphoramidates as metabolic prototypes for diversifying nucleic acid polymerization in vivo</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">2541</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp1246</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkp1246" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20097909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsFGnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=2541-2550&author=A.+Girautauthor=X.-p.+Songauthor=M.+Froeyenauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=Iminodiacetic-phosphoramidates+as+metabolic+prototypes+for+diversifying+nucleic+acid+polymerization+in+vivo&doi=10.1093%2Fnar%2Fgkp1246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Iminodiacetic-phosphoramidates as metabolic prototypes for diversifying nucleic acid polymerization in vivo</span></div><div class="casAuthors">Giraut, Anne; Song, Xiao-Ping; Froeyen, Matheus; Marliere, Philippe; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2541-2550</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Previous studies in our lab. proved that certain functional groups are able to mimic the pyrophosphate moiety and act as leaving groups in the enzymic polymn. of DNAs by HIV-1 reverse transcriptase.  When the potential leaving group possesses two carboxylic acid moieties linked to the nucleoside via a phosphoramidate bond, it is efficiently recognized by this error-prone enzyme, resulting in nucleotide incorporation into DNA.  Here, we present a new efficient alternative leaving group, iminodiacetic acid, which displays enhanced kinetics and an enhanced elongation capacity compared to previous results obtained with amino acid deoxyadenosine phosphoramidates.  Iminodiacetic acid phosphoramidate of deoxyadenosine monophosphate (IDA-dAMP) is processed by HIV-1 RT as a substrate for single nucleotide incorporation and displays a typical Michaelis-Menten kinetic profile.  This novel substrate also proved to be successful in primer strand elongation of a seven-base template overhang.  Modeling of this new substrate in the active site of the enzyme revealed that the interactions formed between the triphosphate moiety, magnesium ions and enzyme's residues could be different from those of the natural triphosphate substrate and is likely to involve addnl. amino acid residues.  Preliminary testing for a potential metabolic accessibility lets us to envision its possible use in an orthogonal system for nucleic acid synthesis that would not influence or be influenced by genetic information from the outside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobcJNqBOdKGbVg90H21EOLACvtfcHk0liVZ7jE_eo0yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsFGnu74%253D&md5=5b632d1c2cb21440cf6ead8bf69aa494</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp1246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp1246%26sid%3Dliteratum%253Aachs%26aulast%3DGiraut%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DX.-p.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DIminodiacetic-phosphoramidates%2520as%2520metabolic%2520prototypes%2520for%2520diversifying%2520nucleic%2520acid%2520polymerization%2520in%2520vivo%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D2541%26epage%3D2550%26doi%3D10.1093%2Fnar%2Fgkp1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Iminodipropionic acid as the leaving group for DNA polymerization by HIV-1 reverse transcriptase</span>. <i>Chembiochem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1868</span>– <span class="NLM_lpage">1880</span>, <span class="refDoi"> DOI: 10.1002/cbic.201100160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbic.201100160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21714056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotVCmu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1868-1880&author=X.-P.+Songauthor=C.+Bouillonauthor=E.+Lescrinierauthor=P.+Herdewijn&title=Iminodipropionic+acid+as+the+leaving+group+for+DNA+polymerization+by+HIV-1+reverse+transcriptase&doi=10.1002%2Fcbic.201100160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Iminodipropionic Acid as the Leaving Group for DNA Polymerization by HIV-1 Reverse Transcriptase</span></div><div class="casAuthors">Song, Xiao-Ping; Bouillon, Camille; Lescrinier, Eveline; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1868-1880</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Previous studies have demonstrated that some selected amino monoacids and amino diacids can function as leaving groups in the polymerase-catalyzed incorporation of deoxynucleotides into DNA.  Among these, the iminodiacetic acid phosphoramidate of deoxyadenosine monophosphate (IDA-dAMP) represents an interesting example, as it could overcome some of the problems obsd. when using L-aspartic acid as the leaving group, i.e., poor chain elongation.  We have now synthesized and evaluated a series of IDA-dAMP analogs that bear either an extended aliph. chain in the amino acid function, or a phosphonic acid moiety (substituting for the carboxylic acid function).  Among these compds., the nucleotide with an iminodipropionic acid leaving group (IDP-dAMP) was identified as the best substrate; the excellent single incorporation (91% conversion to a P+1 strand at 50 μM) was at a substrate concn. ten times lower than that used for IDA-dAMP.  This nucleotide also presented improved kinetics and elongation capability compared to IDA-dAMP.  The analogs with T, G, and C base moieties were also investigated for their incorporation ability with HIV-1 RT.  The incorporation efficiency was found to decrease in the order A > T > G > C.  The properties of the iminodipropionic acid as the leaving group surpass those of previously evaluated leaving groups; this acid will be a prime candidate for in vivo testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjsahfrtqUgLVg90H21EOLACvtfcHk0ljH_XtDB6JK8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotVCmu78%253D&md5=eea9ba713fb278dd1ab7589500d9dc57</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201100160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201100160%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.-P.%26aulast%3DBouillon%26aufirst%3DC.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DIminodipropionic%2520acid%2520as%2520the%2520leaving%2520group%2520for%2520DNA%2520polymerization%2520by%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DChembiochem%26date%3D2011%26volume%3D12%26spage%3D1868%26epage%3D1880%26doi%3D10.1002%2Fcbic.201100160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlière, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">3-Phosphono-L-alanine as pyrophosphate mimic for DNA synthesis using HIV-1 reverse transcriptase</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/c0ob00554a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1039%2FC0OB00554A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21103490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=111-119&author=S.+Yangauthor=M.+Froeyenauthor=E.+Lescrinierauthor=P.+Marli%C3%A8reauthor=P.+Herdewijn&title=3-Phosphono-L-alanine+as+pyrophosphate+mimic+for+DNA+synthesis+using+HIV-1+reverse+transcriptase&doi=10.1039%2Fc0ob00554a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">3-Phosphono-L-alanine as pyrophosphate mimic for DNA synthesis using HIV-1 reverse transcriptase</span></div><div class="casAuthors">Yang, Shiqiong; Froeyen, Mathy; Lescrinier, Eveline; Marliere, Philippe; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of sulf(on)ate and phosph(on)ate amino acid phosphoramidate analogs of deoxynucleotides were synthesized as potential substrates for HIV-1 reverse transcriptase.  Taurine, L-cysteic acid, 3-phosphono-L-alanine, O-sulfonato-L-serine, and O-phospho-l-serine were investigated as leaving groups in an enzyme catalyzed DNA synthesis protocol.  Among these analogs, the phosphonate congener performed best and 3-phosphono-L-alanine can be considered as an excellent mimic of the pyrophosphate (PPi) moiety of deoxyadenosine triphosphate, to be used in enzymic synthesis of nucleic acids.  During a single nucleotide incorporation assay the use of 3-phosphono-L-Ala-dAMP as substrate resulted in 95% conversion to a P + 1 strand in 60 min at 50 μM (a concn. 10 times less than found for L-Asp-dAMP) and with improved incorporation kinetics and less stalling.  For the sequences investigated, the efficiency of the incorporation is base dependent and decreases in the order (A ≥ T = G > C).  In all cases, the incorporation follows Watson-Crick rules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeYzCNJ85TSbVg90H21EOLACvtfcHk0ljH_XtDB6JK8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FO&md5=c9ab71936ae04d10cd5a787f06feb556</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC0OB00554A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00554A%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DMarli%25C3%25A8re%26aufirst%3DP.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3D3-Phosphono-L-alanine%2520as%2520pyrophosphate%2520mimic%2520for%2520DNA%2520synthesis%2520using%2520HIV-1%2520reverse%2520transcriptase%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D111%26epage%3D119%26doi%3D10.1039%2Fc0ob00554a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Dipeptides as leaving group in the enzyme-catalyzed DNA synthesis</span>. <i>Chem. Biodivers.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2685</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1002/cbdv.201200261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fcbdv.201200261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=23255441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jsVOrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=2685-2700&author=X.-P.+Songauthor=C.+Bouillonauthor=E.+Lescrinierauthor=P.+Herdewijn&title=Dipeptides+as+leaving+group+in+the+enzyme-catalyzed+DNA+synthesis&doi=10.1002%2Fcbdv.201200261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Dipeptides as leaving group in the enzyme-catalyzed DNA synthesis</span></div><div class="casAuthors">Song Xiao-Ping; Bouillon Camille; Lescrinier Eveline; Herdewijn Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & biodiversity</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2685-700</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Conjugates of 2'-deoxyadenosine monophosphate with dipeptides have been synthesized and tested as substrates for several polymerases.  Although the incorporation efficiency is not very high, it demonstrates that some of these dipeptides can be accommodated in the active site of polymerases and function as leaving groups in the enzymatic synthesis of DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8LuHMtpFx6CA3UY9EwTNrfW6udTcc2ebj5eJpKgEV0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jsVOrtA%253D%253D&md5=7497c0debe23bc82146910349a07e2f9</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fcbdv.201200261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbdv.201200261%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.-P.%26aulast%3DBouillon%26aufirst%3DC.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DDipeptides%2520as%2520leaving%2520group%2520in%2520the%2520enzyme-catalyzed%2520DNA%2520synthesis%26jtitle%3DChem.%2520Biodivers.%26date%3D2012%26volume%3D9%26spage%3D2685%26epage%3D2700%26doi%3D10.1002%2Fcbdv.201200261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalman, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntee, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, C. R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Biological Activity of Aromatic Amino Acid Phosphoramidates of 5-Fluoro-2’-deoxyuridine and 1-β-Arabinofuranosylcytosine: Evidence of Phosphoramidase Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">4569</span>– <span class="NLM_lpage">4575</span>, <span class="refDoi"> DOI: 10.1021/jm9603680</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9603680" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=4569-4575&author=T.+W.+Abrahamauthor=T.+I.+Kalmanauthor=E.+J.+McInteeauthor=C.+R.+Wagner&title=Synthesis+and+Biological+Activity+of+Aromatic+Amino+Acid+Phosphoramidates+of+5-Fluoro-2%E2%80%99-deoxyuridine+and+1-%CE%B2-Arabinofuranosylcytosine%3A+Evidence+of+Phosphoramidase+Activity&doi=10.1021%2Fjm9603680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Biological Activity of Aromatic Amino Acid Phosphoramidates of 5-Fluoro-2'-deoxyuridine and 1-β-Arabinofuranosylcytosine: Evidence of Phosphoramidase Activity</span></div><div class="casAuthors">Abraham, Timothy W.; Kalman, Thomas I.; McIntee, Edward J.; Wagner, Carston R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4569-4575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The amino acid phosphoramidate diesters of FUdR and Ara-C, fluorodeoxyuridyl N-(carbomethoxyphenylethyl)phosphoramidate I, 5-fluoro-2'-deoxy-5'-uridyl N-(1-carbomethoxy-2-indolylethyl)phosphoramidate, 1-β-arabinofuranosylcytosine 5'-N-(1-carbomethoxy-2-phenylethyl)phosphoramidate, and 1-β-arabinofuranosylcytosine 5'-N-(1-carbomethoxy-2-indolylethyl)phosphoramidate, were synthesized and tested for their antitumor activity against L1210 mouse lymphocytic leukemia cells and CCRF-CEM human T-cell lymphoblastic leukemia cells.  Stability studies revealed that >99% of the phosphoramidates remained intact after incubation in 20% calf or 20% human serum.  Intracellular thymidylate synthase (TS) inhibition studies revealed that treatment of L1210 and CCRF-CEM cells with I resulted in significant inhibition of TS in intact and permeabilized cells, while treatment of L929 TK- cells with these compds. did not result in inhibition of TS activity in intact cells.  However, permeabilization of L929 TK- cells enhanced the activity of 5a,b toward intracellular TS by 900- and 1500-fold, resp.  Consequently, it is proposed that the metab. of the phosphoramidate diesters of FUdR in proliferating tissue proceeds through two sep. enzymic steps involving P-N bond cleavage by an unknown phosphoramidase followed by P-O bond cleavage by phosphatases such as 5'-nucleotidase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmqZMoJ5CQFrVg90H21EOLACvtfcHk0li_iC4aJxF3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1ylsL8%253D&md5=2698a91970ae6f18d34c68b6e885997c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm9603680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9603680%26sid%3Dliteratum%253Aachs%26aulast%3DAbraham%26aufirst%3DT.%2BW.%26aulast%3DKalman%26aufirst%3DT.%2BI.%26aulast%3DMcIntee%26aufirst%3DE.%2BJ.%26aulast%3DWagner%26aufirst%3DC.%2BR.%26atitle%3DSynthesis%2520and%2520Biological%2520Activity%2520of%2520Aromatic%2520Amino%2520Acid%2520Phosphoramidates%2520of%25205-Fluoro-2%25E2%2580%2599-deoxyuridine%2520and%25201-%25CE%25B2-Arabinofuranosylcytosine%253A%2520Evidence%2520of%2520Phosphoramidase%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D4569%26epage%3D4575%26doi%3D10.1021%2Fjm9603680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singer, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haugland, R. P.</span></span> <span> </span><span class="NLM_article-title">Characterization of PicoGreen reagent and development of a fluorescence-based solution assay for double-stranded DNA quantitation</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>249</i></span>,  <span class="NLM_fpage">228</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1006/abio.1997.2177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1006%2Fabio.1997.2177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=9212875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2sXktF2rsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1997&pages=228-238&author=V.+L.+Singerauthor=L.+J.+Jonesauthor=S.+T.+Yueauthor=R.+P.+Haugland&title=Characterization+of+PicoGreen+reagent+and+development+of+a+fluorescence-based+solution+assay+for+double-stranded+DNA+quantitation&doi=10.1006%2Fabio.1997.2177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of PicoGreen reagent and development of a fluorescence-based solution assay for double-stranded DNA quantitation</span></div><div class="casAuthors">Singer, Victoria L.; Jones, Laurie J.; Yue, Stephen T.; Haugland, Richard P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">228-238</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A sensitive assay for detecting double-stranded (ds) DNA in soln. is described.  This assay employs a new dye, PicoGreen dsDNA quantitation reagent, which becomes intensely fluorescent upon binding nucleic acids.  The brightness of this reagent is due to its high quantum yield (∼0.5, bound to ds calf thymus DNA) and large molar extinction coeff. (∼70,000 cm-1 M-1).  The fluorescence enhancement of this dye upon binding dsDNA is >1000-fold, with excitation and emission maxima near those of fluorescein.  Unlike Hoechst 33258, PicoGreen reagent fluorescence intensity was the same upon binding to poly(dA)·poly(dT) and poly(dG)·poly(dC) homopolymers.  The PicoGreen assay allowed the detection of 25 pg/mL dsDNA, surpassing the sensitivity achieved with Hoechst 33258 by 400-fold.  The linear concn. range for DNA quantitation extended over four orders of magnitude-25 pg/mL to 1 μg/m-with a single dye concn.  Assay linearity was maintained even in the presence of salts, proteins, poly(ethylene glycol), urea, chloroform, ethanol, and agarose; some ionic detergents and heparin interfered.  Linear DNAs yielded slightly brighter signals than supercoiled plasmids.  Finally, the assay showed greater dsDNA:RNA selectivity than Hoechst 33258 in low ionic strength buffer and better dsDNA:single-stranded DNA selectivity in 1 M NaCl.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomOWcQ1NrPjLVg90H21EOLACvtfcHk0li_iC4aJxF3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktF2rsbo%253D&md5=d331528a1da7ed5b724022a0e53139d5</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1006%2Fabio.1997.2177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1997.2177%26sid%3Dliteratum%253Aachs%26aulast%3DSinger%26aufirst%3DV.%2BL.%26aulast%3DJones%26aufirst%3DL.%2BJ.%26aulast%3DYue%26aufirst%3DS.%2BT.%26aulast%3DHaugland%26aufirst%3DR.%2BP.%26atitle%3DCharacterization%2520of%2520PicoGreen%2520reagent%2520and%2520development%2520of%2520a%2520fluorescence-based%2520solution%2520assay%2520for%2520double-stranded%2520DNA%2520quantitation%26jtitle%3DAnal.%2520Biochem.%26date%3D1997%26volume%3D249%26spage%3D228%26epage%3D238%26doi%3D10.1006%2Fabio.1997.2177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">HIV-1 reverse transcriptase complex with DNA and nevirapine reveals non-nucleoside inhibition mechanism</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnsmb.2223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22266819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVGqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=253-259&author=K.+Dasauthor=S.+E.+Martinezauthor=J.+D.+Baumanauthor=E.+Arnold&title=HIV-1+reverse+transcriptase+complex+with+DNA+and+nevirapine+reveals+non-nucleoside+inhibition+mechanism&doi=10.1038%2Fnsmb.2223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 reverse transcriptase complex with DNA and nevirapine reveals non-nucleoside inhibition mechanism</span></div><div class="casAuthors">Das, Kalyan; Martinez, Sergio E.; Bauman, Joseph D.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">253-259</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Combinations of nucleoside and non-nucleoside inhibitors (NNRTIs) of HIV-1 reverse transcriptase (RT) are widely used in anti-AIDS therapies.  Five NNRTIs, including nevirapine, are clin. drugs; however, the mol. mechanism of inhibition by NNRTIs is not clear.  We detd. the crystal structures of RT-DNA-nevirapine, RT-DNA, and RT-DNA-AZT-triphosphate complexes at 2.85-, 2.70-, and 2.80-Å resoln., resp.  The RT-DNA complex in the crystal could bind nevirapine or AZT-triphosphate but not both.  Binding of nevirapine led to opening of the NNRTI-binding pocket.  The pocket formation caused shifting of the 3' end of the DNA primer by ∼5.5 Å away from its polymerase active site position.  Nucleic acid interactions with fingers and palm subdomains were reduced, the dNTP-binding pocket was distorted and the thumb opened up.  The structures elucidate complementary roles of nucleoside and non-nucleoside inhibitors in inhibiting RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxsRKzSWU5S7Vg90H21EOLACvtfcHk0li_iC4aJxF3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVGqtrg%253D&md5=8acb0ebef279d7fe999080d12298792c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2223%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHIV-1%2520reverse%2520transcriptase%2520complex%2520with%2520DNA%2520and%2520nevirapine%2520reveals%2520non-nucleoside%2520inhibition%2520mechanism%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D253%26epage%3D259%26doi%3D10.1038%2Fnsmb.2223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandwar, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structures of HIV-1 RT-RNA/DNA ternary complexes with dATP and nevirapine reveal conformational flexibility of RNA/DNA: insights into requirements for RNase H cleavage</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">8125</span>– <span class="NLM_lpage">8137</span>, <span class="refDoi"> DOI: 10.1093/nar/gku487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgku487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=24880687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=8125-8137&author=K.+Dasauthor=S.+E.+Martinezauthor=R.+P.+Bandwarauthor=E.+Arnold&title=Structures+of+HIV-1+RT-RNA%2FDNA+ternary+complexes+with+dATP+and+nevirapine+reveal+conformational+flexibility+of+RNA%2FDNA%3A+insights+into+requirements+for+RNase+H+cleavage&doi=10.1093%2Fnar%2Fgku487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of HIV-1 RT-RNA/DNA ternary complexes with dATP and nevirapine reveal conformational flexibility of RNA/DNA: insights into requirements for RNase H cleavage</span></div><div class="casAuthors">Das, Kalyan; Martinez, Sergio E.; Bandwar, Rajiv P.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8125-8137</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In synthesizing a double-stranded DNA from viral RNA, HIV-1 reverse transcriptase (RT) generates an RNA/DNA intermediate.  RT also degrades the RNA strand and synthesizes the second DNA strand.  The RNase H active site of RT functions as a nuclease to cleave the RNA strand; however, the structural basis for endonucleolytic cleavage of the RNA strand remains elusive.  Here we report crystal structures of RT-RNA/DNA-dATP and RT-RNA/DNA-nevirapine (NVP) ternary complexes at 2.5 and 2.9 Å resoln., resp.  The polymerase region of RT-RNA/DNA-dATP complex resembles DNA/DNA ternary complexes apart from addnl. interactions of 2'-OH groups of the RNA strand.  The conformation and binding of RNA/DNA deviates significantly after the seventh nucleotide vs. a DNA/DNA substrate.  Binding of NVP slides the RNA/DNA non-uniformly over RT, and the RNA strand moves closer to the RNase H active site.  Two addnl. structures, one contg. a gapped RNA and another a bulged RNA, reveal that conformational changes of an RNA/DNA and increased interactions with the RNase H domain, including the interaction of a 2'-OH with N474, help to position the RNA nearer to the active site.  The structures and existing biochem. data suggest a nucleic acid conformation-induced mechanism for guiding cleavage of the RNA strand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt_GxtIkYGgLVg90H21EOLACvtfcHk0li_iC4aJxF3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnO&md5=3f7b7e2daa6511407dc1d292420c275d</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku487%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DMartinez%26aufirst%3DS.%2BE.%26aulast%3DBandwar%26aufirst%3DR.%2BP.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructures%2520of%2520HIV-1%2520RT-RNA%252FDNA%2520ternary%2520complexes%2520with%2520dATP%2520and%2520nevirapine%2520reveal%2520conformational%2520flexibility%2520of%2520RNA%252FDNA%253A%2520insights%2520into%2520requirements%2520for%2520RNase%2520H%2520cleavage%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D8125%26epage%3D8137%26doi%3D10.1093%2Fnar%2Fgku487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandwar, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the role of the K65R mutation in HIV-1 reverse transcriptase polymerization, excision antagonism, and tenofovir resistance</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">35092</span>– <span class="NLM_lpage">35100</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.022525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M109.022525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=19812032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWitbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=35092-35100&author=K.+Dasauthor=R.+P.+Bandwarauthor=K.+L.+Whiteauthor=J.+Y.+Fengauthor=S.+G.+Sarafianosauthor=S.+Tuskeauthor=X.+Tuauthor=A.+D.+Clarkauthor=P.+L.+Boyerauthor=X.+Houauthor=B.+L.+Gaffneyauthor=R.+A.+Jonesauthor=M.+D.+Millerauthor=S.+H.+Hughesauthor=E.+Arnold&title=Structural+basis+for+the+role+of+the+K65R+mutation+in+HIV-1+reverse+transcriptase+polymerization%2C+excision+antagonism%2C+and+tenofovir+resistance&doi=10.1074%2Fjbc.m109.022525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Role of the K65R Mutation in HIV-1 Reverse Transcriptase Polymerization, Excision Antagonism, and Tenofovir Resistance</span></div><div class="casAuthors">Das, Kalyan; Bandwar, Rajiv P.; White, Kirsten L.; Feng, Joy Y.; Sarafianos, Stefan G.; Tuske, Steven; Tu, Xiong-Ying; Clark, Arthur D., Jr.; Boyer, Paul L.; Hou, Xiao-Rong; Gaffney, Barbara L.; Jones, Roger A.; Miller, Michael D.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">35092-35100</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">K65R is a primary reverse transcriptase (RT) mutation selected in human immunodeficiency virus type 1-infected patients taking antiretroviral regimens contg. tenofovir disoproxil fumarate or other nucleoside analog RT drugs.  The authors detd. the crystal structures of K65R mutant RT crosslinked to double-stranded DNA and in complexes with tenofovir diphosphate (TFV-DP) or dATP.  The crystals permit substitution of TFV-DP with dATP at the dNTP-binding site.  The guanidinium planes of the arginines K65R and Arg72 were stacked to form a mol. platform that restricts the conformational adaptability of both of the residues, which explains the neg. effects of the K65R mutation on nucleotide incorporation and on excision.  Furthermore, the guanidinium planes of K65R and Arg72 were stacked in two different rotameric conformations in TFV-DP- and dATP-bound structures that may help explain how K65R RT discriminates the drug from substrates.  These K65R-mediated effects on RT structure and function help the authors to visualize the complex interaction with other key nucleotide RT drug resistance mutations, such as M184V, L74V, and thymidine analog resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVMu34Ijj-abVg90H21EOLACvtfcHk0lgqHMuW_5rn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWitbzF&md5=3bda12aca3dbce964898817070c306c1</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.022525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.022525%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBandwar%26aufirst%3DR.%2BP.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DTu%26aufirst%3DX.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructural%2520basis%2520for%2520the%2520role%2520of%2520the%2520K65R%2520mutation%2520in%2520HIV-1%2520reverse%2520transcriptase%2520polymerization%252C%2520excision%2520antagonism%252C%2520and%2520tenofovir%2520resistance%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D35092%26epage%3D35100%26doi%3D10.1074%2Fjbc.m109.022525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modak, M. J.</span></span> <span> </span><span class="NLM_article-title">Site-directed Mutagenesis of Arginine 72 of HIV-1 Reverse Transcriptase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">19729</span>– <span class="NLM_lpage">19735</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.34.19729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.270.34.19729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=7544345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2MXnvVCmtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=19729-19735&author=S.+G.+Sarafianosauthor=V.+N.+Pandeyauthor=N.+Kaushikauthor=M.+J.+Modak&title=Site-directed+Mutagenesis+of+Arginine+72+of+HIV-1+Reverse+Transcriptase&doi=10.1074%2Fjbc.270.34.19729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Site-directed mutagenesis of arginine 72 of HIV-1 reverse transcriptase. Catalytic role and inhibitor sensitivity</span></div><div class="casAuthors">Sarafianos, Stefanos g.; Pandey, Virendra N.; Kaushik, Neerja; Modak, Mukund J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">19729-35</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">To det. the catalytic role of Arg72 of HIV-1 reverse transcriptase (RT), the authors carried out site-directed mutagenesis at codon 72.  Two mutant proteins (R72A and R72K) were purified and characterized.  With Arg to Ala substitution the kcat of the polymerase reaction was reduced by nearly 100-fold with poly(rA) template, but only about 5-15-fold with poly(rC) and poly(dC) templates.  The Arg to Lys substitution exhibited a qual. similar pattern, although the overall redn. in kcat was less severe.  Most interestingly, the authors noted a large difference in the rate const. of the first and second nucleotide incorporation by R72A, suggesting that Arg72 participates in the reaction after the formation of the first phosphodiester bond.  The authors propose this step to be the pyrophosphate binding and removal step following the nucleotidyltransferase reaction.  Support for this proposal is obtained from the observation that the R72A mutant (i) exhibited a pronounced translocation defect in the processivity anal., (ii) lacked the ability to catalyze pyrophosphorolysis, and (iii) showed complete resistance to phosphonoformate, an analog of PPi.  Arg72 is the first residue of HIV-1 RT proposed to be involved in the pyrophosphate binding/removal function of RT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGUwF5rYC_-LVg90H21EOLACvtfcHk0lgqHMuW_5rn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvVCmtLg%253D&md5=2ebbd8aa30c11149c7bec11e944c1c8f</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.34.19729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.34.19729%26sid%3Dliteratum%253Aachs%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DPandey%26aufirst%3DV.%2BN.%26aulast%3DKaushik%26aufirst%3DN.%26aulast%3DModak%26aufirst%3DM.%2BJ.%26atitle%3DSite-directed%2520Mutagenesis%2520of%2520Arginine%252072%2520of%2520HIV-1%2520Reverse%2520Transcriptase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D19729%26epage%3D19735%26doi%3D10.1074%2Fjbc.270.34.19729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naeger, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Structures of HIV-1 RT-DNA complexes before and after incorporation of the anti-AIDS drug tenofovir</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">469</span>– <span class="NLM_lpage">474</span>, <span class="refDoi"> DOI: 10.1038/nsmb760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnsmb760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=15107837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVCqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=469-474&author=S.+Tuskeauthor=S.+G.+Sarafianosauthor=A.+D.+Clarkauthor=J.+Dingauthor=L.+K.+Naegerauthor=K.+L.+Whiteauthor=M.+D.+Millerauthor=C.+S.+Gibbsauthor=P.+L.+Boyerauthor=P.+Clarkauthor=G.+Wangauthor=B.+L.+Gaffneyauthor=R.+A.+Jonesauthor=D.+M.+Jerinaauthor=S.+H.+Hughesauthor=E.+Arnold&title=Structures+of+HIV-1+RT-DNA+complexes+before+and+after+incorporation+of+the+anti-AIDS+drug+tenofovir&doi=10.1038%2Fnsmb760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of HIV-1 RT-DNA complexes before and after incorporation of the anti-AIDS drug tenofovir</span></div><div class="casAuthors">Tuske, Steve; Sarafianos, Stefan G.; Clark, Arthur D.; Ding, Jianping; Naeger, Lisa K.; White, Kirsten L.; Miller, Michael D.; Gibbs, Craig S.; Boyer, Paul L.; Clark, Patrick; Wang, Gang; Gaffney, Barbara L.; Jones, Roger A.; Jerina, Donald M.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">469-474</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tenofovir, also known as PMPA, R-9-2-(phosphonomethoxypropyl)adenine, is a nucleotide reverse transcriptase (RT) inhibitor.  We have detd. the crystal structures of two related complexes of HIV-1 RT with template primer and tenofovir: (i) a ternary complex at a resoln. of 3.0 Å of RT crosslinked to a dideoxy-terminated DNA with tenofovir-diphosphate bound as the incoming substrate; and (ii) a RT-DNA complex at a resoln. of 3.1 Å with tenofovir at the 3' primer terminus.  The tenofovir nucleotide in the tenofovir-terminated structure seems to adopt multiple conformations.  Some nucleoside reverse transcriptase inhibitors, including 3TC and AZT, have elements ('handles') that project beyond the corresponding elements on normal dNTPs (the 'substrate envelope').  HIV-1 RT resistance mechanisms to AZT and 3TC take advantage of these handles; tenofovir's structure lacks handles that could protrude through the substrate envelope to cause resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi1ei68A2R6bVg90H21EOLACvtfcHk0lgqHMuW_5rn5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVCqsL8%253D&md5=da428445f0276f07fd66dbde46732fbf</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnsmb760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb760%26sid%3Dliteratum%253Aachs%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DNaeger%26aufirst%3DL.%2BK.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DGibbs%26aufirst%3DC.%2BS.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DClark%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DJerina%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DStructures%2520of%2520HIV-1%2520RT-DNA%2520complexes%2520before%2520and%2520after%2520incorporation%2520of%2520the%2520anti-AIDS%2520drug%2520tenofovir%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D469%26epage%3D474%26doi%3D10.1038%2Fnsmb760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1499</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1126/science.abc1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1126%2Fscience.abc1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=32358203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1499-1504&author=W.+Yinauthor=C.+Maoauthor=X.+Luanauthor=D.-D.+Shenauthor=Q.+Shenauthor=H.+Suauthor=X.+Wangauthor=F.+Zhouauthor=W.+Zhaoauthor=M.+Gaoauthor=S.+Changauthor=Y.-C.+Xieauthor=G.+Tianauthor=H.-W.+Jiangauthor=S.-C.+Taoauthor=J.+Shenauthor=Y.+Jiangauthor=H.+Jiangauthor=Y.+Xuauthor=S.+Zhangauthor=Y.+Zhangauthor=H.+E.+Xu&title=Structural+basis+for+inhibition+of+the+RNA-dependent+RNA+polymerase+from+SARS-CoV-2+by+remdesivir&doi=10.1126%2Fscience.abc1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span></div><div class="casAuthors">Yin, Wanchao; Mao, Chunyou; Luan, Xiaodong; Shen, Dan-Dan; Shen, Qingya; Su, Haixia; Wang, Xiaoxi; Zhou, Fulai; Zhao, Wenfeng; Gao, Minqi; Chang, Shenghai; Xie, Yuan-Chao; Tian, Guanghui; Jiang, He-Wei; Tao, Sheng-Ce; Shen, Jingshan; Jiang, Yi; Jiang, Hualiang; Xu, Yechun; Zhang, Shuyang; Zhang, Yan; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">1499-1504</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has become a global crisis.  Replication of SARS-CoV-2 requires the viral RNA-dependent RNA polymerase (RdRp) enzyme, a target of the antiviral drug remdesivir.  Here, we report the cryo-electron microscopy structure of the SARS-CoV-2 RdRp, both in the apo form at 2.8-angstrom resoln. and in complex with a 50-base template-primer RNA and remdesivir at 2.5-angstrom resoln.  The complex structure reveals that the partial double-stranded RNA template is inserted into the central channel of the RdRp, where remdesivir is covalently incorporated into the primer strand at the first replicated base pair, and terminates chain elongation.  Our structures provide insights into the mechanism of viral RNA replication and a rational template for drug design to combat the viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlizem_f4e1LVg90H21EOLACvtfcHk0lgEkCi61gTGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK&md5=e5d7c8a1c27a02fdbfa9381482a0c3e8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc1560%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLuan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DD.-D.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.-C.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.-W.%26aulast%3DTao%26aufirst%3DS.-C.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520RNA-dependent%2520RNA%2520polymerase%2520from%2520SARS-CoV-2%2520by%2520remdesivir%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1499%26epage%3D1504%26doi%3D10.1126%2Fscience.abc1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hillen, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokic, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farnung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dienemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegunov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, P.</span></span> <span> </span><span class="NLM_article-title">Structure of replicating SARS-CoV-2 polymerase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>584</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2368-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fs41586-020-2368-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=32438371" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=584&publication_year=2020&pages=154-156&author=H.+S.+Hillenauthor=G.+Kokicauthor=L.+Farnungauthor=C.+Dienemannauthor=D.+Tegunovauthor=P.+Cramer&title=Structure+of+replicating+SARS-CoV-2+polymerase&doi=10.1038%2Fs41586-020-2368-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of replicating SARS-CoV-2 polymerase</span></div><div class="casAuthors">Hillen, Hauke S.; Kokic, Goran; Farnung, Lucas; Dienemann, Christian; Tegunov, Dimitry; Cramer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">584</span>
        (<span class="NLM_cas:issue">7819</span>),
    <span class="NLM_cas:pages">154-156</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The new coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) uses an RNA-dependent RNA polymerase (RdRp) for the replication of its genome and the transcription of its genes1-3.  Here we present a cryo-electron microscopy structure of the SARS-CoV-2 RdRp in an active form that mimics the replicating enzyme.  The structure comprises the viral proteins non-structural protein 12 (nsp12), nsp8 and nsp7, and more than two turns of RNA template-product duplex.  The active-site cleft of nsp12 binds to the first turn of RNA and mediates RdRp activity with conserved residues.  Two copies of nsp8 bind to opposite sides of the cleft and position the second turn of RNA.  Long helical extensions in nsp8 protrude along exiting RNA, forming pos. charged 'sliding poles'.  These sliding poles can account for the known processivity of RdRp that is required for replicating the long genome of coronaviruses3.  Our results enable a detailed anal. of the inhibitory mechanisms that underlie the antiviral activity of substances such as remdesivir, a drug for the treatment of coronavirus disease 2019 (COVID-19)4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvvV9anYxFpbVg90H21EOLACvtfcHk0lgEkCi61gTGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVygtLjN&md5=a6c1595799a31219d6a54228f5df82c9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2368-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2368-8%26sid%3Dliteratum%253Aachs%26aulast%3DHillen%26aufirst%3DH.%2BS.%26aulast%3DKokic%26aufirst%3DG.%26aulast%3DFarnung%26aufirst%3DL.%26aulast%3DDienemann%26aufirst%3DC.%26aulast%3DTegunov%26aufirst%3DD.%26aulast%3DCramer%26aufirst%3DP.%26atitle%3DStructure%2520of%2520replicating%2520SARS-CoV-2%2520polymerase%26jtitle%3DNature%26date%3D2020%26volume%3D584%26spage%3D154%26epage%3D156%26doi%3D10.1038%2Fs41586-020-2368-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michielssens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyubankova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceulemans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Influence of the nucleobase and anchimeric assistance of the carboxyl acid groups in the hydrolysis of amino acid nucleoside phosphoramidates</span>. <i>Chemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">857</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1002/chem.201102279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1002%2Fchem.201102279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22173724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1elur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=857-868&author=M.+Maitiauthor=S.+Michielssensauthor=N.+Dyubankovaauthor=M.+Maitiauthor=E.+Lescrinierauthor=A.+Ceulemansauthor=P.+Herdewijn&title=Influence+of+the+nucleobase+and+anchimeric+assistance+of+the+carboxyl+acid+groups+in+the+hydrolysis+of+amino+acid+nucleoside+phosphoramidates&doi=10.1002%2Fchem.201102279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Influence of the Nucleobase and Anchimeric Assistance of the Carboxyl Acid Groups in the Hydrolysis of Amino Acid Nucleoside Phosphoramidates</span></div><div class="casAuthors">Maiti, Munmun; Michielssens, Servaas; Dyubankova, Natalia; Maiti, Mohitosh; Lescrinier, Eveline; Ceulemans, Arnout; Herdewijn, Piet</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">857-868, S857/1-S857/35</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Nucleoside phosphoramidates (NPs) are a class of nucleotide analogs that has been developed as potential antiviral/antitumor prodrugs.  Recently, we have shown that some amino acid nucleoside phosphoramidates (aaNPs) can act as substrates for viral polymerases like HIV-1 RT.  Herein, we report the synthesis and hydrolysis of a series of new aaNPs, contg. either natural or modified nucleobases to define the basis for their differential reactivity.  Aq. stability, kinetics, and hydrolysis pathways were studied by NMR spectroscopy at different soln. pD values (5-7) and temps.  It was obsd. that the kinetics and mechanism (P-N and/or P-O bond cleavage) of the hydrolysis reaction largely depend on the nature of the nucleobase and amino acid moieties.  Aspartyl NPs were found to be more reactive than Gly or β-Ala NPs.  For aspartyl NPs, the order of reactivity of the nucleobase was 1-deazaadenine>7-deazaadenine>adenine>thymine≥3-deazaadenine.  Notably, neutral aq. solns. of Asp-1-deaza-dAMP degraded spontaneously even at 4 °C through exclusive P-O bond hydrolysis (a 50-fold reactivity difference for Asp-1-deaza-dAMP vs. Asp-3-deaza-dAMP at pD 5 and 70 °C).  Conformational studies by NMR spectroscopy and mol. modeling suggest the involvement of the protonated N3 atom in adenine and 1- and 7-deazaadenine in the intramol. catalysis of the hydrolysis reaction through the rare syn conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVaANmdiC86rVg90H21EOLACvtfcHk0lgEkCi61gTGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1elur7J&md5=097824e68615f4ba50cb55c6f24c56df</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fchem.201102279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201102279%26sid%3Dliteratum%253Aachs%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DMichielssens%26aufirst%3DS.%26aulast%3DDyubankova%26aufirst%3DN.%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DCeulemans%26aufirst%3DA.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DInfluence%2520of%2520the%2520nucleobase%2520and%2520anchimeric%2520assistance%2520of%2520the%2520carboxyl%2520acid%2520groups%2520in%2520the%2520hydrolysis%2520of%2520amino%2520acid%2520nucleoside%2520phosphoramidates%26jtitle%3DChemistry%26date%3D2012%26volume%3D18%26spage%3D857%26epage%3D868%26doi%3D10.1002%2Fchem.201102279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baweja, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmel, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shatkin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Crystal engineering of HIV-1 reverse transcriptase for structure-based drug design</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5092</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1093%2Fnar%2Fgkn464" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=18676450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGkt7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=5083-5092&author=J.+D.+Baumanauthor=K.+Dasauthor=W.+C.+Hoauthor=M.+Bawejaauthor=D.+M.+Himmelauthor=A.+D.+Clarkauthor=D.+A.+Orenauthor=P.+L.+Boyerauthor=S.+H.+Hughesauthor=A.+J.+Shatkinauthor=E.+Arnold&title=Crystal+engineering+of+HIV-1+reverse+transcriptase+for+structure-based+drug+design&doi=10.1093%2Fnar%2Fgkn464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal engineering of HIV-1 reverse transcriptase for structure-based drug design</span></div><div class="casAuthors">Bauman, Joseph D.; Das, Kalyan; Ho, William C.; Baweja, Mukta; Himmel, Daniel M.; Clark, Arthur D., Jr.; Oren, Deena A.; Boyer, Paul L.; Hughes, Stephen H.; Shatkin, Aaron J.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5083-5092</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">HIV-1 reverse transcriptase (RT) is a primary target for anti-AIDS drugs.  Structures of HIV-1 RT, usually detd. at ∼2.5-3.0 Å resoln., are important for understanding enzyme function and mechanisms of drug resistance in addn. to being helpful in the design of RT inhibitors.  Despite hundreds of attempts, it was not possible to obtain the structure of a complex of HIV-1 RT with TMC278, a nonnucleoside RT inhibitor (NNRTI) in advanced clin. trials.  A systematic and iterative protein crystal engineering approach was developed to optimize RT for obtaining crystals in complexes with TMC278 and other NNRTIs that diffract X-rays to 1.8 Å resoln.  Another form of engineered RT was optimized to produce a high-resoln. apo-RT crystal form, reported here at 1.85 Å resoln., with a distinct RT conformation.  Engineered RTs were mutagenized using a new, flexible and cost effective method called methylated overlap-extension ligation independent cloning.  Our anal. suggests that reducing the solvent content, increasing lattice contacts, and stabilizing the internal low-energy conformations of RT are crit. for the growth of crystals that diffract to high resoln.  The new RTs enable rapid crystn. and yield high-resoln. structures that are useful in designing/developing new anti-AIDS drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8sHD_iC_LsLVg90H21EOLACvtfcHk0lj-_w88xIk58w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGkt7%252FI&md5=c7957e67c3a017b289e03de94a78cbbd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn464%26sid%3Dliteratum%253Aachs%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DHo%26aufirst%3DW.%2BC.%26aulast%3DBaweja%26aufirst%3DM.%26aulast%3DHimmel%26aufirst%3DD.%2BM.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DOren%26aufirst%3DD.%2BA.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DCrystal%2520engineering%2520of%2520HIV-1%2520reverse%2520transcriptase%2520for%2520structure-based%2520drug%2520design%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D5083%26epage%3D5092%26doi%3D10.1093%2Fnar%2Fgkn464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeStefano, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seehra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laston, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bambara, R. A.</span></span> <span> </span><span class="NLM_article-title">Characterization of an RNase H deficient mutant of human immunodeficiency virus-1 reverse transcriptase having an aspartate to asparagine change at position 498</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>1219</i></span>,  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1016/0167-4781(94)90062-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1016%2F0167-4781%2894%2990062-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=7522572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADyaK2MXhsVSksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1219&publication_year=1994&pages=380-388&author=J.+J.+DeStefanoauthor=W.+Wuauthor=J.+Seehraauthor=J.+McCoyauthor=D.+Lastonauthor=E.+Alboneauthor=P.+J.+Fayauthor=R.+A.+Bambara&title=Characterization+of+an+RNase+H+deficient+mutant+of+human+immunodeficiency+virus-1+reverse+transcriptase+having+an+aspartate+to+asparagine+change+at+position+498&doi=10.1016%2F0167-4781%2894%2990062-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of an RNase H deficient mutant of human immunodeficiency virus-1 reverse transcriptase having an aspartate to asparagine change at position 498</span></div><div class="casAuthors">DeStefano, Jeffrey J.; Wu, Weimin; Seehra, Jasbir; McCoy, John; Laston, David; Albone, Earl; Fay, Philip J.; Bambara, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Structure and Expression</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">1219</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">380-8</span>CODEN:
                <span class="NLM_cas:coden">BBGSD5</span>;
        ISSN:<span class="NLM_cas:issn">0167-4781</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">It has been proposed that Asp-443, Glu-478, and Asp-498 are important in RNase H mediated catalysis by human immunodeficiency virus-1 reverse transcriptase (Davies J.F., Hostomska, Z. Hostomsky, Z., Jordan, S.R. and Matthews, D.A. (1991) Science 251, 88-95; Mizrahi, V., Usdin, M.T., Harington, A. and Dudding, L.R. (1990) Nucleic Acids Res. 18, 5359-5363).  Single point mutations at either position 443 (Mizrahi, V., Usdin, M.T., Harington, A. and Dudding, L.R. (1990) Nucleic Acids Res. 18, 5359-5363) or 478 (Schatz, O., Cromme, F.V., Greuninger-Leitch, F. and Le Grice, S.F.J. (1989) FEBS Lett. 257, 311-314) severely inhibit RNase H activity but have only small effects on polymerase activity.  The authors show here that a single mutation at position 498 of Asp to Asn (mutant D498N) results in a stable enzyme with a 20-fold redn. in the ratio of RNase H to polymerase activity.  The mutant and wild type enzymes were equally processive, paused in the same locations, and extended primers at the same rate during DNA synthesis on a heteropolymeric RNA template.  The rate of elongation on the homopolymeric template poly(rA) was also the same.  The results indicate that the mutation does not affect normally measured catalytic parameters of the polymerase function of the enzyme.  D498N catalyzed strand transfer synthesis on homopolymeric, but not heteropolymeric templates, indicating that RNase H activity is not required for the former activity, but is for the latter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvnmPl3U2jbVg90H21EOLACvtfcHk0lj-_w88xIk58w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhsVSksr4%253D&md5=bdd9b7a2287839ee73c9f4c459b7ec69</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0167-4781%2894%2990062-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0167-4781%252894%252990062-0%26sid%3Dliteratum%253Aachs%26aulast%3DDeStefano%26aufirst%3DJ.%2BJ.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DSeehra%26aufirst%3DJ.%26aulast%3DMcCoy%26aufirst%3DJ.%26aulast%3DLaston%26aufirst%3DD.%26aulast%3DAlbone%26aufirst%3DE.%26aulast%3DFay%26aufirst%3DP.%2BJ.%26aulast%3DBambara%26aufirst%3DR.%2BA.%26atitle%3DCharacterization%2520of%2520an%2520RNase%2520H%2520deficient%2520mutant%2520of%2520human%2520immunodeficiency%2520virus-1%2520reverse%2520transcriptase%2520having%2520an%2520aspartate%2520to%2520asparagine%2520change%2520at%2520position%2520498%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1994%26volume%3D1219%26spage%3D380%26epage%3D388%26doi%3D10.1016%2F0167-4781%2894%2990062-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaffney, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, R. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of guanosine and deoxyguanosine phosphoramidites with cross-linkable thioalkyl tethers for direct incorporation into RNA and DNA</span>. <i>Nucleos Nucleot. Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1076</span>– <span class="NLM_lpage">1094</span>, <span class="refDoi"> DOI: 10.1080/15257770903368385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1080%2F15257770903368385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20183575" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1076-1094&author=X.+Houauthor=G.+Wangauthor=B.+L.+Gaffneyauthor=R.+A.+Jones&title=Synthesis+of+guanosine+and+deoxyguanosine+phosphoramidites+with+cross-linkable+thioalkyl+tethers+for+direct+incorporation+into+RNA+and+DNA&doi=10.1080%2F15257770903368385"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1080%2F15257770903368385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15257770903368385%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DGaffney%26aufirst%3DB.%2BL.%26aulast%3DJones%26aufirst%3DR.%2BA.%26atitle%3DSynthesis%2520of%2520guanosine%2520and%2520deoxyguanosine%2520phosphoramidites%2520with%2520cross-linkable%2520thioalkyl%2520tethers%2520for%2520direct%2520incorporation%2520into%2520RNA%2520and%2520DNA%26jtitle%3DNucleos%2520Nucleot.%2520Nucleic%2520Acids%26date%3D2009%26volume%3D28%26spage%3D1076%26epage%3D1094%26doi%3D10.1080%2F15257770903368385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuske, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilankumaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesha, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroth, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, E.</span></span> <span> </span><span class="NLM_article-title">Trapping HIV-1 reverse transcriptase before and after translocation on DNA</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>278</i></span>,  <span class="NLM_fpage">16280</span>– <span class="NLM_lpage">16288</span>, <span class="refDoi"> DOI: 10.1074/jbc.m212911200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M212911200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=12554739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjt1ehtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=16280-16288&author=S.+G.+Sarafianosauthor=A.+D.+Clarkauthor=S.+Tuskeauthor=C.+J.+Squireauthor=K.+Dasauthor=D.+Shengauthor=P.+Ilankumaranauthor=A.+R.+Rameshaauthor=H.+Krothauthor=J.+M.+Sayerauthor=D.+M.+Jerinaauthor=P.+L.+Boyerauthor=S.+H.+Hughesauthor=E.+Arnold&title=Trapping+HIV-1+reverse+transcriptase+before+and+after+translocation+on+DNA&doi=10.1074%2Fjbc.m212911200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping HIV-1 Reverse Transcriptase Before and After Translocation on DNA</span></div><div class="casAuthors">Sarafianos, Stefan G.; Clark, Arthur D., Jr.; Tuske, Steve; Squire, Christopher J.; Das, Kalyan; Sheng, Dequan; Ilankumaran, Palanichamy; Ramesha, Andagar R.; Kroth, Heiko; Sayer, Jane M.; Jerina, Donald M.; Boyer, Paul L.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">16280-16288</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A disulfide crosslinking strategy was used to covalently trap as a stable complex (complex N) a short-lived, kinetic intermediate in DNA polymn.  This intermediate corresponds to the product of polymn. prior to translocation.  We also prepd. the trapped complex that corresponds to the product of polymn. after translocation (complex P).  The crosslinking method that we used is a variation of a technique developed by the Verdine and Harrison labs.  It involves disulfide interchange between an engineered sulfhydryl group of the protein (Q258C mutation) and a disulfide-contg. tether attached at the N2 amino group of a modified dG in either the template or the primer strand of the nucleic acid.  We report here a highly efficient synthesis of the precursor, bis(3-aminopropyl)disulfide dihydrochloride, used to introduce this substituent into the oligonucleotide.  Efficient crosslinking takes place when the base pair contg. the substituent is positioned seven registers from the dNTP-binding site (N site) and the N site is occupied.  Complex N, but not complex P, is a substrate for the ATP-based excision reaction that unblocks nucleoside reverse transcriptase inhibitor (NRTI)-terminated primers and causes resistance to several NRTIs, confirming predictions that the excision reaction takes place only when the 3'-end of the primer is bound at the N site.  These techniques can be used for biochem. and structural studies of the mechanism of DNA polymn., translocation, and excision-based resistance of RT to NRTIs.  They may also be useful in studying other DNA or RNA polymerases or other enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7jRh1ft9lT7Vg90H21EOLACvtfcHk0lgBMyRbOQtlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjt1ehtLk%253D&md5=8fe58e8d0b1c383fe2552880bb2925ae</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M212911200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M212911200%26sid%3Dliteratum%253Aachs%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DTuske%26aufirst%3DS.%26aulast%3DSquire%26aufirst%3DC.%2BJ.%26aulast%3DDas%26aufirst%3DK.%26aulast%3DSheng%26aufirst%3DD.%26aulast%3DIlankumaran%26aufirst%3DP.%26aulast%3DRamesha%26aufirst%3DA.%2BR.%26aulast%3DKroth%26aufirst%3DH.%26aulast%3DSayer%26aufirst%3DJ.%2BM.%26aulast%3DJerina%26aufirst%3DD.%2BM.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DTrapping%2520HIV-1%2520reverse%2520transcriptase%2520before%2520and%2520after%2520translocation%2520on%2520DNA%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D16280%26epage%3D16288%26doi%3D10.1074%2Fjbc.m212911200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">Integrating macromolecular X-ray diffraction data with the graphical user interface iMosflm</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1310</span>– <span class="NLM_lpage">1325</span>, <span class="refDoi"> DOI: 10.1038/nprot.2017.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1038%2Fnprot.2017.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=28569763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovFCqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1310-1325&author=H.+R.+Powellauthor=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=A.+G.+W.+Leslie&title=Integrating+macromolecular+X-ray+diffraction+data+with+the+graphical+user+interface+iMosflm&doi=10.1038%2Fnprot.2017.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating macromolecular X-ray diffraction data with the graphical user interface iMosflm</span></div><div class="casAuthors">Powell, Harold R.; Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1310-1325</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">X-ray crystallog. is the predominant source of structural information for biol. macromols., providing fundamental insights into biol. function.  The availability of robust and user-friendly software to process the collected X-ray diffraction images makes the technique accessible to a wider range of scientists. iMosflm/MOSFLM (http://www.mrc-lmb.cam.ac.uk/harry/imosflm) is a software package designed to achieve this goal.  The graphical user interface (GUI) version of MOSFLM (called iMosflm) is designed to guide inexperienced users through the steps of data integration, while retaining powerful features for more experienced users.  Images from almost all com. available X-ray detectors can be handled using this software.  Although the program uses only 2D profile fitting, it can readily integrate data collected in the 'fine phi-slicing' mode (in which the rotation angle per image is less than the crystal mosaic spread by a factor of at least 2), which is commonly used with modern very fast readout detectors.  The GUI provides real-time feedback on the success of the indexing step and the progress of data processing.  This feedback includes the ability to monitor detector and crystal parameter refinement and to display the av. spot shape in different regions of the detector.  Data scaling and merging tasks can be initiated directly from the interface.  Using this protocol, a data set of 360 images with ∼2,000 reflections per image can be processed in ∼4 min.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUd2z_blS8nrVg90H21EOLACvtfcHk0lgBMyRbOQtlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovFCqu7w%253D&md5=eb357d1354d93138e2100e0693ce0e0f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2017.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2017.037%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DIntegrating%2520macromolecular%2520X-ray%2520diffraction%2520data%2520with%2520the%2520graphical%2520user%2520interface%2520iMosflm%26jtitle%3DNat.%2520Protoc.%26date%3D2017%26volume%3D12%26spage%3D1310%26epage%3D1325%26doi%3D10.1038%2Fnprot.2017.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgBMyRbOQtlbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lg9DAWNuudpfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lg9DAWNuudpfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lg9DAWNuudpfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljDrM5KAiitzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span> <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+PyMOL+Molecular+Graphics+System%2C+version+1.8%3B+Schr%C3%B6dinger%2C+LLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michailidis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzek, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagedorn, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norden, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachiya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parniak, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wainberg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarafianos, S. G.</span></span> <span> </span><span class="NLM_article-title">Biochemical mechanism of HIV-1 resistance to rilpivirine</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">38110</span>– <span class="NLM_lpage">38123</span>, <span class="refDoi"> DOI: 10.1074/jbc.m112.398180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=10.1074%2Fjbc.M112.398180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=22955279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WjtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=38110-38123&author=K.+Singhauthor=B.+Marchandauthor=D.+K.+Raiauthor=B.+Sharmaauthor=E.+Michailidisauthor=E.+M.+Ryanauthor=K.+B.+Matzekauthor=M.+D.+Leslieauthor=A.+N.+Hagedornauthor=Z.+Liauthor=P.+R.+Nordenauthor=A.+Hachiyaauthor=M.+A.+Parniakauthor=H.-T.+Xuauthor=M.+A.+Wainbergauthor=S.+G.+Sarafianos&title=Biochemical+mechanism+of+HIV-1+resistance+to+rilpivirine&doi=10.1074%2Fjbc.m112.398180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical Mechanism of HIV-1 Resistance to Rilpivirine</span></div><div class="casAuthors">Singh, Kamalendra; Marchand, Bruno; Rai, Devendra K.; Sharma, Bechan; Michailidis, Eleftherios; Ryan, Emily M.; Matzek, Kayla B.; Leslie, Maxwell D.; Hagedorn, Ariel N.; Li, Zhe; Norden, Pieter R.; Hachiya, Atsuko; Parniak, Michael A.; Xu, Hong-Tao; Wainberg, Mark A.; Sarafianos, Stefan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">38110-38123</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Rilpivirine (RPV) is a second generation nonnucleoside reverse transcriptase (RT) inhibitor (NNRTI) that efficiently inhibits HIV-1 resistant to first generation NNRTIs.  Virol. failure during therapy with RPV and emtricitabine is assocd. with the appearance of E138K and M184I mutations in RT.  Here we investigate the biochem. mechanism of RT inhibition and resistance to RPV.  We used two transient kinetics approaches (quench-flow and stopped-flow) to det. how subunit-specific mutations in RT p66 or p51 affect assocn. and dissocn. of RPV to RT as well as their impact on binding of dNTP and DNA and the catalytic incorporation of nucleotide.  We compared WT with four subunit-specific RT mutants, p66M184I/p51WT, p66E138K/p51E138K, p66E138K/M184I/p51E138K, and p66M184I/p51E138K.  Ile-184 in p66 (p66184I) decreased the catalytic efficiency of RT (kpol/Kd.dNTP), primarily through a decrease in dNTP binding (Kd.dNTP).  Lys-138 either in both subunits or in p51 alone abrogated the neg. effect of p66184I by restoring dNTP binding.  Furthermore, p51138K reduced RPV susceptibility by altering the ratio of RPV dissocn. to RPV assocn., resulting in a net redn. in RPV equil. binding affinity (Kd.RPV = koff.RPV/kon.RPV).  Quantum mechanics/mol. mechanics hybrid mol. modeling revealed that p51E138K affects access to the RPV binding site by disrupting the salt bridge between p51E138 and p66K101. p66184I caused repositioning of the Tyr-183 active site residue and decreased the efficiency of RT, whereas the addn. of p51138K restored Tyr-183 to a WT-like conformation, thus abrogating the Ile-184-induced functional defects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_d23tgjqi-7Vg90H21EOLACvtfcHk0ljDrM5KAiitzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WjtLfJ&md5=2f7ed4760eaa646e49d181e75d16ee94</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.398180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.398180%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DB.%26aulast%3DRai%26aufirst%3DD.%2BK.%26aulast%3DSharma%26aufirst%3DB.%26aulast%3DMichailidis%26aufirst%3DE.%26aulast%3DRyan%26aufirst%3DE.%2BM.%26aulast%3DMatzek%26aufirst%3DK.%2BB.%26aulast%3DLeslie%26aufirst%3DM.%2BD.%26aulast%3DHagedorn%26aufirst%3DA.%2BN.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DNorden%26aufirst%3DP.%2BR.%26aulast%3DHachiya%26aufirst%3DA.%26aulast%3DParniak%26aufirst%3DM.%2BA.%26aulast%3DXu%26aufirst%3DH.-T.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DSarafianos%26aufirst%3DS.%2BG.%26atitle%3DBiochemical%2520mechanism%2520of%2520HIV-1%2520resistance%2520to%2520rilpivirine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D38110%26epage%3D38123%26doi%3D10.1074%2Fjbc.m112.398180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PQU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PQU','PDB','4PQU'); return false;">PDB: 4PQU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG','PDB','7AIG'); return false;">PDB: 7AIG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF','PDB','7AIF'); return false;">PDB: 7AIF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID','PDB','7AID'); return false;">PDB: 7AID</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX','PDB','7AHX'); return false;">PDB: 7AHX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ','PDB','7AIJ'); return false;">PDB: 7AIJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII','PDB','7AII'); return false;">PDB: 7AII</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V4I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V4I','PDB','3V4I'); return false;">PDB: 3V4I</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i35"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01747">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_67744"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01747?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01747</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table of summary of incorporation, resolution, and inhibition data for <span class="smallcaps smallerCapital">l</span>- and <span class="smallcaps smallerCapital">d</span>-amino acid phosphoramidates, and phosphonoamidates in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup>; table of X-ray data and refinement statistics; ligplots of <b>4d</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; ligplots of <b>4b</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; ligplots of <b>4e</b> with Mg<sup>2+</sup> or Mn<sup>2+</sup>; and <sup>1</sup>H, <sup>13</sup>C, and <sup>31</sup>P NMR spectra of compounds <b>3a</b>–<b>g</b> and <b>4a–g</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_001.pdf">jm0c01747_si_001.pdf (1.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01747/suppl_file/jm0c01747_si_002.csv">jm0c01747_si_002.csv (1.02 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates for RT/DNA plus compounds were deposited in the Protein Data Bank as follows: compound <b>4b</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AHX">7AHX</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AID">7AID</a>), compound <b>4d</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIF">7AIF</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIG">7AIG</a>), and compound <b>4e</b> with Mn<sup>2+</sup>/Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AII">7AII</a>) or Mg<sup>2+</sup> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7AIJ">7AIJ</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01747&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01747%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-1%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01747" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d5568af13cdc","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
