
<html lang="en"     class="pb-page"  data-request-id="869b24b3-9d78-4f5f-824c-77c3abc3b184"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-12;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01336;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen" /></meta><meta name="dc.Creator" content="Alex M.  Ayoub" /></meta><meta name="dc.Creator" content="Laura M. L.  Hawk" /></meta><meta name="dc.Creator" content="Ryan J.  Herzig" /></meta><meta name="dc.Creator" content="Jiewei  Jiang" /></meta><meta name="dc.Creator" content="Andrea J.  Wisniewski" /></meta><meta name="dc.Creator" content="Clifford T.  Gee" /></meta><meta name="dc.Creator" content="Peiliang  Zhao" /></meta><meta name="dc.Creator" content="Jin-Yi  Zhu" /></meta><meta name="dc.Creator" content="Norbert  Berndt" /></meta><meta name="dc.Creator" content="Nana K.  Offei-Addo" /></meta><meta name="dc.Creator" content="Thomas G.  Scott" /></meta><meta name="dc.Creator" content="Jun  Qi" /></meta><meta name="dc.Creator" content="James E.  Bradner" /></meta><meta name="dc.Creator" content="Timothy R.  Ward" /></meta><meta name="dc.Creator" content="Ernst  Schönbrunn" /></meta><meta name="dc.Creator" content="Gunda I.  Georg" /></meta><meta name="dc.Creator" content="William C. K.  Pomerantz" /></meta><meta name="dc.Description" content="Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihy..." /></meta><meta name="Description" content="Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihy..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 7, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01336" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01336" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01336" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01336" /></link>
        
    
    

<title>BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01336" /></meta><meta property="og:title" content="BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0011.jpeg" /></meta><meta property="og:description" content="Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay. Dihydropyridopyimidine 3 was subjected to further characterization and is highly selective for the BET family of bromodomains. Structure–activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity. Compound 3 was also cocrystallized with Brd4 for determining the ligand binding pose and rationalizing subsequent structure–activity data. An additional series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compounds with submicromolar affinity and cellular target engagement. Given these findings, novel and easily synthesized inhibitors are being introduced to the growing field of bromodomain inhibitor development." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay. Dihydropyridopyimidine 3 was subjected to further characterization and is highly selective for the BET family of bromodomains. Structure–activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity. Compound 3 was also cocrystallized with Brd4 for determining the ligand binding pose and rationalizing subsequent structure–activity data. An additional series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compounds with submicromolar affinity and cellular target engagement. Given these findings, novel and easily synthesized inhibitors are being introduced to the growing field of bromodomain inhibitor development." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0011.jpeg" /></meta><meta name="twitter:title" content="BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01336"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01336">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01336&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01336&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01336&amp;href=/doi/10.1021/acs.jmedchem.6b01336" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 4805-4817</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01595" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01796" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.7b00943"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alex+M.++Ayoub">Alex M. Ayoub</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura+M.+L.++Hawk">Laura M. L. Hawk</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ryan+J.++Herzig">Ryan J. Herzig</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiewei++Jiang">Jiewei Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea+J.++Wisniewski">Andrea J. Wisniewski</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Clifford+T.++Gee">Clifford T. Gee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peiliang++Zhao">Peiliang Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jin-Yi++Zhu">Jin-Yi Zhu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Norbert++Berndt">Norbert Berndt</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nana+K.++Offei-Addo">Nana K. Offei-Addo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+G.++Scott">Thomas G. Scott</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jun++Qi">Jun Qi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+E.++Bradner">James E. Bradner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Timothy+R.++Ward">Timothy R. Ward</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ernst++Sch%C3%B6nbrunn">Ernst Schönbrunn</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gunda+I.++Georg">Gunda I. Georg</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8900-9460" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+C.+K.++Pomerantz">William C. K. Pomerantz</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-0163-4078" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Medicinal Chemistry and Institute for Therapeutics Discovery and Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Drug Discovery Department, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Medical Oncology, Dana-Farber Cancer Institute, and Department of Medicine, Harvard Medical School, 360 Longwood Avenue, Boston, Massachusetts 02215, United States</span></div><div class="corresp-info"><strong>*</strong>G.I.G.: e-mail, <a href="/cdn-cgi/l/email-protection#5b3c3e34293c1b2e3635753e3f2e"><span class="__cf_email__" data-cfemail="cfa8aaa0bda88fbaa2a1e1aaabba">[email protected]</span></a>; phone, (612) 626-6320.</div><div class="corresp-info"><strong>*</strong>W.C.K.P.; e-mail, <a href="/cdn-cgi/l/email-protection#e3948093a3968e8dcd868796"><span class="__cf_email__" data-cfemail="fc8b9f8cbc899192d2999889">[email protected]</span></a>(612) 624-9091.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01336&amp;href=/doi/10.1021%2Facs.jmedchem.6b01336" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 4805–4817</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 23, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 September 2016</li><li><span class="item_label"><b>Published</b> online</span>7 June 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 June 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01336" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01336</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4805%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAlex%2BM.%2BAyoub%252C%2BLaura%2BM.%2BL.%2BHawk%252C%2BRyan%2BJ.%2BHerzig%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D12%26contentID%3Dacs.jmedchem.6b01336%26title%3DBET%2BBromodomain%2BInhibitors%2Bwith%2BOne-Step%2BSynthesis%2BDiscovered%2Bfrom%2BVirtual%2BScreen%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4817%26publicationDate%3DJune%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01336"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2478</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">26</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01336" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;M. Ayoub&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;M. L. Hawk&quot;},{&quot;first_name&quot;:&quot;Ryan&quot;,&quot;last_name&quot;:&quot;J. Herzig&quot;},{&quot;first_name&quot;:&quot;Jiewei&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;J. Wisniewski&quot;},{&quot;first_name&quot;:&quot;Clifford&quot;,&quot;last_name&quot;:&quot;T. Gee&quot;},{&quot;first_name&quot;:&quot;Peiliang&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Jin-Yi&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Norbert&quot;,&quot;last_name&quot;:&quot;Berndt&quot;},{&quot;first_name&quot;:&quot;Nana&quot;,&quot;last_name&quot;:&quot;K. Offei-Addo&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;G. Scott&quot;},{&quot;first_name&quot;:&quot;Jun&quot;,&quot;last_name&quot;:&quot;Qi&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;E. Bradner&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;R. Ward&quot;},{&quot;first_name&quot;:&quot;Ernst&quot;,&quot;last_name&quot;:&quot;Schönbrunn&quot;},{&quot;first_name&quot;:&quot;Gunda&quot;,&quot;last_name&quot;:&quot;I. Georg&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;C. K. Pomerantz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4805-4817&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01336&quot;},&quot;abstract&quot;:&quot;Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay. Dihydropyridopyimidine 3 was subjected to further characterization and is highly selective for the BET family of bromodomains. Structure–activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity. Compound 3 was also cocrystallized with Brd4 for determining the ligand binding pose and rationalizing subsequent structure–activity data. An additional series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compounds with submicromolar affinity and cellular target engagement. Given these findings, novel and easily synthesiz&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01336&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01336" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01336&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01336" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01336&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01336" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01336&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01336&amp;href=/doi/10.1021/acs.jmedchem.6b01336" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01336" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01336" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01336%26sid%3Dliteratum%253Aachs%26pmid%3D28535045%26genre%3Darticle%26aulast%3DAyoub%26date%3D2017%26atitle%3DBET%2BBromodomain%2BInhibitors%2Bwith%2BOne-Step%2BSynthesis%2BDiscovered%2Bfrom%2BVirtual%2BScreen%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D12%26spage%3D4805%26epage%3D4817%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=290998" title="Magnetic properties">Magnetic properties</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/12" title="Go to Volume 60, Issue 12"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/jmcmar.2017.60.issue-12/20170622/jmcmar.2017.60.issue-12.largecover.jpg" alt="Go to Volume 60, Issue 12"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Chemical inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease. We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay. Dihydropyridopyimidine <b>3</b> was subjected to further characterization and is highly selective for the BET family of bromodomains. Structure–activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity. Compound <b>3</b> was also cocrystallized with Brd4 for determining the ligand binding pose and rationalizing subsequent structure–activity data. An additional series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compounds with submicromolar affinity and cellular target engagement. Given these findings, novel and easily synthesized inhibitors are being introduced to the growing field of bromodomain inhibitor development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41004" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41004" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomain modules are essential protein recognition domains for transcriptional regulation. Bromodomains function through selective binding to N-ε-acetylated lysine side chains, most notably on histones and transcription factors. Additional enzymatic and scaffolding function can then be recruited to DNA via other domains within the bromodomain-containing protein for modulating gene transcription. The BET (bromodomain and extra-terminal) proteins are a subfamily of bromodomain-containing proteins consisting of Brd2, -3, -4, and -T, each of which contains tandem bromodomains, an extra-terminal motif, and in the case of Brd4, a C-terminal protein–protein interaction domain. BET proteins have attracted growing clinical interest due to their significant role in human health and disease. BrdT, which is testis-specific, has emerged as a promising target for male contraceptive agents due to its role in regulating spermatogenesis.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The other BET bromodomains play a critical role in a number of pathologies.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> In particular, Brd4 regulates the transcription of the oncogene c-Myc.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In the clinic, several pan-BET inhibitors are under study against various cancers including multiple myeloma, NUT-midline carcinoma, and glioblastoma multiforme.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Outside of cancer, Brd4 also plays a role in affecting viral activation and cardiac disease.<a onclick="showRef(event, 'ref3 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref5 ref6 ref7">(3, 5-7)</a></div><div class="NLM_p">Small molecules I-BET762 (<b>1</b>)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and (+)-JQ1 (<b>2</b>),<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> both based on a triazoloazepine scaffold, were the first described pan-BET inhibitors in the primary literature. Recently, compound <b>2</b> related memory deficits in mice<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and drug resistance against both triazoloazepines<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> have been described. These issues motivate the development of additional BET inhibitors. New chemotypes are therefore desirable for the optimization of the pharmaceutical properties of BET inhibitors as alternatives for clinical development. The first reported synthesis of <b>2</b> was completed in four steps, including three transformations carried out in one pot in the final step.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Subsequently, a scalable synthesis of <b>2</b> that utilized less toxic reagents and the same number of steps was described.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Several other scaffolds have been reported as pan-BET inhibitors.<a onclick="showRef(event, 'ref2 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref2 ref13 ref14 ref15 ref16 ref17">(2, 13-17)</a> The most accessible inhibitor reported to date was synthesized in two steps, the first of which was a multicomponent reaction, highlighting the demand for rapid, facile syntheses.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">In this report, we describe using a high-throughput virtual screen optimized for the first bromodomain of BrdT (hereafter referred to as BrdT(1)) to discover new binding scaffolds, followed by validation of these lead compounds (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Structure–activity relationships (SARs), ligand deconstruction, and binding pose were subsequently explored for the most potent compound found from the virtual screen, compound <b>3</b>, which is especially accessible due to its facile synthesis. Structure-guided attempts to access a new binding site on the protein surface of Brd4(1) based on the cocrystal structure with <b>3</b> led to additional gains in potency for analogs of <b>3</b>. To facilitate future studies using <b>3</b> and subsequent analogs as chemical probes, a bromodomain selectivity screen was also carried out, demonstrating high selectivity for the BET class of bromodomains. Herein, we describe a facile synthesis of pan-BET inhibitors with submicromolar potency based on a novel dihydropyridopyrimidine scaffold.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) BET bromodomain inhibitors <b>1</b>, <b>2</b>, BI2536 (<b>12</b>), and BI6727 (<b>13</b>). (B) Members of the BET branch of the bromodomain phylogenetic tree. Hits were identified by virtual screening, and binding potential of hits was assessed by fluorescence anisotropy (FA), differential scanning fluorimetry (DSF), and protein-observed <sup>19</sup>F NMR (PrOF NMR). Structure–activity relationship (SAR) studies were subsequently conducted to probe the indicated dihydropyridopyrimidine hit compound. Optimization of compound <b>3</b> based on an X-ray cocrystal structure led to the development of compound <b>3s</b>, with improved potency for both Brd4(1) and BrdT(1). The <i>K</i><sub>i</sub> values shown were determined by fluorescence anisotropy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We chose to pursue inhibition of BrdT because of its role in spermatogenesis, making BrdT a promising target for male contraception. The high-throughput virtual screen was performed with a library of six million compounds from the ZINC database that were docked against BrdT(1) using the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FLP">4FLP</a> in a low-precision mode. The molecules scoring in the top 0.4% were selected, refining the list to 24 000 hits. Increasing precision and applying computational filters for removal of potential pan-assay interference compounds (PAINS) and nondruglike compounds ultimately refined the number of hits to 200.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> A total of 22 commercially available compounds (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S1 in Supporting Information</a>) were selected from the 200 compounds for further testing. The choice of compounds was made qualitatively by visual inspection and applying multiple parameters, which included Lipinski and Weber rules, identification of toxicophores, scaffold novelty, and commercial availability at an affordable price. In addition, optimal shape complementarity in the binding pocket and the presence of multiple hits of the same scaffold were major factors in the selection of compounds for testing.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds from high-throughput virtual screen tested for binding to BrdT(1) in a fluorescence anisotropy assay. (A) Of the 22 compounds tested, the nine shown here bound to BrdT(1). (B) Fluorescence anisotropy binding of compounds <b>3</b>–<b>12</b> to BrdT(1). Compound <b>3</b> (open circles), the tightest binder, was selected for further studies. (C) PrOF NMR titration of compound <b>3</b> into 5FW-BrdT(1). The dashed lines indicate the location of the unperturbed resonance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The selected compounds were initially tested by fluorescence anisotropy because of the assay’s modest requirements of material and ability to provide quantitative binding information. We recently reported a fluorescence anisotropy assay for the first bromodomain of Brd4 using BI-BODIPY, a fluorescently labeled version of pan-BET inhibitor <b>13</b>.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> In that study, we limited ourselves to compounds soluble in ethylene glycol because of concerns regarding the known binding interactions of DMSO with bromodomains as a potential source of assay interference.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Although fluorescence anisotropy assays on BET bromodomains have been described by others,<a onclick="showRef(event, 'ref13 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref13 ref24 ref25 ref26">(13, 24-26)</a> we sought to find optimal screening conditions for first bromodomains of both Brd4 and BrdT. To this end, we tested experimental conditions with respect to chemical additives, incubation time, and protein stability. A detailed description of the optimization can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Supporting Information</a>. Optimization led us to a standard set of conditions: 4 mM CHAPS as detergent, 0.5% (v/v) DMSO, and a 30 min binding equilibration time.</div><div class="NLM_p">Of the 22 compounds selected from the virtual screen for further testing by fluorescence anisotropy, nine showed unambiguous binding to BrdT(1) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A and <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). Six of the nine hits contained a pyrimidine moiety; the remaining three hits were a diazocinone (compound <b>5</b>), a benzimidiazolone (compound <b>10</b>), and a pyrrolopyrazolone (compound <b>11</b>). Dihydropyridopyrimidine <b>3</b> was selected for further optimization due to its high binding potential for BrdT(1) (IC<sub>50</sub> = 5.2 μM, <i>K</i><sub>i</sub> = 0.69 μM). For comparison, the known BET inhibitor <b>12</b> had an IC<sub>50</sub> of 1.4 μM under these conditions. The <i>S</i> enantiomer of <b>3</b> was identified to bind in the virtual screen; however, in this study only racemates were tested based on commercial availability and synthetic accessibility. Although our starting point was a virtual screen against BrdT(1), we observed more potent binding to Brd4(1) with a <i>K</i><sub>i</sub> value of 0.37 μM.</div><div class="NLM_p">Compound <b>3</b> binding was further verified by differential scanning fluorimetry (DSF), which yields an increase in protein melting temperature upon ligand binding.<a onclick="showRef(event, 'ref9 ref27'); return false;" href="javascript:void(0);" class="ref ref9 ref27">(9, 27)</a> Compound <b>3</b> caused a 4.8 °C increase of the melting temperature of BrdT(1), consistent with the binding of <b>3</b> to BrdT(1) being significantly stabilizing. Finally, a competitive alpha-screen assay carried out against the compound <b>2</b>-BET bromodomain interaction yielded IC<sub>50</sub> values of 1.0 ± 0.2 μM and 2.3 ± 0.5 μM Brd4(1) and BrdT(1), respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S2</a>). Compound <b>3</b> also bound competitively with the native acetylated histone substrate in a similar alpha-screen assay (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S3</a>), giving a comparable IC<sub>50</sub> of 0.90 μM with Brd4(1).</div><div class="NLM_p">The structural effects of compound <b>3</b> on binding to BET bromodomains was verified by protein-observed <sup>19</sup>F (PrOF) NMR, which is emerging as a structure-based technique for detecting binding events for small molecule discovery.<a onclick="showRef(event, 'ref21 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref21 ref28 ref29 ref30 ref31">(21, 28-31)</a> PrOF NMR utilizes <sup>19</sup>F NMR spectra of fluorine-labeled proteins, in this case 5-fluorotryptophan (5FW) labeled Brd4(1) and BrdT(1). Because of the high environmental sensitivity of the <sup>19</sup>F protein resonances, perturbations to the <sup>19</sup>F NMR spectrum of the protein, such as resonance broadening or shifting in the presence of ligand, correlate to ligand binding. Binding is minimally perturbed by the presence of the fluorinated residues, demonstrated by the similar affinities of 5FW-labeled Brd4(1) and unlabeled Brd4(1) for BI-BODIPY (0.055 μM and 0.11 μM, respectively).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Similarly, the affinities of 5FW-BrdT(1) and BrdT(1) for BI-BODIPY (0.69 μM and 0.32 μM, respectively) are within approximately 2-fold of each other (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S4</a>), supporting the minimally perturbing nature of the fluorinated protein for quantifying bromodomain–ligand interactions.</div><div class="NLM_p">PrOF NMR corroborated the results from the fluorescence anisotropy assay, with compound <b>3</b> exhibiting behavior consistent with slow to intermediate exchange (<a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a>C and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). Chemical exchange phenomenon in the slow to intermediate exchange regime, which is based on the residence time of the ligand bound to the protein and the relative frequency difference between the resonances for the bound and unbound states, is consistent with a low micromolar to submicromolar binding ligand.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> For both 5FW-BrdT(1) and 5FW-Brd4(1), the resonance corresponding to the tryptophan in the acetylated lysine binding pocket (W50 and W81, respectively) broadens into baseline at or below 50 μM compound <b>3</b>, with a new resonance growing in upfield of the original peak at higher ligand concentrations. The presence of serum did not interfere with the binding of compound <b>3</b> to 5FW-Brd4(1) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S5</a>) for cell-based studies that are discussed below.</div><div class="NLM_p">Neither compound <b>3</b> nor close structural analogs (as low as 70% similarity as defined by ChEMBL) have been reported as bioactive compounds for bromodomains. However, other dihydropyridopyrimidine scaffolds have been described as active against a variety of targets.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38">(35-38)</a> While dihydropyridine moieties are prone to oxidation, they are commonly utilized in calcium channel blockers<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and the dihydropyridine moiety in <b>3</b> remains stable over several freeze–thaw cycles when stored as a DMSO solution at −20 °C as measured by ESI-MS (data not shown). After oxidation of compound <b>3</b> to a pyridine (<b>3n</b>) under forcing conditions with chloranil (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), no binding was observed to BrdT(1) or Brd4(1) by fluorescence anisotropy.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dihydropyridopyrimidines <b>3</b>–<b>3m</b> and pyridine <b>3n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and reaction conditions: (a) acetic acid, 110 °C, 6 h, N<sub>2</sub>, 11–81%; (b) compound <b>3</b>, chloranil, DMSO, rt, 2 h, 87%.</p></p></figure><div class="NLM_p">Compound <b>3</b> is an especially attractive lead due to its synthesis in a single step from commercially available starting materials (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The synthesis was performed using the Hantzsch dihydropyridine three-component cyclization based on the methods described by Tanifum and colleagues.<a onclick="showRef(event, 'ref35 ref40'); return false;" href="javascript:void(0);" class="ref ref35 ref40">(35, 40)</a> Pure compounds (>95% purity for 11 of the 12 synthesized compounds) were obtained by simply washing the product; no chromatography was necessary. Analogs were generated by varying the substituents on the 1,3-dicarbonyl, benzaldehyde, and uracil starting materials.</div><div class="NLM_p">To optimize the potency of <b>3</b>, the substitution on the para position of the phenyl ring and N-1 (R<sup>3</sup>) and N-3 (R<sup>2</sup>) of the pyrimidine ring were varied (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Additionally, the lactone from tetronic acid was exchanged for a lactam or a ketone. The lactone is the most potent of these variants, with a ketone leading to loss of detectable potency. The substituent on N-1 of the pyrimidine ring was critical for potency, with an ethyl group proving optimal. A propyl group was also tolerated, albeit with a substantial decrease in potency compared to an ethyl group. No binding was observed with a cyclopropyl group at this position. We observed a mild loss in potency against BrdT(1) upon exchanging the ethyl on N-1 for a 2,2,2-trifluoroethyl group. This compound, <b>3j</b>, did not cause a positive change in melting temperature by DSF for BrdT(1) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S7</a>). Although tighter binding to Brd4(1) was measured in our fluorescence anisotropy assay (<i>K</i><sub>i</sub> = 0.16 μM for <b>3j</b> vs 0.37 μM for <b>3</b>), alpha-screen competition data did not reproduce this higher affinity and only showed higher micromolar inhibition (4.6 μM for <b>3j</b> vs 1.0 μM for <b>3</b>) with a steeper Hill slope (−1.6 for <b>3j</b> vs −1 for <b>3</b>), suggesting potential nonspecific binding (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figures S2 and S3</a>). PrOF NMR of <b>3j</b> with 5FW-Brd4(1) also shows slow to intermediate exchange kinetics (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S8</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effects of Varying the Substitution of Compound <b>3</b> on Potency against Brd4 and BrdT<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">N/D = not determined.</p></div></div><div></div></div><div class="NLM_p">Substitution of a methyl group at N-1 led to levels of binding too low (<i>K</i><sub>i</sub> > 40 μM) to be detected by fluorescence anisotropy. By comparison, varying the para substituent on the phenyl ring had a smaller impact on potency, with chloro, methyl, trifluoromethyl, and no substituent all tolerated. Adding a methyl group at N-3 of the pyrimidine ring led to loss of detectable potency.</div><div class="NLM_p">To determine the active pharmacophore of <b>3</b>, this compound was deconstructed into its parent pyrimidine due to the high frequency of pyrimidine compounds binding to BrdT(1) in the virtual screen. A fragment corresponding to the pyrimidine moiety of <b>3</b>, 6-amino-1-ethyluracil, was tested for binding to 5FW-BrdT(1) and 5FW-Brd4(1) by PrOF NMR to determine the dissociation constants. Titration of 6-amino-1-ethyluracil with 5FW-Brd4(1) yielded a <i>K</i><sub>d</sub> of 660 ± 60 μM and a ligand efficiency of 0.39 (kcal/mol)/non-hydrogen atom (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). W81, which is part of WPF shelf in the binding pocket (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), is perturbed significantly; the other 5FW resonances remain nearly unchanged, consistent with the binding mode of other known 5FW-BrdT(1) ligands. BrdT(1) binding was also detected with 6-amino-1-ethyluracil, in this case perturbing the two 5FW resonances with similar affinity. Nonlinear regression analysis of the chemical shift perturbation for W50, which is the residue located in the WPF shelf of BrdT(1), yielded a <i>K</i><sub>d</sub> of 1.5 ± 0.2 mM. Similarly, analysis of the chemical shift perturbation for the resonance corresponding to W44, the residue located beneath W50, also yielded a <i>K</i><sub>d</sub> of 1.5 ± 0.5 mM, resulting in a slightly lower ligand efficiency of 0.33 (kcal/mol)/non-hydrogen atom. No binding was observed to 5FW-Brd4(1) when tetronic acid, a fragment corresponding to the lactone moiety of <b>3</b>, was tested by PrOF NMR.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding isotherm showing chemical shift perturbation from a PrOF NMR titration of 6-amino-1-ethyluracil into 50 μM 5FW-Brd4(1), -BrdT(1), and -BPTF. Perturbed resonances for each protein and their respective <i>K</i><sub>d</sub> values are shown. The ribbon structure for each protein is also shown with tryptophan residues labeled in red. Residue assignment for 5FW-BrdT(1) is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Other pyrimidine-containing compounds have been reported to bind bromodomains. ATAD2, a non-BET bromodomain-containing protein, binds to the pyrimidine thymine with 10 mM affinity.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Several higher affinity pyrimidine-containing dual kinase-bromodomain inhibitors have also been described.<a onclick="showRef(event, 'ref27 ref42'); return false;" href="javascript:void(0);" class="ref ref27 ref42">(27, 42)</a> On the basis of these results, we postulated that uracil derivatives are broad-spectrum bromodomain binders. In support of this hypothesis, we found that 6-amino-1-ethyluracil bound non-BET bromodomain 5FW-BPTF, albeit weakly (<i>K</i><sub>d</sub> = 6.8 mM ± 1.8 mM) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).</div><div class="NLM_p">Chemical probe molecules must have both high functional activity and high selectivity to provide useful biological information. On the basis of the binding of 6-amino-1-ethyluracil to BPTF, we tested binding of compound <b>3</b> to non-BET bromodomain 5FW-BPTF by PrOF NMR as a representative off target (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). In this case no binding was detected, suggesting that while 6-amino-1-ethyluracil may bind to multiple bromodomains, elaboration of the uracil into the substituted dihydropyridopyrimidine scaffold generates selectivity toward BET bromodomains. PrOF NMR experiments on both 5FW-BrdT(1) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C) and 5FW-Brd4(1) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A) are consistent with potent binding of <b>3</b>. To test the selectivity of compound <b>3</b> against additional bromodomains, compound <b>3</b> was subjected to a selectivity screen against a panel of bromodomains in a bromodomain qPCR-based displacement assay (BromoScan, DiscoverX). Because <i>K</i><sub>i</sub> values are an indirect measure of dissociation constant and may differ based on the protein’s interaction with the fluorescently labeled ligand and experimental conditions, a direct titration was first carried out. BromoScan titration of <b>3</b> against Brd4(1) and BrdT(1) provided attenuated but similar <i>K</i><sub>d</sub> values compared to <i>K</i><sub>i</sub> values determined by fluorescence anisotropy, within 2-fold for BrdT(1) (<i>K</i><sub>d</sub> = 1.6 μM) and within 3-fold for Brd4(1) (<i>K</i><sub>d</sub> = 0.84 μM) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S10</a>). Additionally, single point measurements at 20 μM showed that <b>3</b> is highly selective for BET bromodomains. Low levels of activity were observed against non-BET bromodomains, with the most significant binding observed to the WDR9(2) (72% inhibition), SMARCA2 (71% inhibition), CBP (68% inhibition), and p300 (66% inhibition) bromodomains (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). We have yet to express the majority of these bromodomains for validating inhibition in complementary biophysical assays as described above for BPTF. Notably, the most significant inhibition apparent was for the second bromodomain of BrdT (BrdT(2)). To verify this inhibition, we further tested our fluorescence anisotropy conditions with BrdT(2). For these experiments, the lowest <i>K</i><sub>i</sub> value that can be determined is equal to the <i>K</i><sub>d</sub> of the fluorescent tracer.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> This limit of resolvable detection was reached in the interaction of <b>3</b> and <b>3j</b>, yielding a <i>K</i><sub>i</sub> of 0.45 μM or less for <b>3</b> and <b>3j</b>. Together these results support selective pan-BET inhibition with <b>3</b> and its analogs.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) PrOF NMR confirms binding of compound <b>3</b> to 5FW-Brd4(1). W81 in Brd4 and W50 in BrdT are broadened at 25 μM <b>3</b>, and a new resonance begins to grow in at higher concentrations. Dashed lines reference the position of the resonance in the absence of ligand. The protein concentration was 45 μM in all experiments. (B) Compound <b>3</b> does not bind to non-BET bromodomain BPTF, shown by the similar spectra in the presence and absence of ligand. Protein concentration was 50 μM in all experiments. (C) Selectivity of <b>3</b> against a panel of bromodomains. BET bromodomains are in bromodomain family II.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The high selectivity for BET bromodomains guided our attempts to obtain a cocrystal structure of Brd4(1) with <b>3</b> at atomic resolution. Brd4(1) was cocrystallized with compound <b>3</b>, and the structure of the complex was determined at 1.5 Å resolution (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Table S2</a>). Although the racemate of <b>3</b> was used for crystallization, only electron density corresponding to the <i>S</i>-stereoisomer was observed, consistent with our docking studies. The N-3 nitrogen and C-2 carbonyl oxygen directly interact with the conserved Asn140 side chain via H-bonds. Consequently, methylation at N-3 abolishes binding potential and renders compounds <b>3e</b>, <b>3f</b>, and <b>3m</b> inactive. Additionally, water-mediated H-bonds are observed between the C-2 carbonyl oxygen and the side chain of Tyr97 and between N-9 and the carbonyl oxygen of Pro82. Oxidation of <b>3</b> to <b>3n</b> may lead to the loss of the latter interaction, resulting in loss of potency in <b>3n</b>. Several hydrophobic van der Waals interactions exist between the inhibitor and acetylated lysine binding site residues. The pyrimidine N-1 ethyl side chain interacts with the aromatic ring of Phe83 which serves as a backstop for binding further into the histone binding pocket of Brd4(1). Such an interaction would be missed with shorter methyl analogs. Finally, the tolyl group of <b>3</b> in the <i>S</i>-configuration is 3.7 Å away from the indole of W81 at the WPF shelf, consistent with an offset π–π interaction. More extended aromatic ring systems may be able to capitalize on this interaction in future designs. This interaction may be responsible for the upfield shift of the W81 resonance in PrOF NMR spectra in the presence of <b>3</b>, similar to that observed for <b>12</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Together, the noncovalent interactions from the cocrystal structure support the <i>N</i>-ethylpyrimidine ring of <b>3</b> being the dominant pharmacophore and help rationalize the SAR trends observed in our related analogs.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of Brd4(1) with compound <b>3</b>. (A) Direct hydrogen bonding interactions (black dotted lines) of compound <b>3</b> with Asn140 and water-mediated interactions with Tyr97 and Pro82. The inhibitor is shown in yellow, and the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map (contoured at 1σ) is indicated as blue mesh. Water molecules are shown as cyan spheres. (B) Space filling model of the inhibitor bound to the acetylated lysine binding site. (C) van der Waals interactions (green dotted lines) with mostly hydrophobic residues of the acetylated lysine binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The X-ray cocrystal structure led us to take initial steps toward optimization of <b>3</b>, leading to a series of more potent inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Scheme S1</a>, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). To take advantage of the proximity of a channel near Trp81 in the ZA loop observed in the cocrystal structure of <b>3</b> and Brd4(1), the <i>N</i>-methyl group of lactam <b>3l</b> was extended to an allyl (<b>3o</b>), benzyl (<b>3p</b>), phenethyl (<b>3q</b>), phenethyl methoxy (<b>3r</b>), or phenethyl 4-hydroxyl (<b>3s</b>). These extensions capitalize on accessing potential interactions that extend into the channel beyond those of <b>2</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S11</a>). We also explored a series of ring-opened lactones <b>3t</b>, <b>3u</b>, and <b>3v</b>, but these compounds either did not show gains in potency or only showed partial inhibition so were not investigated further (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S12</a>). Compounds <b>3p</b>, <b>3q</b>, <b>3r</b>, and <b>3s</b>, each of which contain a phenyl ring, were substantially more potent than compound <b>3</b> in binding to both Brd4(1) and BrdT(1). Competitive alpha-screen data using <b>2</b> as a displaceable probe revealed similar trends in binding behavior (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). PrOF NMR experiments corroborated the potent binding observed by fluorescence anisotropy, showing slow exchange upon titration of <b>3s</b> into 5FW-Brd4(1) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S13</a>) and affecting both W81 near the histone binding site and W75 outside the pocket, indicating a subtle conformational change. Slow exchange binding behavior is also consistent with a long resonance time of <b>3s</b> bound to Brd4(1).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effects of Varying the N-Substitution of Lactam on Potency against Brd4 and BrdT<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0010.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center"><i>K</i><sub>i</sub> (μM)</th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">BrdT(1)</th><th class="colsep0 rowsep0" align="center">Brd4(1)</th><th class="colsep0 rowsep0" align="center">BrdT(1)</th><th class="colsep0 rowsep0" align="center">Brd4(1)</th><th class="colsep0 rowsep0" align="center">MM1.S cells</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3o</b></td><td class="colsep0 rowsep0" align="left">N/D</td><td class="colsep0 rowsep0" align="left">14 ± 9</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">0.63</td><td class="colsep0 rowsep0" align="left">N/D</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3p</b></td><td class="colsep0 rowsep0" align="left">0.23 ± 0.13</td><td class="colsep0 rowsep0" align="left">0.19 ± 0.04</td><td class="colsep0 rowsep0" align="left">0.55</td><td class="colsep0 rowsep0" align="left">0.24</td><td class="colsep0 rowsep0" align="left">2.1 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3q</b></td><td class="colsep0 rowsep0" align="left">0.22 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.14 ± 0.04</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3r</b></td><td class="colsep0 rowsep0" align="left">0.20 ± 0.17</td><td class="colsep0 rowsep0" align="left">0.10 ± 0.01</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">0.48</td><td class="colsep0 rowsep0" align="left">2.1 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3s</b></td><td class="colsep0 rowsep0" align="left">0.20 ± 0.18</td><td class="colsep0 rowsep0" align="left">0.11 ± 0.01</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">0.46 ± 0.04</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">N/D = not determined.</p></div></div></div><div class="NLM_p">We tested for cellular activity using the multiple myeloma cell line MM1.S, which is highly sensitive to Brd4 inhibition.<a onclick="showRef(event, 'ref3 ref44'); return false;" href="javascript:void(0);" class="ref ref3 ref44">(3, 44)</a> Compound <b>2</b> strongly inhibited the growth of MM1.S cells with an IC<sub>50</sub> value of 0.02 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S14A</a>), which was accompanied by a dose-dependent reduction of c-Myc levels and upregulation of p21Cip1 levels (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Both lactone-containing inhibitors <b>3</b> and <b>3j</b> lacked growth inhibition activity and were ineffective in altering c-Myc or p21Cip1 levels, indicating poor cellular uptake or stability. However, the lactam series of compounds <b>3p</b>, <b>3q</b>, <b>3r</b>, and <b>3s</b>, all of which are both more potent and more lipophilic than <b>3</b> and <b>3j</b>, showed cellular activity with low to submicromolar IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S15</a>, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). These data are supported mechanistically with this series of compounds showing dose-dependent target engagement via decreasing c-Myc expression levels and concomitant increase in p21Cip1 via Western blot analysis (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Western blot analysis of the effect of compounds <b>3p</b>, <b>3q</b>, <b>3r</b>, and <b>3s</b> on c-Myc and p21Cip1 levels. Compound <b>2</b> served as a positive control, and vinculin served as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41073" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41073" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The pan-BET inhibitor described here is based on a dihydropyridopyrimidine scaffold that is readily accessible through a one-step synthesis. Beyond BET bromodomains, our NMR studies show that uracil derivatives, including a deconstructed fragment of compound <b>3</b>, can bind with bromodomains of wide sequence diversity. The uracil scaffold may therefore be a useful starting point for bromodomain inhibitor design. As we have shown in this work, elaboration upon this scaffold leads to the development of selectivity and potency. The dihydropyridopyrimidine compounds described herein maintain a molecular weight below 500 g/mol (339 g/mol for <b>3</b> and 458 g/mol for <b>3s</b>) and favorable cLogP values (2.1–3.6). We have shown that the substituent on the para position of the aromatic ring on the dihydropyridine is solvent exposed and binding is not substantially perturbed for a variety of substituents. Therefore, either solubilizing groups, affinity tags for future proteomic studies, bivalent inhibitor development,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> or protein degradation motifs<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> can readily be appended starting from commercially available benzaldehyde derivatives, adding to the utility of this scaffold.</div><div class="NLM_p last">McKeown and colleagues described an imidazopyrazine pan-BET inhibitor with a facile synthesis consisting of two steps: a multicomponent reaction followed by a Suzuki coupling.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The potency of their most potent compound via isothermal titration calorimetry for Brd4 is 0.55 μM at 15 °C. Compound <b>3</b> compares favorably to the most potent compound from McKeown and colleagues in overall yield (70% vs 45%) and number of synthetic steps (1 vs 2), and compound <b>3s</b> shows improvements in potency over compound <b>3</b>. Although our compounds are selective for BET bromodomains, the imidazopyrazine also opens the door for targeting TAF1 if selectivity were optimized. Together, these two classes of compounds offer an opportunity for rapid structure–activity data to be generated against BET bromodomains using non-diazepine scaffolds.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31527" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31527" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Through performing a high-throughput virtual screen against bromodomain BrdT(1), we have discovered several new inhibitors, based on a dihydropyridopyrimidine scaffold from <b>3</b>, that are highly selective for BET bromodomains with submicromolar affinity for the first and second bromodomains of BrdT. Although multiple BET inhibitors are known, compound <b>3</b> is already commercially available, and many analogs of <b>3</b> can be generated in a single step in a metal-free process. Diversity can be readily accomplished by varying the composition of the aldehyde, uracil, and 1,3-dicarbonyl compound starting materials. Compound <b>3</b> and related analogs such as molecules <b>3p</b>–<b>3s</b> can thus serve as starting points for improving potency, solubility, and cell permeability for future probe development.</div><div class="NLM_p last">The development of inhibitors selective for a particular member of the BET family represents a significant challenge due to the high sequence similarity between BET family members. Two domain 1 and one domain 2 BET bromodomain selective molecules have been reported.<a onclick="showRef(event, 'ref16 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref16 ref47 ref48">(16, 47, 48)</a> No potent, isoform selective inhibitor has yet been described, although a small molecule containing a uracil moiety with micromolar affinity for Brd4 has recently been reported.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> An optimized fluorescence anisotropy assay based on competition with a fluorescently labeled pan-BET inhibitor can be used to profile binding to multiple bromodomains, offering a path toward discovery and development of isoform-selective inhibitors.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> General Methods</h3><div class="NLM_p last">Solvents and reagents were purchased from commercial suppliers and used without additional purification. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded at 400 and 100 MHz, respectively, in DMSO-<i>d</i><sub>6</sub>. Chemical shifts, δ, are reported in ppm, and coupling constants, <i>J</i>, are expressed in hertz (Hz). Abbreviations for peaks are s = singlet, d = doublet, t = triplet, and m = multiplet. High-resolution mass spectrometry was performed using positive mode electrospray ionization methods (ESMS) with a Bruker BioTOF II spectrometer. The purity of compounds was determined by analytical reverse-phase high performance liquid chromatography (HPLC) analysis using Waters 2695 HPLC (column: Phenomenex Synergi Fusion-RP, 4.0 μm, 4.6 mm × 150 mm); mobile phase isocratic 50:50 0.1% TFA in H<sub>2</sub>O/MeOH; detector, Waters 2996; injector, automated injector; detection wavelength, 254 nm; flow rate, 1.0 mL/min. Sample diluent 1:5 DMSO/MeOH. Compounds from the virtual screen predicted to bind (<b>3</b>–<b>12</b> and compounds in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S1</a>), and <b>3b</b>–<b>d</b>, were purchased from Ambinter and fully characterized. All synthesized compounds and purchased analogs of <b>3</b> were >95% pure except <b>3d</b>, which was 93% pure, and <b>3l</b>, which was 87% pure.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> General Procedure for the Synthesis of the Dihydropyridopyrimidine Derivatives</h3><div class="NLM_p">Tetronic acid, cyclopentane-1,3-dione or 1-methylpyrrolidine-2.4-dione (0.50 mmol), the benzaldehyde (0.50 mmol), and the uracil (0.50 mmol) were added to a dry microwave vial (5 mL), and glacial AcOH (2 mL) was added. The vial was placed under N<sub>2</sub> for 1 min and sealed. The vial was placed in an oil bath at 110 °C and stirred for 6 h or until the solution solidified. The solution was allowed to cool to room temperature. AcOH was evaporated under N<sub>2</sub>, and the solid was washed with DCM (3 × 2 mL) to remove excess AcOH. The remaining solid was purified by washing with MeOH (8 × 1 mL) and dried to afford the dihydropyridopyrimidine.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> 1-Ethyl-5-(<i>p</i>-tolyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3</b>)</h4><div class="NLM_p last">Yield (119 mg, 70%) as a white solid; mp 318–319 °C dec; with a purity of 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.07 (s, 1H), 10.12 (s, 1H), 7.12–7.10 (d, <i>J</i> = 8.0 Hz, 2H), 7.05–7.03 (d, <i>J</i> = 8.0 Hz, 2H), 4.97–4.93 (d, <i>J</i> = 16.5 Hz, 1H), 4.88–4.84 (d, <i>J</i> = 16.5 Hz, 1H), 4.63 (s, 1H), 3.98–3.87 (m, 2H), 2.22 (s, 3H), 1.21–1.17 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 161.6, 156.1, 149.8, 145.2, 142.0, 135.3, 128.5, 127.5, 102.3, 90.1, 65.8, 37.0, 33.8, 20.6, 13.7. HRMS: calcd for C<sub>18</sub>H<sub>17</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 362.1111; found 362.1109.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> 5-(4-Methoxyphenyl)-1-methyl-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3a</b>)</h4><div class="NLM_p last">Yield (136 mg, 80%) as a white solid; mp 306–307 °C dec; with a purity of 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.05 (s, 1H), 10.15 (s, 1H), 7.17–7.15 (d, <i>J</i> = 8.3 Hz, 2H), 6.80–6.78 (d, <i>J</i> = 8.3 Hz, 2H), 4.96–4.91 (d, <i>J</i> = 16.5 Hz, 1H), 4.86–4.81 (d, <i>J</i> = 16.5 Hz, 1H), 4.61 (s, 1H), 3.69 (s, 3H), 3.35 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.0, 161.6, 157.8, 155.9, 150.1, 146.0, 137.2, 128.7, 113.3, 102.5, 90.0, 65.8, 55.0, 33.4, 29.1. HRMS: calcd for C<sub>17</sub>H<sub>15</sub>NaN<sub>3</sub>O<sub>5</sub> [M + Na]<sup>+</sup>, 364.0904; found 364.0904.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> 1-Methyl-5-phenyl-8,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,5<i>H</i>)-trione (<b>3b</b>)</h4><div class="NLM_p last">White solid; mp 308–309 °C dec; with a purity 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.07 (s, 1H), 10.19 (s, 1H), 7.28–7.25 (m, 4H), 7.17–7.12 (m, 1H), 4.95 (d, <i>J</i> = 16.6 Hz, 1H), 4.85 (d, <i>J</i> = 16.7 Hz, 1H), 4.67 (s, 1H), 3.36 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 161.6, 156.2, 150.1, 146.2, 144.8, 127.9, 127.7, 126.3, 102.3, 89.7, 65.8, 34.3, 29.1. HRMS: calcd for C<sub>16</sub>H<sub>13</sub>N<sub>3</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>, 334.0798; found 334.0798.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 1-Ethyl-5-phenyl-8,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,5<i>H</i>)-trione (<b>3c</b>)</h4><div class="NLM_p last">White solid; mp 305–306 °C dec; with a purity 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 10.15 (s, 1H), 7.27–7.21 (m, 4H), 7.18–7.12 (m, 1H), 4.96 (d, <i>J</i> = 16.6 Hz, 1H), 4.87 (d, <i>J</i> = 16.6 Hz, 1H), 4.67 (s, 1H), 4.02–3.85 (m, 2H), 1.19 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 161.6, 156.2, 149.8, 145.3, 144.8, 128.0, 127.6, 126.3, 102.2, 89.9, 65.8, 37.0, 34.2, 13.7. HRMS: calcd for C<sub>17</sub>H<sub>15</sub>N<sub>3</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>, 348.0955; found 348.0955.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 5-Phenyl-1-propyl-8,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,5<i>H</i>)-trione (<b>3d</b>)</h4><div class="NLM_p last">White solid; mp 300–301 °C dec; with a purity of 95%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 1H), 10.07 (s, 1H), 7.28–7.21 (m, 4H), 7.18–7.12 (m, 1H), 4.97 (d, <i>J</i> = 16.5 Hz, 1H), 4.85 (d, <i>J</i> = 16.5 Hz, 1H). 4.67 (s, 1H), 3.85 (t, <i>J</i> = 7.4 Hz, 2H), 1.66–1.55 (m, 2H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.0, 161.5, 156.3, 150.0, 145.5, 144.9, 128.0, 127.6, 126.3, 102.2, 89.9, 65.9, 42.9, 34.2, 21.2, 10.7. HRMS: calcd for C<sub>18</sub>H<sub>17</sub>N<sub>3</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>, 362.1111; found 362.1111.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 1,3-Dimethyl-5-(<i>p</i>-tolyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3e</b>)</h4><div class="NLM_p last">Yield (115 mg, 68%) as white solid; mp 294–296 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (s, 1H), 7.14–7.12 (d, <i>J</i> = 7.2 Hz, 2H), 7.04–7.02 (d, <i>J</i> = 7.2 Hz, 2H), 4.96–4.82 (m, 2H), 4.65 (s, 1H), 3.42 (s, 3H), 3.08 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 160.8, 155.7, 150.4, 144.8, 142.0, 135.3, 128.4, 127.7, 102.5, 89.3, 65.7, 34.5, 30.1, 27.6, 20.6. HRMS: calcd for C<sub>18</sub>H<sub>17</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 362.1111; found 362.1111.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-(4-Methoxyphenyl)-1,3-dimethyl-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3f</b>)</h4><div class="NLM_p last">Yield (133 mg, 74%) as a white solid; mp 329–330 °C dec; with a purity of 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.21 (s, 1H), 7.17–7.15 (d, <i>J</i> = 8.6 Hz, 2H), 6.79–6.77 (d, <i>J</i> = 8.6 Hz, 2H), 4.97–4.93 (d, <i>J</i> = 16.6 Hz, 1H), 4.86–4.82 (d, <i>J</i> = 16.6 Hz, 1H), 4.64 (s, 1H), 3.69 (s, 3H), 3.42 (s, 3H), 3.08 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 160.8, 157.8, 155.7, 150.4, 144.7, 137.1, 128.7, 113.3, 102.6, 89.5, 65.7, 55.0, 34.0, 30.2, 27.6. HRMS: calcd for C<sub>18</sub>H<sub>17</sub>NaN<sub>3</sub>O<sub>5</sub> [M + Na]<sup>+</sup>, 378.1060; found 378.1060.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 5-(4-Chlorophenyl)-1-ethyl-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3g</b>)</h4><div class="NLM_p last">Yield (146 mg, 81%) as a white solid; mp 298–300 °C dec; with a purity of 98%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (s, 1H), 10.19 (s, 1H), 7.31–7.29 (d, <i>J</i> = 8.7 Hz, 2H), 7.27–7.25 (d, <i>J</i> = 8.7 Hz, 2H), 4.98–4.94 (d, <i>J</i> = 16.6 Hz, 1H), 4.90–4.86 (d, <i>J</i> = 16.6 Hz, 1H), 4.67 (s, 1H), 4.00–3.84 (m, 2H), 1.21–1.17 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 161.6, 156.5, 149.8, 145.5, 143.7, 130.9, 129.6, 127.9, 101.7, 89.5, 65.9, 37.1, 34.0,13.7. HRMS: calcd for C<sub>17</sub>H<sub>14</sub>ClNaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 382.0565; found 382.0565.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1-Ethyl-5-(4-(trifluoromethyl)phenyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3h</b>)</h4><div class="NLM_p last">Yield (140 mg, 71%) as a white solid; mp 297–298 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 10.24 (s, 1H), 7.62–7.60 (d, <i>J</i> = 8.1 Hz, 2H), 7.49–7.47 (d, <i>J</i> = 8.1 Hz, 2H), 4.99–4.95 (d, <i>J</i> = 16.7 Hz, 1H), 4.91–4.87 (d, <i>J</i> = 16.7 Hz, 1H), 4.78 (s, 1H), 4.02–3.85 (m, 2H), 1.22–1.19 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.8, 161.6, 156.8, 149.8, 149.2, 145.7, 128.6, 127.0, 124.89, 124.85, 124.82, 124.4, 101.4, 89.2, 66.0, 37.2, 34.6, 13.7. HRMS: calcd for C<sub>18</sub>H<sub>14</sub>F<sub>3</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 416.0829; found 416.0829.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 1-Cyclopropyl-5-(<i>p</i>-tolyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3i</b>)</h4><div class="NLM_p last">Yield (125 mg, 71%) as a white solid; mp 280–281 °C dec; with a purity of 96%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.91 (s, 1H), 9.97 (s, 1H), 7.12–7.10 (d, <i>J</i> = 8.0 Hz, 2H), 7.04–7.02 (d, <i>J</i> = 8.0 Hz, 2H), 4.97–4.93 (d, <i>J</i> = 16.7 Hz, 1H), 4.88–4.84 (d, <i>J</i> = 16.7 Hz, 1H), 4.60 (s, 1H), 2.74–2.69 (m, 1H), 2.22 (s, 3H), 1.24–1.14 (m, 2H), 0.97–0.89 (m, 1H), 0.89–0.81 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.0, 161.9, 156.0, 150.1, 146.9, 141.9, 135.3, 128.5, 127.5, 102.2, 90.0, 66.0, 33.8, 24.5, 20.6, 10.1, 10.0. HRMS: calcd for C<sub>19</sub>H<sub>17</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 374.1111; found 374.1110.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 5-(<i>p</i>-Tolyl)-1-(2,2,2-trifluoroethyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3j</b>)</h4><div class="NLM_p last">Yield (21 mg, 11%) as a white solid; mp 305–306 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.42 (s, 1H), 10.34 (s, 1H), 7.11–7.09 (d, <i>J</i> = 8.1 Hz, 2H), 7.06–7.04 (d, <i>J</i> = 8.1 Hz, 2H), 5.03–4.99 (d, <i>J</i> = 16.5 Hz, 1H), 4.88–4.75 (m, 3H), 4.64 (s, 1H), 2.23 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.7, 161.3, 155.6, 149.7, 144.8, 141.7, 135.6, 128.6, 127.4, 125.2, 122.4, 102.6, 90.9, 65.6, 33.8, 20.6. HRMS: calcd for C<sub>18</sub>H<sub>14</sub>F<sub>3</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 416.0829; found 416.0829.</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-Ethyl-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-cyclopenta[5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3k</b>)</h4><div class="NLM_p last">Yield (66.0 mg, 59%) as a white solid; mp 308–310 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.02 (s, 1H), 9.92 (s, 1H), 7.07–7.05 (d, <i>J</i> = 8.0 Hz, 2H), 7.00–6.98 (d, <i>J</i> = 8.0 Hz, 2H), 4.60 (s, 1H), 4.06–3.94 (m, 2H), 2.76–2.64 (m, 2H), 2.30–2.27 (m, 2H), 2.20 (s, 3H), 1.20–1.17 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 201.1, 164.1, 161.5, 149.9, 145.3, 142.6, 134.9, 128.4, 127.4, 117.3, 90.7, 36.8, 33.7, 33.6, 24.6, 20.6, 13.8. HRMS: calcd for C<sub>19</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub> [M + Na]<sup>+</sup>, 360.1319; found 360.1319.</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 1-Ethyl-7-methyl-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3l</b>)</h4><div class="NLM_p last">Yield (88 mg, 50%) as a white solid; mp 293–295 °C dec; with a purity of 87%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.93 (s, 1H), 9.70 (s, 1H), 7.09–7.07 (d, <i>J</i> = 8.0 Hz, 2H), 7.01–6.99 (d, <i>J</i> = 8.0 Hz, 2H), 4.62 (s, 1H), 4.12–3.87 (m, 4H), 2.80 (s, 3H), 2.21 (s, 3H), 1.19–1.15 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.9, 161.7, 149.8, 146.4, 145.7, 142.9, 134.9, 128.3, 127.4, 109.3, 89.2, 49.4, 36.8, 34.0, 28.3, 20.6, 13.6. HRMS: calcd for C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub> [M + Na]<sup>+</sup>, 375.1428; found 375.1428.</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-Ethyl-3-methyl-5-(<i>p</i>-tolyl)-5,9-dihydrofuro[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3m</b>)</h4><div class="NLM_p last">Yield (118.2 mg, 67%) as a white solid; mp 299–300 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 7.13–7.11 (d, <i>J</i> = 7.9 Hz, 2H), 7.04–7.02 (d, <i>J</i> = 7.9 Hz, 2H), 4.98–4.94 (d, <i>J</i> = 16.5 Hz, 1H), 4.89–4.85 (d, <i>J</i> = 16.5 Hz, 1H), 4.66 (s, 1H), 4.06–3.92 (m, 2H), 3.08 (s, 3H), 2.22 (s, 3H), 1.23–1.20 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 160.8, 155.9, 150.1, 143.9, 141.9, 135.4, 128.5, 127.6, 102.4, 89.6, 65.8, 38.1, 34.4, 27.5, 20.6, 13.6. HRMS: calcd for C<sub>19</sub>H<sub>19</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 376.1268; found 376.1260.</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-Ethyl-5-(<i>p</i>-tolyl)furo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(1<i>H</i>,3<i>H</i>,8<i>H</i>)-trione (<b>3n</b>)</h4><div class="NLM_p last">Compound <b>3</b> (22.0 mg, 0.07 mmol, 1.0 equiv) and chloranil (32.1 mg, 0.13 mmol, 2.0 equiv) were added to a dry microwave vial, and DMSO (3 mL) was added. The vial was capped. The reaction was stirred at room temperature for 2 h. DMSO was then evaporated under N<sub>2</sub>. Purification by semiautomated silica gel chromatography (4:1 hexanes/EtOAc) yielded a white solid (19.0 mg, 87%); mp 278–279 °C dec; with a purity of 99%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.65 (s, 1H), 7.20–7.18 (d, <i>J</i> = 8.0 Hz, 2H), 7.15–7.13 (d, <i>J</i> = 8.0 Hz, 2H), 5.39 (s, 2H), 4.29–4.27 (q, <i>J</i> = 6.9 Hz, 2H), 2.38 (s, 3H), 1.27–1.23 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.9, 166.7, 159.7, 156.3, 154.3, 149.7, 137.3, 130.4, 127.8, 127.7, 112.2, 109.4, 68.9, 37.4, 21.0, 12.8. HRMS: calcd for C<sub>18</sub>H<sub>15</sub>NaN<sub>3</sub>O<sub>4</sub> [M + Na]<sup>+</sup>, 360.0955; found 360.0948.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure for the Synthesis of the Dihydropyridopyrimidine Lactam Derivatives</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Ethyl 7-(Chloromethyl)-1-ethyl-2,4-dioxo-5-(<i>p</i>-tolyl)-1,2,3,4,5,8-hexahydropyrido[2,3-<i>d</i>]pyrimidine-6-carboxylate (<b>S1</b>)</h4><div class="NLM_p">6-Amino-1-ethylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (12.90 mmol) and ethyl 4-chloro-2-(4-methylbenzylidene)-3-oxobutanoate (6.45 mmol), which were synthesized according to the reported procedures,<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50, 51)</a> were placed into a round-bottom flask (250 mL). Magnesium sulfate was added, and the mixture was heated to 45 °C for 48 h under a nitrogen atmosphere. After filtration, the solvent was removed in vacuo and the residue was purified by flash column chromatography on silica gel (40 g column, DCM + 1.5% MeOH) to furnish <b>S1</b> as white solid (364 mg, 14%); mp 266–267 °C dec; with a purity of 96%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.03 (s, 1H), 8.99 (s, 1H), 7.07 (d, <i>J</i> = 8.0 Hz, 2H), 7.02 (d, <i>J</i> = 8.0 Hz, 2H), 5.17 (d, <i>J</i> = 11.0 Hz, 1H), 4.88 (s, 1H), 4.72 (d, <i>J</i> = 11.0 Hz, 1H), 4.07 (m, 3H), 3.96–3.86 (m, 1H), 2.20 (s, 3H), 1.16 (q, <i>J</i> = 7.0 Hz, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.2, 161.3, 149.8, 144.0, 143.2, 142.7, 135.5, 128.7, 127.1, 107.1, 89.8, 60.2, 48.6, 36.4, 35.7, 20.6, 13.9, 13.5. HRMS: calcd for C<sub>20</sub>H<sub>22</sub>ClN<sub>3</sub>NaO<sub>4</sub> [M + Na]<sup>+</sup>, 426.1191; found 426.1199.</div><div class="NLM_p last">Compound <b>S1</b> (0.05 mmol) and the substituted primary amine (0.05 mmol) were added into a dry microwave tube (2 mL) containing ethanol (1 mL). The mixture was heated to 150 °C for 1 h in a microwave reactor. The mixture was purified by flash column chromatography on silica gel (12 g column, DCM + 2.5% MeOH) to yield the target compound.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 7-Allyl-1-ethyl-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3o</b>)</h4><div class="NLM_p last">Yield (6.92 mg, 31%) as a yellow solid; mp 246–247 °C dec; with a purity of 93%. <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.39 (s, 1H), 8.89 (s, 1H), 7.15 (d, <i>J</i> = 7.9 Hz, 2H), 7.00 (d, <i>J</i> = 7.8 Hz, 2H), 5.78 (ddd, <i>J</i> = 15.3, 9.9, 4.8 Hz, 1H), 5.28 (d, <i>J</i> = 16.6 Hz, 1H), 5.09 (d, <i>J</i> = 16.6 Hz, 1H), 5.04 (d, <i>J</i> = 19.1 Hz, 1H), 4.93 (d, <i>J</i> = 19.1 Hz, 1H), 4.89 (s, 1H), 4.01 (q, <i>J</i> = 6.8 Hz, 2H), 3.86 (s, 2H), 2.20 (s, 3H), 1.09 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO<i>-d</i><sub>6</sub>) δ 175.4, 172.8, 162.0, 155.3, 150.9, 143.7, 134.8, 133.6, 128.3, 127.1, 115.9, 93.4, 90.0, 75.1, 43.3, 35.7, 32.3, 20.6, 14.0. HRMS: calcd for C<sub>21</sub>H<sub>23</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 379.1765; found 379.1771.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 7-Benzyl-1-ethyl-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3p</b>)</h4><div class="NLM_p last">Yield (9.01 mg, 35%) as a yellow solid; mp 223–224 °C dec; with a purity of 97%. <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.40 (s, 1H), 9.18 (s, 1H), 7.26 (d, <i>J</i> = 6.8 Hz, 3H), 7.17 (d, <i>J</i> = 7.9 Hz, 2H), 7.08 (d, <i>J</i> = 6.1 Hz, 2H), 7.02 (d, <i>J</i> = 7.9 Hz, 2H), 5.28 (d, <i>J</i> = 16.6 Hz, 1H), 5.12 (d, <i>J</i> = 16.6 Hz, 1H), 4.91 (s, 1H), 4.45 (d, <i>J</i> = 5.7 Hz, 2H), 4.02 (q, <i>J</i> = 6.8, 2H), 2.23 (s, 3H), 1.09 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO<i>-d</i><sub>6</sub>) δ 175.6, 172.8, 162.1, 155.3, 150.9, 143.7, 137.4, 134.9, 128.4, 128.4, 127.3, 127.2, 126.8, 93.5, 90.0, 75.2, 44.5, 35.7, 32.4, 20.6, 14.0. HRMS: calcd for C<sub>25</sub>H<sub>25</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 429.1921; found 429.1929.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1-Ethyl-7-phenethyl-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3q</b>)</h4><div class="NLM_p last">Yield (6.61 mg, 25%) as a yellow solid; mp 252–253 °C dec; with a purity of 93%. <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.37 (s, 1H), 8.76 (s, 1H), 7.17 (s, 3H), 7.12 (d, <i>J</i> = 7.8 Hz, 2H), 7.01 (d, <i>J</i> = 7.8 Hz, 4H), 5.13 (d, <i>J</i> = 16.5 Hz, 1H), 5.00 (d, <i>J</i> = 16.5 Hz, 1H), 4.85 (s, 1H), 4.00 (dd, <i>J</i> = 14.6, 7.5 Hz, 2H), 3.46 (dd, <i>J</i> = 12.2, 6.1 Hz, 2H), 2.77–2.68 (m, 2H), 2.23 (s, 3H), 1.08 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 175.1, 172.8, 162.1, 155.4, 150.9, 143.7, 138.3, 134.8, 128.7, 128.4, 128.2, 127.1, 126.2, 93.4, 89.9, 75.0, 43.0, 35.7, 35.2, 32.3, 20.6, 14.0. HRMS: calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 443.2078; found 443.2068.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1-Ethyl-7-(4-methoxyphenethyl)-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3r</b>)</h4><div class="NLM_p last">Yield (5.93 mg, 22%) as a yellow solid; mp 222–223 °C dec; with a purity of 96%. <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.37 (s, 1H), 8.73 (s, 1H), 7.12 (d, <i>J</i> = 7.9 Hz, 2H), 7.02 (d, <i>J</i> = 7.9 Hz, 2H), 6.92 (d, <i>J</i> = 8.4 Hz, 2H), 6.73 (d, <i>J</i> = 8.4 Hz, 2H), 5.17 (d, <i>J</i> = 16.5 Hz, 1H), 5.05–4.98 (d, <i>J</i> = 16.5 Hz, 1H), 4.85 (s, 1H), 4.00 (q, <i>J</i> = 6.7 Hz, 2H), 3.70 (s, 3H), 3.41 (dd, <i>J</i> = 13.6, 6.8 Hz, 2H), 2.70–2.62 (m, 2H), 2.23 (s, 3H), 1.08 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO<i>-d</i><sub>6</sub>) δ 175.0, 172.8, 162.0, 157.7, 155.4, 150.9, 143.7, 134.8, 130.2, 129.7, 128.4, 127.1, 113.6, 93.4, 89.8, 75.1, 54.9, 43.2, 35.7, 34.2, 32.3, 20.6, 14.0. HRMS: calcd for C<sub>27</sub>H<sub>29</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 473.2183; found 473.2180.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1-Ethyl-7-(4-hydroxyphenethyl)-5-(<i>p</i>-tolyl)-5,7,8,9-tetrahydro-1<i>H</i>-pyrrolo[3′,4′:5,6]pyrido[2,3-<i>d</i>]pyrimidine-2,4,6(3<i>H</i>)-trione (<b>3s</b>)</h4><div class="NLM_p last">Yield (5.94 mg, 22%) as a yellow solid; mp 26<b>1</b>-262 °C dec; with a purity of 92%. <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub>6</sub>) δ 10.37 (s, 1H), 9.16 (s, 1H), 8.73 (s, 1H), 7.12 (d, <i>J</i> = 7.9 Hz, 2H), 7.01 (d, <i>J</i> = 7.9 Hz, 2H), 6.79 (d, <i>J</i> = 8.2 Hz, 2H), 6.57 (d, <i>J</i> = 8.2 Hz, 2H), 5.15 (d, <i>J</i> = 16.4 Hz, 1H), 5.01 (d, <i>J</i> = 16.4 Hz, 1H), 4.86 (s, 1H), 4.00 (q, <i>J</i> = 6.8 Hz, 2H), 3.41–3.33 (m, 2H), 2.59 (m, 2H), 2.23 (s, 3H), 1.08 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO<i>-d</i><sub>6</sub>) δ 175.0, 172.8, 162.0, 155.7, 155.4, 150.9, 143.7, 134.8, 129.6, 128.3, 127.1, 115.0, 93.4, 89.8, 75.0, 43.4, 35.6, 34.4, 32.3, 20.6, 14.0. HRMS: calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>, 459.2027; found 459.2038.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Fluorescence Anisotropy</h3><div class="NLM_p">All fluorescence anisotropy experiments were carried out in 50 mM Tris, 100 mM NaCl, pH = 7.4, on 384-well plates (Corning 4511). BrdT solutions also contained 3 mM DTT. Solutions for all experiments except those testing other detergents also contained 4 mM CHAPS. Other additives were present as described below. Stock solutions of fluorescent tracer BI-BODIPY in DMSO (25 μM) were diluted to 0.025 μM for these experiments. Plates were read on a Tecan Infinity 500 with an excitation wavelength at 485 nm and emission at 535 nm. For direct binding experiments, the protein was serially diluted across the plate, and the resulting anisotropy values were fit using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> in GraphPad Prism 6. In this equation, <i>b</i> and <i>c</i> are the maximum and minimum anisotropy values, respectively, <i>a</i> is the concentration of fluorescent tracer, <i>x</i> is the concentration of protein, and <i>y</i> is the observed anisotropy value.<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/jm-2016-01336f_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p last">The protein concentration for competition experiments was determined from the protein concentration in direct binding experiments at which the fluorescent tracer is 80% bound. Stock solutions of inhibitor molecules, which are 50 mM in DMSO, were serially diluted from 250 μM to subnanomolar concentrations, keeping the concentration of protein, BI-BODIPY, and other components constant. The DMSO concentration was held constant at 0.5% (v/v). The anisotropy values were then fit using GraphPad Prism’s log(inhibitor) vs response (four parameters) function. The IC<sub>50</sub> values thus obtained were converted to <i>K</i><sub>i</sub> values using a variant of the Cheng–Prusoff equation, where the <i>K</i><sub>i</sub> is the indirectly determined dissociation constant of the competitor ligand (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Reported errors were determined from three or more independent experiments. Direct binding with BI-BODIPY and competition experiments with positive control <b>12</b> were carried out prior to each set of competition experiments to assess protein quality and assay stability.<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/jm-2016-01336f_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Differential Scanning Fluorimetry</h3><div class="NLM_p last">DSF experiments were carried out using a CFX384 Touch real-time PCR detection system (Bio-Rad). The assay solution was composed of 10 μM BrdT(1) in a buffer of 50 mM HEPES (pH = 7.5) and 100 mM NaCl with a 5× concentration of SYPRO Orange (Life Technologies) as a fluorescent reporter. Compounds were tested in an eight-point dose response assay at concentrations ranging from 1250 to 9 μM (final DMSO concentration of 2.5%); experiments were performed in duplicate with average values reported. Protein denaturation was measured from 25 to 75 °C after a 5 min incubation at 25 °C, in 0.2 °C increments with a dwell time of 5 s (ramping rate of 2.4 °C per min). The fluorescent response of the protein unfolding was recorded, and the peak of the negative first derivative of the resulting melting curve was used to identify the apparent melting temperature (<i>T</i><sub>m</sub>) of the samples; calculations were performed by the Bio-Rad CFX Manager software. The change in melting temperature (Δ<i>T</i><sub>m</sub>) was determined by subtracting the <i>T</i><sub>m</sub> of the DMSO control (2.5% final concentration) from the <i>T</i><sub>m</sub> of the compound treated samples. Additional controls include a constant concentration of compound <b>2</b> at 100 μM and an eight-point dose response of <b>2</b> from 300 to 0.1 μM, included on each plate. A <i>Z</i>′ was calculated for each plate using <b>2</b> and DMSO controls; plates with a <i>Z</i>′ value less than 0.5 were excluded from analysis.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Virtual Screen</h3><div class="NLM_p last">A high throughput virtual screen (HTVS) was performed using the BrdT(1) X-ray crystal structure with (+)-JQ-1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FLP">4FLP</a>), using ∼6 000 000 compounds from the ZINC clean druglike molecules set.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The crystal structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FLP">4FLP</a>) was retrieved from the Protein Data Bank and downloaded into Schrodinger Maestro 9.3. The Protein Preparation Wizard in the Schrödinger suite was used to process the BrdT(1) structure. The protein structure was prepared by assigning bond orders, adding hydrogens, creating disulfide bonds, and optimizing H-bonding networks with PROPKA. Missing side chains for some amino acids were found and added using Prime version 3.1. Energy minimization with root-mean-square deviation value of 0.30 was then applied using the Schrodinger OPLS_2005 force field. Compound <b>2</b> was removed, and a receptor grid was generated. Subsets of ∼100 000 molecules of the ZINC clean druglike commercially available molecules set were then docked using Glide 5.8.<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54, 55)</a> The ZINC clean druglike commercially available molecules set consists of molecules with molecular weight of <500 g/mol, log <i>P</i> < 5, fewer than 5 H-bond donors, fewer than 10 H bond acceptors, fewer than 7 rotatable bonds, and polar surface area less than 150 Å<sup>2</sup>, and molecules with reactive groups, thiols, and aldehydes were removed.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56, 57)</a> The ligands were docked in low precision subsets of ∼100 000. The best scoring 0.4% of each subset were retained and combined into one file and then reordered. After reordering, the top 6000 molecules were then redocked using standard precision. The top 4000 were redocked with standard precision, with no significant change. The best 200 compounds were then examined, and high molecular weight compounds, PAINS, and phenols were removed. Twenty currently available compounds including two examples from the best scoring scaffold were ordered from a commercial source.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> PrOF NMR</h3><div class="NLM_p last"><sup>19</sup>F NMR spectra were acquired on a 500 MHz Bruker Avance III HD equipped with a 5 mM Prodigy TCI cryoprobe. Samples contained 40–50 μM 5FW-labeled bromodomain in 50 mM Tris, 100 mM NaCl, pH = 7.4, with 5% D<sub>2</sub>O. 5FW-BrdT samples also contained 3 mM DTT. Experimental parameters included a 90° flip angle, a relaxation delay of 0.7 s, an acquisition time of 0.1 s, and 500 transients. Spectra are referenced to TFA at −76.55 ppm. Small molecules were titrated in concentrated DMSO stocks. DMSO concentration was kept below 1% because of its potential for bromodomain binding. <i>K</i><sub>d</sub> values were obtained using a one-site binding equation accounting for ligand depletion.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Brd4(1) and BrdT(1) AlphaScreen</h3><div class="NLM_p last">Assays were performed with minor modifications from the manufacturer’s protocol (PerkinElmer, USA). All reagents were diluted in AlphaScreen buffer (50 mM HEPES, 150 mM NaCl, 0.01% v/v Tween-20, 0.1% w/v BSA, pH = 7.4). After addition of Alpha beads to master solutions, all subsequent steps were performed under low light conditions. A 2× solution of components with final concentrations of His-Brd4(1) or His-BrdT(1) at 0.020 μM, Ni-coated acceptor bead at 10 μg/mL, and biotinylated compound <b>2</b> at 0.010 μM were added in 10 μL to 384-well plates (AlphaPlate-384, PerkinElmer) using an EL406 liquid handler (Biotek, USA). Plates were spun down at 1000 rpm. A 10-point 1:<span class="NLM_inline-formula"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/jm-2016-01336f_m003.gif" alt="" /></img></span> serial dilution of compounds in DMSO was prepared at 200× the final concentration. An amount of 100 nL of compound from these stock plates was added by pin transfer using a Janus Workstation (PerkinElmer). A 2× solution of streptavidin-coated donor beads with a final concentration of 10 μg/mL was added in a 10 μL volume. The plates were spun down again at 1000 rpm and sealed with foil to prevent light exposure and evaporation. The plates were then incubated at room temperature for 1 h and read on an Envision 2104 (PerkinElmer) using the manufacturer’s protocol. IC<sub>50</sub> values were calculated using a four-parameter logistic curve in Prism 6 (GraphPad Software, USA) after normalization to DMSO-treated negative control wells (0.2% DMSO, v/v). AlphaScreen assays were performed by Reaction Biology using an acetylated histone 4 (1–21) against Brd4(1) with a constant DMSO concentration of 1% (v/v). Compound <b>2</b> was used as a positive control.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Protein Crystallography</h3><div class="NLM_p last">Reagents for crystallographic experiments were purchased from Hampton Research. Brd4(1) was purified as described previously.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Crystals of Brd4(1) were grown by vapor diffusion in hanging drops using 0.2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.1 M Tris/HCl (pH = 8.5), and 25% (w/v) PEG 3350 as precipitant supplemented with 5 mM compound <b>1</b> and 10% (v/v) DMSO. Crystals were harvested in cryoprotectant (3 parts precipitant including 0.5 mM compound <b>1</b> and 1 part 100% (v/v) ethylene glycol) and flash frozen in a stream of nitrogen gas. X-ray diffraction data were recorded at −180 °C at the beamline 22-ID, SER-CAT, Advanced Photon Source, Argonne National Laboratories. Data were reduced and scaled with XDS.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> PHENIX<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> was employed for phasing and refinement, and model building was performed using Coot.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The structure was solved by molecular replacement using Phaser<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> with the monomer of PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7A">4O7A</a> as the search model. An initial model of the inhibitor was generated using MarvinSketch (ChemAxon, Cambridge, MA) with ligand restraints from eLBOW of the PHENIX suite. All structures were validated by MolProbity<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and phenix.model_vs_data.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Figures were prepared using PyMOL (Schrödinger, LLC). Atomic coordinates and structure factors have been deposited in the Protein Data Bank (PDB) under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KDH">5KDH</a>.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Cell Viability Assays</h3><div class="NLM_p last">MM1.S cells were purchased from ATCC and maintained in RPMI-1640 medium (Life Technologies) containing 10% fetal bovine serum. Cells were grown and maintained at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>. Antibodies for c-MYC (no. 5605) and p21Cip1 (no. 2946) were purchased from Cell Signaling. Vinculin antibody (no. V9131) was from Sigma-Aldrich. Peroxidase-conjugated secondary antibodies were purchased from Jackson ImmunoResearch, West Grove, PA. MM1.S cells were seeded in 96-well plates at approximately 20 000 cells per well and incubated with drugs or vehicle (0.1% DMSO) for 72 h, with six replicates per drug concentration. After drug treatment, half volume of MTT reagent (Calbiochem) (2 mg/mL filtered PBS) was added to each well, followed by an incubation for 3 h at 37 °C. The supernatant was then aspirated and replaced with 100 μL of DMSO to solubilize the formazan crystals. Plates were placed on an orbital shaker for 5 min, and absorbance at 540 nm was determined with a SpectraMax 340PC microplate reader (Molecular Devices) with SoftMax Pro 5.2 software. The data were analyzed with Prism 6 (GraphPad) to determine IC<sub>50</sub> values.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Immunoblotting</h3><div class="NLM_p last">Cells were seeded in six-well plates (2 × 10<sup>5</sup> adherent or 1 × 10<sup>6</sup> suspension cells per well) and incubated for 6 h with increasing inhibitor concentrations. Cells were harvested by centrifugation at 300<i>g</i> for 5 min and resuspended in CelLytic M cell lysis reagent (Sigma-Aldrich) containing Halt protease inhibitor cocktail and Halt phosphatase inhibitor cocktail (Thermo Scientific) and 5 mM EDTA at 4 °C. Protein concentration was determined with Bio-Rad protein assay reagent (Hercules, CA), and samples were diluted with <sup>1</sup>/<sub>3</sub> volume 4× SDS sample buffer and heated at 95 °C for 5 min. Samples were subjected to 10% or 12.5% SDS–PAGE and transferred to PVDF or nitrocellulose membranes. Western blots were developed with the appropriate pairs of primary and secondary antibodies, and signals were visualized using HyGLO chemiluminescent reagent (Denville Scientific).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01336">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01336" class="ext-link">10.1021/acs.jmedchem.6b01336</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Additional data and methods (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_001.csv">jm6b01336_si_001.csv (3.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf">jm6b01336_si_002.pdf (8.24 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of Brd4 bound to compound <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KDH">5KDH</a>) has been deposited with the Protein Data Bank. Authors will release atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01336" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68356" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68356" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gunda I. Georg</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8900-9460" title="Orcid link">http://orcid.org/0000-0002-8900-9460</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69193998491b6839b98d8939283"><span class="__cf_email__" data-cfemail="a0c7c5cfd2c7e0d5cdce8ec5c4d5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William C. K. Pomerantz</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0163-4078" title="Orcid link">http://orcid.org/0000-0002-0163-4078</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#592e3a29192c3437773c3d2c"><span class="__cf_email__" data-cfemail="4631253606332b2868232233">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex M. Ayoub</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura M. L. Hawk</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ryan J. Herzig</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiewei Jiang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea J. Wisniewski</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clifford T. Gee</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Minnesota, 207 Pleasant Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peiliang Zhao</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jin-Yi Zhu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Norbert Berndt</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nana K. Offei-Addo</span> - <span class="hlFld-Affiliation affiliation">Department of
Medical Oncology, Dana-Farber Cancer Institute, and Department of
Medicine, Harvard Medical School, 360
Longwood Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas G. Scott</span> - <span class="hlFld-Affiliation affiliation">Department of
Medical Oncology, Dana-Farber Cancer Institute, and Department of
Medicine, Harvard Medical School, 360
Longwood Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jun Qi</span> - <span class="hlFld-Affiliation affiliation">Department of
Medical Oncology, Dana-Farber Cancer Institute, and Department of
Medicine, Harvard Medical School, 360
Longwood Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James E. Bradner</span> - <span class="hlFld-Affiliation affiliation">Department of
Medical Oncology, Dana-Farber Cancer Institute, and Department of
Medicine, Harvard Medical School, 360
Longwood Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy R. Ward</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Institute for Therapeutics Discovery and
Development, University of Minnesota, 717 Delaware Street SE, Minneapolis, Minnesota 55455, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernst Schönbrunn</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United
States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.M.A. and L.M.L.H. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ACK-d88e3562-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43021" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43021" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Contraception Research Branch, Eunice Kennedy Shriver National Institute of Child Health and Human Development Grant 5 U01 HD076542 (G.I.G. and E.S.) and NICHD Grant HHSN275201300017C (G.I.G. and E.S.), NIH Grant 1U01HD76508 (J.E.B. and J.Q.), and the NSF CAREER award (Grant CHE-1352019, W.C.K.P.). These studies were supported by the University of Minnesota through the Vince and McKnight Endowed Chairs (G.I.G.). We thank the Southeast Regional Collaborative Access Team (SER-CAT, University of Georgia) for assistance with synchrotron data collection at Argonne National Laboratory. We also thank the Moffitt Chemical Biology Core for use of the protein crystallography facility (NCI Grant P30-CA076292). BI-BODIPY was a generous gift from Wei Zhang at the University of Massachusetts, Boston.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i45" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i45"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i46" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i46"> Abbreviations Used</h2><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">BrdT(1)</td><td class="NLM_def"><p class="first last">BrdT domain 1</p></td></tr><tr><td class="NLM_term">BrdT(2)</td><td class="NLM_def"><p class="first last">BrdT domain 2</p></td></tr><tr><td class="NLM_term">Brd4(1)</td><td class="NLM_def"><p class="first last">Brd4 domain 1</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">PrOF NMR</td><td class="NLM_def"><p class="first last">protein-observed fluorine NMR</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 63 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Matzuk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agno, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Small-Molecule Inhibition of BRDT for Male Contraception</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">684</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+McKeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-Molecule+Inhibition+of+BRDT+for+Male+Contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0lgaVWoUGBa8Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-Molecule%2520Inhibition%2520of%2520BRDT%2520for%2520Male%2520Contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span> </span><span class="NLM_article-title">Place Your BETs: the Therapeutic Potential of Bromodomains</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+Your+BETs%3A+the+Therapeutic+Potential+of+Bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lh78sc0n1dryg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520Your%2520BETs%253A%2520the%2520Therapeutic%2520Potential%2520of%2520Bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET Bromodomain Inhibition as a Therapeutic Strategy to Target C-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+Bromodomain+Inhibition+as+a+Therapeutic+Strategy+to+Target+C-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lh78sc0n1dryg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520Bromodomain%2520Inhibition%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Target%2520C-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhGFihSNsG9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Q.</span><span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor JQ1 Activates HIV Latency Through Antagonizing Brd4 Inhibition of Tat-Transactivation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1093/nar/gks976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1093%2Fnar%2Fgks976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23087374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=277-287&author=Z.+Liauthor=J.+Guoauthor=Y.+Wuauthor=Q.+Zhou&title=The+BET+Bromodomain+Inhibitor+JQ1+Activates+HIV+Latency+Through+Antagonizing+Brd4+Inhibition+of+Tat-Transactivation&doi=10.1093%2Fnar%2Fgks976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The BET bromodomain inhibitor JQ1 activates HIV latency through antagonizing Brd4 inhibition of Tat-transactivation</span></div><div class="casAuthors">Li, Zichong; Guo, Jia; Wu, Yuntao; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Latent HIV reservoirs are the primary hurdle to eradication of infection.  Identification of agents, pathways and mol. mechanisms that activate latent provirus may, in the presence of highly active antiretroviral therapy, permit clearance of infected cells by the immune system.  Promoter-proximal pausing of RNA polymerase (Pol) II is a major rate-limiting step in HIV gene expression.  The viral Tat protein recruits human Super Elongation Complex (SEC) to paused Pol II to overcome this limitation.  Here, we identify the bromodomain protein Brd4 and its inhibition of Tat-transactivation as a major impediment to latency reactivation.  Brd4 competitively blocks the Tat-SEC interaction on HIV promoter.  The BET bromodomain inhibitor JQ1 dissocs. Brd4 from the HIV promoter to allow Tat recruitment of SEC to stimulate HIV elongation.  JQ1 synergizes with another latency activator prostratin, which promotes Pol II loading onto the viral promoter.  Because JQ1 activates viral latency without inducing global T cell activation, this and other closely related compds. and their antagonization of Brd4 to promote Tat-SEC interaction merit further investigations as effective agents/strategies for eliminating latent HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_cOKnnk8oLVg90H21EOLACvtfcHk0lhGFihSNsG9Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D&md5=87866d81d13af3d5051844ed6c17c98d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks976%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520JQ1%2520Activates%2520HIV%2520Latency%2520Through%2520Antagonizing%2520Brd4%2520Inhibition%2520of%2520Tat-Transactivation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D277%26epage%3D287%26doi%3D10.1093%2Fnar%2Fgks976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Spiltoir, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demos-Davies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinsey, T. A.</span><span> </span><span class="NLM_article-title">BET Acetyl-Lysine Binding Proteins Control Pathological Cardiac Hypertrophy</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+I.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+Acetyl-Lysine+Binding+Proteins+Control+Pathological+Cardiac+Hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0lgeMf9fDROTtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%2BI.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520Acetyl-Lysine%2520Binding%2520Proteins%2520Control%2520Pathological%2520Cardiac%2520Hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Anand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artero, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alaiti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alazem, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margulies, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappola, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plutzky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haldar, S. M.</span><span> </span><span class="NLM_article-title">BET Bromodomains Mediate Transcriptional Pause Release in Heart Failure</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23911322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WrsbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=569-582&author=P.+Anandauthor=J.+D.+Brownauthor=C.+Y.+Linauthor=J.+Qiauthor=R.+Zhangauthor=P.+C.+Arteroauthor=M.+A.+Alaitiauthor=J.+Bullardauthor=K.+Alazemauthor=K.+B.+Marguliesauthor=T.+P.+Cappolaauthor=M.+Lemieuxauthor=J.+Plutzkyauthor=J.+E.+Bradnerauthor=S.+M.+Haldar&title=BET+Bromodomains+Mediate+Transcriptional+Pause+Release+in+Heart+Failure&doi=10.1016%2Fj.cell.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomains Mediate Transcriptional Pause Release in Heart Failure</span></div><div class="casAuthors">Anand, Priti; Brown, Jonathan D.; Lin, Charles Y.; Qi, Jun; Zhang, Rongli; Artero, Pedro Calderon; Alaiti, M. Amer; Bullard, Jace; Alazem, Kareem; Margulies, Kenneth B.; Cappola, Thomas P.; Lemieux, Madeleine; Plutzky, Jorge; Bradner, James E.; Haldar, Saptarsi M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-582</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heart failure (HF) is driven by the interplay between regulatory transcription factors and dynamic alterations in chromatin structure.  Pathol. gene transactivation in HF is assocd. with recruitment of histone acetyl-transferases and local chromatin hyperacetylation.  We therefore assessed the role of acetyl-lysine reader proteins, or bromodomains, in HF.  Using a chem. genetic approach, we establish a central role for BET family bromodomain proteins in gene control during HF pathogenesis.  BET inhibition potently suppresses cardiomyocyte hypertrophy in vitro and pathol. cardiac remodeling in vivo.  Integrative transcriptional and epigenomic analyses reveal that BET proteins function mechanistically as pause-release factors crit. to expression of genes that are central to HF pathogenesis and relevant to the pathobiol. of failing human hearts.  This study implicates epigenetic readers as essential effectors of transcriptional pause release during HF pathogenesis and identifies BET coactivator proteins as therapeutic targets in the heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZbQWpJYqt7bVg90H21EOLACvtfcHk0lipfl6RKPIZlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WrsbzI&md5=33410607098cb42f1a7a52fa7ec0108b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DJ.%2BD.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DArtero%26aufirst%3DP.%2BC.%26aulast%3DAlaiti%26aufirst%3DM.%2BA.%26aulast%3DBullard%26aufirst%3DJ.%26aulast%3DAlazem%26aufirst%3DK.%26aulast%3DMargulies%26aufirst%3DK.%2BB.%26aulast%3DCappola%26aufirst%3DT.%2BP.%26aulast%3DLemieux%26aufirst%3DM.%26aulast%3DPlutzky%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHaldar%26aufirst%3DS.%2BM.%26atitle%3DBET%2520Bromodomains%2520Mediate%2520Transcriptional%2520Pause%2520Release%2520in%2520Heart%2520Failure%26jtitle%3DCell%26date%3D2013%26volume%3D154%26spage%3D569%26epage%3D582%26doi%3D10.1016%2Fj.cell.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of Inflammation by a Synthetic Histone Mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+Inflammation+by+a+Synthetic+Histone+Mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lipfl6RKPIZlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520Inflammation%2520by%2520a%2520Synthetic%2520Histone%2520Mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective Inhibition of BET Bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+Inhibition+of+BET+Bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhEigRHWvvQkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520Inhibition%2520of%2520BET%2520Bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Korb, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zucker-Scharff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darnell, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">BET Protein Brd4 Activates Transcription in Neurons and BET Inhibitor Jq1 Blocks Memory in Mice</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1464</span><span class="NLM_x">–</span> <span class="NLM_lpage">1473</span><span class="refDoi"> DOI: 10.1038/nn.4095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnn.4095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26301327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1464-1473&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=R.+B.+Darnellauthor=C.+D.+Allis&title=BET+Protein+Brd4+Activates+Transcription+in+Neurons+and+BET+Inhibitor+Jq1+Blocks+Memory+in+Mice&doi=10.1038%2Fnn.4095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BET protein Brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span></div><div class="casAuthors">Korb, Erica; Herre, Margo; Zucker-Scharff, Ilana; Darnell, Robert B.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1464-1473</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precise regulation of transcription is crucial for the cellular mechanisms underlying memory formation.  However, the link between neuronal stimulation and the proteins that directly interact with histone modifications to activate transcription in neurons remains unclear.  Brd4 is a member of the bromodomain and extra-terminal domain (BET) protein family, which binds acetylated histones and is a crit. regulator of transcription in many cell types, including transcription in response to external cues.  Small mol. BET inhibitors are in clin. trials, yet almost nothing is known about Brd4 function in the brain.  Here we show that Brd4 mediates the transcriptional regulation underlying learning and memory.  The loss of Brd4 function affects crit. synaptic proteins, which results in memory deficits in mice but also decreases seizure susceptibility.  Thus Brd4 provides a crit. link between neuronal activation and the transcriptional responses that occur during memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuMwCm6NDBLVg90H21EOLACvtfcHk0lgHp30_s6Db8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN&md5=0a09dea39378424cfc003a1e9e40ab9a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnn.4095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4095%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DBET%2520Protein%2520Brd4%2520Activates%2520Transcription%2520in%2520Neurons%2520and%2520BET%2520Inhibitor%2520Jq1%2520Blocks%2520Memory%2520in%2520Mice%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1464%26epage%3D1473%26doi%3D10.1038%2Fnn.4095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Fong, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, E. Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ftouni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morison, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregory, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsay, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, S.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papenfuss, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, M. A.</span><span> </span><span class="NLM_article-title">BET Inhibitor Resistance Emerges From Leukaemia Stem Cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">525</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1038/nature14888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature14888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26367796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=538-542&author=C.+Y.+Fongauthor=O.+Gilanauthor=E.+Y.+N.+Lamauthor=A.+F.+Rubinauthor=S.+Ftouniauthor=D.+Tylerauthor=K.+Stanleyauthor=D.+Sinhaauthor=P.+Yehauthor=J.+Morisonauthor=G.+Giotopoulosauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+C.-W.+Leeauthor=C.+Carpenterauthor=R.+Gregoryauthor=R.+G.+Ramsayauthor=S.+W.+Laneauthor=O.+Abdel-Wahabauthor=T.+Kouzaridesauthor=R.+W.+Johnstoneauthor=S.-J.+Dawsonauthor=B.+J.+P.+Huntlyauthor=R.+K.+Prinjhaauthor=A.+T.+Papenfussauthor=M.+A.+Dawson&title=BET+Inhibitor+Resistance+Emerges+From+Leukaemia+Stem+Cells&doi=10.1038%2Fnature14888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor resistance emerges from leukaemia stem cells</span></div><div class="casAuthors">Fong, Chun Yew; Gilan, Omer; Lam, Enid Y. N.; Rubin, Alan F.; Ftouni, Sarah; Tyler, Dean; Stanley, Kym; Sinha, Devbarna; Yeh, Paul; Morison, Jessica; Giotopoulos, George; Lugo, Dave; Jeffrey, Philip; Lee, Stanley Chun-Wei; Carpenter, Christopher; Gregory, Richard; Ramsay, Robert G.; Lane, Steven W.; Abdel-Wahab, Omar; Kouzarides, Tony; Johnstone, Ricky W.; Dawson, Sarah-Jane; Huntly, Brian J. P.; Prinjha, Rab K.; Papenfuss, Anthony T.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">538-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain and extra terminal protein (BET) inhibitors are first-in-class targeted therapies that deliver a new therapeutic opportunity by directly targeting bromodomain proteins that bind acetylated chromatin marks.  Early clin. trials have shown promise, esp. in acute myeloid leukemia, and therefore the evaluation of resistance mechanisms is crucial to optimize the clin. efficacy of these drugs.  Here we use primary mouse hematopoietic stem and progenitor cells immortalized with the fusion protein MLL-AF9 to generate several single-cell clones that demonstrate resistance, in vitro and in vivo, to the prototypical BET inhibitor, I-BET.  Resistance to I-BET confers cross-resistance to chem. distinct BET inhibitors such as JQ1, as well as resistance to genetic knockdown of BET proteins.  Resistance is not mediated through increased drug efflux or metab., but is shown to emerge from leukemia stem cells both ex vivo and in vivo.  Chromatin-bound BRD4 is globally reduced in resistant cells, whereas the expression of key target genes such as Myc remains unaltered, highlighting the existence of alternative mechanisms to regulate transcription.  We demonstrate that resistance to BET inhibitors, in human and mouse leukemia cells, is in part a consequence of increased Wnt/β-catenin signaling, and neg. regulation of this pathway results in restoration of sensitivity to I-BET in vitro and in vivo.  Together, these findings provide new insights into the biol. of acute myeloid leukemia, highlight potential therapeutic limitations of BET inhibitors, and identify strategies that may enhance the clin. utility of these unique targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Lg1n64SCWrVg90H21EOLACvtfcHk0lgHp30_s6Db8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL&md5=47d1769e077b1c6c92ff52a051368404</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature14888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14888%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DC.%2BY.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DRubin%26aufirst%3DA.%2BF.%26aulast%3DFtouni%26aufirst%3DS.%26aulast%3DTyler%26aufirst%3DD.%26aulast%3DStanley%26aufirst%3DK.%26aulast%3DSinha%26aufirst%3DD.%26aulast%3DYeh%26aufirst%3DP.%26aulast%3DMorison%26aufirst%3DJ.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%2BC.-W.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamsay%26aufirst%3DR.%2BG.%26aulast%3DLane%26aufirst%3DS.%2BW.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPapenfuss%26aufirst%3DA.%2BT.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBET%2520Inhibitor%2520Resistance%2520Emerges%2520From%2520Leukaemia%2520Stem%2520Cells%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D538%26epage%3D542%26doi%3D10.1038%2Fnature14888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Syeda, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakkaraj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span> </span><span class="NLM_article-title">Scalable Syntheses of the BET Bromodomain Inhibitor JQ1</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3454</span><span class="NLM_x">–</span> <span class="NLM_lpage">3457</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2015.02.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.tetlet.2015.02.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2015&pages=3454-3457&author=S.+S.+Syedaauthor=S.+Jakkarajauthor=G.+I.+Georg&title=Scalable+Syntheses+of+the+BET+Bromodomain+Inhibitor+JQ1&doi=10.1016%2Fj.tetlet.2015.02.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable syntheses of the BET bromodomain inhibitor JQ1</span></div><div class="casAuthors">Syeda, Shameem Sultana; Jakkaraj, Sudhakar; Georg, Gunda I.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3454-3457</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have developed methods involving the use of alternate, safer reagents for the scalable syntheses of the potent BET bromodomain inhibitor JQ1 (I).  A one-pot three step method, involving the conversion of a benzodiazepine to a thioamide using Lawesson's reagent, followed by amidrazone formation and installation of the triazole moiety furnished JQ1.  This method provides good yields and a facile purifn. process.  For the synthesis of enantiomerically enriched (+)-JQ1 (II), the highly toxic reagent di-Et chlorophosphate, used in a previous synthesis, was replaced with the safer reagent di-Ph chlorophosphate in the three-step one-pot triazole formation without effecting yields and enantiomeric purity of (+)-JQ1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpULsbWhpozUbVg90H21EOLACvtfcHk0lgzFMMTgNc8Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGntLo%253D&md5=2e86bfe5c83a36e13d5016b6a3a25b35</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2015.02.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2015.02.062%26sid%3Dliteratum%253Aachs%26aulast%3DSyeda%26aufirst%3DS.%2BS.%26aulast%3DJakkaraj%26aufirst%3DS.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26atitle%3DScalable%2520Syntheses%2520of%2520the%2520BET%2520Bromodomain%2520Inhibitor%2520JQ1%26jtitle%3DTetrahedron%2520Lett.%26date%3D2015%26volume%3D56%26spage%3D3454%26epage%3D3457%26doi%3D10.1016%2Fj.tetlet.2015.02.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Ran, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meagher, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaswamy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">4927</span><span class="NLM_x">–</span> <span class="NLM_lpage">4939</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%93-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lgeVMgc-kJQBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%2593-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Hay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttakar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">The Design and Synthesis of 5- and 6-Isoxazolylbenzimidazoles as Selective Inhibitors of the BET Bromodomains</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1039/C2MD20189E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2FC2MD20189E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=140-144&author=D.+Hayauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=M.+Philpottauthor=S.+Picaudauthor=D.+S.+Hewingsauthor=S.+Uttakarauthor=T.+D.+Heightmanauthor=S.+J.+Conwayauthor=S.+Knappauthor=P.+E.+Brennan&title=The+Design+and+Synthesis+of+5-+and+6-Isoxazolylbenzimidazoles+as+Selective+Inhibitors+of+the+BET+Bromodomains&doi=10.1039%2FC2MD20189E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC2MD20189E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20189E%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DUttakar%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Design%2520and%2520Synthesis%2520of%25205-%2520and%25206-Isoxazolylbenzimidazoles%2520as%2520Selective%2520Inhibitors%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D140%26epage%3D144%26doi%3D10.1039%2FC2MD20189E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clément, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnathan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordaunt, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimes, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajakane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7501</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span><span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+Epigenetic+Regulator+I-BET762%3A+Lead+Optimization+to+Afford+a+Clinical+Candidate+Inhibitor+of+the+BET+Bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lhq5Tj7v591gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520Epigenetic%2520Regulator%2520I-BET762%253A%2520Lead%2520Optimization%2520to%2520Afford%2520a%2520Clinical%2520Candidate%2520Inhibitor%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brotherton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez-Dacal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">RVX-208, an Inhibitor of BET Transcriptional Regulators with Selectivity for the Second Bromodomain</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">19754</span><span class="NLM_x">–</span> <span class="NLM_lpage">19759</span><span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+Inhibitor+of+BET+Transcriptional+Regulators+with+Selectivity+for+the+Second+Bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lh3wl2Hc93UKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520Inhibitor%2520of%2520BET%2520Transcriptional%2520Regulators%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Discovery of Chemical Inhibitors of Human Bromodomains</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">11625</span><span class="NLM_x">–</span> <span class="NLM_lpage">11688</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11688&author=G.+Zhangauthor=S.+G.+Smithauthor=M.-M.+Zhou&title=Discovery+of+Chemical+Inhibitors+of+Human+Bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0lh3wl2Hc93UKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DDiscovery%2520of%2520Chemical%2520Inhibitors%2520of%2520Human%2520Bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D11625%26epage%3D11688%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blacklow, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9019</span><span class="NLM_x">–</span> <span class="NLM_lpage">9027</span><span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+Multicomponent+Reactions+to+Develop+Novel+Bromodomain+Inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0ljcmId89Jc0yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520Multicomponent%2520Reactions%2520to%2520Develop%2520Novel%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="note"><p class="first last">Despite our attempts to remove all PAINS compounds, a rhodanine was tested (see structure in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S1</a>). No measurable binding was detected for this compound in our fluorescence anisotropy assay.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Rudolph, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steegmaier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haslinger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garin-Chesa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adolf, G. R.</span><span> </span><span class="NLM_article-title">BI 6727, a Polo-Like Kinase Inhibitor with Improved Pharmacokinetic Profile and Broad Antitumor Activity</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3094</span><span class="NLM_x">–</span> <span class="NLM_lpage">3102</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-08-2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1158%2F1078-0432.CCR-08-2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=19383823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFOrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=3094-3102&author=D.+Rudolphauthor=M.+Steegmaierauthor=M.+Hoffmannauthor=M.+Grauertauthor=A.+Baumauthor=J.+Quantauthor=C.+Haslingerauthor=P.+Garin-Chesaauthor=G.+R.+Adolf&title=BI+6727%2C+a+Polo-Like+Kinase+Inhibitor+with+Improved+Pharmacokinetic+Profile+and+Broad+Antitumor+Activity&doi=10.1158%2F1078-0432.CCR-08-2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">BI 6727, A Polo-like Kinase Inhibitor with Improved Pharmacokinetic Profile and Broad Antitumor Activity</span></div><div class="casAuthors">Rudolph, Dorothea; Steegmaier, Martin; Hoffmann, Matthias; Grauert, Matthias; Baum, Anke; Quant, Jens; Haslinger, Christian; Garin-Chesa, Pilar; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3094-3102</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Antimitotic chemotherapy remains a cornerstone of multimodality treatment for locally advanced and metastatic cancers.  To identify novel mitosis-specific agents with higher selectivity than approved tubulin-binding agents (taxanes, Vinca alkaloids), we have generated inhibitors of Polo-like kinase 1, a target that functions predominantly in mitosis.  The first compd. in this series, suitable for i.v. administration, has entered clin. development.  To fully explore the potential of Polo-like kinase 1 inhibition in oncol., we have profiled addnl. compds. and now describe a novel clin. candidate.  BI 6727 is a highly potent (enzyme IC50 = 0.87 nmol/L, EC50 = 11-37 nmol/L on a panel of cancer cell lines) and selective dihydropteridinone with distinct properties.  First, BI 6727 has a pharmacokinetic profile favoring sustained exposure of tumor tissues with a high vol. of distribution and a long terminal half-life in mice (Vss = 7.6 L/kg, t1/2 = 46 h) and rats (Vss = 22 L/kg, t1/2 = 54 h).  Second, BI 6727 has physicochem. and pharmacokinetic properties that allow in vivo testing of i.v. as well as oral formulations, adding flexibility to dosing schedules.  Finally, BI 6727 shows marked antitumor activity in multiple cancer models, including a model of taxane-resistant colorectal cancer.  With oral and i.v. routes of administration, the total weekly dose of BI 6727 is most relevant for efficacy, supporting the use of a variety of well-tolerated dosing schedules.  These findings warrant further investigation of BI 6727 as a tailored antimitotic agent; clin. studies have been initiated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-YEX91EOHHLVg90H21EOLACvtfcHk0ljcmId89Jc0yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFOrtbo%253D&md5=daed550be9d9b99ac6797c359039d295</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-08-2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-08-2445%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DSteegmaier%26aufirst%3DM.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DGrauert%26aufirst%3DM.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DQuant%26aufirst%3DJ.%26aulast%3DHaslinger%26aufirst%3DC.%26aulast%3DGarin-Chesa%26aufirst%3DP.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DBI%25206727%252C%2520a%2520Polo-Like%2520Kinase%2520Inhibitor%2520with%2520Improved%2520Pharmacokinetic%2520Profile%2520and%2520Broad%2520Antitumor%2520Activity%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D3094%26epage%3D3102%26doi%3D10.1158%2F1078-0432.CCR-08-2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cassel, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhikari, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">dos Santos, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2246</span><span class="NLM_x">–</span> <span class="NLM_lpage">2256</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2246-2256&author=A.+K.+Urickauthor=L.+M.+L.+Hawkauthor=M.+K.+Casselauthor=N.+K.+Mishraauthor=S.+Liuauthor=N.+Adhikariauthor=W.+Zhangauthor=C.+O.+dos+Santosauthor=J.+L.+Hallauthor=W.+C.+K.+Pomerantz&title=Dual+Screening+of+BPTF+and+Brd4+Using+Protein-Observed+Fluorine+NMR+Uncovers+New+Bromodomain+Probe+Molecules&doi=10.1021%2Facschembio.5b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span></div><div class="casAuthors">Urick, Andrew K.; Hawk, Laura M. L.; Cassel, Melissa K.; Mishra, Neeraj K.; Liu, Shuai; Adhikari, Neeta; Zhang, Wei; dos Santos, Camila O.; Hall, Jennifer L.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2246-2256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein dysregulation is linked to cancer, diabetes, and inflammation.  Selective inhibition of bromodomain function is a newly proposed therapeutic strategy.  The authors describe a 19F NMR dual screening method for small mol. discovery using fluorinated tryptophan resonances on two bromodomain-contg. proteins.  The chem. shift dispersion of 19F resonances within fluorine-labeled proteins enables the simultaneous anal. of two fluorinated bromodomains by NMR.  A library of 229 small mols. was screened against the first bromodomain of Brd4 and the BPTF bromodomain.  The authors report the first small mol. selective for BPTF over Brd4, termed AU1.  The Kd = 2.8 μM for AU1, which is active in a cell-based reporter assay.  No binding was detected with Brd4.  Three new Brd4 inhibitors with submicromolar affinity were also discovered.  Brd4 hits were validated in a thermal stability assay and potency detd. via fluorescence anisotropy.  The speed, ease of interpretation, and low protein concn. needed for protein-obsd. 19F NMR expts. in a multiprotein format offers a new method to discover and characterize selective ligands for bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjhnKasUdNL7Vg90H21EOLACvtfcHk0lgxmnwnqTDNgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI&md5=584fa6b83f8bfa661f55edb1078e570c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00483%26sid%3Dliteratum%253Aachs%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DCassel%26aufirst%3DM.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3Ddos%2BSantos%26aufirst%3DC.%2BO.%26aulast%3DHall%26aufirst%3DJ.%2BL.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DDual%2520Screening%2520of%2520BPTF%2520and%2520Brd4%2520Using%2520Protein-Observed%2520Fluorine%2520NMR%2520Uncovers%2520New%2520Bromodomain%2520Probe%2520Molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2246%26epage%3D2256%26doi%3D10.1021%2Facschembio.5b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span> </span><span class="NLM_article-title">Leveraging Kinase Inhibitors to Develop Small Molecule Tools for Imaging Kinases by Fluorescence Microscopy</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2523</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1039/c2mb25099c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2Fc2mb25099c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22673640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2523-2524&author=Z.+Zhangauthor=N.+Kwiatkowskiauthor=H.+Zengauthor=S.+M.+Limauthor=N.+S.+Grayauthor=W.+Zhangauthor=P.+L.+Yang&title=Leveraging+Kinase+Inhibitors+to+Develop+Small+Molecule+Tools+for+Imaging+Kinases+by+Fluorescence+Microscopy&doi=10.1039%2Fc2mb25099c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span></div><div class="casAuthors">Zhang, Zijuan; Kwiatkowski, Nicholas; Zeng, Hong; Lim, Sang Min; Gray, Nathanael S.; Zhang, Wei; Yang, Priscilla L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2526</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the usage of fluorescence microscopy as a tool to study biol. systems continues to grow, so does the need for addnl. tools that permit the selective detection of proteins of interest.  Existing selective and well-characterized kinase inhibitors may be exploited to develop novel small mol. probes useful in imaging kinases by fluorescence microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTi2Qtkh07Vg90H21EOLACvtfcHk0lhnUxJOhTSSIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF&md5=e59fd8a1bbd080bcfa2653fd7250328e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2Fc2mb25099c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb25099c%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DP.%2BL.%26atitle%3DLeveraging%2520Kinase%2520Inhibitors%2520to%2520Develop%2520Small%2520Molecule%2520Tools%2520for%2520Imaging%2520Kinases%2520by%2520Fluorescence%2520Microscopy%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D2523%26epage%3D2524%26doi%3D10.1039%2Fc2mb25099c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">Bromodomain-Peptide Displacement Assays for Interactome Mapping and Inhibitor Discovery</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2899</span><span class="NLM_x">–</span> <span class="NLM_lpage">2908</span><span class="refDoi"> DOI: 10.1039/c1mb05099k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2Fc1mb05099k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21804994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=2899-2908&author=M.+Philpottauthor=J.+Yangauthor=T.+Tumberauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=T.+Keatesauthor=I.+Felletarauthor=A.+Ciulliauthor=S.+Knappauthor=T.+D.+Heightman&title=Bromodomain-Peptide+Displacement+Assays+for+Interactome+Mapping+and+Inhibitor+Discovery&doi=10.1039%2Fc1mb05099k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-peptide displacement assays for interactome mapping and inhibitor discovery</span></div><div class="casAuthors">Philpott, Martin; Yang, Jing; Tumber, Tony; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Keates, Tracy; Felletar, Ildiko; Ciulli, Alessio; Knapp, Stefan; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2899-2908</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone lysine acetylation is a key component of epigenetic regulation of gene transcription.  Bromodomains, found in histone acetyl transferases and other chromatin-assocd. proteins, bind selectively to acetylated lysines, acting as "readers" of the histone code, and have recently been shown to contain a druggable binding pocket.  Here we report the development of high-throughput assays that quantify the binding of bromodomains to acetylated histone peptides.  We have used these assays to screen for histone binding partners of as yet uncharacterized bromodomains, adding to current knowledge of the histone code and expanding the repertoire of assays for chem. probe discovery.  We have also demonstrated that these assays can be used to detect small mol. binding from the very weak to the nanomolar range.  This assay methodol. is thereby anticipated to provide the basis both for broader interactome profiling and for small mol. inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mWM_E4IUzbVg90H21EOLACvtfcHk0lhnUxJOhTSSIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P&md5=0f9eccb2d2761e1707c9b04bccc76050</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2Fc1mb05099k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1mb05099k%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DT.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DBromodomain-Peptide%2520Displacement%2520Assays%2520for%2520Interactome%2520Mapping%2520and%2520Inhibitor%2520Discovery%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26spage%3D2899%26epage%3D2908%26doi%3D10.1039%2Fc1mb05099k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: 2. Structure-Based Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1021/jm501504k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501504k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1281-1297&author=L.+Zhaoauthor=Y.+Wangauthor=D.+Caoauthor=T.+Chenauthor=Q.+Wangauthor=Y.+Liauthor=Y.+Xuauthor=N.+Zhangauthor=X.+Wangauthor=D.+Chenauthor=L.+Chenauthor=Y.-L.+Chenauthor=G.+Xiaauthor=Z.+Shiauthor=Y.-C.+Liuauthor=Y.+Linauthor=Z.+Miaoauthor=J.+Shenauthor=B.+Xiong&title=Fragment-Based+Drug+Discovery+of+2-Thiazolidinones+as+BRD4+Inhibitors%3A+2.+Structure-Based+Optimization&doi=10.1021%2Fjm501504k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: Structure-Based Optimization</span></div><div class="casAuthors">Zhao, Lele; Wang, Yingqing; Cao, Danyan; Chen, Tiantian; Wang, Qi; Li, Yanlian; Xu, Yechun; Zhang, Naixia; Wang, Xin; Chen, Danqi; Chen, Lin; Chen, Yue-Lei; Xia, Guangxin; Shi, Zhe; Liu, Yu-Chih; Lin, Yijyun; Miao, Zehong; Shen, Jingkang; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1281-1297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The signal transduction of acetylated histone can be processed through a recognition module, bromodomain.  Several inhibitors targeting BRD4, one of the bromodomain members, are in clin. trials as anticancer drugs.  Hereby, the authors report the efforts on discovery and optimization of a new series of 2-thiazolidinones as BRD4 inhibitors along the previous study.  In this work, guided by crystal structure anal., the authors reversed the sulfonamide group and identified a new binding mode.  A structure-activity relationship study on this new series led to several potent BRD4 inhibitors with IC50 of about 0.05-0.1 μM in FP binding assay and GI50 of 0.1-0.3 μM in cell based assays.  To complete the lead-like assessment of this series, the authors further checked its effects on BRD4 downstream protein c-Myc, investigated its selectivity among five different bromodomain proteins, as well as the metabolic stability test, and reinforced the utility of 2-thiazolidinone scaffold as BET bromodomain inhibitors in novel anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4x-5lE2iPzbVg90H21EOLACvtfcHk0lhwhvBga8vYgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D&md5=4056ac1a6d6598dc61be7cc7ef82a3da</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm501504k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501504k%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520of%25202-Thiazolidinones%2520as%2520BRD4%2520Inhibitors%253A%25202.%2520Structure-Based%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1281%26epage%3D1297%26doi%3D10.1021%2Fjm501504k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamotte, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delannée, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudit, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trottet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Naphthyridines as Novel BET Family Bromodomain Inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">589</span><span class="refDoi"> DOI: 10.1002/cmdc.201300259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fcmdc.201300259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24000170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGlsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=580-589&author=O.+Mirguetauthor=Y.+Lamotteauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=D.+Delann%C3%A9eauthor=A.+Bouillotauthor=F.+Gellibertauthor=G.+Krysaauthor=A.+Lewisauthor=J.+Witheringtonauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=E.+Nicodeme&title=Naphthyridines+as+Novel+BET+Family+Bromodomain+Inhibitors&doi=10.1002%2Fcmdc.201300259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Naphthyridines as Novel BET Family Bromodomain Inhibitors</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Chung, Chun-wa; Bamborough, Paul; Delannee, Delphine; Bouillot, Anne; Gellibert, Francoise; Krysa, Gael; Lewis, Antonia; Witherington, Jason; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-589</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are small protein domains found in a variety of proteins that recognize and bind to acetylated histone tails.  This binding affects chromatin structure and facilitates the localization of transcriptional complexes to specific genes, thereby regulating epigenetically controlled processes including gene transcription and mRNA elongation.  Inhibitors of the bromodomain and extra-terminal (BET) proteins BRD2-4 and T, which prevent bromodomain binding to acetyl-modified histone tails, have shown therapeutic promise in several diseases.  We report here the discovery of 1,5-naphthyridine derivs., e.g. I, as potent inhibitors of the BET bromodomain family with good cell activity and oral pharmacokinetic parameters.  X-ray crystal structures of naphthyridine isomers have been solved and quantum mech. calcns. have been used to explain the higher affinity of the 1,5-isomer over the others.  The best compds. were progressed in a mouse model of inflammation and exhibited dose-dependent anti-inflammatory pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEqN70fkD1t7Vg90H21EOLACvtfcHk0ljDSn-0ZisIUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGlsLzI&md5=bb69d7ae0f7de3788617315a76bc8005</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300259%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDelann%25C3%25A9e%26aufirst%3DD.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DNaphthyridines%2520as%2520Novel%2520BET%2520Family%2520Bromodomain%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D580%26epage%3D589%26doi%3D10.1002%2Fcmdc.201300259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-W.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Bromodomain+Inhibitors+Part+1%3A+Inhibitor+Binding+Modes+and+Implications+for+Lead+Discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0ljDSn-0ZisIUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Bromodomain%2520Inhibitors%2520Part%25201%253A%2520Inhibitor%2520Binding%2520Modes%2520and%2520Implications%2520for%2520Lead%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Ember, S. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berndt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">1171</span><span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28BRD4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0lhmRWDA7jewEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Pomerantz, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipinski, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cierpicki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapp, A. K.</span><span> </span><span class="NLM_article-title">Profiling the Dynamic Interfaces of Fluorinated Transcription Complexes for Ligand Discovery and Characterization</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1345</span><span class="NLM_x">–</span> <span class="NLM_lpage">1350</span><span class="refDoi"> DOI: 10.1021/cb3002733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3002733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVWms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1345-1350&author=W.+C.+Pomerantzauthor=N.+Wangauthor=A.+K.+Lipinskiauthor=R.+Wangauthor=T.+Cierpickiauthor=A.+K.+Mapp&title=Profiling+the+Dynamic+Interfaces+of+Fluorinated+Transcription+Complexes+for+Ligand+Discovery+and+Characterization&doi=10.1021%2Fcb3002733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling the Dynamic Interfaces of Fluorinated Transcription Complexes for Ligand Discovery and Characterization</span></div><div class="casAuthors">Pomerantz, William C.; Wang, Ningkun; Lipinski, Ashley K.; Wang, Rurun; Cierpicki, Tomasz; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1345-1350</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The conformationally dynamic binding surfaces of transcription complexes present a particular challenge for ligand discovery and characterization.  In the case of the KIX domain of the master coactivator CBP/p300, few small mols. have been reported that target its two allosterically regulated binding sites despite the important roles that KIX plays in processes ranging from memory formation to hematopoiesis.  Taking advantage of the enrichment of arom. amino acids at protein interfaces, here we show that the incorporation of six 19F-labeled arom. side chains within the KIX domain enables recapitulation of the differential binding footprints of three natural activator peptides (MLL, c-Myb, and pKID) in complex with KIX and effectively reports on allosteric changes upon binding using 1D NMR spectroscopy.  Addnl., the examn. of both the previously described KIX protein-protein interaction inhibitor Naphthol-ASE-phosphate and newly discovered ligand 1-10 rapidly revealed both the binding sites and the affinities of these small mols.  Significantly, the utility of using fluorinated transcription factors for ligand discovery was demonstrated through a fragment screen leading to a new low mol. wt. fragment ligand for CBP/p300, 1G7.  Arom. amino acids are enriched at protein-biomol. interfaces; therefore, this quant. and facile approach will be broadly useful for studying dynamic transcription complexes and screening campaigns complementing existing biophys. methods for studying these dynamic interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3u6Duitlu5LVg90H21EOLACvtfcHk0lhmRWDA7jewEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVWms70%253D&md5=5528eab7191432daab201cee2324436c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fcb3002733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3002733%26sid%3Dliteratum%253Aachs%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DLipinski%26aufirst%3DA.%2BK.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DProfiling%2520the%2520Dynamic%2520Interfaces%2520of%2520Fluorinated%2520Transcription%2520Complexes%2520for%2520Ligand%2520Discovery%2520and%2520Characterization%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1345%26epage%3D1350%26doi%3D10.1021%2Fcb3002733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, S. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C.</span><span> </span><span class="NLM_article-title">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2755</span><span class="NLM_x">–</span> <span class="NLM_lpage">2760</span><span class="refDoi"> DOI: 10.1021/cb5007344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2755-2760&author=N.+K.+Mishraauthor=A.+K.+Urickauthor=S.+W.+J.+Emberauthor=E.+Sch%C3%B6nbrunnauthor=W.+C.+Pomerantz&title=Fluorinated+Aromatic+Amino+Acids+Are+Sensitive+19F+NMR+Probes+for+Bromodomain-Ligand+Interactions&doi=10.1021%2Fcb5007344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span></div><div class="casAuthors">Mishra, Neeraj K.; Urick, Andrew K.; Ember, Stuart W. J.; Schonbrunn, Ernst; Pomerantz, William C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2755-2760</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe a 19F NMR method for detecting bromodomain-ligand interactions using fluorine-labeled arom. amino acids due to the conservation of arom. residues in the bromodomain binding site.  The authors test the sensitivity, accuracy, and speed of this method with small mol. ligands (+)-JQ1, BI2536, Dinaciclib, TG101348, and acetaminophen using three bromodomains Brd4, BrdT, and BPTF.  Simplified 19F NMR spectra allowed for simultaneous testing of multiple bromodomains to assess selectivity and identification of a new BPTF ligand.  Fluorine labeling only modestly affected the Brd4 structure and function assessed by isothermal titrn. calorimetry, CD, and x-ray crystallog.  The speed, ease of interpretation, and low concn. of protein needed for binding expts. affords a new method to discover and characterize both native and new ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoejxq3KKVddrVg90H21EOLACvtfcHk0lhmRWDA7jewEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP&md5=97f9edadc91cba8670f54bcca57fa03f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb5007344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007344%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26atitle%3DFluorinated%2520Aromatic%2520Amino%2520Acids%2520Are%2520Sensitive%252019F%2520NMR%2520Probes%2520for%2520Bromodomain-Ligand%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2755%26epage%3D2760%26doi%3D10.1021%2Fcb5007344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koleski, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Fragment Screening and Druggability Assessment for the CBP/P300 KIX Domain Through Protein-Observed 19F NMR Spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3735</span><span class="NLM_x">–</span> <span class="NLM_lpage">3739</span><span class="refDoi"> DOI: 10.1002/anie.201411658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fanie.201411658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVKgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=3735-3739&author=C.+T.+Geeauthor=E.+J.+Koleskiauthor=W.+C.+K.+Pomerantz&title=Fragment+Screening+and+Druggability+Assessment+for+the+CBP%2FP300+KIX+Domain+Through+Protein-Observed+19F+NMR+Spectroscopy&doi=10.1002%2Fanie.201411658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Screening and Druggability Assessment for the CBP/p300 KIX Domain through Protein-Observed 19F NMR Spectroscopy</span></div><div class="casAuthors">Gee, Clifford T.; Koleski, Edward J.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3735-3739</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">19F NMR spectroscopy of labeled proteins is a sensitive method for characterizing structure, conformational dynamics, higher-order assembly, and ligand binding.  Fluorination of arom. side chains has been suggested as a labeling strategy for small-mol. ligand discovery for protein-protein interaction interfaces.  Using a model transcription factor binding domain of the CREB binding protein (CBP)/p300, KIX, we report the first full small-mol. screen using protein-obsd. 19F NMR spectroscopy.  Screening of 508 compds. and validation by 1H-15N HSQC NMR spectroscopy led to the identification of a minimal pharmacaphore for the MLL-KIX interaction site.  Hit rate anal. for the CREB-KIX and MLL-KIX sites provided a metric to assess the ligandability or "druggability" of each interface informing future medicinal chem. efforts.  The structural information from the simplified spectra and data collection speed, affords a new screening tool for anal. of protein interfaces and discovery of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhRb2BYwsiILVg90H21EOLACvtfcHk0lh3WOi726pc7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVKgsbw%253D&md5=3094371871c8ded8074112b98f145289</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201411658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201411658%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DKoleski%26aufirst%3DE.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DFragment%2520Screening%2520and%2520Druggability%2520Assessment%2520for%2520the%2520CBP%252FP300%2520KIX%2520Domain%2520Through%2520Protein-Observed%252019F%2520NMR%2520Spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D3735%26epage%3D3739%26doi%3D10.1002%2Fanie.201411658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacRaild, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devine, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debono, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scammells, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanlon, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anders, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, R. S.</span><span> </span><span class="NLM_article-title">Ligand-Induced Conformational Change of Plasmodium Falciparum AMA1 Detected Using 19F NMR</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6419</span><span class="NLM_x">–</span> <span class="NLM_lpage">6427</span><span class="refDoi"> DOI: 10.1021/jm500390g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500390g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6419-6427&author=X.+Geauthor=C.+A.+MacRaildauthor=S.+M.+Devineauthor=C.+O.+Debonoauthor=G.+Wangauthor=P.+J.+Scammellsauthor=M.+J.+Scanlonauthor=R.+F.+Andersauthor=M.+Foleyauthor=R.+S.+Norton&title=Ligand-Induced+Conformational+Change+of+Plasmodium+Falciparum+AMA1+Detected+Using+19F+NMR&doi=10.1021%2Fjm500390g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm500390g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500390g%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DX.%26aulast%3DMacRaild%26aufirst%3DC.%2BA.%26aulast%3DDevine%26aufirst%3DS.%2BM.%26aulast%3DDebono%26aufirst%3DC.%2BO.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DAnders%26aufirst%3DR.%2BF.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DLigand-Induced%2520Conformational%2520Change%2520of%2520Plasmodium%2520Falciparum%2520AMA1%2520Detected%2520Using%252019F%2520NMR%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6419%26epage%3D6427%26doi%3D10.1021%2Fjm500390g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arntson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisniewski, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Protein-Observed 19F NMR for Fragment Screening, Affinity, Quantification, and Druggability Assessment</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1427</span><span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+19F+NMR+for+Fragment+Screening%2C+Affinity%2C+Quantification%2C+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%252019F%2520NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%252C%2520Quantification%252C%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arntson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisniewski, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Protein-Observed (19)F-NMR for Fragment Screening, Affinity Quantification and Druggability Assessment</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1427</span><span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+%2819%29F-NMR+for+Fragment+Screening%2C+Affinity+Quantification+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%2520%252819%2529F-NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%2520Quantification%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Fielding, L.</span><span> </span><span class="NLM_article-title">NMR Methods for the Determination of Protein-Ligand Dissociation Constants</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.2174/1568026033392705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.2174%2F1568026033392705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=12577990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD3sXis1KhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=39-53&author=L.+Fielding&title=NMR+Methods+for+the+Determination+of+Protein-Ligand+Dissociation+Constants&doi=10.2174%2F1568026033392705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">NMR methods for the determination of protein-ligand dissociation constants</span></div><div class="casAuthors">Fielding, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-53</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This article is a review with 83 refs. of the application of NMR to the measurement of the dissocn. consts. of protein-ligand complexes.  After briefly discussing some general concepts of mol. stability, the text turns to consider which NMR parameters are reporters of complex formation.  The available data treatments required to translate obsd. NMR effects into quant. measurements of the stability of the complex in the form of the dissocn. const. (KD) are introduced.  Linearization methods and curve fitting methods are explained in detail and are illustrated with examples drawn from recent reports of protein-small mol. interactions.  Throughout the text examples of the commonly obsd. NMR parameters Δδ, 1/T1 and 1/T2 are drawn from biol. studies of 1H, 31P, 19F 15N (and other nuclei).  The advantages of NMR diffusion expts. as a measure of KD are considered.  Some less frequently used NMR approaches, some new ideas and some non-general methods are grouped together in a misc. section.  The major sources of errors in the detn. of KD are identified.  This allows recommendations for optimal exptl. set up.  Options for dealing with strong binding are reviewed.  Finally, the implications of abstracting KD data from high throughput screening expts. are considered and several different approaches to generate this data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYJuFztEcewrVg90H21EOLACvtfcHk0lh3WOi726pc7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXis1KhsQ%253D%253D&md5=cd37fa1c247f9c14eb57e6f44971f7fe</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568026033392705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026033392705%26sid%3Dliteratum%253Aachs%26aulast%3DFielding%26aufirst%3DL.%26atitle%3DNMR%2520Methods%2520for%2520the%2520Determination%2520of%2520Protein-Ligand%2520Dissociation%2520Constants%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26spage%3D39%26epage%3D53%26doi%3D10.2174%2F1568026033392705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Tanifum, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kots, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, B.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbertson, S. R.</span><span> </span><span class="NLM_article-title">Novel Pyridopyrimidine Derivatives as Inhibitors of Stable Toxin a (STa) Induced cGMP Synthesis</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3067</span><span class="NLM_x">–</span> <span class="NLM_lpage">3071</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.bmcl.2009.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3067-3071&author=E.+A.+Tanifumauthor=A.+Y.+Kotsauthor=B.-K.+Choiauthor=F.+Muradauthor=S.+R.+Gilbertson&title=Novel+Pyridopyrimidine+Derivatives+as+Inhibitors+of+Stable+Toxin+a+%28STa%29+Induced+cGMP+Synthesis&doi=10.1016%2Fj.bmcl.2009.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DTanifum%26aufirst%3DE.%2BA.%26aulast%3DKots%26aufirst%3DA.%2BY.%26aulast%3DChoi%26aufirst%3DB.-K.%26aulast%3DMurad%26aufirst%3DF.%26aulast%3DGilbertson%26aufirst%3DS.%2BR.%26atitle%3DNovel%2520Pyridopyrimidine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Stable%2520Toxin%2520a%2520%2528STa%2529%2520Induced%2520cGMP%2520Synthesis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3067%26epage%3D3071%26doi%3D10.1016%2Fj.bmcl.2009.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Magedov, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frolova, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manpadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhoga, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evdokimov, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiou, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnibrugh, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van slambrouck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steelant, W. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuster, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Otterlo, W. A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornienko, A.</span><span> </span><span class="NLM_article-title">Anticancer Properties of an Important Drug Lead Podophyllotoxin Can Be Efficiently Mimicked by Diverse Heterocyclic Scaffolds Accessible via One-Step Synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4234</span><span class="NLM_x">–</span> <span class="NLM_lpage">4246</span><span class="refDoi"> DOI: 10.1021/jm200410r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200410r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4234-4246&author=I.+V.+Magedovauthor=L.+Frolovaauthor=M.+Manpadiauthor=U.+D.+Bhogaauthor=H.+Tangauthor=N.+M.+Evdokimovauthor=O.+Georgeauthor=K.+H.+Georgiouauthor=S.+Rennerauthor=M.+Getlikauthor=T.+L.+Kinnibrughauthor=M.+A.+Fernandesauthor=S.+Van+slambrouckauthor=W.+F.+A.+Steelantauthor=C.+B.+Shusterauthor=S.+Rogeljauthor=W.+A.+L.+van+Otterloauthor=A.+Kornienko&title=Anticancer+Properties+of+an+Important+Drug+Lead+Podophyllotoxin+Can+Be+Efficiently+Mimicked+by+Diverse+Heterocyclic+Scaffolds+Accessible+via+One-Step+Synthesis&doi=10.1021%2Fjm200410r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm200410r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200410r%26sid%3Dliteratum%253Aachs%26aulast%3DMagedov%26aufirst%3DI.%2BV.%26aulast%3DFrolova%26aufirst%3DL.%26aulast%3DManpadi%26aufirst%3DM.%26aulast%3DBhoga%26aufirst%3DU.%2BD.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DEvdokimov%26aufirst%3DN.%2BM.%26aulast%3DGeorge%26aufirst%3DO.%26aulast%3DGeorgiou%26aufirst%3DK.%2BH.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DKinnibrugh%26aufirst%3DT.%2BL.%26aulast%3DFernandes%26aufirst%3DM.%2BA.%26aulast%3DVan%2Bslambrouck%26aufirst%3DS.%26aulast%3DSteelant%26aufirst%3DW.%2BF.%2BA.%26aulast%3DShuster%26aufirst%3DC.%2BB.%26aulast%3DRogelj%26aufirst%3DS.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DKornienko%26aufirst%3DA.%26atitle%3DAnticancer%2520Properties%2520of%2520an%2520Important%2520Drug%2520Lead%2520Podophyllotoxin%2520Can%2520Be%2520Efficiently%2520Mimicked%2520by%2520Diverse%2520Heterocyclic%2520Scaffolds%2520Accessible%2520via%2520One-Step%2520Synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4234%26epage%3D4246%26doi%3D10.1021%2Fjm200410r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Evdokimov, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van slambrouck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffeter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Calvé, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamoral-Theys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooten, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uglinskii, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steelant, W. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornienko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magedov, I. V.</span><span> </span><span class="NLM_article-title">Structural Simplification of Bioactive Natural Products with Multicomponent Synthesis. 3. Fused Uracil-Containing Heterocycles as Novel Topoisomerase-Targeting Agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2012</span><span class="NLM_x">–</span> <span class="NLM_lpage">2021</span><span class="refDoi"> DOI: 10.1021/jm1009428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1009428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2012-2021&author=N.+M.+Evdokimovauthor=S.+Van+slambrouckauthor=P.+Heffeterauthor=L.+Tuauthor=B.+Le+Calv%C3%A9author=D.+Lamoral-Theysauthor=C.+J.+Hootenauthor=P.+Y.+Uglinskiiauthor=S.+Rogeljauthor=R.+Kissauthor=W.+F.+A.+Steelantauthor=W.+Bergerauthor=J.+J.+Yangauthor=C.+G.+Bologaauthor=A.+Kornienkoauthor=I.+V.+Magedov&title=Structural+Simplification+of+Bioactive+Natural+Products+with+Multicomponent+Synthesis.+3.+Fused+Uracil-Containing+Heterocycles+as+Novel+Topoisomerase-Targeting+Agents&doi=10.1021%2Fjm1009428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm1009428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1009428%26sid%3Dliteratum%253Aachs%26aulast%3DEvdokimov%26aufirst%3DN.%2BM.%26aulast%3DVan%2Bslambrouck%26aufirst%3DS.%26aulast%3DHeffeter%26aufirst%3DP.%26aulast%3DTu%26aufirst%3DL.%26aulast%3DLe%2BCalv%25C3%25A9%26aufirst%3DB.%26aulast%3DLamoral-Theys%26aufirst%3DD.%26aulast%3DHooten%26aufirst%3DC.%2BJ.%26aulast%3DUglinskii%26aufirst%3DP.%2BY.%26aulast%3DRogelj%26aufirst%3DS.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DSteelant%26aufirst%3DW.%2BF.%2BA.%26aulast%3DBerger%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DJ.%2BJ.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKornienko%26aufirst%3DA.%26aulast%3DMagedov%26aufirst%3DI.%2BV.%26atitle%3DStructural%2520Simplification%2520of%2520Bioactive%2520Natural%2520Products%2520with%2520Multicomponent%2520Synthesis.%25203.%2520Fused%2520Uracil-Containing%2520Heterocycles%2520as%2520Novel%2520Topoisomerase-Targeting%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2012%26epage%3D2021%26doi%3D10.1021%2Fjm1009428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kamal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suresh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallareddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raju, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamboli, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaik, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalivendi, S. V.</span><span> </span><span class="NLM_article-title">Synthesis of a New 4-Aza-2,3-Didehydropodophyllotoxin Analogues as Potent Cytotoxic and Antimitotic Agents</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2349</span><span class="NLM_x">–</span> <span class="NLM_lpage">2358</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.bmc.2011.02.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2349-2358&author=A.+Kamalauthor=P.+Sureshauthor=A.+Mallareddyauthor=B.+A.+Kumarauthor=P.+V.+Reddyauthor=P.+Rajuauthor=J.+R.+Tamboliauthor=T.+B.+Shaikauthor=N.+Jainauthor=S.+V.+Kalivendi&title=Synthesis+of+a+New+4-Aza-2%2C3-Didehydropodophyllotoxin+Analogues+as+Potent+Cytotoxic+and+Antimitotic+Agents&doi=10.1016%2Fj.bmc.2011.02.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DKamal%26aufirst%3DA.%26aulast%3DSuresh%26aufirst%3DP.%26aulast%3DMallareddy%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DB.%2BA.%26aulast%3DReddy%26aufirst%3DP.%2BV.%26aulast%3DRaju%26aufirst%3DP.%26aulast%3DTamboli%26aufirst%3DJ.%2BR.%26aulast%3DShaik%26aufirst%3DT.%2BB.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DKalivendi%26aufirst%3DS.%2BV.%26atitle%3DSynthesis%2520of%2520a%2520New%25204-Aza-2%252C3-Didehydropodophyllotoxin%2520Analogues%2520as%2520Potent%2520Cytotoxic%2520and%2520Antimitotic%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2349%26epage%3D2358%26doi%3D10.1016%2Fj.bmc.2011.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ioan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micucci, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broccatelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhorov, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budriesi, R.</span><span> </span><span class="NLM_article-title">1,4-Dihydropyridine Scaffold in Medicinal Chemistry, the Story So Far and Perspectives (Part 1): Action in Ion Channels and GPCRs</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4901</span><span class="NLM_x">–</span> <span class="NLM_lpage">4922</span><span class="refDoi"> DOI: 10.2174/092986711797535173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.2174%2F092986711797535173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22050742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=4901-4922&author=P.+Ioanauthor=E.+Carosatiauthor=M.+Micucciauthor=G.+Crucianiauthor=F.+Broccatelliauthor=B.+S.+Zhorovauthor=A.+Chiariniauthor=R.+Budriesi&title=1%2C4-Dihydropyridine+Scaffold+in+Medicinal+Chemistry%2C+the+Story+So+Far+and+Perspectives+%28Part+1%29%3A+Action+in+Ion+Channels+and+GPCRs&doi=10.2174%2F092986711797535173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">1,4-Dihydropyridine scaffold in medicinal chemistry, the story so far and perspectives (Part 1): action in Ion channels and GPCRs</span></div><div class="casAuthors">Ioan, P.; Carosati, E.; Micucci, M.; Cruciani, G.; Broccatelli, F.; Zhorov, B. S.; Chiarini, A.; Budriesi, R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">4901-4922</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the pioneering studies of Fleckenstein and co-workers, L-Type Calcium Channel (LTCC) blockers have attracted large interest due to their effectiveness in treating several cardiovascular diseases.  Medicinal chemists achieved high potency and tissue selectivity by decorating the 1-4-DHP nucleus, the most studied scaffold among LTCC blockers.  Nowadays it is clear that the 1,4-DHP nucleus is a privileged scaffold since, when appropriately substituted, it can selectively modulate diverse receptors, channels and enzymes.  Therefore, the 1,4-DHP scaffold could be used to treat various diseases by a single-ligand multi-target approach.  In this review, we describe the structure-activity relationships of 1,4-DHPs at ion channels, G-protein coupled receptors, and outline the potential for future therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGwJSlDl7P57Vg90H21EOLACvtfcHk0lhagdb4oBizBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtL3K&md5=25335bf1582b042b4d2c4e58557bb896</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.2174%2F092986711797535173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535173%26sid%3Dliteratum%253Aachs%26aulast%3DIoan%26aufirst%3DP.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DMicucci%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DZhorov%26aufirst%3DB.%2BS.%26aulast%3DChiarini%26aufirst%3DA.%26aulast%3DBudriesi%26aufirst%3DR.%26atitle%3D1%252C4-Dihydropyridine%2520Scaffold%2520in%2520Medicinal%2520Chemistry%252C%2520the%2520Story%2520So%2520Far%2520and%2520Perspectives%2520%2528Part%25201%2529%253A%2520Action%2520in%2520Ion%2520Channels%2520and%2520GPCRs%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D4901%26epage%3D4922%26doi%3D10.2174%2F092986711797535173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Hantzsch, A.</span><span> </span><span class="NLM_article-title">Condensationsprodukte Aus Aldehydammoniak Und Ketonartigen Verbindungen</span> <span class="citation_source-journal">Ber. Dtsch. Chem. Ges.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1881</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1637</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span><span class="refDoi"> DOI: 10.1002/cber.18810140214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fcber.18810140214" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1881&pages=1637-1638&author=A.+Hantzsch&title=Condensationsprodukte+Aus+Aldehydammoniak+Und+Ketonartigen+Verbindungen&doi=10.1002%2Fcber.18810140214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcber.18810140214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.18810140214%26sid%3Dliteratum%253Aachs%26aulast%3DHantzsch%26aufirst%3DA.%26atitle%3DCondensationsprodukte%2520Aus%2520Aldehydammoniak%2520Und%2520Ketonartigen%2520Verbindungen%26jtitle%3DBer.%2520Dtsch.%2520Chem.%2520Ges.%26date%3D1881%26volume%3D14%26spage%3D1637%26epage%3D1638%26doi%3D10.1002%2Fcber.18810140214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Structure-Based Approaches Towards Identification of Fragments for the Low-Druggability ATAD2 Bromodomain</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-Based+Approaches+Towards+Identification+of+Fragments+for+the+Low-Druggability+ATAD2+Bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-Based%2520Approaches%2520Towards%2520Identification%2520of%2520Fragments%2520for%2520the%2520Low-Druggability%2520ATAD2%2520Bromodomain%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2360</span><span class="NLM_x">–</span> <span class="NLM_lpage">2365</span><span class="refDoi"> DOI: 10.1021/cb4003283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Cyclin-Dependent+Kinase+Inhibitor+Dinaciclib+Interacts+with+the+Acetyl-Lysine+Recognition+Site+of+Bromodomains&doi=10.1021%2Fcb4003283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0lgB0UPGGPDUUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DCyclin-Dependent%2520Kinase%2520Inhibitor%2520Dinaciclib%2520Interacts%2520with%2520the%2520Acetyl-Lysine%2520Recognition%2520Site%2520of%2520Bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365%26doi%3D10.1021%2Fcb4003283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Fluorescence Polarization Competition Assay: the Range of Resolvable Inhibitor Potency Is Limited by the Affinity of the Fluorescent Ligand</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=X.+Huang&title=Fluorescence+Polarization+Competition+Assay%3A+the+Range+of+Resolvable+Inhibitor+Potency+Is+Limited+by+the+Affinity+of+the+Fluorescent+Ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0lgB0UPGGPDUUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520Polarization%2520Competition%2520Assay%253A%2520the%2520Range%2520of%2520Resolvable%2520Inhibitor%2520Potency%2520Is%2520Limited%2520by%2520the%2520Affinity%2520of%2520the%2520Fluorescent%2520Ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0ljOYoAG7gnP9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Tanaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeAngelo, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Design and Characterization of Bivalent BET Inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+Characterization+of+Bivalent+BET+Inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0ljOYoAG7gnP9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Bivalent%2520BET%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in Vivo Protein Knockdown by Small-Molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+Vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0ljOYoAG7gnP9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520Vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlmeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9251</span><span class="NLM_x">–</span> <span class="NLM_lpage">9264</span><span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0li4NTBzsrej6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Baud, M. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Shiao, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardote, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pschibul, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwan, T. T.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Chemical Biology. a Bump-and-Hole Approach to Engineer Controlled Selectivity of BET Bromodomain Chemical Probes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">641</span><span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=Chemical+Biology.+a+Bump-and-Hole+Approach+to+Engineer+Controlled+Selectivity+of+BET+Bromodomain+Chemical+Probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0li4NTBzsrej6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DChemical%2520Biology.%2520a%2520Bump-and-Hole%2520Approach%2520to%2520Engineer%2520Controlled%2520Selectivity%2520of%2520BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Raux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voitovich, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derviaux, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebuffet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milhas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillemot, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunel, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collette, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roche, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morelli, X.</span><span> </span><span class="NLM_article-title">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-Terminal Domain (BET) Proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1634</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+Selective+Inhibition+of+the+First+Bromodomain+of+the+Human+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0li4NTBzsrej6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520Selective%2520Inhibition%2520of%2520the%2520First%2520Bromodomain%2520of%2520the%2520Human%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Elzein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalla, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gimbel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustig, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zablocki, J.</span><span> </span><span class="NLM_article-title">Discovery of a Novel A2B Adenosine Receptor Antagonist as a Clinical Candidate for Chronic Inflammatory Airway Diseases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2267</span><span class="NLM_x">–</span> <span class="NLM_lpage">2278</span><span class="refDoi"> DOI: 10.1021/jm7014815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7014815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2267-2278&author=E.+Elzeinauthor=R.+V.+Kallaauthor=X.+Liauthor=T.+Perryauthor=A.+Gimbelauthor=D.+Zengauthor=D.+Lustigauthor=K.+Leungauthor=J.+Zablocki&title=Discovery+of+a+Novel+A2B+Adenosine+Receptor+Antagonist+as+a+Clinical+Candidate+for+Chronic+Inflammatory+Airway+Diseases&doi=10.1021%2Fjm7014815"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm7014815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7014815%26sid%3Dliteratum%253Aachs%26aulast%3DElzein%26aufirst%3DE.%26aulast%3DKalla%26aufirst%3DR.%2BV.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPerry%26aufirst%3DT.%26aulast%3DGimbel%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLustig%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DZablocki%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520A2B%2520Adenosine%2520Receptor%2520Antagonist%2520as%2520a%2520Clinical%2520Candidate%2520for%2520Chronic%2520Inflammatory%2520Airway%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2267%26epage%3D2278%26doi%3D10.1021%2Fjm7014815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Devasthale, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fevig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrity, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cap, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levesque, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moulin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirby, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamann, L. G.</span><span> </span><span class="NLM_article-title">Optimization of Activity, Selectivity, and Liability Profiles in 5-Oxopyrrolopyridine DPP4 Inhibitors Leading to Clinical Candidate (Sa)-2-(3-(Aminomethyl)-4-(2,4-Dichlorophenyl)-2-Methyl-5-Oxo-5H-Pyrrolo[3,4-B]Pyridin-6(7H)-Yl)-N,N-Dimethylacetamide (BMS-767778)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7343</span><span class="NLM_x">–</span> <span class="NLM_lpage">7357</span><span class="refDoi"> DOI: 10.1021/jm4008906</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4008906" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7343-7357&author=P.+Devasthaleauthor=Y.+Wangauthor=W.+Wangauthor=J.+Fevigauthor=J.+Fengauthor=A.+Wangauthor=T.+Harrityauthor=D.+Eganauthor=N.+Morganauthor=M.+Capauthor=A.+Furaauthor=H.+E.+Kleiauthor=K.+Kishauthor=C.+Weigeltauthor=L.+Sunauthor=P.+Levesqueauthor=F.+Moulinauthor=Y.-X.+Liauthor=R.+Zahlerauthor=M.+S.+Kirbyauthor=L.+G.+Hamann&title=Optimization+of+Activity%2C+Selectivity%2C+and+Liability+Profiles+in+5-Oxopyrrolopyridine+DPP4+Inhibitors+Leading+to+Clinical+Candidate+%28Sa%29-2-%283-%28Aminomethyl%29-4-%282%2C4-Dichlorophenyl%29-2-Methyl-5-Oxo-5H-Pyrrolo%5B3%2C4-B%5DPyridin-6%287H%29-Yl%29-N%2CN-Dimethylacetamide+%28BMS-767778%29&doi=10.1021%2Fjm4008906"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm4008906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4008906%26sid%3Dliteratum%253Aachs%26aulast%3DDevasthale%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFevig%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DHarrity%26aufirst%3DT.%26aulast%3DEgan%26aufirst%3DD.%26aulast%3DMorgan%26aufirst%3DN.%26aulast%3DCap%26aufirst%3DM.%26aulast%3DFura%26aufirst%3DA.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DWeigelt%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLevesque%26aufirst%3DP.%26aulast%3DMoulin%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DY.-X.%26aulast%3DZahler%26aufirst%3DR.%26aulast%3DKirby%26aufirst%3DM.%2BS.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DOptimization%2520of%2520Activity%252C%2520Selectivity%252C%2520and%2520Liability%2520Profiles%2520in%25205-Oxopyrrolopyridine%2520DPP4%2520Inhibitors%2520Leading%2520to%2520Clinical%2520Candidate%2520%2528Sa%2529-2-%25283-%2528Aminomethyl%2529-4-%25282%252C4-Dichlorophenyl%2529-2-Methyl-5-Oxo-5H-Pyrrolo%255B3%252C4-B%255DPyridin-6%25287H%2529-Yl%2529-N%252CN-Dimethylacetamide%2520%2528BMS-767778%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7343%26epage%3D7357%26doi%3D10.1021%2Fjm4008906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Kenakin, T. P.</span> <span class="citation_source-book">Pharmacologic Analysis of Drug-Receptor Interaction</span>; <span class="NLM_publisher-name">Raven</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1993</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=T.+P.+Kenakin&title=Pharmacologic+Analysis+of+Drug-Receptor+Interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26btitle%3DPharmacologic%2520Analysis%2520of%2520Drug-Receptor%2520Interaction%26pub%3DRaven%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC – a Free Database of Commercially Available Compounds for Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+a+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eavrEwiBRK50bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520a%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgBN3YJDSSdmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgBN3YJDSSdmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span> </span><span class="NLM_article-title">Drug-Like Properties and the Causes of Poor Solubility and Poor Permeability</span> <span class="citation_source-journal">J. Pharmacol. Toxicol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/S1056-8719(00)00107-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2FS1056-8719%2800%2900107-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=11274893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A280%3ADC%252BD3M7os1OjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2000&pages=235-249&author=C.+A.+Lipinski&title=Drug-Like+Properties+and+the+Causes+of+Poor+Solubility+and+Poor+Permeability&doi=10.1016%2FS1056-8719%2800%2900107-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-like properties and the causes of poor solubility and poor permeability</span></div><div class="casAuthors">Lipinski C A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmacological and toxicological methods</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-49</span>
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    </div><div class="casAbstract">There are currently about 10000 drug-like compounds.  These are sparsely, rather than uniformly, distributed through chemistry space.  True diversity does not exist in experimental combinatorial chemistry screening libraries.  Absorption, distribution, metabolism, and excretion (ADME) and chemical reactivity-related toxicity is low, while biological receptor activity is higher dimensional in chemistry space, and this is partly explainable by evolutionary pressures on ADME to deal with endobiotics and exobiotics.  ADME is hard to predict for large data sets because current ADME experimental screens are multi-mechanisms, and predictions get worse as more data accumulates.  Currently, screening for biological receptor activity precedes or is concurrent with screening for properties related to "drugability." In the future, "drugability" screening may precede biological receptor activity screening.  The level of permeability or solubility needed for oral absorption is related to potency.  The relative importance of poor solubility and poor permeability towards the problem of poor oral absorption depends on the research approach used for lead generation.  A "rational drug design" approach as exemplified by Merck advanced clinical candidates leads to time-dependent higher molecular weight, higher H-bonding properties, unchanged lipophilicity, and, hence, poorer permeability.  A high throughput screening (HTS)-based approach as exemplified by unpublished data on Pfizer (Groton, CT) early candidates leads to higher molecular weight, unchanged H-bonding properties, higher lipophilicity, and, hence, poorer aqueous solubility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmUcEou3dTa_EeosoOnpOLfW6udTcc2ebYElzUB7NBIrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7os1OjsQ%253D%253D&md5=aeb705afe1eccaceaf441c7a19f13ecc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS1056-8719%2800%2900107-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1056-8719%252800%252900107-6%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DDrug-Like%2520Properties%2520and%2520the%2520Causes%2520of%2520Poor%2520Solubility%2520and%2520Poor%2520Permeability%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2000%26volume%3D44%26spage%3D235%26epage%3D249%26doi%3D10.1016%2FS1056-8719%2800%2900107-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lj-nNVYMRQVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lj-nNVYMRQVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lj-nNVYMRQVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lj-nNVYMRQVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljaIpGcdILzKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljaIpGcdILzKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Phenix.Model_vs_Data: a High-Level Tool for the Calculation of Crystallographic Model and Data Statistics</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1107/S0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Phenix.Model_vs_Data%3A+a+High-Level+Tool+for+the+Calculation+of+Crystallographic+Model+and+Data+Statistics&doi=10.1107%2FS0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0ljaIpGcdILzKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPhenix.Model_vs_Data%253A%2520a%2520High-Level%2520Tool%2520for%2520the%2520Calculation%2520of%2520Crystallographic%2520Model%2520and%2520Data%2520Statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2FS0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':[],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 26 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yangfeng Li, Jiong Zhao, Lauren M. Gutgesell, Zhengnan Shen, Kiira Ratia, Katherine Dye, Oleksii Dubrovskyi, Huiping Zhao, Fei Huang, Debra A. Tonetti, Gregory R. J. Thatcher, <span class="NLM_string-name hlFld-ContribAuthor">Rui Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 7186-7210. <a href="https://doi.org/10.1021/acs.jmedchem.0c00456" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00456%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPyrrolopyridone%252BBromodomain%252Band%252BExtra-Terminal%252BMotif%252B%252528BET%252529%252BInhibitors%252BEffective%252Bin%252BEndocrine-Resistant%252BER%25252B%252BBreast%252BCancer%252Bwith%252BAcquired%252BResistance%252Bto%252BFulvestrant%252Band%252BPalbociclib%26aulast%3DLi%26aufirst%3DYangfeng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18032020%26date%3D15062020%26date%3D26052020%26volume%3D63%26issue%3D13%26spage%3D7186%26epage%3D7210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anand Divakaran, Steven E. Kirberger, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">SAR by (Protein-Observed) 19F NMR. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2019,</strong> <em>52 </em>
                                    (12)
                                     , 3407-3418. <a href="https://doi.org/10.1021/acs.accounts.9b00377" title="DOI URL">https://doi.org/10.1021/acs.accounts.9b00377</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.9b00377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.9b00377%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DSAR%252Bby%252B%252528Protein-Observed%252529%252B19F%252BNMR%26aulast%3DDivakaran%26aufirst%3DAnand%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D11072019%26date%3D13112019%26volume%3D52%26issue%3D12%26spage%3D3407%26epage%3D3418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Angela
S. Carlson, Huarui Cui, Anand Divakaran, Jorden A. Johnson, Ryan M. Brunner, William C. K. Pomerantz, <span class="NLM_string-name hlFld-ContribAuthor">Joseph J. Topczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Systematically Mitigating the p38α Activity of Triazole-based BET Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1296-1301. <a href="https://doi.org/10.1021/acsmedchemlett.9b00227" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00227</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00227%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSystematically%252BMitigating%252Bthe%252Bp38%2525CE%2525B1%252BActivity%252Bof%252BTriazole-based%252BBET%252BInhibitors%26aulast%3DCarlson%26aufirst%3DAngela%2BS.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D20052019%26date%3D02082019%26date%3D07082019%26date%3D02082019%26volume%3D10%26issue%3D9%26spage%3D1296%26epage%3D1301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carolyn
N. Paulson, Xianghong Guan, Alex M. Ayoub, Alice Chan, Rezaul M. Karim, William C. K. Pomerantz, Ernst Schönbrunn, Gunda I. Georg, <span class="NLM_string-name hlFld-ContribAuthor">Jon E. Hawkinson</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Characterization of a Fluorescence Polarization Pan-BET Bromodomain Probe. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (12)
                                     , 1223-1229. <a href="https://doi.org/10.1021/acsmedchemlett.8b00380" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00380%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BCharacterization%252Bof%252Ba%252BFluorescence%252BPolarization%252BPan-BET%252BBromodomain%252BProbe%26aulast%3DPaulson%26aufirst%3DCarolyn%2BN.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16082018%26date%3D31102018%26date%3D09112018%26date%3D31102018%26volume%3D9%26issue%3D12%26spage%3D1223%26epage%3D1229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chong Qin, Yang Hu, Bing Zhou, Ester Fernandez-Salas, Chao-Yie Yang, Liu Liu, Donna McEachern, Sally Przybranowski, Mi Wang, Jeanne Stuckey, Jennifer Meagher, Longchuan Bai, Zhuo Chen, Mei Lin, Jiuling Yang, Danya N. Ziazadeh, Fuming Xu, Jiantao Hu, Weiguo Xiang, Liyue Huang, Siwei Li, Bo Wen, Duxin Sun, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (15)
                                     , 6685-6704. <a href="https://doi.org/10.1021/acs.jmedchem.8b00506" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00506</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BQCA570%252Bas%252Ban%252BExceptionally%252BPotent%252Band%252BEfficacious%252BProteolysis%252BTargeting%252BChimera%252B%252528PROTAC%252529%252BDegrader%252Bof%252Bthe%252BBromodomain%252Band%252BExtra-Terminal%252B%252528BET%252529%252BProteins%252BCapable%252Bof%252BInducing%252BComplete%252Band%252BDurable%252BTumor%252BRegression%26aulast%3DQin%26aufirst%3DChong%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D30032018%26date%3D18072018%26volume%3D61%26issue%3D15%26spage%3D6685%26epage%3D6704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Qi Zhang, Jingyu Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Evripidis Gavathiotis</span>. </span><span class="cited-content_cbyCitation_article-title">ICBS 2017 in Shanghai—Illuminating Life with Chemical Innovation. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , 1111-1122. <a href="https://doi.org/10.1021/acschembio.8b00220" title="DOI URL">https://doi.org/10.1021/acschembio.8b00220</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00220%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DICBS%252B2017%252Bin%252BShanghai%2525E2%252580%252594Illuminating%252BLife%252Bwith%252BChemical%252BInnovation%26aulast%3DZhang%26aufirst%3DQi%26date%3D2018%26date%3D2018%26date%3D02052018%26date%3D20042018%26volume%3D13%26issue%3D5%26spage%3D1111%26epage%3D1122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zuzanna  Dominiak</span>, <span class="hlFld-ContribAuthor ">Hubert  Huras</span>, <span class="hlFld-ContribAuthor ">Paweł  Kręcisz</span>, <span class="hlFld-ContribAuthor ">Waldemar  Krzeszowski</span>, <span class="hlFld-ContribAuthor ">Paweł  Szymański</span>, <span class="hlFld-ContribAuthor ">Kamila  Czarnecka</span>. </span><span class="cited-content_cbyCitation_article-title">Promising results in development of male contraception. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>41 </em>, 128005. <a href="https://doi.org/10.1016/j.bmcl.2021.128005" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128005%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPromising%252Bresults%252Bin%252Bdevelopment%252Bof%252Bmale%252Bcontraception%26aulast%3DDominiak%26aufirst%3DZuzanna%26date%3D2021%26volume%3D41%26spage%3D128005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jill E  Long</span>, <span class="hlFld-ContribAuthor ">Min S  Lee</span>, <span class="hlFld-ContribAuthor ">Diana L  Blithe</span>. </span><span class="cited-content_cbyCitation_article-title">Update on Novel Hormonal and Nonhormonal Male Contraceptive Development. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Clinical Endocrinology & Metabolism</span><span> <strong>2021,</strong> <em>106 </em>
                                    (6)
                                     , e2381-e2392. <a href="https://doi.org/10.1210/clinem/dgab034" title="DOI URL">https://doi.org/10.1210/clinem/dgab034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1210/clinem/dgab034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1210%2Fclinem%2Fdgab034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Clinical%2520Endocrinology%2520%2526%2520Metabolism%26atitle%3DUpdate%252Bon%252BNovel%252BHormonal%252Band%252BNonhormonal%252BMale%252BContraceptive%252BDevelopment%26aulast%3DLong%26aufirst%3DJill%2BE%26date%3D2021%26date%3D2021%26volume%3D106%26issue%3D6%26spage%3De2381%26epage%3De2392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leonardo L.G.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Adriano D.  Andricopulo</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-54. <a href="https://doi.org/10.1002/0471266949.bmc141.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc141.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc141.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc141.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DStructure%2525E2%252580%252590Based%252BDrug%252BDesign%26aulast%3DFerreira%26aufirst%3DLeonardo%2BL.G.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D54%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Yang</span>, <span class="hlFld-ContribAuthor ">Jian-Qiong  Yang</span>, <span class="hlFld-ContribAuthor ">Shi-He  Luo</span>, <span class="hlFld-ContribAuthor ">Wen-Jie  Mei</span>, <span class="hlFld-ContribAuthor ">Jian-Yun  Lin</span>, <span class="hlFld-ContribAuthor ">Jia-Qi  Zhan</span>, <span class="hlFld-ContribAuthor ">Zhao-Yang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of N-2(5H)-furanonyl sulfonyl hydrazone derivatives and their biological evaluation in vitro and in vivo activity against MCF-7 breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>107 </em>, 104518. <a href="https://doi.org/10.1016/j.bioorg.2020.104518" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104518</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104518%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BN-2%2525285H%252529-furanonyl%252Bsulfonyl%252Bhydrazone%252Bderivatives%252Band%252Btheir%252Bbiological%252Bevaluation%252Bin%252Bvitro%252Band%252Bin%252Bvivo%252Bactivity%252Bagainst%252BMCF-7%252Bbreast%252Bcancer%252Bcells%26aulast%3DYang%26aufirst%3DKai%26date%3D2021%26volume%3D107%26spage%3D104518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohd.  Muddassir</span>, <span class="hlFld-ContribAuthor ">Kunjal  Soni</span>, <span class="hlFld-ContribAuthor ">Chetan B.  Sangani</span>, <span class="hlFld-ContribAuthor ">Abdullah  Alarifi</span>, <span class="hlFld-ContribAuthor ">Mohd.  Afzal</span>, <span class="hlFld-ContribAuthor ">Naaser A. Y.  Abduh</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>, <span class="hlFld-ContribAuthor ">Poonam  Bhadja</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain and BET family proteins as epigenetic targets in cancer therapy: their degradation, present drugs, and possible PROTACs. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 612-636. <a href="https://doi.org/10.1039/D0RA07971E" title="DOI URL">https://doi.org/10.1039/D0RA07971E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA07971E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA07971E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DBromodomain%252Band%252BBET%252Bfamily%252Bproteins%252Bas%252Bepigenetic%252Btargets%252Bin%252Bcancer%252Btherapy%25253A%252Btheir%252Bdegradation%25252C%252Bpresent%252Bdrugs%25252C%252Band%252Bpossible%252BPROTACs%26aulast%3DMuddassir%26aufirst%3DMohd.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D612%26epage%3D636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Current advances on the development of BET inhibitors: insights from computational methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 127-180. <a href="https://doi.org/10.1016/bs.apcsb.2020.06.002" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.06.002%26sid%3Dliteratum%253Aachs%26atitle%3DCurrent%252Badvances%252Bon%252Bthe%252Bdevelopment%252Bof%252BBET%252Binhibitors%25253A%252Binsights%252Bfrom%252Bcomputational%252Bmethods%26aulast%3DPrieto-Mart%25C3%25ADnez%26aufirst%3DFernando%2BD.%26date%3D2020%26spage%3D127%26epage%3D180%26pub%3DElsevier%26date%3D2020%26volume%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemische Epigenetik: der Einfluss chemischer und chemo‐biologischer Techniken auf die Zielstruktur‐Validierung von Bromodomänen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2019,</strong> <em>131 </em>
                                    (50)
                                     , 18096-18120. <a href="https://doi.org/10.1002/ange.201812164" title="DOI URL">https://doi.org/10.1002/ange.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DChemische%252BEpigenetik%25253A%252Bder%252BEinfluss%252Bchemischer%252Bund%252Bchemo%2525E2%252580%252590biologischer%252BTechniken%252Bauf%252Bdie%252BZielstruktur%2525E2%252580%252590Validierung%252Bvon%252BBromodom%2525C3%2525A4nen%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26date%3D2019%26volume%3D131%26issue%3D50%26spage%3D18096%26epage%3D18120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2019,</strong> <em>58 </em>
                                    (50)
                                     , 17930-17952. <a href="https://doi.org/10.1002/anie.201812164" title="DOI URL">https://doi.org/10.1002/anie.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DChemical%252BEpigenetics%25253A%252BThe%252BImpact%252Bof%252BChemical%252Band%252BChemical%252BBiology%252BTechniques%252Bon%252BBromodomain%252BTarget%252BValidation%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26volume%3D58%26issue%3D50%26spage%3D17930%26epage%3D17952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deheng  Chen</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Ziqin  Yan</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Feilong  Zhou</span>, <span class="hlFld-ContribAuthor ">Xilin  Lyu</span>, <span class="hlFld-ContribAuthor ">Biling  Xu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Yujun  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>182 </em>, 111633. <a href="https://doi.org/10.1016/j.ejmech.2019.111633" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252Bstructural%252Binsight%25252C%252Band%252Bbioactivities%252Bof%252BBY27%252Bas%252Ba%252Bselective%252Binhibitor%252Bof%252Bthe%252Bsecond%252Bbromodomains%252Bof%252BBET%252Bproteins%26aulast%3DChen%26aufirst%3DDeheng%26date%3D2019%26volume%3D182%26spage%3D111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alejandro  Speck-Planche</span>, <span class="hlFld-ContribAuthor ">Marcus T.  Scotti</span>. </span><span class="cited-content_cbyCitation_article-title">BET bromodomain inhibitors: fragment-based in silico design using multi-target QSAR models. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2019,</strong> <em>23 </em>
                                    (3)
                                     , 555-572. <a href="https://doi.org/10.1007/s11030-018-9890-8" title="DOI URL">https://doi.org/10.1007/s11030-018-9890-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-018-9890-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-018-9890-8%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DBET%252Bbromodomain%252Binhibitors%25253A%252Bfragment-based%252Bin%252Bsilico%252Bdesign%252Busing%252Bmulti-target%252BQSAR%252Bmodels%26aulast%3DSpeck-Planche%26aufirst%3DAlejandro%26date%3D2019%26date%3D2018%26volume%3D23%26issue%3D3%26spage%3D555%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven E.  Kirberger</span>, <span class="hlFld-ContribAuthor ">Peter D.  Ycas</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Chen  Chen</span>, <span class="hlFld-ContribAuthor ">Michael F.  Ciccone</span>, <span class="hlFld-ContribAuthor ">Rinette W. L.  Woo</span>, <span class="hlFld-ContribAuthor ">Andrew K.  Urick</span>, <span class="hlFld-ContribAuthor ">Huda  Zahid</span>, <span class="hlFld-ContribAuthor ">Ke  Shi</span>, <span class="hlFld-ContribAuthor ">Hideki  Aihara</span>, <span class="hlFld-ContribAuthor ">Sean D.  McAllister</span>, <span class="hlFld-ContribAuthor ">Mohammed  Kashani-Sabet</span>, <span class="hlFld-ContribAuthor ">Junwei  Shi</span>, <span class="hlFld-ContribAuthor ">Alex  Dickson</span>, <span class="hlFld-ContribAuthor ">Camila O.  dos Santos</span>, <span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Selectivity, ligand deconstruction, and cellular activity analysis of a BPTF bromodomain inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2019,</strong> <em>17 </em>
                                    (7)
                                     , 2020-2027. <a href="https://doi.org/10.1039/C8OB02599A" title="DOI URL">https://doi.org/10.1039/C8OB02599A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8OB02599A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8OB02599A%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DSelectivity%25252C%252Bligand%252Bdeconstruction%25252C%252Band%252Bcellular%252Bactivity%252Banalysis%252Bof%252Ba%252BBPTF%252Bbromodomain%252Binhibitor%26aulast%3DKirberger%26aufirst%3DSteven%2BE.%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D7%26spage%3D2020%26epage%3D2027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William C. K.  Pomerantz</span>, <span class="hlFld-ContribAuthor ">Jorden A.  Johnson</span>, <span class="hlFld-ContribAuthor ">Peter D.  Ycas</span>. </span><span class="cited-content_cbyCitation_article-title">Applied Biophysics for Bromodomain Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 287-337. <a href="https://doi.org/10.1007/7355_2019_79" title="DOI URL">https://doi.org/10.1007/7355_2019_79</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_79%26sid%3Dliteratum%253Aachs%26atitle%3DApplied%252BBiophysics%252Bfor%252BBromodomain%252BDrug%252BDiscovery%26aulast%3DPomerantz%26aufirst%3DWilliam%2BC.%2BK.%26date%3D2019%26date%3D2019%26spage%3D287%26epage%3D337%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jill E  Long</span>, <span class="hlFld-ContribAuthor ">Min S  Lee</span>, <span class="hlFld-ContribAuthor ">Diana L  Blithe</span>. </span><span class="cited-content_cbyCitation_article-title">Male Contraceptive Development: Update on Novel Hormonal and Nonhormonal Methods. </span><span class="cited-content_cbyCitation_journal-name">Clinical Chemistry</span><span> <strong>2019,</strong> <em>65 </em>
                                    (1)
                                     , 153-160. <a href="https://doi.org/10.1373/clinchem.2018.295089" title="DOI URL">https://doi.org/10.1373/clinchem.2018.295089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1373/clinchem.2018.295089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1373%2Fclinchem.2018.295089%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Chemistry%26atitle%3DMale%252BContraceptive%252BDevelopment%25253A%252BUpdate%252Bon%252BNovel%252BHormonal%252Band%252BNonhormonal%252BMethods%26aulast%3DLong%26aufirst%3DJill%2BE%26date%3D2019%26date%3D2019%26volume%3D65%26issue%3D1%26spage%3D153%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramakanth  Pagadala</span>, <span class="hlFld-ContribAuthor ">Uppalaiah  Kusampally</span>. </span><span class="cited-content_cbyCitation_article-title">Benzyltrimethylammonium Dichloroiodate Catalyzed Four‐Component Synthesis of Fully Functionalized Pyrroles Under Eco‐compatible Conditions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2018,</strong> <em>23 </em><a href="https://doi.org/10.1002/jhet.3414" title="DOI URL">https://doi.org/10.1002/jhet.3414</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.3414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.3414%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DBenzyltrimethylammonium%252BDichloroiodate%252BCatalyzed%252BFour%2525E2%252580%252590Component%252BSynthesis%252Bof%252BFully%252BFunctionalized%252BPyrroles%252BUnder%252BEco%2525E2%252580%252590compatible%252BConditions%26aulast%3DPagadala%26aufirst%3DRamakanth%26date%3D2018%26date%3D2018%26volume%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Brd4 for cancer therapy: inhibitors and degraders. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (11)
                                     , 1779-1802. <a href="https://doi.org/10.1039/C8MD00198G" title="DOI URL">https://doi.org/10.1039/C8MD00198G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00198G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00198G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DTargeting%252BBrd4%252Bfor%252Bcancer%252Btherapy%25253A%252Binhibitors%252Band%252Bdegraders%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D11%26spage%3D1779%26epage%3D1802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Musawwer  Khan</span>, <span class="hlFld-ContribAuthor ">Sarfaraz  Khan</span>, <span class="hlFld-ContribAuthor ">Saigal</span>, <span class="hlFld-ContribAuthor ">Subash C.  Sahoo</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient and Eco-Friendly One-Pot Synthesis of Functionalized Furan-2-one, Pyrrol-2-one, and Tetrahydropyridine Using Lemon Juice as a Biodegradable Catalyst. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2018,</strong> <em>3 </em>
                                    (5)
                                     , 1371-1380. <a href="https://doi.org/10.1002/slct.201702933" title="DOI URL">https://doi.org/10.1002/slct.201702933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201702933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201702933%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DEfficient%252Band%252BEco-Friendly%252BOne-Pot%252BSynthesis%252Bof%252BFunctionalized%252BFuran-2-one%25252C%252BPyrrol-2-one%25252C%252Band%252BTetrahydropyridine%252BUsing%252BLemon%252BJuice%252Bas%252Ba%252BBiodegradable%252BCatalyst%26aulast%3DKhan%26aufirst%3DM.%2BMusawwer%26date%3D2018%26date%3D2018%26volume%3D3%26issue%3D5%26spage%3D1371%26epage%3D1380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenyuan  Miao</span>, <span class="hlFld-ContribAuthor ">Xianghong  Guan</span>, <span class="hlFld-ContribAuthor ">Jiewei  Jiang</span>, <span class="hlFld-ContribAuthor ">Gunda I.  Georg</span>. </span><span class="cited-content_cbyCitation_article-title">BRDT Inhibitors for Male Contraceptive Drug Discovery: Current Status. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 287-315. <a href="https://doi.org/10.1007/978-981-13-0773-7_11" title="DOI URL">https://doi.org/10.1007/978-981-13-0773-7_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0773-7_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0773-7_11%26sid%3Dliteratum%253Aachs%26atitle%3DBRDT%252BInhibitors%252Bfor%252BMale%252BContraceptive%252BDrug%252BDiscovery%25253A%252BCurrent%252BStatus%26aulast%3DMiao%26aufirst%3DZhenyuan%26date%3D2018%26date%3D2018%26spage%3D287%26epage%3D315%26pub%3DSpringer%2520Singapore%26atitle%3DTargeting%252BProtein-Protein%252BInteractions%252Bby%252BSmall%252BMolecules%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2018%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng  Gao</span>, <span class="hlFld-ContribAuthor ">Wenting  Zhao</span>, <span class="hlFld-ContribAuthor ">Hongyi  Zhao</span>, <span class="hlFld-ContribAuthor ">Ziyun  Lin</span>, <span class="hlFld-ContribAuthor ">Dongfeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Haihong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">An efficient and facile access to highly functionalized pyrrole derivatives. </span><span class="cited-content_cbyCitation_journal-name">Beilstein Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>14 </em>, 884-890. <a href="https://doi.org/10.3762/bjoc.14.75" title="DOI URL">https://doi.org/10.3762/bjoc.14.75</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3762/bjoc.14.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3762%2Fbjoc.14.75%26sid%3Dliteratum%253Aachs%26jtitle%3DBeilstein%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DAn%252Befficient%252Band%252Bfacile%252Baccess%252Bto%252Bhighly%252Bfunctionalized%252Bpyrrole%252Bderivatives%26aulast%3DGao%26aufirst%3DMeng%26date%3D2018%26date%3D2018%26volume%3D14%26spage%3D884%26epage%3D890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Imran  Ali</span>, <span class="hlFld-ContribAuthor ">Jooyun  Lee</span>, <span class="hlFld-ContribAuthor ">Areum  Go</span>, <span class="hlFld-ContribAuthor ">Gildon  Choi</span>, <span class="hlFld-ContribAuthor ">Kwangho  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel [1,2,4]triazolo[4,3- a ]quinoxaline aminophenyl derivatives as BET inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (20)
                                     , 4606-4613. <a href="https://doi.org/10.1016/j.bmcl.2017.09.025" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.09.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.09.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.09.025%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnovel%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B4%25252C3-%252Ba%252B%25255Dquinoxaline%252Baminophenyl%252Bderivatives%252Bas%252BBET%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DAli%26aufirst%3DImran%26date%3D2017%26volume%3D27%26issue%3D20%26spage%3D4606%26epage%3D4613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) BET bromodomain inhibitors <b>1</b>, <b>2</b>, BI2536 (<b>12</b>), and BI6727 (<b>13</b>). (B) Members of the BET branch of the bromodomain phylogenetic tree. Hits were identified by virtual screening, and binding potential of hits was assessed by fluorescence anisotropy (FA), differential scanning fluorimetry (DSF), and protein-observed <sup>19</sup>F NMR (PrOF NMR). Structure–activity relationship (SAR) studies were subsequently conducted to probe the indicated dihydropyridopyrimidine hit compound. Optimization of compound <b>3</b> based on an X-ray cocrystal structure led to the development of compound <b>3s</b>, with improved potency for both Brd4(1) and BrdT(1). The <i>K</i><sub>i</sub> values shown were determined by fluorescence anisotropy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Compounds from high-throughput virtual screen tested for binding to BrdT(1) in a fluorescence anisotropy assay. (A) Of the 22 compounds tested, the nine shown here bound to BrdT(1). (B) Fluorescence anisotropy binding of compounds <b>3</b>–<b>12</b> to BrdT(1). Compound <b>3</b> (open circles), the tightest binder, was selected for further studies. (C) PrOF NMR titration of compound <b>3</b> into 5FW-BrdT(1). The dashed lines indicate the location of the unperturbed resonance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dihydropyridopyrimidines <b>3</b>–<b>3m</b> and pyridine <b>3n</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and reaction conditions: (a) acetic acid, 110 °C, 6 h, N<sub>2</sub>, 11–81%; (b) compound <b>3</b>, chloranil, DMSO, rt, 2 h, 87%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding isotherm showing chemical shift perturbation from a PrOF NMR titration of 6-amino-1-ethyluracil into 50 μM 5FW-Brd4(1), -BrdT(1), and -BPTF. Perturbed resonances for each protein and their respective <i>K</i><sub>d</sub> values are shown. The ribbon structure for each protein is also shown with tryptophan residues labeled in red. Residue assignment for 5FW-BrdT(1) is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S9</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) PrOF NMR confirms binding of compound <b>3</b> to 5FW-Brd4(1). W81 in Brd4 and W50 in BrdT are broadened at 25 μM <b>3</b>, and a new resonance begins to grow in at higher concentrations. Dashed lines reference the position of the resonance in the absence of ligand. The protein concentration was 45 μM in all experiments. (B) Compound <b>3</b> does not bind to non-BET bromodomain BPTF, shown by the similar spectra in the presence and absence of ligand. Protein concentration was 50 μM in all experiments. (C) Selectivity of <b>3</b> against a panel of bromodomains. BET bromodomains are in bromodomain family II.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of Brd4(1) with compound <b>3</b>. (A) Direct hydrogen bonding interactions (black dotted lines) of compound <b>3</b> with Asn140 and water-mediated interactions with Tyr97 and Pro82. The inhibitor is shown in yellow, and the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map (contoured at 1σ) is indicated as blue mesh. Water molecules are shown as cyan spheres. (B) Space filling model of the inhibitor bound to the acetylated lysine binding site. (C) van der Waals interactions (green dotted lines) with mostly hydrophobic residues of the acetylated lysine binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/medium/jm-2016-01336f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Western blot analysis of the effect of compounds <b>3p</b>, <b>3q</b>, <b>3r</b>, and <b>3s</b> on c-Myc and p21Cip1 levels. Compound <b>2</b> served as a positive control, and vinculin served as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-12/acs.jmedchem.6b01336/20170616/images/large/jm-2016-01336f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01336&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i47">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00432" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00432" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 63 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Matzuk, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agno, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Small-Molecule Inhibition of BRDT for Male Contraception</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">684</span><span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+McKeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-Molecule+Inhibition+of+BRDT+for+Male+Contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0ljvR7Mc89pimg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-Molecule%2520Inhibition%2520of%2520BRDT%2520for%2520Male%2520Contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span> </span><span class="NLM_article-title">Place Your BETs: the Therapeutic Potential of Bromodomains</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+Your+BETs%3A+the+Therapeutic+Potential+of+Bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0ljvR7Mc89pimg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520Your%2520BETs%253A%2520the%2520Therapeutic%2520Potential%2520of%2520Bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET Bromodomain Inhibition as a Therapeutic Strategy to Target C-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+C.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+Bromodomain+Inhibition+as+a+Therapeutic+Strategy+to+Target+C-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lgX_ft8VIz5-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%2BC.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520Bromodomain%2520Inhibition%2520as%2520a%2520Therapeutic%2520Strategy%2520to%2520Target%2520C-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Targeting Bromodomains: Epigenetic Readers of Lysine Acetylation</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span><span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+Bromodomains%3A+Epigenetic+Readers+of+Lysine+Acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lgX_ft8VIz5-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520Bromodomains%253A%2520Epigenetic%2520Readers%2520of%2520Lysine%2520Acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Q.</span><span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor JQ1 Activates HIV Latency Through Antagonizing Brd4 Inhibition of Tat-Transactivation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span><span class="refDoi"> DOI: 10.1093/nar/gks976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1093%2Fnar%2Fgks976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23087374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=277-287&author=Z.+Liauthor=J.+Guoauthor=Y.+Wuauthor=Q.+Zhou&title=The+BET+Bromodomain+Inhibitor+JQ1+Activates+HIV+Latency+Through+Antagonizing+Brd4+Inhibition+of+Tat-Transactivation&doi=10.1093%2Fnar%2Fgks976"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The BET bromodomain inhibitor JQ1 activates HIV latency through antagonizing Brd4 inhibition of Tat-transactivation</span></div><div class="casAuthors">Li, Zichong; Guo, Jia; Wu, Yuntao; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">277-287</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Latent HIV reservoirs are the primary hurdle to eradication of infection.  Identification of agents, pathways and mol. mechanisms that activate latent provirus may, in the presence of highly active antiretroviral therapy, permit clearance of infected cells by the immune system.  Promoter-proximal pausing of RNA polymerase (Pol) II is a major rate-limiting step in HIV gene expression.  The viral Tat protein recruits human Super Elongation Complex (SEC) to paused Pol II to overcome this limitation.  Here, we identify the bromodomain protein Brd4 and its inhibition of Tat-transactivation as a major impediment to latency reactivation.  Brd4 competitively blocks the Tat-SEC interaction on HIV promoter.  The BET bromodomain inhibitor JQ1 dissocs. Brd4 from the HIV promoter to allow Tat recruitment of SEC to stimulate HIV elongation.  JQ1 synergizes with another latency activator prostratin, which promotes Pol II loading onto the viral promoter.  Because JQ1 activates viral latency without inducing global T cell activation, this and other closely related compds. and their antagonization of Brd4 to promote Tat-SEC interaction merit further investigations as effective agents/strategies for eliminating latent HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod_cOKnnk8oLVg90H21EOLACvtfcHk0ljK2nam-bLcHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFertQ%253D%253D&md5=87866d81d13af3d5051844ed6c17c98d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks976%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520JQ1%2520Activates%2520HIV%2520Latency%2520Through%2520Antagonizing%2520Brd4%2520Inhibition%2520of%2520Tat-Transactivation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D277%26epage%3D287%26doi%3D10.1093%2Fnar%2Fgks976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Spiltoir, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demos-Davies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinsey, T. A.</span><span> </span><span class="NLM_article-title">BET Acetyl-Lysine Binding Proteins Control Pathological Cardiac Hypertrophy</span> <span class="citation_source-journal">J. Mol. Cell. Cardiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1016/j.yjmcc.2013.07.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.yjmcc.2013.07.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23939492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2013&pages=175-179&author=J.+I.+Spiltoirauthor=M.+S.+Strattonauthor=M.+A.+Cavasinauthor=K.+Demos-Daviesauthor=B.+G.+Reidauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+A.+McKinsey&title=BET+Acetyl-Lysine+Binding+Proteins+Control+Pathological+Cardiac+Hypertrophy&doi=10.1016%2Fj.yjmcc.2013.07.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">BET acetyl-lysine binding proteins control pathological cardiac hypertrophy</span></div><div class="casAuthors">Spiltoir, Jessica I.; Stratton, Matthew S.; Cavasin, Maria A.; Demos-Davies, Kim; Reid, Brian G.; Qi, Jun; Bradner, James E.; McKinsey, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular and Cellular Cardiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">175-179</span>CODEN:
                <span class="NLM_cas:coden">JMCDAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-2828</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cardiac hypertrophy is an independent predictor of adverse outcomes in patients with heart failure, and thus represents an attractive target for novel therapeutic intervention.  JQ1, a small mol. inhibitor of bromodomain and extraterminal (BET) acetyl-lysine reader proteins, was identified in a high throughput screen designed to discover novel small mol. regulators of cardiomyocyte hypertrophy.  JQ1 dose-dependently blocked agonist-dependent hypertrophy of cultured neonatal rat ventricular myocytes (NRVMs) and reversed the prototypical gene program assocd. with pathol. cardiac hypertrophy.  JQ1 also blocked left ventricular hypertrophy (LVH) and improved cardiac function in adult mice subjected to transverse aortic constriction (TAC).  The BET family consists of BRD2, BRD3, BRD4 and BRDT.  BRD4 protein expression was increased during cardiac hypertrophy, and hypertrophic stimuli promoted recruitment of BRD4 to the transcriptional start site (TSS) of the gene encoding atrial natriuretic factor (ANF).  Binding of BRD4 to the ANF TSS was assocd. with increased phosphorylation of local RNA polymerase II.  These findings define a novel function for BET proteins as signal-responsive regulators of cardiac hypertrophy, and suggest that small mol. inhibitors of these epigenetic reader proteins have potential as therapeutics for heart failure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDUzCSA2tNtrVg90H21EOLACvtfcHk0ljK2nam-bLcHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCju7%252FF&md5=ffe6d80a94c27e5d89b38baf33af365c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.yjmcc.2013.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.yjmcc.2013.07.017%26sid%3Dliteratum%253Aachs%26aulast%3DSpiltoir%26aufirst%3DJ.%2BI.%26aulast%3DStratton%26aufirst%3DM.%2BS.%26aulast%3DCavasin%26aufirst%3DM.%2BA.%26aulast%3DDemos-Davies%26aufirst%3DK.%26aulast%3DReid%26aufirst%3DB.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26atitle%3DBET%2520Acetyl-Lysine%2520Binding%2520Proteins%2520Control%2520Pathological%2520Cardiac%2520Hypertrophy%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Cardiol.%26date%3D2013%26volume%3D63%26spage%3D175%26epage%3D179%26doi%3D10.1016%2Fj.yjmcc.2013.07.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Anand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artero, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alaiti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alazem, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margulies, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappola, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plutzky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haldar, S. M.</span><span> </span><span class="NLM_article-title">BET Bromodomains Mediate Transcriptional Pause Release in Heart Failure</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">582</span><span class="refDoi"> DOI: 10.1016/j.cell.2013.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.cell.2013.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=23911322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WrsbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=569-582&author=P.+Anandauthor=J.+D.+Brownauthor=C.+Y.+Linauthor=J.+Qiauthor=R.+Zhangauthor=P.+C.+Arteroauthor=M.+A.+Alaitiauthor=J.+Bullardauthor=K.+Alazemauthor=K.+B.+Marguliesauthor=T.+P.+Cappolaauthor=M.+Lemieuxauthor=J.+Plutzkyauthor=J.+E.+Bradnerauthor=S.+M.+Haldar&title=BET+Bromodomains+Mediate+Transcriptional+Pause+Release+in+Heart+Failure&doi=10.1016%2Fj.cell.2013.07.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomains Mediate Transcriptional Pause Release in Heart Failure</span></div><div class="casAuthors">Anand, Priti; Brown, Jonathan D.; Lin, Charles Y.; Qi, Jun; Zhang, Rongli; Artero, Pedro Calderon; Alaiti, M. Amer; Bullard, Jace; Alazem, Kareem; Margulies, Kenneth B.; Cappola, Thomas P.; Lemieux, Madeleine; Plutzky, Jorge; Bradner, James E.; Haldar, Saptarsi M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-582</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heart failure (HF) is driven by the interplay between regulatory transcription factors and dynamic alterations in chromatin structure.  Pathol. gene transactivation in HF is assocd. with recruitment of histone acetyl-transferases and local chromatin hyperacetylation.  We therefore assessed the role of acetyl-lysine reader proteins, or bromodomains, in HF.  Using a chem. genetic approach, we establish a central role for BET family bromodomain proteins in gene control during HF pathogenesis.  BET inhibition potently suppresses cardiomyocyte hypertrophy in vitro and pathol. cardiac remodeling in vivo.  Integrative transcriptional and epigenomic analyses reveal that BET proteins function mechanistically as pause-release factors crit. to expression of genes that are central to HF pathogenesis and relevant to the pathobiol. of failing human hearts.  This study implicates epigenetic readers as essential effectors of transcriptional pause release during HF pathogenesis and identifies BET coactivator proteins as therapeutic targets in the heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZbQWpJYqt7bVg90H21EOLACvtfcHk0ljK2nam-bLcHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WrsbzI&md5=33410607098cb42f1a7a52fa7ec0108b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DJ.%2BD.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DArtero%26aufirst%3DP.%2BC.%26aulast%3DAlaiti%26aufirst%3DM.%2BA.%26aulast%3DBullard%26aufirst%3DJ.%26aulast%3DAlazem%26aufirst%3DK.%26aulast%3DMargulies%26aufirst%3DK.%2BB.%26aulast%3DCappola%26aufirst%3DT.%2BP.%26aulast%3DLemieux%26aufirst%3DM.%26aulast%3DPlutzky%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHaldar%26aufirst%3DS.%2BM.%26atitle%3DBET%2520Bromodomains%2520Mediate%2520Transcriptional%2520Pause%2520Release%2520in%2520Heart%2520Failure%26jtitle%3DCell%26date%3D2013%26volume%3D154%26spage%3D569%26epage%3D582%26doi%3D10.1016%2Fj.cell.2013.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Nicodeme, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of Inflammation by a Synthetic Histone Mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+Inflammation+by+a+Synthetic+Histone+Mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lh8AJd0T3h9Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520Inflammation%2520by%2520a%2520Synthetic%2520Histone%2520Mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective Inhibition of BET Bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+Inhibition+of+BET+Bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lh8AJd0T3h9Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520Inhibition%2520of%2520BET%2520Bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Korb, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herre, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zucker-Scharff, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darnell, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">BET Protein Brd4 Activates Transcription in Neurons and BET Inhibitor Jq1 Blocks Memory in Mice</span> <span class="citation_source-journal">Nat. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1464</span><span class="NLM_x">–</span> <span class="NLM_lpage">1473</span><span class="refDoi"> DOI: 10.1038/nn.4095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnn.4095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26301327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2015&pages=1464-1473&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=R.+B.+Darnellauthor=C.+D.+Allis&title=BET+Protein+Brd4+Activates+Transcription+in+Neurons+and+BET+Inhibitor+Jq1+Blocks+Memory+in+Mice&doi=10.1038%2Fnn.4095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BET protein Brd4 activates transcription in neurons and BET inhibitor Jq1 blocks memory in mice</span></div><div class="casAuthors">Korb, Erica; Herre, Margo; Zucker-Scharff, Ilana; Darnell, Robert B.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1464-1473</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precise regulation of transcription is crucial for the cellular mechanisms underlying memory formation.  However, the link between neuronal stimulation and the proteins that directly interact with histone modifications to activate transcription in neurons remains unclear.  Brd4 is a member of the bromodomain and extra-terminal domain (BET) protein family, which binds acetylated histones and is a crit. regulator of transcription in many cell types, including transcription in response to external cues.  Small mol. BET inhibitors are in clin. trials, yet almost nothing is known about Brd4 function in the brain.  Here we show that Brd4 mediates the transcriptional regulation underlying learning and memory.  The loss of Brd4 function affects crit. synaptic proteins, which results in memory deficits in mice but also decreases seizure susceptibility.  Thus Brd4 provides a crit. link between neuronal activation and the transcriptional responses that occur during memory formation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuMwCm6NDBLVg90H21EOLACvtfcHk0lh8AJd0T3h9Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlynsL%252FN&md5=0a09dea39378424cfc003a1e9e40ab9a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnn.4095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.4095%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DBET%2520Protein%2520Brd4%2520Activates%2520Transcription%2520in%2520Neurons%2520and%2520BET%2520Inhibitor%2520Jq1%2520Blocks%2520Memory%2520in%2520Mice%26jtitle%3DNat.%2520Neurosci.%26date%3D2015%26volume%3D18%26spage%3D1464%26epage%3D1473%26doi%3D10.1038%2Fnn.4095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Fong, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, E. Y. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ftouni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morison, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giotopoulos, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S. C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregory, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramsay, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kouzarides, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, S.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huntly, B. J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papenfuss, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawson, M. A.</span><span> </span><span class="NLM_article-title">BET Inhibitor Resistance Emerges From Leukaemia Stem Cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">525</span><span class="NLM_x">, </span> <span class="NLM_fpage">538</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1038/nature14888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnature14888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26367796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=538-542&author=C.+Y.+Fongauthor=O.+Gilanauthor=E.+Y.+N.+Lamauthor=A.+F.+Rubinauthor=S.+Ftouniauthor=D.+Tylerauthor=K.+Stanleyauthor=D.+Sinhaauthor=P.+Yehauthor=J.+Morisonauthor=G.+Giotopoulosauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+C.-W.+Leeauthor=C.+Carpenterauthor=R.+Gregoryauthor=R.+G.+Ramsayauthor=S.+W.+Laneauthor=O.+Abdel-Wahabauthor=T.+Kouzaridesauthor=R.+W.+Johnstoneauthor=S.-J.+Dawsonauthor=B.+J.+P.+Huntlyauthor=R.+K.+Prinjhaauthor=A.+T.+Papenfussauthor=M.+A.+Dawson&title=BET+Inhibitor+Resistance+Emerges+From+Leukaemia+Stem+Cells&doi=10.1038%2Fnature14888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor resistance emerges from leukaemia stem cells</span></div><div class="casAuthors">Fong, Chun Yew; Gilan, Omer; Lam, Enid Y. N.; Rubin, Alan F.; Ftouni, Sarah; Tyler, Dean; Stanley, Kym; Sinha, Devbarna; Yeh, Paul; Morison, Jessica; Giotopoulos, George; Lugo, Dave; Jeffrey, Philip; Lee, Stanley Chun-Wei; Carpenter, Christopher; Gregory, Richard; Ramsay, Robert G.; Lane, Steven W.; Abdel-Wahab, Omar; Kouzarides, Tony; Johnstone, Ricky W.; Dawson, Sarah-Jane; Huntly, Brian J. P.; Prinjha, Rab K.; Papenfuss, Anthony T.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">538-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain and extra terminal protein (BET) inhibitors are first-in-class targeted therapies that deliver a new therapeutic opportunity by directly targeting bromodomain proteins that bind acetylated chromatin marks.  Early clin. trials have shown promise, esp. in acute myeloid leukemia, and therefore the evaluation of resistance mechanisms is crucial to optimize the clin. efficacy of these drugs.  Here we use primary mouse hematopoietic stem and progenitor cells immortalized with the fusion protein MLL-AF9 to generate several single-cell clones that demonstrate resistance, in vitro and in vivo, to the prototypical BET inhibitor, I-BET.  Resistance to I-BET confers cross-resistance to chem. distinct BET inhibitors such as JQ1, as well as resistance to genetic knockdown of BET proteins.  Resistance is not mediated through increased drug efflux or metab., but is shown to emerge from leukemia stem cells both ex vivo and in vivo.  Chromatin-bound BRD4 is globally reduced in resistant cells, whereas the expression of key target genes such as Myc remains unaltered, highlighting the existence of alternative mechanisms to regulate transcription.  We demonstrate that resistance to BET inhibitors, in human and mouse leukemia cells, is in part a consequence of increased Wnt/β-catenin signaling, and neg. regulation of this pathway results in restoration of sensitivity to I-BET in vitro and in vivo.  Together, these findings provide new insights into the biol. of acute myeloid leukemia, highlight potential therapeutic limitations of BET inhibitors, and identify strategies that may enhance the clin. utility of these unique targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Lg1n64SCWrVg90H21EOLACvtfcHk0lgjMdeEQS4cBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL&md5=47d1769e077b1c6c92ff52a051368404</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature14888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14888%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DC.%2BY.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DRubin%26aufirst%3DA.%2BF.%26aulast%3DFtouni%26aufirst%3DS.%26aulast%3DTyler%26aufirst%3DD.%26aulast%3DStanley%26aufirst%3DK.%26aulast%3DSinha%26aufirst%3DD.%26aulast%3DYeh%26aufirst%3DP.%26aulast%3DMorison%26aufirst%3DJ.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%2BC.-W.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamsay%26aufirst%3DR.%2BG.%26aulast%3DLane%26aufirst%3DS.%2BW.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DDawson%26aufirst%3DS.-J.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPapenfuss%26aufirst%3DA.%2BT.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBET%2520Inhibitor%2520Resistance%2520Emerges%2520From%2520Leukaemia%2520Stem%2520Cells%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D538%26epage%3D542%26doi%3D10.1038%2Fnature14888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Syeda, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jakkaraj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span> </span><span class="NLM_article-title">Scalable Syntheses of the BET Bromodomain Inhibitor JQ1</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">3454</span><span class="NLM_x">–</span> <span class="NLM_lpage">3457</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2015.02.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.tetlet.2015.02.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFGntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2015&pages=3454-3457&author=S.+S.+Syedaauthor=S.+Jakkarajauthor=G.+I.+Georg&title=Scalable+Syntheses+of+the+BET+Bromodomain+Inhibitor+JQ1&doi=10.1016%2Fj.tetlet.2015.02.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable syntheses of the BET bromodomain inhibitor JQ1</span></div><div class="casAuthors">Syeda, Shameem Sultana; Jakkaraj, Sudhakar; Georg, Gunda I.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3454-3457</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">We have developed methods involving the use of alternate, safer reagents for the scalable syntheses of the potent BET bromodomain inhibitor JQ1 (I).  A one-pot three step method, involving the conversion of a benzodiazepine to a thioamide using Lawesson's reagent, followed by amidrazone formation and installation of the triazole moiety furnished JQ1.  This method provides good yields and a facile purifn. process.  For the synthesis of enantiomerically enriched (+)-JQ1 (II), the highly toxic reagent di-Et chlorophosphate, used in a previous synthesis, was replaced with the safer reagent di-Ph chlorophosphate in the three-step one-pot triazole formation without effecting yields and enantiomeric purity of (+)-JQ1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpULsbWhpozUbVg90H21EOLACvtfcHk0lgjMdeEQS4cBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFGntLo%253D&md5=2e86bfe5c83a36e13d5016b6a3a25b35</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2015.02.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2015.02.062%26sid%3Dliteratum%253Aachs%26aulast%3DSyeda%26aufirst%3DS.%2BS.%26aulast%3DJakkaraj%26aufirst%3DS.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26atitle%3DScalable%2520Syntheses%2520of%2520the%2520BET%2520Bromodomain%2520Inhibitor%2520JQ1%26jtitle%3DTetrahedron%2520Lett.%26date%3D2015%26volume%3D56%26spage%3D3454%26epage%3D3457%26doi%3D10.1016%2Fj.tetlet.2015.02.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Ran, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meagher, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaswamy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">4927</span><span class="NLM_x">–</span> <span class="NLM_lpage">4939</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%93-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0ljdmgI1UkG_Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%2593-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Hay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttakar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span> </span><span class="NLM_article-title">The Design and Synthesis of 5- and 6-Isoxazolylbenzimidazoles as Selective Inhibitors of the BET Bromodomains</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1039/C2MD20189E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2FC2MD20189E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=140-144&author=D.+Hayauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=S.+Martinauthor=M.+Philpottauthor=S.+Picaudauthor=D.+S.+Hewingsauthor=S.+Uttakarauthor=T.+D.+Heightmanauthor=S.+J.+Conwayauthor=S.+Knappauthor=P.+E.+Brennan&title=The+Design+and+Synthesis+of+5-+and+6-Isoxazolylbenzimidazoles+as+Selective+Inhibitors+of+the+BET+Bromodomains&doi=10.1039%2FC2MD20189E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC2MD20189E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20189E%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DUttakar%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Design%2520and%2520Synthesis%2520of%25205-%2520and%25206-Isoxazolylbenzimidazoles%2520as%2520Selective%2520Inhibitors%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D140%26epage%3D144%26doi%3D10.1039%2FC2MD20189E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clément, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnathan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordaunt, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimes, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajakane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7501</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span><span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+Epigenetic+Regulator+I-BET762%3A+Lead+Optimization+to+Afford+a+Clinical+Candidate+Inhibitor+of+the+BET+Bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0ljdmgI1UkG_Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520Epigenetic%2520Regulator%2520I-BET762%253A%2520Lead%2520Optimization%2520to%2520Afford%2520a%2520Clinical%2520Candidate%2520Inhibitor%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brotherton, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diez-Dacal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingard, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span> </span><span class="NLM_article-title">RVX-208, an Inhibitor of BET Transcriptional Regulators with Selectivity for the Second Bromodomain</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">19754</span><span class="NLM_x">–</span> <span class="NLM_lpage">19759</span><span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+Mullerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+Inhibitor+of+BET+Transcriptional+Regulators+with+Selectivity+for+the+Second+Bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lgUZ0X1m_zf_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520Inhibitor%2520of%2520BET%2520Transcriptional%2520Regulators%2520with%2520Selectivity%2520for%2520the%2520Second%2520Bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Discovery of Chemical Inhibitors of Human Bromodomains</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">115</span><span class="NLM_x">, </span> <span class="NLM_fpage">11625</span><span class="NLM_x">–</span> <span class="NLM_lpage">11688</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00205</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00205" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=11625-11688&author=G.+Zhangauthor=S.+G.+Smithauthor=M.-M.+Zhou&title=Discovery+of+Chemical+Inhibitors+of+Human+Bromodomains&doi=10.1021%2Facs.chemrev.5b00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Chemical Inhibitors of Human Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11625-11668</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-guided drug design strategy has played an important role in the recent discovery and development of bromodomain inhibitors that has yielded a multitude of chemotypes as described in this review.  Going forward, we expect that this structure-based approach will continue to guide medicinal chemists in lead optimization of bromodomain inhibitors and facilitate the efforts to develop new inhibitors for less druggable bromodomains.  Collectively, these emerging new chem. inhibitors will empower our ongoing investigation to better understand the roles of bromodomain proteins in control of gene transcription in human biol. and disease pathol.  These small-mol. inhibitors can also be developed into new epigenetic therapies for new effective treatments of various human diseases including cancer and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHbsgfBguOabVg90H21EOLACvtfcHk0lgUZ0X1m_zf_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qurzP&md5=4014c771384b4a768a80644af22c4485</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00205%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DDiscovery%2520of%2520Chemical%2520Inhibitors%2520of%2520Human%2520Bromodomains%26jtitle%3DChem.%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D11625%26epage%3D11688%26doi%3D10.1021%2Facs.chemrev.5b00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blacklow, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9019</span><span class="NLM_x">–</span> <span class="NLM_lpage">9027</span><span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9027&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+Multicomponent+Reactions+to+Develop+Novel+Bromodomain+Inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0lgUZ0X1m_zf_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520Multicomponent%2520Reactions%2520to%2520Develop%2520Novel%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9027%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="note"><p class="first last">Despite our attempts to remove all PAINS compounds, a rhodanine was tested (see structure in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf" class="ext-link">Figure S1</a>). No measurable binding was detected for this compound in our fluorescence anisotropy assay.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Rudolph, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steegmaier, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grauert, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haslinger, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garin-Chesa, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adolf, G. R.</span><span> </span><span class="NLM_article-title">BI 6727, a Polo-Like Kinase Inhibitor with Improved Pharmacokinetic Profile and Broad Antitumor Activity</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3094</span><span class="NLM_x">–</span> <span class="NLM_lpage">3102</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-08-2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1158%2F1078-0432.CCR-08-2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=19383823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFOrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=3094-3102&author=D.+Rudolphauthor=M.+Steegmaierauthor=M.+Hoffmannauthor=M.+Grauertauthor=A.+Baumauthor=J.+Quantauthor=C.+Haslingerauthor=P.+Garin-Chesaauthor=G.+R.+Adolf&title=BI+6727%2C+a+Polo-Like+Kinase+Inhibitor+with+Improved+Pharmacokinetic+Profile+and+Broad+Antitumor+Activity&doi=10.1158%2F1078-0432.CCR-08-2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">BI 6727, A Polo-like Kinase Inhibitor with Improved Pharmacokinetic Profile and Broad Antitumor Activity</span></div><div class="casAuthors">Rudolph, Dorothea; Steegmaier, Martin; Hoffmann, Matthias; Grauert, Matthias; Baum, Anke; Quant, Jens; Haslinger, Christian; Garin-Chesa, Pilar; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3094-3102</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Antimitotic chemotherapy remains a cornerstone of multimodality treatment for locally advanced and metastatic cancers.  To identify novel mitosis-specific agents with higher selectivity than approved tubulin-binding agents (taxanes, Vinca alkaloids), we have generated inhibitors of Polo-like kinase 1, a target that functions predominantly in mitosis.  The first compd. in this series, suitable for i.v. administration, has entered clin. development.  To fully explore the potential of Polo-like kinase 1 inhibition in oncol., we have profiled addnl. compds. and now describe a novel clin. candidate.  BI 6727 is a highly potent (enzyme IC50 = 0.87 nmol/L, EC50 = 11-37 nmol/L on a panel of cancer cell lines) and selective dihydropteridinone with distinct properties.  First, BI 6727 has a pharmacokinetic profile favoring sustained exposure of tumor tissues with a high vol. of distribution and a long terminal half-life in mice (Vss = 7.6 L/kg, t1/2 = 46 h) and rats (Vss = 22 L/kg, t1/2 = 54 h).  Second, BI 6727 has physicochem. and pharmacokinetic properties that allow in vivo testing of i.v. as well as oral formulations, adding flexibility to dosing schedules.  Finally, BI 6727 shows marked antitumor activity in multiple cancer models, including a model of taxane-resistant colorectal cancer.  With oral and i.v. routes of administration, the total weekly dose of BI 6727 is most relevant for efficacy, supporting the use of a variety of well-tolerated dosing schedules.  These findings warrant further investigation of BI 6727 as a tailored antimitotic agent; clin. studies have been initiated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-YEX91EOHHLVg90H21EOLACvtfcHk0liPz6wXJidlPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFOrtbo%253D&md5=daed550be9d9b99ac6797c359039d295</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-08-2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-08-2445%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DD.%26aulast%3DSteegmaier%26aufirst%3DM.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DGrauert%26aufirst%3DM.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DQuant%26aufirst%3DJ.%26aulast%3DHaslinger%26aufirst%3DC.%26aulast%3DGarin-Chesa%26aufirst%3DP.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DBI%25206727%252C%2520a%2520Polo-Like%2520Kinase%2520Inhibitor%2520with%2520Improved%2520Pharmacokinetic%2520Profile%2520and%2520Broad%2520Antitumor%2520Activity%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D3094%26epage%3D3102%26doi%3D10.1158%2F1078-0432.CCR-08-2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cassel, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhikari, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">dos Santos, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">2246</span><span class="NLM_x">–</span> <span class="NLM_lpage">2256</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2246-2256&author=A.+K.+Urickauthor=L.+M.+L.+Hawkauthor=M.+K.+Casselauthor=N.+K.+Mishraauthor=S.+Liuauthor=N.+Adhikariauthor=W.+Zhangauthor=C.+O.+dos+Santosauthor=J.+L.+Hallauthor=W.+C.+K.+Pomerantz&title=Dual+Screening+of+BPTF+and+Brd4+Using+Protein-Observed+Fluorine+NMR+Uncovers+New+Bromodomain+Probe+Molecules&doi=10.1021%2Facschembio.5b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Screening of BPTF and Brd4 Using Protein-Observed Fluorine NMR Uncovers New Bromodomain Probe Molecules</span></div><div class="casAuthors">Urick, Andrew K.; Hawk, Laura M. L.; Cassel, Melissa K.; Mishra, Neeraj K.; Liu, Shuai; Adhikari, Neeta; Zhang, Wei; dos Santos, Camila O.; Hall, Jennifer L.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2246-2256</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein dysregulation is linked to cancer, diabetes, and inflammation.  Selective inhibition of bromodomain function is a newly proposed therapeutic strategy.  The authors describe a 19F NMR dual screening method for small mol. discovery using fluorinated tryptophan resonances on two bromodomain-contg. proteins.  The chem. shift dispersion of 19F resonances within fluorine-labeled proteins enables the simultaneous anal. of two fluorinated bromodomains by NMR.  A library of 229 small mols. was screened against the first bromodomain of Brd4 and the BPTF bromodomain.  The authors report the first small mol. selective for BPTF over Brd4, termed AU1.  The Kd = 2.8 μM for AU1, which is active in a cell-based reporter assay.  No binding was detected with Brd4.  Three new Brd4 inhibitors with submicromolar affinity were also discovered.  Brd4 hits were validated in a thermal stability assay and potency detd. via fluorescence anisotropy.  The speed, ease of interpretation, and low protein concn. needed for protein-obsd. 19F NMR expts. in a multiprotein format offers a new method to discover and characterize selective ligands for bromodomain-contg. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjhnKasUdNL7Vg90H21EOLACvtfcHk0liPz6wXJidlPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCiurfI&md5=584fa6b83f8bfa661f55edb1078e570c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00483%26sid%3Dliteratum%253Aachs%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DCassel%26aufirst%3DM.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3Ddos%2BSantos%26aufirst%3DC.%2BO.%26aulast%3DHall%26aufirst%3DJ.%2BL.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DDual%2520Screening%2520of%2520BPTF%2520and%2520Brd4%2520Using%2520Protein-Observed%2520Fluorine%2520NMR%2520Uncovers%2520New%2520Bromodomain%2520Probe%2520Molecules%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2246%26epage%3D2256%26doi%3D10.1021%2Facschembio.5b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span> </span><span class="NLM_article-title">Leveraging Kinase Inhibitors to Develop Small Molecule Tools for Imaging Kinases by Fluorescence Microscopy</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2523</span><span class="NLM_x">–</span> <span class="NLM_lpage">2524</span><span class="refDoi"> DOI: 10.1039/c2mb25099c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2Fc2mb25099c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22673640" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2523-2524&author=Z.+Zhangauthor=N.+Kwiatkowskiauthor=H.+Zengauthor=S.+M.+Limauthor=N.+S.+Grayauthor=W.+Zhangauthor=P.+L.+Yang&title=Leveraging+Kinase+Inhibitors+to+Develop+Small+Molecule+Tools+for+Imaging+Kinases+by+Fluorescence+Microscopy&doi=10.1039%2Fc2mb25099c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging kinase inhibitors to develop small molecule tools for imaging kinases by fluorescence microscopy</span></div><div class="casAuthors">Zhang, Zijuan; Kwiatkowski, Nicholas; Zeng, Hong; Lim, Sang Min; Gray, Nathanael S.; Zhang, Wei; Yang, Priscilla L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2523-2526</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">As the usage of fluorescence microscopy as a tool to study biol. systems continues to grow, so does the need for addnl. tools that permit the selective detection of proteins of interest.  Existing selective and well-characterized kinase inhibitors may be exploited to develop novel small mol. probes useful in imaging kinases by fluorescence microscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkTi2Qtkh07Vg90H21EOLACvtfcHk0liPz6wXJidlPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ygtbnF&md5=e59fd8a1bbd080bcfa2653fd7250328e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2Fc2mb25099c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb25099c%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DP.%2BL.%26atitle%3DLeveraging%2520Kinase%2520Inhibitors%2520to%2520Develop%2520Small%2520Molecule%2520Tools%2520for%2520Imaging%2520Kinases%2520by%2520Fluorescence%2520Microscopy%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D2523%26epage%3D2524%26doi%3D10.1039%2Fc2mb25099c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uttarkar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">Bromodomain-Peptide Displacement Assays for Interactome Mapping and Inhibitor Discovery</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2899</span><span class="NLM_x">–</span> <span class="NLM_lpage">2908</span><span class="refDoi"> DOI: 10.1039/c1mb05099k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2Fc1mb05099k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=21804994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=2899-2908&author=M.+Philpottauthor=J.+Yangauthor=T.+Tumberauthor=O.+Fedorovauthor=S.+Uttarkarauthor=P.+Filippakopoulosauthor=S.+Picaudauthor=T.+Keatesauthor=I.+Felletarauthor=A.+Ciulliauthor=S.+Knappauthor=T.+D.+Heightman&title=Bromodomain-Peptide+Displacement+Assays+for+Interactome+Mapping+and+Inhibitor+Discovery&doi=10.1039%2Fc1mb05099k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-peptide displacement assays for interactome mapping and inhibitor discovery</span></div><div class="casAuthors">Philpott, Martin; Yang, Jing; Tumber, Tony; Fedorov, Oleg; Uttarkar, Sagar; Filippakopoulos, Panagis; Picaud, Sarah; Keates, Tracy; Felletar, Ildiko; Ciulli, Alessio; Knapp, Stefan; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2899-2908</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone lysine acetylation is a key component of epigenetic regulation of gene transcription.  Bromodomains, found in histone acetyl transferases and other chromatin-assocd. proteins, bind selectively to acetylated lysines, acting as "readers" of the histone code, and have recently been shown to contain a druggable binding pocket.  Here we report the development of high-throughput assays that quantify the binding of bromodomains to acetylated histone peptides.  We have used these assays to screen for histone binding partners of as yet uncharacterized bromodomains, adding to current knowledge of the histone code and expanding the repertoire of assays for chem. probe discovery.  We have also demonstrated that these assays can be used to detect small mol. binding from the very weak to the nanomolar range.  This assay methodol. is thereby anticipated to provide the basis both for broader interactome profiling and for small mol. inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mWM_E4IUzbVg90H21EOLACvtfcHk0lio4vaLDA6MFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFakt77P&md5=0f9eccb2d2761e1707c9b04bccc76050</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2Fc1mb05099k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1mb05099k%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DT.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DUttarkar%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DBromodomain-Peptide%2520Displacement%2520Assays%2520for%2520Interactome%2520Mapping%2520and%2520Inhibitor%2520Discovery%26jtitle%3DMol.%2520BioSyst.%26date%3D2011%26volume%3D7%26spage%3D2899%26epage%3D2908%26doi%3D10.1039%2Fc1mb05099k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Zhao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, B.</span><span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: 2. Structure-Based Optimization</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1281</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1021/jm501504k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501504k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1281-1297&author=L.+Zhaoauthor=Y.+Wangauthor=D.+Caoauthor=T.+Chenauthor=Q.+Wangauthor=Y.+Liauthor=Y.+Xuauthor=N.+Zhangauthor=X.+Wangauthor=D.+Chenauthor=L.+Chenauthor=Y.-L.+Chenauthor=G.+Xiaauthor=Z.+Shiauthor=Y.-C.+Liuauthor=Y.+Linauthor=Z.+Miaoauthor=J.+Shenauthor=B.+Xiong&title=Fragment-Based+Drug+Discovery+of+2-Thiazolidinones+as+BRD4+Inhibitors%3A+2.+Structure-Based+Optimization&doi=10.1021%2Fjm501504k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery of 2-Thiazolidinones as BRD4 Inhibitors: Structure-Based Optimization</span></div><div class="casAuthors">Zhao, Lele; Wang, Yingqing; Cao, Danyan; Chen, Tiantian; Wang, Qi; Li, Yanlian; Xu, Yechun; Zhang, Naixia; Wang, Xin; Chen, Danqi; Chen, Lin; Chen, Yue-Lei; Xia, Guangxin; Shi, Zhe; Liu, Yu-Chih; Lin, Yijyun; Miao, Zehong; Shen, Jingkang; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1281-1297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The signal transduction of acetylated histone can be processed through a recognition module, bromodomain.  Several inhibitors targeting BRD4, one of the bromodomain members, are in clin. trials as anticancer drugs.  Hereby, the authors report the efforts on discovery and optimization of a new series of 2-thiazolidinones as BRD4 inhibitors along the previous study.  In this work, guided by crystal structure anal., the authors reversed the sulfonamide group and identified a new binding mode.  A structure-activity relationship study on this new series led to several potent BRD4 inhibitors with IC50 of about 0.05-0.1 μM in FP binding assay and GI50 of 0.1-0.3 μM in cell based assays.  To complete the lead-like assessment of this series, the authors further checked its effects on BRD4 downstream protein c-Myc, investigated its selectivity among five different bromodomain proteins, as well as the metabolic stability test, and reinforced the utility of 2-thiazolidinone scaffold as BET bromodomain inhibitors in novel anticancer drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4x-5lE2iPzbVg90H21EOLACvtfcHk0lio4vaLDA6MFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSqtw%253D%253D&md5=4056ac1a6d6598dc61be7cc7ef82a3da</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm501504k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501504k%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.-L.%26aulast%3DXia%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520of%25202-Thiazolidinones%2520as%2520BRD4%2520Inhibitors%253A%25202.%2520Structure-Based%2520Optimization%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1281%26epage%3D1297%26doi%3D10.1021%2Fjm501504k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamotte, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delannée, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouillot, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gellibert, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krysa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudit, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trottet, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodeme, E.</span><span> </span><span class="NLM_article-title">Naphthyridines as Novel BET Family Bromodomain Inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">589</span><span class="refDoi"> DOI: 10.1002/cmdc.201300259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fcmdc.201300259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24000170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGlsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=580-589&author=O.+Mirguetauthor=Y.+Lamotteauthor=C.-W.+Chungauthor=P.+Bamboroughauthor=D.+Delann%C3%A9eauthor=A.+Bouillotauthor=F.+Gellibertauthor=G.+Krysaauthor=A.+Lewisauthor=J.+Witheringtonauthor=P.+Huetauthor=Y.+Duditauthor=L.+Trottetauthor=E.+Nicodeme&title=Naphthyridines+as+Novel+BET+Family+Bromodomain+Inhibitors&doi=10.1002%2Fcmdc.201300259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Naphthyridines as Novel BET Family Bromodomain Inhibitors</span></div><div class="casAuthors">Mirguet, Olivier; Lamotte, Yann; Chung, Chun-wa; Bamborough, Paul; Delannee, Delphine; Bouillot, Anne; Gellibert, Francoise; Krysa, Gael; Lewis, Antonia; Witherington, Jason; Huet, Pascal; Dudit, Yann; Trottet, Lionel; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">580-589</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are small protein domains found in a variety of proteins that recognize and bind to acetylated histone tails.  This binding affects chromatin structure and facilitates the localization of transcriptional complexes to specific genes, thereby regulating epigenetically controlled processes including gene transcription and mRNA elongation.  Inhibitors of the bromodomain and extra-terminal (BET) proteins BRD2-4 and T, which prevent bromodomain binding to acetyl-modified histone tails, have shown therapeutic promise in several diseases.  We report here the discovery of 1,5-naphthyridine derivs., e.g. I, as potent inhibitors of the BET bromodomain family with good cell activity and oral pharmacokinetic parameters.  X-ray crystal structures of naphthyridine isomers have been solved and quantum mech. calcns. have been used to explain the higher affinity of the 1,5-isomer over the others.  The best compds. were progressed in a mouse model of inflammation and exhibited dose-dependent anti-inflammatory pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEqN70fkD1t7Vg90H21EOLACvtfcHk0lio4vaLDA6MFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGlsLzI&md5=bb69d7ae0f7de3788617315a76bc8005</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300259%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DDelann%25C3%25A9e%26aufirst%3DD.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DHuet%26aufirst%3DP.%26aulast%3DDudit%26aufirst%3DY.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DNaphthyridines%2520as%2520Novel%2520BET%2520Family%2520Bromodomain%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D580%26epage%3D589%26doi%3D10.1002%2Fcmdc.201300259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Chung, C.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dean, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span> </span><span class="NLM_article-title">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">576</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span><span class="refDoi"> DOI: 10.1021/jm201320w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201320w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=576-586&author=C.-W.+Chungauthor=A.+W.+Deanauthor=J.+M.+Woolvenauthor=P.+Bamborough&title=Fragment-Based+Discovery+of+Bromodomain+Inhibitors+Part+1%3A+Inhibitor+Binding+Modes+and+Implications+for+Lead+Discovery&doi=10.1021%2Fjm201320w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of Bromodomain Inhibitors Part 1: Inhibitor Binding Modes and Implications for Lead Discovery</span></div><div class="casAuthors">Chung, Chun-wa; Dean, Anthony W.; Woolven, James M.; Bamborough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">576-586</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are key epigenetic regulators of gene transcription and readers of the histone code.  However, the therapeutic benefits of modulating this target class are largely unexplored due to the lack of suitable chem. probes.  This article describes the generation of lead mols. for the BET bromodomains through screening a fragment set chosen using structural insights and computational approaches.  Anal. of 40 BRD2/fragment x-ray complexes highlights both shared and disparate interaction features that may be exploited for affinity and selectivity.  Six representative crystal structures are then exemplified in detail.  Two of the fragments are completely new bromodomain chemotypes, and three have never before been crystd. in a bromodomain, so our results significantly extend the limited public knowledge-base of crystallog. small mol./bromodomain interactions.  Certain fragments (including paracetamol) bind in a consistent mode to different bromodomains such as CREBBP, suggesting their potential to act as generic bromodomain templates.  An important implication is that the bromodomains are not only a phylogenetic family but also a system in which chem. and structural knowledge of one bromodomain gives insights transferable to others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonNb7GnZnS_rVg90H21EOLACvtfcHk0lg1nR6olh2kog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKktrnN&md5=5b7d7239d0de57266d94beef1ea080ae</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm201320w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201320w%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DDean%26aufirst%3DA.%2BW.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26atitle%3DFragment-Based%2520Discovery%2520of%2520Bromodomain%2520Inhibitors%2520Part%25201%253A%2520Inhibitor%2520Binding%2520Modes%2520and%2520Implications%2520for%2520Lead%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D576%26epage%3D586%26doi%3D10.1021%2Fjm201320w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Ember, S. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berndt, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span> </span><span class="NLM_article-title">Acetyl-Lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1160</span><span class="NLM_x">–</span> <span class="NLM_lpage">1171</span><span class="refDoi"> DOI: 10.1021/cb500072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1160-1171&author=S.+W.+J.+Emberauthor=J.-Y.+Zhuauthor=S.+H.+Olesenauthor=M.+P.+Martinauthor=A.+Beckerauthor=N.+Berndtauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Acetyl-Lysine+Binding+Site+of+Bromodomain-Containing+Protein+4+%28BRD4%29+Interacts+with+Diverse+Kinase+Inhibitors&doi=10.1021%2Fcb500072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-lysine Binding Site of Bromodomain-Containing Protein 4 (BRD4) Interacts with Diverse Kinase Inhibitors</span></div><div class="casAuthors">Ember, Stuart W. J.; Zhu, Jin-Yi; Olesen, Sanne H.; Martin, Mathew P.; Becker, Andreas; Berndt, Norbert; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1160-1171</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Members of the bromodomain and extra terminal (BET) family of proteins are essential for the recognition of acetylated lysine (KAc) residues in histones and have emerged as promising drug targets in cancer, inflammation, and contraception research.  In co-crystn. screening campaigns using the first bromodomain of BRD4 (BRD4-1) against kinase inhibitor libraries, we identified and characterized 14 kinase inhibitors (10 distinct chem. scaffolds) as ligands of the KAc binding site.  Among these, the PLK1 inhibitor BI2536 and the JAK2 inhibitor TG101209 displayed strongest inhibitory potential against BRD4 (IC50 = 25 nM and 130 nM, resp.) and high selectivity for BET bromodomains.  Comparative structural anal. revealed markedly different binding modes of kinase hinge-binding scaffolds in the KAc binding site, suggesting that BET proteins are potential off-targets of diverse kinase inhibitors.  Combined, these findings provide a new structural framework for the rational design of next-generation BET-selective and dual-activity BET-kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0X1hjt7cBbVg90H21EOLACvtfcHk0lg1nR6olh2kog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1ymt7g%253D&md5=5ca37df00cdaa0ba81c0cb4a0f70ba5d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fcb500072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500072z%26sid%3Dliteratum%253Aachs%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DAcetyl-Lysine%2520Binding%2520Site%2520of%2520Bromodomain-Containing%2520Protein%25204%2520%2528BRD4%2529%2520Interacts%2520with%2520Diverse%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1160%26epage%3D1171%26doi%3D10.1021%2Fcb500072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Pomerantz, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipinski, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cierpicki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapp, A. K.</span><span> </span><span class="NLM_article-title">Profiling the Dynamic Interfaces of Fluorinated Transcription Complexes for Ligand Discovery and Characterization</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1345</span><span class="NLM_x">–</span> <span class="NLM_lpage">1350</span><span class="refDoi"> DOI: 10.1021/cb3002733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3002733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVWms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=1345-1350&author=W.+C.+Pomerantzauthor=N.+Wangauthor=A.+K.+Lipinskiauthor=R.+Wangauthor=T.+Cierpickiauthor=A.+K.+Mapp&title=Profiling+the+Dynamic+Interfaces+of+Fluorinated+Transcription+Complexes+for+Ligand+Discovery+and+Characterization&doi=10.1021%2Fcb3002733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling the Dynamic Interfaces of Fluorinated Transcription Complexes for Ligand Discovery and Characterization</span></div><div class="casAuthors">Pomerantz, William C.; Wang, Ningkun; Lipinski, Ashley K.; Wang, Rurun; Cierpicki, Tomasz; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1345-1350</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The conformationally dynamic binding surfaces of transcription complexes present a particular challenge for ligand discovery and characterization.  In the case of the KIX domain of the master coactivator CBP/p300, few small mols. have been reported that target its two allosterically regulated binding sites despite the important roles that KIX plays in processes ranging from memory formation to hematopoiesis.  Taking advantage of the enrichment of arom. amino acids at protein interfaces, here we show that the incorporation of six 19F-labeled arom. side chains within the KIX domain enables recapitulation of the differential binding footprints of three natural activator peptides (MLL, c-Myb, and pKID) in complex with KIX and effectively reports on allosteric changes upon binding using 1D NMR spectroscopy.  Addnl., the examn. of both the previously described KIX protein-protein interaction inhibitor Naphthol-ASE-phosphate and newly discovered ligand 1-10 rapidly revealed both the binding sites and the affinities of these small mols.  Significantly, the utility of using fluorinated transcription factors for ligand discovery was demonstrated through a fragment screen leading to a new low mol. wt. fragment ligand for CBP/p300, 1G7.  Arom. amino acids are enriched at protein-biomol. interfaces; therefore, this quant. and facile approach will be broadly useful for studying dynamic transcription complexes and screening campaigns complementing existing biophys. methods for studying these dynamic interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3u6Duitlu5LVg90H21EOLACvtfcHk0lg1nR6olh2kog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVWms70%253D&md5=5528eab7191432daab201cee2324436c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fcb3002733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3002733%26sid%3Dliteratum%253Aachs%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DLipinski%26aufirst%3DA.%2BK.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DCierpicki%26aufirst%3DT.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DProfiling%2520the%2520Dynamic%2520Interfaces%2520of%2520Fluorinated%2520Transcription%2520Complexes%2520for%2520Ligand%2520Discovery%2520and%2520Characterization%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D1345%26epage%3D1350%26doi%3D10.1021%2Fcb3002733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ember, S. W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C.</span><span> </span><span class="NLM_article-title">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2755</span><span class="NLM_x">–</span> <span class="NLM_lpage">2760</span><span class="refDoi"> DOI: 10.1021/cb5007344</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5007344" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2755-2760&author=N.+K.+Mishraauthor=A.+K.+Urickauthor=S.+W.+J.+Emberauthor=E.+Sch%C3%B6nbrunnauthor=W.+C.+Pomerantz&title=Fluorinated+Aromatic+Amino+Acids+Are+Sensitive+19F+NMR+Probes+for+Bromodomain-Ligand+Interactions&doi=10.1021%2Fcb5007344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorinated Aromatic Amino Acids Are Sensitive 19F NMR Probes for Bromodomain-Ligand Interactions</span></div><div class="casAuthors">Mishra, Neeraj K.; Urick, Andrew K.; Ember, Stuart W. J.; Schonbrunn, Ernst; Pomerantz, William C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2755-2760</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe a 19F NMR method for detecting bromodomain-ligand interactions using fluorine-labeled arom. amino acids due to the conservation of arom. residues in the bromodomain binding site.  The authors test the sensitivity, accuracy, and speed of this method with small mol. ligands (+)-JQ1, BI2536, Dinaciclib, TG101348, and acetaminophen using three bromodomains Brd4, BrdT, and BPTF.  Simplified 19F NMR spectra allowed for simultaneous testing of multiple bromodomains to assess selectivity and identification of a new BPTF ligand.  Fluorine labeling only modestly affected the Brd4 structure and function assessed by isothermal titrn. calorimetry, CD, and x-ray crystallog.  The speed, ease of interpretation, and low concn. of protein needed for binding expts. affords a new method to discover and characterize both native and new ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoejxq3KKVddrVg90H21EOLACvtfcHk0ljzUdsULgg2sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12gtLfP&md5=97f9edadc91cba8670f54bcca57fa03f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb5007344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5007344%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%26atitle%3DFluorinated%2520Aromatic%2520Amino%2520Acids%2520Are%2520Sensitive%252019F%2520NMR%2520Probes%2520for%2520Bromodomain-Ligand%2520Interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2755%26epage%3D2760%26doi%3D10.1021%2Fcb5007344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koleski, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Fragment Screening and Druggability Assessment for the CBP/P300 KIX Domain Through Protein-Observed 19F NMR Spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3735</span><span class="NLM_x">–</span> <span class="NLM_lpage">3739</span><span class="refDoi"> DOI: 10.1002/anie.201411658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fanie.201411658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVKgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=3735-3739&author=C.+T.+Geeauthor=E.+J.+Koleskiauthor=W.+C.+K.+Pomerantz&title=Fragment+Screening+and+Druggability+Assessment+for+the+CBP%2FP300+KIX+Domain+Through+Protein-Observed+19F+NMR+Spectroscopy&doi=10.1002%2Fanie.201411658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Screening and Druggability Assessment for the CBP/p300 KIX Domain through Protein-Observed 19F NMR Spectroscopy</span></div><div class="casAuthors">Gee, Clifford T.; Koleski, Edward J.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3735-3739</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">19F NMR spectroscopy of labeled proteins is a sensitive method for characterizing structure, conformational dynamics, higher-order assembly, and ligand binding.  Fluorination of arom. side chains has been suggested as a labeling strategy for small-mol. ligand discovery for protein-protein interaction interfaces.  Using a model transcription factor binding domain of the CREB binding protein (CBP)/p300, KIX, we report the first full small-mol. screen using protein-obsd. 19F NMR spectroscopy.  Screening of 508 compds. and validation by 1H-15N HSQC NMR spectroscopy led to the identification of a minimal pharmacaphore for the MLL-KIX interaction site.  Hit rate anal. for the CREB-KIX and MLL-KIX sites provided a metric to assess the ligandability or "druggability" of each interface informing future medicinal chem. efforts.  The structural information from the simplified spectra and data collection speed, affords a new screening tool for anal. of protein interfaces and discovery of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhRb2BYwsiILVg90H21EOLACvtfcHk0ljzUdsULgg2sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVKgsbw%253D&md5=3094371871c8ded8074112b98f145289</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201411658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201411658%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DKoleski%26aufirst%3DE.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DFragment%2520Screening%2520and%2520Druggability%2520Assessment%2520for%2520the%2520CBP%252FP300%2520KIX%2520Domain%2520Through%2520Protein-Observed%252019F%2520NMR%2520Spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D3735%26epage%3D3739%26doi%3D10.1002%2Fanie.201411658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Ge, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacRaild, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devine, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debono, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scammells, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scanlon, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anders, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, R. S.</span><span> </span><span class="NLM_article-title">Ligand-Induced Conformational Change of Plasmodium Falciparum AMA1 Detected Using 19F NMR</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6419</span><span class="NLM_x">–</span> <span class="NLM_lpage">6427</span><span class="refDoi"> DOI: 10.1021/jm500390g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500390g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6419-6427&author=X.+Geauthor=C.+A.+MacRaildauthor=S.+M.+Devineauthor=C.+O.+Debonoauthor=G.+Wangauthor=P.+J.+Scammellsauthor=M.+J.+Scanlonauthor=R.+F.+Andersauthor=M.+Foleyauthor=R.+S.+Norton&title=Ligand-Induced+Conformational+Change+of+Plasmodium+Falciparum+AMA1+Detected+Using+19F+NMR&doi=10.1021%2Fjm500390g"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm500390g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500390g%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DX.%26aulast%3DMacRaild%26aufirst%3DC.%2BA.%26aulast%3DDevine%26aufirst%3DS.%2BM.%26aulast%3DDebono%26aufirst%3DC.%2BO.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26aulast%3DAnders%26aufirst%3DR.%2BF.%26aulast%3DFoley%26aufirst%3DM.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DLigand-Induced%2520Conformational%2520Change%2520of%2520Plasmodium%2520Falciparum%2520AMA1%2520Detected%2520Using%252019F%2520NMR%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6419%26epage%3D6427%26doi%3D10.1021%2Fjm500390g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arntson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisniewski, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Protein-Observed 19F NMR for Fragment Screening, Affinity, Quantification, and Druggability Assessment</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1427</span><span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+19F+NMR+for+Fragment+Screening%2C+Affinity%2C+Quantification%2C+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%252019F%2520NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%252C%2520Quantification%252C%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Gee, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arntson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urick, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mishra, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawk, L. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wisniewski, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, W. C. K.</span><span> </span><span class="NLM_article-title">Protein-Observed (19)F-NMR for Fragment Screening, Affinity Quantification and Druggability Assessment</span> <span class="citation_source-journal">Nat. Protoc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1414</span><span class="NLM_x">–</span> <span class="NLM_lpage">1427</span><span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+%2819%29F-NMR+for+Fragment+Screening%2C+Affinity+Quantification+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%2520%252819%2529F-NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%2520Quantification%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Fielding, L.</span><span> </span><span class="NLM_article-title">NMR Methods for the Determination of Protein-Ligand Dissociation Constants</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">39</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span><span class="refDoi"> DOI: 10.2174/1568026033392705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.2174%2F1568026033392705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=12577990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD3sXis1KhsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=39-53&author=L.+Fielding&title=NMR+Methods+for+the+Determination+of+Protein-Ligand+Dissociation+Constants&doi=10.2174%2F1568026033392705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">NMR methods for the determination of protein-ligand dissociation constants</span></div><div class="casAuthors">Fielding, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Hilversum, Netherlands)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-53</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  This article is a review with 83 refs. of the application of NMR to the measurement of the dissocn. consts. of protein-ligand complexes.  After briefly discussing some general concepts of mol. stability, the text turns to consider which NMR parameters are reporters of complex formation.  The available data treatments required to translate obsd. NMR effects into quant. measurements of the stability of the complex in the form of the dissocn. const. (KD) are introduced.  Linearization methods and curve fitting methods are explained in detail and are illustrated with examples drawn from recent reports of protein-small mol. interactions.  Throughout the text examples of the commonly obsd. NMR parameters Δδ, 1/T1 and 1/T2 are drawn from biol. studies of 1H, 31P, 19F 15N (and other nuclei).  The advantages of NMR diffusion expts. as a measure of KD are considered.  Some less frequently used NMR approaches, some new ideas and some non-general methods are grouped together in a misc. section.  The major sources of errors in the detn. of KD are identified.  This allows recommendations for optimal exptl. set up.  Options for dealing with strong binding are reviewed.  Finally, the implications of abstracting KD data from high throughput screening expts. are considered and several different approaches to generate this data are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYJuFztEcewrVg90H21EOLACvtfcHk0lgWOEEkuv1xMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXis1KhsQ%253D%253D&md5=cd37fa1c247f9c14eb57e6f44971f7fe</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568026033392705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026033392705%26sid%3Dliteratum%253Aachs%26aulast%3DFielding%26aufirst%3DL.%26atitle%3DNMR%2520Methods%2520for%2520the%2520Determination%2520of%2520Protein-Ligand%2520Dissociation%2520Constants%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26spage%3D39%26epage%3D53%26doi%3D10.2174%2F1568026033392705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Tanifum, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kots, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, B.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbertson, S. R.</span><span> </span><span class="NLM_article-title">Novel Pyridopyrimidine Derivatives as Inhibitors of Stable Toxin a (STa) Induced cGMP Synthesis</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3067</span><span class="NLM_x">–</span> <span class="NLM_lpage">3071</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.bmcl.2009.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3067-3071&author=E.+A.+Tanifumauthor=A.+Y.+Kotsauthor=B.-K.+Choiauthor=F.+Muradauthor=S.+R.+Gilbertson&title=Novel+Pyridopyrimidine+Derivatives+as+Inhibitors+of+Stable+Toxin+a+%28STa%29+Induced+cGMP+Synthesis&doi=10.1016%2Fj.bmcl.2009.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DTanifum%26aufirst%3DE.%2BA.%26aulast%3DKots%26aufirst%3DA.%2BY.%26aulast%3DChoi%26aufirst%3DB.-K.%26aulast%3DMurad%26aufirst%3DF.%26aulast%3DGilbertson%26aufirst%3DS.%2BR.%26atitle%3DNovel%2520Pyridopyrimidine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Stable%2520Toxin%2520a%2520%2528STa%2529%2520Induced%2520cGMP%2520Synthesis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3067%26epage%3D3071%26doi%3D10.1016%2Fj.bmcl.2009.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Magedov, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frolova, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manpadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhoga, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evdokimov, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georgiou, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinnibrugh, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van slambrouck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steelant, W. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shuster, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Otterlo, W. A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornienko, A.</span><span> </span><span class="NLM_article-title">Anticancer Properties of an Important Drug Lead Podophyllotoxin Can Be Efficiently Mimicked by Diverse Heterocyclic Scaffolds Accessible via One-Step Synthesis</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4234</span><span class="NLM_x">–</span> <span class="NLM_lpage">4246</span><span class="refDoi"> DOI: 10.1021/jm200410r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200410r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4234-4246&author=I.+V.+Magedovauthor=L.+Frolovaauthor=M.+Manpadiauthor=U.+D.+Bhogaauthor=H.+Tangauthor=N.+M.+Evdokimovauthor=O.+Georgeauthor=K.+H.+Georgiouauthor=S.+Rennerauthor=M.+Getlikauthor=T.+L.+Kinnibrughauthor=M.+A.+Fernandesauthor=S.+Van+slambrouckauthor=W.+F.+A.+Steelantauthor=C.+B.+Shusterauthor=S.+Rogeljauthor=W.+A.+L.+van+Otterloauthor=A.+Kornienko&title=Anticancer+Properties+of+an+Important+Drug+Lead+Podophyllotoxin+Can+Be+Efficiently+Mimicked+by+Diverse+Heterocyclic+Scaffolds+Accessible+via+One-Step+Synthesis&doi=10.1021%2Fjm200410r"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm200410r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200410r%26sid%3Dliteratum%253Aachs%26aulast%3DMagedov%26aufirst%3DI.%2BV.%26aulast%3DFrolova%26aufirst%3DL.%26aulast%3DManpadi%26aufirst%3DM.%26aulast%3DBhoga%26aufirst%3DU.%2BD.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DEvdokimov%26aufirst%3DN.%2BM.%26aulast%3DGeorge%26aufirst%3DO.%26aulast%3DGeorgiou%26aufirst%3DK.%2BH.%26aulast%3DRenner%26aufirst%3DS.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DKinnibrugh%26aufirst%3DT.%2BL.%26aulast%3DFernandes%26aufirst%3DM.%2BA.%26aulast%3DVan%2Bslambrouck%26aufirst%3DS.%26aulast%3DSteelant%26aufirst%3DW.%2BF.%2BA.%26aulast%3DShuster%26aufirst%3DC.%2BB.%26aulast%3DRogelj%26aufirst%3DS.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DKornienko%26aufirst%3DA.%26atitle%3DAnticancer%2520Properties%2520of%2520an%2520Important%2520Drug%2520Lead%2520Podophyllotoxin%2520Can%2520Be%2520Efficiently%2520Mimicked%2520by%2520Diverse%2520Heterocyclic%2520Scaffolds%2520Accessible%2520via%2520One-Step%2520Synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4234%26epage%3D4246%26doi%3D10.1021%2Fjm200410r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Evdokimov, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van slambrouck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffeter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Calvé, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamoral-Theys, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooten, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uglinskii, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogelj, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiss, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steelant, W. F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bologa, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornienko, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magedov, I. V.</span><span> </span><span class="NLM_article-title">Structural Simplification of Bioactive Natural Products with Multicomponent Synthesis. 3. Fused Uracil-Containing Heterocycles as Novel Topoisomerase-Targeting Agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2012</span><span class="NLM_x">–</span> <span class="NLM_lpage">2021</span><span class="refDoi"> DOI: 10.1021/jm1009428</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1009428" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2012-2021&author=N.+M.+Evdokimovauthor=S.+Van+slambrouckauthor=P.+Heffeterauthor=L.+Tuauthor=B.+Le+Calv%C3%A9author=D.+Lamoral-Theysauthor=C.+J.+Hootenauthor=P.+Y.+Uglinskiiauthor=S.+Rogeljauthor=R.+Kissauthor=W.+F.+A.+Steelantauthor=W.+Bergerauthor=J.+J.+Yangauthor=C.+G.+Bologaauthor=A.+Kornienkoauthor=I.+V.+Magedov&title=Structural+Simplification+of+Bioactive+Natural+Products+with+Multicomponent+Synthesis.+3.+Fused+Uracil-Containing+Heterocycles+as+Novel+Topoisomerase-Targeting+Agents&doi=10.1021%2Fjm1009428"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm1009428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1009428%26sid%3Dliteratum%253Aachs%26aulast%3DEvdokimov%26aufirst%3DN.%2BM.%26aulast%3DVan%2Bslambrouck%26aufirst%3DS.%26aulast%3DHeffeter%26aufirst%3DP.%26aulast%3DTu%26aufirst%3DL.%26aulast%3DLe%2BCalv%25C3%25A9%26aufirst%3DB.%26aulast%3DLamoral-Theys%26aufirst%3DD.%26aulast%3DHooten%26aufirst%3DC.%2BJ.%26aulast%3DUglinskii%26aufirst%3DP.%2BY.%26aulast%3DRogelj%26aufirst%3DS.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DSteelant%26aufirst%3DW.%2BF.%2BA.%26aulast%3DBerger%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DJ.%2BJ.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKornienko%26aufirst%3DA.%26aulast%3DMagedov%26aufirst%3DI.%2BV.%26atitle%3DStructural%2520Simplification%2520of%2520Bioactive%2520Natural%2520Products%2520with%2520Multicomponent%2520Synthesis.%25203.%2520Fused%2520Uracil-Containing%2520Heterocycles%2520as%2520Novel%2520Topoisomerase-Targeting%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2012%26epage%3D2021%26doi%3D10.1021%2Fjm1009428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kamal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suresh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mallareddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raju, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamboli, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaik, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalivendi, S. V.</span><span> </span><span class="NLM_article-title">Synthesis of a New 4-Aza-2,3-Didehydropodophyllotoxin Analogues as Potent Cytotoxic and Antimitotic Agents</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2349</span><span class="NLM_x">–</span> <span class="NLM_lpage">2358</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2Fj.bmc.2011.02.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2349-2358&author=A.+Kamalauthor=P.+Sureshauthor=A.+Mallareddyauthor=B.+A.+Kumarauthor=P.+V.+Reddyauthor=P.+Rajuauthor=J.+R.+Tamboliauthor=T.+B.+Shaikauthor=N.+Jainauthor=S.+V.+Kalivendi&title=Synthesis+of+a+New+4-Aza-2%2C3-Didehydropodophyllotoxin+Analogues+as+Potent+Cytotoxic+and+Antimitotic+Agents&doi=10.1016%2Fj.bmc.2011.02.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DKamal%26aufirst%3DA.%26aulast%3DSuresh%26aufirst%3DP.%26aulast%3DMallareddy%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DB.%2BA.%26aulast%3DReddy%26aufirst%3DP.%2BV.%26aulast%3DRaju%26aufirst%3DP.%26aulast%3DTamboli%26aufirst%3DJ.%2BR.%26aulast%3DShaik%26aufirst%3DT.%2BB.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DKalivendi%26aufirst%3DS.%2BV.%26atitle%3DSynthesis%2520of%2520a%2520New%25204-Aza-2%252C3-Didehydropodophyllotoxin%2520Analogues%2520as%2520Potent%2520Cytotoxic%2520and%2520Antimitotic%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2349%26epage%3D2358%26doi%3D10.1016%2Fj.bmc.2011.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ioan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Micucci, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broccatelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhorov, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Budriesi, R.</span><span> </span><span class="NLM_article-title">1,4-Dihydropyridine Scaffold in Medicinal Chemistry, the Story So Far and Perspectives (Part 1): Action in Ion Channels and GPCRs</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4901</span><span class="NLM_x">–</span> <span class="NLM_lpage">4922</span><span class="refDoi"> DOI: 10.2174/092986711797535173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.2174%2F092986711797535173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=22050742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=4901-4922&author=P.+Ioanauthor=E.+Carosatiauthor=M.+Micucciauthor=G.+Crucianiauthor=F.+Broccatelliauthor=B.+S.+Zhorovauthor=A.+Chiariniauthor=R.+Budriesi&title=1%2C4-Dihydropyridine+Scaffold+in+Medicinal+Chemistry%2C+the+Story+So+Far+and+Perspectives+%28Part+1%29%3A+Action+in+Ion+Channels+and+GPCRs&doi=10.2174%2F092986711797535173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">1,4-Dihydropyridine scaffold in medicinal chemistry, the story so far and perspectives (Part 1): action in Ion channels and GPCRs</span></div><div class="casAuthors">Ioan, P.; Carosati, E.; Micucci, M.; Cruciani, G.; Broccatelli, F.; Zhorov, B. S.; Chiarini, A.; Budriesi, R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">4901-4922</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the pioneering studies of Fleckenstein and co-workers, L-Type Calcium Channel (LTCC) blockers have attracted large interest due to their effectiveness in treating several cardiovascular diseases.  Medicinal chemists achieved high potency and tissue selectivity by decorating the 1-4-DHP nucleus, the most studied scaffold among LTCC blockers.  Nowadays it is clear that the 1,4-DHP nucleus is a privileged scaffold since, when appropriately substituted, it can selectively modulate diverse receptors, channels and enzymes.  Therefore, the 1,4-DHP scaffold could be used to treat various diseases by a single-ligand multi-target approach.  In this review, we describe the structure-activity relationships of 1,4-DHPs at ion channels, G-protein coupled receptors, and outline the potential for future therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGwJSlDl7P57Vg90H21EOLACvtfcHk0ljTMEIHi_gh_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtL3K&md5=25335bf1582b042b4d2c4e58557bb896</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.2174%2F092986711797535173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535173%26sid%3Dliteratum%253Aachs%26aulast%3DIoan%26aufirst%3DP.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DMicucci%26aufirst%3DM.%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DZhorov%26aufirst%3DB.%2BS.%26aulast%3DChiarini%26aufirst%3DA.%26aulast%3DBudriesi%26aufirst%3DR.%26atitle%3D1%252C4-Dihydropyridine%2520Scaffold%2520in%2520Medicinal%2520Chemistry%252C%2520the%2520Story%2520So%2520Far%2520and%2520Perspectives%2520%2528Part%25201%2529%253A%2520Action%2520in%2520Ion%2520Channels%2520and%2520GPCRs%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D4901%26epage%3D4922%26doi%3D10.2174%2F092986711797535173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Hantzsch, A.</span><span> </span><span class="NLM_article-title">Condensationsprodukte Aus Aldehydammoniak Und Ketonartigen Verbindungen</span> <span class="citation_source-journal">Ber. Dtsch. Chem. Ges.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1881</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1637</span><span class="NLM_x">–</span> <span class="NLM_lpage">1638</span><span class="refDoi"> DOI: 10.1002/cber.18810140214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1002%2Fcber.18810140214" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1881&pages=1637-1638&author=A.+Hantzsch&title=Condensationsprodukte+Aus+Aldehydammoniak+Und+Ketonartigen+Verbindungen&doi=10.1002%2Fcber.18810140214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fcber.18810140214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.18810140214%26sid%3Dliteratum%253Aachs%26aulast%3DHantzsch%26aufirst%3DA.%26atitle%3DCondensationsprodukte%2520Aus%2520Aldehydammoniak%2520Und%2520Ketonartigen%2520Verbindungen%26jtitle%3DBer.%2520Dtsch.%2520Chem.%2520Ges.%26date%3D1881%26volume%3D14%26spage%3D1637%26epage%3D1638%26doi%3D10.1002%2Fcber.18810140214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petros, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Structure-Based Approaches Towards Identification of Fragments for the Low-Druggability ATAD2 Bromodomain</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1843</span><span class="NLM_x">–</span> <span class="NLM_lpage">1848</span><span class="refDoi"> DOI: 10.1039/C4MD00237G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1039%2FC4MD00237G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1843-1848&author=A.+Chaikuadauthor=A.+M.+Petrosauthor=O.+Fedorovauthor=J.+Xuauthor=S.+Knapp&title=Structure-Based+Approaches+Towards+Identification+of+Fragments+for+the+Low-Druggability+ATAD2+Bromodomain&doi=10.1039%2FC4MD00237G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2FC4MD00237G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00237G%26sid%3Dliteratum%253Aachs%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DStructure-Based%2520Approaches%2520Towards%2520Identification%2520of%2520Fragments%2520for%2520the%2520Low-Druggability%2520ATAD2%2520Bromodomain%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1843%26epage%3D1848%26doi%3D10.1039%2FC4MD00237G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Martin, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olesen, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Georg, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönbrunn, E.</span><span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2360</span><span class="NLM_x">–</span> <span class="NLM_lpage">2365</span><span class="refDoi"> DOI: 10.1021/cb4003283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Cyclin-Dependent+Kinase+Inhibitor+Dinaciclib+Interacts+with+the+Acetyl-Lysine+Recognition+Site+of+Bromodomains&doi=10.1021%2Fcb4003283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0ljTMEIHi_gh_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DCyclin-Dependent%2520Kinase%2520Inhibitor%2520Dinaciclib%2520Interacts%2520with%2520the%2520Acetyl-Lysine%2520Recognition%2520Site%2520of%2520Bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365%26doi%3D10.1021%2Fcb4003283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Huang, X.</span><span> </span><span class="NLM_article-title">Fluorescence Polarization Competition Assay: the Range of Resolvable Inhibitor Potency Is Limited by the Affinity of the Fluorescent Ligand</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1177/1087057102239666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1177%2F1087057102239666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=12854996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=34-38&author=X.+Huang&title=Fluorescence+Polarization+Competition+Assay%3A+the+Range+of+Resolvable+Inhibitor+Potency+Is+Limited+by+the+Affinity+of+the+Fluorescent+Ligand&doi=10.1177%2F1087057102239666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescence polarization competition assay: the range of resolvable inhibitor potency is limited by the affinity of the fluorescent ligand</span></div><div class="casAuthors">Huang, Xinyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">For the development of fluorescence polarization (FP) competition assays, there is a widespread belief that tight-binding fluorescent ligands should be avoided to identify inhibitors of low or intermediate potency in the screening of small-mol. compd. libraries.  It is demonstrated herein that this statement is a misconception; in fact, the higher the affinity of the fluorescent ligand, the wider the range of inhibitor potency that can be resolved.  An approx. est. for the low end of inhibitor Ki values that can be resolved is the Kd value of the fluorescent ligand.  Because FP competition assays are typically conducted under nonstoichiometric titrn. conditions, it is suggested that a fluorescent ligand of highest affinity that also has an adequate quantum yield to satisfy such conditions be selected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUYlJzQaNj7Vg90H21EOLACvtfcHk0lgdLlHcdfczXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlSqu78%253D&md5=7c706c96b6e534486f6780c93a0d441b</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1177%2F1087057102239666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057102239666%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DFluorescence%2520Polarization%2520Competition%2520Assay%253A%2520the%2520Range%2520of%2520Resolvable%2520Inhibitor%2520Potency%2520Is%2520Limited%2520by%2520the%2520Affinity%2520of%2520the%2520Fluorescent%2520Ligand%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D34%26epage%3D38%26doi%3D10.1177%2F1087057102239666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Mahony, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lasater, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">Dual Kinase-Bromodomain Inhibitors for Rationally Designed Polypharmacology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">305</span><span class="NLM_x">–</span> <span class="NLM_lpage">312</span><span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+Kinase-Bromodomain+Inhibitors+for+Rationally+Designed+Polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lgdLlHcdfczXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520Kinase-Bromodomain%2520Inhibitors%2520for%2520Rationally%2520Designed%2520Polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Tanaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seo, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Souza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paulk, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeAngelo, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Design and Characterization of Bivalent BET Inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1038/nchembio.2209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.2209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=27775715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslehurrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1089-1096&author=M.+Tanakaauthor=J.+M.+Robertsauthor=H.-S.+Seoauthor=A.+Souzaauthor=J.+Paulkauthor=T.+G.+Scottauthor=S.+L.+DeAngeloauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Design+and+Characterization+of+Bivalent+BET+Inhibitors&doi=10.1038%2Fnchembio.2209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of bivalent BET inhibitors</span></div><div class="casAuthors">Tanaka, Minoru; Roberts, Justin M.; Seo, Hyuk-Soo; Souza, Amanda; Paulk, Joshiawa; Scott, Thomas G.; DeAngelo, Stephen L.; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1089-1096</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cellular signaling is often propagated by multivalent interactions.  Multivalency creates avidity, allowing stable biophys. recognition.  Multivalency is an attractive strategy for achieving potent binding to protein targets, as the affinity of bivalent ligands is often greater than the sum of monovalent affinities.  The bromodomain and extraterminal domain (BET) family of transcriptional coactivators features tandem bromodomains through which BET proteins bind acetylated histones and transcription factors.  All reported antagonists of the BET protein BRD4 bind in a monovalent fashion.  Here the authors describe, to the knowledge for the first time, a bivalent BET bromodomain inhibitor-MT1-which has unprecedented potency.  Biophys. and biochem. studies suggest MT1 is an intramol. bivalent BRD4 binder that is more than 100-fold more potent, in cellular assays, than the corresponding monovalent antagonist, JQ1.  MT1 significantly (P < 0.05) delayed leukemia progression in mice, as compared to JQ1.  These data qualify a powerful chem. probe for BET bromodomains and a rationale for further development of multidomain inhibitors of epigenetic reader proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQwc6X0VYYbVg90H21EOLACvtfcHk0lgdLlHcdfczXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslehurrM&md5=7319203dac409d5dee3c580b49fcf185</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2209%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DDeAngelo%26aufirst%3DS.%2BL.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Bivalent%2520BET%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D1089%26epage%3D1096%26doi%3D10.1038%2Fnchembio.2209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Bondeson, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mares, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, I. E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miah, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulholland, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Routly, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buckley, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafson, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinn, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grandi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergamini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faelth-Savitski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanagan, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casillas, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Votta, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">den Besten, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famm, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harling, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span> </span><span class="NLM_article-title">Catalytic in Vivo Protein Knockdown by Small-Molecule PROTACs</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">611</span><span class="NLM_x">–</span> <span class="NLM_lpage">617</span><span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+Vivo+Protein+Knockdown+by+Small-Molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0ljxX2_X8f7mUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520Vivo%2520Protein%2520Knockdown%2520by%2520Small-Molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusinova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshua, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mujtaba, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohlmeyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.-M.</span><span> </span><span class="NLM_article-title">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9251</span><span class="NLM_x">–</span> <span class="NLM_lpage">9264</span><span class="refDoi"> DOI: 10.1021/jm401334s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401334s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9251-9264&author=G.+Zhangauthor=A.+N.+Plotnikovauthor=E.+Rusinovaauthor=T.+Shenauthor=K.+Morohashiauthor=J.+Joshuaauthor=L.+Zengauthor=S.+Mujtabaauthor=M.+Ohlmeyerauthor=M.-M.+Zhou&title=Structure-Guided+Design+of+Potent+Diazobenzene+Inhibitors+for+the+BET+Bromodomains&doi=10.1021%2Fjm401334s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Potent Diazobenzene Inhibitors for the BET Bromodomains</span></div><div class="casAuthors">Zhang, Guangtao; Plotnikov, Alexander N.; Rusinova, Elena; Shen, Tong; Morohashi, Keita; Joshua, Jennifer; Zeng, Lei; Mujtaba, Shiraz; Ohlmeyer, Michael; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9251-9264</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRD4, characterized by two acetyl-lysine binding bromodomains and an extra-terminal (ET) domain, is a key chromatin organizer that directs gene activation in chromatin through transcription factor recruitment, enhancer assembly, and pause release of the RNA polymerase II complex for transcription elongation.  BRD4 has been recently validated as a new epigenetic drug target for cancer and inflammation.  Our current knowledge of the functional differences of the two bromodomains of BRD4, however, is limited and is hindered by the lack of selective inhibitors.  Here, we report our structure-guided development of diazobenzene-based small-mol. inhibitors for the BRD4 bromodomains that have over 90% sequence identity at the acetyl-lysine binding site.  Our lead compd., MS436, through a set of water-mediated interactions, exhibits low nanomolar affinity (estd. Ki of 30-50 nM), with preference for the first bromodomain over the second.  We demonstrated that MS436 effectively inhibits BRD4 activity in NF-κB-directed prodn. of nitric oxide and proinflammatory cytokine interleukin-6 in murine macrophages.  MS436 represents a new class of bromodomain inhibitors and will facilitate further investigation of the biol. functions of the two bromodomains of BRD4 in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlT4IPx_3s7rVg90H21EOLACvtfcHk0ljxX2_X8f7mUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Ckt7jO&md5=24c7b50636fc4e0bbbfb80446e4dd23c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm401334s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401334s%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DJoshua%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure-Guided%2520Design%2520of%2520Potent%2520Diazobenzene%2520Inhibitors%2520for%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9251%26epage%3D9264%26doi%3D10.1021%2Fjm401334s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Baud, M. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin-Shiao, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardote, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pschibul, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zengerle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwan, T. T.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Chemical Biology. a Bump-and-Hole Approach to Engineer Controlled Selectivity of BET Bromodomain Chemical Probes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">346</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">641</span><span class="refDoi"> DOI: 10.1126/science.1249830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1126%2Fscience.1249830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=25323695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=346&publication_year=2014&pages=638-641&author=M.+G.+J.+Baudauthor=E.+Lin-Shiaoauthor=T.+Cardoteauthor=C.+Tallantauthor=A.+Pschibulauthor=K.-H.+Chanauthor=M.+Zengerleauthor=J.+R.+Garciaauthor=T.+T.-L.+Kwanauthor=F.+M.+Fergusonauthor=A.+Ciulli&title=Chemical+Biology.+a+Bump-and-Hole+Approach+to+Engineer+Controlled+Selectivity+of+BET+Bromodomain+Chemical+Probes&doi=10.1126%2Fscience.1249830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A bump-and-hole approach to engineer controlled selectivity of BET bromodomain chemical probes</span></div><div class="casAuthors">Baud, Matthias G. J.; Lin-Shiao, Enrique; Cardote, Teresa; Tallant, Cynthia; Pschibul, Annica; Chan, Kwok-Ho; Zengerle, Michael; Garcia, Jordi R.; Kwan, Terence T.-L.; Ferguson, Fleur M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">346</span>
        (<span class="NLM_cas:issue">6209</span>),
    <span class="NLM_cas:pages">638-641</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Small mols. are useful tools for probing the biol. function and therapeutic potential of individual proteins, but achieving selectivity is challenging when the target protein shares structural domains with other proteins.  The bromodomain and extra-terminal (BET) proteins have attracted interest because of their roles in transcriptional regulation, epigenetics, and cancer.  The BET bromodomains (protein interaction modules that bind acetyl-lysine) have been targeted by potent small-mol. inhibitors, but these inhibitors lack selectivity for individual family members.  We developed an Et deriv. of an existing small-mol. inhibitor, I-BET/JQ1, and showed that it binds leucine/alanine mutant bromodomains with nanomolar affinity and achieves up to 540-fold selectivity relative to wild-type bromodomains.  Cell culture studies showed that blockade of the first bromodomain alone is sufficient to displace a specific BET protein, Brd4, from chromatin.  Expansion of this approach could help identify the individual roles of single BET proteins in human physiol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9EnzLtvUKwbVg90H21EOLACvtfcHk0ljxX2_X8f7mUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWlt7nL&md5=04fe967e4681fb4b2b3168f4d5f8ff81</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1126%2Fscience.1249830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1249830%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLin-Shiao%26aufirst%3DE.%26aulast%3DCardote%26aufirst%3DT.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DPschibul%26aufirst%3DA.%26aulast%3DChan%26aufirst%3DK.-H.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DGarcia%26aufirst%3DJ.%2BR.%26aulast%3DKwan%26aufirst%3DT.%2BT.-L.%26aulast%3DFerguson%26aufirst%3DF.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DChemical%2520Biology.%2520a%2520Bump-and-Hole%2520Approach%2520to%2520Engineer%2520Controlled%2520Selectivity%2520of%2520BET%2520Bromodomain%2520Chemical%2520Probes%26jtitle%3DScience%26date%3D2014%26volume%3D346%26spage%3D638%26epage%3D641%26doi%3D10.1126%2Fscience.1249830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Raux, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voitovich, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derviaux, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebuffet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milhas, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roux, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trinquet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guillemot, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunel, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collette, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roche, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Combes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morelli, X.</span><span> </span><span class="NLM_article-title">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-Terminal Domain (BET) Proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1634</span><span class="NLM_x">–</span> <span class="NLM_lpage">1641</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1634-1641&author=B.+Rauxauthor=Y.+Voitovichauthor=C.+Derviauxauthor=A.+Lugariauthor=E.+Rebuffetauthor=S.+Milhasauthor=S.+Prietauthor=T.+Rouxauthor=E.+Trinquetauthor=J.-C.+Guillemotauthor=S.+Knappauthor=J.-M.+Brunelauthor=A.+Y.+Fedorovauthor=Y.+Colletteauthor=P.+Rocheauthor=S.+Betziauthor=S.+Combesauthor=X.+Morelli&title=Exploring+Selective+Inhibition+of+the+First+Bromodomain+of+the+Human+Bromodomain+and+Extra-Terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.5b01708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Selective Inhibition of the First Bromodomain of the Human Bromodomain and Extra-terminal Domain (BET) Proteins</span></div><div class="casAuthors">Raux, Brigitt; Voitovich, Yuliia; Derviaux, Carine; Lugari, Adrien; Rebuffet, Etienne; Milhas, Sabine; Priet, Stephane; Roux, Thomas; Trinquet, Eric; Guillemot, Jean-Claude; Knapp, Stefan; Brunel, Jean-Michel; Fedorov, Alexey Yu.; Collette, Yves; Roche, Philippe; Betzi, Stephane; Combes, Sebastien; Morelli, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1634-1641</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A midthroughput screening follow-up program targeting the first bromodomain of the human BRD4 protein, BRD4(BD1), identified an acetylated-mimic xanthine deriv. inhibitor.  This compd. binds with an affinity in the low micromolar range yet exerts suitable unexpected selectivity in vitro against the other members of the bromodomain and extra-terminal domain (BET) family.  A structure-based program pinpointed a role of the ZA loop, paving the way for the development of potent and selective BET-BRDi probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouQdXvF3r4zLVg90H21EOLACvtfcHk0lggeF0vWkLIeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKhsr0%253D&md5=fb9cfc20c76f65fcf409370b60aecc0a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01708%26sid%3Dliteratum%253Aachs%26aulast%3DRaux%26aufirst%3DB.%26aulast%3DVoitovich%26aufirst%3DY.%26aulast%3DDerviaux%26aufirst%3DC.%26aulast%3DLugari%26aufirst%3DA.%26aulast%3DRebuffet%26aufirst%3DE.%26aulast%3DMilhas%26aufirst%3DS.%26aulast%3DPriet%26aufirst%3DS.%26aulast%3DRoux%26aufirst%3DT.%26aulast%3DTrinquet%26aufirst%3DE.%26aulast%3DGuillemot%26aufirst%3DJ.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrunel%26aufirst%3DJ.-M.%26aulast%3DFedorov%26aufirst%3DA.%2BY.%26aulast%3DCollette%26aufirst%3DY.%26aulast%3DRoche%26aufirst%3DP.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DCombes%26aufirst%3DS.%26aulast%3DMorelli%26aufirst%3DX.%26atitle%3DExploring%2520Selective%2520Inhibition%2520of%2520the%2520First%2520Bromodomain%2520of%2520the%2520Human%2520Bromodomain%2520and%2520Extra-Terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1634%26epage%3D1641%26doi%3D10.1021%2Facs.jmedchem.5b01708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Elzein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalla, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gimbel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustig, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zablocki, J.</span><span> </span><span class="NLM_article-title">Discovery of a Novel A2B Adenosine Receptor Antagonist as a Clinical Candidate for Chronic Inflammatory Airway Diseases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2267</span><span class="NLM_x">–</span> <span class="NLM_lpage">2278</span><span class="refDoi"> DOI: 10.1021/jm7014815</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7014815" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2267-2278&author=E.+Elzeinauthor=R.+V.+Kallaauthor=X.+Liauthor=T.+Perryauthor=A.+Gimbelauthor=D.+Zengauthor=D.+Lustigauthor=K.+Leungauthor=J.+Zablocki&title=Discovery+of+a+Novel+A2B+Adenosine+Receptor+Antagonist+as+a+Clinical+Candidate+for+Chronic+Inflammatory+Airway+Diseases&doi=10.1021%2Fjm7014815"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm7014815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7014815%26sid%3Dliteratum%253Aachs%26aulast%3DElzein%26aufirst%3DE.%26aulast%3DKalla%26aufirst%3DR.%2BV.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPerry%26aufirst%3DT.%26aulast%3DGimbel%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLustig%26aufirst%3DD.%26aulast%3DLeung%26aufirst%3DK.%26aulast%3DZablocki%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520A2B%2520Adenosine%2520Receptor%2520Antagonist%2520as%2520a%2520Clinical%2520Candidate%2520for%2520Chronic%2520Inflammatory%2520Airway%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2267%26epage%3D2278%26doi%3D10.1021%2Fjm7014815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Devasthale, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fevig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrity, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cap, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klei, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kish, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weigelt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levesque, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moulin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirby, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamann, L. G.</span><span> </span><span class="NLM_article-title">Optimization of Activity, Selectivity, and Liability Profiles in 5-Oxopyrrolopyridine DPP4 Inhibitors Leading to Clinical Candidate (Sa)-2-(3-(Aminomethyl)-4-(2,4-Dichlorophenyl)-2-Methyl-5-Oxo-5H-Pyrrolo[3,4-B]Pyridin-6(7H)-Yl)-N,N-Dimethylacetamide (BMS-767778)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7343</span><span class="NLM_x">–</span> <span class="NLM_lpage">7357</span><span class="refDoi"> DOI: 10.1021/jm4008906</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4008906" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7343-7357&author=P.+Devasthaleauthor=Y.+Wangauthor=W.+Wangauthor=J.+Fevigauthor=J.+Fengauthor=A.+Wangauthor=T.+Harrityauthor=D.+Eganauthor=N.+Morganauthor=M.+Capauthor=A.+Furaauthor=H.+E.+Kleiauthor=K.+Kishauthor=C.+Weigeltauthor=L.+Sunauthor=P.+Levesqueauthor=F.+Moulinauthor=Y.-X.+Liauthor=R.+Zahlerauthor=M.+S.+Kirbyauthor=L.+G.+Hamann&title=Optimization+of+Activity%2C+Selectivity%2C+and+Liability+Profiles+in+5-Oxopyrrolopyridine+DPP4+Inhibitors+Leading+to+Clinical+Candidate+%28Sa%29-2-%283-%28Aminomethyl%29-4-%282%2C4-Dichlorophenyl%29-2-Methyl-5-Oxo-5H-Pyrrolo%5B3%2C4-B%5DPyridin-6%287H%29-Yl%29-N%2CN-Dimethylacetamide+%28BMS-767778%29&doi=10.1021%2Fjm4008906"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm4008906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4008906%26sid%3Dliteratum%253Aachs%26aulast%3DDevasthale%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DFevig%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DHarrity%26aufirst%3DT.%26aulast%3DEgan%26aufirst%3DD.%26aulast%3DMorgan%26aufirst%3DN.%26aulast%3DCap%26aufirst%3DM.%26aulast%3DFura%26aufirst%3DA.%26aulast%3DKlei%26aufirst%3DH.%2BE.%26aulast%3DKish%26aufirst%3DK.%26aulast%3DWeigelt%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLevesque%26aufirst%3DP.%26aulast%3DMoulin%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DY.-X.%26aulast%3DZahler%26aufirst%3DR.%26aulast%3DKirby%26aufirst%3DM.%2BS.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26atitle%3DOptimization%2520of%2520Activity%252C%2520Selectivity%252C%2520and%2520Liability%2520Profiles%2520in%25205-Oxopyrrolopyridine%2520DPP4%2520Inhibitors%2520Leading%2520to%2520Clinical%2520Candidate%2520%2528Sa%2529-2-%25283-%2528Aminomethyl%2529-4-%25282%252C4-Dichlorophenyl%2529-2-Methyl-5-Oxo-5H-Pyrrolo%255B3%252C4-B%255DPyridin-6%25287H%2529-Yl%2529-N%252CN-Dimethylacetamide%2520%2528BMS-767778%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7343%26epage%3D7357%26doi%3D10.1021%2Fjm4008906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Kenakin, T. P.</span> <span class="citation_source-book">Pharmacologic Analysis of Drug-Receptor Interaction</span>; <span class="NLM_publisher-name">Raven</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1993</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=T.+P.+Kenakin&title=Pharmacologic+Analysis+of+Drug-Receptor+Interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKenakin%26aufirst%3DT.%2BP.%26btitle%3DPharmacologic%2520Analysis%2520of%2520Drug-Receptor%2520Interaction%26pub%3DRaven%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC – a Free Database of Commercially Available Compounds for Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+a+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eZ3ycCEVKJYnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520a%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhwM7ia9oMEgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhwM7ia9oMEgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span> </span><span class="NLM_article-title">Drug-Like Properties and the Causes of Poor Solubility and Poor Permeability</span> <span class="citation_source-journal">J. Pharmacol. Toxicol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/S1056-8719(00)00107-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1016%2FS1056-8719%2800%2900107-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=11274893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A280%3ADC%252BD3M7os1OjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2000&pages=235-249&author=C.+A.+Lipinski&title=Drug-Like+Properties+and+the+Causes+of+Poor+Solubility+and+Poor+Permeability&doi=10.1016%2FS1056-8719%2800%2900107-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-like properties and the causes of poor solubility and poor permeability</span></div><div class="casAuthors">Lipinski C A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmacological and toxicological methods</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-49</span>
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    </div><div class="casAbstract">There are currently about 10000 drug-like compounds.  These are sparsely, rather than uniformly, distributed through chemistry space.  True diversity does not exist in experimental combinatorial chemistry screening libraries.  Absorption, distribution, metabolism, and excretion (ADME) and chemical reactivity-related toxicity is low, while biological receptor activity is higher dimensional in chemistry space, and this is partly explainable by evolutionary pressures on ADME to deal with endobiotics and exobiotics.  ADME is hard to predict for large data sets because current ADME experimental screens are multi-mechanisms, and predictions get worse as more data accumulates.  Currently, screening for biological receptor activity precedes or is concurrent with screening for properties related to "drugability." In the future, "drugability" screening may precede biological receptor activity screening.  The level of permeability or solubility needed for oral absorption is related to potency.  The relative importance of poor solubility and poor permeability towards the problem of poor oral absorption depends on the research approach used for lead generation.  A "rational drug design" approach as exemplified by Merck advanced clinical candidates leads to time-dependent higher molecular weight, higher H-bonding properties, unchanged lipophilicity, and, hence, poorer permeability.  A high throughput screening (HTS)-based approach as exemplified by unpublished data on Pfizer (Groton, CT) early candidates leads to higher molecular weight, unchanged H-bonding properties, higher lipophilicity, and, hence, poorer aqueous solubility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQmUcEou3dTa_EeosoOnpOLfW6udTcc2eZaT5PErcsl7bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7os1OjsQ%253D%253D&md5=aeb705afe1eccaceaf441c7a19f13ecc</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2FS1056-8719%2800%2900107-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1056-8719%252800%252900107-6%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26atitle%3DDrug-Like%2520Properties%2520and%2520the%2520Causes%2520of%2520Poor%2520Solubility%2520and%2520Poor%2520Permeability%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2000%26volume%3D44%26spage%3D235%26epage%3D249%26doi%3D10.1016%2FS1056-8719%2800%2900107-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">2615</span><span class="NLM_x">–</span> <span class="NLM_lpage">2623</span><span class="refDoi"> DOI: 10.1021/jm020017n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.-Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+Properties+That+Influence+the+Oral+Bioavailability+of+Drug+Candidates&doi=10.1021%2Fjm020017n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area ≤140 Å2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0liDfSXJpYidLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.-Y.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520Properties%2520That%2520Influence%2520the%2520Oral%2520Bioavailability%2520of%2520Drug%2520Candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623%26doi%3D10.1021%2Fjm020017n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liDfSXJpYidLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljLxsWHmCzJzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljLxsWHmCzJzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljLxsWHmCzJzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0liNcpWklwtlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Phenix.Model_vs_Data: a High-Level Tool for the Calculation of Crystallographic Model and Data Statistics</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">676</span><span class="refDoi"> DOI: 10.1107/S0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=10.1107%2FS0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Phenix.Model_vs_Data%3A+a+High-Level+Tool+for+the+Calculation+of+Crystallographic+Model+and+Data+Statistics&doi=10.1107%2FS0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0liNcpWklwtlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1107%2FS0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DPhenix.Model_vs_Data%253A%2520a%2520High-Level%2520Tool%2520for%2520the%2520Calculation%2520of%2520Crystallographic%2520Model%2520and%2520Data%2520Statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2FS0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FLP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FLP','PDB','4FLP'); return false;">PDB: 4FLP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4O7A','PDB','4O7A'); return false;">PDB: 4O7A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KDH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KDH','PDB','5KDH'); return false;">PDB: 5KDH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01336">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_65143"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01336">10.1021/acs.jmedchem.6b01336</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_001.csv">CSV</a>)</p></li><li><p class="inline">Additional data and methods (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_001.csv">jm6b01336_si_001.csv (3.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01336/suppl_file/jm6b01336_si_002.pdf">jm6b01336_si_002.pdf (8.24 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates of Brd4 bound to compound <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KDH">5KDH</a>) has been deposited with the Protein Data Bank. Authors will release atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-12%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01336%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01336" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667989f77968d19b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
