
<html lang="en"     class="pb-page"  data-request-id="f21e7e85-ff1b-4c41-9ef5-0cafc8e99c44"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b00794;website:website:acspubs;issue:issue:10.1021/jmcmar.2016.59.issue-17;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES)" /></meta><meta name="dc.Creator" content="Kuan-Wei  Huang" /></meta><meta name="dc.Creator" content="Kai-Cheng  Hsu" /></meta><meta name="dc.Creator" content="Lee-Ya  Chu" /></meta><meta name="dc.Creator" content="Jinn-Moon  Yang" /></meta><meta name="dc.Creator" content="Hanna S.  Yuan" /></meta><meta name="dc.Creator" content="Yu-Yuan  Hsiao" /></meta><meta name="dc.Description" content="The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their..." /></meta><meta name="Description" content="The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 29, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00794" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00794" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00794" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00794" /></link>
        
    
    

<title>Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00794" /></meta><meta property="og:title" content="Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0009.jpeg" /></meta><meta property="og:description" content="The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their critical roles in virus replication. Here using RNase T and CRN-4 as the model systems, we identify potential inhibitors for DEDDh exonucleases. We further show that two of the inhibitors, ATA and PV6R, indeed inhibit the exonuclease activity of the viral protein NP exonuclease of Lassa fever virus in vitro. Moreover, we determine the crystal structure of CRN-4 in complex with MES that reveals a unique inhibition mechanism by inducing the general base His179 to shift out of the active site. Our results not only provide the structural basis for the inhibition mechanism but also suggest potential lead inhibitors for the DEDDh exonucleases that may pave the way for designing nuclease inhibitors for biochemical and biomedical applications." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their critical roles in virus replication. Here using RNase T and CRN-4 as the model systems, we identify potential inhibitors for DEDDh exonucleases. We further show that two of the inhibitors, ATA and PV6R, indeed inhibit the exonuclease activity of the viral protein NP exonuclease of Lassa fever virus in vitro. Moreover, we determine the crystal structure of CRN-4 in complex with MES that reveals a unique inhibition mechanism by inducing the general base His179 to shift out of the active site. Our results not only provide the structural basis for the inhibition mechanism but also suggest potential lead inhibitors for the DEDDh exonucleases that may pave the way for designing nuclease inhibitors for biochemical and biomedical applications." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0009.jpeg" /></meta><meta name="twitter:title" content="Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES)" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00794"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00794">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00794&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00794&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00794&amp;href=/doi/10.1021/acs.jmedchem.6b00794" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 8019-8029</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00793" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00810" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kuan-Wei++Huang">Kuan-Wei Huang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kai-Cheng++Hsu">Kai-Cheng Hsu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lee-Ya++Chu">Lee-Ya Chu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jinn-Moon++Yang">Jinn-Moon Yang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hanna+S.++Yuan">Hanna S. Yuan</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu-Yuan++Hsiao">Yu-Yuan Hsiao</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Biological Science and Technology, National Chiao Tung University, Hsinchu 30068, Taiwan, ROC</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Institute of Molecular Biology, Academia Sinica, Taipei 11529, Taiwan, ROC</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Graduate Institute of Cancer Biology and Drug Discovery, College of Medical Science and Technology, Taipei Medical University, Taipei 11031, Taiwan</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Chemical Biology and Molecular Biophysics Program, Taiwan International Graduate Program, Academia Sinica, Nankang, Taipei 11529, Taiwan</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Institute of Bioinformatics and Structural Biology, National Tsing Hua University, 101 Kuang-Fu Road Section 2, Hsinchu 30013, Taiwan</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Institute of Bioinformatics and Systems Biology, National Chiao Tung University, Hsinchu, 30050, Taiwan</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">∇</span> <span class="aff-text">Center for Bioinformatics Research, National Chiao Tung University, Hsinchu 30068, Taiwan</span></div><div class="aff-info" id="aff8"><span class="aff-symbol">○</span> <span class="aff-text">Institute of Molecular Medicine and Bioengineering, National Chiao Tung University, Hsinchu 30068, Taiwan</span></div><div class="corresp-info"><strong>*</strong>H.S.Y.: e-mail, <a href="/cdn-cgi/l/email-protection#345c555a5a5574475d5a5d57551a5150411a4043"><span class="__cf_email__" data-cfemail="f199909f9f90b182989f989290df949584df8586">[email protected]</span></a>; phone, +886-2-27884151.</div><div class="corresp-info"><strong>*</strong>Y.-Y.H.: e-mail, <a href="/cdn-cgi/l/email-protection#660b0f0d0356505751260805121348030213481211"><span class="__cf_email__" data-cfemail="b7dadedcd287818680f7d9d4c3c299d2d3c299c3c0">[email protected]</span></a>; phone, +886-3-571-2121, extension 56999.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00794&amp;href=/doi/10.1021%2Facs.jmedchem.6b00794" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 8019–8029</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 16, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 May 2016</li><li><span class="item_label"><b>Published</b> online</span>29 August 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 September 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00794" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00794</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00794"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4277</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00794" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of Inhibitors for the DEDDh Family of Exonucleases and a Unique Inhibition Mechanism by Crystal Structure Analysis of CRN-4 Bound with 2-Morpholin-4-ylethanesulfonate (MES)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Kuan-Wei&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Kai-Cheng&quot;,&quot;last_name&quot;:&quot;Hsu&quot;},{&quot;first_name&quot;:&quot;Lee-Ya&quot;,&quot;last_name&quot;:&quot;Chu&quot;},{&quot;first_name&quot;:&quot;Jinn-Moon&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Hanna&quot;,&quot;last_name&quot;:&quot;S. Yuan&quot;},{&quot;first_name&quot;:&quot;Yu-Yuan&quot;,&quot;last_name&quot;:&quot;Hsiao&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;29&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;8019-8029&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00794&quot;},&quot;abstract&quot;:&quot;The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their critical roles in virus replication. Here using RNase T and CRN-4 as the model systems, we identify potential inhibitors for DEDDh exonucleases. We further show that two of the inhibitors, ATA and PV6R, indeed inhibit the exonuclease activity of the viral protein NP exonuclease of Lassa fever virus in vitro. Moreover, we determine the crystal structure of CRN-4 in complex with MES that reveals a unique inhibition mechanism by inducing the general base His179 to shift out of the active site. Our results not only provide the structural basis for the inhibition mechanism but also suggest potential lead inhibitors for the DEDDh exonucleases that may pave the way for designing nuclease inhibitors for biochemical and biomedical applications.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00794&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00794" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00794&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00794" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00794&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00794" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00794&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00794&amp;href=/doi/10.1021/acs.jmedchem.6b00794" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00794" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00794" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00794%26sid%3Dliteratum%253Aachs%26pmid%3D27529560%26genre%3Darticle%26aulast%3DHuang%26date%3D2016%26atitle%3DIdentification%2Bof%2BInhibitors%2Bfor%2Bthe%2BDEDDh%2BFamily%2Bof%2BExonucleases%2Band%2Ba%2BUnique%2BInhibition%2BMechanism%2Bby%2BCrystal%2BStructure%2BAnalysis%2Bof%2BCRN-4%2BBound%2Bwith%2B2-Morpholin-4-ylethanesulfonate%2B%2528MES%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D17%26spage%3D8019%26epage%3D8029%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/17" title="Go to Volume 59, Issue 17"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/jmcmar.2016.59.issue-17/20160908/jmcmar.2016.59.issue-17.largecover.jpg" alt="Go to Volume 59, Issue 17"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The DEDDh family of exonucleases plays essential roles in DNA and RNA metabolism in all kingdoms of life. Several viral and human DEDDh exonucleases can serve as antiviral drug targets due to their critical roles in virus replication. Here using RNase T and CRN-4 as the model systems, we identify potential inhibitors for DEDDh exonucleases. We further show that two of the inhibitors, ATA and PV6R, indeed inhibit the exonuclease activity of the viral protein NP exonuclease of Lassa fever virus in vitro. Moreover, we determine the crystal structure of CRN-4 in complex with MES that reveals a unique inhibition mechanism by inducing the general base His179 to shift out of the active site. Our results not only provide the structural basis for the inhibition mechanism but also suggest potential lead inhibitors for the DEDDh exonucleases that may pave the way for designing nuclease inhibitors for biochemical and biomedical applications.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14304" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14304" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The DEDDh family of exonucleases, also named DnaQ-like or RNase T family, constitutes more than 70 000 members widely distributed in all species from prokaryotes to eukaryotes (see RNase T family in the Pfam database). These DEDDh exonucleases bear 3′-to-5′ exonuclease activity, capable of digesting RNA and/or DNA by removing one nucleotide at a time from the 3′ end of a nucleic acid chain. They all have a DEDDh domain with a similar mixed αβ fold and four conserved acidic aspartate and glutamate residues (DEDD) for binding two magnesium ions in the active site with a nearby general base residue, histidine or tyrosine.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Although they all have a conserved fold and active site, DEDDh exonucleases have distinctive preferences in substrate binding and digestion. As examples, CRN-4 and NP exonuclease prefer to digest double-stranded DNA or RNA.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Exo I and ISG20 prefer to digest single-stranded DNA or RNA,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> while Snp, 3′hExo, and RNase T digest 3′ overhang of duplex nucleic acids.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a></div><div class="NLM_p">DEDDh exonucleases play essential roles in various cellular processes, including RNA maturation, RNA turnover, DNA replication, and DNA repair.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Dysfunction of human DEDDh exonucleases is deleterious and associated with diseases. Mutations in the exonuclease domain of WRN are linked to Werner syndrome,<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> whereas mutations in TREX1 are associated with the autoimmune diseases Aicardi-Goutieres syndrome and systemic lupus erythematosus.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> Moreover, several viral and human DEDDh exonucleases are essential for virus infection and gene replication, including Nsp14 of SARS coronavirus,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> NP exonuclease of Lassa fever virus,<a onclick="showRef(event, 'ref3 ref15'); return false;" href="javascript:void(0);" class="ref ref3 ref15">(3, 15)</a> and human TREX1 that is hijacked by HIV to digest viral nonproductive transcripts.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> NP exonuclease in Lassa fever virus specifically digests double-stranded RNA to prevent viral RNA detected by the host cytosolic RNA sensors, causing suppression of innate immune signaling in the infected cells.<a onclick="showRef(event, 'ref3 ref15'); return false;" href="javascript:void(0);" class="ref ref3 ref15">(3, 15)</a> Lassa fever virus causes severe hemorrhagic fever and infects 300 000 to 500 000 people per year in Western Africa, but there is no vaccine and limited therapeutic treatment.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> These viral or human exonucleases, including NP exonuclease, can serve as antiviral drug targets; however, no inhibitor for DEDDh exonucleases has ever been identified.</div><div class="NLM_p">All DEDDh exonucleases contain a conserved fold and active site but have divergent substrate-binding sites due to their different substrate preferences. These features are beneficial for designing general inhibitors targeted to the conserved active site and specific inhibitors targeted to different substrate-binding sites. In the first step toward a search for specific inhibitors, here we use RNase T and CRN-4 as model systems to screen for general inhibitors of DEDDh exonucleases. RNase T is a well-studied exonuclease digesting both RNA and DNA during RNA maturation and DNA repair in <i>E. coli</i>.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19-21)</a> CRN-4 is an apoptotic nuclease participating in chromosomal DNA fragmentation during apoptosis in <i>C. elegans</i>.<a onclick="showRef(event, 'ref4 ref22'); return false;" href="javascript:void(0);" class="ref ref4 ref22">(4, 22)</a> The crystal structures of both RNase T and CRN-4 are already reported, and their biochemical properties and catalytic mechanisms are well studied.</div><div class="NLM_p">We searched in the literature and found the following potential nuclease inhibitors: citric acid (citrate), 2-(<i>N</i>-morpholino)ethanesulfonic acid (MES), 4-[(4,6-dichloro-1,3,5-triazin-2-yl)amino]-2-(3-hydroxy-6-oxoxanthen-9-yl)benzoic acid (DR396), pontacyl violet 6R (PV6R), fmoc-<span class="smallcaps smallerCapital">d</span>-Cha-OH (FDCO), <i>p</i>-chloromercuriphenyl sulfonate (PCMPS), 5,5′-dithiobis(2-nitrobenzoic acid) (DTNB), 7-nitroindole-2-carboxylic acid (NCA), and aurintricarboxylic acid (ATA) (see the full name and chemical structure of each inhibitor in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Table S1 in Supporting Information</a>). Citrate is a metabolite of the Krebs cycle that binds polynucleotide phosphorylase (PNPase) and inhibits its exoribonuclease activity in <i>Escherichia coli</i>.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In addition, the nuclease activity of thermophilic bacteriophage GBSV1 (GBSV1-NSN) was reduced by 40% upon application of 10 mM citrate.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The sulfuric group in MES (2-(<i>N</i>-morpholino)ethanesulfonic acid) likely mimics phosphate groups in nucleic acids, as contaminant oligomers of vinylsulfonic acid (OVS) of a MES buffer were potent inhibitors of RNase A.<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25-27)</a> DR396, PV6R, and FDCO inhibit DNase γ in a dose-dependent manner,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> whereas the two phosphatase inhibitors (PCMPS and DTNB) have been used during nuclei isolation to prevent chromatin digestion by DNase I, DNase II, and micrococcal nuclease.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> DTNB has been used as a sulfhydryl modifying agent to react with cysteine residues in RNase T to inactivate its nuclease activity.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> NCA is a small molecule inhibitor of the apurinic/apyrimidinic (AP) endonuclease,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and preclinical and clinical data indicate that inhibitors of APE1 can block base excision repair, resulting in potentiation of radiotherapy for cancer cells.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> ATA is a general inhibitor of nucleases widely inhibiting of DNase I, RNase A, Sl nuclease, exonuclease III, and the restriction endonucleases Sal I, Bam HI, Pst I, and Sma I.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Besides nuclease inhibitor activity, ATA is also an inhibitor of nucleic acid processing enzymes,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> kinase,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and the JAK-STAT pathway.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Recently, ATA has been reported as a potent inhibitor of various viruses, such as influenza A and B,<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38, 39)</a> human immunodeficiency virus,<a onclick="showRef(event, 'ref35 ref40'); return false;" href="javascript:void(0);" class="ref ref35 ref40">(35, 40)</a> hepatitis C virus,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> SARS-CoV,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> and vaccinia virus.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> It is intriguing why ATA is such a broad-spectrum inhibitor. It is likely that ATA has many hydroxyl and keto groups that can form hydrogen bonds with the active-site residues<a onclick="showRef(event, 'ref39 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref44">(39, 44)</a> and that ATA may polymerize to block both nucleic acid and ATP binding on target proteins.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of Inhibitor Activities for the Inhibition of Exonuclease and Nucleic Acid Binding Activities for CRN-4 and RNase T</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">inhibition of exonuclease activity<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="2" align="center">inhibition of nucleic acid-binding activity<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitors<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">CRN-4</th><th class="colsep0 rowsep0" align="center">RNase T</th><th class="colsep0 rowsep0" align="center">CRN-4</th><th class="colsep0 rowsep0" align="center">RNase T</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">citrate</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MES</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PV6R</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PCMPS</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCA</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DR396</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DTNB</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ATA</td><td class="colsep0 rowsep0" align="left">++</td><td class="colsep0 rowsep0" align="left">+++</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FDCO</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Full names are the following: citrate, citrate acid; MES, 2-morpholin-4-ylethanesulfonate; PV6R, pontacyl violet 6R; PCMPS, <i>p</i>-chloromercuriphenyl sulfonate; NCA, 7-nitroindole-2-carboxylic acid; DR396, 4-[(4,6-dichloro-1,3,5-triazin-2-yl)amino]-2-(3-hydroxy-6-oxoxanthen-9-yl)benzoic acid; DTNB, 5,5′-dithiobis(2-nitrobenzoic acid); ATA, aurintricarboxylic acid; FDCO, fmoc-<span class="smallcaps smallerCapital">d</span>-Cha-OH.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Inhibitors in inhibitor-coupled nuclease activity assays are grouped into strong (+++), medium (++), weak (+), and noninhibitor (−).</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Inhibitors in inhibitor-coupled nucleic acid-binding assays are grouped into effective (+) and ineffective (−) compounds.</p></div></div></div><div class="NLM_p last">Here, using inhibitor-coupled nuclease activity and nucleic acid-binding assays, we found that PV6R, PCMPS, DTNB, and ATA can efficiently inhibit the nuclease activities of RNase T and CRN-4. As proof-of-concept, we further treated NP exonuclease from Lassa fever virus with the potential inhibitors and show that indeed some of these compounds can efficiently inhibit the viral protein in vitro. The crystal structure of CRN-4 in complex with the inhibitor MES revealed a unique inhibition mechanism that involved inducing the general base His179 to shift out of the active site upon inhibitor binding. Molecular docking analysis identified the key residues for inhibitor binding that will be helpful for the design of more effective and selective inhibitors in the future. Our biochemical, structural, and molecular docking studies suggest the general lead inhibitors for the DEDDh family of exonucleases and pave the way for designing nuclease inhibitors for biochemical and biomedical applications.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38832" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38832" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Screening Inhibitors for CRN-4 and RNase T</h3><div class="NLM_p">We searched the literature and found nine potential inhibitors that have been used to inhibit the activities of various nucleases (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Table S1</a>). We first set out to identify a good substrate for both CRN-4 and RNase T that can be used for inhibitor screening experiments. RNase T digests single-stranded 11-nucleotide DNA with a higher efficiency than RNA.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> We tested the nuclease activity of CRN-4 in digesting these single-stranded 11-nt DNA and RNA. We found that CRN-4 digested 5′-end <sup>32</sup>P-labeled single-stranded RNA with a high efficiency at concentrations greater than 0.25 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). In contrast, CRN-4 exhibited a low activity in digesting the 5′-end <sup>32</sup>P-labeled single-stranded DNA at concentrations up to 4 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). The difference between 0.25 μM and 4 μM is 16-fold, so these results suggest that CRN-4 is a more efficient RNase than DNase, as RNA digestion efficiency is more than 16 times that of DNA. Moreover, CRN-4 bound RNA at a low concentration of 12.5 μM, but it bound DNA weakly at a high concentration of 200 μM as revealed by the gel retardation assays (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). Taken together, these results show that CRN-4 binds RNA with a higher affinity and that it prefers to digest single-stranded RNA over DNA.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. CRN-4 prefers to digest and bind single-stranded RNA than DNA. (A) CRN-4 cleaved a single-stranded 11-nt RNA more efficiently than DNA. (B) CRN-4 prefers to bind single-stranded RNA than DNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To screen for effective inhibitors, we incubated CRN-4 (2 μM) and RNase T (0.1 μM) with each inhibitor for 10 min and then added the single-stranded 11-nt RNA or DNA for the digestion assays, respectively. After the RNA digestion reaction (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>), the exonuclease/RNA/inhibitor mixture was resolved by 20% denaturing polyacrylamide gels to determine if the RNA substrate had been digested (see <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). On the basis of the cleavage results, the inhibitors were classified into four groups: strong, medium, weak, and noninhibiting. Strong inhibitors can limit nuclease activity at inhibitor concentrations of 0.01–0.1 mM, whereas medium inhibitors are effective at 0.1–1 mM. Weak inhibitors can only moderately inhibit nuclease activity at 1 mM, whereas noninhibitors cannot interfere with nuclease activity at concentrations up to 1 mM. For CRN-4, both PV6R and PCMPS were strong inhibitors, DTNB and ATA were medium inhibitors, MES, DR396, and FDCO were weak inhibitors, and citrate and NCA were noninhibitors (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). For RNase T, both PCMPS and ATA were strong inhibitors, PV6R and DTNB were medium inhibitors, whereas citrate, MES, NCA, DR396, and FDCO were all weak inhibitors (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). The inhibitory effects of these agents on CRN-4 and RNase T were consistent in that PV6R, PCMPS, DTNB, and ATA were all strong-to-medium inhibitors, whereas citrate, MES, NCA, DR396, and FDCO were weak to noninhibiting (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). This result suggests that these inhibitors may share similar inhibition mechanisms or binding sites on CRN-4 and RNase T.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The exonuclease activities of CRN-4 and RNase T are inhibited by several compounds. (A) The exonuclease activity of CRN-4 was assayed by incubating with citrate, PV6R, PCMPS, NCA, DR396, DTNB, ATA, and FDCO in concentrations ranging from 0.01 to 1 mM. The concentration for MES was from 0.1 to 10 mM. (B) The exonuclease activity of RNase T was analyzed by incubating with the same compounds as in (A). The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt DNA or RNA was used in the inhibitor-coupled nuclease assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To understand the inhibition mechanism for these inhibitors, we further examined the RNA-binding activity of CRN-4 in the presence of the inhibitors by gel shift assays. In the absence of an inhibitor, CRN-4 bound the single-stranded 11-nt RNA at a concentration of 100 μM in a buffer with or without 10% dimethyl sulfoxide (DMSO) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). Except for FDCO (1 mM), all the other weak-to-noninhibitors (citrate (1 mM), MES (1 and 10 mM), NCA (1 mM), and DR396 (1 mM)) did not interfere with the interactions between CRN-4 and RNA. Among the medium-to-strong inhibitors, PV6R (1 mM) reduced the interactions between CRN-4 and RNA, whereas PCMPS (1 mM), DTNB (1 mM), and ATA (1 mM) altered the binding between CRN-4 and RNA, as the location of the bands of the CRN-4-RNA complex differed from that of the control group. On the basis of the results of these RNA-binding assays, we could classify these inhibitors into two groups: the effective inhibitors that interfere with RNA-binding activity, including PV6R, PCMPS, DTNB, ATA, and FDCO, and the ineffective compounds that do not interfere with the RNA-binding activity of CRN-4, including citrate, MES, NCA, and DR396 (see summary in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibitor-coupled RNA/DNA-binding assays of CRN-4 and RNase T. (A) The RNA-binding activity of CRN-4 was analyzed by electrophoresis mobility shift assays with or without the presence of different inhibitors. The concentrations of MES were 1 and 10 mM, and for all other inhibitors it was 1 mM. The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt RNA was used in the assays. (B) The DNA-binding activity of RNase T was analyzed by electrophoresis mobility shift assays with or without the presence of different inhibitors. The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt DNA was used in the assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For a comparison, we tested the DNA-binding activity of RNase T with or without each inhibitor using gel shift assays. We found that similar to the effect on CRN-4, PCMPS, DTNB, and ATA interfered with the DNA-binding activity of RNase T (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). The remaining inhibitors (citrate, MES, PC6R, NCA, DR396, and FDCO) did not interfere with the DNA-binding activity of RNase T (see summary in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Taken together, these results suggest that PCMPS, DTNB, and ATA are general inhibitors for the DEDDh family of exonucleases, since they inhibit the exonuclease activities and block the nucleic acid-binding activities for both CRN-4 and RNase T.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Inhibitors for Viral Protein NP Exonuclease</h3><div class="NLM_p">To demonstrate that these compounds could inhibit the exonuclease activity of viral proteins in the DEDDh family of exonucleases, we further cloned and purified the NP exonuclease from Lassa fever virus (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S1A</a>). We found that NP exonuclease digests both single-stranded and stem-loop RNA (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S1B</a>), and therefore a 20-nucleotide ssRNA and stem-loop RNA were used as substrates for inhibitor-coupled nuclease activity assays. We found that ATA was a strong inhibitor for NP exonuclease as the inhibitor concentration of 0.01 mM was sufficient for enzyme inhibition, whereas PV6R was a medium inhibitor as the concentration of at least 0.1 mM was required for effective enzyme inhibition (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). On the other hand, citrate, MES, PCMPS, NCA, DTNB, FDCO, and DR396 were weak inhibitors, as a concentration of at least 1 mM was required for effective enzyme inhibition. The inhibition efficiencies of these inhibitors are similar to those of CRN-4 and RNase T (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), confirming that the strong inhibitors, PV6R and ATA, are likely general inhibitors for the DEDDh family of exonucleases. Moreover, we also found that PV6R and ATA inhibited NP in a dose-dependent manner when we incubated NP with a stem-loop RNA with a 3′-end overhang (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C, <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D, and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S1C</a>). The IC<sub>50</sub> values for ATA, PV6R, and MES were estimated to be 6.8, 66.2, and 6693 μM at the NP concentration of 2 μM and the DNA concentration of 1 μM (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>E). Taken together, these results confirm that the strong-to-medium inhibitors for CRN-4 and RNase T can also inhibit the exonuclease activity of the Lassa fever NP exonuclease in vitro. ATA and PV6R are therefore potential lead compounds for inhibition of viral exonucleases of the DEDDh subfamily.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. NP exonuclease of Lassa fever virus is inhibited by several potential inhibitors. (A, B) The exonuclease activity of NP exonuclease was analyzed by incubating NP exonuclease with singled-stranded 20-nt RNA and stem loop RNA labeled with FAM at 5′ end in the presence of each inhibitor. DR396 absorbed UV light, so the cleavage pattern was different from those of other inhibitors. (C) IC<sub>50</sub> of ATA in inhibiting NP exonuclease was estimated by incubating NP exonuclease (2 μM) and stem-loop RNA (1 μM) with different concentrations of ATA, and the digested DNA was analyzed by 20% denaturing polyacrylamide gels. (D) Quantification results of IC<sub>50</sub> of three independent experiments. (E) Summary of inhibition activity and IC<sub>50</sub> for the potential inhibitors of NP exonuclease.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Crystal Structure of the CRN-4-MES Complex</h3><div class="NLM_p">To reveal the inhibition mechanisms of these inhibitors, we screened the conditions for cocrystallization of CRN-4 with inhibitors. We obtained co-crystals of the CRN-4-MES complex by mixing CRN-4 with DNA in 50 mM MES by the hanging-drop vapor diffusion method. The crystal structure of the CRN-4-MES complex was solved by molecular replacement using the apo-form CRN-4 structure (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CG7">3CG7</a>) as the searching model. The structure was refined to a resolution of 2.1 Å, and all of the diffraction and refinement statistics are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Table S3</a>. No DNA molecule was observed in the structure, but only a CRN-4 dimer bound with two MES molecules was crystallized in one asymmetric unit of the orthorhombic unit cell. CRN-4 folds into a dimeric structure, with the two protomers related to each other by a noncrystallographic 2-fold axis, similar to the structures of the apo-form of CRN-4<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). CRN-4 consists of an N-terminal DEDDh exonuclease domain and a C-terminal Zn domain (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). The electrostatic surface potential of the Zn domain is basic, suggesting that it is involved in nucleic acid binding (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). The nucleic acid substrate is likely bound at the Zn domain, and its 3′ end is extended and inserted into the active site in the DEDDh exonuclease domain.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of the CRN-4-MES complex. (A) Overall crystal structure of the dimeric CRN-4 in complex with MES. The DEDDh domain is displayed in light blue, whereas the Zn domain is displayed in pink. Two MES molecules are bound in the active sites of the CRN-4 dimer. (B) Electrostatic surface potential of one CRN-4 monomer. The MES molecule is bound deeply in the active site of CRN-4. (C) OMIT electron density map (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>, 1σ) and the interacting residues of the MES molecule. Hydrogen bonds are displayed as red dotted lines. (D) Active site of MES-bound CRN-4. One Mg<sup>2+</sup> ion is bound in the active site. The general base His179 shifts out of the active site upon MES binding there.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">MES in the CRN-4-MES complex was bound to Thr161 and Asp184 by hydrogen bonds in the active site of the DEDDh exonuclease domain (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). The electron density for the MES molecule in the active site of CRN-4 clearly revealed the shape of MES as a ring-like domain linked to a sulfuric group (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). Next to MES, only one Mg<sup>2+</sup> was bound in the active site in an octahedral geometry with six coordinated ligands: Asp15, Asp184, and four water molecules (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). Due to the presence of MES in the active site, the general base His179 in the CRN-4-MES complex had shifted out of the active site.</div><div class="NLM_p">The superimposition of the CRN-4-MES complex, Mn<sup>2+</sup>-bound apo-form of CRN-4 (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CM5">3CM5</a>), and DNA/Mg<sup>2+</sup>-bound RNase T (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9W">3V9W</a>) shows that their active sites are highly conserved, with the four acidic DEDD residues positioned at the same locations and orientations (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A). The MES molecule in the CRN-4-MES complex is superimposed on the general base His179 in the Mn<sup>2+</sup>-bound apo-CRN-4 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>B). As a result, the general base His179 in the CRN-4-MES complex is shifted more than 6 Å away from the active site. In summary, the MES-bound CRN-4 adopts an inactive conformation that is different from the active conformation observed in several DEDDh exonucleases with two Mg<sup>2+</sup> ions and a nearby general base in the active site.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MES inhibits the exonuclease activity of CRN-4 by inducing a conformational change in the active site but not by blocking substrate binding. (A) Superimposition of apo-form CRN-4/Mn<sup>2+</sup>, the CRN-4-MES complex, and the RNase T-DNA complex reveals a similar geometry of the active site with four acidic residues (DEDD) and one general base (H) residue. His179 in the CRN-4-MES complex is shifted away from the active site. (B) Superimposition of apo-CRN-4/Mn<sup>2+</sup> and the CRN-4-MES complex shows that MES occupies the position of His179 in apo-CRN-4. The general base His179 in the CRN-4-MES complex is consequently shifted away from the active site. (C) Structural model of CRN-4 bound with RNA. One monomer is displayed as a molecular surface, whereas the other is displayed as a ribbon model. The 3′ end of the stem-looped RNA is inserted into the active site. (D) Close look at the active site of the MES-bound CRN-4. MES does not overlap with the RNA in the CRN-4-RNA model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">MES does not interfere with the RNA-binding activity of CRN-4 as shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A. To understand the inhibition mechanism of MES, we generated a model of the CRN-4-RNA complex using the structure of the 3′hExo-RNA complex (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L8R">4L8R</a>) as the template. The stem-loop RNA substrate is bound on the Zn domain and fitted well onto the CRN-4 surface. The 3′ end of the stem-loop RNA is inserted into the active site of CRN-4 (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>C). The MES molecule is located in the active site, but it does not interfere with the binding of the substrate, as the scissile phosphate is bound next to MES (see <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D). This result is consistent with the RNA-binding assays, suggesting that MES is a noncompetitive inhibitor of CRN-4.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Key Inhibitor-Binding Residues in CRN-4</h3><div class="NLM_p">To further decipher the possible inhibitor-binding sites, we used the docking program iGEMDOCK<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> to fit six of the weak-to-strong exonuclease activity inhibitors (listed in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), except for MES, into CRN-4. Two noninhibitors, citrate and NCA, were also used in docking analysis to serve as the negative controls. The six inhibitors and two noninhibitors were all bound in the active site pocket in the docking models, probably because the active site is located in the most concave surface of CRN-4. On the basis of the interacting residues in the docking models, the six inhibitors were classified into three groups (groups 2–4), while the noninhibitors were classified as group 5 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). The six inhibitors all overlapped with RNA substrates in the CRN-4-RNA model, suggesting that they were bound at the RNA-binding sites in the active site of CRN-4 (see <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B–D and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S2</a>). In comparison to the crystal structure of the CRN-4-MES complex, MES was bound at the active site but did not overlap with RNA (see <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>D).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Important amino acids for inhibitor binding in CRN-4 are revealed by molecular docking. (A) Eight compounds, except MES, are docked into CRN-4 using iGEMDOCK,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and they are divided into four groups (groups 2–5) based on the interactions with CRN-4 in the docking models. The interactions of MES in CRN-4 are derived from the crystal structure of CRN-4-MES complex, and it is classified into a single group (group 1). Interactions listed on the top row marked by E indicate electrostatic interactions, by H indicate hydrogen bonding, by C indicate covalent bond, and by V indicate van der Waals interactions. A cell is colored green for a positive interaction; otherwise the cell is colored black. The symbols +++, ++, + , and – represent strong, medium, weak, and no inhibition, respectively, based on the inhibitor-coupled nuclease activity assay from <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A. (B–E) Docking models for ATA, DR396, PCMPS, and citrate in the active site of CRN-4. Scissile phosphates are displayed as orange balls marked by P. Hydrogen and covalent bonds are displayed as red and green dotted lines, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The interaction profile shows that Glu17 and Phe71 were the two key residues that most frequently formed interactions with inhibitors in CRN-4 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). The inhibitors interacted with the side chains of these two residues through their ring functional groups, such as the morpholine group in MES in the crystal structure and the naphthalene group in PV6R in the docking model (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S2</a>). Furthermore, all inhibitors consistently formed electrostatic interactions with the catalytic Mg<sup>2+</sup>. In contrast, citrate and NCA did not interact with these two residues (Glu17 and Phe71) or Mg<sup>2+</sup>, suggesting that these two compounds could not make stable interactions with CRN-4. This docking study thereby may explain why the two compounds, citrate and NCA, are noninhibitors for CRN-4 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S2</a>).</div><div class="NLM_p last">The inhibitors in groups 2 and 3 had similar binding modes and interacted with similar residues, such as Asp15, Glu17, Thr18, and Phe71. The main difference between the inhibitors in groups 2 and 3 was that group 2 inhibitors presented additional interactions with Arg118 and Gln25 (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A–C). These additional interactions may account for the enhanced inhibition activity of the group 2 inhibitors (ATA, DTNB, and PV6R) compared to the group 3 inhibitors (DR396 and FDCO). The docking study also revealed that PCMPS (group 4) could be anchored through a covalent linkage to the sulfur atom of Cys68, and so PCMPS inhibited CRN-4 with a higher potency than other compounds (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>D). Taken together, these docking studies suggest that inhibitors that could interact with Glu17, Gln25, Phe71, Asp111, Asp115, Arg118 and the metal ion would be potent inhibitors of the exonuclease activity of CRN-4. To verify if these key residues are conserved in viral DEDDh exonucleases, we superposed two structures, apo-CRN-4 and NP exonuclease. Even though there is no significant sequence similarity between these two proteins, their overall structure and orientation of the five active-site residues are comparable (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figures S4A and S4B</a>). The metal ions in these two structures, Mn<sup>2+</sup> and Mg<sup>2+</sup>, are located at a similar position (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S4B</a>). Key residues for inhibitor binding in CRN-4, including Glu17, Asp111, Asp115, and Arg118, are also superimposed well with those in NP exonuclease, Glu388, Gln459, Asp463, and Lys466 (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S4C</a>). These structural features suggest that these inhibitor binding residues are conserved in viral DEDDh exonucleases.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we identified seven compounds that can inhibit the activity of CRN-4 and RNase T from the DEDDh family of exonucleases, with weak (∼mM) to strong effects (∼10 μM). The four inhibitors PCMPS, PV6R, ATA, and DTNB are the most potent in inhibiting the exonuclease activity of both CRN-4 and RNase T (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). As proof-of-concept for showing these inhibitors may serve as leading compounds to inhibit the viral proteins in the DEDDh family of exonucleases, we further tested if these compounds could inhibit NP exonuclease from Lassa fever virus. Indeed, ATA and PV6R efficiently inhibit the exonuclease activity of NP exonuclease in vitro with an IC<sub>50</sub> value in the single or double-digit μM range. Our study thus identifies, for the first time, a group of effective inhibitors for the DEDDh family of exonucleases that have potential to be developed into antiviral agents.</div><div class="NLM_p">We also reveal a unique inhibition mechanism for DEDDh exonucleases through crystal structure determination of the CRN-4-MES complex. In our biochemical assays, we showed that MES is a weak inhibitor and does not interfere with the RNA-binding activity of CRN-4. The crystal structure of this association reveals that MES indeed does not interfere with substrate binding, but it does occupy the position of the general base residue. As a result, the general base, His179, which is located on a flexible loop, is shifted away from the active site upon MES binding, thereby inhibiting the activity of CRN-4. Interestingly, previous structural studies of RNase T<a onclick="showRef(event, 'ref7 ref47'); return false;" href="javascript:void(0);" class="ref ref7 ref47">(7, 47)</a> and TREX1<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> also revealed a similar mechanism whereby the general base located on a flexible loop is shifted from the active site when unfavored substrates are bound to the enzyme, which consequently inhibits exonuclease activity (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figure S3</a>). We note that the general base His residue is always located on a loop region in the DEDDh family of exonucleases (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Table S2</a>). Therefore, it seems feasible to trap the exonuclease in an inactive conformation by means of a small molecule, such as MES, to inhibit the nuclease activity of the DEDDh family of exonucleases. Nevertheless, the interactions between MES and CRN-4 are only maintained by a few hydrogen bonds and van der Waals interactions (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C), suggesting that MES binds weakly to CRN-4. As MES does not bind at the same location as the other four potent inhibitors, MES could be modified by adding small functional groups to increase its interactions and improve its specificity toward a specific DEDDh exonuclease.</div><div class="NLM_p last">Many enzyme inhibitors have been used in antiviral therapy, including inhibitors for neuraminidases, proteases, polymerases, integrases, and reverse transcriptases.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49, 50)</a> Neuraminidase inhibitors are usually the first choice for influenza prevention and treatment, although many cases of viral resistance to neuraminidase inhibitors have been reported.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> Similarly, a significant degree of drug resistance through mutations in reverse transcriptases, integrases, and proteases has been reported for HIV.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53-55)</a> Thus, designing new types of antiviral drugs or combining new drugs with existing ones in combination therapies is an emerging trend of drug development.<a onclick="showRef(event, 'ref50 ref56'); return false;" href="javascript:void(0);" class="ref ref50 ref56">(50, 56)</a> The design of inhibitors for exonucleases and endonucleases is a new area for exploration. In the identification of lead inhibitors for the DEDDh family of exonucleases, our study is the first step in discovering new targets and new inhibitors for biochemical and biomedical applications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45470" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45470" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Protein Expression and Purification of CRN-4, RNase T, and NP exonuclease</h3><div class="NLM_p last">The plasmid pGEX4T-2-CRN-4<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> was transformed into the <i>E. coli</i> BL21(DE3)pLysS strain and cultured in LB medium supplemented with 50 μg/mL ampicillin to an OD<sub>600</sub> of 0.5–0.6. Expression of GST-tagged CRN-4 was induced by 1 mM IPTG at 18 °C for 20 h. The harvested cells were disrupted by a microfluidizer in lysis buffer (1 mM KH<sub>2</sub>PO<sub>4</sub>, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 20 mM NaCl, 2.7 mM KCl, pH 7.4). The crude cell extract was passed through a glutathione-Sepharose column (GE Healthcare) in a standard protocol, and the collected protein fractions were digested by thrombin at room temperature for 16 h to remove the GST tag. The protein sample was further purified by a HiTrap heparin column (GE Healthcare) and a gel filtration column (Superdex 75, GE Healthcare). Purified CRN-4 samples were concentrated to 10–15 mg/mL in 50 mM Tris-HCl, pH 7.5, 300 mM NaCl. His-tagged RNase T was purified as described<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and concentrated to 20–30 mg/mL in 50 mM Tris-HCl, pH 7.5, 300 mM NaCl. The exonuclease domain of nucleoprotein protein (NP exonuclease) from Lassa virus was cloned into expression vectors pET-28c (Novagen, USA) to generate the N-terminal His-tagged fused protein (342–569 aa). The plasmid was transformed into <i>E. coli</i> Tuner (DE3) strain cultured in LB medium supplemented with 50 μg/mL kanamycin. Cells were grown to an OD<sub>600</sub> of 0.5–0.6 and induced by 1 mM IPTG at 18 °C for 18–22 h. The harvested cells were disrupted by a microfluidizer and passed through Ni-NTA resin affinity column (Qiagen Inc., USA) using a standard protocol. The protein sample was further purified by HiTrap heparin column (GE Healthcare, USA) and gel filtration chromatography column (Superdex 200, GE Healthcare, USA). Purified NP exonuclease samples were concentrated to 24 mg/mL in 300 mM NaCl and 50 mM Tris-HCl (pH 7.0). The purity of CRN-4, RNase T, and NP exonuclease was higher than 95% by SDS–PAGE analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Figures S1 and S5</a>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Nuclease Activity Assays</h3><div class="NLM_p last">The single-stranded 11-nt DNA (5′-AACCTTACAAA-3′) or RNA (5′-AACCUUACAAA-3′) were labeled respectively at the 5′ end with [γ-<sup>32</sup>P]ATP by T4 polynucleotide kinase and then purified by a Microspin G-25 column (GE Healthcare) to remove the nonincorporated nucleotides. Inhibitors were mixed with CRN-4 or RNase T at room temperature for 10 min. The full names and chemical structures of each inhibitor are given in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">Table S1</a>. Citrate, MES, PV6R, and PCMPS were dissolved in water. NCA, DR396, DTNB, ATA, and FDCO were dissolved in 100% DMSO. The 5′-end labeled 11-nt ssDNA/ssRNA (20 nM) was then incubated with the exonuclease/inhibitor mixture in 120 mM NaCl, 2 mM MgCl<sub>2</sub>, and 20 mM Tris-HCl pH 7.0, with or without the presence of 10% DMSO, at room temperature for 30–60 min. The reaction times for CRN-4, RNase T, and NP exonuclease are 30, 40, and 60 min, respectively. The reaction was stopped by the addition of DNA loading dye at 95 °C for 5 min. The DNA/RNA digestion patterns were resolved on 20% denaturing polyacrylamide gels and visualized by autoradiography (Fujifilm, FLA-5000). The 5′-end FAM labeled 20-nt ssRNA (5′-ACUGGACAAAUACUCCGAGG-3′) and a stem-loop RNA with a 4-nt 3′ overhang (5′-GGCCCTCTTTAGGGCCTTGG-3′) were used as substrates for NP exonuclease activity assays. These substrates (1 μM) were incubated with the NP or NP/inhibitor in 120 mM NaCl, 2 mM MgCl<sub>2</sub>, and 20 mM Tris-HCl pH 7.0, with or without the presence of 10% DMSO, at room temperature for 60 min. The samples were resolved on 20% denaturing polyacrylamide gels and visualized by ultraviolet light. Software programs, ImageJ and GraphPad Prism 6, were used in quantification of cleavage patterns and calculation of the half maximal inhibitory concentration (IC<sub>50</sub>).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> DNA/RNA-Binding Assays</h3><div class="NLM_p last">The single-stranded 11-nt DNA or RNA substrates were labeled as described in the previous section. Inhibitors in different concentrations as shown in <a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> were mixed with CRN-4 or RNase T at room temperature for 10 min. The 5′-end labeled 11-nt DNA/RNA substrates (20 nM) were then incubated respectively with different exonuclease/inhibitor mixtures in a buffer containing 50 mM Tris-HCl, pH 7.0, 120 mM NaCl, 10 mM EGTA, and 40 mM EDTA, with or without 10% DMSO, at room temperature for 20 min. After incubation, the reaction mixture was applied to 20% TBE gels, which were exposed to a phosphorimaging plate (Fujifilm) for autoradiographic visualization.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystallization, Data Collection, and Structure Determination of the CRN-4-MES Complex</h3><div class="NLM_p">CRN-4-MES complex crystals were grown by the hanging-drop vapor diffusion method at 4 °C. Purified CRN-4 samples were mixed with single-stranded DNA (5′-GCTTAC-3′) in the molar ratio of 1:1.2. The crystallization drop was made by mixing 1 μL of protein–DNA mixed solution and 1 μL of reservoir solution (50 mM MES monohydrate, pH 6.0, 20 mM magnesium chloride hexahydrate, 15% v/v 2-propanol). Crystals appeared after 3–5 months. The crystal was cryoprotected by a soaking buffer (50 mM MES monohydrate, pH 6.0, 20 mM magnesium chloride hexahydrate, 15% v/v 2-propanol, 30% glycerol) for data collection at 100 K. The single-wavelength diffraction data of the CRN-4-MES crystal were collected via beamline 13B1 at NSRRC, Taiwan.</div><div class="NLM_p last">The diffraction data were processed by HKL2000. The structure was solved by a molecular replacement method using the crystal structure of the apo-form CRN-4 (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CG7">3CG7</a>) as the search model by MOLREP in CCP4 suite.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Interestingly, no DNA was found in the crystal structure, but only two CRN-4-MES complexes were identified in one asymmetric unit. The models were built by Coot<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and refined by Phenix.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Structural coordinates and diffraction structure factors were deposited in the RCSB Protein Data Bank with the PDB code of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DK5">5DK5</a> for CRN-4-MES complex.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Molecular Docking</h3><div class="NLM_p last">The Mn-bound (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CM5">3CM5</a>) and MES-bound structures of CRN-4 were used for the molecular docking. To determine the substrate-binding site, a structure of a homologous exonuclease (3′hExo, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L8R">4L8R</a>) in complex with a stem-loop RNA was aligned with CRN-4 using a structural alignment tool.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> The inhibitor-binding site was defined by residues ≤10 Å from the inhibitor. The structures of each inhibitor were generated using the CORINA software.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> These inhibitor structures were docked into CRN-4 using the in-house docking tool, iGEMDOCK.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> iGEMDOCK is an extension of GEMDOCK<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and provides a graphical interface to visualize protein–inhibitor interactions. Since the chlorine atom could be dissociated from PCMPS, which formed a covalent linkage with the cysteine,<a onclick="showRef(event, 'ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65">(63-65)</a> PCMPS was in the dissociated form during docking and the scoring function of GEMDOCK was modified to include the possible covalent linkage. Finally, iGEMDOCK was used to generate interactions between inhibitors and amino acid residues, including electrostatic, hydrogen-bonding, and van der Waals interactions.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00794">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00794" class="ext-link">10.1021/acs.jmedchem.6b00794</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables S1–S3 listing candidate inhibitors, general base residues, and crystal data; Figures S1–S5 showing IC<sub>50</sub>, docking models, general base residues, structural comparisons, and SDS–PAGE analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf">jm6b00794_si_001.pdf (1.26 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB entry code for CRN-4-MES complex is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DK5">5DK5</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b00794" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17485" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17485" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hanna S. Yuan</span> - <span class="hlFld-Affiliation affiliation">Institute of Molecular
Biology, Academia Sinica, Taipei 11529, Taiwan, ROC</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a3cbc2cdcdc2e3d0cacdcac0c28dc6c7d68dd7d4"><span class="__cf_email__" data-cfemail="630b020d0d0223100a0d0a00024d0607164d1714">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu-Yuan Hsiao</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Science and Technology, National Chiao Tung University, Hsinchu 30068, Taiwan, ROC</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Molecular Medicine and Bioengineering, National Chiao Tung University, Hsinchu 30068, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#87eaeeece2b7b1b6b0c7e9e4f3f2a9e2e3f2a9f3f0"><span class="__cf_email__" data-cfemail="75181c1e1045434442351b1601005b1011005b0102">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kuan-Wei Huang</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Science and Technology, National Chiao Tung University, Hsinchu 30068, Taiwan, ROC</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Molecular
Biology, Academia Sinica, Taipei 11529, Taiwan, ROC</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kai-Cheng Hsu</span> - <span class="hlFld-Affiliation affiliation">Graduate Institute of Cancer
Biology and Drug Discovery, College of Medical Science and Technology, Taipei Medical University, Taipei 11031, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lee-Ya Chu</span> - <span class="hlFld-Affiliation affiliation">Institute of Molecular
Biology, Academia Sinica, Taipei 11529, Taiwan, ROC</span>; 
    <span class="hlFld-Affiliation affiliation">Chemical Biology and Molecular Biophysics Program, Taiwan
International Graduate Program, Academia
Sinica, Nankang, Taipei 11529, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Bioinformatics and Structural Biology, National Tsing Hua University, 101 Kuang-Fu Road Section 2, Hsinchu 30013, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinn-Moon Yang</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Science and Technology, National Chiao Tung University, Hsinchu 30068, Taiwan, ROC</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Bioinformatics Research, National Chiao
Tung University, Hsinchu 30068, Taiwan</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare no competing financial interest.<br /></br></p></li></ul></div><div class="ack" id="ACK-d84e1206-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Portions of this research were carried out at the National Synchrotron Radiation Research Center. The Synchrotron Radiation Protein Crystallography Facility is supported by the National Core Facility Program for Biotechnology. Additional assistance was provided by the Institute of Molecular Biology English Editing Core. Funding is from Academia Sinica and Ministry of Science and Technology, Taiwan, ROC, to Y.-Y.H. (Grant MOST103-2311-B-009-001-MY3) and H.S.Y. Funding for open access charge is from Academia Sinica, Taiwan.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i18" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i18"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> Abbreviations Used</h2><tr><td class="NLM_term">MES</td><td class="NLM_def"><p class="first last">2-morpholin-4-ylethanesulfonate</p></td></tr><tr><td class="NLM_term">citrate</td><td class="NLM_def"><p class="first last">citrate acid</p></td></tr><tr><td class="NLM_term">PV6R</td><td class="NLM_def"><p class="first last">pontacyl violet 6R</p></td></tr><tr><td class="NLM_term">PCMPS</td><td class="NLM_def"><p class="first last"><i>p</i>-chloromercuriphenyl sulfonate</p></td></tr><tr><td class="NLM_term">NCA</td><td class="NLM_def"><p class="first last">7-nitroindole-2-carboxylic acid</p></td></tr><tr><td class="NLM_term">DR396</td><td class="NLM_def"><p class="first last">4-[(4,6-dichloro-1,3,5-triazin-2-yl)amino]-2-(3-hydroxy-6-oxoxanthen-9-yl)benzoic acid</p></td></tr><tr><td class="NLM_term">DTNB</td><td class="NLM_def"><p class="first last">5,5′-dithiobis(2-nitrobenzoic acid)</p></td></tr><tr><td class="NLM_term">ATA</td><td class="NLM_def"><p class="first last">aurintricarboxylic acid</p></td></tr><tr><td class="NLM_term">FDCO</td><td class="NLM_def"><p class="first last">fmoc-<span class="smallcaps smallerCapital">d</span>-Cha-OH</p></td></tr><tr><td class="NLM_term">PNPase</td><td class="NLM_def"><p class="first last">polynucleotide phosphorylase</p></td></tr><tr><td class="NLM_term">GBSV1-NSN</td><td class="NLM_def"><p class="first last">thermophilic bacteriophage GBSV1</p></td></tr><tr><td class="NLM_term">OVS</td><td class="NLM_def"><p class="first last">vinylsulfonic acid</p></td></tr><tr><td class="NLM_term">AP</td><td class="NLM_def"><p class="first last">apurinic/apyrimidinic</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximum inhibitory concentration</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43277" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43277" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 65 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Zuo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">Exoribonuclease superfamilies: structural analysis and phylogenetic distribution</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1026</span><span class="refDoi"> DOI: 10.1093/nar/29.5.1017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2F29.5.1017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=11222749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvFaqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=1017-1026&author=Y.+Zuoauthor=M.+P.+Deutscher&title=Exoribonuclease+superfamilies%3A+structural+analysis+and+phylogenetic+distribution&doi=10.1093%2Fnar%2F29.5.1017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Exoribonuclease superfamilies: structural analysis and phylogenetic distribution</span></div><div class="casAuthors">Zuo, Yuhong; Deutscher, Murray P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1017-1026</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review with 56 refs.  Exoribonucleases play an important role in all aspects of RNA metab.  Biochem. and genetic analyses in recent years have identified many new RNases and it is now clear that a single cell can contain multiple enzymes of this class.  Here, we analyze the structure and phylogenetic distribution of the known exoribonucleases.  Based on extensive sequence anal. and on their catalytic properties, all of the exoribonucleases and their homologs have been grouped into six superfamilies and various subfamilies.  We identify common motifs that can be used to characterize newly-discovered exoribonucleases, and based on these motifs we correct some previously misassigned proteins.  This anal. may serve as a useful first step for developing a nomenclature for this group of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcVrS04JSdSrVg90H21EOLACvtfcHk0lgfS3vQgBkmcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvFaqs7w%253D&md5=dca9ea9312120b40d30e20cbab240018</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F29.5.1017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F29.5.1017%26sid%3Dliteratum%253Aachs%26aulast%3DZuo%26aufirst%3DY.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DExoribonuclease%2520superfamilies%253A%2520structural%2520analysis%2520and%2520phylogenetic%2520distribution%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2001%26volume%3D29%26spage%3D1017%26epage%3D1026%26doi%3D10.1093%2Fnar%2F29.5.1017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Yang, W.</span><span> </span><span class="NLM_article-title">Nucleases: diversity of structure, function and mechanism</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1017/S0033583510000181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1017%2FS0033583510000181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20854710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1-93&author=W.+Yang&title=Nucleases%3A+diversity+of+structure%2C+function+and+mechanism&doi=10.1017%2FS0033583510000181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleases: diversity of structure, function and mechanism</span></div><div class="casAuthors">Yang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-93</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Nucleases cleave the phosphodiester bonds of nucleic acids and may be endo or exo, DNase or RNase, topoisomerases, recombinases, ribozymes, or RNA splicing enzymes.  In this review, I survey nuclease activities with known structures and catalytic machinery and classify them by reaction mechanism and metal-ion dependence and by their biol. function ranging from DNA replication, recombination, repair, RNA maturation, processing, interference, to defense, nutrient regeneration or cell death.  Several general principles emerge from this anal.  There is little correlation between catalytic mechanism and biol. function.  A single catalytic mechanism can be adapted in a variety of reactions and biol. pathways.  Conversely, a single biol. process can often be accomplished by multiple tertiary and quaternary folds and by more than one catalytic mechanism.  Two-metal-ion-dependent nucleases comprise the largest no. of different tertiary folds and mediate the most diverse set of biol. functions.  Metal-ion-dependent cleavage is exclusively assocd. with exonucleases producing mononucleotides and endonucleases that cleave double- or single-stranded substrates in helical and base-stacked conformations.  All metal-ion-independent RNases generate 2'.3'-cyclic phosphate products, and all metal-ion-independent DNases form phospho-protein intermediates.  I also find several previously unnoted relationships between different nucleases and shared catalytic configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7b0a8_b1YbVg90H21EOLACvtfcHk0lgfS3vQgBkmcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D&md5=4cd0bd5cec91b4572adf183ba2d90e7a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000181%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26atitle%3DNucleases%253A%2520diversity%2520of%2520structure%252C%2520function%2520and%2520mechanism%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2011%26volume%3D44%26spage%3D1%26epage%3D93%26doi%3D10.1017%2FS0033583510000181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Hastie, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zandonatti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saphire, E. O.</span><span> </span><span class="NLM_article-title">Structural basis for the dsRNA specificity of the Lassa virus NP exonuclease</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e44211</span><span class="refDoi"> DOI: 10.1371/journal.pone.0044211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pone.0044211" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e44211&author=K.+M.+Hastieauthor=L.+B.+Kingauthor=M.+A.+Zandonattiauthor=E.+O.+Saphire&title=Structural+basis+for+the+dsRNA+specificity+of+the+Lassa+virus+NP+exonuclease&doi=10.1371%2Fjournal.pone.0044211"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0044211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0044211%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DK.%2BM.%26aulast%3DKing%26aufirst%3DL.%2BB.%26aulast%3DZandonatti%26aufirst%3DM.%2BA.%26aulast%3DSaphire%26aufirst%3DE.%2BO.%26atitle%3DStructural%2520basis%2520for%2520the%2520dsRNA%2520specificity%2520of%2520the%2520Lassa%2520virus%2520NP%2520exonuclease%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De44211%26doi%3D10.1371%2Fjournal.pone.0044211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Crystal structure of CRN-4: implications for domain function in apoptotic DNA degradation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">457</span><span class="refDoi"> DOI: 10.1128/MCB.01006-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1128%2FMCB.01006-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18981218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=448-457&author=Y.+Y.+Hsiaoauthor=A.+Nakagawaauthor=Z.+Shiauthor=S.+Mitaniauthor=D.+Xueauthor=H.+S.+Yuan&title=Crystal+structure+of+CRN-4%3A+implications+for+domain+function+in+apoptotic+DNA+degradation&doi=10.1128%2FMCB.01006-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of CRN-4: implications for domain function in apoptotic DNA degradation</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Nakagawa, Akihisa; Shi, Zhonghao; Mitani, Shohei; Xue, Ding; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-457</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cell death related nuclease 4 (CRN-4) is one of the apoptotic nucleases involved in DNA degrdn. in Caenorhabditis elegans.  To understand how CRN-4 is involved in apoptotic DNA fragmentation, we analyzed CRN-4's biochem. properties, in vivo cell functions, and the crystal structures of CRN-4 in apo-form, Mn2+-bound active form, and Er3+-bound inactive form.  CRN-4 is a dimeric nuclease with the optimal enzyme activity in cleaving double-stranded DNA in apoptotic salt conditions.  Both mutational studies and the structures of the Mn2+-bound CRN-4 revealed the geometry of the functional nuclease active site in the N-terminal DEDDh domain.  The C-terminal domain, termed the Zn-domain, contains basic surface residues ideal for nucleic acid recognition and is involved in DNA binding, as confirmed by deletion assays.  Cell death anal. in C. elegans further demonstrated that both the nuclease active site and the Zn-domain are required for crn-4's function in apoptosis.  Combining all of the data, we suggest a structural model where chromosomal DNA is bound at the Zn-domain and cleaved at the DEDDh nuclease domain in CRN-4 when the cell is undergoing apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrerdQCbulCTrVg90H21EOLACvtfcHk0lhCXs38iEnpAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChurc%253D&md5=161b104a4169ebf613fe1e99a3e2ffbb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FMCB.01006-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01006-08%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DNakagawa%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DMitani%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DCrystal%2520structure%2520of%2520CRN-4%253A%2520implications%2520for%2520domain%2520function%2520in%2520apoptotic%2520DNA%2520degradation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D448%26epage%3D457%26doi%3D10.1128%2FMCB.01006-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Korada, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, C. E.</span><span> </span><span class="NLM_article-title">Crystal structures of Escherichia coli exonuclease I in complex with single-stranded DNA provide insights into the mechanism of processive digestion</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">5887</span><span class="NLM_x">–</span> <span class="NLM_lpage">5897</span><span class="refDoi"> DOI: 10.1093/nar/gkt278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgkt278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23609540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFCgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=5887-5897&author=S.+K.+Koradaauthor=T.+D.+Johnsauthor=C.+E.+Smithauthor=N.+D.+Jonesauthor=K.+A.+McCabeauthor=C.+E.+Bell&title=Crystal+structures+of+Escherichia+coli+exonuclease+I+in+complex+with+single-stranded+DNA+provide+insights+into+the+mechanism+of+processive+digestion&doi=10.1093%2Fnar%2Fgkt278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of Escherichia coli exonuclease I in complex with single-stranded DNA provide insights into the mechanism of processive digestion</span></div><div class="casAuthors">Korada, Sai Krishna C.; Johns, Trevor D.; Smith, Christopher E.; Jones, Nathan D.; McCabe, Kimberly A.; Bell, Charles E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5887-5897</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Escherichia coli Exonuclease I (ExoI) digests single-stranded DNA (ssDNA) in the 3'-5' direction in a highly processive manner.  The crystal structure of ExoI, detd. previously in the absence of DNA, revealed a C-shaped mol. with three domains that form a central pos. charged groove.  The active site is at the bottom of the groove, while an extended loop, proposed to encircle the DNA, crosses over the groove.  Here, we present crystal structures of ExoI in complex with four different ssDNA substrates.  The structures all have the ssDNA bound in essentially the predicted manner, with the 3'-end in the active site and the downstream end under the crossover loop.  The central nucleotides of the DNA form a prominent bulge that contacts the SH3-like domain, while the nucleotides at the downstream end of the DNA form extensive interactions with an anchor' site.  Seven of the complexes are similar to one another, but one has the ssDNA bound in a distinct conformation.  The highest-resoln. structure, detd. at 1.95 Å, reveals an Mg2+ ion bound to the scissile phosphate in a position corresponding to MgB in related two-metal nucleases.  The structures provide new insights into the mechanism of processive digestion that will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6XunF5jiSqLVg90H21EOLACvtfcHk0lhCXs38iEnpAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFCgu74%253D&md5=e60a1b8ac88aa33674831a042a9435a2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt278%26sid%3Dliteratum%253Aachs%26aulast%3DKorada%26aufirst%3DS.%2BK.%26aulast%3DJohns%26aufirst%3DT.%2BD.%26aulast%3DSmith%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DN.%2BD.%26aulast%3DMcCabe%26aufirst%3DK.%2BA.%26aulast%3DBell%26aufirst%3DC.%2BE.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520exonuclease%2520I%2520in%2520complex%2520with%2520single-stranded%2520DNA%2520provide%2520insights%2520into%2520the%2520mechanism%2520of%2520processive%2520digestion%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D5887%26epage%3D5897%26doi%3D10.1093%2Fnar%2Fgkt278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Horio, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamasaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, T.</span><span> </span><span class="NLM_article-title">Crystal structure of human ISG20, an interferon-induced antiviral ribonuclease</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">577</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.1016/j.febslet.2004.09.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.febslet.2004.09.074" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=577&publication_year=2004&pages=111-116&author=T.+Horioauthor=M.+Muraiauthor=T.+Inoueauthor=T.+Hamasakiauthor=T.+Tanakaauthor=T.+Ohgi&title=Crystal+structure+of+human+ISG20%2C+an+interferon-induced+antiviral+ribonuclease&doi=10.1016%2Fj.febslet.2004.09.074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2004.09.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2004.09.074%26sid%3Dliteratum%253Aachs%26aulast%3DHorio%26aufirst%3DT.%26aulast%3DMurai%26aufirst%3DM.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHamasaki%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DOhgi%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520human%2520ISG20%252C%2520an%2520interferon-induced%2520antiviral%2520ribonuclease%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D577%26spage%3D111%26epage%3D116%26doi%3D10.1016%2Fj.febslet.2004.09.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">How an exonuclease decides where to stop in trimming of nucleic acids: crystal structures of RNase T-product complexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">8144</span><span class="NLM_x">–</span> <span class="NLM_lpage">8154</span><span class="refDoi"> DOI: 10.1093/nar/gks548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgks548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22718982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlKltrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=8144-8154&author=Y.+Y.+Hsiaoauthor=Y.+Duhauthor=Y.+P.+Chenauthor=Y.+T.+Wangauthor=H.+S.+Yuan&title=How+an+exonuclease+decides+where+to+stop+in+trimming+of+nucleic+acids%3A+crystal+structures+of+RNase+T-product+complexes&doi=10.1093%2Fnar%2Fgks548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">How an exonuclease decides where to stop in trimming of nucleic acids: crystal structures of RNase T-product complexes</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Duh, Yulander; Chen, Yi-Ping; Wang, Yi-Ting; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8144-8154</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exonucleases are key enzymes in the maintenance of genome stability, processing of immature RNA precursors and degrdn. of unnecessary nucleic acids.  However, it remains unclear how exonucleases digest nucleic acids to generate correct end products for next-step processing.  Here we show how the exonuclease RNase T stops its trimming precisely.  The crystal structures of RNase T in complex with a stem-loop DNA, a GG dinucleotide and single-stranded DNA with different 3'-end sequences demonstrate why a duplex with a short 3'-overhang, a dinucleotide and a ssDNA with a 3'-end C cannot be further digested by RNase T.  Several hydrophobic residues in RNase T change their conformation upon substrate binding and induce an active or inactive conformation in the active site that construct a precise machine to det. which substrate should be digested based on its sequence, length and structure.  These studies thus provide mechanistic insights into how RNase T prevents over digestion of its various substrates, and the results can be extrapolated to the thousands of members of the DEDDh family of exonucleases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwS0gbeUZVZLVg90H21EOLACvtfcHk0lhCXs38iEnpAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlKltrfJ&md5=15052613cbcc50891518739f660671f0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks548%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DDuh%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%2BP.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DHow%2520an%2520exonuclease%2520decides%2520where%2520to%2520stop%2520in%2520trimming%2520of%2520nucleic%2520acids%253A%2520crystal%2520structures%2520of%2520RNase%2520T-product%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D8144%26epage%3D8154%26doi%3D10.1093%2Fnar%2Fgks548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Kupsco, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marzluff, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thapar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duronio, R. J.</span><span> </span><span class="NLM_article-title">Genetic and biochemical characterization of Drosophila Snipper: A promiscuous member of the metazoan 3′hExo/ERI-1 family of 3′ to 5′ exonucleases</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">2117</span><span class="refDoi"> DOI: 10.1261/rna.186706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1261%2Frna.186706" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=2103-2117&author=J.+M.+Kupscoauthor=M.+J.+Wuauthor=W.+F.+Marzluffauthor=R.+Thaparauthor=R.+J.+Duronio&title=Genetic+and+biochemical+characterization+of+Drosophila+Snipper%3A+A+promiscuous+member+of+the+metazoan+3%E2%80%B2hExo%2FERI-1+family+of+3%E2%80%B2+to+5%E2%80%B2+exonucleases&doi=10.1261%2Frna.186706"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1261%2Frna.186706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.186706%26sid%3Dliteratum%253Aachs%26aulast%3DKupsco%26aufirst%3DJ.%2BM.%26aulast%3DWu%26aufirst%3DM.%2BJ.%26aulast%3DMarzluff%26aufirst%3DW.%2BF.%26aulast%3DThapar%26aufirst%3DR.%26aulast%3DDuronio%26aufirst%3DR.%2BJ.%26atitle%3DGenetic%2520and%2520biochemical%2520characterization%2520of%2520Drosophila%2520Snipper%253A%2520A%2520promiscuous%2520member%2520of%2520the%2520metazoan%25203%25E2%2580%25B2hExo%252FERI-1%2520family%2520of%25203%25E2%2580%25B2%2520to%25205%25E2%2580%25B2%2520exonucleases%26jtitle%3DRNA%26date%3D2006%26volume%3D12%26spage%3D2103%26epage%3D2117%26doi%3D10.1261%2Frna.186706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Zhao, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. X.</span><span> </span><span class="NLM_article-title">A novel mutation of the WRN gene in a Chinese patient with Werner syndrome</span> <span class="citation_source-journal">Clin. Exp. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1111/j.1365-2230.2007.02641.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1111%2Fj.1365-2230.2007.02641.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18205852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1c3lslyntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=278-281&author=N.+Zhaoauthor=F.+Haoauthor=T.+Quauthor=Y.+G.+Zuoauthor=B.+X.+Wang&title=A+novel+mutation+of+the+WRN+gene+in+a+Chinese+patient+with+Werner+syndrome&doi=10.1111%2Fj.1365-2230.2007.02641.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A novel mutation of the WRN gene in a Chinese patient with Werner syndrome</span></div><div class="casAuthors">Zhao N; Hao F; Qu T; Zuo Y-G; Wang B-X</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and experimental dermatology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">278-81</span>
        ISSN:<span class="NLM_cas:issn">0307-6938</span>.
    </div><div class="casAbstract">Werner syndrome (WS) is an autosomal recessive inherited disease characterized by features of premature ageing.  It is caused by mutations of the WRN gene encoding a protein with both exonuclease and helicase activities.  The aim of this study was to identify gene mutations in a Chinese patient with WS.  A 31-year-old Chinese man with typical features of WS was diagnosed as having probable WS.  We performed PCR to scan 33 exons of the WRN gene of the patient, six members of his family, and 50 unrelated controls.  Automated DNA sequencing identified the mutation in the patient as 3250delG.  The proband's parents, son, younger brother and paternal grandmother were heterozygous.  We did not find this heterozygous mutation in the proband's maternal grandmother or in any of 50 normal controls.  The novel mutation in the WRN gene is responsible for the pathogenesis of WS and genetic detection is a useful method to confirm the diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgR4CIUtwleUQ0lWbKiCflfW6udTcc2eYewL_fIHyW77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3lslyntw%253D%253D&md5=8f467ba3b0c7351159d9ed7f35ce0f04</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2230.2007.02641.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2230.2007.02641.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DN.%26aulast%3DHao%26aufirst%3DF.%26aulast%3DQu%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DY.%2BG.%26aulast%3DWang%26aufirst%3DB.%2BX.%26atitle%3DA%2520novel%2520mutation%2520of%2520the%2520WRN%2520gene%2520in%2520a%2520Chinese%2520patient%2520with%2520Werner%2520syndrome%26jtitle%3DClin.%2520Exp.%2520Dermatol.%26date%3D2008%26volume%3D33%26spage%3D278%26epage%3D281%26doi%3D10.1111%2Fj.1365-2230.2007.02641.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Perry, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yannone, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holden, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitomi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asaithamby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span> </span><span class="NLM_article-title">WRN exonuclease structure and molecular mechanism imply an editing role in DNA end processing</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1038/nsmb1088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnsmb1088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16622405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD28XltFWlsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=414-422&author=J.+J.+Perryauthor=S.+M.+Yannoneauthor=L.+G.+Holdenauthor=C.+Hitomiauthor=A.+Asaithambyauthor=S.+Hanauthor=P.+K.+Cooperauthor=D.+J.+Chenauthor=J.+A.+Tainer&title=WRN+exonuclease+structure+and+molecular+mechanism+imply+an+editing+role+in+DNA+end+processing&doi=10.1038%2Fnsmb1088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">WRN exonuclease structure and molecular mechanism imply an editing role in DNA end processing</span></div><div class="casAuthors">Perry, J. Jefferson P.; Yannone, Steven M.; Holden, Lauren G.; Hitomi, Chiharu; Asaithamby, Aroumougame; Han, Seungil; Cooper, Priscilla K.; Chen, David J.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">WRN is unique among the five human RecQ DNA helicases in having a functional exonuclease domain (WRN-exo) and being defective in the premature aging and cancer-related disorder Werner syndrome.  Here, we characterize WRN-exo crystal structures, biochem. activity and participation in DNA end joining.  Metal-ion complex structures, active site mutations and activity assays reveal a nuclease mechanism mediated by two metal ions.  The DNA end-binding Ku70/80 complex specifically stimulates WRN-exo activity, and structure-based mutational inactivation of WRN-exo alters DNA end joining in human cells.  We furthermore establish structural and biochem. similarities of WRN-exo to DnaQ-family replicative proofreading exonucleases, describing WRN-specific adaptations consistent with double-stranded DNA specificity and functionally important conformational changes.  These results indicate WRN-exo is a human DnaQ family member and support DnaQ-like proofreading activities stimulated by Ku70/80, with implications for WRN functions in age-related pathologies and maintenance of genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4cx5lqWQDh7Vg90H21EOLACvtfcHk0liLGhRUk1tu2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltFWlsrs%253D&md5=db2f70f2d735904d7e0221f218aa6d83</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1088%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DJ.%2BJ.%26aulast%3DYannone%26aufirst%3DS.%2BM.%26aulast%3DHolden%26aufirst%3DL.%2BG.%26aulast%3DHitomi%26aufirst%3DC.%26aulast%3DAsaithamby%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DP.%2BK.%26aulast%3DChen%26aufirst%3DD.%2BJ.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DWRN%2520exonuclease%2520structure%2520and%2520molecular%2520mechanism%2520imply%2520an%2520editing%2520role%2520in%2520DNA%2520end%2520processing%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D414%26epage%3D422%26doi%3D10.1038%2Fnsmb1088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Crow, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayward, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parmar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robins, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamel, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corry, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frints, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepper, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingston, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massey, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meritet, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponsot, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voit, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonthron, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnes, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, T.</span><span> </span><span class="NLM_article-title">Mutations in the gene encoding the 3′-5′ DNA exonuclease TREX1 cause Aicardi-Goutieres syndrome at the AGS1 locus</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">920</span><span class="refDoi"> DOI: 10.1038/ng1845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fng1845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16845398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsVCgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=917-920&author=Y.+J.+Crowauthor=B.+E.+Haywardauthor=R.+Parmarauthor=P.+Robinsauthor=A.+Leitchauthor=M.+Aliauthor=D.+N.+Blackauthor=H.+van+Bokhovenauthor=H.+G.+Brunnerauthor=B.+C.+Hamelauthor=P.+C.+Corryauthor=F.+M.+Cowanauthor=S.+G.+Frintsauthor=J.+Klepperauthor=J.+H.+Livingstonauthor=S.+A.+Lynchauthor=R.+F.+Masseyauthor=J.+F.+Meritetauthor=J.+L.+Michaudauthor=G.+Ponsotauthor=T.+Voitauthor=P.+Lebonauthor=D.+T.+Bonthronauthor=A.+P.+Jacksonauthor=D.+E.+Barnesauthor=T.+Lindahl&title=Mutations+in+the+gene+encoding+the+3%E2%80%B2-5%E2%80%B2+DNA+exonuclease+TREX1+cause+Aicardi-Goutieres+syndrome+at+the+AGS1+locus&doi=10.1038%2Fng1845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the gene encoding the 3'-5' DNA exonuclease TREX1 cause Aicardi-Goutieres syndrome at the AGS1 locus</span></div><div class="casAuthors">Crow, Yanick J.; Hayward, Bruce E.; Parmar, Rekha; Robins, Peter; Leitch, Andrea; Ali, Manir; Black, Deborah N.; van Bokhoven, Hans; Brunner, Han G.; Hamel, Ben C.; Corry, Peter C.; Cowan, Frances M.; Frints, Suzanne G.; Klepper, Joerg; Livingston, John H.; Lynch, Sally Ann; Massey, Roger F.; Meritet, Jean Francois; Michaud, Jacques L.; Ponsot, Gerard; Voit, Thomas; Lebon, Pierre; Bonthron, David T.; Jackson, Andrew P.; Barnes, Deborah E.; Lindahl, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">917-920</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aicardi-Goutieres syndrome (AGS) presents as a severe neurol. brain disease and is a genetic mimic of the sequelae of transplacentally acquired viral infection.  Evidence exists for a perturbation of innate immunity as a primary pathogenic event in the disease phenotype.  Here, we show that TREX1, encoding the major mammalian 3' → 5' DNA exonuclease, is the AGS1 gene, and AGS-causing mutations result in abrogation of TREX1 enzyme activity.  Similar loss of function in the Trex1-/- mouse leads to an inflammatory phenotype.  Our findings suggest an unanticipated role for TREX1 in processing or clearing anomalous DNA structures, failure of which results in the triggering of an abnormal innate immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIwU5CVVO0DrVg90H21EOLACvtfcHk0ljQYYXRocK37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsVCgsro%253D&md5=b36e47cc93a79cbc47c41cf7a33f94a7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fng1845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1845%26sid%3Dliteratum%253Aachs%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26aulast%3DHayward%26aufirst%3DB.%2BE.%26aulast%3DParmar%26aufirst%3DR.%26aulast%3DRobins%26aufirst%3DP.%26aulast%3DLeitch%26aufirst%3DA.%26aulast%3DAli%26aufirst%3DM.%26aulast%3DBlack%26aufirst%3DD.%2BN.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3DHamel%26aufirst%3DB.%2BC.%26aulast%3DCorry%26aufirst%3DP.%2BC.%26aulast%3DCowan%26aufirst%3DF.%2BM.%26aulast%3DFrints%26aufirst%3DS.%2BG.%26aulast%3DKlepper%26aufirst%3DJ.%26aulast%3DLivingston%26aufirst%3DJ.%2BH.%26aulast%3DLynch%26aufirst%3DS.%2BA.%26aulast%3DMassey%26aufirst%3DR.%2BF.%26aulast%3DMeritet%26aufirst%3DJ.%2BF.%26aulast%3DMichaud%26aufirst%3DJ.%2BL.%26aulast%3DPonsot%26aufirst%3DG.%26aulast%3DVoit%26aufirst%3DT.%26aulast%3DLebon%26aufirst%3DP.%26aulast%3DBonthron%26aufirst%3DD.%2BT.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DBarnes%26aufirst%3DD.%2BE.%26aulast%3DLindahl%26aufirst%3DT.%26atitle%3DMutations%2520in%2520the%2520gene%2520encoding%2520the%25203%25E2%2580%25B2-5%25E2%2580%25B2%2520DNA%2520exonuclease%2520TREX1%2520cause%2520Aicardi-Goutieres%2520syndrome%2520at%2520the%2520AGS1%2520locus%26jtitle%3DNat.%2520Genet.%26date%3D2006%26volume%3D38%26spage%3D917%26epage%3D920%26doi%3D10.1038%2Fng1845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senenko, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Silva, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyse, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kere, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeiffer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koskenmies, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominiczak, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gahr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrino, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubner, N.</span><span> </span><span class="NLM_article-title">Mutations in the gene encoding the 3′-5′ DNA exonuclease TREX1 are associated with systemic lupus erythematosus</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1065</span><span class="NLM_x">–</span> <span class="NLM_lpage">1067</span><span class="refDoi"> DOI: 10.1038/ng2091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fng2091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17660818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps12gtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2007&pages=1065-1067&author=M.+A.+Lee-Kirschauthor=M.+Gongauthor=D.+Chowdhuryauthor=L.+Senenkoauthor=K.+Engelauthor=Y.+A.+Leeauthor=U.+de+Silvaauthor=S.+L.+Baileyauthor=T.+Witteauthor=T.+J.+Vyseauthor=J.+Kereauthor=C.+Pfeifferauthor=S.+Harveyauthor=A.+Wongauthor=S.+Koskenmiesauthor=O.+Hummelauthor=K.+Rohdeauthor=R.+E.+Schmidtauthor=A.+F.+Dominiczakauthor=M.+Gahrauthor=T.+Hollisauthor=F.+W.+Perrinoauthor=J.+Liebermanauthor=N.+Hubner&title=Mutations+in+the+gene+encoding+the+3%E2%80%B2-5%E2%80%B2+DNA+exonuclease+TREX1+are+associated+with+systemic+lupus+erythematosus&doi=10.1038%2Fng2091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the gene encoding the 3'-5' DNA exonuclease TREX1 are associated with systemic lupus erythematosus</span></div><div class="casAuthors">Lee-Kirsch, Min Ae; Gong, Maolian; Chowdhury, Dipanjan; Senenko, Lydia; Engel, Kerstin; Lee, Young-Ae; de Silva, Udesh; Bailey, Suzanna L.; Witte, Torsten; Vyse, Timothy J.; Kere, Juha; Pfeiffer, Christiane; Harvey, Scott; Wong, Andrew; Koskenmies, Sari; Hummel, Oliver; Rohde, Klaus; Schmidt, Reinhold E.; Dominiczak, Anna F.; Gahr, Manfred; Hollis, Thomas; Perrino, Fred W.; Lieberman, Judy; Huebner, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1065-1067</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TREX1 acts in concert with the SET complex in granzyme A-mediated apoptosis, and mutations in TREX1 cause Aicardi-Goutieres syndrome and familial chilblain lupus.  Here, we report monoallelic frameshift or missense mutations and one 3' UTR variant of TREX1 present in 9/417 individuals with systemic lupus erythematosus but absent in 1,712 controls (P = 4.1 × 10-7).  We demonstrate that two mutant TREX1 alleles alter subcellular targeting.  Our findings implicate TREX1 in the pathogenesis of SLE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkgyqOJHcmtbVg90H21EOLACvtfcHk0ljQYYXRocK37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps12gtLg%253D&md5=f91c180a5614571fe136aa36159d51f0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fng2091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng2091%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DGong%26aufirst%3DM.%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DSenenko%26aufirst%3DL.%26aulast%3DEngel%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DY.%2BA.%26aulast%3Dde%2BSilva%26aufirst%3DU.%26aulast%3DBailey%26aufirst%3DS.%2BL.%26aulast%3DWitte%26aufirst%3DT.%26aulast%3DVyse%26aufirst%3DT.%2BJ.%26aulast%3DKere%26aufirst%3DJ.%26aulast%3DPfeiffer%26aufirst%3DC.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DKoskenmies%26aufirst%3DS.%26aulast%3DHummel%26aufirst%3DO.%26aulast%3DRohde%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DR.%2BE.%26aulast%3DDominiczak%26aufirst%3DA.%2BF.%26aulast%3DGahr%26aufirst%3DM.%26aulast%3DHollis%26aufirst%3DT.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DLieberman%26aufirst%3DJ.%26aulast%3DHubner%26aufirst%3DN.%26atitle%3DMutations%2520in%2520the%2520gene%2520encoding%2520the%25203%25E2%2580%25B2-5%25E2%2580%25B2%2520DNA%2520exonuclease%2520TREX1%2520are%2520associated%2520with%2520systemic%2520lupus%2520erythematosus%26jtitle%3DNat.%2520Genet.%26date%3D2007%26volume%3D39%26spage%3D1065%26epage%3D1067%26doi%3D10.1038%2Fng2091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Graham, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerle, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolles, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denison, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baric, R. S.</span><span> </span><span class="NLM_article-title">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1820</span><span class="NLM_x">–</span> <span class="NLM_lpage">1826</span><span class="refDoi"> DOI: 10.1038/nm.2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnm.2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23142821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CktLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1820-1826&author=R.+L.+Grahamauthor=M.+M.+Beckerauthor=L.+D.+Eckerleauthor=M.+Bollesauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=A+live%2C+impaired-fidelity+coronavirus+vaccine+protects+in+an+aged%2C+immunocompromised+mouse+model+of+lethal+disease&doi=10.1038%2Fnm.2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span></div><div class="casAuthors">Graham, Rachel L.; Becker, Michelle M.; Eckerle, Lance D.; Bolles, Meagan; Denison, Mark R.; Baric, Ralph S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1820-1826</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Live, attenuated RNA virus vaccines are efficacious but subject to reversion to virulence.  Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses.  Coronavirus (CoV) replication fidelity is approx. 20-fold greater than that of other RNA viruses and is mediated by a 3'→5' exonuclease (ExoN) activity that probably functions in RNA proofreading.  In this study we demonstrate that engineered inactivation of severe acute respiratory syndrome (SARS)-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged and immunocompromised mice.  The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo.  ExoN inactivation has potential for broad applications in the stable attenuation of CoVs and, perhaps, other RNA viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYHcgTwksD3bVg90H21EOLACvtfcHk0ljQYYXRocK37Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CktLvF&md5=e59f2b04c0ad2a6d8bbe62b7034adb66</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2972%26sid%3Dliteratum%253Aachs%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DBecker%26aufirst%3DM.%2BM.%26aulast%3DEckerle%26aufirst%3DL.%2BD.%26aulast%3DBolles%26aufirst%3DM.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DA%2520live%252C%2520impaired-fidelity%2520coronavirus%2520vaccine%2520protects%2520in%2520an%2520aged%252C%2520immunocompromised%2520mouse%2520model%2520of%2520lethal%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D1820%26epage%3D1826%26doi%3D10.1038%2Fnm.2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Minskaia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertzig, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbalenya, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campanacci, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cambillau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebuhr, J.</span><span> </span><span class="NLM_article-title">Discovery of an RNA virus 3′- > 5′ exoribonuclease that is critically involved in coronavirus RNA synthesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">5108</span><span class="NLM_x">–</span> <span class="NLM_lpage">5113</span><span class="refDoi"> DOI: 10.1073/pnas.0508200103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.0508200103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=5108-5113&author=E.+Minskaiaauthor=T.+Hertzigauthor=A.+E.+Gorbalenyaauthor=V.+Campanacciauthor=C.+Cambillauauthor=B.+Canardauthor=J.+Ziebuhr&title=Discovery+of+an+RNA+virus+3%E2%80%B2-+%3E+5%E2%80%B2+exoribonuclease+that+is+critically+involved+in+coronavirus+RNA+synthesis&doi=10.1073%2Fpnas.0508200103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0508200103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0508200103%26sid%3Dliteratum%253Aachs%26aulast%3DMinskaia%26aufirst%3DE.%26aulast%3DHertzig%26aufirst%3DT.%26aulast%3DGorbalenya%26aufirst%3DA.%2BE.%26aulast%3DCampanacci%26aufirst%3DV.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DCanard%26aufirst%3DB.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520RNA%2520virus%25203%25E2%2580%25B2-%2520%253E%25205%25E2%2580%25B2%2520exoribonuclease%2520that%2520is%2520critically%2520involved%2520in%2520coronavirus%2520RNA%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D5108%26epage%3D5113%26doi%3D10.1073%2Fpnas.0508200103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Qi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schelde, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallat, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span> </span><span class="NLM_article-title">Cap binding and immune evasion revealed by Lassa nucleoprotein structure</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">783</span><span class="refDoi"> DOI: 10.1038/nature09605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnature09605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21085117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=779-783&author=X.+Qiauthor=S.+Lanauthor=W.+Wangauthor=L.+M.+Scheldeauthor=H.+Dongauthor=G.+D.+Wallatauthor=H.+Lyauthor=Y.+Liangauthor=C.+Dong&title=Cap+binding+and+immune+evasion+revealed+by+Lassa+nucleoprotein+structure&doi=10.1038%2Fnature09605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cap binding and immune evasion revealed by Lassa nucleoprotein structure</span></div><div class="casAuthors">Qi, Xiao-Xuan; Lan, Shui-Yun; Wang, Wen-Jian; Schelde, Lisa McLay; Dong, Hao-Hao; Wallat, Gregor D.; Ly, Hinh; Liang, Yu-Ying; Dong, Chang-Jiang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">779-783</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lassa virus, the causative agent of Lassa fever, causes thousands of deaths annually and is a biol. threat agent, for which there is no vaccine and limited therapy.  The nucleoprotein (NP) of Lassa virus has essential roles in viral RNA synthesis and immune suppression, the mol. mechanisms of which are poorly understood.  Here we report the crystal structure of Lassa virus NP at 1.80 Å resoln., which reveals amino (N)- and carboxy (C)-terminal domains with structures unlike any of the reported viral NPs.  The N domain folds into a novel structure with a deep cavity for binding the m7GpppN cap structure that is required for viral RNA transcription, whereas the C domain contains 3'-5' exoribonuclease activity involved in suppressing interferon induction.  To our knowledge this is the first X-ray crystal structure solved for an arenaviral NP, which reveals its unexpected functions and indicates unique mechanisms in cap binding and immune evasion.  These findings provide great potential for vaccine and drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZleegizK2j7Vg90H21EOLACvtfcHk0lggGo3l9d15fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr3F&md5=4ae1f6e2dcdb2196c3aff97aca5d39bb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature09605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09605%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DX.%26aulast%3DLan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSchelde%26aufirst%3DL.%2BM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWallat%26aufirst%3DG.%2BD.%26aulast%3DLy%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DCap%2520binding%2520and%2520immune%2520evasion%2520revealed%2520by%2520Lassa%2520nucleoprotein%2520structure%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D779%26epage%3D783%26doi%3D10.1038%2Fnature09605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Hasan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rakheja, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattnaik, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brugarolas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dozmorov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakeland, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, N.</span><span> </span><span class="NLM_article-title">Trex1 regulates lysosomal biogenesis and interferon-independent activation of antiviral genes</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1038/ni.2475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fni.2475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23160154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=61-71&author=M.+Hasanauthor=J.+Kochauthor=D.+Rakhejaauthor=A.+K.+Pattnaikauthor=J.+Brugarolasauthor=I.+Dozmorovauthor=B.+Levineauthor=E.+K.+Wakelandauthor=M.+A.+Lee-Kirschauthor=N.+Yan&title=Trex1+regulates+lysosomal+biogenesis+and+interferon-independent+activation+of+antiviral+genes&doi=10.1038%2Fni.2475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Trex1 regulates lysosomal biogenesis and interferon-independent activation of antiviral genes</span></div><div class="casAuthors">Hasan, Maroof; Koch, James; Rakheja, Dinesh; Pattnaik, Asit K.; Brugarolas, James; Dozmorov, Igor; Levine, Beth; Wakeland, Edward K.; Lee-Kirsch, Min Ae; Yan, Nan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-71</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sensing of viral nucleic acids by the innate immune system triggers the prodn. of type I interferons, which activates interferon-stimulated genes (ISGs) and directs a multifaceted antiviral response.  ISGs can also be activated through interferon-independent pathways, although the precise mechanisms remain elusive.  Here we found that the cytosolic exonuclease Trex1 regulated the activation of a subset of ISGs independently of interferon.  Both Trex1-/- mouse cells and Trex1-mutant human cells had high expression of genes encoding antiviral mols. ('antiviral genes') and were refractory to viral infection.  The interferon-independent activation of antiviral genes in Trex1-/- cells required the adaptor STING, the kinase TBK1 and the transcription factors IRF3 and IRF7.  We also found that Trex1-deficient cells had an expanded lysosomal compartment, altered subcellular localization of the transcription factor TFEB and diminished activity of the regulator mTORC1.  Together our data identify Trex1 as a regulator of lysosomal biogenesis and interferon-independent activation of antiviral genes and show that dysregulation of lysosomes can elicit innate immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY-tMxMVIKlrVg90H21EOLACvtfcHk0lggGo3l9d15fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtrvI&md5=c54d3da15545b1f0fbf8d2f10480c041</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fni.2475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.2475%26sid%3Dliteratum%253Aachs%26aulast%3DHasan%26aufirst%3DM.%26aulast%3DKoch%26aufirst%3DJ.%26aulast%3DRakheja%26aufirst%3DD.%26aulast%3DPattnaik%26aufirst%3DA.%2BK.%26aulast%3DBrugarolas%26aufirst%3DJ.%26aulast%3DDozmorov%26aufirst%3DI.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DWakeland%26aufirst%3DE.%2BK.%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DTrex1%2520regulates%2520lysosomal%2520biogenesis%2520and%2520interferon-independent%2520activation%2520of%2520antiviral%2520genes%26jtitle%3DNat.%2520Immunol.%26date%3D2013%26volume%3D14%26spage%3D61%26epage%3D71%26doi%3D10.1038%2Fni.2475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Yan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regalado-Magdos, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stiggelbout, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span> </span><span class="NLM_article-title">The cytosolic exonuclease TREX1 inhibits the innate immune response to human immunodeficiency virus type 1</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1005</span><span class="NLM_x">–</span> <span class="NLM_lpage">1013</span><span class="refDoi"> DOI: 10.1038/ni.1941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fni.1941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20871604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Slur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1005-1013&author=N.+Yanauthor=A.+D.+Regalado-Magdosauthor=B.+Stiggelboutauthor=M.+A.+Lee-Kirschauthor=J.+Lieberman&title=The+cytosolic+exonuclease+TREX1+inhibits+the+innate+immune+response+to+human+immunodeficiency+virus+type+1&doi=10.1038%2Fni.1941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The cytosolic exonuclease TREX1 inhibits the innate immune response to human immunodeficiency virus type 1</span></div><div class="casAuthors">Yan, Nan; Regalado-Magdos, Ashton D.; Stiggelbout, Bart; Lee-Kirsch, Min Ae; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1005-1013</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Viral infection triggers innate immune sensors to produce type I interferon.  However, infection of T cells and macrophages with human immunodeficiency virus (HIV) does not trip those alarms.  How HIV avoids activating nucleic acid sensors is unknown.  Here we found that the cytosolic exonuclease TREX1 suppressed interferon triggered by HIV.  In Trex1-/- mouse cells and human CD4+ T cells and macrophages in which TREX1 was inhibited by RNA-mediated interference, cytosolic HIV DNA accumulated and HIV infection induced type I interferon that inhibited HIV replication and spreading.  TREX1 bound to cytosolic HIV DNA and digested excess HIV DNA that would otherwise activate interferon expression via a pathway dependent on the kinase TBK1, the adaptor STING and the transcription factor IRF3.  HIV-stimulated interferon prodn. in cells deficient in TREX1 did not involve known nucleic acid sensors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMq3IR745M27Vg90H21EOLACvtfcHk0lg74exMqNbHhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Slur7E&md5=2b1846d165468c434ced9eedebfb098f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fni.1941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1941%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DN.%26aulast%3DRegalado-Magdos%26aufirst%3DA.%2BD.%26aulast%3DStiggelbout%26aufirst%3DB.%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520cytosolic%2520exonuclease%2520TREX1%2520inhibits%2520the%2520innate%2520immune%2520response%2520to%2520human%2520immunodeficiency%2520virus%2520type%25201%26jtitle%3DNat.%2520Immunol.%26date%3D2010%26volume%3D11%26spage%3D1005%26epage%3D1013%26doi%3D10.1038%2Fni.1941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ogbu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajuluchukwu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uneke, C. J.</span><span> </span><span class="NLM_article-title">Lassa fever in West African sub-region: an overview</span> <span class="citation_source-journal">J. Vector Borne Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17378212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD2s7nslerug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2007&pages=1-11&author=O.+Ogbuauthor=E.+Ajuluchukwuauthor=C.+J.+Uneke&title=Lassa+fever+in+West+African+sub-region%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lassa fever in West African sub-region: an overview</span></div><div class="casAuthors">Ogbu O; Ajuluchukwu E; Uneke C J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of vector borne diseases</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>
        ISSN:<span class="NLM_cas:issn">0972-9062</span>.
    </div><div class="casAbstract">Lassa fever is an acute viral zoonotic illness caused by Lassa virus, an arenavirus known to be responsible for a severe haemorrhagic fever characterised by fever, muscle aches, sore throat, nausea, vomiting and, chest and abdominal pain.  The virus exhibits persistent, asymptomatic infection with profuse urinary virus excretion in the ubiquitous rodent vector, Mastomys natalensis.  Lassa fever is endemic in West Africa and has been reported from Sierra Leone, Guinea, Liberia, and Nigeria.  Some studies indicate that 300,000 to 500,000 cases of Lassa fever and 5000 deaths occur yearly across West Africa.  Studies reported in English, that investigated Lassa fever with reference to West Africa were identified using the Medline Entrez-PubMed search and were used for this review.  The scarcity of resources available for health care delivery system and the political instability that characterise the West African countries would continue to impede efforts for the control of Lassa fever in the sub-region.  There is need for adequate training of health care workers regarding diagnostics, intensive care of patients under isolation, contact tracing, adequate precautionary measures in handling infectious laboratory specimens, control of the vector as well as care and disposal of infectious waste.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRW5RJYcP5UICO2kE7krGOBfW6udTcc2eYHQMIrkMKvP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7nslerug%253D%253D&md5=a9f0f345352f31f279532f4445eb52c5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOgbu%26aufirst%3DO.%26aulast%3DAjuluchukwu%26aufirst%3DE.%26aulast%3DUneke%26aufirst%3DC.%2BJ.%26atitle%3DLassa%2520fever%2520in%2520West%2520African%2520sub-region%253A%2520an%2520overview%26jtitle%3DJ.%2520Vector%2520Borne%2520Dis.%26date%3D2007%26volume%3D44%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">3′ exoribonucleolytic trimming is a common feature of the maturation of small, stable RNAs in Escherichia coli</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2856</span><span class="NLM_x">–</span> <span class="NLM_lpage">2861</span><span class="refDoi"> DOI: 10.1073/pnas.95.6.2856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.95.6.2856" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=2856-2861&author=Z.+Liauthor=S.+Panditauthor=M.+P.+Deutscher&title=3%E2%80%B2+exoribonucleolytic+trimming+is+a+common+feature+of+the+maturation+of+small%2C+stable+RNAs+in+Escherichia+coli&doi=10.1073%2Fpnas.95.6.2856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.6.2856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.6.2856%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPandit%26aufirst%3DS.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3D3%25E2%2580%25B2%2520exoribonucleolytic%2520trimming%2520is%2520a%2520common%2520feature%2520of%2520the%2520maturation%2520of%2520small%252C%2520stable%2520RNAs%2520in%2520Escherichia%2520coli%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D2856%26epage%3D2861%26doi%3D10.1073%2Fpnas.95.6.2856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">Maturation of 23S ribosomal RNA requires the exoribonuclease RNase T</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1017/S1355838299981669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1017%2FS1355838299981669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=139-146&author=Z.+Liauthor=S.+Panditauthor=M.+P.+Deutscher&title=Maturation+of+23S+ribosomal+RNA+requires+the+exoribonuclease+RNase+T&doi=10.1017%2FS1355838299981669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1017%2FS1355838299981669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1355838299981669%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPandit%26aufirst%3DS.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DMaturation%2520of%252023S%2520ribosomal%2520RNA%2520requires%2520the%2520exoribonuclease%2520RNase%2520T%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D139%26epage%3D146%26doi%3D10.1017%2FS1355838299981669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Structural insights into DNA repair by RNase T--an exonuclease processing 3′ end of structured DNA in repair pathways</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001803</span><span class="refDoi"> DOI: 10.1371/journal.pbio.1001803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pbio.1001803" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=e1001803&author=Y.+Y.+Hsiaoauthor=W.+H.+Fangauthor=C.+C.+Leeauthor=Y.+P.+Chenauthor=H.+S.+Yuan&title=Structural+insights+into+DNA+repair+by+RNase+T%2D%2Dan+exonuclease+processing+3%E2%80%B2+end+of+structured+DNA+in+repair+pathways&doi=10.1371%2Fjournal.pbio.1001803"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1001803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1001803%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DFang%26aufirst%3DW.%2BH.%26aulast%3DLee%26aufirst%3DC.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BP.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DStructural%2520insights%2520into%2520DNA%2520repair%2520by%2520RNase%2520T--an%2520exonuclease%2520processing%25203%25E2%2580%25B2%2520end%2520of%2520structured%2520DNA%2520in%2520repair%2520pathways%26jtitle%3DPLoS%2520Biol.%26date%3D2014%26volume%3D12%26spage%3De1001803%26doi%3D10.1371%2Fjournal.pbio.1001803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Parrish, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, D.</span><span> </span><span class="NLM_article-title">Functional genomic analysis of apoptotic DNA degradation in C. elegans</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">996</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00095-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS1097-2765%2803%2900095-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=987-996&author=J.+Z.+Parrishauthor=D.+Xue&title=Functional+genomic+analysis+of+apoptotic+DNA+degradation+in+C.+elegans&doi=10.1016%2FS1097-2765%2803%2900095-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900095-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900095-9%26sid%3Dliteratum%253Aachs%26aulast%3DParrish%26aufirst%3DJ.%2BZ.%26aulast%3DXue%26aufirst%3DD.%26atitle%3DFunctional%2520genomic%2520analysis%2520of%2520apoptotic%2520DNA%2520degradation%2520in%2520C.%2520elegans%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D11%26spage%3D987%26epage%3D996%26doi%3D10.1016%2FS1097-2765%2803%2900095-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Nurmohamed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Titman, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandran, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pears, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luisi, B. F.</span><span> </span><span class="NLM_article-title">Polynucleotide phosphorylase activity may be modulated by metabolites in escherichia coli</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">14315</span><span class="NLM_x">–</span> <span class="NLM_lpage">14323</span><span class="refDoi"> DOI: 10.1074/jbc.M110.200741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M110.200741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=14315-14323&author=S.+Nurmohamedauthor=H.+A.+Vincentauthor=C.+M.+Titmanauthor=V.+Chandranauthor=M.+R.+Pearsauthor=D.+J.+Duauthor=J.+L.+Griffinauthor=A.+J.+Callaghanauthor=B.+F.+Luisi&title=Polynucleotide+phosphorylase+activity+may+be+modulated+by+metabolites+in+escherichia+coli&doi=10.1074%2Fjbc.M110.200741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.200741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.200741%26sid%3Dliteratum%253Aachs%26aulast%3DNurmohamed%26aufirst%3DS.%26aulast%3DVincent%26aufirst%3DH.%2BA.%26aulast%3DTitman%26aufirst%3DC.%2BM.%26aulast%3DChandran%26aufirst%3DV.%26aulast%3DPears%26aufirst%3DM.%2BR.%26aulast%3DDu%26aufirst%3DD.%2BJ.%26aulast%3DGriffin%26aufirst%3DJ.%2BL.%26aulast%3DCallaghan%26aufirst%3DA.%2BJ.%26aulast%3DLuisi%26aufirst%3DB.%2BF.%26atitle%3DPolynucleotide%2520phosphorylase%2520activity%2520may%2520be%2520modulated%2520by%2520metabolites%2520in%2520escherichia%2520coli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D14315%26epage%3D14323%26doi%3D10.1074%2Fjbc.M110.200741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Song, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span> </span><span class="NLM_article-title">Characterization of a novel non-specific nuclease from thermophilic bacteriophage GBSV1</span> <span class="citation_source-journal">BMC Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="refDoi"> DOI: 10.1186/1472-6750-8-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1186%2F1472-6750-8-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18439318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1czivVCguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=43&author=Q.+Songauthor=X.+Zhang&title=Characterization+of+a+novel+non-specific+nuclease+from+thermophilic+bacteriophage+GBSV1&doi=10.1186%2F1472-6750-8-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel non-specific nuclease from thermophilic bacteriophage GBSV1</span></div><div class="casAuthors">Song Qing; Zhang Xiaobo</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Thermostable enzymes from thermophiles have attracted extensive studies.  In this investigation, a nuclease-encoding gene (designated as GBSV1-NSN) was obtained from a thermophilic bacteriophage GBSV1 for the first time.  RESULTS:  After recombinant expression in Escherichia coli, the purified GBSV1-NSN exhibited non-specific nuclease activity, being able to degrade various nucleic acids, including RNA, single-stranded DNA and double-stranded DNA that was circular or linear.  Based on sequence analysis, the nuclease shared no homology with any known nucleases, suggesting that it was a novel nuclease.  The characterization of the recombinant GBSV1-NSN showed that its optimal temperature and pH were 60 degrees C and 7.5, respectively.  The results indicated that the enzymatic activity was inhibited by enzyme inhibitors or detergents, such as ethylene diamine tetraacetic acid, citrate, dithiothreitol, beta-mercaptoethanol, guanidine hydrochloride, urea and SDS.  In contrast, the nuclease activity was enhanced by TritonX-100, Tween-20 or chaps to approximately 124.5% - 141.6%.  The Km of GBSV1-NSN nuclease was 231, 61 and 92 microM, while its kcat was 1278, 241 and 300 s-1 for the cleavage of dsDNA, ssDNA and RNA, respectively.  CONCLUSION:  Our study, therefore, presented a novel thermostable non-specific nuclease from thermophilic bacteriophage and its overexpression and purification for scientific research and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSC9vQdvXAByxe4jJGKT5N0fW6udTcc2ebApMj1XWhrVbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czivVCguw%253D%253D&md5=27616f53f271951fe9c8c188a3eeb194</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-8-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-8-43%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520novel%2520non-specific%2520nuclease%2520from%2520thermophilic%2520bacteriophage%2520GBSV1%26jtitle%3DBMC%2520Biotechnol.%26date%3D2008%26volume%3D8%26spage%3D43%26doi%3D10.1186%2F1472-6750-8-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stauffacher, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Etten, R. L.</span><span> </span><span class="NLM_article-title">Crystal structure of a human low molecular weight phosphotyrosyl phosphatase. Implications for substrate specificity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">21714</span><span class="NLM_x">–</span> <span class="NLM_lpage">21720</span><span class="refDoi"> DOI: 10.1074/jbc.273.34.21714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.273.34.21714" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=21714-21720&author=M.+Zhangauthor=C.+V.+Stauffacherauthor=D.+Linauthor=R.+L.+Van+Etten&title=Crystal+structure+of+a+human+low+molecular+weight+phosphotyrosyl+phosphatase.+Implications+for+substrate+specificity&doi=10.1074%2Fjbc.273.34.21714"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.34.21714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.34.21714%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DStauffacher%26aufirst%3DC.%2BV.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BL.%26atitle%3DCrystal%2520structure%2520of%2520a%2520human%2520low%2520molecular%2520weight%2520phosphotyrosyl%2520phosphatase.%2520Implications%2520for%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D21714%26epage%3D21720%26doi%3D10.1074%2Fjbc.273.34.21714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Smith, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soellner, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raines, R. T.</span><span> </span><span class="NLM_article-title">Potent inhibition of ribonuclease A by oligo(vinylsulfonic acid)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">20934</span><span class="NLM_x">–</span> <span class="NLM_lpage">20938</span><span class="refDoi"> DOI: 10.1074/jbc.M301852200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M301852200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12649287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=20934-20938&author=B.+D.+Smithauthor=M.+B.+Soellnerauthor=R.+T.+Raines&title=Potent+inhibition+of+ribonuclease+A+by+oligo%28vinylsulfonic+acid%29&doi=10.1074%2Fjbc.M301852200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Inhibition of Ribonuclease A by Oligo(vinylsulfonic Acid)</span></div><div class="casAuthors">Smith, Bryan D.; Soellner, Matthew B.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">20934-20938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">RNase A (RNase A) can make multiple contacts with an RNA substrate.  In particular, the enzymic active site and adjacent subsites bind sequential phosphoryl groups in the RNA backbone through Coulombic interactions.  Here, oligomers of vinylsulfonic acid (OVS) are shown to be potent inhibitors of RNase A that exploit these interactions.  Inhibition is competitive with substrate and has Ki = 11 pM in assays at low salt concn.  The effect of salt concn. on inhibition indicates that nearly eight favorable Coulombic interactions occur in the RNase A·OVS complex.  The phosphonic acid and sulfuric acid analogs of OVS are also potent inhibitors although slightly less effective.  OVS is also shown to be a contaminant of MES and other buffers that contain sulfonylethyl groups.  Oligomers greater than nine units in length can be isolated from com. MES buffer.  Inhibition by contaminating OVS is responsible for the apparent decrease in catalytic activity that has been obsd. in assays of RNase A at low salt concn.  Thus, OVS is both a useful inhibitor of RNase A and a potential bane to chemists and biochemists who use ethanesulfonic acid buffers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktvWKV_gUDrVg90H21EOLACvtfcHk0liuMjUquU6YUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wisb0%253D&md5=412479c7e592c5f4b493c963f3c26cc5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301852200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301852200%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BD.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DPotent%2520inhibition%2520of%2520ribonuclease%2520A%2520by%2520oligo%2528vinylsulfonic%2520acid%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D20934%26epage%3D20938%26doi%3D10.1074%2Fjbc.M301852200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Park, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raines, R. T.</span><span> </span><span class="NLM_article-title">Origin of the “inactivation” of ribonuclease A at low salt concentration</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1016/S0014-5793(00)01227-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0014-5793%2800%2901227-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2000&pages=199-202&author=C.+Parkauthor=R.+T.+Raines&title=Origin+of+the+%E2%80%9Cinactivation%E2%80%9D+of+ribonuclease+A+at+low+salt+concentration&doi=10.1016%2FS0014-5793%2800%2901227-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2800%2901227-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252800%252901227-8%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DC.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DOrigin%2520of%2520the%2520%25E2%2580%259Cinactivation%25E2%2580%259D%2520of%2520ribonuclease%2520A%2520at%2520low%2520salt%2520concentration%26jtitle%3DFEBS%2520Lett.%26date%3D2000%26volume%3D468%26spage%3D199%26epage%3D202%26doi%3D10.1016%2FS0014-5793%2800%2901227-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Sunaga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiokawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanuma, S.</span><span> </span><span class="NLM_article-title">A novel inhibitor that protects apoptotic DNA fragmentation catalyzed by DNase gamma</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1292</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2004.10.171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.bbrc.2004.10.171" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2004&pages=1292-1297&author=S.+Sunagaauthor=T.+Kobayashiauthor=A.+Yoshimoriauthor=D.+Shiokawaauthor=S.+Tanuma&title=A+novel+inhibitor+that+protects+apoptotic+DNA+fragmentation+catalyzed+by+DNase+gamma&doi=10.1016%2Fj.bbrc.2004.10.171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2004.10.171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2004.10.171%26sid%3Dliteratum%253Aachs%26aulast%3DSunaga%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DYoshimori%26aufirst%3DA.%26aulast%3DShiokawa%26aufirst%3DD.%26aulast%3DTanuma%26aufirst%3DS.%26atitle%3DA%2520novel%2520inhibitor%2520that%2520protects%2520apoptotic%2520DNA%2520fragmentation%2520catalyzed%2520by%2520DNase%2520gamma%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2004%26volume%3D325%26spage%3D1292%26epage%3D1297%26doi%3D10.1016%2Fj.bbrc.2004.10.171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Prentice, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurley, L. R.</span><span> </span><span class="NLM_article-title">Effects of phosphatase inhibitors on nuclease activity</span> <span class="citation_source-journal">Cell Biol. Int. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1016/0309-1651(85)90069-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2F0309-1651%2885%2990069-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1985&pages=1027-1034&author=D.+A.+Prenticeauthor=P.+A.+Kitosauthor=L.+R.+Gurley&title=Effects+of+phosphatase+inhibitors+on+nuclease+activity&doi=10.1016%2F0309-1651%2885%2990069-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2F0309-1651%2885%2990069-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0309-1651%252885%252990069-4%26sid%3Dliteratum%253Aachs%26aulast%3DPrentice%26aufirst%3DD.%2BA.%26aulast%3DKitos%26aufirst%3DP.%2BA.%26aulast%3DGurley%26aufirst%3DL.%2BR.%26atitle%3DEffects%2520of%2520phosphatase%2520inhibitors%2520on%2520nuclease%2520activity%26jtitle%3DCell%2520Biol.%2520Int.%2520Rep.%26date%3D1985%26volume%3D9%26spage%3D1027%26epage%3D1034%26doi%3D10.1016%2F0309-1651%2885%2990069-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Li, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">The role of individual cysteine residues in the activity of Escherichia coli RNase T</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.1074/jbc.271.2.1127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.271.2.1127" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=1127-1132&author=Z.+W.+Liauthor=L.+J.+Zhanauthor=M.+P.+Deutscher&title=The+role+of+individual+cysteine+residues+in+the+activity+of+Escherichia+coli+RNase+T&doi=10.1074%2Fjbc.271.2.1127"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.2.1127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.2.1127%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BW.%26aulast%3DZhan%26aufirst%3DL.%2BJ.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DThe%2520role%2520of%2520individual%2520cysteine%2520residues%2520in%2520the%2520activity%2520of%2520Escherichia%2520coli%2520RNase%2520T%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D1127%26epage%3D1132%26doi%3D10.1074%2Fjbc.271.2.1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Madhusudan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrimpton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houlbrook, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talbot, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammonds, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freemont, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternberg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dianov, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickson, I. D.</span><span> </span><span class="NLM_article-title">Isolation of a small molecule inhibitor of DNA base excision repair</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">4711</span><span class="NLM_x">–</span> <span class="NLM_lpage">4724</span><span class="refDoi"> DOI: 10.1093/nar/gki781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgki781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16113242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=4711-4724&author=S.+Madhusudanauthor=F.+Smartauthor=P.+Shrimptonauthor=J.+L.+Parsonsauthor=L.+Gardinerauthor=S.+Houlbrookauthor=D.+C.+Talbotauthor=T.+Hammondsauthor=P.+A.+Freemontauthor=M.+J.+Sternbergauthor=G.+L.+Dianovauthor=I.+D.+Hickson&title=Isolation+of+a+small+molecule+inhibitor+of+DNA+base+excision+repair&doi=10.1093%2Fnar%2Fgki781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a small molecule inhibitor of DNA base excision repair</span></div><div class="casAuthors">Madhusudan, Srinivasan; Smart, Fiona; Shrimpton, Paul; Parsons, Jason L.; Gardiner, Laurence; Houlbrook, Sue; Talbot, Denis C.; Hammonds, Timothy; Freemont, Paul A.; Sternberg, Michael J. E.; Dianov, Grigory L.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4711-4724</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The base excision repair (BER) pathway is essential for the removal of DNA bases damaged by alkylation or oxidn.  A key step in BER is the processing of an apurinic/apyrimidinic (AP) site intermediate by an AP endonuclease.  The major AP endonuclease in human cells (APE1, also termed HAP1 and Ref-1) accounts for >95% of the total AP endonuclease activity, and is essential for the protection of cells against the toxic effects of several classes of DNA damaging agents.  Moreover, APE1 overexpression has been linked to radio- and chemo-resistance in human tumors.  Using a newly developed high-throughput screen, several chem. inhibitors of APE1 have been isolated.  Amongst these, CRT0044876 was identified as a potent and selective APE1 inhibitor.  CRT0044876 inhibits the AP endonuclease, 3'-phosphodiesterase and 3'-phosphatase activities of APE1 at low micromolar concns., and is a specific inhibitor of the exonuclease III family of enzymes to which APE1 belongs.  At non-cytotoxic concns., CRT0044876 potentiates the cytotoxicity of several DNA base-targeting compds.  This enhancement of cytotoxicity is assocd. with an accumulation of unrepaired AP sites.  In silico modeling studies suggest that CRT0044876 binds to the active site of APE1.  These studies provide both a novel reagent for probing APE1 function in human cells, and a rational basis for the development of APE1-targeting drugs for antitumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo35mTihPN15bVg90H21EOLACvtfcHk0li0xkrYjlYdSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D&md5=01a434b054d60d670d683dc0fa38e649</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki781%26sid%3Dliteratum%253Aachs%26aulast%3DMadhusudan%26aufirst%3DS.%26aulast%3DSmart%26aufirst%3DF.%26aulast%3DShrimpton%26aufirst%3DP.%26aulast%3DParsons%26aufirst%3DJ.%2BL.%26aulast%3DGardiner%26aufirst%3DL.%26aulast%3DHoulbrook%26aufirst%3DS.%26aulast%3DTalbot%26aufirst%3DD.%2BC.%26aulast%3DHammonds%26aufirst%3DT.%26aulast%3DFreemont%26aufirst%3DP.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DDianov%26aufirst%3DG.%2BL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DIsolation%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520DNA%2520base%2520excision%2520repair%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D4711%26epage%3D4724%26doi%3D10.1093%2Fnar%2Fgki781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Raffoul, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heydari, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillman, G. G.</span><span> </span><span class="NLM_article-title">DNA repair and cancer therapy: targeting APE1/Ref-1 using dietary agents</span> <span class="citation_source-journal">J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">370481</span><span class="refDoi"> DOI: 10.1155/2012/370481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1155%2F2012%2F370481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22997517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BC38bovVOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=370481&author=J.+J.+Raffoulauthor=A.+R.+Heydariauthor=G.+G.+Hillman&title=DNA+repair+and+cancer+therapy%3A+targeting+APE1%2FRef-1+using+dietary+agents&doi=10.1155%2F2012%2F370481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Repair and Cancer Therapy: Targeting APE1/Ref-1 Using Dietary Agents</span></div><div class="casAuthors">Raffoul Julian J; Heydari Ahmad R; Hillman Gilda G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">370481</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epidemiological studies have demonstrated the cancer protective effects of dietary agents and other natural compounds isolated from fruits, soybeans, and vegetables on neoplasia.  Studies have also revealed the potential for these natural products to be combined with chemotherapy or radiotherapy for the more effective treatment of cancer.  In this paper we discuss the potential for targeting the DNA base excision repair enzyme APE1/Ref-1 using dietary agents such as soy isoflavones, resveratrol, curcumin, and the vitamins ascorbate and α-tocopherol.  We also discuss the potential role of soy isoflavones in sensitizing cancer cells to the effects of radiotherapy.  A comprehensive review of the dual nature of APE1/Ref-1 in DNA repair and redox activation of cellular transcription factors, NF-κB and HIF-1α, is also discussed.  Further research efforts dedicated to delineating the role of APE1/Ref-1 DNA repair versus redox activity in sensitizing cancer cells to conventional treatment are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQfkGkuJsup8NIRw0GKUoEfW6udTcc2ea2sIFwGcdz9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38bovVOltw%253D%253D&md5=5f5ee8ee8ef54e5484f38cf042a13b02</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1155%2F2012%2F370481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F370481%26sid%3Dliteratum%253Aachs%26aulast%3DRaffoul%26aufirst%3DJ.%2BJ.%26aulast%3DHeydari%26aufirst%3DA.%2BR.%26aulast%3DHillman%26aufirst%3DG.%2BG.%26atitle%3DDNA%2520repair%2520and%2520cancer%2520therapy%253A%2520targeting%2520APE1%252FRef-1%2520using%2520dietary%2520agents%26jtitle%3DJ.%2520Oncol.%26date%3D2012%26volume%3D2012%26spage%3D370481%26doi%3D10.1155%2F2012%2F370481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, M. R.</span><span> </span><span class="NLM_article-title">The DNA base excision repair protein Ape1/Ref-1 as a therapeutic and chemopreventive target</span> <span class="citation_source-journal">Mol. Aspects Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1016/j.mam.2007.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.mam.2007.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17560642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvF2ktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=375-395&author=M.+L.+Fishelauthor=M.+R.+Kelley&title=The+DNA+base+excision+repair+protein+Ape1%2FRef-1+as+a+therapeutic+and+chemopreventive+target&doi=10.1016%2Fj.mam.2007.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA base excision repair protein Ape1/Ref-1 as a therapeutic and chemopreventive target</span></div><div class="casAuthors">Fishel, Melissa L.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">375-395</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With our growing understanding of the pathways involved in cell proliferation and signaling, targeted therapies, in the treatment of cancer are entering the clin. arena.  New and emerging targets are proteins involved in DNA repair pathways.  Inhibition of various proteins in the DNA repair pathways sensitizes cancer cells to DNA damaging agents such as chemotherapy and/or radiation.  We study the apurinic endonuclease 1/redox factor-1 (Ape1/Ref-1) and believe that its crucial function in DNA repair and redn.-oxidn. or redox signaling make it an excellent target for sensitizing tumor cells to chemotherapy.  Ape1/Ref-1 is an essential enzyme in the base excision repair (BER) pathway which is responsible for the repair of DNA caused by oxidative and alkylation damage.  As importantly, Ape1/Ref-1 also functions as a redox factor maintaining transcription factors in an active reduced state.  Ape1/Ref-1 stimulates the DNA binding activity of numerous transcription factors that are involved in cancer promotion and progression such as AP-1 (Fos/Jun), NFκB, HIF-1α, CREB, p53 and others.  We will discuss what is known regarding the pharmacol. targeting of the DNA repair activity, as well as the redox activity of Ape1/Ref-1, and explore the budding clin. utility of inhibition of either of these functions in cancer treatment.  A brief discussion of the effect of polymorphisms in its DNA sequence is included because of Ape1/Ref-1's importance to maintenance and integrity of the genome.  Exptl. modification of Ape1/Ref-1 activity changes the response of cells and of organisms to DNA damaging agents, suggesting that Ape1/Ref-1 may also be a productive target of chemoprevention.  In this review, we will provide an overview of Ape1/Ref-1's activities and explore the potential of this protein as a target in cancer treatment as well as its role in chemoprevention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCI5d4nCXTnbVg90H21EOLACvtfcHk0li760uE0qCJRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvF2ktrk%253D&md5=b037a714cdc8bd0f1df7e762ae3d1b92</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2007.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2007.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DThe%2520DNA%2520base%2520excision%2520repair%2520protein%2520Ape1%252FRef-1%2520as%2520a%2520therapeutic%2520and%2520chemopreventive%2520target%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2007%26volume%3D28%26spage%3D375%26epage%3D395%26doi%3D10.1016%2Fj.mam.2007.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Hallick, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chelm, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, E. M.</span><span> </span><span class="NLM_article-title">Use of aurintricarboxylic acid as an inhibitor of nucleases during nucleic-acid isolation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3055</span><span class="NLM_x">–</span> <span class="NLM_lpage">3064</span><span class="refDoi"> DOI: 10.1093/nar/4.9.3055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2F4.9.3055" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1977&pages=3055-3064&author=R.+B.+Hallickauthor=B.+K.+Chelmauthor=P.+W.+Grayauthor=E.+M.+Orozco&title=Use+of+aurintricarboxylic+acid+as+an+inhibitor+of+nucleases+during+nucleic-acid+isolation&doi=10.1093%2Fnar%2F4.9.3055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F4.9.3055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F4.9.3055%26sid%3Dliteratum%253Aachs%26aulast%3DHallick%26aufirst%3DR.%2BB.%26aulast%3DChelm%26aufirst%3DB.%2BK.%26aulast%3DGray%26aufirst%3DP.%2BW.%26aulast%3DOrozco%26aufirst%3DE.%2BM.%26atitle%3DUse%2520of%2520aurintricarboxylic%2520acid%2520as%2520an%2520inhibitor%2520of%2520nucleases%2520during%2520nucleic-acid%2520isolation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1977%26volume%3D4%26spage%3D3055%26epage%3D3064%26doi%3D10.1093%2Fnar%2F4.9.3055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Cushman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Clercq, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schols, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldman, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowen, J. A.</span><span> </span><span class="NLM_article-title">Preparation and anti-HIV activities of aurintricarboxylic acid fractions and analogues: direct correlation of antiviral potency with molecular weight</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1021/jm00105a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00105a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK3MXjsFChsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=329-337&author=M.+Cushmanauthor=P.+L.+Wangauthor=S.+H.+Changauthor=C.+Wildauthor=E.+De+Clercqauthor=D.+Scholsauthor=M.+E.+Goldmanauthor=J.+A.+Bowen&title=Preparation+and+anti-HIV+activities+of+aurintricarboxylic+acid+fractions+and+analogues%3A+direct+correlation+of+antiviral+potency+with+molecular+weight&doi=10.1021%2Fjm00105a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and anti-HIV activities of aurintricarboxylic acid fractions and analogs:  direct correlation of antiviral potency with molecular weight</span></div><div class="casAuthors">Cushman, Mark; Wang, Pinglang; Chang, Steve H.; Wild, Carl; De Clercq, Erik; Schols, Dominique; Goldman, Mark E.; Bowen, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">329-37</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Aurintricarboxylic acid (ATA) was fractionated by a combination of dialysis, ultrafiltration, and gel permeation chromatog.  The no. av. and wt. av. mol. wts. of the ATA fractions were detd. by the universal calibration method.  The sulfonic acid analog of ATA was prepd. and sepd. in high and low mol. wt. fractions.  The phosphonic acid analog of ATA was also synthesized.  All of the ATA fractions were tested for prevention of the cytopathic effect of HIV-1 and HIV-2 in MT-4 cell culture as well as against HIV-1 in CEM cell culture.  The abilities of the fractions and analogs to inhibit syncytium formation between HIV-1- and HIV-2-infected HUT-78 cells and uninfected MOLT-4 cells were evaluated.  In addn., the fractions and analogs were tested for cytotoxicity in mock-infected MT-4 cells, prevention of the binding of the OKT4A monoclonal antibody to the CD4 receptor, inhibition of the binding of anti-gp 120 monoclonal antibody to gp120, inhibition of attachment of HIV-1 virions to MT-4 cells, and inhibition of HIV-1 reverse transcriptase.  In all of these assays except cytotoxicity, there was a correlation of potency with mol. wt.  The higher the mol. wt., the higher the activity.  Several of the lower mol. wt. fractions of ATA, which bound to gp120 but not to CD4, prevented HIV-1 and HIV-2 cytopathicity.  A similar profile was obsd. for the phosphonic acid analog of ATA and the lower mol. wt. fraction of the sulfonic acid analog.  The results on the ATA fractions indicate that the binding of ATA to gp120 in the absence of CD4 binding is sufficient for anti-HIV activity.  The active compds. bind more avidly to gp120 than to CD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQvZPzjBX5f7Vg90H21EOLACvtfcHk0li760uE0qCJRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXjsFChsg%253D%253D&md5=44d9335d0c1bbd6e970847a5b9b4d595</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm00105a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00105a052%26sid%3Dliteratum%253Aachs%26aulast%3DCushman%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DP.%2BL.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DWild%26aufirst%3DC.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DGoldman%26aufirst%3DM.%2BE.%26aulast%3DBowen%26aufirst%3DJ.%2BA.%26atitle%3DPreparation%2520and%2520anti-HIV%2520activities%2520of%2520aurintricarboxylic%2520acid%2520fractions%2520and%2520analogues%253A%2520direct%2520correlation%2520of%2520antiviral%2520potency%2520with%2520molecular%2520weight%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D329%26epage%3D337%26doi%3D10.1021%2Fjm00105a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Tsi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid protects against cell death caused by lipopolysaccharide in macrophages by decreasing inducible nitric-oxide synthase induction via IkappaB kinase, extracellular signal-regulated kinase, and p38 mitogen-activated protein kinase inhibition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1124/mol.62.1.90</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1124%2Fmol.62.1.90" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=90-101&author=C.+J.+Tsiauthor=Y.+Chaoauthor=C.+W.+Chenauthor=W.+W.+Lin&title=Aurintricarboxylic+acid+protects+against+cell+death+caused+by+lipopolysaccharide+in+macrophages+by+decreasing+inducible+nitric-oxide+synthase+induction+via+IkappaB+kinase%2C+extracellular+signal-regulated+kinase%2C+and+p38+mitogen-activated+protein+kinase+inhibition&doi=10.1124%2Fmol.62.1.90"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1124%2Fmol.62.1.90&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.62.1.90%26sid%3Dliteratum%253Aachs%26aulast%3DTsi%26aufirst%3DC.%2BJ.%26aulast%3DChao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DLin%26aufirst%3DW.%2BW.%26atitle%3DAurintricarboxylic%2520acid%2520protects%2520against%2520cell%2520death%2520caused%2520by%2520lipopolysaccharide%2520in%2520macrophages%2520by%2520decreasing%2520inducible%2520nitric-oxide%2520synthase%2520induction%2520via%2520IkappaB%2520kinase%252C%2520extracellular%2520signal-regulated%2520kinase%252C%2520and%2520p38%2520mitogen-activated%2520protein%2520kinase%2520inhibition%26jtitle%3DMol.%2520Pharmacol.%26date%3D2002%26volume%3D62%26spage%3D90%26epage%3D101%26doi%3D10.1124%2Fmol.62.1.90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span> </span><span class="NLM_article-title">Inhibition of cytokine-induced JAK-STAT signalling pathways by an endonuclease inhibitor aurintricarboxylic acid</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1020</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0704955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fsj.bjp.0704955" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12429573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsVantLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2002&pages=1011-1020&author=C.+W.+Chenauthor=Y.+Chaoauthor=Y.+H.+Changauthor=M.+J.+Hsuauthor=W.+W.+Lin&title=Inhibition+of+cytokine-induced+JAK-STAT+signalling+pathways+by+an+endonuclease+inhibitor+aurintricarboxylic+acid&doi=10.1038%2Fsj.bjp.0704955"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of cytokine-induced JAK-STAT signalling pathways by an endonuclease inhibitor aurintricarboxylic acid</span></div><div class="casAuthors">Chen, Ching-Wen; Chao, Yee; Chang, Ying-Hsin; Hsu, Ming-Jen; Lin, Wan-Wan</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1011-1020</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 Inducible nitric oxide (iNOS) is thought to involve in host defense and tissue damage in inflammatory loci.  In previous study, we have found that the endonuclease inhibitor aurintricarboxylic acid (ATA) can protect macrophages from cell death induced by bacterial lipopolysaccharide.  This action is through the interruption with signalling pathways for NF-κB and AP-1 activation, and thus iNOS expression.  In this study we have addressed the effects of ATA on JAK-STAT signalling pathways.  2 In murine RAW 264.7 macrophages, IFN-γ-mediated NO prodn. and iNOS expression were concn.-dependently reduced by the presence of 3-100 μM ATA.  3 IFN-γ-induced STAT1 activation, as assessed from its tyrosine phosphorylation, nuclear translocation, binding to specific DNA response element and evoked IRF-1 reporter gene assay, were concomitantly inhibited by ATA.  However, ATA did not alter IFN-γ binding to RAW 264.7 cells.  4 The activities of JAK1 and JAK2, the upstream kinases essential for STAT1 signalling in response to IFN-γ, were also reduced by ATA.  5 Moreover, IL-4, IL-10, GM-CSF and M-CSF elicited tyrosine phosphorylation of STAT3, STAT5 and/or STAT6 in macrophages were diminished by the presence of ATA.  6 Taken together, we conclude that ATA can interfere JAK-STAT signalling pathways in response to cytokines.  This action contributes to the inhibition of IFN-γ-induced iNOS expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnff4gpCUlirVg90H21EOLACvtfcHk0li5uDut4A7RIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsVantLY%253D&md5=7678f3151186ec8ecfa7cc879918350e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704955%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DChao%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DY.%2BH.%26aulast%3DHsu%26aufirst%3DM.%2BJ.%26aulast%3DLin%26aufirst%3DW.%2BW.%26atitle%3DInhibition%2520of%2520cytokine-induced%2520JAK-STAT%2520signalling%2520pathways%2520by%2520an%2520endonuclease%2520inhibitor%2520aurintricarboxylic%2520acid%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D137%26spage%3D1011%26epage%3D1020%26doi%3D10.1038%2Fsj.bjp.0704955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Hashem, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaman, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farnsworth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Domselaar, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid is a potent inhibitor of influenza A and B virus neuraminidases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e8350</span><span class="refDoi"> DOI: 10.1371/journal.pone.0008350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pone.0008350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20020057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1Mfjt1ehuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e8350&author=A.+M.+Hashemauthor=A.+S.+Flamanauthor=A.+Farnsworthauthor=E.+G.+Brownauthor=G.+Van+Domselaarauthor=R.+Heauthor=X.+Li&title=Aurintricarboxylic+acid+is+a+potent+inhibitor+of+influenza+A+and+B+virus+neuraminidases&doi=10.1371%2Fjournal.pone.0008350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aurintricarboxylic acid is a potent inhibitor of influenza A and B virus neuraminidases</span></div><div class="casAuthors">Hashem Anwar M; Flaman Anathea S; Farnsworth Aaron; Brown Earl G; Van Domselaar Gary; He Runtao; Li Xuguang</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e8350</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Influenza viruses cause serious infections that can be prevented or treated using vaccines or antiviral agents, respectively.  While vaccines are effective, they have a number of limitations, and influenza strains resistant to currently available anti-influenza drugs are increasingly isolated.  This necessitates the exploration of novel anti-influenza therapies.  METHODOLOGY/PRINCIPAL FINDINGS:  We investigated the potential of aurintricarboxylic acid (ATA), a potent inhibitor of nucleic acid processing enzymes, to protect Madin-Darby canine kidney cells from influenza infection.  We found, by neutral red assay, that ATA was protective, and by RT-PCR and ELISA, respectively, confirmed that ATA reduced viral replication and release.  Furthermore, while pre-treating cells with ATA failed to inhibit viral replication, pre-incubation of virus with ATA effectively reduced viral titers, suggesting that ATA may elicit its inhibitory effects by directly interacting with the virus.  Electron microscopy revealed that ATA induced viral aggregation at the cell surface, prompting us to determine if ATA could inhibit neuraminidase.  ATA was found to compromise the activities of virus-derived and recombinant neuraminidase.  Moreover, an oseltamivir-resistant H1N1 strain with H274Y was also found to be sensitive to ATA.  Finally, we observed additive protective value when infected cells were simultaneously treated with ATA and amantadine hydrochloride, an anti-influenza drug that inhibits M2-ion channels of influenza A virus.  CONCLUSIONS/SIGNIFICANCE:  Collectively, these data suggest that ATA is a potent anti-influenza agent by directly inhibiting the neuraminidase and could be a more effective antiviral compound when used in combination with amantadine hydrochloride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT-CVIozU9TSJagfuraLLzSfW6udTcc2eYd4Uxw3MqWp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mfjt1ehuw%253D%253D&md5=0c143587cfab62cb6636eec41dd5e307</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008350%26sid%3Dliteratum%253Aachs%26aulast%3DHashem%26aufirst%3DA.%2BM.%26aulast%3DFlaman%26aufirst%3DA.%2BS.%26aulast%3DFarnsworth%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DE.%2BG.%26aulast%3DVan%2BDomselaar%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26atitle%3DAurintricarboxylic%2520acid%2520is%2520a%2520potent%2520inhibitor%2520of%2520influenza%2520A%2520and%2520B%2520virus%2520neuraminidases%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De8350%26doi%3D10.1371%2Fjournal.pone.0008350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Hung, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid inhibits influenza virus neuraminidase</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2008.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.antiviral.2008.10.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=123-131&author=H.+C.+Hungauthor=C.+P.+Tsengauthor=J.+M.+Yangauthor=Y.+W.+Juauthor=S.+N.+Tsengauthor=Y.+F.+Chenauthor=Y.+S.+Chaoauthor=H.+P.+Hsiehauthor=S.+R.+Shihauthor=J.+T.+Hsu&title=Aurintricarboxylic+acid+inhibits+influenza+virus+neuraminidase&doi=10.1016%2Fj.antiviral.2008.10.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DH.%2BC.%26aulast%3DTseng%26aufirst%3DC.%2BP.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DJu%26aufirst%3DY.%2BW.%26aulast%3DTseng%26aufirst%3DS.%2BN.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DChao%26aufirst%3DY.%2BS.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26atitle%3DAurintricarboxylic%2520acid%2520inhibits%2520influenza%2520virus%2520neuraminidase%26jtitle%3DAntiviral%2520Res.%26date%3D2009%26volume%3D81%26spage%3D123%26epage%3D131%26doi%3D10.1016%2Fj.antiviral.2008.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Schols, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauwels, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmyter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Specific interaction of aurintricarboxylic acid with the human immunodeficiency virus/CD4 cell receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">3322</span><span class="NLM_x">–</span> <span class="NLM_lpage">3326</span><span class="refDoi"> DOI: 10.1073/pnas.86.9.3322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.86.9.3322" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=3322-3326&author=D.+Scholsauthor=M.+Babaauthor=R.+Pauwelsauthor=J.+Desmyterauthor=E.+De+Clercq&title=Specific+interaction+of+aurintricarboxylic+acid+with+the+human+immunodeficiency+virus%2FCD4+cell+receptor&doi=10.1073%2Fpnas.86.9.3322"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.9.3322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.9.3322%26sid%3Dliteratum%253Aachs%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DPauwels%26aufirst%3DR.%26aulast%3DDesmyter%26aufirst%3DJ.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DSpecific%2520interaction%2520of%2520aurintricarboxylic%2520acid%2520with%2520the%2520human%2520immunodeficiency%2520virus%252FCD4%2520cell%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D3322%26epage%3D3326%26doi%3D10.1073%2Fpnas.86.9.3322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bopda-Waffo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silberstein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talele, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik-Basu, N.</span><span> </span><span class="NLM_article-title">Characterization of aurintricarboxylic acid as a potent hepatitis C virus replicase inhibitor</span> <span class="citation_source-journal">Antivir. Chem. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.3851/IMP1286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.3851%2FIMP1286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=19794229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSlurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=19-36&author=Y.+Chenauthor=A.+Bopda-Waffoauthor=A.+Basuauthor=R.+Krishnanauthor=E.+Silbersteinauthor=D.+R.+Taylorauthor=T.+T.+Taleleauthor=P.+Aroraauthor=N.+Kaushik-Basu&title=Characterization+of+aurintricarboxylic+acid+as+a+potent+hepatitis+C+virus+replicase+inhibitor&doi=10.3851%2FIMP1286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of aurintricarboxylic acid as a potent hepatitis C virus replicase inhibitor</span></div><div class="casAuthors">Chen, Ye; Bopda-Waffo, Alain; Basu, Amartya; Krishnan, Ramalingam; Silberstein, Erica; Taylor, Deborah R.; Talele, Tanaji; Arora, Payal; Kaushik-Basu, Neerja</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Chemistry & Chemotherapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-36</span>CODEN:
                <span class="NLM_cas:coden">ACCHEH</span>;
        ISSN:<span class="NLM_cas:issn">0956-3202</span>.
    
            (<span class="NLM_cas:orgname">International Medical Press, Ltd.</span>)
        </div><div class="casAbstract">Background: Hepatitis C virus (HCV) NS5B is an essential component of the viral replication machinery and an important target for antiviral intervention.  Aurintricarboxylic acid (ATA), a broad-spectrum antiviral agent, was evaluated and characterized for its anti-NS5B activity in vitro and in HCV replicon cells.  Methods: Recombinant NS5B, HCV replicase and Huh-7 cells harboring the subgenomic HCV replicon of genotype 1b were employed for biochem. and mechanistic investigations.  Results: Anal. of ATA activity in vitro yielded equipotent inhibition of recombinant NS5B and HCV replicase in the submicromolar range (50% inhibition concn. [IC50] approx. 150 nM).  Biochem. and mechanistic studies revealed a bimodal mechanism of ATA inhibition with characteristics of pyrophosphate mimics and non-nucleoside inhibitors.  Mol. modeling and competition displacement studies were consistent with these parameters, suggesting that ATA might bind to the benzothiadiazine allosteric pocket 3 of NS5B or at its catalytic center.  Kinetic studies revealed a mixed mode of ATA inhibition with respect to both RNA and UTP substrates.  Under single-cycle assay conditions, ATA inhibited HCV NS5B initiation and elongation from pre-bound RNA, but with ≥fivefold decreased potency compared with continuous polymn. conditions.  The IC50 value of ATA for the native replicase complex was 145 nM.  In HCV replicon cells, ATA treatment ablated HCV RNA replication (50% effective concn. = 75 nM) with concomitant decrease in NS5B expression and no apparent cytotoxic effects.  Conclusions: This study identified ATA as a potent anti-NS5B inhibitor and suggests that its unique mode of action might be exploited for structural refinement and development of novel anti-NS5B agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_z7hctkbzBLVg90H21EOLACvtfcHk0lhHfv2SYE_2Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSlurjJ&md5=985e30923e99a6992e1b041e8c4065ae</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.3851%2FIMP1286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3851%252FIMP1286%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBopda-Waffo%26aufirst%3DA.%26aulast%3DBasu%26aufirst%3DA.%26aulast%3DKrishnan%26aufirst%3DR.%26aulast%3DSilberstein%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DD.%2BR.%26aulast%3DTalele%26aufirst%3DT.%2BT.%26aulast%3DArora%26aufirst%3DP.%26aulast%3DKaushik-Basu%26aufirst%3DN.%26atitle%3DCharacterization%2520of%2520aurintricarboxylic%2520acid%2520as%2520a%2520potent%2520hepatitis%2520C%2520virus%2520replicase%2520inhibitor%26jtitle%3DAntivir.%2520Chem.%2520Chemother.%26date%3D2009%26volume%3D20%26spage%3D19%26epage%3D36%26doi%3D10.3851%2FIMP1286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traykova-Adonova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutts, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grudesky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschambaul, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drebot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span> </span><span class="NLM_article-title">Potent and selective inhibition of SARS coronavirus replication by aurintricarboxylic acid</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1203</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2004.06.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.bbrc.2004.06.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2004&pages=1199-1203&author=R.+Heauthor=A.+Adonovauthor=M.+Traykova-Adonovaauthor=J.+Caoauthor=T.+Cuttsauthor=E.+Grudeskyauthor=Y.+Deschambaulauthor=J.+Berryauthor=M.+Drebotauthor=X.+Li&title=Potent+and+selective+inhibition+of+SARS+coronavirus+replication+by+aurintricarboxylic+acid&doi=10.1016%2Fj.bbrc.2004.06.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2004.06.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2004.06.076%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DR.%26aulast%3DAdonov%26aufirst%3DA.%26aulast%3DTraykova-Adonova%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCutts%26aufirst%3DT.%26aulast%3DGrudesky%26aufirst%3DE.%26aulast%3DDeschambaul%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DJ.%26aulast%3DDrebot%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26atitle%3DPotent%2520and%2520selective%2520inhibition%2520of%2520SARS%2520coronavirus%2520replication%2520by%2520aurintricarboxylic%2520acid%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2004%26volume%3D320%26spage%3D1199%26epage%3D1203%26doi%3D10.1016%2Fj.bbrc.2004.06.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Myskiw, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschambault, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid inhibits the early stage of vaccinia virus replication by targeting both cellular and viral factors</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">3027</span><span class="NLM_x">–</span> <span class="NLM_lpage">3032</span><span class="refDoi"> DOI: 10.1128/JVI.02531-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1128%2FJVI.02531-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17192307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=3027-3032&author=C.+Myskiwauthor=Y.+Deschambaultauthor=K.+Jefferiesauthor=R.+Heauthor=J.+Cao&title=Aurintricarboxylic+acid+inhibits+the+early+stage+of+vaccinia+virus+replication+by+targeting+both+cellular+and+viral+factors&doi=10.1128%2FJVI.02531-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Aurintricarboxylic acid inhibits the early stage of vaccinia virus replication by targeting both cellular and viral factors</span></div><div class="casAuthors">Myskiw, Chad; Deschambault, Yvon; Jefferies, Kristel; He, Runtao; Cao, Jingxin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3027-3032</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Aurintricarboxylic acid (ATA) has been shown to inhibit the replication of viruses from several different families, including human immunodeficiency virus, vesicular stomatitis virus, and the coronavirus causing severe acute respiratory syndrome.  This study characterizes the inhibitory effect of ATA on vaccinia virus replication in HeLa, Huh7, and AD293 cells.  Vaccinia virus replication is significantly abrogated upon ATA treatment, which is assocd. with the inhibition of early viral gene transcription.  This inhibitory effect may be attributed to two findings.  First, ATA blocks the phosphorylation of extracellular signal-regulated kinase 1/2, an event shown to be essential for vaccinia virus replication.  Second, ATA inhibits the phosphatase activity of the viral enzyme H1L, which is required to initiate viral transcription.  Thus, ATA inhibits vaccinia virus replication by targeting both cellular and viral factors essential for the early stage of replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq6eG_fwi1mLVg90H21EOLACvtfcHk0ljD0JP7WILFcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GrtLc%253D&md5=198029b1652f9388e88bc027a37fabed</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FJVI.02531-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02531-06%26sid%3Dliteratum%253Aachs%26aulast%3DMyskiw%26aufirst%3DC.%26aulast%3DDeschambault%26aufirst%3DY.%26aulast%3DJefferies%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DJ.%26atitle%3DAurintricarboxylic%2520acid%2520inhibits%2520the%2520early%2520stage%2520of%2520vaccinia%2520virus%2520replication%2520by%2520targeting%2520both%2520cellular%2520and%2520viral%2520factors%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D3027%26epage%3D3032%26doi%3D10.1128%2FJVI.02531-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Yap, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span> </span><span class="NLM_article-title">Structural analysis of inhibition mechanisms of aurintricarboxylic acid on SARS-CoV polymerase and other proteins</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2005.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.compbiolchem.2005.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15979041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsVeqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2005&pages=212-219&author=Y.+Yapauthor=X.+Zhangauthor=A.+Andonovauthor=R.+He&title=Structural+analysis+of+inhibition+mechanisms+of+aurintricarboxylic+acid+on+SARS-CoV+polymerase+and+other+proteins&doi=10.1016%2Fj.compbiolchem.2005.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of inhibition mechanisms of Aurintricarboxylic Acid on SARS-CoV polymerase and other proteins</span></div><div class="casAuthors">Yap, YeeLeng; Zhang, XueWu; Andonov, Anton; He, RunTao</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">212-219</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The authors recently published exptl. results that indicated Aurintricarboxylic Acid (ATA) could selectively inhibit SARS-Co-V replication inside host cells by greater than 1000 times.  This inhibition suggested that ATA could be developed as potent antiviral drug.  Here, to extend our exptl. observation, the authors have incorporated protein structural studies (with pos./neg. controls) to investigate the potential binding modes/sites of ATA onto RNA-dependent RNA polymerase (RdRp) from SARS-Co-V and other pathogenic pos.-strand RNA-viruses, as well as other proteins in SARS-Co-V based on the fact that ATA binds to Ca2+-activated neutral protease (m-calpain), the protein tyrosine phosphatase (PTP) and HIV integrase which have existing crystal structures.  Eight regions with homologous 3D-conformation were derived for 10 proteins of interest.  One of the region, Rbinding (754-766 in SARS-Co-V's RdRp), located in the palm sub-domain mainly constituted of antiparallel β-strand-turn-β-strand hairpin structures that covers two of the three RdRp catalytic sites (Asp 760, Asp761), was also predicted by mol. docking method (based on free energy of binding ΔG) to be important binding motif recognized by ATA.  The existence of this strictly conserved region that incorporated catalytic residues, coupled with the homologous ATA binding pockets and their consistent ΔG values, suggested strongly ATA may be involved in an analogous inhibition mechanism of SARS-COV's RdRp in concomitant to the case in m-calpain, PTP and HIV integrase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXw0p-RxaEGbVg90H21EOLACvtfcHk0ljD0JP7WILFcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsVeqtrg%253D&md5=36b9b416ce9c4405bba137bc52c7923f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2005.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2005.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DAndonov%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DR.%26atitle%3DStructural%2520analysis%2520of%2520inhibition%2520mechanisms%2520of%2520aurintricarboxylic%2520acid%2520on%2520SARS-CoV%2520polymerase%2520and%2520other%2520proteins%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D29%26spage%3D212%26epage%3D219%26doi%3D10.1016%2Fj.compbiolchem.2005.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Shadrick, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frick, D. N.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid modulates the affinity of hepatitis C virus NS3 helicase for both nucleic acid and ATP</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6151</span><span class="NLM_x">–</span> <span class="NLM_lpage">6159</span><span class="refDoi"> DOI: 10.1021/bi4006495</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4006495" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6151-6159&author=W.+R.+Shadrickauthor=S.+Mukherjeeauthor=A.+M.+Hansonauthor=N.+L.+Sweeneyauthor=D.+N.+Frick&title=Aurintricarboxylic+acid+modulates+the+affinity+of+hepatitis+C+virus+NS3+helicase+for+both+nucleic+acid+and+ATP&doi=10.1021%2Fbi4006495"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi4006495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4006495%26sid%3Dliteratum%253Aachs%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BM.%26aulast%3DSweeney%26aufirst%3DN.%2BL.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26atitle%3DAurintricarboxylic%2520acid%2520modulates%2520the%2520affinity%2520of%2520hepatitis%2520C%2520virus%2520NS3%2520helicase%2520for%2520both%2520nucleic%2520acid%2520and%2520ATP%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D6151%26epage%3D6159%26doi%3D10.1021%2Fbi4006495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">iGEMDOCK: a graphical environment of enhancing GEMDOCK using pharmacological interactions and post-screening analysis</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S33</span><span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1186%2F1471-2105-12-S1-S33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21342564" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&issue=Suppl.+1&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+a+graphical+environment+of+enhancing+GEMDOCK+using+pharmacological+interactions+and+post-screening+analysis&doi=10.1186%2F1471-2105-12-S1-S33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S33%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520a%2520graphical%2520environment%2520of%2520enhancing%2520GEMDOCK%2520using%2520pharmacological%2520interactions%2520and%2520post-screening%2520analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS33%26doi%3D10.1186%2F1471-2105-12-S1-S33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Structural basis for RNA trimming by RNase T in stable RNA 3′-end maturation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1038/nchembio.524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnchembio.524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21317904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=236-243&author=Y.+Y.+Hsiaoauthor=C.+C.+Yangauthor=C.+L.+Linauthor=J.+L.+Linauthor=Y.+Duhauthor=H.+S.+Yuan&title=Structural+basis+for+RNA+trimming+by+RNase+T+in+stable+RNA+3%E2%80%B2-end+maturation&doi=10.1038%2Fnchembio.524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for RNA trimming by RNase T in stable RNA 3'-end maturation</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Yang, Che-Chuan; Lin, Chia-Liang; Lin, Jason L.-J.; Duh, Yulander; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">236-243</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA maturation relies on various exonucleases to remove nucleotides successively from the 5' or 3' end of nucleic acids.  However, little is known regarding the mol. basis for substrate and cleavage preference of exonucleases.  Our biochem. and structural analyses on RNase T-DNA complexes show that the RNase T dimer has an ideal architecture for binding a duplex with a short 3' overhang to produce a digestion product of a duplex with a 2-nucleotide (nt) or 1-nt 3' overhang, depending on the compn. of the last base pair in the duplex.  A 'C-filter' in RNase T screens out the nucleic acids with 3'-terminal cytosines for hydrolysis by inducing a disruptive conformational change at the active site.  Our results reveal the general principles and the working mechanism for the final trimming step made by RNase T in the maturation of stable RNA and pave the way for the understanding of other DEDD family exonucleases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3kWl8g05MgLVg90H21EOLACvtfcHk0ljD0JP7WILFcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtLY%253D&md5=b8e87a61752441f3313028889090485f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.524%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DJ.%2BL.%26aulast%3DDuh%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DStructural%2520basis%2520for%2520RNA%2520trimming%2520by%2520RNase%2520T%2520in%2520stable%2520RNA%25203%25E2%2580%25B2-end%2520maturation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D236%26epage%3D243%26doi%3D10.1038%2Fnchembio.524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">de Silva, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrino, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, T.</span><span> </span><span class="NLM_article-title">The crystal structure of TREX1 explains the 3′ nucleotide specificity and reveals a polyproline II helix for protein partnering</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">10537</span><span class="NLM_x">–</span> <span class="NLM_lpage">10543</span><span class="refDoi"> DOI: 10.1074/jbc.M700039200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M700039200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=10537-10543&author=U.+de+Silvaauthor=S.+Choudhuryauthor=S.+L.+Baileyauthor=S.+Harveyauthor=F.+W.+Perrinoauthor=T.+Hollis&title=The+crystal+structure+of+TREX1+explains+the+3%E2%80%B2+nucleotide+specificity+and+reveals+a+polyproline+II+helix+for+protein+partnering&doi=10.1074%2Fjbc.M700039200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M700039200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M700039200%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSilva%26aufirst%3DU.%26aulast%3DChoudhury%26aufirst%3DS.%26aulast%3DBailey%26aufirst%3DS.%2BL.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DHollis%26aufirst%3DT.%26atitle%3DThe%2520crystal%2520structure%2520of%2520TREX1%2520explains%2520the%25203%25E2%2580%25B2%2520nucleotide%2520specificity%2520and%2520reveals%2520a%2520polyproline%2520II%2520helix%2520for%2520protein%2520partnering%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D10537%26epage%3D10543%26doi%3D10.1074%2Fjbc.M700039200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Strategies in the design of antiviral drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1038/nrd703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnrd703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12119605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1Wqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=13-25&author=E.+De+Clercq&title=Strategies+in+the+design+of+antiviral+drugs&doi=10.1038%2Fnrd703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies in the design of antiviral drugs</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-25</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A decade ago, just five drugs were licensed for the treatment of viral infections.  Since then, greater understanding of viral life cycles, prompted in particular by the need to combat human immunodeficiency virus, has resulted in the discovery and validation of several targets for therapeutic intervention.  Consequently, the current antiviral repertoire now includes more than 30 drugs.  But we still lack effective therapies for several viral infections, and established treatments are not always effective or well tolerated, highlighting the need for further refinement of antiviral drug design and development.  Here, I describe the rationale behind current and future drug-based strategies for combating viral infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2t4izHL0-7Vg90H21EOLACvtfcHk0ljC7cN5IpHNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhs1Wqt7s%253D&md5=8ee06ad97a7859c1eed044ce1db62fdf</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrd703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd703%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DStrategies%2520in%2520the%2520design%2520of%2520antiviral%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D13%26epage%3D25%26doi%3D10.1038%2Fnrd703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span> <span class="citation_source-journal">Int. J. Antimicrob. Agents</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">320</span><span class="refDoi"> DOI: 10.1016/j.ijantimicag.2008.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.ijantimicag.2008.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=19108994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=307-320&author=E.+De+Clercq&title=Anti-HIV+drugs%3A+25+compounds+approved+within+25+years+after+the+discovery+of+HIV&doi=10.1016%2Fj.ijantimicag.2008.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-320</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In 2008, 25 years after the human immunodeficiency virus (HIV) was discovered as the then tentative etiol. agent of acquired immune deficiency syndrome (AIDS), exactly 25 anti-HIV compds. have been formally approved for clin. use in the treatment of AIDS.  These compds. fall into six categories: nucleoside reverse transcriptase inhibitors (NRTIs: zidovudine, didanosine, zalcitabine, stavudine, lamivudine, abacavir and emtricitabine); nucleotide reverse transcriptase inhibitors (NtRTIs: tenofovir); non-nucleoside reverse transcriptase inhibitors (NNRTIs: nevirapine, delavirdine, efavirenz and etravirine); protease inhibitors (PIs: saquinavir, ritonavir, indinavir, nelfinavir, amprenavir, lopinavir, atazanavir, fosamprenavir, tipranavir and darunavir); cell entry inhibitors [fusion inhibitors (FIs: enfuvirtide) and co-receptor inhibitors (CRIs: maraviroc)]; and integrase inhibitors (INIs: raltegravir).  These compds. should be used in drug combination regimens to achieve the highest possible benefit, tolerability and compliance and to diminish the risk of resistance development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsT6dqAjvKIrVg90H21EOLACvtfcHk0ljC7cN5IpHNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D&md5=6d8263455e9c912d353ef2ea230a5e15</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2008.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2008.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DAnti-HIV%2520drugs%253A%252025%2520compounds%2520approved%2520within%252025%2520years%2520after%2520the%2520discovery%2520of%2520HIV%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2009%26volume%3D33%26spage%3D307%26epage%3D320%26doi%3D10.1016%2Fj.ijantimicag.2008.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Ison, M. G.</span><span> </span><span class="NLM_article-title">Antivirals and resistance: influenza virus</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1016/j.coviro.2011.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2011.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22440914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=563-573&author=M.+G.+Ison&title=Antivirals+and+resistance%3A+influenza+virus&doi=10.1016%2Fj.coviro.2011.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Antivirals and resistance: influenza virus</span></div><div class="casAuthors">Ison, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-573</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Influenza causes annual epidemics of respiratory viral infections are assocd. with significant morbidity and mortality.  Influenza vaccines have been shown to reduce the risk of infection and mitigate against some of the virus' sequellae.  Likewise, two classes of antivirals, the adamantanes (amantadine and rimantadine) and the neuraminidase inhibitors (laninamivir, oseltamivir, peramivir, and zanamivir) are currently approved for the prevention and treatment of influenza; several other classes of antivirals and immune modulators are also currently under investigation.  One of the greatest challenges to our armamentarium of antivirals is the emergence of resistant mutants.  In this paper, we will review the currently approved and investigational antiviral agents and the mechanisms of resistance that impact their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMhlhzdOc5lLVg90H21EOLACvtfcHk0ljC7cN5IpHNHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrfE&md5=7414bf2c9f6d902f7ba696ff3c8bf63e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2011.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2011.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DIson%26aufirst%3DM.%2BG.%26atitle%3DAntivirals%2520and%2520resistance%253A%2520influenza%2520virus%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2011%26volume%3D1%26spage%3D563%26epage%3D573%26doi%3D10.1016%2Fj.coviro.2011.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Samson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pizzorno, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abed, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boivin, G.</span><span> </span><span class="NLM_article-title">Influenza virus resistance to neuraminidase inhibitors</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">174</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2013.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.antiviral.2013.03.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=174-185&author=M.+Samsonauthor=A.+Pizzornoauthor=Y.+Abedauthor=G.+Boivin&title=Influenza+virus+resistance+to+neuraminidase+inhibitors&doi=10.1016%2Fj.antiviral.2013.03.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2013.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2013.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DSamson%26aufirst%3DM.%26aulast%3DPizzorno%26aufirst%3DA.%26aulast%3DAbed%26aufirst%3DY.%26aulast%3DBoivin%26aufirst%3DG.%26atitle%3DInfluenza%2520virus%2520resistance%2520to%2520neuraminidase%2520inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D98%26spage%3D174%26epage%3D185%26doi%3D10.1016%2Fj.antiviral.2013.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Hosseinipour, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Zyl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachega, J. B.</span><span> </span><span class="NLM_article-title">Emergence of HIV drug resistance during first- and second-line antiretroviral therapy in resource-limited settings</span> <span class="citation_source-journal">J. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">S49</span><span class="NLM_x">–</span> <span class="NLM_lpage">S56</span><span class="refDoi"> DOI: 10.1093/infdis/jit107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Finfdis%2Fjit107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23687289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotlOgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2013&pages=S49-S56&issue=Suppl.+2&author=M.+C.+Hosseinipourauthor=R.+K.+Guptaauthor=G.+Van+Zylauthor=J.+J.+Eronauthor=J.+B.+Nachega&title=Emergence+of+HIV+drug+resistance+during+first-+and+second-line+antiretroviral+therapy+in+resource-limited+settings&doi=10.1093%2Finfdis%2Fjit107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of HIV Drug Resistance During First- and Second-Line Antiretroviral Therapy in Resource-Limited Settings</span></div><div class="casAuthors">Hosseinipour, Mina C.; Gupta, Ravindra K.; Van Zyl, Gert; Eron, Joseph J.; Nachega, Jean B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S49-S56</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Introduction Antiretroviral therapy (ART) in resource-limited settings has expanded in the last decade, reaching >8 million individuals and reducing AIDS mortality and morbidity.  Continued success of ART programs will require understanding the emergence of HIV drug resistance patterns among individuals in whom treatment has failed and managing ART from both an individual and public health perspective.  We review data on the emergence of HIV drug resistance among individuals in whom first-line therapy has failed and clin. and resistance outcomes of those receiving second-line therapy in resource-limited settings.  Results Resistance surveys among patients initiating first-line nonnucleoside reverse-transcriptase inhibitor (NNRTI)-based therapy suggest that 76%-90% of living patients achieve HIV RNA suppression by 12 mo after ART initiation.  Among patients with detectable HIV RNA at 12 mo, HIV drug resistance, primarily due to M184V and NNRTI mutations, has been identified in 60%-72%, although the antiretroviral activity of proposed second-line regimens has been preserved.  Complex mutation patterns, including thymidine-analog mutations, K65R, and multinucleoside mutations, are prevalent among cases of treatment failure identified by clin. or immunol. methods.  Approx. 22% of patients receiving second-line therapy do not achieve HIV RNA suppression by 6 mo, with poor adherence, rather than HIV drug resistance, driving most failures.  Major protease inhibitor resistance at the time of second-line failure ranges from 0% to 50%, but studies are limited.  Conclusions Resistance of HIV to first-line therapy is predictable at 12 mo when evaluated by means of HIV RNA monitoring and, when detected, largely preserves second-line therapy options.  Optimizing adherence, performing resistance surveillance, and improving treatment monitoring are crit. for long-term prevention of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHVDfsLEAp7Vg90H21EOLACvtfcHk0lj6SSBNHZfINg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotlOgtrg%253D&md5=1b90c5824f6c03db0d3156563d4f4a9b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjit107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjit107%26sid%3Dliteratum%253Aachs%26aulast%3DHosseinipour%26aufirst%3DM.%2BC.%26aulast%3DGupta%26aufirst%3DR.%2BK.%26aulast%3DVan%2BZyl%26aufirst%3DG.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26aulast%3DNachega%26aufirst%3DJ.%2BB.%26atitle%3DEmergence%2520of%2520HIV%2520drug%2520resistance%2520during%2520first-%2520and%2520second-line%2520antiretroviral%2520therapy%2520in%2520resource-limited%2520settings%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D207%26issue%3DSuppl.%25202%26spage%3DS49%26epage%3DS56%26doi%3D10.1093%2Finfdis%2Fjit107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Berkhout, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggink, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, R. W.</span><span> </span><span class="NLM_article-title">Is there a future for antiviral fusion inhibitors?</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/j.coviro.2012.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2012.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22440966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=50-59&author=B.+Berkhoutauthor=D.+Egginkauthor=R.+W.+Sanders&title=Is+there+a+future+for+antiviral+fusion+inhibitors%3F&doi=10.1016%2Fj.coviro.2012.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Is there a future for antiviral fusion inhibitors?</span></div><div class="casAuthors">Berkhout, Ben; Eggink, Dirk; Sanders, Rogier W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-59</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Entry of human immunodeficiency virus type 1 (HIV-1) into cells is mediated by attachment of the envelope glycoproteins, gp120 and gp41, to the CD4 receptor and a chemokine receptor (CCR5 or CXCR4) and subsequent fusion of viral and cellular membranes.  Several steps of the entry process can be targeted by drugs.  Receptor antagonists prevent virus attachment and fusion inhibitors block conformational changes that are required for membrane fusion.  The T20 peptide (Enfuvirtide, Fuzeon), which is homologous to part of the gp41-encoded fusion machinery, is the only clin. approved fusion inhibitor, but over the last decade new generations of T20-like peptides have been developed with improved potency and stability, as well as fusion inhibitors that target alternative gp41 domains.  Here, we will review the field of HIV-1 fusion inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUhAj7F6DFc7Vg90H21EOLACvtfcHk0lj6SSBNHZfINg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSksb4%253D&md5=a9fe1bb65fa0e440015402fce8b5d86c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2012.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2012.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DBerkhout%26aufirst%3DB.%26aulast%3DEggink%26aufirst%3DD.%26aulast%3DSanders%26aufirst%3DR.%2BW.%26atitle%3DIs%2520there%2520a%2520future%2520for%2520antiviral%2520fusion%2520inhibitors%253F%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2012%26volume%3D2%26spage%3D50%26epage%3D59%26doi%3D10.1016%2Fj.coviro.2012.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Das, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, E.</span><span> </span><span class="NLM_article-title">HIV-1 reverse transcriptase and antiviral drug resistance. Part 1</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">118</span><span class="refDoi"> DOI: 10.1016/j.coviro.2013.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2013.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23602471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsFanur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=111-118&author=K.+Dasauthor=E.+Arnold&title=HIV-1+reverse+transcriptase+and+antiviral+drug+resistance.+Part+1&doi=10.1016%2Fj.coviro.2013.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 reverse transcriptase and antiviral drug resistance. Part 1</span></div><div class="casAuthors">Das, Kalyan; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-118</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  HIV-1 reverse transcriptase (RT) contributes to the development of resistance to all anti-AIDS drugs by introducing mutations into the viral genome.  At the mol. level, mutations in RT result in resistance to RT inhibitors.  Eight nucleoside/nucleotide analogs (NRTIs) and five non-nucleoside inhibitors (NNRTIs) are approved HIV-1 drugs.  Structures of RT have been detd. in complexes with substrates and/or inhibitors, and the structures have illuminated different conformational and functional states of the enzyme.  Understanding the mol. mechanisms of resistance to NRTIs and NNRTIs, and their complex relationships, may help in designing new drugs that are periodically required to overcome existing as well as emerging trends of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKm2E0DCBa77Vg90H21EOLACvtfcHk0lhhAFATLcwSJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsFanur0%253D&md5=7c1049b8cdad61bdd7d73c0aefbe496e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2013.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2013.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHIV-1%2520reverse%2520transcriptase%2520and%2520antiviral%2520drug%2520resistance.%2520Part%25201%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2013%26volume%3D3%26spage%3D111%26epage%3D118%26doi%3D10.1016%2Fj.coviro.2013.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Bean, P.</span><span> </span><span class="NLM_article-title">New drug targets for HIV</span> <span class="citation_source-journal">Clin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S96</span><span class="NLM_x">–</span> <span class="NLM_lpage">S100</span><span class="refDoi"> DOI: 10.1086/429504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1086%2F429504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16265623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtValurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=S96-S100&issue=Suppl.+1&author=P.+Bean&title=New+drug+targets+for+HIV&doi=10.1086%2F429504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">New drug targets for HIV</span></div><div class="casAuthors">Bean, Pamela</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S96-S100</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">A review.  A significant no. of human immunodeficiency virus (HIV) infections have become resistant to antiretroviral treatment, which means that there is a paramount need for novel drug targets to defeat the virus.  Until recently, all HIV drugs inhibited HIV replication by mechanisms operating inside infected cells.  In contrast, new antiretroviral drugs operate outside infected cells.  Their mechanism of action consists in inhibiting entry of the virus into cells, thereby halting the very first step of HIV replication.  Examples of this new class of drugs include entry inhibitors, coreceptor antagonists, and fusion inhibitors.  In addn. to their novel mechanism of action, this new class of drugs also has potential action against drug-resistant HIV strains, causes minimal adverse effects, and may be administered in a simplified, once-daily dosing regimen.  New classes of anti-HIV drugs-and new drugs in existing classes-represent the best hope for people infected with HIV, esp. those who have exhausted current therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIMgCCnqQaHbVg90H21EOLACvtfcHk0lhhAFATLcwSJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtValurk%253D&md5=0c76d4c0bfd32bc75d9f6a3ec4dff808</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1086%2F429504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F429504%26sid%3Dliteratum%253Aachs%26aulast%3DBean%26aufirst%3DP.%26atitle%3DNew%2520drug%2520targets%2520for%2520HIV%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2005%26volume%3D41%26issue%3DSuppl.%25201%26spage%3DS96%26epage%3DS100%26doi%3D10.1086%2F429504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Potterton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turkenburg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E.</span><span> </span><span class="NLM_article-title">A graphical user interface to the CCP4 program suite</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1131</span><span class="NLM_x">–</span> <span class="NLM_lpage">1137</span><span class="refDoi"> DOI: 10.1107/S0907444903008126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444903008126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12832755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=1131-1137&author=E.+Pottertonauthor=P.+Briggsauthor=M.+Turkenburgauthor=E.+Dodson&title=A+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS0907444903008126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Elizabeth; Briggs, Peter; Turkenburg, Maria; Dodson, Eleanor</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1131-1137</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4i is a graphical user interface that makes running programs from the CCP4 suite simpler and quicker.  It is particularly directed at inexperienced users and tightly linked to introductory and scientific documentation.  It also provides a simple project-management system and visualization tools.  The system is readily extensible and not specific to CCP4 software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDH-BQSvi5qbVg90H21EOLACvtfcHk0lhhAFATLcwSJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D&md5=25a0168dd7ea9f930036453a829e1e3d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444903008126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903008126%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DBriggs%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DE.%26atitle%3DA%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D1131%26epage%3D1137%26doi%3D10.1107%2FS0907444903008126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lg0u0Zz8YD-sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lg0u0Zz8YD-sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=9796821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK1cXmsVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span></div><div class="casAuthors">Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">739-747</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new algorithm is reported which builds an alignment between two protein structures.  The algorithm involves a combinatorial extension (CE) of an alignment path defined by aligned fragment pairs (AFPs) rather than the more conventional techniques using dynamic programming and Monte Carlo optimization.  AFPs, as the name suggests, are pairs of fragments, one from each protein, which confer structure similarity.  AFPs are based on local geometry, rather than global features such as orientation of secondary structures and overall topol.  Combinations of AFPs that represent possible continuous alignment paths are selectively extended or discarded thereby leading to a single optimal alignment.  The algorithm is fast and accurate in finding an optimal structure alignment and hence suitable for database scanning and detailed anal. of large protein families.  The method has been tested and compared with results from Dali and VAST using a representative sample of similar structures.  Several new structural similarities not detected by these other methods are reported.  Specific one-on-one alignments and searches against all structures as found in the Protein Data Bank (PDB) can be performed via the Web at http://cl.sdsc.edu/ce.html.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxq1Bh1NtRcbVg90H21EOLACvtfcHk0liXEObo0MkLkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsVynsbk%253D&md5=ef3e3dc633168dfd49cc871a72608727</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520structure%2520alignment%2520by%2520incremental%2520combinatorial%2520extension%2520%2528CE%2529%2520of%2520the%2520optimal%2520path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Sadowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Comparison of Automatic Three-Dimensional Model Builders Using 639 X-ray Structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1000</span><span class="NLM_x">–</span> <span class="NLM_lpage">1008</span><span class="refDoi"> DOI: 10.1021/ci00020a039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci00020a039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK2cXkvVajsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=1000-1008&author=J.+Sadowskiauthor=J.+Gasteigerauthor=G.+Klebe&title=Comparison+of+Automatic+Three-Dimensional+Model+Builders+Using+639+X-ray+Structures&doi=10.1021%2Fci00020a039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Automatic Three-Dimensional Model Builders Using 639 X-ray Structures</span></div><div class="casAuthors">Sadowski, Jens; Gasteiger, Johann; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1000-8</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">Several criteria were defined to select a dataset of high-quality x-ray structures from the Cambridge file resulting in 639 mols.  Six currently available programs for automatic 3D structure generation were compared by converting the connectivity tables, including appropriate stereodescriptors from this dataset of 639 mol. structures into 3D geometries: CONCORD, ALCOGEN, Chem-X, MOLGEO, COBRA, and CORINA.  The geometries produced by the different programs were evaluated in terms of several quality criteria and discussed in detail.  These criteria measure how well the different programs reproduce the x-ray geometries of the 639 input structures.  Accordingly, the major strengths and weaknesses of the programs are indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8vEtyDrrlq7Vg90H21EOLACvtfcHk0liXEObo0MkLkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkvVajsbY%253D&md5=246e4d66f11178a18daad2563c48331c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci00020a039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00020a039%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DJ.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DComparison%2520of%2520Automatic%2520Three-Dimensional%2520Model%2520Builders%2520Using%2520639%2520X-ray%2520Structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D1994%26volume%3D34%26spage%3D1000%26epage%3D1008%26doi%3D10.1021%2Fci00020a039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span> </span><span class="NLM_article-title">GEMDOCK: a generic evolutionary method for molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1002/prot.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1002%2Fprot.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15048822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=288-304&author=J.+M.+Yangauthor=C.+C.+Chen&title=GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking&doi=10.1002%2Fprot.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">GEMDOCK: a generic evolutionary method for molecular docking</span></div><div class="casAuthors">Yang, Jinn-Moon; Chen, Chun-Chen</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">288-304</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We have developed an evolutionary approach for flexible ligand docking.  This approval, GEMDOCK, uses a Generic Evolutionary Method for mol. DOCKing and an empirical scoring function.  The former combines both discrete and continuous global search strategies with local search strategies to speed up convergence, whereas the latter results in rapid recognition of potential ligands.  GEMDOCK was tested on a diverse data set of 100 protein-ligand complexes from the Protein Data Bank.  In 79% of these complexes, the docked lowest energy ligand structures had root-mean-square derivations (RMSDs) below 2.0 Å with respect to the corresponding crystal structures.  The success rate increased to 85% if the structure water mols. were retained.  We evaluated GEMDOCK on two cross-docking expts. in which each ligand of a protein ensemble was docked into each protein of the ensemble.  Seventy-six percent of the docked structures had RMSDs below 2.0 Å when the ligands were docked into foreign structures.  We analyzed and validated GEMDOCK with respect to various search spaces and scoring functions, and found that if the scoring function was perfect, then the predicted accuracy was also essentially perfect.  This study suggests that GEMDOCK is a useful tool for mol. recognition and may be used to systematically evaluate and thus improve scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_k6KGw_uV-bVg90H21EOLACvtfcHk0ljKdl-lGtSyGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsro%253D&md5=02248206ca276b2da2b784224d087f12</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fprot.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20035%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BC.%26atitle%3DGEMDOCK%253A%2520a%2520generic%2520evolutionary%2520method%2520for%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D288%26epage%3D304%26doi%3D10.1002%2Fprot.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Pike, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brzozowski, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsell, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlquist, M.</span><span> </span><span class="NLM_article-title">Structural insights into the mode of action of a pure antiestrogen</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">145</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/S0969-2126(01)00568-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0969-2126%2801%2900568-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=11250199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslKgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=145-153&author=A.+C.+Pikeauthor=A.+M.+Brzozowskiauthor=J.+Waltonauthor=R.+E.+Hubbardauthor=A.+G.+Thorsellauthor=Y.+L.+Liauthor=J.+A.+Gustafssonauthor=M.+Carlquist&title=Structural+insights+into+the+mode+of+action+of+a+pure+antiestrogen&doi=10.1016%2FS0969-2126%2801%2900568-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the mode of action of a pure antiestrogen</span></div><div class="casAuthors">Pike, Ashley C. W.; Brzozowski, A. Marek; Walton, Julia; Hubbard, Roderick E.; Thorsell, Ann-Gerd; Li, Yi-Lin; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-153</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Estrogens exert their effects on target tissues by binding to a nuclear transcription factor termed the estrogen receptor (ER).  Previous structural studies have demonstrated that each class of ER ligand (agonist, partial agonist, and SERM antagonist) induces distinctive orientations in the receptor's carboxy-terminal transactivation helix.  The conformation of this portion of the receptor dets. whether ER can recruit and interact with the components of the transcriptional machinery, thereby facilitating target gene expression.  Results: We have detd. the structure of rat ERβ ligand binding domain (LBD) in complex with the pure antiestrogen ICI 164,384 at 2.3 Å resoln.  The binding of this compd. to the receptor completely abolishes the assocn. between the transactivation helix (H12) and the rest of the LBD.  The structure reveals that the terminal portion of ICI's bulky side chain substituent protrudes from the hormone binding pocket, binds along the coactivator recruitment site, and phys. prevents H12 from adopting either its characteristic agonist or AF2 antagonist orientation.  Conclusions: The binding mode adopted by the pure antiestrogen is similar to that seen for other ER antagonists.  However, the size and resultant positioning of the ligand's side chain substituent produces a receptor conformation that is distinct from that adopted in the presence of other classes of ER ligands.  The novel observation that binding of ICI results in the complete destabilization of H12 provides some indications as to a possible mechanism for pure receptor antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_dBAqpVZPtrVg90H21EOLACvtfcHk0ljKdl-lGtSyGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslKgsrg%253D&md5=ac139f6c39a2605e15314176397f11d4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900568-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900568-8%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%2BC.%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DWalton%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DThorsell%26aufirst%3DA.%2BG.%26aulast%3DLi%26aufirst%3DY.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DStructural%2520insights%2520into%2520the%2520mode%2520of%2520action%2520of%2520a%2520pure%2520antiestrogen%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D145%26epage%3D153%26doi%3D10.1016%2FS0969-2126%2801%2900568-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Birck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengus, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavigne, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moras, D.</span><span> </span><span class="NLM_article-title">Human TAF(II)28 and TAF(II)18 interact through a histone fold encoded by atypical evolutionary conserved motifs also found in the SPT3 family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81423-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0092-8674%2800%2981423-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1998&pages=239-249&author=C.+Birckauthor=O.+Pochauthor=C.+Romierauthor=M.+Ruffauthor=G.+Mengusauthor=A.+C.+Lavigneauthor=I.+Davidsonauthor=D.+Moras&title=Human+TAF%28II%2928+and+TAF%28II%2918+interact+through+a+histone+fold+encoded+by+atypical+evolutionary+conserved+motifs+also+found+in+the+SPT3+family&doi=10.1016%2FS0092-8674%2800%2981423-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981423-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981423-3%26sid%3Dliteratum%253Aachs%26aulast%3DBirck%26aufirst%3DC.%26aulast%3DPoch%26aufirst%3DO.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DRuff%26aufirst%3DM.%26aulast%3DMengus%26aufirst%3DG.%26aulast%3DLavigne%26aufirst%3DA.%2BC.%26aulast%3DDavidson%26aufirst%3DI.%26aulast%3DMoras%26aufirst%3DD.%26atitle%3DHuman%2520TAF%2528II%252928%2520and%2520TAF%2528II%252918%2520interact%2520through%2520a%2520histone%2520fold%2520encoded%2520by%2520atypical%2520evolutionary%2520conserved%2520motifs%2520also%2520found%2520in%2520the%2520SPT3%2520family%26jtitle%3DCell%26date%3D1998%26volume%3D94%26spage%3D239%26epage%3D249%26doi%3D10.1016%2FS0092-8674%2800%2981423-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Ackerman, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwatia, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonidas, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span> </span><span class="NLM_article-title">Charcot-Leyden crystal protein (galectin-10) is not a dual function galectin with lysophospholipase activity but binds a lysophospholipase inhibitor in a novel structural fashion</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">14859</span><span class="NLM_x">–</span> <span class="NLM_lpage">14868</span><span class="refDoi"> DOI: 10.1074/jbc.M200221200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M200221200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=14859-14868&author=S.+J.+Ackermanauthor=L.+Liuauthor=M.+A.+Kwatiaauthor=M.+P.+Savageauthor=D.+D.+Leonidasauthor=G.+J.+Swaminathanauthor=K.+R.+Acharya&title=Charcot-Leyden+crystal+protein+%28galectin-10%29+is+not+a+dual+function+galectin+with+lysophospholipase+activity+but+binds+a+lysophospholipase+inhibitor+in+a+novel+structural+fashion&doi=10.1074%2Fjbc.M200221200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M200221200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M200221200%26sid%3Dliteratum%253Aachs%26aulast%3DAckerman%26aufirst%3DS.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKwatia%26aufirst%3DM.%2BA.%26aulast%3DSavage%26aufirst%3DM.%2BP.%26aulast%3DLeonidas%26aufirst%3DD.%2BD.%26aulast%3DSwaminathan%26aufirst%3DG.%2BJ.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26atitle%3DCharcot-Leyden%2520crystal%2520protein%2520%2528galectin-10%2529%2520is%2520not%2520a%2520dual%2520function%2520galectin%2520with%2520lysophospholipase%2520activity%2520but%2520binds%2520a%2520lysophospholipase%2520inhibitor%2520in%2520a%2520novel%2520structural%2520fashion%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D14859%26epage%3D14868%26doi%3D10.1074%2Fjbc.M200221200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Simon L. Dürr, Olga Bohuszewicz, Dénes Berta, Reynier Suardiaz, Pablo G. Jambrina, Christine Peter, Yihan Shao, <span class="NLM_string-name hlFld-ContribAuthor">Edina Rosta</span>. </span><span class="cited-content_cbyCitation_article-title">The Role of Conserved Residues in the DEDDh Motif: the Proton-Transfer Mechanism of HIV-1 RNase H. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.1c01493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.1c01493%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DThe%252BRole%252Bof%252BConserved%252BResidues%252Bin%252Bthe%252BDEDDh%252BMotif%25253A%252Bthe%252BProton-Transfer%252BMechanism%252Bof%252BHIV-1%252BRNase%252BH%26aulast%3DD%25C3%25BCrr%26aufirst%3DSimon%2BL.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01042021%26date%3D21052021%26volume%3D0%26issue%3D0%26spage%3D7915%26epage%3D7927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu-Jin  Kim</span>, <span class="hlFld-ContribAuthor ">Victor  Venturini</span>, <span class="hlFld-ContribAuthor ">Juan C.  de la Torre</span>. </span><span class="cited-content_cbyCitation_article-title">Progress in Anti-Mammarenavirus Drug Development. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (7)
                                     , 1187. <a href="https://doi.org/10.3390/v13071187" title="DOI URL">https://doi.org/10.3390/v13071187</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13071187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13071187%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DProgress%252Bin%252BAnti-Mammarenavirus%252BDrug%252BDevelopment%26aulast%3DKim%26aufirst%3DYu-Jin%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D7%26spage%3D1187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mya Myat  Ngwe Tun</span>, <span class="hlFld-ContribAuthor ">Kouichi  Morita</span>, <span class="hlFld-ContribAuthor ">Takeshi  Ishikawa</span>, <span class="hlFld-ContribAuthor ">Shuzo  Urata</span>. </span><span class="cited-content_cbyCitation_article-title">The Antiviral Effect of the Chemical Compounds Targeting DED/EDh Motifs of the Viral Proteins on Lymphocytic Choriomeningitis Virus and SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (7)
                                     , 1220. <a href="https://doi.org/10.3390/v13071220" title="DOI URL">https://doi.org/10.3390/v13071220</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13071220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13071220%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DThe%252BAntiviral%252BEffect%252Bof%252Bthe%252BChemical%252BCompounds%252BTargeting%252BDED%25252FEDh%252BMotifs%252Bof%252Bthe%252BViral%252BProteins%252Bon%252BLymphocytic%252BChoriomeningitis%252BVirus%252Band%252BSARS-CoV-2%26aulast%3DNgwe%2BTun%26aufirst%3DMya%2BMyat%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D7%26spage%3D1220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wayne O.  Hemphill</span>, <span class="hlFld-ContribAuthor ">Sean R.  Simpson</span>, <span class="hlFld-ContribAuthor ">Mingyong  Liu</span>, <span class="hlFld-ContribAuthor ">Freddie R.  Salsbury</span>, <span class="hlFld-ContribAuthor ">Thomas  Hollis</span>, <span class="hlFld-ContribAuthor ">Jason M.  Grayson</span>, <span class="hlFld-ContribAuthor ">Fred W.  Perrino</span>. </span><span class="cited-content_cbyCitation_article-title">TREX1 as a Novel Immunotherapeutic Target. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fimmu.2021.660184" title="DOI URL">https://doi.org/10.3389/fimmu.2021.660184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2021.660184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2021.660184%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DTREX1%252Bas%252Ba%252BNovel%252BImmunotherapeutic%252BTarget%26aulast%3DHemphill%26aufirst%3DWayne%2BO.%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rangika  Munaweera</span>, <span class="hlFld-ContribAuthor ">Ying S.  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Characterizations of the Interactions Between the Pontacyl Violet 6R and Exoribonuclease as a Potential Drug Target Against SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.627340" title="DOI URL">https://doi.org/10.3389/fchem.2020.627340</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.627340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.627340%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputational%252BCharacterizations%252Bof%252Bthe%252BInteractions%252BBetween%252Bthe%252BPontacyl%252BViolet%252B6R%252Band%252BExoribonuclease%252Bas%252Ba%252BPotential%252BDrug%252BTarget%252BAgainst%252BSARS-CoV-2%26aulast%3DMunaweera%26aufirst%3DRangika%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Biswajit  Naik</span>, <span class="hlFld-ContribAuthor ">Nidhi  Gupta</span>, <span class="hlFld-ContribAuthor ">Rupal  Ojha</span>, <span class="hlFld-ContribAuthor ">Satyendra  Singh</span>, <span class="hlFld-ContribAuthor ">Vijay Kumar  Prajapati</span>, <span class="hlFld-ContribAuthor ">Dhaneswar  Prusty</span>. </span><span class="cited-content_cbyCitation_article-title">High throughput virtual screening reveals SARS-CoV-2 multi-target binding natural compounds to lead instant therapy for COVID-19 treatment. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2020,</strong> <em>160 </em>, 1-17. <a href="https://doi.org/10.1016/j.ijbiomac.2020.05.184" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2020.05.184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2020.05.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2020.05.184%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DHigh%252Bthroughput%252Bvirtual%252Bscreening%252Breveals%252BSARS-CoV-2%252Bmulti-target%252Bbinding%252Bnatural%252Bcompounds%252Bto%252Blead%252Binstant%252Btherapy%252Bfor%252BCOVID-19%252Btreatment%26aulast%3DNaik%26aufirst%3DBiswajit%26date%3D2020%26volume%3D160%26spage%3D1%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea J  Pruijssers</span>, <span class="hlFld-ContribAuthor ">Mark R  Denison</span>. </span><span class="cited-content_cbyCitation_article-title">Nucleoside analogues for the treatment of coronavirus infections. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Virology</span><span> <strong>2019,</strong> <em>35 </em>, 57-62. <a href="https://doi.org/10.1016/j.coviro.2019.04.002" title="DOI URL">https://doi.org/10.1016/j.coviro.2019.04.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coviro.2019.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coviro.2019.04.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Virology%26atitle%3DNucleoside%252Banalogues%252Bfor%252Bthe%252Btreatment%252Bof%252Bcoronavirus%252Binfections%26aulast%3DPruijssers%26aufirst%3DAndrea%2BJ%26date%3D2019%26volume%3D35%26spage%3D57%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  Vénien-Bryan</span>, <span class="hlFld-ContribAuthor ">Zhuolun  Li</span>, <span class="hlFld-ContribAuthor ">Laurent  Vuillard</span>, <span class="hlFld-ContribAuthor ">Jean Albert  Boutin</span>. </span><span class="cited-content_cbyCitation_article-title">Cryo-electron microscopy and X-ray crystallography: complementary approaches to structural biology and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section F Structural Biology Communications</span><span> <strong>2017,</strong> <em>73 </em>
                                    (4)
                                     , 174-183. <a href="https://doi.org/10.1107/S2053230X17003740" title="DOI URL">https://doi.org/10.1107/S2053230X17003740</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2053230X17003740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2053230X17003740%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520F%2520Structural%2520Biology%2520Communications%26atitle%3DCryo-electron%252Bmicroscopy%252Band%252BX-ray%252Bcrystallography%25253A%252Bcomplementary%252Bapproaches%252Bto%252Bstructural%252Bbiology%252Band%252Bdrug%252Bdiscovery%26aulast%3DV%25C3%25A9nien-Bryan%26aufirst%3DCatherine%26date%3D2017%26date%3D2017%26volume%3D73%26issue%3D4%26spage%3D174%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. CRN-4 prefers to digest and bind single-stranded RNA than DNA. (A) CRN-4 cleaved a single-stranded 11-nt RNA more efficiently than DNA. (B) CRN-4 prefers to bind single-stranded RNA than DNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. The exonuclease activities of CRN-4 and RNase T are inhibited by several compounds. (A) The exonuclease activity of CRN-4 was assayed by incubating with citrate, PV6R, PCMPS, NCA, DR396, DTNB, ATA, and FDCO in concentrations ranging from 0.01 to 1 mM. The concentration for MES was from 0.1 to 10 mM. (B) The exonuclease activity of RNase T was analyzed by incubating with the same compounds as in (A). The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt DNA or RNA was used in the inhibitor-coupled nuclease assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibitor-coupled RNA/DNA-binding assays of CRN-4 and RNase T. (A) The RNA-binding activity of CRN-4 was analyzed by electrophoresis mobility shift assays with or without the presence of different inhibitors. The concentrations of MES were 1 and 10 mM, and for all other inhibitors it was 1 mM. The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt RNA was used in the assays. (B) The DNA-binding activity of RNase T was analyzed by electrophoresis mobility shift assays with or without the presence of different inhibitors. The 5′-end <sup>32</sup>P-labeled single-stranded 11-nt DNA was used in the assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. NP exonuclease of Lassa fever virus is inhibited by several potential inhibitors. (A, B) The exonuclease activity of NP exonuclease was analyzed by incubating NP exonuclease with singled-stranded 20-nt RNA and stem loop RNA labeled with FAM at 5′ end in the presence of each inhibitor. DR396 absorbed UV light, so the cleavage pattern was different from those of other inhibitors. (C) IC<sub>50</sub> of ATA in inhibiting NP exonuclease was estimated by incubating NP exonuclease (2 μM) and stem-loop RNA (1 μM) with different concentrations of ATA, and the digested DNA was analyzed by 20% denaturing polyacrylamide gels. (D) Quantification results of IC<sub>50</sub> of three independent experiments. (E) Summary of inhibition activity and IC<sub>50</sub> for the potential inhibitors of NP exonuclease.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structure of the CRN-4-MES complex. (A) Overall crystal structure of the dimeric CRN-4 in complex with MES. The DEDDh domain is displayed in light blue, whereas the Zn domain is displayed in pink. Two MES molecules are bound in the active sites of the CRN-4 dimer. (B) Electrostatic surface potential of one CRN-4 monomer. The MES molecule is bound deeply in the active site of CRN-4. (C) OMIT electron density map (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub>, 1σ) and the interacting residues of the MES molecule. Hydrogen bonds are displayed as red dotted lines. (D) Active site of MES-bound CRN-4. One Mg<sup>2+</sup> ion is bound in the active site. The general base His179 shifts out of the active site upon MES binding there.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. MES inhibits the exonuclease activity of CRN-4 by inducing a conformational change in the active site but not by blocking substrate binding. (A) Superimposition of apo-form CRN-4/Mn<sup>2+</sup>, the CRN-4-MES complex, and the RNase T-DNA complex reveals a similar geometry of the active site with four acidic residues (DEDD) and one general base (H) residue. His179 in the CRN-4-MES complex is shifted away from the active site. (B) Superimposition of apo-CRN-4/Mn<sup>2+</sup> and the CRN-4-MES complex shows that MES occupies the position of His179 in apo-CRN-4. The general base His179 in the CRN-4-MES complex is consequently shifted away from the active site. (C) Structural model of CRN-4 bound with RNA. One monomer is displayed as a molecular surface, whereas the other is displayed as a ribbon model. The 3′ end of the stem-looped RNA is inserted into the active site. (D) Close look at the active site of the MES-bound CRN-4. MES does not overlap with the RNA in the CRN-4-RNA model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/medium/jm-2016-00794c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Important amino acids for inhibitor binding in CRN-4 are revealed by molecular docking. (A) Eight compounds, except MES, are docked into CRN-4 using iGEMDOCK,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and they are divided into four groups (groups 2–5) based on the interactions with CRN-4 in the docking models. The interactions of MES in CRN-4 are derived from the crystal structure of CRN-4-MES complex, and it is classified into a single group (group 1). Interactions listed on the top row marked by E indicate electrostatic interactions, by H indicate hydrogen bonding, by C indicate covalent bond, and by V indicate van der Waals interactions. A cell is colored green for a positive interaction; otherwise the cell is colored black. The symbols +++, ++, + , and – represent strong, medium, weak, and no inhibition, respectively, based on the inhibitor-coupled nuclease activity assay from <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A. (B–E) Docking models for ATA, DR396, PCMPS, and citrate in the active site of CRN-4. Scissile phosphates are displayed as orange balls marked by P. Hydrogen and covalent bonds are displayed as red and green dotted lines, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00794/20170816/images/large/jm-2016-00794c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00794&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i20">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22565" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22565" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 65 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Zuo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">Exoribonuclease superfamilies: structural analysis and phylogenetic distribution</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1026</span><span class="refDoi"> DOI: 10.1093/nar/29.5.1017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2F29.5.1017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=11222749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhvFaqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=1017-1026&author=Y.+Zuoauthor=M.+P.+Deutscher&title=Exoribonuclease+superfamilies%3A+structural+analysis+and+phylogenetic+distribution&doi=10.1093%2Fnar%2F29.5.1017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Exoribonuclease superfamilies: structural analysis and phylogenetic distribution</span></div><div class="casAuthors">Zuo, Yuhong; Deutscher, Murray P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1017-1026</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review with 56 refs.  Exoribonucleases play an important role in all aspects of RNA metab.  Biochem. and genetic analyses in recent years have identified many new RNases and it is now clear that a single cell can contain multiple enzymes of this class.  Here, we analyze the structure and phylogenetic distribution of the known exoribonucleases.  Based on extensive sequence anal. and on their catalytic properties, all of the exoribonucleases and their homologs have been grouped into six superfamilies and various subfamilies.  We identify common motifs that can be used to characterize newly-discovered exoribonucleases, and based on these motifs we correct some previously misassigned proteins.  This anal. may serve as a useful first step for developing a nomenclature for this group of enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcVrS04JSdSrVg90H21EOLACvtfcHk0ljOIjQ3hmzgVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhvFaqs7w%253D&md5=dca9ea9312120b40d30e20cbab240018</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F29.5.1017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F29.5.1017%26sid%3Dliteratum%253Aachs%26aulast%3DZuo%26aufirst%3DY.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DExoribonuclease%2520superfamilies%253A%2520structural%2520analysis%2520and%2520phylogenetic%2520distribution%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2001%26volume%3D29%26spage%3D1017%26epage%3D1026%26doi%3D10.1093%2Fnar%2F29.5.1017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Yang, W.</span><span> </span><span class="NLM_article-title">Nucleases: diversity of structure, function and mechanism</span> <span class="citation_source-journal">Q. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1017/S0033583510000181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1017%2FS0033583510000181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20854710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=1-93&author=W.+Yang&title=Nucleases%3A+diversity+of+structure%2C+function+and+mechanism&doi=10.1017%2FS0033583510000181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleases: diversity of structure, function and mechanism</span></div><div class="casAuthors">Yang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-93</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Nucleases cleave the phosphodiester bonds of nucleic acids and may be endo or exo, DNase or RNase, topoisomerases, recombinases, ribozymes, or RNA splicing enzymes.  In this review, I survey nuclease activities with known structures and catalytic machinery and classify them by reaction mechanism and metal-ion dependence and by their biol. function ranging from DNA replication, recombination, repair, RNA maturation, processing, interference, to defense, nutrient regeneration or cell death.  Several general principles emerge from this anal.  There is little correlation between catalytic mechanism and biol. function.  A single catalytic mechanism can be adapted in a variety of reactions and biol. pathways.  Conversely, a single biol. process can often be accomplished by multiple tertiary and quaternary folds and by more than one catalytic mechanism.  Two-metal-ion-dependent nucleases comprise the largest no. of different tertiary folds and mediate the most diverse set of biol. functions.  Metal-ion-dependent cleavage is exclusively assocd. with exonucleases producing mononucleotides and endonucleases that cleave double- or single-stranded substrates in helical and base-stacked conformations.  All metal-ion-independent RNases generate 2'.3'-cyclic phosphate products, and all metal-ion-independent DNases form phospho-protein intermediates.  I also find several previously unnoted relationships between different nucleases and shared catalytic configurations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU7b0a8_b1YbVg90H21EOLACvtfcHk0ljOIjQ3hmzgVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOgt7c%253D&md5=4cd0bd5cec91b4572adf183ba2d90e7a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1017%2FS0033583510000181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583510000181%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26atitle%3DNucleases%253A%2520diversity%2520of%2520structure%252C%2520function%2520and%2520mechanism%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2011%26volume%3D44%26spage%3D1%26epage%3D93%26doi%3D10.1017%2FS0033583510000181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Hastie, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zandonatti, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saphire, E. O.</span><span> </span><span class="NLM_article-title">Structural basis for the dsRNA specificity of the Lassa virus NP exonuclease</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e44211</span><span class="refDoi"> DOI: 10.1371/journal.pone.0044211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pone.0044211" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e44211&author=K.+M.+Hastieauthor=L.+B.+Kingauthor=M.+A.+Zandonattiauthor=E.+O.+Saphire&title=Structural+basis+for+the+dsRNA+specificity+of+the+Lassa+virus+NP+exonuclease&doi=10.1371%2Fjournal.pone.0044211"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0044211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0044211%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DK.%2BM.%26aulast%3DKing%26aufirst%3DL.%2BB.%26aulast%3DZandonatti%26aufirst%3DM.%2BA.%26aulast%3DSaphire%26aufirst%3DE.%2BO.%26atitle%3DStructural%2520basis%2520for%2520the%2520dsRNA%2520specificity%2520of%2520the%2520Lassa%2520virus%2520NP%2520exonuclease%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De44211%26doi%3D10.1371%2Fjournal.pone.0044211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Crystal structure of CRN-4: implications for domain function in apoptotic DNA degradation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">448</span><span class="NLM_x">–</span> <span class="NLM_lpage">457</span><span class="refDoi"> DOI: 10.1128/MCB.01006-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1128%2FMCB.01006-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18981218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVChurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=448-457&author=Y.+Y.+Hsiaoauthor=A.+Nakagawaauthor=Z.+Shiauthor=S.+Mitaniauthor=D.+Xueauthor=H.+S.+Yuan&title=Crystal+structure+of+CRN-4%3A+implications+for+domain+function+in+apoptotic+DNA+degradation&doi=10.1128%2FMCB.01006-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of CRN-4: implications for domain function in apoptotic DNA degradation</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Nakagawa, Akihisa; Shi, Zhonghao; Mitani, Shohei; Xue, Ding; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">448-457</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cell death related nuclease 4 (CRN-4) is one of the apoptotic nucleases involved in DNA degrdn. in Caenorhabditis elegans.  To understand how CRN-4 is involved in apoptotic DNA fragmentation, we analyzed CRN-4's biochem. properties, in vivo cell functions, and the crystal structures of CRN-4 in apo-form, Mn2+-bound active form, and Er3+-bound inactive form.  CRN-4 is a dimeric nuclease with the optimal enzyme activity in cleaving double-stranded DNA in apoptotic salt conditions.  Both mutational studies and the structures of the Mn2+-bound CRN-4 revealed the geometry of the functional nuclease active site in the N-terminal DEDDh domain.  The C-terminal domain, termed the Zn-domain, contains basic surface residues ideal for nucleic acid recognition and is involved in DNA binding, as confirmed by deletion assays.  Cell death anal. in C. elegans further demonstrated that both the nuclease active site and the Zn-domain are required for crn-4's function in apoptosis.  Combining all of the data, we suggest a structural model where chromosomal DNA is bound at the Zn-domain and cleaved at the DEDDh nuclease domain in CRN-4 when the cell is undergoing apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrerdQCbulCTrVg90H21EOLACvtfcHk0li8BP_rKCgq_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVChurc%253D&md5=161b104a4169ebf613fe1e99a3e2ffbb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FMCB.01006-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01006-08%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DNakagawa%26aufirst%3DA.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DMitani%26aufirst%3DS.%26aulast%3DXue%26aufirst%3DD.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DCrystal%2520structure%2520of%2520CRN-4%253A%2520implications%2520for%2520domain%2520function%2520in%2520apoptotic%2520DNA%2520degradation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D448%26epage%3D457%26doi%3D10.1128%2FMCB.01006-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Korada, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, C. E.</span><span> </span><span class="NLM_article-title">Crystal structures of Escherichia coli exonuclease I in complex with single-stranded DNA provide insights into the mechanism of processive digestion</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">5887</span><span class="NLM_x">–</span> <span class="NLM_lpage">5897</span><span class="refDoi"> DOI: 10.1093/nar/gkt278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgkt278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23609540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFCgu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=5887-5897&author=S.+K.+Koradaauthor=T.+D.+Johnsauthor=C.+E.+Smithauthor=N.+D.+Jonesauthor=K.+A.+McCabeauthor=C.+E.+Bell&title=Crystal+structures+of+Escherichia+coli+exonuclease+I+in+complex+with+single-stranded+DNA+provide+insights+into+the+mechanism+of+processive+digestion&doi=10.1093%2Fnar%2Fgkt278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of Escherichia coli exonuclease I in complex with single-stranded DNA provide insights into the mechanism of processive digestion</span></div><div class="casAuthors">Korada, Sai Krishna C.; Johns, Trevor D.; Smith, Christopher E.; Jones, Nathan D.; McCabe, Kimberly A.; Bell, Charles E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5887-5897</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Escherichia coli Exonuclease I (ExoI) digests single-stranded DNA (ssDNA) in the 3'-5' direction in a highly processive manner.  The crystal structure of ExoI, detd. previously in the absence of DNA, revealed a C-shaped mol. with three domains that form a central pos. charged groove.  The active site is at the bottom of the groove, while an extended loop, proposed to encircle the DNA, crosses over the groove.  Here, we present crystal structures of ExoI in complex with four different ssDNA substrates.  The structures all have the ssDNA bound in essentially the predicted manner, with the 3'-end in the active site and the downstream end under the crossover loop.  The central nucleotides of the DNA form a prominent bulge that contacts the SH3-like domain, while the nucleotides at the downstream end of the DNA form extensive interactions with an anchor' site.  Seven of the complexes are similar to one another, but one has the ssDNA bound in a distinct conformation.  The highest-resoln. structure, detd. at 1.95 Å, reveals an Mg2+ ion bound to the scissile phosphate in a position corresponding to MgB in related two-metal nucleases.  The structures provide new insights into the mechanism of processive digestion that will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6XunF5jiSqLVg90H21EOLACvtfcHk0li8BP_rKCgq_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFCgu74%253D&md5=e60a1b8ac88aa33674831a042a9435a2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt278%26sid%3Dliteratum%253Aachs%26aulast%3DKorada%26aufirst%3DS.%2BK.%26aulast%3DJohns%26aufirst%3DT.%2BD.%26aulast%3DSmith%26aufirst%3DC.%2BE.%26aulast%3DJones%26aufirst%3DN.%2BD.%26aulast%3DMcCabe%26aufirst%3DK.%2BA.%26aulast%3DBell%26aufirst%3DC.%2BE.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520exonuclease%2520I%2520in%2520complex%2520with%2520single-stranded%2520DNA%2520provide%2520insights%2520into%2520the%2520mechanism%2520of%2520processive%2520digestion%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D5887%26epage%3D5897%26doi%3D10.1093%2Fnar%2Fgkt278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Horio, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamasaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, T.</span><span> </span><span class="NLM_article-title">Crystal structure of human ISG20, an interferon-induced antiviral ribonuclease</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">577</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span><span class="refDoi"> DOI: 10.1016/j.febslet.2004.09.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.febslet.2004.09.074" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=577&publication_year=2004&pages=111-116&author=T.+Horioauthor=M.+Muraiauthor=T.+Inoueauthor=T.+Hamasakiauthor=T.+Tanakaauthor=T.+Ohgi&title=Crystal+structure+of+human+ISG20%2C+an+interferon-induced+antiviral+ribonuclease&doi=10.1016%2Fj.febslet.2004.09.074"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2004.09.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2004.09.074%26sid%3Dliteratum%253Aachs%26aulast%3DHorio%26aufirst%3DT.%26aulast%3DMurai%26aufirst%3DM.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHamasaki%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DOhgi%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520human%2520ISG20%252C%2520an%2520interferon-induced%2520antiviral%2520ribonuclease%26jtitle%3DFEBS%2520Lett.%26date%3D2004%26volume%3D577%26spage%3D111%26epage%3D116%26doi%3D10.1016%2Fj.febslet.2004.09.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">How an exonuclease decides where to stop in trimming of nucleic acids: crystal structures of RNase T-product complexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">8144</span><span class="NLM_x">–</span> <span class="NLM_lpage">8154</span><span class="refDoi"> DOI: 10.1093/nar/gks548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgks548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22718982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlKltrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=8144-8154&author=Y.+Y.+Hsiaoauthor=Y.+Duhauthor=Y.+P.+Chenauthor=Y.+T.+Wangauthor=H.+S.+Yuan&title=How+an+exonuclease+decides+where+to+stop+in+trimming+of+nucleic+acids%3A+crystal+structures+of+RNase+T-product+complexes&doi=10.1093%2Fnar%2Fgks548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">How an exonuclease decides where to stop in trimming of nucleic acids: crystal structures of RNase T-product complexes</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Duh, Yulander; Chen, Yi-Ping; Wang, Yi-Ting; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8144-8154</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exonucleases are key enzymes in the maintenance of genome stability, processing of immature RNA precursors and degrdn. of unnecessary nucleic acids.  However, it remains unclear how exonucleases digest nucleic acids to generate correct end products for next-step processing.  Here we show how the exonuclease RNase T stops its trimming precisely.  The crystal structures of RNase T in complex with a stem-loop DNA, a GG dinucleotide and single-stranded DNA with different 3'-end sequences demonstrate why a duplex with a short 3'-overhang, a dinucleotide and a ssDNA with a 3'-end C cannot be further digested by RNase T.  Several hydrophobic residues in RNase T change their conformation upon substrate binding and induce an active or inactive conformation in the active site that construct a precise machine to det. which substrate should be digested based on its sequence, length and structure.  These studies thus provide mechanistic insights into how RNase T prevents over digestion of its various substrates, and the results can be extrapolated to the thousands of members of the DEDDh family of exonucleases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwS0gbeUZVZLVg90H21EOLACvtfcHk0li8BP_rKCgq_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlKltrfJ&md5=15052613cbcc50891518739f660671f0</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks548%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DDuh%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%2BP.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DHow%2520an%2520exonuclease%2520decides%2520where%2520to%2520stop%2520in%2520trimming%2520of%2520nucleic%2520acids%253A%2520crystal%2520structures%2520of%2520RNase%2520T-product%2520complexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D8144%26epage%3D8154%26doi%3D10.1093%2Fnar%2Fgks548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Kupsco, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marzluff, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thapar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duronio, R. J.</span><span> </span><span class="NLM_article-title">Genetic and biochemical characterization of Drosophila Snipper: A promiscuous member of the metazoan 3′hExo/ERI-1 family of 3′ to 5′ exonucleases</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">2117</span><span class="refDoi"> DOI: 10.1261/rna.186706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1261%2Frna.186706" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=2103-2117&author=J.+M.+Kupscoauthor=M.+J.+Wuauthor=W.+F.+Marzluffauthor=R.+Thaparauthor=R.+J.+Duronio&title=Genetic+and+biochemical+characterization+of+Drosophila+Snipper%3A+A+promiscuous+member+of+the+metazoan+3%E2%80%B2hExo%2FERI-1+family+of+3%E2%80%B2+to+5%E2%80%B2+exonucleases&doi=10.1261%2Frna.186706"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1261%2Frna.186706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.186706%26sid%3Dliteratum%253Aachs%26aulast%3DKupsco%26aufirst%3DJ.%2BM.%26aulast%3DWu%26aufirst%3DM.%2BJ.%26aulast%3DMarzluff%26aufirst%3DW.%2BF.%26aulast%3DThapar%26aufirst%3DR.%26aulast%3DDuronio%26aufirst%3DR.%2BJ.%26atitle%3DGenetic%2520and%2520biochemical%2520characterization%2520of%2520Drosophila%2520Snipper%253A%2520A%2520promiscuous%2520member%2520of%2520the%2520metazoan%25203%25E2%2580%25B2hExo%252FERI-1%2520family%2520of%25203%25E2%2580%25B2%2520to%25205%25E2%2580%25B2%2520exonucleases%26jtitle%3DRNA%26date%3D2006%26volume%3D12%26spage%3D2103%26epage%3D2117%26doi%3D10.1261%2Frna.186706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Zhao, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuo, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B. X.</span><span> </span><span class="NLM_article-title">A novel mutation of the WRN gene in a Chinese patient with Werner syndrome</span> <span class="citation_source-journal">Clin. Exp. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1111/j.1365-2230.2007.02641.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1111%2Fj.1365-2230.2007.02641.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18205852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1c3lslyntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=278-281&author=N.+Zhaoauthor=F.+Haoauthor=T.+Quauthor=Y.+G.+Zuoauthor=B.+X.+Wang&title=A+novel+mutation+of+the+WRN+gene+in+a+Chinese+patient+with+Werner+syndrome&doi=10.1111%2Fj.1365-2230.2007.02641.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A novel mutation of the WRN gene in a Chinese patient with Werner syndrome</span></div><div class="casAuthors">Zhao N; Hao F; Qu T; Zuo Y-G; Wang B-X</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and experimental dermatology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">278-81</span>
        ISSN:<span class="NLM_cas:issn">0307-6938</span>.
    </div><div class="casAbstract">Werner syndrome (WS) is an autosomal recessive inherited disease characterized by features of premature ageing.  It is caused by mutations of the WRN gene encoding a protein with both exonuclease and helicase activities.  The aim of this study was to identify gene mutations in a Chinese patient with WS.  A 31-year-old Chinese man with typical features of WS was diagnosed as having probable WS.  We performed PCR to scan 33 exons of the WRN gene of the patient, six members of his family, and 50 unrelated controls.  Automated DNA sequencing identified the mutation in the patient as 3250delG.  The proband's parents, son, younger brother and paternal grandmother were heterozygous.  We did not find this heterozygous mutation in the proband's maternal grandmother or in any of 50 normal controls.  The novel mutation in the WRN gene is responsible for the pathogenesis of WS and genetic detection is a useful method to confirm the diagnosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgR4CIUtwleUQ0lWbKiCflfW6udTcc2eZy2O9mQqM_urntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3lslyntw%253D%253D&md5=8f467ba3b0c7351159d9ed7f35ce0f04</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2230.2007.02641.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2230.2007.02641.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DN.%26aulast%3DHao%26aufirst%3DF.%26aulast%3DQu%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DY.%2BG.%26aulast%3DWang%26aufirst%3DB.%2BX.%26atitle%3DA%2520novel%2520mutation%2520of%2520the%2520WRN%2520gene%2520in%2520a%2520Chinese%2520patient%2520with%2520Werner%2520syndrome%26jtitle%3DClin.%2520Exp.%2520Dermatol.%26date%3D2008%26volume%3D33%26spage%3D278%26epage%3D281%26doi%3D10.1111%2Fj.1365-2230.2007.02641.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Perry, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yannone, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holden, L. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitomi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asaithamby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tainer, J. A.</span><span> </span><span class="NLM_article-title">WRN exonuclease structure and molecular mechanism imply an editing role in DNA end processing</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1038/nsmb1088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnsmb1088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16622405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD28XltFWlsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=414-422&author=J.+J.+Perryauthor=S.+M.+Yannoneauthor=L.+G.+Holdenauthor=C.+Hitomiauthor=A.+Asaithambyauthor=S.+Hanauthor=P.+K.+Cooperauthor=D.+J.+Chenauthor=J.+A.+Tainer&title=WRN+exonuclease+structure+and+molecular+mechanism+imply+an+editing+role+in+DNA+end+processing&doi=10.1038%2Fnsmb1088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">WRN exonuclease structure and molecular mechanism imply an editing role in DNA end processing</span></div><div class="casAuthors">Perry, J. Jefferson P.; Yannone, Steven M.; Holden, Lauren G.; Hitomi, Chiharu; Asaithamby, Aroumougame; Han, Seungil; Cooper, Priscilla K.; Chen, David J.; Tainer, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">WRN is unique among the five human RecQ DNA helicases in having a functional exonuclease domain (WRN-exo) and being defective in the premature aging and cancer-related disorder Werner syndrome.  Here, we characterize WRN-exo crystal structures, biochem. activity and participation in DNA end joining.  Metal-ion complex structures, active site mutations and activity assays reveal a nuclease mechanism mediated by two metal ions.  The DNA end-binding Ku70/80 complex specifically stimulates WRN-exo activity, and structure-based mutational inactivation of WRN-exo alters DNA end joining in human cells.  We furthermore establish structural and biochem. similarities of WRN-exo to DnaQ-family replicative proofreading exonucleases, describing WRN-specific adaptations consistent with double-stranded DNA specificity and functionally important conformational changes.  These results indicate WRN-exo is a human DnaQ family member and support DnaQ-like proofreading activities stimulated by Ku70/80, with implications for WRN functions in age-related pathologies and maintenance of genomic integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4cx5lqWQDh7Vg90H21EOLACvtfcHk0lgIsJqBLodguw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltFWlsrs%253D&md5=db2f70f2d735904d7e0221f218aa6d83</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1088%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DJ.%2BJ.%26aulast%3DYannone%26aufirst%3DS.%2BM.%26aulast%3DHolden%26aufirst%3DL.%2BG.%26aulast%3DHitomi%26aufirst%3DC.%26aulast%3DAsaithamby%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DP.%2BK.%26aulast%3DChen%26aufirst%3DD.%2BJ.%26aulast%3DTainer%26aufirst%3DJ.%2BA.%26atitle%3DWRN%2520exonuclease%2520structure%2520and%2520molecular%2520mechanism%2520imply%2520an%2520editing%2520role%2520in%2520DNA%2520end%2520processing%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D414%26epage%3D422%26doi%3D10.1038%2Fnsmb1088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Crow, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayward, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parmar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robins, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Bokhoven, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunner, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamel, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corry, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frints, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepper, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Livingston, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massey, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meritet, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michaud, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponsot, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voit, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebon, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonthron, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnes, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, T.</span><span> </span><span class="NLM_article-title">Mutations in the gene encoding the 3′-5′ DNA exonuclease TREX1 cause Aicardi-Goutieres syndrome at the AGS1 locus</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">917</span><span class="NLM_x">–</span> <span class="NLM_lpage">920</span><span class="refDoi"> DOI: 10.1038/ng1845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fng1845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16845398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsVCgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2006&pages=917-920&author=Y.+J.+Crowauthor=B.+E.+Haywardauthor=R.+Parmarauthor=P.+Robinsauthor=A.+Leitchauthor=M.+Aliauthor=D.+N.+Blackauthor=H.+van+Bokhovenauthor=H.+G.+Brunnerauthor=B.+C.+Hamelauthor=P.+C.+Corryauthor=F.+M.+Cowanauthor=S.+G.+Frintsauthor=J.+Klepperauthor=J.+H.+Livingstonauthor=S.+A.+Lynchauthor=R.+F.+Masseyauthor=J.+F.+Meritetauthor=J.+L.+Michaudauthor=G.+Ponsotauthor=T.+Voitauthor=P.+Lebonauthor=D.+T.+Bonthronauthor=A.+P.+Jacksonauthor=D.+E.+Barnesauthor=T.+Lindahl&title=Mutations+in+the+gene+encoding+the+3%E2%80%B2-5%E2%80%B2+DNA+exonuclease+TREX1+cause+Aicardi-Goutieres+syndrome+at+the+AGS1+locus&doi=10.1038%2Fng1845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the gene encoding the 3'-5' DNA exonuclease TREX1 cause Aicardi-Goutieres syndrome at the AGS1 locus</span></div><div class="casAuthors">Crow, Yanick J.; Hayward, Bruce E.; Parmar, Rekha; Robins, Peter; Leitch, Andrea; Ali, Manir; Black, Deborah N.; van Bokhoven, Hans; Brunner, Han G.; Hamel, Ben C.; Corry, Peter C.; Cowan, Frances M.; Frints, Suzanne G.; Klepper, Joerg; Livingston, John H.; Lynch, Sally Ann; Massey, Roger F.; Meritet, Jean Francois; Michaud, Jacques L.; Ponsot, Gerard; Voit, Thomas; Lebon, Pierre; Bonthron, David T.; Jackson, Andrew P.; Barnes, Deborah E.; Lindahl, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">917-920</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aicardi-Goutieres syndrome (AGS) presents as a severe neurol. brain disease and is a genetic mimic of the sequelae of transplacentally acquired viral infection.  Evidence exists for a perturbation of innate immunity as a primary pathogenic event in the disease phenotype.  Here, we show that TREX1, encoding the major mammalian 3' → 5' DNA exonuclease, is the AGS1 gene, and AGS-causing mutations result in abrogation of TREX1 enzyme activity.  Similar loss of function in the Trex1-/- mouse leads to an inflammatory phenotype.  Our findings suggest an unanticipated role for TREX1 in processing or clearing anomalous DNA structures, failure of which results in the triggering of an abnormal innate immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIwU5CVVO0DrVg90H21EOLACvtfcHk0lgIsJqBLodguw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsVCgsro%253D&md5=b36e47cc93a79cbc47c41cf7a33f94a7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fng1845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1845%26sid%3Dliteratum%253Aachs%26aulast%3DCrow%26aufirst%3DY.%2BJ.%26aulast%3DHayward%26aufirst%3DB.%2BE.%26aulast%3DParmar%26aufirst%3DR.%26aulast%3DRobins%26aufirst%3DP.%26aulast%3DLeitch%26aufirst%3DA.%26aulast%3DAli%26aufirst%3DM.%26aulast%3DBlack%26aufirst%3DD.%2BN.%26aulast%3Dvan%2BBokhoven%26aufirst%3DH.%26aulast%3DBrunner%26aufirst%3DH.%2BG.%26aulast%3DHamel%26aufirst%3DB.%2BC.%26aulast%3DCorry%26aufirst%3DP.%2BC.%26aulast%3DCowan%26aufirst%3DF.%2BM.%26aulast%3DFrints%26aufirst%3DS.%2BG.%26aulast%3DKlepper%26aufirst%3DJ.%26aulast%3DLivingston%26aufirst%3DJ.%2BH.%26aulast%3DLynch%26aufirst%3DS.%2BA.%26aulast%3DMassey%26aufirst%3DR.%2BF.%26aulast%3DMeritet%26aufirst%3DJ.%2BF.%26aulast%3DMichaud%26aufirst%3DJ.%2BL.%26aulast%3DPonsot%26aufirst%3DG.%26aulast%3DVoit%26aufirst%3DT.%26aulast%3DLebon%26aufirst%3DP.%26aulast%3DBonthron%26aufirst%3DD.%2BT.%26aulast%3DJackson%26aufirst%3DA.%2BP.%26aulast%3DBarnes%26aufirst%3DD.%2BE.%26aulast%3DLindahl%26aufirst%3DT.%26atitle%3DMutations%2520in%2520the%2520gene%2520encoding%2520the%25203%25E2%2580%25B2-5%25E2%2580%25B2%2520DNA%2520exonuclease%2520TREX1%2520cause%2520Aicardi-Goutieres%2520syndrome%2520at%2520the%2520AGS1%2520locus%26jtitle%3DNat.%2520Genet.%26date%3D2006%26volume%3D38%26spage%3D917%26epage%3D920%26doi%3D10.1038%2Fng1845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senenko, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engel, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Silva, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witte, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vyse, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kere, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeiffer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koskenmies, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hummel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominiczak, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gahr, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrino, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubner, N.</span><span> </span><span class="NLM_article-title">Mutations in the gene encoding the 3′-5′ DNA exonuclease TREX1 are associated with systemic lupus erythematosus</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">1065</span><span class="NLM_x">–</span> <span class="NLM_lpage">1067</span><span class="refDoi"> DOI: 10.1038/ng2091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fng2091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17660818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXps12gtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2007&pages=1065-1067&author=M.+A.+Lee-Kirschauthor=M.+Gongauthor=D.+Chowdhuryauthor=L.+Senenkoauthor=K.+Engelauthor=Y.+A.+Leeauthor=U.+de+Silvaauthor=S.+L.+Baileyauthor=T.+Witteauthor=T.+J.+Vyseauthor=J.+Kereauthor=C.+Pfeifferauthor=S.+Harveyauthor=A.+Wongauthor=S.+Koskenmiesauthor=O.+Hummelauthor=K.+Rohdeauthor=R.+E.+Schmidtauthor=A.+F.+Dominiczakauthor=M.+Gahrauthor=T.+Hollisauthor=F.+W.+Perrinoauthor=J.+Liebermanauthor=N.+Hubner&title=Mutations+in+the+gene+encoding+the+3%E2%80%B2-5%E2%80%B2+DNA+exonuclease+TREX1+are+associated+with+systemic+lupus+erythematosus&doi=10.1038%2Fng2091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the gene encoding the 3'-5' DNA exonuclease TREX1 are associated with systemic lupus erythematosus</span></div><div class="casAuthors">Lee-Kirsch, Min Ae; Gong, Maolian; Chowdhury, Dipanjan; Senenko, Lydia; Engel, Kerstin; Lee, Young-Ae; de Silva, Udesh; Bailey, Suzanna L.; Witte, Torsten; Vyse, Timothy J.; Kere, Juha; Pfeiffer, Christiane; Harvey, Scott; Wong, Andrew; Koskenmies, Sari; Hummel, Oliver; Rohde, Klaus; Schmidt, Reinhold E.; Dominiczak, Anna F.; Gahr, Manfred; Hollis, Thomas; Perrino, Fred W.; Lieberman, Judy; Huebner, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1065-1067</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TREX1 acts in concert with the SET complex in granzyme A-mediated apoptosis, and mutations in TREX1 cause Aicardi-Goutieres syndrome and familial chilblain lupus.  Here, we report monoallelic frameshift or missense mutations and one 3' UTR variant of TREX1 present in 9/417 individuals with systemic lupus erythematosus but absent in 1,712 controls (P = 4.1 × 10-7).  We demonstrate that two mutant TREX1 alleles alter subcellular targeting.  Our findings implicate TREX1 in the pathogenesis of SLE.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkgyqOJHcmtbVg90H21EOLACvtfcHk0lgPxYTekWUMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXps12gtLg%253D&md5=f91c180a5614571fe136aa36159d51f0</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fng2091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng2091%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DGong%26aufirst%3DM.%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DSenenko%26aufirst%3DL.%26aulast%3DEngel%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DY.%2BA.%26aulast%3Dde%2BSilva%26aufirst%3DU.%26aulast%3DBailey%26aufirst%3DS.%2BL.%26aulast%3DWitte%26aufirst%3DT.%26aulast%3DVyse%26aufirst%3DT.%2BJ.%26aulast%3DKere%26aufirst%3DJ.%26aulast%3DPfeiffer%26aufirst%3DC.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%26aulast%3DKoskenmies%26aufirst%3DS.%26aulast%3DHummel%26aufirst%3DO.%26aulast%3DRohde%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DR.%2BE.%26aulast%3DDominiczak%26aufirst%3DA.%2BF.%26aulast%3DGahr%26aufirst%3DM.%26aulast%3DHollis%26aufirst%3DT.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DLieberman%26aufirst%3DJ.%26aulast%3DHubner%26aufirst%3DN.%26atitle%3DMutations%2520in%2520the%2520gene%2520encoding%2520the%25203%25E2%2580%25B2-5%25E2%2580%25B2%2520DNA%2520exonuclease%2520TREX1%2520are%2520associated%2520with%2520systemic%2520lupus%2520erythematosus%26jtitle%3DNat.%2520Genet.%26date%3D2007%26volume%3D39%26spage%3D1065%26epage%3D1067%26doi%3D10.1038%2Fng2091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Graham, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerle, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolles, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denison, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baric, R. S.</span><span> </span><span class="NLM_article-title">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1820</span><span class="NLM_x">–</span> <span class="NLM_lpage">1826</span><span class="refDoi"> DOI: 10.1038/nm.2972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnm.2972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23142821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CktLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1820-1826&author=R.+L.+Grahamauthor=M.+M.+Beckerauthor=L.+D.+Eckerleauthor=M.+Bollesauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=A+live%2C+impaired-fidelity+coronavirus+vaccine+protects+in+an+aged%2C+immunocompromised+mouse+model+of+lethal+disease&doi=10.1038%2Fnm.2972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A live, impaired-fidelity coronavirus vaccine protects in an aged, immunocompromised mouse model of lethal disease</span></div><div class="casAuthors">Graham, Rachel L.; Becker, Michelle M.; Eckerle, Lance D.; Bolles, Meagan; Denison, Mark R.; Baric, Ralph S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1820-1826</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Live, attenuated RNA virus vaccines are efficacious but subject to reversion to virulence.  Among RNA viruses, replication fidelity is recognized as a key determinant of virulence and escape from antiviral therapy; increased fidelity is attenuating for some viruses.  Coronavirus (CoV) replication fidelity is approx. 20-fold greater than that of other RNA viruses and is mediated by a 3'→5' exonuclease (ExoN) activity that probably functions in RNA proofreading.  In this study we demonstrate that engineered inactivation of severe acute respiratory syndrome (SARS)-CoV ExoN activity results in a stable mutator phenotype with profoundly decreased fidelity in vivo and attenuation of pathogenesis in young, aged and immunocompromised mice.  The ExoN inactivation genotype and mutator phenotype are stable and do not revert to virulence, even after serial passage or long-term persistent infection in vivo.  ExoN inactivation has potential for broad applications in the stable attenuation of CoVs and, perhaps, other RNA viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYHcgTwksD3bVg90H21EOLACvtfcHk0lgPxYTekWUMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CktLvF&md5=e59f2b04c0ad2a6d8bbe62b7034adb66</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.2972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2972%26sid%3Dliteratum%253Aachs%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DBecker%26aufirst%3DM.%2BM.%26aulast%3DEckerle%26aufirst%3DL.%2BD.%26aulast%3DBolles%26aufirst%3DM.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DA%2520live%252C%2520impaired-fidelity%2520coronavirus%2520vaccine%2520protects%2520in%2520an%2520aged%252C%2520immunocompromised%2520mouse%2520model%2520of%2520lethal%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D1820%26epage%3D1826%26doi%3D10.1038%2Fnm.2972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Minskaia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertzig, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorbalenya, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campanacci, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cambillau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebuhr, J.</span><span> </span><span class="NLM_article-title">Discovery of an RNA virus 3′- > 5′ exoribonuclease that is critically involved in coronavirus RNA synthesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">5108</span><span class="NLM_x">–</span> <span class="NLM_lpage">5113</span><span class="refDoi"> DOI: 10.1073/pnas.0508200103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.0508200103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=5108-5113&author=E.+Minskaiaauthor=T.+Hertzigauthor=A.+E.+Gorbalenyaauthor=V.+Campanacciauthor=C.+Cambillauauthor=B.+Canardauthor=J.+Ziebuhr&title=Discovery+of+an+RNA+virus+3%E2%80%B2-+%3E+5%E2%80%B2+exoribonuclease+that+is+critically+involved+in+coronavirus+RNA+synthesis&doi=10.1073%2Fpnas.0508200103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0508200103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0508200103%26sid%3Dliteratum%253Aachs%26aulast%3DMinskaia%26aufirst%3DE.%26aulast%3DHertzig%26aufirst%3DT.%26aulast%3DGorbalenya%26aufirst%3DA.%2BE.%26aulast%3DCampanacci%26aufirst%3DV.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DCanard%26aufirst%3DB.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520RNA%2520virus%25203%25E2%2580%25B2-%2520%253E%25205%25E2%2580%25B2%2520exoribonuclease%2520that%2520is%2520critically%2520involved%2520in%2520coronavirus%2520RNA%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D5108%26epage%3D5113%26doi%3D10.1073%2Fpnas.0508200103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Qi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schelde, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallat, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ly, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span> </span><span class="NLM_article-title">Cap binding and immune evasion revealed by Lassa nucleoprotein structure</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">783</span><span class="refDoi"> DOI: 10.1038/nature09605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnature09605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21085117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=779-783&author=X.+Qiauthor=S.+Lanauthor=W.+Wangauthor=L.+M.+Scheldeauthor=H.+Dongauthor=G.+D.+Wallatauthor=H.+Lyauthor=Y.+Liangauthor=C.+Dong&title=Cap+binding+and+immune+evasion+revealed+by+Lassa+nucleoprotein+structure&doi=10.1038%2Fnature09605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cap binding and immune evasion revealed by Lassa nucleoprotein structure</span></div><div class="casAuthors">Qi, Xiao-Xuan; Lan, Shui-Yun; Wang, Wen-Jian; Schelde, Lisa McLay; Dong, Hao-Hao; Wallat, Gregor D.; Ly, Hinh; Liang, Yu-Ying; Dong, Chang-Jiang</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7325</span>),
    <span class="NLM_cas:pages">779-783</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lassa virus, the causative agent of Lassa fever, causes thousands of deaths annually and is a biol. threat agent, for which there is no vaccine and limited therapy.  The nucleoprotein (NP) of Lassa virus has essential roles in viral RNA synthesis and immune suppression, the mol. mechanisms of which are poorly understood.  Here we report the crystal structure of Lassa virus NP at 1.80 Å resoln., which reveals amino (N)- and carboxy (C)-terminal domains with structures unlike any of the reported viral NPs.  The N domain folds into a novel structure with a deep cavity for binding the m7GpppN cap structure that is required for viral RNA transcription, whereas the C domain contains 3'-5' exoribonuclease activity involved in suppressing interferon induction.  To our knowledge this is the first X-ray crystal structure solved for an arenaviral NP, which reveals its unexpected functions and indicates unique mechanisms in cap binding and immune evasion.  These findings provide great potential for vaccine and drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZleegizK2j7Vg90H21EOLACvtfcHk0ljlReB8hZTjKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGhsr3F&md5=4ae1f6e2dcdb2196c3aff97aca5d39bb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature09605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09605%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DX.%26aulast%3DLan%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSchelde%26aufirst%3DL.%2BM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWallat%26aufirst%3DG.%2BD.%26aulast%3DLy%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DCap%2520binding%2520and%2520immune%2520evasion%2520revealed%2520by%2520Lassa%2520nucleoprotein%2520structure%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D779%26epage%3D783%26doi%3D10.1038%2Fnature09605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Hasan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rakheja, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattnaik, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brugarolas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dozmorov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakeland, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, N.</span><span> </span><span class="NLM_article-title">Trex1 regulates lysosomal biogenesis and interferon-independent activation of antiviral genes</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1038/ni.2475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fni.2475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23160154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=61-71&author=M.+Hasanauthor=J.+Kochauthor=D.+Rakhejaauthor=A.+K.+Pattnaikauthor=J.+Brugarolasauthor=I.+Dozmorovauthor=B.+Levineauthor=E.+K.+Wakelandauthor=M.+A.+Lee-Kirschauthor=N.+Yan&title=Trex1+regulates+lysosomal+biogenesis+and+interferon-independent+activation+of+antiviral+genes&doi=10.1038%2Fni.2475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Trex1 regulates lysosomal biogenesis and interferon-independent activation of antiviral genes</span></div><div class="casAuthors">Hasan, Maroof; Koch, James; Rakheja, Dinesh; Pattnaik, Asit K.; Brugarolas, James; Dozmorov, Igor; Levine, Beth; Wakeland, Edward K.; Lee-Kirsch, Min Ae; Yan, Nan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-71</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sensing of viral nucleic acids by the innate immune system triggers the prodn. of type I interferons, which activates interferon-stimulated genes (ISGs) and directs a multifaceted antiviral response.  ISGs can also be activated through interferon-independent pathways, although the precise mechanisms remain elusive.  Here we found that the cytosolic exonuclease Trex1 regulated the activation of a subset of ISGs independently of interferon.  Both Trex1-/- mouse cells and Trex1-mutant human cells had high expression of genes encoding antiviral mols. ('antiviral genes') and were refractory to viral infection.  The interferon-independent activation of antiviral genes in Trex1-/- cells required the adaptor STING, the kinase TBK1 and the transcription factors IRF3 and IRF7.  We also found that Trex1-deficient cells had an expanded lysosomal compartment, altered subcellular localization of the transcription factor TFEB and diminished activity of the regulator mTORC1.  Together our data identify Trex1 as a regulator of lysosomal biogenesis and interferon-independent activation of antiviral genes and show that dysregulation of lysosomes can elicit innate immune responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY-tMxMVIKlrVg90H21EOLACvtfcHk0ljlReB8hZTjKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ymtrvI&md5=c54d3da15545b1f0fbf8d2f10480c041</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fni.2475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.2475%26sid%3Dliteratum%253Aachs%26aulast%3DHasan%26aufirst%3DM.%26aulast%3DKoch%26aufirst%3DJ.%26aulast%3DRakheja%26aufirst%3DD.%26aulast%3DPattnaik%26aufirst%3DA.%2BK.%26aulast%3DBrugarolas%26aufirst%3DJ.%26aulast%3DDozmorov%26aufirst%3DI.%26aulast%3DLevine%26aufirst%3DB.%26aulast%3DWakeland%26aufirst%3DE.%2BK.%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DTrex1%2520regulates%2520lysosomal%2520biogenesis%2520and%2520interferon-independent%2520activation%2520of%2520antiviral%2520genes%26jtitle%3DNat.%2520Immunol.%26date%3D2013%26volume%3D14%26spage%3D61%26epage%3D71%26doi%3D10.1038%2Fni.2475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Yan, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Regalado-Magdos, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stiggelbout, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee-Kirsch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lieberman, J.</span><span> </span><span class="NLM_article-title">The cytosolic exonuclease TREX1 inhibits the innate immune response to human immunodeficiency virus type 1</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1005</span><span class="NLM_x">–</span> <span class="NLM_lpage">1013</span><span class="refDoi"> DOI: 10.1038/ni.1941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fni.1941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20871604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Slur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=1005-1013&author=N.+Yanauthor=A.+D.+Regalado-Magdosauthor=B.+Stiggelboutauthor=M.+A.+Lee-Kirschauthor=J.+Lieberman&title=The+cytosolic+exonuclease+TREX1+inhibits+the+innate+immune+response+to+human+immunodeficiency+virus+type+1&doi=10.1038%2Fni.1941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The cytosolic exonuclease TREX1 inhibits the innate immune response to human immunodeficiency virus type 1</span></div><div class="casAuthors">Yan, Nan; Regalado-Magdos, Ashton D.; Stiggelbout, Bart; Lee-Kirsch, Min Ae; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1005-1013</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Viral infection triggers innate immune sensors to produce type I interferon.  However, infection of T cells and macrophages with human immunodeficiency virus (HIV) does not trip those alarms.  How HIV avoids activating nucleic acid sensors is unknown.  Here we found that the cytosolic exonuclease TREX1 suppressed interferon triggered by HIV.  In Trex1-/- mouse cells and human CD4+ T cells and macrophages in which TREX1 was inhibited by RNA-mediated interference, cytosolic HIV DNA accumulated and HIV infection induced type I interferon that inhibited HIV replication and spreading.  TREX1 bound to cytosolic HIV DNA and digested excess HIV DNA that would otherwise activate interferon expression via a pathway dependent on the kinase TBK1, the adaptor STING and the transcription factor IRF3.  HIV-stimulated interferon prodn. in cells deficient in TREX1 did not involve known nucleic acid sensors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMq3IR745M27Vg90H21EOLACvtfcHk0ljlReB8hZTjKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Slur7E&md5=2b1846d165468c434ced9eedebfb098f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fni.1941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni.1941%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DN.%26aulast%3DRegalado-Magdos%26aufirst%3DA.%2BD.%26aulast%3DStiggelbout%26aufirst%3DB.%26aulast%3DLee-Kirsch%26aufirst%3DM.%2BA.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520cytosolic%2520exonuclease%2520TREX1%2520inhibits%2520the%2520innate%2520immune%2520response%2520to%2520human%2520immunodeficiency%2520virus%2520type%25201%26jtitle%3DNat.%2520Immunol.%26date%3D2010%26volume%3D11%26spage%3D1005%26epage%3D1013%26doi%3D10.1038%2Fni.1941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ogbu, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajuluchukwu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uneke, C. J.</span><span> </span><span class="NLM_article-title">Lassa fever in West African sub-region: an overview</span> <span class="citation_source-journal">J. Vector Borne Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17378212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD2s7nslerug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2007&pages=1-11&author=O.+Ogbuauthor=E.+Ajuluchukwuauthor=C.+J.+Uneke&title=Lassa+fever+in+West+African+sub-region%3A+an+overview"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lassa fever in West African sub-region: an overview</span></div><div class="casAuthors">Ogbu O; Ajuluchukwu E; Uneke C J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of vector borne diseases</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-11</span>
        ISSN:<span class="NLM_cas:issn">0972-9062</span>.
    </div><div class="casAbstract">Lassa fever is an acute viral zoonotic illness caused by Lassa virus, an arenavirus known to be responsible for a severe haemorrhagic fever characterised by fever, muscle aches, sore throat, nausea, vomiting and, chest and abdominal pain.  The virus exhibits persistent, asymptomatic infection with profuse urinary virus excretion in the ubiquitous rodent vector, Mastomys natalensis.  Lassa fever is endemic in West Africa and has been reported from Sierra Leone, Guinea, Liberia, and Nigeria.  Some studies indicate that 300,000 to 500,000 cases of Lassa fever and 5000 deaths occur yearly across West Africa.  Studies reported in English, that investigated Lassa fever with reference to West Africa were identified using the Medline Entrez-PubMed search and were used for this review.  The scarcity of resources available for health care delivery system and the political instability that characterise the West African countries would continue to impede efforts for the control of Lassa fever in the sub-region.  There is need for adequate training of health care workers regarding diagnostics, intensive care of patients under isolation, contact tracing, adequate precautionary measures in handling infectious laboratory specimens, control of the vector as well as care and disposal of infectious waste.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRW5RJYcP5UICO2kE7krGOBfW6udTcc2eYshyd2t1fg3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s7nslerug%253D%253D&md5=a9f0f345352f31f279532f4445eb52c5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOgbu%26aufirst%3DO.%26aulast%3DAjuluchukwu%26aufirst%3DE.%26aulast%3DUneke%26aufirst%3DC.%2BJ.%26atitle%3DLassa%2520fever%2520in%2520West%2520African%2520sub-region%253A%2520an%2520overview%26jtitle%3DJ.%2520Vector%2520Borne%2520Dis.%26date%3D2007%26volume%3D44%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">3′ exoribonucleolytic trimming is a common feature of the maturation of small, stable RNAs in Escherichia coli</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2856</span><span class="NLM_x">–</span> <span class="NLM_lpage">2861</span><span class="refDoi"> DOI: 10.1073/pnas.95.6.2856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.95.6.2856" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=2856-2861&author=Z.+Liauthor=S.+Panditauthor=M.+P.+Deutscher&title=3%E2%80%B2+exoribonucleolytic+trimming+is+a+common+feature+of+the+maturation+of+small%2C+stable+RNAs+in+Escherichia+coli&doi=10.1073%2Fpnas.95.6.2856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.6.2856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.6.2856%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPandit%26aufirst%3DS.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3D3%25E2%2580%25B2%2520exoribonucleolytic%2520trimming%2520is%2520a%2520common%2520feature%2520of%2520the%2520maturation%2520of%2520small%252C%2520stable%2520RNAs%2520in%2520Escherichia%2520coli%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D2856%26epage%3D2861%26doi%3D10.1073%2Fpnas.95.6.2856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandit, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">Maturation of 23S ribosomal RNA requires the exoribonuclease RNase T</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span><span class="refDoi"> DOI: 10.1017/S1355838299981669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1017%2FS1355838299981669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=139-146&author=Z.+Liauthor=S.+Panditauthor=M.+P.+Deutscher&title=Maturation+of+23S+ribosomal+RNA+requires+the+exoribonuclease+RNase+T&doi=10.1017%2FS1355838299981669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1017%2FS1355838299981669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1355838299981669%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPandit%26aufirst%3DS.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DMaturation%2520of%252023S%2520ribosomal%2520RNA%2520requires%2520the%2520exoribonuclease%2520RNase%2520T%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D139%26epage%3D146%26doi%3D10.1017%2FS1355838299981669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Structural insights into DNA repair by RNase T--an exonuclease processing 3′ end of structured DNA in repair pathways</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001803</span><span class="refDoi"> DOI: 10.1371/journal.pbio.1001803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pbio.1001803" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=e1001803&author=Y.+Y.+Hsiaoauthor=W.+H.+Fangauthor=C.+C.+Leeauthor=Y.+P.+Chenauthor=H.+S.+Yuan&title=Structural+insights+into+DNA+repair+by+RNase+T%2D%2Dan+exonuclease+processing+3%E2%80%B2+end+of+structured+DNA+in+repair+pathways&doi=10.1371%2Fjournal.pbio.1001803"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1001803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1001803%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DFang%26aufirst%3DW.%2BH.%26aulast%3DLee%26aufirst%3DC.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BP.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DStructural%2520insights%2520into%2520DNA%2520repair%2520by%2520RNase%2520T--an%2520exonuclease%2520processing%25203%25E2%2580%25B2%2520end%2520of%2520structured%2520DNA%2520in%2520repair%2520pathways%26jtitle%3DPLoS%2520Biol.%26date%3D2014%26volume%3D12%26spage%3De1001803%26doi%3D10.1371%2Fjournal.pbio.1001803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Parrish, J. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, D.</span><span> </span><span class="NLM_article-title">Functional genomic analysis of apoptotic DNA degradation in C. elegans</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">996</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00095-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS1097-2765%2803%2900095-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=987-996&author=J.+Z.+Parrishauthor=D.+Xue&title=Functional+genomic+analysis+of+apoptotic+DNA+degradation+in+C.+elegans&doi=10.1016%2FS1097-2765%2803%2900095-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900095-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900095-9%26sid%3Dliteratum%253Aachs%26aulast%3DParrish%26aufirst%3DJ.%2BZ.%26aulast%3DXue%26aufirst%3DD.%26atitle%3DFunctional%2520genomic%2520analysis%2520of%2520apoptotic%2520DNA%2520degradation%2520in%2520C.%2520elegans%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D11%26spage%3D987%26epage%3D996%26doi%3D10.1016%2FS1097-2765%2803%2900095-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Nurmohamed, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Titman, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandran, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pears, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luisi, B. F.</span><span> </span><span class="NLM_article-title">Polynucleotide phosphorylase activity may be modulated by metabolites in escherichia coli</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">14315</span><span class="NLM_x">–</span> <span class="NLM_lpage">14323</span><span class="refDoi"> DOI: 10.1074/jbc.M110.200741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M110.200741" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=14315-14323&author=S.+Nurmohamedauthor=H.+A.+Vincentauthor=C.+M.+Titmanauthor=V.+Chandranauthor=M.+R.+Pearsauthor=D.+J.+Duauthor=J.+L.+Griffinauthor=A.+J.+Callaghanauthor=B.+F.+Luisi&title=Polynucleotide+phosphorylase+activity+may+be+modulated+by+metabolites+in+escherichia+coli&doi=10.1074%2Fjbc.M110.200741"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.200741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.200741%26sid%3Dliteratum%253Aachs%26aulast%3DNurmohamed%26aufirst%3DS.%26aulast%3DVincent%26aufirst%3DH.%2BA.%26aulast%3DTitman%26aufirst%3DC.%2BM.%26aulast%3DChandran%26aufirst%3DV.%26aulast%3DPears%26aufirst%3DM.%2BR.%26aulast%3DDu%26aufirst%3DD.%2BJ.%26aulast%3DGriffin%26aufirst%3DJ.%2BL.%26aulast%3DCallaghan%26aufirst%3DA.%2BJ.%26aulast%3DLuisi%26aufirst%3DB.%2BF.%26atitle%3DPolynucleotide%2520phosphorylase%2520activity%2520may%2520be%2520modulated%2520by%2520metabolites%2520in%2520escherichia%2520coli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D14315%26epage%3D14323%26doi%3D10.1074%2Fjbc.M110.200741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Song, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span> </span><span class="NLM_article-title">Characterization of a novel non-specific nuclease from thermophilic bacteriophage GBSV1</span> <span class="citation_source-journal">BMC Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="refDoi"> DOI: 10.1186/1472-6750-8-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1186%2F1472-6750-8-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=18439318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1czivVCguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=43&author=Q.+Songauthor=X.+Zhang&title=Characterization+of+a+novel+non-specific+nuclease+from+thermophilic+bacteriophage+GBSV1&doi=10.1186%2F1472-6750-8-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel non-specific nuclease from thermophilic bacteriophage GBSV1</span></div><div class="casAuthors">Song Qing; Zhang Xiaobo</div><div class="citationInfo"><span class="NLM_cas:title">BMC biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Thermostable enzymes from thermophiles have attracted extensive studies.  In this investigation, a nuclease-encoding gene (designated as GBSV1-NSN) was obtained from a thermophilic bacteriophage GBSV1 for the first time.  RESULTS:  After recombinant expression in Escherichia coli, the purified GBSV1-NSN exhibited non-specific nuclease activity, being able to degrade various nucleic acids, including RNA, single-stranded DNA and double-stranded DNA that was circular or linear.  Based on sequence analysis, the nuclease shared no homology with any known nucleases, suggesting that it was a novel nuclease.  The characterization of the recombinant GBSV1-NSN showed that its optimal temperature and pH were 60 degrees C and 7.5, respectively.  The results indicated that the enzymatic activity was inhibited by enzyme inhibitors or detergents, such as ethylene diamine tetraacetic acid, citrate, dithiothreitol, beta-mercaptoethanol, guanidine hydrochloride, urea and SDS.  In contrast, the nuclease activity was enhanced by TritonX-100, Tween-20 or chaps to approximately 124.5% - 141.6%.  The Km of GBSV1-NSN nuclease was 231, 61 and 92 microM, while its kcat was 1278, 241 and 300 s-1 for the cleavage of dsDNA, ssDNA and RNA, respectively.  CONCLUSION:  Our study, therefore, presented a novel thermostable non-specific nuclease from thermophilic bacteriophage and its overexpression and purification for scientific research and applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSC9vQdvXAByxe4jJGKT5N0fW6udTcc2eY27ee-tDFzXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czivVCguw%253D%253D&md5=27616f53f271951fe9c8c188a3eeb194</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2F1472-6750-8-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6750-8-43%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520novel%2520non-specific%2520nuclease%2520from%2520thermophilic%2520bacteriophage%2520GBSV1%26jtitle%3DBMC%2520Biotechnol.%26date%3D2008%26volume%3D8%26spage%3D43%26doi%3D10.1186%2F1472-6750-8-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stauffacher, C. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Etten, R. L.</span><span> </span><span class="NLM_article-title">Crystal structure of a human low molecular weight phosphotyrosyl phosphatase. Implications for substrate specificity</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">21714</span><span class="NLM_x">–</span> <span class="NLM_lpage">21720</span><span class="refDoi"> DOI: 10.1074/jbc.273.34.21714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.273.34.21714" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=21714-21720&author=M.+Zhangauthor=C.+V.+Stauffacherauthor=D.+Linauthor=R.+L.+Van+Etten&title=Crystal+structure+of+a+human+low+molecular+weight+phosphotyrosyl+phosphatase.+Implications+for+substrate+specificity&doi=10.1074%2Fjbc.273.34.21714"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.34.21714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.34.21714%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DStauffacher%26aufirst%3DC.%2BV.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DVan%2BEtten%26aufirst%3DR.%2BL.%26atitle%3DCrystal%2520structure%2520of%2520a%2520human%2520low%2520molecular%2520weight%2520phosphotyrosyl%2520phosphatase.%2520Implications%2520for%2520substrate%2520specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D21714%26epage%3D21720%26doi%3D10.1074%2Fjbc.273.34.21714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Smith, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soellner, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raines, R. T.</span><span> </span><span class="NLM_article-title">Potent inhibition of ribonuclease A by oligo(vinylsulfonic acid)</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">20934</span><span class="NLM_x">–</span> <span class="NLM_lpage">20938</span><span class="refDoi"> DOI: 10.1074/jbc.M301852200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M301852200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12649287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=20934-20938&author=B.+D.+Smithauthor=M.+B.+Soellnerauthor=R.+T.+Raines&title=Potent+inhibition+of+ribonuclease+A+by+oligo%28vinylsulfonic+acid%29&doi=10.1074%2Fjbc.M301852200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Inhibition of Ribonuclease A by Oligo(vinylsulfonic Acid)</span></div><div class="casAuthors">Smith, Bryan D.; Soellner, Matthew B.; Raines, Ronald T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">278</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">20934-20938</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">RNase A (RNase A) can make multiple contacts with an RNA substrate.  In particular, the enzymic active site and adjacent subsites bind sequential phosphoryl groups in the RNA backbone through Coulombic interactions.  Here, oligomers of vinylsulfonic acid (OVS) are shown to be potent inhibitors of RNase A that exploit these interactions.  Inhibition is competitive with substrate and has Ki = 11 pM in assays at low salt concn.  The effect of salt concn. on inhibition indicates that nearly eight favorable Coulombic interactions occur in the RNase A·OVS complex.  The phosphonic acid and sulfuric acid analogs of OVS are also potent inhibitors although slightly less effective.  OVS is also shown to be a contaminant of MES and other buffers that contain sulfonylethyl groups.  Oligomers greater than nine units in length can be isolated from com. MES buffer.  Inhibition by contaminating OVS is responsible for the apparent decrease in catalytic activity that has been obsd. in assays of RNase A at low salt concn.  Thus, OVS is both a useful inhibitor of RNase A and a potential bane to chemists and biochemists who use ethanesulfonic acid buffers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktvWKV_gUDrVg90H21EOLACvtfcHk0lgRwi8P_05-Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkt1Wisb0%253D&md5=412479c7e592c5f4b493c963f3c26cc5</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M301852200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M301852200%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DB.%2BD.%26aulast%3DSoellner%26aufirst%3DM.%2BB.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DPotent%2520inhibition%2520of%2520ribonuclease%2520A%2520by%2520oligo%2528vinylsulfonic%2520acid%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D20934%26epage%3D20938%26doi%3D10.1074%2Fjbc.M301852200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Park, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raines, R. T.</span><span> </span><span class="NLM_article-title">Origin of the “inactivation” of ribonuclease A at low salt concentration</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">199</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span><span class="refDoi"> DOI: 10.1016/S0014-5793(00)01227-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0014-5793%2800%2901227-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2000&pages=199-202&author=C.+Parkauthor=R.+T.+Raines&title=Origin+of+the+%E2%80%9Cinactivation%E2%80%9D+of+ribonuclease+A+at+low+salt+concentration&doi=10.1016%2FS0014-5793%2800%2901227-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2800%2901227-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252800%252901227-8%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DC.%26aulast%3DRaines%26aufirst%3DR.%2BT.%26atitle%3DOrigin%2520of%2520the%2520%25E2%2580%259Cinactivation%25E2%2580%259D%2520of%2520ribonuclease%2520A%2520at%2520low%2520salt%2520concentration%26jtitle%3DFEBS%2520Lett.%26date%3D2000%26volume%3D468%26spage%3D199%26epage%3D202%26doi%3D10.1016%2FS0014-5793%2800%2901227-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Sunaga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiokawa, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanuma, S.</span><span> </span><span class="NLM_article-title">A novel inhibitor that protects apoptotic DNA fragmentation catalyzed by DNase gamma</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1292</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2004.10.171</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.bbrc.2004.10.171" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2004&pages=1292-1297&author=S.+Sunagaauthor=T.+Kobayashiauthor=A.+Yoshimoriauthor=D.+Shiokawaauthor=S.+Tanuma&title=A+novel+inhibitor+that+protects+apoptotic+DNA+fragmentation+catalyzed+by+DNase+gamma&doi=10.1016%2Fj.bbrc.2004.10.171"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2004.10.171&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2004.10.171%26sid%3Dliteratum%253Aachs%26aulast%3DSunaga%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DYoshimori%26aufirst%3DA.%26aulast%3DShiokawa%26aufirst%3DD.%26aulast%3DTanuma%26aufirst%3DS.%26atitle%3DA%2520novel%2520inhibitor%2520that%2520protects%2520apoptotic%2520DNA%2520fragmentation%2520catalyzed%2520by%2520DNase%2520gamma%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2004%26volume%3D325%26spage%3D1292%26epage%3D1297%26doi%3D10.1016%2Fj.bbrc.2004.10.171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Prentice, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurley, L. R.</span><span> </span><span class="NLM_article-title">Effects of phosphatase inhibitors on nuclease activity</span> <span class="citation_source-journal">Cell Biol. Int. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1027</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1016/0309-1651(85)90069-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2F0309-1651%2885%2990069-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1985&pages=1027-1034&author=D.+A.+Prenticeauthor=P.+A.+Kitosauthor=L.+R.+Gurley&title=Effects+of+phosphatase+inhibitors+on+nuclease+activity&doi=10.1016%2F0309-1651%2885%2990069-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2F0309-1651%2885%2990069-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0309-1651%252885%252990069-4%26sid%3Dliteratum%253Aachs%26aulast%3DPrentice%26aufirst%3DD.%2BA.%26aulast%3DKitos%26aufirst%3DP.%2BA.%26aulast%3DGurley%26aufirst%3DL.%2BR.%26atitle%3DEffects%2520of%2520phosphatase%2520inhibitors%2520on%2520nuclease%2520activity%26jtitle%3DCell%2520Biol.%2520Int.%2520Rep.%26date%3D1985%26volume%3D9%26spage%3D1027%26epage%3D1034%26doi%3D10.1016%2F0309-1651%2885%2990069-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Li, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deutscher, M. P.</span><span> </span><span class="NLM_article-title">The role of individual cysteine residues in the activity of Escherichia coli RNase T</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">271</span><span class="NLM_x">, </span> <span class="NLM_fpage">1127</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.1074/jbc.271.2.1127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.271.2.1127" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=1127-1132&author=Z.+W.+Liauthor=L.+J.+Zhanauthor=M.+P.+Deutscher&title=The+role+of+individual+cysteine+residues+in+the+activity+of+Escherichia+coli+RNase+T&doi=10.1074%2Fjbc.271.2.1127"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.2.1127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.2.1127%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BW.%26aulast%3DZhan%26aufirst%3DL.%2BJ.%26aulast%3DDeutscher%26aufirst%3DM.%2BP.%26atitle%3DThe%2520role%2520of%2520individual%2520cysteine%2520residues%2520in%2520the%2520activity%2520of%2520Escherichia%2520coli%2520RNase%2520T%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D1127%26epage%3D1132%26doi%3D10.1074%2Fjbc.271.2.1127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Madhusudan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smart, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shrimpton, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houlbrook, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talbot, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammonds, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freemont, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternberg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dianov, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickson, I. D.</span><span> </span><span class="NLM_article-title">Isolation of a small molecule inhibitor of DNA base excision repair</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">4711</span><span class="NLM_x">–</span> <span class="NLM_lpage">4724</span><span class="refDoi"> DOI: 10.1093/nar/gki781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2Fgki781" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16113242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=4711-4724&author=S.+Madhusudanauthor=F.+Smartauthor=P.+Shrimptonauthor=J.+L.+Parsonsauthor=L.+Gardinerauthor=S.+Houlbrookauthor=D.+C.+Talbotauthor=T.+Hammondsauthor=P.+A.+Freemontauthor=M.+J.+Sternbergauthor=G.+L.+Dianovauthor=I.+D.+Hickson&title=Isolation+of+a+small+molecule+inhibitor+of+DNA+base+excision+repair&doi=10.1093%2Fnar%2Fgki781"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of a small molecule inhibitor of DNA base excision repair</span></div><div class="casAuthors">Madhusudan, Srinivasan; Smart, Fiona; Shrimpton, Paul; Parsons, Jason L.; Gardiner, Laurence; Houlbrook, Sue; Talbot, Denis C.; Hammonds, Timothy; Freemont, Paul A.; Sternberg, Michael J. E.; Dianov, Grigory L.; Hickson, Ian D.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4711-4724</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The base excision repair (BER) pathway is essential for the removal of DNA bases damaged by alkylation or oxidn.  A key step in BER is the processing of an apurinic/apyrimidinic (AP) site intermediate by an AP endonuclease.  The major AP endonuclease in human cells (APE1, also termed HAP1 and Ref-1) accounts for >95% of the total AP endonuclease activity, and is essential for the protection of cells against the toxic effects of several classes of DNA damaging agents.  Moreover, APE1 overexpression has been linked to radio- and chemo-resistance in human tumors.  Using a newly developed high-throughput screen, several chem. inhibitors of APE1 have been isolated.  Amongst these, CRT0044876 was identified as a potent and selective APE1 inhibitor.  CRT0044876 inhibits the AP endonuclease, 3'-phosphodiesterase and 3'-phosphatase activities of APE1 at low micromolar concns., and is a specific inhibitor of the exonuclease III family of enzymes to which APE1 belongs.  At non-cytotoxic concns., CRT0044876 potentiates the cytotoxicity of several DNA base-targeting compds.  This enhancement of cytotoxicity is assocd. with an accumulation of unrepaired AP sites.  In silico modeling studies suggest that CRT0044876 binds to the active site of APE1.  These studies provide both a novel reagent for probing APE1 function in human cells, and a rational basis for the development of APE1-targeting drugs for antitumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo35mTihPN15bVg90H21EOLACvtfcHk0liXx65sTnELIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFKjt7w%253D&md5=01a434b054d60d670d683dc0fa38e649</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki781%26sid%3Dliteratum%253Aachs%26aulast%3DMadhusudan%26aufirst%3DS.%26aulast%3DSmart%26aufirst%3DF.%26aulast%3DShrimpton%26aufirst%3DP.%26aulast%3DParsons%26aufirst%3DJ.%2BL.%26aulast%3DGardiner%26aufirst%3DL.%26aulast%3DHoulbrook%26aufirst%3DS.%26aulast%3DTalbot%26aufirst%3DD.%2BC.%26aulast%3DHammonds%26aufirst%3DT.%26aulast%3DFreemont%26aufirst%3DP.%2BA.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26aulast%3DDianov%26aufirst%3DG.%2BL.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26atitle%3DIsolation%2520of%2520a%2520small%2520molecule%2520inhibitor%2520of%2520DNA%2520base%2520excision%2520repair%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D4711%26epage%3D4724%26doi%3D10.1093%2Fnar%2Fgki781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Raffoul, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heydari, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillman, G. G.</span><span> </span><span class="NLM_article-title">DNA repair and cancer therapy: targeting APE1/Ref-1 using dietary agents</span> <span class="citation_source-journal">J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2012</span><span class="NLM_x">, </span> <span class="NLM_fpage">370481</span><span class="refDoi"> DOI: 10.1155/2012/370481</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1155%2F2012%2F370481" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22997517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BC38bovVOltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2012&publication_year=2012&pages=370481&author=J.+J.+Raffoulauthor=A.+R.+Heydariauthor=G.+G.+Hillman&title=DNA+repair+and+cancer+therapy%3A+targeting+APE1%2FRef-1+using+dietary+agents&doi=10.1155%2F2012%2F370481"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Repair and Cancer Therapy: Targeting APE1/Ref-1 Using Dietary Agents</span></div><div class="casAuthors">Raffoul Julian J; Heydari Ahmad R; Hillman Gilda G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2012</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">370481</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epidemiological studies have demonstrated the cancer protective effects of dietary agents and other natural compounds isolated from fruits, soybeans, and vegetables on neoplasia.  Studies have also revealed the potential for these natural products to be combined with chemotherapy or radiotherapy for the more effective treatment of cancer.  In this paper we discuss the potential for targeting the DNA base excision repair enzyme APE1/Ref-1 using dietary agents such as soy isoflavones, resveratrol, curcumin, and the vitamins ascorbate and α-tocopherol.  We also discuss the potential role of soy isoflavones in sensitizing cancer cells to the effects of radiotherapy.  A comprehensive review of the dual nature of APE1/Ref-1 in DNA repair and redox activation of cellular transcription factors, NF-κB and HIF-1α, is also discussed.  Further research efforts dedicated to delineating the role of APE1/Ref-1 DNA repair versus redox activity in sensitizing cancer cells to conventional treatment are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQfkGkuJsup8NIRw0GKUoEfW6udTcc2eZsyMgzMSZRF7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38bovVOltw%253D%253D&md5=5f5ee8ee8ef54e5484f38cf042a13b02</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1155%2F2012%2F370481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2012%252F370481%26sid%3Dliteratum%253Aachs%26aulast%3DRaffoul%26aufirst%3DJ.%2BJ.%26aulast%3DHeydari%26aufirst%3DA.%2BR.%26aulast%3DHillman%26aufirst%3DG.%2BG.%26atitle%3DDNA%2520repair%2520and%2520cancer%2520therapy%253A%2520targeting%2520APE1%252FRef-1%2520using%2520dietary%2520agents%26jtitle%3DJ.%2520Oncol.%26date%3D2012%26volume%3D2012%26spage%3D370481%26doi%3D10.1155%2F2012%2F370481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Fishel, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelley, M. R.</span><span> </span><span class="NLM_article-title">The DNA base excision repair protein Ape1/Ref-1 as a therapeutic and chemopreventive target</span> <span class="citation_source-journal">Mol. Aspects Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1016/j.mam.2007.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.mam.2007.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17560642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvF2ktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2007&pages=375-395&author=M.+L.+Fishelauthor=M.+R.+Kelley&title=The+DNA+base+excision+repair+protein+Ape1%2FRef-1+as+a+therapeutic+and+chemopreventive+target&doi=10.1016%2Fj.mam.2007.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA base excision repair protein Ape1/Ref-1 as a therapeutic and chemopreventive target</span></div><div class="casAuthors">Fishel, Melissa L.; Kelley, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">375-395</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With our growing understanding of the pathways involved in cell proliferation and signaling, targeted therapies, in the treatment of cancer are entering the clin. arena.  New and emerging targets are proteins involved in DNA repair pathways.  Inhibition of various proteins in the DNA repair pathways sensitizes cancer cells to DNA damaging agents such as chemotherapy and/or radiation.  We study the apurinic endonuclease 1/redox factor-1 (Ape1/Ref-1) and believe that its crucial function in DNA repair and redn.-oxidn. or redox signaling make it an excellent target for sensitizing tumor cells to chemotherapy.  Ape1/Ref-1 is an essential enzyme in the base excision repair (BER) pathway which is responsible for the repair of DNA caused by oxidative and alkylation damage.  As importantly, Ape1/Ref-1 also functions as a redox factor maintaining transcription factors in an active reduced state.  Ape1/Ref-1 stimulates the DNA binding activity of numerous transcription factors that are involved in cancer promotion and progression such as AP-1 (Fos/Jun), NFκB, HIF-1α, CREB, p53 and others.  We will discuss what is known regarding the pharmacol. targeting of the DNA repair activity, as well as the redox activity of Ape1/Ref-1, and explore the budding clin. utility of inhibition of either of these functions in cancer treatment.  A brief discussion of the effect of polymorphisms in its DNA sequence is included because of Ape1/Ref-1's importance to maintenance and integrity of the genome.  Exptl. modification of Ape1/Ref-1 activity changes the response of cells and of organisms to DNA damaging agents, suggesting that Ape1/Ref-1 may also be a productive target of chemoprevention.  In this review, we will provide an overview of Ape1/Ref-1's activities and explore the potential of this protein as a target in cancer treatment as well as its role in chemoprevention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCI5d4nCXTnbVg90H21EOLACvtfcHk0liXx65sTnELIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvF2ktrk%253D&md5=b037a714cdc8bd0f1df7e762ae3d1b92</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2007.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2007.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DFishel%26aufirst%3DM.%2BL.%26aulast%3DKelley%26aufirst%3DM.%2BR.%26atitle%3DThe%2520DNA%2520base%2520excision%2520repair%2520protein%2520Ape1%252FRef-1%2520as%2520a%2520therapeutic%2520and%2520chemopreventive%2520target%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2007%26volume%3D28%26spage%3D375%26epage%3D395%26doi%3D10.1016%2Fj.mam.2007.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Hallick, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chelm, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orozco, E. M.</span><span> </span><span class="NLM_article-title">Use of aurintricarboxylic acid as an inhibitor of nucleases during nucleic-acid isolation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3055</span><span class="NLM_x">–</span> <span class="NLM_lpage">3064</span><span class="refDoi"> DOI: 10.1093/nar/4.9.3055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fnar%2F4.9.3055" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1977&pages=3055-3064&author=R.+B.+Hallickauthor=B.+K.+Chelmauthor=P.+W.+Grayauthor=E.+M.+Orozco&title=Use+of+aurintricarboxylic+acid+as+an+inhibitor+of+nucleases+during+nucleic-acid+isolation&doi=10.1093%2Fnar%2F4.9.3055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F4.9.3055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F4.9.3055%26sid%3Dliteratum%253Aachs%26aulast%3DHallick%26aufirst%3DR.%2BB.%26aulast%3DChelm%26aufirst%3DB.%2BK.%26aulast%3DGray%26aufirst%3DP.%2BW.%26aulast%3DOrozco%26aufirst%3DE.%2BM.%26atitle%3DUse%2520of%2520aurintricarboxylic%2520acid%2520as%2520an%2520inhibitor%2520of%2520nucleases%2520during%2520nucleic-acid%2520isolation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1977%26volume%3D4%26spage%3D3055%26epage%3D3064%26doi%3D10.1093%2Fnar%2F4.9.3055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Cushman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wild, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Clercq, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schols, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldman, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowen, J. A.</span><span> </span><span class="NLM_article-title">Preparation and anti-HIV activities of aurintricarboxylic acid fractions and analogues: direct correlation of antiviral potency with molecular weight</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span><span class="refDoi"> DOI: 10.1021/jm00105a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00105a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK3MXjsFChsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1991&pages=329-337&author=M.+Cushmanauthor=P.+L.+Wangauthor=S.+H.+Changauthor=C.+Wildauthor=E.+De+Clercqauthor=D.+Scholsauthor=M.+E.+Goldmanauthor=J.+A.+Bowen&title=Preparation+and+anti-HIV+activities+of+aurintricarboxylic+acid+fractions+and+analogues%3A+direct+correlation+of+antiviral+potency+with+molecular+weight&doi=10.1021%2Fjm00105a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and anti-HIV activities of aurintricarboxylic acid fractions and analogs:  direct correlation of antiviral potency with molecular weight</span></div><div class="casAuthors">Cushman, Mark; Wang, Pinglang; Chang, Steve H.; Wild, Carl; De Clercq, Erik; Schols, Dominique; Goldman, Mark E.; Bowen, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">329-37</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Aurintricarboxylic acid (ATA) was fractionated by a combination of dialysis, ultrafiltration, and gel permeation chromatog.  The no. av. and wt. av. mol. wts. of the ATA fractions were detd. by the universal calibration method.  The sulfonic acid analog of ATA was prepd. and sepd. in high and low mol. wt. fractions.  The phosphonic acid analog of ATA was also synthesized.  All of the ATA fractions were tested for prevention of the cytopathic effect of HIV-1 and HIV-2 in MT-4 cell culture as well as against HIV-1 in CEM cell culture.  The abilities of the fractions and analogs to inhibit syncytium formation between HIV-1- and HIV-2-infected HUT-78 cells and uninfected MOLT-4 cells were evaluated.  In addn., the fractions and analogs were tested for cytotoxicity in mock-infected MT-4 cells, prevention of the binding of the OKT4A monoclonal antibody to the CD4 receptor, inhibition of the binding of anti-gp 120 monoclonal antibody to gp120, inhibition of attachment of HIV-1 virions to MT-4 cells, and inhibition of HIV-1 reverse transcriptase.  In all of these assays except cytotoxicity, there was a correlation of potency with mol. wt.  The higher the mol. wt., the higher the activity.  Several of the lower mol. wt. fractions of ATA, which bound to gp120 but not to CD4, prevented HIV-1 and HIV-2 cytopathicity.  A similar profile was obsd. for the phosphonic acid analog of ATA and the lower mol. wt. fraction of the sulfonic acid analog.  The results on the ATA fractions indicate that the binding of ATA to gp120 in the absence of CD4 binding is sufficient for anti-HIV activity.  The active compds. bind more avidly to gp120 than to CD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQvZPzjBX5f7Vg90H21EOLACvtfcHk0liLRanq2dhMFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXjsFChsg%253D%253D&md5=44d9335d0c1bbd6e970847a5b9b4d595</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm00105a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00105a052%26sid%3Dliteratum%253Aachs%26aulast%3DCushman%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DP.%2BL.%26aulast%3DChang%26aufirst%3DS.%2BH.%26aulast%3DWild%26aufirst%3DC.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DGoldman%26aufirst%3DM.%2BE.%26aulast%3DBowen%26aufirst%3DJ.%2BA.%26atitle%3DPreparation%2520and%2520anti-HIV%2520activities%2520of%2520aurintricarboxylic%2520acid%2520fractions%2520and%2520analogues%253A%2520direct%2520correlation%2520of%2520antiviral%2520potency%2520with%2520molecular%2520weight%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1991%26volume%3D34%26spage%3D329%26epage%3D337%26doi%3D10.1021%2Fjm00105a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Tsi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid protects against cell death caused by lipopolysaccharide in macrophages by decreasing inducible nitric-oxide synthase induction via IkappaB kinase, extracellular signal-regulated kinase, and p38 mitogen-activated protein kinase inhibition</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="NLM_x">–</span> <span class="NLM_lpage">101</span><span class="refDoi"> DOI: 10.1124/mol.62.1.90</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1124%2Fmol.62.1.90" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=90-101&author=C.+J.+Tsiauthor=Y.+Chaoauthor=C.+W.+Chenauthor=W.+W.+Lin&title=Aurintricarboxylic+acid+protects+against+cell+death+caused+by+lipopolysaccharide+in+macrophages+by+decreasing+inducible+nitric-oxide+synthase+induction+via+IkappaB+kinase%2C+extracellular+signal-regulated+kinase%2C+and+p38+mitogen-activated+protein+kinase+inhibition&doi=10.1124%2Fmol.62.1.90"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1124%2Fmol.62.1.90&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.62.1.90%26sid%3Dliteratum%253Aachs%26aulast%3DTsi%26aufirst%3DC.%2BJ.%26aulast%3DChao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DLin%26aufirst%3DW.%2BW.%26atitle%3DAurintricarboxylic%2520acid%2520protects%2520against%2520cell%2520death%2520caused%2520by%2520lipopolysaccharide%2520in%2520macrophages%2520by%2520decreasing%2520inducible%2520nitric-oxide%2520synthase%2520induction%2520via%2520IkappaB%2520kinase%252C%2520extracellular%2520signal-regulated%2520kinase%252C%2520and%2520p38%2520mitogen-activated%2520protein%2520kinase%2520inhibition%26jtitle%3DMol.%2520Pharmacol.%26date%3D2002%26volume%3D62%26spage%3D90%26epage%3D101%26doi%3D10.1124%2Fmol.62.1.90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Chen, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, W. W.</span><span> </span><span class="NLM_article-title">Inhibition of cytokine-induced JAK-STAT signalling pathways by an endonuclease inhibitor aurintricarboxylic acid</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">137</span><span class="NLM_x">, </span> <span class="NLM_fpage">1011</span><span class="NLM_x">–</span> <span class="NLM_lpage">1020</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0704955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fsj.bjp.0704955" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12429573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD38XpsVantLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2002&pages=1011-1020&author=C.+W.+Chenauthor=Y.+Chaoauthor=Y.+H.+Changauthor=M.+J.+Hsuauthor=W.+W.+Lin&title=Inhibition+of+cytokine-induced+JAK-STAT+signalling+pathways+by+an+endonuclease+inhibitor+aurintricarboxylic+acid&doi=10.1038%2Fsj.bjp.0704955"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of cytokine-induced JAK-STAT signalling pathways by an endonuclease inhibitor aurintricarboxylic acid</span></div><div class="casAuthors">Chen, Ching-Wen; Chao, Yee; Chang, Ying-Hsin; Hsu, Ming-Jen; Lin, Wan-Wan</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1011-1020</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 Inducible nitric oxide (iNOS) is thought to involve in host defense and tissue damage in inflammatory loci.  In previous study, we have found that the endonuclease inhibitor aurintricarboxylic acid (ATA) can protect macrophages from cell death induced by bacterial lipopolysaccharide.  This action is through the interruption with signalling pathways for NF-κB and AP-1 activation, and thus iNOS expression.  In this study we have addressed the effects of ATA on JAK-STAT signalling pathways.  2 In murine RAW 264.7 macrophages, IFN-γ-mediated NO prodn. and iNOS expression were concn.-dependently reduced by the presence of 3-100 μM ATA.  3 IFN-γ-induced STAT1 activation, as assessed from its tyrosine phosphorylation, nuclear translocation, binding to specific DNA response element and evoked IRF-1 reporter gene assay, were concomitantly inhibited by ATA.  However, ATA did not alter IFN-γ binding to RAW 264.7 cells.  4 The activities of JAK1 and JAK2, the upstream kinases essential for STAT1 signalling in response to IFN-γ, were also reduced by ATA.  5 Moreover, IL-4, IL-10, GM-CSF and M-CSF elicited tyrosine phosphorylation of STAT3, STAT5 and/or STAT6 in macrophages were diminished by the presence of ATA.  6 Taken together, we conclude that ATA can interfere JAK-STAT signalling pathways in response to cytokines.  This action contributes to the inhibition of IFN-γ-induced iNOS expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnff4gpCUlirVg90H21EOLACvtfcHk0liLRanq2dhMFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XpsVantLY%253D&md5=7678f3151186ec8ecfa7cc879918350e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0704955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0704955%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BW.%26aulast%3DChao%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DY.%2BH.%26aulast%3DHsu%26aufirst%3DM.%2BJ.%26aulast%3DLin%26aufirst%3DW.%2BW.%26atitle%3DInhibition%2520of%2520cytokine-induced%2520JAK-STAT%2520signalling%2520pathways%2520by%2520an%2520endonuclease%2520inhibitor%2520aurintricarboxylic%2520acid%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2002%26volume%3D137%26spage%3D1011%26epage%3D1020%26doi%3D10.1038%2Fsj.bjp.0704955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Hashem, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flaman, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farnsworth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Domselaar, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid is a potent inhibitor of influenza A and B virus neuraminidases</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e8350</span><span class="refDoi"> DOI: 10.1371/journal.pone.0008350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1371%2Fjournal.pone.0008350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20020057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A280%3ADC%252BD1Mfjt1ehuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e8350&author=A.+M.+Hashemauthor=A.+S.+Flamanauthor=A.+Farnsworthauthor=E.+G.+Brownauthor=G.+Van+Domselaarauthor=R.+Heauthor=X.+Li&title=Aurintricarboxylic+acid+is+a+potent+inhibitor+of+influenza+A+and+B+virus+neuraminidases&doi=10.1371%2Fjournal.pone.0008350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Aurintricarboxylic acid is a potent inhibitor of influenza A and B virus neuraminidases</span></div><div class="casAuthors">Hashem Anwar M; Flaman Anathea S; Farnsworth Aaron; Brown Earl G; Van Domselaar Gary; He Runtao; Li Xuguang</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e8350</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Influenza viruses cause serious infections that can be prevented or treated using vaccines or antiviral agents, respectively.  While vaccines are effective, they have a number of limitations, and influenza strains resistant to currently available anti-influenza drugs are increasingly isolated.  This necessitates the exploration of novel anti-influenza therapies.  METHODOLOGY/PRINCIPAL FINDINGS:  We investigated the potential of aurintricarboxylic acid (ATA), a potent inhibitor of nucleic acid processing enzymes, to protect Madin-Darby canine kidney cells from influenza infection.  We found, by neutral red assay, that ATA was protective, and by RT-PCR and ELISA, respectively, confirmed that ATA reduced viral replication and release.  Furthermore, while pre-treating cells with ATA failed to inhibit viral replication, pre-incubation of virus with ATA effectively reduced viral titers, suggesting that ATA may elicit its inhibitory effects by directly interacting with the virus.  Electron microscopy revealed that ATA induced viral aggregation at the cell surface, prompting us to determine if ATA could inhibit neuraminidase.  ATA was found to compromise the activities of virus-derived and recombinant neuraminidase.  Moreover, an oseltamivir-resistant H1N1 strain with H274Y was also found to be sensitive to ATA.  Finally, we observed additive protective value when infected cells were simultaneously treated with ATA and amantadine hydrochloride, an anti-influenza drug that inhibits M2-ion channels of influenza A virus.  CONCLUSIONS/SIGNIFICANCE:  Collectively, these data suggest that ATA is a potent anti-influenza agent by directly inhibiting the neuraminidase and could be a more effective antiviral compound when used in combination with amantadine hydrochloride.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT-CVIozU9TSJagfuraLLzSfW6udTcc2ebj077U2W5n3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mfjt1ehuw%253D%253D&md5=0c143587cfab62cb6636eec41dd5e307</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0008350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0008350%26sid%3Dliteratum%253Aachs%26aulast%3DHashem%26aufirst%3DA.%2BM.%26aulast%3DFlaman%26aufirst%3DA.%2BS.%26aulast%3DFarnsworth%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DE.%2BG.%26aulast%3DVan%2BDomselaar%26aufirst%3DG.%26aulast%3DHe%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%26atitle%3DAurintricarboxylic%2520acid%2520is%2520a%2520potent%2520inhibitor%2520of%2520influenza%2520A%2520and%2520B%2520virus%2520neuraminidases%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De8350%26doi%3D10.1371%2Fjournal.pone.0008350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Hung, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chao, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. T.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid inhibits influenza virus neuraminidase</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">123</span><span class="NLM_x">–</span> <span class="NLM_lpage">131</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2008.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.antiviral.2008.10.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=123-131&author=H.+C.+Hungauthor=C.+P.+Tsengauthor=J.+M.+Yangauthor=Y.+W.+Juauthor=S.+N.+Tsengauthor=Y.+F.+Chenauthor=Y.+S.+Chaoauthor=H.+P.+Hsiehauthor=S.+R.+Shihauthor=J.+T.+Hsu&title=Aurintricarboxylic+acid+inhibits+influenza+virus+neuraminidase&doi=10.1016%2Fj.antiviral.2008.10.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DH.%2BC.%26aulast%3DTseng%26aufirst%3DC.%2BP.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DJu%26aufirst%3DY.%2BW.%26aulast%3DTseng%26aufirst%3DS.%2BN.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DChao%26aufirst%3DY.%2BS.%26aulast%3DHsieh%26aufirst%3DH.%2BP.%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26atitle%3DAurintricarboxylic%2520acid%2520inhibits%2520influenza%2520virus%2520neuraminidase%26jtitle%3DAntiviral%2520Res.%26date%3D2009%26volume%3D81%26spage%3D123%26epage%3D131%26doi%3D10.1016%2Fj.antiviral.2008.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Schols, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pauwels, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desmyter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Specific interaction of aurintricarboxylic acid with the human immunodeficiency virus/CD4 cell receptor</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">86</span><span class="NLM_x">, </span> <span class="NLM_fpage">3322</span><span class="NLM_x">–</span> <span class="NLM_lpage">3326</span><span class="refDoi"> DOI: 10.1073/pnas.86.9.3322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1073%2Fpnas.86.9.3322" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1989&pages=3322-3326&author=D.+Scholsauthor=M.+Babaauthor=R.+Pauwelsauthor=J.+Desmyterauthor=E.+De+Clercq&title=Specific+interaction+of+aurintricarboxylic+acid+with+the+human+immunodeficiency+virus%2FCD4+cell+receptor&doi=10.1073%2Fpnas.86.9.3322"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.86.9.3322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.86.9.3322%26sid%3Dliteratum%253Aachs%26aulast%3DSchols%26aufirst%3DD.%26aulast%3DBaba%26aufirst%3DM.%26aulast%3DPauwels%26aufirst%3DR.%26aulast%3DDesmyter%26aufirst%3DJ.%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DSpecific%2520interaction%2520of%2520aurintricarboxylic%2520acid%2520with%2520the%2520human%2520immunodeficiency%2520virus%252FCD4%2520cell%2520receptor%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1989%26volume%3D86%26spage%3D3322%26epage%3D3326%26doi%3D10.1073%2Fpnas.86.9.3322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bopda-Waffo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silberstein, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talele, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arora, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaushik-Basu, N.</span><span> </span><span class="NLM_article-title">Characterization of aurintricarboxylic acid as a potent hepatitis C virus replicase inhibitor</span> <span class="citation_source-journal">Antivir. Chem. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.3851/IMP1286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.3851%2FIMP1286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=19794229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSlurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=19-36&author=Y.+Chenauthor=A.+Bopda-Waffoauthor=A.+Basuauthor=R.+Krishnanauthor=E.+Silbersteinauthor=D.+R.+Taylorauthor=T.+T.+Taleleauthor=P.+Aroraauthor=N.+Kaushik-Basu&title=Characterization+of+aurintricarboxylic+acid+as+a+potent+hepatitis+C+virus+replicase+inhibitor&doi=10.3851%2FIMP1286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of aurintricarboxylic acid as a potent hepatitis C virus replicase inhibitor</span></div><div class="casAuthors">Chen, Ye; Bopda-Waffo, Alain; Basu, Amartya; Krishnan, Ramalingam; Silberstein, Erica; Taylor, Deborah R.; Talele, Tanaji; Arora, Payal; Kaushik-Basu, Neerja</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Chemistry & Chemotherapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-36</span>CODEN:
                <span class="NLM_cas:coden">ACCHEH</span>;
        ISSN:<span class="NLM_cas:issn">0956-3202</span>.
    
            (<span class="NLM_cas:orgname">International Medical Press, Ltd.</span>)
        </div><div class="casAbstract">Background: Hepatitis C virus (HCV) NS5B is an essential component of the viral replication machinery and an important target for antiviral intervention.  Aurintricarboxylic acid (ATA), a broad-spectrum antiviral agent, was evaluated and characterized for its anti-NS5B activity in vitro and in HCV replicon cells.  Methods: Recombinant NS5B, HCV replicase and Huh-7 cells harboring the subgenomic HCV replicon of genotype 1b were employed for biochem. and mechanistic investigations.  Results: Anal. of ATA activity in vitro yielded equipotent inhibition of recombinant NS5B and HCV replicase in the submicromolar range (50% inhibition concn. [IC50] approx. 150 nM).  Biochem. and mechanistic studies revealed a bimodal mechanism of ATA inhibition with characteristics of pyrophosphate mimics and non-nucleoside inhibitors.  Mol. modeling and competition displacement studies were consistent with these parameters, suggesting that ATA might bind to the benzothiadiazine allosteric pocket 3 of NS5B or at its catalytic center.  Kinetic studies revealed a mixed mode of ATA inhibition with respect to both RNA and UTP substrates.  Under single-cycle assay conditions, ATA inhibited HCV NS5B initiation and elongation from pre-bound RNA, but with ≥fivefold decreased potency compared with continuous polymn. conditions.  The IC50 value of ATA for the native replicase complex was 145 nM.  In HCV replicon cells, ATA treatment ablated HCV RNA replication (50% effective concn. = 75 nM) with concomitant decrease in NS5B expression and no apparent cytotoxic effects.  Conclusions: This study identified ATA as a potent anti-NS5B inhibitor and suggests that its unique mode of action might be exploited for structural refinement and development of novel anti-NS5B agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_z7hctkbzBLVg90H21EOLACvtfcHk0ljqlMpnC4nEEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSlurjJ&md5=985e30923e99a6992e1b041e8c4065ae</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.3851%2FIMP1286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3851%252FIMP1286%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBopda-Waffo%26aufirst%3DA.%26aulast%3DBasu%26aufirst%3DA.%26aulast%3DKrishnan%26aufirst%3DR.%26aulast%3DSilberstein%26aufirst%3DE.%26aulast%3DTaylor%26aufirst%3DD.%2BR.%26aulast%3DTalele%26aufirst%3DT.%2BT.%26aulast%3DArora%26aufirst%3DP.%26aulast%3DKaushik-Basu%26aufirst%3DN.%26atitle%3DCharacterization%2520of%2520aurintricarboxylic%2520acid%2520as%2520a%2520potent%2520hepatitis%2520C%2520virus%2520replicase%2520inhibitor%26jtitle%3DAntivir.%2520Chem.%2520Chemother.%26date%3D2009%26volume%3D20%26spage%3D19%26epage%3D36%26doi%3D10.3851%2FIMP1286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traykova-Adonova, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutts, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grudesky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschambaul, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drebot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span> </span><span class="NLM_article-title">Potent and selective inhibition of SARS coronavirus replication by aurintricarboxylic acid</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">1199</span><span class="NLM_x">–</span> <span class="NLM_lpage">1203</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2004.06.076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.bbrc.2004.06.076" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2004&pages=1199-1203&author=R.+Heauthor=A.+Adonovauthor=M.+Traykova-Adonovaauthor=J.+Caoauthor=T.+Cuttsauthor=E.+Grudeskyauthor=Y.+Deschambaulauthor=J.+Berryauthor=M.+Drebotauthor=X.+Li&title=Potent+and+selective+inhibition+of+SARS+coronavirus+replication+by+aurintricarboxylic+acid&doi=10.1016%2Fj.bbrc.2004.06.076"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2004.06.076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2004.06.076%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DR.%26aulast%3DAdonov%26aufirst%3DA.%26aulast%3DTraykova-Adonova%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCutts%26aufirst%3DT.%26aulast%3DGrudesky%26aufirst%3DE.%26aulast%3DDeschambaul%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DJ.%26aulast%3DDrebot%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26atitle%3DPotent%2520and%2520selective%2520inhibition%2520of%2520SARS%2520coronavirus%2520replication%2520by%2520aurintricarboxylic%2520acid%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2004%26volume%3D320%26spage%3D1199%26epage%3D1203%26doi%3D10.1016%2Fj.bbrc.2004.06.076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Myskiw, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deschambault, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferies, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, J.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid inhibits the early stage of vaccinia virus replication by targeting both cellular and viral factors</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">3027</span><span class="NLM_x">–</span> <span class="NLM_lpage">3032</span><span class="refDoi"> DOI: 10.1128/JVI.02531-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1128%2FJVI.02531-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=17192307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1GrtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=3027-3032&author=C.+Myskiwauthor=Y.+Deschambaultauthor=K.+Jefferiesauthor=R.+Heauthor=J.+Cao&title=Aurintricarboxylic+acid+inhibits+the+early+stage+of+vaccinia+virus+replication+by+targeting+both+cellular+and+viral+factors&doi=10.1128%2FJVI.02531-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Aurintricarboxylic acid inhibits the early stage of vaccinia virus replication by targeting both cellular and viral factors</span></div><div class="casAuthors">Myskiw, Chad; Deschambault, Yvon; Jefferies, Kristel; He, Runtao; Cao, Jingxin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3027-3032</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Aurintricarboxylic acid (ATA) has been shown to inhibit the replication of viruses from several different families, including human immunodeficiency virus, vesicular stomatitis virus, and the coronavirus causing severe acute respiratory syndrome.  This study characterizes the inhibitory effect of ATA on vaccinia virus replication in HeLa, Huh7, and AD293 cells.  Vaccinia virus replication is significantly abrogated upon ATA treatment, which is assocd. with the inhibition of early viral gene transcription.  This inhibitory effect may be attributed to two findings.  First, ATA blocks the phosphorylation of extracellular signal-regulated kinase 1/2, an event shown to be essential for vaccinia virus replication.  Second, ATA inhibits the phosphatase activity of the viral enzyme H1L, which is required to initiate viral transcription.  Thus, ATA inhibits vaccinia virus replication by targeting both cellular and viral factors essential for the early stage of replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq6eG_fwi1mLVg90H21EOLACvtfcHk0liqG1-Ea29r5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1GrtLc%253D&md5=198029b1652f9388e88bc027a37fabed</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1128%2FJVI.02531-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02531-06%26sid%3Dliteratum%253Aachs%26aulast%3DMyskiw%26aufirst%3DC.%26aulast%3DDeschambault%26aufirst%3DY.%26aulast%3DJefferies%26aufirst%3DK.%26aulast%3DHe%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DJ.%26atitle%3DAurintricarboxylic%2520acid%2520inhibits%2520the%2520early%2520stage%2520of%2520vaccinia%2520virus%2520replication%2520by%2520targeting%2520both%2520cellular%2520and%2520viral%2520factors%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D3027%26epage%3D3032%26doi%3D10.1128%2FJVI.02531-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Yap, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, R.</span><span> </span><span class="NLM_article-title">Structural analysis of inhibition mechanisms of aurintricarboxylic acid on SARS-CoV polymerase and other proteins</span> <span class="citation_source-journal">Comput. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">212</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1016/j.compbiolchem.2005.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.compbiolchem.2005.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15979041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsVeqtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2005&pages=212-219&author=Y.+Yapauthor=X.+Zhangauthor=A.+Andonovauthor=R.+He&title=Structural+analysis+of+inhibition+mechanisms+of+aurintricarboxylic+acid+on+SARS-CoV+polymerase+and+other+proteins&doi=10.1016%2Fj.compbiolchem.2005.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of inhibition mechanisms of Aurintricarboxylic Acid on SARS-CoV polymerase and other proteins</span></div><div class="casAuthors">Yap, YeeLeng; Zhang, XueWu; Andonov, Anton; He, RunTao</div><div class="citationInfo"><span class="NLM_cas:title">Computational Biology and Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">212-219</span>CODEN:
                <span class="NLM_cas:coden">CBCOCH</span>;
        ISSN:<span class="NLM_cas:issn">1476-9271</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The authors recently published exptl. results that indicated Aurintricarboxylic Acid (ATA) could selectively inhibit SARS-Co-V replication inside host cells by greater than 1000 times.  This inhibition suggested that ATA could be developed as potent antiviral drug.  Here, to extend our exptl. observation, the authors have incorporated protein structural studies (with pos./neg. controls) to investigate the potential binding modes/sites of ATA onto RNA-dependent RNA polymerase (RdRp) from SARS-Co-V and other pathogenic pos.-strand RNA-viruses, as well as other proteins in SARS-Co-V based on the fact that ATA binds to Ca2+-activated neutral protease (m-calpain), the protein tyrosine phosphatase (PTP) and HIV integrase which have existing crystal structures.  Eight regions with homologous 3D-conformation were derived for 10 proteins of interest.  One of the region, Rbinding (754-766 in SARS-Co-V's RdRp), located in the palm sub-domain mainly constituted of antiparallel β-strand-turn-β-strand hairpin structures that covers two of the three RdRp catalytic sites (Asp 760, Asp761), was also predicted by mol. docking method (based on free energy of binding ΔG) to be important binding motif recognized by ATA.  The existence of this strictly conserved region that incorporated catalytic residues, coupled with the homologous ATA binding pockets and their consistent ΔG values, suggested strongly ATA may be involved in an analogous inhibition mechanism of SARS-COV's RdRp in concomitant to the case in m-calpain, PTP and HIV integrase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXw0p-RxaEGbVg90H21EOLACvtfcHk0liqG1-Ea29r5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsVeqtrg%253D&md5=36b9b416ce9c4405bba137bc52c7923f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.compbiolchem.2005.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.compbiolchem.2005.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DYap%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DAndonov%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DR.%26atitle%3DStructural%2520analysis%2520of%2520inhibition%2520mechanisms%2520of%2520aurintricarboxylic%2520acid%2520on%2520SARS-CoV%2520polymerase%2520and%2520other%2520proteins%26jtitle%3DComput.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D29%26spage%3D212%26epage%3D219%26doi%3D10.1016%2Fj.compbiolchem.2005.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Shadrick, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frick, D. N.</span><span> </span><span class="NLM_article-title">Aurintricarboxylic acid modulates the affinity of hepatitis C virus NS3 helicase for both nucleic acid and ATP</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6151</span><span class="NLM_x">–</span> <span class="NLM_lpage">6159</span><span class="refDoi"> DOI: 10.1021/bi4006495</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4006495" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6151-6159&author=W.+R.+Shadrickauthor=S.+Mukherjeeauthor=A.+M.+Hansonauthor=N.+L.+Sweeneyauthor=D.+N.+Frick&title=Aurintricarboxylic+acid+modulates+the+affinity+of+hepatitis+C+virus+NS3+helicase+for+both+nucleic+acid+and+ATP&doi=10.1021%2Fbi4006495"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi4006495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4006495%26sid%3Dliteratum%253Aachs%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BM.%26aulast%3DSweeney%26aufirst%3DN.%2BL.%26aulast%3DFrick%26aufirst%3DD.%2BN.%26atitle%3DAurintricarboxylic%2520acid%2520modulates%2520the%2520affinity%2520of%2520hepatitis%2520C%2520virus%2520NS3%2520helicase%2520for%2520both%2520nucleic%2520acid%2520and%2520ATP%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D6151%26epage%3D6159%26doi%3D10.1021%2Fbi4006495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">iGEMDOCK: a graphical environment of enhancing GEMDOCK using pharmacological interactions and post-screening analysis</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S33</span><span class="refDoi"> DOI: 10.1186/1471-2105-12-S1-S33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1186%2F1471-2105-12-S1-S33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21342564" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&issue=Suppl.+1&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+a+graphical+environment+of+enhancing+GEMDOCK+using+pharmacological+interactions+and+post-screening+analysis&doi=10.1186%2F1471-2105-12-S1-S33"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-S1-S33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-S1-S33%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520a%2520graphical%2520environment%2520of%2520enhancing%2520GEMDOCK%2520using%2520pharmacological%2520interactions%2520and%2520post-screening%2520analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26issue%3DSuppl.%25201%26spage%3DS33%26doi%3D10.1186%2F1471-2105-12-S1-S33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Hsiao, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H. S.</span><span> </span><span class="NLM_article-title">Structural basis for RNA trimming by RNase T in stable RNA 3′-end maturation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">236</span><span class="NLM_x">–</span> <span class="NLM_lpage">243</span><span class="refDoi"> DOI: 10.1038/nchembio.524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnchembio.524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=21317904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=236-243&author=Y.+Y.+Hsiaoauthor=C.+C.+Yangauthor=C.+L.+Linauthor=J.+L.+Linauthor=Y.+Duhauthor=H.+S.+Yuan&title=Structural+basis+for+RNA+trimming+by+RNase+T+in+stable+RNA+3%E2%80%B2-end+maturation&doi=10.1038%2Fnchembio.524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for RNA trimming by RNase T in stable RNA 3'-end maturation</span></div><div class="casAuthors">Hsiao, Yu-Yuan; Yang, Che-Chuan; Lin, Chia-Liang; Lin, Jason L.-J.; Duh, Yulander; Yuan, Hanna S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">236-243</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA maturation relies on various exonucleases to remove nucleotides successively from the 5' or 3' end of nucleic acids.  However, little is known regarding the mol. basis for substrate and cleavage preference of exonucleases.  Our biochem. and structural analyses on RNase T-DNA complexes show that the RNase T dimer has an ideal architecture for binding a duplex with a short 3' overhang to produce a digestion product of a duplex with a 2-nucleotide (nt) or 1-nt 3' overhang, depending on the compn. of the last base pair in the duplex.  A 'C-filter' in RNase T screens out the nucleic acids with 3'-terminal cytosines for hydrolysis by inducing a disruptive conformational change at the active site.  Our results reveal the general principles and the working mechanism for the final trimming step made by RNase T in the maturation of stable RNA and pave the way for the understanding of other DEDD family exonucleases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3kWl8g05MgLVg90H21EOLACvtfcHk0libRy92bpm1vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFSjtLY%253D&md5=b8e87a61752441f3313028889090485f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.524%26sid%3Dliteratum%253Aachs%26aulast%3DHsiao%26aufirst%3DY.%2BY.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DC.%2BL.%26aulast%3DLin%26aufirst%3DJ.%2BL.%26aulast%3DDuh%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DH.%2BS.%26atitle%3DStructural%2520basis%2520for%2520RNA%2520trimming%2520by%2520RNase%2520T%2520in%2520stable%2520RNA%25203%25E2%2580%25B2-end%2520maturation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D236%26epage%3D243%26doi%3D10.1038%2Fnchembio.524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">de Silva, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhury, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harvey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrino, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollis, T.</span><span> </span><span class="NLM_article-title">The crystal structure of TREX1 explains the 3′ nucleotide specificity and reveals a polyproline II helix for protein partnering</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">10537</span><span class="NLM_x">–</span> <span class="NLM_lpage">10543</span><span class="refDoi"> DOI: 10.1074/jbc.M700039200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M700039200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=10537-10543&author=U.+de+Silvaauthor=S.+Choudhuryauthor=S.+L.+Baileyauthor=S.+Harveyauthor=F.+W.+Perrinoauthor=T.+Hollis&title=The+crystal+structure+of+TREX1+explains+the+3%E2%80%B2+nucleotide+specificity+and+reveals+a+polyproline+II+helix+for+protein+partnering&doi=10.1074%2Fjbc.M700039200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M700039200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M700039200%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSilva%26aufirst%3DU.%26aulast%3DChoudhury%26aufirst%3DS.%26aulast%3DBailey%26aufirst%3DS.%2BL.%26aulast%3DHarvey%26aufirst%3DS.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DHollis%26aufirst%3DT.%26atitle%3DThe%2520crystal%2520structure%2520of%2520TREX1%2520explains%2520the%25203%25E2%2580%25B2%2520nucleotide%2520specificity%2520and%2520reveals%2520a%2520polyproline%2520II%2520helix%2520for%2520protein%2520partnering%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D10537%26epage%3D10543%26doi%3D10.1074%2Fjbc.M700039200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Strategies in the design of antiviral drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">13</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1038/nrd703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1038%2Fnrd703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12119605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1Wqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=13-25&author=E.+De+Clercq&title=Strategies+in+the+design+of+antiviral+drugs&doi=10.1038%2Fnrd703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies in the design of antiviral drugs</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-25</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A decade ago, just five drugs were licensed for the treatment of viral infections.  Since then, greater understanding of viral life cycles, prompted in particular by the need to combat human immunodeficiency virus, has resulted in the discovery and validation of several targets for therapeutic intervention.  Consequently, the current antiviral repertoire now includes more than 30 drugs.  But we still lack effective therapies for several viral infections, and established treatments are not always effective or well tolerated, highlighting the need for further refinement of antiviral drug design and development.  Here, I describe the rationale behind current and future drug-based strategies for combating viral infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2t4izHL0-7Vg90H21EOLACvtfcHk0libRy92bpm1vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhs1Wqt7s%253D&md5=8ee06ad97a7859c1eed044ce1db62fdf</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnrd703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd703%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DStrategies%2520in%2520the%2520design%2520of%2520antiviral%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D13%26epage%3D25%26doi%3D10.1038%2Fnrd703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">De Clercq, E.</span><span> </span><span class="NLM_article-title">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span> <span class="citation_source-journal">Int. J. Antimicrob. Agents</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">320</span><span class="refDoi"> DOI: 10.1016/j.ijantimicag.2008.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.ijantimicag.2008.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=19108994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=307-320&author=E.+De+Clercq&title=Anti-HIV+drugs%3A+25+compounds+approved+within+25+years+after+the+discovery+of+HIV&doi=10.1016%2Fj.ijantimicag.2008.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-HIV drugs: 25 compounds approved within 25 years after the discovery of HIV</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-320</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In 2008, 25 years after the human immunodeficiency virus (HIV) was discovered as the then tentative etiol. agent of acquired immune deficiency syndrome (AIDS), exactly 25 anti-HIV compds. have been formally approved for clin. use in the treatment of AIDS.  These compds. fall into six categories: nucleoside reverse transcriptase inhibitors (NRTIs: zidovudine, didanosine, zalcitabine, stavudine, lamivudine, abacavir and emtricitabine); nucleotide reverse transcriptase inhibitors (NtRTIs: tenofovir); non-nucleoside reverse transcriptase inhibitors (NNRTIs: nevirapine, delavirdine, efavirenz and etravirine); protease inhibitors (PIs: saquinavir, ritonavir, indinavir, nelfinavir, amprenavir, lopinavir, atazanavir, fosamprenavir, tipranavir and darunavir); cell entry inhibitors [fusion inhibitors (FIs: enfuvirtide) and co-receptor inhibitors (CRIs: maraviroc)]; and integrase inhibitors (INIs: raltegravir).  These compds. should be used in drug combination regimens to achieve the highest possible benefit, tolerability and compliance and to diminish the risk of resistance development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsT6dqAjvKIrVg90H21EOLACvtfcHk0libRy92bpm1vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXivVKgs7g%253D&md5=6d8263455e9c912d353ef2ea230a5e15</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2008.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2008.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DAnti-HIV%2520drugs%253A%252025%2520compounds%2520approved%2520within%252025%2520years%2520after%2520the%2520discovery%2520of%2520HIV%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2009%26volume%3D33%26spage%3D307%26epage%3D320%26doi%3D10.1016%2Fj.ijantimicag.2008.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Ison, M. G.</span><span> </span><span class="NLM_article-title">Antivirals and resistance: influenza virus</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">573</span><span class="refDoi"> DOI: 10.1016/j.coviro.2011.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2011.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22440914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=563-573&author=M.+G.+Ison&title=Antivirals+and+resistance%3A+influenza+virus&doi=10.1016%2Fj.coviro.2011.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Antivirals and resistance: influenza virus</span></div><div class="casAuthors">Ison, Michael G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-573</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Influenza causes annual epidemics of respiratory viral infections are assocd. with significant morbidity and mortality.  Influenza vaccines have been shown to reduce the risk of infection and mitigate against some of the virus' sequellae.  Likewise, two classes of antivirals, the adamantanes (amantadine and rimantadine) and the neuraminidase inhibitors (laninamivir, oseltamivir, peramivir, and zanamivir) are currently approved for the prevention and treatment of influenza; several other classes of antivirals and immune modulators are also currently under investigation.  One of the greatest challenges to our armamentarium of antivirals is the emergence of resistant mutants.  In this paper, we will review the currently approved and investigational antiviral agents and the mechanisms of resistance that impact their activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMhlhzdOc5lLVg90H21EOLACvtfcHk0ljw6-rLVo-QEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrfE&md5=7414bf2c9f6d902f7ba696ff3c8bf63e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2011.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2011.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DIson%26aufirst%3DM.%2BG.%26atitle%3DAntivirals%2520and%2520resistance%253A%2520influenza%2520virus%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2011%26volume%3D1%26spage%3D563%26epage%3D573%26doi%3D10.1016%2Fj.coviro.2011.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Samson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pizzorno, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abed, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boivin, G.</span><span> </span><span class="NLM_article-title">Influenza virus resistance to neuraminidase inhibitors</span> <span class="citation_source-journal">Antiviral Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">174</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1016/j.antiviral.2013.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.antiviral.2013.03.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=174-185&author=M.+Samsonauthor=A.+Pizzornoauthor=Y.+Abedauthor=G.+Boivin&title=Influenza+virus+resistance+to+neuraminidase+inhibitors&doi=10.1016%2Fj.antiviral.2013.03.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2013.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2013.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DSamson%26aufirst%3DM.%26aulast%3DPizzorno%26aufirst%3DA.%26aulast%3DAbed%26aufirst%3DY.%26aulast%3DBoivin%26aufirst%3DG.%26atitle%3DInfluenza%2520virus%2520resistance%2520to%2520neuraminidase%2520inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D98%26spage%3D174%26epage%3D185%26doi%3D10.1016%2Fj.antiviral.2013.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Hosseinipour, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Zyl, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachega, J. B.</span><span> </span><span class="NLM_article-title">Emergence of HIV drug resistance during first- and second-line antiretroviral therapy in resource-limited settings</span> <span class="citation_source-journal">J. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">207</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 2</span><span class="NLM_x">) </span> <span class="NLM_fpage">S49</span><span class="NLM_x">–</span> <span class="NLM_lpage">S56</span><span class="refDoi"> DOI: 10.1093/infdis/jit107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Finfdis%2Fjit107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23687289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXotlOgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2013&pages=S49-S56&issue=Suppl.+2&author=M.+C.+Hosseinipourauthor=R.+K.+Guptaauthor=G.+Van+Zylauthor=J.+J.+Eronauthor=J.+B.+Nachega&title=Emergence+of+HIV+drug+resistance+during+first-+and+second-line+antiretroviral+therapy+in+resource-limited+settings&doi=10.1093%2Finfdis%2Fjit107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of HIV Drug Resistance During First- and Second-Line Antiretroviral Therapy in Resource-Limited Settings</span></div><div class="casAuthors">Hosseinipour, Mina C.; Gupta, Ravindra K.; Van Zyl, Gert; Eron, Joseph J.; Nachega, Jean B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S49-S56</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Introduction Antiretroviral therapy (ART) in resource-limited settings has expanded in the last decade, reaching >8 million individuals and reducing AIDS mortality and morbidity.  Continued success of ART programs will require understanding the emergence of HIV drug resistance patterns among individuals in whom treatment has failed and managing ART from both an individual and public health perspective.  We review data on the emergence of HIV drug resistance among individuals in whom first-line therapy has failed and clin. and resistance outcomes of those receiving second-line therapy in resource-limited settings.  Results Resistance surveys among patients initiating first-line nonnucleoside reverse-transcriptase inhibitor (NNRTI)-based therapy suggest that 76%-90% of living patients achieve HIV RNA suppression by 12 mo after ART initiation.  Among patients with detectable HIV RNA at 12 mo, HIV drug resistance, primarily due to M184V and NNRTI mutations, has been identified in 60%-72%, although the antiretroviral activity of proposed second-line regimens has been preserved.  Complex mutation patterns, including thymidine-analog mutations, K65R, and multinucleoside mutations, are prevalent among cases of treatment failure identified by clin. or immunol. methods.  Approx. 22% of patients receiving second-line therapy do not achieve HIV RNA suppression by 6 mo, with poor adherence, rather than HIV drug resistance, driving most failures.  Major protease inhibitor resistance at the time of second-line failure ranges from 0% to 50%, but studies are limited.  Conclusions Resistance of HIV to first-line therapy is predictable at 12 mo when evaluated by means of HIV RNA monitoring and, when detected, largely preserves second-line therapy options.  Optimizing adherence, performing resistance surveillance, and improving treatment monitoring are crit. for long-term prevention of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHVDfsLEAp7Vg90H21EOLACvtfcHk0ljw6-rLVo-QEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXotlOgtrg%253D&md5=1b90c5824f6c03db0d3156563d4f4a9b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjit107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjit107%26sid%3Dliteratum%253Aachs%26aulast%3DHosseinipour%26aufirst%3DM.%2BC.%26aulast%3DGupta%26aufirst%3DR.%2BK.%26aulast%3DVan%2BZyl%26aufirst%3DG.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26aulast%3DNachega%26aufirst%3DJ.%2BB.%26atitle%3DEmergence%2520of%2520HIV%2520drug%2520resistance%2520during%2520first-%2520and%2520second-line%2520antiretroviral%2520therapy%2520in%2520resource-limited%2520settings%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D207%26issue%3DSuppl.%25202%26spage%3DS49%26epage%3DS56%26doi%3D10.1093%2Finfdis%2Fjit107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Berkhout, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggink, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, R. W.</span><span> </span><span class="NLM_article-title">Is there a future for antiviral fusion inhibitors?</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">50</span><span class="NLM_x">–</span> <span class="NLM_lpage">59</span><span class="refDoi"> DOI: 10.1016/j.coviro.2012.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2012.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=22440966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSksb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=50-59&author=B.+Berkhoutauthor=D.+Egginkauthor=R.+W.+Sanders&title=Is+there+a+future+for+antiviral+fusion+inhibitors%3F&doi=10.1016%2Fj.coviro.2012.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Is there a future for antiviral fusion inhibitors?</span></div><div class="casAuthors">Berkhout, Ben; Eggink, Dirk; Sanders, Rogier W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">50-59</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Entry of human immunodeficiency virus type 1 (HIV-1) into cells is mediated by attachment of the envelope glycoproteins, gp120 and gp41, to the CD4 receptor and a chemokine receptor (CCR5 or CXCR4) and subsequent fusion of viral and cellular membranes.  Several steps of the entry process can be targeted by drugs.  Receptor antagonists prevent virus attachment and fusion inhibitors block conformational changes that are required for membrane fusion.  The T20 peptide (Enfuvirtide, Fuzeon), which is homologous to part of the gp41-encoded fusion machinery, is the only clin. approved fusion inhibitor, but over the last decade new generations of T20-like peptides have been developed with improved potency and stability, as well as fusion inhibitors that target alternative gp41 domains.  Here, we will review the field of HIV-1 fusion inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUhAj7F6DFc7Vg90H21EOLACvtfcHk0ljw6-rLVo-QEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSksb4%253D&md5=a9fe1bb65fa0e440015402fce8b5d86c</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2012.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2012.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DBerkhout%26aufirst%3DB.%26aulast%3DEggink%26aufirst%3DD.%26aulast%3DSanders%26aufirst%3DR.%2BW.%26atitle%3DIs%2520there%2520a%2520future%2520for%2520antiviral%2520fusion%2520inhibitors%253F%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2012%26volume%3D2%26spage%3D50%26epage%3D59%26doi%3D10.1016%2Fj.coviro.2012.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Das, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, E.</span><span> </span><span class="NLM_article-title">HIV-1 reverse transcriptase and antiviral drug resistance. Part 1</span> <span class="citation_source-journal">Curr. Opin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">118</span><span class="refDoi"> DOI: 10.1016/j.coviro.2013.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2Fj.coviro.2013.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=23602471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsFanur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=111-118&author=K.+Dasauthor=E.+Arnold&title=HIV-1+reverse+transcriptase+and+antiviral+drug+resistance.+Part+1&doi=10.1016%2Fj.coviro.2013.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 reverse transcriptase and antiviral drug resistance. Part 1</span></div><div class="casAuthors">Das, Kalyan; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-118</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  HIV-1 reverse transcriptase (RT) contributes to the development of resistance to all anti-AIDS drugs by introducing mutations into the viral genome.  At the mol. level, mutations in RT result in resistance to RT inhibitors.  Eight nucleoside/nucleotide analogs (NRTIs) and five non-nucleoside inhibitors (NNRTIs) are approved HIV-1 drugs.  Structures of RT have been detd. in complexes with substrates and/or inhibitors, and the structures have illuminated different conformational and functional states of the enzyme.  Understanding the mol. mechanisms of resistance to NRTIs and NNRTIs, and their complex relationships, may help in designing new drugs that are periodically required to overcome existing as well as emerging trends of drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKm2E0DCBa77Vg90H21EOLACvtfcHk0li5eNIanA5kYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsFanur0%253D&md5=7c1049b8cdad61bdd7d73c0aefbe496e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2013.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2013.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHIV-1%2520reverse%2520transcriptase%2520and%2520antiviral%2520drug%2520resistance.%2520Part%25201%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2013%26volume%3D3%26spage%3D111%26epage%3D118%26doi%3D10.1016%2Fj.coviro.2013.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Bean, P.</span><span> </span><span class="NLM_article-title">New drug targets for HIV</span> <span class="citation_source-journal">Clin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S96</span><span class="NLM_x">–</span> <span class="NLM_lpage">S100</span><span class="refDoi"> DOI: 10.1086/429504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1086%2F429504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=16265623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtValurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=S96-S100&issue=Suppl.+1&author=P.+Bean&title=New+drug+targets+for+HIV&doi=10.1086%2F429504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">New drug targets for HIV</span></div><div class="casAuthors">Bean, Pamela</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S96-S100</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">A review.  A significant no. of human immunodeficiency virus (HIV) infections have become resistant to antiretroviral treatment, which means that there is a paramount need for novel drug targets to defeat the virus.  Until recently, all HIV drugs inhibited HIV replication by mechanisms operating inside infected cells.  In contrast, new antiretroviral drugs operate outside infected cells.  Their mechanism of action consists in inhibiting entry of the virus into cells, thereby halting the very first step of HIV replication.  Examples of this new class of drugs include entry inhibitors, coreceptor antagonists, and fusion inhibitors.  In addn. to their novel mechanism of action, this new class of drugs also has potential action against drug-resistant HIV strains, causes minimal adverse effects, and may be administered in a simplified, once-daily dosing regimen.  New classes of anti-HIV drugs-and new drugs in existing classes-represent the best hope for people infected with HIV, esp. those who have exhausted current therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIMgCCnqQaHbVg90H21EOLACvtfcHk0li5eNIanA5kYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtValurk%253D&md5=0c76d4c0bfd32bc75d9f6a3ec4dff808</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1086%2F429504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F429504%26sid%3Dliteratum%253Aachs%26aulast%3DBean%26aufirst%3DP.%26atitle%3DNew%2520drug%2520targets%2520for%2520HIV%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2005%26volume%3D41%26issue%3DSuppl.%25201%26spage%3DS96%26epage%3DS100%26doi%3D10.1086%2F429504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Potterton, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turkenburg, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E.</span><span> </span><span class="NLM_article-title">A graphical user interface to the CCP4 program suite</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1131</span><span class="NLM_x">–</span> <span class="NLM_lpage">1137</span><span class="refDoi"> DOI: 10.1107/S0907444903008126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444903008126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=12832755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=1131-1137&author=E.+Pottertonauthor=P.+Briggsauthor=M.+Turkenburgauthor=E.+Dodson&title=A+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS0907444903008126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Elizabeth; Briggs, Peter; Turkenburg, Maria; Dodson, Eleanor</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1131-1137</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4i is a graphical user interface that makes running programs from the CCP4 suite simpler and quicker.  It is particularly directed at inexperienced users and tightly linked to introductory and scientific documentation.  It also provides a simple project-management system and visualization tools.  The system is readily extensible and not specific to CCP4 software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDH-BQSvi5qbVg90H21EOLACvtfcHk0li5eNIanA5kYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D&md5=25a0168dd7ea9f930036453a829e1e3d</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444903008126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903008126%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DBriggs%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DE.%26atitle%3DA%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D1131%26epage%3D1137%26doi%3D10.1107%2FS0907444903008126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li5eNIanA5kYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgQMZK8HnxKXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span> <span class="citation_source-journal">Protein Eng., Des. Sel.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">747</span><span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=9796821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK1cXmsVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Protein structure alignment by incremental combinatorial extension (CE) of the optimal path</span></div><div class="casAuthors">Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">739-747</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new algorithm is reported which builds an alignment between two protein structures.  The algorithm involves a combinatorial extension (CE) of an alignment path defined by aligned fragment pairs (AFPs) rather than the more conventional techniques using dynamic programming and Monte Carlo optimization.  AFPs, as the name suggests, are pairs of fragments, one from each protein, which confer structure similarity.  AFPs are based on local geometry, rather than global features such as orientation of secondary structures and overall topol.  Combinations of AFPs that represent possible continuous alignment paths are selectively extended or discarded thereby leading to a single optimal alignment.  The algorithm is fast and accurate in finding an optimal structure alignment and hence suitable for database scanning and detailed anal. of large protein families.  The method has been tested and compared with results from Dali and VAST using a representative sample of similar structures.  Several new structural similarities not detected by these other methods are reported.  Specific one-on-one alignments and searches against all structures as found in the Protein Data Bank (PDB) can be performed via the Web at http://cl.sdsc.edu/ce.html.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxq1Bh1NtRcbVg90H21EOLACvtfcHk0lgQMZK8HnxKXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsVynsbk%253D&md5=ef3e3dc633168dfd49cc871a72608727</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520structure%2520alignment%2520by%2520incremental%2520combinatorial%2520extension%2520%2528CE%2529%2520of%2520the%2520optimal%2520path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Sadowski, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Comparison of Automatic Three-Dimensional Model Builders Using 639 X-ray Structures</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1000</span><span class="NLM_x">–</span> <span class="NLM_lpage">1008</span><span class="refDoi"> DOI: 10.1021/ci00020a039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci00020a039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADyaK2cXkvVajsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=1000-1008&author=J.+Sadowskiauthor=J.+Gasteigerauthor=G.+Klebe&title=Comparison+of+Automatic+Three-Dimensional+Model+Builders+Using+639+X-ray+Structures&doi=10.1021%2Fci00020a039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Automatic Three-Dimensional Model Builders Using 639 X-ray Structures</span></div><div class="casAuthors">Sadowski, Jens; Gasteiger, Johann; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1000-8</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">Several criteria were defined to select a dataset of high-quality x-ray structures from the Cambridge file resulting in 639 mols.  Six currently available programs for automatic 3D structure generation were compared by converting the connectivity tables, including appropriate stereodescriptors from this dataset of 639 mol. structures into 3D geometries: CONCORD, ALCOGEN, Chem-X, MOLGEO, COBRA, and CORINA.  The geometries produced by the different programs were evaluated in terms of several quality criteria and discussed in detail.  These criteria measure how well the different programs reproduce the x-ray geometries of the 639 input structures.  Accordingly, the major strengths and weaknesses of the programs are indicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8vEtyDrrlq7Vg90H21EOLACvtfcHk0lgQMZK8HnxKXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkvVajsbY%253D&md5=246e4d66f11178a18daad2563c48331c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci00020a039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00020a039%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DJ.%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DComparison%2520of%2520Automatic%2520Three-Dimensional%2520Model%2520Builders%2520Using%2520639%2520X-ray%2520Structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D1994%26volume%3D34%26spage%3D1000%26epage%3D1008%26doi%3D10.1021%2Fci00020a039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span> </span><span class="NLM_article-title">GEMDOCK: a generic evolutionary method for molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span><span class="refDoi"> DOI: 10.1002/prot.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1002%2Fprot.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=15048822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=288-304&author=J.+M.+Yangauthor=C.+C.+Chen&title=GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking&doi=10.1002%2Fprot.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">GEMDOCK: a generic evolutionary method for molecular docking</span></div><div class="casAuthors">Yang, Jinn-Moon; Chen, Chun-Chen</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">288-304</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We have developed an evolutionary approach for flexible ligand docking.  This approval, GEMDOCK, uses a Generic Evolutionary Method for mol. DOCKing and an empirical scoring function.  The former combines both discrete and continuous global search strategies with local search strategies to speed up convergence, whereas the latter results in rapid recognition of potential ligands.  GEMDOCK was tested on a diverse data set of 100 protein-ligand complexes from the Protein Data Bank.  In 79% of these complexes, the docked lowest energy ligand structures had root-mean-square derivations (RMSDs) below 2.0 Å with respect to the corresponding crystal structures.  The success rate increased to 85% if the structure water mols. were retained.  We evaluated GEMDOCK on two cross-docking expts. in which each ligand of a protein ensemble was docked into each protein of the ensemble.  Seventy-six percent of the docked structures had RMSDs below 2.0 Å when the ligands were docked into foreign structures.  We analyzed and validated GEMDOCK with respect to various search spaces and scoring functions, and found that if the scoring function was perfect, then the predicted accuracy was also essentially perfect.  This study suggests that GEMDOCK is a useful tool for mol. recognition and may be used to systematically evaluate and thus improve scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_k6KGw_uV-bVg90H21EOLACvtfcHk0lgDzbHlz5_Zvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsro%253D&md5=02248206ca276b2da2b784224d087f12</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fprot.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20035%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BC.%26atitle%3DGEMDOCK%253A%2520a%2520generic%2520evolutionary%2520method%2520for%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2004%26volume%3D55%26spage%3D288%26epage%3D304%26doi%3D10.1002%2Fprot.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Pike, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brzozowski, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorsell, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gustafsson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlquist, M.</span><span> </span><span class="NLM_article-title">Structural insights into the mode of action of a pure antiestrogen</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">145</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/S0969-2126(01)00568-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0969-2126%2801%2900568-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=11250199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhslKgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=145-153&author=A.+C.+Pikeauthor=A.+M.+Brzozowskiauthor=J.+Waltonauthor=R.+E.+Hubbardauthor=A.+G.+Thorsellauthor=Y.+L.+Liauthor=J.+A.+Gustafssonauthor=M.+Carlquist&title=Structural+insights+into+the+mode+of+action+of+a+pure+antiestrogen&doi=10.1016%2FS0969-2126%2801%2900568-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the mode of action of a pure antiestrogen</span></div><div class="casAuthors">Pike, Ashley C. W.; Brzozowski, A. Marek; Walton, Julia; Hubbard, Roderick E.; Thorsell, Ann-Gerd; Li, Yi-Lin; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-153</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Background: Estrogens exert their effects on target tissues by binding to a nuclear transcription factor termed the estrogen receptor (ER).  Previous structural studies have demonstrated that each class of ER ligand (agonist, partial agonist, and SERM antagonist) induces distinctive orientations in the receptor's carboxy-terminal transactivation helix.  The conformation of this portion of the receptor dets. whether ER can recruit and interact with the components of the transcriptional machinery, thereby facilitating target gene expression.  Results: We have detd. the structure of rat ERβ ligand binding domain (LBD) in complex with the pure antiestrogen ICI 164,384 at 2.3 Å resoln.  The binding of this compd. to the receptor completely abolishes the assocn. between the transactivation helix (H12) and the rest of the LBD.  The structure reveals that the terminal portion of ICI's bulky side chain substituent protrudes from the hormone binding pocket, binds along the coactivator recruitment site, and phys. prevents H12 from adopting either its characteristic agonist or AF2 antagonist orientation.  Conclusions: The binding mode adopted by the pure antiestrogen is similar to that seen for other ER antagonists.  However, the size and resultant positioning of the ligand's side chain substituent produces a receptor conformation that is distinct from that adopted in the presence of other classes of ER ligands.  The novel observation that binding of ICI results in the complete destabilization of H12 provides some indications as to a possible mechanism for pure receptor antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_dBAqpVZPtrVg90H21EOLACvtfcHk0lgDzbHlz5_Zvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhslKgsrg%253D&md5=ac139f6c39a2605e15314176397f11d4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900568-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900568-8%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%2BC.%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DWalton%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DThorsell%26aufirst%3DA.%2BG.%26aulast%3DLi%26aufirst%3DY.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DStructural%2520insights%2520into%2520the%2520mode%2520of%2520action%2520of%2520a%2520pure%2520antiestrogen%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D145%26epage%3D153%26doi%3D10.1016%2FS0969-2126%2801%2900568-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Birck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poch, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mengus, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavigne, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moras, D.</span><span> </span><span class="NLM_article-title">Human TAF(II)28 and TAF(II)18 interact through a histone fold encoded by atypical evolutionary conserved motifs also found in the SPT3 family</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">249</span><span class="refDoi"> DOI: 10.1016/S0092-8674(00)81423-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1016%2FS0092-8674%2800%2981423-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1998&pages=239-249&author=C.+Birckauthor=O.+Pochauthor=C.+Romierauthor=M.+Ruffauthor=G.+Mengusauthor=A.+C.+Lavigneauthor=I.+Davidsonauthor=D.+Moras&title=Human+TAF%28II%2928+and+TAF%28II%2918+interact+through+a+histone+fold+encoded+by+atypical+evolutionary+conserved+motifs+also+found+in+the+SPT3+family&doi=10.1016%2FS0092-8674%2800%2981423-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2981423-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252981423-3%26sid%3Dliteratum%253Aachs%26aulast%3DBirck%26aufirst%3DC.%26aulast%3DPoch%26aufirst%3DO.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DRuff%26aufirst%3DM.%26aulast%3DMengus%26aufirst%3DG.%26aulast%3DLavigne%26aufirst%3DA.%2BC.%26aulast%3DDavidson%26aufirst%3DI.%26aulast%3DMoras%26aufirst%3DD.%26atitle%3DHuman%2520TAF%2528II%252928%2520and%2520TAF%2528II%252918%2520interact%2520through%2520a%2520histone%2520fold%2520encoded%2520by%2520atypical%2520evolutionary%2520conserved%2520motifs%2520also%2520found%2520in%2520the%2520SPT3%2520family%26jtitle%3DCell%26date%3D1998%26volume%3D94%26spage%3D239%26epage%3D249%26doi%3D10.1016%2FS0092-8674%2800%2981423-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Ackerman, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwatia, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savage, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonidas, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swaminathan, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, K. R.</span><span> </span><span class="NLM_article-title">Charcot-Leyden crystal protein (galectin-10) is not a dual function galectin with lysophospholipase activity but binds a lysophospholipase inhibitor in a novel structural fashion</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">14859</span><span class="NLM_x">–</span> <span class="NLM_lpage">14868</span><span class="refDoi"> DOI: 10.1074/jbc.M200221200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;key=10.1074%2Fjbc.M200221200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=14859-14868&author=S.+J.+Ackermanauthor=L.+Liuauthor=M.+A.+Kwatiaauthor=M.+P.+Savageauthor=D.+D.+Leonidasauthor=G.+J.+Swaminathanauthor=K.+R.+Acharya&title=Charcot-Leyden+crystal+protein+%28galectin-10%29+is+not+a+dual+function+galectin+with+lysophospholipase+activity+but+binds+a+lysophospholipase+inhibitor+in+a+novel+structural+fashion&doi=10.1074%2Fjbc.M200221200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M200221200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M200221200%26sid%3Dliteratum%253Aachs%26aulast%3DAckerman%26aufirst%3DS.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKwatia%26aufirst%3DM.%2BA.%26aulast%3DSavage%26aufirst%3DM.%2BP.%26aulast%3DLeonidas%26aufirst%3DD.%2BD.%26aulast%3DSwaminathan%26aufirst%3DG.%2BJ.%26aulast%3DAcharya%26aufirst%3DK.%2BR.%26atitle%3DCharcot-Leyden%2520crystal%2520protein%2520%2528galectin-10%2529%2520is%2520not%2520a%2520dual%2520function%2520galectin%2520with%2520lysophospholipase%2520activity%2520but%2520binds%2520a%2520lysophospholipase%2520inhibitor%2520in%2520a%2520novel%2520structural%2520fashion%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D14859%26epage%3D14868%26doi%3D10.1074%2Fjbc.M200221200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CG7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CG7','PDB','3CG7'); return false;">PDB: 3CG7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CM5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CM5','PDB','3CM5'); return false;">PDB: 3CM5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3V9W','PDB','3V9W'); return false;">PDB: 3V9W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L8R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L8R','PDB','4L8R'); return false;">PDB: 4L8R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DK5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DK5','PDB','5DK5'); return false;">PDB: 5DK5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i15"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00794">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28578"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00794">10.1021/acs.jmedchem.6b00794</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables S1–S3 listing candidate inhibitors, general base residues, and crystal data; Figures S1–S5 showing IC<sub>50</sub>, docking models, general base residues, structural comparisons, and SDS–PAGE analysis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00794/suppl_file/jm6b00794_si_001.pdf">jm6b00794_si_001.pdf (1.26 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB entry code for CRN-4-MES complex is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DK5">5DK5</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00794&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00794%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-17%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00794" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"667987e2497c24c7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
