
<html lang="en"     class="pb-page"  data-request-id="6b752bfd-2228-4ab6-9b9d-b3bfca2738c5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01855;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-7;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment Hits: What do They Look Like and How do They Bind?" /></meta><meta name="dc.Creator" content="Fabrizio  Giordanetto" /></meta><meta name="dc.Creator" content="Chentian  Jin" /></meta><meta name="dc.Creator" content="Lindsay  Willmore" /></meta><meta name="dc.Creator" content="Miklos  Feher" /></meta><meta name="dc.Creator" content="David E.  Shaw" /></meta><meta name="dc.Description" content="A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to fi..." /></meta><meta name="Description" content="A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to fi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 15, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01855" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01855" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01855" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01855" /></link>
        
    
    

<title>Fragment Hits: What do They Look Like and How do They Bind? | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01855" /></meta><meta property="og:title" content="Fragment Hits: What do They Look Like and How do They Bind?" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0013.jpeg" /></meta><meta property="og:description" content="A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to find and progress fragment hits could potentially be improved by enhancing our understanding of their binding properties, which to date has largely been based on tacit knowledge and reports from individual projects. In the work reported here, we systematically analyzed the molecular and binding properties of fragment hits using 489 published protein–fragment complexes. We identified a number of notable features that these hits tend to have in common, including preferences in buried surface area upon binding, hydrogen bonding and other directional interactions with the protein targets, structural topology, functional-group occurrence, and degree of carbon saturation. In the future, taking account of these preferences in designing and selecting fragments to screen against protein targets may increase the chances of success in fragment screening campaigns." /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta><meta name="twitter:description" content="A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to find and progress fragment hits could potentially be improved by enhancing our understanding of their binding properties, which to date has largely been based on tacit knowledge and reports from individual projects. In the work reported here, we systematically analyzed the molecular and binding properties of fragment hits using 489 published protein–fragment complexes. We identified a number of notable features that these hits tend to have in common, including preferences in buried surface area upon binding, hydrogen bonding and other directional interactions with the protein targets, structural topology, functional-group occurrence, and degree of carbon saturation. In the future, taking account of these preferences in designing and selecting fragments to screen against protein targets may increase the chances of success in fragment screening campaigns." /></meta><meta name="twitter:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0013.jpeg" /></meta><meta name="twitter:title" content="Fragment Hits: What do They Look Like and How do They Bind?" /></meta>
        
            <meta property="og:type" content="Article" /></meta>
        

        
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01855"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/ke1q~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-6ee29527b3d8ff00a106.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01855">
            
        





































    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-12/achre4.2021.54.issue-12/20210615/achre4.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-25/aamick.2021.13.issue-25/20210630/aamick.2021.13.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-6/aapmcd.2021.3.issue-6/20210611/aapmcd.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-6/abseba.2021.7.issue-6/20210614/abseba.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-12/accacs.2021.11.issue-12/20210618/accacs.2021.11.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-12/acncdm.2021.12.issue-12/20210616/acncdm.2021.12.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-6/aelccp.2021.6.issue-6/20210611/aelccp.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-6/aeecco.2021.1.issue-6/20210611/aeecco.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-6/aewcaa.2021.1.issue-6/20210611/aewcaa.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-6/aidcbc.2021.7.issue-6/20210611/aidcbc.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-6/amlcef.2021.3.issue-6/20210607/amlcef.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-6/amclct.2021.12.issue-6/20210610/amclct.2021.12.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-25/acsodf.2021.6.issue-25/20210629/acsodf.2021.6.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-25/ascecg.2021.9.issue-25/20210628/ascecg.2021.9.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-25/ancham.2021.93.issue-25/20210629/ancham.2021.93.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-25/bichaw.2021.60.issue-25/20210629/bichaw.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-6/bomaf6.2021.22.issue-6/20210614/bomaf6.2021.22.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-12/cmatex.2021.33.issue-12/20210622/cmatex.2021.33.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-6/cgdefu.2021.21.issue-6/20210602/cgdefu.2021.21.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-12/enfuem.2021.35.issue-12/20210617/enfuem.2021.35.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-12/esthag.2021.55.issue-12/20210615/esthag.2021.55.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-6/estlcu.2021.8.issue-6/20210608/estlcu.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-25/iecred.2021.60.issue-25/20210630/iecred.2021.60.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-12/inocaj.2021.60.issue-12/20210621/inocaj.2021.60.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-25/jafcau.2021.69.issue-25/20210630/jafcau.2021.69.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-6/jceaax.2021.66.issue-6/20210610/jceaax.2021.66.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-6/jceda8.2021.98.issue-6/20210608/jceda8.2021.98.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-6/jctcce.2021.17.issue-6/20210608/jctcce.2021.17.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-12/joceah.2021.86.issue-12/20210618/joceah.2021.86.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-24/jpcafh.2021.125.issue-24/20210624/jpcafh.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-24/jpcbfk.2021.125.issue-24/20210624/jpcbfk.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-24/jpccck.2021.125.issue-24/20210624/jpccck.2021.125.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-24/jpclcd.2021.12.issue-24/20210624/jpclcd.2021.12.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-6/jprobs.2021.20.issue-6/20210604/jprobs.2021.20.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-25/jacsat.2021.143.issue-25/20210630/jacsat.2021.143.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-6/jamsef.2021.32.issue-6/20210602/jamsef.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-25/langd5.2021.37.issue-25/20210629/langd5.2021.37.issue-25.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-12/mamobx.2021.54.issue-12/20210622/mamobx.2021.54.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-6/mpohbp.2021.18.issue-6/20210607/mpohbp.2021.18.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-12/orlef7.2021.23.issue-12/20210618/orlef7.2021.23.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-12/orgnd7.2021.40.issue-12/20210628/orgnd7.2021.40.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01855&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01855&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01855&amp;href=/doi/10.1021/acs.jmedchem.8b01855" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3381-3394</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01790" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01860" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment Hits: What do They Look Like and How do They Bind?</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Fabrizio Giordanetto</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabrizio Giordanetto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">D. E. Shaw Research, New York, New York 10036, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#8ec8efecfce7f4e7e1a0c9e7e1fceaefe0ebfafae1cecacbdde6eff9dcebfdebeffcede6a0ede1e3"><span class="__cf_email__" data-cfemail="85c3e4e7f7ecffeceaabc2eceaf7e1e4ebe0f1f1eac5c1c0d6ede4f2d7e0f6e0e4f7e6edabe6eae8">[email protected]</span></a>. Tel: (212) 478-0822. Fax: (212) 845-1822 (F.G.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabrizio++Giordanetto">Fabrizio Giordanetto</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9876-9552" title="Orcid link">http://orcid.org/0000-0001-9876-9552</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chentian Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chentian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">D. E. Shaw Research, New York, New York 10036, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chentian++Jin">Chentian Jin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lindsay Willmore</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lindsay Willmore</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">D. E. Shaw Research, New York, New York 10036, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lindsay++Willmore">Lindsay Willmore</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Miklos Feher</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Miklos Feher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">D. E. Shaw Research, New York, New York 10036, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Miklos++Feher">Miklos Feher</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">David E. Shaw</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David E. Shaw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">D. E. Shaw Research, New York, New York 10036, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, New York 10032, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#1d597c6b7479334e757c6a5d59584e757c6a4f786e787c6f7e75337e7270"><span class="__cf_email__" data-cfemail="6e2a0f18070a403d060f192e2a2b3d060f193c0b1d0b0f1c0d06400d0103">[email protected]</span></a>. Tel: (212) 478-0260. Fax: (212) 845-1286. (D.E.S.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+E.++Shaw">David E. Shaw</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8265-5761" title="Orcid link">http://orcid.org/0000-0001-8265-5761</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01855&amp;href=/doi/10.1021%2Facs.jmedchem.8b01855" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3381–3394</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 November 2018</li><li><span class="item_label"><b>Published</b> online</span>15 March 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 April 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01855" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01855</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc text">with CC-BY-NC-ND<a href="/page/policy/authorchoice_ccbyncnd_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01855"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">11642</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">26</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01855" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment Hits: What do They Look Like and How do They Bind?&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fabrizio&quot;,&quot;last_name&quot;:&quot;Giordanetto&quot;},{&quot;first_name&quot;:&quot;Chentian&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Lindsay&quot;,&quot;last_name&quot;:&quot;Willmore&quot;},{&quot;first_name&quot;:&quot;Miklos&quot;,&quot;last_name&quot;:&quot;Feher&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;E. Shaw&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;3381-3394&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01855&quot;},&quot;abstract&quot;:&quot;A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to find and progress fragment hits could potentially be improved by enhancing our understanding of their binding properties, which to date has largely been based on tacit knowledge and reports from individual projects. In the work reported here, we systematically analyzed the molecular and binding properties of fragment hits using 489 published protein–fragment complexes. We identified a number of notable features that these hits tend to have in common, including preferences in buried surface area upon binding, hydrogen bonding and other directional interactions with the protein targets, structural topology, functional-group occurrence, and degree of carbon saturation. In the future, taking account of these preferences in designing and selecting fragments to screen against protein targets may increase the chances of succ&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01855&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01855" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01855&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01855" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01855&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01855" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01855&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01855&amp;href=/doi/10.1021/acs.jmedchem.8b01855" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01855" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01855" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01855%26sid%3Dliteratum%253Aachs%26pmid%3D30875465%26genre%3Darticle%26aulast%3DGiordanetto%26date%3D2019%26atitle%3DFragment%2BHits%253A%2BWhat%2Bdo%2BThey%2BLook%2BLike%2Band%2BHow%2Bdo%2BThey%2BBind%253F%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D7%26spage%3D3381%26epage%3D3394%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292534" title="Oxygen">Oxygen</a>,</li><li><a href="/action/doSearch?ConceptID=290865" title="Nitrogen">Nitrogen</a>,</li><li><a href="/action/doSearch?ConceptID=291960" title="Molecular interactions">Molecular interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/7" title="Go to Volume 62, Issue 7"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/jmcmar.2019.62.issue-7/20190411/jmcmar.2019.62.issue-7.largecover.jpg" alt="Go to Volume 62, Issue 7"/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A “fragment hit”, a molecule of low molecular weight that has been validated to bind to a target protein, can be an effective chemical starting point for a drug discovery project. Our ability to find and progress fragment hits could potentially be improved by enhancing our understanding of their binding properties, which to date has largely been based on tacit knowledge and reports from individual projects. In the work reported here, we systematically analyzed the molecular and binding properties of fragment hits using 489 published protein–fragment complexes. We identified a number of notable features that these hits tend to have in common, including preferences in buried surface area upon binding, hydrogen bonding and other directional interactions with the protein targets, structural topology, functional-group occurrence, and degree of carbon saturation. In the future, taking account of these preferences in designing and selecting fragments to screen against protein targets may increase the chances of success in fragment screening campaigns.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18183" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18183" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A major challenge in drug discovery research is the identification of a suitable small molecule that binds the target and can serve as a starting point for chemistry exploration and optimization. This critical early hit-identification step significantly influences the overall likelihood of success of a drug discovery project. Very small molecules (molecular weight <300 Da),<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> referred to as “fragments”, can be ideal hits in some respects, as they tend to have favorable physical properties and form high-quality interactions with the target protein. Screening a library of fragments by evaluating their binding to a target protein has proven to be an effective method for identifying hits. From its origins as an infrequently used NMR-based method,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> fragment screening has evolved into an approach that has been widely adopted by industry and academia<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10">(3−10)</a> and has played a central role in the discovery of two approved drugs to date<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> and in the identification of a number of clinical candidates.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">Fragment screening has a number of potential advantages over screening larger compounds as an approach to the identification of starting points for drug development. Although, in contrast to larger compounds, a fragment typically has fewer interactions with the target protein and thus lower affinity overall, thermodynamic<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and probabilistic<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> models of small molecule–macromolecule binding suggest that these fragment–protein interactions are individually of greater energetic reward. Additionally, screening can more effectively sample the chemical space of smaller, less complex compounds,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> which improves the odds of identifying binders that have high ligand efficiency.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Fragments may thus provide medicinal chemists with a greater number of promising opportunities and, owing to their smaller size, greater flexibility in the optimization process.</div><div class="NLM_p">These advantages notwithstanding, fragment-based approaches have a number of limitations. It can be challenging to detect weak-affinity fragment hits<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and effectively distinguish them from false positives.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Furthermore, structural information describing the atomic interactions between the fragment hit and its protein target is typically necessary for successful fragment optimization, but such information can be difficult to obtain.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These caveats hinder fragment-hit identification, evaluation, and prioritization, thus significantly limiting the reliability and success of fragment-based screening protocols.</div><div class="NLM_p last">Fragment-based approaches could potentially be improved with a deeper understanding of the molecular properties of fragment hits and how such fragments bind to their target proteins. To date, our understanding of fragment–protein binding has largely been derived from individual case studies rather than from a broad structural analysis of validated fragment hits. Here, we have collected and reviewed publicly available fragment-hit data, of which a substantial amount was deposited since 2012, from a variety of fragment-based screening campaigns.<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a> We first describe the protein systems used in these fragment screens, followed by a cheminformatics analysis of the fragment hits to assess their properties (such as lipophilicity and structural topology). We then analyze how the fragments interact with their targets using various measures (such as the amount of buried polar surface area (SA) and the number of hydrogen bonds (H-bonds) between the fragments and the proteins). By studying a large set of fragment hits in this manner, we elucidate some of their salient properties, quantifying and adding to what is typically tacit knowledge among fragment-based screening practitioners and medicinal chemists. Our findings could potentially be used to improve the chances of success for fragment-based screening methods, in particular by informing fragment design and the evaluation of fragment hits.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Structures of the complexes were extracted from the Protein Data Bank (PDB)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> using the keyword “fragment” in a text-based query performed on Jan 13, 2019 and then filtered by keeping only the structures with a crystallographic resolution ≤2.5 Å and containing a ligand with a total number of nonhydrogen atoms ≤20. The resulting list of 5115 complexes was further refined by removing all entries that did not relate to fragment screening, identification, and characterization by analysis of the primary literature citation associated with each structure as well as the examination of the components of the structure. PDB entries where the chemical component consisted only of typical crystallization buffer elements (e.g., glycerol), adjuvants (e.g., carboxylates), or native cofactors/cosubstrates (e.g., pyridoxal phosphate), for example, were not considered. Visual inspection of the resulting 1623 complexes supported by the corresponding literature sources served to distinguish bona fide fragment hits from optimized fragments or lead compounds and to identify the associated binding pockets. Since crystallization buffers and conditions vary greatly across the considered complexes, any chemical component from the crystallization buffer that did not have biological relevance to the system (e.g., dimethyl sulfoxide) was removed from the structure before further analysis.</div><div class="NLM_p">Structural complexes where multiple fragments bound to the protein within close structural proximity (<4 Å for their shortest interatomic distance) were also removed. Structures in which the fragment was tethered to the protein through covalent linkages were discarded. Fragments bound to different binding sites on the same protein were treated independently in cases where the evidence supports the relevance of the alternate binding pockets, based on the primary literature source and electron density analysis. In cases where multiple structures were found of a single fragment bound to the same protein site, preference was given to the structure with the highest atomic resolution and clearest electron density, unless a significant change in the fragment-binding conformation was detected (root-mean-square deviation > 1 Å for the fragment heavy atoms between binding conformations), in which case both structures were included. Fragments that displayed a real-space correlation coefficient of less than 0.8 (which is the threshold value proposed by Deller and Rupp for unambiguous fragment binding<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a>) were not considered further. Fragment-binding sites located at the interface between the protein and its copies in the crystal lattice were not considered, to create a final data set in which every pocket involves just one copy of the protein. The 489 complexes that remained on the list after these filtering steps were then analyzed with respect to the observed protein–fragment interactions.</div><div class="NLM_p">The ionization, tautomer, and rotamer states of the fragment and the amino acids in the binding pocket were generated using Protonate3D<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and manually refined as needed based on typical geometric features for molecular interactions (e.g., H-bonds),<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> the local protein environment and potential interaction networks, the calculated p<i>K</i><sub>a</sub><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> for the fragment, and the pH of the experimental crystallization conditions used. No heavy-atom coordinates were modified during the preparation process. Water molecules are only discussed for structures having a crystallographic resolution ≤1.5 Å,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and only if the water molecules appear to be structurally relevant and are clearly supported by the electron density maps (i.e., a real-space correlation coefficient ≥0.8). We define structurally relevant water molecules to be those with ≥2 H-bonds to the protein and ≥1 H-bonds to the fragment.</div><div class="NLM_p">Molecular properties of the fragments, including the number of heavy atoms, rotatable bonds, chiral centers, formal charges, and H-bond donors and acceptors in their protonation state at pH 7, were calculated as implemented in MOE.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The cxcalc command line script from ChemAxon<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> was used to compute the p<i>K</i><sub>a</sub> values for ionizable groups on the fragment hits. The dissimilarity distribution of fragments was evaluated using extended connectivity fingerprints (ECFP4)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with the screenmd command line script within JChem from ChemAxon.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Octanol–water partition coefficient (clog <i>P</i>) values for the fragments were calculated using the classic algorithm within the batch version of ACD/Percepta.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> The number and identity of ring assemblies, Bemis–Murcko frameworks, and the fraction of sp<sup>3</sup>-hybridized carbon atoms (Fsp<sup>3</sup>) were calculated using Vortex.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Protein pocket descriptors were computed using default parameters in dpocket and implemented as part of the α spheres-based methodology available in fpocket.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Here, selection of the relevant fragment hit was used to explicitly define the associated binding pocket.</div><div class="NLM_p">Surface-based descriptors including total, polar, and apolar solvent-accessible surface areas were calculated in MOE<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> using a 1.4 Å solvent radius probe. The differences in these values between the unbound and bound states of fragment hit and protein yielded the corresponding buried solvent-accessible surface areas. Molecular interaction counts between the fragment and corresponding protein, water, and metal ions were computed using a probabilistic receptor potential within MOE.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Here, H-bonds, metal coordination bonds, arene-based interactions, carbon–hydrogen bonds, halogen bonds, and sulfur-mediated contacts are scored with empirical type-based scoring functions using the extended Hückel theory. These functions are trained using statistics derived from contacts in the RCSB PDB, and each interaction is scored in terms of the percentage likelihood of being geometrically ideal. The default energy threshold of interaction (0.5 kcal mol<sup>–1</sup>) was used to identify relevant interactions. The H-bond count only includes interactions involving oxygen or nitrogen atoms (whereas weak H-bonds, which are interactions in which the hydrogen is covalently bonded to a carbon atom or in which the acceptor is a halogen, are omitted). Water molecules are only discussed in cases where they are potentially relevant to the fragment–protein interaction, as defined above. The degree of burial of an H-bond was calculated as the difference between the solvent-accessible surface area in the unbound and bound states for the protein atom involved in the H-bond.</div><div class="NLM_p">Atom types for the fragment hits used in the frequency of distribution analysis were assigned based on the MMFF94 force field<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> as implemented in MOE. The atom-type-based frequencies of molecular interactions were obtained by dividing the number of observed interactions by the total number of occurrences for a given atom type across the entire fragment set to control for over-representation of specific functional groups and thus provide useful background information for molecular design purposes.</div><div class="NLM_p">A procedure analogous to the one described above was used to generate a comparative data set in which larger ligands were bound to the protein pockets that were observed in the original (fragment hit) data set; the construction of this second data set was such that any given protein pocket has a similar frequency of occurrence in each data set. Structures of the complexes were extracted from the PDB<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> using the UniProtKB accession numbers<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> of the fragment–protein complexes previously identified in a protein-based query, and then filtered by keeping only the structures with a crystallographic resolution ≤2.5 Å and containing a ligand with a total number of nonhydrogen atoms ≥25. This list was further refined by removing all entries that did not contain a chemical component that resulted from ligand screening and optimization, using analysis of the primary literature citation associated with each structure as well as examination of the components of the structure. Structures in which the ligand was covalently bound to the protein were discarded. Preparation of the ligand–protein complexes for calculation of ligand–protein interactions and ligand properties followed the same procedure described for the fragment–protein complexes.</div><div class="NLM_p last">To ensure that the analysis was not skewed by particular protein pockets that are over-represented in the data set, each data point was normalized by the number of occurrences of the bound protein pocket in the entire data set. A given observation for one of the 19 PDE10A–fragment complexes surveyed in this work, for example, carries a weight of 5.26% in the final analysis. When a data normalization based on a pocket-sequence-identity cutoff of 60%<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> was used, there were no significant changes in the results (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figures S1 and S2</a>).</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55426" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55426" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> General Characteristics of Fragment Hits and Their Protein Targets</h3><div class="NLM_p">Our data set consists of 489 structures of a fragment bound to a pocket of a protein or protein domain. The set contains 126 unique proteins spanning 20 different protein families and 79 structural domains (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">S3</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, 67% of the structures of the complexes have a crystallographic resolution of ≤2 Å, with 1.03 Å as the best-reported resolution (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4J">4Y4J</a>). The structural data is for the most part relatively recent; 79% of the structures were deposited in the PDB since the beginning of 2012 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S1</a>). Transferases and hydrolases account for 58% of the complexes, with 44 and 26 unique protein entries from each class, respectively. The remainder of the data set is distributed across 18 protein families, including DNA-binding proteins, oxidoreductases, isomerases, and viral proteins (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Several important drug targets appear in the data set, including poly ADP-ribose polymerase, carbonic anhydrase, β-lactamase, estrogen receptor, DNA gyrase, Bruton’s tyrosine kinase, and Janus kinase 2. The most frequent protein entries, representing 19% of the data surveyed here, are the aspartic protease endothiapepsin (<i>N</i> = 57),<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> cAMP and cAMP-inhibited cGMP 3′,5′-cyclic phosphodiesterase 10A2 (PDE10A2, <i>N</i> = 19),<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> and heat-shock protein 90 (Hsp90, <i>N</i> = 18).<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characteristics of the proteins, structures, and binding pockets. (a) Distribution of protein classes, as codified by the PDB.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Numeric labels indicate the number of unique proteins belonging to each protein class, as indicated by their respective UniProt access codes.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (b) Distribution of crystallographic resolution values. (c) Distribution of fragment-binding pocket volumes using dpocket,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> normalized by the occurrence of a given protein binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A total of 168 unique fragment-binding pockets were identified in the 126 proteins in the data set, according to the selection criteria described in the <a class="ref internalNav" href="#sec2" aria-label="Methods">Methods</a> section. To ensure that the analysis was not skewed by the over-representation of certain protein pockets, each binding data point was normalized by the number of occurrences of the bound protein pocket in the data set. 82% of the proteins (<i>N</i> = 103) feature only a single binding site. The remaining 23 proteins bind fragments at more than one site, with HIV-1 reverse transcriptase containing 7 different sites (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S1</a>). The fragment-binding pockets span a 6-fold difference in size, as estimated using a Voronoi tessellation and α sphere-based method<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), ranging from a small cleft on human cyclophilin D accommodating pyrrolidine-1-carbaldehyde (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R54">3R54</a>) to a voluminous funnel-shaped pocket on hepatitis C virus polymerase NS5B bound to 4-(2-phenylhydrazinyl)-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidine (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IH5">4IH5</a>).</div><div class="NLM_p">In total, 462 unique fragments are covered in the present analysis. 21 of these fragments occur more than once in the data set, either because they bind different proteins or because they bind to distinct pockets on the same protein (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). None of these 21 fragments match the pan-assay interference compounds (PAINS) structural filters defined by Saubern et al.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Across the whole data set, only four fragments (pyrocathecol, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7I">4K7I</a>; 4-methylbenzene-1,2-diol, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7N">4K7N</a>; 4-(<i>tert</i>-butyl)benzene-1,2-diol, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7O">4K7O</a>; 4-(2-amino-1-hydroxyethyl)benzene-1,2-diol, PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4J">4Y4J</a>) were flagged as potential PAINS hits, all with the polyphenolic structural alert. The fragments range in size from 6 to 20 heavy atoms, with 81% of the fragment set containing between 10 and 16 heavy atoms (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). At the extremes, 2-chloro-1<i>H</i>-imidazole (with 6 heavy atoms) binds to the BAZ2B bromodomain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9K">5E9K</a>), and <i>N</i>-[2-(morpholin-4-yl)phenyl]thiophene-3-carboxamide (with 20 heavy atoms) binds to soluble epoxide hydrolase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WKD">3WKD</a>). Calculated octanol–water partition coefficient (clog <i>P</i>) values<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> range from −2.4 to 4.8, with 68% of the compounds displaying clog <i>P</i> < 2, as summarized in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Examples of the extremes of lipophilicity in this data set include 4-acetylpiperazin-2-one complexed with the bromodomain-containing protein 1 (clog <i>P</i>: −2.4, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AME">5AME</a>) and 2-(5-chloro-3-methylbenzo[<i>b</i>]thiophen-2-yl)acetic acid complexed with farnesyl pyrophosphate synthase (clog <i>P</i>: 4.8, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N1V">3N1V</a>). 71% of the fragment hits have a significant proportion (20–30%) of their heavy atoms as nitrogen or oxygen atoms that are capable of accepting or donating H-bonds. More than half of the fragments (68%) display a net formal charge of 0, and for the remainder of the (charged) fragments, twice as many have a net negative formal charge (22%) as a net positive charge (11%). Only eight fragments could have zwitterionic character, with both strongly basic and acidic functionalities (predicted p<i>K</i><sub>a</sub> > 9 and <5, respectively).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characteristics of the fragment hits (<i>N</i> = 462, black profiles). Distributions of the (a) number of heavy atoms, (b) ACD calculated log <i>P</i>, (c) number of atoms capable of accepting or donating H-bonds expressed as a percentage of the fragment’s heavy atoms, (d) fraction of sp<sup>3</sup>-hybridized carbon atoms (Fsp<sup>3</sup>), (e) number of rotatable bonds, (f) number of chiral centers, (g) number of ring assemblies, and (h) formal charges are shown. (i) Frequency distribution of Tanimoto distances calculated based on ECFP4 for all possible fragment pairs. Distributions for a representative commercial fragment library<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (<i>N</i> = 1794, gray profiles) are included.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Fragments Hits That Bind to More than One Protein Pocket<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0011.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The captions show the PDB IDs of the corresponding complexes. In cases where the fragment bound distinct pockets on the same protein, the number of pockets bound is indicated in parentheses.</p></div></div><div></div></div><div class="NLM_p">In terms of structural complexity, the vast majority (>90%) of the fragment hits are achiral, with limited carbon saturation (fraction of sp<sup>3</sup>-hybridized carbon atoms Fsp<sup>3</sup> < 0.5), and up to three rotatable bonds and two ring assemblies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The fragment hits are structurally diverse (extended connectivity fingerprint 4 (ECFP4)-based Tanimoto distance >0.7 for 97% of the set; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The fragment hits presented here compare well to an established commercial fragment library;<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> the most notable difference is that the latter has a ∼20% greater proportion of fragments with a high degree of carbon saturation and a high number of ring assemblies (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The fragment hits in this work are described by 52 unique Bemis–Murcko frameworks.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Of the 52 unique molecular frameworks, 33 (63%) occur only once, whereas five frameworks account for 71% of the total. These five frameworks are: monocyclic 6- and 5-membered rings; bicyclic 5-6- and 6-6-fused ring systems; and 5-6 rings connected by one bond (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">3</a></a>). When hybridization, heteroatoms, and exocyclic carbonyl groups are considered, the present set encompasses 138 unique ring assemblies, of which 78 (57%) appear only once. Benzene, pyridine, pyrazole, thiophene, and indole taken together account for 49% of the ring assemblies, as shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">3</a></a>. Among the other ring assemblies with more than 10 representatives, piperidine and indazole each occur in several distinct binding pockets (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">3</a></a>).</div><figure id="fig11" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0012.jpeg" id="GRAPHIC-d7e595-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Bemis–Murcko frameworks occurring more than 10 times across the fragment hits. The number of observations and frequency of occurrence as a percentage of the total observed frameworks (<i>N</i> = 52, see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S2</a> for the full list) are indicated in the caption. Values in parentheses indicate the number of unique proteins the framework was found bound to. (b) The most frequently occurring individual ring assemblies. The number of observations and frequency of occurrence as a percentage of the total observed individual ring assemblies (<i>N</i> = 138, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S3</a>) are indicated in the caption. Values in parentheses indicate the number of unique proteins bound by the ring assembly in this data set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Characteristics of Fragment Hits: Biological–Target Interactions</h3><div class="NLM_p">We analyzed the fragment–pocket complexes to assess the molecular interactions between the two binding partners. As molecular interactions are conceptual models that describe complex physical phenomena, general considerations regarding surface contacts and atomic contacts are summarized first, followed by more specific interaction models.</div><div class="NLM_p">Most of the fragment hits (73%) bury more than 80% of their total solvent-accessible surface area (SA) upon binding. Even the two fragments that are the most exposed to solvent upon binding still hide 50 and 57% of their surface in the bound pose (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAN">5JAN</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J4H">5J4H</a>, respectively); these fragments are the only ones that bury <60% of their SA. In contrast, 21 fragments are completely engulfed by the protein (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">5</a>a,b). Notably, 77% of fragment hits bury more than 80% of their polar SA, and 53% of hits bury over 90% of their polar SA. The apolar SA of fragments is also largely buried upon binding, although to a slightly lesser extent, with 70 and 42% of the set burying >80 and >90% of the lipophilic surface, respectively. The polar fraction of the buried SA of a protein pocket varies significantly, ranging from 5 to 74% of the total protein buried surface. In terms of absolute values, fragments bury on average 336 Å<sup>2</sup> upon binding, with a substantial tendency to bury more apolar than polar SA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">4</a></a>). Accordingly, the ratio of apolar to polar SA buried has a mean of 3.1 and a median of 2.2 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S4</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Surface characteristics of the fragment–pocket complexes. (a) Polar and apolar buried SA of fragments, expressed as a percentage of their total polar and apolar SA, respectively. The total buried SA of fragments is also indicated as a percentage of the whole-fragment SA. (b) Extents of the total, apolar, and polar SA that are buried by the fragments upon binding. (c) Buried polar SA of the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selected fragment hits bound to proteins. (a) Maximum and (b) minimum solvent exposure of fragments (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OMQ">3OMQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAH">5JAH</a>, respectively). (c) Maximum number of polar interactions (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>). (d) Maximum number of additional directional interactions (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>). (e) and (f) Examples of fragments binding to multiple pockets within the same protein (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FPO">5FPO</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CLP">5CLP</a>, respectively). Fragments are depicted as bold sticks (cyan carbon atoms). Relevant H-bonds and additional directional interactions are indicated as dashed black lines. Relevant protein pocket surfaces are displayed as gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having analyzed the binding of fragment hits to their targets with surface-based descriptors, we next evaluated their molecular interactions. H-bonds to protein and water molecules, as well as metal coordination bonds, were identified based on geometric criteria.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>, 92% of the fragment–protein complexes are stabilized by at least one H-bond to the protein or to a structural water or by a coordination bond to a structural metal ion. In one complex, seven such molecular interactions were noted (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">5</a></a>c). A large majority of H-bonds between fragments and proteins (88%) are completely buried (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Polar interactions established by the fragment hits. (a) Frequency distribution of polar interaction counts per fragment complex, including H-bonds to protein or water and coordination bonds to metal ions. Fragment complexes that do not display any water H-bonds or metal coordination bonds (54 and 95% of the total, respectively) have been omitted from the histogram for clarity. (b) Frequency distribution of fragment H-bonds to protein amino acids at the side-chain and the backbone level. (c) Frequency distribution of buried surface area for protein atoms involved in H-bonds to fragments. (d) Frequency distribution of H-bonds between fragment atoms (neutral and ionizable) and protein amino acid side chains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A group of 37 complexes (from 8 unique proteins) feature fragment hits bound to the structural metal ions present in the pockets, namely, zinc, manganese, and iron ions (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S5</a>). Negatively charged oxygen atoms from fragment carboxylic acid groups are the atoms that most frequently establish coordination bonds to these metal ions (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S5</a>). A maximum of two metal coordination bonds was recorded for a given fragment-pocket entry in the current data set (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACW">5ACW</a>).</div><div class="NLM_p">53 of the 116 X-ray structures with resolution ≤1.5 Å (46% of the total, with 20 unique binding sites) have structural water molecules with at least two H-bonds to the protein and one to the fragment hit. The maximum number of individual water-based H-bonds observed for a fragment was 3 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MOH">5MOH</a>). Nitrogen fragment atoms have a higher occurrence of H-bonds to water than do oxygen fragment atoms (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S6</a>). Anilinic nitrogen atoms and sp<sup>3</sup>-hybridized nitrogen atoms in aliphatic amine groups display the highest frequency of water H-bonds in the data set (0.33 and 0.52, respectively; <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S6</a>).</div><div class="NLM_p">Fragment H-bonds to proteins stabilize 89% of the bound complexes, with 74% of the entries displaying between one and three H-bonds. The highest number of fragment–protein H-bonds in a complex is 6 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4G">4Y4G</a>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>). Side-chain atoms account for 58% of the total protein–fragment interactions. H-bonds occurring between ionizable functional groups on both the fragment and the amino acid side chain represent 17% of the total number of H-bonds in the data set (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>). As expected, the side chains of polar amino acids make the greatest contribution to H-bonds between proteins and fragments. Aspartic acid and serine each provide more than 10% of the total H-bonds observed in the fragment–protein complexes. Interestingly, glycine, which establishes H-bonds exclusively through its backbone atoms, also accounts for greater than 10% of total H-bonds observed, occurring in 27 unique protein pockets (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>). Cysteine, isoleucine, phenylalanine, proline, and tryptophan each account for fewer than 2% of H-bonds. Histidine and glutamic acid have a preference for H-bonds to ionizable and neutral fragment atoms, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>).</div><div class="NLM_p">The occurrence of H-bonds to the protein based on specific fragment atom types is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>. After normalizing for the overall atom-type occurrence in our data set, nitrogen and oxygen atoms show a similar preference for H-bond formation, with 0.62 and 0.61 H-bonds per atom, respectively. Positively charged nitrogen atoms establish the highest number of H-bonds to protein residues, with aliphatic amines displaying a higher likelihood of forming such bonds than aromatic or conjugated ones (cf., sp<sup>2</sup> NH<sup>+</sup> and sp<sup>3</sup> NH<sup>+</sup>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>). The only negatively charged nitrogen atoms that form H-bonds to proteins are embedded in heterocyclic systems (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S5</a>). Tetrazole moieties, for example, were found to establish up to three H-bonds to serine and threonine residues of CTX-M-9 class A β-lactamase (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G2Y">3G2Y</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G32">3G32</a>). In contrast, the deprotonated nitrogen atoms of sulfonamide mainly coordinate metal ions (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S5</a>). H-bond donors in the form of neutral nitrogen atoms are represented by five different functional groups (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>). Among the functional groups with more than 50 observations, anilinic nitrogen atoms form the highest number of H-bonds per atom (0.80), followed by heterocyclic and amidic NHs (0.66 and 0.61, respectively). H-bond acceptors featuring a neutral, unprotonated nitrogen atom engage in H-bonds with the protein in one-third of cases. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>, 4 out of 10 nitrile functionalities form an H-bond in this data set (e.g., PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4T">4Y4T</a>). Interestingly, one aliphatic amine may potentially accept an H-bond from the protein, based on the surrounding atomic environment, crystallization pH (7.5), and predicted p<i>K</i><sub>a</sub> (7.3) for the fragment species (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FYU">5FYU</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Fragment Atoms Involved in H-Bonds to the Protein<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">atom type</th><th class="colsep0 rowsep0" align="center" char=".">total occurrence</th><th class="colsep0 rowsep0" align="center" char=".">H-bond occurrence</th><th class="colsep0 rowsep0" align="center" char=".">ratio</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>nitrogen</b></td><td class="colsep0 rowsep0" align="char" char=".">833</td><td class="colsep0 rowsep0" align="char" char=".">517</td><td class="colsep0 rowsep0" align="char" char=".">0.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>N (neutral)</b></td><td class="colsep0 rowsep0" align="char" char=".">328</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sp<sup>2</sup></td><td class="colsep0 rowsep0" align="char" char=".">317</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sp</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sp<sup>3</sup></td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>N<sup>–</sup></b></td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heterocyclic</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>N<sup>+</sup></b></td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">150</td><td class="colsep0 rowsep0" align="char" char=".">1.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sp<sup>2</sup></td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">1.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sp<sup>3</sup></td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">1.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>NH</b></td><td class="colsep0 rowsep0" align="char" char=".">372</td><td class="colsep0 rowsep0" align="char" char=".">263</td><td class="colsep0 rowsep0" align="char" char=".">0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">amide</td><td class="colsep0 rowsep0" align="char" char=".">137</td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="char" char=".">0.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">anilinic</td><td class="colsep0 rowsep0" align="char" char=".">138</td><td class="colsep0 rowsep0" align="char" char=".">110</td><td class="colsep0 rowsep0" align="char" char=".">0.80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heterocyclic</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfonamide</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hydrazide</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">1.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>oxygen</b></td><td class="colsep0 rowsep0" align="char" char=".">720</td><td class="colsep0 rowsep0" align="char" char=".">441</td><td class="colsep0 rowsep0" align="char" char=".">0.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>O (neutral)</b></td><td class="colsep0 rowsep0" align="char" char=".">363</td><td class="colsep0 rowsep0" align="char" char=".">171</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carbonyl</td><td class="colsep0 rowsep0" align="char" char=".">214</td><td class="colsep0 rowsep0" align="char" char=".">142</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfonamide</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">0.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ether</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aromatic</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>O<sup>–</sup></b></td><td class="colsep0 rowsep0" align="char" char=".">270</td><td class="colsep0 rowsep0" align="char" char=".">202</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carboxylic acid</td><td class="colsep0 rowsep0" align="char" char=".">192</td><td class="colsep0 rowsep0" align="char" char=".">188</td><td class="colsep0 rowsep0" align="char" char=".">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfonic acid</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">phenol</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nitro</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">N-oxide</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>OH</b></td><td class="colsep0 rowsep0" align="char" char=".">87</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aliphatic</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">1.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aromatic</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">0.39</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The total occurrence of the various atomic types in the fragment set based on the MMFF94 force field definitions,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> corresponding subclass based on atomic hybridization or associated functional group, and the calculated number of H-bonds and occurrence ratios are presented.</p></div></div></div><div class="NLM_p">Negatively charged oxygen atoms occur in five different chemical environments in this data set. Carboxylic acid groups, which occur the most frequently, form on average one H-bond per oxygen atom (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>). All other functional groups within this subclass occur much less frequently (<i>N</i> ≤ 57). Notably, there is one phenolic group in the data set that may capture two H-bonds (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GVB">3GVB</a>), based on the crystallization pH (8.7) and calculated p<i>K</i><sub>a</sub> (8.7). Alcoholic groups on fragment hits secure on average 0.78 H-bonds, with aliphatic alcohols having a higher H-bond formation frequency than their phenolic counterparts (1.53 and 0.39, respectively). Unprotonated oxygen atoms as H-bond acceptors represent the most conspicuous oxygen atom class (<i>N</i> = 363, 50%), with an average of 0.47 H-bonds per atom. Carbonyl oxygen atoms from amide, urea, ester, and ketone moieties display, on average, 0.66 H-bonds each, compared to 0.53 H-bonds each for oxygen atoms in sulfonamide groups. Ether and aromatic oxygen atoms display H-bonds in only 6 and 12% of cases, respectively.</div><div class="NLM_p">Only 7% (<i>N</i> = 34) of the fragment–pocket complexes analyzed here do not contain any of the polar interactions described above. The distribution of the fragment buried surface values of this subset of fragments was not significantly skewed compared to the remainder of the set. In half of these complexes, the fragment establishes at least one H-bond to a water molecule, which in turn makes an H-bond to the protein, thus acting as a bridge between the fragment hit and the protein. Interestingly, when additional directional interactions including arene- and sulfur-mediated contacts,<a onclick="showRef(event, 'ref27 ref48'); return false;" href="javascript:void(0);" class="ref ref27 ref48">(27,48)</a> halogen bonds,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and carbon–H-bonds<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> are also considered, all complexes displayed at least one such interaction between the fragment and protein. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">7</a></a> summarizes the distribution of additional directional interactions across the whole data set, as assessed using MOE’s nonbonded-contacts detection algorithm<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Supporting Information</a>). 56% of fragment hits establish at least one such interaction with the corresponding protein. Arene-based interactions occur most frequently (42% of cases), followed by carbon H-bonds (12%), sulfur-mediated contacts (11%), and halogen bonds (3%). As many as seven different non-H-bond interactions in a single complex have been found in the current set (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">5</a></a>d). Sulfur atoms in fragment hits establish, on average, 0.84 interactions, with sulfur-oxygen contacts the most frequent (0.4 per sulfur atom). Halogen bonds and carbon H-bonds occur markedly less often: 0.13 and 0.02 per atom, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S7</a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Frequency distribution of additional directional interactions per fragment pocket including: arene-based interactions (i.e., arene–arene, arene–cation, and arene–hydrogen), carbon H-bonds, sulfur-mediated contacts, and halogen (i.e., iodine, bromine, and chlorine) bonds to protein atoms. The fractions of fragment complexes without arene (58%), carbon H-bonds (88%), sulfur contacts (89%), and halogen bonds (96%) have been omitted from the graph for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Comparison of Fragment Hits and Larger Ligands</h3><div class="NLM_p">We constructed a comparative data set of 445 protein–ligand complexes (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S8</a>) such that any given protein pocket occurs with similar frequency as it does in the fragment data set. The 439 unique ligands are on average twice as large and twice as lipophilic as the fragment hits previously discussed and have a greater number of ring assemblies, rotatable bonds, and sp<sup>3</sup>-hybridized atoms in their structures. There are no significant differences between the fragments and the ligands with respect to the distribution of formal charges or the ratios of atoms capable of establishing H-bonds (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p">The ligands are ∼10% less likely than the fragments to bury their total SA, and they are equally less likely to bury their polar and apolar SA. In absolute terms, owing to their larger size and lipophilicity, ligands bury a greater amount of total SA, mainly of apolar nature, when compared to fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p last">On average, ligands establish one additional protein H-bond and twice as many arene interactions compared to fragments, whereas water H-bonds and additional directional interactions do not substantially change. Despite the higher number of protein H-bonds established, these bonds are 10% less likely to be fully buried, and the polar atoms of ligands show a reduced share of H-bonds compared to those of fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S6</a>).</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We compiled and analyzed a data set composed of 462 unique fragments bound to 168 different pockets on 126 individual proteins, resulting in a total of 489 fragment–pocket complexes (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). This work was made possible by a large number of crystallographic studies performed by scientists across numerous organizations in the past two decades (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S9</a>). Several systems are over-represented due to the large number of fragment-binding studies published for these proteins,<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43 ref44">(39−44)</a> but the data set is nevertheless diverse in terms of structural domains and distinct protein families (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S1</a>). From the analysis of this diverse data set, we observe a number of notable fragment features, discussed below, that can be used as a guide for the design, selection, and evaluation of fragments.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Binding Versatility of the Fragment Hits</h3><div class="NLM_p">21 fragments bound to more than one protein pocket, in many cases on different proteins (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The ability of fragments to bind to multiple proteins<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> reinforces the appeal of using fragment-based methods to generate chemical starting points for drug discovery. In seven cases, a single fragment bound to different binding sites on its target protein (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">5</a></a>e,f). Fragment screening is thus well suited to uncover and evaluate alternative binding sites and target interaction mechanisms of potential therapeutic relevance.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Interestingly, with the exception of saturated carbocyclic rings, these 21 fragments recapitulate most of the pharmacophoric elements typically exploited for molecular interactions. If aptly complemented with missing fragment pharmacophores and normalized for relative pharmacophoric occurrence, we recommend these fragments as a useful choice for a minimalistic, “first pass” library for pilot fragment screens, particularly for X-ray crystallography screening. Here, the so-called “promiscuity” of fragments in a well-defined structural context is understood as a practical advantage for mapping hotspots on a protein<a onclick="showRef(event, 'ref39 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref39 ref52 ref53">(39,52,53)</a> and identifying fragment binders for further optimization. In the absence of relevant structural information, however, fragment promiscuity could be detrimental, especially when relying on biophysical screening.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div><div class="NLM_p">The fragment hits are remarkably versatile in their interactions with different binding pockets, as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">8</a></a> for selected examples. 5-Hydroxyindole, for example, is fully engulfed by leukotriene A-4 hydrolase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUH">3FUH</a>), in contrast to the complex this fragment forms with the DNA repair and recombination protein RadA, on which it binds to a highly solvent-exposed cleft (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B3C">4B3C</a>). Interestingly, in both complexes, the fragment nitrogen atom (and not its 5-hydroxy group) is H-bonded to the protein. In another example of fragment versatility, adenine exploits several of its pharmacophoric elements when binding to the BAZ2B bromodomain and Hsp90 (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYX">5DYX</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YED">2YED</a>, respectively). The tautomerism of adenine’s imidazole moiety further adds to its adaptability; both the 7- and 9-position nitrogen atoms are independently H-bonded to backbone carbonyl groups on the two proteins. Another fragment, 5-nitro-benzimidazole, exemplifies how a structural element typically frowned upon in recent medicinal chemistry practice (i.e., the nitro group) can serve an important molecular-recognition function in the early phases of drug discovery. Here, it secures H-bonds to both neutral (serine) and charged (arginine) side chains in the binding pockets of PDE10A2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSA">4MSA</a>) and nicotinamide phosphoribosyltransferase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N9C">4N9C</a>), respectively. These examples highlight the ability of fragments to effectively sample chemical space during screening.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Selected fragment hits bound to different proteins. (a, b) Different degrees of solvent exposure of 5-hydroxyindole bound to RadA (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B3C">4B3C</a>) and leukotriene A-4 hydrolase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUH">3FUH</a>), respectively. (c, d) Different tautomers of adenine bound to BAZ2B bromodomain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYX">5DYX</a>) and Hsp90 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YED">2YED</a>), respectively. (e, f) Different interactions of the nitro group of 5-nitro-benzimidazole bound to nicotinamide phosphoribosyltransferase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N9C">4N9C</a>) and PDE10A2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSA">4MSA</a>), respectively. Fragments are depicted as bold sticks (cyan carbon atoms) and relevant H-bonds as dashed black lines. Relevant protein pocket surfaces are displayed as gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Properties of the Fragment Hits and Relevance to Fragment Design</h3><div class="NLM_p">The fragment hits described here mostly comply with the rule-of-three (Ro3) guidelines,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> with less than 5% of the set deviating from any of the Ro3 parameters. The propensity of fragment hits to display a quarter of their atoms as H-bond recognition elements (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) hints at a particular balance between exposed polarity and lipophilicity that is most conducive to productive interactions with different proteins, while ensuring that physicochemical properties are compatible with fragment-screening experiments. We thus strongly suggest favoring fragments with a polar atom fraction of ∼0.25 when evaluating novel fragment topologies and pharmacophores during fragment-library enrichment campaigns. Interestingly, in our independent data set of protein–ligand complexes, we found that the ligands have a similar fraction of polar atoms as the fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S4</a>), suggesting that this ∼0.25 polar atom fraction may be of general utility in ensuring favorable interactions with target proteins.</div><div class="NLM_p">The limited degree of chirality and carbon sp<sup>3</sup> saturation of the validated fragment hits is noteworthy, especially when coupled to their wide chemical diversity and over-reliance on a handful of topological skeletons (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>−<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">3</a>). This might reflect historic trends in the first generation of fragment libraries, especially given the retrospective nature of the present study. Recently, there has been renewed interest in fragment structural complexity and three-dimensionality as driving forces in fragment design. This is borne out by the fact that the primary difference between the fragment hits presented here and those in a representative commercial fragment library from an established vendor in the fragment-based community<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> is the higher Fsp<sup>3</sup> distribution of the latter (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Nevertheless, it is noteworthy that achiral, heteroaromatic assemblies belonging to five topological frameworks can result in productive binding against a diverse set of pocket shapes and features. We suggest that the three-dimensional character of fragments (and molecules in general) is misrepresented by typical two-dimensional molecular descriptors such as Fsp<sup>3</sup>. Several of the aromatic fragment hits in this study are indeed nonplanar both in their shape and, more importantly, in the way that they interact with the protein. This nonplanarity is afforded by virtue of ortho substituents (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAO">5JAO</a>), monoatomic linkers connecting individual rings (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YE7">2YE7</a>), and a small amount of hydrocarbon saturation (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FYU">5FYU</a>).</div><div class="NLM_p last">In our opinion, fragments should be kept relatively simple during fragment screening to maximize their potential interactions with proteins. To this end, we view the current set of 462 validated fragment hits as a relevant first approximation of a diverse fragment library with adequate structural complexity, which could then be subjected to further refinement based on, for example, target-specific hypotheses or diversity-optimization goals. In accordance with probabilistic interaction models,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> the comparison in this work between the molecular properties of fragments and larger ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S5</a>) suggests that structural complexity can and should be built in at a later stage, during fragment optimization. The challenge (and an underappreciated differentiation element) is to devise adequate synthetic protocols for fragment diversification.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S2</a>, there are still ample opportunities to generate novel fragment matter even when considering limited structural complexity (to illustrate this, it is worth noting that although the data set of fragment hits presented here and the representative commercial fragment library we analyze for comparison display comparable distributions of molecular properties (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>), there is not a single fragment structure shared between the two sets). Furthermore, 7-membered rings are under-represented in both sets, both as individual rings and as part of fused systems. Their peculiar conformational preferences<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and projected substitution vectors represent interesting design features that are well suited to a fragment-based context. A number of additional fragment design considerations, directly derived from the analysis of the molecular interactions between the fragment hits and target proteins studied here, will be presented in the following sections.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Interactions of the Fragment Hits</h3><div class="NLM_p">Fragments are smaller than lead compounds, and thus tend to have fewer productive interactions with target proteins (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figures S4 and S6</a>). Although the fragment–protein interactions are typically referred to as “higher-quality” interactions, the net effect is a weaker binding affinity. The thermodynamics underlying fragment–protein binding is an area of active study.<a onclick="showRef(event, 'ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59">(57−59)</a> Our current analysis of fragment hit–protein complexes serves to identify propensities in these interactions as approximated with surface-based and interaction-count descriptors. It is envisaged that these descriptors, together with the associated functional-group preference, could support triaging of fragment-screening results in virtual campaigns (e.g., molecular dynamics–based screening of a cocktail of different fragments) by, for example, reducing the number of false positives.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Solvent Exposure and Protein Complementarity</h4><div class="NLM_p">More than 70% of the fragment hits surveyed here, as validated by X-ray crystallography, reduce their total solvent exposure by >80% upon binding, a finding that is consistent with previously published comparisons of primary and secondary fragment-binding sites from proprietary databases.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Accordingly, we find that the polar fraction of the surfaces of fragments in this data set is almost entirely buried (>80%) upon binding. The fragments consistently bury their polar SA regardless of the diverse physicochemical features of the observed protein pockets. Fragments tend to bury on average about twice as much apolar surface as polar surface (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">4</a></a>), in line with the observed polar/apolar atomic composition of the fragments (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Importantly, larger ligands of higher affinity reduce their total, polar, and apolar solvent exposure to a lesser degree than do fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S5</a>). Thermodynamic analyses have shown, however, that fragments cannot rely entirely on apolar desolvation as the main driver for binding.<a onclick="showRef(event, 'ref57 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref60">(57,60)</a> Indeed, the large amounts of buried polar areas suggest a very effective use of fragment H-bond donor and acceptor functionalities. In most cases, the protein H-bond donors and acceptors are completely isolated from solvent (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>) by virtue of significant apolar surface burial and are fully engaged in H-bonds with the fragment (see next section).</div><div class="NLM_p last">Taken together, these results are consistent with previous findings of enthalpically favored binding events at protein hotspots that are composed of polar sites buried in a lipophilic environment.<a onclick="showRef(event, 'ref57 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref60">(57,60)</a> Given the observed fragment size and fraction of polar atoms, as well as typical H-bond chemical functionalities, a potential strategy to maximize molecular interaction diversity would be to present a minimum set of individual polar pharmacophoric elements, as opposed to distributing several pharmacophores on a given fragment. This strategy would provide fragments with greater freedom to satisfy the geometric constraints for optimal interactions. It would also result in better sampling of the reduced pharmacophoric space during fragment screening,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and additional pharmacophores could be built in and evaluated during the subsequent phase of fragment growing.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> H-Bonds to Protein and Water</h4><div class="NLM_p">Stabilizing polar interactions are a recurring feature of the majority of the fragment–hit complexes (93%, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>). These include strong attractive interactions, such as H-bonds to protein and to structural water molecules, as well as coordination bonds to catalytic metal ions. 87% and 58% of the complexes are stabilized by at least one or two H-bonds to the protein, respectively, most of which are completely isolated from solvent upon fragment binding (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">6</a></a>). This result supports the previous finding of an average of two H-bonds per fragment from a minimally overlapping data set (the two overlapping entries are PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ESS">3ESS</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Importantly, the substantial network of observed H-bonds provides an interaction context for the systematic and significant degree of fragment- and protein-polarity burial observed. The desolvation of polar groups on the fragment and the protein, as directed by the formed H-bonds, may result in an important enthalpic contribution to fragment binding, enhancing apolar desolvation, as previously reported.<a onclick="showRef(event, 'ref57 ref58 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref60">(57,58,60)</a> It is noteworthy that the larger ligands cannot match the fragments’ share of H-bonds per polar atom, and that the additional H-bonds formed tend to be more solvent-exposed than the ones established by fragments (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Figure S6</a>).</div><div class="NLM_p">Fragment hits display a slight preference (58%) for establishing H-bonds to side-chain groups. Glycine is also highly represented as an H-bond target. These findings emphasize the importance of H-bond-site accessibility and geometric constraints, in addition to the need to populate diverse H-bond functionalities in fragment libraries, as summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">2</a></a>. Functional groups with dual H-bond accepting and donating character (e.g., amide or alcohol groups) are particularly attractive for interaction-sampling purposes and can be further complemented by groups whose protomeric and tautomeric states can be influenced by the protein environment. Overall, the functional-group diversity observed in the fragments analyzed in the current study strengthens the conceptual appeal of fragment-based methods. An instructive example of fragment adaptability to molecular interactions is provided by selected fragments that demonstrate paired H-bonds to the side chain of asparagine residues across the present data set (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">9</a></a>). In these fragments, nine individual atom types engaged the asparagine in paired H-bonds. The observed chemical and topological diversity is very inspiring for molecular design purposes, as it indicates opportunities for original bioisosteric replacements as well as for the optimization and diversification of pharmacophoric elements.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Fragment hits H-bonded to the side chain of asparagine. (a) Spatial distribution of the fragment atoms interacting with asparagine side chains (<i>N</i> = 70). (b–h) Diverse selection of fragment hits engaged in paired H-bonds to the side chain of asparagine (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CUR">4CUR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LR6">4LR6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ8">4TZ8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YK0">4YK0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYU">5DYU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E3G">5E3G</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9Y">5E9Y</a>). Fragment hits and the asparagine side chain (backbone atoms omitted for clarity) are depicted as bold sticks (cyan and light gray carbon atoms, respectively) and relevant H-bonds as dashed black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Water plays an important role in the binding of fragments to proteins.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> When only high-resolution (≤1.5 Å) structures are considered, 46% of the fragment hits establish at least one H-bond to structural water molecules in the binding pocket. Water molecules could form, for example, extended nonbonded interaction networks by filling pocket cavities and offering interaction hotspots for fragments (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">10</a></a>a,b). Importantly, in several cases, water-mediated H-bonds are the only polar interaction for the fragment hits (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">10</a></a>b,c). This limits the ability of the energetic and solvation approximations used in current modeling software to adequately characterize and predict fragment binding using computational methods such as docking and molecular dynamics.<a onclick="showRef(event, 'ref59 ref61 ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref59 ref61 ref62 ref63">(59,61−63)</a></div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Fragment hits H-bonded to structural water molecules. (a, b) Water molecules as part of extended nonbonded interaction networks (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MOH">5MOH</a>), (b, c) water molecules as the only H-bond partners for the fragment (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NOW">5NOW</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y3P">4Y3P</a>, respectively). Fragment hits are depicted as bold sticks (cyan carbon atoms), water molecules as red spheres, and relevant H-bonds as dashed black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Beyond H-Bonds</h4><div class="NLM_p">Although we are still far from a complete understanding of the energetics associated with H-bonds and metal-coordination bonds, these classes of polar interactions are relatively well studied, and medicinal chemists are accustomed to optimizing compounds based on them. Additional types of directional molecular interactions have only recently started to become more widely recognized, including arene-based contacts,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> weak H-bonds, such as carbon H-bonds,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> CH/π H-bonds,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> halogen bonds,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and sulfur-mediated contacts.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> More than half of the fragment hits display at least one such interaction, with arene contacts being the most frequent (42%). Although their occurrence is limited in comparison to canonical H-bonds, these additional directional interactions are likely to make important contributions to overall affinity in the context of fragment binding, where a reduced number of atoms is available for interactions. In a number of fragment hit–protein structures, such interactions stabilize the complex in the absence of more specific polar interactions (such as H-bonds), as shown by selected examples in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">11</a></a>. Here, arene groups on the fragment hits are sandwiched against peptide bonds (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">11</a></a>a) and stacked against the aromatic side chains of phenylalanine and tryptophan residues (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">11</a></a>b–d).</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Fragment hits engaged in arene-based interactions with the protein as the main attractive interaction in the absence of protein- and water-mediated H-bonds. (a) Arene interaction with backbone amide bonds (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K2Y">4K2Y</a>), (b, c) arene interaction with the side chains of phenylalanine (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y37">4Y37</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I5W">5I5W</a>), and (d) arene interaction with the side chain of tryptophan (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAN">5JAN</a>). Fragment hits are depicted as bold sticks (cyan carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The large variety of fragment heteroaromatic arrangements that are able to establish arene-type interactions is an indication of future opportunities for the design of novel fragments and, more importantly, for the generation of intellectual property during fragment optimization. The thienodiazaborinine scaffold engaged in a face-to-face stacking interaction with phenylalanine 291 in the binding pocket of endothiapepsin (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y37">4Y37</a>, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">11</a></a>) is an excellent example of under-represented and innovative heterocycles that could open up relevant pharmacophoric and chemical spaces for exploitation in fragment-based campaigns. The fine-tuning and optimization of such interactions at a fragment level still represent a significant challenge, given the conspicuous polarization and marked dispersive characteristics. To this end, the ability to query and mine existing structural data for nonbonded interactions, and to readily visualize<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> them in the context of a fragment-evolution effort, would greatly facilitate progress in this area.</div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04036" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04036" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Analysis of the fragment–protein complexes curated here highlights salient features of validated fragment hits originating from fragment-based screening efforts. Despite limitations in sample size at both the fragment and the protein levels, we believe that this data set offers important insights relevant to hit-discovery activities, including the design and selection of fragments for fragment-screening libraries and the evaluation of the quality of fragment hits. The observed topological and functional-group diversity of fragments coupled with their polarity–lipophilicity balance could, for example, inform fragment-library selection and expansion schemes. Likewise, the observed surface and interaction-based propensities of the fragment–protein complexes could support the development of intuitive classification methods during in silico pocket and fragment-hit identification. As the number of deposited protein structures with bound fragment hits increases, the preliminary analysis presented here could be updated and used to refine empirical potentials for protein–fragment interactions and develop probabilistic models for molecular design applications. Our analysis emphasizes the essential role played by crystallographers and the importance of structural information in fragment-based thinking and methodologies.</div><div class="NLM_p last">The structural and chemical details revealed by publicly available protein–fragment hits have the potential to significantly impact molecular design and drug discovery. We believe that the ability of users of fragment-based approaches to distill this information for compound design can be a significant determinant of success during the fragment-hit evaluation and the fragment hit-to-lead phases. In these processes, interactive visualization of bound fragment hits across drug discovery projects could further enhance design and idea generation.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01855" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01855" class="ext-link">10.1021/acs.jmedchem.8b01855</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figure S1: frequency distribution of interactions per fragment pocket; Figure S2: frequency distribution of total, polar, and apolar buried surface area (SA) of the fragments; Figure S3: characteristics of fragment hit protein targets; Figure S4: comparison between distributions of fragment hit and ligand features; Figure S5: comparison between surface characteristics of the fragment– and ligand–pocket complexes; Figure S6: comparison between interactions of the fragment– and ligand–pocket complexes; Table S1: occurrence of multiple fragment hit binding sites across various protein targets; Table S2: occurrence of Bemis–Murcko frameworks across 462 unique fragment hits; Table S3: occurrence of explicit individual ring assemblies across 462 unique fragment hits; Table S4: descriptive statistics of the fragment buried SA upon binding; Table S5: fragment atoms involved in coordination bonds to structural metal ions in protein binding pockets; Table S6: fragment atoms involved in H-bonds with structural water molecules in protein binding pockets with crystallographic resolution ≤1.5 Å; Table S7: fragment atoms involved in sulfur contacts, halogen bonds, and carbon hydrogen bonds with the protein; Table S8: PDB codes for the 445 protein–ligand complexes; Table S9: PDB codes, SMILES strings, and citation information for the 489 pocket–fragment complexes (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf">jm8b01855_si_001.pdf (1.26 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01855" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabrizio Giordanetto</span> - <span class="hlFld-Affiliation affiliation">D.
E. Shaw Research, New York, New York 10036, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9876-9552" title="Orcid link">http://orcid.org/0000-0001-9876-9552</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#feb89f9c8c97849791d0b997918c9a9f909b8a8a91bebabbad969f89ac9b8d9b9f8c9d96d09d9193"><span class="__cf_email__" data-cfemail="33755251415a495a5c1d745a5c4157525d5647475c737776605b5244615640565241505b1d505c5e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David E. Shaw</span> - <span class="hlFld-Affiliation affiliation">D.
E. Shaw Research, New York, New York 10036, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biophysics, Columbia University, New York, New York 10032, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8265-5761" title="Orcid link">http://orcid.org/0000-0001-8265-5761</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#90d4f1e6f9f4bec3f8f1e7d0d4d5c3f8f1e7c2f5e3f5f1e2f3f8bef3fffd"><span class="__cf_email__" data-cfemail="ace8cddac5c882ffc4cddbece8e9ffc4cddbfec9dfc9cddecfc482cfc3c1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chentian Jin</span> - <span class="hlFld-Affiliation affiliation">D.
E. Shaw Research, New York, New York 10036, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsay Willmore</span> - <span class="hlFld-Affiliation affiliation">D.
E. Shaw Research, New York, New York 10036, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miklos Feher</span> - <span class="hlFld-Affiliation affiliation">D.
E. Shaw Research, New York, New York 10036, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><p>The authors declare the following competing financial interest(s): This study was conducted and funded internally by D. E. Shaw Research, of which D.E.S. is the sole beneficial owner and Chief Scientist, and with which all authors are affiliated.<br /></br></p></li></ul></div><div class="ack" id="ACK-d7e1632-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Michael Eastwood, David Borhani, Yakov Pechersky, and Dina Sharon for helpful discussions and a critical reading of the manuscript and Rebecca Bish-Cornelissen and Berkman Frank for editorial assistance.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ECFP4</td><td class="NLM_def"><p class="first last">extended connectivity fingerprints</p></td></tr><tr><td class="NLM_term">PDE10A2</td><td class="NLM_def"><p class="first last">cAMP and cAMP-inhibited cGMP 3′,5′-cyclic phosphodiesterase 10A2</p></td></tr><tr><td class="NLM_term">Hsp90</td><td class="NLM_def"><p class="first last">heat-shock protein 90</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">BAZ2B</td><td class="NLM_def"><p class="first last">bromodomain adjacent to zinc finger domain protein 2B</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57444" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57444" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 66 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A “Rule of Three” for Fragment-Based Lead Discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%9CRule+of+Three%E2%80%9D+for+Fragment-Based+Lead+Discovery%3F&doi=10.1016%2FS1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2eY5-X3mrvX8s7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%259CRule%2520of%2520Three%25E2%2580%259D%2520for%2520Fragment-Based%2520Lead%2520Discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877%26doi%3D10.1016%2FS1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuker, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovering High-Affinity Ligands for Proteins: SAR by NMR</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+High-Affinity+Ligands+for+Proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0liAjUTBOEMQpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520High-Affinity%2520Ligands%2520for%2520Proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">5003</span>, <span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-Based+Approaches+in+Drug+Discovery+and+Chemical+Biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0liO0qq2I6__sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-Based%2520Approaches%2520in%2520Drug%2520Discovery%2520and%2520Chemical%2520Biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span> <span> </span><span class="NLM_article-title">Introduction to Fragment-Based Drug Discovery</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1007/128_2011_180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21695633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=1-32&author=D.+A.+Erlanson&title=Introduction+to+Fragment-Based+Drug+Discovery&doi=10.1007%2F128_2011_180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to fragment-based drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has emerged in the past decade as a powerful tool for discovering drug leads.  The approach first identifies starting points: very small mols. (fragments) that are about half the size of typical drugs.  These fragments are then expanded or linked together to generate drug leads.  Although the origins of the technique date back some 30 years, it was only in the mid-1990s that exptl. techniques became sufficiently sensitive and rapid for the concept to be become practical.  Since that time, the field has exploded: FBDD has played a role in discovery of at least 18 drugs that have entered the clinic, and practitioners of FBDD can be found throughout the world in both academia and industry.  Literally dozens of reviews have been published on various aspects of FBDD or on the field as a whole, as have three books (Jahnke and Erlanson, Fragment-based approaches in drug discovery, 2006; Zartler and Shapiro, Fragment-based drug discovery: a practical approach, 2008; Kuo, Fragment based drug design: tools, practical approaches, and examples, 2011).  However, this chapter will assume that the reader is approaching the field with little prior knowledge.  It will introduce some of the key concepts, set the stage for the chapters to follow, and demonstrate how X-ray crystallog. plays a central role in fragment identification and advancement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcNmFPvb7c5LVg90H21EOLACvtfcHk0liO0qq2I6__sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvM&md5=735c05a6b5f1d4ee9e4d34e15ac2b8fe</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2012%26volume%3D317%26spage%3D1%26epage%3D32%26doi%3D10.1007%2F128_2011_180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">The Rise of Fragment-Based Drug Discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+Rise+of+Fragment-Based+Drug+Discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0liO0qq2I6__sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520Rise%2520of%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Visegrády, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery on G-Protein-Coupled Receptors</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1517/17460441.2013.794135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1517%2F17460441.2013.794135" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=811-820&author=A.+Visegr%C3%A1dyauthor=G.+M.+Keser%C5%B1&title=Fragment-Based+Lead+Discovery+on+G-Protein-Coupled+Receptors&doi=10.1517%2F17460441.2013.794135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.794135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.794135%26sid%3Dliteratum%253Aachs%26aulast%3DVisegr%25C3%25A1dy%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DFragment-Based%2520Lead%2520Discovery%2520on%2520G-Protein-Coupled%2520Receptors%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D811%26epage%3D820%26doi%3D10.1517%2F17460441.2013.794135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Experiences in Fragment-Based Drug Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2012.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.tips.2012.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=22459076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=224-232&author=C.+W.+Murrayauthor=M.+L.+Verdonkauthor=D.+C.+Rees&title=Experiences+in+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.tips.2012.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">224-232</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has become established in both industry and academia as an alternative approach to high-throughput screening for the generation of chem. leads for drug targets.  In FBDD, specialised detection methods are used to identify small chem. compds. (fragments) that bind to the drug target, and structural biol. is usually employed to establish their binding mode and to facilitate their optimization.  In this article, we present three recent and successful case histories in FBDD.  We then re-examine the key concepts and challenges of FBDD with particular emphasis on recent literature and our own experience from a substantial no. of FBDD applications.  Our opinion is that careful application of FBDD is living up to its promise of delivering high quality leads with good phys. properties and that in future many drug mols. will be derived from fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUzcsuUdmx4bVg90H21EOLACvtfcHk0li_-OdcfYTuZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D&md5=c97b6b84a4afeadb87ee7d46e6d2bb91</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DExperiences%2520in%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D224%26epage%3D232%26doi%3D10.1016%2Fj.tips.2012.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heikkila, T. J.</span>; <span class="NLM_string-name">Surade, S.</span>; <span class="NLM_string-name">Silvestre, H. L.</span>; <span class="NLM_string-name">Dias, M. V. B.</span>; <span class="NLM_string-name">Ciulli, A.</span>; <span class="NLM_string-name">Bromfield, K.</span>; <span class="NLM_string-name">Scott, D.</span>; <span class="NLM_string-name">Howard, N.</span>; <span class="NLM_string-name">Wen, S.</span>; <span class="NLM_string-name">Wei, A. H.</span>; <span class="NLM_string-name">Osborne, D.</span>; <span class="NLM_string-name">Abell, C.</span>; <span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery in Academia: Experiences From a Tuberculosis Programme</span>. In  <i>From Molecules to Medicines</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sussman, J. L.</span>; <span class="NLM_string-name">Spadon, P.</span></span>, Eds.; NATO Science for Peace and Security Series A: Chemistry and Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Netherlands</span>, <span class="NLM_year">2009</span>; pp  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">36</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2F978-90-481-2339-1_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=21-36&author=T.+J.+Heikkila&author=S.+Surade&author=H.+L.+Silvestre&author=M.+V.+B.+Dias&author=A.+Ciulli&author=K.+Bromfield&author=D.+Scott&author=N.+Howard&author=S.+Wen&author=A.+H.+Wei&author=D.+Osborne&author=C.+Abell&author=T.+L.+Blundellauthor=J.+L.+Sussman&author=P.+Spadon&title=From+Molecules+to+Medicines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2F978-90-481-2339-1_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-90-481-2339-1_3%26sid%3Dliteratum%253Aachs%26aulast%3DHeikkila%26aufirst%3DT.%2BJ.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520in%2520Academia%253A%2520Experiences%2520From%2520a%2520Tuberculosis%2520Programme%26btitle%3DFrom%2520Molecules%2520to%2520Medicines%26aulast%3DSussman%26aufirst%3DJ.%2BL.%26pub%3DSpringer%26date%3D2009%26spage%3D21%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zartler, E.
R.</span></span> <span> </span><span class="NLM_article-title">Fragonomics: The -omics with Real Impact</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1021/ml5003212</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5003212" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtleqtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=952-953&author=E.%0AR.+Zartler&title=Fragonomics%3A+The+-omics+with+Real+Impact&doi=10.1021%2Fml5003212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Fragonomics: The -Omics with Real Impact</span></div><div class="casAuthors">Zartler, Edward R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-953</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragonomics is the process of using small, relatively simple mols. to generate chem. starting points for hit generation.  Fragonomics has come of age and is now one of the major concepts in hit generation.  What is its future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1kUkPpbIBrVg90H21EOLACvtfcHk0lh4mxCsk2y6rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtleqtLjK&md5=2c57c10c7f6f74f9dfb804342ea66a70</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml5003212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5003212%26sid%3Dliteratum%253Aachs%26aulast%3DZartler%26aufirst%3DE.%2BR.%26atitle%3DFragonomics%253A%2520The%2520-omics%2520with%2520Real%2520Impact%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D952%26epage%3D953%26doi%3D10.1021%2Fml5003212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romasanta, A. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Sijde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellsten, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Muijlwijk-Koezen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span> <span> </span><span class="NLM_article-title">When Fragments Link: A Bibliometric Perspective on the Development of Fragment-Based Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.drudis.2018.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=29738823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1596-1609&author=A.+K.+S.+Romasantaauthor=P.+van+der+Sijdeauthor=I.+Hellstenauthor=R.+E.+Hubbardauthor=G.+M.+Keseruauthor=J.+van+Muijlwijk-Koezenauthor=I.+J.+P.+de+Esch&title=When+Fragments+Link%3A+A+Bibliometric+Perspective+on+the+Development+of+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.drudis.2018.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">When fragments link: a bibliometric perspective on the development of fragment-based drug discovery</span></div><div class="casAuthors">Romasanta, Angelo K. S.; van der Sijde, Peter; Hellsten, Iina; Hubbard, Roderick E.; Keseru, Gyorgy M.; van Muijlwijk-Koezen, Jacqueline; de Esch, Iwan J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1596-1609</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a highly interdisciplinary field, rich in ideas integrated from pharmaceutical sciences, chem., biol., and physics, among others.  To enrich our understanding of the development of the field, we used bibliometric techniques to analyze 3642 publications in FBDD, complementing accounts by key practitioners.  Mapping its core papers, we found the transfer of knowledge from academia to industry.  Co-authorship anal. showed that university-industry collaboration has grown over time.  Moreover, we show how ideas from other scientific disciplines have been integrated into the FBDD paradigm.  Keyword anal. showed that the field is organized into four interconnected practices: library design, fragment screening, computational methods, and optimization.  This study highlights the importance of interactions among various individuals and institutions from diverse disciplines in newly emerging scientific fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5WVEQd1QaULVg90H21EOLACvtfcHk0ljBH2bk5hsmnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlaqtb0%253D&md5=5b8042da1162e24116f29867d0c4b97b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DRomasanta%26aufirst%3DA.%2BK.%2BS.%26aulast%3Dvan%2Bder%2BSijde%26aufirst%3DP.%26aulast%3DHellsten%26aufirst%3DI.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26atitle%3DWhen%2520Fragments%2520Link%253A%2520A%2520Bibliometric%2520Perspective%2520on%2520the%2520Development%2520of%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D1596%26epage%3D1609%26doi%3D10.1016%2Fj.drudis.2018.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haass, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproesser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settachatgul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Selective Inhibitor of Oncogenic B-Raf Kinase with Potent Antimelanoma Activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">3041</span>– <span class="NLM_lpage">3046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=W.+Wangauthor=J.+Zhangauthor=H.+Choauthor=S.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+Smalleyauthor=D.+Fongauthor=Y.+L.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.+H.+Kimauthor=J.+Schlessingerauthor=K.+Y.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+Selective+Inhibitor+of+Oncogenic+B-Raf+Kinase+with+Potent+Antimelanoma+Activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0ljBH2bk5hsmnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DS.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BL.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520Selective%2520Inhibitor%2520of%2520Oncogenic%2520B-Raf%2520Kinase%2520with%2520Potent%2520Antimelanoma%2520Activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolop, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span> <span> </span><span class="NLM_article-title">Vemurafenib: The First Drug Approved for BRAF-Mutant Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">886</span>, <span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+The+First+Drug+Approved+for+BRAF-Mutant+Cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0ljBH2bk5hsmnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520The%2520First%2520Drug%2520Approved%2520for%2520BRAF-Mutant%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Souers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leverson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boghaert, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catron, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enschede, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymowitz, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maecker, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitten, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleksijew, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahir, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humerickhouse, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span> <span> </span><span class="NLM_article-title">ABT-199, a Potent and Selective BCL-2 Inhibitor, Achieves Antitumor Activity While Sparing Platelets</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1038/nm.3048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnm.3048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23291630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=202-208&author=A.+J.+Souersauthor=J.+D.+Leversonauthor=E.+R.+Boghaertauthor=S.+L.+Acklerauthor=N.+D.+Catronauthor=J.+Chenauthor=B.+D.+Daytonauthor=H.+Dingauthor=S.+H.+Enschedeauthor=W.+J.+Fairbrotherauthor=D.+C.+S.+Huangauthor=S.+G.+Hymowitzauthor=S.+Jinauthor=S.+L.+Khawauthor=P.+J.+Kovarauthor=L.+T.+Lamauthor=J.+Leeauthor=H.+L.+Maeckerauthor=K.+C.+Marshauthor=K.+D.+Masonauthor=M.+J.+Mittenauthor=P.+M.+Nimmerauthor=A.+Oleksijewauthor=C.+H.+Parkauthor=C.-M.+Parkauthor=D.+C.+Phillipsauthor=A.+W.+Robertsauthor=D.+Sampathauthor=J.+F.+Seymourauthor=M.+L.+Smithauthor=G.+M.+Sullivanauthor=S.+K.+Tahirauthor=C.+Tseauthor=M.+D.+Wendtauthor=Y.+Xiaoauthor=J.+C.+Xueauthor=H.+Zhangauthor=R.+A.+Humerickhouseauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=ABT-199%2C+a+Potent+and+Selective+BCL-2+Inhibitor%2C+Achieves+Antitumor+Activity+While+Sparing+Platelets&doi=10.1038%2Fnm.3048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-199, a potent and selective BCL-2 inhibitor, achieves antitumor activity while sparing platelets</span></div><div class="casAuthors">Souers, Andrew J.; Leverson, Joel D.; Boghaert, Erwin R.; Ackler, Scott L.; Catron, Nathaniel D.; Chen, Jun; Dayton, Brian D.; Ding, Hong; Enschede, Sari H.; Fairbrother, Wayne J.; Huang, David C. S.; Hymowitz, Sarah G.; Jin, Sha; Khaw, Seong Lin; Kovar, Peter J.; Lam, Lloyd T.; Lee, Jackie; Maecker, Heather L.; Marsh, Kennan C.; Mason, Kylie D.; Mitten, Michael J.; Nimmer, Paul M.; Oleksijew, Anatol; Park, Chang H.; Park, Cheol-Min; Phillips, Darren C.; Roberts, Andrew W.; Sampath, Deepak; Seymour, John F.; Smith, Morey L.; Sullivan, Gerard M.; Tahir, Stephen K.; Tse, Chris; Wendt, Michael D.; Xiao, Yu; Xue, John C.; Zhang, Haichao; Humerickhouse, Rod A.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-208</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the B cell CLL/lymphoma 2 (BCL-2) family are key regulators of the apoptotic process.  This family comprises proapoptotic and prosurvival proteins, and shifting the balance toward the latter is an established mechanism whereby cancer cells evade apoptosis.  The therapeutic potential of directly inhibiting prosurvival proteins was unveiled with the development of navitoclax, a selective inhibitor of both BCL-2 and BCL-2-like 1 (BCL-XL), which has shown clin. efficacy in some BCL-2-dependent hematol. cancers.  However, concomitant on-target thrombocytopenia caused by BCL-XL inhibition limits the efficacy achievable with this agent.  Here we report the re-engineering of navitoclax to create a highly potent, orally bioavailable and BCL-2-selective inhibitor, ABT-199.  This compd. inhibits the growth of BCL-2-dependent tumors in vivo and spares human platelets.  A single dose of ABT-199 in three patients with refractory chronic lymphocytic leukemia resulted in tumor lysis within 24 h.  These data indicate that selective pharmacol. inhibition of BCL-2 shows promise for the treatment of BCL-2-dependent hematol. cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7P7koWwJHLVg90H21EOLACvtfcHk0liXVE9DNNfBhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D&md5=6a24b72303eb2062da653f1f61ac74fb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.3048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3048%26sid%3Dliteratum%253Aachs%26aulast%3DSouers%26aufirst%3DA.%2BJ.%26aulast%3DLeverson%26aufirst%3DJ.%2BD.%26aulast%3DBoghaert%26aufirst%3DE.%2BR.%26aulast%3DAckler%26aufirst%3DS.%2BL.%26aulast%3DCatron%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDayton%26aufirst%3DB.%2BD.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DEnschede%26aufirst%3DS.%2BH.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKhaw%26aufirst%3DS.%2BL.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DMaecker%26aufirst%3DH.%2BL.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMason%26aufirst%3DK.%2BD.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNimmer%26aufirst%3DP.%2BM.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPhillips%26aufirst%3DD.%2BC.%26aulast%3DRoberts%26aufirst%3DA.%2BW.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSeymour%26aufirst%3DJ.%2BF.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DSullivan%26aufirst%3DG.%2BM.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHumerickhouse%26aufirst%3DR.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DABT-199%252C%2520a%2520Potent%2520and%2520Selective%2520BCL-2%2520Inhibitor%252C%2520Achieves%2520Antitumor%2520Activity%2520While%2520Sparing%2520Platelets%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D202%26epage%3D208%26doi%3D10.1038%2Fnm.3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty Years on: The Impact of Fragments on Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+Years+on%3A+The+Impact+of+Fragments+on+Drug+Discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0liXVE9DNNfBhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520Years%2520on%253A%2520The%2520Impact%2520of%2520Fragments%2520on%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jencks, W. P.</span></span> <span> </span><span class="NLM_article-title">On the Attribution and Additivity of Binding Energies</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">4046</span>– <span class="NLM_lpage">4050</span>, <span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+Attribution+and+Additivity+of+Binding+Energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0liEf7C_Uw87Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520Attribution%2520and%2520Additivity%2520of%2520Binding%2520Energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span> <span> </span><span class="NLM_article-title">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+Complexity+and+Its+Impact+on+the+Probability+of+Finding+Leads+for+Drug+Discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0liEf7C_Uw87Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520Complexity%2520and%2520Its%2520Impact%2520on%2520the%2520Probability%2520of%2520Finding%2520Leads%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">The Chemical Space Project</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1021/ar500432k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500432k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=722-730&author=J.-L.+Reymond&title=The+Chemical+Space+Project&doi=10.1021%2Far500432k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemical Space Project</span></div><div class="casAuthors">Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-730</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the simplest questions that can be asked about mol. diversity is how many org. mols. are possible in total.  To answer this question, my research group has computationally enumerated all possible org. mols. up to a certain size to gain an unbiased insight into the entire chem. space.  Our latest database, GDB-17, contains 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens, by far the largest small mol. database reported to date.  Mols. allowed by valency rules but unstable or nonsynthesizable due to strained topologies or reactive functional groups were not considered, which reduced the enumeration by at least 10 orders of magnitude and was essential to arrive at a manageable database size.  Despite these restrictions, GDB-17 is highly relevant with respect to known mols.  Beyond enumeration, understanding and exploiting GDBs (generated databases) led us to develop methods for virtual screening and visualization of very large databases in the form of a "periodic system of mols." comprising six different fingerprint spaces, with web-browsers for nearest neighbor searches, and the MQN- and SMIfp-Mapplet application for exploring color-coded principal component maps of GDB and other large databases.  Proof-of-concept applications of GDB for drug discovery were realized by combining virtual screening with chem. synthesis and activity testing for neurotransmitter receptor and transporter ligands.  One surprising lesson from using GDB for drug analog searches is the incredible depth of chem. space, i.e., the fact that millions of very close analogs of any mol. can be readily identified by nearest-neighbor searches in the MQN-space of the various GDBs.  The chem. space project has opened an unprecedented door on chem. diversity.  Ongoing and yet unmet challenges concern enumerating mols. beyond 17 atoms and synthesizing GDB mols. with innovative scaffolds and pharmacophores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxJYiz7ZE3ALVg90H21EOLACvtfcHk0liEf7C_Uw87Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D&md5=d3199b2273a0e8d92a4b1ffa59c6dffc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Far500432k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500432k%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3DThe%2520Chemical%2520Space%2520Project%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D722%26epage%3D730%26doi%3D10.1021%2Far500432k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruedisser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span> <span> </span><span class="NLM_article-title">Library Design for Fragment Based Screening</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.2174/1568026054637700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.2174%2F1568026054637700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=16101415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=751-762&author=A.+Schuffenhauerauthor=S.+Ruedisserauthor=A.+L.+Marzinzikauthor=W.+Jahnkeauthor=M.+Blommersauthor=P.+Selzerauthor=E.+Jacoby&title=Library+Design+for+Fragment+Based+Screening&doi=10.2174%2F1568026054637700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Library design for fragment based screening</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ruedisser, Simon; Marzinzik, Andreas L.; Jahnke, Wolfgang; Blommers, Marcel; Selzer, Paul; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">751-762</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  According to Hann's model of mol. complexity an increased probability of detection binding to a target protein can be expected when small, low complex mol. fragments are screened with high sensitivity instead of full-sized ligands with lower sensitivity.  Anal. of the HTS summary data of Novartis and comparison with NMR screening results obtained on generic fragment libraries indicate this expectation to be true with hitrates of 0.001% - 0.151% obsd. in the identification of ligands with an IC50 threshold in the micromolar range in an HTS setup and hitrates above or equal to 3% obsd. in NMR screening of fragments with an affinity threshold in the millimolar range.  It is however necessary to keep in mind that the sets of target studied were not identical for both method and the experience in NMR screening is too limited for a final conclusion.  The term hitrate as used here reflects only the success rate in the observation of ligand binding event.  It must not be confused with the overall success rate of fragment and high throughput screening in the lead finding process, which can be entirely different, since the steps required to follow-up a ligand binding event to a lead are different for both methods.  A survey of fragment-based lead discovery case studies given in the literature shows that in approx. half of the cases the initial hit fragment was discovered by screening a generic library, whereas in the other cases some knowledge about an initial ligands or the protein binding site has been used, whereas systematic virtual screening of fragment databases has been only rarely reported.  As comparatively high hitrates were obtained, further consideration to optimize the generic fragment screening library were directed to the chem. tractability of the fragment.  As several functional groups preferred by chemists for modification and linking of the fragments are also preferentially involved in interactions between the fragments and the target protein, a set of screening fragments was derived from chem. building blocks by masking its linker group by a chem. transformation which can be later on used in the chem. follow-up of the fragment hit.  For example primary amines can be masked as acetamides.  If the screening fragment is active the related building block can then be used for synthesis of a follow-up library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCOn-N__v0t7Vg90H21EOLACvtfcHk0lhDnaloPJpB1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht78%253D&md5=7fd149986a1e2472be22bcd4e142b47c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1568026054637700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637700%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DRuedisser%26aufirst%3DS.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBlommers%26aufirst%3DM.%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DLibrary%2520Design%2520for%2520Fragment%2520Based%2520Screening%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D751%26epage%3D762%26doi%3D10.2174%2F1568026054637700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrmann, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1693-1701&author=J.+Schiebelauthor=N.+Radevaauthor=S.+G.+Krimmerauthor=X.+Wangauthor=M.+Stielerauthor=F.+R.+Ehrmannauthor=K.+Fuauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=A.+Heineauthor=G.+Klebe&title=Six+Biophysical+Screening+Methods+Miss+a+Large+Proportion+of+Crystallographically+Discovered+Fragment+Hits%3A+A+Case+Study&doi=10.1021%2Facschembio.5b01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Krimmer, Stefan G.; Wang, Xiaojie; Stieler, Martin; Ehrmann, Frederik R.; Fu, Kan; Metz, Alexander; Huschmann, Franziska U.; Weiss, Manfred S.; Mueller, Uwe; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1693-1701</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) has become a pillar in drug development.  Typical applications of this method comprise at least two biophys. screens as prefilter and a follow-up crystallog. expt. on a subset of fragments.  Clearly, structural information is pivotal in FBLD, but a key question is whether such a screening cascade strategy will retrieve the majority of fragment-bound structures.  We therefore set out to screen 361 fragments for binding to endothiapepsin, a representative of the challenging group of aspartic proteases, employing six screening techniques and crystallog. in parallel.  Crystallog. resulted in the very high no. of 71 structures.  Yet alarmingly, 44% of these hits were not detected by any biophys. screening approach.  Moreover, any screening cascade, building on the results from two or more screening methods, would have failed to predict at least 73% of these hits.  We thus conclude that, at least in the present case, the frequently applied biophys. prescreening filters deteriorate the no. of possible X-ray hits while only the immediate use of crystallog. enables exhaustive retrieval of a max. of fragment structures, which represent a rich source guiding hit-to-lead-to-drug evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgOpXVyCfeiLVg90H21EOLACvtfcHk0lhDnaloPJpB1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D&md5=30f16cf75361d0ca11e7b75ad796d207</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01034%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSix%2520Biophysical%2520Screening%2520Methods%2520Miss%2520a%2520Large%2520Proportion%2520of%2520Crystallographically%2520Discovered%2520Fragment%2520Hits%253A%2520A%2520Case%2520Study%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1693%26epage%3D1701%26doi%3D10.1021%2Facschembio.5b01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural Biology in Fragment-Based Drug Design</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.sbi.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=20471246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=497-507&author=C.+W.+Murrayauthor=T.+L.+Blundell&title=Structural+Biology+in+Fragment-Based+Drug+Design&doi=10.1016%2Fj.sbi.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology in fragment-based drug design</span></div><div class="casAuthors">Murray, Christopher W.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based ligand screening is now established as an emerging paradigm for drug discovery.  Here we examine the recent literature looking at how structural biol. has been used in a variety of successful fragment-screening applications.  We argue that the detn. of exptl. binding modes has proved to be one of the mainstays of successful fragment-based approaches and that this reflects the difficulty in optimizing a fragment to a lead mol. in the absence of structural information.  We focus on antimicrobial research where fragment-based drug discovery allows control of the phys. properties of the emerging lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1S2lx8J-FX7Vg90H21EOLACvtfcHk0lhDnaloPJpB1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF&md5=ae8cfc0f8be47a316bd92073e95e0c8a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520Biology%2520in%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D497%26epage%3D507%26doi%3D10.1016%2Fj.sbi.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2015</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01123</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01123" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1KlsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=89-99&author=C.+N.+Johnsonauthor=D.+A.+Erlansonauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2015&doi=10.1021%2Facs.jmedchem.6b01123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-to-Lead Medicinal Chemistry Publications in 2015</span></div><div class="casAuthors">Johnson, Christopher N.; Erlanson, Daniel A.; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-99</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) is now well-established as a technol. for generating new chem. leads and drugs.  This Miniperspective provides a tabulated overview of the fragment-to-lead literature published in the year 2015, together with a commentary on trends obsd. across the FBDD field during this time.  It is hoped that this tabulated summary will provide a useful point of ref. for both FBDD practitioners and the wider medicinal chem. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojX6ZlmVXQrLVg90H21EOLACvtfcHk0lhDnaloPJpB1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1KlsbnP&md5=da9d027787d70d4276378cb01f8e0ce2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01123%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202015%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D89%26epage%3D99%26doi%3D10.1021%2Facs.jmedchem.6b01123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2016</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1774</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslentrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1774-1784&author=C.+N.+Johnsonauthor=D.+A.+Erlansonauthor=W.+Jahnkeauthor=P.+N.+Mortensonauthor=D.+C.+Rees&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2016&doi=10.1021%2Facs.jmedchem.7b01298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-to-Lead Medicinal Chemistry Publications in 2016</span></div><div class="casAuthors">Johnson, Christopher N.; Erlanson, Daniel A.; Jahnke, Wolfgang; Mortenson, Paul N.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1774-1784</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The popularity of fragment-based drug discovery (FBDD) is demonstrated by the no. of recent successful fragment-to-lead (F2L) publications.  This Miniperspective provides a tabulated summary of the F2L literature published in the year 2016, along with discussion of general trends.  It uses the same format as our summary of the 2015 literature and is intended to be a resource for both FBDD practitioners and medicinal chemists in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDiwnepXDgrbVg90H21EOLACvtfcHk0lh0zNfZE-ZenQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslentrfP&md5=861d9adf1530250a060fc254ca7b493c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01298%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202016%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1774%26epage%3D1784%26doi%3D10.1021%2Facs.jmedchem.7b01298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=P.+N.+Mortensonauthor=D.+A.+Erlansonauthor=I.+J.+P.+de+Eschauthor=W.+Jahnkeauthor=C.+N.+Johnson&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2017&doi=10.1021%2Facs.jmedchem.8b01472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202017%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26doi%3D10.1021%2Facs.jmedchem.8b01472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lh0zNfZE-ZenQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deller, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupp, B.</span></span> <span> </span><span class="NLM_article-title">Models of Protein-Ligand Crystal Structures: Trust, but Verify</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">817</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9833-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2Fs10822-015-9833-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25665575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFektb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=817-836&author=M.+C.+Dellerauthor=B.+Rupp&title=Models+of+Protein-Ligand+Crystal+Structures%3A+Trust%2C+but+Verify&doi=10.1007%2Fs10822-015-9833-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Models of protein-ligand crystal structures: trust, but verify</span></div><div class="casAuthors">Deller, Marc C.; Rupp, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">817-836</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">X-ray crystallog. provides the most accurate models of protein-ligand structures.  These models serve as the foundation of many computational methods including structure prediction, mol. modeling, and structure-based drug design.  The success of these computational methods ultimately depends on the quality of the underlying protein-ligand models.  X-ray crystallog. offers the unparalleled advantage of a clear math. formalism relating the exptl. data to the protein-ligand model.  In the case of X-ray crystallog., the primary exptl. evidence is the electron d. of the mols. forming the crystal.  The first step in the generation of an accurate and precise crystallog. model is the interpretation of the electron d. of the crystal, typically carried out by construction of an at. model.  The at. model must then be validated for fit to the exptl. electron d. and also for agreement with prior expectations of stereochem.  Stringent validation of protein-ligand models has become possible as a result of the mandatory deposition of primary diffraction data, and many computational tools are now available to aid in the validation process.  Validation of protein-ligand complexes has revealed some instances of overenthusiastic interpretation of ligand d.  Fundamental concepts and metrics of protein-ligand quality validation are discussed and we highlight software tools to assist in this process.  It is essential that end users select high quality protein-ligand models for their computational and biol. studies, and we provide an overview of how this can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnJmfgZbDk2bVg90H21EOLACvtfcHk0lh0zNfZE-ZenQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFektb8%253D&md5=aec55797c308266a49bd34aff9b95bbd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9833-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9833-8%26sid%3Dliteratum%253Aachs%26aulast%3DDeller%26aufirst%3DM.%2BC.%26aulast%3DRupp%26aufirst%3DB.%26atitle%3DModels%2520of%2520Protein-Ligand%2520Crystal%2520Structures%253A%2520Trust%252C%2520but%2520Verify%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D817%26epage%3D836%26doi%3D10.1007%2Fs10822-015-9833-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Protonate3D: Assignment of Ionization States and Hydrogen Coordinates to Macromolecular Structures</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+Assignment+of+Ionization+States+and+Hydrogen+Coordinates+to+Macromolecular+Structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0liLrKvMXpwouQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520Assignment%2520of%2520Ionization%2520States%2520and%2520Hydrogen%2520Coordinates%2520to%2520Macromolecular%2520Structures%26jtitle%3DProteins%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0liLrKvMXpwouQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span> <i>JChem</i>, Version 16.4.25; <span class="NLM_publisher-name">ChemAxon</span>, <span class="NLM_year">2016</span>. <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+JChem%2C+Version+16.4.25%3B+ChemAxon%2C+2016.+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DJChem%26pub%3DChemAxon%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carugo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordo, D.</span></span> <span> </span><span class="NLM_article-title">How Many Water Molecules Can Be Detected by Protein Crystallography?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1107/S0907444998012086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1107%2FS0907444998012086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10089359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=479-483&author=O.+Carugoauthor=D.+Bordo&title=How+Many+Water+Molecules+Can+Be+Detected+by+Protein+Crystallography%3F&doi=10.1107%2FS0907444998012086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How many water molecules can be detected by protein crystallography?</span></div><div class="casAuthors">Carugo, Oliviero; Bordo, Domenico</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">D55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">479-483</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The no. of water mols. which are expected to be exptl. located by protein crystallog. was detd. by multiple regression anal. on a test set of 873 known protein crystal structures detd. at room temp. and on another set of 33 structures detd. at low temp.  The dependence of the no. of water mols. included in the protein models as a function of a no. of significant regressors, such as resoln., fraction of crystal vol. occupied by the solvent, no. of residues in the asym. unit, fraction of apolar protein surface or secondary structure, has been studied.  The no. of water mols. included in crystallog. models depends primarily on the resoln. at which the structure has been solved, while the temp. of the data collection has only marginal influence.  On av., at 2.0 Å resoln. one water mol. per residue is included in the model, while at 1.0 Å resoln. about 1.6-1.7 are crystallog. located.  At 2.0 Å resoln. the well known rule-of-thumb of "one water per protein residue" is confirmed, though the no. of water mols. exptl. obsd. is strongly dependent on resoln.  The results presented are useful in assessing the quality of a protein crystal structure, in selecting structural results to be compared and in evaluating the expected improvement on the solvent structure when increasing the crystallog. resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2Ji9xBF-6R7Vg90H21EOLACvtfcHk0liLrKvMXpwouQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFGnsLc%253D&md5=c7321eca9f64b228b8a26264f77b1d53</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444998012086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998012086%26sid%3Dliteratum%253Aachs%26aulast%3DCarugo%26aufirst%3DO.%26aulast%3DBordo%26aufirst%3DD.%26atitle%3DHow%2520Many%2520Water%2520Molecules%2520Can%2520Be%2520Detected%2520by%2520Protein%2520Crystallography%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1999%26volume%3D55%26spage%3D479%26epage%3D483%26doi%3D10.1107%2FS0907444998012086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span> <i>Molecular Operating Environment (MOE)</i>,
2015.10; <span class="NLM_publisher-name">Chemical Computing Group, Inc.</span>: <span class="NLM_publisher-loc">1010
Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C%0A2015.10%3B+Chemical+Computing+Group%2C+Inc.%3A+1010%0ASherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%252C%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljH4MiHZECVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrauskas, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolovanov, E. A.</span></span> <span> </span><span class="NLM_article-title">ACD/Log P Method Description</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1023/A:1008719622770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1023%2FA%3A1008719622770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnslaitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=99-116&author=A.+A.+Petrauskasauthor=E.+A.+Kolovanov&title=ACD%2FLog+P+Method+Description&doi=10.1023%2FA%3A1008719622770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ACD/Log P method description</span></div><div class="casAuthors">Petrauskas, Alanas A.; Kolovanov, Eduard A.</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">Hydrophobicity and Solvation in Drug Design, Pt. 3</span>),
    <span class="NLM_cas:pages">99-116</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">This study describes the development of the ACD/Log P calcn. method.  Anal. of 14 calcn. methods revealed that the most accurate calcns. are obtained when correction factors are used.  The authors evaluated the correction factors used by Hansch and Leo in CLOGP to simplify their method.  Most of the CLOGP structural factors are included in fragmental increments.  Aliph. and arom. factors are replaced with additive interfragmental increments.  Missing increments are estd. by two empirical equations with simple phys. interpretation.  The final method uses three simple equations with several types of parameters.  The training set included 3601 compds. and the correlation between exptl. and calcd. Log P values gave R = 0.992, S = 0.21.  The method was validated by comparing it with 17 other methods on various data sets of independently selected drugs and other compds.  In all cases, method produced the best results.  The weakness of this method is that it uses a large no. of individual increments for arom. interactions.  Each increment represents a combination of several effects which presently cannot be sepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0Ih18c8CWcrVg90H21EOLACvtfcHk0ljH4MiHZECVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnslaitbc%253D&md5=9980bde748cf510386050ba5369446aa</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008719622770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008719622770%26sid%3Dliteratum%253Aachs%26aulast%3DPetrauskas%26aufirst%3DA.%2BA.%26aulast%3DKolovanov%26aufirst%3DE.%2BA.%26atitle%3DACD%252FLog%2520P%2520Method%2520Description%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D19%26spage%3D99%26epage%3D116%26doi%3D10.1023%2FA%3A1008719622770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span> <i>ACD Percepta Batch,
2012 Release</i>; <span class="NLM_publisher-name">Advanced Chemistry Development,
Inc.</span>: <span class="NLM_publisher-loc">8 King Street East, Suite 107,
Toronto, Ontario, M5C 1B5, Canada</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ACD+Percepta+Batch%2C%0A2012+Release%3B+Advanced+Chemistry+Development%2C%0AInc.%3A+8+King+Street+East%2C+Suite+107%2C%0AToronto%2C+Ontario%2C+M5C+1B5%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACD%2520Percepta%2520Batch%252C%250A2012%2520Release%26pub%3DAdvanced%2520Chemistry%2520Development%252C%250AInc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span> <i>Vortex</i>, 2016.08; <span class="NLM_publisher-name">Dotmatics Limited</span>: <span class="NLM_publisher-loc">The Old Monastery, Windhill, Bishops Stortford,
Herts CM23 2ND UK</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Vortex%2C+2016.08%3B+Dotmatics+Limited%3A+The+Old+Monastery%2C+Windhill%2C+Bishops+Stortford%2C%0AHerts+CM23+2ND+UK%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DVortex%26pub%3DDotmatics%2520Limited%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Guilloux, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuffery, P.</span></span> <span> </span><span class="NLM_article-title">Fpocket: An Open Source Platform for Ligand Pocket Detection</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">168</span> <span class="refDoi"> DOI: 10.1186/1471-2105-10-168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1186%2F1471-2105-10-168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=19486540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BD1MvjsFWltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&author=V.+Le%0AGuillouxauthor=P.+Schmidtkeauthor=P.+Tuffery&title=Fpocket%3A+An+Open+Source+Platform+for+Ligand+Pocket+Detection&doi=10.1186%2F1471-2105-10-168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fpocket: an open source platform for ligand pocket detection</span></div><div class="casAuthors">Le Guilloux Vincent; Schmidtke Peter; Tuffery Pierre</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">168</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Virtual screening methods start to be well established as effective approaches to identify hits, candidates and leads for drug discovery research.  Among those, structure based virtual screening (SBVS) approaches aim at docking collections of small compounds in the target structure to identify potent compounds.  For SBVS, the identification of candidate pockets in protein structures is a key feature, and the recent years have seen increasing interest in developing methods for pocket and cavity detection on protein surfaces.  RESULTS:  Fpocket is an open source pocket detection package based on Voronoi tessellation and alpha spheres built on top of the publicly available package Qhull.  The modular source code is organised around a central library of functions, a basis for three main programs: (i) Fpocket, to perform pocket identification, (ii) Tpocket, to organise pocket detection benchmarking on a set of known protein-ligand complexes, and (iii) Dpocket, to collect pocket descriptor values on a set of proteins.  Fpocket is written in the C programming language, which makes it a platform well suited for the scientific community willing to develop new scoring functions and extract various pocket descriptors on a large scale level.  Fpocket 1.0, relying on a simple scoring function, is able to detect 94% and 92% of the pockets within the best three ranked pockets from the holo and apo proteins respectively, outperforming the standards of the field, while being faster.  CONCLUSION:  Fpocket provides a rapid, open source and stable basis for further developments related to protein pocket detection, efficient pocket descriptor extraction, or drugablity prediction purposes.  Fpocket is freely available under the GNU GPL license at http://fpocket.sourceforge.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRTfW9d1-8T8XWEEzDrymxmfW6udTcc2eaH6pR5mtXnYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvjsFWltw%253D%253D&md5=7d16f53ed64eac9cdeb33ea40b61adfd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-168%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGuilloux%26aufirst%3DV.%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DTuffery%26aufirst%3DP.%26atitle%3DFpocket%253A%2520An%2520Open%2520Source%2520Platform%2520for%2520Ligand%2520Pocket%2520Detection%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26doi%3D10.1186%2F1471-2105-10-168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+Molecular+Force+Field.+I.+Basis%2C+Form%2C+Scope%2C+Parameterization%2C+and+Performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Å root mean square [rms]), bond angles (1.2° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0liacebnG0VLbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520Molecular%2520Force%2520Field.%2520I.%2520Basis%252C%2520Form%252C%2520Scope%252C%2520Parameterization%252C%2520and%2520Performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meireles, L.</span></span> <span> </span><span class="NLM_article-title">Prediction of Protein Pairs Sharing Common Active Ligands Using Protein Sequence, Structure, and Ligand Similarity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1745</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSqt7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1734-1745&author=Y.+C.+Chenauthor=R.+Tolbertauthor=A.+M.+Aronovauthor=G.+McGaugheyauthor=W.+P.+Waltersauthor=L.+Meireles&title=Prediction+of+Protein+Pairs+Sharing+Common+Active+Ligands+Using+Protein+Sequence%2C+Structure%2C+and+Ligand+Similarity&doi=10.1021%2Facs.jcim.6b00118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Protein Pairs Sharing Common Active Ligands Using Protein Sequence, Structure, and Ligand Similarity</span></div><div class="casAuthors">Chen, Yu-Chen; Tolbert, Robert; Aronov, Alex M.; McGaughey, Georgia; Walters, W. Patrick; Meireles, Lidio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1734-1745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors benchmarked the ability of comparative computational approaches to correctly discriminate protein pairs sharing a common active ligand (pos. protein pairs) from protein pairs with no common active ligands (neg. protein pairs).  Since the target and the off-targets of a drug share at least a common ligand, i.e. the drug itself, the prediction of pos. protein pairs may help identify off-targets.  The authors evaluated representative protein-centric and ligand centric approaches, including (1) 2D and 3D ligand similarity, (2) several measures of protein sequence similarity in conjunction with different sequence sources (e.g., full protein sequence vs. binding site residues), and( 3) a newly described pocket shape similarity and alignment program called SiteHopper.  While the sequence-based alignment of pocket residues achieved the best overall performance, SiteHopper outperformed sequence-based approaches for unrelated proteins with only 20-30% pocket residue identity.  Analogously, among ligand-centric approaches, path-based fingerprints achieved the best overall performance, but ROCS-based ligand shape similarity outperformed path-based fingerprints for structurally dissimilar ligands (Tanimoto 25%-40%).  A significant drop in recognition performance was obsd. for ligand-centric approaches when PDB ligands were used instead of ChEMBL ligands.  Finally, the authors analyzed the relation between pocket shape and ligand shape in the authors' dataset, and found that similar ligands tend to bind to similar pockets, while similar pockets may accept a range of different shape ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgs65nWQ5rdLVg90H21EOLACvtfcHk0liacebnG0VLbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSqt7zF&md5=0bdc84d1c3596d99d976ce5b3548cb62</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00118%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTolbert%26aufirst%3DR.%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcGaughey%26aufirst%3DG.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DMeireles%26aufirst%3DL.%26atitle%3DPrediction%2520of%2520Protein%2520Pairs%2520Sharing%2520Common%2520Active%2520Ligands%2520Using%2520Protein%2520Sequence%252C%2520Structure%252C%2520and%2520Ligand%2520Similarity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1734%26epage%3D1745%26doi%3D10.1021%2Facs.jcim.6b00118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The%0AUniProt+Consortium&title=UniProt%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrmann, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Experimental Active-Site Mapping by Fragments: Hot Spots Remote from the Catalytic Center of Endothiapepsin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7561</span>– <span class="NLM_lpage">7575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7561-7575&author=N.+Radevaauthor=S.+G.+Krimmerauthor=M.+Stielerauthor=K.+Fuauthor=X.+Wangauthor=F.+R.+Ehrmannauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=J.+Schiebelauthor=A.+Heineauthor=G.+Klebe&title=Experimental+Active-Site+Mapping+by+Fragments%3A+Hot+Spots+Remote+from+the+Catalytic+Center+of+Endothiapepsin&doi=10.1021%2Facs.jmedchem.6b00645"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00645%26sid%3Dliteratum%253Aachs%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DExperimental%2520Active-Site%2520Mapping%2520by%2520Fragments%253A%2520Hot%2520Spots%2520Remote%2520from%2520the%2520Catalytic%2520Center%2520of%2520Endothiapepsin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7561%26epage%3D7575%26doi%3D10.1021%2Facs.jmedchem.6b00645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brass, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herhaus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentgraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">A Small Nonrule of 3 Compatible Fragment Library Provides High Hit Rate of Endothiapepsin Crystal Structures with Various Fragment Chemotypes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7784</span>– <span class="NLM_lpage">7796</span>, <span class="refDoi"> DOI: 10.1021/jm200642w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200642w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FgtVyhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7784-7796&author=H.+K%C3%B6sterauthor=T.+Craanauthor=S.+Brassauthor=C.+Herhausauthor=M.+Zentgrafauthor=L.+Neumannauthor=A.+Heineauthor=G.+Klebe&title=A+Small+Nonrule+of+3+Compatible+Fragment+Library+Provides+High+Hit+Rate+of+Endothiapepsin+Crystal+Structures+with+Various+Fragment+Chemotypes&doi=10.1021%2Fjm200642w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A small nonrule of 3 compatible fragment library provides high hit rate of endothiapepsin crystal structures with various fragment chemotypes</span></div><div class="casAuthors">Koster Helene; Craan Tobias; Brass Sascha; Herhaus Christian; Zentgraf Matthias; Neumann Lars; Heine Andreas; Klebe Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7784-96</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Druglike molecules are defined by Lipinski's rule of 5, to characterize fragment thresholds, they have been reduced from 5 to 3 (Astex's rule of 3).  They are applied to assemble fragment libraries, and providers use them to select fragments for commercial offer.  We question whether these rules are too stringent to compose fragment libraries with candidates exhibiting sufficient room for chemical subsequent growing and merging modifications as appropriate functional groups for chemical transformations are required.  Usually these groups exhibit properties as hydrogen bond donors/acceptors and provide entry points for optimization chemistry.  We therefore designed a fragment library (364 entries) without strictly applying the rule of 3.  For initial screening for endothiapepsin binding, we performed a biochemical cleavage assay of a fluorogenic substrate at 1 mM. "Hits" were defined to inhibit the enzyme by at least 40%.  Fifty-five hits were suggested and subsequently soaked into endothiapepsin crystals.  Eleven crystal structures could be determined covering fragments with diverse binding modes: (i) direct binding to the catalytic dyad aspartates, (ii) water-mediated binding to the aspartates, (iii) no direct interaction with the dyad.  They occupy different specificity pockets.  Only 4 of the 11 fragments are consistent with the rule of 3.  Restriction to this rule would have limited the fragment hits to a strongly reduced variety of chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuxdu_QoLSWZotVjB3IRIyfW6udTcc2eZBFx6BQY8-K7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FgtVyhtQ%253D%253D&md5=bb86da923d5251f50e4aa20bf04c05af</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm200642w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200642w%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6ster%26aufirst%3DH.%26aulast%3DCraan%26aufirst%3DT.%26aulast%3DBrass%26aufirst%3DS.%26aulast%3DHerhaus%26aufirst%3DC.%26aulast%3DZentgraf%26aufirst%3DM.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520Small%2520Nonrule%2520of%25203%2520Compatible%2520Fragment%2520Library%2520Provides%2520High%2520Hit%2520Rate%2520of%2520Endothiapepsin%2520Crystal%2520Structures%2520with%2520Various%2520Fragment%2520Chemotypes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7784%26epage%3D7796%26doi%3D10.1021%2Fjm200642w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Recht, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bounaud, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chie-Leon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nienaber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, F. E.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimization of PDE10A Inhibitors Using Fragment-Based Screening by Nanocalorimetry and X-Ray Crystallography</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1177/1087057113516493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1177%2F1087057113516493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=24375910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWgs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=497-507&author=M.+I.+Rechtauthor=V.+Sridharauthor=J.+Badgerauthor=P.-Y.+Bounaudauthor=C.+Loganauthor=B.+Chie-Leonauthor=V.+Nienaberauthor=F.+E.+Torres&title=Identification+and+Optimization+of+PDE10A+Inhibitors+Using+Fragment-Based+Screening+by+Nanocalorimetry+and+X-Ray+Crystallography&doi=10.1177%2F1087057113516493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and optimization of PDE10A inhibitors using fragment-based screening by nanocalorimetry and X-ray crystallography</span></div><div class="casAuthors">Recht, Michael I.; Sridhar, Vandana; Badger, John; Bounaud, Pierre-Yves; Logan, Cheyenne; Chie-Leon, Barbara; Nienaber, Vicki; Torres, Francisco E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507, 11</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) is a technique in which small, low-complexity chem. fragments of 6 to 15 heavy atoms are screened for binding to or inhibiting activity of the target.  Hits are then linked and/or elaborated into tightly binding ligands, ideally yielding early lead compds. for drug discovery.  Calorimetry provides a label-free method to assay binding and enzymic activity that is unaffected by the spectroscopic properties of the sample.  Conventional microcalorimetry is hampered by requiring large quantities of reagents and long measurement times.  Nanocalorimeters can overcome these limitations of conventional isothermal titrn. calorimetry.  Here we use enthalpy arrays, which are arrays of nanocalorimeters, to perform an enzyme activity-based fragment screen for competitive inhibitors of phosphodiesterase 10A (PDE10A).  Two dozen fragments with KI <2 mM were identified and moved to crystal soaking trials.  All soak expts. yielded high-resoln. diffraction, with two-thirds of the fragments yielding high-resoln. co-crystal structures with PDE10A.  The structural information was used to elaborate fragment hits, yielding leads with KI <1 μM.  This study shows how array calorimetry can be used as a prescreening method for fragment-based lead discovery with enzyme targets and paired successfully with an X-ray crystallog. secondary screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjSq4QZ0ZQdLVg90H21EOLACvtfcHk0ljFuKLujgEIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWgs73E&md5=d63ee1e675ba628c75a0276c975e1fd0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1177%2F1087057113516493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516493%26sid%3Dliteratum%253Aachs%26aulast%3DRecht%26aufirst%3DM.%2BI.%26aulast%3DSridhar%26aufirst%3DV.%26aulast%3DBadger%26aufirst%3DJ.%26aulast%3DBounaud%26aufirst%3DP.-Y.%26aulast%3DLogan%26aufirst%3DC.%26aulast%3DChie-Leon%26aufirst%3DB.%26aulast%3DNienaber%26aufirst%3DV.%26aulast%3DTorres%26aufirst%3DF.%2BE.%26atitle%3DIdentification%2520and%2520Optimization%2520of%2520PDE10A%2520Inhibitors%2520Using%2520Fragment-Based%2520Screening%2520by%2520Nanocalorimetry%2520and%2520X-Ray%2520Crystallography%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D497%26epage%3D507%26doi%3D10.1177%2F1087057113516493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shipe, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharik, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theberge, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uslaner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Series of Pyrimidine-Based Phosphodiesterase 10A (PDE10A) Inhibitors through Fragment Screening, Structure-Based Design, and Parallel Synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7888</span>– <span class="NLM_lpage">7894</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00983</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00983" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWktL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7888-7894&author=W.+D.+Shipeauthor=S.+S.+Sharikauthor=J.+C.+Barrowauthor=G.+B.+McGaugheyauthor=C.+R.+Thebergeauthor=J.+M.+Uslanerauthor=Y.+Yanauthor=J.+J.+Rengerauthor=S.+M.+Smithauthor=P.+J.+Colemanauthor=C.+D.+Cox&title=Discovery+and+Optimization+of+a+Series+of+Pyrimidine-Based+Phosphodiesterase+10A+%28PDE10A%29+Inhibitors+through+Fragment+Screening%2C+Structure-Based+Design%2C+and+Parallel+Synthesis&doi=10.1021%2Facs.jmedchem.5b00983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Series of Pyrimidine-Based Phosphodiesterase 10A (PDE10A) Inhibitors through Fragment Screening, Structure-Based Design, and Parallel Synthesis</span></div><div class="casAuthors">Shipe, William D.; Sharik, Steven S.; Barrow, James C.; McGaughey, Georgia B.; Theberge, Cory R.; Uslaner, Jason M.; Yan, Youwei; Renger, John J.; Smith, Sean M.; Coleman, Paul J.; Cox, Christopher D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7888-7894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of a fragment library for PDE10A inhibitors identified a low mol. wt. pyrimidine hit with PDE10A Ki of 8700 nM and LE of 0.59.  Initial optimization by catalog followed by iterative parallel synthesis guided by X-ray cocrystal structures resulted in rapid potency improvements with minimal loss of ligand efficiency.  Compd. 15h, with PDE10A Ki of 8.2 pM, LE of 0.49, and >5000-fold selectivity over other PDEs, fully attenuates MK-801-induced hyperlocomotor activity after i.p. dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTIq75ec_7MrVg90H21EOLACvtfcHk0ljFuKLujgEIZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWktL3F&md5=7deabe7e71abef929fee9a81d090e1b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00983%26sid%3Dliteratum%253Aachs%26aulast%3DShipe%26aufirst%3DW.%2BD.%26aulast%3DSharik%26aufirst%3DS.%2BS.%26aulast%3DBarrow%26aufirst%3DJ.%2BC.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DTheberge%26aufirst%3DC.%2BR.%26aulast%3DUslaner%26aufirst%3DJ.%2BM.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DSmith%26aufirst%3DS.%2BM.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DCox%26aufirst%3DC.%2BD.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Series%2520of%2520Pyrimidine-Based%2520Phosphodiesterase%252010A%2520%2528PDE10A%2529%2520Inhibitors%2520through%2520Fragment%2520Screening%252C%2520Structure-Based%2520Design%252C%2520and%2520Parallel%2520Synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7888%26epage%3D7894%26doi%3D10.1021%2Facs.jmedchem.5b00983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodhead, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5955</span>, <span class="refDoi"> DOI: 10.1021/jm100059d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5942-5955&author=C.+W.+Murrayauthor=M.+G.+Carrauthor=O.+Callaghanauthor=G.+Chessariauthor=M.+Congreveauthor=S.+Cowanauthor=J.+E.+Coyleauthor=R.+Downhamauthor=E.+Figueroaauthor=M.+Fredericksonauthor=B.+Grahamauthor=R.+McMenaminauthor=M.+A.+O%E2%80%99Brienauthor=S.+Patelauthor=T.+R.+Phillipsauthor=G.+Williamsauthor=A.+J.+Woodheadauthor=A.+J.-A.+Woolford&title=Fragment-Based+Drug+Discovery+Applied+to+Hsp90.+Discovery+of+Two+Lead+Series+with+High+Ligand+Efficiency&doi=10.1021%2Fjm100059d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span></div><div class="casAuthors">Murray, Christopher W.; Carr, Maria G.; Callaghan, Owen; Chessari, Gianni; Congreve, Miles; Cowan, Suzanna; Coyle, Joseph E.; Downham, Robert; Figueroa, Eva; Frederickson, Martyn; Graham, Brent; McMenamin, Rachel; O'Brien, M. Alistair; Patel, Sahil; Phillips, Theresa R.; Williams, Glyn; Woodhead, Andrew J.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5942-5955</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of the chaperone Hsp90 are potentially useful as chemotherapeutic agents in cancer.  This paper describes an application of fragment screening to Hsp90 using a combination of NMR and high throughput x-ray crystallog.  The screening identified an aminopyrimidine with affinity in the high micromolar range and subsequent structure-based design allowed its optimization into a low nanomolar series with good ligand efficiency.  A phenolic chemotype was also identified in fragment screening and was found to bind with affinity close to 1 mM.  This fragment was optimized using structure based design into a resorcinol lead which has subnanomolar affinity for Hsp90, excellent cell potency, and good ligand efficiency.  This fragment to lead campaign improved affinity for Hsp90 by over 1,000,000 fold with the addn. of only 6 heavy atoms.  The companion paper (DOI: 10.1021/jm100060b) describes how the resorcinol lead was optimized into a compd. that is now in clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISC3cw2labbVg90H21EOLACvtfcHk0lhsxkfZXSKonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D&md5=d1ad900aa5a45ba6ba3a5e9ccdcd96b3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm100059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100059d%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DFigueroa%26aufirst%3DE.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPhillips%26aufirst%3DT.%2BR.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520Applied%2520to%2520Hsp90.%2520Discovery%2520of%2520Two%2520Lead%2520Series%2520with%2520High%2520Ligand%2520Efficiency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5942%26epage%3D5955%26doi%3D10.1021%2Fjm100059d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.
G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scriven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, P.</span></span> <span> </span><span class="NLM_article-title">Targeting Conserved Water Molecules: Design of 4-Aryl-5-cyanopyrrolo[2,3-D]pyrimidine Hsp90 Inhibitors Using Fragment-Based Screening and Structure-Based Optimization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">6770</span>– <span class="NLM_lpage">6789</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.08.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.bmc.2012.08.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23018093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVSltrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=6770-6789&author=N.%0AG.+M.+Daviesauthor=H.+Browneauthor=B.+Davisauthor=M.+J.+Drysdaleauthor=N.+Foloppeauthor=S.+Geoffreyauthor=B.+Gibbonsauthor=T.+Hartauthor=R.+Hubbardauthor=M.+R.+Jensenauthor=H.+Mansellauthor=A.+Masseyauthor=N.+Matassovaauthor=J.+D.+Mooreauthor=J.+Murrayauthor=R.+Prattauthor=S.+Rayauthor=A.+Robertsonauthor=S.+D.+Roughleyauthor=J.+Schoepferauthor=K.+Scrivenauthor=H.+Simmoniteauthor=S.+Stokesauthor=A.+Surgenorauthor=P.+Webbauthor=M.+Woodauthor=L.+Wrightauthor=P.+Brough&title=Targeting+Conserved+Water+Molecules%3A+Design+of+4-Aryl-5-cyanopyrrolo%5B2%2C3-D%5Dpyrimidine+Hsp90+Inhibitors+Using+Fragment-Based+Screening+and+Structure-Based+Optimization&doi=10.1016%2Fj.bmc.2012.08.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting conserved water molecules: Design of 4-aryl-5-cyanopyrrolo[2,3-d]pyrimidine Hsp90 inhibitors using fragment-based screening and structure-based optimization</span></div><div class="casAuthors">Davies, Nicholas G. M.; Browne, Helen; Davis, Ben; Drysdale, Martin J.; Foloppe, Nicolas; Geoffrey, Stephanie; Gibbons, Ben; Hart, Terance; Hubbard, Roderick; Jensen, Michael Rugaard; Mansell, Howard; Massey, Andrew; Matassova, Natalia; Moore, Jonathan D.; Murray, James; Pratt, Robert; Ray, Stuart; Robertson, Alan; Roughley, Stephen D.; Schoepfer, Joseph; Scriven, Kirsten; Simmonite, Heather; Stokes, Stephen; Surgenor, Allan; Webb, Paul; Wood, Mike; Wright, Lisa; Brough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6770-6789</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibitors of the Hsp90 mol. chaperone are showing promise as anti-cancer agents.  Here the authors describe a series of 4-aryl-5-cyanopyrrolo[2,3-d]pyrimidine ATP competitive Hsp90 inhibitors that were identified following structure-driven optimization of purine hits revealed by NMR based screening of a proprietary fragment library.  Ligand-Hsp90 x-ray structures combined with mol. modeling led to the rational displacement of a conserved water mol. leading to enhanced affinity for Hsp90 as measured by fluorescence polarization, isothermal titrn. calorimetry and surface plasmon resonance assays.  This displacement was achieved with a nitrile group, presenting an example of efficient gain in binding affinity with minimal increase in mol. wt.  Some compds. in this chem. series inhibit the proliferation of human cancer cell lines in vitro and cause depletion of oncogenic Hsp90 client proteins and concomitant elevation of the co-chaperone Hsp70.  In addn., one compd. was demonstrated to be orally bioavailable in the mouse.  This work demonstrates the power of structure-based design for the rapid evolution of potent Hsp90 inhibitors and the importance of considering conserved water mols. in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAwVhZUzKD7Vg90H21EOLACvtfcHk0lhsxkfZXSKonQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVSltrvN&md5=69c85da95d07bd488f07eaf503c978f4</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.08.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.08.050%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DN.%2BG.%2BM.%26aulast%3DBrowne%26aufirst%3DH.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DGeoffrey%26aufirst%3DS.%26aulast%3DGibbons%26aufirst%3DB.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DMansell%26aufirst%3DH.%26aulast%3DMassey%26aufirst%3DA.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DPratt%26aufirst%3DR.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DRobertson%26aufirst%3DA.%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DScriven%26aufirst%3DK.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DSurgenor%26aufirst%3DA.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DL.%26aulast%3DBrough%26aufirst%3DP.%26atitle%3DTargeting%2520Conserved%2520Water%2520Molecules%253A%2520Design%2520of%25204-Aryl-5-cyanopyrrolo%255B2%252C3-D%255Dpyrimidine%2520Hsp90%2520Inhibitors%2520Using%2520Fragment-Based%2520Screening%2520and%2520Structure-Based%2520Optimization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D6770%26epage%3D6789%26doi%3D10.1016%2Fj.bmc.2012.08.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saubern, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">KNIME Workflow to Assess PAINS Filters in SMARTS Format. Comparison of RDKit and Indigo Cheminformatics Libraries</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1002/minf.201100076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fminf.201100076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27468104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12itL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=847-850&author=S.+Saubernauthor=R.+Guhaauthor=J.+B.+Baell&title=KNIME+Workflow+to+Assess+PAINS+Filters+in+SMARTS+Format.+Comparison+of+RDKit+and+Indigo+Cheminformatics+Libraries&doi=10.1002%2Fminf.201100076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">KNIME Workflow to Assess PAINS Filters in SMARTS Format. Comparison of RDKit and Indigo Cheminformatics Libraries</span></div><div class="casAuthors">Saubern, Simon; Guha, Rajarshi; Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">847-850</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2_WZKnI-T-LVg90H21EOLACvtfcHk0ljvYIBQ5yyZgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12itL%252FF&md5=314effbb1e4205e273858c6c70e7591e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201100076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201100076%26sid%3Dliteratum%253Aachs%26aulast%3DSaubern%26aufirst%3DS.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DKNIME%2520Workflow%2520to%2520Assess%2520PAINS%2520Filters%2520in%2520SMARTS%2520Format.%2520Comparison%2520of%2520RDKit%2520and%2520Indigo%2520Cheminformatics%2520Libraries%26jtitle%3DMol.%2520Inf.%26date%3D2011%26volume%3D30%26spage%3D847%26epage%3D850%26doi%3D10.1002%2Fminf.201100076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span> </span><span class="NLM_article-title">Enamine: Golden Fragment Library (2016)</span>. <a href="https://enamine.net/download/FL/Enamine_Golden_Fragment_Library.pdf" class="extLink">https://enamine.net/download/FL/Enamine_Golden_Fragment_Library.pdf</a> retrieved Dec 23, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Enamine%3A+Golden+Fragment+Library+%282016%29.+https%3A%2F%2Fenamine.net%2Fdownload%2FFL%2FEnamine_Golden_Fragment_Library.pdf+retrieved+Dec+23%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DEnamine%253A%2520Golden%2520Fragment%2520Library%2520%25282016%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The Properties of Known Drugs. 1. Molecular Frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+Properties+of+Known+Drugs.+1.+Molecular+Frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0ljvYIBQ5yyZgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Properties%2520of%2520Known%2520Drugs.%25201.%2520Molecular%2520Frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2138</span>– <span class="NLM_lpage">2153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2138-2153&author=X.+Zhang&title=Intermolecular+Sulfur%C2%B7%C2%B7%C2%B7Oxygen+Interactions%3A+Theoretical+and+Statistical+Investigations&doi=10.1021%2Facs.jcim.5b00177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span></div><div class="casAuthors">Zhang, Xuejin; Gong, Zhen; Li, Jian; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2138-2153</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intermol. S···O interactions are very common and are important in biol. systems, but until recently, the presence of these contacts in protein-ligand systems largely depended on serendipitous discovery instead of rational design.  Here we provide insight into the phenomenon of intermol. S···O contacts by focusing on three sulfur-contg. arom. rings.  Quantum mechanics is employed to characterize the strength and directionality of the S···O interactions and to det. their energy dependence on their geometric parameters.  Protein Data Bank mining is performed to systematically det. the occurrence and geometry of intermol. S···O interactions, and several representative examples are discussed.  Three typical cases are investigated using a combined quantum mechanics/mol. mechanics approach to demonstrate the potential of these interactions in improving binding affinities and physiochem. properties.  Overall, our work elucidates the structures and energy features of intermol. S···O interactions and addresses their use in mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2J_2Nq9PDQLVg90H21EOLACvtfcHk0ljvYIBQ5yyZgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI&md5=7192e3b57e66fc6eb9dd53165427d449</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00177%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DIntermolecular%2520Sulfur%25C2%25B7%25C2%25B7%25C2%25B7Oxygen%2520Interactions%253A%2520Theoretical%2520and%2520Statistical%2520Investigations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2138%26epage%3D2153%26doi%3D10.1021%2Facs.jcim.5b00177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metrangolo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilati, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priimagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terraneo, G.</span></span> <span> </span><span class="NLM_article-title">The Halogen Bond</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2601</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+Halogen+Bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0ljvYIBQ5yyZgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520Halogen%2520Bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">41576</span>– <span class="NLM_lpage">41582</span>, <span class="refDoi"> DOI: 10.1074/jbc.R112.418574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1074%2Fjbc.R112.418574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23048026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=41576-41582&author=S.+Horowitzauthor=R.+C.+Trievel&title=Carbon-Oxygen+Hydrogen+Bonding+in+Biological+Structure+and+Function&doi=10.1074%2Fjbc.R112.418574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span></div><div class="casAuthors">Horowitz, Scott; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41576-41582</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Carbon-oxygen (CH···O) hydrogen bonding represents an unusual category of mol. interactions first documented in biol. structures over 4 decades ago.  Although CH···O hydrogen bonding has remained generally underappreciated in the biochem. literature, studies over the last 15 years have begun to yield direct evidence of these interactions in biol. systems.  In this minireview, we provide a historical context of biol. CH···O hydrogen bonding and summarize some major advancements from exptl. studies over the past several years that have elucidated the importance, prevalence, and functions of these interactions.  In particular, we examine the impact of CH···O bonds on protein and nucleic acid structure, mol. recognition, and enzyme catalysis and conclude by exploring overarching themes and unresolved questions regarding unconventional interactions in biomol. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHj0osWPCzV7Vg90H21EOLACvtfcHk0lg7iyhusve77w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP&md5=e96e7e30d5f5e03b0733fea323ecd6a2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.418574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.418574%26sid%3Dliteratum%253Aachs%26aulast%3DHorowitz%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCarbon-Oxygen%2520Hydrogen%2520Bonding%2520in%2520Biological%2520Structure%2520and%2520Function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D41576%26epage%3D41582%26doi%3D10.1074%2Fjbc.R112.418574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saini, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Detection of Secondary Binding Sites in Proteins Using Fragment Screening</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">15910</span>– <span class="NLM_lpage">15915</span>, <span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+Secondary+Binding+Sites+in+Proteins+Using+Fragment+Screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lg7iyhusve77w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520Secondary%2520Binding%2520Sites%2520in%2520Proteins%2520Using%2520Fragment%2520Screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from Hot Spot Analysis for Fragment-Based Drug Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+Hot+Spot+Analysis+for+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lg7iyhusve77w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520Hot%2520Spot%2520Analysis%2520for%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Identifying Interactions That Determine Fragment Binding at Protein Hotspots</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4314</span>– <span class="NLM_lpage">4325</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4314-4325&author=C.+J.+Radouxauthor=T.+S.+G.+Olssonauthor=W.+R.+Pittauthor=C.+R.+Groomauthor=T.+L.+Blundell&title=Identifying+Interactions+That+Determine+Fragment+Binding+at+Protein+Hotspots&doi=10.1021%2Facs.jmedchem.5b01980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Interactions that Determine Fragment Binding at Protein Hotspots</span></div><div class="casAuthors">Radoux, Chris J.; Olsson, Tjelvar S. G.; Pitt, Will R.; Groom, Colin R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4314-4325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Locating a ligand-binding site is an important first step in structure-guided drug discovery, but current methods do little to suggest which interactions within a pocket are the most important for binding.  Here the authors illustrate a method that samples at. hotspots with simple mol. probes to produce fragment hotspot maps.  These maps specifically highlight fragment-binding sites and their corresponding pharmacophores.  For ligand-bound structures, they provide an intuitive visual guide within the binding site, directing medicinal chemists where to grow the mol. and alerting them to suboptimal interactions within the original hit.  The fragment hotspot map calcn. is validated using exptl. binding positions of 21 fragments and subsequent lead mols.  The ligands are found in high scoring areas of the fragment hotspot maps, with fragment atoms having a median percentage rank of 97%.  Protein kinase B and pantothenate synthetase are examd. in detail.  In each case, the fragment hotspot maps are able to rationalize a Free-Wilson anal. of SAR data from a fragment-based drug design project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8BzZp1bkHarVg90H21EOLACvtfcHk0livFd58W59iSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D&md5=fa25eabfe291f9519f5701bcbc6c52c3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01980%26sid%3Dliteratum%253Aachs%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DIdentifying%2520Interactions%2520That%2520Determine%2520Fragment%2520Binding%2520at%2520Protein%2520Hotspots%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4314%26epage%3D4325%26doi%3D10.1021%2Facs.jmedchem.5b01980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devine, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcair, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debono, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrashekaran, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohanty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Promiscuous 2-Aminothiazoles (PrATs): A Frequent Hitting Scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1205</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1021/jm501402x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501402x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1205-1214&author=S.+M.+Devineauthor=M.+D.+Mulcairauthor=C.+O.+Debonoauthor=E.+W.+W.+Leungauthor=J.+W.+M.+Nissinkauthor=S.+S.+Limauthor=I.+R.+Chandrashekaranauthor=M.+Vaziraniauthor=B.+Mohantyauthor=J.+S.+Simpsonauthor=J.+B.+Baellauthor=P.+J.+Scammellsauthor=R.+S.+Nortonauthor=M.+J.+Scanlon&title=Promiscuous+2-Aminothiazoles+%28PrATs%29%3A+A+Frequent+Hitting+Scaffold&doi=10.1021%2Fjm501402x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous 2-Aminothiazoles (PrATs): A Frequent Hitting Scaffold</span></div><div class="casAuthors">Devine, Shane M.; Mulcair, Mark D.; Debono, Cael O.; Leung, Eleanor W. W.; Nissink, J. Willem M.; Lim, San Sui; Chandrashekaran, Indu R.; Vazirani, Mansha; Mohanty, Biswaranjan; Simpson, Jamie S.; Baell, Jonathan B.; Scammells, Peter J.; Norton, Raymond S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1205-1214</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have identified a class of mols., known as 2-aminothiazoles (2-ATs), as frequent-hitting fragments in biophys. binding assays.  This was exemplified by 4-phenylthiazol-2-amine being identified as a hit in 14/14 screens against a diverse range of protein targets, suggesting that this scaffold is a poor starting point for fragment-based drug discovery.  This prompted us to analyze this scaffold in the context of an academic fragment library used for fragment-based drug discovery (FBDD) and two larger compd. libraries used for high-throughput screening (HTS).  This anal. revealed that such "promiscuous 2-aminothiazoles" (PrATs) behaved as frequent hitters under both FBDD and HTS settings, although the problem was more pronounced in the fragment-based studies.  As 2-ATs are present in known drugs, they cannot necessarily be deemed undesirable, but the combination of their promiscuity and difficulties assocd. with optimizing them into a lead compd. makes them, in our opinion, poor scaffolds for fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryZfJtzgcrZ7Vg90H21EOLACvtfcHk0livFd58W59iSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsw%253D%253D&md5=8d989c9ee48293efe32a331049b59f42</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm501402x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501402x%26sid%3Dliteratum%253Aachs%26aulast%3DDevine%26aufirst%3DS.%2BM.%26aulast%3DMulcair%26aufirst%3DM.%2BD.%26aulast%3DDebono%26aufirst%3DC.%2BO.%26aulast%3DLeung%26aufirst%3DE.%2BW.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DLim%26aufirst%3DS.%2BS.%26aulast%3DChandrashekaran%26aufirst%3DI.%2BR.%26aulast%3DVazirani%26aufirst%3DM.%26aulast%3DMohanty%26aufirst%3DB.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DPromiscuous%25202-Aminothiazoles%2520%2528PrATs%2529%253A%2520A%2520Frequent%2520Hitting%2520Scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1205%26epage%3D1214%26doi%3D10.1021%2Fjm501402x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1002/anie.201506783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fanie.201506783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=488-492&author=C.+W.+Murrayauthor=D.+C.+Rees&title=Opportunity+Knocks%3A+Organic+Chemistry+for+Fragment-Based+Drug+Discovery+%28FBDD%29&doi=10.1002%2Fanie.201506783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">488-492</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj8GEY4dDXxrVg90H21EOLACvtfcHk0livFd58W59iSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O&md5=97a92911033a2304da101888da0c3642</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506783%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DOpportunity%2520Knocks%253A%2520Organic%2520Chemistry%2520for%2520Fragment-Based%2520Drug%2520Discovery%2520%2528FBDD%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D488%26epage%3D492%26doi%3D10.1002%2Fanie.201506783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Conformational Preferences Derived from Crystal Structure Data. A Medicinal Chemistry Focused Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/ci7002494</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci7002494" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1-24&author=K.+A.+Brameldauthor=B.+Kuhnauthor=D.+C.+Reuterauthor=M.+Stahl&title=Small+Molecule+Conformational+Preferences+Derived+from+Crystal+Structure+Data.+A+Medicinal+Chemistry+Focused+Analysis&doi=10.1021%2Fci7002494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule conformational preferences derived from crystal structure data. a medicinal chemistry focused analysis</span></div><div class="casAuthors">Brameld, Ken A.; Kuhn, Bernd; Reuter, Deborah C.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Based on torsion angle distributions of frequently occurring substructures, conformation preferences of druglike mols. are presented, accompanied by a review of the relevant literature.  First, the relevance of the Cambridge Structural Database (CSD) for drug design is demonstrated by comparing substructures present in compds. entering clin. trials with those found in the CSD and protein-bound ligands in the Protein Data Bank (PDB).  Next, we briefly highlight preferred conformations of elementary acyclic systems, followed by a discussion of sulfonamide conformations.  Due to their central role in medicinal chem., we discuss properties of aryl ring substituents in depth, including biaryl systems and systems of two aryl rings connected by two acyclic bonds.  For a subset of torsion motifs, we also compare torsion angle histograms derived from CSD structures with those derived from ligands in the PDB.  Furthermore, selected properties of some six- and seven-membered ring systems are discussed.  The article closes with a section on attractive sulfur-oxygen contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrdvMrRvKirVg90H21EOLACvtfcHk0lgokQ2KuTTZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D&md5=289969338dcb94a145ce736d19d05cf4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci7002494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci7002494%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DReuter%26aufirst%3DD.%2BC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DSmall%2520Molecule%2520Conformational%2520Preferences%2520Derived%2520from%2520Crystal%2520Structure%2520Data.%2520A%2520Medicinal%2520Chemistry%2520Focused%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1%26epage%3D24%26doi%3D10.1021%2Fci7002494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Thermodynamics of Fragment Binding</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1021/ci200608b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200608b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslWjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1039-1045&author=G.+G.+Ferenczyauthor=G.+M.+Keser%C5%B1&title=Thermodynamics+of+Fragment+Binding&doi=10.1021%2Fci200608b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamics of Fragment Binding</span></div><div class="casAuthors">Ferenczy, Gyorgy G.; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ligand binding pockets of proteins have preponderance of hydrophobic amino acids and are typically within the apolar interior of the protein; nevertheless, they are able to bind low complexity, polar, water-sol. fragments.  In order to understand this phenomenon, we analyzed high resoln. X-ray data of protein-ligand complexes from the Protein Data Bank and found that fragments bind to proteins with two near optimal geometry H-bonds on av.  The linear extent of the fragment binding site was found not to be larger than 10 Å, and the H-bonding region was found to be restricted to about 5 Å on av.  The no. of conserved H-bonds in proteins cocrystd. with multiple different fragments is also near to 2.  These fragment binding sites that are able to form limited no. of strong H-bonds in a hydrophobic environment are identified as hot spots.  An est. of the free energy gain of H-bond formation vs. apolar desolvation supports the idea that fragment-sized compds. need H-bonds to achieve detectable binding.  This suggests that fragment binding is mostly enthalpic, in line with their obsd. binding thermodn. documented in isothermal titrn. calorimetry (ITC) data sets and gives a thermodn. rationale for fragment-based approaches.  The binding of larger compds. tends to more rely on apolar desolvation with a corresponding increase of the entropy content of their binding free energy.  These findings explain the reported size-dependence of maximal available affinity and ligand efficiency both behaving differently in the small mol. region featured by strong H-bond formation and in the larger mol. region featured by apolar desolvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvPwUrHRwL1LVg90H21EOLACvtfcHk0lgokQ2KuTTZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslWjurw%253D&md5=d1dde67220cd13f4f222ae68a8870892</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci200608b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200608b%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DThermodynamics%2520of%2520Fragment%2520Binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1039%26epage%3D1045%26doi%3D10.1021%2Fci200608b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">On the Enthalpic Preference of Fragment Binding</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1039/C5MD00542F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1039%2FC5MD00542F" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=332-337&author=G.%0AG.+Ferenczyauthor=G.+M.+Keser%C5%B1&title=On+the+Enthalpic+Preference+of+Fragment+Binding&doi=10.1039%2FC5MD00542F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FC5MD00542F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00542F%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DOn%2520the%2520Enthalpic%2520Preference%2520of%2520Fragment%2520Binding%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D332%26epage%3D337%26doi%3D10.1039%2FC5MD00542F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rühmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Fragment Binding Can Be Either More Enthalpy-Driven or Entropy-Driven: Crystal Structures and Residual Hydration Patterns Suggest Why</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6960</span>– <span class="NLM_lpage">6971</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00812</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00812" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlehu7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6960-6971&author=E.+R%C3%BChmannauthor=M.+Betzauthor=A.+Heineauthor=G.+Klebe&title=Fragment+Binding+Can+Be+Either+More+Enthalpy-Driven+or+Entropy-Driven%3A+Crystal+Structures+and+Residual+Hydration+Patterns+Suggest+Why&doi=10.1021%2Facs.jmedchem.5b00812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Binding Can Be Either More Enthalpy-Driven or Entropy-Driven: Crystal Structures and Residual Hydration Patterns Suggest Why</span></div><div class="casAuthors">Ruehmann, Eggert; Betz, Michael; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6960-6971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In lead optimization, small, enthalpically advantaged fragments have been suggested to be superior, as an entropic component will be added inevitably during late-stage optimization.  Detn. of thermodn. signatures of weak-binding fragments is essential to support the decision-making process, to decide which fragment to take to further optimization.  High-resoln. crystal structures of six fragments binding to the S1 pocket of thrombin were detd. and analyzed with respect to their thermodn. profile.  The two most potent fragments exhibiting an amidine-type scaffold are not the most enthalpic binders; instead a chloro-thiophene fragment binds more enthalpically.  Two chem. very similar chloro-arom. fragments differ strongly in their potency (430 μM vs 10 mM); their binding modes are related, but the surrounding residual water network differs.  The more potent one recruits a water mol. and involves Glu192 in binding, thus succeeding in firmly capping the S1 pocket.  Fragments exhibiting a rather perfect solvation pattern in their binding mode also experience the highest potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjIOp5Fbyf3bVg90H21EOLACvtfcHk0lgokQ2KuTTZ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlehu7%252FF&md5=97c80bfedfb189af3be987f0bcc75e24</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00812%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25BChmann%26aufirst%3DE.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFragment%2520Binding%2520Can%2520Be%2520Either%2520More%2520Enthalpy-Driven%2520or%2520Entropy-Driven%253A%2520Crystal%2520Structures%2520and%2520Residual%2520Hydration%2520Patterns%2520Suggest%2520Why%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6960%26epage%3D6971%26doi%3D10.1021%2Facs.jmedchem.5b00812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">The Thermodynamics of Protein–Ligand Interaction and Solvation: Insights for Ligand Design</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">1002</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.jmb.2008.09.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=1002-1017&author=T.+S.+G.+Olssonauthor=M.+A.+Williamsauthor=W.+R.+Pittauthor=J.+E.+Ladbury&title=The+Thermodynamics+of+Protein%E2%80%93Ligand+Interaction+and+Solvation%3A+Insights+for+Ligand+Design&doi=10.1016%2Fj.jmb.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DWilliams%26aufirst%3DM.%2BA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520Thermodynamics%2520of%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520and%2520Solvation%253A%2520Insights%2520for%2520Ligand%2520Design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26spage%3D1002%26epage%3D1017%26doi%3D10.1016%2Fj.jmb.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sándor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Virtual Fragment Docking by Glide: A Validation Study on 190 Protein–Fragment Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1165</span>– <span class="NLM_lpage">1172</span>, <span class="refDoi"> DOI: 10.1021/ci1000407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1165-1172&author=M.+S%C3%A1ndorauthor=R.+Kissauthor=G.+M.+Keser%C5%B1&title=Virtual+Fragment+Docking+by+Glide%3A+A+Validation+Study+on+190+Protein%E2%80%93Fragment+Complexes&doi=10.1021%2Fci1000407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Fragment Docking by Glide: a Validation Study on 190 Protein-Fragment Complexes</span></div><div class="casAuthors">Sandor, Mark; Kiss, Robert; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1165-1172</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The docking accuracy of Glide was evaluated using 16 different docking protocols on 190 protein-fragment complexes representing 78 targets.  Std. precision docking (Glide SP) based protocols showed the best performance.  The av. root-mean-square deviation (RMSD) between the docked and cocrystd. poses achieved by Glide SP with pre- and postprocessing was 1.17 Å, and an acceptable binding mode with RMSD. < 2 Å could be found in 80% of the cases.  Comparison of the docking results produced by different protocols suggests that the sampling efficacy of Glide is adequate for fragment docking.  The docking accuracy seems to be limited by the performance of scoring schemes, which is supported by the weak correlation between exptl. binding affinities and GlideScores.  Cross-docking expts. performed on 8 targets represented by 63 complexes revealed that Glide SP gave similar results to that of the computationally more intensive Glide XP.  The av. RMSD achieved by Glide SP with pre- and postprocessing was 2.06 Å, and an acceptable binding mode with RMSD < 2 Å could be found in 63% of the cases.  These cross-docking results were improved significantly selecting the optimal X-ray structure for each target (av. RMSD = 1.3 Å, success rate = 77%), indicating the importance of enrichment studies and the use of multiple X-ray structures in virtual fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKfjH9BbnrLVg90H21EOLACvtfcHk0lh8r5xzbnVIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D&md5=35d7acb7374360828beb668e2f207bab</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci1000407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000407%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1ndor%26aufirst%3DM.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DVirtual%2520Fragment%2520Docking%2520by%2520Glide%253A%2520A%2520Validation%2520Study%2520on%2520190%2520Protein%25E2%2580%2593Fragment%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1165%26epage%3D1172%26doi%3D10.1021%2Fci1000407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giangreco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korb, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">Docking Performance of Fragments and Druglike Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5422</span>– <span class="NLM_lpage">5431</span>, <span class="refDoi"> DOI: 10.1021/jm200558u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200558u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5422-5431&author=M.%0AL.+Verdonkauthor=I.+Giangrecoauthor=R.+J.+Hallauthor=O.+Korbauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Docking+Performance+of+Fragments+and+Druglike+Compounds&doi=10.1021%2Fjm200558u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Performance of Fragments and Druglike Compounds</span></div><div class="casAuthors">Verdonk, Marcel L.; Giangreco, Ilenia; Hall, Richard J.; Korb, Oliver; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5431</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper addresses two questions of key interest to researchers working with protein-ligand docking methods: (i) Why is there such a large variation in docking performance between different test sets reported in the literature (ii) Are fragments more difficult to dock than druglike compds..  To answer these, the authors construct a test set of inhouse x-ray structures of protein-ligand complexes from drug discovery projects, half of which contain fragment ligands, the other half drug-like ligands.  The authors find that a key factor affecting docking performance is ligand efficiency (LE).  High LE compds. are significantly easier to dock than low LE compds., which the authors believe could explain the differences obsd. between test sets reported in the literature.  There is no significant difference in docking performance between fragments and drug-like compds., but the reasons why dockings fail appear to be different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFTOFvmiFxrVg90H21EOLACvtfcHk0lh8r5xzbnVIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D&md5=8d7a5519d6e7fa2ab403ea76b8b9a586</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm200558u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200558u%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDocking%2520Performance%2520of%2520Fragments%2520and%2520Druglike%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5422%26epage%3D5431%26doi%3D10.1021%2Fjm200558u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Applying Thermodynamic Profiling in Lead Finding and Optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1038/nrd4486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd4486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25614222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=95-110&author=G.+Klebe&title=Applying+Thermodynamic+Profiling+in+Lead+Finding+and+Optimization&doi=10.1038%2Fnrd4486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Applying thermodynamic profiling in lead finding and optimization</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-110</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery involves the optimization of various physicochem. properties of a ligand, particularly its binding affinity for its target receptor (or receptors).  In recent years, there has been growing interest in using thermodn. profiling of ligand-receptor interactions in order to select and optimize those ligands that might be most likely to become drug candidates with desirable physicochem. properties.  The thermodn. of binding is influenced by multiple factors, including hydrogen bonding and hydrophobic interactions, desolvation, residual mobility, dynamics and the local water structure.  This article discusses key issues in understanding the effects of these factors and applying this knowledge in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtHj7SsL6rrVg90H21EOLACvtfcHk0lh8r5xzbnVIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D&md5=c2e12760623de81335ce252cb804c602</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrd4486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4486%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApplying%2520Thermodynamic%2520Profiling%2520in%2520Lead%2520Finding%2520and%2520Optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D95%26epage%3D110%26doi%3D10.1038%2Fnrd4486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishio, M.</span></span> <span> </span><span class="NLM_article-title">The CH/π hydrogen bond in chemistry. Conformation, supramolecules, optical resolution and interactions involving carbohydrates</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">13873</span>– <span class="NLM_lpage">13900</span>, <span class="refDoi"> DOI: 10.1039/c1cp20404a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1039%2Fc1cp20404a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21611676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlCgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=13873-13900&author=M.+Nishio&title=The+CH%2F%CF%80+hydrogen+bond+in+chemistry.+Conformation%2C+supramolecules%2C+optical+resolution+and+interactions+involving+carbohydrates&doi=10.1039%2Fc1cp20404a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The CH/π hydrogen bond in chemistry. Conformation, supramolecules, optical resolution and interactions involving carbohydrates</span></div><div class="casAuthors">Nishio, Motohiro</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">13873-13900</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The CH/π hydrogen bond is an attractive mol. force occurring between a soft acid and a soft base.  Contribution from the dispersion energy is important in typical cases where aliph. or arom. CH groups are involved.  Coulombic energy is of minor importance as compared to the other weak hydrogen bonds.  The hydrogen bond nature of this force, however, has been confirmed by AIM analyses.  The dual characteristic of the CH/π hydrogen bond is the basis for ubiquitous existence of this force in various fields of chem.  A salient feature is that the CH/π hydrogen bond works cooperatively.  Another significant point is that it works in nonpolar as well as polar, protic solvents such as water.  The interaction energy depends on the nature of the mol. fragments, CH as well as π-groups: the stronger the proton donating ability of the CH group, the larger the stabilizing effect.  This Perspective focuses on the consequence of this mol. force in the conformation of org. compds. and supramol. chem.  Implication of the CH/π hydrogen bond extends to the specificity of mol. recognition or selectivity in org. reactions, polymer science, surface phenomena and interactions involving proteins.  Many problems, unsettled to date, will become clearer in the light of the CH/π paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoQqvxrPj-PbVg90H21EOLACvtfcHk0lh8r5xzbnVIQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlCgsbs%253D&md5=18eff250217a75244a10c8d2ef69710e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc1cp20404a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cp20404a%26sid%3Dliteratum%253Aachs%26aulast%3DNishio%26aufirst%3DM.%26atitle%3DThe%2520CH%252F%25CF%2580%2520hydrogen%2520bond%2520in%2520chemistry.%2520Conformation%252C%2520supramolecules%252C%2520optical%2520resolution%2520and%2520interactions%2520involving%2520carbohydrates%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2011%26volume%3D13%26spage%3D13873%26epage%3D13900%26doi%3D10.1039%2Fc1cp20404a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Interactive and Versatile Navigation of Structural Databases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4257</span>– <span class="NLM_lpage">4266</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01756</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01756" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4257-4266&author=O.+Korbauthor=B.+Kuhnauthor=J.+Hertauthor=N.+Taylorauthor=J.+Coleauthor=C.+Groomauthor=M.+Stahl&title=Interactive+and+Versatile+Navigation+of+Structural+Databases&doi=10.1021%2Facs.jmedchem.5b01756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive and Versatile Navigation of Structural Databases</span></div><div class="casAuthors">Korb, Oliver; Kuhn, Bernd; Hert, Jerome; Taylor, Neil; Cole, Jason; Groom, Colin; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4257-4266</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases.  Intuitive pharmacophore queries describing, among others, protein-ligand interaction patterns, ligand scaffolds, or protein environments can be built and modified interactively.  Matching crystal structures are overlaid onto the query and visualized as soon as they are available, enabling the researcher to quickly modify a hypothesis on the fly.  We exemplify the utility of the approach by showing applications relevant to real-world drug discovery projects, including the identification of novel fragments for a specific protein environment or scaffold hopping.  The ability to concurrently search protein-ligand binding sites extd. from the Protein Data Bank (PDB) and small org. mols. from the Cambridge Structural Database (CSD) using the same pharmacophore query further emphasizes the flexibility of CSD-CrossMiner.  We believe that CSD-CrossMiner closes an important gap in mining structural data and will allow users to ext. more value from the growing no. of available crystal structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVHoEyOPb96bVg90H21EOLACvtfcHk0lg0t7wG_MdYcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGms70%253D&md5=1b872969057778cbcc0be80db9bb2bd3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01756%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DJ.%26aulast%3DGroom%26aufirst%3DC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DInteractive%2520and%2520Versatile%2520Navigation%2520of%2520Structural%2520Databases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4257%26epage%3D4266%26doi%3D10.1021%2Facs.jmedchem.5b01756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt:
A Hub for Protein Information</span>. <i>Nucleic Acids
Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">D204</span>– <span class="NLM_lpage">D212</span>, <span class="refDoi"> DOI: 10.1093/nar/gku989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2Fgku989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25348405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D204-D212&author=The%0AUniProt+Consortium&title=UniProt%3A%0AA+Hub+for+Protein+Information&doi=10.1093%2Fnar%2Fgku989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku989%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AA%2520Hub%2520for%2520Protein%2520Information%26jtitle%3DNucleic%2520Acids%250ARes.%26date%3D2015%26volume%3D43%26spage%3DD204%26epage%3DD212%26doi%3D10.1093%2Fnar%2Fgku989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 26 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alan E. Bilsland, Kirsten McAulay, Ryan West, Angelo Pugliese, <span class="NLM_string-name hlFld-ContribAuthor">Justin Bower</span>. </span><span class="cited-content_cbyCitation_article-title">Automated Generation of Novel Fragments Using Screening Data, a Dual SMILES Autoencoder, Transfer Learning and Syntax Correction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (6)
                                     , 2547-2559. <a href="https://doi.org/10.1021/acs.jcim.0c01226" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01226</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01226%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAutomated%252BGeneration%252Bof%252BNovel%252BFragments%252BUsing%252BScreening%252BData%25252C%252Ba%252BDual%252BSMILES%252BAutoencoder%25252C%252BTransfer%252BLearning%252Band%252BSyntax%252BCorrection%26aulast%3DBilsland%26aufirst%3DAlan%2BE.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22102020%26date%3D24052021%26volume%3D61%26issue%3D6%26spage%3D2547%26epage%3D2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhuo Wang, Tao Chen, Hua Liu, Xiao-Li Zhao, Wei-Bo Hu, Hui Yang, Yahu A. Liu, <span class="NLM_string-name hlFld-ContribAuthor">Ke Wen</span>. </span><span class="cited-content_cbyCitation_article-title">Pillar[5]arene-Derived endo-Functionalized Molecular Tube for Mimicking Protein–Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (9)
                                     , 6467-6477. <a href="https://doi.org/10.1021/acs.joc.1c00314" title="DOI URL">https://doi.org/10.1021/acs.joc.1c00314</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.1c00314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.1c00314%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DPillar%25255B5%25255Darene-Derived%252Bendo-Functionalized%252BMolecular%252BTube%252Bfor%252BMimicking%252BProtein%2525E2%252580%252593Ligand%252BInteractions%26aulast%3DWang%26aufirst%3DZhuo%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08022021%26date%3D19042021%26volume%3D86%26issue%3D9%26spage%3D6467%26epage%3D6477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wolfgang Jahnke, Daniel A. Erlanson, Iwan J. P. de Esch, Christopher N. Johnson, Paul N. Mortenson, Yuji Ochi, <span class="NLM_string-name hlFld-ContribAuthor">Tatsuya Urushima</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2019. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15494-15507. <a href="https://doi.org/10.1021/acs.jmedchem.0c01608" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2019%26aulast%3DJahnke%26aufirst%3DWolfgang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14092020%26date%3D23112020%26volume%3D63%26issue%3D24%26spage%3D15494%26epage%3D15507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fei Cong, Xin-Yang Lv, Craig S. Day, <span class="NLM_string-name hlFld-ContribAuthor">Ruben Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Dual Catalytic Strategy for Forging sp2–sp3 and sp3–sp3 Architectures via β-Scission of Aliphatic Alcohol Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (49)
                                     , 20594-20599. <a href="https://doi.org/10.1021/jacs.0c11172" title="DOI URL">https://doi.org/10.1021/jacs.0c11172</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c11172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c11172%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDual%252BCatalytic%252BStrategy%252Bfor%252BForging%252Bsp2%2525E2%252580%252593sp3%252Band%252Bsp3%2525E2%252580%252593sp3%252BArchitectures%252Bvia%252B%2525CE%2525B2-Scission%252Bof%252BAliphatic%252BAlcohol%252BDerivatives%26aulast%3DCong%26aufirst%3DFei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D23102020%26date%3D30112020%26volume%3D142%26issue%3D49%26spage%3D20594%26epage%3D20599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kenneth Goossens, Berthold Wroblowski, Cassiano Langini, Herman van Vlijmen, Amedeo Caflisch, <span class="NLM_string-name hlFld-ContribAuthor">Hans De Winter</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of the Fragment Docking Program SEED. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4881-4893. <a href="https://doi.org/10.1021/acs.jcim.0c00556" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00556</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00556%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAssessment%252Bof%252Bthe%252BFragment%252BDocking%252BProgram%252BSEED%26aulast%3DGoossens%26aufirst%3DKenneth%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D31082020%26date%3D21082020%26volume%3D60%26issue%3D10%26spage%3D4881%26epage%3D4893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anil K. Pandey, Steven E. Kirberger, Jorden A. Johnson, Jennifer R. Kimbrough, Danika K. D. Partridge, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient Synthesis of 1,4-Thiazepanones and 1,4-Thiazepanes as 3D Fragments for Screening Libraries. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (10)
                                     , 3946-3950. <a href="https://doi.org/10.1021/acs.orglett.0c01230" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c01230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c01230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c01230%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DEfficient%252BSynthesis%252Bof%252B1%25252C4-Thiazepanones%252Band%252B1%25252C4-Thiazepanes%252Bas%252B3D%252BFragments%252Bfor%252BScreening%252BLibraries%26aulast%3DPandey%26aufirst%3DAnil%2BK.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07042020%26date%3D29042020%26volume%3D22%26issue%3D10%26spage%3D3946%26epage%3D3950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Damien Bosc, Virgyl Camberlein, Ronan Gealageas, Omar Castillo-Aguilera, Benoit Deprez, <span class="NLM_string-name hlFld-ContribAuthor">Rebecca Deprez-Poulain</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetic Target-Guided Synthesis: Reaching the Age of Maturity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (8)
                                     , 3817-3833. <a href="https://doi.org/10.1021/acs.jmedchem.9b01183" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01183%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DKinetic%252BTarget-Guided%252BSynthesis%25253A%252BReaching%252Bthe%252BAge%252Bof%252BMaturity%26aulast%3DBosc%26aufirst%3DDamien%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D22072019%26date%3D24122019%26date%3D10122019%26volume%3D63%26issue%3D8%26spage%3D3817%26epage%3D3833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jorden A. Johnson, Christos A. Nicolaou, Steven E. Kirberger, Anil K. Pandey, Haitao Hu, <span class="NLM_string-name hlFld-ContribAuthor">William C. K. Pomerantz</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluating the Advantages of Using 3D-Enriched Fragments for Targeting BET Bromodomains. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (12)
                                     , 1648-1654. <a href="https://doi.org/10.1021/acsmedchemlett.9b00414" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00414</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00414%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEvaluating%252Bthe%252BAdvantages%252Bof%252BUsing%252B3D-Enriched%252BFragments%252Bfor%252BTargeting%252BBET%252BBromodomains%26aulast%3DJohnson%26aufirst%3DJorden%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09092019%26date%3D22112019%26date%3D27112019%26date%3D22112019%26volume%3D10%26issue%3D12%26spage%3D1648%26epage%3D1654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dávid  Bajusz</span>, <span class="hlFld-ContribAuthor ">Warren S.  Wade</span>, <span class="hlFld-ContribAuthor ">Grzegorz  Satała</span>, <span class="hlFld-ContribAuthor ">Andrzej J.  Bojarski</span>, <span class="hlFld-ContribAuthor ">Janez  Ilaš</span>, <span class="hlFld-ContribAuthor ">Jessica  Ebner</span>, <span class="hlFld-ContribAuthor ">Florian  Grebien</span>, <span class="hlFld-ContribAuthor ">Henrietta  Papp</span>, <span class="hlFld-ContribAuthor ">Ferenc  Jakab</span>, <span class="hlFld-ContribAuthor ">Alice  Douangamath</span>, <span class="hlFld-ContribAuthor ">Daren  Fearon</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Marion  Schuller</span>, <span class="hlFld-ContribAuthor ">Ivan  Ahel</span>, <span class="hlFld-ContribAuthor ">Amanda  Wakefield</span>, <span class="hlFld-ContribAuthor ">Sándor  Vajda</span>, <span class="hlFld-ContribAuthor ">János  Gerencsér</span>, <span class="hlFld-ContribAuthor ">Péter  Pallai</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring protein hotspots by optimized fragment pharmacophores. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-021-23443-y" title="DOI URL">https://doi.org/10.1038/s41467-021-23443-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-021-23443-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-021-23443-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DExploring%252Bprotein%252Bhotspots%252Bby%252Boptimized%252Bfragment%252Bpharmacophores%26aulast%3DBajusz%26aufirst%3DD%25C3%25A1vid%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José T.  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Arthur C.  Silva</span>, <span class="hlFld-ContribAuthor ">Rafael F.  Dantas</span>, <span class="hlFld-ContribAuthor ">Barbara F.  Gomes</span>, <span class="hlFld-ContribAuthor ">Lauro R.  Souza Neto</span>, <span class="hlFld-ContribAuthor ">Jose  Brandao-Neto</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Owens</span>, <span class="hlFld-ContribAuthor ">Nicholas  Furnham</span>, <span class="hlFld-ContribAuthor ">Bruno J.  Neves</span>, <span class="hlFld-ContribAuthor ">Floriano P.  Silva-Junior</span>, <span class="hlFld-ContribAuthor ">Carolina H.  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">Schistosomiasis Drug Discovery in the Era of Automation and Artificial Intelligence. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fimmu.2021.642383" title="DOI URL">https://doi.org/10.3389/fimmu.2021.642383</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2021.642383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2021.642383%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DSchistosomiasis%252BDrug%252BDiscovery%252Bin%252Bthe%252BEra%252Bof%252BAutomation%252Band%252BArtificial%252BIntelligence%26aulast%3DMoreira-Filho%26aufirst%3DJos%25C3%25A9%2BT.%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew D.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Maksims  Yevglevskis</span>, <span class="hlFld-ContribAuthor ">Amit  Nathubhai</span>, <span class="hlFld-ContribAuthor ">Tony D.  James</span>, <span class="hlFld-ContribAuthor ">Michael D.  Threadgill</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Woodman</span>. </span><span class="cited-content_cbyCitation_article-title">Racemases and epimerases operating through a 1,1-proton transfer mechanism: reactivity, mechanism and inhibition. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>50 </em>
                                    (10)
                                     , 5952-5984. <a href="https://doi.org/10.1039/D0CS00540A" title="DOI URL">https://doi.org/10.1039/D0CS00540A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00540A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00540A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DRacemases%252Band%252Bepimerases%252Boperating%252Bthrough%252Ba%252B1%25252C1-proton%252Btransfer%252Bmechanism%25253A%252Breactivity%25252C%252Bmechanism%252Band%252Binhibition%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2021%26date%3D2021%26volume%3D50%26issue%3D10%26spage%3D5952%26epage%3D5984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roderick E.  Hubbard</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment‐Based Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-35. <a href="https://doi.org/10.1002/0471266949.bmc265" title="DOI URL">https://doi.org/10.1002/0471266949.bmc265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc265%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%2525E2%252580%252590Based%252BLead%252BDiscovery%26aulast%3DHubbard%26aufirst%3DRoderick%2BE.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D35%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clinton G. L.  Veale</span>. </span><span class="cited-content_cbyCitation_article-title">Into the Fray! A Beginner's Guide to Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (8)
                                     , 1199-1225. <a href="https://doi.org/10.1002/cmdc.202000929" title="DOI URL">https://doi.org/10.1002/cmdc.202000929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000929%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DInto%252Bthe%252BFray%252521%252BA%252BBeginner%252527s%252BGuide%252Bto%252BMedicinal%252BChemistry%26aulast%3DVeale%26aufirst%3DClinton%2BG.%2BL.%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D8%26spage%3D1199%26epage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brett  Cosgrove</span>, <span class="hlFld-ContribAuthor ">Kenneth  Down</span>, <span class="hlFld-ContribAuthor ">Sophie  Bertrand</span>, <span class="hlFld-ContribAuthor ">Nicholas C.O.  Tomkinson</span>, <span class="hlFld-ContribAuthor ">Michael D.  Barker</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the effects of the core nitrogen atom configuration on the thermodynamic solubility of 6,5-bicyclic heterocycles. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>33 </em>, 127752. <a href="https://doi.org/10.1016/j.bmcl.2020.127752" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127752</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127752%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DInvestigating%252Bthe%252Beffects%252Bof%252Bthe%252Bcore%252Bnitrogen%252Batom%252Bconfiguration%252Bon%252Bthe%252Bthermodynamic%252Bsolubility%252Bof%252B6%25252C5-bicyclic%252Bheterocycles%26aulast%3DCosgrove%26aufirst%3DBrett%26date%3D2021%26volume%3D33%26spage%3D127752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyu  Qing</span>, <span class="hlFld-ContribAuthor ">Shiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Pei</span>, <span class="hlFld-ContribAuthor ">Luhua  Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple Target Drug Design Using LigBuilder 3. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 279-298. <a href="https://doi.org/10.1007/978-1-0716-1209-5_16" title="DOI URL">https://doi.org/10.1007/978-1-0716-1209-5_16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-0716-1209-5_16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-0716-1209-5_16%26sid%3Dliteratum%253Aachs%26atitle%3DMultiple%252BTarget%252BDrug%252BDesign%252BUsing%252BLigBuilder%252B3%26aulast%3DQing%26aufirst%3DXiaoyu%26date%3D2021%26date%3D2021%26spage%3D279%26epage%3D298%26pub%3DSpringer%2520US%26atitle%3DProtein-Ligand%252BInteractions%252Band%252BDrug%252BDesign%26aulast%3DBallante%26aufirst%3DFlavio%26date%3D2021%26volume%3D2266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Ferrari</span>, <span class="hlFld-ContribAuthor ">Maicol  Bissaro</span>, <span class="hlFld-ContribAuthor ">Simone  Fabbian</span>, <span class="hlFld-ContribAuthor ">Jessica  De Almeida Roger</span>, <span class="hlFld-ContribAuthor ">Stefano  Mammi</span>, <span class="hlFld-ContribAuthor ">Stefano  Moro</span>, <span class="hlFld-ContribAuthor ">Massimo  Bellanda</span>, <span class="hlFld-ContribAuthor ">Mattia  Sturlese</span>. </span><span class="cited-content_cbyCitation_article-title">HT-SuMD: making molecular dynamics simulations suitable for fragment-based screening. A comparative study with NMR. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2021,</strong> <em>36 </em>
                                    (1)
                                     , 1-14. <a href="https://doi.org/10.1080/14756366.2020.1838499" title="DOI URL">https://doi.org/10.1080/14756366.2020.1838499</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2020.1838499&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2020.1838499%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DHT-SuMD%25253A%252Bmaking%252Bmolecular%252Bdynamics%252Bsimulations%252Bsuitable%252Bfor%252Bfragment-based%252Bscreening.%252BA%252Bcomparative%252Bstudy%252Bwith%252BNMR%26aulast%3DFerrari%26aufirst%3DFrancesca%26date%3D2021%26date%3D2020%26volume%3D36%26issue%3D1%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kirill  Kulish</span>, <span class="hlFld-ContribAuthor ">Cosimo  Boldrini</span>, <span class="hlFld-ContribAuthor ">Marta  Castiñeira Reis</span>, <span class="hlFld-ContribAuthor ">Juana M.  Pérez</span>, <span class="hlFld-ContribAuthor ">Syuzanna R.  Harutyunyan</span>. </span><span class="cited-content_cbyCitation_article-title">Lewis Acid Promoted Dearomatization of Naphthols. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (68)
                                     , 15843-15846. <a href="https://doi.org/10.1002/chem.202003392" title="DOI URL">https://doi.org/10.1002/chem.202003392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202003392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202003392%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DLewis%252BAcid%252BPromoted%252BDearomatization%252Bof%252BNaphthols%26aulast%3DKulish%26aufirst%3DKirill%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D68%26spage%3D15843%26epage%3D15846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rodolfo  Cadilla</span>, <span class="hlFld-ContribAuthor ">David N.  Deaton</span>, <span class="hlFld-ContribAuthor ">Young  Do</span>, <span class="hlFld-ContribAuthor ">Patricia A.  Elkins</span>, <span class="hlFld-ContribAuthor ">Daniela  Ennulat</span>, <span class="hlFld-ContribAuthor ">Jeffrey H.  Guss</span>, <span class="hlFld-ContribAuthor ">Jason  Holt</span>, <span class="hlFld-ContribAuthor ">Michael R.  Jeune</span>, <span class="hlFld-ContribAuthor ">Andrew G.  King</span>, <span class="hlFld-ContribAuthor ">Jan C.  Klapwijk</span>, <span class="hlFld-ContribAuthor ">H. Fritz  Kramer</span>, <span class="hlFld-ContribAuthor ">Nicholas J.  Kramer</span>, <span class="hlFld-ContribAuthor ">Susan B.  Laffan</span>, <span class="hlFld-ContribAuthor ">Paresh I.  Masuria</span>, <span class="hlFld-ContribAuthor ">Alan V.  McDougal</span>, <span class="hlFld-ContribAuthor ">Paul N.  Mortenson</span>, <span class="hlFld-ContribAuthor ">Caterina  Musetti</span>, <span class="hlFld-ContribAuthor ">Gregory E.  Peckham</span>, <span class="hlFld-ContribAuthor ">Beth L.  Pietrak</span>, <span class="hlFld-ContribAuthor ">Chuck  Poole</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Price</span>, <span class="hlFld-ContribAuthor ">Alan R.  Rendina</span>, <span class="hlFld-ContribAuthor ">Girish  Sati</span>, <span class="hlFld-ContribAuthor ">Gordon  Saxty</span>, <span class="hlFld-ContribAuthor ">Barry G.  Shearer</span>, <span class="hlFld-ContribAuthor ">Lisa M.  Shewchuk</span>, <span class="hlFld-ContribAuthor ">Helen F.  Sneddon</span>, <span class="hlFld-ContribAuthor ">Eugene L.  Stewart</span>, <span class="hlFld-ContribAuthor ">J. Darren  Stuart</span>, <span class="hlFld-ContribAuthor ">Dean N.  Thomas</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Thomson</span>, <span class="hlFld-ContribAuthor ">Paris  Ward</span>, <span class="hlFld-ContribAuthor ">Joseph W.  Wilson</span>, <span class="hlFld-ContribAuthor ">Tiahshun  Xu</span>, <span class="hlFld-ContribAuthor ">Mark A.  Youngman</span>. </span><span class="cited-content_cbyCitation_article-title">The exploration of aza-quinolines as hematopoietic prostaglandin D synthase (H-PGDS) inhibitors with low brain exposure. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (23)
                                     , 115791. <a href="https://doi.org/10.1016/j.bmc.2020.115791" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115791</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115791%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bexploration%252Bof%252Baza-quinolines%252Bas%252Bhematopoietic%252Bprostaglandin%252BD%252Bsynthase%252B%252528H-PGDS%252529%252Binhibitors%252Bwith%252Blow%252Bbrain%252Bexposure%26aulast%3DCadilla%26aufirst%3DRodolfo%26date%3D2020%26volume%3D28%26issue%3D23%26spage%3D115791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">James S  Fraser</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of allosteric binding sites by crystallographic fragment screening. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>65 </em>, 209-216. <a href="https://doi.org/10.1016/j.sbi.2020.08.004" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.08.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.08.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DDiscovery%252Bof%252Ballosteric%252Bbinding%252Bsites%252Bby%252Bcrystallographic%252Bfragment%252Bscreening%26aulast%3DKrojer%26aufirst%3DTobias%26date%3D2020%26volume%3D65%26spage%3D209%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rupesh  Agarwal</span>, <span class="hlFld-ContribAuthor ">Barbara A.  Bensing</span>, <span class="hlFld-ContribAuthor ">Dehui  Mi</span>, <span class="hlFld-ContribAuthor ">Paige N.  Vinson</span>, <span class="hlFld-ContribAuthor ">Jerome  Baudry</span>, <span class="hlFld-ContribAuthor ">Tina M.  Iverson</span>, <span class="hlFld-ContribAuthor ">Jeremy C.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Structure based virtual screening identifies small molecule effectors for the sialoglycan binding protein Hsa. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2020,</strong> <em>477 </em>
                                    (19)
                                     , 3695-3707. <a href="https://doi.org/10.1042/BCJ20200332" title="DOI URL">https://doi.org/10.1042/BCJ20200332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20200332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20200332%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DStructure%252Bbased%252Bvirtual%252Bscreening%252Bidentifies%252Bsmall%252Bmolecule%252Beffectors%252Bfor%252Bthe%252Bsialoglycan%252Bbinding%252Bprotein%252BHsa%26aulast%3DAgarwal%26aufirst%3DRupesh%26date%3D2020%26date%3D2020%26volume%3D477%26issue%3D19%26spage%3D3695%26epage%3D3707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maicol  Bissaro</span>, <span class="hlFld-ContribAuthor ">Mattia  Sturlese</span>, <span class="hlFld-ContribAuthor ">Stefano  Moro</span>. </span><span class="cited-content_cbyCitation_article-title">The rise of molecular simulations in fragment-based drug design (FBDD): an overview. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1693-1701. <a href="https://doi.org/10.1016/j.drudis.2020.06.023" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.06.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.06.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.06.023%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DThe%252Brise%252Bof%252Bmolecular%252Bsimulations%252Bin%252Bfragment-based%252Bdrug%252Bdesign%252B%252528FBDD%252529%25253A%252Ban%252Boverview%26aulast%3DBissaro%26aufirst%3DMaicol%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1693%26epage%3D1701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauro Ribeiro  de Souza Neto</span>, <span class="hlFld-ContribAuthor ">José Teófilo  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Bruno Junior  Neves</span>, <span class="hlFld-ContribAuthor ">Rocío Lucía Beatriz Riveros  Maidana</span>, <span class="hlFld-ContribAuthor ">Ana Carolina Ramos  Guimarães</span>, <span class="hlFld-ContribAuthor ">Nicholas  Furnham</span>, <span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>, <span class="hlFld-ContribAuthor ">Floriano Paes  Silva</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Strategies to Support Fragment-to-Lead Optimization in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00093" title="DOI URL">https://doi.org/10.3389/fchem.2020.00093</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00093%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BStrategies%252Bto%252BSupport%252BFragment-to-Lead%252BOptimization%252Bin%252BDrug%252BDiscovery%26aulast%3Dde%2BSouza%2BNeto%26aufirst%3DLauro%2BRibeiro%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">György G.  Ferenczy</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Thermodynamic profiling for fragment-based lead discovery and optimization. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , 117-129. <a href="https://doi.org/10.1080/17460441.2020.1691166" title="DOI URL">https://doi.org/10.1080/17460441.2020.1691166</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2020.1691166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2020.1691166%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThermodynamic%252Bprofiling%252Bfor%252Bfragment-based%252Blead%252Bdiscovery%252Band%252Boptimization%26aulast%3DFerenczy%26aufirst%3DGy%25C3%25B6rgy%2BG.%26date%3D2020%26date%3D2019%26volume%3D15%26issue%3D1%26spage%3D117%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Geschwindner</span>, <span class="hlFld-ContribAuthor ">Johan  Ulander</span>. </span><span class="cited-content_cbyCitation_article-title">The current impact of water thermodynamics for small-molecule drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 1221-1225. <a href="https://doi.org/10.1080/17460441.2019.1664468" title="DOI URL">https://doi.org/10.1080/17460441.2019.1664468</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1664468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1664468%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bcurrent%252Bimpact%252Bof%252Bwater%252Bthermodynamics%252Bfor%252Bsmall-molecule%252Bdrug%252Bdiscovery%26aulast%3DGeschwindner%26aufirst%3DStefan%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D12%26spage%3D1221%26epage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Sergio  Ruiz-Carmona</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">An investigation of structural stability in protein-ligand complexes reveals the balance between order and disorder. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2019,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-019-0205-5" title="DOI URL">https://doi.org/10.1038/s42004-019-0205-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-019-0205-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-019-0205-5%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DAn%252Binvestigation%252Bof%252Bstructural%252Bstability%252Bin%252Bprotein-ligand%252Bcomplexes%252Breveals%252Bthe%252Bbalance%252Bbetween%252Border%252Band%252Bdisorder%26aulast%3DMajewski%26aufirst%3DMaciej%26date%3D2019%26date%3D2019%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jason C.  Cole</span>, <span class="hlFld-ContribAuthor ">Seth  Wiggin</span>, <span class="hlFld-ContribAuthor ">Francesca  Stanzione</span>. </span><span class="cited-content_cbyCitation_article-title">New insights and innovation from a million crystal structures in the Cambridge Structural Database. </span><span class="cited-content_cbyCitation_journal-name">Structural Dynamics</span><span> <strong>2019,</strong> <em>6 </em>
                                    (5)
                                     , 054301. <a href="https://doi.org/10.1063/1.5116878" title="DOI URL">https://doi.org/10.1063/1.5116878</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.5116878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.5116878%26sid%3Dliteratum%253Aachs%26jtitle%3DStructural%2520Dynamics%26atitle%3DNew%252Binsights%252Band%252Binnovation%252Bfrom%252Ba%252Bmillion%252Bcrystal%252Bstructures%252Bin%252Bthe%252BCambridge%252BStructural%252BDatabase%26aulast%3DCole%26aufirst%3DJason%2BC.%26date%3D2019%26volume%3D6%26issue%3D5%26spage%3D054301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characteristics of the proteins, structures, and binding pockets. (a) Distribution of protein classes, as codified by the PDB.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Numeric labels indicate the number of unique proteins belonging to each protein class, as indicated by their respective UniProt access codes.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (b) Distribution of crystallographic resolution values. (c) Distribution of fragment-binding pocket volumes using dpocket,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> normalized by the occurrence of a given protein binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Characteristics of the fragment hits (<i>N</i> = 462, black profiles). Distributions of the (a) number of heavy atoms, (b) ACD calculated log <i>P</i>, (c) number of atoms capable of accepting or donating H-bonds expressed as a percentage of the fragment’s heavy atoms, (d) fraction of sp<sup>3</sup>-hybridized carbon atoms (Fsp<sup>3</sup>), (e) number of rotatable bonds, (f) number of chiral centers, (g) number of ring assemblies, and (h) formal charges are shown. (i) Frequency distribution of Tanimoto distances calculated based on ECFP4 for all possible fragment pairs. Distributions for a representative commercial fragment library<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (<i>N</i> = 1794, gray profiles) are included.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0012.jpeg" id="rightTab-GRAPHIC-d7e595-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Bemis–Murcko frameworks occurring more than 10 times across the fragment hits. The number of observations and frequency of occurrence as a percentage of the total observed frameworks (<i>N</i> = 52, see <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S2</a> for the full list) are indicated in the caption. Values in parentheses indicate the number of unique proteins the framework was found bound to. (b) The most frequently occurring individual ring assemblies. The number of observations and frequency of occurrence as a percentage of the total observed individual ring assemblies (<i>N</i> = 138, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf" class="ext-link">Table S3</a>) are indicated in the caption. Values in parentheses indicate the number of unique proteins bound by the ring assembly in this data set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Surface characteristics of the fragment–pocket complexes. (a) Polar and apolar buried SA of fragments, expressed as a percentage of their total polar and apolar SA, respectively. The total buried SA of fragments is also indicated as a percentage of the whole-fragment SA. (b) Extents of the total, apolar, and polar SA that are buried by the fragments upon binding. (c) Buried polar SA of the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selected fragment hits bound to proteins. (a) Maximum and (b) minimum solvent exposure of fragments (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OMQ">3OMQ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAH">5JAH</a>, respectively). (c) Maximum number of polar interactions (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>). (d) Maximum number of additional directional interactions (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS">5EGS</a>). (e) and (f) Examples of fragments binding to multiple pockets within the same protein (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FPO">5FPO</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CLP">5CLP</a>, respectively). Fragments are depicted as bold sticks (cyan carbon atoms). Relevant H-bonds and additional directional interactions are indicated as dashed black lines. Relevant protein pocket surfaces are displayed as gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Polar interactions established by the fragment hits. (a) Frequency distribution of polar interaction counts per fragment complex, including H-bonds to protein or water and coordination bonds to metal ions. Fragment complexes that do not display any water H-bonds or metal coordination bonds (54 and 95% of the total, respectively) have been omitted from the histogram for clarity. (b) Frequency distribution of fragment H-bonds to protein amino acids at the side-chain and the backbone level. (c) Frequency distribution of buried surface area for protein atoms involved in H-bonds to fragments. (d) Frequency distribution of H-bonds between fragment atoms (neutral and ionizable) and protein amino acid side chains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Frequency distribution of additional directional interactions per fragment pocket including: arene-based interactions (i.e., arene–arene, arene–cation, and arene–hydrogen), carbon H-bonds, sulfur-mediated contacts, and halogen (i.e., iodine, bromine, and chlorine) bonds to protein atoms. The fractions of fragment complexes without arene (58%), carbon H-bonds (88%), sulfur contacts (89%), and halogen bonds (96%) have been omitted from the graph for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Selected fragment hits bound to different proteins. (a, b) Different degrees of solvent exposure of 5-hydroxyindole bound to RadA (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B3C">4B3C</a>) and leukotriene A-4 hydrolase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUH">3FUH</a>), respectively. (c, d) Different tautomers of adenine bound to BAZ2B bromodomain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYX">5DYX</a>) and Hsp90 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YED">2YED</a>), respectively. (e, f) Different interactions of the nitro group of 5-nitro-benzimidazole bound to nicotinamide phosphoribosyltransferase (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N9C">4N9C</a>) and PDE10A2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSA">4MSA</a>), respectively. Fragments are depicted as bold sticks (cyan carbon atoms) and relevant H-bonds as dashed black lines. Relevant protein pocket surfaces are displayed as gray mesh.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Fragment hits H-bonded to the side chain of asparagine. (a) Spatial distribution of the fragment atoms interacting with asparagine side chains (<i>N</i> = 70). (b–h) Diverse selection of fragment hits engaged in paired H-bonds to the side chain of asparagine (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CUR">4CUR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LR6">4LR6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ8">4TZ8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YK0">4YK0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYU">5DYU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E3G">5E3G</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9Y">5E9Y</a>). Fragment hits and the asparagine side chain (backbone atoms omitted for clarity) are depicted as bold sticks (cyan and light gray carbon atoms, respectively) and relevant H-bonds as dashed black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Fragment hits H-bonded to structural water molecules. (a, b) Water molecules as part of extended nonbonded interaction networks (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MOH">5MOH</a>), (b, c) water molecules as the only H-bond partners for the fragment (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NOW">5NOW</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y3P">4Y3P</a>, respectively). Fragment hits are depicted as bold sticks (cyan carbon atoms), water molecules as red spheres, and relevant H-bonds as dashed black lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/medium/jm-2018-01855d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Fragment hits engaged in arene-based interactions with the protein as the main attractive interaction in the absence of protein- and water-mediated H-bonds. (a) Arene interaction with backbone amide bonds (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K2Y">4K2Y</a>), (b, c) arene interaction with the side chains of phenylalanine (PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y37">4Y37</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I5W">5I5W</a>), and (d) arene interaction with the side chain of tryptophan (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAN">5JAN</a>). Fragment hits are depicted as bold sticks (cyan carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.8b01855/20190404/images/large/jm-2018-01855d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01855&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71684" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71684" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 66 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A “Rule of Three” for Fragment-Based Lead Discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%9CRule+of+Three%E2%80%9D+for+Fragment-Based+Lead+Discovery%3F&doi=10.1016%2FS1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2eYw3Tx4u0M54bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%259CRule%2520of%2520Three%25E2%2580%259D%2520for%2520Fragment-Based%2520Lead%2520Discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26spage%3D876%26epage%3D877%26doi%3D10.1016%2FS1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuker, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span> <span> </span><span class="NLM_article-title">Discovering High-Affinity Ligands for Proteins: SAR by NMR</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">1531</span>– <span class="NLM_lpage">1534</span>, <span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+High-Affinity+Ligands+for+Proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0lgo5wRCq7iicQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520High-Affinity%2520Ligands%2520for%2520Proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyne, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">5003</span>, <span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-Based+Approaches+in+Drug+Discovery+and+Chemical+Biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lgoFZLB5zVTmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-Based%2520Approaches%2520in%2520Drug%2520Discovery%2520and%2520Chemical%2520Biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span> <span> </span><span class="NLM_article-title">Introduction to Fragment-Based Drug Discovery</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>317</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1007/128_2011_180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21695633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2012&pages=1-32&author=D.+A.+Erlanson&title=Introduction+to+Fragment-Based+Drug+Discovery&doi=10.1007%2F128_2011_180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction to fragment-based drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">317</span>
        (<span class="NLM_cas:issue">Fragment-Based Drug Discovery and X-Ray Crystallography</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">0340-1022</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has emerged in the past decade as a powerful tool for discovering drug leads.  The approach first identifies starting points: very small mols. (fragments) that are about half the size of typical drugs.  These fragments are then expanded or linked together to generate drug leads.  Although the origins of the technique date back some 30 years, it was only in the mid-1990s that exptl. techniques became sufficiently sensitive and rapid for the concept to be become practical.  Since that time, the field has exploded: FBDD has played a role in discovery of at least 18 drugs that have entered the clinic, and practitioners of FBDD can be found throughout the world in both academia and industry.  Literally dozens of reviews have been published on various aspects of FBDD or on the field as a whole, as have three books (Jahnke and Erlanson, Fragment-based approaches in drug discovery, 2006; Zartler and Shapiro, Fragment-based drug discovery: a practical approach, 2008; Kuo, Fragment based drug design: tools, practical approaches, and examples, 2011).  However, this chapter will assume that the reader is approaching the field with little prior knowledge.  It will introduce some of the key concepts, set the stage for the chapters to follow, and demonstrate how X-ray crystallog. plays a central role in fragment identification and advancement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcNmFPvb7c5LVg90H21EOLACvtfcHk0lgoFZLB5zVTmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ShurvM&md5=735c05a6b5f1d4ee9e4d34e15ac2b8fe</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2012%26volume%3D317%26spage%3D1%26epage%3D32%26doi%3D10.1007%2F128_2011_180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">The Rise of Fragment-Based Drug Discovery</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">192</span>, <span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+Rise+of+Fragment-Based+Drug+Discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lgoFZLB5zVTmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520Rise%2520of%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Visegrády, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery on G-Protein-Coupled Receptors</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">820</span>, <span class="refDoi"> DOI: 10.1517/17460441.2013.794135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1517%2F17460441.2013.794135" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=811-820&author=A.+Visegr%C3%A1dyauthor=G.+M.+Keser%C5%B1&title=Fragment-Based+Lead+Discovery+on+G-Protein-Coupled+Receptors&doi=10.1517%2F17460441.2013.794135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.794135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.794135%26sid%3Dliteratum%253Aachs%26aulast%3DVisegr%25C3%25A1dy%26aufirst%3DA.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DFragment-Based%2520Lead%2520Discovery%2520on%2520G-Protein-Coupled%2520Receptors%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D811%26epage%3D820%26doi%3D10.1517%2F17460441.2013.794135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Experiences in Fragment-Based Drug Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2012.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.tips.2012.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=22459076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=224-232&author=C.+W.+Murrayauthor=M.+L.+Verdonkauthor=D.+C.+Rees&title=Experiences+in+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.tips.2012.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Experiences in fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Verdonk, Marcel L.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">224-232</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) has become established in both industry and academia as an alternative approach to high-throughput screening for the generation of chem. leads for drug targets.  In FBDD, specialised detection methods are used to identify small chem. compds. (fragments) that bind to the drug target, and structural biol. is usually employed to establish their binding mode and to facilitate their optimization.  In this article, we present three recent and successful case histories in FBDD.  We then re-examine the key concepts and challenges of FBDD with particular emphasis on recent literature and our own experience from a substantial no. of FBDD applications.  Our opinion is that careful application of FBDD is living up to its promise of delivering high quality leads with good phys. properties and that in future many drug mols. will be derived from fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUzcsuUdmx4bVg90H21EOLACvtfcHk0lj5afupzIjHSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslejsLs%253D&md5=c97b6b84a4afeadb87ee7d46e6d2bb91</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2012.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2012.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DExperiences%2520in%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D224%26epage%3D232%26doi%3D10.1016%2Fj.tips.2012.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heikkila, T. J.</span>; <span class="NLM_string-name">Surade, S.</span>; <span class="NLM_string-name">Silvestre, H. L.</span>; <span class="NLM_string-name">Dias, M. V. B.</span>; <span class="NLM_string-name">Ciulli, A.</span>; <span class="NLM_string-name">Bromfield, K.</span>; <span class="NLM_string-name">Scott, D.</span>; <span class="NLM_string-name">Howard, N.</span>; <span class="NLM_string-name">Wen, S.</span>; <span class="NLM_string-name">Wei, A. H.</span>; <span class="NLM_string-name">Osborne, D.</span>; <span class="NLM_string-name">Abell, C.</span>; <span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery in Academia: Experiences From a Tuberculosis Programme</span>. In  <i>From Molecules to Medicines</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sussman, J. L.</span>; <span class="NLM_string-name">Spadon, P.</span></span>, Eds.; NATO Science for Peace and Security Series A: Chemistry and Biology; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Netherlands</span>, <span class="NLM_year">2009</span>; pp  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">36</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2F978-90-481-2339-1_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=21-36&author=T.+J.+Heikkila&author=S.+Surade&author=H.+L.+Silvestre&author=M.+V.+B.+Dias&author=A.+Ciulli&author=K.+Bromfield&author=D.+Scott&author=N.+Howard&author=S.+Wen&author=A.+H.+Wei&author=D.+Osborne&author=C.+Abell&author=T.+L.+Blundellauthor=J.+L.+Sussman&author=P.+Spadon&title=From+Molecules+to+Medicines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2F978-90-481-2339-1_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-90-481-2339-1_3%26sid%3Dliteratum%253Aachs%26aulast%3DHeikkila%26aufirst%3DT.%2BJ.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520in%2520Academia%253A%2520Experiences%2520From%2520a%2520Tuberculosis%2520Programme%26btitle%3DFrom%2520Molecules%2520to%2520Medicines%26aulast%3DSussman%26aufirst%3DJ.%2BL.%26pub%3DSpringer%26date%3D2009%26spage%3D21%26epage%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zartler, E.
R.</span></span> <span> </span><span class="NLM_article-title">Fragonomics: The -omics with Real Impact</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">952</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1021/ml5003212</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5003212" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtleqtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=952-953&author=E.%0AR.+Zartler&title=Fragonomics%3A+The+-omics+with+Real+Impact&doi=10.1021%2Fml5003212"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Fragonomics: The -Omics with Real Impact</span></div><div class="casAuthors">Zartler, Edward R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">952-953</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragonomics is the process of using small, relatively simple mols. to generate chem. starting points for hit generation.  Fragonomics has come of age and is now one of the major concepts in hit generation.  What is its future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1kUkPpbIBrVg90H21EOLACvtfcHk0lj3LPBINmoUwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtleqtLjK&md5=2c57c10c7f6f74f9dfb804342ea66a70</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fml5003212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5003212%26sid%3Dliteratum%253Aachs%26aulast%3DZartler%26aufirst%3DE.%2BR.%26atitle%3DFragonomics%253A%2520The%2520-omics%2520with%2520Real%2520Impact%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D952%26epage%3D953%26doi%3D10.1021%2Fml5003212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romasanta, A. K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Sijde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellsten, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Muijlwijk-Koezen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span> <span> </span><span class="NLM_article-title">When Fragments Link: A Bibliometric Perspective on the Development of Fragment-Based Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2018.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.drudis.2018.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=29738823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlaqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1596-1609&author=A.+K.+S.+Romasantaauthor=P.+van+der+Sijdeauthor=I.+Hellstenauthor=R.+E.+Hubbardauthor=G.+M.+Keseruauthor=J.+van+Muijlwijk-Koezenauthor=I.+J.+P.+de+Esch&title=When+Fragments+Link%3A+A+Bibliometric+Perspective+on+the+Development+of+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.drudis.2018.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">When fragments link: a bibliometric perspective on the development of fragment-based drug discovery</span></div><div class="casAuthors">Romasanta, Angelo K. S.; van der Sijde, Peter; Hellsten, Iina; Hubbard, Roderick E.; Keseru, Gyorgy M.; van Muijlwijk-Koezen, Jacqueline; de Esch, Iwan J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1596-1609</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a highly interdisciplinary field, rich in ideas integrated from pharmaceutical sciences, chem., biol., and physics, among others.  To enrich our understanding of the development of the field, we used bibliometric techniques to analyze 3642 publications in FBDD, complementing accounts by key practitioners.  Mapping its core papers, we found the transfer of knowledge from academia to industry.  Co-authorship anal. showed that university-industry collaboration has grown over time.  Moreover, we show how ideas from other scientific disciplines have been integrated into the FBDD paradigm.  Keyword anal. showed that the field is organized into four interconnected practices: library design, fragment screening, computational methods, and optimization.  This study highlights the importance of interactions among various individuals and institutions from diverse disciplines in newly emerging scientific fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5WVEQd1QaULVg90H21EOLACvtfcHk0lj3LPBINmoUwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlaqtb0%253D&md5=5b8042da1162e24116f29867d0c4b97b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2018.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2018.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DRomasanta%26aufirst%3DA.%2BK.%2BS.%26aulast%3Dvan%2Bder%2BSijde%26aufirst%3DP.%26aulast%3DHellsten%26aufirst%3DI.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26aulast%3Dvan%2BMuijlwijk-Koezen%26aufirst%3DJ.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26atitle%3DWhen%2520Fragments%2520Link%253A%2520A%2520Bibliometric%2520Perspective%2520on%2520the%2520Development%2520of%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D1596%26epage%3D1609%26doi%3D10.1016%2Fj.drudis.2018.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haass, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproesser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smalley, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settachatgul, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artis, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Selective Inhibitor of Oncogenic B-Raf Kinase with Potent Antimelanoma Activity</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">3041</span>– <span class="NLM_lpage">3046</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711741105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.0711741105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=18287029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=3041-3046&author=J.+Tsaiauthor=J.+T.+Leeauthor=W.+Wangauthor=J.+Zhangauthor=H.+Choauthor=S.+Mamoauthor=R.+Bremerauthor=S.+Gilletteauthor=J.+Kongauthor=N.+K.+Haassauthor=K.+Sproesserauthor=L.+Liauthor=K.+S.+Smalleyauthor=D.+Fongauthor=Y.+L.+Zhuauthor=A.+Marimuthuauthor=H.+Nguyenauthor=B.+Lamauthor=J.+Liuauthor=I.+Cheungauthor=J.+Riceauthor=Y.+Suzukiauthor=C.+Luuauthor=C.+Settachatgulauthor=R.+Shellooeauthor=J.+Cantwellauthor=S.+H.+Kimauthor=J.+Schlessingerauthor=K.+Y.+Zhangauthor=B.+L.+Westauthor=B.+Powellauthor=G.+Habetsauthor=C.+Zhangauthor=P.+N.+Ibrahimauthor=P.+Hirthauthor=D.+R.+Artisauthor=M.+Herlynauthor=G.+Bollag&title=Discovery+of+a+Selective+Inhibitor+of+Oncogenic+B-Raf+Kinase+with+Potent+Antimelanoma+Activity&doi=10.1073%2Fpnas.0711741105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a selective inhibitor of oncogenic B-Raf kinase with potent antimelanoma activity</span></div><div class="casAuthors">Tsai, James; Lee, John T.; Wang, Weiru; Zhang, Jiazhong; Cho, Hanna; Mamo, Shumeye; Bremer, Ryan; Gillette, Sam; Kong, Jun; Haass, Nikolas K.; Sproesser, Katrin; Li, Ling; Smalley, Keiran S. M.; Fong, Daniel; Zhu, Yong-Liang; Marimuthu, Adhirai; Nguyen, Hoa; Lam, Billy; Liu, Jennifer; Cheung, Ivana; Rice, Julie; Suzuki, Yoshihisa; Luu, Catherine; Settachatgul, Calvin; Shellooe, Rafe; Cantwell, John; Kim, Sung-Hou; Schlessinger, Joseph; Zhang, Kam Y. J.; West, Brian L.; Powell, Ben; Habets, Gaston; Zhang, Chao; Ibrahim, Prabha N.; Hirth, Peter; Artis, Dean R.; Herlyn, Meenhard; Bollag, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3041-3046</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">BRAFV600E is the most frequent oncogenic protein kinase mutation known.  Furthermore, inhibitors targeting "active" protein kinases have demonstrated significant utility in the therapeutic repertoire against cancer.  Therefore, the authors pursued the development of specific kinase inhibitors targeting B-Raf, and the V600E allele in particular.  By using a structure-guided discovery approach, a potent and selective inhibitor of active B-Raf has been discovered.  PLX4720, a 7-azaindole deriv. that inhibits B-RafV600E with an IC50 of 13 nM, defines a class of kinase inhibitor with marked selectivity in both biochem. and cellular assays.  PLX4720 preferentially inhibits the active B-RafV600E kinase compared with a broad spectrum of other kinases, and potent cytotoxic effects are also exclusive to cells bearing the V600E allele.  Consistent with the high degree of selectivity, ERK phosphorylation is potently inhibited by PLX4720 in B-RafV600E-bearing tumor cell lines but not in cells lacking oncogenic B-Raf.  In melanoma models, PLX4720 induces cell cycle arrest and apoptosis exclusively in B-RafV600E-pos. cells.  In B-RafV600E-dependent tumor xenograft models, orally dosed PLX4720 causes significant tumor growth delays, including tumor regressions, without evidence of toxicity.  The work described here represents the entire discovery process, from initial identification through structural and biol. studies in animal models to a promising therapeutic for testing in cancer patients bearing B-RafV600E-driven tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPX53IQCbDJ7Vg90H21EOLACvtfcHk0lj3LPBINmoUwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtVSitrk%253D&md5=273b94202f99984e6eb24723d0c6141f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711741105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711741105%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DMamo%26aufirst%3DS.%26aulast%3DBremer%26aufirst%3DR.%26aulast%3DGillette%26aufirst%3DS.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHaass%26aufirst%3DN.%2BK.%26aulast%3DSproesser%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSmalley%26aufirst%3DK.%2BS.%26aulast%3DFong%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BL.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DH.%26aulast%3DLam%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DI.%26aulast%3DRice%26aufirst%3DJ.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DLuu%26aufirst%3DC.%26aulast%3DSettachatgul%26aufirst%3DC.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DK.%2BY.%26aulast%3DWest%26aufirst%3DB.%2BL.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%2BN.%26aulast%3DHirth%26aufirst%3DP.%26aulast%3DArtis%26aufirst%3DD.%2BR.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBollag%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520Selective%2520Inhibitor%2520of%2520Oncogenic%2520B-Raf%2520Kinase%2520with%2520Potent%2520Antimelanoma%2520Activity%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2008%26volume%3D105%26spage%3D3041%26epage%3D3046%26doi%3D10.1073%2Fpnas.0711741105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolop, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirth, P.</span></span> <span> </span><span class="NLM_article-title">Vemurafenib: The First Drug Approved for BRAF-Mutant Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">873</span>– <span class="NLM_lpage">886</span>, <span class="refDoi"> DOI: 10.1038/nrd3847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd3847" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23060265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=873-886&author=G.+Bollagauthor=J.+Tsaiauthor=J.+Zhangauthor=C.+Zhangauthor=P.+Ibrahimauthor=K.+Nolopauthor=P.+Hirth&title=Vemurafenib%3A+The+First+Drug+Approved+for+BRAF-Mutant+Cancer&doi=10.1038%2Fnrd3847"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Vemurafenib: the first drug approved for BRAF-mutant cancer</span></div><div class="casAuthors">Bollag, Gideon; Tsai, James; Zhang, Jiazhong; Zhang, Chao; Ibrahim, Prabha; Nolop, Keith; Hirth, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">873-886</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The identification of driver oncogenes has provided important targets for drugs that can change the landscape of cancer therapies.  One such example is the BRAF oncogene, which is found in about half of all melanomas as well as several other cancers.  As a druggable kinase, oncogenic BRAF has become a crucial target of small-mol. drug discovery efforts.  Following a rapid clin. development path, vemurafenib (Zelboraf; Plexxikon/Roche) was approved for the treatment of BRAF-mutated metastatic melanoma in the United States in August 2011 and the European Union in Feb. 2012.  This Review describes the underlying biol. of BRAF, the technol. used to identify vemurafenib and its clin. development milestones, along with future prospects based on lessons learned during its development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT6Fbb43oRTbVg90H21EOLACvtfcHk0lh2r1gXaWa7Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2rsLjL&md5=b1b38cad542be9b3a30060b77c584bc1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3847%26sid%3Dliteratum%253Aachs%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DTsai%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DNolop%26aufirst%3DK.%26aulast%3DHirth%26aufirst%3DP.%26atitle%3DVemurafenib%253A%2520The%2520First%2520Drug%2520Approved%2520for%2520BRAF-Mutant%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D873%26epage%3D886%26doi%3D10.1038%2Fnrd3847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Souers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leverson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boghaert, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catron, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dayton, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enschede, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairbrother, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hymowitz, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maecker, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitten, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nimmer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oleksijew, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seymour, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahir, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humerickhouse, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span> <span> </span><span class="NLM_article-title">ABT-199, a Potent and Selective BCL-2 Inhibitor, Achieves Antitumor Activity While Sparing Platelets</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">202</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1038/nm.3048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnm.3048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23291630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=202-208&author=A.+J.+Souersauthor=J.+D.+Leversonauthor=E.+R.+Boghaertauthor=S.+L.+Acklerauthor=N.+D.+Catronauthor=J.+Chenauthor=B.+D.+Daytonauthor=H.+Dingauthor=S.+H.+Enschedeauthor=W.+J.+Fairbrotherauthor=D.+C.+S.+Huangauthor=S.+G.+Hymowitzauthor=S.+Jinauthor=S.+L.+Khawauthor=P.+J.+Kovarauthor=L.+T.+Lamauthor=J.+Leeauthor=H.+L.+Maeckerauthor=K.+C.+Marshauthor=K.+D.+Masonauthor=M.+J.+Mittenauthor=P.+M.+Nimmerauthor=A.+Oleksijewauthor=C.+H.+Parkauthor=C.-M.+Parkauthor=D.+C.+Phillipsauthor=A.+W.+Robertsauthor=D.+Sampathauthor=J.+F.+Seymourauthor=M.+L.+Smithauthor=G.+M.+Sullivanauthor=S.+K.+Tahirauthor=C.+Tseauthor=M.+D.+Wendtauthor=Y.+Xiaoauthor=J.+C.+Xueauthor=H.+Zhangauthor=R.+A.+Humerickhouseauthor=S.+H.+Rosenbergauthor=S.+W.+Elmore&title=ABT-199%2C+a+Potent+and+Selective+BCL-2+Inhibitor%2C+Achieves+Antitumor+Activity+While+Sparing+Platelets&doi=10.1038%2Fnm.3048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ABT-199, a potent and selective BCL-2 inhibitor, achieves antitumor activity while sparing platelets</span></div><div class="casAuthors">Souers, Andrew J.; Leverson, Joel D.; Boghaert, Erwin R.; Ackler, Scott L.; Catron, Nathaniel D.; Chen, Jun; Dayton, Brian D.; Ding, Hong; Enschede, Sari H.; Fairbrother, Wayne J.; Huang, David C. S.; Hymowitz, Sarah G.; Jin, Sha; Khaw, Seong Lin; Kovar, Peter J.; Lam, Lloyd T.; Lee, Jackie; Maecker, Heather L.; Marsh, Kennan C.; Mason, Kylie D.; Mitten, Michael J.; Nimmer, Paul M.; Oleksijew, Anatol; Park, Chang H.; Park, Cheol-Min; Phillips, Darren C.; Roberts, Andrew W.; Sampath, Deepak; Seymour, John F.; Smith, Morey L.; Sullivan, Gerard M.; Tahir, Stephen K.; Tse, Chris; Wendt, Michael D.; Xiao, Yu; Xue, John C.; Zhang, Haichao; Humerickhouse, Rod A.; Rosenberg, Saul H.; Elmore, Steven W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">202-208</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteins in the B cell CLL/lymphoma 2 (BCL-2) family are key regulators of the apoptotic process.  This family comprises proapoptotic and prosurvival proteins, and shifting the balance toward the latter is an established mechanism whereby cancer cells evade apoptosis.  The therapeutic potential of directly inhibiting prosurvival proteins was unveiled with the development of navitoclax, a selective inhibitor of both BCL-2 and BCL-2-like 1 (BCL-XL), which has shown clin. efficacy in some BCL-2-dependent hematol. cancers.  However, concomitant on-target thrombocytopenia caused by BCL-XL inhibition limits the efficacy achievable with this agent.  Here we report the re-engineering of navitoclax to create a highly potent, orally bioavailable and BCL-2-selective inhibitor, ABT-199.  This compd. inhibits the growth of BCL-2-dependent tumors in vivo and spares human platelets.  A single dose of ABT-199 in three patients with refractory chronic lymphocytic leukemia resulted in tumor lysis within 24 h.  These data indicate that selective pharmacol. inhibition of BCL-2 shows promise for the treatment of BCL-2-dependent hematol. cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi7P7koWwJHLVg90H21EOLACvtfcHk0lh2r1gXaWa7Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslagtQ%253D%253D&md5=6a24b72303eb2062da653f1f61ac74fb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnm.3048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3048%26sid%3Dliteratum%253Aachs%26aulast%3DSouers%26aufirst%3DA.%2BJ.%26aulast%3DLeverson%26aufirst%3DJ.%2BD.%26aulast%3DBoghaert%26aufirst%3DE.%2BR.%26aulast%3DAckler%26aufirst%3DS.%2BL.%26aulast%3DCatron%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DDayton%26aufirst%3DB.%2BD.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DEnschede%26aufirst%3DS.%2BH.%26aulast%3DFairbrother%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DHymowitz%26aufirst%3DS.%2BG.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DKhaw%26aufirst%3DS.%2BL.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DMaecker%26aufirst%3DH.%2BL.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DMason%26aufirst%3DK.%2BD.%26aulast%3DMitten%26aufirst%3DM.%2BJ.%26aulast%3DNimmer%26aufirst%3DP.%2BM.%26aulast%3DOleksijew%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DC.-M.%26aulast%3DPhillips%26aufirst%3DD.%2BC.%26aulast%3DRoberts%26aufirst%3DA.%2BW.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSeymour%26aufirst%3DJ.%2BF.%26aulast%3DSmith%26aufirst%3DM.%2BL.%26aulast%3DSullivan%26aufirst%3DG.%2BM.%26aulast%3DTahir%26aufirst%3DS.%2BK.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXue%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHumerickhouse%26aufirst%3DR.%2BA.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3DABT-199%252C%2520a%2520Potent%2520and%2520Selective%2520BCL-2%2520Inhibitor%252C%2520Achieves%2520Antitumor%2520Activity%2520While%2520Sparing%2520Platelets%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D202%26epage%3D208%26doi%3D10.1038%2Fnm.3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty Years on: The Impact of Fragments on Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+Years+on%3A+The+Impact+of+Fragments+on+Drug+Discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0ljSDD_fdiVANQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520Years%2520on%253A%2520The%2520Impact%2520of%2520Fragments%2520on%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jencks, W. P.</span></span> <span> </span><span class="NLM_article-title">On the Attribution and Additivity of Binding Energies</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">4046</span>– <span class="NLM_lpage">4050</span>, <span class="refDoi"> DOI: 10.1073/pnas.78.7.4046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.78.7.4046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=16593049" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1981&pages=4046-4050&author=W.+P.+Jencks&title=On+the+Attribution+and+Additivity+of+Binding+Energies&doi=10.1073%2Fpnas.78.7.4046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">On the attribution and additivity of binding energies</span></div><div class="casAuthors">Jencks, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4046-50</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">It can be useful to describe the Gibbs free energy changes for the binding to a protein of a mol., A-B, and of its component parts, A and B, in terms of the intrinsic binding energies of A and B, ΔGiA and ΔGiB, and a connection Gibbs energy, ΔGs, that is derived largely from changes in translational and rotational entropy.  The relation between ΔGsand ΔGiA and ΔGiB is discussed.  This empirical approach avoids the difficult of interpreting obsd. ΔH and TΔS values for aq. solns.  The ΔGi and ΔGsterms can be large for binding to enzymes and other proteins.  The use and interpretation of these binding energies are discussed, esp. with respect to ATP interaction with myosin ATPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2X3Thp5VESLVg90H21EOLACvtfcHk0ljSDD_fdiVANQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXlvVamtbo%253D&md5=1cdbbb8b6e3f9d3ff0ba2c662c772375</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.78.7.4046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.78.7.4046%26sid%3Dliteratum%253Aachs%26aulast%3DJencks%26aufirst%3DW.%2BP.%26atitle%3DOn%2520the%2520Attribution%2520and%2520Additivity%2520of%2520Binding%2520Energies%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1981%26volume%3D78%26spage%3D4046%26epage%3D4050%26doi%3D10.1073%2Fpnas.78.7.4046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, G.</span></span> <span> </span><span class="NLM_article-title">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">856</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+Complexity+and+Its+Impact+on+the+Probability+of+Finding+Leads+for+Drug+Discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0ljSDD_fdiVANQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520Complexity%2520and%2520Its%2520Impact%2520on%2520the%2520Probability%2520of%2520Finding%2520Leads%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, J.-L.</span></span> <span> </span><span class="NLM_article-title">The Chemical Space Project</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1021/ar500432k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar500432k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=722-730&author=J.-L.+Reymond&title=The+Chemical+Space+Project&doi=10.1021%2Far500432k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemical Space Project</span></div><div class="casAuthors">Reymond, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-730</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the simplest questions that can be asked about mol. diversity is how many org. mols. are possible in total.  To answer this question, my research group has computationally enumerated all possible org. mols. up to a certain size to gain an unbiased insight into the entire chem. space.  Our latest database, GDB-17, contains 166.4 billion mols. of up to 17 atoms of C, N, O, S, and halogens, by far the largest small mol. database reported to date.  Mols. allowed by valency rules but unstable or nonsynthesizable due to strained topologies or reactive functional groups were not considered, which reduced the enumeration by at least 10 orders of magnitude and was essential to arrive at a manageable database size.  Despite these restrictions, GDB-17 is highly relevant with respect to known mols.  Beyond enumeration, understanding and exploiting GDBs (generated databases) led us to develop methods for virtual screening and visualization of very large databases in the form of a "periodic system of mols." comprising six different fingerprint spaces, with web-browsers for nearest neighbor searches, and the MQN- and SMIfp-Mapplet application for exploring color-coded principal component maps of GDB and other large databases.  Proof-of-concept applications of GDB for drug discovery were realized by combining virtual screening with chem. synthesis and activity testing for neurotransmitter receptor and transporter ligands.  One surprising lesson from using GDB for drug analog searches is the incredible depth of chem. space, i.e., the fact that millions of very close analogs of any mol. can be readily identified by nearest-neighbor searches in the MQN-space of the various GDBs.  The chem. space project has opened an unprecedented door on chem. diversity.  Ongoing and yet unmet challenges concern enumerating mols. beyond 17 atoms and synthesizing GDB mols. with innovative scaffolds and pharmacophores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxJYiz7ZE3ALVg90H21EOLACvtfcHk0liKMWxuizZPpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFWisb4%253D&md5=d3199b2273a0e8d92a4b1ffa59c6dffc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Far500432k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far500432k%26sid%3Dliteratum%253Aachs%26aulast%3DReymond%26aufirst%3DJ.-L.%26atitle%3DThe%2520Chemical%2520Space%2520Project%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2015%26volume%3D48%26spage%3D722%26epage%3D730%26doi%3D10.1021%2Far500432k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schuffenhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruedisser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span> <span> </span><span class="NLM_article-title">Library Design for Fragment Based Screening</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.2174/1568026054637700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.2174%2F1568026054637700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=16101415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=751-762&author=A.+Schuffenhauerauthor=S.+Ruedisserauthor=A.+L.+Marzinzikauthor=W.+Jahnkeauthor=M.+Blommersauthor=P.+Selzerauthor=E.+Jacoby&title=Library+Design+for+Fragment+Based+Screening&doi=10.2174%2F1568026054637700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Library design for fragment based screening</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ruedisser, Simon; Marzinzik, Andreas L.; Jahnke, Wolfgang; Blommers, Marcel; Selzer, Paul; Jacoby, Edgar</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">751-762</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  According to Hann's model of mol. complexity an increased probability of detection binding to a target protein can be expected when small, low complex mol. fragments are screened with high sensitivity instead of full-sized ligands with lower sensitivity.  Anal. of the HTS summary data of Novartis and comparison with NMR screening results obtained on generic fragment libraries indicate this expectation to be true with hitrates of 0.001% - 0.151% obsd. in the identification of ligands with an IC50 threshold in the micromolar range in an HTS setup and hitrates above or equal to 3% obsd. in NMR screening of fragments with an affinity threshold in the millimolar range.  It is however necessary to keep in mind that the sets of target studied were not identical for both method and the experience in NMR screening is too limited for a final conclusion.  The term hitrate as used here reflects only the success rate in the observation of ligand binding event.  It must not be confused with the overall success rate of fragment and high throughput screening in the lead finding process, which can be entirely different, since the steps required to follow-up a ligand binding event to a lead are different for both methods.  A survey of fragment-based lead discovery case studies given in the literature shows that in approx. half of the cases the initial hit fragment was discovered by screening a generic library, whereas in the other cases some knowledge about an initial ligands or the protein binding site has been used, whereas systematic virtual screening of fragment databases has been only rarely reported.  As comparatively high hitrates were obtained, further consideration to optimize the generic fragment screening library were directed to the chem. tractability of the fragment.  As several functional groups preferred by chemists for modification and linking of the fragments are also preferentially involved in interactions between the fragments and the target protein, a set of screening fragments was derived from chem. building blocks by masking its linker group by a chem. transformation which can be later on used in the chem. follow-up of the fragment hit.  For example primary amines can be masked as acetamides.  If the screening fragment is active the related building block can then be used for synthesis of a follow-up library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCOn-N__v0t7Vg90H21EOLACvtfcHk0liKMWxuizZPpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht78%253D&md5=7fd149986a1e2472be22bcd4e142b47c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1568026054637700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637700%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DRuedisser%26aufirst%3DS.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBlommers%26aufirst%3DM.%26aulast%3DSelzer%26aufirst%3DP.%26aulast%3DJacoby%26aufirst%3DE.%26atitle%3DLibrary%2520Design%2520for%2520Fragment%2520Based%2520Screening%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D751%26epage%3D762%26doi%3D10.2174%2F1568026054637700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrmann, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1701</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1693-1701&author=J.+Schiebelauthor=N.+Radevaauthor=S.+G.+Krimmerauthor=X.+Wangauthor=M.+Stielerauthor=F.+R.+Ehrmannauthor=K.+Fuauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=A.+Heineauthor=G.+Klebe&title=Six+Biophysical+Screening+Methods+Miss+a+Large+Proportion+of+Crystallographically+Discovered+Fragment+Hits%3A+A+Case+Study&doi=10.1021%2Facschembio.5b01034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study</span></div><div class="casAuthors">Schiebel, Johannes; Radeva, Nedyalka; Krimmer, Stefan G.; Wang, Xiaojie; Stieler, Martin; Ehrmann, Frederik R.; Fu, Kan; Metz, Alexander; Huschmann, Franziska U.; Weiss, Manfred S.; Mueller, Uwe; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1693-1701</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) has become a pillar in drug development.  Typical applications of this method comprise at least two biophys. screens as prefilter and a follow-up crystallog. expt. on a subset of fragments.  Clearly, structural information is pivotal in FBLD, but a key question is whether such a screening cascade strategy will retrieve the majority of fragment-bound structures.  We therefore set out to screen 361 fragments for binding to endothiapepsin, a representative of the challenging group of aspartic proteases, employing six screening techniques and crystallog. in parallel.  Crystallog. resulted in the very high no. of 71 structures.  Yet alarmingly, 44% of these hits were not detected by any biophys. screening approach.  Moreover, any screening cascade, building on the results from two or more screening methods, would have failed to predict at least 73% of these hits.  We thus conclude that, at least in the present case, the frequently applied biophys. prescreening filters deteriorate the no. of possible X-ray hits while only the immediate use of crystallog. enables exhaustive retrieval of a max. of fragment structures, which represent a rich source guiding hit-to-lead-to-drug evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgOpXVyCfeiLVg90H21EOLACvtfcHk0liKMWxuizZPpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D&md5=30f16cf75361d0ca11e7b75ad796d207</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01034%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSix%2520Biophysical%2520Screening%2520Methods%2520Miss%2520a%2520Large%2520Proportion%2520of%2520Crystallographically%2520Discovered%2520Fragment%2520Hits%253A%2520A%2520Case%2520Study%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1693%26epage%3D1701%26doi%3D10.1021%2Facschembio.5b01034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Structural Biology in Fragment-Based Drug Design</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.sbi.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=20471246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=497-507&author=C.+W.+Murrayauthor=T.+L.+Blundell&title=Structural+Biology+in+Fragment-Based+Drug+Design&doi=10.1016%2Fj.sbi.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology in fragment-based drug design</span></div><div class="casAuthors">Murray, Christopher W.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based ligand screening is now established as an emerging paradigm for drug discovery.  Here we examine the recent literature looking at how structural biol. has been used in a variety of successful fragment-screening applications.  We argue that the detn. of exptl. binding modes has proved to be one of the mainstays of successful fragment-based approaches and that this reflects the difficulty in optimizing a fragment to a lead mol. in the absence of structural information.  We focus on antimicrobial research where fragment-based drug discovery allows control of the phys. properties of the emerging lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1S2lx8J-FX7Vg90H21EOLACvtfcHk0liKMWxuizZPpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF&md5=ae8cfc0f8be47a316bd92073e95e0c8a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520Biology%2520in%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D497%26epage%3D507%26doi%3D10.1016%2Fj.sbi.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2015</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01123</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01123" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1KlsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=89-99&author=C.+N.+Johnsonauthor=D.+A.+Erlansonauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2015&doi=10.1021%2Facs.jmedchem.6b01123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-to-Lead Medicinal Chemistry Publications in 2015</span></div><div class="casAuthors">Johnson, Christopher N.; Erlanson, Daniel A.; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-99</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based drug discovery (FBDD) is now well-established as a technol. for generating new chem. leads and drugs.  This Miniperspective provides a tabulated overview of the fragment-to-lead literature published in the year 2015, together with a commentary on trends obsd. across the FBDD field during this time.  It is hoped that this tabulated summary will provide a useful point of ref. for both FBDD practitioners and the wider medicinal chem. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojX6ZlmVXQrLVg90H21EOLACvtfcHk0lgw-qsUL7GAvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1KlsbnP&md5=da9d027787d70d4276378cb01f8e0ce2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01123%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202015%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D89%26epage%3D99%26doi%3D10.1021%2Facs.jmedchem.6b01123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2016</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1774</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslentrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1774-1784&author=C.+N.+Johnsonauthor=D.+A.+Erlansonauthor=W.+Jahnkeauthor=P.+N.+Mortensonauthor=D.+C.+Rees&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2016&doi=10.1021%2Facs.jmedchem.7b01298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-to-Lead Medicinal Chemistry Publications in 2016</span></div><div class="casAuthors">Johnson, Christopher N.; Erlanson, Daniel A.; Jahnke, Wolfgang; Mortenson, Paul N.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1774-1784</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The popularity of fragment-based drug discovery (FBDD) is demonstrated by the no. of recent successful fragment-to-lead (F2L) publications.  This Miniperspective provides a tabulated summary of the F2L literature published in the year 2016, along with discussion of general trends.  It uses the same format as our summary of the 2015 literature and is intended to be a resource for both FBDD practitioners and medicinal chemists in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDiwnepXDgrbVg90H21EOLACvtfcHk0lgw-qsUL7GAvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslentrfP&md5=861d9adf1530250a060fc254ca7b493c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01298%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202016%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1774%26epage%3D1784%26doi%3D10.1021%2Facs.jmedchem.7b01298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Esch, I. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span> <span> </span><span class="NLM_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=P.+N.+Mortensonauthor=D.+A.+Erlansonauthor=I.+J.+P.+de+Eschauthor=W.+Jahnkeauthor=C.+N.+Johnson&title=Fragment-to-Lead+Medicinal+Chemistry+Publications+in+2017&doi=10.1021%2Facs.jmedchem.8b01472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%2BP.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26atitle%3DFragment-to-Lead%2520Medicinal%2520Chemistry%2520Publications%2520in%25202017%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26doi%3D10.1021%2Facs.jmedchem.8b01472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhYVC2vzy0-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deller, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupp, B.</span></span> <span> </span><span class="NLM_article-title">Models of Protein-Ligand Crystal Structures: Trust, but Verify</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">817</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9833-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1007%2Fs10822-015-9833-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25665575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisFektb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=817-836&author=M.+C.+Dellerauthor=B.+Rupp&title=Models+of+Protein-Ligand+Crystal+Structures%3A+Trust%2C+but+Verify&doi=10.1007%2Fs10822-015-9833-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Models of protein-ligand crystal structures: trust, but verify</span></div><div class="casAuthors">Deller, Marc C.; Rupp, Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">817-836</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">X-ray crystallog. provides the most accurate models of protein-ligand structures.  These models serve as the foundation of many computational methods including structure prediction, mol. modeling, and structure-based drug design.  The success of these computational methods ultimately depends on the quality of the underlying protein-ligand models.  X-ray crystallog. offers the unparalleled advantage of a clear math. formalism relating the exptl. data to the protein-ligand model.  In the case of X-ray crystallog., the primary exptl. evidence is the electron d. of the mols. forming the crystal.  The first step in the generation of an accurate and precise crystallog. model is the interpretation of the electron d. of the crystal, typically carried out by construction of an at. model.  The at. model must then be validated for fit to the exptl. electron d. and also for agreement with prior expectations of stereochem.  Stringent validation of protein-ligand models has become possible as a result of the mandatory deposition of primary diffraction data, and many computational tools are now available to aid in the validation process.  Validation of protein-ligand complexes has revealed some instances of overenthusiastic interpretation of ligand d.  Fundamental concepts and metrics of protein-ligand quality validation are discussed and we highlight software tools to assist in this process.  It is essential that end users select high quality protein-ligand models for their computational and biol. studies, and we provide an overview of how this can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnJmfgZbDk2bVg90H21EOLACvtfcHk0lhYVC2vzy0-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisFektb8%253D&md5=aec55797c308266a49bd34aff9b95bbd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9833-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9833-8%26sid%3Dliteratum%253Aachs%26aulast%3DDeller%26aufirst%3DM.%2BC.%26aulast%3DRupp%26aufirst%3DB.%26atitle%3DModels%2520of%2520Protein-Ligand%2520Crystal%2520Structures%253A%2520Trust%252C%2520but%2520Verify%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D817%26epage%3D836%26doi%3D10.1007%2Fs10822-015-9833-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">Protonate3D: Assignment of Ionization States and Hydrogen Coordinates to Macromolecular Structures</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">187</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.1002/prot.22234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fprot.22234" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=18814299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2009&pages=187-205&author=P.+Labute&title=Protonate3D%3A+Assignment+of+Ionization+States+and+Hydrogen+Coordinates+to+Macromolecular+Structures&doi=10.1002%2Fprot.22234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protonate3D: assignment of ionization states and hydrogen coordinates to macromolecular structures</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">187-205</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A new method, called Protonate3D, is presented for the automated prediction of hydrogen coordinates given the 3D coordinates of the heavy atoms of a macromol. structure.  Protonate3D considers side-chain "flip," rotamer, tautomer, and ionization states of all chem. groups, ligands, and solvent, provided suitable templates are available in a parameter file.  The energy model includes van der Waals, Coulomb, solvation, rotamer, tautomer, and titrn. effects.  The results of computational validation expts. suggest that Protonate3D can accurately predict the location of hydrogen atoms in macromol. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIRzxpxpCrVg90H21EOLACvtfcHk0lhYVC2vzy0-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisV2isrs%253D&md5=89803fe2028fb9257eb09854a27ef2dd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fprot.22234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22234%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DProtonate3D%253A%2520Assignment%2520of%2520Ionization%2520States%2520and%2520Hydrogen%2520Coordinates%2520to%2520Macromolecular%2520Structures%26jtitle%3DProteins%26date%3D2009%26volume%3D75%26spage%3D187%26epage%3D205%26doi%3D10.1002%2Fprot.22234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bissantz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">A Medicinal Chemist’s Guide to Molecular Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5061</span>– <span class="NLM_lpage">5084</span>, <span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+Medicinal+Chemist%E2%80%99s+Guide+to+Molecular+Interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lhYVC2vzy0-mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Guide%2520to%2520Molecular%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span> <i>JChem</i>, Version 16.4.25; <span class="NLM_publisher-name">ChemAxon</span>, <span class="NLM_year">2016</span>. <a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+JChem%2C+Version+16.4.25%3B+ChemAxon%2C+2016.+http%3A%2F%2Fwww.chemaxon.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DJChem%26pub%3DChemAxon%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carugo, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordo, D.</span></span> <span> </span><span class="NLM_article-title">How Many Water Molecules Can Be Detected by Protein Crystallography?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">479</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1107/S0907444998012086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1107%2FS0907444998012086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10089359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFGnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1999&pages=479-483&author=O.+Carugoauthor=D.+Bordo&title=How+Many+Water+Molecules+Can+Be+Detected+by+Protein+Crystallography%3F&doi=10.1107%2FS0907444998012086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How many water molecules can be detected by protein crystallography?</span></div><div class="casAuthors">Carugo, Oliviero; Bordo, Domenico</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">D55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">479-483</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The no. of water mols. which are expected to be exptl. located by protein crystallog. was detd. by multiple regression anal. on a test set of 873 known protein crystal structures detd. at room temp. and on another set of 33 structures detd. at low temp.  The dependence of the no. of water mols. included in the protein models as a function of a no. of significant regressors, such as resoln., fraction of crystal vol. occupied by the solvent, no. of residues in the asym. unit, fraction of apolar protein surface or secondary structure, has been studied.  The no. of water mols. included in crystallog. models depends primarily on the resoln. at which the structure has been solved, while the temp. of the data collection has only marginal influence.  On av., at 2.0 Å resoln. one water mol. per residue is included in the model, while at 1.0 Å resoln. about 1.6-1.7 are crystallog. located.  At 2.0 Å resoln. the well known rule-of-thumb of "one water per protein residue" is confirmed, though the no. of water mols. exptl. obsd. is strongly dependent on resoln.  The results presented are useful in assessing the quality of a protein crystal structure, in selecting structural results to be compared and in evaluating the expected improvement on the solvent structure when increasing the crystallog. resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2Ji9xBF-6R7Vg90H21EOLACvtfcHk0liZZ1WRseb_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFGnsLc%253D&md5=c7321eca9f64b228b8a26264f77b1d53</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444998012086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998012086%26sid%3Dliteratum%253Aachs%26aulast%3DCarugo%26aufirst%3DO.%26aulast%3DBordo%26aufirst%3DD.%26atitle%3DHow%2520Many%2520Water%2520Molecules%2520Can%2520Be%2520Detected%2520by%2520Protein%2520Crystallography%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1999%26volume%3D55%26spage%3D479%26epage%3D483%26doi%3D10.1107%2FS0907444998012086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span> <i>Molecular Operating Environment (MOE)</i>,
2015.10; <span class="NLM_publisher-name">Chemical Computing Group, Inc.</span>: <span class="NLM_publisher-loc">1010
Sherbooke St. West, Suite #910, Montreal, QC, Canada, H3A 2R7</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C%0A2015.10%3B+Chemical+Computing+Group%2C+Inc.%3A+1010%0ASherbooke+St.+West%2C+Suite+%23910%2C+Montreal%2C+QC%2C+Canada%2C+H3A+2R7%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%252C%2520Inc%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0liZZ1WRseb_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrauskas, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolovanov, E. A.</span></span> <span> </span><span class="NLM_article-title">ACD/Log P Method Description</span>. <i>Perspect. Drug Discovery Des.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">116</span>, <span class="refDoi"> DOI: 10.1023/A:1008719622770</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1023%2FA%3A1008719622770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnslaitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=99-116&author=A.+A.+Petrauskasauthor=E.+A.+Kolovanov&title=ACD%2FLog+P+Method+Description&doi=10.1023%2FA%3A1008719622770"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">ACD/Log P method description</span></div><div class="casAuthors">Petrauskas, Alanas A.; Kolovanov, Eduard A.</div><div class="citationInfo"><span class="NLM_cas:title">Perspectives in Drug Discovery and Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">Hydrophobicity and Solvation in Drug Design, Pt. 3</span>),
    <span class="NLM_cas:pages">99-116</span>CODEN:
                <span class="NLM_cas:coden">PDDDEC</span>;
        ISSN:<span class="NLM_cas:issn">0928-2866</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">This study describes the development of the ACD/Log P calcn. method.  Anal. of 14 calcn. methods revealed that the most accurate calcns. are obtained when correction factors are used.  The authors evaluated the correction factors used by Hansch and Leo in CLOGP to simplify their method.  Most of the CLOGP structural factors are included in fragmental increments.  Aliph. and arom. factors are replaced with additive interfragmental increments.  Missing increments are estd. by two empirical equations with simple phys. interpretation.  The final method uses three simple equations with several types of parameters.  The training set included 3601 compds. and the correlation between exptl. and calcd. Log P values gave R = 0.992, S = 0.21.  The method was validated by comparing it with 17 other methods on various data sets of independently selected drugs and other compds.  In all cases, method produced the best results.  The weakness of this method is that it uses a large no. of individual increments for arom. interactions.  Each increment represents a combination of several effects which presently cannot be sepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0Ih18c8CWcrVg90H21EOLACvtfcHk0liZZ1WRseb_sQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnslaitbc%253D&md5=9980bde748cf510386050ba5369446aa</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008719622770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008719622770%26sid%3Dliteratum%253Aachs%26aulast%3DPetrauskas%26aufirst%3DA.%2BA.%26aulast%3DKolovanov%26aufirst%3DE.%2BA.%26atitle%3DACD%252FLog%2520P%2520Method%2520Description%26jtitle%3DPerspect.%2520Drug%2520Discovery%2520Des.%26date%3D2000%26volume%3D19%26spage%3D99%26epage%3D116%26doi%3D10.1023%2FA%3A1008719622770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span> <i>ACD Percepta Batch,
2012 Release</i>; <span class="NLM_publisher-name">Advanced Chemistry Development,
Inc.</span>: <span class="NLM_publisher-loc">8 King Street East, Suite 107,
Toronto, Ontario, M5C 1B5, Canada</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ACD+Percepta+Batch%2C%0A2012+Release%3B+Advanced+Chemistry+Development%2C%0AInc.%3A+8+King+Street+East%2C+Suite+107%2C%0AToronto%2C+Ontario%2C+M5C+1B5%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACD%2520Percepta%2520Batch%252C%250A2012%2520Release%26pub%3DAdvanced%2520Chemistry%2520Development%252C%250AInc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span> <i>Vortex</i>, 2016.08; <span class="NLM_publisher-name">Dotmatics Limited</span>: <span class="NLM_publisher-loc">The Old Monastery, Windhill, Bishops Stortford,
Herts CM23 2ND UK</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Vortex%2C+2016.08%3B+Dotmatics+Limited%3A+The+Old+Monastery%2C+Windhill%2C+Bishops+Stortford%2C%0AHerts+CM23+2ND+UK%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DVortex%26pub%3DDotmatics%2520Limited%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Guilloux, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuffery, P.</span></span> <span> </span><span class="NLM_article-title">Fpocket: An Open Source Platform for Ligand Pocket Detection</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">168</span> <span class="refDoi"> DOI: 10.1186/1471-2105-10-168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1186%2F1471-2105-10-168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=19486540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BD1MvjsFWltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&author=V.+Le%0AGuillouxauthor=P.+Schmidtkeauthor=P.+Tuffery&title=Fpocket%3A+An+Open+Source+Platform+for+Ligand+Pocket+Detection&doi=10.1186%2F1471-2105-10-168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fpocket: an open source platform for ligand pocket detection</span></div><div class="casAuthors">Le Guilloux Vincent; Schmidtke Peter; Tuffery Pierre</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">168</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Virtual screening methods start to be well established as effective approaches to identify hits, candidates and leads for drug discovery research.  Among those, structure based virtual screening (SBVS) approaches aim at docking collections of small compounds in the target structure to identify potent compounds.  For SBVS, the identification of candidate pockets in protein structures is a key feature, and the recent years have seen increasing interest in developing methods for pocket and cavity detection on protein surfaces.  RESULTS:  Fpocket is an open source pocket detection package based on Voronoi tessellation and alpha spheres built on top of the publicly available package Qhull.  The modular source code is organised around a central library of functions, a basis for three main programs: (i) Fpocket, to perform pocket identification, (ii) Tpocket, to organise pocket detection benchmarking on a set of known protein-ligand complexes, and (iii) Dpocket, to collect pocket descriptor values on a set of proteins.  Fpocket is written in the C programming language, which makes it a platform well suited for the scientific community willing to develop new scoring functions and extract various pocket descriptors on a large scale level.  Fpocket 1.0, relying on a simple scoring function, is able to detect 94% and 92% of the pockets within the best three ranked pockets from the holo and apo proteins respectively, outperforming the standards of the field, while being faster.  CONCLUSION:  Fpocket provides a rapid, open source and stable basis for further developments related to protein pocket detection, efficient pocket descriptor extraction, or drugablity prediction purposes.  Fpocket is freely available under the GNU GPL license at http://fpocket.sourceforge.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRTfW9d1-8T8XWEEzDrymxmfW6udTcc2eYLoCK_-dU3Bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvjsFWltw%253D%253D&md5=7d16f53ed64eac9cdeb33ea40b61adfd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-10-168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-10-168%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BGuilloux%26aufirst%3DV.%26aulast%3DSchmidtke%26aufirst%3DP.%26aulast%3DTuffery%26aufirst%3DP.%26atitle%3DFpocket%253A%2520An%2520Open%2520Source%2520Platform%2520for%2520Ligand%2520Pocket%2520Detection%26jtitle%3DBMC%2520Bioinf.%26date%3D2009%26volume%3D10%26doi%3D10.1186%2F1471-2105-10-168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+Molecular+Force+Field.+I.+Basis%2C+Form%2C+Scope%2C+Parameterization%2C+and+Performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Å root mean square [rms]), bond angles (1.2° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0liJwtanoGscvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520Molecular%2520Force%2520Field.%2520I.%2520Basis%252C%2520Form%252C%2520Scope%252C%2520Parameterization%252C%2520and%2520Performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronov, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meireles, L.</span></span> <span> </span><span class="NLM_article-title">Prediction of Protein Pairs Sharing Common Active Ligands Using Protein Sequence, Structure, and Ligand Similarity</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1734</span>– <span class="NLM_lpage">1745</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSqt7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1734-1745&author=Y.+C.+Chenauthor=R.+Tolbertauthor=A.+M.+Aronovauthor=G.+McGaugheyauthor=W.+P.+Waltersauthor=L.+Meireles&title=Prediction+of+Protein+Pairs+Sharing+Common+Active+Ligands+Using+Protein+Sequence%2C+Structure%2C+and+Ligand+Similarity&doi=10.1021%2Facs.jcim.6b00118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Protein Pairs Sharing Common Active Ligands Using Protein Sequence, Structure, and Ligand Similarity</span></div><div class="casAuthors">Chen, Yu-Chen; Tolbert, Robert; Aronov, Alex M.; McGaughey, Georgia; Walters, W. Patrick; Meireles, Lidio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1734-1745</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors benchmarked the ability of comparative computational approaches to correctly discriminate protein pairs sharing a common active ligand (pos. protein pairs) from protein pairs with no common active ligands (neg. protein pairs).  Since the target and the off-targets of a drug share at least a common ligand, i.e. the drug itself, the prediction of pos. protein pairs may help identify off-targets.  The authors evaluated representative protein-centric and ligand centric approaches, including (1) 2D and 3D ligand similarity, (2) several measures of protein sequence similarity in conjunction with different sequence sources (e.g., full protein sequence vs. binding site residues), and( 3) a newly described pocket shape similarity and alignment program called SiteHopper.  While the sequence-based alignment of pocket residues achieved the best overall performance, SiteHopper outperformed sequence-based approaches for unrelated proteins with only 20-30% pocket residue identity.  Analogously, among ligand-centric approaches, path-based fingerprints achieved the best overall performance, but ROCS-based ligand shape similarity outperformed path-based fingerprints for structurally dissimilar ligands (Tanimoto 25%-40%).  A significant drop in recognition performance was obsd. for ligand-centric approaches when PDB ligands were used instead of ChEMBL ligands.  Finally, the authors analyzed the relation between pocket shape and ligand shape in the authors' dataset, and found that similar ligands tend to bind to similar pockets, while similar pockets may accept a range of different shape ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgs65nWQ5rdLVg90H21EOLACvtfcHk0liJwtanoGscvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSqt7zF&md5=0bdc84d1c3596d99d976ce5b3548cb62</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00118%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BC.%26aulast%3DTolbert%26aufirst%3DR.%26aulast%3DAronov%26aufirst%3DA.%2BM.%26aulast%3DMcGaughey%26aufirst%3DG.%26aulast%3DWalters%26aufirst%3DW.%2BP.%26aulast%3DMeireles%26aufirst%3DL.%26atitle%3DPrediction%2520of%2520Protein%2520Pairs%2520Sharing%2520Common%2520Active%2520Ligands%2520Using%2520Protein%2520Sequence%252C%2520Structure%252C%2520and%2520Ligand%2520Similarity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1734%26epage%3D1745%26doi%3D10.1021%2Facs.jcim.6b00118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The%0AUniProt+Consortium&title=UniProt%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radeva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrmann, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huschmann, F. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Experimental Active-Site Mapping by Fragments: Hot Spots Remote from the Catalytic Center of Endothiapepsin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7561</span>– <span class="NLM_lpage">7575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7561-7575&author=N.+Radevaauthor=S.+G.+Krimmerauthor=M.+Stielerauthor=K.+Fuauthor=X.+Wangauthor=F.+R.+Ehrmannauthor=A.+Metzauthor=F.+U.+Huschmannauthor=M.+S.+Weissauthor=U.+Muellerauthor=J.+Schiebelauthor=A.+Heineauthor=G.+Klebe&title=Experimental+Active-Site+Mapping+by+Fragments%3A+Hot+Spots+Remote+from+the+Catalytic+Center+of+Endothiapepsin&doi=10.1021%2Facs.jmedchem.6b00645"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00645%26sid%3Dliteratum%253Aachs%26aulast%3DRadeva%26aufirst%3DN.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DStieler%26aufirst%3DM.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DEhrmann%26aufirst%3DF.%2BR.%26aulast%3DMetz%26aufirst%3DA.%26aulast%3DHuschmann%26aufirst%3DF.%2BU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DExperimental%2520Active-Site%2520Mapping%2520by%2520Fragments%253A%2520Hot%2520Spots%2520Remote%2520from%2520the%2520Catalytic%2520Center%2520of%2520Endothiapepsin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7561%26epage%3D7575%26doi%3D10.1021%2Facs.jmedchem.6b00645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köster, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brass, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herhaus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentgraf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">A Small Nonrule of 3 Compatible Fragment Library Provides High Hit Rate of Endothiapepsin Crystal Structures with Various Fragment Chemotypes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">7784</span>– <span class="NLM_lpage">7796</span>, <span class="refDoi"> DOI: 10.1021/jm200642w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200642w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FgtVyhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7784-7796&author=H.+K%C3%B6sterauthor=T.+Craanauthor=S.+Brassauthor=C.+Herhausauthor=M.+Zentgrafauthor=L.+Neumannauthor=A.+Heineauthor=G.+Klebe&title=A+Small+Nonrule+of+3+Compatible+Fragment+Library+Provides+High+Hit+Rate+of+Endothiapepsin+Crystal+Structures+with+Various+Fragment+Chemotypes&doi=10.1021%2Fjm200642w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A small nonrule of 3 compatible fragment library provides high hit rate of endothiapepsin crystal structures with various fragment chemotypes</span></div><div class="casAuthors">Koster Helene; Craan Tobias; Brass Sascha; Herhaus Christian; Zentgraf Matthias; Neumann Lars; Heine Andreas; Klebe Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7784-96</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Druglike molecules are defined by Lipinski's rule of 5, to characterize fragment thresholds, they have been reduced from 5 to 3 (Astex's rule of 3).  They are applied to assemble fragment libraries, and providers use them to select fragments for commercial offer.  We question whether these rules are too stringent to compose fragment libraries with candidates exhibiting sufficient room for chemical subsequent growing and merging modifications as appropriate functional groups for chemical transformations are required.  Usually these groups exhibit properties as hydrogen bond donors/acceptors and provide entry points for optimization chemistry.  We therefore designed a fragment library (364 entries) without strictly applying the rule of 3.  For initial screening for endothiapepsin binding, we performed a biochemical cleavage assay of a fluorogenic substrate at 1 mM. "Hits" were defined to inhibit the enzyme by at least 40%.  Fifty-five hits were suggested and subsequently soaked into endothiapepsin crystals.  Eleven crystal structures could be determined covering fragments with diverse binding modes: (i) direct binding to the catalytic dyad aspartates, (ii) water-mediated binding to the aspartates, (iii) no direct interaction with the dyad.  They occupy different specificity pockets.  Only 4 of the 11 fragments are consistent with the rule of 3.  Restriction to this rule would have limited the fragment hits to a strongly reduced variety of chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuxdu_QoLSWZotVjB3IRIyfW6udTcc2eamLFQkAqpMprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FgtVyhtQ%253D%253D&md5=bb86da923d5251f50e4aa20bf04c05af</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm200642w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200642w%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6ster%26aufirst%3DH.%26aulast%3DCraan%26aufirst%3DT.%26aulast%3DBrass%26aufirst%3DS.%26aulast%3DHerhaus%26aufirst%3DC.%26aulast%3DZentgraf%26aufirst%3DM.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520Small%2520Nonrule%2520of%25203%2520Compatible%2520Fragment%2520Library%2520Provides%2520High%2520Hit%2520Rate%2520of%2520Endothiapepsin%2520Crystal%2520Structures%2520with%2520Various%2520Fragment%2520Chemotypes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7784%26epage%3D7796%26doi%3D10.1021%2Fjm200642w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Recht, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bounaud, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Logan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chie-Leon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nienaber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, F. E.</span></span> <span> </span><span class="NLM_article-title">Identification and Optimization of PDE10A Inhibitors Using Fragment-Based Screening by Nanocalorimetry and X-Ray Crystallography</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1177/1087057113516493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1177%2F1087057113516493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=24375910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWgs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=497-507&author=M.+I.+Rechtauthor=V.+Sridharauthor=J.+Badgerauthor=P.-Y.+Bounaudauthor=C.+Loganauthor=B.+Chie-Leonauthor=V.+Nienaberauthor=F.+E.+Torres&title=Identification+and+Optimization+of+PDE10A+Inhibitors+Using+Fragment-Based+Screening+by+Nanocalorimetry+and+X-Ray+Crystallography&doi=10.1177%2F1087057113516493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and optimization of PDE10A inhibitors using fragment-based screening by nanocalorimetry and X-ray crystallography</span></div><div class="casAuthors">Recht, Michael I.; Sridhar, Vandana; Badger, John; Bounaud, Pierre-Yves; Logan, Cheyenne; Chie-Leon, Barbara; Nienaber, Vicki; Torres, Francisco E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507, 11</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Fragment-based lead discovery (FBLD) is a technique in which small, low-complexity chem. fragments of 6 to 15 heavy atoms are screened for binding to or inhibiting activity of the target.  Hits are then linked and/or elaborated into tightly binding ligands, ideally yielding early lead compds. for drug discovery.  Calorimetry provides a label-free method to assay binding and enzymic activity that is unaffected by the spectroscopic properties of the sample.  Conventional microcalorimetry is hampered by requiring large quantities of reagents and long measurement times.  Nanocalorimeters can overcome these limitations of conventional isothermal titrn. calorimetry.  Here we use enthalpy arrays, which are arrays of nanocalorimeters, to perform an enzyme activity-based fragment screen for competitive inhibitors of phosphodiesterase 10A (PDE10A).  Two dozen fragments with KI <2 mM were identified and moved to crystal soaking trials.  All soak expts. yielded high-resoln. diffraction, with two-thirds of the fragments yielding high-resoln. co-crystal structures with PDE10A.  The structural information was used to elaborate fragment hits, yielding leads with KI <1 μM.  This study shows how array calorimetry can be used as a prescreening method for fragment-based lead discovery with enzyme targets and paired successfully with an X-ray crystallog. secondary screen.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjSq4QZ0ZQdLVg90H21EOLACvtfcHk0lg4eDEqGhiPOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWgs73E&md5=d63ee1e675ba628c75a0276c975e1fd0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1177%2F1087057113516493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113516493%26sid%3Dliteratum%253Aachs%26aulast%3DRecht%26aufirst%3DM.%2BI.%26aulast%3DSridhar%26aufirst%3DV.%26aulast%3DBadger%26aufirst%3DJ.%26aulast%3DBounaud%26aufirst%3DP.-Y.%26aulast%3DLogan%26aufirst%3DC.%26aulast%3DChie-Leon%26aufirst%3DB.%26aulast%3DNienaber%26aufirst%3DV.%26aulast%3DTorres%26aufirst%3DF.%2BE.%26atitle%3DIdentification%2520and%2520Optimization%2520of%2520PDE10A%2520Inhibitors%2520Using%2520Fragment-Based%2520Screening%2520by%2520Nanocalorimetry%2520and%2520X-Ray%2520Crystallography%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2014%26volume%3D19%26spage%3D497%26epage%3D507%26doi%3D10.1177%2F1087057113516493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shipe, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharik, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaughey, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theberge, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uslaner, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renger, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of a Series of Pyrimidine-Based Phosphodiesterase 10A (PDE10A) Inhibitors through Fragment Screening, Structure-Based Design, and Parallel Synthesis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7888</span>– <span class="NLM_lpage">7894</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00983</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00983" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWktL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7888-7894&author=W.+D.+Shipeauthor=S.+S.+Sharikauthor=J.+C.+Barrowauthor=G.+B.+McGaugheyauthor=C.+R.+Thebergeauthor=J.+M.+Uslanerauthor=Y.+Yanauthor=J.+J.+Rengerauthor=S.+M.+Smithauthor=P.+J.+Colemanauthor=C.+D.+Cox&title=Discovery+and+Optimization+of+a+Series+of+Pyrimidine-Based+Phosphodiesterase+10A+%28PDE10A%29+Inhibitors+through+Fragment+Screening%2C+Structure-Based+Design%2C+and+Parallel+Synthesis&doi=10.1021%2Facs.jmedchem.5b00983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of a Series of Pyrimidine-Based Phosphodiesterase 10A (PDE10A) Inhibitors through Fragment Screening, Structure-Based Design, and Parallel Synthesis</span></div><div class="casAuthors">Shipe, William D.; Sharik, Steven S.; Barrow, James C.; McGaughey, Georgia B.; Theberge, Cory R.; Uslaner, Jason M.; Yan, Youwei; Renger, John J.; Smith, Sean M.; Coleman, Paul J.; Cox, Christopher D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7888-7894</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of a fragment library for PDE10A inhibitors identified a low mol. wt. pyrimidine hit with PDE10A Ki of 8700 nM and LE of 0.59.  Initial optimization by catalog followed by iterative parallel synthesis guided by X-ray cocrystal structures resulted in rapid potency improvements with minimal loss of ligand efficiency.  Compd. 15h, with PDE10A Ki of 8.2 pM, LE of 0.49, and >5000-fold selectivity over other PDEs, fully attenuates MK-801-induced hyperlocomotor activity after i.p. dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTIq75ec_7MrVg90H21EOLACvtfcHk0lgGi3vnxs63Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWktL3F&md5=7deabe7e71abef929fee9a81d090e1b9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00983%26sid%3Dliteratum%253Aachs%26aulast%3DShipe%26aufirst%3DW.%2BD.%26aulast%3DSharik%26aufirst%3DS.%2BS.%26aulast%3DBarrow%26aufirst%3DJ.%2BC.%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DTheberge%26aufirst%3DC.%2BR.%26aulast%3DUslaner%26aufirst%3DJ.%2BM.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DRenger%26aufirst%3DJ.%2BJ.%26aulast%3DSmith%26aufirst%3DS.%2BM.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26aulast%3DCox%26aufirst%3DC.%2BD.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520a%2520Series%2520of%2520Pyrimidine-Based%2520Phosphodiesterase%252010A%2520%2528PDE10A%2529%2520Inhibitors%2520through%2520Fragment%2520Screening%252C%2520Structure-Based%2520Design%252C%2520and%2520Parallel%2520Synthesis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7888%26epage%3D7894%26doi%3D10.1021%2Facs.jmedchem.5b00983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coyle, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Downham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederickson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMenamin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodhead, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, A. J.-A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5955</span>, <span class="refDoi"> DOI: 10.1021/jm100059d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5942-5955&author=C.+W.+Murrayauthor=M.+G.+Carrauthor=O.+Callaghanauthor=G.+Chessariauthor=M.+Congreveauthor=S.+Cowanauthor=J.+E.+Coyleauthor=R.+Downhamauthor=E.+Figueroaauthor=M.+Fredericksonauthor=B.+Grahamauthor=R.+McMenaminauthor=M.+A.+O%E2%80%99Brienauthor=S.+Patelauthor=T.+R.+Phillipsauthor=G.+Williamsauthor=A.+J.+Woodheadauthor=A.+J.-A.+Woolford&title=Fragment-Based+Drug+Discovery+Applied+to+Hsp90.+Discovery+of+Two+Lead+Series+with+High+Ligand+Efficiency&doi=10.1021%2Fjm100059d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span></div><div class="casAuthors">Murray, Christopher W.; Carr, Maria G.; Callaghan, Owen; Chessari, Gianni; Congreve, Miles; Cowan, Suzanna; Coyle, Joseph E.; Downham, Robert; Figueroa, Eva; Frederickson, Martyn; Graham, Brent; McMenamin, Rachel; O'Brien, M. Alistair; Patel, Sahil; Phillips, Theresa R.; Williams, Glyn; Woodhead, Andrew J.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5942-5955</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of the chaperone Hsp90 are potentially useful as chemotherapeutic agents in cancer.  This paper describes an application of fragment screening to Hsp90 using a combination of NMR and high throughput x-ray crystallog.  The screening identified an aminopyrimidine with affinity in the high micromolar range and subsequent structure-based design allowed its optimization into a low nanomolar series with good ligand efficiency.  A phenolic chemotype was also identified in fragment screening and was found to bind with affinity close to 1 mM.  This fragment was optimized using structure based design into a resorcinol lead which has subnanomolar affinity for Hsp90, excellent cell potency, and good ligand efficiency.  This fragment to lead campaign improved affinity for Hsp90 by over 1,000,000 fold with the addn. of only 6 heavy atoms.  The companion paper (DOI: 10.1021/jm100060b) describes how the resorcinol lead was optimized into a compd. that is now in clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISC3cw2labbVg90H21EOLACvtfcHk0lgGi3vnxs63Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D&md5=d1ad900aa5a45ba6ba3a5e9ccdcd96b3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm100059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100059d%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DFigueroa%26aufirst%3DE.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPhillips%26aufirst%3DT.%2BR.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.-A.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520Applied%2520to%2520Hsp90.%2520Discovery%2520of%2520Two%2520Lead%2520Series%2520with%2520High%2520Ligand%2520Efficiency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5942%26epage%3D5955%26doi%3D10.1021%2Fjm100059d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.
G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drysdale, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansell, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matassova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roughley, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scriven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmonite, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surgenor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, P.</span></span> <span> </span><span class="NLM_article-title">Targeting Conserved Water Molecules: Design of 4-Aryl-5-cyanopyrrolo[2,3-D]pyrimidine Hsp90 Inhibitors Using Fragment-Based Screening and Structure-Based Optimization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">6770</span>– <span class="NLM_lpage">6789</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.08.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.bmc.2012.08.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23018093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVSltrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=6770-6789&author=N.%0AG.+M.+Daviesauthor=H.+Browneauthor=B.+Davisauthor=M.+J.+Drysdaleauthor=N.+Foloppeauthor=S.+Geoffreyauthor=B.+Gibbonsauthor=T.+Hartauthor=R.+Hubbardauthor=M.+R.+Jensenauthor=H.+Mansellauthor=A.+Masseyauthor=N.+Matassovaauthor=J.+D.+Mooreauthor=J.+Murrayauthor=R.+Prattauthor=S.+Rayauthor=A.+Robertsonauthor=S.+D.+Roughleyauthor=J.+Schoepferauthor=K.+Scrivenauthor=H.+Simmoniteauthor=S.+Stokesauthor=A.+Surgenorauthor=P.+Webbauthor=M.+Woodauthor=L.+Wrightauthor=P.+Brough&title=Targeting+Conserved+Water+Molecules%3A+Design+of+4-Aryl-5-cyanopyrrolo%5B2%2C3-D%5Dpyrimidine+Hsp90+Inhibitors+Using+Fragment-Based+Screening+and+Structure-Based+Optimization&doi=10.1016%2Fj.bmc.2012.08.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting conserved water molecules: Design of 4-aryl-5-cyanopyrrolo[2,3-d]pyrimidine Hsp90 inhibitors using fragment-based screening and structure-based optimization</span></div><div class="casAuthors">Davies, Nicholas G. M.; Browne, Helen; Davis, Ben; Drysdale, Martin J.; Foloppe, Nicolas; Geoffrey, Stephanie; Gibbons, Ben; Hart, Terance; Hubbard, Roderick; Jensen, Michael Rugaard; Mansell, Howard; Massey, Andrew; Matassova, Natalia; Moore, Jonathan D.; Murray, James; Pratt, Robert; Ray, Stuart; Robertson, Alan; Roughley, Stephen D.; Schoepfer, Joseph; Scriven, Kirsten; Simmonite, Heather; Stokes, Stephen; Surgenor, Allan; Webb, Paul; Wood, Mike; Wright, Lisa; Brough, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6770-6789</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibitors of the Hsp90 mol. chaperone are showing promise as anti-cancer agents.  Here the authors describe a series of 4-aryl-5-cyanopyrrolo[2,3-d]pyrimidine ATP competitive Hsp90 inhibitors that were identified following structure-driven optimization of purine hits revealed by NMR based screening of a proprietary fragment library.  Ligand-Hsp90 x-ray structures combined with mol. modeling led to the rational displacement of a conserved water mol. leading to enhanced affinity for Hsp90 as measured by fluorescence polarization, isothermal titrn. calorimetry and surface plasmon resonance assays.  This displacement was achieved with a nitrile group, presenting an example of efficient gain in binding affinity with minimal increase in mol. wt.  Some compds. in this chem. series inhibit the proliferation of human cancer cell lines in vitro and cause depletion of oncogenic Hsp90 client proteins and concomitant elevation of the co-chaperone Hsp70.  In addn., one compd. was demonstrated to be orally bioavailable in the mouse.  This work demonstrates the power of structure-based design for the rapid evolution of potent Hsp90 inhibitors and the importance of considering conserved water mols. in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMAwVhZUzKD7Vg90H21EOLACvtfcHk0lg4tCVpgeh3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVSltrvN&md5=69c85da95d07bd488f07eaf503c978f4</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.08.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.08.050%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DN.%2BG.%2BM.%26aulast%3DBrowne%26aufirst%3DH.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DGeoffrey%26aufirst%3DS.%26aulast%3DGibbons%26aufirst%3DB.%26aulast%3DHart%26aufirst%3DT.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DMansell%26aufirst%3DH.%26aulast%3DMassey%26aufirst%3DA.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DJ.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DPratt%26aufirst%3DR.%26aulast%3DRay%26aufirst%3DS.%26aulast%3DRobertson%26aufirst%3DA.%26aulast%3DRoughley%26aufirst%3DS.%2BD.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DScriven%26aufirst%3DK.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DSurgenor%26aufirst%3DA.%26aulast%3DWebb%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DL.%26aulast%3DBrough%26aufirst%3DP.%26atitle%3DTargeting%2520Conserved%2520Water%2520Molecules%253A%2520Design%2520of%25204-Aryl-5-cyanopyrrolo%255B2%252C3-D%255Dpyrimidine%2520Hsp90%2520Inhibitors%2520Using%2520Fragment-Based%2520Screening%2520and%2520Structure-Based%2520Optimization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D6770%26epage%3D6789%26doi%3D10.1016%2Fj.bmc.2012.08.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saubern, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span> <span> </span><span class="NLM_article-title">KNIME Workflow to Assess PAINS Filters in SMARTS Format. Comparison of RDKit and Indigo Cheminformatics Libraries</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">850</span>, <span class="refDoi"> DOI: 10.1002/minf.201100076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fminf.201100076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=27468104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht12itL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=847-850&author=S.+Saubernauthor=R.+Guhaauthor=J.+B.+Baell&title=KNIME+Workflow+to+Assess+PAINS+Filters+in+SMARTS+Format.+Comparison+of+RDKit+and+Indigo+Cheminformatics+Libraries&doi=10.1002%2Fminf.201100076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">KNIME Workflow to Assess PAINS Filters in SMARTS Format. Comparison of RDKit and Indigo Cheminformatics Libraries</span></div><div class="casAuthors">Saubern, Simon; Guha, Rajarshi; Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">847-850</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2_WZKnI-T-LVg90H21EOLACvtfcHk0lg4tCVpgeh3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht12itL%252FF&md5=314effbb1e4205e273858c6c70e7591e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201100076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201100076%26sid%3Dliteratum%253Aachs%26aulast%3DSaubern%26aufirst%3DS.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DKNIME%2520Workflow%2520to%2520Assess%2520PAINS%2520Filters%2520in%2520SMARTS%2520Format.%2520Comparison%2520of%2520RDKit%2520and%2520Indigo%2520Cheminformatics%2520Libraries%26jtitle%3DMol.%2520Inf.%26date%3D2011%26volume%3D30%26spage%3D847%26epage%3D850%26doi%3D10.1002%2Fminf.201100076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span> </span><span class="NLM_article-title">Enamine: Golden Fragment Library (2016)</span>. <a href="https://enamine.net/download/FL/Enamine_Golden_Fragment_Library.pdf" class="extLink">https://enamine.net/download/FL/Enamine_Golden_Fragment_Library.pdf</a> retrieved Dec 23, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Enamine%3A+Golden+Fragment+Library+%282016%29.+https%3A%2F%2Fenamine.net%2Fdownload%2FFL%2FEnamine_Golden_Fragment_Library.pdf+retrieved+Dec+23%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DEnamine%253A%2520Golden%2520Fragment%2520Library%2520%25282016%2529%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bemis, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcko, M. A.</span></span> <span> </span><span class="NLM_article-title">The Properties of Known Drugs. 1. Molecular Frameworks</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm9602928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+Properties+of+Known+Drugs.+1.+Molecular+Frameworks&doi=10.1021%2Fjm9602928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0lg4tCVpgeh3qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520Properties%2520of%2520Known%2520Drugs.%25201.%2520Molecular%2520Frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893%26doi%3D10.1021%2Fjm9602928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2138</span>– <span class="NLM_lpage">2153</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2138-2153&author=X.+Zhang&title=Intermolecular+Sulfur%C2%B7%C2%B7%C2%B7Oxygen+Interactions%3A+Theoretical+and+Statistical+Investigations&doi=10.1021%2Facs.jcim.5b00177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Intermolecular Sulfur···Oxygen Interactions: Theoretical and Statistical Investigations</span></div><div class="casAuthors">Zhang, Xuejin; Gong, Zhen; Li, Jian; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2138-2153</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intermol. S···O interactions are very common and are important in biol. systems, but until recently, the presence of these contacts in protein-ligand systems largely depended on serendipitous discovery instead of rational design.  Here we provide insight into the phenomenon of intermol. S···O contacts by focusing on three sulfur-contg. arom. rings.  Quantum mechanics is employed to characterize the strength and directionality of the S···O interactions and to det. their energy dependence on their geometric parameters.  Protein Data Bank mining is performed to systematically det. the occurrence and geometry of intermol. S···O interactions, and several representative examples are discussed.  Three typical cases are investigated using a combined quantum mechanics/mol. mechanics approach to demonstrate the potential of these interactions in improving binding affinities and physiochem. properties.  Overall, our work elucidates the structures and energy features of intermol. S···O interactions and addresses their use in mol. design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2J_2Nq9PDQLVg90H21EOLACvtfcHk0lg4ZnCk70Ug0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFart7rI&md5=7192e3b57e66fc6eb9dd53165427d449</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00177%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DIntermolecular%2520Sulfur%25C2%25B7%25C2%25B7%25C2%25B7Oxygen%2520Interactions%253A%2520Theoretical%2520and%2520Statistical%2520Investigations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2138%26epage%3D2153%26doi%3D10.1021%2Facs.jcim.5b00177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavallo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metrangolo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilati, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priimagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Resnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terraneo, G.</span></span> <span> </span><span class="NLM_article-title">The Halogen Bond</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">2478</span>– <span class="NLM_lpage">2601</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+Halogen+Bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0lg4ZnCk70Ug0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520Halogen%2520Bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">41576</span>– <span class="NLM_lpage">41582</span>, <span class="refDoi"> DOI: 10.1074/jbc.R112.418574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1074%2Fjbc.R112.418574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=23048026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=41576-41582&author=S.+Horowitzauthor=R.+C.+Trievel&title=Carbon-Oxygen+Hydrogen+Bonding+in+Biological+Structure+and+Function&doi=10.1074%2Fjbc.R112.418574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span></div><div class="casAuthors">Horowitz, Scott; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41576-41582</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Carbon-oxygen (CH···O) hydrogen bonding represents an unusual category of mol. interactions first documented in biol. structures over 4 decades ago.  Although CH···O hydrogen bonding has remained generally underappreciated in the biochem. literature, studies over the last 15 years have begun to yield direct evidence of these interactions in biol. systems.  In this minireview, we provide a historical context of biol. CH···O hydrogen bonding and summarize some major advancements from exptl. studies over the past several years that have elucidated the importance, prevalence, and functions of these interactions.  In particular, we examine the impact of CH···O bonds on protein and nucleic acid structure, mol. recognition, and enzyme catalysis and conclude by exploring overarching themes and unresolved questions regarding unconventional interactions in biomol. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHj0osWPCzV7Vg90H21EOLACvtfcHk0lg4ZnCk70Ug0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP&md5=e96e7e30d5f5e03b0733fea323ecd6a2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.418574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.418574%26sid%3Dliteratum%253Aachs%26aulast%3DHorowitz%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCarbon-Oxygen%2520Hydrogen%2520Bonding%2520in%2520Biological%2520Structure%2520and%2520Function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D41576%26epage%3D41582%26doi%3D10.1074%2Fjbc.R112.418574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saini, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tickle, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Detection of Secondary Binding Sites in Proteins Using Fragment Screening</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">15910</span>– <span class="NLM_lpage">15915</span>, <span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+Secondary+Binding+Sites+in+Proteins+Using+Fragment+Screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lgPlXr_wXr40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520Secondary%2520Binding%2520Sites%2520in%2520Proteins%2520Using%2520Fragment%2520Screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Lessons from Hot Spot Analysis for Fragment-Based Drug Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.tips.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=26538314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=724-736&author=D.+R.+Hallauthor=D.+Kozakovauthor=A.+Whittyauthor=S.+Vajda&title=Lessons+from+Hot+Spot+Analysis+for+Fragment-Based+Drug+Discovery&doi=10.1016%2Fj.tips.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from hot spot analysis for fragment-based drug discovery</span></div><div class="casAuthors">Hall, David R.; Kozakov, Dima; Whitty, Adrian; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">724-736</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Anal. of binding energy hot spots at protein surfaces can provide crucial insights into the prospects for successful application of fragment-based drug discovery (FBDD), and whether a fragment hit can be advanced into a high-affinity, drug-like ligand.  The key factor is the strength of the top ranking hot spot, and how well a given fragment complements it.  We show that published data are sufficient to provide a sophisticated and quant. understanding of how hot spots derive from a protein 3D structure, and how their strength, no., and spatial arrangement govern the potential for a surface site to bind to fragment-sized and larger ligands.  This improved understanding provides important guidance for the effective application of FBDD in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdQmPulmhPx7Vg90H21EOLACvtfcHk0lgPlXr_wXr40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGmtb%252FF&md5=3e682f28482d162590826f8f2e321b96</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLessons%2520from%2520Hot%2520Spot%2520Analysis%2520for%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D724%26epage%3D736%26doi%3D10.1016%2Fj.tips.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Identifying Interactions That Determine Fragment Binding at Protein Hotspots</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4314</span>– <span class="NLM_lpage">4325</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01980</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01980" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4314-4325&author=C.+J.+Radouxauthor=T.+S.+G.+Olssonauthor=W.+R.+Pittauthor=C.+R.+Groomauthor=T.+L.+Blundell&title=Identifying+Interactions+That+Determine+Fragment+Binding+at+Protein+Hotspots&doi=10.1021%2Facs.jmedchem.5b01980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Interactions that Determine Fragment Binding at Protein Hotspots</span></div><div class="casAuthors">Radoux, Chris J.; Olsson, Tjelvar S. G.; Pitt, Will R.; Groom, Colin R.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4314-4325</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Locating a ligand-binding site is an important first step in structure-guided drug discovery, but current methods do little to suggest which interactions within a pocket are the most important for binding.  Here the authors illustrate a method that samples at. hotspots with simple mol. probes to produce fragment hotspot maps.  These maps specifically highlight fragment-binding sites and their corresponding pharmacophores.  For ligand-bound structures, they provide an intuitive visual guide within the binding site, directing medicinal chemists where to grow the mol. and alerting them to suboptimal interactions within the original hit.  The fragment hotspot map calcn. is validated using exptl. binding positions of 21 fragments and subsequent lead mols.  The ligands are found in high scoring areas of the fragment hotspot maps, with fragment atoms having a median percentage rank of 97%.  Protein kinase B and pantothenate synthetase are examd. in detail.  In each case, the fragment hotspot maps are able to rationalize a Free-Wilson anal. of SAR data from a fragment-based drug design project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8BzZp1bkHarVg90H21EOLACvtfcHk0lgPlXr_wXr40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOms7Y%253D&md5=fa25eabfe291f9519f5701bcbc6c52c3</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01980%26sid%3Dliteratum%253Aachs%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DIdentifying%2520Interactions%2520That%2520Determine%2520Fragment%2520Binding%2520at%2520Protein%2520Hotspots%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4314%26epage%3D4325%26doi%3D10.1021%2Facs.jmedchem.5b01980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devine, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulcair, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debono, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, E. W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissink, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrashekaran, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohanty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">Promiscuous 2-Aminothiazoles (PrATs): A Frequent Hitting Scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1205</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1021/jm501402x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501402x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1205-1214&author=S.+M.+Devineauthor=M.+D.+Mulcairauthor=C.+O.+Debonoauthor=E.+W.+W.+Leungauthor=J.+W.+M.+Nissinkauthor=S.+S.+Limauthor=I.+R.+Chandrashekaranauthor=M.+Vaziraniauthor=B.+Mohantyauthor=J.+S.+Simpsonauthor=J.+B.+Baellauthor=P.+J.+Scammellsauthor=R.+S.+Nortonauthor=M.+J.+Scanlon&title=Promiscuous+2-Aminothiazoles+%28PrATs%29%3A+A+Frequent+Hitting+Scaffold&doi=10.1021%2Fjm501402x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous 2-Aminothiazoles (PrATs): A Frequent Hitting Scaffold</span></div><div class="casAuthors">Devine, Shane M.; Mulcair, Mark D.; Debono, Cael O.; Leung, Eleanor W. W.; Nissink, J. Willem M.; Lim, San Sui; Chandrashekaran, Indu R.; Vazirani, Mansha; Mohanty, Biswaranjan; Simpson, Jamie S.; Baell, Jonathan B.; Scammells, Peter J.; Norton, Raymond S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1205-1214</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have identified a class of mols., known as 2-aminothiazoles (2-ATs), as frequent-hitting fragments in biophys. binding assays.  This was exemplified by 4-phenylthiazol-2-amine being identified as a hit in 14/14 screens against a diverse range of protein targets, suggesting that this scaffold is a poor starting point for fragment-based drug discovery.  This prompted us to analyze this scaffold in the context of an academic fragment library used for fragment-based drug discovery (FBDD) and two larger compd. libraries used for high-throughput screening (HTS).  This anal. revealed that such "promiscuous 2-aminothiazoles" (PrATs) behaved as frequent hitters under both FBDD and HTS settings, although the problem was more pronounced in the fragment-based studies.  As 2-ATs are present in known drugs, they cannot necessarily be deemed undesirable, but the combination of their promiscuity and difficulties assocd. with optimizing them into a lead compd. makes them, in our opinion, poor scaffolds for fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryZfJtzgcrZ7Vg90H21EOLACvtfcHk0lgPlXr_wXr40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsw%253D%253D&md5=8d989c9ee48293efe32a331049b59f42</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm501402x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501402x%26sid%3Dliteratum%253Aachs%26aulast%3DDevine%26aufirst%3DS.%2BM.%26aulast%3DMulcair%26aufirst%3DM.%2BD.%26aulast%3DDebono%26aufirst%3DC.%2BO.%26aulast%3DLeung%26aufirst%3DE.%2BW.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DLim%26aufirst%3DS.%2BS.%26aulast%3DChandrashekaran%26aufirst%3DI.%2BR.%26aulast%3DVazirani%26aufirst%3DM.%26aulast%3DMohanty%26aufirst%3DB.%26aulast%3DSimpson%26aufirst%3DJ.%2BS.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DPromiscuous%25202-Aminothiazoles%2520%2528PrATs%2529%253A%2520A%2520Frequent%2520Hitting%2520Scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1205%26epage%3D1214%26doi%3D10.1021%2Fjm501402x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1002/anie.201506783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1002%2Fanie.201506783" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=488-492&author=C.+W.+Murrayauthor=D.+C.+Rees&title=Opportunity+Knocks%3A+Organic+Chemistry+for+Fragment-Based+Drug+Discovery+%28FBDD%29&doi=10.1002%2Fanie.201506783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunity Knocks: Organic Chemistry for Fragment-Based Drug Discovery (FBDD)</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">488-492</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj8GEY4dDXxrVg90H21EOLACvtfcHk0ljCgeozGQLGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt77O&md5=97a92911033a2304da101888da0c3642</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506783%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DOpportunity%2520Knocks%253A%2520Organic%2520Chemistry%2520for%2520Fragment-Based%2520Drug%2520Discovery%2520%2528FBDD%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D488%26epage%3D492%26doi%3D10.1002%2Fanie.201506783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuter, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Conformational Preferences Derived from Crystal Structure Data. A Medicinal Chemistry Focused Analysis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/ci7002494</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci7002494" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1-24&author=K.+A.+Brameldauthor=B.+Kuhnauthor=D.+C.+Reuterauthor=M.+Stahl&title=Small+Molecule+Conformational+Preferences+Derived+from+Crystal+Structure+Data.+A+Medicinal+Chemistry+Focused+Analysis&doi=10.1021%2Fci7002494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule conformational preferences derived from crystal structure data. a medicinal chemistry focused analysis</span></div><div class="casAuthors">Brameld, Ken A.; Kuhn, Bernd; Reuter, Deborah C.; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Based on torsion angle distributions of frequently occurring substructures, conformation preferences of druglike mols. are presented, accompanied by a review of the relevant literature.  First, the relevance of the Cambridge Structural Database (CSD) for drug design is demonstrated by comparing substructures present in compds. entering clin. trials with those found in the CSD and protein-bound ligands in the Protein Data Bank (PDB).  Next, we briefly highlight preferred conformations of elementary acyclic systems, followed by a discussion of sulfonamide conformations.  Due to their central role in medicinal chem., we discuss properties of aryl ring substituents in depth, including biaryl systems and systems of two aryl rings connected by two acyclic bonds.  For a subset of torsion motifs, we also compare torsion angle histograms derived from CSD structures with those derived from ligands in the PDB.  Furthermore, selected properties of some six- and seven-membered ring systems are discussed.  The article closes with a section on attractive sulfur-oxygen contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBrdvMrRvKirVg90H21EOLACvtfcHk0ljCgeozGQLGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXksV2isA%253D%253D&md5=289969338dcb94a145ce736d19d05cf4</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci7002494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci7002494%26sid%3Dliteratum%253Aachs%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DReuter%26aufirst%3DD.%2BC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DSmall%2520Molecule%2520Conformational%2520Preferences%2520Derived%2520from%2520Crystal%2520Structure%2520Data.%2520A%2520Medicinal%2520Chemistry%2520Focused%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D1%26epage%3D24%26doi%3D10.1021%2Fci7002494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Thermodynamics of Fragment Binding</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1021/ci200608b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200608b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC38XkslWjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1039-1045&author=G.+G.+Ferenczyauthor=G.+M.+Keser%C5%B1&title=Thermodynamics+of+Fragment+Binding&doi=10.1021%2Fci200608b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamics of Fragment Binding</span></div><div class="casAuthors">Ferenczy, Gyorgy G.; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ligand binding pockets of proteins have preponderance of hydrophobic amino acids and are typically within the apolar interior of the protein; nevertheless, they are able to bind low complexity, polar, water-sol. fragments.  In order to understand this phenomenon, we analyzed high resoln. X-ray data of protein-ligand complexes from the Protein Data Bank and found that fragments bind to proteins with two near optimal geometry H-bonds on av.  The linear extent of the fragment binding site was found not to be larger than 10 Å, and the H-bonding region was found to be restricted to about 5 Å on av.  The no. of conserved H-bonds in proteins cocrystd. with multiple different fragments is also near to 2.  These fragment binding sites that are able to form limited no. of strong H-bonds in a hydrophobic environment are identified as hot spots.  An est. of the free energy gain of H-bond formation vs. apolar desolvation supports the idea that fragment-sized compds. need H-bonds to achieve detectable binding.  This suggests that fragment binding is mostly enthalpic, in line with their obsd. binding thermodn. documented in isothermal titrn. calorimetry (ITC) data sets and gives a thermodn. rationale for fragment-based approaches.  The binding of larger compds. tends to more rely on apolar desolvation with a corresponding increase of the entropy content of their binding free energy.  These findings explain the reported size-dependence of maximal available affinity and ligand efficiency both behaving differently in the small mol. region featured by strong H-bond formation and in the larger mol. region featured by apolar desolvation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvPwUrHRwL1LVg90H21EOLACvtfcHk0ljCgeozGQLGiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkslWjurw%253D&md5=d1dde67220cd13f4f222ae68a8870892</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci200608b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200608b%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DThermodynamics%2520of%2520Fragment%2520Binding%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1039%26epage%3D1045%26doi%3D10.1021%2Fci200608b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferenczy, G.
G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">On the Enthalpic Preference of Fragment Binding</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1039/C5MD00542F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1039%2FC5MD00542F" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=332-337&author=G.%0AG.+Ferenczyauthor=G.+M.+Keser%C5%B1&title=On+the+Enthalpic+Preference+of+Fragment+Binding&doi=10.1039%2FC5MD00542F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FC5MD00542F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5MD00542F%26sid%3Dliteratum%253Aachs%26aulast%3DFerenczy%26aufirst%3DG.%2BG.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DOn%2520the%2520Enthalpic%2520Preference%2520of%2520Fragment%2520Binding%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D332%26epage%3D337%26doi%3D10.1039%2FC5MD00542F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rühmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Fragment Binding Can Be Either More Enthalpy-Driven or Entropy-Driven: Crystal Structures and Residual Hydration Patterns Suggest Why</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6960</span>– <span class="NLM_lpage">6971</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00812</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00812" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlehu7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6960-6971&author=E.+R%C3%BChmannauthor=M.+Betzauthor=A.+Heineauthor=G.+Klebe&title=Fragment+Binding+Can+Be+Either+More+Enthalpy-Driven+or+Entropy-Driven%3A+Crystal+Structures+and+Residual+Hydration+Patterns+Suggest+Why&doi=10.1021%2Facs.jmedchem.5b00812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment Binding Can Be Either More Enthalpy-Driven or Entropy-Driven: Crystal Structures and Residual Hydration Patterns Suggest Why</span></div><div class="casAuthors">Ruehmann, Eggert; Betz, Michael; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6960-6971</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In lead optimization, small, enthalpically advantaged fragments have been suggested to be superior, as an entropic component will be added inevitably during late-stage optimization.  Detn. of thermodn. signatures of weak-binding fragments is essential to support the decision-making process, to decide which fragment to take to further optimization.  High-resoln. crystal structures of six fragments binding to the S1 pocket of thrombin were detd. and analyzed with respect to their thermodn. profile.  The two most potent fragments exhibiting an amidine-type scaffold are not the most enthalpic binders; instead a chloro-thiophene fragment binds more enthalpically.  Two chem. very similar chloro-arom. fragments differ strongly in their potency (430 μM vs 10 mM); their binding modes are related, but the surrounding residual water network differs.  The more potent one recruits a water mol. and involves Glu192 in binding, thus succeeding in firmly capping the S1 pocket.  Fragments exhibiting a rather perfect solvation pattern in their binding mode also experience the highest potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjIOp5Fbyf3bVg90H21EOLACvtfcHk0lg-b2qkyCLB4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlehu7%252FF&md5=97c80bfedfb189af3be987f0bcc75e24</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00812%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25BChmann%26aufirst%3DE.%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFragment%2520Binding%2520Can%2520Be%2520Either%2520More%2520Enthalpy-Driven%2520or%2520Entropy-Driven%253A%2520Crystal%2520Structures%2520and%2520Residual%2520Hydration%2520Patterns%2520Suggest%2520Why%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6960%26epage%3D6971%26doi%3D10.1021%2Facs.jmedchem.5b00812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">The Thermodynamics of Protein–Ligand Interaction and Solvation: Insights for Ligand Design</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">1002</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1016%2Fj.jmb.2008.09.073" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2008&pages=1002-1017&author=T.+S.+G.+Olssonauthor=M.+A.+Williamsauthor=W.+R.+Pittauthor=J.+E.+Ladbury&title=The+Thermodynamics+of+Protein%E2%80%93Ligand+Interaction+and+Solvation%3A+Insights+for+Ligand+Design&doi=10.1016%2Fj.jmb.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DWilliams%26aufirst%3DM.%2BA.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DThe%2520Thermodynamics%2520of%2520Protein%25E2%2580%2593Ligand%2520Interaction%2520and%2520Solvation%253A%2520Insights%2520for%2520Ligand%2520Design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D384%26spage%3D1002%26epage%3D1017%26doi%3D10.1016%2Fj.jmb.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sándor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Virtual Fragment Docking by Glide: A Validation Study on 190 Protein–Fragment Complexes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1165</span>– <span class="NLM_lpage">1172</span>, <span class="refDoi"> DOI: 10.1021/ci1000407</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1000407" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1165-1172&author=M.+S%C3%A1ndorauthor=R.+Kissauthor=G.+M.+Keser%C5%B1&title=Virtual+Fragment+Docking+by+Glide%3A+A+Validation+Study+on+190+Protein%E2%80%93Fragment+Complexes&doi=10.1021%2Fci1000407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Fragment Docking by Glide: a Validation Study on 190 Protein-Fragment Complexes</span></div><div class="casAuthors">Sandor, Mark; Kiss, Robert; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1165-1172</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The docking accuracy of Glide was evaluated using 16 different docking protocols on 190 protein-fragment complexes representing 78 targets.  Std. precision docking (Glide SP) based protocols showed the best performance.  The av. root-mean-square deviation (RMSD) between the docked and cocrystd. poses achieved by Glide SP with pre- and postprocessing was 1.17 Å, and an acceptable binding mode with RMSD. < 2 Å could be found in 80% of the cases.  Comparison of the docking results produced by different protocols suggests that the sampling efficacy of Glide is adequate for fragment docking.  The docking accuracy seems to be limited by the performance of scoring schemes, which is supported by the weak correlation between exptl. binding affinities and GlideScores.  Cross-docking expts. performed on 8 targets represented by 63 complexes revealed that Glide SP gave similar results to that of the computationally more intensive Glide XP.  The av. RMSD achieved by Glide SP with pre- and postprocessing was 2.06 Å, and an acceptable binding mode with RMSD < 2 Å could be found in 63% of the cases.  These cross-docking results were improved significantly selecting the optimal X-ray structure for each target (av. RMSD = 1.3 Å, success rate = 77%), indicating the importance of enrichment studies and the use of multiple X-ray structures in virtual fragment screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodKfjH9BbnrLVg90H21EOLACvtfcHk0lg-b2qkyCLB4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvVaksLY%253D&md5=35d7acb7374360828beb668e2f207bab</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci1000407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1000407%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1ndor%26aufirst%3DM.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DVirtual%2520Fragment%2520Docking%2520by%2520Glide%253A%2520A%2520Validation%2520Study%2520on%2520190%2520Protein%25E2%2580%2593Fragment%2520Complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1165%26epage%3D1172%26doi%3D10.1021%2Fci1000407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giangreco, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korb, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span> <span> </span><span class="NLM_article-title">Docking Performance of Fragments and Druglike Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">5422</span>– <span class="NLM_lpage">5431</span>, <span class="refDoi"> DOI: 10.1021/jm200558u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200558u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5422-5431&author=M.%0AL.+Verdonkauthor=I.+Giangrecoauthor=R.+J.+Hallauthor=O.+Korbauthor=P.+N.+Mortensonauthor=C.+W.+Murray&title=Docking+Performance+of+Fragments+and+Druglike+Compounds&doi=10.1021%2Fjm200558u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Performance of Fragments and Druglike Compounds</span></div><div class="casAuthors">Verdonk, Marcel L.; Giangreco, Ilenia; Hall, Richard J.; Korb, Oliver; Mortenson, Paul N.; Murray, Christopher W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5431</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper addresses two questions of key interest to researchers working with protein-ligand docking methods: (i) Why is there such a large variation in docking performance between different test sets reported in the literature (ii) Are fragments more difficult to dock than druglike compds..  To answer these, the authors construct a test set of inhouse x-ray structures of protein-ligand complexes from drug discovery projects, half of which contain fragment ligands, the other half drug-like ligands.  The authors find that a key factor affecting docking performance is ligand efficiency (LE).  High LE compds. are significantly easier to dock than low LE compds., which the authors believe could explain the differences obsd. between test sets reported in the literature.  There is no significant difference in docking performance between fragments and drug-like compds., but the reasons why dockings fail appear to be different.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVFTOFvmiFxrVg90H21EOLACvtfcHk0lgiK4Ktcdv_LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFCgsLo%253D&md5=8d7a5519d6e7fa2ab403ea76b8b9a586</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm200558u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200558u%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DGiangreco%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26atitle%3DDocking%2520Performance%2520of%2520Fragments%2520and%2520Druglike%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5422%26epage%3D5431%26doi%3D10.1021%2Fjm200558u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Applying Thermodynamic Profiling in Lead Finding and Optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1038/nrd4486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1038%2Fnrd4486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25614222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=95-110&author=G.+Klebe&title=Applying+Thermodynamic+Profiling+in+Lead+Finding+and+Optimization&doi=10.1038%2Fnrd4486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Applying thermodynamic profiling in lead finding and optimization</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">95-110</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery involves the optimization of various physicochem. properties of a ligand, particularly its binding affinity for its target receptor (or receptors).  In recent years, there has been growing interest in using thermodn. profiling of ligand-receptor interactions in order to select and optimize those ligands that might be most likely to become drug candidates with desirable physicochem. properties.  The thermodn. of binding is influenced by multiple factors, including hydrogen bonding and hydrophobic interactions, desolvation, residual mobility, dynamics and the local water structure.  This article discusses key issues in understanding the effects of these factors and applying this knowledge in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtHj7SsL6rrVg90H21EOLACvtfcHk0lgiK4Ktcdv_LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVemtr4%253D&md5=c2e12760623de81335ce252cb804c602</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnrd4486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4486%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApplying%2520Thermodynamic%2520Profiling%2520in%2520Lead%2520Finding%2520and%2520Optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D95%26epage%3D110%26doi%3D10.1038%2Fnrd4486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishio, M.</span></span> <span> </span><span class="NLM_article-title">The CH/π hydrogen bond in chemistry. Conformation, supramolecules, optical resolution and interactions involving carbohydrates</span>. <i>Phys. Chem. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">13873</span>– <span class="NLM_lpage">13900</span>, <span class="refDoi"> DOI: 10.1039/c1cp20404a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1039%2Fc1cp20404a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=21611676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlCgsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=13873-13900&author=M.+Nishio&title=The+CH%2F%CF%80+hydrogen+bond+in+chemistry.+Conformation%2C+supramolecules%2C+optical+resolution+and+interactions+involving+carbohydrates&doi=10.1039%2Fc1cp20404a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">The CH/π hydrogen bond in chemistry. Conformation, supramolecules, optical resolution and interactions involving carbohydrates</span></div><div class="casAuthors">Nishio, Motohiro</div><div class="citationInfo"><span class="NLM_cas:title">Physical Chemistry Chemical Physics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">13873-13900</span>CODEN:
                <span class="NLM_cas:coden">PPCPFQ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9076</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The CH/π hydrogen bond is an attractive mol. force occurring between a soft acid and a soft base.  Contribution from the dispersion energy is important in typical cases where aliph. or arom. CH groups are involved.  Coulombic energy is of minor importance as compared to the other weak hydrogen bonds.  The hydrogen bond nature of this force, however, has been confirmed by AIM analyses.  The dual characteristic of the CH/π hydrogen bond is the basis for ubiquitous existence of this force in various fields of chem.  A salient feature is that the CH/π hydrogen bond works cooperatively.  Another significant point is that it works in nonpolar as well as polar, protic solvents such as water.  The interaction energy depends on the nature of the mol. fragments, CH as well as π-groups: the stronger the proton donating ability of the CH group, the larger the stabilizing effect.  This Perspective focuses on the consequence of this mol. force in the conformation of org. compds. and supramol. chem.  Implication of the CH/π hydrogen bond extends to the specificity of mol. recognition or selectivity in org. reactions, polymer science, surface phenomena and interactions involving proteins.  Many problems, unsettled to date, will become clearer in the light of the CH/π paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoQqvxrPj-PbVg90H21EOLACvtfcHk0lgiK4Ktcdv_LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlCgsbs%253D&md5=18eff250217a75244a10c8d2ef69710e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1039%2Fc1cp20404a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cp20404a%26sid%3Dliteratum%253Aachs%26aulast%3DNishio%26aufirst%3DM.%26atitle%3DThe%2520CH%252F%25CF%2580%2520hydrogen%2520bond%2520in%2520chemistry.%2520Conformation%252C%2520supramolecules%252C%2520optical%2520resolution%2520and%2520interactions%2520involving%2520carbohydrates%26jtitle%3DPhys.%2520Chem.%2520Chem.%2520Phys.%26date%3D2011%26volume%3D13%26spage%3D13873%26epage%3D13900%26doi%3D10.1039%2Fc1cp20404a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Interactive and Versatile Navigation of Structural Databases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4257</span>– <span class="NLM_lpage">4266</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01756</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01756" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4257-4266&author=O.+Korbauthor=B.+Kuhnauthor=J.+Hertauthor=N.+Taylorauthor=J.+Coleauthor=C.+Groomauthor=M.+Stahl&title=Interactive+and+Versatile+Navigation+of+Structural+Databases&doi=10.1021%2Facs.jmedchem.5b01756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Interactive and Versatile Navigation of Structural Databases</span></div><div class="casAuthors">Korb, Oliver; Kuhn, Bernd; Hert, Jerome; Taylor, Neil; Cole, Jason; Groom, Colin; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4257-4266</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present CSD-CrossMiner, a novel tool for pharmacophore-based searches in crystal structure databases.  Intuitive pharmacophore queries describing, among others, protein-ligand interaction patterns, ligand scaffolds, or protein environments can be built and modified interactively.  Matching crystal structures are overlaid onto the query and visualized as soon as they are available, enabling the researcher to quickly modify a hypothesis on the fly.  We exemplify the utility of the approach by showing applications relevant to real-world drug discovery projects, including the identification of novel fragments for a specific protein environment or scaffold hopping.  The ability to concurrently search protein-ligand binding sites extd. from the Protein Data Bank (PDB) and small org. mols. from the Cambridge Structural Database (CSD) using the same pharmacophore query further emphasizes the flexibility of CSD-CrossMiner.  We believe that CSD-CrossMiner closes an important gap in mining structural data and will allow users to ext. more value from the growing no. of available crystal structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVHoEyOPb96bVg90H21EOLACvtfcHk0lgiK4Ktcdv_LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGms70%253D&md5=1b872969057778cbcc0be80db9bb2bd3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01756%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DJ.%26aulast%3DGroom%26aufirst%3DC.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DInteractive%2520and%2520Versatile%2520Navigation%2520of%2520Structural%2520Databases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4257%26epage%3D4266%26doi%3D10.1021%2Facs.jmedchem.5b01756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">UniProt:
A Hub for Protein Information</span>. <i>Nucleic Acids
Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">D204</span>– <span class="NLM_lpage">D212</span>, <span class="refDoi"> DOI: 10.1093/nar/gku989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=10.1093%2Fnar%2Fgku989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;key=25348405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D204-D212&author=The%0AUniProt+Consortium&title=UniProt%3A%0AA+Hub+for+Protein+Information&doi=10.1093%2Fnar%2Fgku989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku989%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AA%2520Hub%2520for%2520Protein%2520Information%26jtitle%3DNucleic%2520Acids%250ARes.%26date%3D2015%26volume%3D43%26spage%3DD204%26epage%3DD212%26doi%3D10.1093%2Fnar%2Fgku989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4J','PDB','4Y4J'); return false;">PDB: 4Y4J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R54" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R54','PDB','3R54'); return false;">PDB: 3R54</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IH5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IH5','PDB','4IH5'); return false;">PDB: 4IH5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7I','PDB','4K7I'); return false;">PDB: 4K7I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7N','PDB','4K7N'); return false;">PDB: 4K7N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K7O','PDB','4K7O'); return false;">PDB: 4K7O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9K','PDB','5E9K'); return false;">PDB: 5E9K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WKD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3WKD','PDB','3WKD'); return false;">PDB: 3WKD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AME" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AME','PDB','5AME'); return false;">PDB: 5AME</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N1V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N1V','PDB','3N1V'); return false;">PDB: 3N1V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAN','PDB','5JAN'); return false;">PDB: 5JAN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J4H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J4H','PDB','5J4H'); return false;">PDB: 5J4H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OMQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OMQ','PDB','3OMQ'); return false;">PDB: 3OMQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAH','PDB','5JAH'); return false;">PDB: 5JAH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD','PDB','3FGD'); return false;">PDB: 3FGD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EGS','PDB','5EGS'); return false;">PDB: 5EGS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FPO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FPO','PDB','5FPO'); return false;">PDB: 5FPO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CLP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5CLP','PDB','5CLP'); return false;">PDB: 5CLP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACW','PDB','5ACW'); return false;">PDB: 5ACW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MOH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MOH','PDB','5MOH'); return false;">PDB: 5MOH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4G','PDB','4Y4G'); return false;">PDB: 4Y4G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G2Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G2Y','PDB','3G2Y'); return false;">PDB: 3G2Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G32" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3G32','PDB','3G32'); return false;">PDB: 3G32</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y4T','PDB','4Y4T'); return false;">PDB: 4Y4T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FYU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FYU','PDB','5FYU'); return false;">PDB: 5FYU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GVB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GVB','PDB','3GVB'); return false;">PDB: 3GVB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUH','PDB','3FUH'); return false;">PDB: 3FUH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B3C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4B3C','PDB','4B3C'); return false;">PDB: 4B3C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYX','PDB','5DYX'); return false;">PDB: 5DYX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YED" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YED','PDB','2YED'); return false;">PDB: 2YED</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MSA','PDB','4MSA'); return false;">PDB: 4MSA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N9C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4N9C','PDB','4N9C'); return false;">PDB: 4N9C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5JAO','PDB','5JAO'); return false;">PDB: 5JAO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YE7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YE7','PDB','2YE7'); return false;">PDB: 2YE7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ESS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ESS','PDB','3ESS'); return false;">PDB: 3ESS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CUR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CUR','PDB','4CUR'); return false;">PDB: 4CUR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LR6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LR6','PDB','4LR6'); return false;">PDB: 4LR6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4TZ8','PDB','4TZ8'); return false;">PDB: 4TZ8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YK0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YK0','PDB','4YK0'); return false;">PDB: 4YK0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DYU','PDB','5DYU'); return false;">PDB: 5DYU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E3G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E3G','PDB','5E3G'); return false;">PDB: 5E3G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E9Y','PDB','5E9Y'); return false;">PDB: 5E9Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NOW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NOW','PDB','5NOW'); return false;">PDB: 5NOW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y3P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y3P','PDB','4Y3P'); return false;">PDB: 4Y3P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K2Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K2Y','PDB','4K2Y'); return false;">PDB: 4K2Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y37" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Y37','PDB','4Y37'); return false;">PDB: 4Y37</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I5W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5I5W','PDB','5I5W'); return false;">PDB: 5I5W</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i18"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01855">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28655"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01855">10.1021/acs.jmedchem.8b01855</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figure S1: frequency distribution of interactions per fragment pocket; Figure S2: frequency distribution of total, polar, and apolar buried surface area (SA) of the fragments; Figure S3: characteristics of fragment hit protein targets; Figure S4: comparison between distributions of fragment hit and ligand features; Figure S5: comparison between surface characteristics of the fragment– and ligand–pocket complexes; Figure S6: comparison between interactions of the fragment– and ligand–pocket complexes; Table S1: occurrence of multiple fragment hit binding sites across various protein targets; Table S2: occurrence of Bemis–Murcko frameworks across 462 unique fragment hits; Table S3: occurrence of explicit individual ring assemblies across 462 unique fragment hits; Table S4: descriptive statistics of the fragment buried SA upon binding; Table S5: fragment atoms involved in coordination bonds to structural metal ions in protein binding pockets; Table S6: fragment atoms involved in H-bonds with structural water molecules in protein binding pockets with crystallographic resolution ≤1.5 Å; Table S7: fragment atoms involved in sulfur contacts, halogen bonds, and carbon hydrogen bonds with the protein; Table S8: PDB codes for the 445 protein–ligand complexes; Table S9: PDB codes, SMILES strings, and citation information for the 489 pocket–fragment complexes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01855/suppl_file/jm8b01855_si_001.pdf">jm8b01855_si_001.pdf (1.26 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01855&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01855%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-7%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01855" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-b425991822a79ada59b3.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-fb88834b824b41b135ad.js"></script>

<script type="text/javascript" src="/wro/ke1q~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6679804fbfb823b0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
